{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "name": "COVAQA.ipynb",
      "provenance": [],
      "collapsed_sections": [],
      "include_colab_link": true
    },
    "kernelspec": {
      "name": "python3",
      "display_name": "Python 3"
    },
    "language_info": {
      "name": "python"
    }
  },
  "cells": [
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "view-in-github",
        "colab_type": "text"
      },
      "source": [
        "<a href=\"https://colab.research.google.com/github/MBux/COVAQA/blob/CAiRE-COVID/NLP/COVAQA.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "ZOuehPFukM8W",
        "outputId": "d2b275a6-5d57-4b74-cb08-6247109c44d1"
      },
      "source": [
        "!pip install caireCovid"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Collecting caireCovid\n",
            "\u001b[?25l  Downloading https://files.pythonhosted.org/packages/fb/77/b003b34210945368aa04890b47b183e6b92b07c2dade3fc9190c11c6f07a/caireCovid-0.1.8-py3-none-any.whl (40kB)\n",
            "\u001b[K     |████████████████████████████████| 40kB 2.4MB/s \n",
            "\u001b[?25hRequirement already satisfied: numpy in /usr/local/lib/python3.7/dist-packages (from caireCovid) (1.19.5)\n",
            "Requirement already satisfied: six in /usr/local/lib/python3.7/dist-packages (from caireCovid) (1.15.0)\n",
            "Collecting sentencepiece\n",
            "\u001b[?25l  Downloading https://files.pythonhosted.org/packages/f5/99/e0808cb947ba10f575839c43e8fafc9cc44e4a7a2c8f79c60db48220a577/sentencepiece-0.1.95-cp37-cp37m-manylinux2014_x86_64.whl (1.2MB)\n",
            "\u001b[K     |████████████████████████████████| 1.2MB 3.7MB/s \n",
            "\u001b[?25hRequirement already satisfied: requests in /usr/local/lib/python3.7/dist-packages (from caireCovid) (2.23.0)\n",
            "Requirement already satisfied: tqdm in /usr/local/lib/python3.7/dist-packages (from caireCovid) (4.41.1)\n",
            "Collecting unidecode\n",
            "\u001b[?25l  Downloading https://files.pythonhosted.org/packages/9e/25/723487ca2a52ebcee88a34d7d1f5a4b80b793f179ee0f62d5371938dfa01/Unidecode-1.2.0-py2.py3-none-any.whl (241kB)\n",
            "\u001b[K     |████████████████████████████████| 245kB 16.9MB/s \n",
            "\u001b[?25hRequirement already satisfied: nltk in /usr/local/lib/python3.7/dist-packages (from caireCovid) (3.2.5)\n",
            "Collecting jsonlines\n",
            "  Downloading https://files.pythonhosted.org/packages/d4/58/06f430ff7607a2929f80f07bfd820acbc508a4e977542fefcc522cde9dff/jsonlines-2.0.0-py3-none-any.whl\n",
            "Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.7/dist-packages (from requests->caireCovid) (2020.12.5)\n",
            "Requirement already satisfied: chardet<4,>=3.0.2 in /usr/local/lib/python3.7/dist-packages (from requests->caireCovid) (3.0.4)\n",
            "Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /usr/local/lib/python3.7/dist-packages (from requests->caireCovid) (1.24.3)\n",
            "Requirement already satisfied: idna<3,>=2.5 in /usr/local/lib/python3.7/dist-packages (from requests->caireCovid) (2.10)\n",
            "Installing collected packages: sentencepiece, unidecode, jsonlines, caireCovid\n",
            "Successfully installed caireCovid-0.1.8 jsonlines-2.0.0 sentencepiece-0.1.95 unidecode-1.2.0\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "2vF0QrTtkQMt",
        "outputId": "b8592ec2-7954-4356-d8cb-a42576eb861d"
      },
      "source": [
        "!git clone https://github.com/HLTCHKUST/CAiRE-COVID.git"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Cloning into 'CAiRE-COVID'...\n",
            "remote: Enumerating objects: 144, done.\u001b[K\n",
            "remote: Counting objects: 100% (144/144), done.\u001b[K\n",
            "remote: Compressing objects: 100% (117/117), done.\u001b[K\n",
            "remote: Total 144 (delta 50), reused 105 (delta 24), pack-reused 0\u001b[K\n",
            "Receiving objects: 100% (144/144), 337.91 KiB | 2.66 MiB/s, done.\n",
            "Resolving deltas: 100% (50/50), done.\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "b0iomR8VkSOr",
        "outputId": "ab3e60b1-a356-43e6-f016-e81c57950c65"
      },
      "source": [
        "pip install -r CAiRE-COVID/requirements.txt"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Collecting pyserini==0.8.1.0\n",
            "\u001b[?25l  Downloading https://files.pythonhosted.org/packages/72/f4/45a9175211da4b7d4aed77af707eab938fc068d6124fc6673da748bc74bd/pyserini-0.8.1.0-py3-none-any.whl (57.7MB)\n",
            "\u001b[K     |████████████████████████████████| 57.7MB 67kB/s \n",
            "\u001b[?25hCollecting tensorflow-gpu==1.15.2\n",
            "\u001b[33m  WARNING: Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError(\"HTTPSConnectionPool(host='files.pythonhosted.org', port=443): Read timed out. (read timeout=15)\")': /packages/92/42/7bddc0d5bc169596fbd13b6e1b844f832491ab671381e483da3bf5292ca9/tensorflow_gpu-1.15.2-cp37-cp37m-manylinux2010_x86_64.whl\u001b[0m\n",
            "\u001b[?25l  Downloading https://files.pythonhosted.org/packages/92/42/7bddc0d5bc169596fbd13b6e1b844f832491ab671381e483da3bf5292ca9/tensorflow_gpu-1.15.2-cp37-cp37m-manylinux2010_x86_64.whl (410.9MB)\n",
            "\u001b[K     |████████████████████████████████| 411.0MB 37kB/s \n",
            "\u001b[?25hRequirement already satisfied: numpy in /usr/local/lib/python3.7/dist-packages (from -r CAiRE-COVID/requirements.txt (line 6)) (1.19.5)\n",
            "Requirement already satisfied: unidecode in /usr/local/lib/python3.7/dist-packages (from -r CAiRE-COVID/requirements.txt (line 7)) (1.2.0)\n",
            "Requirement already satisfied: requests in /usr/local/lib/python3.7/dist-packages (from -r CAiRE-COVID/requirements.txt (line 8)) (2.23.0)\n",
            "Requirement already satisfied: six in /usr/local/lib/python3.7/dist-packages (from -r CAiRE-COVID/requirements.txt (line 9)) (1.15.0)\n",
            "Requirement already satisfied: tqdm in /usr/local/lib/python3.7/dist-packages (from -r CAiRE-COVID/requirements.txt (line 10)) (4.41.1)\n",
            "Requirement already satisfied: sentencepiece in /usr/local/lib/python3.7/dist-packages (from -r CAiRE-COVID/requirements.txt (line 11)) (0.1.95)\n",
            "Requirement already satisfied: jsonlines in /usr/local/lib/python3.7/dist-packages (from -r CAiRE-COVID/requirements.txt (line 12)) (2.0.0)\n",
            "Requirement already satisfied: nltk in /usr/local/lib/python3.7/dist-packages (from -r CAiRE-COVID/requirements.txt (line 13)) (3.2.5)\n",
            "Requirement already satisfied: easydict in /usr/local/lib/python3.7/dist-packages (from -r CAiRE-COVID/requirements.txt (line 16)) (1.9)\n",
            "Collecting spacy==2.1.3\n",
            "\u001b[?25l  Downloading https://files.pythonhosted.org/packages/c7/29/ede5977ea144bb5758407542eb363ebfb11bbb459d26dea5dd0545563854/spacy-2.1.3-cp37-cp37m-manylinux1_x86_64.whl (27.7MB)\n",
            "\u001b[K     |████████████████████████████████| 27.7MB 174kB/s \n",
            "\u001b[?25hCollecting neuralcoref\n",
            "\u001b[?25l  Downloading https://files.pythonhosted.org/packages/06/6d/c90e5bfd1b8ef32f1b231a32f2f625bf33df7525324d2bbcd08992791d64/neuralcoref-4.0-cp37-cp37m-manylinux1_x86_64.whl (286kB)\n",
            "\u001b[K     |████████████████████████████████| 286kB 35.6MB/s \n",
            "\u001b[?25hRequirement already satisfied: Cython in /usr/local/lib/python3.7/dist-packages (from pyserini==0.8.1.0->-r CAiRE-COVID/requirements.txt (line 2)) (0.29.23)\n",
            "Collecting pyjnius\n",
            "\u001b[?25l  Downloading https://files.pythonhosted.org/packages/ea/b1/e33db12a20efe28b20fbcf4efc9b95a934954587cd7aa5998987a22e8885/pyjnius-1.3.0-cp37-cp37m-manylinux2010_x86_64.whl (1.1MB)\n",
            "\u001b[K     |████████████████████████████████| 1.1MB 18.7MB/s \n",
            "\u001b[?25hRequirement already satisfied: keras-preprocessing>=1.0.5 in /usr/local/lib/python3.7/dist-packages (from tensorflow-gpu==1.15.2->-r CAiRE-COVID/requirements.txt (line 5)) (1.1.2)\n",
            "Requirement already satisfied: astor>=0.6.0 in /usr/local/lib/python3.7/dist-packages (from tensorflow-gpu==1.15.2->-r CAiRE-COVID/requirements.txt (line 5)) (0.8.1)\n",
            "Requirement already satisfied: wrapt>=1.11.1 in /usr/local/lib/python3.7/dist-packages (from tensorflow-gpu==1.15.2->-r CAiRE-COVID/requirements.txt (line 5)) (1.12.1)\n",
            "Collecting keras-applications>=1.0.8\n",
            "\u001b[?25l  Downloading https://files.pythonhosted.org/packages/71/e3/19762fdfc62877ae9102edf6342d71b28fbfd9dea3d2f96a882ce099b03f/Keras_Applications-1.0.8-py3-none-any.whl (50kB)\n",
            "\u001b[K     |████████████████████████████████| 51kB 6.8MB/s \n",
            "\u001b[?25hCollecting tensorboard<1.16.0,>=1.15.0\n",
            "\u001b[?25l  Downloading https://files.pythonhosted.org/packages/1e/e9/d3d747a97f7188f48aa5eda486907f3b345cd409f0a0850468ba867db246/tensorboard-1.15.0-py3-none-any.whl (3.8MB)\n",
            "\u001b[K     |████████████████████████████████| 3.8MB 19.5MB/s \n",
            "\u001b[?25hRequirement already satisfied: wheel>=0.26; python_version >= \"3\" in /usr/local/lib/python3.7/dist-packages (from tensorflow-gpu==1.15.2->-r CAiRE-COVID/requirements.txt (line 5)) (0.36.2)\n",
            "Requirement already satisfied: google-pasta>=0.1.6 in /usr/local/lib/python3.7/dist-packages (from tensorflow-gpu==1.15.2->-r CAiRE-COVID/requirements.txt (line 5)) (0.2.0)\n",
            "Requirement already satisfied: grpcio>=1.8.6 in /usr/local/lib/python3.7/dist-packages (from tensorflow-gpu==1.15.2->-r CAiRE-COVID/requirements.txt (line 5)) (1.34.1)\n",
            "Requirement already satisfied: opt-einsum>=2.3.2 in /usr/local/lib/python3.7/dist-packages (from tensorflow-gpu==1.15.2->-r CAiRE-COVID/requirements.txt (line 5)) (3.3.0)\n",
            "Requirement already satisfied: termcolor>=1.1.0 in /usr/local/lib/python3.7/dist-packages (from tensorflow-gpu==1.15.2->-r CAiRE-COVID/requirements.txt (line 5)) (1.1.0)\n",
            "Collecting tensorflow-estimator==1.15.1\n",
            "\u001b[?25l  Downloading https://files.pythonhosted.org/packages/de/62/2ee9cd74c9fa2fa450877847ba560b260f5d0fb70ee0595203082dafcc9d/tensorflow_estimator-1.15.1-py2.py3-none-any.whl (503kB)\n",
            "\u001b[K     |████████████████████████████████| 512kB 32.5MB/s \n",
            "\u001b[?25hRequirement already satisfied: protobuf>=3.6.1 in /usr/local/lib/python3.7/dist-packages (from tensorflow-gpu==1.15.2->-r CAiRE-COVID/requirements.txt (line 5)) (3.12.4)\n",
            "Collecting gast==0.2.2\n",
            "  Downloading https://files.pythonhosted.org/packages/4e/35/11749bf99b2d4e3cceb4d55ca22590b0d7c2c62b9de38ac4a4a7f4687421/gast-0.2.2.tar.gz\n",
            "Requirement already satisfied: absl-py>=0.7.0 in /usr/local/lib/python3.7/dist-packages (from tensorflow-gpu==1.15.2->-r CAiRE-COVID/requirements.txt (line 5)) (0.12.0)\n",
            "Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.7/dist-packages (from requests->-r CAiRE-COVID/requirements.txt (line 8)) (2020.12.5)\n",
            "Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /usr/local/lib/python3.7/dist-packages (from requests->-r CAiRE-COVID/requirements.txt (line 8)) (1.24.3)\n",
            "Requirement already satisfied: idna<3,>=2.5 in /usr/local/lib/python3.7/dist-packages (from requests->-r CAiRE-COVID/requirements.txt (line 8)) (2.10)\n",
            "Requirement already satisfied: chardet<4,>=3.0.2 in /usr/local/lib/python3.7/dist-packages (from requests->-r CAiRE-COVID/requirements.txt (line 8)) (3.0.4)\n",
            "Collecting thinc<7.1.0,>=7.0.2\n",
            "\u001b[?25l  Downloading https://files.pythonhosted.org/packages/36/42/d7ea7539af3852fd8c1f0b3adf4a100fb3d72b40b69cef1a764ff979a743/thinc-7.0.8-cp37-cp37m-manylinux1_x86_64.whl (2.1MB)\n",
            "\u001b[K     |████████████████████████████████| 2.1MB 37.4MB/s \n",
            "\u001b[?25hRequirement already satisfied: wasabi<1.1.0,>=0.2.0 in /usr/local/lib/python3.7/dist-packages (from spacy==2.1.3->-r CAiRE-COVID/requirements.txt (line 17)) (0.8.2)\n",
            "Requirement already satisfied: srsly<1.1.0,>=0.0.5 in /usr/local/lib/python3.7/dist-packages (from spacy==2.1.3->-r CAiRE-COVID/requirements.txt (line 17)) (1.0.5)\n",
            "Collecting blis<0.3.0,>=0.2.2\n",
            "\u001b[?25l  Downloading https://files.pythonhosted.org/packages/fa/5f/47b7b29ad202b2210020e2f33bfb06d1db2abe0e709c2a84736e8a9d1bd5/blis-0.2.4-cp37-cp37m-manylinux1_x86_64.whl (3.2MB)\n",
            "\u001b[K     |████████████████████████████████| 3.2MB 31.8MB/s \n",
            "\u001b[?25hCollecting preshed<2.1.0,>=2.0.1\n",
            "\u001b[?25l  Downloading https://files.pythonhosted.org/packages/bc/2b/3ecd5d90d2d6fd39fbc520de7d80db5d74defdc2d7c2e15531d9cc3498c7/preshed-2.0.1-cp37-cp37m-manylinux1_x86_64.whl (82kB)\n",
            "\u001b[K     |████████████████████████████████| 92kB 10.1MB/s \n",
            "\u001b[?25hRequirement already satisfied: jsonschema<3.0.0,>=2.6.0 in /usr/local/lib/python3.7/dist-packages (from spacy==2.1.3->-r CAiRE-COVID/requirements.txt (line 17)) (2.6.0)\n",
            "Collecting plac<1.0.0,>=0.9.6\n",
            "  Downloading https://files.pythonhosted.org/packages/9e/9b/62c60d2f5bc135d2aa1d8c8a86aaf84edb719a59c7f11a4316259e61a298/plac-0.9.6-py2.py3-none-any.whl\n",
            "Requirement already satisfied: murmurhash<1.1.0,>=0.28.0 in /usr/local/lib/python3.7/dist-packages (from spacy==2.1.3->-r CAiRE-COVID/requirements.txt (line 17)) (1.0.5)\n",
            "Requirement already satisfied: cymem<2.1.0,>=2.0.2 in /usr/local/lib/python3.7/dist-packages (from spacy==2.1.3->-r CAiRE-COVID/requirements.txt (line 17)) (2.0.5)\n",
            "Collecting boto3\n",
            "\u001b[?25l  Downloading https://files.pythonhosted.org/packages/ab/d6/e750641a7e23ecb531b0b00ec8a0708e7687b921d691f4d199b6fcd8b3d5/boto3-1.17.81.tar.gz (98kB)\n",
            "\u001b[K     |████████████████████████████████| 102kB 11.1MB/s \n",
            "\u001b[?25hRequirement already satisfied: h5py in /usr/local/lib/python3.7/dist-packages (from keras-applications>=1.0.8->tensorflow-gpu==1.15.2->-r CAiRE-COVID/requirements.txt (line 5)) (3.1.0)\n",
            "Requirement already satisfied: setuptools>=41.0.0 in /usr/local/lib/python3.7/dist-packages (from tensorboard<1.16.0,>=1.15.0->tensorflow-gpu==1.15.2->-r CAiRE-COVID/requirements.txt (line 5)) (56.1.0)\n",
            "Requirement already satisfied: markdown>=2.6.8 in /usr/local/lib/python3.7/dist-packages (from tensorboard<1.16.0,>=1.15.0->tensorflow-gpu==1.15.2->-r CAiRE-COVID/requirements.txt (line 5)) (3.3.4)\n",
            "Requirement already satisfied: werkzeug>=0.11.15 in /usr/local/lib/python3.7/dist-packages (from tensorboard<1.16.0,>=1.15.0->tensorflow-gpu==1.15.2->-r CAiRE-COVID/requirements.txt (line 5)) (1.0.1)\n",
            "Collecting botocore<1.21.0,>=1.20.81\n",
            "\u001b[?25l  Downloading https://files.pythonhosted.org/packages/46/fd/219a45d005a328bd1c11708519d1c580dddad4f5945b92c25f9e56198f3e/botocore-1.20.81-py2.py3-none-any.whl (7.6MB)\n",
            "\u001b[K     |████████████████████████████████| 7.6MB 27.4MB/s \n",
            "\u001b[?25hCollecting jmespath<1.0.0,>=0.7.1\n",
            "  Downloading https://files.pythonhosted.org/packages/07/cb/5f001272b6faeb23c1c9e0acc04d48eaaf5c862c17709d20e3469c6e0139/jmespath-0.10.0-py2.py3-none-any.whl\n",
            "Collecting s3transfer<0.5.0,>=0.4.0\n",
            "\u001b[?25l  Downloading https://files.pythonhosted.org/packages/63/d0/693477c688348654ddc21dcdce0817653a294aa43f41771084c25e7ff9c7/s3transfer-0.4.2-py2.py3-none-any.whl (79kB)\n",
            "\u001b[K     |████████████████████████████████| 81kB 9.1MB/s \n",
            "\u001b[?25hRequirement already satisfied: cached-property; python_version < \"3.8\" in /usr/local/lib/python3.7/dist-packages (from h5py->keras-applications>=1.0.8->tensorflow-gpu==1.15.2->-r CAiRE-COVID/requirements.txt (line 5)) (1.5.2)\n",
            "Requirement already satisfied: importlib-metadata; python_version < \"3.8\" in /usr/local/lib/python3.7/dist-packages (from markdown>=2.6.8->tensorboard<1.16.0,>=1.15.0->tensorflow-gpu==1.15.2->-r CAiRE-COVID/requirements.txt (line 5)) (4.0.1)\n",
            "Requirement already satisfied: python-dateutil<3.0.0,>=2.1 in /usr/local/lib/python3.7/dist-packages (from botocore<1.21.0,>=1.20.81->boto3->neuralcoref->-r CAiRE-COVID/requirements.txt (line 18)) (2.8.1)\n",
            "Requirement already satisfied: zipp>=0.5 in /usr/local/lib/python3.7/dist-packages (from importlib-metadata; python_version < \"3.8\"->markdown>=2.6.8->tensorboard<1.16.0,>=1.15.0->tensorflow-gpu==1.15.2->-r CAiRE-COVID/requirements.txt (line 5)) (3.4.1)\n",
            "Requirement already satisfied: typing-extensions>=3.6.4; python_version < \"3.8\" in /usr/local/lib/python3.7/dist-packages (from importlib-metadata; python_version < \"3.8\"->markdown>=2.6.8->tensorboard<1.16.0,>=1.15.0->tensorflow-gpu==1.15.2->-r CAiRE-COVID/requirements.txt (line 5)) (3.7.4.3)\n",
            "Building wheels for collected packages: gast, boto3\n",
            "  Building wheel for gast (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "  Created wheel for gast: filename=gast-0.2.2-cp37-none-any.whl size=7540 sha256=6a386e7286237b273f7a9d2a2d0cde9777811892e02f672adc689078d9c7219b\n",
            "  Stored in directory: /root/.cache/pip/wheels/5c/2e/7e/a1d4d4fcebe6c381f378ce7743a3ced3699feb89bcfbdadadd\n",
            "  Building wheel for boto3 (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "  Created wheel for boto3: filename=boto3-1.17.81-py2.py3-none-any.whl size=128921 sha256=fb17962e2a7520efc50e56ed02bd8959d700cc6c546c266ea03135712a5f1780\n",
            "  Stored in directory: /root/.cache/pip/wheels/62/2c/8b/051d4e4933cda3659c4912652fd03cdcad9eee8c15fa06ec4d\n",
            "Successfully built gast boto3\n",
            "\u001b[31mERROR: tensorflow 2.5.0 has requirement gast==0.4.0, but you'll have gast 0.2.2 which is incompatible.\u001b[0m\n",
            "\u001b[31mERROR: tensorflow 2.5.0 has requirement tensorboard~=2.5, but you'll have tensorboard 1.15.0 which is incompatible.\u001b[0m\n",
            "\u001b[31mERROR: tensorflow 2.5.0 has requirement tensorflow-estimator<2.6.0,>=2.5.0rc0, but you'll have tensorflow-estimator 1.15.1 which is incompatible.\u001b[0m\n",
            "\u001b[31mERROR: tensorflow-probability 0.12.1 has requirement gast>=0.3.2, but you'll have gast 0.2.2 which is incompatible.\u001b[0m\n",
            "\u001b[31mERROR: en-core-web-sm 2.2.5 has requirement spacy>=2.2.2, but you'll have spacy 2.1.3 which is incompatible.\u001b[0m\n",
            "\u001b[31mERROR: botocore 1.20.81 has requirement urllib3<1.27,>=1.25.4, but you'll have urllib3 1.24.3 which is incompatible.\u001b[0m\n",
            "Installing collected packages: pyjnius, pyserini, keras-applications, tensorboard, tensorflow-estimator, gast, tensorflow-gpu, blis, preshed, plac, thinc, spacy, jmespath, botocore, s3transfer, boto3, neuralcoref\n",
            "  Found existing installation: tensorboard 2.5.0\n",
            "    Uninstalling tensorboard-2.5.0:\n",
            "      Successfully uninstalled tensorboard-2.5.0\n",
            "  Found existing installation: tensorflow-estimator 2.5.0\n",
            "    Uninstalling tensorflow-estimator-2.5.0:\n",
            "      Successfully uninstalled tensorflow-estimator-2.5.0\n",
            "  Found existing installation: gast 0.4.0\n",
            "    Uninstalling gast-0.4.0:\n",
            "      Successfully uninstalled gast-0.4.0\n",
            "  Found existing installation: blis 0.4.1\n",
            "    Uninstalling blis-0.4.1:\n",
            "      Successfully uninstalled blis-0.4.1\n",
            "  Found existing installation: preshed 3.0.5\n",
            "    Uninstalling preshed-3.0.5:\n",
            "      Successfully uninstalled preshed-3.0.5\n",
            "  Found existing installation: plac 1.1.3\n",
            "    Uninstalling plac-1.1.3:\n",
            "      Successfully uninstalled plac-1.1.3\n",
            "  Found existing installation: thinc 7.4.0\n",
            "    Uninstalling thinc-7.4.0:\n",
            "      Successfully uninstalled thinc-7.4.0\n",
            "  Found existing installation: spacy 2.2.4\n",
            "    Uninstalling spacy-2.2.4:\n",
            "      Successfully uninstalled spacy-2.2.4\n",
            "Successfully installed blis-0.2.4 boto3-1.17.81 botocore-1.20.81 gast-0.2.2 jmespath-0.10.0 keras-applications-1.0.8 neuralcoref-4.0 plac-0.9.6 preshed-2.0.1 pyjnius-1.3.0 pyserini-0.8.1.0 s3transfer-0.4.2 spacy-2.1.3 tensorboard-1.15.0 tensorflow-estimator-1.15.1 tensorflow-gpu-1.15.2 thinc-7.0.8\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "wLi1rvCBkwQl",
        "outputId": "c5ae3a1b-6905-4f22-864b-dc4920e19ceb"
      },
      "source": [
        "!pip install gast==0.2.2"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Requirement already satisfied: gast==0.2.2 in /usr/local/lib/python3.7/dist-packages (0.2.2)\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "IKcGqufgpWB3",
        "outputId": "f0430751-0530-46f9-b0a2-f9356b3beeba"
      },
      "source": [
        "!pip3 install torch torchvision torchaudio"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Requirement already satisfied: torch in /usr/local/lib/python3.7/dist-packages (1.8.1+cu101)\n",
            "Requirement already satisfied: torchvision in /usr/local/lib/python3.7/dist-packages (0.9.1+cu101)\n",
            "Collecting torchaudio\n",
            "\u001b[?25l  Downloading https://files.pythonhosted.org/packages/aa/55/01ad9244bcd595e39cea5ce30726a7fe02fd963d07daeb136bfe7e23f0a5/torchaudio-0.8.1-cp37-cp37m-manylinux1_x86_64.whl (1.9MB)\n",
            "\u001b[K     |████████████████████████████████| 1.9MB 2.9MB/s \n",
            "\u001b[?25hRequirement already satisfied: numpy in /usr/local/lib/python3.7/dist-packages (from torch) (1.19.5)\n",
            "Requirement already satisfied: typing-extensions in /usr/local/lib/python3.7/dist-packages (from torch) (3.7.4.3)\n",
            "Requirement already satisfied: pillow>=4.1.1 in /usr/local/lib/python3.7/dist-packages (from torchvision) (7.1.2)\n",
            "Installing collected packages: torchaudio\n",
            "Successfully installed torchaudio-0.8.1\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "XORfLoRFpcOP",
        "outputId": "8e76a060-5333-46d9-cfb6-40162229bfd4"
      },
      "source": [
        "! curl -O https://download.java.net/java/GA/jdk11/9/GPL/openjdk-11.0.2_linux-x64_bin.tar.gz\n",
        "! mv openjdk-11.0.2_linux-x64_bin.tar.gz /usr/lib/jvm/; cd /usr/lib/jvm/; tar -zxvf openjdk-11.0.2_linux-x64_bin.tar.gz\n",
        "! update-alternatives --install /usr/bin/java java /usr/lib/jvm/jdk-11.0.2/bin/java 1\n",
        "! update-alternatives --set java /usr/lib/jvm/jdk-11.0.2/bin/java"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current\n",
            "                                 Dload  Upload   Total   Spent    Left  Speed\n",
            "100  178M  100  178M    0     0  72.1M      0  0:00:02  0:00:02 --:--:-- 72.0M\n",
            "jdk-11.0.2/bin/jaotc\n",
            "jdk-11.0.2/bin/jar\n",
            "jdk-11.0.2/bin/jarsigner\n",
            "jdk-11.0.2/bin/java\n",
            "jdk-11.0.2/bin/javac\n",
            "jdk-11.0.2/bin/javadoc\n",
            "jdk-11.0.2/bin/javap\n",
            "jdk-11.0.2/bin/jcmd\n",
            "jdk-11.0.2/bin/jconsole\n",
            "jdk-11.0.2/bin/jdb\n",
            "jdk-11.0.2/bin/jdeprscan\n",
            "jdk-11.0.2/bin/jdeps\n",
            "jdk-11.0.2/bin/jhsdb\n",
            "jdk-11.0.2/bin/jimage\n",
            "jdk-11.0.2/bin/jinfo\n",
            "jdk-11.0.2/bin/jjs\n",
            "jdk-11.0.2/bin/jlink\n",
            "jdk-11.0.2/bin/jmap\n",
            "jdk-11.0.2/bin/jmod\n",
            "jdk-11.0.2/bin/jps\n",
            "jdk-11.0.2/bin/jrunscript\n",
            "jdk-11.0.2/bin/jshell\n",
            "jdk-11.0.2/bin/jstack\n",
            "jdk-11.0.2/bin/jstat\n",
            "jdk-11.0.2/bin/jstatd\n",
            "jdk-11.0.2/bin/keytool\n",
            "jdk-11.0.2/bin/pack200\n",
            "jdk-11.0.2/bin/rmic\n",
            "jdk-11.0.2/bin/rmid\n",
            "jdk-11.0.2/bin/rmiregistry\n",
            "jdk-11.0.2/bin/serialver\n",
            "jdk-11.0.2/bin/unpack200\n",
            "jdk-11.0.2/conf/logging.properties\n",
            "jdk-11.0.2/conf/management/jmxremote.access\n",
            "jdk-11.0.2/conf/management/jmxremote.password.template\n",
            "jdk-11.0.2/conf/management/management.properties\n",
            "jdk-11.0.2/conf/net.properties\n",
            "jdk-11.0.2/conf/security/java.policy\n",
            "jdk-11.0.2/conf/security/java.security\n",
            "jdk-11.0.2/conf/security/policy/README.txt\n",
            "jdk-11.0.2/conf/security/policy/limited/default_US_export.policy\n",
            "jdk-11.0.2/conf/security/policy/limited/default_local.policy\n",
            "jdk-11.0.2/conf/security/policy/limited/exempt_local.policy\n",
            "jdk-11.0.2/conf/security/policy/unlimited/default_US_export.policy\n",
            "jdk-11.0.2/conf/security/policy/unlimited/default_local.policy\n",
            "jdk-11.0.2/conf/sound.properties\n",
            "jdk-11.0.2/include/classfile_constants.h\n",
            "jdk-11.0.2/include/jawt.h\n",
            "jdk-11.0.2/include/jdwpTransport.h\n",
            "jdk-11.0.2/include/jni.h\n",
            "jdk-11.0.2/include/jvmti.h\n",
            "jdk-11.0.2/include/jvmticmlr.h\n",
            "jdk-11.0.2/include/linux/jawt_md.h\n",
            "jdk-11.0.2/include/linux/jni_md.h\n",
            "jdk-11.0.2/jmods/java.base.jmod\n",
            "jdk-11.0.2/jmods/java.compiler.jmod\n",
            "jdk-11.0.2/jmods/java.datatransfer.jmod\n",
            "jdk-11.0.2/jmods/java.desktop.jmod\n",
            "jdk-11.0.2/jmods/java.instrument.jmod\n",
            "jdk-11.0.2/jmods/java.logging.jmod\n",
            "jdk-11.0.2/jmods/java.management.jmod\n",
            "jdk-11.0.2/jmods/java.management.rmi.jmod\n",
            "jdk-11.0.2/jmods/java.naming.jmod\n",
            "jdk-11.0.2/jmods/java.net.http.jmod\n",
            "jdk-11.0.2/jmods/java.prefs.jmod\n",
            "jdk-11.0.2/jmods/java.rmi.jmod\n",
            "jdk-11.0.2/jmods/java.scripting.jmod\n",
            "jdk-11.0.2/jmods/java.se.jmod\n",
            "jdk-11.0.2/jmods/java.security.jgss.jmod\n",
            "jdk-11.0.2/jmods/java.security.sasl.jmod\n",
            "jdk-11.0.2/jmods/java.smartcardio.jmod\n",
            "jdk-11.0.2/jmods/java.sql.jmod\n",
            "jdk-11.0.2/jmods/java.sql.rowset.jmod\n",
            "jdk-11.0.2/jmods/java.transaction.xa.jmod\n",
            "jdk-11.0.2/jmods/java.xml.crypto.jmod\n",
            "jdk-11.0.2/jmods/java.xml.jmod\n",
            "jdk-11.0.2/jmods/jdk.accessibility.jmod\n",
            "jdk-11.0.2/jmods/jdk.aot.jmod\n",
            "jdk-11.0.2/jmods/jdk.attach.jmod\n",
            "jdk-11.0.2/jmods/jdk.charsets.jmod\n",
            "jdk-11.0.2/jmods/jdk.compiler.jmod\n",
            "jdk-11.0.2/jmods/jdk.crypto.cryptoki.jmod\n",
            "jdk-11.0.2/jmods/jdk.crypto.ec.jmod\n",
            "jdk-11.0.2/jmods/jdk.dynalink.jmod\n",
            "jdk-11.0.2/jmods/jdk.editpad.jmod\n",
            "jdk-11.0.2/jmods/jdk.hotspot.agent.jmod\n",
            "jdk-11.0.2/jmods/jdk.httpserver.jmod\n",
            "jdk-11.0.2/jmods/jdk.internal.ed.jmod\n",
            "jdk-11.0.2/jmods/jdk.internal.jvmstat.jmod\n",
            "jdk-11.0.2/jmods/jdk.internal.le.jmod\n",
            "jdk-11.0.2/jmods/jdk.internal.opt.jmod\n",
            "jdk-11.0.2/jmods/jdk.internal.vm.ci.jmod\n",
            "jdk-11.0.2/jmods/jdk.internal.vm.compiler.jmod\n",
            "jdk-11.0.2/jmods/jdk.internal.vm.compiler.management.jmod\n",
            "jdk-11.0.2/jmods/jdk.jartool.jmod\n",
            "jdk-11.0.2/jmods/jdk.javadoc.jmod\n",
            "jdk-11.0.2/jmods/jdk.jcmd.jmod\n",
            "jdk-11.0.2/jmods/jdk.jconsole.jmod\n",
            "jdk-11.0.2/jmods/jdk.jdeps.jmod\n",
            "jdk-11.0.2/jmods/jdk.jdi.jmod\n",
            "jdk-11.0.2/jmods/jdk.jdwp.agent.jmod\n",
            "jdk-11.0.2/jmods/jdk.jfr.jmod\n",
            "jdk-11.0.2/jmods/jdk.jlink.jmod\n",
            "jdk-11.0.2/jmods/jdk.jshell.jmod\n",
            "jdk-11.0.2/jmods/jdk.jsobject.jmod\n",
            "jdk-11.0.2/jmods/jdk.jstatd.jmod\n",
            "jdk-11.0.2/jmods/jdk.localedata.jmod\n",
            "jdk-11.0.2/jmods/jdk.management.agent.jmod\n",
            "jdk-11.0.2/jmods/jdk.management.jfr.jmod\n",
            "jdk-11.0.2/jmods/jdk.management.jmod\n",
            "jdk-11.0.2/jmods/jdk.naming.dns.jmod\n",
            "jdk-11.0.2/jmods/jdk.naming.rmi.jmod\n",
            "jdk-11.0.2/jmods/jdk.net.jmod\n",
            "jdk-11.0.2/jmods/jdk.pack.jmod\n",
            "jdk-11.0.2/jmods/jdk.rmic.jmod\n",
            "jdk-11.0.2/jmods/jdk.scripting.nashorn.jmod\n",
            "jdk-11.0.2/jmods/jdk.scripting.nashorn.shell.jmod\n",
            "jdk-11.0.2/jmods/jdk.sctp.jmod\n",
            "jdk-11.0.2/jmods/jdk.security.auth.jmod\n",
            "jdk-11.0.2/jmods/jdk.security.jgss.jmod\n",
            "jdk-11.0.2/jmods/jdk.unsupported.desktop.jmod\n",
            "jdk-11.0.2/jmods/jdk.unsupported.jmod\n",
            "jdk-11.0.2/jmods/jdk.xml.dom.jmod\n",
            "jdk-11.0.2/jmods/jdk.zipfs.jmod\n",
            "jdk-11.0.2/legal/java.base/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/java.base/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/java.base/LICENSE\n",
            "jdk-11.0.2/legal/java.base/aes.md\n",
            "jdk-11.0.2/legal/java.base/asm.md\n",
            "jdk-11.0.2/legal/java.base/c-libutl.md\n",
            "jdk-11.0.2/legal/java.base/cldr.md\n",
            "jdk-11.0.2/legal/java.base/icu.md\n",
            "jdk-11.0.2/legal/java.base/public_suffix.md\n",
            "jdk-11.0.2/legal/java.base/unicode.md\n",
            "jdk-11.0.2/legal/java.compiler/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/java.compiler/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/java.compiler/LICENSE\n",
            "jdk-11.0.2/legal/java.datatransfer/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/java.datatransfer/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/java.datatransfer/LICENSE\n",
            "jdk-11.0.2/legal/java.desktop/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/java.desktop/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/java.desktop/LICENSE\n",
            "jdk-11.0.2/legal/java.desktop/colorimaging.md\n",
            "jdk-11.0.2/legal/java.desktop/giflib.md\n",
            "jdk-11.0.2/legal/java.desktop/harfbuzz.md\n",
            "jdk-11.0.2/legal/java.desktop/jpeg.md\n",
            "jdk-11.0.2/legal/java.desktop/lcms.md\n",
            "jdk-11.0.2/legal/java.desktop/libpng.md\n",
            "jdk-11.0.2/legal/java.desktop/mesa3d.md\n",
            "jdk-11.0.2/legal/java.desktop/opengl.md\n",
            "jdk-11.0.2/legal/java.desktop/xwindows.md\n",
            "jdk-11.0.2/legal/java.instrument/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/java.instrument/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/java.instrument/LICENSE\n",
            "jdk-11.0.2/legal/java.logging/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/java.logging/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/java.logging/LICENSE\n",
            "jdk-11.0.2/legal/java.management.rmi/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/java.management.rmi/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/java.management.rmi/LICENSE\n",
            "jdk-11.0.2/legal/java.management/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/java.management/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/java.management/LICENSE\n",
            "jdk-11.0.2/legal/java.naming/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/java.naming/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/java.naming/LICENSE\n",
            "jdk-11.0.2/legal/java.net.http/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/java.net.http/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/java.net.http/LICENSE\n",
            "jdk-11.0.2/legal/java.prefs/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/java.prefs/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/java.prefs/LICENSE\n",
            "jdk-11.0.2/legal/java.rmi/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/java.rmi/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/java.rmi/LICENSE\n",
            "jdk-11.0.2/legal/java.scripting/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/java.scripting/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/java.scripting/LICENSE\n",
            "jdk-11.0.2/legal/java.se/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/java.se/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/java.se/LICENSE\n",
            "jdk-11.0.2/legal/java.security.jgss/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/java.security.jgss/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/java.security.jgss/LICENSE\n",
            "jdk-11.0.2/legal/java.security.sasl/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/java.security.sasl/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/java.security.sasl/LICENSE\n",
            "jdk-11.0.2/legal/java.smartcardio/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/java.smartcardio/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/java.smartcardio/LICENSE\n",
            "jdk-11.0.2/legal/java.smartcardio/pcsclite.md\n",
            "jdk-11.0.2/legal/java.sql.rowset/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/java.sql.rowset/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/java.sql.rowset/LICENSE\n",
            "jdk-11.0.2/legal/java.sql/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/java.sql/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/java.sql/LICENSE\n",
            "jdk-11.0.2/legal/java.transaction.xa/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/java.transaction.xa/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/java.transaction.xa/LICENSE\n",
            "jdk-11.0.2/legal/java.xml.crypto/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/java.xml.crypto/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/java.xml.crypto/LICENSE\n",
            "jdk-11.0.2/legal/java.xml.crypto/santuario.md\n",
            "jdk-11.0.2/legal/java.xml/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/java.xml/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/java.xml/LICENSE\n",
            "jdk-11.0.2/legal/java.xml/bcel.md\n",
            "jdk-11.0.2/legal/java.xml/dom.md\n",
            "jdk-11.0.2/legal/java.xml/jcup.md\n",
            "jdk-11.0.2/legal/java.xml/xalan.md\n",
            "jdk-11.0.2/legal/java.xml/xerces.md\n",
            "jdk-11.0.2/legal/jdk.accessibility/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/jdk.accessibility/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/jdk.accessibility/LICENSE\n",
            "jdk-11.0.2/legal/jdk.aot/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/jdk.aot/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/jdk.aot/LICENSE\n",
            "jdk-11.0.2/legal/jdk.attach/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/jdk.attach/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/jdk.attach/LICENSE\n",
            "jdk-11.0.2/legal/jdk.charsets/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/jdk.charsets/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/jdk.charsets/LICENSE\n",
            "jdk-11.0.2/legal/jdk.compiler/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/jdk.compiler/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/jdk.compiler/LICENSE\n",
            "jdk-11.0.2/legal/jdk.crypto.cryptoki/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/jdk.crypto.cryptoki/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/jdk.crypto.cryptoki/LICENSE\n",
            "jdk-11.0.2/legal/jdk.crypto.cryptoki/pkcs11cryptotoken.md\n",
            "jdk-11.0.2/legal/jdk.crypto.cryptoki/pkcs11wrapper.md\n",
            "jdk-11.0.2/legal/jdk.crypto.ec/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/jdk.crypto.ec/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/jdk.crypto.ec/LICENSE\n",
            "jdk-11.0.2/legal/jdk.crypto.ec/ecc.md\n",
            "jdk-11.0.2/legal/jdk.dynalink/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/jdk.dynalink/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/jdk.dynalink/LICENSE\n",
            "jdk-11.0.2/legal/jdk.dynalink/dynalink.md\n",
            "jdk-11.0.2/legal/jdk.editpad/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/jdk.editpad/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/jdk.editpad/LICENSE\n",
            "jdk-11.0.2/legal/jdk.hotspot.agent/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/jdk.hotspot.agent/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/jdk.hotspot.agent/LICENSE\n",
            "jdk-11.0.2/legal/jdk.httpserver/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/jdk.httpserver/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/jdk.httpserver/LICENSE\n",
            "jdk-11.0.2/legal/jdk.internal.ed/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/jdk.internal.ed/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/jdk.internal.ed/LICENSE\n",
            "jdk-11.0.2/legal/jdk.internal.jvmstat/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/jdk.internal.jvmstat/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/jdk.internal.jvmstat/LICENSE\n",
            "jdk-11.0.2/legal/jdk.internal.le/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/jdk.internal.le/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/jdk.internal.le/LICENSE\n",
            "jdk-11.0.2/legal/jdk.internal.le/jline.md\n",
            "jdk-11.0.2/legal/jdk.internal.opt/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/jdk.internal.opt/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/jdk.internal.opt/LICENSE\n",
            "jdk-11.0.2/legal/jdk.internal.opt/jopt-simple.md\n",
            "jdk-11.0.2/legal/jdk.internal.vm.ci/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/jdk.internal.vm.ci/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/jdk.internal.vm.ci/LICENSE\n",
            "jdk-11.0.2/legal/jdk.internal.vm.compiler.management/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/jdk.internal.vm.compiler.management/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/jdk.internal.vm.compiler.management/LICENSE\n",
            "jdk-11.0.2/legal/jdk.internal.vm.compiler/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/jdk.internal.vm.compiler/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/jdk.internal.vm.compiler/LICENSE\n",
            "jdk-11.0.2/legal/jdk.jartool/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/jdk.jartool/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/jdk.jartool/LICENSE\n",
            "jdk-11.0.2/legal/jdk.javadoc/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/jdk.javadoc/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/jdk.javadoc/LICENSE\n",
            "jdk-11.0.2/legal/jdk.javadoc/jquery-migrate.md\n",
            "jdk-11.0.2/legal/jdk.javadoc/jquery.md\n",
            "jdk-11.0.2/legal/jdk.javadoc/jqueryUI.md\n",
            "jdk-11.0.2/legal/jdk.javadoc/jszip.md\n",
            "jdk-11.0.2/legal/jdk.javadoc/pako.md\n",
            "jdk-11.0.2/legal/jdk.jcmd/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/jdk.jcmd/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/jdk.jcmd/LICENSE\n",
            "jdk-11.0.2/legal/jdk.jconsole/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/jdk.jconsole/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/jdk.jconsole/LICENSE\n",
            "jdk-11.0.2/legal/jdk.jdeps/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/jdk.jdeps/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/jdk.jdeps/LICENSE\n",
            "jdk-11.0.2/legal/jdk.jdi/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/jdk.jdi/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/jdk.jdi/LICENSE\n",
            "jdk-11.0.2/legal/jdk.jdwp.agent/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/jdk.jdwp.agent/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/jdk.jdwp.agent/LICENSE\n",
            "jdk-11.0.2/legal/jdk.jfr/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/jdk.jfr/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/jdk.jfr/LICENSE\n",
            "jdk-11.0.2/legal/jdk.jlink/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/jdk.jlink/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/jdk.jlink/LICENSE\n",
            "jdk-11.0.2/legal/jdk.jshell/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/jdk.jshell/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/jdk.jshell/LICENSE\n",
            "jdk-11.0.2/legal/jdk.jsobject/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/jdk.jsobject/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/jdk.jsobject/LICENSE\n",
            "jdk-11.0.2/legal/jdk.jstatd/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/jdk.jstatd/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/jdk.jstatd/LICENSE\n",
            "jdk-11.0.2/legal/jdk.localedata/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/jdk.localedata/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/jdk.localedata/LICENSE\n",
            "jdk-11.0.2/legal/jdk.localedata/cldr.md\n",
            "jdk-11.0.2/legal/jdk.localedata/thaidict.md\n",
            "jdk-11.0.2/legal/jdk.management.agent/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/jdk.management.agent/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/jdk.management.agent/LICENSE\n",
            "jdk-11.0.2/legal/jdk.management.jfr/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/jdk.management.jfr/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/jdk.management.jfr/LICENSE\n",
            "jdk-11.0.2/legal/jdk.management/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/jdk.management/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/jdk.management/LICENSE\n",
            "jdk-11.0.2/legal/jdk.naming.dns/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/jdk.naming.dns/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/jdk.naming.dns/LICENSE\n",
            "jdk-11.0.2/legal/jdk.naming.rmi/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/jdk.naming.rmi/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/jdk.naming.rmi/LICENSE\n",
            "jdk-11.0.2/legal/jdk.net/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/jdk.net/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/jdk.net/LICENSE\n",
            "jdk-11.0.2/legal/jdk.pack/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/jdk.pack/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/jdk.pack/LICENSE\n",
            "jdk-11.0.2/legal/jdk.rmic/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/jdk.rmic/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/jdk.rmic/LICENSE\n",
            "jdk-11.0.2/legal/jdk.scripting.nashorn.shell/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/jdk.scripting.nashorn.shell/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/jdk.scripting.nashorn.shell/LICENSE\n",
            "jdk-11.0.2/legal/jdk.scripting.nashorn/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/jdk.scripting.nashorn/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/jdk.scripting.nashorn/LICENSE\n",
            "jdk-11.0.2/legal/jdk.scripting.nashorn/double-conversion.md\n",
            "jdk-11.0.2/legal/jdk.scripting.nashorn/joni.md\n",
            "jdk-11.0.2/legal/jdk.sctp/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/jdk.sctp/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/jdk.sctp/LICENSE\n",
            "jdk-11.0.2/legal/jdk.security.auth/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/jdk.security.auth/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/jdk.security.auth/LICENSE\n",
            "jdk-11.0.2/legal/jdk.security.jgss/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/jdk.security.jgss/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/jdk.security.jgss/LICENSE\n",
            "jdk-11.0.2/legal/jdk.unsupported.desktop/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/jdk.unsupported.desktop/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/jdk.unsupported.desktop/LICENSE\n",
            "jdk-11.0.2/legal/jdk.unsupported/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/jdk.unsupported/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/jdk.unsupported/LICENSE\n",
            "jdk-11.0.2/legal/jdk.xml.dom/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/jdk.xml.dom/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/jdk.xml.dom/LICENSE\n",
            "jdk-11.0.2/legal/jdk.zipfs/ADDITIONAL_LICENSE_INFO\n",
            "jdk-11.0.2/legal/jdk.zipfs/ASSEMBLY_EXCEPTION\n",
            "jdk-11.0.2/legal/jdk.zipfs/LICENSE\n",
            "jdk-11.0.2/lib/classlist\n",
            "jdk-11.0.2/lib/ct.sym\n",
            "jdk-11.0.2/lib/jexec\n",
            "jdk-11.0.2/lib/jfr/default.jfc\n",
            "jdk-11.0.2/lib/jfr/profile.jfc\n",
            "jdk-11.0.2/lib/jli/libjli.so\n",
            "jdk-11.0.2/lib/jrt-fs.jar\n",
            "jdk-11.0.2/lib/jvm.cfg\n",
            "jdk-11.0.2/lib/libattach.so\n",
            "jdk-11.0.2/lib/libawt.so\n",
            "jdk-11.0.2/lib/libawt_headless.so\n",
            "jdk-11.0.2/lib/libawt_xawt.so\n",
            "jdk-11.0.2/lib/libdt_socket.so\n",
            "jdk-11.0.2/lib/libextnet.so\n",
            "jdk-11.0.2/lib/libfontmanager.so\n",
            "jdk-11.0.2/lib/libinstrument.so\n",
            "jdk-11.0.2/lib/libj2gss.so\n",
            "jdk-11.0.2/lib/libj2pcsc.so\n",
            "jdk-11.0.2/lib/libj2pkcs11.so\n",
            "jdk-11.0.2/lib/libjaas.so\n",
            "jdk-11.0.2/lib/libjava.so\n",
            "jdk-11.0.2/lib/libjavajpeg.so\n",
            "jdk-11.0.2/lib/libjawt.so\n",
            "jdk-11.0.2/lib/libjdwp.so\n",
            "jdk-11.0.2/lib/libjimage.so\n",
            "jdk-11.0.2/lib/libjsig.so\n",
            "jdk-11.0.2/lib/libjsound.so\n",
            "jdk-11.0.2/lib/liblcms.so\n",
            "jdk-11.0.2/lib/libmanagement.so\n",
            "jdk-11.0.2/lib/libmanagement_agent.so\n",
            "jdk-11.0.2/lib/libmanagement_ext.so\n",
            "jdk-11.0.2/lib/libmlib_image.so\n",
            "jdk-11.0.2/lib/libnet.so\n",
            "jdk-11.0.2/lib/libnio.so\n",
            "jdk-11.0.2/lib/libprefs.so\n",
            "jdk-11.0.2/lib/librmi.so\n",
            "jdk-11.0.2/lib/libsaproc.so\n",
            "jdk-11.0.2/lib/libsctp.so\n",
            "jdk-11.0.2/lib/libsplashscreen.so\n",
            "jdk-11.0.2/lib/libsunec.so\n",
            "jdk-11.0.2/lib/libunpack.so\n",
            "jdk-11.0.2/lib/libverify.so\n",
            "jdk-11.0.2/lib/libzip.so\n",
            "jdk-11.0.2/lib/modules\n",
            "jdk-11.0.2/lib/psfont.properties.ja\n",
            "jdk-11.0.2/lib/psfontj2d.properties\n",
            "jdk-11.0.2/lib/security/blacklisted.certs\n",
            "jdk-11.0.2/lib/security/cacerts\n",
            "jdk-11.0.2/lib/security/default.policy\n",
            "jdk-11.0.2/lib/security/public_suffix_list.dat\n",
            "jdk-11.0.2/lib/server/Xusage.txt\n",
            "jdk-11.0.2/lib/server/libjsig.so\n",
            "jdk-11.0.2/lib/server/libjvm.so\n",
            "jdk-11.0.2/lib/src.zip\n",
            "jdk-11.0.2/lib/tzdb.dat\n",
            "jdk-11.0.2/release\n",
            "update-alternatives: using /usr/lib/jvm/jdk-11.0.2/bin/java to provide /usr/bin/java (java) in manual mode\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "_4MKXTmbpiU8"
      },
      "source": [
        "import os\n",
        "os.environ[\"JAVA_HOME\"] = \"/usr/lib/jvm/jdk-11.0.2\""
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "L9dsADtwp71a",
        "outputId": "30a761b4-4f74-44df-f18c-c454f39214eb"
      },
      "source": [
        "! pip install pyserini==0.8.1.0"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Requirement already satisfied: pyserini==0.8.1.0 in /usr/local/lib/python3.7/dist-packages (0.8.1.0)\n",
            "Requirement already satisfied: Cython in /usr/local/lib/python3.7/dist-packages (from pyserini==0.8.1.0) (0.29.23)\n",
            "Requirement already satisfied: pyjnius in /usr/local/lib/python3.7/dist-packages (from pyserini==0.8.1.0) (1.3.0)\n",
            "Requirement already satisfied: six>=1.7.0 in /usr/local/lib/python3.7/dist-packages (from pyjnius->pyserini==0.8.1.0) (1.15.0)\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "ZVyso7Ocp9mo"
      },
      "source": [
        "from pyserini.search import pysearch\n",
        "COVID_INDEX = '../input/luceneindexcovidparagraph20200410/lucene-index-covid-paragraph-2020-04-24'"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "WJhUdpEwai8Y"
      },
      "source": [
        "#### Change into CAiRE-COVID source-code directory\n",
        "\n",
        "For this section, in order to get the CAiRE-COVID source code function,\n",
        "I had to make a series of changes to the source code.\n",
        "* The src directory had to be moved into the projects directory.\n",
        "* Re-versioning had to be done for several installed packages.\n"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "b_ZAQdtEuu5V",
        "outputId": "f35c9967-64a6-4a23-d2f1-bed12b220e1a"
      },
      "source": [
        "cd CAiRE-COVID/"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "/content/CAiRE-COVID\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "Vnnc7chkapJG"
      },
      "source": [
        "#### Perform reversioning of packages"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "RSThBoIhxOS8",
        "outputId": "9a8ae9e1-5792-41eb-ec57-f92b706aa6c0"
      },
      "source": [
        "!pip install fairseq\n",
        "!pip uninstall transformers\n",
        "!pip install transformers==3.5"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Requirement already satisfied: fairseq in /usr/local/lib/python3.7/dist-packages (0.10.2)\n",
            "Requirement already satisfied: torch in /usr/local/lib/python3.7/dist-packages (from fairseq) (1.8.1+cu101)\n",
            "Requirement already satisfied: sacrebleu>=1.4.12 in /usr/local/lib/python3.7/dist-packages (from fairseq) (1.5.1)\n",
            "Requirement already satisfied: numpy in /usr/local/lib/python3.7/dist-packages (from fairseq) (1.19.5)\n",
            "Requirement already satisfied: regex in /usr/local/lib/python3.7/dist-packages (from fairseq) (2019.12.20)\n",
            "Requirement already satisfied: cffi in /usr/local/lib/python3.7/dist-packages (from fairseq) (1.14.5)\n",
            "Requirement already satisfied: hydra-core in /usr/local/lib/python3.7/dist-packages (from fairseq) (1.0.6)\n",
            "Requirement already satisfied: tqdm in /usr/local/lib/python3.7/dist-packages (from fairseq) (4.41.1)\n",
            "Requirement already satisfied: cython in /usr/local/lib/python3.7/dist-packages (from fairseq) (0.29.23)\n",
            "Requirement already satisfied: dataclasses in /usr/local/lib/python3.7/dist-packages (from fairseq) (0.6)\n",
            "Requirement already satisfied: typing-extensions in /usr/local/lib/python3.7/dist-packages (from torch->fairseq) (3.7.4.3)\n",
            "Requirement already satisfied: portalocker==2.0.0 in /usr/local/lib/python3.7/dist-packages (from sacrebleu>=1.4.12->fairseq) (2.0.0)\n",
            "Requirement already satisfied: pycparser in /usr/local/lib/python3.7/dist-packages (from cffi->fairseq) (2.20)\n",
            "Requirement already satisfied: importlib-resources; python_version < \"3.9\" in /usr/local/lib/python3.7/dist-packages (from hydra-core->fairseq) (5.1.3)\n",
            "Requirement already satisfied: antlr4-python3-runtime==4.8 in /usr/local/lib/python3.7/dist-packages (from hydra-core->fairseq) (4.8)\n",
            "Requirement already satisfied: omegaconf<2.1,>=2.0.5 in /usr/local/lib/python3.7/dist-packages (from hydra-core->fairseq) (2.0.6)\n",
            "Requirement already satisfied: zipp>=0.4; python_version < \"3.8\" in /usr/local/lib/python3.7/dist-packages (from importlib-resources; python_version < \"3.9\"->hydra-core->fairseq) (3.4.1)\n",
            "Requirement already satisfied: PyYAML>=5.1.* in /usr/local/lib/python3.7/dist-packages (from omegaconf<2.1,>=2.0.5->hydra-core->fairseq) (5.4.1)\n",
            "ERROR: unknown command \"uninstrall\" - maybe you meant \"uninstall\"\n",
            "Collecting transformers==3.5\n",
            "\u001b[?25l  Downloading https://files.pythonhosted.org/packages/9c/34/fb092588df61bf33f113ade030d1cbe74fb73a0353648f8dd938a223dce7/transformers-3.5.0-py3-none-any.whl (1.3MB)\n",
            "\u001b[K     |████████████████████████████████| 1.3MB 3.0MB/s \n",
            "\u001b[?25hRequirement already satisfied: packaging in /usr/local/lib/python3.7/dist-packages (from transformers==3.5) (20.9)\n",
            "Collecting sentencepiece==0.1.91\n",
            "\u001b[?25l  Downloading https://files.pythonhosted.org/packages/f2/e2/813dff3d72df2f49554204e7e5f73a3dc0f0eb1e3958a4cad3ef3fb278b7/sentencepiece-0.1.91-cp37-cp37m-manylinux1_x86_64.whl (1.1MB)\n",
            "\u001b[K     |████████████████████████████████| 1.1MB 20.0MB/s \n",
            "\u001b[?25hRequirement already satisfied: numpy in /usr/local/lib/python3.7/dist-packages (from transformers==3.5) (1.19.5)\n",
            "Requirement already satisfied: filelock in /usr/local/lib/python3.7/dist-packages (from transformers==3.5) (3.0.12)\n",
            "Requirement already satisfied: requests in /usr/local/lib/python3.7/dist-packages (from transformers==3.5) (2.23.0)\n",
            "Requirement already satisfied: tqdm>=4.27 in /usr/local/lib/python3.7/dist-packages (from transformers==3.5) (4.41.1)\n",
            "Requirement already satisfied: sacremoses in /usr/local/lib/python3.7/dist-packages (from transformers==3.5) (0.0.45)\n",
            "Requirement already satisfied: protobuf in /usr/local/lib/python3.7/dist-packages (from transformers==3.5) (3.12.4)\n",
            "Collecting tokenizers==0.9.3\n",
            "\u001b[?25l  Downloading https://files.pythonhosted.org/packages/7b/ac/f5ba028f0f097d855e1541301e946d4672eb0f30b6e25cb2369075f916d2/tokenizers-0.9.3-cp37-cp37m-manylinux1_x86_64.whl (2.9MB)\n",
            "\u001b[K     |████████████████████████████████| 2.9MB 23.8MB/s \n",
            "\u001b[?25hRequirement already satisfied: regex!=2019.12.17 in /usr/local/lib/python3.7/dist-packages (from transformers==3.5) (2019.12.20)\n",
            "Requirement already satisfied: pyparsing>=2.0.2 in /usr/local/lib/python3.7/dist-packages (from packaging->transformers==3.5) (2.4.7)\n",
            "Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.7/dist-packages (from requests->transformers==3.5) (2020.12.5)\n",
            "Requirement already satisfied: idna<3,>=2.5 in /usr/local/lib/python3.7/dist-packages (from requests->transformers==3.5) (2.10)\n",
            "Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /usr/local/lib/python3.7/dist-packages (from requests->transformers==3.5) (1.24.3)\n",
            "Requirement already satisfied: chardet<4,>=3.0.2 in /usr/local/lib/python3.7/dist-packages (from requests->transformers==3.5) (3.0.4)\n",
            "Requirement already satisfied: click in /usr/local/lib/python3.7/dist-packages (from sacremoses->transformers==3.5) (7.1.2)\n",
            "Requirement already satisfied: joblib in /usr/local/lib/python3.7/dist-packages (from sacremoses->transformers==3.5) (1.0.1)\n",
            "Requirement already satisfied: six in /usr/local/lib/python3.7/dist-packages (from sacremoses->transformers==3.5) (1.15.0)\n",
            "Requirement already satisfied: setuptools in /usr/local/lib/python3.7/dist-packages (from protobuf->transformers==3.5) (56.1.0)\n",
            "Installing collected packages: sentencepiece, tokenizers, transformers\n",
            "  Found existing installation: sentencepiece 0.1.95\n",
            "    Uninstalling sentencepiece-0.1.95:\n",
            "      Successfully uninstalled sentencepiece-0.1.95\n",
            "  Found existing installation: tokenizers 0.10.3\n",
            "    Uninstalling tokenizers-0.10.3:\n",
            "      Successfully uninstalled tokenizers-0.10.3\n",
            "  Found existing installation: transformers 4.6.1\n",
            "    Uninstalling transformers-4.6.1:\n",
            "      Successfully uninstalled transformers-4.6.1\n",
            "Successfully installed sentencepiece-0.1.91 tokenizers-0.9.3 transformers-3.5.0\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "c7LNOr5tz-Pf",
        "outputId": "3a420ee9-3370-4232-fcc7-7630297fafba"
      },
      "source": [
        "!pip install torch==1.4.0"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Collecting torch==1.4.0\n",
            "\u001b[?25l  Downloading https://files.pythonhosted.org/packages/1a/3b/fa92ece1e58a6a48ec598bab327f39d69808133e5b2fb33002ca754e381e/torch-1.4.0-cp37-cp37m-manylinux1_x86_64.whl (753.4MB)\n",
            "\u001b[K     |████████████████████████████████| 753.4MB 23kB/s \n",
            "\u001b[31mERROR: torchvision 0.9.1+cu101 has requirement torch==1.8.1, but you'll have torch 1.4.0 which is incompatible.\u001b[0m\n",
            "\u001b[31mERROR: torchtext 0.9.1 has requirement torch==1.8.1, but you'll have torch 1.4.0 which is incompatible.\u001b[0m\n",
            "\u001b[31mERROR: torchaudio 0.8.1 has requirement torch==1.8.1, but you'll have torch 1.4.0 which is incompatible.\u001b[0m\n",
            "\u001b[?25hInstalling collected packages: torch\n",
            "Successfully installed torch-1.4.0\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "U5Ct-pbfaupU"
      },
      "source": [
        "#### Get the lucene index"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "tj7dsUmrJ4b4",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "9fd83bc8-3393-46b6-9c4b-ed5ccadfccef"
      },
      "source": [
        "!wget https://www.dropbox.com/s/jdfbrnohtkrvds5/lucene-index-cord19-paragraph-2020-07-16.tar.gz?dl=1\n"
      ],
      "execution_count": 40,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "--2021-05-27 00:56:06--  https://www.dropbox.com/s/jdfbrnohtkrvds5/lucene-index-cord19-paragraph-2020-07-16.tar.gz?dl=1\n",
            "Resolving www.dropbox.com (www.dropbox.com)... 162.125.82.18, 2620:100:6032:18::a27d:5212\n",
            "Connecting to www.dropbox.com (www.dropbox.com)|162.125.82.18|:443... connected.\n",
            "HTTP request sent, awaiting response... 301 Moved Permanently\n",
            "Location: /s/dl/jdfbrnohtkrvds5/lucene-index-cord19-paragraph-2020-07-16.tar.gz [following]\n",
            "--2021-05-27 00:56:06--  https://www.dropbox.com/s/dl/jdfbrnohtkrvds5/lucene-index-cord19-paragraph-2020-07-16.tar.gz\n",
            "Reusing existing connection to www.dropbox.com:443.\n",
            "HTTP request sent, awaiting response... 302 Found\n",
            "Location: https://uc9d48e50ec4960281550e78291b.dl.dropboxusercontent.com/cd/0/get/BPTY4WEOQh96Wt1k1n0mIXVDhV-x1FWjFkoimpJuW0i0_HzdnSMaWdXAMy_7ASAfLDhqbgJkb_mDs5xoqKusvKDEKIjCbwqowArE0n9-m5f4Kzn2KnkwJfAMyTiIDIoYK5gPqQE6zxYUM_w3i1P5HOFO/file?dl=1# [following]\n",
            "--2021-05-27 00:56:07--  https://uc9d48e50ec4960281550e78291b.dl.dropboxusercontent.com/cd/0/get/BPTY4WEOQh96Wt1k1n0mIXVDhV-x1FWjFkoimpJuW0i0_HzdnSMaWdXAMy_7ASAfLDhqbgJkb_mDs5xoqKusvKDEKIjCbwqowArE0n9-m5f4Kzn2KnkwJfAMyTiIDIoYK5gPqQE6zxYUM_w3i1P5HOFO/file?dl=1\n",
            "Resolving uc9d48e50ec4960281550e78291b.dl.dropboxusercontent.com (uc9d48e50ec4960281550e78291b.dl.dropboxusercontent.com)... 162.125.82.15, 2620:100:6032:15::a27d:520f\n",
            "Connecting to uc9d48e50ec4960281550e78291b.dl.dropboxusercontent.com (uc9d48e50ec4960281550e78291b.dl.dropboxusercontent.com)|162.125.82.15|:443... connected.\n",
            "HTTP request sent, awaiting response... 200 OK\n",
            "Length: 6363337237 (5.9G) [application/binary]\n",
            "Saving to: ‘lucene-index-cord19-paragraph-2020-07-16.tar.gz?dl=1’\n",
            "\n",
            "lucene-index-cord19 100%[===================>]   5.93G  17.1MB/s    in 6m 30s  \n",
            "\n",
            "2021-05-27 01:02:37 (15.6 MB/s) - ‘lucene-index-cord19-paragraph-2020-07-16.tar.gz?dl=1’ saved [6363337237/6363337237]\n",
            "\n",
            "tar (child): lucene-index-cord19-paragraph-2020-07-16.tar.gz: Cannot open: No such file or directory\n",
            "tar (child): Error is not recoverable: exiting now\n",
            "tar: Child returned status 2\n",
            "tar: Error is not recoverable: exiting now\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "I-JSbvlvaz0U"
      },
      "source": [
        "#### Un-package the lucene index\n",
        "** Note. re-name may need to rename file to remove the trailing '?dl=1'"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "L9D_UNaxWKF_",
        "outputId": "a7b8951a-6e07-48c2-d326-ce46a0ca9821"
      },
      "source": [
        "!tar xvfz lucene-index-cord19-paragraph-2020-07-16.tar.gz"
      ],
      "execution_count": 41,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "lucene-index-cord19-paragraph-2020-07-16/\n",
            "lucene-index-cord19-paragraph-2020-07-16/_2.tvd\n",
            "lucene-index-cord19-paragraph-2020-07-16/segments_2\n",
            "lucene-index-cord19-paragraph-2020-07-16/_2.tvx\n",
            "lucene-index-cord19-paragraph-2020-07-16/_2.fdx\n",
            "lucene-index-cord19-paragraph-2020-07-16/_2.fnm\n",
            "lucene-index-cord19-paragraph-2020-07-16/_2.dii\n",
            "lucene-index-cord19-paragraph-2020-07-16/_2.nvm\n",
            "lucene-index-cord19-paragraph-2020-07-16/_2.nvd\n",
            "lucene-index-cord19-paragraph-2020-07-16/_2_Lucene80_0.dvm\n",
            "lucene-index-cord19-paragraph-2020-07-16/_2_Lucene80_0.dvd\n",
            "lucene-index-cord19-paragraph-2020-07-16/_2.fdt\n",
            "lucene-index-cord19-paragraph-2020-07-16/_2.si\n",
            "lucene-index-cord19-paragraph-2020-07-16/write.lock\n",
            "lucene-index-cord19-paragraph-2020-07-16/_2.dim\n",
            "lucene-index-cord19-paragraph-2020-07-16/_2_Lucene50_0.doc\n",
            "lucene-index-cord19-paragraph-2020-07-16/_2_Lucene50_0.tim\n",
            "lucene-index-cord19-paragraph-2020-07-16/_2_Lucene50_0.tip\n",
            "lucene-index-cord19-paragraph-2020-07-16/_2_Lucene50_0.pos\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "8FY80Qt_bFJO"
      },
      "source": [
        "#### Run the example retrieval script"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "ak1oYWL7qEcp",
        "outputId": "462e3bbe-5495-420b-9b3b-09518064ff3c"
      },
      "source": [
        "!python project/retrieval.py"
      ],
      "execution_count": 55,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Better speed can be achieved with apex installed from https://www.github.com/nvidia/apex.\n",
            "05/27/2021 01:15:23 - INFO - neuralcoref -   Loading model from /root/.neuralcoref_cache/neuralcoref\n",
            "1000\n",
            "1000 967 0\n",
            "[{'rank': 1, 'abstract': 'Background: Many countries have implemented population-wide interventions such as physical distancing measures, in efforts to control COVID-19. The extent and success of such measures has varied. Many jurisdictions with declines in reported COVID-19 cases are moving to relax measures, while others are continuing to intensify efforts to reduce transmission. Aim: We aim to determine the time frame between a change in COVID-19 measures at the population level and the observable impact of such a change on cases. Methods: We examine how long it takes for there to be a substantial difference between the cases that occur following a change in control measures and those that would have occurred at baseline. We then examine how long it takes to detect a difference, given delays and noise in reported cases. We use changes in population-level (e.g., distancing) control measures informed by data and estimates from British Columbia, Canada. Results: We find that the time frames are long: it takes three weeks or more before we might expect a substantial difference in cases given a change in population-level COVID-19 control, and it takes slightly longer to detect the impacts of the change. The time frames are shorter (11-15 days) for dramatic changes in control, and they are impacted by noise and delays in the testing and reporting process, with delays reaching up to 25-40 days. Conclusion: The time until a change in broad control measures has an observed impact is longer than is typically understood, and is longer than the mean incubation period (time between exposure than onset) and the often used 14 day time period. Policy makers and public health planners should consider this when assessing the impact of policy change, and efforts should be made to develop rapid, consistent real-time COVID-19 surveillance.', 'body_text': [{'text': 'In response to the COVID-19 global pandemic, many countries have implemented large-scale physical (\"social\") distancing measures. The details of physical distancing vary significantly between and within countries. While government-mandated lockdowns have been enforced in many parts of the world, certain regions have introduced less strict distancing measures. Such differences in the severity and timeliness of the response, as well as variation in patterns of social contact within a community and public compliance, all modulate the effect of distancing measures on a region\\'s epidemic trajectory. The relaxation of physical distancing must be done gradually and with care. Accordingly, countries experiencing increasing case counts have considered, and will continue to need to consider, what degree of distancing measures to implement in order to reduce disease spread whilst maintaining public compliance and minimising negative effects of distancing (such as adverse health and economic impacts). Understanding the possible trajectories which may arise from changing physical distancing measures is crucial to ensuring that sufficient measures are still taken after relaxation begins. Quantifying the effect and time scale of such relaxations will contribute to health care capacity forecasting, and delivery of timely advice and instructions to communities.', 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'A number of recent studies have examined the effectiveness of government control measures on the spread of COVID-19 [2, 8, 34, 35, 23, 32, 18, 9] . One approach has been to retrospectively compare observed data to baseline model output [34, 8] . Many studies have also focused on estimating changes in the effective reproduction number over time [11, 6, 35] . Similarly, [2] directly estimates the impact of control measures on COVID-19 transmission patterns using a Bayesian model with explicit physical distancing. However, these studies do not focus on reporting the time until a given amount of change is predicted between many simulations of future case counts. In-stead, they compare simulations to a constant baseline, or report quantitative measures on only a few simulations.', 'cite_spans': [{'start': 116, 'end': 119, 'text': '[2,', 'ref_id': 'BIBREF1'}, {'start': 120, 'end': 122, 'text': '8,', 'ref_id': 'BIBREF7'}, {'start': 123, 'end': 126, 'text': '34,', 'ref_id': 'BIBREF33'}, {'start': 127, 'end': 130, 'text': '35,', 'ref_id': 'BIBREF34'}, {'start': 131, 'end': 134, 'text': '23,', 'ref_id': 'BIBREF22'}, {'start': 135, 'end': 138, 'text': '32,', 'ref_id': 'BIBREF31'}, {'start': 139, 'end': 142, 'text': '18,', 'ref_id': 'BIBREF17'}, {'start': 143, 'end': 145, 'text': '9]', 'ref_id': 'BIBREF8'}, {'start': 236, 'end': 240, 'text': '[34,', 'ref_id': 'BIBREF33'}, {'start': 241, 'end': 243, 'text': '8]', 'ref_id': 'BIBREF7'}, {'start': 346, 'end': 350, 'text': '[11,', 'ref_id': 'BIBREF10'}, {'start': 351, 'end': 353, 'text': '6,', 'ref_id': 'BIBREF5'}, {'start': 354, 'end': 357, 'text': '35]', 'ref_id': 'BIBREF34'}, {'start': 371, 'end': 374, 'text': '[2]', 'ref_id': 'BIBREF1'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'In this work, we use a likelihood-based approach to determine when we may expect to see the effects of implementing or relaxing physical distancing measures using case count data. This is in contrast to the work described above which focuses on quantifying the effect of such measures. We further use this method to estimate the changes in growth rate that would be expected after beginning to strengthen or relax physical distancing. We use the SEIR-type model developed in [2] to simulate scenarios over a broad range of simulation parameters and strengths of distancing. We consider the impact on the time to detect a change in distancing of data collection delays and inconsistencies, and apply our methods to publicly available case count data from British Columbia, Canada. While we focus here on the \"first\" implementation and relaxation of physical distancing for COVID-19, these methods can also be used to detect the first time we would expect to see a change in reported case counts in response to modifications of any non-pharmaceutical intervention (NPI). For example, we could use this model to explore the effects of introducing digital contact tracing or improved testing.', 'cite_spans': [{'start': 475, 'end': 478, 'text': '[2]', 'ref_id': 'BIBREF1'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'We use a compartmental model describing susceptible, exposed, infectious and removed individuals (SEIR) [2] . We fit the model to data on daily case counts, and incorporate knowledge of the incubation period and duration of the infectious period in the model. The model includes a fixed proportion of the population who are willing and able to practise physical distancing, although individuals can transition between distancing and non-distancing modes. Upon infectious contact, individuals move from a susceptible state (S) to an exposed state (E 1 ). From here, the disease progresses through the infectious, pre-symptomatic state (E 2 ), to infectious and symptomatic (I), and finally to removed (R). Removed individuals are assumed to be not susceptible. The model furthermore incorporates a possible quarantined state (Q) following symptom onset, in which individuals are still infected but are unable to infect others. For each state, there is a corresponding \"distanced\" state. A schematic for this model is shown in Figure 1 .', 'cite_spans': [{'start': 104, 'end': 107, 'text': '[2]', 'ref_id': 'BIBREF1'}], 'ref_spans': [{'start': 1025, 'end': 1033, 'text': 'Figure 1', 'ref_id': None}], 'section': 'Transmission model'}, {'text': 'The differential equations for the non-distancing compartments are given as follows:', 'cite_spans': [], 'ref_spans': [], 'section': 'Transmission model'}, {'text': 'The differential equations for individuals practising physical distancing are analogous:', 'cite_spans': [], 'ref_spans': [], 'section': 'Transmission model'}, {'text': 'In these equations, the strength of physical distancing is represented by the parameter 0 ≤ f (t) ≤ 1, with f (t) = 1 indicating no physical distancing. This parameter modulates transmission within the distancing compartments. In addition, individuals in the distancing compartments are less likely to be encountered by others, and so contribute a reduced portion of the force of infection as well. The value of f may be allowed to vary in time as physical distancing is strengthened or relaxed in the population. In particular, for the purpose of this work we define: We take the mean number of new cases reported on day t, µ t , to be a weighted sum of those 4 . CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'cite_spans': [], 'ref_spans': [], 'section': 'Transmission model'}, {'text': 'The copyright holder for this preprint this version posted June 16, 2020. . https://doi.org/10.1101/2020.06.14.20131177 doi: medRxiv preprint Figure 1 : Schematic of the epidemiological model from [2] . Compartments are: susceptible to the virus (S); exposed (E 1 ); exposed, pre-symptomatic, and infectious (E 2 ); symptomatic and infectious (I); quarantined (Q); and removed (recovered or deceased; R). There are analogous states for individuals practising physical distancing (bottom row). An individual in state X can begin distancing and move to the corresponding distanced state X d at rate u d . The reverse transition occurs at rate u r . The model quickly settles on a fraction e = u d /(u d + u r ) participating in distancing, and dynamics depend on this fraction, rather than on the rates u d and u r . The red and tinted red squares represent the compartments that we identify as active cases. The orange/orange tinted and red/red tinted compartments indicate infectious cases.', 'cite_spans': [{'start': 197, 'end': 200, 'text': '[2]', 'ref_id': 'BIBREF1'}], 'ref_spans': [{'start': 142, 'end': 150, 'text': 'Figure 1', 'ref_id': None}], 'section': 'Transmission model'}, {'text': 'who became symptomatic at some time s days ago:', 'cite_spans': [], 'ref_spans': [], 'section': 'Transmission model'}, {'text': 'Here w c represents the distribution of delay between the onset of symptoms and a positive test being reported and ψ t is the fraction of eligible cases on day t that will be tested and reported. The upper limit on the integral, M , represents the maximum possible delay in days between onset and reporting, for which we use twice the mean delay. We seek to determine how soon changes in distancing measures will cause substantial changes in the number of prevalent cases. And then, determine how well we may be able to estimate the strength of these measures given the delay and noise in the reported case counts and uncertainty about the model parameters. Given a set of daily case count data {C t }, where C t is the number of cases identified on day t, we use a negative binomial likelihood with mean µ t and dispersion parameter φ, NB(C t | µ t , φ) as in [2] , to write the likelihood of the data given the model parameters accounting for dispersion:', 'cite_spans': [{'start': 861, 'end': 864, 'text': '[2]', 'ref_id': 'BIBREF1'}], 'ref_spans': [], 'section': 'Transmission model'}, {'text': 'Here T is the final day for which case counts are available (or, of concern), f = (f 0 , f 1 , f 2 ),', 'cite_spans': [], 'ref_spans': [], 'section': 'Transmission model'}, {'text': '. CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'cite_spans': [], 'ref_spans': [], 'section': '5'}, {'text': 'The copyright holder for this preprint this version posted June 16, 2020. . and θ = (u d , u r , q, D, k 1 , k 2 , ψ t ) are the remaining model parameters, which are assumed to be fixed in all of our experiments except for the simulations in Figure S6 . In this parameterization of the negative binomial distribution, the variance scales with the mean according to the dispersion parameter φ: Var[{C t }] = µ + µ 2 /φ. Values of the model parameters for our analysis using British Columbia case count data are provided in Table S1 .', 'cite_spans': [], 'ref_spans': [{'start': 243, 'end': 252, 'text': 'Figure S6', 'ref_id': None}, {'start': 523, 'end': 531, 'text': 'Table S1', 'ref_id': 'TABREF1'}], 'section': '5'}, {'text': \"The ODEs described above require parameters which can be sampled from prior distributions. In particular, the parameter R 0 , the basic reproduction number, is a fundamental determinant of such models' dynamics. In the model above,\", 'cite_spans': [], 'ref_spans': [], 'section': 'Time to pairwise model divergence'}, {'text': 'in the absence of distancing (i.e., when f = 1). We use uncertainty in R 0 to capture uncertainty in transmission and duration of infection simultaneously: given information about the duration, the transmission parameter β can simply be scaled in order to match the exponential growth phase of a dataset. The ODEs can be simulated numerically, drawing R 0 from the prior and keeping other parameters fixed. This procedure induces a distribution over the case counts seen on each day.', 'cite_spans': [], 'ref_spans': [], 'section': 'Time to pairwise model divergence'}, {'text': 'We introduce a method for determining the first date at which the case counts of two models begin to diverge. The simulated datasets for a model are treated as random samples from a distribution of prevalence in each component of the ODE. In order to determine when the effect of physical distancing first appears in a model (i.e., determining when a model diverges from a baseline model), we compute the empirical probability that there is a difference between the two models, according to the distributions of prevalence implied by the two models. In this work, we mainly consider a difference of 10 prevalent and non-quarantined cases, which correspond to I +I d in the model, to be a substantial difference, as this may be a number for which the risk of a return to community spreading is imminent. We refer to this threshold as the number of active cases (i.e., the I + I d component is the number of active cases). To compute the empirical probability that a substantial difference occurs on a given day after introducing distancing measures, we consider the proportion of simulated samples for which the model with physical distancing shows fewer infected counts than the non-distancing model by at least 10. (Alternatively, if distancing measures are being relaxed rather than introduced, then prevalent cases would increase and we consider the proportion of simulated samples for which the model without physical distancing shows more than 10 active cases.) We form this empirical probability for each day. The first day on which the empirical probability of observing a 10-case difference is at least 0.95 is our estimate for the first day at which the effect of modifying physical distancing is substantial (this is the days until threshold in our figures). In most of our experiments involving this threshold, we use a value of 10 for the threshold. In addition, we conduct an experiment in which the threshold is varied in the set 6 . CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'cite_spans': [], 'ref_spans': [], 'section': 'Time to pairwise model divergence'}, {'text': 'The copyright holder for this preprint this version posted June 16, 2020. . In predictive investigations of the effects of relaxing physical distancing, we simulate the SEIRtype model displayed in Section 2.1 forwards in time, starting from May 1st and ending on July 1st. We investigate situations in which the amount of physical distancing is modulated on May 17th (the strategy for restarting British Columbia through relaxation of physical distancing recommendations began in the middle of May [28] ) through a forcing of the ODE parameter f 2 , and in which many of the ODE parameters are modulated through grid-searches. We also consider a retrospective analysis of the introduction of physical distancing in British Columbia, assuming that physical distancing is introduced in mid-March (March 18th) by modulation of parameter f from f 0 = 1 to f 1 = 0.4 and f 1 = 0.7, and compare to a baseline model in which f remains at 1. We investigate 25% to 50% confidence intervals around the number of active case in early/mid-April, by considering 100 replicates in the ODE simulations, each with a value of R 0 drawn independently from the prior.', 'cite_spans': [{'start': 498, 'end': 502, 'text': '[28]', 'ref_id': 'BIBREF27'}], 'ref_spans': [], 'section': 'Time to pairwise model divergence'}, {'text': 'In Section 2.3, we provide some evidence that the physical distancing undertaken by people in British Columbia corresponds to a modeled value for f 1 of 0.36. We consider a baseline model in which the value of f remains at 0.36 for the duration of the time. We consider alternative relaxation models in which the value of f changes from f 1 = 0.36 to either f 2 = 0.5 (weak relaxation), f 2 = 0.65 (medium relaxation) or f 2 = 0.9 (strong relaxation) on May 17th. For each condition, we again consider 100 replicates each with a value of R 0 drawn from the prior.', 'cite_spans': [], 'ref_spans': [], 'section': 'Time to pairwise model divergence'}, {'text': 'In addition to estimating the first time at which we would expect a substantial difference in incidence to occur after a change in physical distancing, we also aim to understand the time it would take to observe such a difference in reported case counts. We use a likelihood-based approach to estimate if a change in distancing measures is yet having an effect on observed case counts, and to estimate the size of this effect.', 'cite_spans': [], 'ref_spans': [], 'section': 'Time to observe a change in the strength of distancing'}, {'text': 'The likelihood expression, as described in Section 2.1, relates predicted case counts from the SEIR-type model to reported case counts, over time and corrected for the delay between symptomonset and reporting which is assumed to have a Weibull distribution (w c ). This likelihood, written as a function of a given strength of physical distancing f X , (e.g., f 1 , the strength of distancing when it is first implemented or f 2 , the strength of distancing after relaxation) may be maximised to find f n , the maximum likelihood estimate (MLE) of f X on day n. Estimation of f n uses data from the start of the outbreak up to and including day n. Since case count data can be quite noisy, there can also be considerable variation in the day-by-day MLEs. For example, in the first few days after a change in physical distancing, a single day with a low number of identified cases 7 . CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Time to observe a change in the strength of distancing'}, {'text': 'The copyright holder for this preprint this version posted June 16, 2020. . https://doi.org/10.1101/2020.06.14.20131177 doi: medRxiv preprint can have a large impact on the MLE for f X . To mitigate the effect of this noise, we introduce a stopping rule in which an estimate f n is \"accepted\" once the MLE changes by less than 5% over a 3-day period. We use this day-by-day MLE approach to estimate the value of f 1 in the period following the introduction of distancing recommendations in British Columbia, using daily case counts reported between March 1st and April 22nd, 2020. We find a credible band by simulating the SEIR model with 50 different values of the baseline R 0 parameter, normally distributed about the mean R 0 = 2.57 estimated by a pre-distancing model fit, and with standard deviation 0.05.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'Although, as of the time of writing, we have only started to see the effects of relaxing physical distancing in British Columbia, we simulate the effects of relaxing distancing to 65% of the normal (unrelaxed) level, under the assumption that the noise and delay in observing case counts remains as during March/April in British Columbia (φ = 5, delay shape parameter 9.85, delay scale parameter 1.73 as estimated in [2] ). We also consider other levels of distancing relaxation (50%, 90%) and record the resulting impact on the number of days n to accept the MLE f n . For the credible bands, we increased the standard deviation to 0.15 to obtain a similar level of variation in the model fit as compared to the introduction-of-distancing estimation (the majority of individuals distancing causes the overall reproduction number to be lower).', 'cite_spans': [{'start': 417, 'end': 420, 'text': '[2]', 'ref_id': 'BIBREF1'}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'We also explore the impact of changing the noise/delay of case count reporting on the time to detect a change in the strength of physical distancing. We simulate outbreak trajectories for strengthening distancing under varying levels of the dispersion parameter φ. In this investigation, we control the variance in the number of daily reported cases: note that this is not the same as the dispersion parameter about R 0 , commonly referred to as k. We also control the shape and scale parameters of the onset-to-reporting Weibull distribution. We then perform the same daily MLE estimation procedure on these simulated time series. Note that this back-estimation requires the assumption that the true noise level and delay are known. If we did not have estimates of these quantities in reality, we would need to co-estimate them along with the strength of distancing, which may introduce identifiability issues or a longer time to acceptance.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'All statistical analysis was performed in R [24] , and datasets and R code are available * on GitHub under an open source license.', 'cite_spans': [{'start': 44, 'end': 48, 'text': '[24]', 'ref_id': 'BIBREF23'}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'We find that it takes 21-23 days (March 18th until April 9th or 11th) before there is a substantial difference between the baseline trajectory (the exponential growth phase of the SEIR-type model) and a trajectory with distancing measures of f 1 = 0.4, 0.7. And it takes 26 days to estimate the strength of distancing using reported data. The situation is similar for relaxing measures: it * http://github.com/carolinecolijn/ObserveEffectsDistancing 8 . CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Results'}, {'text': 'The copyright holder for this preprint this version posted June 16, 2020. . takes approximately 45 days before a substantial difference arises due to relaxing distancing from f 1 = 0.36 to f 2 = 0.65, and 23 days to detect such a difference using observed cases. Figure 2 shows rising case trajectories for which distancing is introduced, and declining ones in which distancing is relaxed, and illustrates the dependence of the timing on the severity of the change. If the change is weak (e.g., a weak relaxation from f 1 = 0.36 to f 2 = 0.5) there is no discernible difference between the trajectories before July 1 (47 days). The time until a substantial difference arises naturally depends on what is considered a substantial difference, and on the underlying uncertainty. We explore the impact of the threshold choice 9 . CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 59, 'end': 75, 'text': 'June 16, 2020. .', 'ref_id': None}], 'ref_spans': [{'start': 263, 'end': 271, 'text': 'Figure 2', 'ref_id': 'FIGREF2'}], 'section': '(which was not certified by peer review)'}, {'text': \"The copyright holder for this preprint this version posted June 16, 2020. . and strength of relaxation ( Figure 3) ; naturally, stronger relaxation (all the way to baseline contact levels with f = 1) produces a difference quickly (10-20 days) . Similarly a smaller threshold difference of 5 cases is reached relatively soon. Overall it takes 10-40 days for a substantial difference to arise. The uncertainty in R 0 and other parameters also impacts the time before there is a substantial difference in trajectories ( Figure S6 ). We find that halving uncertainty in the underlying growth rate (by reducing the standard deviation in R 0 ) can reduce the time until detection from 20-40 days to 12-25 days. Uncertainty in other fixed parameters does not have a strong impact on the time ( Figure S6 ) unless the relaxation conditions are strong (f 2 > 0.7). Epidemiological parameters are now somewhat well-established for COVID-19 [26] , but we can be less certain of the true value of more behaviour-influenced parameters (q, u r , u d ). We explicitly explore the impact of the incubation period on the time until a substantial difference arises, and find little impact ( Figure S7 ), largely because a few days' uncertainty in the incubation period is overwhelmed by the the overall time scale and threshold of 10 cases to define a substantial difference. In some instances we cannot differentiate between trajectories for the models with distancing and the models without distancing (these are the missing values in the related figures). Also, if the standard deviation in the underlying parameter R 0 is too large, a substantial difference may not be possible to predict in the time frame of interest.\", 'cite_spans': [{'start': 230, 'end': 242, 'text': '(10-20 days)', 'ref_id': None}, {'start': 930, 'end': 934, 'text': '[26]', 'ref_id': 'BIBREF25'}], 'ref_spans': [{'start': 105, 'end': 114, 'text': 'Figure 3)', 'ref_id': None}, {'start': 517, 'end': 526, 'text': 'Figure S6', 'ref_id': None}, {'start': 787, 'end': 796, 'text': 'Figure S6', 'ref_id': None}, {'start': 1173, 'end': 1182, 'text': 'Figure S7', 'ref_id': None}], 'section': '(which was not certified by peer review)'}, {'text': 'We have conceptually separated the time until there is any substantial difference in model trajectories and the time until we would be able to estimate the impact of a change in distancing, using reported case counts under conditions in British Columbia. These times may differ as a result of reporting delay, but also in some circumstances it may be sufficient to observe signal of a change in distancing under a lower probability or a lower case threshold. Figure 4 shows the daily maximum likelihood estimate of the physical distancing parameter f over time. We find that it took approximately 26 days to accept the value of f 1 (according to the methods described in Section 2.3). We estimate f 1 to be 0.22 on the day of acceptance (April 13th), but this estimate increases to 0.36 by the end of the observation period at 35 days after distancing: April 22nd. Several larger cluster outbreaks had begun to be observed in British Columbia by this time [4] , which may have contributed to this increase in the estimated distancing parameter.', 'cite_spans': [{'start': 956, 'end': 959, 'text': '[4]', 'ref_id': 'BIBREF3'}], 'ref_spans': [{'start': 459, 'end': 467, 'text': 'Figure 4', 'ref_id': 'FIGREF4'}], 'section': '(which was not certified by peer review)'}, {'text': 'In the first few days after the introduction of distancing, we see considerable noise in the daily MLEs. This is expected, as limited data observed in those first few days implies that even small variations in the case counts can have a large effect. The likelihood is flat for a period around the first week after distancing is introduced. The credible intervals are thus more informative than the point estimates. The time at which the credible interval no longer includes 1.0 may indicate the time at which we are confident that distancing has a positive effect (even if we cannot yet determine the size of this effect). We are able to observe this over a much shorter time frame: 8 days until the 95% interval drops below f 1 = 0.99 and 11 days until it drops below f 1 = 0.9, in contrast to 10 . CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. Figure 3 : The size of the threshold used to define \"substantial differences\" impacts the time until such a difference occurs, as does the severity of the change in contact rate f . The baseline model has f 1 = 0.36 (on May 17th), and so a change to f 2 = 1 is a complete relaxation of distancing measures, and is detected sooner than a partial relaxation. Each density is produced based on the assumption that R 0 is normally distributed with a mean 2.5 and a standard deviation of 0.15. We vary the threshold for the number of cases of interest in the range {5, 10, 15, 20}. Missing values on the y-axis (for example for values of the active cases level of 0.6) indicate that our methods cannot differentiate between the models with and without relaxed physical distancing when the active cases threshold is 15 or 20 cases. For each f 2 and threshold level, we simulate 10,000 ODEs with random initial R 0 values drawn independently from a normal distribution with mean 2.5 and variance 0.15, and for each group of 100 simulated ODEs, we compute the days since May 17th until the excess active cases threshold is detected between the baseline and the condition. the 26 days to accept the MLE estimate. In Figure 4b , we simulate the effects of relaxing physical distancing in British Columbia, from f 1 = 0.36 as estimated in Figure 4a to f 2 = 0.65. Under the assumption that the noise and delay in observing the true outbreak process remains constant, we estimate that it would take 23 days from initiating relaxation of distancing until accepting the MLE f n . We also performed the same analysis with f 2 = 0.9 and f 2 = 0.5, and estimated 23 days and 30 days respectively. We also used the time-dependent reproductive number [7] (R t ) estimates directly derived from reported cases, using an assumed serial interval of 5 (sd 1) days ( Figure S8 ). The time until the 95% quantile for R t drops below 1.0 is 14 days after distancing was introduced (April 1), and it takes 18, 24 days after distancing is relaxed for R t \\'s 95% quantile to be above 1.0 (f 2 = 0.9, 0.65). Even with weekly smoothing, the R t values fluctuate greatly, particularly after relaxation of distancing. In contrast, the estimates of f 1 and f 2 do not. However, they are not directly comparable to R t because we estimate a single f value over an extended time period, whereas R t is a daily (smoothed) value.', 'cite_spans': [{'start': 2692, 'end': 2695, 'text': '[7]', 'ref_id': 'BIBREF6'}], 'ref_spans': [{'start': 960, 'end': 968, 'text': 'Figure 3', 'ref_id': None}, {'start': 2167, 'end': 2176, 'text': 'Figure 4b', 'ref_id': 'FIGREF4'}, {'start': 2288, 'end': 2297, 'text': 'Figure 4a', 'ref_id': 'FIGREF4'}, {'start': 2803, 'end': 2812, 'text': 'Figure S8', 'ref_id': None}], 'section': '(which was not certified by peer review)'}, {'text': 'The time to observe the strength of distancing is impacted by both the level of noise in the 11 . CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted June 16, 2020. . https://doi.org/10.1101/2020.06.14.20131177 doi: medRxiv preprint 12 . CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted June 16, 2020. . case counts and the delay between symptom onset and reporting. We explore the effects of this in Figure 5 . As the noise in the daily case counts is reduced (Figure 5a) , so is the time to detect the strength of distancing. Under a level of noise realistic for observed case counts in BC (φ = 5), we estimated 26 days to accept f n . If φ is doubled (corresponding to halving the variance, approximately, if on the order of 50 cases are observed per day), this reduces to 21 days. Under the most optimistic scenario, where there is no noise (but when there is still an average 8.7 day delay), the time-to-accept is 15 days.', 'cite_spans': [], 'ref_spans': [{'start': 173, 'end': 181, 'text': 'Figure 5', 'ref_id': 'FIGREF5'}, {'start': 233, 'end': 244, 'text': '(Figure 5a)', 'ref_id': 'FIGREF5'}], 'section': '(which was not certified by peer review)'}, {'text': 'As would be expected, a longer delay between symptom onset and case reporting also results in a longer time taken to estimate the strength of distancing (Figure 5b) . With a Weibull distributed delay of shape 1.73 and scale 9.85 (mean 8.78, variance 27.4) and φ = 5, as estimated for British Columbia in [2] , we estimate 32 days to accept the MLE of f 1 . If the delay is reduced to shape 2 and scale 5 (mean 4.43, variance 5.37), the time to accept is 20 days. Lastly, with a delay shape 10 and scale 1 (mean 0.95, variance 0.013), the time to accept is 15 days.', 'cite_spans': [{'start': 304, 'end': 307, 'text': '[2]', 'ref_id': 'BIBREF1'}], 'ref_spans': [{'start': 153, 'end': 164, 'text': '(Figure 5b)', 'ref_id': 'FIGREF5'}], 'section': '(which was not certified by peer review)'}, {'text': 'As jurisdictions begin to ease physical distancing measures, we must understand how long it may take to observe a statistically significant difference in reported case counts. We find that it generally takes between 10-60 days before changes in distancing have a substantial, detectable impact on the reported case counts. In certain cases, we find that these methods are unable to differentiate between the relaxed and unrelaxed scenarios; for example, if the deviation in R 0 is high and the relative change in physical distancing is low. Through computing daily estimates of the parameter controlling strength of physical distancing in our model, we find that, at least under public health systems in British Columbia, the time taken to detect changes in distancing is on the order 3-4 weeks. Halving the case count variance or the mean onset-to-reporting delay in British Columbia during March and April could have reduced the time taken to understand the strength of social distancing by approximately 20% and 38%, respectively. This highlights the high potential benefit of improved, consistent surveillance systems, and perhaps contact-tracing apps, which minimise delays and e.g. weekly patterns or discrepancies in case reporting.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': \"The question of determining when two curves begin to differ has been discussed in general in other literature. Such lines of work include Simultaneous Equations Models (SEM), a field within econometrics [25] and growth rates in bacterial culture [16] . These prior works require strong loglinear and normality assumptions and cannot be easily adapted to SEIR-type models. Approaches accounting for the issue of delayed case reporting in epidemiology are often referred to as 'nowcasting' [21, 33] . These are distinct from the methodology we introduce in this work in that that they focus on inference of the true number of cases at a particular time from the observed num-13 . CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\", 'cite_spans': [{'start': 203, 'end': 207, 'text': '[25]', 'ref_id': 'BIBREF24'}, {'start': 246, 'end': 250, 'text': '[16]', 'ref_id': 'BIBREF15'}, {'start': 488, 'end': 492, 'text': '[21,', 'ref_id': 'BIBREF20'}, {'start': 493, 'end': 496, 'text': '33]', 'ref_id': 'BIBREF32'}], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'The copyright holder for this preprint this version posted June 16, 2020. Results are coloured grey before the MLE is accepted. Simulated daily case counts are also provided for reference (top panels). 14 . CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted June 16, 2020. . ber. In contrast, we work backwards from observed cases, using a model which takes observation delay and imperfect sampling into account, to understand the impact of physical distancing on COVID-19.', 'cite_spans': [{'start': 59, 'end': 75, 'text': 'June 16, 2020. .', 'ref_id': None}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'Our analysis has a number of limitations. Although policy changes happen at defined times, distancing behaviour and other COVID-19 control practices do not change instantaneously, and the time frame to detect changes may therefore be longer than we have estimated. We have not explored a wide range of growth rates or baseline prevalence levels and these may affect the results. The transmission model used is a deterministic SEIR variant, and does not include stochastic effects (except in the observation model), or age or risk structure. The model includes a fraction of the population practising physical distancing, thereby reducing their contact rates, but does not otherwise include heterogeneity in contact patterns. In particular, we focus on the case onset-toreporting delay, but certain high-risk groups such as healthcare workers may be more likely to get tested and have expedited testing available. Our methodology could readily be extended to structured models, but this requires age-stratified counts and knowledge of the mixing terms. Indeed, any disease model will include exponential growth and decay, and this work is somewhat modelagnostic in that, whatever level of detail goes into producing this exponential behaviour, we can still perform the same eventual inference.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'Our approach to determining when the effect of modifying measures is observable relies on using case count data as the indicator for increased community-based transmission. However, public health officials may find outbreaks-even where they do not contribute to statistically higher case counts-by noting epidemiological links among cases (e.g., links through workplace, family, healthcare or gatherings). Changes in these smaller outbreaks may be detected much faster than our 3 week estimates, but it remains the case that measures directed towards the general population are the main intervention for COVID-19. Our results focus on estimating impacts on this broader population level, and this has long time scales. Sentinel surveillance systems, contact tracing and outbreak detection are among the tools used by public health agencies to gather rapid and more detailed information than case counts during a disease outbreak. These form multi-faceted surveillance networks-including hospitals, primary care, symptom trackers-which may often be faster than confirmed and lab-tested cases. However, these networks also have complex limitations, and greatly vary by jurisdiction. Their data are not always consistently or widely published. Confirmed case counts remain a widely used source for population-level understanding of COVID-19 control, particular for those seeking a broad assessment of COVID-19 in multiple jurisdictions to support policy on borders and travel or to compare effectiveness of control measures.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'While we focus on data from British Columbia in this work, our methods may be applied to any region or country with case reporting to determine the relevant time lags and interpret reported cases accordingly. In addition, SEIR-type models are used to forecast COVID-19 in all 15 . CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted June 16, 2020. . 50 US states with the Covid Act Now project [15] : each state is associated with a COVID-19 risk level based on how soon their projections arrive at certain constant thresholds on measures such as case counts and ICU headroom used. Our methods could improve such region-based work by allowing the parameters and baselines to be calibrated, reflecting population density, testing protocols, demographics and cultural factors regarding social contact. The times to observe the impacts of changes in control measures are likely to be region specific.', 'cite_spans': [{'start': 120, 'end': 124, 'text': '[15]', 'ref_id': 'BIBREF14'}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'We have found in this work that the time-to-detection for a return to widespread transmission owing to relaxed physical distancing measures can be long, indeed considerably longer than than the mean incubation period [26, 14] or often used 14 day time period [5, 27, 17] . In order to decrease the time-to-detection, we need less noisy testing and faster ways to monitor community transmission. Outbreaks within communities contribute both to \"noise\" in case counts and to uncertainty in R 0 , particularly if they reveal areas of previous under-detection. It is therefore also important to maintain consistent surveillance spatially and temporally. Other surveillance techniques could facilitate faster and smoother case detection, but suffer from their own limitations. Consistent sentinel surveillance, e.g. as are seen in influenza-like-illness data, may have a slightly longer delay, but ultimately less noise. Symptom tracker apps could show changes in incidence sooner than laboratory-confirmed case counts, but could suffer from false positives and may be affected by coverage and usage limitations. Digital contact tracing may also support rapid case finding, but will likely ultimately rely on testing data for confirmation. However, case confirmation delays in a contact tracing context are often considerably shorter than through symptom-based testing [3] . Promising recent research has also investigated detection of SARS-CoV-2 in wastewater [22, 1] , which, although potentially not revealing of individual-level infection, may bypass case detection delays and provide an early warning system. Similarly, investigation of live mobility data during the disease outbreak [10, 13] may reveal changes in population behaviour, even if such work requires some assumptions about the link to changes in incidence. Given the long time frames for reported case count data it is urgent that robust combinations of diverse surveillance systems be developed, and for those seeking an overview of COVID-19 trajectories without reference to multi-faceted local surveillance data (perhaps at the national level or to support decisions about travel to and from other jurisdictions), it is important not to over-interpret short-term fluctuations in reported case counts. ', 'cite_spans': [{'start': 217, 'end': 221, 'text': '[26,', 'ref_id': 'BIBREF25'}, {'start': 222, 'end': 225, 'text': '14]', 'ref_id': 'BIBREF13'}, {'start': 259, 'end': 262, 'text': '[5,', 'ref_id': 'BIBREF4'}, {'start': 263, 'end': 266, 'text': '27,', 'ref_id': 'BIBREF26'}, {'start': 267, 'end': 270, 'text': '17]', 'ref_id': 'BIBREF16'}, {'start': 1364, 'end': 1367, 'text': '[3]', 'ref_id': 'BIBREF2'}, {'start': 1456, 'end': 1460, 'text': '[22,', 'ref_id': 'BIBREF21'}, {'start': 1461, 'end': 1463, 'text': '1]', 'ref_id': 'BIBREF0'}, {'start': 1684, 'end': 1688, 'text': '[10,', 'ref_id': 'BIBREF9'}, {'start': 1689, 'end': 1692, 'text': '13]', 'ref_id': 'BIBREF12'}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'We consider two experiments wherein the uncertainty about the ODE parameters are modulated, and record the time to detect a threshold of excess in the active cases above baseline models. In our first experiment we focus on R 0 . In the main text, we consider ODEs in which the R 0 parameter is initialized to a draw from a normal distribution with mean 2.5 and standard deviation 0.15. This reflects our broad understanding about the uncertainty in R 0 . In all conditions, the baseline indicates a situation in which physical distancing is not relaxed, and remains in effect.', 'cite_spans': [], 'ref_spans': [], 'section': 'Impact of uncertainty in model parameters'}, {'text': 'If the standard deviation is varied away from 0.15, then the days until the threshold is reached (i.e., a significant difference between the baseline and a relaxed condition is seen) is affected. The results of varying the standard deviation between 0.01 and 0.4 (considering the values in the set {0.01, 0.11, 0.21, 0.31, 0.41}) are shown in Figure S6a . In that Figure, we see that the days until a significant increase in the active cases can vary from 10 to 50 days. Missing values in the lines in this plot indicate that our methods cannot determine whether or not the condition will ever differ from the baseline: if the standard deviation of R 0 is > 0.11, we do not predict that a weak relaxation of physical distancing to f 2 = 0.6 will result in a substantial increase in active cases. However, if the standard deviation of R 0 is < 0.11 a relaxation of physical distancing to f 2 = 0.6 could lead to a significant increase in active case count after 20 or 40 days.', 'cite_spans': [], 'ref_spans': [{'start': 343, 'end': 353, 'text': 'Figure S6a', 'ref_id': None}, {'start': 364, 'end': 371, 'text': 'Figure,', 'ref_id': None}], 'section': 'Impact of uncertainty in model parameters'}, {'text': '. CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '21'}, {'text': 'The copyright holder for this preprint this version posted June 16, 2020. . https://doi.org/10.1101/2020.06.14.20131177 doi: medRxiv preprint Figure S6 : (a) Days until active cases in the relaxed model exceeds the baseline model (y-axis), with varying levels of the standard deviation for R 0 (x-axis). There are four relaxation levels (all starting relaxation on May 17th). Parameter settings are identical to other figures aside from sd.', 'cite_spans': [], 'ref_spans': [{'start': 142, 'end': 151, 'text': 'Figure S6', 'ref_id': None}], 'section': '(which was not certified by peer review)'}, {'text': 'The missing values indicate that our methods could not differentiate between the models. (b) The variables D, k 1 , k 2 , q, u d and u r are all varied between 10% and 30% around their fitted values (with the three levels described in the legend). The value that f is relaxed to is shown in the x-axis (before relaxation, f 1 is set to 0.36). So, R 0 is sampled at ODE initialization, but so too now are the 6 variables mentioned (with independent draws from the uniform distribution). For each f 2 level, 100 iterations are considered each with 100 replicates.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'In another experiment, we consider variation of the ODE parameters D, k 1 , k 2 , q, u d and u r . We vary each of these parameters around their default values (given in Table S1 ) by X%, with X varied in the set {10, 20, 30}. For each X (uncertainty) level, we consider simulations in which the 22 . CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [{'start': 170, 'end': 178, 'text': 'Table S1', 'ref_id': 'TABREF1'}], 'section': '(which was not certified by peer review)'}, {'text': \"The copyright holder for this preprint this version posted June 16, 2020. . ODE is initialized with random parameters. For the basic reproduction number, R 0 is initialized as a normal distribution with mean 2.5 and variance 0.15 (as is done in the main text), and for the parameters D, k 1 , k 2 , q, u d and u r we initialize each parameter by drawing from a uniform distribution. For each parameter, the range of the uniform distribution has a minimum value of the 'default' value of that parameter (given by Table S1 ) minus X% of the default value, and maximum value of the 'default' plus X%. We then record the days until an active case count excess of 10 is obtained in 95% of the simulations. The number of iterations and the number of replicates is identical to Figure 3 (described in the main text). Density plots for the number of days until the threshold on the excess of active case counts are shown in Figure S6b for a variety of relaxation levels. These results show that if the uncertainty about the parameters of our SEIR-type model are even 10%, then even relaxing distancing totally f 2 = 1 can result in 20 days before the threshold of 10 excess cases is reached. In addition, if social physical is relaxed only moderately (to f 2 = 0.6 for example), then it may be 70 days before an excess of 10 active cases is seen. Figure S7 : The incubation period has a limited effect on the time before a substantial difference occurs between two simulated scenarios. The initial value f 1 is 0.36. The curves are obtained under the uncertainty assumption that R 0 is normally distributed with a mean 2.5, and a standard deviation of 0.15. The incubation period is 1/k 1 + 1/k 2 . For each setting of the incubation period, we rescale k 1 and k 2 evenly from their fitted values (k 1 = 1 and k 2 = 1/5) to achieve the incubation period in days. (e.g., for an incubation period of 7 days 1/k 1 + 1/k 2 = 7, and so the values for k 1 and k 2 used are 1 * (6/7) and 1/5 * (6/7) respectively.) Missing values indicate that our methods cannot distinguish between simulations in which physical distancing is relaxed and simulations in which physical distancing is not relaxed (indicating that a relaxation of f 2 to 0.6 may not have an effect for much of the range of incubation periods considered)\", 'cite_spans': [], 'ref_spans': [{'start': 512, 'end': 520, 'text': 'Table S1', 'ref_id': 'TABREF1'}, {'start': 771, 'end': 779, 'text': 'Figure 3', 'ref_id': None}, {'start': 916, 'end': 926, 'text': 'Figure S6b', 'ref_id': None}, {'start': 1339, 'end': 1348, 'text': 'Figure S7', 'ref_id': None}], 'section': '(which was not certified by peer review)'}, {'text': 'The above results indicate that uncertainty about the model parameters substantially increases the number of days until definitive statements about the effects of relaxing social distancing can be made, with the total range of the number of days until effects are seen varying between 20 and 70.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': '. CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'cite_spans': [], 'ref_spans': [], 'section': '23'}, {'text': 'The copyright holder for this preprint this version posted June 16, 2020. .', 'cite_spans': [], 'ref_spans': [], 'section': '23'}, {'text': 'We vary the incubation period to explore the impact on the time before the active cases threshold of 10 is reached. The incubation period is defined as 1/k 1 +1/k 2 , with default values k 1 = 1/5 and k 2 = 1 throughout this work. We vary the incubation period between 6 and 10 days by modulating k 1 and k 2 appropriately (with the same scaling factor for each) and show the results in Figure S7 .', 'cite_spans': [], 'ref_spans': [{'start': 387, 'end': 396, 'text': 'Figure S7', 'ref_id': None}], 'section': 'Effect of the incubation period'}, {'text': 'For each incubation period and relaxation f 2 level, we perform 50 replicates and plot the median number of days until the active cases threshold is reached. We find that there is not much variation in days to threshold as the incubation period is varied. This suggests that the variance in active case due to uncertainty about R 0 may overwhelm variation in the incubation period over the range considered.', 'cite_spans': [], 'ref_spans': [], 'section': 'Effect of the incubation period'}, {'text': 'For comparison with the daily MLE method applied to British Columbia case count data in Figure 4 , we also perform estimation of R t using EpiEstim package [7] in R. Results are shown in Figure S8 . Note that EpiEstim assumes that all cases from the first time step are imported cases, leading to inflated R t estimates during the first week. To account for this, when finding the the earliest time that R t is less than/greater than 1.0 at the 95% level, we ignore this first week of estimation.', 'cite_spans': [{'start': 156, 'end': 159, 'text': '[7]', 'ref_id': 'BIBREF6'}], 'ref_spans': [{'start': 88, 'end': 96, 'text': 'Figure 4', 'ref_id': 'FIGREF4'}, {'start': 187, 'end': 196, 'text': 'Figure S8', 'ref_id': None}], 'section': 'Estimation of R t'}, {'text': '. CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'cite_spans': [], 'ref_spans': [], 'section': '24'}, {'text': 'The copyright holder for this preprint this version posted June 16, 2020. Figure S8 : Median estimates of the time-dependent reproduction number R t in British Columbia, using a weekly sliding window and assumed serial interval of 5 (sd 1) days. Top panel: after implementing distancing, using true observed case counts (14 days until R t < 1 at the 95% level). Bottom panel: after relaxing distancing, using simulated data (18, 25 days until R t > 1 at the 95% level for f 2 = 0.9, 0.65, respectively), under the assumption that f 1 = 0.36 and observation noise and delay remain as pre-relaxation during March/April in BC. Coloured bands correspond to 95% quantiles.', 'cite_spans': [], 'ref_spans': [{'start': 74, 'end': 83, 'text': 'Figure S8', 'ref_id': None}], 'section': '24'}, {'text': '. CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'cite_spans': [], 'ref_spans': [], 'section': '25'}, {'text': 'The copyright holder for this preprint this version posted June 16, 2020. . https://doi.org/10.1101/2020.06.14.20131177 doi: medRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': '25'}], 'paragraphs': [{'text': 'For each incubation period and relaxation f 2 level, we perform 50 replicates and plot the median number of days until the active cases threshold is reached. We find that there is not much variation in days to threshold as the incubation period is varied. This suggests that the variance in active case due to uncertainty about R 0 may overwhelm variation in the incubation period over the range considered.', 'id': '00054'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 10.934499740600586}, {'rank': 2, 'paragraphs': [{'text': 'We vary the incubation period to explore the impact on the time before the active cases threshold of 10 is reached. The incubation period is defined as 1/k 1 +1/k 2 , with default values k 1 = 1/5 and k 2 = 1 throughout this work. We vary the incubation period between 6 and 10 days by modulating k 1 and k 2 appropriately (with the same scaling factor for each) and show the results in Figure S7 .', 'id': '00053'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 10.873900413513184}, {'rank': 3, 'paragraphs': [{'text': \"The copyright holder for this preprint this version posted June 16, 2020. . ODE is initialized with random parameters. For the basic reproduction number, R 0 is initialized as a normal distribution with mean 2.5 and variance 0.15 (as is done in the main text), and for the parameters D, k 1 , k 2 , q, u d and u r we initialize each parameter by drawing from a uniform distribution. For each parameter, the range of the uniform distribution has a minimum value of the 'default' value of that parameter (given by Table S1 ) minus X% of the default value, and maximum value of the 'default' plus X%. We then record the days until an active case count excess of 10 is obtained in 95% of the simulations. The number of iterations and the number of replicates is identical to Figure 3 (described in the main text). Density plots for the number of days until the threshold on the excess of active case counts are shown in Figure S6b for a variety of relaxation levels. These results show that if the uncertainty about the parameters of our SEIR-type model are even 10%, then even relaxing distancing totally f 2 = 1 can result in 20 days before the threshold of 10 excess cases is reached. In addition, if social physical is relaxed only moderately (to f 2 = 0.6 for example), then it may be 70 days before an excess of 10 active cases is seen. Figure S7 : The incubation period has a limited effect on the time before a substantial difference occurs between two simulated scenarios. The initial value f 1 is 0.36. The curves are obtained under the uncertainty assumption that R 0 is normally distributed with a mean 2.5, and a standard deviation of 0.15. The incubation period is 1/k 1 + 1/k 2 . For each setting of the incubation period, we rescale k 1 and k 2 evenly from their fitted values (k 1 = 1 and k 2 = 1/5) to achieve the incubation period in days. (e.g., for an incubation period of 7 days 1/k 1 + 1/k 2 = 7, and so the values for k 1 and k 2 used are 1 * (6/7) and 1/5 * (6/7) respectively.) Missing values indicate that our methods cannot distinguish between simulations in which physical distancing is relaxed and simulations in which physical distancing is not relaxed (indicating that a relaxation of f 2 to 0.6 may not have an effect for much of the range of incubation periods considered)\", 'id': '00049'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 10.631799697875977}, {'rank': 4, 'abstract': '', 'body_text': [{'text': 'Dear Editor,', 'cite_spans': [], 'section': '', 'ref_spans': []}, {'text': 'We read with great interest the recent publication entitled “A Chinese case of coronavirus disease 2019 (COVID-19) did not show infectivity during the incubation period: based on an epidemiological survey” [1]. Bae concluded that “the epidemiological findings support the claim that the COVID-19 virus does not have infectivity during the incubation period [1].” In fact, a pathogen should not have infectivity during the incubation period or development of disease. However, the exact incubation period of COVID-19 is still unknown. In a recent report by Linton et al. [2], the incubation range was estimated as between 2 days and 14 days with 95% confidence. Exceptional cases might occur with an unusually short or long incubation period. Nevertheless, the reliability of history-taking should also be addressed. How the author was able to confirm the reliability of the patient’s self-reported history is an interesting issue for further discussion. In our country, Thailand, it is not uncommon for patients to disguise their clinical history, which can cause delays in the diagnosis of COVID-19 and exacerbate the local transmission of the disease.', 'cite_spans': [{'start': 207, 'end': 208, 'mention': '1', 'ref_id': 'BIBREF0'}, {'start': 358, 'end': 359, 'mention': '1', 'ref_id': 'BIBREF0'}, {'start': 571, 'end': 572, 'mention': '2', 'ref_id': 'BIBREF1'}], 'section': '', 'ref_spans': []}], 'paragraphs': [{'text': 'We read with great interest the recent publication entitled “A Chinese case of coronavirus disease 2019 (COVID-19) did not show infectivity during the incubation period: based on an epidemiological survey” [1]. Bae concluded that “the epidemiological findings support the claim that the COVID-19 virus does not have infectivity during the incubation period [1].” In fact, a pathogen should not have infectivity during the incubation period or development of disease. However, the exact incubation period of COVID-19 is still unknown. In a recent report by Linton et al. [2], the incubation range was estimated as between 2 days and 14 days with 95% confidence. Exceptional cases might occur with an unusually short or long incubation period. Nevertheless, the reliability of history-taking should also be addressed. How the author was able to confirm the reliability of the patient’s self-reported history is an interesting issue for further discussion. In our country, Thailand, it is not uncommon for patients to disguise their clinical history, which can cause delays in the diagnosis of COVID-19 and exacerbate the local transmission of the disease.', 'id': '00002'}], 'title': 'Letter to the Editor: Coronavirus Disease 2019 (COVID-19), Infectivity, and the Incubation Period', 'sha': 'f4a58859fbef59081909da083e09f61a1834d8e2', 'doi': '10.3961/jpmph.20.065', 'score': 10.556900024414062}, {'rank': 5, 'abstract': \"Objective: to undertake a review and critical appraisal of all published/preprint reports that offer an estimation of incubation periods for novel coronavirus (COVID-19). Design: a rapid and systematic review/critical appraisal Data sources: COVID-19 Open Research Dataset supplied by Georgetown's Centre for Security and Emerging Technology as well as PubMed and Embase via Arxiv, medRxiv, and bioRxiv. Results: screening was undertaken 44,000 articles with a final selection of 25 studies referring to 18 different experimental projects related to the estimation of the incubation period of COVID-19. Findings: The majority of extant published estimates offer empirical evidence showing that the incubation period for the virus is a mean of 7.8 days, with a median of 5.01 days, which falls into the ranges proposed by the WHO (0 to 14 days) and the ECDC (2 to 12 days). Nevertheless, a number of authors proposed that quarantine time should be a minimum of 14 days and that for estimates of mortality risks a median time delay of 13 days between illness and mortality should be under consideration. It is unclear as to whether any correlation exists between the age of patients and the length of time they incubate the virus. Finally, it is generally agreed that robust precautions must be put in place for the prevention and/or mitigation of asymptomatic transmission to high-risk patients caused by those incubating the virus.\", 'body_text': [{'text': 'At the start of 2020, a new form of coronavirus was found to be the source of infection responsible for an epidemic of viral pneumonia in Wuhan, China, a region in which the first patients began to show symptoms in December 2019. At the time of writing (April 29, 2020) over 3 million people globally have caught the virus, of whom more than 200,000 have died. The novel virus, causing severe acute respiratory disease, is thought to be from the same family as Middle East Respiratory Syndrome (MERS) coronavirus and Severe Acute Respiratory Syndrome (SARS) coronavirus, but it is unique in its own right. This means that central epidemiological parameters, which includes the incubation period, are being urgently researched in real-time from case reports while the epidemic is continuing [1] . The incubation period of a virus represents the time span from the probable earliest contact with a source of transmission and the earliest recognition of the first symptoms. Accurately estimating the length of incubation period is essential for effective contemporary public health measures to be taken [2] . If health authorities know what the incubation period is then they will know for how long a healthy individual has to be monitored and have their movement restricted (quarantine period) [3] . Correctly estimating the incubation period will also help us to comprehend how infectious COVID-19 is, make estimations of the size of the pandemic, and decide on the best course of action [4, 5, 6, 7] . With insufficient data available to definitively state what the incubation period for this virus is, the World Health Organisation (WHO) is working with a broad range of 0-14 days, the European Centre for Disease Prevention and Control (CDCDC) is working with a range of 2-14 days, and a number of studies have made the assumption that the incubation period is similar to that of the MERS and SARS coronaviruses [1] .', 'cite_spans': [{'start': 790, 'end': 793, 'text': '[1]', 'ref_id': None}, {'start': 1100, 'end': 1103, 'text': '[2]', 'ref_id': None}, {'start': 1292, 'end': 1295, 'text': '[3]', 'ref_id': 'BIBREF0'}, {'start': 1487, 'end': 1490, 'text': '[4,', 'ref_id': 'BIBREF1'}, {'start': 1491, 'end': 1493, 'text': '5,', 'ref_id': 'BIBREF2'}, {'start': 1494, 'end': 1496, 'text': '6,', 'ref_id': 'BIBREF3'}, {'start': 1497, 'end': 1499, 'text': '7]', 'ref_id': 'BIBREF4'}, {'start': 1914, 'end': 1917, 'text': '[1]', 'ref_id': None}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'Various infectious/viral diseases have a variety of incubation periods. Nevertheless, for some infectious diseases, we are relatively certain about the incubation period. For every individual the level of pathogens invading the body, their ability to reproduce and resist treatment will differ, and so for any specific disease the data related to incubation periods should be treated with logarithm normal distribution [8] . Log-normal distribution represents the continuous probability distribution for a random variable with normal logarithm distribution. Incubation periods can usually be measured via biological experimentation and physiological observation. Accurately determining the incubation period will significantly influence controls to prevent transmission of the disease and official policy regarding it. Nevertheless, determining the incubation period for COVID-19 is no simple matter, due to the fact that there is no consistency in the quality of the available data. One reason for this is that generally we can only discover the times when the patient was in contact with persons carrying the virus, and then assume that the incubation period runs from the earliest date of exposure to the appearance of clinical symptoms or medical diagnosis. This way of calculating the incubation period may well be responsible for overestimation. This paper represents a systematic review of the literature in order to answer the essential question of what length the COVID-19 incubation period is. Due to the fact that no method currently exists that can make an accurate estimation of the incubation period, the only option is to draw lessons from past experience/practice.', 'cite_spans': [{'start': 419, 'end': 422, 'text': '[8]', 'ref_id': 'BIBREF5'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'A search was run with the \"COVID-19 Open Research Base set\" provided by Georgetown\\'s Centre for Security and Emerging Technology. This dataset can be found at https://cset.georgetown.edu/covid-19-open-research-datasetcord-19/. It comprises more than 55,000 academic articles, and more than 44,000 of them have some reference to COVID-19, SARS-CoV-2, and other forms of coronavirus. This dataset is openly available to the global research community to employ using new techniques in Natural Language Processing (NLP) and other forms of Artificial Intelligence for the generation of novel insights supporting the continuing battle with the pandemic. This dataset is regularly updated when new research appears in peer-reviewed publications and archive services, e.g. bioRxiv, medRxiv, et cetera. This search was undertaken on April 25, 2020.', 'cite_spans': [], 'ref_spans': [], 'section': 'Methodology'}, {'text': 'For reviewing every article in the dataset we employed the Bidirectional Encoder Representations from Transformers for Biomedical Text Mining (BioBERT) model [9] , which is a biomedical language representation model created to assist in mining biomedical texts. The inspiration for the model comes from the pre-trained language model BERT [10] created by Devlin J.et al. BioBERT resolves the difficulties caused by moving from a trained corpus for general use to biomedical use and assists in the understanding of complex biomedical texts as are found with the work on COVID-19. The model was taken from the Github https://github.com/dmis-lab/biobert . Filtering of the articles was then undertaken using keywords and questions, e.g. \"what is the incubation period of COVID/normal coronavirus/SARS-CoV-2/nCoV\".', 'cite_spans': [{'start': 158, 'end': 161, 'text': '[9]', 'ref_id': 'BIBREF6'}, {'start': 339, 'end': 343, 'text': '[10]', 'ref_id': 'BIBREF7'}], 'ref_spans': [], 'section': 'Methodology'}, {'text': 'As illustrated in Figure 1 , a systematic approach was used to screen the full \"COVID-19 Open Research Dataset\". Through employing BioBERT, just 25 articles were found that scored highly on confidence rating. An additional search was run employing Google Scholar to make sure that every relevant article was included. Every paper previously found was in the top 29 results alongside four other articles. Each of the 29 papers have been read thoroughly, causing nine articles to be excluded from the list for the following reasons: one was simply a report of earlier studies into incubation periods, one related to techniques for estimating incubation periods, two were letters to editors, one was a literature review regarding the genometric features of SARS-CoV-2, one was a perspective paper, and the other four were either irrelevant or redundant. The ultimate list of 18 articles was split into five categories, with four articles focusing on the study of incubation periods, seven on characteristics of transmission, two on clinical characteristics, four case studies, and one last paper related to epidemiological characteristics. ', 'cite_spans': [], 'ref_spans': [{'start': 18, 'end': 26, 'text': 'Figure 1', 'ref_id': 'FIGREF0'}], 'section': 'Results'}, {'text': 'Backer J.et al [1) employed travel histories and symptom onsets for 88 confirmed cases discovered beyond the boundaries of Wuhan, China, in the early stage of the coronavirus outbreak. These authors made an estimation that the incubation period ranged from 2.1 to 11.1 days (2.5 th to 97.5 th percentile) and that the mean incubation period was 6.4 days (95% CI: 5.6-7.7). This research offers empirical evidence and falls into the previously mentioned incubation periods estimated by both the ECDC and the WHO [11]. The researchers employed three parametric forms related to the distribution of the incubation period: lognormal distribution, gamma distribution, and the Weibull distribution. They employed uniform prior probability distribution for the exposure interval related to the point of infection for all 88 individuals. The researchers used posterior distribution samples employing the RStan package within R software version 3.6.0 (R Foundation, Vienna, Austria) [12] .', 'cite_spans': [{'start': 974, 'end': 978, 'text': '[12]', 'ref_id': 'BIBREF9'}], 'ref_spans': [], 'section': 'Estimating the incubation period'}, {'text': \"Natalie ML. Et al [2] examined COVID-19's epidemiological characteristics and incubation period. The researchers harvested information relating to confirmed diagnoses of COVID-19 infection beyond the disease epicentre in Wuhan, China, using official reporting from state institutes and reporting on mortalities both within and outwith Wuhan. The data used by the authors was either directly harvested from government sources or from news websites reporting government statements. The data collection process was real-time, so it was added to as further details emerged. The final data selection represented a selection of reported cases up to January 31, 2020. The outcomes of this research concluded that the incubation period falls into a range of 2-14 days (95% CI), with the mean being approximately five days as found by employing best-fit lognormal distribution. Mean time between onset of symptoms and admission to hospital (either for treatment or isolation) was estimated to be between three and four days with no truncation and between five and nine days with right truncation. On the basis of the 95 th percentile estimate for the incubation period, the researchers recommended that exposed individuals should be quarantined for a minimum of 14 days. When making estimates of the risk of fatality in . CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)\", 'cite_spans': [], 'ref_spans': [], 'section': 'Estimating the incubation period'}, {'text': 'The copyright holder for this preprint this version posted May 23, 2020. . https://doi.org/10.1101/2020.05.20.20108340 doi: medRxiv preprint COVID-19 cases, the median time delay between illness onset and death of 13 days (17 days with right truncation) should be taken into account.', 'cite_spans': [], 'ref_spans': [], 'section': 'Estimating the incubation period'}, {'text': 'Jiang X et al [13] undertook research making a comparison between incubation periods for MERS, SARS, and SARS-CoV-2. The researchers reported that SARS-CoV-2 has an extended incubation period, which has led to modifications in official policy for control and screening. To prevent the virus spreading, any individual who may have been exposed should go into isolation for 14 days, this being the outer limit of predictions for incubation times. Nevertheless, by analyzing a large dataset for this research, researchers report that no identifiable difference exists between incubation times for SARS-CoV-2, severe acute respiratory syndrome coronavirus (SARS-CoV), and the Middle East respiratory syndrome coronavirus (MERS-CoV), which highlights the requirement for more extensive and better-annotated datasets. This research covered 49 patients with SARS-CoV-2 who had definite dates for first exposure, end of exposure and beginning of symptoms, 153 patients with SARS-CoV, and 70 MERS-CoV patients; this data was amalgamated from seven separate papers. The results indicated that MERS incubates on average 5.8 days (95% CI: 5-6.5), SARS-CoV 4.7 days (95% CI: 4.3-5.1), and SARS-CoV2 4.9 days (95% CI: 4.4-5.5). This demonstrates that the longest incubation period is MERS-CoV, with SARS-CoV2 second longest.', 'cite_spans': [{'start': 14, 'end': 18, 'text': '[13]', 'ref_id': 'BIBREF10'}], 'ref_spans': [], 'section': 'Estimating the incubation period'}, {'text': \"Lauer Stephen et al [14] researched the COVID-19 incubation period by looking at diagnosed cases that have been publicly reported. The aim of the study was to ascertain COVID-19's incubation period and to detail its implications for public health. The researchers examined diagnosed cases of COVID-19 occurring between January 4, 2020, and February 24, 2020. The research covered 181 subjects diagnosed with SARS-CoV-2 infection outwith Hubei province, China by examining press releases and news reports from 50 different provinces, regions, and nations. The researchers harvested information regarding patient demographics, dates/time of possible exposure, onset of symptoms, onset of fever, and admission to hospital. The researchers estimate that, conservatively, 101/10,000 cases (99 th percentile, 482) will experience symptom onset more than 14 days after being quarantined or actively monitored. Nevertheless, the researchers noted that severe cases could be overrepresented in public reporting; it is possible that severe and mild COVID 19 infections have different incubation periods. This research adds to the evidence that COVID-19 is similar to SARS in having a median incubation period of around five days. This recommendation comes from the research looking at proposed quarantine/active monitoring times for subjects with potential exposure to the virus.\", 'cite_spans': [{'start': 20, 'end': 24, 'text': '[14]', 'ref_id': 'BIBREF11'}], 'ref_spans': [], 'section': 'Estimating the incubation period'}, {'text': \"Li Q. et al [15] researched the early data regarding transmission dynamics for the virus in Wuhan, estimating the mean incubation period at 5.2 days (95% CI: 4.1-7.0), with the distribution's 95 th percentile being 12.5 days. The researchers fitted a log-normal distribution to data regarding history of exposure and date of onset using only those cases where detailed information was available to estimate the length of incubation. Their preliminary estimate of distribution of the incubation period offers strong support for the case that exposed subjects should be quarantined or put under medical observation for 14 days. Nevertheless, this study's accuracy may be questioned as the estimate was made using data from just 10 patients.\", 'cite_spans': [{'start': 12, 'end': 16, 'text': '[15]', 'ref_id': 'BIBREF12'}], 'ref_spans': [], 'section': 'Transmission Characteristics'}, {'text': \"Meili L. Et al [16] researched the transmission characteristics of China's COVID-19 outbreak. They took individual patient histories from COVID-19 subjects in China (not from Hubei Province) for estimating the distribution of the time for generation, incubation, and the time span between onset of symptoms and isolation/diagnosis. On average, patients were isolated 3.7 days after the onset of symptoms, and diagnosed after 6.6 days. The average patient was found to be infectious 3.9 days prior to displaying any notable symptoms. This militates against effective quarantining or contact tracing. With contact tracing, isolation, and quarantine, the baseline reproduction number is estimated at 1.54, with the majority of infection attributable to super spreaders. The authors of this research, on this basis, suggested that 14 day quarantine periods would fail for 6.7% of subjects; they proposed 22day quarantine becoming standard. This research gave an estimation of mean incubation period as 7.2 days. Lauren C.T.et al [17] researched the estimations for transmission intervals for COVID-19. The researchers used transmission clusters for estimations of serial interval distribution and incubation period. They used information on outbreaks in Tianjin, China (between January 21 and February 27) and Singapore (between January 19 and   1  2  3  4  5  6  7  8  9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41  42  43  44  45  46  47  48  49  50  51  52  53  54  55  56  57  58  59  60  61  62  63  64  65 . CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\", 'cite_spans': [{'start': 15, 'end': 19, 'text': '[16]', 'ref_id': 'BIBREF13'}, {'start': 1025, 'end': 1029, 'text': '[17]', 'ref_id': 'BIBREF14'}], 'ref_spans': [{'start': 1316, 'end': 1591, 'text': '(between January 19 and   1  2  3  4  5  6  7  8  9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41  42  43  44  45  46  47  48  49  50  51  52  53  54  55  56  57  58  59  60  61  62  63  64  65', 'ref_id': None}], 'section': 'Transmission Characteristics'}, {'text': 'The copyright holder for this preprint this version posted May 23, 2020. . February 26). Interval censoring was employed (R package icenReg [18] ) for making parametric estimations of the distribution of the incubation period. Mean incubation periods for Tianjin were estimated at nine days (7.92, 10.2) and for Singapore 7.1 days (6113, 8.25) . It was additionally recorded by the researchers that in both datasets cases that occurred earlier showed shorter incubation periods.', 'cite_spans': [{'start': 140, 'end': 144, 'text': '[18]', 'ref_id': 'BIBREF15'}, {'start': 338, 'end': 343, 'text': '8.25)', 'ref_id': None}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': \"China Chengfeng Q. et al [19] undertook research into the virus' transmission and clinical characteristics. This research comprised contact investigation involving 104 patients in special hospitals in Hunan province from January 22, 2020 and February 12, 2020. Collection and analysis was made of information regarding patient demographics, clinical, laboratory, and radiological findings, medication administered, and patient outcomes. Confirmation of patient illness was made with the PCR test. The patients had a mean age of 43 (ranging from 8 to 84), with 52.88% female. The researchers found a median incubation period of six (range 1-32) days; eight patients incubated between 14 and 17 days, and eight patients incubated between 18 and 30 days.\", 'cite_spans': [{'start': 25, 'end': 29, 'text': '[19]', 'ref_id': 'BIBREF16'}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'Yan Bei et al [20] undertook research into cases where it was assumed that the virus had been passed by an asymptomatic carrier. This research looked at a family from Anyang, China, of which five members were suffering COVID-19 pneumonia who had, prior to developing symptoms, been in contact with an asymptomatic member of the family who had travelled to see them from Wuhan, the origin of the pandemic. The timeline they uncovered implies that the coronavirus could have been passed on by this asymptomatic carrier. The first patient to develop symptoms incubated for 19 days, a long period but one which falls within the reported range (0 to 24 days). This patient initially produced a negative return for the RT-PCR test; RT-PCR is a common test for diagnostic virology and does not often return false positives, so her second result from this test was probably not a false positive, and so it was assumed that she was infected with the coronavirus that is responsible for COVID-19.', 'cite_spans': [{'start': 14, 'end': 18, 'text': '[20]', 'ref_id': 'BIBREF17'}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': \"Wei Xia et al [21] undertook research into how the coronavirus was transmitted in incubation periods in 2019. The researchers harvested data on the demographics, possible exposure, and time to symptom onset for confirmed cases published by local Chinese authorities. They assessed the possibilities of transmission in the course of the incubation period for 50 clusters of infection; these included 124 cases outwith Wuhan/Hubei province. Every secondary case examined and been in contact with a first-generation case prior to experiencing symptoms. This research found that the mean incubation period for COVID-19 was 4.9 days (95% CI, 4.4-5.4), with a range of 0.8 to 11.1 days (2.5 th to 97.5 th percentile). The infectious curve demonstrated that 73% of secondary cases became infected prior to symptoms appearing for first-generation cases; this was especially the case in the final three days of incubation. These findings demonstrated that COVID-19 is transmitted between those who are in close contact in the course of incubation, which could indicate a weakness in the quarantine system. Robust workable countermeasures are required for the prevention or mitigation of the virus being transmitted asymptomatically amongst high-risk populations in the course of the incubation period. Juanjuan Z. [22] undertook research into the evolution of COVID-19's transmission dynamics and epidemiology outwith Hubei province. It was hoped that being able to understand the evolution of the transmission dynamics and epidemiology outwith the center of the epidemic would provide useful information that could be used as guidance for intervention policies. The researchers harvested data on individuals whose diagnosis was confirmed by laboratory testing in mainland China (apart from Hubei) that was reported by official public sources between January 19 and February 17, 2020. The date of the fourth time the case definition was revised (January 27) was employed as a means of dividing the epidemic into two phases (December 24-January 27 and January 28-February 17) as dates for symptom onset. Trends were estimated in terms of central time-event periods and subject demographics. This research encompassed 8579 cases, covering 30 provinces. The median age of the subjects was 44 years (range 33 to 56); as the epidemic continued, more cases emerged in the younger age groups and for the elderly (those aged over 64). Mean time between symptom onset and hospitalization fell from 4.4 days (% 95% CI 0.0-14.0) between December 24 and January 27, down to 2.6 days (0.0-9.0) between January 28 and February 17. For the whole period, mean incubation period was calculated at 5.2 days (1. 8-12.4 ) with the mean serial interval being 5.1 days (1.3-11.6).\", 'cite_spans': [{'start': 14, 'end': 18, 'text': '[21]', 'ref_id': 'BIBREF18'}, {'start': 1305, 'end': 1309, 'text': '[22]', 'ref_id': 'BIBREF19'}, {'start': 2684, 'end': 2690, 'text': '8-12.4', 'ref_id': None}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'Shaoqing et al [23] undertook research investigating the clinical characteristics and outcomes for patients submitting to surgery in the COVID-19 incubation period. The researchers undertook analysis of clinical data for 34 subjects submitting to elective surgery during the COVID-19 incubation period at four Chinese hospitals (Renmin, Tongji, Zhongnan, and Central) in Wuhan between January 1 and February 5, 2020. The patients had a median age of 55, with 20 of them (58.8%) being female. Every patient exhibited COVID-19 pneumonia symptoms within a short time of surgery, with abnormalities showing on chest CT scans. Symptoms exhibited by these patients encompassed fever (31 (91.2%)), fatigue (25 (73.5%)) and a dry cough (18 (52.9%)). 15 of the patients (44.1%) had to be admitted to the intensive care unit (ICU) as the disease progressed, and seven of these died (20.5%). Every patient examined in this research had being directly exposed to the environment in Wuhan before being admitted to hospital; no patient had exhibited any symptoms of COVID-19 prior to surgery. It was notable how swiftly the COVID-19 symptoms appeared once surgery had been completed, with the infection being confirmed by laboratory within a short period. The time gap between hospital admission and surgery (median time 2.5 days) is less than the median incubation period of 5.2 days found in patients with laboratory-confirmed infections in Wuhan [15] ; it is also less than the general incubation time in hospitals in China (median time 4.0 days, from research into infected patients from 552 Chinese hospitals [19] ). Taken together, this evidence is confirmation of the hypothesis that patients within this research were incubating COVID-19 prior to submitting to surgery.', 'cite_spans': [{'start': 15, 'end': 19, 'text': '[23]', 'ref_id': 'BIBREF20'}, {'start': 1435, 'end': 1439, 'text': '[15]', 'ref_id': 'BIBREF12'}, {'start': 1600, 'end': 1604, 'text': '[19]', 'ref_id': 'BIBREF16'}], 'ref_spans': [], 'section': 'Clinical Characteristics'}, {'text': \"Guan et al [24] undertook research into coronavirus' clinical characteristics within China. They reviewed data for 1099 patients who had a COVID-19 diagnosis confirmed by laboratory; data came from 552 hospitals across 32 provinces and municipalities of mainland China up to January 29, 2020. Only data from 291 patients was used to calculate incubation periods, these being patients who had a clear idea of the date they had been exposed. The patients had a median age of 47 years, with 58.1% male and 41.9% female. 1.18% of the subjects had been in direct wildlife contact, 31.30% had visited Wuhan, and 71.80% had been in contact with people from Wuhan. There was found to be a median incubation period of four days (interquartile range between two and seven days).\", 'cite_spans': [{'start': 11, 'end': 15, 'text': '[24]', 'ref_id': 'BIBREF21'}], 'ref_spans': [], 'section': 'Clinical Characteristics'}, {'text': 'Jasper F.C. et al [25] looked at a cluster of pneumonia within a family associated with COVID-19. The researchers examined clinical, laboratory, epidemiological, microbiological, and radiological outcomes for five patients (aged between 36 and 66 years) from the same family who all manifested idiopathic pneumonia upon their return to Shenzen, Guangdong province, having visited Wuhan from December 29, 2019 to January 4, 2020. All patients exhibited at least one and sometimes more of the symptoms of diarrhoea, upper/lower respiratory tract problems, or fever, within 3 to 6 days after being exposed. The patients attended hospital between six and ten days from symptoms appearing. The findings of this research accord with other accounts of COVID-19 transmission in family/hospital environments, and share features with reports of travellers with the infection in other areas. Jin-Wei et al [26] undertook research into 102 cases of COVID-19 (51% male, 49% female) in Xiangyang, China, all of whom tested positive via RT-PCR. The cohort ranged in age from 1.5 years to 90 years, with a mean age of 50.38. The majority of subjects were aged between 50 and 70 years. Of the 71 subjects who had confirmed contact histories, 7 lived in Wuhan, 37 had travelled there, 4 had contact with patients who had been diagnosed with the virus, and 23 were members of families where infection clusters appeared. Analysing 44 subjects where it was obvious where contact occurred, incubation periods were between 1 and 20 days, with the mean being 8.09 days (4.99). Severe illness/death rates were lower than average, with certain patients incubating the virus for longer periods than would be predicted. The researchers made a recommendation that quarantine periods should be extended to 3 weeks where appropriate.', 'cite_spans': [{'start': 18, 'end': 22, 'text': '[25]', 'ref_id': 'BIBREF22'}, {'start': 895, 'end': 899, 'text': '[26]', 'ref_id': 'BIBREF23'}], 'ref_spans': [], 'section': 'Case studies'}, {'text': 'Yuanyuan H. l [27] undertook research into COVID-19 patients who were long-term users of glucocorticoids. Across the world, clusters of patients having COVID-19 have been reported, with research demonstrating that person-toperson transmission is the chief route of infection. The research states that average incubation periods are between 2 and 14 days, with between 3 and 7 days being the most common. The research notes that incubation periods may be extended in some patients. The research reports on a family cluster of COVID-19; a 47-year-old female member of the family on long-term glucocorticoid therapy had no symptoms in a 14 day quarantine period but tested positive for COVID-19 antibodies 40 days after leaving Wuhan. These findings imply that long-term glucocorticoid use could be responsible for long incubation periods, atypical infection, and additional transmissions of the virus.', 'cite_spans': [{'start': 14, 'end': 18, 'text': '[27]', 'ref_id': 'BIBREF24'}], 'ref_spans': [], 'section': 'Case studies'}, {'text': \"Rachael P.F et al [28] undertook an investigation into Singapore's surveillance/response measures. This research examined three COVID-19 clusters associated with a church in Singapore, a business conference, and a visiting Chinese tour group in February 2020. The research employed inpatient medical records and interviews to harvest clinical and epidemiological data regarding subjects with a confirmed diagnosis of COVID-19; field investigations were undertaken for assessment of the ways these subjects had interacted with others and the potential ways in which they had acquired the virus. With overseas cases, open-source reports were used. At the time of the research (February 15, 2020) there were 36 infections with epidemiological links to the three Singaporean clusters mentioned above. 425 subjects who had had close contact with people from these clusters were put in quarantine. Those affected had generally had prolonged direct close contact with others, although it was not possible to exclude indirect transmission, e.g. through shared food or fomites. This research found a median incubation period for COVID-19 of four days (IQR 3-6). Serial intervals for transmission pairs were between three and eight days. The researchers came to the conclusion that the virus can be transmitted within communities and that local clusters of infection will appear in countries that welcomed high numbers of Chinese visitors prior to Wuhan being locked down and travel restrictions being imposed. The research further concluded that contact tracing and increased surveillance is necessary to prevent the virus spreading widely across communities. China Pei W. et al [29] undertook an investigation into COVID-19's epidemiological characteristics as found in the Chinese province of Henan, centered on publicly available data related to 1212 patients. The report mentions that these patients are 55% male and 45% female, with 81% being aged from 21 to 60 years. Statistical analysis undertaken with 483 patients on this cohort revealed that the estimated average median period was 7.4 days, the mode 4 days, and the median 7 days. 92% of patients did not have an incubation period in excess of 14 days. Table 1 summarises all of the articles under review/analysis, including the methodology employed for estimation of incubation periods. Table 1 : overview of articles investigating and reporting on the incubation period of COVID-19.\", 'cite_spans': [{'start': 18, 'end': 22, 'text': '[28]', 'ref_id': 'BIBREF25'}, {'start': 1651, 'end': 1674, 'text': 'China Pei W. et al [29]', 'ref_id': None}], 'ref_spans': [{'start': 2206, 'end': 2213, 'text': 'Table 1', 'ref_id': None}, {'start': 2341, 'end': 2348, 'text': 'Table 1', 'ref_id': None}], 'section': 'Case studies'}, {'text': 'Method for estimating the incubation period. 1  2  3  4  5  6  7  8  9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41  42  43  44  45  46  47  48  49  50  51  52  53  54  55  56  57  58  59  60  61  62  63  64  65 . CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [{'start': 45, 'end': 294, 'text': '1  2  3  4  5  6  7  8  9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41  42  43  44  45  46  47  48  49  50  51  52  53  54  55  56  57  58  59  60  61  62  63  64  65', 'ref_id': None}], 'section': 'Ref. Incubation Period'}, {'text': 'The copyright holder for this preprint this version posted May 23, 2020. ', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'Throughout human history epidemics have been a disrupter of human civilizations and caused staggering amounts of mortality and illness for both humans and animals [30] . 1  2  3  4  5  6  7  8  9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41  42  43  44  45  46  47  48  49  50  51  52  53  54  55  56  57  58  59  60  61  62  63  64  65 . CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 163, 'end': 167, 'text': '[30]', 'ref_id': 'BIBREF27'}], 'ref_spans': [{'start': 170, 'end': 419, 'text': '1  2  3  4  5  6  7  8  9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41  42  43  44  45  46  47  48  49  50  51  52  53  54  55  56  57  58  59  60  61  62  63  64  65', 'ref_id': None}], 'section': 'Discussion/conclusion'}, {'text': 'The copyright holder for this preprint this version posted May 23, 2020. .', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': \"On the basis of this research, it appears that the only means of estimating the incubation period for COVID-19 at present is logarithm normal distribution. Novel ways of effecting improvements to the estimation's accuracy in terms of estimating incubation periods should be investigated. This is an important issue because date of infection and/or onset of symptoms are difficult to precisely identify. Nicholas G.R.et al [31] have made a comparison of two means of making such estimates. One method uses doubly interval-censored data, and the other employs data reduction techniques making the calculation more tractable. The researchers used both methods on historical data relating to the incubation periods for respiratory syncytial virus and influenza A. The outcomes demonstrate that these methods reduce the demands for computational power to analyze the reduced data and make good estimates of median incubation times in many different experimental conditions. Nevertheless, the researchers do recommend that doubly interval-censored analysis should be used to estimate the distribution tails. The systematic review undertaken by this research has demonstrated that the current published estimations have offered empirical evidence that the virus' incubation period is approximately a median of 5.01 and a mean of 7.8 days, which falls into the 2 to 12 days assumption of the ECDC and the 0 to 14 days of the WHO [11]. Researchers like [2] propose a recommended quarantine length of a minimum of 14 days, suggesting that fatality risks should be estimated using a median time of 13 days between first symptoms and mortality. [14] offered evidence that it is possible for symptoms to appear once patients have left a 14 day period of quarantine or active monitoring; the authors suggest that the median incubation time is around five days, similar to SARS. In [16] , the authors stated that 14-day quarantines may be insufficient to protect the public in 6.7% of cases; they therefore proposed that quarantine should be a minimum of 22 days. In [19] it was found that eight patients developed symptoms after more than 14 days from infection. In [20] the authors referred to a patient who incubated the virus for 19 days, a substantial period but one which falls into the reported range (0-24 days).\", 'cite_spans': [{'start': 422, 'end': 426, 'text': '[31]', 'ref_id': 'BIBREF28'}, {'start': 1633, 'end': 1637, 'text': '[14]', 'ref_id': 'BIBREF11'}, {'start': 1867, 'end': 1871, 'text': '[16]', 'ref_id': 'BIBREF13'}, {'start': 2052, 'end': 2056, 'text': '[19]', 'ref_id': 'BIBREF16'}, {'start': 2152, 'end': 2156, 'text': '[20]', 'ref_id': 'BIBREF17'}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'It has yet to be conclusively demonstrated whether age group as any impact on the time a patient incubates the virus. Out of 291 patients [24] with an average age of 47, the incubation period was 4.0 days, for five patients [25] with an average age of 49.5 years it was 4.5 days, for 44 patients [26] with an average age of 60 years it was 4.99 days, and for two patients [27] with an average age of 47 years, it was 4.5 days.', 'cite_spans': [{'start': 138, 'end': 142, 'text': '[24]', 'ref_id': 'BIBREF21'}, {'start': 224, 'end': 228, 'text': '[25]', 'ref_id': 'BIBREF22'}, {'start': 296, 'end': 300, 'text': '[26]', 'ref_id': 'BIBREF23'}, {'start': 372, 'end': 376, 'text': '[27]', 'ref_id': 'BIBREF24'}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The authors of [27] suggested that long-term glucocorticoid use could cause a delay in the onset of symptoms, so that they would not appear until after a 14 day quarantine period had been completed. Long-term glucocorticoid use could be responsible for atypical infection, longer incubation periods, and additional COVID-19 transmission. [13] states that COVID-19 has a mean incubation period lower than MERS coronavirus but higher than SARS coronavirus. [21] offers a final insight that robust and workable countermeasures must be put in place for the prevention and/or mitigation of asymptomatic transmission in the course of incubation amongst high-risk populations.', 'cite_spans': [{'start': 15, 'end': 19, 'text': '[27]', 'ref_id': 'BIBREF24'}, {'start': 338, 'end': 342, 'text': '[13]', 'ref_id': 'BIBREF10'}, {'start': 455, 'end': 459, 'text': '[21]', 'ref_id': 'BIBREF18'}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The authors declare that they have no competing interests.', 'cite_spans': [], 'ref_spans': [], 'section': 'Competing interests'}, {'text': 'Not required. 1  2  3  4  5  6  7  8  9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41  42  43  44  45  46  47  48  49  50  51  52  53  54  55  56  57  58  59  60  61  62  63  64  65 . CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'cite_spans': [], 'ref_spans': [{'start': 14, 'end': 263, 'text': '1  2  3  4  5  6  7  8  9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41  42  43  44  45  46  47  48  49  50  51  52  53  54  55  56  57  58  59  60  61  62  63  64  65', 'ref_id': None}], 'section': 'Ethical approval'}, {'text': 'The copyright holder for this preprint this version posted May 23, 2020. 1  2  3  4  5  6  7  8  9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41  42  43  44  45  46  47  48  49  50  51  52  53  54  55  56  57  58  59  60  61  62  63  64  65 . CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'cite_spans': [], 'ref_spans': [{'start': 73, 'end': 322, 'text': '1  2  3  4  5  6  7  8  9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41  42  43  44  45  46  47  48  49  50  51  52  53  54  55  56  57  58  59  60  61  62  63  64  65', 'ref_id': None}], 'section': 'Bibliography'}, {'text': 'The copyright holder for this preprint this version posted May 23, 2020. 1  2  3  4  5  6  7  8  9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41  42  43  44  45  46  47  48  49  50  51  52  53  54  55  56  57  58  59  60  61  62  63  64  65 . CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'cite_spans': [], 'ref_spans': [{'start': 73, 'end': 322, 'text': '1  2  3  4  5  6  7  8  9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41  42  43  44  45  46  47  48  49  50  51  52  53  54  55  56  57  58  59  60  61  62  63  64  65', 'ref_id': None}], 'section': 'Bibliography'}, {'text': 'The copyright holder for this preprint this version posted May 23, 2020. . https://doi.org/10.1101/2020.05.20.20108340 doi: medRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': 'Bibliography'}], 'paragraphs': [{'text': 'At the start of 2020, a new form of coronavirus was found to be the source of infection responsible for an epidemic of viral pneumonia in Wuhan, China, a region in which the first patients began to show symptoms in December 2019. At the time of writing (April 29, 2020) over 3 million people globally have caught the virus, of whom more than 200,000 have died. The novel virus, causing severe acute respiratory disease, is thought to be from the same family as Middle East Respiratory Syndrome (MERS) coronavirus and Severe Acute Respiratory Syndrome (SARS) coronavirus, but it is unique in its own right. This means that central epidemiological parameters, which includes the incubation period, are being urgently researched in real-time from case reports while the epidemic is continuing [1] . The incubation period of a virus represents the time span from the probable earliest contact with a source of transmission and the earliest recognition of the first symptoms. Accurately estimating the length of incubation period is essential for effective contemporary public health measures to be taken [2] . If health authorities know what the incubation period is then they will know for how long a healthy individual has to be monitored and have their movement restricted (quarantine period) [3] . Correctly estimating the incubation period will also help us to comprehend how infectious COVID-19 is, make estimations of the size of the pandemic, and decide on the best course of action [4, 5, 6, 7] . With insufficient data available to definitively state what the incubation period for this virus is, the World Health Organisation (WHO) is working with a broad range of 0-14 days, the European Centre for Disease Prevention and Control (CDCDC) is working with a range of 2-14 days, and a number of studies have made the assumption that the incubation period is similar to that of the MERS and SARS coronaviruses [1] .', 'id': '00001'}], 'title': 'The estimations of the COVID-19 incubation period: a systematic review of the literature', 'sha': '09ff6011e4e95bff0b9221c89ffdb91d958e4ef1', 'doi': '10.1101/2020.05.20.20108340', 'score': 10.437600135803223}, {'rank': 6, 'abstract': \"A whole new pathogen, to which humans have virtually no pre-existing immunity, has caused fear all over the world. Severe acute respiratory syndrome coronavirus (SARS CoV-2) is one of the types of human novel-coronavirus of the family coronavirus. The nature of transmission of the virus makes it one of the most infectious pathogenic diseases that has ever existed. Though the human coronaviruses have existed since the discovery of the human coronavirus 229E (HCoV-229E) and human coronavirus OC43 (HCoV-OC43) in 1960, it has been a challenge to develop an effective cure as well as vaccine for the diseases associated with coronaviruses. Commonly, human coronaviruses cause illnesses such as intestinal and respiratory tract illnesses. Nevertheless, the symptoms reflected after infection from the coronaviruses take some time before being identified. Thus, viruses can replicate and cause more harm to the human body before being detected. Moreover, research continues to explain why some gene variations in some individuals increase the risk of some infectious diseases, while others are not affected. Looking at gene variations in people infected with Coronavirus Disease 2019 (COVID-19) and studying how genes influence people's response to infection will help to develop a vaccine that will help strengthen the immune system. Knowing how the human genes respond to the virus COVID-19 will help to cure people more effectively.\", 'body_text': [{'text': 'Some viral diseases have caused fatal havoc on the human species since they are associated with high mortality rates as well as low prevention and treatment approaches. Currently, there is a high risk of the spread of the COVID-19 disease caused by the SARS CoV-2 viruses. SARS CoV-2 virus is one of the types of human novel-coronavirus of the family coronavirus. The nature of transmission of the virus makes it one of the most infectious pathogenic diseases that has ever existed. Though the human coronaviruses have existed since the discovery of the human coronavirus 229E (HCoV-229E) and human coronavirus OC43 (HCoV-OC43) in 1960, it has been a challenge to develop an effective cure as well as vaccine for the diseases associated with coronaviruses. Later in 2003, a coronavirus pandemic erupted in Guangdong, a Chinese province famous for live wild animal trade, and was identified as the severe acute respiratory syndrome coronavirus (SARS-CoV). SARS-CoV pandemic affected at least 37 nations causing 1879 deaths across the globe by 2005 when it was effectively controlled [1]. Commonly, human coronaviruses cause illnesses such as intestinal and respiratory tract illnesses. Nevertheless, the symptoms reflected after infection from the coronaviruses take some time before being identified; thus, the viruses can replicate and cause more harm to the human body before being detected. Antibiotic treatment has been ineffective in treating infections due to coronaviruses, which acts as another factor contributing to the very high death toll [1]. Through conducting this research, this paper will establish reliable information on the transmission mode of coronaviruses and explore the various traits of both SARS-CoV and SARS-CoV-2 while offering various measures to control the implications of the viruses on the human population. Older people with pre-existing diseases and men are more susceptible to this infection and make up the majority of the victims. However, most people with severe symptoms are middle-aged adults who were stable at an early stage. Several studies conducted to determine the cause of such variation by collecting data from around the world, in particular from countries with a large number of cases, to find clues to help find a common denominator that can help in finding a cure. Age, sexual orientation, socio-economic status, and access to the health care system are some of the variants that were not very helpful, but researching genome could help answer the question we were looking for, a cure to combat this deadly viral infection and avoid another wave of similar diseases.', 'cite_spans': [{'start': 1083, 'end': 1084, 'mention': '1', 'ref_id': 'BIBREF0'}, {'start': 1552, 'end': 1553, 'mention': '1', 'ref_id': 'BIBREF0'}], 'section': 'Introduction and background', 'ref_spans': []}, {'text': 'We used PubMed as our main database to search for the relevant published literature. We searched for all types of the articles that were related to the topic.', 'cite_spans': [], 'section': 'Review', 'ref_spans': []}, {'text': 'Animal to human transmission', 'cite_spans': [], 'section': 'Review', 'ref_spans': []}, {'text': 'The infections caused by the coronaviruses family to human beings since the onset of the 21st century influenced researches to identify the origin and transmission mode of the viruses across human beings. A study conducted to identify the initial cause of the severe acute respiratory syndrome disease revealed that the SARS-CoV virus was transmitted from the wild animal reservoirs in the Guangdong animal market. The direct or indirect interaction between human beings and animals, such as bats, was identified as the critical source of the coronavirus. Direct consumption of bats or being close to them, facilitated the spread of the virus from the wild animals to human beings, thus increasing the impacts of the disease onto the human population [2,3]. Similarly, the COVID-19 disease caused by SARS CoV-2 coronaviruses originally erupted from the interaction of human beings and wild animals such as bats in the Wuhan market in the Hubei province of China. Coronaviruses have been identified in various wild animals, especially mammals hence, illustrating their potential impacts on the human species. Though the viruses do not have a severe effect on the wild animals, they have been identified to influence severe respiratory infections that are resistant to known antibiotics as well as treatments.', 'cite_spans': [{'start': 752, 'end': 753, 'mention': '2', 'ref_id': 'BIBREF11'}, {'start': 754, 'end': 755, 'mention': '3', 'ref_id': 'BIBREF21'}], 'section': 'Review', 'ref_spans': []}, {'text': 'Human to human transmission', 'cite_spans': [], 'section': 'Review', 'ref_spans': []}, {'text': 'The contact between infected persons and uninfected persons influences the spread of both the SARS CoV-2 and COVID-19, making them one of the most infectious diseases on the globe. Since both infections affect the respiratory tract, the viruses can spread via the cough droplets, sputum, and closeness, such as touching an infected individual. Moreover, the SARS CoV-2 viruses were discovered in the feces of a child infected by the disease, thus, depicting that fecal contact could facilitate the spread of the disease [4]. Further, the diseases influence the human species respiratory issues such as influenza, thus possible to spread via the air from one individual to another. COVID-19 has been identified as a disease affecting the upper respiratory tract during the first few days after an infection where it causes pain in the throat and later moves down to the lower respiratory tract. Disposing of the viruses on the skin after circulating across the body system increases the possibility of spreading the viruses via sharing clothes, body contact, shaking hands, and hugging [5]. Crowding and high socialization among human beings increase the chances of infections across the human population; hence, people need to avoid such situations.', 'cite_spans': [{'start': 521, 'end': 522, 'mention': '4', 'ref_id': 'BIBREF22'}, {'start': 1086, 'end': 1087, 'mention': '5', 'ref_id': 'BIBREF23'}], 'section': 'Review', 'ref_spans': []}, {'text': 'Incubation period', 'cite_spans': [], 'section': 'Review', 'ref_spans': []}, {'text': 'The incubation period of a disease is the duration that the symptoms take to first appear after exposure to infection. A long incubation period depicts that an individual has a high tendency to infect other individuals before the disease can be medically diagnosed. During the incubation period, the pathogens causing the disease replicate, and the host immune response is activated producing cytokines such as interferon that can have global effects, hence, reducing the body resistance posed by the immune system [6]. Although an incubation period might be longer such as 14 to 21 days after exposure, the ability to infect other individuals can be before the symptoms are identified. Moreover, determining the length of the incubation period of certain infectious diseases provides the most probable time frame, during which an individual can be subjected to isolation and intensive medical observation [6]. The outbreak of severe acute respiratory syndrome in 2003 caused by the novel coronaviruses influenced studies on the coronavirus family and its traits. During the study, the incubation period for the disease was estimated to be a mean of 6.4 days. Currently, the estimated incubation period for the prevalent COVID-19 has faced various changes since different people are affected differently by the disease. According to the World Health Organization, the incubation period of COVID-19 is between two to 10 days [7]. The China National Health Commission had initially approximated the COVID-19 incubation 10 to 14 days.', 'cite_spans': [{'start': 516, 'end': 517, 'mention': '6', 'ref_id': 'BIBREF24'}, {'start': 907, 'end': 908, 'mention': '6', 'ref_id': 'BIBREF24'}, {'start': 1425, 'end': 1426, 'mention': '7', 'ref_id': 'BIBREF25'}], 'section': 'Review', 'ref_spans': []}, {'text': 'On the other hand, the Centers for Disease Control and Prevention (CDC) approximates that the incubation period of the COVID-19 is two to 14 days [7]. Quarantine, self-isolation, and intensive medical examination are carried out to prevent the spread of the COVID-19 in public. Since there is no known cure or vaccine for the current global pandemic, many nations have opted to control the spread of the disease across the population to avoid fatal impacts such as high mortality rates.', 'cite_spans': [{'start': 147, 'end': 148, 'mention': '7', 'ref_id': 'BIBREF25'}], 'section': 'Review', 'ref_spans': []}, {'text': 'Genomic differences between SARS-COV and SARS-CoV-2 viruses', 'cite_spans': [], 'section': 'Review', 'ref_spans': []}, {'text': 'Genomic traits of disease-causing pathogens reflect their origin as well as establish their implications on the host body. Further, the study of the genomic characteristics of a disease-causing agent influences the development of possible treatment measures that can be used to address the effects of the infection across the affected population [8]. Current researches conducted on nine patients depicted that the human novel coronavirus 2019 has a ten genomic sequence similar to two bat novel coronavirus-like pathogens with a similarity of 99.98% across the examined nine patients [8,9]. The bat-SL-CoVZ45 and bat-SL-CoVZXC21 are distinct from the SARS-CoV virus that affected at least 37 nations in 2003 [9]. Studies have shown that despite some variations in genomic sequences, COVID-19 has a similar receptor-binding domain structure to that of SARS-CoV. Moreover, the 2019 SARS-CoV-2 is categorized as a subgenus Sarbecovirus of the genus Betacoronavirus [8,10]. Additionally, the close relation of the SARS-CoV-2 infection with the bat-like coronaviruses shows that close contact between humans and bats could have influenced the spread of the virus into the human hosts. Moreover, the mutation of the SARS-CoV is long term since it can take years; hence the resistivity to vaccines could be low but develop within several years of exposure [11]. Currently, the mutation rate of the SARS-CoV-2 virus is not known, though its replication rate is high and poses fatal challenges to human health.', 'cite_spans': [{'start': 347, 'end': 348, 'mention': '8', 'ref_id': 'BIBREF26'}, {'start': 586, 'end': 587, 'mention': '8', 'ref_id': 'BIBREF26'}, {'start': 588, 'end': 589, 'mention': '9', 'ref_id': 'BIBREF27'}, {'start': 710, 'end': 711, 'mention': '9', 'ref_id': 'BIBREF27'}, {'start': 964, 'end': 965, 'mention': '8', 'ref_id': 'BIBREF26'}, {'start': 966, 'end': 968, 'mention': '10', 'ref_id': 'BIBREF1'}, {'start': 1351, 'end': 1353, 'mention': '11', 'ref_id': 'BIBREF2'}], 'section': 'Review', 'ref_spans': []}, {'text': 'Immune response and gene variations', 'cite_spans': [], 'section': 'Review', 'ref_spans': []}, {'text': 'The gene, known as human leukocyte antigen (HLA), produces proteins that bind to the pathogen that acts as warning signs to alert immune cells. The human body has three different classes of proteins to determine how the body can best fight certain infections, including SARS-CoV-2. If HLA binds a viral protein that is critical for the pathogen to replicate and survive, the subsequent antibody activity is likely to be more effective in targeting the virus. If someone was previously exposed to common coronaviruses - which cause the common cold - and had the right HLA types that provide some protection against other viruses, it is theoretically possible that they could also generate an early immune response against the novel SARS-CoV-2 virus [12].', 'cite_spans': [{'start': 749, 'end': 751, 'mention': '12', 'ref_id': 'BIBREF3'}], 'section': 'Review', 'ref_spans': []}, {'text': 'Immune system response is different in different people. Overreaction of the immune system, which is known as cytokine storm, is thought to be the leading cause of death in many severe acute respiratory syndrome (SARS) patients and is believed to be responsible for some of the deaths among young COVID-19 patients. The mortality rate among men infected with COVID-19 is higher than mortality rate among women with COVID-19, partly due to angiotensin-converting enzyme 2 (ACE2) found in the X chromosome, which means that men have one of the genes that make them more vulnerable to this disease. In comparison, women have two copies of the gene; ACE2 would mitigate any viral susceptibility to one variant from the second allele of ACE2 [13].', 'cite_spans': [{'start': 738, 'end': 740, 'mention': '13', 'ref_id': 'BIBREF4'}], 'section': 'Review', 'ref_spans': []}, {'text': 'To date, some genetic studies suggest that variants in HLA genes are likely to play a role, while others point to the difference in ABO blood type and ACE2 genes that SARS-COV 2 binds to human cells. In addition to the genetic variants of the ACE2 receptor, the severity of the disease is influenced by variations in human leukocyte antigen genes that affect the immune response to the virus.', 'cite_spans': [], 'section': 'Review', 'ref_spans': []}, {'text': 'Wuhan University research tested different sources of ABO blood from 369 virus-infected patients and found 32.16%, 24.90%, 9.10%, and 33.84% for A, B, AB, and O, respectively. It has proposed a high type A contingent and a low type O contingent. These results reflected blood tests in another hospital, followed by 37.75%, 26.42%, 10.03%, and 25.80%. A similar pattern has been reported for mortality rates, with 206 deaths at 41.26%, 24.27%, 9.22%, and 25.24%, respectively in classes A, B, AB, and O [14]. Based on these findings, it is suggested that people with blood type \"A\" are at significantly higher risk of infection than people with other ABO blood-types, which may be due to the presence of anti-A antibody in the blood. However, more clinical trials are needed to support this association of ABO blood groups and COVID-19.', 'cite_spans': [{'start': 503, 'end': 505, 'mention': '14', 'ref_id': 'BIBREF5'}], 'section': 'Review', 'ref_spans': []}, {'text': 'The association between the levels of ACE2 expression and the four key levels of immune signature in the tissues, including CD8+T cells, interferon response, B cells, and natural killer (NK) cells, was evaluated. According to the human protein atlas (HPA), significantly higher levels of ACE2 protein and gene expression are found in the gastrointestinal tract, kidney, gallbladder, and male reproductive tissues. This data could also explain the higher number of cases of COVID-19 in males compared to females [15].', 'cite_spans': [{'start': 512, 'end': 514, 'mention': '15', 'ref_id': 'BIBREF6'}], 'section': 'Review', 'ref_spans': []}, {'text': 'Looking at the interaction of ACE2 levels of expression with CD8+T cells, interferon, B cells, NK cells in males, there was a substantially positive linkage in the thyroid, heart, adrenal glands, liver, kidneys, colon, and bladder. Whereas, the females only had interaction between ACE2 and immune markers in thyroid, heart, and colon, which indicates the variations and similarities between male and female expressions of ACE2 and immune markers. It also explains why some people have respiratory symptoms, and others have gastrointestinal symptoms such as diarrhea, nausea, vomiting, and other signs of the system. It has already shown that the lung cells are more easily contaminated with SARS 2 due to higher levels of ACE2. Male and older age groups have a higher degree of association between ACE2 expression and immune signature than females and young adults, which explains the higher level of SARS 2 cases and death in this population. There are no significant differences in ACE2 expression between males and females, but the distinction lies in the immune response that varies from person to person [15].', 'cite_spans': [{'start': 1111, 'end': 1113, 'mention': '15', 'ref_id': 'BIBREF6'}], 'section': 'Review', 'ref_spans': []}, {'text': 'Diagnosis for both viruses', 'cite_spans': [], 'section': 'Review', 'ref_spans': []}, {'text': 'The diagnosis of coronaviruses is based on the symptoms reflected by the affected individual. Since both the SARS-CoV and SARS-CoV-2 cause a severe influenza-like infection that affects the human respiratory system, respiratory symptoms are commonly used to detect the viruses [16]. The initial indications of the presence of the virus in the human body include coughing, running nose, common cold, headaches, body aches, and difficulty in breathing. Further, various more advanced techniques are used to determine the type of the virus as well as its genetic composition in the human respiratory system [17]. Electron microscopy, reverse transcriptase-polymerase chain reaction (RT-PCR) amplification, serology, and sequence-independent amplification are the fundamental techniques used to develop the essential traits of both SARS-CoV and SARS-CoV-2 in the human body [M]. Currently, RT-PCR amplification, serology, and sequence-independent amplification are commonly used approaches to diagnose the SARS-CoV-2 virus across the human population.', 'cite_spans': [{'start': 278, 'end': 280, 'mention': '16', 'ref_id': 'BIBREF7'}, {'start': 605, 'end': 607, 'mention': '17', 'ref_id': 'BIBREF8'}], 'section': 'Review', 'ref_spans': []}, {'text': 'Treatment measures', 'cite_spans': [], 'section': 'Review', 'ref_spans': []}, {'text': 'The treatment process of the SARS-CoV infection varies slightly from the methods currently employed to address the increasing issues of SARS-CoV-2 disease on the globe. SARS-CoV treatment measures involved the initiation of ventilators that assisted the patients in accessing effective breathing ability [18]. Treatment of the subsequent infections facilitated by the virus enhanced the ability to control the disease though it eventually caused at least 1800 deaths [19]. On the other hand, the current treatment measures imposed to counter the fatality of the SARS-CoV-2 include the treatment of the conditions associated with the infection since there is no known cure for the disease. Further, scientists are considering the application of virus antiviral medication to assist in curbing the prevalence of the disease across the human population. The close association of SARS-CoV and SARS-CoV-2 has influenced the use of ventilators to enhance the breathing ability of patients affected by SARS-CoV-2 [20]. Controlling the impact of SARS-CoV-2 has been a problematic issue across the globe because of its high infectious condition and its generic composition. The protein coat over the key genetic ribonucleic acid (RNA) material inside the SARS-CoV-2 virus resists the impacts of any treatment chemical utilized to cure the ailment [21]. Currently, regular washing of hands, avoiding social contact with an infected individual as well as wearing protective masks and gloves are the commonly used measures to prevent the spread of the coronaviruses across the human population. Isolation and quarantine measures are being imposed on the individuals who have tested positive with the disease.', 'cite_spans': [{'start': 305, 'end': 307, 'mention': '18', 'ref_id': 'BIBREF9'}, {'start': 468, 'end': 470, 'mention': '19', 'ref_id': 'BIBREF10'}, {'start': 1007, 'end': 1009, 'mention': '20', 'ref_id': 'BIBREF12'}, {'start': 1339, 'end': 1341, 'mention': '21', 'ref_id': 'BIBREF13'}], 'section': 'Review', 'ref_spans': []}, {'text': 'Plasma transfusion', 'cite_spans': [], 'section': 'Review', 'ref_spans': []}, {'text': 'Convalescent plasma is used as a clinical procedure and as an option for patients who are either seriously ill or who do not respond to other therapies. This plasma comes from patients who have recovered from COVID-19 infection and who have developed antibodies to control the virus. These antibodies are formed in plasma to protect against possible infections with the same virus. However, the potential treatment benefits and risk of developing side-effects need to be further investigated. The use of convalescent plasma in previous Middle East Respiratory Syndrome (MERS-CoV) and SARS outbreaks demonstrated safety and faster viral clearance, particularly when given early in the course of the disease. Transferring plasma from patients who developed neutralizing antibodies (nAbs) to specific SARS-CoV-2 proteins could neutralize the virus, prevent further replication and stop ongoing tissue damage in patients with less severe infections, earlier in the course of the disease, or prophylactically administered to high-risk individuals [22].', 'cite_spans': [{'start': 1043, 'end': 1045, 'mention': '22', 'ref_id': 'BIBREF14'}], 'section': 'Review', 'ref_spans': []}, {'text': 'Understanding the actions of crucial COVID-19 proteins is critical to the development of plausible treatment or vaccines. CoV-2 S protein plays a vital role in the fusion, infection, and transmission of viruses. Moreover, the most likely objective is to build nAbs to block the binding and merging of COVID-19. The S protein is the primary inducer of nAbs, consisting of two components, S1 and S2. The receptor-binding domain (RBD) interacts directly with the host receptor S1. Although SARS 2 uses the same ACE2 receptor as SARS for cell entry, SARS-CoV-2 S protein binds ACE2 with a higher affinity than SARS-CoV, suggesting that its ACE2 recognition may differ from SARS-CoV. Besides, there is a minimal cross-reactivity of the antibody between the two virus S proteins. Some studies indicate that SARS-CoV antibodies may be binding to SARS CoV-2 RBD, but their ability to neutralize is not understood.', 'cite_spans': [], 'section': 'Review', 'ref_spans': []}, {'text': 'SARS and MERS unique nAbs that include monoclonal antibodies (mAbs) target regions S1 and S2 and block RBDs to their respective receptors and interfere with S2-mediated membrane fusion or host cell entry, preventing viral infection. Despite the use of polyclonal antibodies to treat SARS 2 disease in recovered COVID-19 patients, researchers have yet to discover novel neutralizing mAbs for SARS-CoV 2. If these antibodies developed, it will take several years for them to be ready for human use, as their safety and efficacy should be tested [23].', 'cite_spans': [{'start': 544, 'end': 546, 'mention': '23', 'ref_id': 'BIBREF15'}], 'section': 'Review', 'ref_spans': []}, {'text': 'Data from various human and animal-acquired viral infections and vaccines indicate that neutralizing antibodies (nAbs) are needed to prevent or cure viral infections. These nAbs may be either contaminated or vaccinated to avoid viral infections. Lack of data on the role of nAbs in disease progression due to possible heterogeneity of SARS 2-specific nAbs in recovered COVID-19 patients needs further investigation in the development of an appropriate vaccine and patients undergoing convalescent plasma therapy.', 'cite_spans': [], 'section': 'Review', 'ref_spans': []}, {'text': 'In the past, antibody therapy for plasma infusions has been commonly used to treat viral infections, including SARS and MERS. The efficacy of this therapy has been linked to the concentration of nAbs in the recovered plasma donor. If viral infection with SARS-CoV-2 persists, convalescent plasma continues in recovered patients or serum is used as an alternative therapy.', 'cite_spans': [], 'section': 'Review', 'ref_spans': []}, {'text': 'nAb levels in patients recovered from COVID-19 may vary, some with high nAb titers, some with very low titers, indicating that other immune responses, including T cells or cytokines, may contribute to the recovery of these patients. Research has shown that higher C-reactive protein (CRP) and lower lymphocyte counts have a strong innate immune response at the time of admission to elderly patients than younger patients, which may have caused an adverse condition following coronavirus infection. Rapid innate immune responses in older patients often suggest higher rates of nAb titer. It attributes to the fact that neutralizing antibodies (nAbs) have a positive association with CRP but are negatively associated with the indication of patient lymphocytes, a link between cell-mediated antibody response and immune response [24].', 'cite_spans': [{'start': 828, 'end': 830, 'mention': '24', 'ref_id': 'BIBREF16'}], 'section': 'Review', 'ref_spans': []}, {'text': 'Comparative analysis', 'cite_spans': [], 'section': 'Review', 'ref_spans': []}, {'text': 'The impacts of coronaviruses on the globe have caused fatal situations that led to the death of more than 25,000 individuals globally up to this day. SARS-CoV-2 is a very highly infectious disease that can spread via close contact between human beings [25]. Coming into contact with surfaces that contain the virus further facilitates the spread of the virus. Since humans are social beings participating in social events, it increases the risk of spreading the virus from one individual to another. The traits of the virus make it more likely to affect a large number of individuals across the globe. Moreover, the SARS-CoV-2 disease is a viral disease caused by a novel human coronavirus that has a long strand of RNA, hence, it is complex to regulate its replication in the human body [26]. COVID-19 viruses have protein spikes that enhance their compatibility with the human cells; therefore, they can survive in the human body.', 'cite_spans': [{'start': 253, 'end': 255, 'mention': '25', 'ref_id': 'BIBREF17'}, {'start': 789, 'end': 791, 'mention': '26', 'ref_id': 'BIBREF18'}], 'section': 'Review', 'ref_spans': []}, {'text': 'The coronaviruses multiply at a higher rate; hence, they spread through the affected human organs at a high rate. The high replication rate increases the rate by which the viruses spread across the human body, hence it is challenging to cure the disease. The compatibility of the viral cells with human cells further inhibits treatment approaches aimed at denaturing the viral cell. The key difference between the SARS-CoV virus and the SARS-CoV-2 virus is their genomic structure, and their response to different treatment approaches. SARS-CoV genomic make-up varies from the SARS-CoV-2 since the SARS-CoV-2 virus has longer RNA strand than the strand in the SARS virus. The RNA S-sequence in both SARS-CoV-2 and SARS-CoV viruses is similar; hence, the key differences have not been effectively proven [27]. The impacts of the diseases on human bodies are identical though the SARS-CoV affects various organs.', 'cite_spans': [{'start': 804, 'end': 806, 'mention': '27', 'ref_id': 'BIBREF19'}], 'section': 'Review', 'ref_spans': []}, {'text': 'Currently, there are different facts about the SARS-CoV-2 that are not known through current researches that are being conducted to answer the different concerns on the SARS-CoV-2 infection. The treatment approaches in terms of vaccine and a medicinal cure have not been discovered. The survival of the COVID-19 virus in the human corpse after demise is not currently known; thus, the researches conducted are aimed at informing the relevant risk eminent when one comes into contact with the corpse of corona patient [28]. People are encouraged to maintain a social distance to avoid further spread of the disease. Moreover, frequent hand-washing and disinfecting the possibly contamination-prone surfaces is recommended. Using face masks and gloves prevents individuals from the virus from the surfaces as well as from the other infected individuals. Staying at home has been advocated across the globe, where individuals are encouraged to conduct essential activities from their homes and avoid moving out of their houses for unnecessary reasons. Comparison between SARS and COVID-19 is shown in Table 1.', 'cite_spans': [{'start': 518, 'end': 520, 'mention': '28', 'ref_id': 'BIBREF20'}], 'section': 'Review', 'ref_spans': [{'start': 1104, 'end': 1105, 'mention': '1', 'ref_id': None}]}, {'text': 'SARS and COVID-19 have similarities, but can they co-infect a patient? If they do co-infect, then how often do they have a synergistic effect, if they do have a synergistic effect, then is there a way to differentiate them from each other symptomatically? Can they both be differentiated based on signs and symptoms when they co-infect a patient?', 'cite_spans': [], 'section': 'Review', 'ref_spans': []}, {'text': 'Symptom wise COVID-19 symptoms vary. Why these variations exist? Why some patients exhibit some symptoms while others exhibit different symptoms? Is this because of the variant of the virus? Or is it because of the genetic differences between the individuals? And the way COVID-19 manifests with different symptoms in different individuals, is it the same situation with SARS? If yes, then to what extent?', 'cite_spans': [], 'section': 'Review', 'ref_spans': []}, {'text': 'These are the questions that arise, and we recommend future scientists to explore more about these questions.', 'cite_spans': [], 'section': 'Review', 'ref_spans': []}, {'text': 'We have much to learn from previous outbreaks of coronaviruses and the new ongoing pandemic, by looking at the similarities and differences between the current SARS-CoV-2 and the previous outbreak of SARS; a remarkable resemblance emerges with some unique features of its own. Every time a new strain of these viruses appears, it is more aggressive than the previous ones, which makes it more difficult to find treatment and a vaccine. A clear understanding of the host immune response may help in finding how the virus causes disease and a possible cure to eradicate this virus. The wet animal markets have played a significant role in the SARS, and COVID-19 global epidemics emphasize the need for a change in China - away from their use. To prevent zoonotic transmission, the best approach is to suspend or close down these markets; however, due to its long history in China, it will be impossible to shut down permanently as the income of most people relies on selling live animals. The best possible way to prevent and control the disease spread is the use of proper hygiene, and rules for limiting animal-human contact would be ideal, in addition to epidemiological surveillance and monitoring. The knowledge we have so far is limited, and more research work is needed to design prophylactic and therapeutic measures to fight current epidemics and any future outbreak of similar coronavirus.', 'cite_spans': [], 'section': 'Conclusions', 'ref_spans': []}], 'paragraphs': [{'text': 'The incubation period of a disease is the duration that the symptoms take to first appear after exposure to infection. A long incubation period depicts that an individual has a high tendency to infect other individuals before the disease can be medically diagnosed. During the incubation period, the pathogens causing the disease replicate, and the host immune response is activated producing cytokines such as interferon that can have global effects, hence, reducing the body resistance posed by the immune system [6]. Although an incubation period might be longer such as 14 to 21 days after exposure, the ability to infect other individuals can be before the symptoms are identified. Moreover, determining the length of the incubation period of certain infectious diseases provides the most probable time frame, during which an individual can be subjected to isolation and intensive medical observation [6]. The outbreak of severe acute respiratory syndrome in 2003 caused by the novel coronaviruses influenced studies on the coronavirus family and its traits. During the study, the incubation period for the disease was estimated to be a mean of 6.4 days. Currently, the estimated incubation period for the prevalent COVID-19 has faced various changes since different people are affected differently by the disease. According to the World Health Organization, the incubation period of COVID-19 is between two to 10 days [7]. The China National Health Commission had initially approximated the COVID-19 incubation 10 to 14 days.', 'id': '00008'}], 'title': 'Do COVID-19 and SARS Gene Complexities and Variations Help Overcome the Knowledge Gap?', 'sha': 'ca0099bfbf2b65723d4b8a82b49915fc1f7e6403', 'doi': '10.7759/cureus.8439', 'score': 10.417099952697754}, {'rank': 7, 'paragraphs': [{'text': \"The copyright holder for this preprint this version posted June 16, 2020. . and strength of relaxation ( Figure 3) ; naturally, stronger relaxation (all the way to baseline contact levels with f = 1) produces a difference quickly (10-20 days) . Similarly a smaller threshold difference of 5 cases is reached relatively soon. Overall it takes 10-40 days for a substantial difference to arise. The uncertainty in R 0 and other parameters also impacts the time before there is a substantial difference in trajectories ( Figure S6 ). We find that halving uncertainty in the underlying growth rate (by reducing the standard deviation in R 0 ) can reduce the time until detection from 20-40 days to 12-25 days. Uncertainty in other fixed parameters does not have a strong impact on the time ( Figure S6 ) unless the relaxation conditions are strong (f 2 > 0.7). Epidemiological parameters are now somewhat well-established for COVID-19 [26] , but we can be less certain of the true value of more behaviour-influenced parameters (q, u r , u d ). We explicitly explore the impact of the incubation period on the time until a substantial difference arises, and find little impact ( Figure S7 ), largely because a few days' uncertainty in the incubation period is overwhelmed by the the overall time scale and threshold of 10 cases to define a substantial difference. In some instances we cannot differentiate between trajectories for the models with distancing and the models without distancing (these are the missing values in the related figures). Also, if the standard deviation in the underlying parameter R 0 is too large, a substantial difference may not be possible to predict in the time frame of interest.\", 'id': '00027'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 10.245499610900879}, {'rank': 8, 'paragraphs': [{'text': 'We use a compartmental model describing susceptible, exposed, infectious and removed individuals (SEIR) [2] . We fit the model to data on daily case counts, and incorporate knowledge of the incubation period and duration of the infectious period in the model. The model includes a fixed proportion of the population who are willing and able to practise physical distancing, although individuals can transition between distancing and non-distancing modes. Upon infectious contact, individuals move from a susceptible state (S) to an exposed state (E 1 ). From here, the disease progresses through the infectious, pre-symptomatic state (E 2 ), to infectious and symptomatic (I), and finally to removed (R). Removed individuals are assumed to be not susceptible. The model furthermore incorporates a possible quarantined state (Q) following symptom onset, in which individuals are still infected but are unable to infect others. For each state, there is a corresponding \"distanced\" state. A schematic for this model is shown in Figure 1 .', 'id': '00004'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 10.124699592590332}, {'rank': 9, 'abstract': 'Back-projection is an epidemiological analysis method that was developed to estimate HIV incidence using surveillance data on AIDS diagnoses. It was used extensively during the 1990s for this purpose as well as in other epidemiological contexts. Surveillance data on COVID-19 diagnoses can be analysed by the method of back-projection using information about the probability distribution of the time between infection and diagnosis, which is primarily determined by the incubation period. This paper demonstrates the value of such analyses using daily diagnoses from Australia. It is shown how back-projection can be used to assess the pattern of COVID-19 infection incidence over time and to assess the impact of control measures by investigating their temporal association with changes in incidence patterns. For Australia, these analyses reveal that peak infection incidence coincided with the introduction of border closures and social distancing restrictions, while the introduction of subsequent social distancing measures coincided with a continuing decline in incidence to very low levels. These associations were not directly discernible from the daily diagnosis counts, which continued to increase after the first stage of control measures. It is estimated that a one week delay in peak incidence would have led to a fivefold increase in total infections. Furthermore, at the height of the outbreak, half to three-quarters of all infections remained undiagnosed. Automated data analytics of routinely collected surveillance data are a valuable monitoring tool for the COVID-19 pandemic and may be useful for calibrating transmission dynamics models.', 'body_text': [{'text': 'As the COVID-19 pandemic has evolved, daily counts of new diagnoses have been a major focus of governments, researchers and the broader community. However, although these counts provide a window into the progress of the epidemic, they represent only part of the total extent of infection within a population. Furthermore, the pattern of diagnoses over time is a lagged and incomplete representation of the pattern of infections over time. In monitoring the extent and evolution of the epidemic, as well as the effectiveness of control measures such as border closures, social distancing and community lockdowns, it is infections not diagnoses that are of primary interest.', 'cite_spans': [], 'section': 'Introduction', 'ref_spans': []}, {'text': 'The purpose of this paper is to analyse Australian COVID-19 daily diagnosis counts in order to demonstrate how these routinely collected data can be converted into information about the quantity of fundamental interest, which is infection incidence. It is demonstrated that with appropriate statistical analysis, based on back-projection methodology used extensively to monitor the HIV/AIDS epidemic, the daily COVID-19 diagnosis data can provide information about serial infection incidence and the extent of undiagnosed infections in a population. As well as helping to understand the extent of the epidemic, such information can also be used to assess the impact of government control measures. It is argued that analytics systems automating and continuously updating COVID-19 infection estimates and projections are valuable tools for monitoring the outbreak, and should be considered as preparation for a potential second wave of infections.', 'cite_spans': [], 'section': 'Introduction', 'ref_spans': []}, {'text': 'In Australia, each of the eight state and territory health departments provides a daily update on the cumulative number of COVID-19 diagnoses. Various public domain resources provide convenient access to these data [1–5]. This paper uses confirmed case numbers taken directly from the daily health department updates. Daily diagnosis numbers were analysed for the 105 days from the first reported case on 25 January through to 8 May, broken down by state and territory as well as aggregated nationally. Although slight differences exist between the various public domain data resources, the analysis results were not substantively altered when repeated on other versions of the data.', 'cite_spans': [{'start': 216, 'end': 217, 'mention': '1', 'ref_id': 'BIBREF0'}, {'start': 218, 'end': 219, 'mention': '5', 'ref_id': 'BIBREF18'}], 'section': 'Diagnosis data ::: Methods', 'ref_spans': []}, {'text': 'The primary method of analysis was the method of back-projection, also called back-calculation, which was originally developed for the purpose of estimating HIV incidence using AIDS surveillance data [6–8]. This method has also been used extensively in other contexts where an unobserved incident event is followed by a delay until an observed diagnosis [9, Section 3.1], as is the case for COVID-19 infection and diagnosis.', 'cite_spans': [{'start': 201, 'end': 202, 'mention': '6', 'ref_id': 'BIBREF19'}, {'start': 203, 'end': 204, 'mention': '8', 'ref_id': 'BIBREF21'}, {'start': 355, 'end': 356, 'mention': '9', 'ref_id': 'BIBREF22'}], 'section': 'Estimating infection incidence ::: Methods', 'ref_spans': []}, {'text': 'A key piece of information required to implement a back-projection analysis is the probability distribution of the time D between the unobserved event of interest and the subsequently observed diagnosis, which will be referred to as the diagnosis distribution. The diagnosis distribution specifies the probabilities  for a given number of days d, and will be described in detail below. Back-projection then makes use of the fact that the observed serial diagnosis counts Yt, for days t = 1, … , T, reflect the unobserved number of COVID-19 infections Xt, aggregated with the diagnosis distribution. Given the assumed diagnosis distribution, these diagnosis counts can be statistically decomposed to yield estimates of the infection incidence, from which estimates of cumulative infections and undiagnosed infections may also be obtained.', 'cite_spans': [], 'section': 'Estimating infection incidence ::: Methods', 'ref_spans': []}, {'text': 'The analysis is based on the fundamental relationship between the mean diagnosis count at time t, μt = E(Yt), and the mean infection counts λs = E(Xs), for times s up to and including time t, weighted by the diagnosis distribution:Assuming that the infection counts Xt are independent Poisson random variables leads to a high-dimensional linear Poisson regression model for the diagnosis counts Yt. The model requires non-negativity constraints on the parameters λt, reflecting the fact that they are Poisson means. Once fitted, the model provides estimates of the mean infection incidence , which can be used as estimates of the unobserved infection counts over time. This process is called back-projection because it effectively involves projecting the diagnosis counts back in time using the diagnosis distribution, and is also referred to as Poisson deconvolution because it involves disaggregating the mean infection counts from the diagnosis distribution. In theory, the model is an identity link generalised linear model, but in practice it requires special computational software to accommodate the high dimensionality and the parameter constraints. Reliable algorithms for this non-standard analysis are incorporated into freely available open source software and are used here. In particular, the analyses were performed in the R computing environment [10] using the addreg package that contains the nnpois function for performing reliable high-dimensional non-negatively constrained linear Poisson regression [11]. Code for implementing these analyses is available on the GitHub repository linked to the Coronavirus 10-day Forecast resource from The University of Melbourne [4].', 'cite_spans': [{'start': 1363, 'end': 1365, 'mention': '10', 'ref_id': 'BIBREF1'}, {'start': 1521, 'end': 1523, 'mention': '11', 'ref_id': 'BIBREF2'}, {'start': 1686, 'end': 1687, 'mention': '4', 'ref_id': 'BIBREF17'}], 'section': 'Estimating infection incidence ::: Methods', 'ref_spans': []}, {'text': 'The back-projection estimates  provide information about the overall incidence of infection among the population presenting for diagnosis. Importantly, back-projection does not estimate the incidence of community transmission, because it includes both infections derived from community transmission as well as those imported from outside the population. Instead, back-projection provides an assessment of the incidence of infections aggregated over all sources, which can then be used to provide information about the effectiveness of control measures and the pattern of future case diagnoses. Conversely, transmission models incorporate the dynamics of different sources of infection making up the aggregated infection incidence, and for this reason back-projection is useful for calibrating transmission models.', 'cite_spans': [], 'section': 'Estimating infection incidence ::: Methods', 'ref_spans': []}, {'text': 'In view of the high dimensionality of the model, the analysis additionally makes use of smoothing, which was implemented using a simple moving average of width one week. This smoothing was applied first to the daily diagnosis counts, and then in each iteration of the computational algorithm for fitting the high-dimensional linear Poisson model [12, 13]. Confidence intervals were obtained using parametric bootstrap methods [6]. A total of 1000 bootstrap replications of the diagnosis data were generated and the back-projection analysis was applied to each, after which the 2.5% and 97.5% percentiles of the incidence estimates were used as 95% confidence intervals. Adjustment for over-dispersion relative to the Poisson model was incorporated by generating replications from a negative binomial distribution with mean equal to the model fitted values and variance inflated by the over-dispersion factor [6, p. 199]. As in previous smoothed back-projection analyses, the model fitted values for generating bootstrap replications were computed using an unsmoothed back-projection analysis [13]. As described earlier in this section, all data and software for implementing these analyses is available in the public domain.', 'cite_spans': [{'start': 347, 'end': 349, 'mention': '12', 'ref_id': 'BIBREF3'}, {'start': 351, 'end': 353, 'mention': '13', 'ref_id': 'BIBREF4'}, {'start': 427, 'end': 428, 'mention': '6', 'ref_id': 'BIBREF19'}, {'start': 909, 'end': 910, 'mention': '6', 'ref_id': 'BIBREF19'}, {'start': 1093, 'end': 1095, 'mention': '13', 'ref_id': 'BIBREF4'}], 'section': 'Estimating infection incidence ::: Methods', 'ref_spans': []}, {'text': 'As introduced above, estimation of infection incidence using back-projection requires an assumed probability distribution pd for the number of days D between infection and diagnosis. For COVID-19, there are two primary phases contributing to this diagnosis distribution: the incubation period between infection and the development of symptoms, and the testing period between symptoms and final diagnosis. These two components of the diagnosis distribution will be referred to as the incubation distribution and the testing distribution, respectively. Information about the incubation distribution has been published [14] and has been incorporated into transmission modelling used by the Australian government [15, 16]. Based on this information, the analysis presented here used a log-normal incubation distribution with an average of 5.2 days and a 95% percentile of 12.5 days [14]. Likewise, modelling used by the Australian government has assumed a mean testing period of 2 days [16], which is reflected here using a gamma distribution with rate parameter 0.6 per day and shape parameter 1.2. This gamma distribution was chosen so as to closely approximate an exponential distribution with mean 2 days, but with zero probability density at the time origin. Using the incubation and testing distributions, and assuming that the two periods are statistically independent, the diagnosis distribution is the probabilistic convolution of the two distributions. Figure 1 plots the probability density function f(d) and the cumulative distribution function F(d) for the diagnosis period D, as well as the underlying incubation and testing distributions. The delay between infection and diagnosis is consequently assumed to have a mean of 7.2 days and 95% percentile of 15.1 days. To facilitate the linear Poisson regression analysis of discrete-time daily data, the diagnosis distribution is represented as a discretised version of the continuous-time distribution:Since misspecification of the diagnosis distribution can bias the infection incidence estimates, sensitivity analyses were conducted to assess the robustness of the primary conclusions. These were conducted by repeating the back-projection analysis with both short and long diagnosis distributions. The short distribution used an incubation distribution with mean 4.1 days (lower 95% confidence limit reported in [14]), combined with a testing distribution having double the rate of diagnosis and therefore half the mean time from symptoms to diagnosis. The long diagnosis distribution was constructed similarly, with mean incubation period 7.0 days (upper 95% confidence limit) and half the rate of diagnosis or double the mean time from symptoms to diagnosis. The sensitivity analyses therefore had mean diagnosis periods from 5.1 to 11.0 days, compared to the assumed mean of 7.2 days.\\n', 'cite_spans': [{'start': 617, 'end': 619, 'mention': '14', 'ref_id': 'BIBREF5'}, {'start': 710, 'end': 712, 'mention': '15', 'ref_id': 'BIBREF6'}, {'start': 714, 'end': 716, 'mention': '16', 'ref_id': 'BIBREF7'}, {'start': 879, 'end': 881, 'mention': '14', 'ref_id': 'BIBREF5'}, {'start': 983, 'end': 985, 'mention': '16', 'ref_id': 'BIBREF7'}, {'start': 2375, 'end': 2377, 'mention': '14', 'ref_id': 'BIBREF5'}], 'section': 'Incubation, testing and diagnosis periods ::: Methods', 'ref_spans': [{'start': 1459, 'end': 1467, 'mention': 'Figure 1', 'ref_id': 'FIGREF0'}]}, {'text': 'An implicit assumption of the presentation of the diagnosis distribution is that it is a proper distribution in the sense that it sums to 1. In practice this may not be the case because some infected individuals may remain undetected by the testing regime, particularly those with asymptomatic infection. In this case the diagnosis distribution is a sub-distribution, in the sense thatHere P is the proportion of infections that are detected by the testing regime, and 1 − P is the proportion of undetected infections. If P < 1 then the diagnosis distribution has cumulative distribution function P × F(d), which means that the linear Poisson model is re-scaled by a factor P. The consequence of this re-scaling is that, in the presence of undetected infections, the back-projection estimates  are estimates of P × λt rather than the actual infection incidence λt [6, pp. 201–202]. Thus, back-projection applied to diagnosis counts may produce under-estimates of infection incidence, particularly if the extent of undetected infections is substantial.', 'cite_spans': [{'start': 865, 'end': 866, 'mention': '6', 'ref_id': 'BIBREF19'}], 'section': 'Undetected infections ::: Methods', 'ref_spans': []}, {'text': 'Despite the potential for under-estimation, there are two key points that make back-projection estimates useful even in the presence of undetected infections. Firstly, it is possible to make a range of assumptions about P or to estimate it from hospitalisation data, and then to inflate the back-projection estimates by a factor of 1/P in order to adjust for undetected infections [16–18]. For example, the COVID-19 transmission model used by the Australian government [16], as well as other published models [19], incorporate assumptions about the net proportion of cases that present for diagnosis, which corresponds to P, and which can be used straightforwardly to adjust the estimates for undetected infections. One published estimate of P used by the Australian government is 0.93, which would suggest that the infection incidence estimates need to be inflated by 7.5% [17, 18]. Secondly, assuming that P is relatively stable over time, or in other words that the testing regime is relatively stable over time, the shape of the infection incidence curve will be unchanged in the presence of undetected infections. In this case, qualitative features of back-projection estimates, such as the timing of peak incidence and its temporal association with control measures, are unaffected by the existence of undetected infections.', 'cite_spans': [{'start': 382, 'end': 384, 'mention': '16', 'ref_id': 'BIBREF7'}, {'start': 385, 'end': 387, 'mention': '18', 'ref_id': 'BIBREF9'}, {'start': 470, 'end': 472, 'mention': '16', 'ref_id': 'BIBREF7'}, {'start': 510, 'end': 512, 'mention': '19', 'ref_id': 'BIBREF10'}, {'start': 875, 'end': 877, 'mention': '17', 'ref_id': 'BIBREF8'}, {'start': 879, 'end': 881, 'mention': '18', 'ref_id': 'BIBREF9'}], 'section': 'Undetected infections ::: Methods', 'ref_spans': []}, {'text': 'Estimates of infection incidence were compared with the timing of key government control measures. Following early measures such as limiting outdoor and indoor gathering sizes to hundreds, the Australian government implemented staged restrictions during the period 20 March to 31 March [20]. The initial stage involved border closures (20 March) and Stage 1 social distancing restrictions (23 March) including prohibition of many types of face-to-face business and entertainment activities. These were followed by Stages 2 and 3 social distancing restrictions (26–31 March), which included limiting gatherings to two people and restrictions on leaving the home except for essential purposes.', 'cite_spans': [{'start': 287, 'end': 289, 'mention': '20', 'ref_id': 'BIBREF12'}], 'section': 'Control measures ::: Methods', 'ref_spans': []}, {'text': 'The cumulative daily diagnosis counts for Australia are presented in Figure 2 (panel A) for the period subsequent to the national total reaching 50 cases. As at 8 May, 6943 diagnoses were reported nationally. In addition, data from the three states with the largest outbreaks, comprising over 80% of the total epidemic, are also provided. By viewing the diagnosis counts on the logarithmic scale, it is seen that the growth of the epidemics in each state followed a similar pattern to the national trajectory. The smoothed daily national diagnosis counts are also shown in Figure 2 (panel B), together with the timing of key national government control measures. With respect to the pattern of daily diagnoses, border closures and Stage 1 restrictions were accompanied by a continued increase in the incidence of confirmed cases, whereas Stages 2 and 3 restrictions were accompanied by a peak and decline in incidence of new case diagnoses.\\n', 'cite_spans': [], 'section': 'Infection diagnosis ::: Results', 'ref_spans': [{'start': 69, 'end': 77, 'mention': 'Figure 2', 'ref_id': 'FIGREF1'}, {'start': 573, 'end': 581, 'mention': 'Figure 2', 'ref_id': 'FIGREF1'}]}, {'text': 'Figure 3 (panel A) shows the back-projection estimates of daily new infections. The estimates indicate a peak in the incidence of new infections at around 20 March. In contrast to the trend observed in the diagnosis data, border closures and Stage 1 restrictions were accompanied by a peak and decline in the incidence of new infections. To assess model fit and uncertainty, standardised Poisson residuals of observed daily diagnosis counts compared to fitted daily diagnosis counts were inspected and found to be centred around zero indicating good model fit. However, over-dispersion relative to the Poisson model was evident, with an estimated over-dispersion parameter of σ2 = 1.9. This over-dispersion was incorporated into the calculation of 95% confidence intervals displayed in Figure 3.\\n', 'cite_spans': [], 'section': 'Infection incidence ::: Results', 'ref_spans': [{'start': 0, 'end': 8, 'mention': 'Figure 3', 'ref_id': 'FIGREF2'}, {'start': 786, 'end': 794, 'mention': 'Figure 3', 'ref_id': 'FIGREF2'}]}, {'text': 'An important feature of Figure 3 (panel A) is the increase in variability of the back-projection estimates in the very recent past, which is a well-known property of back-projection analyses [6]. This reflects the fact that only a few diagnoses are the result of an infection in the very recent past, so there is less information about infection incidence in the last few days. As demonstrated by the widths of the confidence intervals, there is more information about earlier incidence and about the timing of peak incidence relative to the timing of government control measures.', 'cite_spans': [{'start': 192, 'end': 193, 'mention': '6', 'ref_id': 'BIBREF19'}], 'section': 'Infection incidence ::: Results', 'ref_spans': [{'start': 24, 'end': 32, 'mention': 'Figure 3', 'ref_id': 'FIGREF2'}]}, {'text': 'Due to the delay in diagnosing an infection, only a proportion of all past infections will be diagnosed at any given point in time. By subtracting the cumulative diagnoses from the cumulative infection estimates, it is possible to estimate the number of infections that remain undiagnosed. Figure 3 (panel B) provides the proportion of undiagnosed infections over time together with 95% confidence intervals. At the height of the epidemic, only one-quarter to one-half of all infections were diagnosed. Over time this proportion has increased and is approaching 1 as the incidence of new infections has become lower. Consistent with the confidence intervals in panel A of Figure 3, panel B displays some uncertainty about the undiagnosed proportion in the very recent past. Note that panel B of Figure 3 represents the proportion of all past infections that are undiagnosed, not the proportion of active infections. Furthermore, it is important to note that the undiagnosed infections do not include the undetected infections which are addressed in the next subsection.', 'cite_spans': [], 'section': 'Undiagnosed infections ::: Results', 'ref_spans': [{'start': 290, 'end': 298, 'mention': 'Figure 3', 'ref_id': 'FIGREF2'}, {'start': 672, 'end': 680, 'mention': 'Figure 3', 'ref_id': 'FIGREF2'}, {'start': 795, 'end': 803, 'mention': 'Figure 3', 'ref_id': 'FIGREF2'}]}, {'text': 'Sensitivity analyses exploring the impact of different diagnosis distributions and adjustment for undetected infections, are displayed in Figure 4. The short diagnosis distribution (mean 5.1 days) shifted the incidence peak later while the long diagnosis distribution (mean 11.0 days) shifted it earlier. However, since these are extreme assumptions on the diagnosis distribution, there was not a high level of sensitivity and the broad conclusion that peak incidence occurred contemporaneously with the initial control measures is still applicable. Adjustment of incidence estimates for undetected infections raised the peak infection incidence but did not alter its position. Overall, given estimates of detection rates used by the Australian government [17, 18], adjustment for undetected infections had little effect on the analysis.\\n', 'cite_spans': [{'start': 757, 'end': 759, 'mention': '17', 'ref_id': 'BIBREF8'}, {'start': 761, 'end': 763, 'mention': '18', 'ref_id': 'BIBREF9'}], 'section': 'Sensitivity analyses ::: Results', 'ref_spans': [{'start': 138, 'end': 146, 'mention': 'Figure 4', 'ref_id': 'FIGREF3'}]}, {'text': 'As displayed in Figure 5 (panel A), the infection incidence estimates indicate that peak incidence and the initial government control measures were preceded by two weeks of exponential growth in cumulative infections (solid black points). This is reflected by a strongly linear trend on the logarithmic scale during these two weeks. Subsequently, the growth shifts from exponential to sub-exponential growth in cumulative infections. One possible explanation for this shift is the timing of the border closures and Stage 1 social distancing restrictions, reflected by the shaded region. By delaying the timing of this transition to sub-exponential growth by one week (green points) or moving it forward one week (red points), the effect of earlier or later control on the overall infection numbers can be investigated. Figure 5 (panel B) reflects the effect of these shifts on the estimates of infection incidence. Based on these estimates, one week later control would correspond with 35 050 infections (green line), an almost fivefold increase in total infections, whereas one week earlier control would correspond with only 1735 infections (red line).\\n', 'cite_spans': [], 'section': 'Peak incidence and control measures ::: Results', 'ref_spans': [{'start': 16, 'end': 24, 'mention': 'Figure 5', 'ref_id': 'FIGREF4'}, {'start': 819, 'end': 827, 'mention': 'Figure 5', 'ref_id': 'FIGREF4'}]}, {'text': 'Consistent with Figure 2 (panel A), repeating the analyses on data from each of the three states with greater than 1000 diagnoses (New South Wales, Victoria and Queensland) yielded very similar trends to those displayed in Figure 5 for the national data (results not shown).', 'cite_spans': [], 'section': 'Further analyses ::: Results', 'ref_spans': [{'start': 16, 'end': 24, 'mention': 'Figure 2', 'ref_id': 'FIGREF1'}, {'start': 223, 'end': 231, 'mention': 'Figure 5', 'ref_id': 'FIGREF4'}]}, {'text': 'Also available are short-term projections of new daily diagnosis counts based on the fitted models from the back-projection analyses. Using the fundamental linear relationship between mean diagnoses and mean infections, as expressed in the Methods section, the fitted model can be used to calculate the mean number of diagnoses for a short period into the future. This is possible because the number of diagnoses in the next few days depends primarily on past infection incidence. Figure 6 presents 5-day forecasts for the month of April, subsequent to the date of the final government control measures (31 March). For these forecasts the back-projection analyses were repeated using data that were available prior to multiple days in April. The 5-day forecasts from these analyses were then compared with the actual (smoothed) counts that were subsequently observed. Figure 6 demonstrates the forecasts are a worthwhile predictive tool and a useful by-product of a back-projection analysis. Of course, other simple forecasting methods, such as extrapolation of a regression model fitted to the diagnosis counts, might also provide reasonable predictions. However, the forecasts in Figure 6 make use of the underlying infection incidence that drives future diagnoses, whereas simple extrapolation does not, which can lead to unreliable extrapolated diagnosis counts [6, Section 7.6].\\n', 'cite_spans': [{'start': 1367, 'end': 1368, 'mention': '6', 'ref_id': 'BIBREF19'}], 'section': 'Further analyses ::: Results', 'ref_spans': [{'start': 481, 'end': 489, 'mention': 'Figure 6', 'ref_id': 'FIGREF5'}, {'start': 868, 'end': 876, 'mention': 'Figure 6', 'ref_id': 'FIGREF5'}, {'start': 1182, 'end': 1190, 'mention': 'Figure 6', 'ref_id': 'FIGREF5'}]}, {'text': 'Daily COVID-19 diagnosis counts are a valuable data resource providing information beyond the number of positive tests. Back-projection has been an important analytical method in prior epidemiological contexts, particularly HIV/AIDS, and has great potential for the analysis of COVID-19 data. This paper has presented a suite of analyses that can be undertaken on COVID-19 diagnosis data to assess infection incidence using the method of back-projection.', 'cite_spans': [], 'section': 'Discussion', 'ref_spans': []}, {'text': 'In Australia, the estimates of infection incidence show a peak around 20 March, less than two weeks after reaching 100 diagnoses. They also show that the initial key national government interventions, particularly border closures and Stage 1 social distancing restrictions, corresponded temporally with a peak and subsequent decline in infection incidence. Nonetheless, subsequent restriction stages are likely to have played a role in driving incidence even lower and preventing resurgence.', 'cite_spans': [], 'section': 'Discussion', 'ref_spans': []}, {'text': \"The estimates suggest that a one week delay in the timing of peak infection would have corresponded with a fivefold increase in the total number of people infected. Likewise, shifting the peak one week earlier would have yielded a decrease of similar magnitude in the total number of infections. Furthermore, the estimates suggest that at the height of the epidemic, the proportion of undiagnosed infections was half to three-quarters of the total infection pool. None of this information is directly discernible from inspection of the daily diagnosis counts, which continued to increase after the first stages of government control measures. This shows that the temporal association between control measures and the course of the epidemic may be quite different when the epidemic's extent is measured using estimated infection incidence rather than observed diagnosis counts. Back-projection estimates therefore provide useful additional information for assessing the effectiveness of control measures, as well as for understanding the evolution of infection incidence during the initial wave of infections and in the event of a second wave.\", 'cite_spans': [], 'section': 'Discussion', 'ref_spans': []}, {'text': 'Back-projection may produce under-estimates of infection incidence if a proportion of infected individuals never have their infection diagnosed. This can be adjusted for by inflating the incidence estimates by a factor that depends on the detection proportion, which is an approach previously proposed in the context of HIV/AIDS [6, pp. 201–202]. Based on modelling used by the Australian government [17, 18], which estimates the detection proportion to be P = 0.93, this corresponds to a 7.5% inflation of the incidence estimates, which led to a slightly higher peak incidence when adjusting for undetected infections. Importantly, however, the shape of the infection incidence curve, and hence the timing of key features relative to the timing of government control measures, are unaffected by this under-estimation. Thus, the broad conclusions of the analysis are robust to this feature of the data. As the outbreak progresses, changes in the availability of testing may affect the proportion of undetected infections, which makes P time-dependent and affects the infection incidence curve. This could be accounted for by modelling a time-varying P, however, this is unlikely to be a major issue for the current analysis since testing was restricted to confirmatory case diagnosis.', 'cite_spans': [{'start': 330, 'end': 331, 'mention': '6', 'ref_id': 'BIBREF19'}, {'start': 401, 'end': 403, 'mention': '17', 'ref_id': 'BIBREF8'}, {'start': 405, 'end': 407, 'mention': '18', 'ref_id': 'BIBREF9'}], 'section': 'Discussion', 'ref_spans': []}, {'text': 'Back-projection estimates are more uncertain in the recent past because few cases were infected in the recent past. Also, back-projection estimates may be sensitive to misspecification of the diagnosis distribution. For example, estimates of the incubation distribution may be based on data subject to retrospective ascertainment of infection dates and therefore subject to uncertainties and biases. Sensitivity analyses are therefore important in any back-projection analysis. For the current analysis, the primary qualitative conclusions concerning the pattern of infection incidence were robust to a range of diagnosis distribution assumptions.', 'cite_spans': [], 'section': 'Discussion', 'ref_spans': []}, {'text': 'As the COVID-19 pandemic evolves the diagnosis distribution may change, making it time-dependent. Such changes may result from introducing effective treatments, changes in the operational definition of disease, and changes in testing policy. Individual-specific factors may also affect the incubation distribution, such as age-dependencies. These complexities were present for HIV/AIDS, and generalised back-projection methodology has been developed [6, 7, 13]. Thus, while such influences would not have a substantive effect early in an outbreak, generalised methods may be required for COVID-19 in the longer term.', 'cite_spans': [{'start': 451, 'end': 452, 'mention': '6', 'ref_id': 'BIBREF19'}, {'start': 454, 'end': 455, 'mention': '7', 'ref_id': 'BIBREF20'}, {'start': 457, 'end': 459, 'mention': '13', 'ref_id': 'BIBREF4'}], 'section': 'Discussion', 'ref_spans': []}, {'text': 'This paper has applied back-projection to daily diagnosis counts, however, it could also potentially be applied to routinely reported daily mortality counts. This would require detailed information on the probability distribution of the time from infection to death. Since death from COVID-19 infection is likely to have less under-reporting than case diagnosis, death data would have the advantage of being more complete. Nonetheless, only a small proportion of infections result in death, so back-projection estimates from death data would need to be adjusted using external information about the mortality rate. Another issue is that death data may contain substantially less information than diagnosis data. For example, in Australia where approximately 100 deaths have occurred, such analyses would not be highly informative. Nonetheless, for high mortality countries back-projection of death counts may be informative about past infection incidence and future mortality.', 'cite_spans': [], 'section': 'Discussion', 'ref_spans': []}, {'text': 'An important line of future research is to explore the use of back-projection estimates of detectable infection incidence for calibrating transmission dynamics models. Such models make critical parameter assumptions which should lead the models to track the infection incidence estimated by back-projection. Such an approach to transmission model calibration was used effectively in the HIV/AIDS context and may have applications for COVID-19 modelling [9, 21]. This should be a high priority for COVID-19 epidemiological research, and would require close collaboration between modellers and statisticians.', 'cite_spans': [{'start': 454, 'end': 455, 'mention': '9', 'ref_id': 'BIBREF22'}, {'start': 457, 'end': 459, 'mention': '21', 'ref_id': 'BIBREF13'}], 'section': 'Discussion', 'ref_spans': []}, {'text': 'Future work should also focus on developing automated analytics systems for continuous updating of estimates and projections as the outbreak evolves on a daily basis. Such analyses would add value to the descriptive dashboards that have proliferated in the wake of the highly successful Johns Hopkins University Coronavirus Visual Dashboard [1, 2]. Analytics software for serial updating of estimates of basic reproduction numbers has previously been advocated [22]. However, currently there is only limited implementation of back-projection methodology for COVID-19, through the Coronavirus 10-day Forecast resource from The University of Melbourne [4] and the epiforecasts resource from the London School of Hygiene and Tropical Medicine [23]. More expansive use of back-projection would allow continuous monitoring for resource planning and identification of key milestones, such as an infection incidence peak or a resurgence in infections as control measures are eased.', 'cite_spans': [{'start': 342, 'end': 343, 'mention': '1', 'ref_id': 'BIBREF0'}, {'start': 345, 'end': 346, 'mention': '2', 'ref_id': 'BIBREF11'}, {'start': 462, 'end': 464, 'mention': '22', 'ref_id': 'BIBREF14'}, {'start': 651, 'end': 652, 'mention': '4', 'ref_id': 'BIBREF17'}, {'start': 741, 'end': 743, 'mention': '23', 'ref_id': 'BIBREF15'}], 'section': 'Discussion', 'ref_spans': []}], 'paragraphs': [{'text': '', 'id': '00009'}], 'title': 'Back-projection of COVID-19 diagnosis counts to assess infection incidence and control measures: analysis of Australian data', 'sha': '9b4b5e5bb9941457aeb18f187963086719561fcb', 'doi': '10.1017/s0950268820001065', 'score': 9.97089958190918}, {'rank': 10, 'abstract': 'OBJECTIVE: The aim of this study was to explore any age‐related change in the incubation period of COVID‐19, specifically any difference between older (aged ≥65 years) and younger adults. METHODS: Based on online data released officially by 21 Chinese cities from January 22 to February 15, 2020, the incubation period of COVID‐19 patients who had travelled to Hubei was studied according to age. Previous studies were reviewed and compared. RESULTS: The study recruited 136 COVID‐19 patients who had travelled to Hubei during January 5‐31, 2020, stayed for 1‐2 days, and returned with symptom onset during January 10‐February 6, 2020. The median age was 50.5 years (range 1‐86 years), and 22 patients (16.2%) were aged ≥65 years. The age‐stratified incubation period was U‐shaped with higher values at extremes of age. The median COVID‐19 incubation period was 8.3 (90% confidence interval [CI], 7.4‐9.2) days for all patients, 7.6 (90% CI, 6.7‐8.6) days for younger adults, and 11.2 (90% CI, 9.0‐13.5) days for older adults. The 5th/25th/75th/90th percentiles were 2.3/5.3/11.3/14.2 days for all, 2.0/5.0/10.5/13.2 days for younger adults, and 3.1/7.8/14.4/17.0 days for older adults. There were 11 published studies on COVID‐19 incubation periods up to March 30, 2020, reporting means of 1.8‐7.2 days, and medians of 4‐7.5 days, but there was no specific study on the effect of age on incubation period. One study showed that severe COVID‐19 cases, which included more elderly patients, had longer incubation periods. CONCLUSION: Based on 136 patients with a travel history to Hubei, the epicenter of COVID‐19, the COVID‐19 incubation period was found to be longer in older adults. This finding has important implications for diagnosis, prevention, and control of COVID‐19.', 'body_text': [{'text': 'Coronavirus disease 2019 (COVID‐19), caused by the emergence of the novel pathogen severe acute respiratory syndrome coronavirus 2 (SARS‐CoV‐2), has spread with enormous speed and scale since its first reports in Wuhan, Hubei Province, China, in late December 2019.\\n1\\n A pandemic was declared by the World Health Organization (WHO)\\n2\\n on March 11, 2020. As of April 5, 2020, the reported cumulative number of confirmed patients had reached 1 133 758 globally in over 100 countries and territories, associated with 62 784 deaths.\\n3\\n\\n', 'cite_spans': [], 'section': 'INTRODUCTION', 'ref_spans': []}, {'text': 'Older adults are at higher risk of contracting COVID‐19, developing a severe disease, and dying from the disease,\\n4\\n, \\n5\\n, \\n6\\n similar to what was experienced during the severe acute respiratory syndrome (SARS) epidemic in 2003.\\n7\\n, \\n8\\n In China, 90% of the COVID‐19 deaths occurred in those older than 60 years, and 20% of all deaths were people aged older than 80 years.\\n4\\n In Europe, where the proportion of elderly population (19% aged over 65 years) is higher, the impact is greater: over 95% of the COVID‐19 deaths has occurred in those aged over 60 years, and more than 50% of all deaths have been people aged over 80 years.\\n6\\n Because of immune senescence and common occurrence of secondary immunodeficiency in old age, the presentations and disease course of older adults in response to infectious disease may be altered.\\n9\\n, \\n10\\n\\n', 'cite_spans': [], 'section': 'INTRODUCTION', 'ref_spans': []}, {'text': 'A review of SARS in old age\\n7\\n notes that, according to clinical experience, its incubation period, defined as the time between infection exposure and symptom onset, is apparently longer in older adults, but there is a lack of studies on the incubation periods of SARS‐associated coronavirus, specifically in older adults. Knowing the incubation period of this novel virus is relevant for diagnosis, surveillance, prevention, and control of COVID‐19, and it is important to know whether its incubation period in older adults, who represent a vulnerable group to this disease, differs from that in younger adults. The objective of this study was therefore to explore whether there is any age‐related change in the incubation period of COVID‐19, specifically any difference between older (aged ≥65 years) and younger adults.', 'cite_spans': [], 'section': 'INTRODUCTION', 'ref_spans': []}, {'text': 'For nearly every city in China, daily information on the list of COVID‐19 cases is released officially to the Chinese social media WeChat accounts of respective cities. However, only a minority of cities include in their official release clear information on the day of symptom onset, which is required in estimating incubation period. The information released between January 22 and February 15, 2020 was captured and compiled into a list of patients with COVID‐19 from cities that reported the day of symptom onset and those who had travelled to Hubei, the epicenter of the COVID‐19 epidemic. The following data were entered into an Excel (Microsoft) spreadsheet for COVID‐19 cases reported between January 22 and February 15, 2020 from 21 Chinese cities outside Hubei: patient case number, age, sex, first day to Hubei, last day in Hubei, and first day with symptoms. The data were collected by another person acknowledged at the end of this paper.', 'cite_spans': [], 'section': 'Data source and collection ::: PATIENTS AND METHODS', 'ref_spans': []}, {'text': 'For this study, only those COVID‐19 patients who stayed in Hubei for at most two calendar days were included. The day of exposure was taken as the first day to Hubei if the patient had stayed in Hubei for one calendar day; or as the middle of the first day and second day in Hubei if the patient had stayed for two calendar days. By excluding COVID‐19 patients who had stayed in Hubei for more than 2 days, a narrowly defined exposure window was ensured, thus reducing the uncertainty in estimating the day of exposure. The incubation period for each COVID‐19 patient was derived from the number of days between exposure and symptom onset. Parametric and non‐parametric methods were used in the statistical analysis. The COVID‐19 incubation periods for different age groups of 15‐year interval size were analyzed in terms of their respective mean, median, 25th, and 75th percentiles. Next, the frequency distributions of COVID‐19 incubation periods of the three age groups—children (age 0‐14 years), younger adults (age 15‐64 years), and older adults (aged ≥65 years)—were plotted and compared. The cumulative frequency distributions for the incubation periods for both younger and older adult groups were analyzed using the cumulative frequency analysis software CumFreq.\\n11\\n This computer program fits the observed cumulative frequency data into a best‐fit theoretical distribution by the regression method. Observed data are plotted in ranked order of increasing probability, and the most appropriate theoretical distribution is selected based on the lowest average of absolute values of the differences between observed and calculated cumulative frequency values. Further details are available at the CumFreq website.\\n11\\n With the use of CumFreq, probability distribution curve fitting was performed for the incubation periods for both younger and older adult groups, and their medians, quartiles, 5th, and 90th percentiles estimated.', 'cite_spans': [], 'section': 'Data analysis ::: PATIENTS AND METHODS', 'ref_spans': []}, {'text': 'The author searched the studies reporting on incubation periods of COVID‐19 archived in PubMed (published) and medRxiv (unpublished) until March 30, 2019. The following search terms were used: (“incubation”) and (“COVID‐19” or “SARS‐CoV‐2”). Studies reporting on the incubation period of COVID‐19 were included for further evaluation and comparison with this study. Information on study place, sample size, mean/median age, percentage of subjects studied who were elderly, and incubation period was extracted from these previous studies.', 'cite_spans': [], 'section': 'Literature review ::: PATIENTS AND METHODS', 'ref_spans': []}, {'text': 'A total of 136 COVID‐19 patients were recruited into this study. They had travelled to Hubei from 21 Chinese cities between January 5 and January 31, 2020, had stayed there for at most two calendar days, and had returned to their respective cities with symptom onset between January 10 and February 6, 2020. The median age was 50.5 years (range 1‐86 years, interquartile range, 36.8‐60.5 years), 22 (16.2%) patients were aged 65 years or above, and 72 (53%) patients were male. By parametric tests, the mean incubation period of the 136 patients was 8.5 days (95% confidence interval [CI], 7.8‐9.2 days), with a median of 8.3 days (95% CI, 7.6‐9.0 days) and an interquartile range of 4.9‐12.0 days.', 'cite_spans': [], 'section': 'RESULTS', 'ref_spans': []}, {'text': 'When plotted against age with group size of 15 years, the incubation period showed a U‐shaped curve with higher values at extremes of age (Figure 1). The mean/median incubation period values dropped from 12.0/13.5 days for age 0‐14 years (n = 4) to 8.9/9.5 days for age 15‐29 years (n = 8), 8.4/8.5 days for age 30‐44 years (n = 36), 7.2/6.0 days for age 45‐59 years, and then rose to 9.6/9.0 days for age 60‐74 years (n = 29) and 13.2/13.0 days for age 75‐89 years (n = 6).', 'cite_spans': [], 'section': 'RESULTS', 'ref_spans': [{'start': 146, 'end': 147, 'mention': '1', 'ref_id': 'FIGREF0'}]}, {'text': 'There were 22 patients aged ≥65 years, 110 patients aged 15‐64 years, and four patients aged 0‐14 years in this study. Their frequency distributions of COVID‐19 incubation periods did not fit into a normal pattern (Figure 2). There was a skew towards the left for younger adults, but for older adults, a skew towards the right with wider variation was noted. Cumulative frequency distributions of the entire study population, the younger (15‐64 years), and older (≥65 years) adult age groups were further analyzed by the CumFreq\\n11\\n software (Figure 3). Kumaraswamy distribution returned as the best fit for the entire study population and the younger adult group, while the mirrored generalized Gumbel (also known as the log‐Weibull distribution) returned as the best fit for the older adult group. This gave estimates of the non‐parametric values of the COVID‐19 incubation period for the total study population, and the younger and older adult age groups, as shown in Table 1. The medians were 8.3 (90% CI, 7.4‐9.2) days for all patients, 7.6 (90% CI, 6.7‐8.6) days for younger adults, and 11.2 (90% CI, 9.0‐13.5) days for older adults. The 5th/25th/75th/90th percentiles were 2.3/5.3/11.3/14.2 days for all patients, 2.0/5.0/10.5/13.2 days for younger adults, and 3.1/7.8/14.4/17.0 days for older adults.', 'cite_spans': [], 'section': 'RESULTS', 'ref_spans': [{'start': 222, 'end': 223, 'mention': '2', 'ref_id': 'FIGREF1'}, {'start': 550, 'end': 551, 'mention': '3', 'ref_id': 'FIGREF2'}, {'start': 977, 'end': 978, 'mention': '1', 'ref_id': 'TABREF0'}]}, {'text': 'The literature review retrieved 11 published\\n5\\n, \\n12\\n, \\n13\\n, \\n14\\n, \\n15\\n, \\n16\\n, \\n17\\n, \\n18\\n, \\n19\\n, \\n20\\n, \\n21\\n and two unpublished\\n22\\n, \\n23\\n studies that reported on COVID‐19 incubation periods. Their details are listed in Table 2.', 'cite_spans': [], 'section': 'RESULTS', 'ref_spans': [{'start': 226, 'end': 227, 'mention': '2', 'ref_id': 'TABREF1'}]}, {'text': 'The first estimate of the COVID‐19 incubation period was reported by Li et al\\n12\\n as a mean of 5.2 days (95% CI, 4.1‐7.0) and a 95th percentile of 12.5 days (95% CI, 9.2‐18), based on their study on the exposure information of 10 confirmed COVID‐19 patients in Wuhan, China (Table 2). This is slightly longer than the incubation period estimated for SARS, with a median of 4.0 days (95% CI, 3.6‐4.4) and a 95th percentile of 10.6 days (95% CI, 8.9‐12.2).\\n24\\n Subsequent published studies\\n5\\n, \\n12\\n, \\n13\\n, \\n14\\n, \\n15\\n, \\n16\\n, \\n17\\n, \\n18\\n, \\n19\\n, \\n20\\n, \\n21\\n on the COVID‐19 incubation period (Table 2) have reported means varying from 1.8 days to 7.2 days, medians of 4‐7.5 days, and 95th percentiles of 3.2‐14.6 days, which may be due to differences in methodologies and patient samples.', 'cite_spans': [], 'section': 'DISCUSSION', 'ref_spans': [{'start': 281, 'end': 282, 'mention': '2', 'ref_id': 'TABREF1'}, {'start': 592, 'end': 593, 'mention': '2', 'ref_id': 'TABREF1'}]}, {'text': 'This study of 136 COVID‐19 patients revealed a longer incubation period, with a mean of 8.5 days, an estimated median of 8.3 days, and an interquartile range of 5.3‐11.3 days. Interestingly, two unpublished studies\\n22\\n, \\n23\\n also reported longer COVID‐19 incubation periods with mean/median of 8.62/8.13 days and 9.0/8.62 days, respectively (Table 2). The longer incubation period reported by Jing et al,\\n22\\n who studied a large sample size of 1211 individuals who had been asymptomatic at their time of departure from Wuhan, may be related to the forward follow‐up methodology of a sufficiently long duration of 25 days until symptoms had developed so that those with longer incubation periods were not missed from sample collection. Tindale et al\\n23\\n attributed their finding of longer incubation period to missed intermediate exposure events due to pre‐symptomatic transmission and misperceived exposure times.', 'cite_spans': [], 'section': 'DISCUSSION', 'ref_spans': [{'start': 348, 'end': 349, 'mention': '2', 'ref_id': 'TABREF1'}]}, {'text': 'In this study, the incubation period of COVID‐19, when age‐stratified, displayed a U‐shaped curve with higher values at the extremes of age for the pediatric and geriatric age groups (Figure 1). A study by Cai et al\\n21\\n of COVID‐19 in 10 children (mean age, 6.2 years; range 0.25‐10.9 years) also reported a longer incubation period of 6.5 days in this age group (Table 2). The small sample size of four in the present study’s child group (age 0‐14 years) prevented further statistically significant analysis. With this U‐shaped age‐related finding of COVID‐19 incubation period, as well as the background of an apparent longer incubation period for SARS in old age from clinical experience in 2003,\\n7\\n and also a reported case of a 70‐year‐old COVID‐19 patient with a long incubation period of 27 days in Hubei,\\n25\\n the author proceeded to analyze the incubation period of COVID‐19 in older adults compared with younger adults.', 'cite_spans': [], 'section': 'DISCUSSION', 'ref_spans': [{'start': 191, 'end': 192, 'mention': '1', 'ref_id': 'FIGREF0'}, {'start': 370, 'end': 371, 'mention': '2', 'ref_id': 'TABREF1'}]}, {'text': 'The frequency distribution of COVID‐19 incubation period for 22 older adults (aged ≥65 years) was compared with that of 110 younger adults (aged 15‐64 years). From the frequency distributions of incubation periods for the two age groups, the younger adult group skewed towards the left (shorter incubation period) while the older adult group skewed towards the right (longer incubation period). Neither of the frequency distributions fit into a normal pattern (Figure 2). Sartwell\\n26\\n showed that for most infectious diseases, the frequency curves of the incubation period take the form of logarithmic normal distribution. Based on this observation, lognormal fitting of distribution data has been used to estimate the incubation period parameters of common respiratory viral infections.\\n24\\n However, this lognormal fitting has been challenged in the review by Nishiura,\\n27\\n who noted lack of validity in assuming lognormal distribution and the need for using different distributions to compare for best fit. Previous studies on COVID‐19 have also employed probability distribution fitting methods other than lognormal and some have come up with better fits (Table 2). Using the ComFreq software for cumulative frequency distribution fitting for patients of this study, Kumaraswamy distribution returned as the best fit for the younger adult group, while the mirrored generalized Gumbel (also known as the log‐Weibull distribution) returned as the best fit for the older adult group; the latter also displayed wider 90% confidence limits (Figure 3). Ninety percent instead of 95% confidence limits were used for the analysis because of the smaller sample size of 22 older adults. This gave estimates of the non‐parametric values of the COVID‐19 incubation period for the two age groups (Table 1). The estimated median, interquartile range, and 90th percentile of the COVID‐19 incubation period for older adults were 11.2, 7.8‐14.4, and 17.0 days, respectively, which are longer than the corresponding figures of 7.6, 5.0‐10.5, and 13.2 days for younger adults. The respective 90% confidence intervals of the incubation period distributions for younger and older groups did not overlap despite of the wider 90% confidence limits of the older group.', 'cite_spans': [], 'section': 'DISCUSSION', 'ref_spans': [{'start': 468, 'end': 469, 'mention': '2', 'ref_id': 'FIGREF1'}, {'start': 1546, 'end': 1547, 'mention': '3', 'ref_id': 'FIGREF2'}, {'start': 1165, 'end': 1166, 'mention': '2', 'ref_id': 'TABREF1'}, {'start': 1793, 'end': 1794, 'mention': '1', 'ref_id': 'TABREF0'}]}, {'text': 'Although previous studies on COVID‐19 incubation period did not look specifically at the effect of older age on incubation period, I reviewed these studies for any such clues. Of the ten published studies\\n5\\n, \\n12\\n, \\n13\\n, \\n14\\n, \\n15\\n, \\n16\\n, \\n17\\n, \\n18\\n, \\n19\\n, \\n20\\n on COVID‐19 incubation period (Table 2) that included adults, eight had information on mean or median age. Except for the first study by Li et al,\\n12\\n which included much older subjects in the parent population but without age information for the 10 patients studied in the incubation period subset, the remaining seven adult studies\\n5\\n, \\n13\\n, \\n14\\n, \\n16\\n, \\n18\\n, \\n19\\n, \\n20\\n had median age of 42‐52 years with around 15% aged over 65 years, which is not too dissimilar to this study population (median age, 50.5 years; 16.2% aged over 65 years). However, there was wide variation in the incubation period values reported from these studies (Figure 4), which may be due to differences in methodologies and patient samples. These factors, together with a lack of detailed information on the age structure and the small number of older adults recruited, made direct comparison with this study difficult. Nevertheless, when the clinical and epidemiological study of COVID‐19 by Tian et al\\n20\\n in Beijing was reviewed (Figure 4), it was noted that the incubation period for the severe group (whose subjects were older with a median age of 61.4 years and 43.5% aged over 65 years) had a longer median incubation period of 7.5 ± 7.2 days when compared with 6.5 ± 4.6 days for the milder group (whose subjects were younger with a median age of 44.5 years and 13% aged over 65 years). The unpublished study by Jing et al\\n22\\n reports on a long COVID‐19 incubation period (mean, 8.62 days; median, 8.13 days) that is close to the present figures, but their sample was younger (median age 40 years; 13.2% aged over 60 years) and a different methodology was used as discussed above.', 'cite_spans': [], 'section': 'DISCUSSION', 'ref_spans': [{'start': 908, 'end': 909, 'mention': '4', 'ref_id': 'FIGREF3'}, {'start': 1281, 'end': 1282, 'mention': '4', 'ref_id': 'FIGREF3'}, {'start': 299, 'end': 300, 'mention': '2', 'ref_id': 'TABREF1'}]}, {'text': 'The incubation period of an infection is influenced by infectious dose and immune response. Thus, the shorter COVID‐19 incubation period among travelers to Hubei compared with non‐travelers in the study by Leung\\n19\\n was attributed to exposure to a higher viral load among travelers to Hubei, the epicenter. The longer COVID‐19 incubation period observed in this study for older adults may be accounted for by the blunted immune response due either to age‐associated immune senescence or secondary immunodeficiency in old age.\\n7\\n, \\n9\\n, \\n10\\n The lack of a fever response in elderly subjects, the non‐specific geriatric presentations in an infectious illness (such as falls and delirium), and multi‐comorbidities might result in a delayed awareness of disease onset and its detection by a clinician.\\n6\\n Multiple exposures, instead of a well‐defined exposure event, might occur for frailer elderly people because of their need for close personal and nursing care. Though this latter point may be less relevant for subjects of this study, whose exposure time was narrowed to within two calendar days of travel to Hubei and who were likely to be fitter because of their ambulatory status, it is nevertheless relevant in the setting of long‐term care facilities and hospital wards. Multiple contact dates have been shown to be associated with longer incubation periods of over 10 days in SARS using computer simulation.\\n28\\n\\n', 'cite_spans': [], 'section': 'DISCUSSION', 'ref_spans': []}, {'text': 'The finding in this study of longer COVID‐19 incubation period for older patients has several implications for clinical practice, public‐health policy, and research on this disease. Knowing the incubation period of an infectious disease aids in its detection and diagnosis. Knowledge that an older adult can have a longer COVID‐19 incubation period will minimize underdiagnosis and increase suspicion despite atypical presentations. Delayed diagnoses of SARS and COVID‐19 in older adults were associated with outbreaks in nursing homes and long‐term care facilities.\\n7\\n, \\n29\\n A longer 90th percentile of 17 days (90% CI, 13.9‐17.6 days) for older adults means that 10% of older subjects would have a COVID‐19 incubation period of over 17 days. Figure 3 shows that only 72% of older adults had a COVID‐19 incubation period of ≤14 days, compared with 93% for younger adults. Thus, while the current quarantine period of 14 days that has been adopted for surveillance, prevention, and control of COVID‐19 might appear adequate for younger adults (7% fell outside this period), this would be inadequate for older adults (28% fell outside this period). Older adults would therefore require a longer period of isolation and observation; extending the quarantine period from 14 to 17 days would increase the coverage from 72% to 90%. A longer incubation period may also be associated with a higher rate of asymptomatic, pre‐symptomatic, and subclinical infection towards the end of the long incubation period.\\n23\\n Asymptomatic and pre‐symptomatic SARS‐CoV‐2 infections have been described in half of the 23 residents (mean age, 80.7 years) who tested positive for COVID‐19 during an outbreak at a long‐term‐care skilled‐nursing facility in Washington State, United States.\\n30\\n While there has been no documented asymptomatic transmission,\\n31\\n pre‐symptomatic transmission (transmission of COVID‐19 during incubation period before symptom onset) has been documented during contact tracing and investigation of infection clusters.\\n32\\n, \\n33\\n\\n', 'cite_spans': [], 'section': 'DISCUSSION', 'ref_spans': [{'start': 751, 'end': 752, 'mention': '3', 'ref_id': 'FIGREF2'}]}, {'text': 'The strengths of this study are, first, a clear exposure day by sampling travelers who had stayed in Hubei for at most two calendar days and, second, the inclusion of enough older adults to compare their incubation period with younger adults. Previous studies\\n5\\n, \\n14\\n, \\n17\\n, \\n19\\n based on traveler history to Hubei included cases with longer and imprecise exposure period. Except for two studies,\\n20\\n, \\n22\\n the number of older adults aged over 65 recruited was either unknown or too small (6‐11) to allow age comparison.', 'cite_spans': [], 'section': 'DISCUSSION', 'ref_spans': []}, {'text': 'The limitations of this study are, first, data sourced online instead of clinically. Online data tracking on travel histories and dates of symptom onset to estimate COVID‐19 incubation periods have also been used in seven of the previous studies.\\n5\\n, \\n14\\n, \\n16\\n, \\n17\\n, \\n19\\n, \\n22\\n, \\n23\\n These and the present study assumed that COVID‐19 infection was acquired during travel to Hubei, the epicenter. This may not be true, especially during late January 2020 when the infection had spread to other cities of China. Thus, travelers to Hubei may have acquired the infection on return to their own cities so that the incubation period was overestimated if based on staying in Hubei as infection exposure. Second, elderly people aged ≥65 years do not form a homogenous group in terms of health status. The data collected in this study did not have clinical information on frailty, which may be associated with multiple exposures, blunted immune response, and geriatric presentations that impact on incubation period estimates. On the other hand, older subjects recruited into this study may be biased towards the fitter spectrum because of their traveler status. Nevertheless, the distribution of COVID‐19 incubation period for older adults in this study had a wide variation. There is uncertainty if the wide variation observed arose from heterogeneity in the fitness‐frailty spectrum, a hallmark of aging, or because of the relatively small number of older adults sampled. Further studies involving a larger sample size of older adults and frailty statuses may help to clarify this. Third, the study sample was drawn from a listing of hospitalized individuals. Thus, milder community cases of COVID‐19 were not represented in this study.', 'cite_spans': [], 'section': 'DISCUSSION', 'ref_spans': []}, {'text': 'Nothing to disclose.', 'cite_spans': [], 'section': 'CONFLICTS OF INTEREST', 'ref_spans': []}, {'text': 'The author is responsible for design, conceptualization, literature review, data analysis, and writing of paper.', 'cite_spans': [], 'section': 'AUTHOR CONTRIBUTIONS', 'ref_spans': []}], 'paragraphs': [{'text': '', 'id': '00017'}], 'title': 'Longer incubation period of coronavirus disease 2019 (COVID‐19) in older adults', 'sha': '8f3c0c4095c08ef8213ada5642ba4a36216dbd0d; 19d6cf627e5afd4d56919357e64d211003665028', 'doi': '10.1002/agm2.12114', 'score': 9.946499824523926}, {'rank': 11, 'paragraphs': [{'text': 'Yuanyuan H. l [27] undertook research into COVID-19 patients who were long-term users of glucocorticoids. Across the world, clusters of patients having COVID-19 have been reported, with research demonstrating that person-toperson transmission is the chief route of infection. The research states that average incubation periods are between 2 and 14 days, with between 3 and 7 days being the most common. The research notes that incubation periods may be extended in some patients. The research reports on a family cluster of COVID-19; a 47-year-old female member of the family on long-term glucocorticoid therapy had no symptoms in a 14 day quarantine period but tested positive for COVID-19 antibodies 40 days after leaving Wuhan. These findings imply that long-term glucocorticoid use could be responsible for long incubation periods, atypical infection, and additional transmissions of the virus.', 'id': '00020'}], 'title': 'The estimations of the COVID-19 incubation period: a systematic review of the literature', 'sha': '09ff6011e4e95bff0b9221c89ffdb91d958e4ef1', 'doi': '10.1101/2020.05.20.20108340', 'score': 9.937700271606445}, {'rank': 12, 'abstract': 'Motivation: Wuhan pneumonia is an acute infectious disease caused by the 2019 novel coronavirus (COVID-19). It is being treated as a Class A infectious disease though it was classified as Class B according to the Infectious Disease Prevention Act of China. Accurate estimation of the incubation period of the coronavirus is essential to the prevention and control. However, it remains unclear about its exact incubation period though it is believed that symptoms of COVID-19 can appear in as few as 2 days or as long as 14 or even more after exposure. The accurate incubation period calculation requires original chain-of-infection data that may not be fully available in the Wuhan regions. In this study, we aim to accurately calculate the incubation period of COVID-19 by taking advantage of the chain-of-infection data, which is well-documented and epidemiologically informative, outside the Wuhan regions. Methods: We acquired and collected officially reported COVID-19 data from 10 regions in China except for Hubei province. To achieve the accurate calculation of the incubation period, we only involved the officially confirmed cases with a clear history of exposure and time of onset. We excluded those without relevant epidemiological descriptions, working or living in Wuhan for a long time, or hard to determine the possible exposure time. We proposed a Monte Caro simulation approach to estimate the incubation of COVID-19 as well as employed nonparametric ways. We also employed manifold learning and related statistical analysis to decipher the incubation relationships between different age/gender groups. Result: The incubation period of COVID-19 did not follow general incubation distributions such as lognormal, Weibull, and Gamma distributions. We estimated that the mean and median of its incubation were 5.84 and 5.0 days via bootstrap and proposed Monte Carlo simulations. We found that the incubation periods of the groups with age>=40 years and age<40 years demonstrated a statistically significant difference. The former group had a longer incubation period and a larger variance than the latter. It further suggested that different quarantine time should be applied to the groups for their different incubation periods. Our machine learning analysis also showed that the two groups were linearly separable. incubation of COVID-19 along with previous statistical analysis. Our results further indicated that the incubation difference between males and females did not demonstrate a statistical significance.', 'body_text': [{'text': 'Novel coronaviruses (COVID- 19) , which was found in Wuhan, China in December 2019 presents an acute public health threat to the whole world [1] [2] . The new virus is different from known coronaviruses such as SARS and MERS, though they share some similar respiratory illness symptoms such as fever, cough, or/and shortness of breath [2] . It is believed to root from the animal but spreads from person-to-person.', 'cite_spans': [{'start': 28, 'end': 31, 'text': '19)', 'ref_id': 'BIBREF20'}, {'start': 141, 'end': 144, 'text': '[1]', 'ref_id': 'BIBREF0'}, {'start': 145, 'end': 148, 'text': '[2]', 'ref_id': 'BIBREF1'}, {'start': 335, 'end': 338, 'text': '[2]', 'ref_id': 'BIBREF1'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'The COVID-19 spread even shows that persons without any symptoms or clinically negative in infection can still spread it to others. There is no official vaccine or antiviral drug available up to now (Jan 30, 2020) to treat COVID-19 infected patients [3] [4] . The outbreak of COVID-19 infection is forcing China and many countries to take harsh protection policies. More than eight-thousands of infections have been reported in China and more than a dozen of countries until Jan 30, 2020. More than 15 cities including Wuhan have been quarantined to halt the spread of the COVID-19. It is expected that millions of people can be on lockdown because of COVID-19. WHO declared the coronavirus outbreak a global health emergency on Jan 30, 2020.', 'cite_spans': [{'start': 250, 'end': 253, 'text': '[3]', 'ref_id': 'BIBREF4'}, {'start': 254, 'end': 257, 'text': '[4]', 'ref_id': 'BIBREF5'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'It is essential to know the accurate incubation period of COVID-19 for the sake of deciphering dynamics of its spread. The incubation period is the time from infection to the onset of the disease. It provides the foundation for epidemiological prevention, clinical actions, and drug discovery. Different viruses have different incubation periods that determine their different dynamics epidemiologically. The incubation period of H7N9 (Human Avian Influenza A) is about 6.5 days, but the incubation period for SARS-CoV is typically 2 to 7 days [5] [6] . However, it remains unclear about its exact incubation period of COVID-19, although WHO estimates it is between 2 to 14 days after exposure [8] . It can be difficult to estimate the All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 544, 'end': 547, 'text': '[5]', 'ref_id': 'BIBREF7'}, {'start': 548, 'end': 551, 'text': '[6]', 'ref_id': 'BIBREF8'}, {'start': 694, 'end': 697, 'text': '[8]', 'ref_id': 'BIBREF9'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.24.20027474 doi: medRxiv preprint incubation period of COVID-19 by using original chain-of-infection data that may not be fully available in the Wuhan regions. Or data may lack meaningful exposure history. Furthermore, it is also unknown whether the incubation time will show some statistically significant with respect to Age and Gender. In this study, we aim to accurately estimate the incubation period of COVID-19 by taking advantage of datasets with a well-documented history of exposure. Our results show the incubation mean and median of COVID-19 are 5.84 and 5.0 days respectively and there is a statistical significance with the role of gender. However, the incubation periods of the groups with age>=40 years and age<40 years show a statistically significant difference. Our machine learning analysis also shows that the two groups are linearly separable that demonstrate a clear boundary in knowledge discovery visualization.', 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'We collected a dataset with 59 officially confirmed COVID-19 cases from 10 regions in China except for Hubei province, the assumed origin of the virus. The patient data was dated from Dec 29, 2019, to Feb 5, 2020. We only involved the officially confirmed cases with a clear history of exposure and time of onset in data collection. We exclude those without relevant epidemiological descriptions, working or living in Wuhan for a long time, or hard to determine the possible exposure time.', 'cite_spans': [], 'ref_spans': [], 'section': 'Data collection and preprocessing'}, {'text': 'Data collected for this study included region, age, gender, exposure history, and illness onset. For those cases whose incubation periods locate in an interval [ ! , \" ], we use its midpoint = # ! $# \" \" to represent its incubation period. For example, Case no. 2 in our dataset went on a business trip in Wuhan on Jan 12 th , 2020 and returned to Shaanxi on Jan 15 th , 2020, but had fever symptoms on Jan 20 th 2020. The incubation will be calculated as = ', 'cite_spans': [], 'ref_spans': [], 'section': 'Data collection and preprocessing'}, {'text': 'We propose a Monte Carlo simulation approach that takes advantage of bootstrap techniques to estimate incubation median and mean estimation for the small sample with 59 cases. It is more data-driven compared to traditional parametric approaches to handle parameter estimation for small datasets. The proposed Monte Carlo simulation assumes we have a collected small incubation dataset . We generate a All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Monte Carlo simulation for incubation median and mean estimation'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.24.20027474 doi: medRxiv preprint large incubation sample * by concatenating n=1000 randomly sampled incubation segments *+ each of which contains at least entries in the interval [ ! , \" ] drawn independently from the existing dataset , i.e., * = ⋃ *+ .', 'cite_spans': [], 'ref_spans': [], 'section': 'Monte Carlo simulation for incubation median and mean estimation'}, {'text': 'Then the large incubation sample median is calculated:', 'cite_spans': [], 'ref_spans': [], 'section': ', +-!'}, {'text': 'where (0,1) is added Gaussian noise and ∈ [0,1] is a variance control parameter in simulation. Such a procedure is repeated times, the population median . =', 'cite_spans': [], 'ref_spans': [], 'section': ', +-!'}, {'text': 'The estimated standard deviation . is calculated as the standard deviation of the median sequence ! , \" , ⋯ / . In our simulation, we chose = 100000, ! = 1, \" = 7, = .2 days in simulation and conduct simulations by employing Google', 'cite_spans': [], 'ref_spans': [], 'section': ', +-!'}, {'text': 'Colab with TPU acceleration [9] [10] . The confidence level probability is calculated by the ratio / , where counts the times that * falls in the interval F . − 2 . , . + 2 . H in the simulation.', 'cite_spans': [{'start': 28, 'end': 31, 'text': '[9]', 'ref_id': None}, {'start': 32, 'end': 36, 'text': '[10]', 'ref_id': None}], 'ref_spans': [], 'section': ', +-!'}, {'text': 'Similarly, we can estimate the population mean by using the same way where the large incubation sample mean is calculated as', 'cite_spans': [], 'ref_spans': [], 'section': ', +-!'}, {'text': 'The population mean is estimated as = ! / ∑ * / *-! and the estimated standard deviation . According to the central limit theorem, the population mean will be subject to the normal distribution. The confidence interval probability is calculated by following the same procedure.', 'cite_spans': [], 'ref_spans': [], 'section': ', +-!'}, {'text': 'Our data consists of 34 male cases, 24 female cases and 1 unidentified gender case from ten regions in China.', 'cite_spans': [], 'ref_spans': [], 'section': 'Result'}, {'text': 'All 59 cases have complete epidemiological descriptions about the history of exposure. The total 57 cases have complete information in age and gender One case from Beijing has unknown age. The mean and standard deviation of age are 41.9 and 13.2 years old. The mode of his dataset is 4.0 with 14 support cases.', 'cite_spans': [], 'ref_spans': [], 'section': 'Result'}, {'text': 'The minimum and maximum age are 10 and 70 respectively. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Result'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.24.20027474 doi: medRxiv preprint Table 1 shows incubation statistics for five different groups: Male, Female, Age>=40, Age <40, and All that include all cases. It indicates that the incubation period median and mean of patients more than 40 years old are greater than those of patients less than 40 years old. Similar patterns are also observed for male and female groups. The mean values are always larger than median values for each group suggests the rightskewed distributions of incubation. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [{'start': 144, 'end': 151, 'text': 'Table 1', 'ref_id': 'TABREF0'}], 'section': 'Result'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.24.20027474 doi: medRxiv preprint The incubation of COVID-19 is not subject to neither of the widely used incubation distributions such as normal, lognormal, Gamma, and Weibull distributions well [12] . We employ Shapiro-Wilk tests rather than', 'cite_spans': [{'start': 306, 'end': 310, 'text': '[12]', 'ref_id': 'BIBREF12'}], 'ref_spans': [], 'section': 'Result'}, {'text': 'Kolmogorov-Smirnov tests to conduct normality tests because we only have 59 records and Shapiro-Wilk tests can do a much better job on small datasets with a sample size of from 3 to 5000 [13] . Table 2 shows the All rights reserved. No reuse allowed without permission.', 'cite_spans': [{'start': 187, 'end': 191, 'text': '[13]', 'ref_id': 'BIBREF13'}], 'ref_spans': [{'start': 194, 'end': 201, 'text': 'Table 2', 'ref_id': 'TABREF1'}], 'section': 'Result'}, {'text': 'author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Result'}, {'text': \"The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.24.20027474 doi: medRxiv preprint p-values under the Shapiro-Wilk tests for normal and lognormal distributions as well as Goodness-of-fit tests for Gamma and Weibull distributions by using the R package 'goft' [14] . We can reject normal, lognormal, and Weibull distributions strongly under the significance level of 0.05 cutoff [11] [12] . Although we can't reject Gamma distributions for its boundary line p-value (0.06086), it can be risky to use it to fit and estimate the distribution of the incubation period under a small sample size. We further employ the maximum likelihood estimates (MLEs) for the parameters of the gamma, conduct the goodness-fit test, and obtain p-value=8.6807e-04. As such, we only rely on nonparametric techniques in data analysis rather than use any pre-assumed distributions. \", 'cite_spans': [{'start': 320, 'end': 324, 'text': '[14]', 'ref_id': None}, {'start': 439, 'end': 443, 'text': '[11]', 'ref_id': 'BIBREF11'}, {'start': 444, 'end': 448, 'text': '[12]', 'ref_id': 'BIBREF12'}], 'ref_spans': [], 'section': 'Result'}, {'text': 'Mann-Whitney rank test shows that there are no significant differences between the incubation of males ', 'cite_spans': [], 'ref_spans': [], 'section': 'The incubation difference between males and females does not demonstrate a statistical significance'}, {'text': 'Pearson correlation coefficient analysis shows the R statistics is 0.244 with p-value: 0.06758. It suggests that that the incubation period is somewhat correlated with age though not that strong. To verify whether the incubation of the age>=40 group is different from that of the age<40 group statistically, Figure 3 compares All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [{'start': 308, 'end': 316, 'text': 'Figure 3', 'ref_id': 'FIGREF6'}], 'section': 'The incubation period of the age<40 group is statistically shorter than that of the age>=40 group'}, {'text': 'author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'The incubation period of the age<40 group is statistically shorter than that of the age>=40 group'}, {'text': \"The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.24.20027474 doi: medRxiv preprint their incubation periods groups using different visualization tools. It indicates that the younger group tends to have a shorter incubation period. The variance of their incubation period also seems to be smaller. The Mann-Whitney rank test shows that there's a statistically significant difference between the incubation of age<40 and age>=40 groups with the null hypothesis: the medians of incubation period between two groups are the same. The p-values for corresponding alternative hypotheses: the age<40 group has a smaller incubation median is 0.00474. It suggests that the age<40 group has a shorter incubation period than the age>=40 group. Similarly, the Siegal-Tukey test indicates the younger group's incubation variance is less than the older group's by p-value: 0.0083. It may suggest COVID-19 has a faster but relatively constant spread speed among people <40 years old than people >=40 years old. Figure 4 illustrates biplots of the dataset by removing two cases with missing items by using PCA (principal component analysis), sparse PCA (sparse principal component analysis, t-SNE (t-distributed stochastic neighbor embedding), and LLE (locally linear embedding) [16] [17] [18] [19] . Data is partitioned as the age>=40 and age<40 groups in visualization. t-SNE shows that only one case in the age>=40 group falls in the cluster of the age < 40 group. But PCA, sparse PCA and LLE all show that the incubation of two groups is linearly separable, which means there exists an obvious linear boundary to separate them, in the subspaces generated by PCA, SPCA, t-SNE, or LLE. Such machine learning results indicate that the two groups are actually independent clusters spatially. However, we also find that the incubation data will no longer demonstrate the linear separability property when we partition it as age>=50 and age<50 groups or age>=55 All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\", 'cite_spans': [{'start': 1323, 'end': 1327, 'text': '[16]', 'ref_id': 'BIBREF16'}, {'start': 1328, 'end': 1332, 'text': '[17]', 'ref_id': 'BIBREF17'}, {'start': 1333, 'end': 1337, 'text': '[18]', 'ref_id': None}, {'start': 1338, 'end': 1342, 'text': '[19]', 'ref_id': 'BIBREF20'}], 'ref_spans': [{'start': 1056, 'end': 1064, 'text': 'Figure 4', 'ref_id': 'FIGREF7'}], 'section': 'The incubation period of the age<40 group is statistically shorter than that of the age>=40 group'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.24.20027474 doi: medRxiv preprint and age<55 groups. It suggests that age 40 can be a key age cutoff for the incubation of COVID-19 along with previous statistical analysis. ', 'cite_spans': [], 'ref_spans': [], 'section': 'The incubation data of COVID-19 is linearly separable for Age>=40 and Age<40 groups'}, {'text': 'We further estimated population mean and standard deviation, median and 2.5 th and 97.5 th percentile by using Bootstrap under 10 6 times resampling. Table 3 author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [{'start': 150, 'end': 157, 'text': 'Table 3', 'ref_id': 'TABREF2'}], 'section': 'COVID-19 incubation statistics estimation'}, {'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.24.20027474 doi: medRxiv preprint [21] . It suggests that COVID-19 could have a faster distribution speed than H7N9, but the same spread speed as SARS and MERS in terms of their incubation periods [22] . The existing spread of COVID-19 is faster than SARS partially because it has more complicate spread dynamics [2] . For example, All rights reserved. No reuse allowed without permission.', 'cite_spans': [{'start': 144, 'end': 148, 'text': '[21]', 'ref_id': 'BIBREF22'}, {'start': 307, 'end': 311, 'text': '[22]', 'ref_id': 'BIBREF23'}, {'start': 423, 'end': 426, 'text': '[2]', 'ref_id': 'BIBREF1'}], 'ref_spans': [], 'section': 'COVID-19 incubation statistics estimation'}, {'text': 'author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'COVID-19 incubation statistics estimation'}, {'text': \"The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.24.20027474 doi: medRxiv preprint those without clinical symptoms can still spread the virus even if they are 'officially negative' in the COVID-19 virus test [22] .\", 'cite_spans': [{'start': 269, 'end': 273, 'text': '[22]', 'ref_id': 'BIBREF23'}], 'ref_spans': [], 'section': 'COVID-19 incubation statistics estimation'}, {'text': 'We also investigate the incubation period of 12 family cases and 47 non-family cases in the dataset. The family cases simply refer to the patients who were caught by COVID-19 because their family members had been infected. The Mann-Whitney rank test shows that there are not significant differences between family patients and non-family patients in terms of the median of incubation period. The Siegal-Tukey tests on scale also verify that the incubation scales are at the same level for family patients and non-family patients.', 'cite_spans': [], 'ref_spans': [], 'section': 'COVID-19 incubation statistics estimation'}, {'text': 'Our studies indicate that incubation periods of the age>=40 years and age<40 years groups not only statistically significant but also linearly separable in machine learning. It may suggest different treatments should be considered for the two different groups. It will be more interesting to estimate different incubation time for them separately. That the estimated 97.5 th percentile of COVID-19 incubation is 12.89 days (95% CI: (11.00, 16.13)) may suggest a long isolation or quarantine time (e.g. 17 days) can be better than the widely accepted 14 days. Furthermore, different quarantine time should be applied to the age>=40 years and age<40 years groups for their different incubation periods. Generally speaking, a longer quarantine time can be considered for the old patients (>=40 years) than young patients (<40 years old). Our ongoing work is to collect more qualified data to extend our existing results and investigate incubation of COVID-19 for different groups besides comparing our incubation estimation with other studies [23] .', 'cite_spans': [{'start': 1040, 'end': 1044, 'text': '[23]', 'ref_id': 'BIBREF24'}], 'ref_spans': [], 'section': 'COVID-19 incubation statistics estimation'}], 'paragraphs': [{'text': 'Our studies indicate that incubation periods of the age>=40 years and age<40 years groups not only statistically significant but also linearly separable in machine learning. It may suggest different treatments should be considered for the two different groups. It will be more interesting to estimate different incubation time for them separately. That the estimated 97.5 th percentile of COVID-19 incubation is 12.89 days (95% CI: (11.00, 16.13)) may suggest a long isolation or quarantine time (e.g. 17 days) can be better than the widely accepted 14 days. Furthermore, different quarantine time should be applied to the age>=40 years and age<40 years groups for their different incubation periods. Generally speaking, a longer quarantine time can be considered for the old patients (>=40 years) than young patients (<40 years old). Our ongoing work is to collect more qualified data to extend our existing results and investigate incubation of COVID-19 for different groups besides comparing our incubation estimation with other studies [23] .', 'id': '00033'}], 'title': 'Estimate the incubation period of coronavirus 2019 (COVID-19)', 'sha': 'c1ae608c7ffb926a0f50a6a34c0780983274ea74', 'doi': '10.1101/2020.02.24.20027474', 'score': 9.917099952697754}, {'rank': 13, 'paragraphs': [{'text': 'We collected a dataset with 59 officially confirmed COVID-19 cases from 10 regions in China except for Hubei province, the assumed origin of the virus. The patient data was dated from Dec 29, 2019, to Feb 5, 2020. We only involved the officially confirmed cases with a clear history of exposure and time of onset in data collection. We exclude those without relevant epidemiological descriptions, working or living in Wuhan for a long time, or hard to determine the possible exposure time.', 'id': '00005'}], 'title': 'Estimate the incubation period of coronavirus 2019 (COVID-19)', 'sha': 'c1ae608c7ffb926a0f50a6a34c0780983274ea74', 'doi': '10.1101/2020.02.24.20027474', 'score': 9.906000137329102}, {'rank': 14, 'abstract': 'It is not currently known how long it takes a person infected by the COVID-19 virus to become infectious. Models of the spread of COVID-19 use very different lengths for this latency period, leading to very different estimates of the replication number R, even when models work from the same underlying data sets. In this paper we quantify how much varying the length of the latency period affects estimates of R, and thus the fraction of the population that is predicted to be infected in the first wave of the pandemic. This variation underscores the uncertainty in our understanding of R and raises the possibility that R may be considerably greater than has been assumed by those shaping public policy.', 'body_text': [{'text': 'This paper addresses another point of confusion. While there is general agreement that some infected individuals who are not yet symptomatic (or who never develop symptoms) can still infect others (4) , there is no agreement on how long it takes an infected individual to become infectious. Some researchers treat infected individuals as immediately infectious (1, 5) , while others assume that they only become infectious towards the end of the incubation period (6) . These different assumptions lead to different trajectories for a given R 0 , with earlier infectivity usually leading to faster growth. Conversely, when estimating R 0 from an actual observed trajectory, models with earlier infectivity usually give lower estimates than models with a longer latency period. In this paper we quantify how very different those estimates can be.', 'cite_spans': [{'start': 197, 'end': 200, 'text': '(4)', 'ref_id': 'BIBREF4'}, {'start': 361, 'end': 364, 'text': '(1,', 'ref_id': 'BIBREF0'}, {'start': 365, 'end': 367, 'text': '5)', 'ref_id': 'BIBREF5'}, {'start': 464, 'end': 467, 'text': '(6)', 'ref_id': None}], 'ref_spans': [], 'section': ''}, {'text': 'To be reliable, estimates of R must be made for specific places and times and must acknowledge the large effects of the sometimes-arbitrary choices involved in model-building. We know 1 These numbers are for confirmed cases. Due to testing inefficiencies, the actual number of cases on March 8 was probably much higher than reported, and likewise on March 24. However, if the number of cases in the United States only grew 50-fold instead of 100-fold, the resulting doubling time of 2.8 days would still be far smaller than in Wuhan. much less than we think we know, and there is a substantial chance that the true value of R is much higher than the optimistic 1.3-2.5 range that has been assumed for most public policy discussions.', 'cite_spans': [{'start': 184, 'end': 185, 'text': '1', 'ref_id': 'BIBREF0'}], 'ref_spans': [], 'section': ''}, {'text': 'Most models of disease spread are variants of the SEIR model, which is itself an extension of the popular SIR model (9). In the SEIR model, people are classified as \"Susceptible\", \"Exposed\", \"Infectious\", and \"Removed\" (or \"Recovered\"). Susceptible individuals become Exposed from contact with existing Infectious individuals, Exposed individuals become Infectious after a latency period, and Infectious individuals eventually recover, die, are quarantined, or are otherwise Removed from circulation. We let S(t), E(t), I(t) and R(t) denote the number of Susceptible, Exposed, Infectious and Removed individuals at time t and write equations to govern the evolution of these quantities.', 'cite_spans': [], 'ref_spans': [], 'section': 'Stage models'}, {'text': 'It is important to acknowledge that even the SEIR model is not a complete description of disease spread. Now that the whole world is on alert, most symptomatic individuals are removed from circulation quickly. However, some show mild symptoms and are not noticed or removed from circulation, while some symptomatic individuals defy isolation orders or require a degree of home care that makes true isolation impossible. More accurate predictions require incorporating all of these phenomena (and others) correctly, which requires more complicated models, such as (5).', 'cite_spans': [], 'ref_spans': [], 'section': 'Stage models'}, {'text': 'However, the limitations of the SEIR model are not the point of this paper. Our purpose here is to show that, even in the simplest models, results are highly dependent on the length of the Exposed state. This problem is not fixed by adding additional details to the model. On the contrary, the more complicated a model is, the more its results depend on the choices made in constructing that model.', 'cite_spans': [], 'ref_spans': [], 'section': 'Stage models'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '3'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.04.07.20056960 doi: medRxiv preprint 1. The mean time t 1 spent by patients in the uninfectious \"exposed\" state. We call this time the latency.', 'cite_spans': [], 'ref_spans': [], 'section': '3'}, {'text': '2. The mean time t 2 spent by patients in the infectious state.', 'cite_spans': [], 'ref_spans': [], 'section': '3'}, {'text': '3. The reproduction number R, which is the mean number of people that each infected person infects in turn.', 'cite_spans': [], 'ref_spans': [], 'section': '3'}, {'text': 'The sum t tot = t 1 + t 2 is the total average time from exposure to the appearance of symptoms severe enough to cause an individual to isolate. We call this the extended incubation period.', 'cite_spans': [], 'ref_spans': [], 'section': '3'}, {'text': 't tot is relatively easy to measure, and for COVID-19 is believed to be about one week (10).', 'cite_spans': [], 'ref_spans': [], 'section': '3'}, {'text': 'The individual numbers t 1 and t 2 are much harder to determine. In this paper we will compare different values of the latency t 1 , while holding the sum t tot fixed.', 'cite_spans': [], 'ref_spans': [], 'section': '3'}, {'text': 'In addition to these three parameters, we consider several ways to treat the distribution of incubation times for individuals. The most common choice is to assume that the time each individual spends in the exposed state and the infectious state are independent exponential random variables with means t 1 and t 2 , respectively. This choice leads to a simple system of differential equations. However, in this approach some exposed individuals move to the infectious stage almost immediately, which is not realistic. Alternatively, we can assume that the time spent in the exposed state is always exactly t 1 , and the time spent in the infectious state is exactly t 2 , but that ignores the fact that some people get sick much faster than others, while some people stay undetected and infectious for much longer than others. A promising approach is a hybrid, keeping the time in the exposed state constant and making the time in the infectious state random.', 'cite_spans': [], 'ref_spans': [], 'section': '3'}, {'text': 'In the early stages of an outbreak, when almost all individuals are susceptible, these models (and more complicated extensions of these models) are all linear. Linearity implies that the total number of infected individuals grows exponentially as e t/τ , where τ is the doubling time of the 4 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '3'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.04.07.20056960 doi: medRxiv preprint growth divided by ln(2) ≈ 0.69. Since all models lead to the same qualitative behavior, τ is the only quantity that can be directly deduced from the data. The subsequent step of inferring R from τ then depends on which model we choose and which parameters we choose for that model.', 'cite_spans': [], 'ref_spans': [], 'section': '3'}, {'text': 'We will concentrate on understanding the rapid growth of COVID-19 in the United States in March 2020, when τ was approximately 3.5 days. We apply each of the three models (exponential distribution of latency and the time spent infective, fixed times, and the hybrid) to compute R assuming that τ = 3.5 days and that the average t 2 of the time spent infective is 7 days minus the average t 1 of the latency.', 'cite_spans': [], 'ref_spans': [], 'section': 'Results'}, {'text': 'The results are shown in Figure 1 , which gives the computed value of R, for all three models, as a function of the latency t 1 . The relevant formulas are as follows: When both incubation times are treated as exponential random variables,', 'cite_spans': [], 'ref_spans': [{'start': 25, 'end': 33, 'text': 'Figure 1', 'ref_id': None}], 'section': 'Results'}, {'text': 'When both incubation times are fixed,', 'cite_spans': [], 'ref_spans': [], 'section': 'Results'}, {'text': 'Finally, in the hybrid model', 'cite_spans': [], 'ref_spans': [], 'section': 'Results'}, {'text': 'These formulas are derived in the Supplementary Materials.', 'cite_spans': [], 'ref_spans': [], 'section': 'Results'}, {'text': 'It is worth noting, not only how the inferred value of R depends on the average latency t 1 , but also how it depends on the seemingly innocuous choice of whether to make the lengths of the Exposed and Infectious stages fixed or random. While none of the models fit what is known 5 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Results'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . Figure 1 : Estimated values of R for all three models as a function of the average latency t 1 . While increased latency greatly increases R in the fixed length and hybrid models, the exponential model behaves differently. The exponential model allows a few exposed individuals to reinfect others very quickly and these rapid transmitters drive much of the growth of the outbreak.', 'cite_spans': [], 'ref_spans': [{'start': 78, 'end': 86, 'text': 'Figure 1', 'ref_id': None}], 'section': 'Results'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '6'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.04.07.20056960 doi: medRxiv preprint about virology exactly, it is easy to make a good case for each of them. Yet they give vastly different results, especially for larger values of t 1 ! Without a clear scientific justification for choosing one model over the other two, we must acknowledge that all of our estimates of R are extremely uncertain.', 'cite_spans': [], 'ref_spans': [], 'section': '6'}, {'text': 'The exponential model behaves differently from the other models because it involves some individuals becoming infectious very quickly, even when the average latency is large, and involves some of those individuals infecting others very quickly as well. These rapid transmitters do not account for the majority of the transmissions, which still take a total of 7 days on average, but they have an oversized effect on the growth rate. The number of these rapid transmitters is maximized when either t 1 is small or t 2 is small, which explains why R reaches its peak when', 'cite_spans': [], 'ref_spans': [], 'section': '6'}, {'text': 'The lesson is that when the minimum time from infection to reinfection goes up, so does R.', 'cite_spans': [], 'ref_spans': [], 'section': '6'}, {'text': 'There is a simple explanation for this. If τ = 3.5, then the outbreak doubles every 2.4 days, quadruples every 4.8 days, and multiplies by 8 every 7.2 days. If it takes at least 4.8 days for an exposed individual to infect somebody else, then R must be at least 4 to account for the quadrupling in that time; if the time interval is always at least 7.2 days, then R must be at least 8.', 'cite_spans': [], 'ref_spans': [], 'section': '6'}, {'text': 'The value of R has profound effects on public policy decisions. Suppose that a social distancing policy can reduce transmission by a factor of 3. If we start at R = 2, dividing R by 3 is enough to contain and eventually suppress the outbreak. If we start at R = 4, dividing by 3 will only slow the growth. In that case, more extreme measures are needed to get the outbreak under control.', 'cite_spans': [], 'ref_spans': [], 'section': '6'}, {'text': 'R also determines the fraction of the population that eventually becomes exposed to the 7 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '6'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.04.07.20056960 doi: medRxiv preprint virus. In all of these models, that fraction x is the non-zero solution to the equation Figure 2 : The percentage of the population that eventually becomes infected as a function of the reproduction number R. Note that this percentage is already over 40% when R = 1.3, and rapidly approaches 100% as R increases.', 'cite_spans': [], 'ref_spans': [{'start': 232, 'end': 240, 'text': 'Figure 2', 'ref_id': None}], 'section': '6'}, {'text': \"The dependence of x on R is shown in Figure 2 . When R is as small as 1.15, x is already at 25%, and when R = 1.7, x is already about 70%. When studies such as (11) say that 25%-70% of the population would be infected in the absence of control measures, they are implicitly assuming that R 0 is only between 1.15 and 1.7. Under that assumption, moderate efforts should be enough to bring R below 1 and suppress the outbreak. However, if R 0 is actually larger, and if our social distancing efforts don't bring R below 2, then the vast majority of the population would be infected in the first wave of the pandemic. The resulting herd immunity would be impressive, but almost no vulnerable people would be left to benefit from it.\", 'cite_spans': [], 'ref_spans': [{'start': 37, 'end': 45, 'text': 'Figure 2', 'ref_id': None}], 'section': '6'}, {'text': '8 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '6'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.04.07.20056960 doi: medRxiv preprint Finally, the value of R affects the possibility of having multiple waves of the pandemic.', 'cite_spans': [], 'ref_spans': [], 'section': '6'}, {'text': 'Suppose that a fraction x of the population is infected in the first wave, after which life returns to normal and R returns to its pre-lockdown value. Unless', 'cite_spans': [], 'ref_spans': [], 'section': '6'}, {'text': 'successive outbreaks are extremely likely to occur.', 'cite_spans': [], 'ref_spans': [], 'section': '6'}, {'text': 'All models for understanding the future trajectory of the COVID-19 pandemic depend on knowing how contagious the disease is; that is, on estimating R. This is true both for complicated models involving many different sorts of people and many different stages of disease progression, as well as for simple models such as SEIR. Unfortunately, there is no direct way to measure R. All we can do is measure the time scale τ of the exponential growth of the pandemic and try to estimate R from τ .', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusions'}, {'text': 'Such estimates depend strongly on the details of the model used. Eventually, the correct parameters and the most accurate models will be revealed through clinical studies of the actual distribution of the different stages of infection and especially by tracing individual contacts to determine when infectivity starts. Until those studies are completed for COVID-19, every projection will involve a choosing parameters through educated guesswork.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusions'}, {'text': 'Even in the simple models considered in this paper, changing the parameters of the model can change the estimated value of R dramatically. In the fixed length model the estimate for R ranges from 2.3 to 7.4 and the hybrid model is almost as sensitive. In all three models, having a latency of just one day (t 1 = 1) increases R by 0.5 or more compared to having no latency at all (t 1 = 0). Working with more realistic (and more complicated) models only makes the problem of model dependence worse. The more parameters there are, the more the final answers depend 9 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusions'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.04.07.20056960 doi: medRxiv preprint on the assumptions.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusions'}, {'text': 'As of early April 2020, most of the highly cited estimates of R have been based either on models without latency, on data from Wuhan that shows much slower growth than seen in Europe and the United States, or both. As such, these estimates are likely to have substantially underestimated the true value of R in the United States.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusions'}, {'text': 'The success of any suppression strategy boils down to one thing: getting R below 1. Strategies that could work if R is between 1.3 and 2.5 might well fail if the actual value of R is larger.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusions'}, {'text': \"Meanwhile, mitigation strategies depend on isolating the most vulnerable members of society until the first wave of the pandemic has subsided. The amount of herd immunity required to make that work also depends on R. It won't be safe for the vulnerable to go out until the first wave of the pandemic has subsided and a fraction x > 1 − 1 R of the population has developed immunity through exposure or vaccine. Higher values of R provide a stronger argument for accelerating the testing of a vaccine and accepting the accompanying risks of adverse side effects (by rushing Phase I) or settling on a less effective vaccine (by rushing Phase II). Haste may well be needed.\", 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusions'}, {'text': 'The world has put enormous effort and expense into confronting COVID-19. To succeed in the long term, we need a much better understanding of the infectivity of COVID-19. Models with accurate descriptions of what happens after incubation can help, but only after we have a much better understanding of the minimum time between infection and infectivity. Without that understanding, all of our models will remain unreliable and our policies will remain uninformed.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusions'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusions'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.04.07.20056960 doi: medRxiv preprint 7. Day-by-day statistics for the world, individual countries, and individual states may be found at https://www.worldometers.info/coronavirus. This work is supported by the University of Texas at Austin, but not through any external grants.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusions'}, {'text': 'There are no conflicts of interest. All data is available in the manuscript or the supplementary materials.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusions'}, {'text': '12 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusions'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.04.07.20056960 doi: medRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusions'}, {'text': 'All versions of the SEIR model have the same general structure.', 'cite_spans': [], 'ref_spans': [], 'section': 'Supplemental materials: Mathematical derivations'}, {'text': '1. Susceptibles turn into Exposed at a rate equal to βS(t)I(t), where β is a transmission coefficient. In the early stages of the pandemic, S(t) is approximately the entire population T , so the number of Susceptibles who become Exposed, per unit time, is approximately βT I(t).', 'cite_spans': [], 'ref_spans': [], 'section': 'Supplemental materials: Mathematical derivations'}, {'text': 'Infectious in an average of t 1 days. In the exponential version, the number of Exposed who become Infectious per unit time is αE(t), where α = 1/t 1 . In the fixed-length version, the rate at which Exposed become Infectious today is exactly equal to the rate at which they became Exposed t 1 days ago, namely βT I(t−t 1 ).', 'cite_spans': [], 'ref_spans': [], 'section': 'Exposed individuals become'}, {'text': 'Removed in an average of t 2 days. In the exponential version, the number of Infectious who are Removed per unit time is γI(t), where γ = 1/t 2 . In the fixed-length version, the rate at which Infectious are Removed today is exactly equal to the rate at which they became Infectious t 2 days ago, which in turn is the rate at which they were Exposed t 1 + t 2 days ago, namely βT I(t − t 1 − t 2 ).', 'cite_spans': [], 'ref_spans': [], 'section': 'Infectious individuals are'}, {'text': '4. The reproduction ratio is R = t 2 βT , so we can replace βT with R/t 2 or Rγ.', 'cite_spans': [], 'ref_spans': [], 'section': 'Infectious individuals are'}, {'text': 'In the purely exponential model, these considerations yield the following system of differential equations:', 'cite_spans': [], 'ref_spans': [], 'section': 'Exponential incubation times'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '13'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.04.07.20056960 doi: medRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': '13'}, {'text': 'Making the approximation βS(t) ≈ βT = Rγ, we get d dt', 'cite_spans': [], 'ref_spans': [], 'section': '13'}, {'text': 'The exponential rate of growth (1/τ ) is the positive eigenvalue of the matrix on the right hand side. Since the sum of the two eigenvalues is the trace, namely −(α + γ), the other eigenvalue must be − 1 τ + α + γ . The determinant of the matrix is −(R − 1)αγ. Since the determinant is the product of the two eigenvalues, we have', 'cite_spans': [], 'ref_spans': [], 'section': '13'}, {'text': 'which is equivalent to equation (1).', 'cite_spans': [], 'ref_spans': [], 'section': '13'}, {'text': 'In the model with fixed incubation times, Exposed become Infected at rate RI(t − t 1 )/t 2 , while Infected are Removed at rate RI(t − t 1 − t 2 )/t 2 , so', 'cite_spans': [], 'ref_spans': [], 'section': 'Fixed incubation times'}, {'text': 'Plugging in I(t) = Ce t/τ , where C is an unknown constant, we obtain C τ e t/τ = CRe t/τ t 2 e −t 1 /τ − e −(t 1 +t 2 )/τ 1 τ = R t 2 e −t 1 /τ − e −(t 1 +t 2 )/τ 14 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Fixed incubation times'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.04.07.20056960 doi: medRxiv preprint t 2 τ (e −t 1 /τ − e −(t 1 +t 2 )/τ )', 'cite_spans': [], 'ref_spans': [], 'section': 'Fixed incubation times'}, {'text': 'which is equation (2).', 'cite_spans': [], 'ref_spans': [], 'section': 'Fixed incubation times'}, {'text': 'Here we treat the length of the exposed stage as fixed and the length of the infectious stage as random. The rate at which Exposed become Infectious is RγI(t − t 1 ) = RI(t − t 1 )/t 2 , but the rate at which Infectious individuals are removed is γI(t) = I(t)/t 2 . This gives the differential equation', 'cite_spans': [], 'ref_spans': [], 'section': 'The hybrid model'}, {'text': 'Plugging in I(t) = Ce t/τ as before, and dividing both sides by Ce t/τ , gives', 'cite_spans': [], 'ref_spans': [], 'section': 'The hybrid model'}, {'text': 'which is equation (3).', 'cite_spans': [], 'ref_spans': [], 'section': 'The hybrid model'}, {'text': 'How many people will eventually get sick?', 'cite_spans': [], 'ref_spans': [], 'section': 'The hybrid model'}, {'text': 'Finally, we consider what fraction of people will get sick before the pandemic ends. The pandemic burns itself out when it runs out of Susceptibles to infect, so for this part of the calculation we do not make the simplifying approximation that S(t) ≈ T . Instead we look at the exact equation for S, which is the same in all versions of our model.', 'cite_spans': [], 'ref_spans': [], 'section': 'The hybrid model'}, {'text': 'Dividing both sides by S(t) gives d ln(S(t)) dt = −βI(t).', 'cite_spans': [], 'ref_spans': [], 'section': 'The hybrid model'}, {'text': '15 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. ', 'cite_spans': [], 'ref_spans': [], 'section': 'The hybrid model'}, {'text': 'Let x be the fraction of people who eventually get sick. The left hand side of equation (15) is ln (1 − x) , while the right hand side is −βt 2 S(0)x, since the number of people who get sick is xS(0), and since each sick person is infectious for an average of t 2 days. However, βt 2 S(0) = βT /γ = R, so', 'cite_spans': [], 'ref_spans': [{'start': 99, 'end': 106, 'text': '(1 − x)', 'ref_id': None}], 'section': 'The hybrid model'}, {'text': 'which is equation (4).', 'cite_spans': [], 'ref_spans': [], 'section': 'The hybrid model'}, {'text': 'To understand equation (5), imagine that R = 4, so each infected person has contact, of the sort that spreads the virus, with an average of 4 other people. If more than 3/4 of the population has already had the virus, then fewer than one of those four people (on average) will get infected. Since each infected individual actually infects fewer than one new person (on average), any local outbreak will quickly peter out.', 'cite_spans': [], 'ref_spans': [], 'section': 'Multiple waves and herd immunity'}, {'text': 'More generally, if a fraction x of the population is already immune and a fraction 1 − x is still Susceptible, the average number of new people that each infected individual actually infects is only R(1 − x), not R. As long as R(1 − x) < 1, or equivalently x > 1 − 1 R , local outbreaks cannot get any traction and cannot spread into the population at large. This phenomenon, where immunity in part of the population protects the rest, is called \"herd immunity\".', 'cite_spans': [], 'ref_spans': [], 'section': 'Multiple waves and herd immunity'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '16'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.04.07.20056960 doi: medRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': '16'}], 'paragraphs': [{'text': 'This paper addresses another point of confusion. While there is general agreement that some infected individuals who are not yet symptomatic (or who never develop symptoms) can still infect others (4) , there is no agreement on how long it takes an infected individual to become infectious. Some researchers treat infected individuals as immediately infectious (1, 5) , while others assume that they only become infectious towards the end of the incubation period (6) . These different assumptions lead to different trajectories for a given R 0 , with earlier infectivity usually leading to faster growth. Conversely, when estimating R 0 from an actual observed trajectory, models with earlier infectivity usually give lower estimates than models with a longer latency period. In this paper we quantify how very different those estimates can be.', 'id': '00001'}], 'title': 'Effects of latency on estimates of the COVID-19 replication number', 'sha': 'a017dd5e8dfa5dfd6385d407813e7c26d98becb9', 'doi': '10.1101/2020.04.07.20056960', 'score': 9.876299858093262}, {'rank': 15, 'abstract': 'Background The outbreak of coronavirus disease 2019 (COVID-19) has been declared a pandemic by the World Health Organization, while several key epidemiological parameters of the disease remain to be clarified. This study aimed to obtain robust estimates of the incubation period, upper limit of latent period (interval between exposure of infector and infectee), serial interval, time point of exposure and basic reproduction number (R0) of COVID-19. Methods Between late February and early March of 2020, the individual data of laboratory confirmed cases of COVID-19 were retrieved from 10728 publicly available reports released by the health authorities of and outside China and from 1790 publications identified in PubMed and CNKI. To be eligible, a report had to contain the data that allowed for estimation of at least one parameter. As relevant data mainly came from clustering cases, the clusters for which no evidence was available to establish transmission order were all excluded to ensure accuracy of estimates. Additionally, only the cases with an exposure period spanning 3 days or less were included in the estimation of parameters involving exposure date, and a simple method for determining exposure date was adopted to ensure the error of estimates be small (< 0.3 day). Depending on specific parameters, three or four of normal, lognormal, Weibull, and gamma distributions were fitted to the datasets and the results from appropriate models were presented. Findings In total, 1155 cases from China, Japan, Singapore, South Korea, Vietnam, Germany and Malaysia were included for the final analysis. The mean and standard deviation were 7.44 days and 4.39 days for incubation period, 2.52 days and 3.95 days for the upper limit of latent period, 6.70 days and 5.20 days for serial interval, and -0.19 day (i.e., 0.19 day before symptom onset of infector) and 3.32 days for time point of exposure. R0 was estimated to be 1.70 and 1.78 based on two different formulas. For 39 (6.64%) cases, the incubation periods were longer than 14 days. In 102 (43.78%) infector-infectee pairs, transmission occurred before the symptom onsets of infectors. In 27 (3.92%) infector-infectee pairs, symptom onsets of infectees occurred before those of infectors. Stratified analysis showed that incubation period and serial interval were consistently longer for those with less severe disease and for those whose primary cases had less severe disease. Asymptomatic transmission was also observed. Interpretation This study obtained robust estimates of several key epidemiological parameters of COVID-19. The findings support current practice of 14-day quarantine of persons with potential exposure, but also suggest that longer monitoring periods might be needed for selected groups. The estimates of serial interval, time point of exposure and latent period provide consistent evidence on pre-symptomatic transmission. This together with asymptomatic transmission and the generally longer incubation and serial interval of less severe cases suggests a high risk of long-term epidemic in the absence of appropriate control measures.', 'body_text': [{'text': \"The outbreak of coronavirus disease 2019 (COVID -19) has been declared a pandemic by the World Health Organization, while several key epidemiological parameters of the disease remain to be clarified. This study aimed to obtain robust estimates of the incubation period, upper limit of latent period (interval between infector's exposure and infectee's exposure), serial interval, time point of exposure (the day of infectee's exposure to infector relative to the latter's symptom onset date) and basic reproduction number (R0) of COVID-19.\", 'cite_spans': [{'start': 48, 'end': 52, 'text': '-19)', 'ref_id': None}], 'ref_spans': [], 'section': 'Summary Background'}, {'text': 'Between late February and early March of 2020, the individual data of laboratory confirmed cases of COVID-19 were retrieved from 10728 publicly available reports released by the health authorities of and outside China and from 1790 publications identified in PubMed and CNKI.', 'cite_spans': [], 'ref_spans': [], 'section': 'Methods'}, {'text': 'To be eligible, a report had to contain the data that allowed for estimation of at least one parameter. As relevant data mainly came from clustering cases, the clusters for which no evidence was available to establish transmission order were all excluded to ensure accuracy of estimates. Additionally, only the cases with an exposure period spanning 3 days or less were included in the estimation of parameters involving exposure date, and a simple method for determining exposure date was adopted to ensure the error of estimates be small (< 0.3 day).', 'cite_spans': [], 'ref_spans': [], 'section': 'Methods'}, {'text': 'Depending on specific parameters, three or four of normal, lognormal, Weibull, and gamma distributions were fitted to the datasets and the results from appropriate models were presented.', 'cite_spans': [], 'ref_spans': [], 'section': 'Methods'}, {'text': \"In total, 1155 cases from China, Japan, Singapore, South Korea, Vietnam, Germany and Malaysia were included for the final analysis. The mean and standard deviation were 7.44 days and 4.39 days for incubation period, 2.52 days and 3.95 days for the upper limit of latent period, 6 .70 days and 5.20 days for serial interval, and -0.19 day (i.e., 0.19 day before infector's symptom onset) and 3.32 days for time point of exposure. R0 was estimated to be 1.70 and 1.78 based on two different formulas. For 39 (6.64%) cases, the incubation periods were longer than 14 days. In 102 (43.78%) infector-infectee pairs, transmission occurred before infectors' symptom onsets. In 27 (3.92%) infector-infectee pairs, infectees' symptom onsets occurred before those of infectors. Stratified analysis showed that incubation period and serial interval were consistently longer for those with less severe disease and for those whose primary cases had less severe disease. Asymptomatic transmission was also observed.\", 'cite_spans': [{'start': 278, 'end': 279, 'text': '6', 'ref_id': 'BIBREF5'}], 'ref_spans': [], 'section': 'Findings'}, {'text': 'This study obtained robust estimates of several key epidemiological parameters of COVID-19.', 'cite_spans': [], 'ref_spans': [], 'section': 'Interpretation'}, {'text': 'The findings support current practice of 14-day quarantine of persons with potential exposure, but also suggest that longer monitoring periods might be needed for selected groups. The estimates of serial interval, time point of exposure and latent period provide consistent evidence on pre-symptomatic transmission. This together with asymptomatic transmission and the', 'cite_spans': [], 'ref_spans': [], 'section': 'Interpretation'}, {'text': 'In December 2019, a novel enveloped RNA beta-coronavirus, which was later named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), emerged in Wuhan, the capital city of Hubei province of China. 1 The disease caused by SARS-CoV-2, i.e., coronavirus disease 2019 (COVID-19), spread across and outside China rapidly, and was declared a pandemic by the World Health Organization on 11 March 2020.', 'cite_spans': [{'start': 205, 'end': 206, 'text': '1', 'ref_id': 'BIBREF0'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'Despite the explosive growth of the number of studies on COVID-19, several key epidemiological parameters of the disease remain to be clarified, among which are incubation period and serial interval. The two parameters have important implications for the monitoring, surveillance and control of infectious disease. The mean or median incubation period and serial interval estimated by previous studies were mostly 4 to 5 days, 2-7 and 4 to 4.5 days, [8] [9] [10] [11] [12] respectively. While some of the studies included only a limited number of cases (around or less than 100), 2, 4, 6, 7, [9] [10] [11] [12] others might have suffered from inaccuracy of original data. For example, in some studies, most cases had a long or even unclear interval of exposure, making it difficult to determine the exact exposure date 3, 5, 13 and giving rise to error. Besides, the order of transmission (i.e., who is infector and who is infectee) in clustering cases, which is crucial to estimation of both parameters, is easy to be mistaken given the possibility of pre-symptomatic and asymptomatic transmission, [14] [15] [16] [17] but previous studies rarely described how this issue was handled.', 'cite_spans': [{'start': 450, 'end': 453, 'text': '[8]', 'ref_id': None}, {'start': 454, 'end': 457, 'text': '[9]', 'ref_id': 'BIBREF7'}, {'start': 458, 'end': 462, 'text': '[10]', 'ref_id': 'BIBREF8'}, {'start': 463, 'end': 467, 'text': '[11]', 'ref_id': 'BIBREF9'}, {'start': 468, 'end': 472, 'text': '[12]', 'ref_id': 'BIBREF10'}, {'start': 580, 'end': 582, 'text': '2,', 'ref_id': 'BIBREF1'}, {'start': 583, 'end': 585, 'text': '4,', 'ref_id': 'BIBREF3'}, {'start': 586, 'end': 588, 'text': '6,', 'ref_id': 'BIBREF5'}, {'start': 589, 'end': 591, 'text': '7,', 'ref_id': 'BIBREF6'}, {'start': 592, 'end': 595, 'text': '[9]', 'ref_id': 'BIBREF7'}, {'start': 596, 'end': 600, 'text': '[10]', 'ref_id': 'BIBREF8'}, {'start': 601, 'end': 605, 'text': '[11]', 'ref_id': 'BIBREF9'}, {'start': 606, 'end': 610, 'text': '[12]', 'ref_id': 'BIBREF10'}, {'start': 819, 'end': 821, 'text': '3,', 'ref_id': 'BIBREF2'}, {'start': 822, 'end': 824, 'text': '5,', 'ref_id': 'BIBREF4'}, {'start': 825, 'end': 827, 'text': '13', 'ref_id': 'BIBREF11'}, {'start': 1100, 'end': 1104, 'text': '[14]', 'ref_id': 'BIBREF12'}, {'start': 1105, 'end': 1109, 'text': '[15]', 'ref_id': 'BIBREF13'}, {'start': 1110, 'end': 1114, 'text': '[16]', 'ref_id': 'BIBREF14'}, {'start': 1115, 'end': 1119, 'text': '[17]', 'ref_id': 'BIBREF15'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'Incubation period and serial interval are also key parameters for epidemiological modelling in predicting the transmission dynamics, including the basic reproduction number (R0). R0 is defined as the average number of secondary cases caused by a single infectious individual in a totally susceptible population. 18 Estimation of R0 may require the knowledge of latent period and infectious period, 19, 20 which are both difficult to measure directly. For a disease that is not infectious until onset of symptoms, latent period is equivalent to incubation period, and infectious period can commonly be approximated by the difference of serial interval minus incubation period. 18 However, for COVID-19, the latent period appears to be shorter than incubation period as pre-symptomatic transmission might occur, 16, 17 and this should be accounted for when estimating R0. To our best knowledge, no published studies estimated the latent period of COVID-19, or how often and approximately at which time point the disease could be transmitted prior to the symptomatic onset.', 'cite_spans': [{'start': 312, 'end': 314, 'text': '18', 'ref_id': 'BIBREF16'}, {'start': 398, 'end': 401, 'text': '19,', 'ref_id': 'BIBREF17'}, {'start': 402, 'end': 404, 'text': '20', 'ref_id': 'BIBREF18'}, {'start': 676, 'end': 678, 'text': '18', 'ref_id': 'BIBREF16'}, {'start': 810, 'end': 813, 'text': '16,', 'ref_id': 'BIBREF14'}, {'start': 814, 'end': 816, 'text': '17', 'ref_id': 'BIBREF15'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': \"This study made use of the large amount of data reported by the health authorities of and outside China and those from published studies to address the above issues. Specifically, it was aimed to obtain robust estimates of the following epidemiological parameters of COVID-19: 1) incubation period, 2) the upper limit of latent period, as the exact latent period cannot be observed, 3) serial interval, 4) time point of exposure, referring to the day of infectee's exposure to infector relative to the latter's symptom onset date, and 5) R0. All rights reserved. No reuse allowed without permission. perpetuity. preprint (which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in We also searched for individual cases from relevant departments and public media of the United States, the United Kingdom, Canada and Australia, but failed to find any with details allowing for parameters estimation in this study. Typically, the dates of exposure and symptom onset were lacking, e.g., see reference. 23 Owing to language barrier, we did not do a comprehensive search for other countries. A full list of the abovementioned data sources, including their URLs, can be found in Appendix 1.\", 'cite_spans': [{'start': 1062, 'end': 1064, 'text': '23', 'ref_id': None}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'PubMed was searched to identify relevant publications by using the following terms: ', 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'To be eligible, a report had to contain individual data that allowed for estimation of at least one of the following parameters of laboratory-confirmed cases of COVID-19: incubation period, All rights reserved. No reuse allowed without permission. perpetuity. preprint (which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in The copyright holder for this this version posted March 24, 2020. . https://doi.org/10.1101/2020.03.21.20040329 doi: medRxiv preprint serial interval, time point of exposure, and the upper limit of latent period.', 'cite_spans': [], 'ref_spans': [], 'section': 'Definitions and inclusion criteria'}, {'text': 'Incubation period was defined as the time interval between exposure and onset of disease symptoms. To obtain an accurate estimate, only the cases with an exposure period spanning 3 days or less were included in the analysis. For those exposed for three continuous days and those exposed on two dates with one day apart (i.e., exposed on the first and third days), the second day was uniformly used as the exposure date in estimation. For those exposed for two continuous days, the first day was uniformly used as the exposure date in estimation. This approach ensured the upper limit of error in the estimated incubation period be smaller than 1 day for the cases with a 2-day or 3-day exposure, regardless of when exactly (i.e., first, second, or third day) the transmission actually occurred. In reality, the overall error was bound to be much smaller than 1 day, as most included cases were exposed for only 1 day which would dilute the overall error.', 'cite_spans': [], 'ref_spans': [], 'section': 'Definitions and inclusion criteria'}, {'text': \"Serial interval was defined as the duration between symptom onset of an infector (e.g., a primary case) and that of an infectee (e.g., a secondary case) in a transmission chain, which was typically seen in cluster infections. A negative value meant that the infectee's symptoms occurred before the infector's symptoms. For two cases to qualify as an infector-infectee pair and be included in this study, the following two criteria must both be fulfilled. First, there was evidence that the presumed infector had been exposed outside the cluster (e.g., close contact with a confirmed case, travel history to Hubei, exposure to a person who returned from Hubei) before he/she joined the group activities such as family gathering and business conference that gave rise to cluster infections. Second, the presumed infectee was exposed to the presumed infector only in the 14 days prior to symptom onset, without other exposure histories.\", 'cite_spans': [], 'ref_spans': [], 'section': 'Definitions and inclusion criteria'}, {'text': \"Time point of exposure in this study referred to the day of infectee's exposure to infector relative to the latter's symptom onset date, and was estimated by the infectee's exposure date minus the infector's symptom onset date, where a negative value meant that the exposure occurred before the infector developed symptoms. The upper limit of latent period was defined as the interval between the exposure of infector and that of infectee in a transmission chain, which was used to estimate the longest possible latent period of an individual infector (the exact latent period cannot be observed in reality). If there were more than one infectee caused by an infector, only the one with the earliest exposure date was used in estimating this parameter.\", 'cite_spans': [], 'ref_spans': [], 'section': 'Definitions and inclusion criteria'}, {'text': 'Estimation of the time point of exposure and upper limit of latent period also involved determination of exposure date and judgement about transmission chain, hence the related principles applied in estimating incubation period and serial interval were also followed here.', 'cite_spans': [], 'ref_spans': [], 'section': 'Definitions and inclusion criteria'}, {'text': 'Six researchers were involved in data collection. The reports retrieved through the above searches were scrutinized one by one according to the inclusion criteria specified above. The following data were extracted from eligible reports by using a standard extraction form which was pilot-tested with the reports from Liaoning province of China: the geographical location All rights reserved. No reuse allowed without permission. perpetuity. preprint (which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in The copyright holder for this this version posted March 24, 2020. . https://doi.org/10.1101/2020.03. 21.20040329 doi: medRxiv preprint concerned, age, sex, type of exposure, first date and period (if applicable) of exposure, date of symptom onset, initial symptoms, severity, whether the case was from a cluster. For a clustering case, the generation he/she belonged to and the exposure date, symptom onset date and severity of the infector were also recorded. The retrieved reports were split into six parts, with each researcher responsible for one part. For each part, the eligibility of and data extracted from reports were firstly determined by one researcher and then cross-checked by another three researchers. All uncertainty and disagreements were discussed among the researchers. If no consensus could be reached after discussion, the concerned cases would be excluded to ensure the correctness and accuracy of data. For example, if it could not be determined who was infected through attending a group activity (i.e., infectee) and who had been infected before he/she attended the activity (i.e., infector), then all clustering cases related to the activity had to be excluded.', 'cite_spans': [{'start': 675, 'end': 686, 'text': '21.20040329', 'ref_id': None}], 'ref_spans': [], 'section': 'Screening, data extraction, and quality control'}, {'text': 'The basic characteristics of included cases were summarized descriptively. The lognormal, Weibull, and gamma distributions were fitted to the datasets of incubation period and upper limit of latent period, and the key parameters were estimated by using the maximum likelihood approach. For serial interval and time point of exposure, we assumed that they followed a normal distribution. 8 This was to address the negative values in the serial interval reported by previous studies, in which case the lognormal, Weibull, and gamma distributions may be limited. 24, 25 For each parameter, the range, median, selected percentiles, mean and standard deviation were estimated. The 95% confidence intervals (CI) of mean and standard deviation were estimated by using the bootstrap technique. R0 was estimated by using two widely accepted formula: 8, 9, 19, 20 ', 'cite_spans': [{'start': 560, 'end': 563, 'text': '24,', 'ref_id': 'BIBREF21'}, {'start': 564, 'end': 566, 'text': '25', 'ref_id': 'BIBREF22'}, {'start': 841, 'end': 843, 'text': '8,', 'ref_id': None}, {'start': 844, 'end': 846, 'text': '9,', 'ref_id': 'BIBREF7'}, {'start': 847, 'end': 850, 'text': '19,', 'ref_id': 'BIBREF17'}, {'start': 851, 'end': 853, 'text': '20', 'ref_id': 'BIBREF18'}], 'ref_spans': [], 'section': 'Data analysis'}, {'text': 'where r is exponential growth rate, μ is mean serial interval, σ is standard deviation of serial interval, 1/a is latent period, and 1/b is infectious period. The exponential growth rate was obtained directly from a previous study reporting incidence data of the early stage of epidemic, 1 while the other parameters came from the present study. The latent period was approximated by the upper limit of latent period, and the infectious period was approximated as serial interval minus the upper limit of latent period. Although the true value of latent period was bound to be smaller than the upper limit, we argued that this would not exert a significant impact on the estimation of R0 according to the above formula (2), because when the latent period became shorter the infectious period would become longer accordingly, given a fixed serial interval.', 'cite_spans': [], 'ref_spans': [], 'section': 'Data analysis'}, {'text': 'Sensitivity analyses were conducted to examine the robustness of the three parameters involving exposure date. Specifically, the third day for those whose exposure period spanned All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Data analysis'}, {'text': 'perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Data analysis'}, {'text': 'preprint (which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in', 'cite_spans': [], 'ref_spans': [], 'section': 'Data analysis'}, {'text': 'The copyright holder for this this version posted March 24, 2020. . https://doi.org/10.1101/2020.03.21.20040329 doi: medRxiv preprint three days and the second day for those exposed for two continuous days were used as their exposure dates in sensitivity analyses. The difference between the estimates from sensitivity analysis and those from primary analysis represents the largest possible error in the latter. For serial interval and time point of exposure, which had negative values, the datasets were also fitted with shifted lognormal, Weibull and gamma distributions to see if the mean and standard deviation would change much. Based on the new estimates from sensitivity analyses, R0 was also re-estimated. All statistical analyses were conducted with SAS software, version 9.4.', 'cite_spans': [], 'ref_spans': [], 'section': 'Data analysis'}, {'text': 'The process of data collection is summarized in Figure 1 ', 'cite_spans': [], 'ref_spans': [{'start': 48, 'end': 56, 'text': 'Figure 1', 'ref_id': 'FIGREF5'}], 'section': 'Process and output of data collection'}, {'text': 'The age of included cases ranged from 5 days to 90 years (Table 1 ). There were 122 cases (12.92%) with a travel history to Hubei province where Wuhan is the capital city. The source of exposure of 887 cases was known confirmed cases. The exposure period was exactly 1 day in 426 (50.3%) cases, spanned 2-3 days in 196 (23.1%) cases, and spanned more than 3 days in the other cases (who were not included in estimation of parameters involving exposure date).', 'cite_spans': [], 'ref_spans': [{'start': 57, 'end': 65, 'text': '(Table 1', 'ref_id': 'TABREF1'}], 'section': 'Characteristics of included cases'}, {'text': 'Among the cases with relevant information (n=329, 28.48%), 49 (14.89%) were asymptomatic, 256 (77.81%) mild to moderate, and 24 (7.29%) severe. The top five initial symptoms were fever (73.92%), cough (24.06%), fatigue (7.49%), malaise (7.20%), and chills (4.03%).', 'cite_spans': [], 'ref_spans': [], 'section': 'Characteristics of included cases'}, {'text': 'Clustering cases accounted for 81.4% of all, and most of them were the second generation. It should be noted that for each generation there were some cases excluded from this study, because no information was available for estimating the parameters of interest.', 'cite_spans': [], 'ref_spans': [], 'section': 'Characteristics of included cases'}, {'text': 'Incubation period were estimated from 587 cases, of whom 408 were exposed for 1 day and the others for two or three days. The incubation period of individual cases ranged from 0 to 23 days (Table 2) , with a median of 7 days and 6.64% (n=39) of them longer than 14 days.', 'cite_spans': [], 'ref_spans': [{'start': 189, 'end': 198, 'text': '(Table 2)', 'ref_id': None}], 'section': 'Epidemiological parameters'}, {'text': 'The estimates from lognormal, Weibull and gamma models were almost the same. Thus, lognormal model was used uniformly for this parameter (Figure 2(A) ). The mean incubation period was estimated at 7.44 days (95% CI: 7.10, 7.78) and the standard deviation 4.39 days (95% CI: 3.97, 4.49).', 'cite_spans': [], 'ref_spans': [{'start': 137, 'end': 149, 'text': '(Figure 2(A)', 'ref_id': 'FIGREF6'}], 'section': 'Epidemiological parameters'}, {'text': 'Only 11 pairs of infector-infectee was eligible for estimation of the upper limit of latent period, which ranged from 0 to 5 days (Table 2 ) with a median of 1 day. The mean and standard deviation derived from lognormal model, which fitted the dataset best (Figure 2(B) ), were 2.52 All rights reserved. No reuse allowed without permission. perpetuity. preprint (which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in The copyright holder for this this version posted March 24, 2020. . https://doi.org/10.1101/2020.03.21.20040329 doi: medRxiv preprint days and 3.95 days, respectively. The small sample size precluded further analysis.', 'cite_spans': [], 'ref_spans': [{'start': 130, 'end': 138, 'text': '(Table 2', 'ref_id': None}, {'start': 257, 'end': 269, 'text': '(Figure 2(B)', 'ref_id': 'FIGREF6'}], 'section': 'Epidemiological parameters'}, {'text': 'The serial interval was estimated for 689 pairs of infector-infectee and ranged from −5 to 24 days (Table 2) , with a median of 6 days. It was smaller than 0 (mean: −2.63 days) for 27 (3.92%) pairs, meaning that the infectee showed symptoms earlier than the infector. Assuming a normal distribution (Figure 2(C) ), the mean was 6.70 days (95% CI: 6.31, 7.10) and standard deviation 5.20 days (95% CI: 4.91, 5.46).', 'cite_spans': [], 'ref_spans': [{'start': 99, 'end': 108, 'text': '(Table 2)', 'ref_id': None}, {'start': 299, 'end': 311, 'text': '(Figure 2(C)', 'ref_id': 'FIGREF6'}], 'section': 'Epidemiological parameters'}, {'text': 'The time point of exposure was estimated from 233 pairs of infector-infectee and ranged from −9 to 13 days (Table 2) , with a median of 0 day. It was smaller than 0 (mean: −3.11 days) for 102 (43.78%) pairs, meaning that the transmission occurred before the infectors showed symptoms. Assuming a normal distribution (Figure 2(D) ), the mean was −0.19 days (95% CI: −0.62, 0.25) and standard deviation 3.32 days (95% CI: 2.97, 3.68).', 'cite_spans': [], 'ref_spans': [{'start': 107, 'end': 116, 'text': '(Table 2)', 'ref_id': None}, {'start': 316, 'end': 328, 'text': '(Figure 2(D)', 'ref_id': 'FIGREF6'}], 'section': 'Epidemiological parameters'}, {'text': 'Based on an exponential growth rate of 0.10 per day 1 and the mean (6.7 days) and standard deviation (5.2 days) of the serial interval, R0 was estimated at 1.71 (95% CI: 1.67, 1.75) according to formula (1) . Based on the same exponential growth rate, serial interval and mean of upper limit of latent period (2.52 days), R0 was estimated at 1.78 according to formula (2) .', 'cite_spans': [{'start': 203, 'end': 206, 'text': '(1)', 'ref_id': 'BIBREF0'}, {'start': 368, 'end': 371, 'text': '(2)', 'ref_id': 'BIBREF1'}], 'ref_spans': [], 'section': 'Epidemiological parameters'}, {'text': 'Please note that the 95% CI was not available here as the 95% CI of latent period was not estimated due to limited sample size, i.e., 11 pairs of samples.', 'cite_spans': [], 'ref_spans': [], 'section': 'Epidemiological parameters'}, {'text': 'In sensitivity analysis, the estimates of incubation period, latent period and time point of exposure remained stable, with a difference of 0.14-0.3 day from those in primary analysis (Table 2) Table 2 ). R0 remained almost the same in sensitivity analysis (data not shown).', 'cite_spans': [], 'ref_spans': [{'start': 184, 'end': 193, 'text': '(Table 2)', 'ref_id': None}, {'start': 194, 'end': 201, 'text': 'Table 2', 'ref_id': None}], 'section': 'Sensitivity and stratified and analyses'}, {'text': \"The results of stratified analysis, where asymptomatic cases were also included, are summarized in Table 3 . The three parameters were consistently associated with disease severity and generation in clusters. Specifically, for those with less severe disease or at lower generations in clusters, the incubation period, interval between their exposure and primary cases' symptom onset, and interval between their symptom onset and primary cases' symptom onset all tend to be longer. For those whose primary cases had less severe disease, the incubation period tends to be longer, while the interval between their symptom onset and primary cases' symptom onset tends to be shorter. No such trend was observed for other factors.\", 'cite_spans': [], 'ref_spans': [{'start': 99, 'end': 106, 'text': 'Table 3', 'ref_id': 'TABREF2'}], 'section': 'Sensitivity and stratified and analyses'}, {'text': 'By pooling individual data from seven countries, we estimated the mean incubation period of COVID-19 to be 7.44 days, latent period 2.52 days, serial interval 6.70 days, time point of All rights reserved. No reuse allowed without permission. perpetuity. preprint (which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in Our estimates of incubation period and serial interval are longer than most of the previous estimates. [2] [3] [4] [5] [6] [7] There are several possible reasons for the difference. First, the sample size is generally small in previous studies, 2, 4, 6, 7, [9] [10] [11] [12] but much larger in the present one. Second, most cases in previous studies had a long or even unclear interval of exposure, making it difficult to determine the exact exposure date 3, 5, 13 and giving rise to error. By contrast, this study followed strict inclusion criteria regarding exposure period and method to determine exposure date to ensure the potential error in the estimates be small (< 0.3 day, according to the sensitivity analysis). Third, the approaches to determination of transmission order within clusters are different. For the cases involving cluster infection, which represent the majority of all cases, a common practice of previous studies was to take the case with earliest date of symptom onset as infector and others as infectee. 9 However, we deemed it inappropriate in view of the varying incubation periods of individuals and potential pre-symptomatic or asymptomatic transmission.', 'cite_spans': [{'start': 490, 'end': 493, 'text': '[2]', 'ref_id': 'BIBREF1'}, {'start': 494, 'end': 497, 'text': '[3]', 'ref_id': 'BIBREF2'}, {'start': 498, 'end': 501, 'text': '[4]', 'ref_id': 'BIBREF3'}, {'start': 502, 'end': 505, 'text': '[5]', 'ref_id': 'BIBREF4'}, {'start': 506, 'end': 509, 'text': '[6]', 'ref_id': 'BIBREF5'}, {'start': 510, 'end': 513, 'text': '[7]', 'ref_id': 'BIBREF6'}, {'start': 632, 'end': 634, 'text': '2,', 'ref_id': 'BIBREF1'}, {'start': 635, 'end': 637, 'text': '4,', 'ref_id': 'BIBREF3'}, {'start': 638, 'end': 640, 'text': '6,', 'ref_id': 'BIBREF5'}, {'start': 641, 'end': 643, 'text': '7,', 'ref_id': 'BIBREF6'}, {'start': 644, 'end': 647, 'text': '[9]', 'ref_id': 'BIBREF7'}, {'start': 648, 'end': 652, 'text': '[10]', 'ref_id': 'BIBREF8'}, {'start': 653, 'end': 657, 'text': '[11]', 'ref_id': 'BIBREF9'}, {'start': 658, 'end': 662, 'text': '[12]', 'ref_id': 'BIBREF10'}, {'start': 844, 'end': 846, 'text': '3,', 'ref_id': 'BIBREF2'}, {'start': 847, 'end': 849, 'text': '5,', 'ref_id': 'BIBREF4'}, {'start': 850, 'end': 852, 'text': '13', 'ref_id': 'BIBREF11'}, {'start': 1419, 'end': 1420, 'text': '9', 'ref_id': 'BIBREF7'}], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'In this study, more rigorous criteria were applied to the inclusion of clusters, and the clusters in which transmission order was uncertain were excluded from relevant analyses. Fourth, different studies used data from different stages of the epidemic. The data of this study was retrieved through late February of 2020, which may reflect more of the situation after strong measures were taken to battle the epidemic in China.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'Our finding that the incubation period was within 14 days for 93% of the cases lends support to current practice of 14-day quarantine of persons with potential exposure to SARS-CoV-2. In line with other studies, 13 we also found some cases who developed symptoms 14 days after exposure, indicating that longer quarantine periods might be justified for some people.', 'cite_spans': [{'start': 212, 'end': 214, 'text': '13', 'ref_id': 'BIBREF11'}], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'However, as it is hard to know beforehand who will develop symptoms beyond 14 days of exposure, the cost of extending quarantine of many people and the potential consequence of failure to identify a few symptomatic cases must be weighed carefully.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': \"The negative values of serial interval and time point of exposure provide evidence of presymptomatic transmission, which are consistent with the much shorter latent period than incubation period. The percentages of negative serial interval (3.92%) and negative time point of exposure (43.78%) can be viewed as the lower and upper limits of probability of presymptomatic transmission, respectively. Asymptomatic transmission was also observed in this study (Table 3 ). These findings are in line with those of previous studies. 8,9,14-17 Stratified analysis in this study found a trend towards longer incubation period of less severe cases as compared with that of more severe cases. This implies that severe cases could be detected and 'removed' from the pool of transmission sources more quickly, while less severe cases would All rights reserved. No reuse allowed without permission. perpetuity. preprint (which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in The copyright holder for this this version posted March 24, 2020. . https://doi.org/10.1101/2020.03. 21.20040329 doi: medRxiv preprint stay in the pool for a longer time and constitute a big challenge to the control of epidemic given the possibility of pre-symptomatic transmission.\", 'cite_spans': [{'start': 1132, 'end': 1143, 'text': '21.20040329', 'ref_id': None}], 'ref_spans': [{'start': 456, 'end': 464, 'text': '(Table 3', 'ref_id': 'TABREF2'}], 'section': 'Discussion'}, {'text': 'The R0 we estimated is smaller than those from previous estimates which were mostly between 2 and 4. 7,20,26-28 As discussed above, the difference may either be due to methodological issues in obtaining parameters or reflect different stages of epidemic. In any case, a smaller R0', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'should not be interpreted as low risk of transmission. Slow response of government, presymptomatic and asymptomatic transmission and insufficient protection measures taken by the public together could lead to an out-of-control epidemic, as with the current situation in many countries.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'This study has limitations. First, the cases were retrieved from publicly available reports, and many cases were excluded because there were no details available for estimating the parameters. Thus, the representativeness of cases could be a concern. A comparison of the age and sex of included cases with those of 72314 cases in China showed that they are generally similar. 29 Previous studies suggested that publicly reported cases may overrepresent the severe ones. However, based on 329 cases (28.48%) with relevant information available, non-severe cases appeared to account for a higher proportion in the present study than in the previous one on 72314 cases in China. The second limitation of this study is that the upper limit of latent period was based on only 11 pairs of infector-infectee. Larger studies are needed to obtain a more robust estimate. The third limitation is that by using the upper limit of latent period instead of the real latent period in formula (2), in our case, could lead to a slightly smaller value of R0.', 'cite_spans': [{'start': 376, 'end': 378, 'text': '29', 'ref_id': 'BIBREF26'}], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'If we use a naively small value of the latent period at 0 days (i.e., no latency), the formula (2) would lead to a R0 at 1.67, which can be treated as the lower bound. Hence, under the formula (2), R0 may range from 1.67 to 1.78, which was consistent with those from formula (1), i.e., 1.71 (95% CI: 1.67, 1.75).', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'In conclusion, this study obtained robust estimates of several key epidemiological parameters of COVID-19. It provides additional evidence on the mean incubation period of COVID-19, which supports current practice of 14-day quarantine of persons with potential exposure but also suggests that longer monitoring periods might be needed for selected groups. The estimates of serial interval, time point of exposure and latent period provide consistent evidence on pre-symptomatic transmission. Asymptomatic transmission was also observed. This together with asymptomatic transmission and the generally longer incubation and serial interval of less severe cases suggest a high risk of long-term epidemic in the absence of appropriate control measures.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'YZ conceived the idea and designed the study, has full access to all data in the study, and takes responsibility for the integrity of data and the accuracy of data analysis. MS, ZJ, ZM, YQ, GW, ZY and YZ contributed to data acquisition, statistical analysis, and writing of the report. WMH All rights reserved. No reuse allowed without permission. perpetuity. preprint (which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in The copyright holder for this this version posted March 24, 2020. . https://doi.org/10.1101/2020.03. 21.20040329 doi: medRxiv preprint and ZS contributed to statistical analysis. All authors contributed to data interpretation and critical revision, and reviewed and approved the final version.', 'cite_spans': [{'start': 594, 'end': 627, 'text': '21.20040329 doi: medRxiv preprint', 'ref_id': None}], 'ref_spans': [], 'section': 'Contributors'}, {'text': 'All authors declare no competing interests.', 'cite_spans': [], 'ref_spans': [], 'section': 'Declaration of interests'}, {'text': 'The data that support the findings of this study are available from the corresponding author upon reasonable request via email. After publication of study findings, the data will be available for others to request. Table 2 . The estimates of incubation period, upper limit of latent period, serial interval and time point of exposure (unit: day) # 95% CI was not estimated for the parameter due to the very limited sample size.', 'cite_spans': [], 'ref_spans': [{'start': 215, 'end': 222, 'text': 'Table 2', 'ref_id': None}], 'section': 'Data sharing'}, {'text': '*Results of sensitivity analyses. For incubation period, the third day for those whose exposure period spanned three days and the second day for those exposed for two continuous days were used as their exposure dates in sensitivity analyses. For upper limit of latent period which involved two exposure dates, sensitivity analysis estimated the possible longest (S1) and shortest (S2) periods. For serial interval, sensitivity analysis fitted the dataset with shifted lognormal distribution. For time point of exposure, S1 presents the results with adjustment for exposure dates (similar to those of incubation period), while S2 presents the results based on shifted lognormal distribution. Abbreviations: CI, confidence interval; SD, standard deviation.', 'cite_spans': [], 'ref_spans': [], 'section': 'Data sharing'}, {'text': 'Upper limit of latent period # Serial interval Time point of exposure --*In these analyses the date of diagnosis was used the day of onset for asymptomatic cases. Asymptomatic infectees were excluded from the overall analysis of incubation period and serial interval, while asymptomatic infectors were excluded from the overall analysis of serial interval and time point of exposure. --, not applicable, or not available because of too few cases.', 'cite_spans': [], 'ref_spans': [], 'section': 'Incubation period'}, {'text': 'All rights reserved. No reuse allowed without permission. perpetuity. preprint (which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. preprint (which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in preprint (which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in The copyright holder for this this version posted March 24, 2020. perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Incubation period'}, {'text': 'preprint (which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in', 'cite_spans': [], 'ref_spans': [], 'section': 'Incubation period'}, {'text': 'The copyright holder for this this version posted March 24, 2020. . https://doi.org/10.1101/2020.03. 21.20040329 doi: medRxiv preprint ', 'cite_spans': [{'start': 101, 'end': 134, 'text': '21.20040329 doi: medRxiv preprint', 'ref_id': None}], 'ref_spans': [], 'section': 'Incubation period'}], 'paragraphs': [{'text': 'Despite the explosive growth of the number of studies on COVID-19, several key epidemiological parameters of the disease remain to be clarified, among which are incubation period and serial interval. The two parameters have important implications for the monitoring, surveillance and control of infectious disease. The mean or median incubation period and serial interval estimated by previous studies were mostly 4 to 5 days, 2-7 and 4 to 4.5 days, [8] [9] [10] [11] [12] respectively. While some of the studies included only a limited number of cases (around or less than 100), 2, 4, 6, 7, [9] [10] [11] [12] others might have suffered from inaccuracy of original data. For example, in some studies, most cases had a long or even unclear interval of exposure, making it difficult to determine the exact exposure date 3, 5, 13 and giving rise to error. Besides, the order of transmission (i.e., who is infector and who is infectee) in clustering cases, which is crucial to estimation of both parameters, is easy to be mistaken given the possibility of pre-symptomatic and asymptomatic transmission, [14] [15] [16] [17] but previous studies rarely described how this issue was handled.', 'id': '00009'}], 'title': 'Epidemiological parameters of coronavirus disease 2019: a pooled analysis of publicly reported individual data of 1155 cases from seven countries', 'sha': '3ece7750f80832bf9cc368c5138df12fd19183b9', 'doi': '10.1101/2020.03.21.20040329', 'score': 9.845499992370605}, {'rank': 16, 'paragraphs': [{'text': 'It is essential to know the accurate incubation period of COVID-19 for the sake of deciphering dynamics of its spread. The incubation period is the time from infection to the onset of the disease. It provides the foundation for epidemiological prevention, clinical actions, and drug discovery. Different viruses have different incubation periods that determine their different dynamics epidemiologically. The incubation period of H7N9 (Human Avian Influenza A) is about 6.5 days, but the incubation period for SARS-CoV is typically 2 to 7 days [5] [6] . However, it remains unclear about its exact incubation period of COVID-19, although WHO estimates it is between 2 to 14 days after exposure [8] . It can be difficult to estimate the All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'id': '00003'}], 'title': 'Estimate the incubation period of coronavirus 2019 (COVID-19)', 'sha': 'c1ae608c7ffb926a0f50a6a34c0780983274ea74', 'doi': '10.1101/2020.02.24.20027474', 'score': 9.793299674987793}, {'rank': 17, 'abstract': 'Background: The incubation period of SARS-CoV-2 remains uncertain, which has important implications for estimating transmission potential, forecasting epidemic trends, and decision-making in prevention and control. Purpose: To estimate the central tendency and dispersion for incubation period of COVID-19 and, in turn, assess the effect of a certain length of quarantine for close contacts in active monitoring. Data Sources: PubMed, Embase, medRxiv, bioRxiv, and arXiv, searched up to April 26, 2020 Study Selection: COVID-19 studies that described either individual-level incubation period data or summarized statistics for central tendency and dispersion measures of incubation period were recruited. Data Extraction: From each recruited study, either individual-level incubation period data or summarized statistics for central tendency and dispersion measures were extracted, as well as population characteristics including sample size, average age, and male proportion. Data Synthesis: Fifty-six studies encompassing 4 095 cases were included in this meta-analysis. The estimated median incubation period for general transmissions was 5.8 days [95% confidence interval (95%CI), 5.3 to 6.2 d]. Median and dispersion were higher for SARS-CoV-2 incubation compared to other viral respiratory infections. Furthermore, about 20 in 10 000 contacts in active monitoring would develop symptoms after 14 days, or below 1 in 10 000 for young-age infections or asymptomatic transmissions. Limitation: Small sample sizes for subgroups; some data were possibly used repeatedly in different studies; limited studies for outside mainland China; non-negligible intra-study heterogeneity. Conclusion: The long, dispersive incubation period of SARS-CoV-2 contributes to the global spread of COVID-19. Yet, a 14-day quarantine period is sufficient to trace and identify symptomatic infections, which while could be justified according to a better understanding of the crucial parameters.', 'body_text': [{'text': \"In December 2019, a cluster of pneumonia cases with unclear pathogenesis was reported in Wuhan, Hubei Province, China. This virus was named by World Health Organization (WHO) as the severe acute respiratory syndrome coronavirus (SARS-CoV-2) and the disease it caused was named as coronavirus disease 2019 (COVID- 19) on February 11, 2020 (1). Consequently, COVID-19 was urgently classified as a Class B communicable disease and managed as a Class A communicable disease in accordance with the Law of the People's Republic of China on the Prevention and Treatment of Infectious Disease (2) . Meanwhile, the COVID-19 epidemic continued to spread around the globe, with rapid increases in case numbers in European and American countries, and a looming threat in resource-limited settings across Africa (3) . The World Health Organization (WHO) declared a global pandemic on March 11, 2020. As of May 25, the pandemic had spread to 188 countries on six continents, with a total of over 5 million diagnosed cases worldwide (4) .\", 'cite_spans': [{'start': 313, 'end': 316, 'text': '19)', 'ref_id': 'BIBREF18'}, {'start': 320, 'end': 337, 'text': 'February 11, 2020', 'ref_id': None}, {'start': 585, 'end': 588, 'text': '(2)', 'ref_id': 'BIBREF0'}, {'start': 799, 'end': 802, 'text': '(3)', 'ref_id': 'BIBREF1'}, {'start': 1018, 'end': 1021, 'text': '(4)', 'ref_id': 'BIBREF2'}], 'ref_spans': [], 'section': 'INTRODUCTION'}, {'text': 'Defining the incubation period of any infectious disease is crucial to evaluate transmission potential, estimate epidemic trends, and inform active monitoring and/or mandatory quarantine policies. The novel pathogenesis of COVID-19 has produced varied epidemiological characteristics from previous coronavirus-derived pulmonary infectious diseases, such as severe acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS). SARS and MERS were rarely transmitted during the asymptomatic period (5, 6) . In contrast, increasing evidence indicates that individuals infected with SARS-CoV-2 could be infectious during the asymptomatic incubation period (7) (8) (9) . Thus, knowledge of length and dispersion of incubation period is crucial for SARS-CoV-2 prevention and control. In addition, transmission dynamics models are designed to mimic the spread of SARS-CoV-2 in a nonlinear fashion, and are broadly used for long-term forecasting and evaluating the effect of prevention measures (10) . However, many parameters associated with SARS-CoV-2 transmission are poorly understood, including the incubation period, resulting in a biased prediction (11) . Multiple studies have explored the incubation period for COVID-19, but conclusions remain controversial due to limited sample sizes for each study and considerable heterogeneity between studies (12, 13) .', 'cite_spans': [{'start': 391, 'end': 397, 'text': '(SARS)', 'ref_id': None}, {'start': 512, 'end': 515, 'text': '(5,', 'ref_id': 'BIBREF3'}, {'start': 516, 'end': 518, 'text': '6)', 'ref_id': 'BIBREF4'}, {'start': 668, 'end': 671, 'text': '(7)', 'ref_id': 'BIBREF5'}, {'start': 672, 'end': 675, 'text': '(8)', 'ref_id': 'BIBREF6'}, {'start': 676, 'end': 679, 'text': '(9)', 'ref_id': 'BIBREF7'}, {'start': 1003, 'end': 1007, 'text': '(10)', 'ref_id': 'BIBREF8'}, {'start': 1164, 'end': 1168, 'text': '(11)', 'ref_id': 'BIBREF9'}, {'start': 1365, 'end': 1369, 'text': '(12,', 'ref_id': 'BIBREF10'}, {'start': 1370, 'end': 1373, 'text': '13)', 'ref_id': None}], 'ref_spans': [], 'section': 'INTRODUCTION'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'INTRODUCTION'}, {'text': 'The copyright holder for this preprint this version posted June 22, 2020. . https://doi.org/10.1101/2020.06.20.20134387 doi: medRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'Given the continuing global spread of COVID-19, a further investigation of viral incubation by a systematic review and meta-analysis could provide urgently needed support to improve the understanding of COVID-19 transmission potential and aid prediction and decision-making.', 'cite_spans': [], 'ref_spans': [], 'section': '/ 16'}, {'text': 'In this systematic review and meta-analysis, we searched PubMed, Embase, medRxiv, bioRxiv, and arXiv to identify studies related to COVID-19 published or publicly posted from December 01, 2019 to April 26, 2020 (date of last search) in parallel by two authors (L.W. and Y.L.). Each database was searched using the terms \"(COVID-19) OR (2019-nCoV) OR (novel coronavirus pneumonia)\". The search strategy is detailed in the Supplement Table 1-4. There were no language restrictions on the search.', 'cite_spans': [], 'ref_spans': [], 'section': 'Data Sources and Searches'}, {'text': 'COVID-19 studies that described either individual-level incubation period data or summarized statistics for central tendency and dispersion measures of incubation period were recruited. Studies were excluded if they met either of the below criteria: (1) irrelevant subject to incubation period; (2) no individual-level incubation period or insufficient summarized statistics for incubation period (central tendency and dispersion measures are required); (3) non-human studies; (4) sample size for incubation analysis less than 5; (5) studies of insufficient quality; (6) ambiguous definition of incubation period. Figure 1 describes the literature searching steps. Duplicate studies and studies irrelevant to incubation period were deleted, and studies identified via reference list searches were added. Two reviewers then selected 10% of the retrieved articles at random and independently reviewed the title and abstract according to the predefined set of exclusion criteria. Exclusion criteria were applied consistently, indicating high concordance. In case of uncertainty about inclusion or exclusion, the reviewers consulted together.', 'cite_spans': [], 'ref_spans': [{'start': 614, 'end': 622, 'text': 'Figure 1', 'ref_id': 'FIGREF0'}], 'section': 'Study Selection'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'cite_spans': [], 'ref_spans': [], 'section': 'Data Extraction and Quality Assessment'}, {'text': 'The copyright holder for this preprint this version posted June 22, 2020. . https://doi.org/10.1101/2020.06.20.20134387 doi: medRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': 'Data Extraction and Quality Assessment'}, {'text': 'The literature quality assessment was evaluated in parallel by two researchers (L.W., Y.L.) according to Agency for Healthcare Research and Quality (AHRQ) guidelines (Supplement Table 5 ). Disagreement between the two researchers was resolved by consensus and the resolution was confirmed by two senior authors (F.C., Y.W.). From each recruited study, either individual-level incubation period data or summarized statistics for central tendency (mean or median) and dispersion (variance, standard deviation, interquatiles, or range) measures were extracted, as well as population characteristics including sample size, average age, and male proportion (Supplement Table 6 ). Data were extracted by two independent research coordinators from each publication (L.W., Y.L.); inconsistent inputs were verified and justified by a third author to ensure correctness of data extraction (Y.W.). All literature included in the meta-analysis was labeled as \"General Transmissions\".', 'cite_spans': [], 'ref_spans': [{'start': 178, 'end': 185, 'text': 'Table 5', 'ref_id': None}, {'start': 664, 'end': 671, 'text': 'Table 6', 'ref_id': None}], 'section': '/ 16'}, {'text': 'In addition, three publications reported characteristics of incubation among cases infected by asymptomatic or presymptomatic carriers, and two studies had a subset of cases infected by carriers in the asymptomatic period (14, 15) ; these five studies were labeled as \"Asymptomatic Transmissions\".', 'cite_spans': [{'start': 222, 'end': 226, 'text': '(14,', 'ref_id': 'BIBREF13'}, {'start': 227, 'end': 230, 'text': '15)', 'ref_id': 'BIBREF14'}], 'ref_spans': [], 'section': '/ 16'}, {'text': 'Five studies investigated the incubation period among young-age infections (age < 20 years), which were labeled as \" Young-age Infections\".', 'cite_spans': [], 'ref_spans': [], 'section': '/ 16'}, {'text': 'Incubation period was assumed to follow a log-normal distribution (16) . Parameters of the log-normal distribution including mean and variation were calculated for each study (Supplement Table 6 -12).', 'cite_spans': [{'start': 66, 'end': 70, 'text': '(16)', 'ref_id': 'BIBREF15'}], 'ref_spans': [{'start': 187, 'end': 194, 'text': 'Table 6', 'ref_id': None}], 'section': 'Data Synthesis and Analysis'}, {'text': 'Funnel plots and Egger tests were used to show the potential publication bias and study heterogeneity. The expected mean of log-scaled incubation period was summarized by meta-analysis followed by exponential calculation to obtain the median of incubation period and the corresponding 95% confidence interval (95%CI). Random-effects meta-analysis was used if P value for heterogeneity test ≤ 0.05; otherwise, fixed-effects meta-analysis would be used.', 'cite_spans': [], 'ref_spans': [], 'section': 'Data Synthesis and Analysis'}, {'text': 'On the other hand, the dispersion for the incubation period was estimated by e σ in which the σ is the estimate of standard deviation of the corresponding log-scaled distribution (17) . Variances of log-scaled . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'cite_spans': [{'start': 179, 'end': 183, 'text': '(17)', 'ref_id': 'BIBREF16'}], 'ref_spans': [], 'section': 'Data Synthesis and Analysis'}, {'text': 'The copyright holder for this preprint this version posted June 22, 2020. . https://doi.org/10.1101/2020.06.20.20134387 doi: medRxiv preprint 6 / 16 incubation period of the recruited studies were assumed to follow the inverse gamma distribution to estimate the expectation value. Bootstrap was used to estimate the corresponding 95%CI of the dispersion.', 'cite_spans': [], 'ref_spans': [], 'section': 'Data Synthesis and Analysis'}, {'text': 'Further, the distribution of the incubation period was simulated for the general transmissions, asymptomatic transmissions, and young-age infections, respectively. The 1 000 posterior means of the log-scaled distribution of incubation period were generated using the Bayesian model that produces probability distribution for each parameter, within R package bayesmeta (18) . In addition, the 1 000 standard deviations of the log-scaled distribution were generated by Bootstrap sampling. The 1 000 means and standard deviations of the log-scaled distribution were used to simulate the distribution of incubation period. The proportion of infections developing symptoms after a certain length of quarantine were estimated; the risk of symptomatic SARS-CoV-2 infections being undetected after a certain length of active monitoring among the active-monitoring population was estimated as well (19, 20) .', 'cite_spans': [{'start': 368, 'end': 372, 'text': '(18)', 'ref_id': 'BIBREF17'}, {'start': 889, 'end': 893, 'text': '(19,', 'ref_id': 'BIBREF18'}, {'start': 894, 'end': 897, 'text': '20)', 'ref_id': 'BIBREF20'}], 'ref_spans': [], 'section': 'Data Synthesis and Analysis'}, {'text': 'Last, meta-regression was used to explore the association between the age, sex, and median incubation period. All statistical analyses were performed using R software version 3.6.1 (Foundation for Statistical Computing).', 'cite_spans': [], 'ref_spans': [], 'section': 'Data Synthesis and Analysis'}, {'text': 'This study was funded by the National Natural Science Foundation of China (82041024 to F.C., 82041026 to H.S., 81973142 to Y.W.). Sponsors had no role in design of the study, collection and analysis of data, or preparation of the manuscript.', 'cite_spans': [], 'ref_spans': [], 'section': 'Role of the Funding Source'}, {'text': 'Our search retrieved 13 711 records, of which 13 403 were irrelevant to incubation period and were excluded during screening of titles and abstracts ( Figure 1 ). Fifty-three duplicate studies and 1 study that had no access to full text were removed. By careful screening the full text of the remaining 254 studies, we found that 130 with no summarized statistics for incubation period, 57 non-human studies, . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [{'start': 151, 'end': 159, 'text': 'Figure 1', 'ref_id': 'FIGREF0'}], 'section': 'RESULTS'}, {'text': 'The copyright holder for this preprint this version posted June 22, 2020. . https://doi.org/10.1101/2020.06.20.20134387 doi: medRxiv preprint 7 / 16 11 studies with sample size less than 5, 1 study of insufficient quality, and 1 with ambiguous definition of incubation period were ineligible. Meanwhile, 2 additional studies were identified via reference list searches. Finally, there were 56 studies that met the inclusion criteria, including 34 published studies, 22 preprint studies, and 4 095 COVID-19 infections in total. Literature quality was evaluated for each included study according to AHRQ guidelines (Supplement Table 5 ). Summarized statistics of incubation periods and population characteristics extracted from each study were shown in Supplement Table 6 .', 'cite_spans': [], 'ref_spans': [{'start': 625, 'end': 632, 'text': 'Table 5', 'ref_id': None}, {'start': 762, 'end': 769, 'text': 'Table 6', 'ref_id': None}], 'section': '(which was not certified by peer review)'}, {'text': 'Parameters for log-normal distribution of incubation period were derived for each study (Supplement For comparison with other viral respiratory infections, the summarized statistics of incubation periods for 9 viral respiratory infections were obtained from a previously published systematic review, including measles, adenovirus, respiratory syncytial virus, SARS-CoV, human coronavirus, parainfluenza, rhinovirus, influenza A, and influenza B (17) . In addition, meta-analysis was performed . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. Table 14 ) (21), assuming a 6.15% risk of symptomatic infections among the overall active monitoring population with close contacts, the estimated probability of symptomatic SARS-CoV-2 infections that would be undetected after 14-day active monitoring or quarantine was 20.5 (95%CI,5.2-50.4) per 10 000 monitored individuals ( Figure 4G ). Similarly, one study summarized the proportion of young-age infections at 1.6% among general infections (22) ; thus, among the overall active monitoring population, the estimated probability of symptomatic SARS-CoV-2 infections that would be undetected after 14-day active monitoring was 0.9 in 10 000 (95%CI: 0.2-2.2 in 10 000) ( Figure 4H ). In addition, assuming a 4.11% risk of symptomatic infections among contacts with latent infections and a proportion of latent infections at 9.3% in the overall active monitoring population, the estimated probability of undetected SARS-CoV-2 infections with no sympotom within 14-day active . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 445, 'end': 449, 'text': '(17)', 'ref_id': 'BIBREF16'}, {'start': 1101, 'end': 1105, 'text': '(22)', 'ref_id': 'BIBREF22'}], 'ref_spans': [{'start': 657, 'end': 665, 'text': 'Table 14', 'ref_id': None}, {'start': 984, 'end': 993, 'text': 'Figure 4G', 'ref_id': 'FIGREF7'}, {'start': 1328, 'end': 1337, 'text': 'Figure 4H', 'ref_id': 'FIGREF7'}], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted June 22, 2020. . https://doi.org/10.1101/2020.06.20.20134387 doi: medRxiv preprint 9 / 16 monitoring was 0.2 in 10 000 (95%CI: 0.0-3.1 in 10 000) ( Figure 4I ). Sensitivity analyses considering various settings for the risk of being infected among an active monitoring population were performed as well (Supplement Table 14 ). Overall, in the active monitoring population, the risk of developing symptoms after a 14-day quarantine period was about 20 in 10 000, while the risk of young age infections and asymptomatic transmissions developing symptoms after 14-day quarantine period was below 1 in 10 000.', 'cite_spans': [], 'ref_spans': [{'start': 207, 'end': 216, 'text': 'Figure 4I', 'ref_id': 'FIGREF7'}, {'start': 374, 'end': 382, 'text': 'Table 14', 'ref_id': None}], 'section': '(which was not certified by peer review)'}, {'text': 'Average age and male proportion were extracted from 24 of 56 studies. Meta-regression incorporating two moderators simultaneously was used to explore the impact of individual characteristics on length of incubation period. A linear relationship was identified between age and log-scaled median of incubation period. Average age per 10-year increments resulted in a 16% increment in median incubation period with adjustment for male proportion (incubation period ratio, 1.16, 95%CI 1.01-1.32; P = 0.0250) (Supplement Figure 6A) . No evidence indicated an association between sex and median of incubation period (P = 0.1315) (Supplement Figure 6B ).', 'cite_spans': [], 'ref_spans': [{'start': 516, 'end': 526, 'text': 'Figure 6A)', 'ref_id': None}, {'start': 635, 'end': 644, 'text': 'Figure 6B', 'ref_id': None}], 'section': '(which was not certified by peer review)'}, {'text': 'Finally, an interactive real-time risk assessment application was developed to provide real-time updates of the risk assessment for symptomatic SARS-CoV-2 infections that would be undetected during active monitoring among active monitoring population with close contacts by setting several crucial parameters (Supplement Figure 10 ).', 'cite_spans': [], 'ref_spans': [{'start': 321, 'end': 330, 'text': 'Figure 10', 'ref_id': 'FIGREF0'}], 'section': '(which was not certified by peer review)'}, {'text': 'Increasing evidence supports the transmission potential of SARS-CoV-2 during the latent period (7-9).', 'cite_spans': [], 'ref_spans': [], 'section': 'DISCUSSION'}, {'text': 'Thus, length of incubation period is a crucial parameter to determine the risks for close contacts and guide contact tracing and quarantine policies. The estimated median incubation period in this study was 5.8 days for general transmissions; the estimated mean incubation was 6.9 days which is 33% longer than the previously frequently adopted value-5.2 mean days as reported by Li (16) . Notably, asymptomatic transmissions and young-age infections appear to have an almost two-day longer incubation period than general transmissions. Infections contacted with latent infections may have a . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 383, 'end': 387, 'text': '(16)', 'ref_id': 'BIBREF15'}], 'ref_spans': [], 'section': 'DISCUSSION'}, {'text': 'The copyright holder for this preprint this version posted June 22, 2020. . https://doi.org/10.1101/2020.06.20.20134387 doi: medRxiv preprint 10 / 16 low viral load that requires a longer incubation to develop symptoms (23, 24) . On the other hand, young-age infections probably have a strong immune status that results in a longer incubation.', 'cite_spans': [{'start': 219, 'end': 223, 'text': '(23,', 'ref_id': 'BIBREF23'}, {'start': 224, 'end': 227, 'text': '24)', 'ref_id': 'BIBREF24'}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'Interestingly, preprint studies appear to report longer incubation period than do journal publications, indicating potential publication bias. However, large variation of estimates was observed among studies, which indicates a non-negligible heterogeneity in COVID-19 patients. Evolution of SARS-CoV-2 might partially address this heterogeneity (25) .', 'cite_spans': [{'start': 345, 'end': 349, 'text': '(25)', 'ref_id': 'BIBREF25'}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': \"Our study demonstrates that SARS-CoV-2 has a considerably longer incubation period than most types of viral respiratory infections. Notably, a significant positive association between length of incubation period and magnitude of R 0 was observed. This finding indicates viral respiratory infections beyond SARS-CoV-2 may have transmission potential during their incubation period. In addition, SARS-CoV-2 has a high dispersion of incubation period, which increases the difficulty in tracing and controlling for contacts. These unique epidemiologic characteristics partially contribute to today's global spread of COVID-19. The 14-day quarantine period has been adopted in mainland China and suggested to the international community by WHO (26) . We estimated that 7 out of 100 general infections would develop symptoms after 14 days. Further, nearly 15 out of 100 infected people under age 20 years will develop symptoms after 14 days. However, considering the probability of being infected among contacts of the population in active monitoring or quarantine, about 20 per 10 000 contacts would develop symptoms after 14 days in active monitoring or quarantine; the risk of observing young-age infections or asymptomatic transmissions developing symptoms after 14 days is below 1 in 10 000. Assuming the risks of being infected for the active monitoring population having contacts with symptomatic infections and latent infections were 1% in equal, about 3 in 10 000 contacts would develop symptoms after 14 days, which is more than 1.0 in 10 000 reported in the previous study. 19 Thus, the 14-day quarantine policy is sufficient to trace and identify infections among an active monitoring population. However, precise understanding of the crucial epidemiological parameters related to transmission probability in active monitoring population could aid in further refining the appropriate length of quarantine (27) .\", 'cite_spans': [{'start': 739, 'end': 743, 'text': '(26)', 'ref_id': 'BIBREF26'}, {'start': 1579, 'end': 1581, 'text': '19', 'ref_id': 'BIBREF18'}, {'start': 1911, 'end': 1915, 'text': '(27)', 'ref_id': 'BIBREF27'}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted June 22, 2020. . https://doi.org/10.1101/2020.06. 20.20134387 doi: medRxiv preprint In addition, age is likely to have a positive relationship with the length of incubation period. Notably, the result in our study indicated that older adults had a longer incubation period than younger adults, which was consistent with the findings of previous studies (28, 29) . Older adults tend to have more health complications such as respiratory issues and chronic diseases; thus, pre-existing symptoms may mask the onset of COVID-19 symptoms, which could bias the measurement of incubation period.', 'cite_spans': [{'start': 109, 'end': 142, 'text': '20.20134387 doi: medRxiv preprint', 'ref_id': None}, {'start': 412, 'end': 416, 'text': '(28,', 'ref_id': 'BIBREF28'}, {'start': 417, 'end': 420, 'text': '29)', 'ref_id': 'BIBREF29'}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'However, the underlying mechanism is unclear and warrants further investigation.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'We acknowledge some limitations of this study. First, the sample sizes for asymptomatic transmissions and young-age infections are small, and the results for these subgroups may be less representative.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'Second, most studies obtained data from public resources, and the raw data were not provided; there is a possibility that some data were used repeatedly. Third, 50 of 56 studies were from mainland China; studies from other regions and countries are needed to explore the impact of viral evolution on variation of incubation period and other epidemiological characteristics. Fourth, precisely estimating the exposure window and time of symptom onset related to SARS-CoV-2 infection could be difficult in practice. Studies used different methods to quantify the uncertainty of incubation period for each individual, which may partially explain the non-negligible intra-study heterogeneity. Last, knowledge of the risk of being infected among close contacts is limited and may vary due to different definition of close contacts.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'In conclusion, this study integrated 56 studies and 4 095 COVID-19 infections and estimated the median and dispersion of the SARS-CoV-2 incubation period, both of which ranked second among 11 viral respiratory infections. A long and dispersive incubation period probably contributes to the increasing spread of COVID-19 worldwide. Yet, the 14-day quarantine period is sufficient to trace and identify symptomatic infections among an active monitoring population, and monitoring and quarantine policies may be adjusted to accommodate higher-risk age groups.', 'cite_spans': [], 'ref_spans': [], 'section': 'CONCLUSIONS'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. ', 'cite_spans': [], 'ref_spans': [], 'section': 'CONCLUSIONS'}, {'text': 'The authors have declared that no conflict of interest exists. All authors have completed the ICMJE uniform disclosure form at www.icmje.org/coi_disclosure.pdf and declare: no support from any organization for the submitted work; no financial relationships with any organizations that might have an interest in the submitted work in the previous three years; no other relationships or activities that could appear to have influenced the submitted work.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conflicts of Interest:'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conflicts of Interest:'}, {'text': 'The copyright holder for this preprint this version posted June 22, 2020. . https://doi.org/10.1101/2020.06.20.20134387 doi: medRxiv preprint . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. Studies were ordered by date of post online. Significant heterogeneity was observed among studies (I 2 = 96.1%, P < 0.0001). The random-effects meta-analysis using restricted maximum likelihood (REML) was used to summarize the median incubation period (days) and the corresponding 95% confidence interval (95%CI). CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted June 22, 2020. In addition, assuming a 6.15% risk of being infected among the general quarantine population, the risk of having infections develop symptoms after 14-day active monitoring or quarantine was estimated (G).', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'Further, assuming the young age infections account for 1.6% of general transmissions, the probability of having young age infections among the general quarantine population was set at 6.15%×1.6%; the risk of having infections develop symptoms after 14-day quarantine period among the quarantine population was estimated (H). Similarly, assuming that individuals who contacted latent infections during the asymptomatic or presymptomatic period, which account for 9.3% of the general quarantine population, and 4.11% probability to be infected, the risk of observed asymptomatic transmissions developing symptoms after 14-day quarantine period is shown in Panel (I).', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted June 22, 2020. ', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'Retained after screening of titles and abstracts . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'cite_spans': [], 'ref_spans': [], 'section': 'studies'}, {'text': 'The copyright holder for this preprint this version posted June 22, 2020. . https://doi.org/10.1101/2020.06.20.20134387 doi: medRxiv preprint CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'cite_spans': [], 'ref_spans': [], 'section': 'studies'}, {'text': 'The copyright holder for this preprint this version posted June 22, 2020. . https://doi.org/10.1101/2020.06.20.20134387 doi: medRxiv preprint is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'cite_spans': [], 'ref_spans': [], 'section': 'studies'}, {'text': 'The copyright holder for this preprint this version posted June 22, 2020. . https://doi.org/10.1101/2020.06.20.20134387 doi: medRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': 'studies'}], 'paragraphs': [{'text': 'In conclusion, this study integrated 56 studies and 4 095 COVID-19 infections and estimated the median and dispersion of the SARS-CoV-2 incubation period, both of which ranked second among 11 viral respiratory infections. A long and dispersive incubation period probably contributes to the increasing spread of COVID-19 worldwide. Yet, the 14-day quarantine period is sufficient to trace and identify symptomatic infections among an active monitoring population, and monitoring and quarantine policies may be adjusted to accommodate higher-risk age groups.', 'id': '00036'}], 'title': 'A systematic review and meta-analysis reveals long and dispersive incubation period of COVID-19', 'sha': '3d921c03dc23ee850655ddfd9d5ebe3956d70bf8', 'doi': '10.1101/2020.06.20.20134387', 'score': 9.788299560546875}, {'rank': 18, 'abstract': 'The COVID-19 pandemic is a harsh reminder of the fact that, whether in a single human host or a wave of infection across continents, viral dynamics is often a story about the numbers. In this article we provide a one-stop, curated graphical source for the key numbers (based mostly on the peer-reviewed literature) about the SARS-CoV-2 virus that is responsible for the pandemic. The discussion is framed around two broad themes: i) the biology of the virus itself; ii) the characteristics of the infection of a single human host.', 'body_text': [{'text': \"The COVID-19 pandemic has made brutally clear the need for further research into many aspects of viruses. In this article we compile data about the basic properties of the SARS-CoV-2 virus, and about how it interacts with the body (Figure 1). We also discuss a number of questions about the virus, and perform 'back-of-the-envelope' calculations to show the insights that can be gained from knowing some key numbers and using quantitative reasoning. It is important to note that much uncertainty remains, and while 'back-of-the-envelope' calculations can improve our intuition through sanity checks, they cannot replace detailed epidemiological analysis.\", 'cite_spans': [], 'section': 'Introduction', 'ref_spans': [{'start': 232, 'end': 240, 'mention': 'Figure 1', 'ref_id': 'FIGREF0'}]}, {'text': \"If everybody continued to behave as usual, how long would it take the pandemic to spread from one person to a million infected victims? The basic reproduction number, R0, suggests each infection directly generates 2–4 more infections in the absence of countermeasures like physical distancing. Once a person is infected, it takes a period of time known as the 'latent period' before they are able to transmit the virus. The current best-estimate of the median latent time is ≈3 days followed by ≈4 days of close to maximal infectiousness (Li et al., 2020a; He et al., 2020). The exact durations vary among people, and some are infectious for much longer. Using R0≈4, the number of cases will quadruple every ≈7 days or double every ≈3 days. 1000-fold growth (going from one case to 103) requires 10 doublings since 210 ≈ 103; 3 days × 10 doublings = 30 days, or about one month. So we expect ≈1000x growth in one month, a million-fold (106) in two months, and a billion fold (109) in three months. Even though this calculation is highly simplified, ignoring the effects of 'super-spreaders', herd-immunity and incomplete testing, it emphasizes the fact that viruses can spread at a bewildering pace when no countermeasures are taken. This illustrates why it is crucial to limit the spread of the virus by physical distancing measures. For fuller discussion of the meaning of R0, the latent and infectious periods, as well as various caveats, see the section on 'Definitions and measurement methods' below.\", 'cite_spans': [{'start': 539, 'end': 555, 'mention': 'Li et al., 2020a', 'ref_id': 'BIBREF18'}, {'start': 557, 'end': 572, 'mention': 'He et al., 2020', 'ref_id': 'BIBREF6'}], 'section': '1. How long does it take a single infected person to yield one million infected people? ::: Eight questions about SARS-CoV-2', 'ref_spans': []}, {'text': \"A highly simplified quantitative example helps clarify the need for physical distancing. Suppose that you are infected and you encounter 50 people over the course of a day of working, commuting, socializing and running errands. To make the numbers round, let's further suppose that you have a 2% chance of transmitting the virus in each of these encounters, so that you are likely to infect one new person each day. If you are infectious for 4 days, then you will infect four others on average, which is on the high end of the R0 values for SARS-CoV-2 in the absence of physical distancing. If you instead see five people each day (preferably fewer) because of physical distancing, then you will infect 0.1 people per day, or 0.4 people before you become less infectious. The desired effect of physical distancing is to make each current infection produce <1 new infections. An effective reproduction number (Re) smaller than one will ensure the number of infections eventually dwindles. It is critically important to quickly achieve Re < 1, which is substantially more achievable than pushing Re to near zero through public health measures.\", 'cite_spans': [], 'section': '2. What is the effect of physical distancing? ::: Eight questions about SARS-CoV-2', 'ref_spans': []}, {'text': 'The period of time from infection to symptoms is termed the incubation period. The median SARS-CoV-2 incubation period is estimated to be roughly 5 days (Lauer et al., 2020). Yet there is much person-to-person variation. Approximately 99% of those showing symptoms will show them before day 14, which explains the two week confinement period. Importantly, this analysis neglects infected people who never show symptoms. Since asymptomatic people are not usually tested, it is still not clear how many such cases there are or how long asymptomatic people remain infectious for.', 'cite_spans': [{'start': 154, 'end': 172, 'mention': 'Lauer et al., 2020', 'ref_id': 'BIBREF16'}], 'section': '3. Why was the initial quarantine period two weeks? ::: Eight questions about SARS-CoV-2', 'ref_spans': []}, {'text': \"N95 masks are designed to remove more than 95% of all particles that are at least 0.3 microns (µm) in diameter. In fact, measurements of the particle filtration efficiency of N95 masks show that they are capable of filtering ≈99.8% of particles with a diameter of ≈0.1 μm (Rengasamy et al., 2017). SARS-CoV-2 is an enveloped virus ≈0.1 μm in diameter, so N95 masks are capable of filtering most free virions, but they do more than that. How so? Viruses are often transmitted through respiratory droplets produced by coughing and sneezing. Respiratory droplets are usually divided into two size bins, large droplets (>5 μm in diameter) that fall rapidly to the ground and are thus transmitted only over short distances, and small droplets (≤5 μm in diameter). Small droplets can evaporate into 'droplet nuclei', remain suspended in air for significant periods of time and could be inhaled. Some viruses, such as measles, can be transmitted by droplet nuclei (Tellier et al., 2019). Larger droplets are also known to transmit viruses, usually by settling onto surfaces that are touched and transported by hands onto mucosal membranes such as the eyes, nose and mouth (CDC, 2020). The characteristic diameter of large droplets produced by sneezing is ~100 μm (Han et al., 2013), while the diameter of droplet nuclei produced by coughing is on the order of ~1 μm (Yang et al., 2007). At present, it is unclear whether surfaces or air are the dominant mode of SARS-CoV-2 transmission, but N95 masks should provide some protection against both (Jefferson et al., 2009; Leung et al., 2020).\", 'cite_spans': [{'start': 273, 'end': 295, 'mention': 'Rengasamy et al., 2017', 'ref_id': 'BIBREF34'}, {'start': 958, 'end': 978, 'mention': 'Tellier et al., 2019', 'ref_id': 'BIBREF43'}, {'start': 1166, 'end': 1175, 'mention': 'CDC, 2020', 'ref_id': 'BIBREF11'}, {'start': 1257, 'end': 1273, 'mention': 'Han et al., 2013', 'ref_id': 'BIBREF4'}, {'start': 1360, 'end': 1377, 'mention': 'Yang et al., 2007', 'ref_id': 'BIBREF63'}, {'start': 1539, 'end': 1561, 'mention': 'Jefferson et al., 2009', 'ref_id': 'BIBREF12'}, {'start': 1563, 'end': 1581, 'mention': 'Leung et al., 2020', 'ref_id': 'BIBREF17'}], 'section': '4. How do N95 masks block SARS-CoV-2? ::: Eight questions about SARS-CoV-2', 'ref_spans': []}, {'text': \"SARS-CoV-2 is a beta-coronavirus whose genome is a single ≈30 kb strand of RNA. The flu is caused by an entirely different family of RNA viruses called influenza viruses. Flu viruses have smaller genomes (≈14 kb) encoded in eight distinct strands of RNA, and they infect human cells in a different manner than coronaviruses. The 'common cold' is caused by a variety of viruses, including some coronaviruses and rhinoviruses. Cold-causing coronaviruses (e.g. OC43 and 229E strains) are quite similar to SARS-CoV-2 in genome length (within 10%) and gene content, but different from SARS-CoV-2 in sequence (≈50% nucleotide identity) and infection severity. One interesting facet of coronaviruses is that they have the largest genomes of any known RNA viruses (≈30 kb). These large genomes led researchers to suspect the presence of a 'proofreading mechanism' to reduce the mutation rate and stabilize the genome. Indeed, coronaviruses have a proofreading exonuclease called ExoN, which explains their low mutation rates (~10–6 per site per cycle) in comparison to influenza (≈3 × 10–5 per site per cycle; Sanjuán et al., 2010). This relatively low mutation rate will be of interest for future studies predicting the speed with which coronaviruses can evade our immunization efforts.\", 'cite_spans': [{'start': 1102, 'end': 1122, 'mention': 'Sanjuán et al., 2010', 'ref_id': 'BIBREF37'}], 'section': '5. How similar is SARS-CoV-2 to the common cold and flu viruses? ::: Eight questions about SARS-CoV-2', 'ref_spans': []}, {'text': \"SARS-CoV-2 has a single-stranded positive-sense RNA genome that codes for 10 genes ultimately producing 26 proteins according to an NCBI annotation (NC_045512). How is it that 10 genes code for >20 proteins? One long gene, orf1ab, encodes a polyprotein that is cleaved into 16 proteins by proteases that are themselves part of the polyprotein. In addition to proteases, the polyprotein encodes an RNA polymerase and associated factors to copy the genome, a proofreading exonuclease, and several other non-structural proteins. The remaining genes predominantly code for structural components of the virus: i) the spike protein which binds the cognate receptor on a human or animal cell; ii) a nucleoprotein that packages the genome; iii) two membrane-bound proteins. Though much current work is centered on understanding the role of 'accessory' proteins in the viral life cycle, we estimate that it is currently possible to ascribe clear biochemical or structural functions to only about half of SARS-CoV-2 gene products.\", 'cite_spans': [], 'section': '6. How much is known about the SARS-CoV-2 genome and proteome? ::: Eight questions about SARS-CoV-2', 'ref_spans': []}, {'text': 'Studying viral evolution, researchers commonly use two measures describing the rate of genomic change. The first is the evolutionary rate, which is defined as the average number of substitutions that become fixed per year in strains of the virus, given in units of mutations per site per year. The second is the mutation rate, which is the number of substitutions per site per replication cycle. How can we relate these two values? Consider a single site at the end of a year. The only measurement of a mutation rate in a β-coronavirus suggests that this site will accumulate ~10–6 mutations in each round of replication. Each replication cycle takes ~10 hr, and so there are 103 cycles/year. Multiplying the mutation rate by the number of replications, assuming neutrality and neglecting the effects of evolutionary selection, we arrive at 10–3 mutations per site per year, consistent with the evolutionary rate inferred from sequenced coronavirus genomes. As our estimate is consistent with the measured rate, we infer that the virus undergoes near-continuous replication in the wild, constantly generating new mutations that accumulate over the course of the year. Using our knowledge of the mutation rate, we can also draw inferences about single infections. For example, since the mutation rate is ~10–6 mutations/site/cycle and an mL of sputum might contain upwards of 107 viral RNAs, we infer that every site is mutated more than once in such samples.', 'cite_spans': [], 'section': '7. What can we learn from the mutation rate of the virus? ::: Eight questions about SARS-CoV-2', 'ref_spans': []}, {'text': 'To understand how SARS-CoV-2 can be transmitted, it is vitally important to characterize the stability of infectious virions on different types of surfaces like cardboard, plastics, and various metals. This is a very active area of current research. However, there are significant caveats associated with viral stability measurements. The measured stability depends on the quantity measured, for example, one can measure either infectious virions or viral RNA copies. The number of infectious virions is typically much lower than inferred from measurements of the viral genome (Woelfel et al., 2020). SARS-CoV-2 RNA has been detected on various surfaces several weeks after they were last touched (Moriarty et al., 2020), but infectiousness appears to degrade more quickly than RNA. When researchers measured the stability of infectious virions on surfaces, the numbers depended greatly on the type of surface and the medium carrying the virus, with the stability on plastic being much greater than on copper or steel, for example. Viral stability is also known to depend strongly on temperature and humidity (Chin et al., 2020). Therefore calculating the probability of human infection from exposure to contaminated surfaces is a complex task for which sufficient data is not yet available. As such, caution and protective measures should be taken. To gain some intuition for the importance of surface transmission, we consider an undiagnosed infectious person who touches surfaces tens of times during their infectious period. Prior to lockdown, these public surfaces will subsequently be touched by hundreds of other people. From the basic reproduction number R0 ≈ 2–4 we can infer that not everyone touching those surfaces will be infected. More detailed bounds on the risk of infection from touching surfaces urgently awaits study.', 'cite_spans': [{'start': 578, 'end': 598, 'mention': 'Woelfel et al., 2020', 'ref_id': 'BIBREF54'}, {'start': 698, 'end': 719, 'mention': 'Moriarty et al., 2020', 'ref_id': 'BIBREF24'}, {'start': 1110, 'end': 1127, 'mention': 'Chin et al., 2020', 'ref_id': 'BIBREF22'}], 'section': '8. How stable and infectious is the virion on surfaces? ::: Eight questions about SARS-CoV-2', 'ref_spans': []}, {'text': \"The basic reproduction number, R0, estimates the average number of new infections directly generated by a single infectious person. The 0 subscript connotes that this refers to early stages of an epidemic, when everyone in the region is susceptible (that is, there is no immunity) and no countermeasures have been taken. As geography and culture affect how many people we encounter daily, how much we touch them and share food with them, estimates of R0 can vary between locales. Moreover, because R0 is defined in the absence of countermeasures and immunity, we are usually only able to assess the effective R (Re). At the beginning of an epidemic, before any countermeasures, Re ≈ R0. Several days pass before a newly-infected person becomes infectious themselves. This 'latent period' is typically followed by several days of infectivity called the 'infectious period'.\", 'cite_spans': [], 'section': \"What are the meanings of R0, 'latent period' and 'infectious period'? ::: Definitions and measurement methods\", 'ref_spans': []}, {'text': 'It is important to understand that reported values for all these parameters are population averages inferred from epidemiological models fit to counts of infected, symptomatic, and dying patients. Because testing is always incomplete and model fitting is imperfect, and data will vary between different locations, there is substantial uncertainty associated with reported values. Moreover, these median or average best-fit values do not describe person-to-person variation. For example, viral RNA was detectable in patients with moderate symptoms for more than one week after the onset of symptoms, and more than two weeks in patients with severe symptoms (ECDC, 2020). Though detectable RNA is not the same as active virus, this evidence calls for caution in using uncertain, average parameters to describe a pandemic. Why have detailed distributions of these parameters across people not been published? Direct measurement of latent and infectious periods at the individual level is extremely challenging, as accurately identifying the precise time of infection is usually very difficult.', 'cite_spans': [{'start': 657, 'end': 667, 'mention': 'ECDC, 2020', 'ref_id': 'BIBREF66'}], 'section': \"What are the meanings of R0, 'latent period' and 'infectious period'? ::: Definitions and measurement methods\", 'ref_spans': []}, {'text': \"Diagnosis and quantification of viruses utilizes several different methodologies. One common approach is to quantify the amount of viral RNA in an environmental (e.g., surface) or clinical (e.g., sputum) sample via quantitative reverse-transcription polymerase chain reaction (RT-qPCR). This method measures the number of copies of viral RNA in a sample. The presence of viral RNA does not necessarily imply the presence of infectious virions. Virions could be defective (e.g., by mutation) or might have been deactivated by environmental conditions. To assess the concentration of infectious viruses, researchers typically measure the '50% tissue-culture infectious dose' (TCID50). Measuring TCID50 involves infecting replicate cultures of susceptible cells with dilutions of the virus and noting the dilution at which half the replicate dishes become infected. Viral counts reported by TCID50 tend to be much lower than RT-qPCR measurements, which could be one reason why studies relying on RNA measurements (Moriarty et al., 2020) report the persistence of viral RNA on surfaces for much longer times than studies relying on TCID50 (van Doremalen et al., 2020). It is important to keep this caveat in mind when interpreting data about viral loads, for example a report measuring viral RNA in patient stool samples for several days after recovery (Wu et al., 2020a). Nevertheless, for many viruses even a small dose of virions can lead to infection. For the common cold, for example, ~0.1 TCID50 are sufficient to infect half of the people exposed (Couch et al., 1966).\", 'cite_spans': [{'start': 1011, 'end': 1032, 'mention': 'Moriarty et al., 2020', 'ref_id': 'BIBREF24'}, {'start': 1136, 'end': 1162, 'mention': 'van Doremalen et al., 2020', 'ref_id': 'BIBREF49'}, {'start': 1350, 'end': 1366, 'mention': 'Wu et al., 2020a', 'ref_id': 'BIBREF59'}, {'start': 1551, 'end': 1569, 'mention': 'Couch et al., 1966', 'ref_id': 'BIBREF33'}], 'section': 'What is the difference between measurements of viral RNA and infectious viruses? ::: Definitions and measurement methods', 'ref_spans': []}, {'text': 'Global statistics on new infections and fatalities are pouring in from many countries, providing somewhat different views on the severity and progression of the pandemic. Assessing the severity of the pandemic is critical for policy making and thus much effort has been put into quantifying key measures of its progression. The most common measure for the severity of a disease is the fatality rate. One commonly reported measure is the case fatality rate (CFR), which is the proportion of fatalities out of total diagnosed cases. The CFR reported in different countries varies significantly, from 1% to about 15%. Several key factors affect the CFR. First, demographic parameters and practices associated with increased or decreased risk differ greatly across societies. For example, the prevalence of smoking, the average age of the population, and the capacity of the healthcare system. Indeed, the majority of people dying from SARS-CoV-2 have a preexisting condition such as cardiovascular disease or smoking (The Novel Coronavirus Pneumonia Emergency Response Epidemiology Team, 2020). There is also potential for bias in estimating the CFR. For example, a tendency to identify more severe cases (selection bias) will tend to overestimate the CFR. On the other hand, there is usually a delay between the onset of symptoms and death, which can lead to an underestimate of the CFR early in the progression of an epidemic. We report the uncorrected CFR values, and thus these caveats should be borne in mind. Even when correcting for these factors, the CFR does not give a complete picture as many cases with mild or no symptoms are not tested. Thus, the CFR will tend to overestimate the rate of fatalities per infected person, termed the infection fatality rate (IFR). Estimating the total number of infected people is usually accomplished by testing a random sample for anti-viral antibodies, whose presence indicates that the patient was previously infected. At the time of writing, such assays are not widely available, and so researchers resort to surrogate datasets generated by testing of foreign citizens returning home from infected countries (Verity et al., 2020; Nishiura et al., 2020), large-scale semi-random testing in countries such as Iceland, near complete testing of passengers on the Diamond Princess ship (Russell et al., 2020), or epidemiological models estimating the number of undocumented cases (Li et al., 2020a; Mizumoto et al., 2020). These methods have their own caveats and uncertainties associated with them, and it is not entirely clear how representative they are but they do provide a first glimpse of the true severity of the disease.', 'cite_spans': [{'start': 1015, 'end': 1089, 'mention': 'The Novel Coronavirus Pneumonia Emergency Response Epidemiology Team, 2020', 'ref_id': 'BIBREF45'}, {'start': 2157, 'end': 2176, 'mention': 'Verity et al., 2020', 'ref_id': 'BIBREF50'}, {'start': 2178, 'end': 2199, 'mention': 'Nishiura et al., 2020', 'ref_id': 'BIBREF27'}, {'start': 2330, 'end': 2350, 'mention': 'Russell et al., 2020', 'ref_id': 'BIBREF36'}, {'start': 2424, 'end': 2440, 'mention': 'Li et al., 2020a', 'ref_id': 'BIBREF18'}, {'start': 2442, 'end': 2463, 'mention': 'Mizumoto et al., 2020', 'ref_id': 'BIBREF23'}], 'section': 'What is the difference between the case fatality rate and the infection fatality rate? ::: Definitions and measurement methods', 'ref_spans': []}, {'text': \"Two important characteristics of the viral life cycle are the time it takes them to produce new infectious progeny, and the number of progeny each infected cell produces. The yield of new virions per infected cell is more clearly defined in lytic viruses, such as those infecting bacteria (bacteriophages), as viruses replicate within the cell and subsequently lyse the cell to release a 'burst' of progeny. This measure is usually termed 'burst size'. SARS-CoV-2 does not release its progeny by lysing the cell, but rather by continuous budding (Park et al., 2020b). Even though there is no 'burst', we can still estimate the average number of virions produced by a single infected cell. Measuring the time to complete a replication cycle or the burst size in vivo is very challenging, and thus researchers usually resort to measuring these values in tissue-culture. There are various ways to estimate these quantities, but a common and simple one is using 'one-step' growth dynamics. The key principle of this method is to ensure that only a single replication cycle occurs. This is typically achieved by infecting the cells with a large number of virions, such that every cell gets infected, thus leaving no opportunity for secondary infections.\", 'cite_spans': [{'start': 547, 'end': 565, 'mention': 'Park et al., 2020b', 'ref_id': 'BIBREF31'}], 'section': 'What is the burst size and the replication time of the virus? ::: Definitions and measurement methods', 'ref_spans': []}, {'text': \"Assuming entry of the virus to the cells is rapid (we estimate 10 min for SARS-CoV-2), the time it takes to produce progeny can be estimated by quantifying the lag between inoculation and the appearance of new intracellular virions, also known as the 'eclipse period'. This eclipse period does not account for the time it takes to release new virions from the cell. The time from cell entry until the appearance of the first extracellular viruses, known as the 'latent period' (not to be confused with the epidemiological latent period; see glossary in Box 1), estimates the duration of the full replication cycle. The burst size can be estimated by waiting until virion production saturates, and then dividing the total virion yield by the number of cells infected. While both the time to complete a replication cycle and the burst size may vary significantly in an animal host due to factors including the type of cell infected or the action of the immune system, these numbers provide us with an approximate quantitative view of the viral life-cycle at the cellular level.\", 'cite_spans': [], 'section': 'What is the burst size and the replication time of the virus? ::: Definitions and measurement methods', 'ref_spans': []}, {'text': 'Our personal experience with infectious diseases leaves us with the intuition that we are contagious when we have symptoms. For the seasonal flu, for example, most transmissions indeed occur after a person has developed symptoms (Ip et al., 2017). For SARS-CoV-2, in contrast, it is common to be contagious before symptoms. The SARS-CoV-2 incubation period is about 5 days, while peak infectiousness begins two days before symptoms reveal themselves. As a result, a large fraction of infections occur pre-symptomatically, that is, without the infectious person realizing they have the disease (Ferretti et al., 2020; He et al., 2020). With testing capacity under strain, diagnosis typically occurs ≈5 days after symptom onset, or ≈10 days after infection. By that time, most people have already passed peak infectiousness. In order to effectively slow the growth of the pandemic, it is important to detect infections as early as possible and quarantine those who test positive. In the case of SARS-CoV-2 this means detection before symptoms because there is strong evidence of significant pre-symptomatic transmission. Finally, the situation is further complicated by a large fraction of asymptomatic cases, that is cases in which the infected person never develops noticeable symptoms. This fraction is more than half of children and young adults (Davies et al., 2020). Leading modeling efforts assume that asymptomatic infections are anywhere between 10–80% as contagious as symptomatic ones (Ferretti et al., 2020; Davies et al., 2020). This wide range reflects a crucial gap in our understanding of SARS-CoV-2 transmission: great uncertainty about the magnitude of asymptomatic transmission.', 'cite_spans': [{'start': 230, 'end': 245, 'mention': 'Ip et al., 2017', 'ref_id': 'BIBREF10'}, {'start': 594, 'end': 615, 'mention': 'Ferretti et al., 2020', 'ref_id': 'BIBREF2'}, {'start': 617, 'end': 632, 'mention': 'He et al., 2020', 'ref_id': 'BIBREF6'}, {'start': 1349, 'end': 1368, 'mention': 'Davies et al., 2020', 'ref_id': 'BIBREF55'}, {'start': 1495, 'end': 1516, 'mention': 'Ferretti et al., 2020', 'ref_id': 'BIBREF2'}, {'start': 1518, 'end': 1537, 'mention': 'Davies et al., 2020', 'ref_id': 'BIBREF55'}], 'section': 'Are people usually diagnosed before or after they are contagious? ::: Definitions and measurement methods', 'ref_spans': []}, {'text': 'Diameter. (Figure 3 in Zhu et al., 2020): \"Electron micrographs of negative-stained 2019-nCoV particles were generally spherical with some pleomorphism. Diameter varied from about 60 to 140 nm.\"', 'cite_spans': [{'start': 23, 'end': 39, 'mention': 'Zhu et al., 2020', 'ref_id': 'BIBREF67'}], 'section': 'Size and content ::: Sources of the numbers in Figure 1', 'ref_spans': []}, {'text': 'Volume. Using diameter and assuming the virus is a sphere.', 'cite_spans': [], 'section': 'Size and content ::: Sources of the numbers in Figure 1', 'ref_spans': []}, {'text': 'Mass. Using the volume and a density of ~1 g per mL.', 'cite_spans': [], 'section': 'Size and content ::: Sources of the numbers in Figure 1', 'ref_spans': []}, {'text': 'Number of spike trimers. (Neuman et al., 2011): \"Our model predicts ∼90 spikes per particle.\"', 'cite_spans': [{'start': 26, 'end': 45, 'mention': 'Neuman et al., 2011', 'ref_id': 'BIBREF25'}], 'section': 'Size and content ::: Sources of the numbers in Figure 1', 'ref_spans': []}, {'text': 'Length of spike trimers. (Zhu et al., 2020): \"Virus particles had quite distinctive spikes, about 9 to 12 nm, and gave virions the appearance of a solar corona.\"', 'cite_spans': [{'start': 26, 'end': 42, 'mention': 'Zhu et al., 2020', 'ref_id': 'BIBREF67'}], 'section': 'Size and content ::: Sources of the numbers in Figure 1', 'ref_spans': []}, {'text': 'Receptor binding affinity (Kd). Walls et al., 2020 reports Kd of ≈1 nM for the binding domain using biolayer interferometry with kon of ≈1.5 × 105 M–1 s–1 and koff of ≈1.6 × 10–4 s–1 (Table 1). Wrapp et al., 2020 reports Kd of ≈15 nM for the spike (Figure 3) and ≈35 nM for the binding domain (Figure 4) using surface plasmon resonance with kon of ≈1.9 × 105 M–1 s–1 and koff of ≈2.8 × 10–3 s–1 for the spike, and kon of ≈1.4 × 105 M–1 s–1 and koff of ≈4.7 × 10–3 s–1 for the binding domain. Lan et al., 2020 reports Kd of ≈5 nM for the binding domain (Extended Data Figure 4) using surface plasmon resonance with kon of ≈1.4 × 106 M–1 s–1 and koff of ≈6.5 × 10–3 s–1. Shang et al., 2020 reports Kd of ≈40 nM for the binding domain (Extended Data Figure 6) using surface plasmon resonance with kon of ≈1.8 × 106 M–1 s–1 and koff of ≈7.8 × 10–3 s–1. The main disagreement between the studies seems to be on the koff.', 'cite_spans': [{'start': 32, 'end': 50, 'mention': 'Walls et al., 2020', 'ref_id': 'BIBREF51'}, {'start': 194, 'end': 212, 'mention': 'Wrapp et al., 2020', 'ref_id': 'BIBREF57'}, {'start': 492, 'end': 508, 'mention': 'Lan et al., 2020', 'ref_id': 'BIBREF15'}, {'start': 669, 'end': 687, 'mention': 'Shang et al., 2020', 'ref_id': 'BIBREF39'}], 'section': 'Size and content ::: Sources of the numbers in Figure 1', 'ref_spans': []}, {'text': 'Membrane (M; 222 aa). (Neuman et al., 2011): \"Using the M spacing data for each virus (Figure 6C), this would give ∼1100 M2 molecules per average SARS-CoV, MHV and FCoV particle.\"', 'cite_spans': [{'start': 23, 'end': 42, 'mention': 'Neuman et al., 2011', 'ref_id': 'BIBREF25'}], 'section': 'Size and content ::: Sources of the numbers in Figure 1', 'ref_spans': []}, {'text': 'Envelope (E; 75 aa). (Godet et al., 1992): \"Based on the estimated molar ratio and assuming that coronavirions bear 100 (J Gen Virol\\n63: 241–245) to 200 spikes, each composed of 3 s molecules (Virus Research\\n20:107–120) it can be inferred that approximately 15–30 copies of ORF4 protein are incorporated into TGEV virions (Purdue strain).\"', 'cite_spans': [{'start': 22, 'end': 40, 'mention': 'Godet et al., 1992', 'ref_id': 'BIBREF3'}], 'section': 'Size and content ::: Sources of the numbers in Figure 1', 'ref_spans': []}, {'text': 'Nucleoprotein (364 aa). (Neuman et al., 2011): \"Estimated ratios of M to N protein in purified coronaviruses range from about 3M:1N (Cavanagh, 1983; Escors et al., 2001) to 1M:1N (Hogue and Brian, 1986; Liu and Inglis, 1991), giving 730–2200 N molecules per virion.\"', 'cite_spans': [{'start': 25, 'end': 44, 'mention': 'Neuman et al., 2011', 'ref_id': 'BIBREF25'}, {'start': 133, 'end': 147, 'mention': 'Cavanagh, 1983', 'ref_id': 'BIBREF0'}, {'start': 149, 'end': 168, 'mention': 'Escors et al., 2001', 'ref_id': 'BIBREF68'}, {'start': 180, 'end': 201, 'mention': 'Hogue and Brian, 1986', 'ref_id': 'BIBREF8'}, {'start': 203, 'end': 223, 'mention': 'Liu and Inglis, 1991', 'ref_id': 'BIBREF20'}], 'section': 'Size and content ::: Sources of the numbers in Figure 1', 'ref_spans': []}, {'text': 'Type. (ViralZone) +ssRNA \"Monopartite, linear ssRNA(+) genome\"', 'cite_spans': [], 'section': 'Genome ::: Sources of the numbers in Figure 1', 'ref_spans': []}, {'text': 'Genome length. The initial isolate of SARS-CoV-2 from Wuhan, China has a 29903 nt ≈ 30 kb ssRNA genome (NCBI MN908947.3), which is typical of a coronavirus (Smith and Denison, 2012).', 'cite_spans': [{'start': 157, 'end': 180, 'mention': 'Smith and Denison, 2012', 'ref_id': 'BIBREF41'}], 'section': 'Genome ::: Sources of the numbers in Figure 1', 'ref_spans': []}, {'text': '(Wu et al., 2020b): \"SARS-CoV-2 genome has 10 open reading frames (Figure 2A)\". (Wu et al., 2020c): \"The 2019-nCoV genome was annotated to possess 14 ORFs encoding 27 proteins\". Coronavirus genomes contain several \\'accessory proteins\\' that are not essential for replication and are not always expressed. The \\'nonstructural proteins\\' are expressed as a polyprotein which is proteolytically cleaved into ≈10 proteins. As transcription start and protease cleavage sites are not trivial to identify bioinformatically, there is some uncertainty about the exact number of transcriptional units and proteins expressed by SARS-CoV-2.', 'cite_spans': [{'start': 1, 'end': 17, 'mention': 'Wu et al., 2020b', 'ref_id': 'BIBREF60'}, {'start': 81, 'end': 97, 'mention': 'Wu et al., 2020c', 'ref_id': 'BIBREF61'}], 'section': 'Genome ::: Sources of the numbers in Figure 1', 'ref_spans': []}, {'text': 'Number of proteins. (Wu et al., 2020b): \"By aligning with the amino acid sequence of SARS PP1ab and analyzing the characteristics of restriction cleavage sites recognized by 3CLpro and PLpro, we speculated 14 specific proteolytic sites of 3CLpro and PLpro in SARS-CoV-2 PP1ab (Figure 2B). PLpro cleaves three sites at 181–182, 818–819, and 2763–2764 at the N-terminus and 3CLpro cuts at the other 11 sites at the C-terminus, and forming 15 non-structural proteins.\"', 'cite_spans': [{'start': 21, 'end': 37, 'mention': 'Wu et al., 2020b', 'ref_id': 'BIBREF60'}], 'section': 'Genome ::: Sources of the numbers in Figure 1', 'ref_spans': []}, {'text': 'Evolution rate. (Koyama et al., 2020): \"Mutation rates estimated for SARS, MERS, and OC43 show a large range, covering a span of 0.27 to 2.38 substitutions × 10–3/site/ year (see references 10–16).\" Recent unpublished evidence also suggests this rate is of the same order of magnitude in SARS-CoV-2 ().', 'cite_spans': [{'start': 17, 'end': 36, 'mention': 'Koyama et al., 2020', 'ref_id': 'BIBREF14'}], 'section': 'Genome ::: Sources of the numbers in Figure 1', 'ref_spans': []}, {'text': 'Mutation rate. (Sanjuán et al., 2010): \"Murine hepatitis virus … Therefore, the corrected estimate of the mutation rate is μs/n/c = 1.9x10–6 / 0.55 = 3.5 x 10–6.\"', 'cite_spans': [{'start': 16, 'end': 36, 'mention': 'Sanjuán et al., 2010', 'ref_id': 'BIBREF37'}], 'section': 'Genome ::: Sources of the numbers in Figure 1', 'ref_spans': []}, {'text': 'Genome similarity. For all species except pangolin, genomes were downloaded from NCBI and aligned to the SARS-CoV-2 reference (MN908947) with EMBOSS Stretcher (EMBL-EBI server). Reported values are percent nucleotide sequence identity. Genomes used: bat coronavirus RaTG13 (MN996532.1; 96% id); SARS-CoV-1 (NC_004718.3; 80% id); MERS (NC_019843.3; 55% id); human cold coronavirus strains OC43 (NC_006213.1; 53% id) and 229E (NC_002645.1; 50% id). For pangolin: ‘\"PangolinCoV is 91.02% and 90.55% identical to SARS-CoV-2 and BatCoV RaTG13, respectively, at the whole genome level\" (Zhang et al., 2020).', 'cite_spans': [{'start': 581, 'end': 599, 'mention': 'Zhang et al., 2020', 'ref_id': 'BIBREF64'}], 'section': 'Genome ::: Sources of the numbers in Figure 1', 'ref_spans': []}, {'text': 'Virion entry into cell (for SARS-CoV-1). (Schneider et al., 2012): \"Previous experiments had revealed that virus is internalized within 15 min\". (Ng et al., 2003): \"Within the first 10 min, some virus particles were internalized into vacuoles (arrow) that were just below the plasma membrane surface (Fig. 2, arrows). […] The observation at 15 min postinfection (p.i.), did not differ much from 10 min p.i. (Fig. 4a)\".', 'cite_spans': [{'start': 42, 'end': 64, 'mention': 'Schneider et al., 2012', 'ref_id': 'BIBREF38'}, {'start': 146, 'end': 161, 'mention': 'Ng et al., 2003', 'ref_id': 'BIBREF26'}], 'section': 'Replication timescales ::: Sources of the numbers in Figure 1', 'ref_spans': []}, {'text': 'Eclipse period. (Schneider et al., 2012): \"SARS-CoV replication cycle from adsorption to release of infectious progeny takes about 7 to 8 hr (data not shown)\"; Figure 4 of Harcourt et al., 2020 shows virions are released after 12–36 hr but because this is multi-step growth this represents an upper bound for the replication cycle.', 'cite_spans': [{'start': 17, 'end': 39, 'mention': 'Schneider et al., 2012', 'ref_id': 'BIBREF38'}, {'start': 172, 'end': 193, 'mention': 'Harcourt et al., 2020', 'ref_id': 'BIBREF5'}], 'section': 'Replication timescales ::: Sources of the numbers in Figure 1', 'ref_spans': []}, {'text': 'Burst size. (Hirano et al., 1976): \"The average per-cell yield of active virus was estimated to be about 6–7 × 102 plaque-forming units.\" This data is for MHV, so more research is needed to verify these values for SARS-CoV-2.', 'cite_spans': [{'start': 13, 'end': 32, 'mention': 'Hirano et al., 1976', 'ref_id': 'BIBREF7'}], 'section': 'Replication timescales ::: Sources of the numbers in Figure 1', 'ref_spans': []}, {'text': 'Type. (Shieh et al., 2005): \"Immunohistochemical and in situ\\nhybridization assays demonstrated evidence of SARS-associated coronavirus (SARS-CoV) infection in various respiratory epithelial cells, predominantly type II pneumocytes, and in alveolar macrophages in the lung\". (Walls et al., 2020): \"SARS-CoV-2 uses ACE2 to enter target cells\". (Rockx et al., 2020): \"In SARS-CoV-2-infected macaques, virus was excreted from nose and throat in absence of clinical signs, and detected in type I and II pneumocytes in foci of diffuse alveolar damage and mucous glands of the nasal cavity […] In the upper respiratory tract, there was focal five or locally extensive SARS-CoV-2 antigen expression in epithelial cells of mucous glands in the nasal cavity (septum or concha) of all four macaques, without any associated histological lesions (fig. 2I).\"', 'cite_spans': [{'start': 7, 'end': 25, 'mention': 'Shieh et al., 2005', 'ref_id': 'BIBREF40'}, {'start': 275, 'end': 293, 'mention': 'Walls et al., 2020', 'ref_id': 'BIBREF51'}, {'start': 343, 'end': 361, 'mention': 'Rockx et al., 2020', 'ref_id': 'BIBREF35'}], 'section': 'Host cells ::: Sources of the numbers in Figure 1', 'ref_spans': []}, {'text': 'Type I and Type II pneumocyte and alveolar macrophage cell number. Values taken from table 4 in Crapo et al., 1982, and table 5 in Stone et al., 1992.', 'cite_spans': [{'start': 96, 'end': 114, 'mention': 'Crapo et al., 1982', 'ref_id': 'BIBREF44'}, {'start': 131, 'end': 149, 'mention': 'Stone et al., 1992', 'ref_id': 'BIBREF42'}], 'section': 'Host cells ::: Sources of the numbers in Figure 1', 'ref_spans': []}, {'text': 'Epithelial cells in mucous gland cell number and volume. The value for the surface area of the nasal cavity is taken from ICRP, 1975; the value for the mucous gland density is taken from Tos and Mogensen, 1976; Tos and Morgensen, 1977; the value for the mucous gland volume is taken from Widdicombe, 2019; and the value for the mucous cell volume is taken from Ordoñez et al., 2001 and Mercer et al., 1994. We divide the mucous gland volume by the mucous cell volume to arrive at the total number of mucous cells in a mucous gland. We multiply the surface density of mucous glands by the surface area of the nasal cavity to arrive at the total number of mucous glands, and then multiply the total number of mucous glands by the number of mucous cells per mucous gland.', 'cite_spans': [{'start': 122, 'end': 132, 'mention': 'ICRP, 1975', 'ref_id': 'BIBREF9'}, {'start': 187, 'end': 209, 'mention': 'Tos and Mogensen, 1976', 'ref_id': 'BIBREF47'}, {'start': 211, 'end': 234, 'mention': 'Tos and Morgensen, 1977', 'ref_id': 'BIBREF48'}, {'start': 288, 'end': 304, 'mention': 'Widdicombe, 2019', 'ref_id': 'BIBREF53'}, {'start': 361, 'end': 381, 'mention': 'Ordoñez et al., 2001', 'ref_id': 'BIBREF28'}, {'start': 386, 'end': 405, 'mention': 'Mercer et al., 1994', 'ref_id': 'BIBREF21'}], 'section': 'Host cells ::: Sources of the numbers in Figure 1', 'ref_spans': []}, {'text': 'Type II pneumocyte volume. (Fehrenbach et al., 1995): \"Morphometry revealed that although inter-individual variation due to some oedematous swelling was present, the cells were in a normal size range as indicated by an estimated mean volume of 763 ± 64 μm3.\"', 'cite_spans': [{'start': 28, 'end': 51, 'mention': 'Fehrenbach et al., 1995', 'ref_id': 'BIBREF69'}], 'section': 'Host cells ::: Sources of the numbers in Figure 1', 'ref_spans': []}, {'text': 'Alveolar macrophage volume. (Crapo et al., 1982): \"Alveolar macrophages were found to be the largest cell in the populations studied, having a mean volume of 2,491 μm3.\"', 'cite_spans': [{'start': 29, 'end': 47, 'mention': 'Crapo et al., 1982', 'ref_id': 'BIBREF44'}], 'section': 'Host cells ::: Sources of the numbers in Figure 1', 'ref_spans': []}, {'text': 'Nasopharynx, throat, stool, and sputum. We took the maximal viral load for each patient in nasopharyngeal swabs, throat swabs, stool or in sputum (figure 2 in Wölfel et al., 2020; figure 1 in Kim et al., 2020; Pan et al., 2020).', 'cite_spans': [{'start': 159, 'end': 178, 'mention': 'Wölfel et al., 2020', 'ref_id': 'BIBREF56'}, {'start': 192, 'end': 208, 'mention': 'Kim et al., 2020', 'ref_id': 'BIBREF13'}, {'start': 210, 'end': 226, 'mention': 'Pan et al., 2020', 'ref_id': 'BIBREF29'}], 'section': 'Concentration ::: Sources of the numbers in Figure 1', 'ref_spans': []}, {'text': 'Seroconversion time (time period until a specific antibody becomes detectable in the blood). (Zhao et al., 2020): \"The seroconversion sequentially appeared for Ab, IgM and then IgG, with a median time of 11, 12 and 14 days, respectively\". (To et al., 2020): \"For 16 patients with serum samples available 14 days or longer after symptom onset, rates of seropositivity were 94% for anti-NP IgG (n = 15), 88% for anti-NP IgM (n = 14), 100% for anti-RBD IgG (n = 16), and 94% for anti-RBD IgM (n = 15)\".', 'cite_spans': [{'start': 94, 'end': 111, 'mention': 'Zhao et al., 2020', 'ref_id': 'BIBREF65'}, {'start': 240, 'end': 255, 'mention': 'To et al., 2020', 'ref_id': 'BIBREF46'}], 'section': 'Antibody response – seroconversion ::: Sources of the numbers in Figure 1', 'ref_spans': []}, {'text': 'Maintenance of antibody response to virus. (Wu et al., 2007): \"Among 176 patients who had had severe acute respiratory syndrome (SARS), SARS-specific antibodies were maintained for an average of 2 years, and significant reduction of immunoglobulin G–positive percentage and titers occurred in the third year\".', 'cite_spans': [{'start': 44, 'end': 59, 'mention': 'Wu et al., 2007', 'ref_id': 'BIBREF58'}], 'section': 'Antibody response – seroconversion ::: Sources of the numbers in Figure 1', 'ref_spans': []}, {'text': 'Half-life on surfaces. (van Doremalen et al., 2020): We use half-live values reported in Supplementary Table 1. Chin et al., 2020: We use short-term half-lives reported in the Appendix. Pastorino et al., 2020: We use the slopes of data poitns from the first two hours can calculate the short-term half-life from them. More studies are urgently needed to clarify the implications of virion stability on the probability of infection from aerosols or surfaces.', 'cite_spans': [{'start': 24, 'end': 50, 'mention': 'van Doremalen et al., 2020', 'ref_id': 'BIBREF49'}, {'start': 112, 'end': 129, 'mention': 'Chin et al., 2020', 'ref_id': 'BIBREF22'}, {'start': 186, 'end': 208, 'mention': 'Pastorino et al., 2020', 'ref_id': 'BIBREF32'}], 'section': 'Virus environmental stability ::: Sources of the numbers in Figure 1', 'ref_spans': []}, {'text': 'RNA stability on surfaces (Moriarty et al., 2020): \"SARS-CoV-2 RNA was identified on a variety of surfaces in cabins of both symptomatic and asymptomatic infected passengers up to 17 days after cabins were vacated on the Diamond Princess but before disinfection procedures had been conducted (Takuya Yamagishi, National Institute of Infectious Diseases, personal communication, 2020).”', 'cite_spans': [{'start': 27, 'end': 48, 'mention': 'Moriarty et al., 2020', 'ref_id': 'BIBREF24'}], 'section': 'Virus environmental stability ::: Sources of the numbers in Figure 1', 'ref_spans': []}, {'text': 'Basic reproductive number, R0. (Li et al., 2020a): \"Our median estimate of the effective reproductive number, Re – equivalent to the basic reproductive number (R0) at the beginning of the epidemic – is 2.38 (95% CI: 2.04–2.77)\". (Park et al., 2020a): \"Our estimated R0 from the pooled distribution has a median of 2.9 (95% CI: 2.1–4.5)\".', 'cite_spans': [{'start': 32, 'end': 48, 'mention': 'Li et al., 2020a', 'ref_id': 'BIBREF18'}, {'start': 230, 'end': 248, 'mention': 'Park et al., 2020a', 'ref_id': 'BIBREF30'}], 'section': \"'Characteristic' infection progression in a single patient ::: Sources of the numbers in Figure 1\", 'ref_spans': []}, {'text': 'Latent period (from infection to being able to transmit). (Li et al., 2020a): \"In addition, the median estimates for the latent and infectious periods are approximately 3.69 and 3.48 days, respectively\"; see also table 1 in this paper. (He et al., 2020): We use the time it takes infectiousness to reach half its peak, which happens two days before symptom onset based on Figure 1C. As symptoms arise after five days (see \\'Incubation period\\' below), this implies a three-day latent period.', 'cite_spans': [{'start': 59, 'end': 75, 'mention': 'Li et al., 2020a', 'ref_id': 'BIBREF18'}, {'start': 237, 'end': 252, 'mention': 'He et al., 2020', 'ref_id': 'BIBREF6'}], 'section': \"'Characteristic' infection progression in a single patient ::: Sources of the numbers in Figure 1\", 'ref_spans': [{'start': 372, 'end': 381, 'mention': 'Figure 1C', 'ref_id': 'FIGREF0'}]}, {'text': 'Incubation period (from infection to symptoms). (Lauer et al., 2020): \"The median incubation period was estimated to be 5.1 days (95% CI, 4.5 to 5.8 days), and 97.5% of those who develop symptoms will do so within 11.5 days (CI, 8.2 to 15.6 days) of infection. These estimates imply that, under conservative assumptions, 101 out of every 10 000 cases (99th percentile, 482) will develop symptoms after 14 days of active monitoring or quarantine\". (Li et al., 2020b): \"The mean incubation period was 5.2 days (95% confidence interval [CI], 4.1 to 7.0), with the 95th percentile of the distribution at 12.5 days\".', 'cite_spans': [{'start': 49, 'end': 67, 'mention': 'Lauer et al., 2020', 'ref_id': 'BIBREF16'}, {'start': 448, 'end': 464, 'mention': 'Li et al., 2020b', 'ref_id': 'BIBREF19'}], 'section': \"'Characteristic' infection progression in a single patient ::: Sources of the numbers in Figure 1\", 'ref_spans': []}, {'text': 'Infectious period. (Li et al., 2020a): \"the median estimates for the latent and infectious periods are approximately 3.69 and 3.48 days, respectively\"; see also table 1 in this paper. (He et al., 2020): We quantify the interval over which infectiousness is at least half its maximal value (the interval of half-maximal infectiousness) from the infectiousness profile in Figure 1C.', 'cite_spans': [{'start': 20, 'end': 36, 'mention': 'Li et al., 2020a', 'ref_id': 'BIBREF18'}, {'start': 185, 'end': 200, 'mention': 'He et al., 2020', 'ref_id': 'BIBREF6'}], 'section': \"'Characteristic' infection progression in a single patient ::: Sources of the numbers in Figure 1\", 'ref_spans': [{'start': 370, 'end': 379, 'mention': 'Figure 1C', 'ref_id': 'FIGREF0'}]}, {'text': 'Disease duration. (WHO, 2020): \"Using available preliminary data, the median time from onset to clinical recovery for mild cases is approximately 2 weeks and is 3–6 weeks for patients with severe or critical disease\".', 'cite_spans': [{'start': 19, 'end': 28, 'mention': 'WHO, 2020', 'ref_id': 'BIBREF52'}], 'section': \"'Characteristic' infection progression in a single patient ::: Sources of the numbers in Figure 1\", 'ref_spans': []}, {'text': 'Time until diagnosis. (Xu et al., 2020): We used data on cases with known symptom onset and case confirmation dates and calculated the median time delay between these two dates.', 'cite_spans': [{'start': 23, 'end': 38, 'mention': 'Xu et al., 2020', 'ref_id': 'BIBREF62'}], 'section': \"'Characteristic' infection progression in a single patient ::: Sources of the numbers in Figure 1\", 'ref_spans': []}, {'text': 'Case fatality rate. (ECDC, 2020) - We use data from all countries with more than 50 death cases and calculate the uncorrected raw Case Fatality Rate for each country. The range represents the lowest and highest rates observed using ECDC data up to 14 April 2020.', 'cite_spans': [{'start': 21, 'end': 31, 'mention': 'ECDC, 2020', 'ref_id': 'BIBREF66'}], 'section': \"'Characteristic' infection progression in a single patient ::: Sources of the numbers in Figure 1\", 'ref_spans': []}, {'text': 'Infection fatality rate. We rely on three independent approaches that estimate the IFR. The first relies on data about people who were extensively tested as a result of being repatriated. (Verity et al., 2020): \"We obtain an overall IFR estimate for China of 0.66% (0.39%,1.33%)”. (Ferguson et al., 2020): \"The IFR estimates from Verity et al. have been adjusted to account for a non-uniform attack rate giving an overall IFR of 0.9% (95% credible interval 0.4–1.4%)\". (Nishiura et al., 2020): \"The infection fatality risk (IFR) – the actual risk of death among all infected individuals – is therefore 0.3% to 0.6%\".', 'cite_spans': [{'start': 189, 'end': 208, 'mention': 'Verity et al., 2020', 'ref_id': 'BIBREF50'}, {'start': 282, 'end': 303, 'mention': 'Ferguson et al., 2020', 'ref_id': 'BIBREF1'}, {'start': 470, 'end': 491, 'mention': 'Nishiura et al., 2020', 'ref_id': 'BIBREF27'}], 'section': \"'Characteristic' infection progression in a single patient ::: Sources of the numbers in Figure 1\", 'ref_spans': []}, {'text': 'The second approach relies on data gathered from the Diamond Princess ship, where all passengers were tested. (Russell et al., 2020): \"We estimated that the all-age cIFR on the Diamond Princess was 1.3% (95% confidence interval (CI): 0.38–3.6)\".', 'cite_spans': [{'start': 111, 'end': 131, 'mention': 'Russell et al., 2020', 'ref_id': 'BIBREF36'}], 'section': \"'Characteristic' infection progression in a single patient ::: Sources of the numbers in Figure 1\", 'ref_spans': []}, {'text': 'The third approach relies on epidemiological modeling of case time-series from China. (Mizumoto et al., 2020): \"We also found that most recent crude infection fatality ratio (IFR) and time-delay adjusted IFR is estimated to be 0.04% (95% CrI: 0.03–0.06%) and 0.12% (95%CrI: 0.08–0.17%)\". Combining these three methods, and taking into account the reliability of each report, we estimate a crude range of ≈0.3–1.3% for the IFR.', 'cite_spans': [{'start': 87, 'end': 108, 'mention': 'Mizumoto et al., 2020', 'ref_id': 'BIBREF23'}], 'section': \"'Characteristic' infection progression in a single patient ::: Sources of the numbers in Figure 1\", 'ref_spans': []}], 'paragraphs': [{'text': 'The period of time from infection to symptoms is termed the incubation period. The median SARS-CoV-2 incubation period is estimated to be roughly 5 days (Lauer et al., 2020). Yet there is much person-to-person variation. Approximately 99% of those showing symptoms will show them before day 14, which explains the two week confinement period. Importantly, this analysis neglects infected people who never show symptoms. Since asymptomatic people are not usually tested, it is still not clear how many such cases there are or how long asymptomatic people remain infectious for.', 'id': '00004'}], 'title': 'SARS-CoV-2 (COVID-19) by the numbers', 'sha': 'fc00e01e8d44f9455616abe802e2c79b998f2999', 'doi': '10.7554/elife.57309', 'score': 9.76830005645752}, {'rank': 19, 'abstract': 'Abstract Clusters of patients with novel coronavirus disease 2019 (COVID-19) have been successively reported globally. Studies show clear person-to-person transmission. The average incubation period is 2–14 days, and mostly 3–7 days. However, in some patients, this period may be longer. Here, we report a familial cluster of COVID-19 where a 47-year-old woman with long-term use of glucocorticoids did not develop any symptoms within the 14-day quarantine period but was confirmed with COVID-19 by tested positive of antibody on day 40 after she left Wuhan. Almost at the same time, her father and sister were diagnosed with COVID-19. The results suggest that the long-term use of glucocorticoids might cause atypical infections, a long incubation period, and extra transmission of COVID-19.', 'body_text': [{'text': 'In December 2019, a novel pneumonia resulting from SARS-CoV2, named coronavirus disease 2019 (COVID-19), emerged in Wuhan [1,2]. Due to the high viral contagiousness and transmission during the pre-symptomatic phase [3,4], COVID-19 progressively spread throughout China and has since become a global health issue [5]. In China, Wuhan city, the capital of Hubei province, became the center of a COVID-19 outbreak [6]. In an effort to prevent a COVID-19 epidemic, Wuhan and its surrounding cities in Hubei province were locked down on January 23rd. However, a tremendous number of patients who were still in their incubation period traveled across China before the lockdown date due to Chinese New Year, resulting in the widespread transmission of COVID-19. Hunan province is adjacent to Hubei province. By March 20th, 2020, a total of 1018 confirmed cases had been reported in Hunan, most of which showed familial clustering.', 'cite_spans': [{'start': 123, 'end': 124, 'mention': '1', 'ref_id': 'BIBREF0'}, {'start': 125, 'end': 126, 'mention': '2', 'ref_id': 'BIBREF1'}, {'start': 217, 'end': 218, 'mention': '3', 'ref_id': 'BIBREF2'}, {'start': 219, 'end': 220, 'mention': '4', 'ref_id': 'BIBREF3'}, {'start': 314, 'end': 315, 'mention': '5', 'ref_id': 'BIBREF4'}, {'start': 413, 'end': 414, 'mention': '6', 'ref_id': 'BIBREF5'}], 'section': 'Introduction', 'ref_spans': []}, {'text': 'The average incubation period for COVID-19 is reported to be 2 to 14 days, and most patients show symptoms within 11.5 days [7,8]. To stop the spread of the disease, public health officials in Hunan placed those who may have been from Wuhan and exposed to COVID-19 under quarantine for 14 days. However, the incubation period may be longer in some patients. Here, we report on the last COVID-19 familial cluster in Hunan province, which was hospitalized on February 28. The disease was not confirmed until 40 days after the index patient had returned from Wuhan.', 'cite_spans': [{'start': 125, 'end': 126, 'mention': '7', 'ref_id': 'BIBREF6'}, {'start': 127, 'end': 128, 'mention': '8', 'ref_id': 'BIBREF7'}], 'section': 'Introduction', 'ref_spans': []}, {'text': \"On February 28th, a new COVID-19 case was confirmed in the Fourth People's Hospital of Yiyang. In following the patient's epidemiological history, it emerged that this was a familial cluster. Afterward, we further collected the clinical data of the patient's family members, including epidemiological history, laboratory examinations, lung CTs, pathogen results, treatment course, and recovery. All the clinical data were collected from the Fourth People's Hospital of Yiyang, and all the epidemiological histories of the patients were gathered from the patients themselves; these data were further confirmed by the local Centers for Disease Control.\", 'cite_spans': [], 'section': 'Case source ::: Patients and methods', 'ref_spans': []}, {'text': 'All the information used in this article was approved by the patients.', 'cite_spans': [], 'section': 'Case source ::: Patients and methods', 'ref_spans': []}, {'text': \"The home quarantine involved in our article was completely implemented according to the announcement released by the National Health Commission of the People's Republic of China (http://www.gov.cn/zhengce/zhengceku/2020-02/05/content_5474688.htm.accessed 26 March 2020).\", 'cite_spans': [], 'section': 'Quarantine standards taken in this study ::: Patients and methods', 'ref_spans': []}, {'text': \"Influenza A and B viruses and respiratory syncytial virus were tested by the Xpert Xpress Flu/RSV assay (GeneXpert System, Cepheid, Sunnyvale, CA, USA) according to the manufacturer's instructions [9]. The IgG/IgM of Chlamydophila pneumoniae and Mycoplasma pneumoniae were also tested using the SeroMP kit (Savyon Diagnostics, Ashdod, Israel). The SARS-COV2 nucleic acid test was conducted via real-time RT-PCR according to the protocol of the nucleic acid kit (Kangwei Century Biotechnology Company, China). The SARS-CoV2 antibody kit was used to test for specific IgM and IgG antibodies (Guangzhou Wonfo Biological Technology Co, Ltd., China) via colloidal gold immunochromatography.\", 'cite_spans': [{'start': 198, 'end': 199, 'mention': '9', 'ref_id': 'BIBREF8'}], 'section': 'Detection of respiratory pathogens ::: Patients and methods', 'ref_spans': []}, {'text': \"Case 1 is a Wuhan-settled 47-year-old female. She has a more than 16-year history of systemic lupus erythematosus (SLE) and has been taking oral prednisone (7.5 mg/d) since her diagnosis. She had not taken hydroxychloroquine or other immunosuppressive drugs. On January 10th, 2020, she spent 1 h dancing and singing in a room and one of her partners was later diagnosed with COVID-19. In the following 10 days, she had no contact with any confirmed or suspected patients. She returned to her parents' home in Yiyang on January 20th and after that never went out. Because of the severity of COVID-19, Hunan province carried out a ‘level one response to major public health emergencies’ on January 23. Therefore, Case 1 was placed under home quarantine and observation for 14 days. During the isolation period, she felt good and had no fever, cough or shortness of breath. Therefore, she was released from quarantine on February 6th. Just a day later, however, Case 1 began to have a mild cough, nasal congestion, and runny nose, but no fever. She paid little attention to her symptoms and did not seek further medical help. These symptoms improved 7 days later. On February 28th, she was admitted to hospital for screening because of her father's (Case 2) diagnosis of COVID-19. Her routine blood examination was abnormal, with a white blood cell (WBC) count of 2.98 ∗ 10^9/L, hemoglobin concentration of 106 g/L, and platelet count of 73 ∗ 10^9/L. Her erythrocyte sedimentation rate (ESR) was 33 mm/h, which was above normal. IL-6 was normal. The SARS-CoV2 nuclei acid test from nasopharyngeal swabs was negative, but her IgM and IgG antibodies were positive. Mycoplasma pneumoniae and Chlamydia pneumoniae antibodies were negative. Lung CT (Fig. 2A, B) showed multiple lesions in both lungs. These results supported the diagnosis of COVID-19. Hence, she was given antiviral treatment, including 0.2 g BID of Abidol orally and 5 million IU of interferon nebulization. Nasopharyngeal swab virus nucleic acid tests were performed again on February 29th and March 3rd, both of which were negative. On March 5th, her lung CT (Fig. 2C, D) worsened, and ground-glass opacity changes were found in the right upper lung. She was given extra piperacillin sodium tazobactam sodium (4.5 TID), and then glycyrrhizin (150 mg QD). Her CT showed improvements and she was discharged on March 9th.\", 'cite_spans': [], 'section': 'Case 1 ::: Results', 'ref_spans': [{'start': 1742, 'end': 1748, 'mention': 'Fig. 2', 'ref_id': 'FIGREF1'}, {'start': 2122, 'end': 2128, 'mention': 'Fig. 2', 'ref_id': 'FIGREF1'}]}, {'text': 'Case 2 is an 81-year-old male who has been living in Yiyang for many years. He has a history of prostate cancer and coronary heart disease. He had been living with his eldest daughter (Case 1) since she returned from Wuhan and had not gone out since then. He developed a paroxysmal dry cough and anorexia on February 21st. In the beginning, he did not get treatment. On February 25th, the symptoms aggravated with a fever (38.0 °C) and shortness of breath. Influenza A and B viruses were negative. His chest CT (Fig. 2E) was abnormal with bilateral ground-glass opacities. His symptoms did not improve even after treatment with piperacillin-tazobactam (4.5 g Q48h), ambroxol, ribavirin (500 mg Q12h), and oseltamivir (75 mg BID). The SARS-CoV2 nucleic acid test was positive by both nasopharyngeal swabs and sputum on February 27th. His hemoglobin (83 g/L) and lymphocytes (0.65 ∗ 10^9/L) were below normal, and his D-dimer (5.46 mg/mL), C-reactive protein (134 mg/L), ESR (126 mm/h), IL-6 (7.6 pg/mL) were above normal. His arterial blood gas showed a PO2 of 67 mmHg and a FiO2 of 33%. Therefore, he was diagnosed with severe COVID-19 and admitted to the intensive care unit with support treatments including high fluids, oxygen, interferon, lopinavir-ritonavir and ribavirin for anti-viral therapy, linezolid, meropenem and fluconazole for anti-inflammation, and albumin, gamma globulin, steroids (80 mg/d) and recovery plasma (200 mL∗ 2) to improve his immune system. His virus nucleic acid tests were retested on March 2nd and March 4th, with both showing negative. On March 3rd, his IgM and IgG specific antibodies were positive. Repeat blood gas on March 5th showed a PO2 of 114 mmHg and a FiO2 of 33%. His lung CT (Fig. 2F) showed that his inflammation lesions were absorbed and ground-glass opacities were improved.', 'cite_spans': [], 'section': 'Case 2 ::: Results', 'ref_spans': [{'start': 512, 'end': 518, 'mention': 'Fig. 2', 'ref_id': 'FIGREF1'}, {'start': 1722, 'end': 1728, 'mention': 'Fig. 2', 'ref_id': 'FIGREF1'}]}, {'text': 'Case 3 is a 44-year-old female from Yiyang. She also had close contact with her elder sister (Case 1). She developed a fever (37.8 °C) and inappetence on February 13th. Three days later, she was taken to a hospital. Laboratory examination showed a WBC of 2.88 ∗ 10^9/L, C-reactive protein of 17.06 mg/L, and fibrinogen of 4.27 g/L. Influenza A and B were negative. Lung CT (Fig. 2G) showed multiple patchy high-density shadows on both lungs. On February 17th and February 20th, the nasopharyngeal swab was negative for SARS-CoV2. After treatment, the symptoms disappeared. Hence, she was discharged on February 21st and placed under home quarantine. Since her father (Case 2) was diagnosed with COVID-19 on February 28th, she was admitted to the hospital again for further screening. This time, her SARS-CoV2 nucleic acids and specific IgG and IgM antibodies were positive. The ESR was 55.6 mm/h. Antiviral therapy was given with interferon and lopinavir-ritonavir. On March 2nd, her repeat viral nucleic acid test was still positive. The nasopharyngeal swab was negative twice, respectively on March 4th and March 6th, with simultaneously improved lung CT (Fig. 2H).', 'cite_spans': [], 'section': 'Case 3 ::: Results', 'ref_spans': [{'start': 374, 'end': 380, 'mention': 'Fig. 2', 'ref_id': 'FIGREF1'}, {'start': 1158, 'end': 1164, 'mention': 'Fig. 2', 'ref_id': 'FIGREF1'}]}, {'text': 'Case 4 and Case 5 were the other two family members but were excluded from COVID-19. Remarkably, Case 4, who developed fever, was diagnosed with malignant lymphoma according to the result of her bone marrow puncture.', 'cite_spans': [], 'section': 'Case 4 and Case 5 ::: Results', 'ref_spans': []}, {'text': 'The ongoing outbreak of COVID-19, starting in December 2019, has quickly become a sweeping and unprecedented global challenge. Because COVID-19 can spread from person to person [10,11], a major challenge in controlling the disease is recognizing and quarantining potential infectious sources as soon as possible [12].', 'cite_spans': [{'start': 178, 'end': 180, 'mention': '10', 'ref_id': 'BIBREF9'}, {'start': 181, 'end': 183, 'mention': '11', 'ref_id': 'BIBREF10'}, {'start': 313, 'end': 315, 'mention': '12', 'ref_id': 'BIBREF11'}], 'section': 'Discussion', 'ref_spans': []}, {'text': \"As Li reported, the median incubation period is 4–5.2 days and the 95th percentile of the distribution is 12.5 days [13]. Unfortunately, COVID-19 patients can be infective before the disease onset and maintain infective ability during the convalescent phase [3,4,14]. According to Cao's report [15], the shortest observed duration of viral shedding was 8 days, whereas the longest was 37 days. However, to date, no article has reported what kind of patients have a longer viral shedding time. In the family we report here, the index patient, Case 1, was from Wuhan and had contact with COVID-19 patients before she returned to Yiyang city. The other 4 family members were from Yiyang and had never left Yiyang, nor did they have contact with any other COVID-19 patients. It is most likely that Case 1 was the infective source in this family. By further examining their epidemiological data, we find that the incubation period of COVID-19 and the shedding duration of SARS-CoV2 might be extremely long in this case. The index patient did not develop atypical symptoms (including a mild cough and a runny nose) until 28 days after she had returned from Wuhan. Remarkably, Case 1 was immunocompromised due to her long-term use of oral prednisone, which might explain why she did not develop typical symptoms at first and why she had a longer duration of viral shedding.\", 'cite_spans': [{'start': 117, 'end': 119, 'mention': '13', 'ref_id': 'BIBREF12'}, {'start': 259, 'end': 260, 'mention': '3', 'ref_id': 'BIBREF2'}, {'start': 261, 'end': 262, 'mention': '4', 'ref_id': 'BIBREF3'}, {'start': 263, 'end': 265, 'mention': '14', 'ref_id': 'BIBREF13'}, {'start': 295, 'end': 297, 'mention': '15', 'ref_id': 'BIBREF14'}], 'section': 'Discussion', 'ref_spans': []}, {'text': 'In summary, it is important to recognize potential patients who might have no or only mild symptoms. Therefore, we highly recommend that all people with an exposure history be strictly quarantined, and for immunocompromised people with long-term use of glucocorticoids, this time should not be limited to 14 days.', 'cite_spans': [], 'section': 'Discussion', 'ref_spans': []}, {'text': 'This work was supported by the Emergency Project of Prevention and Control for COVID-19 of Central South University (Grant 16026000 to Jie Meng), and COVID-19 Project of Changsha Science and Technology Bureau (Grant 38065 to Jie Meng).', 'cite_spans': [], 'section': 'Funding', 'ref_spans': []}, {'text': 'We declare no competing interests.', 'cite_spans': [], 'section': 'Declaration of Competing Interest', 'ref_spans': []}], 'paragraphs': [{'text': 'The average incubation period for COVID-19 is reported to be 2 to 14 days, and most patients show symptoms within 11.5 days [7,8]. To stop the spread of the disease, public health officials in Hunan placed those who may have been from Wuhan and exposed to COVID-19 under quarantine for 14 days. However, the incubation period may be longer in some patients. Here, we report on the last COVID-19 familial cluster in Hunan province, which was hospitalized on February 28. The disease was not confirmed until 40 days after the index patient had returned from Wuhan.', 'id': '00002'}], 'title': 'COVID-19 in a patient with long-term use of glucocorticoids: A study of a familial cluster', 'sha': '912c00514457672748738d29efb81f975815cdf6', 'doi': '10.1016/j.clim.2020.108413', 'score': 9.767399787902832}, {'rank': 20, 'abstract': 'BACKGROUND: The current outbreak of coronavirus disease 2019 (COVID-19) has quickly spread across countries and become a global crisis. However, one of the most important clinical characteristics in epidemiology, the distribution of the incubation period, remains unclear. Different estimates of the incubation period of COVID-19 were reported in recent published studies, but all have their own limitations. In this study, we propose a novel low-cost and accurate method to estimate the incubation distribution. METHODS: We have conducted a cross-sectional and forward follow-up study by identifying those asymptomatic individuals at their time of departure from Wuhan and then following them until their symptoms developed. The renewal process is hence adopted by considering the incubation period as a renewal and the duration between departure and symptom onset as a forward recurrence time. Under mild assumptions, the observations of selected forward times can be used to consistently estimate the parameters in the distribution of the incubation period. Such a method enhances the accuracy of estimation by reducing recall bias and utilizing the abundant and readily available forward time data. FINDINGS: The estimated distribution of forward time fits the observations in the collected data well. The estimated median of incubation period is 8·13 days (95% confidence interval [CI]: 7·37–8·91), the mean is 8·62 days (95% CI: 8·02–9·28), the 90th percentile is 14·65 days (95% CI: 14·00–15·26), and the 99th percentile is 20·59 days (95% CI: 19·47, 21·62). Compared with results in other studies, the incubation period estimated in this study is longer. INTERPRETATION: Based on the estimated incubation distribution in this study, about 10% of patients with COVID-19 would not develop symptoms until 14 days after infection. Further study of the incubation distribution is warranted to directly estimate the proportion with long incubation periods.', 'body_text': [{'text': 'The Center for Disease Control and Prevention (CDC) of China and World Health Organization (WHO) are closely monitoring the current outbreak of coronavirus disease 2019 (COVID-19). It was first identified in Wuhan, Hubei province, China, and has quickly spread across countries and become a global crisis. As of February 22, 2020, the National Health Commission (NHC) of China had confirmed a total of 76 936 cases of COVID-19 in mainland China, including 2442 fatalities and 22 888 recoveries.1 Various containment measures, including travel restrictions, isolation, and quarantine have been implemented in China with the aim to minimize virus transmission via human-to-human contact.2 Quarantine of individuals with exposure to infectious pathogens has always been an effective approach for containing contagious diseases in the past. One of the critical factors to determine the optimal quarantine of asymptomatic individuals is a good understanding of the incubation period, and this has been lacking for COVID-19.', 'cite_spans': [{'start': 494, 'end': 495, 'mention': '1', 'ref_id': 'BIBREF0'}, {'start': 685, 'end': 686, 'mention': '2', 'ref_id': 'BIBREF11'}], 'section': 'Introduction', 'ref_spans': []}, {'text': 'The incubation period of an infectious disease is the time elapsed between infection and appearance of the first symptoms and signs. Precise knowledge of the incubation period would help to provide an optimal length of quarantine period for disease control purpose, and also is essential in the investigation of the mechanism of transmission and development of treatment. For example, the distribution of the incubation period is used to estimate the reproductive number, that is, the average number of secondary infections produced by a primary case. The reproductive number is a key quantity that impacts the potential size of an epidemic. Despite the importance of the incubation period, it is often poorly estimated based on limited data.', 'cite_spans': [], 'section': 'Introduction', 'ref_spans': []}, {'text': 'To the best of our knowledge, there is only a handful of studies estimating the incubation period of COVID-19. Among them are Li et al (2020), Guan et al (2020), Backer et al (2020), and Linton et al (2020).3–6 In Li et al, the first 425 lab-confirmed cases, reported as of January 22, 2020, were included in the study, while only ten cases could be identified with the exact dates of exposure.3 The distribution of the incubation period was subsequently approximated by fitting a lognormal distribution to these ten data points, resulting in a mean incubation period of 5·2 days (95% CI: 4·1–7·0), and the 95th percentile is 12·5 days. However, given the limited sample size, it is challenging to make a solid inference on the distribution of the incubation period. A different result was reported by Guan et al, based on 291 patients who had clear information regarding the specific date of exposure as of January 29, 2020, stating that the median incubation period was 4·0 days (interquartile range, 2 to 7).4 However, such study of the incubation period can be highly influenced by the individuals’ recall bias or interviewers’ judgement on the possible dates of exposure rather than the actual dates of exposure that, in turn, might not be accurately monitored and determined, thus leading to a high percentage of error. In Backer et al, 88 confirmed cases detected outside Wuhan were used to estimate the distribution of the incubation period.5 For each selected case, a censored interval for the incubation period can be obtained by travel history and symptoms onset. The distribution of the incubation period can then be estimated by fitting a Weibull, gamma, or lognormal distribution with censored data. However, this method contained two types of sampling biases: (1) with the longer incubation period, the patients who resided at Wuhan but developed symptoms outside Wuhan were easier to be observed and hence lead to an overestimation; (2) if the follow-up time (from infection to the end of the study) is short, only the shorter incubation period would be observed and hence lead to an underestimation. Linton et al proposed a similar approach to estimate the incubation period of Backer et al, but corrected the aforementioned second sampling bias.5,6 However, the first problem in regard to sampling bias is still an unsolved issue. The estimates of the incubation period from these four studies, together with other results of two other coronavirus disease, SARS and MERS, are listed in Table 1.', 'cite_spans': [{'start': 207, 'end': 208, 'mention': '3', 'ref_id': 'BIBREF14'}, {'start': 209, 'end': 210, 'mention': '6', 'ref_id': 'BIBREF17'}, {'start': 394, 'end': 395, 'mention': '3', 'ref_id': 'BIBREF14'}, {'start': 1011, 'end': 1012, 'mention': '4', 'ref_id': 'BIBREF15'}, {'start': 1449, 'end': 1450, 'mention': '5', 'ref_id': 'BIBREF16'}, {'start': 2263, 'end': 2264, 'mention': '5', 'ref_id': 'BIBREF16'}, {'start': 2265, 'end': 2266, 'mention': '6', 'ref_id': 'BIBREF17'}], 'section': 'Introduction', 'ref_spans': [{'start': 2504, 'end': 2511, 'mention': 'Table 1', 'ref_id': 'TABREF0'}]}, {'text': 'To overcome the aforementioned problems, we propose a novel method to estimate the incubation period of COVID-19 by using the well-known renewal theory in probability.18 Such a method enhances the accuracy of estimation by reducing recall bias and utilizing abundance of the readily available forward time with a large sample size of 1211. To the best of our knowledge, this paper is a study of the distribution of the incubation period involving the largest number of samples to date. We find the estimated median of the incubation period is 8·13 days (95% CI: 7·37–8·91), and mean is 8·62 days (95% CI: 8·02–9·28), the 90th percentile is 14·65 days (95% CI: 14·00–15·26), and the 99th percentile is 20·59 days (95% CI: 19·47–21·62). Our estimated incubation period of COVID-19 is longer than the those given by previous researches on SARS, MERS, and COVID-19 in Table 1.', 'cite_spans': [{'start': 167, 'end': 169, 'mention': '18', 'ref_id': 'BIBREF9'}], 'section': 'Introduction', 'ref_spans': [{'start': 864, 'end': 871, 'mention': 'Table 1', 'ref_id': 'TABREF0'}]}, {'text': 'As described in the previous section, the distribution of the incubation period in most of the literature is either described through a parametric model or its empirical distribution based on the observed incubation period from the contact-tracing data. However, the contact-tracing data are challenging and expensive to obtain, and their accuracy can be highly influenced by recall bias. Hence, a low-cost and high-accuracy method to estimate the incubation distribution is needed.', 'cite_spans': [], 'section': 'Motivations ::: Methods', 'ref_spans': []}, {'text': 'Based on the COVID-19 daily updates from provincial and municipal health commissions in China, we notice that there is an abundance of cases who asymptomatically left Wuhan, the epicenter of COVID-19, and developed symptoms outside Wuhan. Assuming that these cases were infected before their departure from Wuhan, the time differences between departure and symptoms onset is the censored observations of their incubation periods. Hence, we conducted a cross-sectional and forward follow-up study by assuming to catch those asymptomatic individuals at their departure time and followed them until their symptoms developed. Using the language of renewal processes, we can treat the development of the disease starting from infection by a pathogen as a stochastic process that could be observed from a specific time point in chronological order. In this study, the specific time point refers to the time of departure from Wuhan. For each prevalent case, the complete process from the infection to the onset of symptoms can be considered as a renewal process. As illustrated in Figure 1, the backward recurrence time is hence defined as the time between infection and departure from Wuhan, and the forward recurrence time is the time between departure from Wuhan and symptom onset. Clearly, the forward time is observable and the corresponding observations are with good veracity, while the backward time is either unable to be observed or the corresponding observations are with large uncertainty due to recall bias. Note that for each infected individual, the backward time and forward time do not have to be same. However, when the renewal process reaches its equilibrium status, it becomes reversible, that is, the statistical properties of this process is the same as the one for time-reversed data in a same process. Hence, at equilibrium, the backward time can be treated as the forward time if time periods are reversed.19', 'cite_spans': [{'start': 1924, 'end': 1926, 'mention': '19', 'ref_id': 'BIBREF10'}], 'section': 'Motivations ::: Methods', 'ref_spans': [{'start': 1074, 'end': 1082, 'mention': 'Figure 1', 'ref_id': 'FIGREF0'}]}, {'text': 'In order to model incubation using the renewal process properly, the following assumptions are established:\\n(A1). The renewal process has reached its equilibrium status;(A2). The distribution of the incubation period is continuous;(A3). The distribution of the incubation period has a finite first moment;(A4). The incubation period for each case is independent and identically distributed;(A5). The cases included in the analysis were infected at Wuhan and developed their symptoms outside Wuhan.', 'cite_spans': [], 'section': 'Motivations ::: Methods', 'ref_spans': []}, {'text': 'In this study, it is reasonable to assume (A1) is satisfied between January 19, 2020, and January 23, 2020, because there are over eleven million residents in the Wuhan metropolitan area and nearby neighborhoods and the daily travel volume in and out of Wuhan exceeded million before January 23, 2020. We justify the use of data between January 19 and January 23 below. With adequate long run, the renewal process would reach the equilibrium status. The assumptions (A2) to (A4) are standard. In fact, we may assume that the incubation period is a continuous variable with range (0, M) for some finite number M. It is well known that the first moment exists for a bounded random variable. The justification for assumption (A5) is below. Therefore, the probability renewal process theory can be applied with confidence, and thus we can avoid the challenging mission of ascertaining the backward time.', 'cite_spans': [], 'section': 'Motivations ::: Methods', 'ref_spans': []}, {'text': 'Publicly available data were retrieved from provincial and municipal health commissions in China and the ministries of health in other countries, including 12 963 confirmed cases outside Hubei province as of February 15, 2020. Detailed information on confirmed cases includes region, gender, age, date of symptom onset, date of confirmation, history of travel or residency in Wuhan, and date of departure from Wuhan. The date of symptoms onset in these data refers to the date reported by the patient on which the clinical symptoms first appeared, where the clinical symptoms include fever, cough, nausea, vomiting, diarrhea, and others. Among 12 963 confirmed cases, 6345 cases had their dates of symptom onset collected; 3169 cases had histories of travel or residency in Wuhan; 2514 cases had their dates of departure recorded; and 1922 cases had records of both dates of departure from Wuhan and dates of symptoms onset.', 'cite_spans': [], 'section': 'Data collection and justification ::: Methods', 'ref_spans': []}, {'text': 'However, not all 1922 cases should be taken in the analysis. We have to ensure that (1) assumption (A5) is satisfied, and (2) the follow-up time is long enough. To make sure that the assumption (A5) is being satisfied as much as possible, we (1) exclude cases whose first symptoms appeared before departure, and (2) exclude cases who left Wuhan before January 19, 2020. This date was used because before January 19, the Chinese public was not aware of the severity of this epidemic, and those who left Wuhan might still have had close contact with other infected cases from Wuhan and hence actually got infected outside Wuhan. However, starting January 19, the China CDC began issuing test reagents to all provinces, confirmed cases were reported outside Hubei province in mainland China, the severity of COVID-19 was widely noted by the public, and various strict containment measures were implemented to minimize human-to-human transmission.2 Thus, it is unlikely that confirmed cases who left Wuhan after January 19, 2020, were infected outside Wuhan and assumption (A5) is supported. To ensure that the follow-up time is long enough such that no additional biased sampling occurred in this study, we excluded all cases who left Wuhan after January 23, 2020, which leaves an average follow-up time of 25 days (from date of departure to February 15, which is the end of this study). A 25-day follow-up period should be long enough based on the various studies on the incubation period of COVID-19.3–6 Note that those who left Wuhan after January 23 might not have enough time to develop symptoms before the end of the follow-up period. Including these cases in the cohort might lead to a downward bias on the incubation period. Note that the latest date of symptom onset in our cohort is February 12, 2020, which is ten days before the end of the follow-up period. This period should be long enough for a case to develop symptoms. Furthermore, there were only 49 cases who left Wuhan after the lockdown of Wuhan city on January 23, 2020.20 After examining the collected data, there were a total of 1211 cases that meet the criteria and were followed forwardly.', 'cite_spans': [{'start': 943, 'end': 944, 'mention': '2', 'ref_id': 'BIBREF11'}, {'start': 1499, 'end': 1500, 'mention': '3', 'ref_id': 'BIBREF14'}, {'start': 1501, 'end': 1502, 'mention': '6', 'ref_id': 'BIBREF17'}, {'start': 2039, 'end': 2041, 'mention': '20', 'ref_id': 'BIBREF12'}], 'section': 'Data collection and justification ::: Methods', 'ref_spans': []}, {'text': 'Figure 1 shows the design of the cross-sectional and forward follow-up study. The dot on the left end of each segment is a date of infection, while the square on the right end is a date of symptoms onset. The date of departure from Wuhan cuts the line segment in between. Note that only solid lines were followed in our cohort, while dashed lines are not in the cohort because the date of departure from Wuhan is not between January 19, 2020, and January 23, 2020.', 'cite_spans': [], 'section': 'Data collection and justification ::: Methods', 'ref_spans': [{'start': 0, 'end': 8, 'mention': 'Figure 1', 'ref_id': 'FIGREF0'}]}, {'text': 'Among the 1209 cases with gender information in the study, 533 (44·09%) are female. The mean age of patients was 41·31 and the median age was 40. Over 80% of the cases were between 20 and 60. The youngest confirmed case in our cohort was six months-old while the oldest was 86 years-old. Table 2 shows the demographic characteristics of patients with COVID-19 in the Wuhan departure cohort and the entire data collected as of February 15, 2020. We can see that there was no significant difference, which means the selected cases present the population well.', 'cite_spans': [], 'section': 'Data collection and justification ::: Methods', 'ref_spans': [{'start': 288, 'end': 295, 'mention': 'Table 2', 'ref_id': 'TABREF1'}]}, {'text': 'Let Y be the incubation period of an infected case with probability density function f(y) where y > 0. Let V denote the duration between departure from Wuhan and onset of symptoms, which can be considered as the forward time in a renewal process. Let A be the time between the infection and departure from Wuhan, which can be considered as the backward time. Clearly, A is not observable. It is known that in the cross-sectional sampling, A + V is a length-biased version of the incubation period Y, as it is easier to observe V if A + V is longer. The joint density of A and V is\\nf(a+v)μ,a,v≥0,\\nwhere μ=∫0∞f(y)dy is the mean incubation period, a and v are the realization of A and V. Marginally, A and V have the same density that is,\\n(1)g(v)=F¯(v)μ,v≥0,\\nwhere F¯(⋅) is the survival function corresponding to f(·). It is worth noting that A + V is a length-biased, version of Y, as with a longer observed Y, the corresponding V is easier to be observed. Hence the mean value E(A + V) is longer than the mean incubation period μ.', 'cite_spans': [], 'section': 'Estimation of incubation period distribution of COVID-19 ::: Methods', 'ref_spans': []}, {'text': 'In our cohort of COVID-19 cases, we assume the incubation period is a Weibull random variable with probability density function\\n(2)f(y)=αλ(yλ)α−1exp{−(yλ)α},y≥0.', 'cite_spans': [], 'section': 'Estimation of incubation period distribution of COVID-19 ::: Methods', 'ref_spans': []}, {'text': 'Using equations (1) and (2), it can be shown that the forward time has the density function as follows\\n(3)g(v)=αλexp{−(vλ)α}Γ(1/α),v≥0.', 'cite_spans': [], 'section': 'Estimation of incubation period distribution of COVID-19 ::: Methods', 'ref_spans': []}, {'text': 'Let vi be the observed forward times, i = 1, 2, …, I, where I = 1211 in the study, the estimates α^ and λ^ can be obtained by maximizing the likelihood function\\n(4)L(α,λ)=∏i=1Iαλexp{−(λvi)α}Γ(1/α),α>0,λ>0.', 'cite_spans': [], 'section': 'Estimation of incubation period distribution of COVID-19 ::: Methods', 'ref_spans': []}, {'text': 'The mean and percentiles of the incubation period can be calculated from the parametric Weibull distribution. The confidence intervals in this study are obtained using bootstrap method with B = 1000 resamples.', 'cite_spans': [], 'section': 'Estimation of incubation period distribution of COVID-19 ::: Methods', 'ref_spans': []}, {'text': 'It is arguable that people who left Wuhan might also be infected at the day of departure since they had a higher chance to be exposed to this highly contagious, human-to-human–transmitted virus in a crowded environment as cases were increasing. In such case, the duration between departure from Wuhan and onset of symptoms is no longer only the forward time, but a mixture of the incubation period and the forward time. Unfortunately, it is unclear who got infected before departure and who got infected at the event of departure. Hence, a mixture sensitivity forward time model is proposed, that is,\\nh(v)=(1−π)g(v)+πf(v),\\nwhere π is the proportion of people who contracted disease at the event of departure. Without additional auxiliary information, in general, it is impossible to identify this model nonparametrically. Even in some fully parametric cases, π is not identifiable. For example, if Y~exp(λ), that is f(y) = λexp(−yλ), y >0, it can be shown that V also follows an exponential distribution with g(v) = λexp(−vλ), v>0. In such a case, π is not identifiable. However, in this study, with the Weibull distribution assumption, we have\\nh(v)=αλ{π(vλ)α−1+(1−π)/Γ(1/α)}exp(−(vλ)α),v≥0.', 'cite_spans': [], 'section': 'Sensitivity analysis ::: Methods', 'ref_spans': []}, {'text': 'If α ≠ 1, it is possible to identify all underlying parameters. We explore the sensitivity of estimates of, incubation period by assuming a range of π, that is π = 0,0.005, 0.1, and 0.2, where π = 0 is the reference, and estimate α and λ by maximizing the product of likelihoods, ∏i=1Ih(vi), with respect to α and λ.', 'cite_spans': [], 'section': 'Sensitivity analysis ::: Methods', 'ref_spans': []}, {'text': 'The funder of the study had no role in study design, data collection, modelling analysis, results interpretation, or writing of this article. The corresponding authors had full access to all the data in the study and had final responsibility for the decision to submit for publication.', 'cite_spans': [], 'section': 'Role of the funding source ::: Methods', 'ref_spans': []}, {'text': 'By fitting the observed forward times vi of the 1211 cases in our cohort to the likelihood function (4), we find the maximum likelihood estimates α^=2.04 (95% CI: 1.80−2⋅32) and λ^=0.103 (95% CI: 0⋅096−0⋅111) in our reference scenario. The estimated 5th, 25th, 50th, 75th, 90th, 95th, 99th, and 99·9th percentiles of the incubation period are 2·27 (95% CI: 1·73–2·86), 5·28 (95% CI: 4·53–6·06), 8·13 (95% CI: 7·37–8·91), 11·42 (95% CI: 10·74–12·11), 14·65 (95% CI: 14·00–15·26), 16·67 (95% CI: 15·94–17·32), 20·59 (95% CI: 19·47–21·62), and 25·12 (95% CI: 23·35–26·87) days, respectively. The mean incubation period is 8·62 (95% CI: 8·02–9·28) days. The average time from leaving Wuhan to symptom onset is 5·44 days, the sample median is 5 days, and the maximum is 21 days. Figure 2 visualizes the fitted density function (3) in a solid line onto the histogram of observed forward times, and the dashed line is the Weibull probability density function (2) for incubation period distribution. Note that (3) is a monotonically decreasing function, which fits the observed forward times well, suggesting that our model is reasonable and the results are therefore trustworthy.', 'cite_spans': [], 'section': 'Results', 'ref_spans': [{'start': 774, 'end': 782, 'mention': 'Figure 2', 'ref_id': 'FIGREF1'}]}, {'text': 'Table 3 summaries the estimates of parameters, and the mean and percentiles of incubation period. We can see that the estimates for mean and percentiles decrease as the proportion of people who got infected at the event of departure, π, increases. However, the results are not shifted significantly. Variation of the results from π = 0 to 0.2 is only about one day.', 'cite_spans': [], 'section': 'Results', 'ref_spans': [{'start': 0, 'end': 7, 'mention': 'Table 3', 'ref_id': 'TABREF2'}]}, {'text': 'A sound estimate of the distribution of the incubation period plays a vital role in epidemiology. Its application includes decisions regarding the length of quarantine for prevention and control, dynamic models that accurately predict the disease process, and determining the contaminated source in foodborne outbreaks. In this paper, we propose a novel method to estimate the incubation distribution which only requires information on travel histories and dates of symptoms onset. This method enhances the accuracy of estimation by reducing recall bias and utilizing abundance of the readily available forward time data. In addition, this is the first article to consider the incubation period for COVID-19 virus as a renewal process which is a well-studied methodology and has a solid theoretical foundation. The estimated incubation period has a median of 8·13 days (95% CI: 7·37–8·91), a mean of 8·62 days (95% CI: 8·02–9·28), the 90th percentile is 14·65 days (95% CI: 14·00–15·26), and the 99th percentile is 20·59 days (95% CI: 19·47–21·62). Compared with the results published in Li et al, Guan et al, Backer et al, and Linton et al, the incubation period estimated in our study is significantly longer.3–6 Below is some evidence that may potentially support our findings of the long incubation period:\\nIn the study of Guan et al on behalf of China Medical Treatment Expert Group for COVID-19, the incubation period had a reported median of 4 days, the first quartile of 2 days and the third quartile of 7 days.4 By fitting a commonly used Weibull distribution to such quartiles, we can obtain α^=1.24 and λ^=0.186 defined in Equation (2). As a consequence, the estimated 90%, 95% and 99% percentiles are, respectively, 10·54, 13·04 and 18·45 days, which indicates that some patients may have extended incubation periods.One particular case reported by Yinbin municipal health commissions in China stated that a 64-year-old female was diagnosed with COVID-19 on February 11, 2020 at Yinbin, Sichuan province 20 days after returning from Wuhan. This patient was under self-quarantine at home with the family for 18 days, from January 23 to February 9. On February 8, the patient developed mild symptoms of cough with sputum production.21', 'cite_spans': [{'start': 1211, 'end': 1212, 'mention': '3', 'ref_id': 'BIBREF14'}, {'start': 1213, 'end': 1214, 'mention': '6', 'ref_id': 'BIBREF17'}, {'start': 1519, 'end': 1520, 'mention': '4', 'ref_id': 'BIBREF15'}, {'start': 2242, 'end': 2244, 'mention': '21', 'ref_id': 'BIBREF13'}], 'section': 'Discussion', 'ref_spans': []}, {'text': 'Based on the estimated incubation distribution in this study, about 10% of patients with COVID-19 would not develop symptoms until 14 days after infection Our approach does require that certain assumptions be met, which we detail below.\\nThe collection of forward time depends on the follow-up time, that is, if the follow-up time is not long enough, we would only be able to include those with a shorter incubation period in the Wuhan departure cohort. This may lead to an underestimation of the incubation period. The same limitation also applies to Backer et al and Linton et al.5,6 However, as explained earlier, we only included cases who left Wuhan before January 23 in this study, which leaves an average follow-up time of 25 days. Hence it is less likely we missed those patients with longer incubation periods based on the largest incubation period of 24 days reported in Guan et al.4 Note that the 24-day incubation period was reported as an outlier in Guan et al.4We assume that the individuals included in our cohort were either infected in Wuhan or on the way to their destination from Wuhan, violation of such assumption lead to an overestimation of incubation period. The same limitation also applies to Backer et al and Linton et al.5,6 However, with a carefully selected cohort justified in the section of Method, the chance for an individual in the Wuhan departure cohort getting infected outside Wuhan should be relatively small. Nonetheless, we acknowledge this possibility exists, for example, a family member could be uninfected by the time of departing Wuhan but got infected by other family members or outside contacts after leaving Wuhan. A sensitivity analysis was also conducted by removing all cases who left Wuhan with their families in the Wuhan departure cohort, and we found it only resulted in a small change of the estimated distribution of the incubation period.Individuals in our selected cohort were those who got infected in the early days of the outbreak. They were likely the first- or second-generation cases. Our results do not apply to higher generation cases if the virus mutates.', 'cite_spans': [{'start': 581, 'end': 582, 'mention': '5', 'ref_id': 'BIBREF16'}, {'start': 583, 'end': 584, 'mention': '6', 'ref_id': 'BIBREF17'}, {'start': 891, 'end': 892, 'mention': '4', 'ref_id': 'BIBREF15'}, {'start': 973, 'end': 974, 'mention': '4', 'ref_id': 'BIBREF15'}, {'start': 1248, 'end': 1249, 'mention': '5', 'ref_id': 'BIBREF16'}, {'start': 1250, 'end': 1251, 'mention': '6', 'ref_id': 'BIBREF17'}], 'section': 'Discussion', 'ref_spans': []}], 'paragraphs': [{'text': 'However, not all 1922 cases should be taken in the analysis. We have to ensure that (1) assumption (A5) is satisfied, and (2) the follow-up time is long enough. To make sure that the assumption (A5) is being satisfied as much as possible, we (1) exclude cases whose first symptoms appeared before departure, and (2) exclude cases who left Wuhan before January 19, 2020. This date was used because before January 19, the Chinese public was not aware of the severity of this epidemic, and those who left Wuhan might still have had close contact with other infected cases from Wuhan and hence actually got infected outside Wuhan. However, starting January 19, the China CDC began issuing test reagents to all provinces, confirmed cases were reported outside Hubei province in mainland China, the severity of COVID-19 was widely noted by the public, and various strict containment measures were implemented to minimize human-to-human transmission.2 Thus, it is unlikely that confirmed cases who left Wuhan after January 19, 2020, were infected outside Wuhan and assumption (A5) is supported. To ensure that the follow-up time is long enough such that no additional biased sampling occurred in this study, we excluded all cases who left Wuhan after January 23, 2020, which leaves an average follow-up time of 25 days (from date of departure to February 15, which is the end of this study). A 25-day follow-up period should be long enough based on the various studies on the incubation period of COVID-19.3–6 Note that those who left Wuhan after January 23 might not have enough time to develop symptoms before the end of the follow-up period. Including these cases in the cohort might lead to a downward bias on the incubation period. Note that the latest date of symptom onset in our cohort is February 12, 2020, which is ten days before the end of the follow-up period. This period should be long enough for a case to develop symptoms. Furthermore, there were only 49 cases who left Wuhan after the lockdown of Wuhan city on January 23, 2020.20 After examining the collected data, there were a total of 1211 cases that meet the criteria and were followed forwardly.', 'id': '00010'}], 'title': 'Estimation of incubation period distribution of COVID-19 using disease onset forward time: a novel cross-sectional and forward follow-up study', 'sha': 'c61be9fc4562d250e2dec241ef75bd22d61f61d1; 6d3b3f4ab80a61c45f82c61c6c756cfc6ddf4bf2', 'doi': '10.1101/2020.03.06.20032417', 'score': 9.764800071716309}, {'rank': 21, 'paragraphs': [{'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.24.20027474 doi: medRxiv preprint and age<55 groups. It suggests that age 40 can be a key age cutoff for the incubation of COVID-19 along with previous statistical analysis. ', 'id': '00027'}], 'title': 'Estimate the incubation period of coronavirus 2019 (COVID-19)', 'sha': 'c1ae608c7ffb926a0f50a6a34c0780983274ea74', 'doi': '10.1101/2020.02.24.20027474', 'score': 9.762100219726562}, {'rank': 22, 'paragraphs': [{'text': 'We have found in this work that the time-to-detection for a return to widespread transmission owing to relaxed physical distancing measures can be long, indeed considerably longer than than the mean incubation period [26, 14] or often used 14 day time period [5, 27, 17] . In order to decrease the time-to-detection, we need less noisy testing and faster ways to monitor community transmission. Outbreaks within communities contribute both to \"noise\" in case counts and to uncertainty in R 0 , particularly if they reveal areas of previous under-detection. It is therefore also important to maintain consistent surveillance spatially and temporally. Other surveillance techniques could facilitate faster and smoother case detection, but suffer from their own limitations. Consistent sentinel surveillance, e.g. as are seen in influenza-like-illness data, may have a slightly longer delay, but ultimately less noise. Symptom tracker apps could show changes in incidence sooner than laboratory-confirmed case counts, but could suffer from false positives and may be affected by coverage and usage limitations. Digital contact tracing may also support rapid case finding, but will likely ultimately rely on testing data for confirmation. However, case confirmation delays in a contact tracing context are often considerably shorter than through symptom-based testing [3] . Promising recent research has also investigated detection of SARS-CoV-2 in wastewater [22, 1] , which, although potentially not revealing of individual-level infection, may bypass case detection delays and provide an early warning system. Similarly, investigation of live mobility data during the disease outbreak [10, 13] may reveal changes in population behaviour, even if such work requires some assumptions about the link to changes in incidence. Given the long time frames for reported case count data it is urgent that robust combinations of diverse surveillance systems be developed, and for those seeking an overview of COVID-19 trajectories without reference to multi-faceted local surveillance data (perhaps at the national level or to support decisions about travel to and from other jurisdictions), it is important not to over-interpret short-term fluctuations in reported case counts. ', 'id': '00042'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.759599685668945}, {'rank': 23, 'paragraphs': [{'text': 'We also investigate the incubation period of 12 family cases and 47 non-family cases in the dataset. The family cases simply refer to the patients who were caught by COVID-19 because their family members had been infected. The Mann-Whitney rank test shows that there are not significant differences between family patients and non-family patients in terms of the median of incubation period. The Siegal-Tukey tests on scale also verify that the incubation scales are at the same level for family patients and non-family patients.', 'id': '00032'}], 'title': 'Estimate the incubation period of coronavirus 2019 (COVID-19)', 'sha': 'c1ae608c7ffb926a0f50a6a34c0780983274ea74', 'doi': '10.1101/2020.02.24.20027474', 'score': 9.752799987792969}, {'rank': 24, 'paragraphs': [{'text': \"The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.24.20027474 doi: medRxiv preprint those without clinical symptoms can still spread the virus even if they are 'officially negative' in the COVID-19 virus test [22] .\", 'id': '00031'}], 'title': 'Estimate the incubation period of coronavirus 2019 (COVID-19)', 'sha': 'c1ae608c7ffb926a0f50a6a34c0780983274ea74', 'doi': '10.1101/2020.02.24.20027474', 'score': 9.750300407409668}, {'rank': 25, 'paragraphs': [{'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.24.20027474 doi: medRxiv preprint [21] . It suggests that COVID-19 could have a faster distribution speed than H7N9, but the same spread speed as SARS and MERS in terms of their incubation periods [22] . The existing spread of COVID-19 is faster than SARS partially because it has more complicate spread dynamics [2] . For example, All rights reserved. No reuse allowed without permission.', 'id': '00029'}], 'title': 'Estimate the incubation period of coronavirus 2019 (COVID-19)', 'sha': 'c1ae608c7ffb926a0f50a6a34c0780983274ea74', 'doi': '10.1101/2020.02.24.20027474', 'score': 9.7431001663208}, {'rank': 26, 'paragraphs': [{'text': 'Then the large incubation sample median is calculated:', 'id': '00009'}], 'title': 'Estimate the incubation period of coronavirus 2019 (COVID-19)', 'sha': 'c1ae608c7ffb926a0f50a6a34c0780983274ea74', 'doi': '10.1101/2020.02.24.20027474', 'score': 9.7298002243042}, {'rank': 27, 'paragraphs': [{'text': 'Similarly, we can estimate the population mean by using the same way where the large incubation sample mean is calculated as', 'id': '00013'}], 'title': 'Estimate the incubation period of coronavirus 2019 (COVID-19)', 'sha': 'c1ae608c7ffb926a0f50a6a34c0780983274ea74', 'doi': '10.1101/2020.02.24.20027474', 'score': 9.729799270629883}, {'rank': 28, 'paragraphs': [{'text': 'Mann-Whitney rank test shows that there are no significant differences between the incubation of males ', 'id': '00023'}], 'title': 'Estimate the incubation period of coronavirus 2019 (COVID-19)', 'sha': 'c1ae608c7ffb926a0f50a6a34c0780983274ea74', 'doi': '10.1101/2020.02.24.20027474', 'score': 9.729798316955566}, {'rank': 29, 'paragraphs': [{'text': 'Incubation period and serial interval are also key parameters for epidemiological modelling in predicting the transmission dynamics, including the basic reproduction number (R0). R0 is defined as the average number of secondary cases caused by a single infectious individual in a totally susceptible population. 18 Estimation of R0 may require the knowledge of latent period and infectious period, 19, 20 which are both difficult to measure directly. For a disease that is not infectious until onset of symptoms, latent period is equivalent to incubation period, and infectious period can commonly be approximated by the difference of serial interval minus incubation period. 18 However, for COVID-19, the latent period appears to be shorter than incubation period as pre-symptomatic transmission might occur, 16, 17 and this should be accounted for when estimating R0. To our best knowledge, no published studies estimated the latent period of COVID-19, or how often and approximately at which time point the disease could be transmitted prior to the symptomatic onset.', 'id': '00010'}], 'title': 'Epidemiological parameters of coronavirus disease 2019: a pooled analysis of publicly reported individual data of 1155 cases from seven countries', 'sha': '3ece7750f80832bf9cc368c5138df12fd19183b9', 'doi': '10.1101/2020.03.21.20040329', 'score': 9.728699684143066}, {'rank': 30, 'paragraphs': [{'text': 'Data collected for this study included region, age, gender, exposure history, and illness onset. For those cases whose incubation periods locate in an interval [ ! , \" ], we use its midpoint = # ! $# \" \" to represent its incubation period. For example, Case no. 2 in our dataset went on a business trip in Wuhan on Jan 12 th , 2020 and returned to Shaanxi on Jan 15 th , 2020, but had fever symptoms on Jan 20 th 2020. The incubation will be calculated as = ', 'id': '00006'}], 'title': 'Estimate the incubation period of coronavirus 2019 (COVID-19)', 'sha': 'c1ae608c7ffb926a0f50a6a34c0780983274ea74', 'doi': '10.1101/2020.02.24.20027474', 'score': 9.719099998474121}, {'rank': 31, 'paragraphs': [{'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.24.20027474 doi: medRxiv preprint incubation period of COVID-19 by using original chain-of-infection data that may not be fully available in the Wuhan regions. Or data may lack meaningful exposure history. Furthermore, it is also unknown whether the incubation time will show some statistically significant with respect to Age and Gender. In this study, we aim to accurately estimate the incubation period of COVID-19 by taking advantage of datasets with a well-documented history of exposure. Our results show the incubation mean and median of COVID-19 are 5.84 and 5.0 days respectively and there is a statistical significance with the role of gender. However, the incubation periods of the groups with age>=40 years and age<40 years show a statistically significant difference. Our machine learning analysis also shows that the two groups are linearly separable that demonstrate a clear boundary in knowledge discovery visualization.', 'id': '00004'}], 'title': 'Estimate the incubation period of coronavirus 2019 (COVID-19)', 'sha': 'c1ae608c7ffb926a0f50a6a34c0780983274ea74', 'doi': '10.1101/2020.02.24.20027474', 'score': 9.718899726867676}, {'rank': 32, 'paragraphs': [{'text': 'where (0,1) is added Gaussian noise and ∈ [0,1] is a variance control parameter in simulation. Such a procedure is repeated times, the population median . =', 'id': '00010'}], 'title': 'Estimate the incubation period of coronavirus 2019 (COVID-19)', 'sha': 'c1ae608c7ffb926a0f50a6a34c0780983274ea74', 'doi': '10.1101/2020.02.24.20027474', 'score': 9.718000411987305}, {'rank': 33, 'paragraphs': [{'text': 'The estimated standard deviation . is calculated as the standard deviation of the median sequence ! , \" , ⋯ / . In our simulation, we chose = 100000, ! = 1, \" = 7, = .2 days in simulation and conduct simulations by employing Google', 'id': '00011'}], 'title': 'Estimate the incubation period of coronavirus 2019 (COVID-19)', 'sha': 'c1ae608c7ffb926a0f50a6a34c0780983274ea74', 'doi': '10.1101/2020.02.24.20027474', 'score': 9.717999458312988}, {'rank': 34, 'paragraphs': [{'text': 'Colab with TPU acceleration [9] [10] . The confidence level probability is calculated by the ratio / , where counts the times that * falls in the interval F . − 2 . , . + 2 . H in the simulation.', 'id': '00012'}], 'title': 'Estimate the incubation period of coronavirus 2019 (COVID-19)', 'sha': 'c1ae608c7ffb926a0f50a6a34c0780983274ea74', 'doi': '10.1101/2020.02.24.20027474', 'score': 9.717998504638672}, {'rank': 35, 'paragraphs': [{'text': 'The population mean is estimated as = ! / ∑ * / *-! and the estimated standard deviation . According to the central limit theorem, the population mean will be subject to the normal distribution. The confidence interval probability is calculated by following the same procedure.', 'id': '00014'}], 'title': 'Estimate the incubation period of coronavirus 2019 (COVID-19)', 'sha': 'c1ae608c7ffb926a0f50a6a34c0780983274ea74', 'doi': '10.1101/2020.02.24.20027474', 'score': 9.717997550964355}, {'rank': 36, 'paragraphs': [{'text': 'Our data consists of 34 male cases, 24 female cases and 1 unidentified gender case from ten regions in China.', 'id': '00015'}], 'title': 'Estimate the incubation period of coronavirus 2019 (COVID-19)', 'sha': 'c1ae608c7ffb926a0f50a6a34c0780983274ea74', 'doi': '10.1101/2020.02.24.20027474', 'score': 9.717996597290039}, {'rank': 37, 'paragraphs': [{'text': 'The minimum and maximum age are 10 and 70 respectively. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'id': '00017'}], 'title': 'Estimate the incubation period of coronavirus 2019 (COVID-19)', 'sha': 'c1ae608c7ffb926a0f50a6a34c0780983274ea74', 'doi': '10.1101/2020.02.24.20027474', 'score': 9.717995643615723}, {'rank': 38, 'paragraphs': [{'text': 'author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'id': '00021'}], 'title': 'Estimate the incubation period of coronavirus 2019 (COVID-19)', 'sha': 'c1ae608c7ffb926a0f50a6a34c0780983274ea74', 'doi': '10.1101/2020.02.24.20027474', 'score': 9.717994689941406}, {'rank': 39, 'paragraphs': [{'text': 'author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'id': '00025'}], 'title': 'Estimate the incubation period of coronavirus 2019 (COVID-19)', 'sha': 'c1ae608c7ffb926a0f50a6a34c0780983274ea74', 'doi': '10.1101/2020.02.24.20027474', 'score': 9.71799373626709}, {'rank': 40, 'paragraphs': [{'text': 'We further estimated population mean and standard deviation, median and 2.5 th and 97.5 th percentile by using Bootstrap under 10 6 times resampling. Table 3 author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'id': '00028'}], 'title': 'Estimate the incubation period of coronavirus 2019 (COVID-19)', 'sha': 'c1ae608c7ffb926a0f50a6a34c0780983274ea74', 'doi': '10.1101/2020.02.24.20027474', 'score': 9.717992782592773}, {'rank': 41, 'paragraphs': [{'text': 'author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'id': '00030'}], 'title': 'Estimate the incubation period of coronavirus 2019 (COVID-19)', 'sha': 'c1ae608c7ffb926a0f50a6a34c0780983274ea74', 'doi': '10.1101/2020.02.24.20027474', 'score': 9.717991828918457}, {'rank': 42, 'paragraphs': [{'text': 'The COVID-19 spread even shows that persons without any symptoms or clinically negative in infection can still spread it to others. There is no official vaccine or antiviral drug available up to now (Jan 30, 2020) to treat COVID-19 infected patients [3] [4] . The outbreak of COVID-19 infection is forcing China and many countries to take harsh protection policies. More than eight-thousands of infections have been reported in China and more than a dozen of countries until Jan 30, 2020. More than 15 cities including Wuhan have been quarantined to halt the spread of the COVID-19. It is expected that millions of people can be on lockdown because of COVID-19. WHO declared the coronavirus outbreak a global health emergency on Jan 30, 2020.', 'id': '00002'}], 'title': 'Estimate the incubation period of coronavirus 2019 (COVID-19)', 'sha': 'c1ae608c7ffb926a0f50a6a34c0780983274ea74', 'doi': '10.1101/2020.02.24.20027474', 'score': 9.709699630737305}, {'rank': 43, 'paragraphs': [{'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.24.20027474 doi: medRxiv preprint The incubation of COVID-19 is not subject to neither of the widely used incubation distributions such as normal, lognormal, Gamma, and Weibull distributions well [12] . We employ Shapiro-Wilk tests rather than', 'id': '00019'}], 'title': 'Estimate the incubation period of coronavirus 2019 (COVID-19)', 'sha': 'c1ae608c7ffb926a0f50a6a34c0780983274ea74', 'doi': '10.1101/2020.02.24.20027474', 'score': 9.707500457763672}, {'rank': 44, 'paragraphs': [{'text': 'The world has put enormous effort and expense into confronting COVID-19. To succeed in the long term, we need a much better understanding of the infectivity of COVID-19. Models with accurate descriptions of what happens after incubation can help, but only after we have a much better understanding of the minimum time between infection and infectivity. Without that understanding, all of our models will remain unreliable and our policies will remain uninformed.', 'id': '00043'}], 'title': 'Effects of latency on estimates of the COVID-19 replication number', 'sha': 'a017dd5e8dfa5dfd6385d407813e7c26d98becb9', 'doi': '10.1101/2020.04.07.20056960', 'score': 9.705599784851074}, {'rank': 45, 'paragraphs': [{'text': 'Pearson correlation coefficient analysis shows the R statistics is 0.244 with p-value: 0.06758. It suggests that that the incubation period is somewhat correlated with age though not that strong. To verify whether the incubation of the age>=40 group is different from that of the age<40 group statistically, Figure 3 compares All rights reserved. No reuse allowed without permission.', 'id': '00024'}], 'title': 'Estimate the incubation period of coronavirus 2019 (COVID-19)', 'sha': 'c1ae608c7ffb926a0f50a6a34c0780983274ea74', 'doi': '10.1101/2020.02.24.20027474', 'score': 9.692999839782715}, {'rank': 46, 'paragraphs': [{'text': 'Defining the incubation period of any infectious disease is crucial to evaluate transmission potential, estimate epidemic trends, and inform active monitoring and/or mandatory quarantine policies. The novel pathogenesis of COVID-19 has produced varied epidemiological characteristics from previous coronavirus-derived pulmonary infectious diseases, such as severe acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS). SARS and MERS were rarely transmitted during the asymptomatic period (5, 6) . In contrast, increasing evidence indicates that individuals infected with SARS-CoV-2 could be infectious during the asymptomatic incubation period (7) (8) (9) . Thus, knowledge of length and dispersion of incubation period is crucial for SARS-CoV-2 prevention and control. In addition, transmission dynamics models are designed to mimic the spread of SARS-CoV-2 in a nonlinear fashion, and are broadly used for long-term forecasting and evaluating the effect of prevention measures (10) . However, many parameters associated with SARS-CoV-2 transmission are poorly understood, including the incubation period, resulting in a biased prediction (11) . Multiple studies have explored the incubation period for COVID-19, but conclusions remain controversial due to limited sample sizes for each study and considerable heterogeneity between studies (12, 13) .', 'id': '00002'}], 'title': 'A systematic review and meta-analysis reveals long and dispersive incubation period of COVID-19', 'sha': '3d921c03dc23ee850655ddfd9d5ebe3956d70bf8', 'doi': '10.1101/2020.06.20.20134387', 'score': 9.692099571228027}, {'rank': 47, 'paragraphs': [{'text': 'Novel coronaviruses (COVID- 19) , which was found in Wuhan, China in December 2019 presents an acute public health threat to the whole world [1] [2] . The new virus is different from known coronaviruses such as SARS and MERS, though they share some similar respiratory illness symptoms such as fever, cough, or/and shortness of breath [2] . It is believed to root from the animal but spreads from person-to-person.', 'id': '00001'}], 'title': 'Estimate the incubation period of coronavirus 2019 (COVID-19)', 'sha': 'c1ae608c7ffb926a0f50a6a34c0780983274ea74', 'doi': '10.1101/2020.02.24.20027474', 'score': 9.684100151062012}, {'rank': 48, 'paragraphs': [{'text': 'We propose a Monte Carlo simulation approach that takes advantage of bootstrap techniques to estimate incubation median and mean estimation for the small sample with 59 cases. It is more data-driven compared to traditional parametric approaches to handle parameter estimation for small datasets. The proposed Monte Carlo simulation assumes we have a collected small incubation dataset . We generate a All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'id': '00007'}], 'title': 'Estimate the incubation period of coronavirus 2019 (COVID-19)', 'sha': 'c1ae608c7ffb926a0f50a6a34c0780983274ea74', 'doi': '10.1101/2020.02.24.20027474', 'score': 9.67389965057373}, {'rank': 49, 'paragraphs': [{'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.24.20027474 doi: medRxiv preprint large incubation sample * by concatenating n=1000 randomly sampled incubation segments *+ each of which contains at least entries in the interval [ ! , \" ] drawn independently from the existing dataset , i.e., * = ⋃ *+ .', 'id': '00008'}], 'title': 'Estimate the incubation period of coronavirus 2019 (COVID-19)', 'sha': 'c1ae608c7ffb926a0f50a6a34c0780983274ea74', 'doi': '10.1101/2020.02.24.20027474', 'score': 9.673898696899414}, {'rank': 50, 'paragraphs': [{'text': 'All 59 cases have complete epidemiological descriptions about the history of exposure. The total 57 cases have complete information in age and gender One case from Beijing has unknown age. The mean and standard deviation of age are 41.9 and 13.2 years old. The mode of his dataset is 4.0 with 14 support cases.', 'id': '00016'}], 'title': 'Estimate the incubation period of coronavirus 2019 (COVID-19)', 'sha': 'c1ae608c7ffb926a0f50a6a34c0780983274ea74', 'doi': '10.1101/2020.02.24.20027474', 'score': 9.650400161743164}, {'rank': 51, 'paragraphs': [{'text': 'Kolmogorov-Smirnov tests to conduct normality tests because we only have 59 records and Shapiro-Wilk tests can do a much better job on small datasets with a sample size of from 3 to 5000 [13] . Table 2 shows the All rights reserved. No reuse allowed without permission.', 'id': '00020'}], 'title': 'Estimate the incubation period of coronavirus 2019 (COVID-19)', 'sha': 'c1ae608c7ffb926a0f50a6a34c0780983274ea74', 'doi': '10.1101/2020.02.24.20027474', 'score': 9.650399208068848}, {'rank': 52, 'paragraphs': [{'text': 'The differential equations for the non-distancing compartments are given as follows:', 'id': '00005'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.648299217224121}, {'rank': 53, 'paragraphs': [{'text': 'The differential equations for individuals practising physical distancing are analogous:', 'id': '00006'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.648298263549805}, {'rank': 54, 'paragraphs': [{'text': 'who became symptomatic at some time s days ago:', 'id': '00009'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.648297309875488}, {'rank': 55, 'paragraphs': [{'text': 'All statistical analysis was performed in R [24] , and datasets and R code are available * on GitHub under an open source license.', 'id': '00024'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.648296356201172}, {'rank': 56, 'paragraphs': [{'text': 'The copyright holder for this preprint this version posted June 16, 2020. .', 'id': '00052'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.648295402526855}, {'rank': 57, 'paragraphs': [{'text': 'The copyright holder for this preprint this version posted June 16, 2020. . https://doi.org/10.1101/2020.06.14.20131177 doi: medRxiv preprint', 'id': '00059'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.648294448852539}, {'rank': 58, 'paragraphs': [{'text': \"As Li reported, the median incubation period is 4–5.2 days and the 95th percentile of the distribution is 12.5 days [13]. Unfortunately, COVID-19 patients can be infective before the disease onset and maintain infective ability during the convalescent phase [3,4,14]. According to Cao's report [15], the shortest observed duration of viral shedding was 8 days, whereas the longest was 37 days. However, to date, no article has reported what kind of patients have a longer viral shedding time. In the family we report here, the index patient, Case 1, was from Wuhan and had contact with COVID-19 patients before she returned to Yiyang city. The other 4 family members were from Yiyang and had never left Yiyang, nor did they have contact with any other COVID-19 patients. It is most likely that Case 1 was the infective source in this family. By further examining their epidemiological data, we find that the incubation period of COVID-19 and the shedding duration of SARS-CoV2 might be extremely long in this case. The index patient did not develop atypical symptoms (including a mild cough and a runny nose) until 28 days after she had returned from Wuhan. Remarkably, Case 1 was immunocompromised due to her long-term use of oral prednisone, which might explain why she did not develop typical symptoms at first and why she had a longer duration of viral shedding.\", 'id': '00012'}], 'title': 'COVID-19 in a patient with long-term use of glucocorticoids: A study of a familial cluster', 'sha': '912c00514457672748738d29efb81f975815cdf6', 'doi': '10.1016/j.clim.2020.108413', 'score': 9.64050006866455}, {'rank': 59, 'paragraphs': [{'text': 'The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.24.20027474 doi: medRxiv preprint Table 1 shows incubation statistics for five different groups: Male, Female, Age>=40, Age <40, and All that include all cases. It indicates that the incubation period median and mean of patients more than 40 years old are greater than those of patients less than 40 years old. Similar patterns are also observed for male and female groups. The mean values are always larger than median values for each group suggests the rightskewed distributions of incubation. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'id': '00018'}], 'title': 'Estimate the incubation period of coronavirus 2019 (COVID-19)', 'sha': 'c1ae608c7ffb926a0f50a6a34c0780983274ea74', 'doi': '10.1101/2020.02.24.20027474', 'score': 9.639100074768066}, {'rank': 60, 'paragraphs': [{'text': 'The copyright holder for this preprint this version posted June 16, 2020. . ber. In contrast, we work backwards from observed cases, using a model which takes observation delay and imperfect sampling into account, to understand the impact of physical distancing on COVID-19.', 'id': '00037'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.618800163269043}, {'rank': 61, 'paragraphs': [{'text': 'The copyright holder for this preprint this version posted June 16, 2020. . 50 US states with the Covid Act Now project [15] : each state is associated with a COVID-19 risk level based on how soon their projections arrive at certain constant thresholds on measures such as case counts and ICU headroom used. Our methods could improve such region-based work by allowing the parameters and baselines to be calibrated, reflecting population density, testing protocols, demographics and cultural factors regarding social contact. The times to observe the impacts of changes in control measures are likely to be region specific.', 'id': '00041'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.612000465393066}, {'rank': 62, 'paragraphs': [{'text': 'Here T is the final day for which case counts are available (or, of concern), f = (f 0 , f 1 , f 2 ),', 'id': '00011'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.570099830627441}, {'rank': 63, 'paragraphs': [{'text': '. CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'id': '00012'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.570098876953125}, {'rank': 64, 'paragraphs': [{'text': \"The ODEs described above require parameters which can be sampled from prior distributions. In particular, the parameter R 0 , the basic reproduction number, is a fundamental determinant of such models' dynamics. In the model above,\", 'id': '00014'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.570097923278809}, {'rank': 65, 'paragraphs': [{'text': 'The time to observe the strength of distancing is impacted by both the level of noise in the 11 . CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'id': '00030'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.570096969604492}, {'rank': 66, 'paragraphs': [{'text': '. CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'id': '00045'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.570096015930176}, {'rank': 67, 'paragraphs': [{'text': '. CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'id': '00051'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.57009506225586}, {'rank': 68, 'paragraphs': [{'text': '. CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'id': '00056'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.570094108581543}, {'rank': 69, 'paragraphs': [{'text': '. CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'id': '00058'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.570093154907227}, {'rank': 70, 'paragraphs': [{'text': 'The authors of [27] suggested that long-term glucocorticoid use could cause a delay in the onset of symptoms, so that they would not appear until after a 14 day quarantine period had been completed. Long-term glucocorticoid use could be responsible for atypical infection, longer incubation periods, and additional COVID-19 transmission. [13] states that COVID-19 has a mean incubation period lower than MERS coronavirus but higher than SARS coronavirus. [21] offers a final insight that robust and workable countermeasures must be put in place for the prevention and/or mitigation of asymptomatic transmission in the course of incubation amongst high-risk populations.', 'id': '00028'}], 'title': 'The estimations of the COVID-19 incubation period: a systematic review of the literature', 'sha': '09ff6011e4e95bff0b9221c89ffdb91d958e4ef1', 'doi': '10.1101/2020.05.20.20108340', 'score': 9.562899589538574}, {'rank': 71, 'paragraphs': [{'text': \"The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.24.20027474 doi: medRxiv preprint p-values under the Shapiro-Wilk tests for normal and lognormal distributions as well as Goodness-of-fit tests for Gamma and Weibull distributions by using the R package 'goft' [14] . We can reject normal, lognormal, and Weibull distributions strongly under the significance level of 0.05 cutoff [11] [12] . Although we can't reject Gamma distributions for its boundary line p-value (0.06086), it can be risky to use it to fit and estimate the distribution of the incubation period under a small sample size. We further employ the maximum likelihood estimates (MLEs) for the parameters of the gamma, conduct the goodness-fit test, and obtain p-value=8.6807e-04. As such, we only rely on nonparametric techniques in data analysis rather than use any pre-assumed distributions. \", 'id': '00022'}], 'title': 'Estimate the incubation period of coronavirus 2019 (COVID-19)', 'sha': 'c1ae608c7ffb926a0f50a6a34c0780983274ea74', 'doi': '10.1101/2020.02.24.20027474', 'score': 9.553600311279297}, {'rank': 72, 'abstract': 'During an infectious disease outbreak, biases in the data and complexities of the underlying dynamics pose significant challenges in mathematically modelling the outbreak and designing policy. Motivated by the ongoing response to COVID-19, we provide a toolkit of statistical and mathematical models beyond the simple SIR-type differential equation models for analysing the early stages of an outbreak and assessing interventions. In particular, we focus on parameter estimation in the presence of known biases in the data, and the effect of non-pharmaceutical interventions in enclosed subpopulations, such as households and care homes. We illustrate these methods by applying them to the COVID-19 pandemic.', 'body_text': [{'text': 'First documented in December 2019, an outbreak of community-acquired pneumonia began in Wuhan, Hubei Province, China. In January, this outbreak was attributed to a novel coronavirus, SARS-CoV-2. The initial spread of the pathogen in Wuhan was fast, and after a period of case-finding and contact tracing, China moved to implement a ‘shutdown’ of Wuhan on January 23, and other cities in China the following days, to try to suppress the growth of the epidemic. These measures may have succeeded at slowing down the rate at which cases have been seeded elsewhere, but in many countries initial importation of cases and transmission has not been contained. Countries around the world are now seeing outbreaks that are overwhelming, or have the potential to overwhelm, healthcare systems and cause a high number of deaths even in high-income countries (Remuzzi & Remuzzi, 2020).', 'cite_spans': [{'start': 849, 'end': 872, 'mention': 'Remuzzi & Remuzzi, 2020', 'ref_id': 'BIBREF59'}], 'section': 'COVID-19 pandemic background ::: Introduction', 'ref_spans': []}, {'text': 'While the majority of documented symptomatic cases are mild, characterised in many reports by a persistent cough and fever, a significant proportion of these individuals go on to develop pneumonia, with some then developing acute respiratory failure and a small proportion of overall cases becoming fatal. Severity of symptoms has been observed to increase with age and with the presence of underlying health conditions such as diabetes (Fang, Karakiulakis, & Roth, 2020) and cardiac conditions, with some evidence that severity of symptoms might depend on gender and ethnicity (Guan et al., 2020; Rimmer, 2020; Wu et al., 2020; Wu & McGoogan, 2020; Zhou et al., 2020).', 'cite_spans': [{'start': 438, 'end': 470, 'mention': 'Fang, Karakiulakis, & Roth, 2020', 'ref_id': 'BIBREF14'}, {'start': 579, 'end': 596, 'mention': 'Guan et al., 2020', 'ref_id': 'BIBREF23'}, {'start': 598, 'end': 610, 'mention': 'Rimmer, 2020', 'ref_id': 'BIBREF60'}, {'start': 612, 'end': 627, 'mention': 'Wu et al., 2020', 'ref_id': 'BIBREF80'}, {'start': 629, 'end': 648, 'mention': 'Wu & McGoogan, 2020', 'ref_id': 'BIBREF81'}, {'start': 650, 'end': 667, 'mention': 'Zhou et al., 2020', 'ref_id': 'BIBREF83'}], 'section': 'COVID-19 pandemic background ::: Introduction', 'ref_spans': []}, {'text': 'SARS-CoV-2 has a fast doubling time (the time it takes for the number of cases in the region to double, estimated at approximately 3 days (Pellis et al., 2020) and, potentially, a very large R0 (the average number of infections caused by each infected individual, with estimates ranging from 1.4 to 6.47 (Liu, Gayle, Wilder-Smith, & Rocklöv, 2020; Mahase, 2020; Majumder & Mandl, ; World Health Organisation, 2020)). It is possible that there is a significant degree of asymptomatic and/or pre-symptomatic transmission (Li et al., 2020; Mizumoto, Kagaya, Zarebski, & Chowell, 2020; Nishiura et al., 2020), though without robust serosurveys, this is difficult to quantify with certainty. These characteristics result in the pathogen being able to spread widely, rapidly and undetected, presenting a significant risk to public health.', 'cite_spans': [{'start': 139, 'end': 158, 'mention': 'Pellis et al., 2020', 'ref_id': 'BIBREF54'}, {'start': 305, 'end': 346, 'mention': 'Liu, Gayle, Wilder-Smith, & Rocklöv, 2020', 'ref_id': 'BIBREF41'}, {'start': 348, 'end': 360, 'mention': 'Mahase, 2020', 'ref_id': 'BIBREF43'}, {'start': 362, 'end': 379, 'mention': 'Majumder & Mandl,', 'ref_id': 'BIBREF45'}, {'start': 382, 'end': 413, 'mention': 'World Health Organisation, 2020', 'ref_id': 'BIBREF79'}, {'start': 520, 'end': 535, 'mention': 'Li et al., 2020', 'ref_id': 'BIBREF40'}, {'start': 537, 'end': 580, 'mention': 'Mizumoto, Kagaya, Zarebski, & Chowell, 2020', 'ref_id': 'BIBREF46'}, {'start': 582, 'end': 603, 'mention': 'Nishiura et al., 2020', 'ref_id': 'BIBREF50'}], 'section': 'COVID-19 pandemic background ::: Introduction', 'ref_spans': []}, {'text': 'Typically, the aim of an intervention strategy would be to push and keep the reproduction number Rt, defined as the average number of cases generated by a typical infective at time t, below 1. At this point each infected individual subsequently infects, on average, less than one individual, such that the number of cases should decline. The basic reproduction number, R0, represents the initial value of Rt, before any intervention is put in place and the population can be assumed to be fully susceptible.', 'cite_spans': [], 'section': 'COVID-19 pandemic background ::: Introduction', 'ref_spans': []}, {'text': 'High R0, fast growth, and possible pre- or asymptomatic infection make the design of potential interventions, and the modelling that would inform them, particularly challenging. Large values of R0 mean a substantial amount of transmission needs to be halted; fast growth causes the number of cases in the absence of interventions to rise rapidly, so that the time scale of interventions to reduce R0 must also be fast in order to effect substantive early changes on a population level; finally, the resulting interventions must encompass possible pre- and asymptomatic cases, a challenging prospect when in many instances these individuals are indistinguishable from healthy individuals. Consequently, we must consider the possibility of interventions that are massively disruptive to society and may have to be sustained for a long period of time in order to cause the number of infections to decline towards zero (Ferguson et al., 2020). If infections remain, and the susceptible proportion of the population remains above the herd immunity threshold, these interventions must be upheld to prevent a second wave of the epidemic. There is not yet conclusive evidence as to the degree and duration of immunity conferred by infection with SARS-CoV-2 nor the feasibility of a vaccine, the timeline for which is unlikely to be any time shorter than 18 months away at the time of writing (Jack, 2020). Therefore, short term extreme interventions are not as effective as they might be in other circumstances, since after their removal there remains a long period of time in which cases can rise again. The longer these significantly suppressive and disruptive interventions are in effect, the more severe the effect on the economy, and broader societal health and well-being. Furthermore, adherence to interventions will likely vary with their duration and severity.', 'cite_spans': [{'start': 916, 'end': 937, 'mention': 'Ferguson et al., 2020', 'ref_id': 'BIBREF16'}, {'start': 1385, 'end': 1395, 'mention': 'Jack, 2020', 'ref_id': 'BIBREF27'}], 'section': 'COVID-19 pandemic background ::: Introduction', 'ref_spans': []}, {'text': 'We are further challenged by the lack of transferable intuition. Early work looked at intuition gained from SARS and MERS outbreaks, also caused by coronaviruses. Some parameters do appear to be similar to these pathogens, such as the average length of the incubation period (Lauer et al., 2020; Varia et al., 2003; Virlogeux, Fang, Park, Wu, & Cowling, 2016). However, there are also clear differences, with both SARS and MERS being more fatal, but seemingly less efficient at spreading since they did not seed major global pandemics. Another complication is the spread of the infection during the Chinese Spring Festival, a time period during which movement, social, and contact patterns vary significantly. This presents significant challenges as experience and intuition from other studies regarding population mixing and spatial patterns must either be modified or are invalid. Furthermore, the pandemic has received a proportionately larger level of public attention than e.g. the 2009 H1N1 pandemic (Chew & Eysenbach, 2009; Rubin, Amlôt, Page, & Wessely, 2009), largely boosted by social media. This greater level of public awareness, and the successive, staggered interventions placed to prevent disease spread are responsible for significant variations in behaviour (Butler, 2014; Funk, Gilad, Watkins, & Jansen, 2009) and adherence to public guidance both in China and abroad.', 'cite_spans': [{'start': 276, 'end': 294, 'mention': 'Lauer et al., 2020', 'ref_id': 'BIBREF37'}, {'start': 296, 'end': 314, 'mention': 'Varia et al., 2003', 'ref_id': 'BIBREF75'}, {'start': 316, 'end': 358, 'mention': 'Virlogeux, Fang, Park, Wu, & Cowling, 2016', 'ref_id': 'BIBREF76'}, {'start': 1007, 'end': 1029, 'mention': 'Chew & Eysenbach, 2009', 'ref_id': 'BIBREF10'}, {'start': 1031, 'end': 1066, 'mention': 'Rubin, Amlôt, Page, & Wessely, 2009', 'ref_id': 'BIBREF63'}, {'start': 1276, 'end': 1288, 'mention': 'Butler, 2014', 'ref_id': 'BIBREF6'}, {'start': 1290, 'end': 1326, 'mention': 'Funk, Gilad, Watkins, & Jansen, 2009', 'ref_id': 'BIBREF18'}], 'section': 'COVID-19 pandemic background ::: Introduction', 'ref_spans': []}, {'text': 'The structure of this paper follows two main themes. In Section 2, we discuss various biases that are present in outbreak data and techniques for estimating epidemiological parameters. Accounting for biases and producing robust parameter estimates is important throughout the duration of an epidemic, both for increasing our understanding of the underlying dynamics, and for feeding into models. Firstly, we discuss a bias-corrected method for estimating the incubation period, which can also be applied to serial intervals, onset-to-death time, and other delay distributions. We then present a method for estimating the true growth rate of the epidemic, accounting for the bias encountered since infected individuals may be exported from the region. Our next method is a tool for estimating the expected size of the next generation of infectives based on the rate of observed cases. This tool provides insight into the size of small outbreaks, which can inform decision making when trying to prevent a major outbreak taking off.', 'cite_spans': [], 'section': 'COVID-19 pandemic background ::: Introduction', 'ref_spans': []}, {'text': 'In Section 3, we propose a variety of mathematical models looking at disease impact and intervention strategies, with particular focus on non-pharmaceutical interventions due to the current lack of widely deployable, targeted pharmaceutical treatments. These models focus on enclosed populations, since this is the level at which most interventions are implemented. Since the disease is particularly fatal in the elderly and other at-risk groups, we develop a care home model to investigate how the pathogen may spread through care homes. We also develop household models to investigate the impact of different intervention/control strategies. These models can inform policy design for mitigating or controlling epidemic spread. Finally, in the context of relaxing strong social distancing policies, we investigate the extinction probability of the pathogen. We first consider the extinction probability after lifting restrictions. We then develop a household-based contact tracing model, with which we investigate the extinction probability under weaker isolation policies paired with contact tracing, thus shedding light on possible combinations of interventions that allow us to feasibly manage the infection while minimising the social impact of control policies.', 'cite_spans': [], 'section': 'COVID-19 pandemic background ::: Introduction', 'ref_spans': []}, {'text': 'Techniques are constantly developing that enable higher volumes of more accurate data to be collected real-time during an epidemic. These data present a large opportunity for analysis to gain insight into the pathogen and the dynamics of the outbreak. However, although the quality of the data is constantly increasing, there are still many biases present. Some of these are due to the data collection methods, and in an ideal world we would be able to eliminate them, and some are simply due to the nature of the outbreak, and will be present regardless of data collection methods.', 'cite_spans': [], 'section': 'Potential biases in the outbreak data ::: Biases and estimation during outbreaks', 'ref_spans': []}, {'text': 'During an outbreak, many parameters depend on delay distributions (the length of time between two events), such as the time from infection to symptom onset (the incubation period). If an individual can be followed indefinitely, it is easy to determine the length of these events. However, in reality only events that occur before a given date are observed. Therefore, the data is subject to censoring and truncation issues. In the incubation period, for example, censoring comes into play since, if we have observed an infection but the individual has not yet developed symptoms, we only have a lower bound on how long it will take them to develop symptoms. To account for this, we can instead condition on observing symptom onset before the cut-off date. However, this leads to a truncation issue, since individuals who were infected close to the cut-off date will only be observed if they have a short incubation period, which leads to an overexpression of short delays.', 'cite_spans': [], 'section': 'Potential biases in the outbreak data ::: Biases and estimation during outbreaks', 'ref_spans': []}, {'text': 'The number of cases tends to grow exponentially during the early stages of an outbreak, causing the force of infection and the number of reported cases to increase with time. This further complicates the truncation issue since not only are recent cases truncated but they also account for the majority of cases. The growing force of infection also needs to be accounted for, since if the potential time of infection is interval-censored rather than observed directly, the probability that the case was infected in each day of that interval is not constant.', 'cite_spans': [], 'section': 'Potential biases in the outbreak data ::: Biases and estimation during outbreaks', 'ref_spans': []}, {'text': 'In theory, both of these biases are relatively straightforward to account for. In practice however, there are other biases in the data. One of the major biases is the reporting rate. Although the total number of cases may be reasonably described as growing exponentially with a constant rate in the early stages of an outbreak, high-resolution data may exhibit more complex behaviour. This can be due to a variety of reasons, such as the workload becoming overwhelming, the availability of individual-level data decreasing, the laboratories or offices slowing down activity over the weekend, the case definition changing, the testing capabilities increasing, and so on.', 'cite_spans': [], 'section': 'Potential biases in the outbreak data ::: Biases and estimation during outbreaks', 'ref_spans': []}, {'text': 'Another uncertainty arises since generally only the date of each event is recorded rather than the time. This presents a large window of uncertainty in the length of the delay, since the time of each event can vary up to 24 h, and for a delay distribution, which depend on two events, it could vary by up to 48 h.', 'cite_spans': [], 'section': 'Potential biases in the outbreak data ::: Biases and estimation during outbreaks', 'ref_spans': []}, {'text': 'Travel rate is another bias present in the data. For example, this changes the density of observed cases in a region, which can change the apparent growth rate. Intervention strategies present a further bias because this can change the growth rate of the epidemic and the reporting rate. Additionally, estimates of certain parameters may vary depending on the interventions that are implemented, so these need to be considered carefully.', 'cite_spans': [], 'section': 'Potential biases in the outbreak data ::: Biases and estimation during outbreaks', 'ref_spans': []}, {'text': 'Here we demonstrate the importance of truncation (Table 1\\n). We use the data truncated at 20 January, which has exposure windows between 1 December and 19 January. This data set is chosen since it is most sensitive to truncation due to the exponentially growing force of infection and high reporting rate. Without accounting for truncation, the length of the incubation period is significantly underestimated, which could have a large impact on the success of intervention strategies.', 'cite_spans': [], 'section': 'Truncation ::: Incubation period ::: Biases and estimation during outbreaks', 'ref_spans': [{'start': 50, 'end': 57, 'mention': 'Table 1', 'ref_id': 'TABREF0'}]}, {'text': 'To demonstrate the effectiveness of the bias correction method, we compare three different data sets (Table 2\\n). The similar distributions predicted across these datasets suggests a robust method. Fig. 1b compares the full distributions for these three estimates.', 'cite_spans': [], 'section': 'Different data sets ::: Incubation period ::: Biases and estimation during outbreaks', 'ref_spans': [{'start': 197, 'end': 203, 'mention': 'Fig. 1', 'ref_id': 'FIGREF0'}, {'start': 102, 'end': 109, 'mention': 'Table 2', 'ref_id': 'TABREF1'}]}, {'text': 'Here we investigate the effect that uncertainty in the reporting date can have on the results, using the data truncated at 9 February (Table 3\\n). The standard interval is the recorded data, wide intervals are obtained by removing a day from the exposure window lower bound and adding a day to the upper bound, and the narrow interval vice versa. The uncertainty in the reporting date can impact the estimated incubation period, showing that it is important to consider this risk when designing interventions.', 'cite_spans': [], 'section': 'Reporting date uncertainty ::: Incubation period ::: Biases and estimation during outbreaks', 'ref_spans': [{'start': 135, 'end': 142, 'mention': 'Table 3', 'ref_id': 'TABREF2'}]}, {'text': 'When constructing intervention strategies for an epidemic, the incubation period is an important parameter. For example, consider the quarantine strategy deployed in many countries during the early stages of the epidemic, aimed at preventing cases being imported from Wuhan. This strategy quarantined individuals upon their return from Wuhan for 14 days. For such a strategy to be effective, we require most incubation periods to be less than 14 days, so that the majority of infected people would develop symptoms before quarantine ended, enabling them to be further isolated. In this analysis, we show that in the worst-case scenario we would expect 1 in 62 cases to slip through this quarantine, with the best fit predicting 1 in 101 cases. Therefore, the 14 day quarantine period would capture the majority of cases. Throughout the epidemic, this seems to have been reasonably successful and prevented early seeding of cases in many countries. However, potentially due to complicated travel patterns or asymptomatic transmission, cases have slipped through detection and not been quarantined, which unfortunately has led to the situation observed today.', 'cite_spans': [], 'section': 'Implications ::: Incubation period ::: Biases and estimation during outbreaks', 'ref_spans': []}, {'text': 'In addition to the incubation period, there are many other delay distributions that must be estimated while an epidemic is growing, which can be estimated using the same technique. These include the generation time, the time between two infection events in a transmission chain; the serial interval, the time between symptom onset of an infector to their infectee; and the onset-to-death delay, the time from symptom onset to death.', 'cite_spans': [], 'section': 'Implications ::: Incubation period ::: Biases and estimation during outbreaks', 'ref_spans': []}, {'text': 'Transportation modelling plays a crucial role in the early stages of an outbreak; an infected individual may travel outside of the region in which they were originally infected and seed further infections across geographical scales which are impossible to contain. Furthermore, as the rate of travelling increases, the number of observed cases within the known ‘‘origin” region decreases, and if exportation is not taken into account this results in an underestimation of the number of cases. These underestimates can be improved by looking at the total number of cases across all known affected regions, but doing so introduces further complications. For example, if an individual has less severe symptoms they may not seek medical assistance, thereby not being recorded as a case at their destination. This underestimation of cases can have significant effects if the traveller is able to infect more people. A new transmission chain can thus be started which remains undetected for some time due to a lacking known connection to the ‘‘origin” region.', 'cite_spans': [], 'section': 'Transportation modelling and under-reporting ::: Biases and estimation during outbreaks', 'ref_spans': []}, {'text': 'In the ‘‘origin” region an individual with mild symptoms may still be tested for an infection due to a higher level of alertness in the local health care system. However, this level of active case-finding may not be present elsewhere, or may not have been allocated a comparable level of resources. Further complications to this model arise from the incubation period of individuals wherein detection is unlikely, and the variations in movement and mixing between people when preventative measures are put in place.', 'cite_spans': [], 'section': 'Transportation modelling and under-reporting ::: Biases and estimation during outbreaks', 'ref_spans': []}, {'text': 'We consider a metapopulation model seeded with an infection in one of the regions, O, and investigate how exportation from this region combined with variability in case-finding can alter estimates for the doubling time and the expected portion of the population we expect to identify. This accordingly bounds the proportion of the infected population one would be able to target for personal intervention (e.g. quarantine or treatment). Note that the proportion of identified cases need not necessarily correlate with the proportion of the infected population who exhibit symptoms.', 'cite_spans': [], 'section': 'Transportation modelling and under-reporting ::: Biases and estimation during outbreaks', 'ref_spans': []}, {'text': \"Let us assume that movement from O begins at time t=tc, and occurs with a constant rate ρ; this can be thought of as the surge in travel in China during the beginning of the Spring Festival. In the early phase of an epidemic, we can assume the incidence I(t)=I0ert of cases to be growing exponentially with a rate r. The number of cases at time t which were infected a time τ ago is denoted by i(t,τ), where the probability of detecting a case that infected a host a time τ ago is given by p(τ,τinc). This probability depends on the incubation period τinc of the infection, and is decomposed into a detection probability f(τ,τinc) after some time τ which may also depend on the individual's incubation period, and the probability density function of said incubation period, g(τinc). Hence, p(τ,τinc)=f(τ,τinc)g(τinc) such that the number of observed cases in O is given byCO(t)=I0e−ρ(t−tc)Θ(t−tc)∫0tg(τinc)∫0ter(t−τ)f(τ,τinc)dτdτinc,where Θ(⋅) is the Heaviside step function, and we have assumed that recovery of cases is negligible over the time scale of case observations. If we consider travel to i other regions from O, the total number of observed cases in all destinations isCD(t)=∑i≠OCi(t)=ωCO(t)(eρ(t−tc)Θ(t−tc)−1),where ω is the mean case-finding ability across all destinations. In the presence of real-time transition probabilities pij of moving between two regions, these estimates can be further elaborated.\", 'cite_spans': [], 'section': 'Transportation modelling and under-reporting ::: Biases and estimation during outbreaks', 'ref_spans': []}, {'text': 'We assume that detection occurs immediately following the end of the incubation period, i.e. f(τ,τinc)=Θ(τ−τinc). Similarly, we assume a gamma-distributed incubation period g(τinc)=1Γ(k)θkτinck−1e−τinc/θ with shape and scale parameters k and θ, respectively. We can parametrise this distribution using the ‘‘non-Wuhan” estimate of the incubation period in Table 2, which yields a gamma-distribution with mean 4.84 and standard deviation 3.22. In contrast to other values in the table, this estimate is obtained from discrete infection events, e.g. contact with a known infected case, and therefore has a constant reporting rate, and a constant force of infection over each exposure window. Therefore, this estimate of the incubation period does not rely on the exponential growth rate unlike other estimates from Section 2.2.', 'cite_spans': [], 'section': 'Transportation modelling and under-reporting ::: Biases and estimation during outbreaks', 'ref_spans': [{'start': 356, 'end': 363, 'mention': 'Table 2', 'ref_id': 'TABREF1'}]}, {'text': 'Historic estimates for Chinese travel data indicate a mean travel rate from Hubei province of ρ=0.029 which began on January 10 (Keju, 2019; Morris). Using the above incubation period, this suggests a rate r=0.22±0.01 when ignoring travel exportation, in contrast to r=0.25±0.01 when accounting for ρ. This difference may seem small, but it reduces the doubling time by approximately 12 h. The expected value of r grows linearly with the exportation rate, which has also been observed with real-time travel models (Kraemer et al., 2020). Further models have also been developed which consider travel and exportation of cases in greater detail (Chinazzi et al., 2020; Gostic, Gomez, Mummah, Kucharski, & Lloyd-Smith, 2020).', 'cite_spans': [{'start': 129, 'end': 139, 'mention': 'Keju, 2019', 'ref_id': 'BIBREF30'}, {'start': 141, 'end': 147, 'mention': 'Morris', 'ref_id': 'BIBREF47'}, {'start': 515, 'end': 535, 'mention': 'Kraemer et al., 2020', 'ref_id': 'BIBREF35'}, {'start': 644, 'end': 665, 'mention': 'Chinazzi et al., 2020', 'ref_id': 'BIBREF12'}, {'start': 667, 'end': 720, 'mention': 'Gostic, Gomez, Mummah, Kucharski, & Lloyd-Smith, 2020', 'ref_id': 'BIBREF21'}], 'section': 'Transportation modelling and under-reporting ::: Biases and estimation during outbreaks', 'ref_spans': []}, {'text': 'The relationship between the observed cases in our origin and destinations can be used to determine the case-finding ability, though it should be noted that ω likely varies with time as burdens are increased on public services and the number of cases grow. Early estimates using data from (Recorded daily case updat, 2020) indicate at most an 80% case-finding ability, suggesting thousands of undetected cases exported to other regions of China, a sufficient quantity to sustain further transmission post-exportation independently of the number of asymptomatic cases present.', 'cite_spans': [{'start': 290, 'end': 321, 'mention': 'Recorded daily case updat, 2020', 'ref_id': 'BIBREF58'}], 'section': 'Transportation modelling and under-reporting ::: Biases and estimation during outbreaks', 'ref_spans': []}, {'text': 'The intention of these estimates is not to provide specific values for the doubling time of the spread of COVID-19 in China (as the estimates above use historic travel data and are limited by the availability of data), but to bring attention to the unusual circumstances surrounding changes in contact patterns, and mobility during the Chinese Spring Festival, the largest human migration on Earth (Keju, 2019). Failing to account for the significant level of dispersion or exportation of cases during these circumstances will significantly skew our estimates.', 'cite_spans': [{'start': 399, 'end': 409, 'mention': 'Keju, 2019', 'ref_id': 'BIBREF30'}], 'section': 'Transportation modelling and under-reporting ::: Biases and estimation during outbreaks', 'ref_spans': []}, {'text': 'In a scenario where a single individual exposes a group to infection, it can be unclear how many people have been infected since they do not immediately develop symptoms. However, knowing the true prevalence in the population is essential to determine the most effective interventions to put in place, and to estimate future burdens on public services. Using the probability density function of the incubation period, we consider the efficacy of using the time it takes for people to present with symptoms as a predictor for the size of the infected group. This analysis is an effective ready reckoner at early stages of a novel infection, or in close contact environments, and is useful for predicting generation size when a complete data set is not yet available. In this analysis we focus on a scenario where infection time is known. In reality, we may only know an exposure window. For short exposure windows this method can still be valid, but for longer exposure windows it will need extending to account for this added uncertainty.', 'cite_spans': [], 'section': 'Estimating the size of the first generation from the observed number of symptomatic individuals ::: Biases and estimation during outbreaks', 'ref_spans': []}, {'text': 'We assume that the number of individuals who have been exposed to potential infection is known, in which case the number of people who are infected can be assumed to be binomially distributed with an unknown probability P that each individual has been infected. To determine the distribution of infected individuals, we use the available information regarding the number of individuals who have expressed symptoms. This yields two cases. In the first case, we assume that the true number of symptomatic individuals are observed. In the second case, we take the number of observed symptomatic individuals as a lower bound on the true value.', 'cite_spans': [], 'section': 'Estimating the size of the first generation from the observed number of symptomatic individuals ::: Biases and estimation during outbreaks', 'ref_spans': []}, {'text': 'We wish to determine the probability that the first generation has e0 individuals, E0=e0, given that iτ symptomatic individuals are observed on day τ, Iτ=iτ. This is given by (see Appendix B)P(E0=eo|Iτ=iτ)=e0!(e0−iτ)!(1−F(τ))e0×(1−F(τ))−iτ(n−iτ)!(iτ+1)(n+1)!2F1(iτ+1,iτ−n,iτ+2,F(τ)).\\n', 'cite_spans': [], 'section': 'Estimating the size of the first generation from the observed number of symptomatic individuals ::: Biases and estimation during outbreaks', 'ref_spans': []}, {'text': 'This gives a distribution of the generation-size based on the number of observed symptomatic individuals by time τ. We can extend it to investigate a scenario where no symptomatic individuals have been observed by time τ by using a value of 0 for Iτ:P(E0=e0|Iτ=0)=(1−F(τ))e0(n+1)F21(1,−n,2,F(τ))\\n', 'cite_spans': [], 'section': 'Estimating the size of the first generation from the observed number of symptomatic individuals ::: Biases and estimation during outbreaks', 'ref_spans': []}, {'text': 'This can be used to illustrate worst and best case scenarios given τ time has passed without symptomatic individuals. Additionally, if we consider the probability that E0 = 0, we can find the value of τ where we can have a 95% confidence that there will not be a second generation:P(E0=0|Iτ=0)=1(n+1)F21(1,−n,2,F(τ))>0.95\\n', 'cite_spans': [], 'section': 'Estimating the size of the first generation from the observed number of symptomatic individuals ::: Biases and estimation during outbreaks', 'ref_spans': []}, {'text': 'This analysis considers the case when the number of observed symptomatic individuals to date is the true number. In practice however, we do not generally observe every symptomatic individual, so the number of observations is only a lower bound on the true number. To address this, rather than considering Iτ as the total number of people who have developed symptoms by time τ, we can define I˜τ as the minimum number of people who have developed symptoms by time τ. We assume that the probability that I˜τ is equal to i˜τ for a given value of iτ is uniform at 1iτ+1. We can then use the same methods as above to infer a distribution for P. Details are provided in Appendix C.', 'cite_spans': [], 'section': 'Estimating the size of the first generation from the observed number of symptomatic individuals ::: Biases and estimation during outbreaks', 'ref_spans': []}, {'text': 'As we can see from Fig. 2\\n, this method can be used to predict the number of infected individuals in the original exposed group. However, we have also demonstrated the importance of caution when interpreting this data. If there is uncertainty surrounding the presentation of symptomatic patients, using I˜τ as a lower bound is a robust method to ensure the size of the generation is not underestimated.', 'cite_spans': [], 'section': 'Estimating the size of the first generation from the observed number of symptomatic individuals ::: Biases and estimation during outbreaks', 'ref_spans': [{'start': 19, 'end': 25, 'mention': 'Fig. 2', 'ref_id': 'FIGREF5'}]}, {'text': 'When designing intervention strategies, we need to consider how adherence may alter their effectiveness. This is important, since highly effective interventions may not be adhered to if they present great individual cost to a population. In this case, a theoretically less effective intervention may perform better, if it has sufficient reduction in individual-level cost. In this section, we illustrate the potential impacts of adherence on the effectiveness of interventions using a toy model.', 'cite_spans': [], 'section': 'Adherence ::: Modelling intervention strategies', 'ref_spans': []}, {'text': 'Consider a standard SIR model, and denote by S(t) and R(t), respectively, the susceptible and recovered/immune fractions of the population at time t. We can write S in terms of R such thatS(t)=S(0)exp(−R0R(t))and let t→∞ to get the final size formula1−R(∞)=S(0)exp(−R0R(∞)),where R(∞) is the fraction of cases at end of outbreak in the absence of behavioural change (Brauer, 2019). This gives a ready reckoner for the eventual attack rate if interventions are not put in place or come in too late to be effective. To illustrate, if we have R0=3 (and S(0)≈1), then R(∞)=0.94. If an intervention is put in place that reduces (with full adherence) R0<1 then the outbreak will be controlled. Indeed, let us assume that R0 is reduced to zero by the intervention: for example, assume that social distancing is perfect and the number of contacts of a fully-adherent individual is zero. If only 50% of people adhere to the intervention then the average number of contacts is effectively reduced by a half and logically R0†=R0/2=1.5 (the † representing quantities post intervention) and R(∞)†=0.58 in this case. However, this assumes that adherence is an independent random process at each contact. This suggests that for each contact an individual would ordinarily make, they ‘‘toss a coin” to decide whether to isolate or not. In reality, individuals are more likely to show polarity, where some individuals reduce all their contacts and follow the measures and a proportion of individuals choose to not adhere to the intervention. If there was distinct polarity in the population such that 50% adhered perfectly and 50% ignored policy, then a toy model can be created with two infectious groups, IA and IB, that behave differently. In this caseS˙=−(RAIA+RBIB)S,\\nI˙A=ϕ(RAIA+RBIB)S−IA,\\nI˙B=(1−ϕ)(RAIA+RBIB)S−IB,where a dot over a variable represents its time derivative. Such an epidemic model, where the two groups have the same susceptibility but different infectivity, has the same final size as an epidemic in a single-type model with the same R0 (e.g. see (Andreasen, 2011)). However, they have different durations as can be seen in Fig. 3\\n, where ϕ=1/2, RB=0 and RA=3. This shows that the assumptions about the nature of adherence predict the same growth rate and final size, but that the more polarised adherence has faster early growth and therefore an earlier peak.', 'cite_spans': [{'start': 367, 'end': 379, 'mention': 'Brauer, 2019', 'ref_id': 'BIBREF4'}, {'start': 2054, 'end': 2069, 'mention': 'Andreasen, 2011', 'ref_id': 'BIBREF33'}], 'section': 'Adherence ::: Modelling intervention strategies', 'ref_spans': [{'start': 2130, 'end': 2136, 'mention': 'Fig. 3', 'ref_id': 'FIGREF6'}]}, {'text': 'More complicated model structures could be constructed by incorporating adherence with intervention by susceptible states, which would lead to core group dynamics (see for example (Keeling & Rohani, 2011)). This issue of independent versus polarised adherence is related to the idea of all-or-nothing versus leaky vaccination (Goldstein et al., 2009; Magpantay, Riolo, Domenech de Cellès, King, & Rohani, 2014), where you either vaccinate a fraction of the population with 100% efficacy or vaccinate 100% of the population with reduced efficacy (House & Keeling, 2011). Note however that vaccination reduces your susceptibility (whether only or also), rather than only your infectivity as in the model discussed above, and variation in susceptibility does reduce the final size, with imperfect coverage with a perfect vaccine (all-or-nothing) leading to a lower final size than full coverage with a leaky vaccine (all individuals having the same mean susceptibility).', 'cite_spans': [{'start': 181, 'end': 203, 'mention': 'Keeling & Rohani, 2011', 'ref_id': 'BIBREF29'}, {'start': 327, 'end': 349, 'mention': 'Goldstein et al., 2009', 'ref_id': 'BIBREF20'}, {'start': 351, 'end': 409, 'mention': 'Magpantay, Riolo, Domenech de Cellès, King, & Rohani, 2014', 'ref_id': 'BIBREF42'}, {'start': 546, 'end': 567, 'mention': 'House & Keeling, 2011', 'ref_id': 'BIBREF26'}], 'section': 'Adherence ::: Modelling intervention strategies', 'ref_spans': []}, {'text': 'The ongoing COVID-19 outbreak is known to have higher mortality rates amongst the elderly, the immunocompromised and those with respiratory and health complications (Guan et al., 2020; Wu et al., 2020; Wu & McGoogan, 2020; Yang et al., 2020; Zhou et al., 2020). In this section, we model the introduction of an infectious disease into care homes, in order to obtain estimates of the final size of the epidemic in the vulnerable population as well as predictions for the number of hospitalisations and fatalities.', 'cite_spans': [{'start': 166, 'end': 183, 'mention': 'Guan et al., 2020', 'ref_id': 'BIBREF23'}, {'start': 185, 'end': 200, 'mention': 'Wu et al., 2020', 'ref_id': 'BIBREF80'}, {'start': 202, 'end': 221, 'mention': 'Wu & McGoogan, 2020', 'ref_id': 'BIBREF81'}, {'start': 223, 'end': 240, 'mention': 'Yang et al., 2020', 'ref_id': 'BIBREF82'}, {'start': 242, 'end': 259, 'mention': 'Zhou et al., 2020', 'ref_id': 'BIBREF83'}], 'section': 'Care home model ::: Modelling intervention strategies', 'ref_spans': []}, {'text': 'Modelling of care homes in the UK is conducted against the backdrop of a wider epidemic in the general population, which we here assume to be following SEIR dynamics with a basic reproduction number R0 that might be different from the within-care home reproduction number RC.', 'cite_spans': [], 'section': 'Care home model ::: Modelling intervention strategies', 'ref_spans': []}, {'text': 'Care homes are assumed to be closed populations, with the infection entering each of them independently with a certain probability. Infection is seeded only once, and within-care home outbreaks then evolve independently from, and do not contribute to, other care home outbreaks and the epidemic in the background population. To keep track of hospitalisations, we model the within-care home infection dynamics using a compartmental model that, in addition to SEIR model, has compartments for mildly symptomatic prodromal cases (P), who show no symptoms but are capable of transmitting the virus, those who recover from the disease after mild symptoms that did not require hospitalisation (M), those who have severe symptoms and are admitted to hospital (H), those who recover after hospitalisation (R), and those that die (D). This is illustrated in Fig. 4\\n.', 'cite_spans': [], 'section': 'Care home model ::: Modelling intervention strategies', 'ref_spans': [{'start': 849, 'end': 855, 'mention': 'Fig. 4', 'ref_id': 'FIGREF7'}]}, {'text': \"The stochastic component of the model, i.e. the random introduction of the infection in care homes, is modelled using the Sellke construction (Andersson & Britton, 2000). Each care home i is given an individual, random threshold of resistance, Qi, which is drawn from an U(0,1) distribution. At time t, we then calculate the infection pressure IP(t) from the background epidemic so that care home i becomes infected at time Ti, where Ti=inf{t|IP(t)>Qi}. The infection pressure up to time t for a median sized care home is the integral from 0 to t of the force-of-infection (FOI) applied to the care home coming from all infectious sources, multiplied by a probability p. This probability represents the probability of the infection being introduced to a median-sized care home. For other care homes, we allow this probability to be proportional to its size, under the assumption that larger care homes employ more staff and are therefore at higher risk of introduction. When the infection pressure becomes higher than an individual care home's resilience threshold, that care home begins its own deterministic infection dynamics with a single initial infected case. The equations describing the background epidemic and the within-care home epidemic are given in Appendix D.\", 'cite_spans': [{'start': 143, 'end': 168, 'mention': 'Andersson & Britton, 2000', 'ref_id': 'BIBREF22'}], 'section': 'Care home model ::: Modelling intervention strategies', 'ref_spans': []}, {'text': 'We run this model on data for the entire care home population in the UK, so that there are approximately 15,000 care homes with a total population of approximately 450,000 residents (Care Quality Commission, 2020). Care home sizes range from 1 to 215, with a mean size of 29.4. In this model we only consider the vulnerable population within care homes. We assume R0=1.5 in the background epidemic, a relatively low value that somehow accounts for a certain degree of control, and an RC=3 to allow relatively explosive epidemics in care homes due to potentially more frail individuals, difficulty in isolation and staff inadvertently passing the infection from one case to the next. The other parameters in the baseline scenario are reported in Table 4\\n. Apart from the reproductive number, the background epidemic uses the same parameters as the care home epidemic. However, in the background model there are only rates from E to I and I to R, which are taken to be the rates from E to P and I to M, respectively.', 'cite_spans': [{'start': 183, 'end': 212, 'mention': 'Care Quality Commission, 2020', 'ref_id': 'BIBREF7'}], 'section': 'Care home model ::: Modelling intervention strategies', 'ref_spans': [{'start': 745, 'end': 752, 'mention': 'Table 4', 'ref_id': 'TABREF3'}]}, {'text': '\\nFig. 5\\na shows number of hospital beds occupied and the cumulative number of deaths for the parameter values chosen and for different values of p. Time is shown in weeks, where week zero represents the peak of the external/background epidemic. Fig. 5b summarises the first, showing the impact of reducing p on the demand for hospital beds and on the final number of deaths. It also shows the impact that changing p has on the timing of the peak.', 'cite_spans': [], 'section': 'Care home model ::: Modelling intervention strategies', 'ref_spans': [{'start': 1, 'end': 7, 'mention': 'Fig. 5', 'ref_id': 'FIGREF8'}, {'start': 245, 'end': 251, 'mention': 'Fig. 5', 'ref_id': 'FIGREF8'}]}, {'text': 'Reducing p corresponds to increasing protection of the vulnerable population in care homes by preventing introduction of infection (e.g. screening, testing and promoting hygiene among staff, etc.), a policy sometimes termed ‘cocooning’ or ‘shielding’. The results show that reducing p from 0.99 to 0.1 corresponds to a reduction of around 22,000 (83%) in the number of deaths and around 131,500 in the number of hospitalisations (83%). Strategies aimed at reducing the probability of introduction into a care home, such as reducing the number of visitors or increased monitoring and protection of care home staff, are therefore predicted to have a large impact on the number of cases in vulnerable care home populations.', 'cite_spans': [], 'section': 'Care home model ::: Modelling intervention strategies', 'ref_spans': []}, {'text': 'There are a variety of assumptions underpinning this model. Firstly, the background epidemic ignores structure and assumes homogeneous mixing. This is likely to make the peak more pronounced, so presents a worst case scenario for the demand on hospital beds. The assumption of R0 for the background epidemic only affects the shape and duration of the background epidemic, since the tuneable parameter p controls the risk of introduction to the care home. That is, if R0 is small, a large p still presents a high force of infection into the care homes. Therefore, for a fixed p, we expect that changing R0 does not affect the total number of deaths, but it changes the peak hospitalisation incidence because a faster and more explosive background epidemic makes epidemics in care homes more synchronised. In fact, when testing the impact of a longer, flatter background epidemic, for example obtained by simulating three slightly desynchronised background SEIR epidemics, results have lower peaks and a much more variable timing (not shown). Assuming each care home is independent might not be realistic, since it is likely that staff are shared between multiple homes, in which case they can act as vectors of transmission between homes. However, in the model current outbreaks are already quite synchronised (the within care home outbreaks occur at similar times), so the effect of this assumption is likely to be minimal. The final major assumption is that the epidemic within the care homes is deterministic. This removes the probability of random extinction and random delays, and should obviously be relaxed with a stochastic model, given half of care homes have size smaller than 25. However, the extinction probability is very low with RC=3, so this stochastic effect is unlikely to have a large impact. Random delays, instead, may change the shape and timing of the epidemic, which could potentially reduce the peak burden. Therefore, this model represents a worst case scenario.', 'cite_spans': [], 'section': 'Care home model ::: Modelling intervention strategies', 'ref_spans': []}, {'text': 'Let Qn,s,e,p,i(t) be the proportion of households in the population at time t of size n, with s susceptibles, e exposed, p prodromal, and i symptomatic infectious individuals. The number of recovered individuals will be n−s−e−p−i. In the absence of household-based interventions, we haveddtQn,s,e,p,i=−(srs→e(t,Q)+ere→p+prp→i+iri→ϕ+n−ηspτp+n−ηsiτi)Qn,s,e,p,i+(s+1)rs→e(t,Q)Qn,s+1,e−1,p,i+(e+1)re→pQn,s,e+1,p−1,i+(p+1)rp→iQn,s,e,p+1,i−1+(i+1)ri→ϕQn,s,e,p,i+1+n−η(s+1)pτpQn,s+1,e−1,p,i+n−η(s+1)iτiQn,s+1,e−1,p,i ,where we take any Q with logically impossible indices just to equal 0, ra→b is the rate from state a to b, and τa is the transmission rate from an individual in state a. Here Q is a vector constructed from some ordering (e.g. lexicographic) of the Qn,s,e,p,i (see the code for details). The transmission into households is given byrs→e(t,Q)=Λ(t)+∑n=1nmax∑s=0n∑e=0(n−s)∑p=0(n−s−e)∑i=0(n−s−e−p)(pβp(t)+iβi(t))Qn,s,e,p,i .Here Λ represents infections imported from outside the population of households, and the other terms represent between-household transmissions. In our code, we assume Λ is a step function. Results are largely insensitive to the precise choice of Λ, but compared to, for example, random seeding of infections in households, starting the whole population susceptible and exposing to a small amount of external infection for a fixed time period has less room for the precise initial condition chosen to influence results, and is more realistic for the situation observed in countries apart from China. We take a ‘global’ intervention as part of the baseline, in particular, we can model phenomena such a school closures that hold during a set of times T asβx(t)={(1−ε)βx(0) if t∈T ,βx(0) otherwise,for x∈{p,i}. We call ε the global reduction. We will generally drop this t-indexing for simplicity, and will also consider only a household isolation strategy (though the other strategies can be considered similarly, with an example of how other strategies could be captured in this model framework given in Appendix E). Instead of isolating for a fixed duration, we assume that a fraction αW of households isolates when there is at least one symptomatic case in the household, and isolating households leave isolation when no symptomatic cases remain. We make this assumption since it may potentially capture the behaviour of real households, who are more likely to remain isolated based on presence of symptoms rather than for a fixed duration. In the non-Markovian household model in Section 3.3.2 we consider a fixed duration of isolation as described in the earlier definition. Isolating households do not experience new infections, meaning that the dynamics becomeddtQn,s,e,p,i=−((1−αW1{i>0})srs→e(t,Q)+ere→p+prp→i+iri→∅+n−ηspτp+n−ηsiτi)Qn,s,e,p,i+(1−αW1{i>0})(s+1)rs→e(t,Q)Qn,s+1,e−1,p,i+(e+1)re→pQn,s,e+1,p−1,i+(p+1)rp→iQn,s,e,p+1,i−1+(i+1)ri→∅Qn,s,e,p,i+1+n−η(s+1)pτpQn,s+1,e−1,p,i+n−η(s+1)iτiQn,s+1,e−1,p,i ,and also do not transmit outside, meaning that the rate of between-household transmission becomesrs→e(t,Q)=Λ(t)+∑n=1nmax∑s=0n∑e=0(n−s)∑p=0(n−s−e)∑i=0(n−s−e−p)(1−αW1{i>0})(pβp+iβi)Qn,s,e,p,i .\\n', 'cite_spans': [], 'section': 'Model ::: Population and household transmission ::: Household isolation modelling ::: Modelling intervention strategies', 'ref_spans': []}, {'text': 'Using the methods in (Black et al., 2017; Ross et al., 2010), it is possible to fit household models of this kind to the overall growth rate, r, which we take to correspond to a doubling time of three days. Natural history parameters can then be set directly based on reasonable estimates: re→p to the inverse of the latent period; rp→i to the inverse of the prodromal period; ri→∅ to the inverse of the symptomatic period. Shaw (Shaw, 2016) analyses various household datasets for respiratory pathogens and estimates values for η close to 1, so this is taken to be 0.8. The remaining degrees of freedom are relative infectiousness of the prodrome (taken as a third) and the probability of transmitting within a pair, which we can take as a typical value given by Shaw (Shaw, 2016). For the numerical results in Fig. 6, Fig. 7\\n, the baseline natural history parameters are chosen to be re→p=1/5, rp→i=1/3, ri→∅=1/4.', 'cite_spans': [{'start': 22, 'end': 40, 'mention': 'Black et al., 2017', 'ref_id': 'BIBREF3'}, {'start': 42, 'end': 59, 'mention': 'Ross et al., 2010', 'ref_id': 'BIBREF62'}, {'start': 430, 'end': 440, 'mention': 'Shaw, 2016', 'ref_id': 'BIBREF67'}, {'start': 770, 'end': 780, 'mention': 'Shaw, 2016', 'ref_id': 'BIBREF67'}], 'section': 'Parameterisation ::: Population and household transmission ::: Household isolation modelling ::: Modelling intervention strategies', 'ref_spans': [{'start': 812, 'end': 818, 'mention': 'Fig. 6', 'ref_id': 'FIGREF9'}, {'start': 820, 'end': 826, 'mention': 'Fig. 7', 'ref_id': 'FIGREF10'}]}, {'text': 'Using the given parameter values for our baseline scenario (Table 5\\n), we consider a combination of household isolation (which follows all-or-nothing adherence) with global reduction in transmission (which follows leaky adherence) for three weeks and show the results in Fig. 6. The distribution of infectious individuals varies with household size, which is shown in Fig. 7 for different durations of global intervention. Applying household isolation at 65% adherence (αW=0.65) manages to reduce the spread of infection, but appears insufficient in this model and with baseline parameters for controlling the outbreak in the long-term, unless other intervention strategies that reduce the global transmission (increasing ε) are adopted at the same time. Alternatively, different levels of adherence can be considered to determine if and when control may be achieved purely through household-based interventions. For the model proposed in the next section, we look into the effectiveness of increasing adherence.', 'cite_spans': [], 'section': 'Summary ::: Population and household transmission ::: Household isolation modelling ::: Modelling intervention strategies', 'ref_spans': [{'start': 271, 'end': 277, 'mention': 'Fig. 6', 'ref_id': 'FIGREF9'}, {'start': 368, 'end': 374, 'mention': 'Fig. 7', 'ref_id': 'FIGREF10'}, {'start': 60, 'end': 67, 'mention': 'Table 5', 'ref_id': 'TABREF4'}]}, {'text': 'We consider independent households of size n=1,2,…,8, for each of which ne stochastic simulations of the within-household epidemic are performed based on the Sellke construction (Andersson & Britton, 2000; Sellke, 1983). Given all infectious contacts outside lead to an actual infection because we are in the early phase of the epidemic and there is no depletion of susceptibles, each case infects, on average, Rg new cases outside. Inside the household, a case would infect on average Rh cases in an infinitely large household, but not all infectious contacts lead to real infections, given local saturation effects: in a household of size n, each infectious individual makes on average Rh/(n−1) infectious contacts with each other specific individual throughout the infectious period, but only the first one will result in an infection, and only if the individual was susceptible at the time of contact.', 'cite_spans': [{'start': 179, 'end': 204, 'mention': 'Andersson & Britton, 2000', 'ref_id': 'BIBREF22'}, {'start': 206, 'end': 218, 'mention': 'Sellke, 1983', 'ref_id': 'BIBREF65'}], 'section': 'Model ::: Non-Markovian models with enhanced within-household transmission ::: Household isolation modelling ::: Modelling intervention strategies', 'ref_spans': []}, {'text': 'Each individual is given an indicator function of whether they are symptomatic or not (individuals show symptoms independently of each other with probability ps) and a resilience threshold. This last quantity is drawn from an exponential distribution with mean 1, and represents the overall infection pressure this individual is able to withstand before they get infected. The infection pressure up to time τ is the integral from 0 to τ of the force-of-infection (FOI) applied to this individual coming from all infectious sources.', 'cite_spans': [], 'section': 'Model ::: Non-Markovian models with enhanced within-household transmission ::: Household isolation modelling ::: Modelling intervention strategies', 'ref_spans': []}, {'text': 'At the beginning of the within-household epidemic, a single initial case is assumed. Time is discretised with a predefined time step dt=0.1 days. At any time step, the current infectivity of all infectives in that time step is summed over, keeping track differently of the infectivity spread outside and inside the household. An overall measure of the accumulated infectivity within the household is updated at each time step and when this crosses the resilience threshold of a susceptible individual, they acquire the infection.', 'cite_spans': [], 'section': 'Model ::: Non-Markovian models with enhanced within-household transmission ::: Household isolation modelling ::: Modelling intervention strategies', 'ref_spans': []}, {'text': 'We assume an individual spends half of their time outside and half inside the household. When self-isolation starts, the assumed adherence ai represents the fraction of the time spent outside that is shifted from outside to within the household. Therefore, for perfect adherence, from the moment symptoms occur, the individual stops transmitting outside but their infectivity within the household grows by 100%. We also explore variations in this compensatory behaviour, so that the time of an individual is split in a more flexible proportion than 1:1. The same argument applies to other control policies, with adherence levels ah for household isolation and aq for household quarantine. When multiple control policies are in place at the same time, their effect is assumed to be multiplicative: if an individual has symptoms and the household isolates, the outside transmission rate from that individual is reduced from baseline by a multiplicative factor (1−ai)×(1−ah) and the within-household transmission rate is the baseline value plus a fraction 1−((1−ai)×(1−ah)) of the baseline value. Therefore, implementing a control policy that reduces transmission outside might lead to more infections in the household (see Fig. 8\\nand the associated accumulated infection pressure in Fig. 10).', 'cite_spans': [], 'section': 'Model ::: Non-Markovian models with enhanced within-household transmission ::: Household isolation modelling ::: Modelling intervention strategies', 'ref_spans': [{'start': 1221, 'end': 1227, 'mention': 'Fig. 8', 'ref_id': 'FIGREF11'}, {'start': 1281, 'end': 1288, 'mention': 'Fig. 10', 'ref_id': 'FIGREF1'}]}, {'text': 'We denote by βgn(τ) the average global infectivity profile of a household of size n, i.e. the time-point average of the rates at which new cases outside are generated by any case infected in all simulated epidemics in a household of size n. During the exponentially growing phase, any global infection starts a new within-household epidemic. Furthermore, larger households are more likely to be infected because they have more members. Therefore, if hn is the probability that a randomly selected household has size n,πn=nhn∑mmhmgives the probability that the household of a randomly selected individual is of size n. This is called the size-biased distribution. The global infectivity profile of the average household infected during the exponentially growing phase is thenβg(τ)=∑nπnβgn(τ)and the area under this curve is known in the literature as the household reproduction number, and is typically denoted by R*. If enough transmission is prevented, so that R*<1, the epidemic is controlled. The basic reproduction number R0 and R* share the same threshold at one, so they are simultaneously larger, equal or smaller than unity. However, in a growing epidemic R0<R* (Ball et al., 2016; Pellis et al., 2012). The real-time growth rate r is related to R* by the Lotka-Euler equation1R*=∫0∞βg(τ)e−rτdτ.\\n', 'cite_spans': [{'start': 1171, 'end': 1188, 'mention': 'Ball et al., 2016', 'ref_id': 'BIBREF1'}, {'start': 1190, 'end': 1209, 'mention': 'Pellis et al., 2012', 'ref_id': 'BIBREF53'}], 'section': 'Model ::: Non-Markovian models with enhanced within-household transmission ::: Household isolation modelling ::: Modelling intervention strategies', 'ref_spans': []}, {'text': 'At baseline we take Rg=2.5 and Rh=2.5, which gives a real-time growth rate r of about 0.245. Each individual, irrespective of whether they will be infected or not, is given (independently of each other) a duration of: incubation period (randomly drawn from a Gamma distribution with mean μE=4.84 and standard deviation σE=2.79 days), prodromal period (dp=1.5 days), and delay from onset of symptoms to when the individual detects the symptoms and enters isolation (dd=0.5 days), and the period of self-isolation (di=7 days). At the end of the incubation period, a symptomatic individual starts showing symptoms, which allow the triggering of control policies (after the delay dd). We assume that two thirds of individuals go on to develop symptoms, with the rest remaining asymptomatic. Asymptomatic individuals do not trigger any policy. After a latent period (defined as the incubation minus the prodromal period) and irrespective of symptoms, any infected case starts an infectious period with an infectivity that changes over time following the probability density function of a Gamma distribution (mean μF=2.2 and standard deviation σF=1.64 days). Asymptomatic cases are assumed half as infectious as symptomatic ones (relative infectivity fa=0.5).', 'cite_spans': [], 'section': 'Parameterisation ::: Non-Markovian models with enhanced within-household transmission ::: Household isolation modelling ::: Modelling intervention strategies', 'ref_spans': []}, {'text': \"Under the baseline parameter values (Table 6\\n), control can in principle be achieved via certain interventions, but only for high levels of adherence, which might be difficult to enforce for a prolonged length of time (Fig. 9\\na). More importantly, the model's conclusions are highly sensitive to variations in parameter choices, which are uncertain. Parameters that present problems here are the delay from symptom onset to isolation (with control failing for 1 day detection delay unless adherence is essentially perfect), proportion of asymptomatic infections (any chance of control lost at 50%) and the strength of asymptomatic transmission. The short delay before symptomatic individuals isolate may be unrealistic unless the susceptible population is very well-informed about symptoms that call for isolation, and so likely does not apply in very early stages of an outbreak. Overall, in the face of the many uncertainties, household-based interventions triggered purely by symptoms appear useful to slow the spread but need to be complemented by other policies (see Table 7).\", 'cite_spans': [], 'section': 'Summary ::: Non-Markovian models with enhanced within-household transmission ::: Household isolation modelling ::: Modelling intervention strategies', 'ref_spans': [{'start': 219, 'end': 225, 'mention': 'Fig. 9', 'ref_id': 'FIGREF12'}, {'start': 37, 'end': 44, 'mention': 'Table 6', 'ref_id': 'TABREF5'}, {'start': 1072, 'end': 1079, 'mention': 'Table 7', 'ref_id': 'TABREF6'}]}, {'text': 'Comparing the different strategies (Fig. 9b), household quarantine can be optimal (as one might expect), but this requires high adherence levels. As adherence drops, this strategy becomes suboptimal to mixed isolation. Mixed isolation is significantly better than household isolation on its own and requires little extra social cost, so should not cause adherence to drop (relative to household isolation adherence levels). The difference between the two strategies comes down to the transmission slipping through after the 14 day household isolation. The cheapest strategy, when considering working age adults, is individual isolation (Fig. 9d), but the effect is limited compared to the other models and cannot achieve control in the baseline scenario even with 100% adherence.', 'cite_spans': [], 'section': 'Summary ::: Non-Markovian models with enhanced within-household transmission ::: Household isolation modelling ::: Modelling intervention strategies', 'ref_spans': [{'start': 36, 'end': 42, 'mention': 'Fig. 9', 'ref_id': 'FIGREF12'}, {'start': 637, 'end': 643, 'mention': 'Fig. 9', 'ref_id': 'FIGREF12'}]}, {'text': 'Overall, the mixed isolation strategy appears to be most cost-effective. However, this is dependent on the assumption that adherence is better for 14 day isolation rather than a very long quarantine. It can be observed that household-based interventions are more effective than individual isolations, demonstrating the importance of these strategies in designing intervention policy. Fig. 8 shows how the different isolation strategies contain the infectious periods of individuals within the household and also indicates the number of individuals being isolated within the household.', 'cite_spans': [], 'section': 'Summary ::: Non-Markovian models with enhanced within-household transmission ::: Household isolation modelling ::: Modelling intervention strategies', 'ref_spans': [{'start': 384, 'end': 390, 'mention': 'Fig. 8', 'ref_id': 'FIGREF11'}]}, {'text': 'To study the impact such an increased within-household transmission has on the chance that a vulnerable individual is infected in the household, we randomly choose one non-primary case in the household as the vulnerable one and count how many of the ne epidemics result in this individual being infected under the different control policies (Fig. 9c). Under these interventions, the risk of a vulnerable individual getting infected within-household, conditional on the infection entering it in the first place, is in the range 5−15%.', 'cite_spans': [], 'section': 'Summary ::: Non-Markovian models with enhanced within-household transmission ::: Household isolation modelling ::: Modelling intervention strategies', 'ref_spans': [{'start': 342, 'end': 348, 'mention': 'Fig. 9', 'ref_id': 'FIGREF12'}]}, {'text': 'Since this model relies on the Sellke construction, we calculate the infection pressure that accumulates (within a household) during the outbreak. In relation to Fig. 8, we report in Fig. 10 the infection pressure that accumulates for the different control policies, showing the different impact each intervention can have on the within household dynamics.', 'cite_spans': [], 'section': 'Summary ::: Non-Markovian models with enhanced within-household transmission ::: Household isolation modelling ::: Modelling intervention strategies', 'ref_spans': [{'start': 162, 'end': 168, 'mention': 'Fig. 8', 'ref_id': 'FIGREF11'}, {'start': 183, 'end': 190, 'mention': 'Fig. 10', 'ref_id': 'FIGREF1'}]}, {'text': 'Social distancing, isolation and lockdowns act to mitigate the spread of an infectious disease and reduce the number of cases. However, such interventions, particularly widespread lockdowns, cannot be maintained indefinitely and must be lifted at some point. For the disease to be controlled, these interventions can be implemented until pharmaceutical interventions are developed, such as a vaccine, or until the case numbers are low enough that the disease may go extinct. Here, we consider the situation where interventions are lifted just before extinction, when the number of cases has reached a low but non-zero initial value n0: at this point, the number of cases might rebound or might go extinct by random chance despite an R0>1. We use a time-inhomogeneous birth-death chain model (Kendallet al., 1948) to investigate the probability of extinction in this context. The n0 ‘‘initial’’ cases give rise to new cases at a time-dependent rate β(t) and recover at rate δ(t). Letting Z(t) denote the random variable that gives the number of cases at time t, we are first interested in obtaining an expression for the probability generating functionQ(t,s)=E[sZ(t)]=∑n=0∞P(Z(t)=n)sn.\\n', 'cite_spans': [{'start': 792, 'end': 811, 'mention': 'Kendallet al., 1948', 'ref_id': 'BIBREF31'}], 'section': 'Extinction probabilities ::: Modelling intervention strategies', 'ref_spans': []}, {'text': 'It follows that Q(t,s) satisfies the differential equation∂Q∂t=(δ(t)−(δ(t)+β(t))s+β(t)s2)∂Q∂s,subject to the initial conditionQ(0,s)=s.\\n', 'cite_spans': [], 'section': 'Extinction probabilities ::: Modelling intervention strategies', 'ref_spans': []}, {'text': 'Solving for Q and setting s=0 gives the probability that, at time t, the number of cases has reached zero and the disease has become extinct. We denote this probability by q(t) (Alexander & Bonhoeffer, 2012), which is given byq(t)=1−(∫0t(β(t1)eI(t1))dt1+eI(t))−1,where I(t)=∫0tδ(t1)−β(t1)dt1.', 'cite_spans': [{'start': 178, 'end': 206, 'mention': 'Alexander & Bonhoeffer, 2012', 'ref_id': 'BIBREF11'}], 'section': 'Extinction probabilities ::: Modelling intervention strategies', 'ref_spans': []}, {'text': 'The above case considers a closed population. Since the virus has spread worldwide, for any population of interest, immigration of infected individuals cannot be ignored. To capture this, we model the case where immigration from external sources is introduced into the system at a rate η(t), and are similarly interested in the random variable Y(t), which denotes the number of cases at time t. The corresponding generating function, R(t,s), for this random variable satisfies∂R∂t=(δ(t)−(δ(t)+β(t))s+β(t)s2)∂R∂s+η(t)(s−1)R.\\n', 'cite_spans': [], 'section': 'Extinction probabilities ::: Modelling intervention strategies', 'ref_spans': []}, {'text': 'Again, solving for R(t,s) and setting s=0 gives the probability, r(t), that there are no cases of infected individuals left, at which time a new case can only arise through immigration from an external source. This probability is given byr(t)=exp(−(1−q(t))∫0tη(t1)dt1)).\\n', 'cite_spans': [], 'section': 'Extinction probabilities ::: Modelling intervention strategies', 'ref_spans': []}, {'text': 'We simulate data based on one initial case n0=1, though this may easily be extended to any number of initial cases. We run simulations both with and without immigration, choosing β(t)=3/(7(1+5e−t)) and δ(t)=1/7 for all t, so that an effective reproduction number given by β(t)/δ(t) grows gradually from 0.5 to 3 after interventions are released, and choosing immigration rate η(t)=W0e−t, where W0 is the initial (constant) rate of importation of cases before any controls on immigration are put into effect. We set W0=5 imported cases per day. With these choices of parameters, the resulting extinction probabilities are given in Fig. 11\\n. Note that we are assuming the immigration rate is decreasing to 0, so if the infection is controlled internally for long enough, an overall ultimate extinction is possible in this model. For these parameter choices, the final probability of extinction, defined as limt→∞q(t) (without immigration) and limt→∞r(t) (with immigration) are approximately 0.446 and 0.002, respectively. It should be noted that q(t) concerns the best case scenario with only one initial case. Increasing the number of initial cases n0 scales the probability of extinction by q(t)n0. These probabilities suggest that, without widespread immunity, stochastic extinction might be aided by social distancing but is heavily compromised by immigration. Border controls, therefore, if of limited use when transmission is self-sustaining, become key when the number of cases is low. Note that we have assumed an importation function η(t) that goes to 0 for large t, in line with a pandemic that goes extinct in other geographical regions. However, the presence of an animal reservoir might lead to an importation function that is non-zero over longer time scales, thus effectively making ultimate extinction impossible unless the effective reproduction number is kept below one by a systematic and permanent intervention (e.g. technology-based change in behaviour) or herd immunity.', 'cite_spans': [], 'section': 'Extinction probabilities ::: Modelling intervention strategies', 'ref_spans': [{'start': 630, 'end': 637, 'mention': 'Fig. 11', 'ref_id': 'FIGREF2'}]}, {'text': 'When choosing contact tracing strategies, a balance must be struck between the effectiveness of a strategy and the resources that it requires. Some strategies are only feasible when there are few infections, since the resources required can grow rapidly depending on the dynamics of the outbreak and the contact tracing process.', 'cite_spans': [], 'section': 'Hitting times of contact tracing capacities ::: Contact tracing and household isolation ::: Modelling intervention strategies', 'ref_spans': []}, {'text': 'To define the capacity of the contact tracing process, we consider the ability of a public health agency to observe the condition of those asked to self-isolate, due to their recent exposure to an infected individual. The health agency must remain in contact for the duration of the 14 day self-isolation period, so that if any individual under isolation develops symptoms and then tests positive, the contact tracing process can be initiated on this node. We will define the capacity of the contact tracing process to be the number of people that can be placed under observation and assume two possible capacities: 800 and 8000. We assume that when a node is contact traced, they are asked to report their global contacts for the last 14 days. All global contacts are assumed to be to a new person since we are in the early stages of an outbreak. Parameters are given in Table 8\\n.', 'cite_spans': [], 'section': 'Hitting times of contact tracing capacities ::: Contact tracing and household isolation ::: Modelling intervention strategies', 'ref_spans': [{'start': 872, 'end': 879, 'mention': 'Table 8', 'ref_id': 'TABREF7'}]}, {'text': 'We carried out 6507 simulations of the contact tracing process for 150 days. Contact tracing capacity was reached in 5000 simulations, and in 180 the epidemic neither went extinct nor was the 8000 capacity reached. In the remaining simulations, the epidemic went extinct. Fig. 12\\na and Table 8 show that increasing the contact tracing capacity tenfold less than doubles the time until that capacity is reached. However, it does increase the odds of driving the epidemic to extinction without hitting the capacity by about 10% (Table 8). Different contact tracing strategies will strain different aspects of the health agency. A strategy that generates large amounts of work is only feasible if there are few active infections. The optimal strategy will need to compromise and may need to change depending on the number of active infections, which cannot be directly observed.', 'cite_spans': [], 'section': 'Hitting times of contact tracing capacities ::: Contact tracing and household isolation ::: Modelling intervention strategies', 'ref_spans': [{'start': 272, 'end': 279, 'mention': 'Fig. 12', 'ref_id': 'FIGREF3'}, {'start': 286, 'end': 293, 'mention': 'Table 8', 'ref_id': 'TABREF7'}, {'start': 527, 'end': 534, 'mention': 'Table 8', 'ref_id': 'TABREF7'}]}, {'text': 'When there is a small number of cases in a single country, it may be possible to drive the pathogen to extinction. This small case number could correspond to the start of an outbreak or removing of severe interventions. We consider the latter case, but conservatively assume a fully susceptible population.', 'cite_spans': [], 'section': 'Extinction time ::: Contact tracing and household isolation ::: Modelling intervention strategies', 'ref_spans': []}, {'text': 'We assume that social distancing is enforced on day 0 and reduces global contacts by 70%. Full parameters are given in Table 8. Since we are interested in extinction, we will no longer consider the contact tracing capacity. Under these baseline parameter assumptions and 10,000 simulations, the combined force of this contact tracing strategy and isolation is enough to drive the epidemic extinct (Fig. 13b), but measures will need to be in place for months in some cases. If the infection is ever re-imported, then the process would begin again, since herd immunity is not achieved. Note that the minimum extinction time is 21 days due to this being the time after which an infected individual is labelled recovered.', 'cite_spans': [], 'section': 'Extinction time ::: Contact tracing and household isolation ::: Modelling intervention strategies', 'ref_spans': [{'start': 398, 'end': 405, 'mention': 'Fig. 13', 'ref_id': 'FIGREF4'}, {'start': 119, 'end': 126, 'mention': 'Table 8', 'ref_id': 'TABREF7'}]}, {'text': 'Additionally, this model only considers extinction under the assumption that no cases are imported. In Section 3.4, we have shown that importation of cases significantly reduces the extinction probability. This suggests that extinction may no longer be guaranteed, and the time to extinction will be significantly increased. This analysis has focused on a single contact tracing strategy using indicative parameters for COVID-19. The proposed model can be extended to more strategies and region specific parameters to inform the design of control policies. Also, as is shown in Table 8, contact tracing capacity is likely to be reached, which may prevent extinction from being achieved. This complication is compounded by the issues of loss of immunity or the presence of an animal reservoir discussed in Section 3.4.', 'cite_spans': [], 'section': 'Extinction time ::: Contact tracing and household isolation ::: Modelling intervention strategies', 'ref_spans': [{'start': 578, 'end': 585, 'mention': 'Table 8', 'ref_id': 'TABREF7'}]}, {'text': 'In this manuscript we have presented a range of mathematical tools to tackle infectious disease outbreaks. In particular, these tools address various technical questions posed by the authors to support the ongoing public health response to COVID-19. This toolkit considers both estimation efforts for key parameters, and investigative efforts (often numerical simulations) in gauging the effectiveness of various intervention or control measures. Joint consideration of estimation and simulation efforts is critical. Parameter estimates are obtained using a certain set of assumptions regarding the data, and investigations or simulations utilising these estimates should ensure that their underlying assumptions are consistent. These challenges in model construction and applicability of statistical methods are compounded by the limitations of the data with which decisions must be made. Some of the biases present in the data can be addressed with an improved data collection methodology - often challenging in the context of a fast-moving outbreak - but many are also inherent to the nature of early outbreak data (Britton & Scalia Tomba, 2019; Lesko, Keil, & Edwards, Edwards). The consequent lack of intuitive insight from this data underscores the need for careful parametric estimates, especially considering the large variability in predicted outcomes resulting from small differences in parameters. Even with robust estimates for some parameters, many other parameters are challenging to estimate using the available data. Therefore, models need to address this variability and uncertainty in order to inform public health policy.', 'cite_spans': [{'start': 1119, 'end': 1147, 'mention': 'Britton & Scalia Tomba, 2019', 'ref_id': 'BIBREF5'}, {'start': 1149, 'end': 1172, 'mention': 'Lesko, Keil, & Edwards,', 'ref_id': 'BIBREF38'}], 'section': 'Discussion', 'ref_spans': []}, {'text': 'We have presented methods to address biases arising from a growing force of infection, changes in the reporting rate, truncated data samples and a varying travel rate. We use these methods to account for these biases when estimating delay distributions, such as the incubation period, and the growth rate/doubling time. These biases can have significant impact when estimating key parameters: the mean incubation period estimates for COVID-19 range from 3.48 days without correcting for truncation to 4.84 days with the correction, and the doubling time in Hubei province decreases from 3.15 days without correcting for travel to 2.77 days. These differences can significantly alter our understanding of the outbreak, and could have a large impact on policy and public health. For instance, underestimating the incubation period may lead to quarantine strategies failing to identify infected individuals if the quarantine length is too short. Overestimating the doubling time (or underestimating the growth rate) will underestimate the risk posed to the host population - both in terms of final size of the epidemic and the rate at which it spreads, which can have significant public health impacts as discussed in (Pellis, Cauchemez, Ferguson and Fraser, 2020).', 'cite_spans': [{'start': 1216, 'end': 1260, 'mention': 'Pellis, Cauchemez, Ferguson and Fraser, 2020', 'ref_id': 'BIBREF57'}], 'section': 'Discussion', 'ref_spans': []}, {'text': 'It is important to note that the above-mentioned biases, and consequent impact of implementing the methods correcting for their presence, may vary across different settings. As an example, the potential underestimation of the COVID-19 growth rate is exacerbated by an overlap in early outbreaks with a period of significant travel and movement in China, and would be less detrimental if first observed in other populations such as Italy. Also, for the incubation period, we have shown two different types of data; one from Wuhan and one from discrete infection events. In the Wuhan data set, truncation and force of infection biases are very important, whereas in the other data set, there is no force of infection bias since the infection events are observed.', 'cite_spans': [], 'section': 'Discussion', 'ref_spans': []}, {'text': 'When an outbreak occurs in an enclosed group, such as a large gathering, we may wish to know how many individuals are likely to be infected. We developed a statistical method to estimate the first generation size based on the number of symptomatic individuals, taking care to account for the uncertainty in this quantity. This ready reckoner can inform testing of large groups to help control the disease spread, but does not apply to later generations or the possible interventions enacted on the population.', 'cite_spans': [], 'section': 'Discussion', 'ref_spans': []}, {'text': 'Building on these enclosed population scenarios, we have developed a set of models that investigate public control measures or interventions on enclosed populations, such as households and care homes. These structured descriptions improve the population risk profiles relative to assumptions of homogeneous mixing. A complementary aspect to a structured population when modelling interventions is adherence. Motivated by vaccination modelling, we consider leaky adherence, where every household chooses to adhere or not whenever an event occurs, and all-or-nothing adherence, where some households adhere every time and some never adhere. We observed that in a homogeneous population, although the two types of adherence predict the same growth rate and final size, the timing of the peak and the early growth can be faster under all-or-nothing adherence. This insight, combined with lessons from the vaccination literature, suggests that efforts should focus on ensuring complete adherence in individuals or households with some level of pre-existing adherence, rather than pushing non-adherent individuals or households to change behaviour.', 'cite_spans': [], 'section': 'Discussion', 'ref_spans': []}, {'text': 'Dedicated modelling of disease spread in care homes is essential due to the documented history of co-morbidity of their residents during pandemics (Guan et al., 2020), (Guan et al., 2020; Wilder-Smith et al., 2020; Wu et al., 2020; Wu & McGoogan, 2020; Yang et al., 2020; Zhou et al., 2020). We do so by regarding care homes as closed populations that are subjected to a force of infection from an external epidemic. We develop a tool for analysing the risk posed to this population by determining the peak size of the epidemic within the care homes and the number of deaths. Applying this model to COVID-19, we find that by ‘‘cocooning” the care homes, i.e. shielding them to reduce the chance of introduction from the external outbreak, we can significantly reduce the size of the peak and therefore reduce the number of deaths. However, assessing the necessary level of shielding requires accurate characterisation of the external force of infection, and underestimating this may invalidate shielding efforts. A limitation to the proposed model is the deterministic within care home epidemic. However, since the average size of care homes is relatively large and we assume a high R0 within care homes, the deterministic assumption is unlikely to significantly alter the conclusions.', 'cite_spans': [{'start': 148, 'end': 165, 'mention': 'Guan et al., 2020', 'ref_id': 'BIBREF23'}, {'start': 169, 'end': 186, 'mention': 'Guan et al., 2020', 'ref_id': 'BIBREF23'}, {'start': 188, 'end': 213, 'mention': 'Wilder-Smith et al., 2020', 'ref_id': 'BIBREF78'}, {'start': 215, 'end': 230, 'mention': 'Wu et al., 2020', 'ref_id': 'BIBREF80'}, {'start': 232, 'end': 251, 'mention': 'Wu & McGoogan, 2020', 'ref_id': 'BIBREF81'}, {'start': 253, 'end': 270, 'mention': 'Yang et al., 2020', 'ref_id': 'BIBREF82'}, {'start': 272, 'end': 289, 'mention': 'Zhou et al., 2020', 'ref_id': 'BIBREF83'}], 'section': 'Discussion', 'ref_spans': []}, {'text': 'When modelling households however, we are concerned with much smaller population sizes. Therefore, it is important to consider stochastic effects within each household, combined with between-household dynamics. We consider two different household models: one which contains features of both within- and between-household transmission, where small-scale transmission can be linked to the epidemic on a population level, and another which facilitates more detail in the within-household transmission and delay distributions, but with reduced correspondence to the population-wide transmission. With the first model, a 65% adherence to household isolation appears insufficient to control the epidemic without severe global reductions in transmission. Coupled with a short term global reduction, the epidemic can be controlled, but upon lifting the global intervention, which could take the form of a lockdown, household isolation is insufficient to maintain control. For the second model, we look into changing the strength of adherence, and the impact this can have on achieving control. Indicative but reasonable parameter values suggest that the COVID-19 outbreak can potentially be controlled using household isolation strategies, provided the level of adherence is sufficiently high. However, such a high level of adherence may be difficult to maintain in the long-term and this modicum of control is anyway highly sensitive to the chosen parameters. We further investigated the efficacy of various isolation or quarantine measures. A policy of individual isolation struggles to curtail the epidemic for any adherence. Instead, mixed isolation, whereby first the whole household isolates and any individual infected during isolation goes on to self isolate after household isolation is lifted, appears to be the most cost-effective strategy.', 'cite_spans': [], 'section': 'Discussion', 'ref_spans': []}, {'text': 'Countries have put into place strict social distancing and lockdown intervention to suppress or regain control of epidemics that threaten to overwhelm the health system and cause massive mortality, but they cannot be sustained in the long term without growing social and economic costs. We have shown however, that the probability of the epidemic becoming extinct once these policies are lifted, even when very few cases remain, is very small. We therefore consider a contact tracing intervention as a potential strategy for managing the COVID-19 outbreak, once severe lockdown interventions are lifted. We developed a household-level contact tracing model to explore the feasibility of combining these strategies to control the epidemic. Firstly, we noted that by using knowledge of household structure, we can reduce the burden on the contact tracing process by isolating household and removing them from the contact tracing process once an infected member has been identified. Secondly, we investigated how contact tracing combined with household isolation may drive the disease to extinction, finding that aggressive contact tracing coupled with household isolation can drive the epidemic to extinction under the indicative parameters assumed, when starting with a single infection. However, the time until extinction can be impractically long, which risks the contact tracing capacity being overwhelmed, suggesting such a strategy may be infeasible in practice. A less aggressive strategy could be implemented that would be less likely to overwhelm local health agencies. Whilst it may not lead to extinction, this can still be beneficial at mitigating and controlling the spread of an outbreak as part of a test, trace and isolate strategy.', 'cite_spans': [], 'section': 'Discussion', 'ref_spans': []}, {'text': 'There are many complexities when modelling an outbreak of a novel infectious disease. To address some of these, we have described a variety of techniques to serve as part of a generally applicable toolkit. However, our proposed models, and many other models, are subject to important limitations which must be considered prior to their application. Key among these is the lack of heterogeneous population mixing, such as through age-stratification (Pellis et al., 2020a) and different risk-groups (Valdano, Poletto, Boelle, & Colizza, 2019, pp. 1–18), and spatio-temporal variations (Lau et al., 2017), all of which influence modelling estimates and predictions. Nevertheless, the relative simplicity of the presented models allows for the development of qualitative intuition regarding the efficacy of various intervention methods, whilst providing tractable theoretical frameworks which can be further developed and better inform policy-makers.', 'cite_spans': [{'start': 449, 'end': 469, 'mention': 'Pellis et al., 2020a', 'ref_id': 'BIBREF54'}, {'start': 498, 'end': 539, 'mention': 'Valdano, Poletto, Boelle, & Colizza, 2019', 'ref_id': 'BIBREF73'}, {'start': 584, 'end': 600, 'mention': 'Lau et al., 2017', 'ref_id': 'BIBREF36'}], 'section': 'Discussion', 'ref_spans': []}, {'text': 'The authors declare no competing interests.', 'cite_spans': [], 'section': 'Declaration of competing interest', 'ref_spans': []}], 'paragraphs': [{'text': 'During an outbreak, many parameters depend on delay distributions (the length of time between two events), such as the time from infection to symptom onset (the incubation period). If an individual can be followed indefinitely, it is easy to determine the length of these events. However, in reality only events that occur before a given date are observed. Therefore, the data is subject to censoring and truncation issues. In the incubation period, for example, censoring comes into play since, if we have observed an infection but the individual has not yet developed symptoms, we only have a lower bound on how long it will take them to develop symptoms. To account for this, we can instead condition on observing symptom onset before the cut-off date. However, this leads to a truncation issue, since individuals who were infected close to the cut-off date will only be observed if they have a short incubation period, which leads to an overexpression of short delays.', 'id': '00010'}], 'title': 'Using statistics and mathematical modelling to understand infectious disease outbreaks: COVID-19 as an example', 'sha': 'b9704e32040a1020ad8701ee03df56825d2e5f77', 'doi': '10.1016/j.idm.2020.06.008', 'score': 9.542799949645996}, {'rank': 73, 'abstract': 'During an infectious disease outbreak, biases in the data and complexities of the underlying dynamics pose significant challenges in mathematically modelling the outbreak and designing policy. Motivated by the ongoing response to COVID-19, we provide a toolkit of statistical and mathematical models beyond the simple SIR-type differential equation models for analysing the early stages of an outbreak and assessing interventions. In particular, we focus on parameter estimation in the presence of known biases in the data, and the effect of non-pharmaceutical interventions in enclosed subpopulations, such as households and care homes. We illustrate these methods by applying them to the COVID-19 pandemic.', 'body_text': [{'text': \"Mathematical epidemiology is a well-developed field. Since the pioneering work of Ross in malaria modelling [73] and Kermack and McKendrick's general epidemic models [42] , there has been gathering interest in using math-ematical tools to investigate infectious diseases. The allure is clear, since mathematical models can provide powerful insight into how these complex systems behave, which in turn can enable these problems to be better controlled/prevented. Not only is the power of the mathematical tools increasing, but the availability of data on infectious diseases, whether this be a rapid release of data during an outbreak or detailed collection of data for endemic pathogens, is increasing. Rapid interpretation of epidemiological data is critical for the development of effective containment, suppression and mitigation interventions, but there are many difficulties to interpreting case data in real-time.\", 'cite_spans': [{'start': 108, 'end': 112, 'text': '[73]', 'ref_id': 'BIBREF73'}, {'start': 117, 'end': 128, 'text': 'Kermack and', 'ref_id': None}, {'start': 166, 'end': 170, 'text': '[42]', 'ref_id': 'BIBREF41'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'These include interpreting symptom progression and fatality ratios with delay distributions and right-censoring, exacerbated by exponential growth in cases leading to the majority of case data being on recently infected individuals; lack of clarity and consistency in denominators; inconsistency of case definitions over time and the eventual impact of interventions and changes to behaviour on transmission dynamics. Mathematical and statistical techniques can help overcome some of these challenges to interpretation, aiding in the development of intervention strategies and management of care. Examining key epidemiological quantities alongside each other in a transmission model can provide quantitative insights into the outbreak, testing the potential impact of intervention strategies and predicting the risk posed to the human (or animal) host population and healthcare preparedness.', 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'Mathematical modelling has been used as part of the planning process during outbreak response by governments worldwide for many recent outbreaks. For example the UK Department of Health has a long established committee Scientific Pandemic Influenza group on Modelling, or SPI-M to advise on new and emerging respiratory infections [23] . One of the largest instances of such an outbreak in recent history was the 2009 H1N1 pandemic.', 'cite_spans': [{'start': 331, 'end': 335, 'text': '[23]', 'ref_id': 'BIBREF22'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'The World Health Organisation developed a network of modelling groups and public health experts to work on exploring various characteristic of the outbreak [14, 84] . These ranged from characterising the dynamics of the outbreak to investigating the effectiveness of different intervention strategies. This integration of mathematics into policy design indicates the important insights that modelling and statistics can provide. This paper is a collection of work-streams addressing various technical questions faced by the group as part of the ongoing response to COVID-19, and as such is written to be reflective of the experience we have gone and are currently going through. Therefore, to aid the reader each section includes results and a short discussion.', 'cite_spans': [{'start': 156, 'end': 160, 'text': '[14,', 'ref_id': 'BIBREF13'}, {'start': 161, 'end': 164, 'text': '84]', 'ref_id': 'BIBREF84'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'Many of the questions and techniques presented here can be further developed as the availability of data and research interests evolves, but are compiled into this manuscript as an overview of methodology and scientific approaches beyond the standard SIR textbook model that benefit the ongoing efforts in tackling this and other outbreaks.', 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction'}, {'text': \"First documented in December 2019, an outbreak of community-acquired pneumonia began in Wuhan, Hubei Province, China. In January, this outbreak was attributed to a novel coronavirus, SARS-CoV2. The initial spread of the pathogen in Wuhan was fast, and after a period of case-finding and contact tracing, China moved to implement a 'shutdown' of Wuhan on January 23, and other cities in China the following days, to try to suppress the growth of the epidemic. These measures may have succeeded at slowing down the rate at which cases have been seeded elsewhere, but in many countries initial importation of cases and transmission has not been contained. Countries around the world are now seeing outbreaks that are overwhelming, or have the potential to overwhelm, healthcare systems and cause a high number of deaths even in high-income countries [70] .\", 'cite_spans': [{'start': 847, 'end': 851, 'text': '[70]', 'ref_id': 'BIBREF70'}], 'ref_spans': [], 'section': 'COVID-19 pandemic background'}, {'text': 'While the majority of documented symptomatic cases are mild, characterised in many reports by persistent cough and fever, a significant proportion of these individuals go on to develop pneumonia, with some then developing acute respiratory failure and a small proportion of overall cases becoming fatal. Severity of symptoms has been observed to increase with age and with the presence of underlying health conditions such as diabetes [25] and cardiac conditions, with some evidence that severity of symptoms might depend on gender and ethnicity [34, 71, 88, 89, 91] .', 'cite_spans': [{'start': 435, 'end': 439, 'text': '[25]', 'ref_id': 'BIBREF24'}, {'start': 546, 'end': 550, 'text': '[34,', 'ref_id': 'BIBREF33'}, {'start': 551, 'end': 554, 'text': '71,', 'ref_id': 'BIBREF71'}, {'start': 555, 'end': 558, 'text': '88,', 'ref_id': 'BIBREF88'}, {'start': 559, 'end': 562, 'text': '89,', 'ref_id': 'BIBREF89'}, {'start': 563, 'end': 566, 'text': '91]', 'ref_id': 'BIBREF91'}], 'ref_spans': [], 'section': 'COVID-19 pandemic background'}, {'text': 'SARS-CoV2 has a fast doubling time (the time it takes for the number of cases in the region to double, estimated at approximately 3 days [68] ) and, potentially, a very large R 0 (the average number of infections caused by each infected individual, with estimates ranging from 1.4 to 6.47 [53, 55, 56, 64] ). It is possible that there is a significant degree of asymptomatic and/or pre-symptomatic transmission [49, 57, 61] , though without robust serosurveys, this is difficult to quantify with certainty. These characteristics result in the pathogen being able to spread widely, rapidly and undetected, presenting a significant risk to public health.', 'cite_spans': [{'start': 137, 'end': 141, 'text': '[68]', 'ref_id': None}, {'start': 289, 'end': 293, 'text': '[53,', 'ref_id': 'BIBREF53'}, {'start': 294, 'end': 297, 'text': '55,', 'ref_id': 'BIBREF55'}, {'start': 298, 'end': 301, 'text': '56,', 'ref_id': 'BIBREF56'}, {'start': 302, 'end': 305, 'text': '64]', 'ref_id': 'BIBREF64'}, {'start': 411, 'end': 415, 'text': '[49,', 'ref_id': 'BIBREF49'}, {'start': 416, 'end': 419, 'text': '57,', 'ref_id': 'BIBREF57'}, {'start': 420, 'end': 423, 'text': '61]', 'ref_id': None}], 'ref_spans': [], 'section': 'COVID-19 pandemic background'}, {'text': 'Typically, the aim of an intervention strategy would be to push and keep the reproduction number R t , defined as the average number of cases generated by a typical infective at time t, below 1. At this point each infected individual subsequently infects, on average, less than one individual, such that the number of cases should decline. The basic reproduction number, R 0 , represents the initial value of R t , before any intervention is put in place.', 'cite_spans': [], 'ref_spans': [], 'section': 'COVID-19 pandemic background'}, {'text': 'High R 0 , fast growth, and possible pre-or asymptomatic infection make the design of potential interventions, and the modelling that would inform them, particularly challenging. Large values of R 0 mean a substantial amount of transmission needs to be halted; fast growth causes the number of cases in the absence of interventions to rise rapidly, so that the time scale of interventions to reduce R 0 must also be fast in order to effect substantive early changes on a population level; finally, the resulting interventions must encompass possible pre-and asymptomatic cases, a challenging prospect when in many instances these individuals are indistinguishable from healthy individuals. Consequently, we must consider the possibility of interventions that are massively disruptive to society and may have to be sustained for a long period of time in order to cause the number of infections to decline towards zero [27] . If infections remain, and the susceptible proportion of the population remains above the herd immunity threshold, these interventions must be upheld to prevent a second wave of the epidemic. There is not yet conclusive evidence as to the degree and duration of immunity conferred by infection with SARS-CoV2 nor the feasibility of a vaccine, the timeline for which is unlikely to be any time shorter than 18 months away at the time of writing [38] . Therefore, short term extreme interventions are not as effective as they might be in other circumstances, since after their removal there remains a long period of time in which cases can rise again. The longer these significantly suppressive and disruptive interventions are in effect, the more severe the effect on the economy, and broader societal health and well-being. Furthermore, adherence to interventions will likely vary with their duration and severity.', 'cite_spans': [{'start': 917, 'end': 921, 'text': '[27]', 'ref_id': 'BIBREF26'}, {'start': 1367, 'end': 1371, 'text': '[38]', 'ref_id': 'BIBREF37'}], 'ref_spans': [], 'section': 'COVID-19 pandemic background'}, {'text': 'We are further challenged by the lack of transferable intuition. Early work looked at intuition gained from SARS and MERS outbreaks, also caused by coronaviruses. Some parameters do appear to be similar to these pathogens, such as the average length of the incubation period [46, 85, 86] . However, there are also clear differences, with both SARS and MERS being more fatal, but seemingly less efficient at spreading since they did not seed major global pandemics. Another complication is the spread of the infection during the Chinese Spring Festival, a time period during which movement, social, and contact patterns vary significantly. This presents significant challenges as experience and intuition from other studies regarding population mixing and spatial patterns must either be modified or are invalid. Furthermore, the pandemic has received a proportionately larger level of public attention than e.g. the 2009 H1N1 pandemic [20, 74] , largely boosted by social media. This greater level of public awareness, and the successive, staggered interventions placed to prevent disease spread are responsible for significant variations in behaviour [17, 30] and adherence to public guidance both in China and abroad.', 'cite_spans': [{'start': 275, 'end': 279, 'text': '[46,', 'ref_id': 'BIBREF46'}, {'start': 280, 'end': 283, 'text': '85,', 'ref_id': 'BIBREF85'}, {'start': 284, 'end': 287, 'text': '86]', 'ref_id': 'BIBREF86'}, {'start': 935, 'end': 939, 'text': '[20,', 'ref_id': 'BIBREF19'}, {'start': 940, 'end': 943, 'text': '74]', 'ref_id': 'BIBREF74'}, {'start': 1152, 'end': 1156, 'text': '[17,', 'ref_id': 'BIBREF16'}, {'start': 1157, 'end': 1160, 'text': '30]', 'ref_id': 'BIBREF29'}], 'ref_spans': [], 'section': 'COVID-19 pandemic background'}, {'text': 'The structure of this paper follows two main themes. In Section 2, we discuss various biases that are present in outbreak data and techniques for estimating epidemiological parameters. Accounting for biases and producing robust parameter estimates is important throughout the duration of an epidemic, both for increasing our understanding of the underlying dynamics, and for feeding into models. Firstly, we discuss a bias-corrected method for estimating the incubation period, which can also be applied to serial intervals, onset-to-death time, and other delay distributions. We then present a method for estimating the true growth rate of the epidemic, accounting for the bias encountered since infected individuals may be exported from the region. Our next method is a tool for estimating the expected size of the next generation of infectives based on the rate of observed cases. This tool provides insight into the size of small outbreaks, which can inform decision making when trying to prevent a major outbreak taking off. We end this section with a selection of bias corrected estimates of: incubation period, doubling time, and onset-to-hospitalisation.', 'cite_spans': [], 'ref_spans': [], 'section': 'COVID-19 pandemic background'}, {'text': 'In Section 3, we propose a variety of mathematical models looking at disease impact and intervention strategies, with particular focus on non-pharmaceutical interventions due to the current lack of widely deployable, targeted pharmaceutical treatments. These models focus on enclosed populations, since this is the level at which most interventions are implemented. Since the disease is particularly fatal in the elderly and other at risk groups, we develop a care home model to investigate how the pathogen may spread through care homes. We also develop household models to investigate the impact of different intervention/control strategies. These models can inform policy design for mitigating or controlling epidemic spread. Finally, in the context of relaxing strong social distancing policies, we investigate the extinction probability of the pathogen. We first consider the extinction probability after lifting restrictions. We then develop a household-based contact tracing model, with which we investigate the extinction probability under weaker isolation policies paired with contact tracing, thus shedding light on possible combination of interventions that allow to feasibly manage the infection while minimising the social impact of control policies.', 'cite_spans': [], 'ref_spans': [], 'section': 'COVID-19 pandemic background'}, {'text': 'Techniques are constantly developing that enable higher volumes of more accurate data to be collected real-time during an epidemic. These data present a large opportunity for analysis to gain insight into the pathogen and the dynamics of the outbreak. However, although the quality of the data is constantly increasing, there are still many biases present. Some of these are due to the data collection methods, and in an ideal world we would be able to eliminate them, and some are simply due to the nature of the outbreak, and will be present regardless of data collection methods.', 'cite_spans': [], 'ref_spans': [], 'section': 'Potential biases in the outbreak data'}, {'text': 'During an outbreak, many parameters depend on delay distributions (the length of time between two events), such as the time from infection to symptoms onset (the incubation period). If an individual can be followed indefinitely, it is easy to determine the length of these events. However, in reality only events that occur before a given date are observed. Therefore, the data is subject to censoring and truncation issues. In the incubation period, for example, censoring comes into play since, if we have observed an infection but the individual has not yet developed symptoms, we only have a lower bound on how long it will take them to develop symptoms. To account for this, we can instead condition on observing symptom onset before the cut-off date. However, this leads to a truncation issue, since individuals who were infected close to the cut-off date will only be observed if they have a short incubation period, which leads to an overexpression of short delays.', 'cite_spans': [], 'ref_spans': [], 'section': 'Potential biases in the outbreak data'}, {'text': 'The number of cases tends to grow exponentially during the early stages of an outbreak, causing the force of infection and the number of reported cases to increase with time. This further complicates the truncation issue since not only are recent cases truncated but they also account for the majority of cases. The growing force of infection also needs to be accounted for, since if the potential time of infection is interval-censored rather than observed directly, the probability that the case was infected in each day of that interval is not constant.', 'cite_spans': [], 'ref_spans': [], 'section': 'Potential biases in the outbreak data'}, {'text': 'In theory, both of these biases are relatively straightforward to account for. In practice however, there are other biases in the data. One of the major biases is the reporting rate. Although the total number of cases may be reasonably described as growing exponentially with a constant rate in the early stages of an outbreak, high-resolution data may exhibit more complex behaviour. This can be due to a variety of reasons, such as the workload becoming overwhelming, the availability of individual-level data decreasing, the laboratories or offices slowing down activity over the weekend, the case definition changing, the testing capabilities increasing, and so on.', 'cite_spans': [], 'ref_spans': [], 'section': 'Potential biases in the outbreak data'}, {'text': 'Another uncertainty arises since generally only the date of each event is recorded rather than the time. This presents a large window of uncertainty in the length of the delay, since the time of each event can vary up to 24 hours, and for a delay distribution, which depend on two events, it could vary by up to 48 hours.', 'cite_spans': [], 'ref_spans': [], 'section': 'Potential biases in the outbreak data'}, {'text': 'Travel rate is another bias present in the data. For example, this changes the density of observed cases in a region, which can change the apparent growth rate. Intervention strategies present a further bias because this can change the growth rate of the epidemic and the reporting rate. Additionally, estimates of certain parameters may vary depending on the interventions that are implemented, so these need to be considered carefully.', 'cite_spans': [], 'ref_spans': [], 'section': 'Potential biases in the outbreak data'}, {'text': 'To model the incubation period, we require information regarding when an individual was infected and when they expressed symptoms. Observing exact time of infection is unlikely, but it can be possible to find potential exposure windows. We consider three different data sets. The first two consist of individuals who travelled from Wuhan before expressing symptoms. We can assume these individuals were infected in Wuhan, since at the time of this data, the force of infection was significantly higher in Wuhan than elsewhere. The length of time spent in Wuhan therefore provides a window during which each individual became infected, and for many of these individuals we also have the date of symptom onset. In the early stages, the growth rate in reported cases was constant, and dependent on the epidemic growth rate in Wuhan and the rate at which people left Wuhan.', 'cite_spans': [], 'ref_spans': [], 'section': 'Incubation period'}, {'text': 'By using travel to estimate the true number of cases, we estimate the exponential growth rate in Wuhan as r = 0.25 (see Section 2.3). Therefore, the force of infection on day i, g(i), is proportional to e 0.25i . After the 23 of January when significant travel bans were introduced, the rate at which individuals left Wuhan diminished significantly, causing the reporting rate to suddenly drop. Therefore, if the data is truncated after the 23 of January, the reporting rate must be appropriately adjusted. This is illustrated in Figure 1a . The difference between these two datasets is the truncation date, with the first truncated at 20 January and the second at 9 February. The third dataset contains cases that were infected through a discrete infection event, such as spending time with a known infected case. In this \"non-Wuhan\" dataset, the reporting rate is constant and the force of infection can be assumed constant over each exposure window. The source we use for these three data sets is a publicly available line-list [80] .', 'cite_spans': [{'start': 1031, 'end': 1035, 'text': '[80]', 'ref_id': 'BIBREF80'}], 'ref_spans': [{'start': 530, 'end': 539, 'text': 'Figure 1a', 'ref_id': 'FIGREF1'}], 'section': 'Incubation period'}, {'text': 'Incubation periods, and many other delay distributions, are generally observed to have right skewed distributions. We therefore choose to use a Gamma distribution, though other distributions can also be applied using the proposed methods, such as Weibull and Log-normal. To fit the data, we use maximum likelihood estimation.', 'cite_spans': [], 'ref_spans': [], 'section': 'Incubation period'}, {'text': 'To adjust for the biases we use a \"forwards\" approach [60, 75, 79, 81] , where we condition on the time of the first event, time of exposure, and find the distribution looking forward to the second event, time of symptom onset. For a data point {a i , b i , y i }, where infection occurs between a i and b i , and y i is the symptom onset date, the likelihood function is given by', 'cite_spans': [{'start': 54, 'end': 58, 'text': '[60,', 'ref_id': 'BIBREF60'}, {'start': 59, 'end': 62, 'text': '75,', 'ref_id': 'BIBREF75'}, {'start': 63, 'end': 66, 'text': '79,', 'ref_id': 'BIBREF79'}, {'start': 67, 'end': 70, 'text': '81]', 'ref_id': 'BIBREF81'}], 'ref_spans': [], 'section': 'Incubation period'}, {'text': 'where g(·) is the density function of the infection date and f θ (·) is the density function of the incubation period parameterised by θ. From this, the likelihood function for our dataset X is given by', 'cite_spans': [], 'ref_spans': [], 'section': 'Incubation period'}, {'text': 'This approach is independent of the reporting rate bias, since the reporting rate depends on the date an individual leaves Wuhan (b i ), which is conditioned against (see Appendix A). We use the mean and standard deviation to characterise the MLE. Since the tail of the incubation period is important when designing quarantine strategies, we then calculate the probability that the incubation period is longer than 14 days and find the minimum day by which 99% of cases will have expressed symptoms (excluding true asymptomatic cases). We also investigate the reporting date uncertainty mentioned in Section 2.1 by considering the different extremes that the data could represent. This is achieved through adding or subtracting a day to all recorded data.', 'cite_spans': [], 'ref_spans': [], 'section': 'Incubation period'}, {'text': 'Methods accounting for truncation and growth biases in epidemic data have been discussed widely in the literature [39, 60, 78, 82] , however there are fewer applications to outbreaks [26] . In the context of COVID-19, estimates have considered growing force of infection, for example [46] , and some approaches have considered truncation, for example [51] . However, these attempts do not adjust for the reporting rate in the data or use the correct force of infection, causing the incubation period to be overestimated.', 'cite_spans': [{'start': 114, 'end': 118, 'text': '[39,', 'ref_id': 'BIBREF38'}, {'start': 119, 'end': 122, 'text': '60,', 'ref_id': 'BIBREF60'}, {'start': 123, 'end': 126, 'text': '78,', 'ref_id': 'BIBREF78'}, {'start': 127, 'end': 130, 'text': '82]', 'ref_id': 'BIBREF82'}, {'start': 183, 'end': 187, 'text': '[26]', 'ref_id': 'BIBREF25'}, {'start': 284, 'end': 288, 'text': '[46]', 'ref_id': 'BIBREF46'}, {'start': 351, 'end': 355, 'text': '[51]', 'ref_id': 'BIBREF51'}], 'ref_spans': [], 'section': 'Incubation period'}, {'text': 'Here we demonstrate the importance of truncation (Table 1) . We use the data truncated at 20 January, which has exposure windows between 1 December and 19 January. This data set is chosen since it is most sensitive to truncation due to the exponentially growing force of infection and high reporting rate. Without accounting for truncation, the length of the incubation period is significantly underestimated, which could have a large impact on the success of intervention strategies. ', 'cite_spans': [], 'ref_spans': [{'start': 49, 'end': 58, 'text': '(Table 1)', 'ref_id': 'TABREF0'}], 'section': 'Truncation'}, {'text': 'To demonstrate the effectiveness of the bias correction method, we compare three different data sets ( Table 2) .', 'cite_spans': [], 'ref_spans': [{'start': 103, 'end': 111, 'text': 'Table 2)', 'ref_id': 'TABREF1'}], 'section': 'Different data sets'}, {'text': 'The similar distributions predicted across these datasets suggests a robust method. Figure 1b compares the full distributions for these three estimates. ', 'cite_spans': [], 'ref_spans': [{'start': 84, 'end': 93, 'text': 'Figure 1b', 'ref_id': 'FIGREF1'}], 'section': 'Different data sets'}, {'text': 'When constructing intervention strategies for an epidemic, the incubation period is an important parameter. For example, consider the quarantine strategy deployed in many countries during the early stages of the epidemic, aimed at preventing cases being imported from Wuhan. This strategy quarantined individuals upon their return from Wuhan for 14 days. For such a strategy to be effective, we require most incubation periods to be less than 14 days, so that the majority of infected people would develop symptoms before quarantine ended, enabling them to be further isolated. In this analysis, we show that in the worst-case scenario we would expect 1 in 62 cases to slip through this quarantine, with the best fit predicting 1 in 101 cases. Therefore, the 14 day quarantine period would capture the majority of cases. Throughout the epidemic, this seems to have been reasonably successful, and prevented early seeding of cases in many countries. However, potentially due to complicated travel patterns or asymptomatic transmission, cases have slipped through detection and not been quarantined, which unfortunately has led to the situation observed today.', 'cite_spans': [], 'ref_spans': [], 'section': 'Implications'}, {'text': 'In addition to the incubation period, there are many other delay distributions that must be estimated while an epidemic is growing related to infectious diseases that can be estimated using the same technique. These include the generation time, the time between two infection events in a transmission chain; the serial interval, the time between symptom onset of an infector to their infectee; and the onset-to-death delay, the time from symptom onset to death.', 'cite_spans': [], 'ref_spans': [], 'section': 'Implications'}, {'text': 'Transportation modelling plays a crucial role in the early stages of an outbreak; an infected individual may travel outside of the region in which they were originally infected and seed further infections across geographical scales which are impossible to contain. Furthermore, as the rate of travelling increases, the number of observed cases within the known \"origin\" region decreases, and if exportation is not taken into account this results in an underestimation of the number of cases. These underestimates can be improved by looking at the total number of cases across all known affected regions, but doing so introduces further complications. For example, if an individual has less severe symptoms they may not seek medical assistance, thereby not being recorded as a case at their destination. This underestimation of cases can have significant effects if the traveller is able to infect more people. A new transmission chain can thus be started which remains undetected for some time due to a lacking known connection to the \"origin\" region.', 'cite_spans': [], 'ref_spans': [], 'section': 'Transportation modelling and under-reporting'}, {'text': 'In the \"origin\" region an individual with mild symptoms may still be tested for an infection due to a higher level of alertness in the local health care system. However, this level of active case-finding may not be present elsewhere, or may not have been allocated a comparable level of resources. Further complications to this model arise from the incubation period of individuals wherein detection is unlikely, and the variations in movement and mixing between people when preventative measures are put in place.', 'cite_spans': [], 'ref_spans': [], 'section': 'Transportation modelling and under-reporting'}, {'text': 'We consider a metapopulation model seeded with an infection in one of the regions, O, and investigate how exportation from this region combined with variability in case-finding can alter estimates for the doubling time and the expected portion of the population we expect to identify. This accordingly bounds the proportion of the infected population one would be able to target for personal intervention (e.g. quarantine or treatment).', 'cite_spans': [], 'ref_spans': [], 'section': 'Transportation modelling and under-reporting'}, {'text': 'Note that the proportion of identified cases need not necessarily correlate with the proportion of the infected population who exhibit symptoms.', 'cite_spans': [], 'ref_spans': [], 'section': 'Transportation modelling and under-reporting'}, {'text': 'Let us assume that movement from O begins at time t = t c , and occurs with a constant rate ρ; this can be thought of as the surge in travel in China during the beginning of the Spring Festival. In the early phase of an epidemic, we can assume the incidence I(t) = I 0 e rt of cases to be growing exponentially with a rate r.', 'cite_spans': [], 'ref_spans': [], 'section': 'Transportation modelling and under-reporting'}, {'text': 'The number of cases at time t which were infected a time τ ago is denoted by i(t, τ ), where the probability of detecting a case that infected a host a time τ ago is given by p(τ ) = Θ(τ − τ inc ) where Θ(·) is the Heaviside step function and τ inc is the incubation period. That is, a case is detected immediately following the end of the incubation period. We assume that the incubation period is characterised by a gamma-distribution with mean 4.84 and standard deviation 3.22 (Table 2) , since this estimate is independent of the exponential growth rate (Section 2.2). The number of observed cases in O is given by', 'cite_spans': [], 'ref_spans': [{'start': 480, 'end': 489, 'text': '(Table 2)', 'ref_id': 'TABREF1'}], 'section': 'Transportation modelling and under-reporting'}, {'text': 'where we have assumed that recovery of cases is negligible over the time scale of case observations. If we consider travel to i other regions from O, the total number of observed cases in all destinations is', 'cite_spans': [], 'ref_spans': [], 'section': 'Transportation modelling and under-reporting'}, {'text': 'where ω is the mean case-finding ability across all destinations. In the presence of real-time transition probabilities p ij of moving between two regions, these estimates can be further elaborated.', 'cite_spans': [], 'ref_spans': [], 'section': 'Transportation modelling and under-reporting'}, {'text': 'Historic estimates for Chinese travel data indicate a mean travel rate from Hubei province of ρ = 0.029 which began on January 10 [40, 58] . Assuming an incubation period of approximately 5 days for the infection, this suggests a rate r = 0.22 ± 0.01 when ignoring travel exportation, in contrast to r = 0.25 ± 0.01 when accounting for ρ. This difference may seem small, but it reduces the doubling time by approximately 12 hours. The expected value of r grows linearly with the exportation rate, which has also been observed with real-time travel models [44] . Further models have also been developed which consider travel and exportation of cases in greater detail [21, 33] .', 'cite_spans': [{'start': 130, 'end': 134, 'text': '[40,', 'ref_id': 'BIBREF39'}, {'start': 135, 'end': 138, 'text': '58]', 'ref_id': 'BIBREF58'}, {'start': 555, 'end': 559, 'text': '[44]', 'ref_id': 'BIBREF43'}, {'start': 667, 'end': 671, 'text': '[21,', 'ref_id': 'BIBREF20'}, {'start': 672, 'end': 675, 'text': '33]', 'ref_id': 'BIBREF32'}], 'ref_spans': [], 'section': 'Transportation modelling and under-reporting'}, {'text': 'The relationship between the observed cases in our origin and destinations can be used to determine the casefinding ability, though it should be noted that ω likely varies with time as burdens are increased on public services and the number of cases grow. Early estimates using data from [1] indicate at most an 80% case-finding ability, suggesting thousands of undetected cases exported to other regions of China, a sufficient quantity to sustain further transmission post-exportation independently of the number of asymptomatic cases present.', 'cite_spans': [], 'ref_spans': [], 'section': 'Transportation modelling and under-reporting'}, {'text': 'The intention of these estimates is not to provide specific values for the doubling time of the spread of COVID-19', 'cite_spans': [], 'ref_spans': [], 'section': 'Transportation modelling and under-reporting'}, {'text': 'in China (as the estimates above use historic travel data and are limited by the availability of data), but to bring attention to the unusual circumstances surrounding changes in contact patterns, and mobility during the Chinese Spring Festival, the largest human migration on Earth [40] . Failing to account for the significant level of dispersion or exportation of cases during these circumstances will significantly skew our estimates.', 'cite_spans': [{'start': 283, 'end': 287, 'text': '[40]', 'ref_id': 'BIBREF39'}], 'ref_spans': [], 'section': 'Transportation modelling and under-reporting'}, {'text': 'In a scenario where a single individual exposes a group, it can be unclear how many people have been infected since they do not immediately develop symptoms. However, knowing the true prevalence in the population is essential to determine the most effective interventions to put in place, and to estimate future burdens on public services. Using the probability density function of the incubation period, we consider the efficacy of using the time it takes for people to present with symptoms as a predictor for the size of the infected group. This analysis is an effective ready reckoner at early stages of a novel infection, or in close contact environments, and is useful for predicting generation size when a complete data set is not yet available. In this analysis we focus on a scenario where infection time is known. In reality, we may only know an exposure window. For short exposure windows this method can still be valid, but for longer exposure windows it will need extending to account for this added uncertainty.', 'cite_spans': [], 'ref_spans': [], 'section': 'Estimating the size of the first generation from the observed number of symptomatic individuals'}, {'text': 'We assume that the number of individuals who have been exposed to potential infection is known, in which case the number of people who are infected can be assumed to be binomially distributed with an unknown probability P that each individual has been infected. To determine the distribution of infected individuals, we use the available information regarding the number of individuals who have expressed symptoms. This yields two cases. In the first case, we assume that the true number of symptomatic individuals are observed. In the second case, we take the number of observed symptomatic individuals as a lower bound on the true value.', 'cite_spans': [], 'ref_spans': [], 'section': 'Estimating the size of the first generation from the observed number of symptomatic individuals'}, {'text': 'We wish to determine the probability that the first generation has e 0 individuals, E 0 = e 0 , given that i τ symptomatic individuals are observed on day τ , I τ = i τ , which is given by', 'cite_spans': [], 'ref_spans': [], 'section': 'Estimating the size of the first generation from the observed number of symptomatic individuals'}, {'text': 'To solve this, we need to determine the distribution of the infection probability P given the number of observed symptomatics. Assuming that P is uniformly distributed, we have', 'cite_spans': [], 'ref_spans': [], 'section': 'Estimating the size of the first generation from the observed number of symptomatic individuals'}, {'text': 'where F (·) is the cumulative density function for the incubation period, n is the number of initially exposed individuals, and c is a normalising constant such that', 'cite_spans': [], 'ref_spans': [], 'section': 'Estimating the size of the first generation from the observed number of symptomatic individuals'}, {'text': ', where 2 F 1 represents the hypergeometric function [2] . Substituting this into Equation (3) gives', 'cite_spans': [{'start': 53, 'end': 56, 'text': '[2]', 'ref_id': 'BIBREF1'}], 'ref_spans': [], 'section': 'Estimating the size of the first generation from the observed number of symptomatic individuals'}, {'text': 'which simplifies to', 'cite_spans': [], 'ref_spans': [], 'section': 'Estimating the size of the first generation from the observed number of symptomatic individuals'}, {'text': 'This gives a distribution of the generation-size based on the number of observed symptomatic individuals by time τ . We can extend it to investigate a scenario where no symptomatic individuals have been observed by time τ by using a value of 0 for I τ :', 'cite_spans': [], 'ref_spans': [], 'section': 'Estimating the size of the first generation from the observed number of symptomatic individuals'}, {'text': 'This can be used to illustrate worst and best case scenarios given τ time has passed without symptomatic individuals. Additionally, if we consider the probability that E 0 = 0, we can find the value of τ where we can have a 95% confidence that there will not be a second generation:', 'cite_spans': [], 'ref_spans': [], 'section': 'Estimating the size of the first generation from the observed number of symptomatic individuals'}, {'text': 'This analysis considers the case when the number of observed symptomatic individuals to date is the true number. In practice however, we do not generally observe every symptomatic individual, so the number of observations is only a lower bound on the true number. To address this, rather than considering I τ as the total number of people who have developed symptoms by time τ , we can defineĨ τ as the minimum number of people who have developed symptoms by time τ . We assume that the probability thatĨ τ is equal toĩ τ for a given value of i τ is uniform at 1 iτ +1 . We can then use the same methods as above to infer a distribution for P . Details are provided in Appendix B. Figure 2 : Prediction of the size of the first generation, e 0 , in an infection event in which 20 people were exposed. A, B and C show the density when the number of observed symptomatics is taken to be the true number of symptomatics, D, E and F consider the case where the observed symptomatics is a lower bound on the true symptomatics. A and D consider the case when zero symptomatics are observed after 5 days, B and E when 5 are observed after 5 days, and C and F when 5 are observed after 10 days. The incubation period for the disease has been modelled as a gamma distribution with a mean of 4.84 and standard deviation of 2.79 ( Table 5 ).', 'cite_spans': [], 'ref_spans': [{'start': 681, 'end': 689, 'text': 'Figure 2', 'ref_id': None}, {'start': 1320, 'end': 1327, 'text': 'Table 5', 'ref_id': 'TABREF4'}], 'section': 'Estimating the size of the first generation from the observed number of symptomatic individuals'}, {'text': 'As we can see from Figure 2 , this method can be used to predict the number of infected individuals in the original exposed group. However, we have also demonstrated the importance of caution when interpreting this data. If there is uncertainty surrounding the presentation of symptomatic patients, usingĨ τ as a lower bound is a robust method to ensure the size of the generation is not underestimated.', 'cite_spans': [], 'ref_spans': [{'start': 19, 'end': 27, 'text': 'Figure 2', 'ref_id': None}], 'section': 'Estimating the size of the first generation from the observed number of symptomatic individuals'}, {'text': 'Here, we provide a table with some of the key parameters for COVID-19. From Table 4 , it can be seen that parameter estimates vary widely across different reports. This can be due to the biases in the data not being correctly accounted for, such as the growth rate of the epidemic, travel patterns and time scale over which data is gathered.', 'cite_spans': [], 'ref_spans': [{'start': 76, 'end': 83, 'text': 'Table 4', 'ref_id': 'TABREF3'}], 'section': 'COVID-19: Indicative parameters'}, {'text': 'In addition to the incubation period distribution, we use the method from Section 2.2 to estimate a variety of delays from UK data provided by Public Health England through a data sharing agreement. This data takes the form of line-lists reporting date of onset, hospitalisation, ICU admission, and recovery or death. These estimates are given in Table 5 , along with the other estimates from Sections 2.2 and 2.3. ', 'cite_spans': [], 'ref_spans': [{'start': 347, 'end': 354, 'text': 'Table 5', 'ref_id': 'TABREF4'}], 'section': 'COVID-19: Indicative parameters'}, {'text': 'When designing intervention strategies, we need to consider how adherence may alter their effectiveness. This is important, since highly effective interventions may not be adhered to if they present great individual cost to a population. In this case, a theoretically less effective intervention may perform better, if it has sufficient reduction in individual-level cost. In this section, we illustrate the potential impacts of adherence on the effectiveness of interventions using a toy model.', 'cite_spans': [], 'ref_spans': [], 'section': 'Adherence'}, {'text': 'Consider a standard SIR model, and denote by S(t) and R(t), respectively, the susceptible and recovered/immune fractions of the population at time t. We can write S in terms of R such that', 'cite_spans': [], 'ref_spans': [], 'section': 'Adherence'}, {'text': 'and let t → ∞ to get the final size formula', 'cite_spans': [], 'ref_spans': [], 'section': 'Adherence'}, {'text': 'where R(∞) is the fraction of cases at end of outbreak. This gives a ready reckoner for the eventual attack rate if interventions are not put in place or come in too late to be effective. To illustrate, if we have R 0 = 3 (and S(0) ≈ 1), then R(∞) = 0.94. If an intervention is put in place that reduces (with full adherence) R 0 < 1 then the outbreak will be controlled. Indeed, let us assume that R 0 is reduced to zero by the intervention: for example, assume that social distancing is perfect and the number of contacts of a fully-adherent individual is zero. If only 50% of people adhere to the intervention then the average number of contacts is effectively reduced by a half and logically R † 0 = R 0 /2 = 1.5 (the † representing quantities post intervention) and R(∞) † = 0.58 in this case. However, this assumes that adherence is an independent random process at each contact. This suggests that for each contact an individual would ordinarily make, they \"toss a coin\" to decide whether to isolate or not. In reality, individuals are more likely to show polarity, where some individuals reduce all their contacts and follow the measures and a proportion of individuals choose to not adhere to the intervention. If there was distinct polarity in the population such that 50% adhered perfectly and 50% ignored policy, then a toy model can be created with two infectious groups, I A and I B , that behave differently. In this casė', 'cite_spans': [], 'ref_spans': [], 'section': 'Adherence'}, {'text': 'where a dot over a variable represents its time derivative. Such an epidemic model, where the two groups have the same susceptibility but different infectivity, has the same final size as an epidemic in a single-type model with the same R 0 (e.g. see [5] ). However, they have different durations as can be seen in Figure 3 , where φ = 1/2, R B = 0 and R A = 3. This shows that the assumptions about the nature of adherence predict the same growth rate and final size, but that the more polarised adherence has faster early growth and therefore an earlier peak.', 'cite_spans': [{'start': 251, 'end': 254, 'text': '[5]', 'ref_id': 'BIBREF4'}], 'ref_spans': [{'start': 315, 'end': 323, 'text': 'Figure 3', 'ref_id': 'FIGREF3'}], 'section': 'Adherence'}, {'text': 'This issue of independent versus polarised adherence is related to the idea of all-or-nothing versus leaky vaccination [32, 54] , where you either vaccinate a fraction of the population with 100% efficacy or vaccinate 100% of the population with reduced efficacy [37] . Note however that vaccination reduces your susceptibility (whether only or also), rather than only your infectivity as in the model discussed above, and variation in susceptibility does reduce the final size, with imperfect coverage with a perfect vaccine (all-or-nothing) leading to a lower final size than full coverage with a leaky vaccine (all individuals having the same mean susceptibility).', 'cite_spans': [{'start': 119, 'end': 123, 'text': '[32,', 'ref_id': 'BIBREF31'}, {'start': 124, 'end': 127, 'text': '54]', 'ref_id': 'BIBREF54'}, {'start': 263, 'end': 267, 'text': '[37]', 'ref_id': 'BIBREF36'}], 'ref_spans': [], 'section': 'Adherence'}, {'text': 'The ongoing COVID-19 outbreak is known to have higher mortality rates amongst the elderly, the immunocompromised and those with respiratory and health complications [34, 88, 89, 90, 91] . In this section, we model the introduction of an infectious disease into care homes, in order to obtain estimates of the final size of the epidemic in the vulnerable population as well as predictions for the number of hospitalisations and fatalities. Modelling of care homes in the UK is conducted against the backdrop of a wider epidemic in the general population, which we here assume to be following SEIR dynamics with a basic reproduction number R 0 that might be different from the within-care home reproduction number R C .', 'cite_spans': [{'start': 165, 'end': 169, 'text': '[34,', 'ref_id': 'BIBREF33'}, {'start': 170, 'end': 173, 'text': '88,', 'ref_id': 'BIBREF88'}, {'start': 174, 'end': 177, 'text': '89,', 'ref_id': 'BIBREF89'}, {'start': 178, 'end': 181, 'text': '90,', 'ref_id': 'BIBREF90'}, {'start': 182, 'end': 185, 'text': '91]', 'ref_id': 'BIBREF91'}], 'ref_spans': [], 'section': 'Care home model'}, {'text': 'Care homes are assumed to be closed populations, with the infection entering each of them independently with a certain probability. Infection is seeded only once, and within-care home outbreaks then evolve independently from, and do not contribute to, other care home outbreaks and the epidemic in the background population. To keep track of hospitalisations, we model the within-care home infection dynamics using a compartmental model that, in addition to SEIR model, has compartments for mildly symptomatic prodromal cases (P), who show no symptoms but are capable of transmitting the virus, those who recover from the disease after mild symptoms that did not require hospitalisation (M), those who have severe symptoms and are admitted to hospital (H), those who recover after hospitalisation (R), and those that die (D). This is illustrated in Figure 4 .', 'cite_spans': [], 'ref_spans': [{'start': 849, 'end': 857, 'text': 'Figure 4', 'ref_id': 'FIGREF4'}], 'section': 'Care home model'}, {'text': 'The stochastic component of the model, i.e. the random introduction of the infection in care homes, is modelled using the Sellke construction [4] . Each care home i is given an individual, random threshold of resistance, Q i , which is drawn from an U(0, 1) distribution. At time t, we then calculate the infection pressure IP(t) from the background epidemic so that care home i becomes infected at time', 'cite_spans': [{'start': 142, 'end': 145, 'text': '[4]', 'ref_id': 'BIBREF3'}], 'ref_spans': [], 'section': 'Care home model'}, {'text': \"infection pressure up to time t for a median sized care home is the integral from 0 to t of the force-of-infection (FOI) applied to the care home coming from all infectious sources, multiplied by a probability p. This probability represents the probability of the infection being introduced to a median-sized care home. For other care homes, we allow this probability to be proportional to its size, under the assumption that larger care homes employ more staff and are therefore at higher risk of introduction. When the infection pressure becomes higher than an individual care home's resilience threshold, that care home begins its own deterministic infection dynamics with a single initial infected case. We run this model on data for the entire care home population in the UK, so that there are approximately 15,000 care homes with a total population of approximately 450,000 residents [22] . In this model we only consider the vulnerable population within care homes. We assume R 0 = 1.5 in the background epidemic, a relatively low value that somehow accounts for a certain degree of control, and an R C = 3 to allow relatively explosive epidemics in care homes due to potentially more frail individuals, difficulty in isolation and staff inadvertently passing the infection from one case to the next. The other parameters in the baseline scenario are reported in Table 6 . There are a variety of assumptions underpinning this model. Firstly, the background epidemic ignores structure and assumes homogeneous mixing. This is likely to make the peak more pronounced, so presents a worst case scenario for the demand on hospital beds. The assumption of R 0 for the background epidemic only affects the shape and duration of the background epidemic, since the tuneable parameter p controls the risk of introduction Week of peak bed occupancy by carehome population\", 'cite_spans': [{'start': 890, 'end': 894, 'text': '[22]', 'ref_id': 'BIBREF21'}], 'ref_spans': [{'start': 1370, 'end': 1377, 'text': 'Table 6', 'ref_id': 'TABREF5'}], 'section': 'Care home model'}, {'text': '(b) Left plot shows the predicted height of the peak in the number of hospital beds occupied (solid blue line, left axis) and total number of deaths (dashed red line, right axis), as a function of the probability p of introduction in a median-sized care home. Right plot shows the timing of the peak, which always appears to occur at a similar time or later, compared to the background epidemic. The peak arrives earlier for larger values of p, as high p corresponds to multiple simultaneous introductions early on in the background epidemic. to the care home. That is, if R 0 is small, a large p still presents a high force of infection into the care homes. Therefore, for a fixed p, we expect that changing R 0 does not affect the total number of deaths, but it changes the peak hospitalisation incidence because a faster and more explosive background epidemic makes epidemics in care homes more synchronised. In fact, when testing the impact of a longer, flatter background epidemic, for example obtained by simulating three slightly desynchronised background SEIR epidemics, results have lower peaks and a much more variable timing (not shown). Assuming each care home is independent might not be realistic, since it is likely that staff are shared between multiple homes, in which case they can act as vectors of transmission between homes. However, current outbreaks are already quite synchronised (the within care home outbreaks occur at similar times), so the effect of this assumption is likely to be minimal. The final major assumption is that the epidemic within the care homes is deterministic. This removes the probability of random extinction and random delays, and should obviously be relaxed with a stochastic model, given half of care homes have size smaller than 25. However, the extinction probability is very low with R C = 3, so this stochastic effect is unlikely to have a large impact. Random delays, instead, may change the shape and timing of the epidemic, which could potentially reduce the peak burden. Therefore, this model represents a worst case scenario.', 'cite_spans': [], 'ref_spans': [], 'section': 'Care home model'}, {'text': 'In the absence of cure or vaccine for COVID-19, governments worldwide must rely on non-pharmaceutical interventions (NPIs) to control the outbreak [35] . A natural such intervention is to ask individuals who express symptoms similar to COVID-19 to isolate themselves, but variants to such individual isolation might include policies sometimes referred to as household isolation, household quarantine and mixed isolation. In this section,', 'cite_spans': [{'start': 147, 'end': 151, 'text': '[35]', 'ref_id': 'BIBREF34'}], 'ref_spans': [], 'section': 'Household isolation modelling'}, {'text': 'we investigate how such strategies affect the spread of the epidemic when bearing in mind that adherence to each intervention may differ. Individual isolation relies on individuals staying in isolation when they express symptoms, thereby stopping transmission. However, there is potential asymptomatic or prodromal transmission before they go into isolation.', 'cite_spans': [], 'ref_spans': [], 'section': 'Household isolation modelling'}, {'text': 'Additionally, isolation strategies generally ask infected individuals to remain at home, which presents an infection risk to the other members of their household, who may go on to spread the infection. This ensures that there are no symptomatic cases evading intervention but applies quite drastic measures to the household.', 'cite_spans': [], 'ref_spans': [], 'section': 'Household isolation modelling'}, {'text': 'A fourth strategy, that reduces the cost relative to household quarantine, is mixed isolation. Here, upon detection of symptoms the entire household is isolated for a fixed length of time. Any subsequent cases within the household then undergo individual isolation as described above. This reduces the risk of cases not being isolated whilst allowing recovered individuals to return to work. There is however still some remaining risk that infected individuals may not yet express symptoms after the end of the isolation period, but this risk can be controlled through the duration of each isolation.', 'cite_spans': [], 'ref_spans': [], 'section': 'Household isolation modelling'}, {'text': 'Although there is now a rich theoretical literature on households models [8, 9, 10] , the mainstream methodological tools in this research area present important limitations that make them not directly applicable to studying these control policies. First, exact theoretical or asymptotic results in these models are mostly restricted to time-integrated quantities, i.e. those quantities that do not depend on the detailed temporal shape at which the infectivity is spread by an individual: these are R 0 (or any other reproduction number [11, 65] , e.g. the household reproduction number R * ), the probability of a large epidemic, and the epidemic final size [4] . For this reason, the vast majority of the literature relies on the standard stochastic SIR model [4] , despite its unrealistic infectivity profile. Even if more recent work has expanded beyond time integrated quantities, for example considering the real-time growth rate [11, 67] , if the interest is on tracking the dynamics of infection spread, a model based on full temporal representation of between-and within-household dynamics [36] appears necessary.', 'cite_spans': [{'start': 73, 'end': 76, 'text': '[8,', 'ref_id': 'BIBREF7'}, {'start': 77, 'end': 79, 'text': '9,', 'ref_id': 'BIBREF8'}, {'start': 80, 'end': 83, 'text': '10]', 'ref_id': 'BIBREF9'}, {'start': 538, 'end': 542, 'text': '[11,', 'ref_id': 'BIBREF10'}, {'start': 543, 'end': 546, 'text': '65]', 'ref_id': 'BIBREF65'}, {'start': 660, 'end': 663, 'text': '[4]', 'ref_id': 'BIBREF3'}, {'start': 763, 'end': 766, 'text': '[4]', 'ref_id': 'BIBREF3'}, {'start': 937, 'end': 941, 'text': '[11,', 'ref_id': 'BIBREF10'}, {'start': 942, 'end': 945, 'text': '67]', 'ref_id': 'BIBREF67'}, {'start': 1100, 'end': 1104, 'text': '[36]', 'ref_id': 'BIBREF35'}], 'ref_spans': [], 'section': 'Household isolation modelling'}, {'text': 'A second limitation of standard household models is the key assumption of constant parameter values. This appears essential for any form of analytical progress. However, in the context of the interventions discussed above, a reduction in transmission between households, as well as a potential increase in within the household, require parameters to change over time.', 'cite_spans': [], 'ref_spans': [], 'section': 'Household isolation modelling'}, {'text': 'To overcome these limitations, we consider two approaches. The first approach fully captures both within and between-household dynamics with a master-equation formalism, i.e. by relying on a Markovian within-household dynamics and keeping track of the expected number of households in each possible state of their internal dynamics. The second approach has a greater emphasis on within-household dynamics, and is fundamentally an independent-households, individual-based, stochastic simulation. The more limited mathematical tractability is the price to pay for an increased flexibility, as the within-household Markov assumption is relaxed and exact distributions for delays between events, typically informed by the data, can be explicitly inputted. Although both approaches can account for increased within-household transmission as isolation and quarantine are imposed, we only consider this for the second method here. This aspect allows us to study the increased risk of infection a vulnerable individual in the household would experience following the implementation of a control policy.', 'cite_spans': [], 'ref_spans': [], 'section': 'Household isolation modelling'}, {'text': 'To model the households in the UK, we construct a realistic distribution of household sizes. We take this demographic data from the 2001 Census [63] . More recent information, though less specific on large household sizes, shows that sizes of smaller households are largely unchanged over time [62] .', 'cite_spans': [{'start': 144, 'end': 148, 'text': '[63]', 'ref_id': None}, {'start': 294, 'end': 298, 'text': '[62]', 'ref_id': None}], 'ref_spans': [], 'section': 'Household isolation modelling'}, {'text': 'In this section, we investigate the above intervention strategies under the assumption that a fraction of households adhere 100% with an intervention and the remaining households ignore the intervention. To model the interventions, we implement a dynamical household model that explicitly represents the small sizes of households.', 'cite_spans': [], 'ref_spans': [], 'section': 'Population and household transmission'}, {'text': 'The dynamics of the outbreak are simulated using an SEPIR model. This model assumes that there are five possible states in which an individual can be. These are, susceptible, latent, mildly symptomatic prodrome, symptomatic infectious and removed. Following [18] , we assume that within-household transmission scales with the inverse of the household size to a specified power η. Such a model can be used to investigate how the pathogen spreads through and between households.', 'cite_spans': [{'start': 258, 'end': 262, 'text': '[18]', 'ref_id': 'BIBREF17'}], 'ref_spans': [], 'section': 'Population and household transmission'}, {'text': 'The methodology involved is the use of self-consistent differential equations, first written down by Ball [7] .', 'cite_spans': [{'start': 106, 'end': 109, 'text': '[7]', 'ref_id': 'BIBREF6'}], 'ref_spans': [], 'section': 'Population and household transmission'}, {'text': 'More recent developments, including numerical methods for these equations, include [36, 72, 15, 43] . Important features of this approach include allowing for a small, finite size of each household in which random effects are important and each pair can only participate in one infection event.', 'cite_spans': [{'start': 83, 'end': 87, 'text': '[36,', 'ref_id': 'BIBREF35'}, {'start': 88, 'end': 91, 'text': '72,', 'ref_id': 'BIBREF72'}, {'start': 92, 'end': 95, 'text': '15,', 'ref_id': 'BIBREF14'}, {'start': 96, 'end': 99, 'text': '43]', 'ref_id': 'BIBREF42'}], 'ref_spans': [], 'section': 'Population and household transmission'}, {'text': 'Let Q n,s,e,p,i (t) be the proportion of households in the population at time t of size n, with s susceptibles, e exposed, p prodromal, and i symptomatic infectious individuals. The number of recovered individuals will be n − s − e − p − i. In the absence of household-based interventions, we have d dt Q n,s,e,p,i = − sr s→e (t, Q) + er e→p + pr p→i + ir i→∅ + n −η spτ p + n −η siτ i Q n,s,e,p,i + (s + 1)r s→e (t, Q)Q n,s+1,e−1,p,i + (e + 1)r e→p Q n,s,e+1,p−1,i + (p + 1)r p→i Q n,s,e,p+1,i−1 + (i + 1)r i→∅ Q n,s,e,p,i+1 + n −η (s + 1)pτ p Q n,s+1,e−1,p,i + n −η (s + 1)iτ i Q n,s+1,e−1,p,i , where we take any Q with logically impossible indices just to equal 0, r a→b is the rate from state a to b, and τ a is the transmission rate from an individual in state a. The transmission into households is given by (pβ p (t) + iβ i (t)) Q n,s,e,p,i .', 'cite_spans': [], 'ref_spans': [], 'section': 'Model'}, {'text': \"Here Λ represents infections imported from outside the population of households, and the other terms represent between-household transmissions. We take a 'global' intervention as part of the baseline, in particular, we can model phenomena such a school closures that hold during a set of times T as\", 'cite_spans': [], 'ref_spans': [], 'section': 'Model'}, {'text': 'We call ε the global reduction. We will generally drop this t-indexing for simplicity, and will also consider only a household isolation strategy (though the other strategies can be considered similarly, with an example of how other strategies could be captured in this model framework given in Appendix C). Suppose a fraction α W of households isolates when there is at least one symptomatic case in the household. These households do not experience new infections, meaning that the dynamics become d dt Q n,s,e,p,i = − 1 − α W 1 {i>0} sr s→e (t, Q) + er e→p + pr p→i + ir i→∅ + n −η spτ p + n −η siτ i Q n,s,e,p,i + 1 − α W 1 {i>0} (s + 1)r s→e (t, Q)Q n,s+1,e−1,p,i + (e + 1)r e→p Q n,s,e+1,p−1,i + (p + 1)r p→i Q n,s,e,p+1,i−1 + (i + 1)r i→∅ Q n,s,e,p,i+1 + n −η (s + 1)pτ p Q n,s+1,e−1,p,i + n −η (s + 1)iτ i Q n,s+1,e−1,p,i , and also do not transmit outside, meaning that the rate of between-household transmission becomes', 'cite_spans': [], 'ref_spans': [], 'section': 'Model'}, {'text': 'Using the methods in [15, 72] , it is possible to fit household models of this kind to the overall growth rate, r, which we take to correspond to a doubling time of three days. Natural history parameters can then be set directly based on reasonable estimates: r e→p to the inverse of the latent period; r p→i to the inverse of the prodromal period; r i→∅ to the inverse of the symptomatic period. Shaw [77] analyses various household datasets for respiratory pathogens and estimates values for η close to 1, so this is taken to be 0.8. The remaining degrees of freedom are relative infectiousness of the prodrome (taken as a third) and the probability of transmitting within a pair, which we can take as a typical value given by Shaw [77] . For the numerical results in figures 6 and 7, the baseline natural history parameters are chosen to be r e→p = 1/5, r p→i = 1/3, r i→∅ = 1/4.', 'cite_spans': [{'start': 21, 'end': 25, 'text': '[15,', 'ref_id': 'BIBREF14'}, {'start': 26, 'end': 29, 'text': '72]', 'ref_id': 'BIBREF72'}, {'start': 402, 'end': 406, 'text': '[77]', 'ref_id': 'BIBREF77'}, {'start': 734, 'end': 738, 'text': '[77]', 'ref_id': 'BIBREF77'}], 'ref_spans': [], 'section': 'Parameterisation'}, {'text': 'Using the given parameter values for our baseline scenario, we consider a combination of household isolation (which follows all-or-nothing adherence) with global reduction in transmission (which follows leaky adherence)', 'cite_spans': [], 'ref_spans': [], 'section': 'Summary'}, {'text': 'for three weeks and show the results in Figure 6 . The distribution of infectious individuals varies with household size, which is shown in Figure 7 for different durations of global intervention. Applying household isolation at 65% adherence (α W = 0.65) manages to reduce the spread of infection, but appears insufficient in this model and with baseline parameters for controlling the outbreak in the long-term, unless other intervention strategies that reduce the global transmission are adopted at the same time. Alternatively, different levels of adherence can be considered to determine if and when control may be achieved purely through household-based interventions..', 'cite_spans': [], 'ref_spans': [{'start': 40, 'end': 48, 'text': 'Figure 6', 'ref_id': 'FIGREF12'}, {'start': 140, 'end': 148, 'text': 'Figure 7', 'ref_id': 'FIGREF13'}], 'section': 'Summary'}, {'text': 'For the model proposed in the next section, we look into the effectiveness of increasing adherence. ', 'cite_spans': [], 'ref_spans': [], 'section': 'Summary'}, {'text': \"The model described above has the advantage of being able to track the dynamics within the household as well as the overall epidemic in the population in a relatively efficient manner. We now discuss a different framework that loses part of the capability in keeping track of the overall epidemic, but offers further flexibility both in the impact of policies on the within-household dynamics and in the distributions between events in the infectious life of an individual. We use this model to investigate the relative effectiveness of the different control policies. We also consider allowing recovered individuals to leave the household, even in the context of household isolation or household quarantine. This has no impact on the transmission dynamics, but reduces the individuals' life disruption and potential economic cost of any policy implemented. This model assumes that there is no reintroduction within households so each household can only be isolated or quarantined once. The assumption that only one household member is infected from outside is approximately satisfied if we assume homogeneous mixing between households and a large number of households, which are all fully susceptible at the start of the epidemic. However, the reality of heterogeneous mixing makes reintroduction a likely possibility even early on in the epidemic. This model, therefore, lacks an explicit description of the social network structure beyond the household. For simplicity, we assume that within households all individuals are identical in terms of their disease dynamics, although the method might be extended to allow for different age/risk groups with different disease dynamics. We assume that the level of within-household transmission in a household of size n scales proportionally to 1/(n − 1), though we acknowledge that true transmission is slightly more complex [19] .\", 'cite_spans': [{'start': 1871, 'end': 1875, 'text': '[19]', 'ref_id': None}], 'ref_spans': [], 'section': 'Non-Markovian models with enhanced within-household transmission'}, {'text': 'We consider independent households of size n = 1, 2, . . . We assume an individual spends half of their time outside and half inside the household. When self-isolation starts, the assumed adherence a i represents the fraction of the time spent outside that is shifted from outside to within the household. Therefore, for perfect adherence, from the moment symptoms occur, the individual stops transmitting outside but their infectivity within the household grows by 100%. We also explore variations in this compensatory behaviour, so that the time of an individual is split in a more flexible proportion than 1 : 1.', 'cite_spans': [], 'ref_spans': [], 'section': 'Model'}, {'text': 'The same argument applies to other control policies, with adherence levels a h for household isolation and a q for household quarantine. When multiple control policies are in place at the same time, their effect is assumed to be multiplicative: if an individual has symptoms and the household isolates, the outside transmission rate from that individual is reduced from baseline by a multiplicative factor (1 − a i ) × (1 − a h ) and the within-household transmission rate is the baseline value plus a fraction 1 − ((1 − a i ) × (1 − a h )) of the baseline value. Therefore, implementing a control policy that reduces transmission outside might lead to more infections in the household (see Figure 8 and the associated accumulated infection pressure in Figure S1 ).', 'cite_spans': [], 'ref_spans': [{'start': 691, 'end': 699, 'text': 'Figure 8', 'ref_id': 'FIGREF14'}, {'start': 753, 'end': 762, 'text': 'Figure S1', 'ref_id': 'FIGREF1'}], 'section': 'Model'}, {'text': 'We denote by β n g (τ ) the average global infectivity profile of a household of size n, i.e. the time-point average of the rates at which new cases outside are generated by any case infected in all simulated epidemics in a household of size n. During the exponentially growing phase, any global infection starts a new within-household epidemic.', 'cite_spans': [], 'ref_spans': [], 'section': 'Model'}, {'text': 'Furthermore, larger households are more likely to be infected because they have more members. Therefore, if h n is the probability that a randomly selected household has size n, π n = nh n m mh m gives the probability that the household of a randomly selected individual is of size n. This is called the sizebiased distribution. The global infectivity profile of the average household infected during the exponentially growing phase is then:', 'cite_spans': [], 'ref_spans': [], 'section': 'Model'}, {'text': 'and the area under this curve is known in the literature as the household reproduction number, and is typically denoted byR * [10, 12, 29] . If enough transmission is prevented, so that R * < 1, the epidemic is controlled.', 'cite_spans': [{'start': 126, 'end': 130, 'text': '[10,', 'ref_id': 'BIBREF9'}, {'start': 131, 'end': 134, 'text': '12,', 'ref_id': 'BIBREF11'}, {'start': 135, 'end': 138, 'text': '29]', 'ref_id': 'BIBREF28'}], 'ref_spans': [], 'section': 'Model'}, {'text': 'At baseline we take R g = 2.5 and R h = 2.5, which gives a real-time growth rate r of about 0.245. Each individual, irrespective of whether they will be infected or not, is given (independently of each other) a duration Comparing the different strategies (Figure 9b ), household quarantine can be optimal (as one might expect), but this requires high adherence levels. As adherence drops, this strategy becomes suboptimal to mixed isolation.', 'cite_spans': [], 'ref_spans': [{'start': 255, 'end': 265, 'text': '(Figure 9b', 'ref_id': 'FIGREF20'}], 'section': 'Parameterisation'}, {'text': 'Mixed isolation is significantly better than household isolation on its own and requires little extra social cost, so should not cause adherence to drop (relative to household isolation adherence levels). The difference between the two strategies comes down to the transmission slipping through after the 14 day household isolation. The cheapest strategy, when considering working age adults, is individual isolation (Figure 9d ), but the effect is limited compared to the other models and cannot achieve control in the baseline scenario even with 100% adherence.', 'cite_spans': [], 'ref_spans': [{'start': 417, 'end': 427, 'text': '(Figure 9d', 'ref_id': 'FIGREF20'}], 'section': 'Parameterisation'}, {'text': 'Overall, the mixed isolation strategy appears to be most cost-effective. However, this is dependent on the assumption that adherence is better for 14 day isolation rather than a very long quarantine. It can be observed that household-based interventions are more effective than individual isolations, demonstrating the importance of these strategies in designing intervention policy. Figure 8 shows how the different isolation strategies contain the infectious periods of individuals within the household and also indicates the number of individuals being isolated within the household. is shown in red (right axis). All random numbers involved in the realisation of the stochastic epidemic are drawn at the start, before the impact of each control policy is implemented. Row 1 shows no isolation and individual and isolation. Rows 2, 3 and 4 show, respectively, household isolation, mixed isolation and household quarantine. The difference between the columns is that the basic policy on the left is \"upgraded\" to the more cost-effective version on the right that allows recovered individuals to leave the house as they cannot transmit outside anymore. When no control is implemented, the primary case (individual A, say, infected at time 0) infects another individual (B, say) around time 11. After a long latent period (i.e. incubation minus prodromal), B becomes infectious and infects a further individual (C, say). The last individual (D, say) escapes infection. When different intervention strategies are in place, within-household infectivity is increased. This can result in individual C becoming infected earlier in the outbreak and individual D no longer escaping infection, both due to the increased force of infection. In this simulation, the dynamics for individual B do not change since they are infected before A becomes symptomatic. Individual D is infected earliest under mixed isolation, because within-household transmission is higher than household isolation alone, due to increased adherence from individual isolation also being in place. Adherence levels to household quarantine are lower than those of household isolation, due to the higher demand of full quarantine, thus leading to less enhanced within-household transmission. The more severe the intervention, the better it captures the infectious periods of infected individuals within the household.', 'cite_spans': [], 'ref_spans': [{'start': 384, 'end': 392, 'text': 'Figure 8', 'ref_id': 'FIGREF14'}], 'section': 'Parameterisation'}, {'text': 'To study the impact such an increased within-household transmission has on the chance that a vulnerable individual is infected in the household, we randomly choose one non-primary case in the household as the ', 'cite_spans': [], 'ref_spans': [], 'section': 'Parameterisation'}, {'text': 'Real-time growth rate (no control r = 0.24 day -1 ) 1.5 days prodromal + 0.5 days to detection; \"2x adherence\" for short closure 40% 60% 80% 100% 7d Individual Isolation 14d Household Isolation (7, 14) (a) Percentage reduction in R * , defined as the total amount of community transmission spread by an average household early in the epidemic, which equals 3.1 for baseline parameters in the absence of control; (b) real-time growth rate, which is assumed to be 0 (rather than negative) when the infection is controlled; (b) increase in risk of infection an initially susceptible vulnerable person experiences in the household; and (d) the average number of days of isolation a person experiences on average in households of different sizes, computed for each size as the average total person-days in isolation divided by the number of individuals in the household. In (a)-(c), line styles refer to different control policies and colours to different levels of adherence to individual isolation. The x-axis gives the adherence to household quarantine, and household isolation (which is less demanding) is assumed to be \"twice as high\", meaning it is at the midpoint between that of household quarantine and 1 (e.g. 0.6 for an x value of 0.2, 0.9 for an x value of 0.8, etc.). The black dash-dotted line in (a) gives the amount needed to control the spread by achieving R * = 1. Notice how: the effect of individual isolation is independent of adherence to household quarantine (dotted lines); the effect of household isolation is independent of adherence to individual isolation (overlapping dash-dotted lines); mixed isolation is always superior to household isolation; household quarantine is only optimal at really high levels of adherence (for these baseline parameters, generally, beyond the level needed to achieve control), but quickly becomes suboptimal to mixed isolation as adherence is reduced. When a sufficiently large reduction in R * is achieved in (a), the growth rate drops to 0 in (b). The increased risk an initially susceptible vulnerable person is infected at home (c) does not reflect this effect, as it represents the increased risk conditional on a introduction: if the infection were controlled in the community, the overall risk of a vulnerable person getting infected would vanish as the risk of introduction in the household vanishes. For these plots, ne = 10000 simulations are performed for each household size. Nevertheless, a large amount of stochastic noise is still visible in (c). In (d), the same control policies are considered, but the household-based ones are considered both in their naïve form (where recovered individuals remain isolated), and in their upgraded version where recovered individuals are free to leave the house: they are identical in terms of transmission but the naïve versions are significantly more costly in terms of person-days of isolation. In a household of size 1 (no within-household transmission), the days in isolation would be exactly 7 or 14 if all cases were symptomatic (here ps = 2/3); similarly, in all households, individual isolation would total exactly 7 days if all cases were symptomatic and all individuals in the household were ultimately infected.', 'cite_spans': [{'start': 194, 'end': 197, 'text': '(7,', 'ref_id': 'BIBREF6'}, {'start': 198, 'end': 201, 'text': '14)', 'ref_id': 'BIBREF13'}], 'ref_spans': [], 'section': 'Real-time growth rate'}, {'text': 'Social distancing, isolation and lockdowns act to mitigate the spread of an infectious disease and reduce the number of cases. However, such interventions, particularly widespread lockdowns, cannot be maintained indefinitely and must be lifted at some point. For the disease to be controlled, these interventions can be implemented until pharmaceutical interventions are developed, such as a vaccine, or until the case numbers are low enough that the disease may go extinct. Here, we consider the situation where interventions are lifted just before extinction, when the number of cases has reached a low but non-zero initial value n 0 : at this point, the number of cases might rebound or might go extinct by random chance despite an R 0 > 1. We use a time-inhomogeneous birth-death chain model [41] to investigate the probability of extinction in this context. The n 0 \"initial\" cases give rise to new cases at a time-dependent rate β(t) and recover at rate δ(t). Letting Z(t) denote the random variable that gives the number of cases at time t, we are first interested in obtaining an expression for the probability generating function', 'cite_spans': [{'start': 796, 'end': 800, 'text': '[41]', 'ref_id': 'BIBREF40'}], 'ref_spans': [], 'section': 'Extinction probabilities'}, {'text': 'It follows that Q(t, s) satisfies the differential equation', 'cite_spans': [], 'ref_spans': [], 'section': 'Extinction probabilities'}, {'text': 'subject to the initial condition Q(0, s) = s.', 'cite_spans': [], 'ref_spans': [], 'section': 'Extinction probabilities'}, {'text': 'Solving for Q and setting s = 0 gives the probability that, at time t, the number of cases has reached zero and the disease has become extinct. We denote this probability by q(t) [3] , which is given by', 'cite_spans': [{'start': 179, 'end': 182, 'text': '[3]', 'ref_id': 'BIBREF2'}], 'ref_spans': [], 'section': 'Extinction probabilities'}, {'text': '(β(t 1 )e I(t1) )dt 1 + e I(t) −1 .', 'cite_spans': [], 'ref_spans': [], 'section': 'Extinction probabilities'}, {'text': 'The above case considers a closed population. Since the virus has spread worldwide, for any population of interest, immigration of infected individuals cannot be ignored. To capture this, we model the case where immigration from external sources is introduced into the system at a rate η(t), and are similarly interested in the random variable Y (t), which denotes the number of cases at time t. The corresponding generating function, R(t, s), for this random variable satisfies', 'cite_spans': [], 'ref_spans': [], 'section': 'Extinction probabilities'}, {'text': 'Again, solving for R(t, s) and setting s = 0 gives the probability, r(t), that there are no cases of infected individuals left, at which time a new case can only arise through immigration from an external source. This probability is given by', 'cite_spans': [], 'ref_spans': [], 'section': 'Extinction probabilities'}, {'text': 'We simulate data based on one initial case n 0 = 1, though this may easily be extended to any number of initial cases. We run simulations both with and without immigration, choosing β(t) = 3/(7(1 + 5e −t )) and δ(t) = 1/7 for all t, so that an effective reproduction number given by β(t)/δ(t) grows gradually from 0.5 to 3', 'cite_spans': [], 'ref_spans': [], 'section': 'Extinction probabilities'}, {'text': 'after interventions are released, and choosing immigration rate η(t) = W 0 e −t , where W 0 is the initial (constant) rate of importation of cases before any controls on immigration are put into effect. We set W 0 = 5 imported cases per day. With these choices of parameters, the resulting extinction probabilities are given in Figure 10 .', 'cite_spans': [], 'ref_spans': [{'start': 328, 'end': 337, 'text': 'Figure 10', 'ref_id': 'FIGREF1'}], 'section': 'Extinction probabilities'}, {'text': 'Note that we are assuming the immigration rate is decreasing to 0, so if the infection is controlled internally for long enough, an overall ultimate extinction is possible in this model. For these parameter choices, the final probability of extinction, defined as lim t→∞ q(t) (without immigration) and lim t→∞ r(t) (with immigration) are approximately 0.446 and 0.002, respectively. It should be noted that q(t) concerns the best case scenario with only one initial case. Increasing the number of initial cases n 0 scales the probability of extinction by q(t) n0 . These probabilities suggest that, without widespread immunity, stochastic extinction might be aided by social distancing but is heavily compromised by immigration. Border controls, therefore, if of limited use when transmission is self-sustaining, become key when the number of cases is low. Note that we have assumed an importation function η(t) that goes to 0 for large t, in line with a pandemic that goes extinct in other geographical regions. However, the presence of an animal reservoir might lead to an importation function that is non-zero over longer time scales, thus effectively making ultimate extinction impossible unless the effective reproduction number is kept below one by a systematic and permanent intervention (e.g. technology-based change in behaviour) or herd immunity. Figure 10 : Extinction probabilities, both analytic and simulated, for the choices of β(t), δ(t) and η(t) described in the main text. The simulated extinction probabilities were calculated from 10,000 simulations of a birth-death chain both with and without immigration, for which the code can be obtained via the supplementary material.', 'cite_spans': [], 'ref_spans': [{'start': 1358, 'end': 1367, 'text': 'Figure 10', 'ref_id': 'FIGREF1'}], 'section': 'Extinction probabilities'}, {'text': 'Contact tracing is a complementary control policy to isolation or quarantine. When a case is discovered, attempts are made to identify and isolate individuals who may have been infected. In doing so, some of the secondary cases will be discovered and isolated early in their infection, decreasing their effective infectious period. If contact tracing is successful, it can greatly reduce the effective reproduction number of the infection, and in combination with other interventions may drive an epidemic extinct, as was seen in the case of SARS [87] .', 'cite_spans': [{'start': 547, 'end': 551, 'text': '[87]', 'ref_id': 'BIBREF87'}], 'ref_spans': [], 'section': 'Contact tracing and household isolation'}, {'text': 'Contact tracing in itself presents numerous challenges, which are exacerbated by its success relying not only on the effectiveness of the tracing process but also the underlying transmission characteristics. For COVID-19, some of these challenges include mild symptoms which cause infections not to be reported, pre-symptomatic transmission which occurs before a case is reported, and short generation times [31] which can cause the epidemic to outrun contact tracing. Additionally, contact tracing is only feasible for smaller case numbers, because each case generates multiple contacts to follow up, so the tracing workload expands dramatically, and an increasing number of chains remain unobserved. This makes it a viable strategy in the early days of an outbreak, or, if containment has failed, following a period of severe interventions, such as a lockdown. Combining contact tracing with isolation is being considered by many countries as part of a test, trace and isolate strategy to be implemented once lockdowns or comparable measures are lifted, provided these lockdowns succeed at driving case numbers sufficiently low. In this section, we develop a household-level contact tracing model for an emerging outbreak, since we do not wish to make assumptions about immunity or depletion of susceptibles. These assumptions can be added to the model as the availability of data into immunity improves. We are interested in the likelihood that the contact tracing process is overwhelmed by large case numbers and the likelihood that, combined with isolation, it can drive the disease to extinction.', 'cite_spans': [{'start': 408, 'end': 412, 'text': '[31]', 'ref_id': 'BIBREF30'}], 'ref_spans': [], 'section': 'Contact tracing and household isolation'}, {'text': \"The early days of an outbreak can be modelled using a branching process, where generations of infections produce infectious offspring. Contact tracing processes can be incorporated as a superinfection along the tree generated by the branching process [13] . When a node is 'superinfected' by the contact tracing process, it is isolated.\", 'cite_spans': [{'start': 251, 'end': 255, 'text': '[13]', 'ref_id': 'BIBREF12'}], 'ref_spans': [], 'section': 'Contact tracing and household isolation'}, {'text': 'We model the infection spreading through a fully susceptible population of individuals, segmented into households of different sizes according to the 2019 ONS survey [62] , and progress through discrete time steps of 1 day.', 'cite_spans': [{'start': 166, 'end': 170, 'text': '[62]', 'ref_id': None}], 'ref_spans': [], 'section': 'Contact tracing and household isolation'}, {'text': 'As such, our branching process is at the household level, coupled with localised within-household epidemics.', 'cite_spans': [], 'ref_spans': [], 'section': 'Contact tracing and household isolation'}, {'text': 'This allows us to model contact tracing strategies that isolate whole households, which may contain several undetected infections. It also enables a wider range of contact tracing strategies to be modelled, each with different intervention scope and costs.', 'cite_spans': [], 'ref_spans': [], 'section': 'Contact tracing and household isolation'}, {'text': 'Each day, individuals (or nodes) make contacts to a random set of individuals; divided into local contacts to members of the same household, and global contacts to members of other households. The number of individuals contacted in a day is distributed using an overdispersed negative binomial distribution and parameterised using estimates from the POLYMOD social contact survey [59] , stratified by household size. Since the probability that a contact causes infection cannot be directly observed, we use improper hazard rates that give rise to the 5 day COVID-19 generation time [28] and R 0 = 3.', 'cite_spans': [{'start': 380, 'end': 384, 'text': '[59]', 'ref_id': 'BIBREF59'}, {'start': 582, 'end': 586, 'text': '[28]', 'ref_id': 'BIBREF27'}], 'ref_spans': [], 'section': 'Contact tracing and household isolation'}, {'text': 'For contact tracing to begin, an infection must be diagnosed, which we assume occurs 70% of the time among infected individuals due to flaws in reporting or very mild symptoms in those infected. We assume a Gamma distributed incubation period with mean 4.84 (Table 5 ) and a Geometric reporting delay from symptom onset with mean 4.8 days [44] . Intuition suggests that if R 0 = 3 then tracing two thirds of contacts will control the epidemic. However, in practice transmission may occur before tracing, so this will not reduce the number of infectious contacts by two thirds. To demonstrate this, we assume that contact tracing successfully traces two thirds of contacts. Trained professionals have to trace all reported contacts from the last 14 days, so we assume that the contact tracing delay follows a Geometric distribution with a mean of 2 days. Individuals are considered recovered 21 days after infection, as the chances that they are still transmitting then are negligible.', 'cite_spans': [{'start': 339, 'end': 343, 'text': '[44]', 'ref_id': 'BIBREF43'}], 'ref_spans': [{'start': 258, 'end': 266, 'text': '(Table 5', 'ref_id': 'TABREF4'}], 'section': 'Contact tracing and household isolation'}, {'text': 'Though our general framework can be modified extensively, we assume the following contact tracing strategy.', 'cite_spans': [], 'ref_spans': [], 'section': 'Contact tracing and household isolation'}, {'text': 'When an individual reports infection, their household is immediately isolated. Contact tracing attempts are then made for all households connected to one of the individuals in this household, whether symptomatic or not. When a connected household is identified (after the contact tracing delay), all individuals within the household are immediately placed under observation. If any of the individuals in the observed households develop symptoms, then the household becomes isolated and the contact tracing process continues to connected households. When a household is isolated, we assume all individuals are isolated with 100% adherence, and cannot transmit the virus within or outside the household. The assumption that isolation prevents local infections is unrealistic, but does not change the overall behaviour of the process as there are no more global infections. This strategy imposes high individual-level cost, since by isolating all individuals within a household, it isolates individuals who have not had direct contact with an infected individual. In practice, such a strategy may have poor adherence. Figure 12a shows an example contact tracing network.', 'cite_spans': [], 'ref_spans': [{'start': 1115, 'end': 1125, 'text': 'Figure 12a', 'ref_id': 'FIGREF1'}], 'section': 'Contact tracing and household isolation'}, {'text': 'When choosing contact tracing strategies, a balance must be struck between the effectiveness of a strategy and the resources that it requires. Some strategies are only feasible when there are few infections, since the resources required can grow rapidly depending on the dynamics of the outbreak and the contact tracing process.', 'cite_spans': [], 'ref_spans': [], 'section': 'Hitting times of contact tracing capacities'}, {'text': 'To define the capacity of the contact tracing process, we consider the ability of a public health agency to observe the condition of those asked to self-isolate, due to their recent exposure to an infected individual. The health agency must remain in contact for the duration of the 14 day self-isolation period, so that if any individual under isolation develops symptoms and then tests positive, the contact tracing process can be initiated on this node. We will define the capacity of the contact tracing process to be the number of people that can be placed under observation and assume two possible capacities: 800 and 8000. We assume that when a node is contact traced, they are asked to report their global contacts for the last 14 days. All global contacts are assumed to be to a new person since we are in the early stages of an outbreak. We carried out 6507 simulations of the contact tracing process for 150 days. Contact tracing capacity was reached in 5000 simulations, and in 180 the epidemic neither went extinct nor was the 8000 capacity reached. In the remaining simulations, the epidemic went extinct. Figure 11a and Table 7 show that increasing the contact tracing capacity tenfold less than doubles the time until that capacity is reached. However, it does increase the odds of driving the epidemic to extinction without hitting the capacity by about 10% (Table 7) . Different contact tracing strategies will strain different aspects of the health agency. A strategy that generates large amounts of work is only feasible if there are few active infections. The optimal strategy will need to compromise and may need to change depending on the number of active infections, which cannot be directly observed.', 'cite_spans': [], 'ref_spans': [{'start': 1120, 'end': 1130, 'text': 'Figure 11a', 'ref_id': 'FIGREF1'}, {'start': 1135, 'end': 1142, 'text': 'Table 7', 'ref_id': 'TABREF6'}, {'start': 1375, 'end': 1384, 'text': '(Table 7)', 'ref_id': 'TABREF6'}], 'section': 'Hitting times of contact tracing capacities'}, {'text': 'When there is a small number of cases in a single country, it may be possible to drive the pathogen to extinction.', 'cite_spans': [], 'ref_spans': [], 'section': 'Extinction Time'}, {'text': 'This small case number could correspond to the start of an outbreak or removing of severe interventions. We consider the latter case, but conservatively assume a fully susceptible population.', 'cite_spans': [], 'ref_spans': [], 'section': 'Extinction Time'}, {'text': 'We assume that social distancing is enforced on day 0 and reduces global contacts by 70%. Since we are interested in extinction, we will no longer consider the contact tracing capacity. Under these baseline parameter assumptions and 10000 simulations, the combined force of this contact tracing strategy and isolation is enough to drive the epidemic extinct (Figure 12b ), but measures will need to be in place for months in some cases.', 'cite_spans': [], 'ref_spans': [{'start': 358, 'end': 369, 'text': '(Figure 12b', 'ref_id': 'FIGREF1'}], 'section': 'Extinction Time'}, {'text': 'If the infection is ever re-imported, then the process would begin again, since herd immunity is not achieved. (a) A visualization of the branching process model after 20 days. Node numbers represent the household that they belong to. Nodes 3, 7 and 18 will be contact traced but the process has not reached them yet, due to contact tracing delays. Node 4 has been contact traced, but has not been quarantined in the plotted scenario, as it does not display symptoms.', 'cite_spans': [], 'ref_spans': [], 'section': 'Extinction Time'}, {'text': '(b) Distribution of extinction times for epidemics that transmit the infection at least once. The distribution takes the shape of a generalised extreme value distribution, and there are several epidemics that still have not gone extinct after interventions have been in effect for long periods of time, which exemplifies how difficult it can be to make an epidemic extinct. Additionally, this model only considers extinction under the assumption that no cases are imported. In Section 3.4, we have shown that importation of cases significantly reduces the extinction probability. This suggests that extinction may no longer be guaranteed, and the time to extinction will be significantly increased. This analysis has focused on a single contact tracing strategy using indicative parameters for COVID-19. The proposed model can be extended to more strategies and region specific parameters to inform the design of control policies. Also, as is shown in Table 7 , contact tracing capacity is likely to be reached, which may prevent extinction from being achieved. This complication is compounded by the issues of loss of immunity or the presence of an animal reservoir discussed in Section 3.4.', 'cite_spans': [], 'ref_spans': [{'start': 952, 'end': 959, 'text': 'Table 7', 'ref_id': 'TABREF6'}], 'section': 'Extinction Time'}, {'text': 'In this manuscript we have presented a range of mathematical tools to tackle infectious disease outbreaks.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'In particular, these tools address various technical questions posed by the authors to support the ongoing public health response to COVID-19. This toolkit considers both estimation efforts for key parameters, and investigative efforts (often numerical simulations) in gauging the effectiveness of various intervention or control measures. Joint consideration of estimation and simulation efforts is critical. Parameter estimates are obtained using a certain set of assumptions regarding the data, and investigations or simulations utilising these estimates should ensure that their underlying assumptions are consistent. These challenges in model construction and applicability of statistical methods are compounded by the limitations of the data with which decisions must be made. Some of the biases present in the data can be addressed with an improved data collection methodology -often challenging in the context of a fast-moving outbreak -but many are also inherent to the nature of early outbreak data [16, 47] . The consequent lack of intuitive insight from this data underscores the need for careful parametric estimates, especially considering the large variability in predicted outcomes resulting from small differences in parameters. Even with robust estimates for some parameters, many other parameters are challenging to estimate using the available data. Therefore, models need to address this variability and uncertainty in order to inform public health policy.', 'cite_spans': [{'start': 1009, 'end': 1013, 'text': '[16,', 'ref_id': 'BIBREF15'}, {'start': 1014, 'end': 1017, 'text': '47]', 'ref_id': 'BIBREF47'}], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'We have presented methods to address biases arising from a growing force of infection, changes in the reporting rate, truncated data samples and a varying travel rate. We use these methods to account for these biases when estimating delay distributions, such as the incubation period and hospitalisation-to-death delay, and the growth rate/doubling time (results given in Table 5 ). These biases can have significant impact when estimating key parameters: the mean incubation period estimates for COVID-19 range from 3.48 days without correcting for truncation to 4.84 days with the correction, and the doubling time in Hubei province decreases from 3.15 days without correcting for travel to 2.77 days. These differences can significantly alter our understanding of the outbreak, and could have a large impact on policy and public health. For instance, underestimating the incubation period may lead to quarantine strategies failing to identify infected individuals if the quarantine length is too short. Overestimating the doubling time (or underestimating the growth rate) will underestimate the risk posed to the host population -both in terms of final size of the epidemic and the rate at which it spreads, which can have significant public health impacts as discussed in [68] .', 'cite_spans': [{'start': 1277, 'end': 1281, 'text': '[68]', 'ref_id': None}], 'ref_spans': [{'start': 372, 'end': 379, 'text': 'Table 5', 'ref_id': 'TABREF4'}], 'section': 'Discussion'}, {'text': 'It is important to note that the above-mentioned biases, and consequent impact of implementing the methods correcting for their presence, may vary across setting. As an example, the potential underestimation of the When an outbreak occurs in an enclosed group, such as a large gathering, we may wish to know how many individuals are likely to be infected. We developed a statistical method to estimate the first generation size based on the number of symptomatic individuals, taking care to account for the uncertainty in this quantity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'This ready reckoner can inform testing of large groups to help control the disease spread, but does not apply to later generations or the possible interventions enacted on the population.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'Building on these enclosed population scenarios, we have developed a set of models that investigate public con-trol measures or interventions on enclosed populations, such as households and care homes. These structured descriptions improve the population risk profiles relative to assumptions of homogeneous mixing. A complementary aspect to a structured population when modelling interventions is adherence. Motivated by vaccination modelling, we consider leaky adherence, where every individual/household chooses to adhere or not whenever an event occurs, and all-or-nothing adherence, where some households adhere every time and some never adhere.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'We observed that in a homogeneous population, although the two types of adherence predict the same growth rate and final size, the timing of the peak and the early growth can be faster under all-or-nothing adherence.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'This insight, combined with lessons from the vaccination literature, suggests that efforts should focus on ensuring complete adherence in individuals/households with some level of pre-existing adherence, rather than pushing non-adherent individuals/households to change behaviour.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'Dedicated modelling of disease spread in care homes is essential due to the documented history of co-morbidity of their residents during pandemics [34, 88, 89, 90, 91] . We do so by regarding care homes as closed populations that are subjected to a force of infection from an external epidemic. We develop a tool for analysing the risk posed to this population by determining the peak size of the epidemic within the care homes and the number of deaths. Applying this model to COVID-19, we find that by \"cocooning\" the care homes, i.e. shielding them to reduce the chance of introduction from the external outbreak, we can significantly reduce the size of the peak and therefore reduce the number of deaths. However, assessing the necessary level of shielding requires accurate characterisation of the external force of infection, and underestimating this may invalidate shielding efforts. A limitation to the proposed model is the deterministic within care home epidemic. However, since the average size of care homes is relatively large and we assume a high R 0 within care homes, the deterministic assumption is unlikely to significantly alter the conclusions.', 'cite_spans': [{'start': 147, 'end': 151, 'text': '[34,', 'ref_id': 'BIBREF33'}, {'start': 152, 'end': 155, 'text': '88,', 'ref_id': 'BIBREF88'}, {'start': 156, 'end': 159, 'text': '89,', 'ref_id': 'BIBREF89'}, {'start': 160, 'end': 163, 'text': '90,', 'ref_id': 'BIBREF90'}, {'start': 164, 'end': 167, 'text': '91]', 'ref_id': 'BIBREF91'}], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'When modelling households however, we are concerned with much smaller population sizes. Therefore, it is important to consider stochastic effects within each household, combined with between-household dynamics. We consider two different household models: one which contains features of both within-and between-household transmission, where small-scale transmission can be linked to the epidemic on a population level, and another which facilitates more detail in the within-household transmission and delay distributions, but with reduced correspondence to the population-wide transmission. With the first model, a 65% adherence to household isolation appears insufficient to control the epidemic without severe global reductions in transmission. Coupled with a short term global reduction, the epidemic can be controlled, but upon lifting the global intervention, which could take the form of a lockdown, household isolation is insufficient to maintain control. For the second model, we look into changing the strength of adherence, and the impact this can have on achieving control.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'Indicative but reasonable parameter values suggest that the COVID-19 outbreak can potentially be controlled using household isolation strategies, provided the level of adherence is sufficiently high. However, such a high level of adherence may be difficult to maintain in the long-term and this modicum of control is anyway highly sensitive to the chosen parameters. We further investigated the efficacy of various isolation or quarantine measures. A policy of individual isolation struggles to curtail the epidemic for any adherence. Instead, mixed isolation, whereby first the whole household isolates and any individual infected during isolation goes on to self isolate after household isolation is lifted, appears to be the most cost-effective strategy.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'Countries have put into place strict social distancing and lockdown intervention to suppress or regain control of epidemics that threaten to overwhelm the health system and cause massive mortality, but they cannot be sustained in the long term without growing social and economic costs. We have shown however, that the probability of the epidemic becoming extinct once these policies are lifted, even when very few cases remain, is very small. We therefore consider a contact tracing intervention as a potential strategy for managing the COVID-19 outbreak, once severe lockdown interventions are lifted. We developed a household-level contact tracing model to explore the feasibility of combining these strategies to control the epidemic. Firstly, we noted that by using knowledge of household structure, we can reduce the burden on the contact tracing process by isolating household and removing them from the contact tracing process once an infected member has been identified. Secondly, we investigated how contact tracing combined with household isolation may drive the disease to extinction, finding that aggressive contact tracing coupled with household isolation can drive the epidemic to extinction under the indicative parameters assumed, when starting with a single infection.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'However, the time until extinction can be impractically long, which risks the contact tracing capacity being overwhelmed, suggesting such a strategy may be infeasible in practice. A less aggressive strategy could be implemented that would be less likely to overwhelm local health agencies. Whilst it may not lead to extinction, this can still be beneficial at mitigating and controlling the spread of an outbreak as part of a test, trace and isolate strategy.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'There are many complexities when modelling an outbreak of a novel infectious disease. To address some of these, we have described a variety of techniques to serve as part of a generally applicable toolkit. However, our proposed models, and many other models, are subject to important limitations which must be considered prior to their application. Key among these is the lack of heterogeneous population mixing, such as through age-stratification [66] and different risk-groups [83] , and spatio-temporal variations [45] , all of which influence modelling estimates and predictions. Nevertheless, the relative simplicity of the presented models allows for the development of qualitative intuition regarding the efficacy of various intervention methods, whilst providing tractable theoretical frameworks which can be further developed and better inform policy-makers.', 'cite_spans': [{'start': 448, 'end': 452, 'text': '[66]', 'ref_id': 'BIBREF66'}, {'start': 479, 'end': 483, 'text': '[83]', 'ref_id': 'BIBREF83'}, {'start': 517, 'end': 521, 'text': '[45]', 'ref_id': None}], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'Appendix A Independence of the incubation period likelihood function and the reporting rate', 'cite_spans': [], 'ref_spans': [], 'section': 'Appendices'}, {'text': 'To estimate the incubation period distribution, we need to find the distribution that maximises the probability of observing the sampled data. However, the sampled data does not directly record incubation period, and instead contains infection exposure window and the symptom onset date. Additionally, the sample does not contain all individuals, and therefore there is a reporting rate that must be incorporated into the likelihood function. If the reporting rate is constant, it can be ignored. However, in the data coming out the Wuhan, the reporting rate varies significantly, since individuals are no longer exported from Wuhan after travel restrictions.', 'cite_spans': [], 'ref_spans': [], 'section': 'Appendices'}, {'text': 'Since the reporting rate depends on individuals leaving Wuhan, the main factor effecting the probability that a case is included in the data set is the date an individual leaves Wuhan. Therefore, the reporting rate depends on the days an individual spends in Wuhan and is independent of their symptom onset date. We need the likelihood function that an individual was infected between days a and b, had symptom onset on day y, and was included in the data set.', 'cite_spans': [], 'ref_spans': [], 'section': 'Appendices'}, {'text': 'We will condition against the infection window, a < I < b, the case being included in the data set, x ∈ D, and symptom onset occurring before the truncation date, Y < T , and determine the probability that for such an individual we observe the given symptom onset date, Y = y. That is, we need to find P (Y = y|{a < I < T', 'cite_spans': [], 'ref_spans': [], 'section': 'Appendices'}, {'text': 'Therefore, the likelihood function P (Y = y|a < I < b, Y ≤ T ) is independent of the reporting rate for the data coming out of Wuhan in the early days of the outbreak.', 'cite_spans': [], 'ref_spans': [], 'section': 'Appendices'}, {'text': 'In the analysis in Section 2.4, it is assumed that every person who has developed symptoms by time τ is known to the observer. However, depending on the disease, symptoms can be subjective. One person may not notice something another person may visit hospital for. Additionally, one person may not want to come forward with symptoms if they are worried about the repercussions of coming forward (for example being isolated against their own will).', 'cite_spans': [], 'ref_spans': [], 'section': 'Appendix B Estimating the generation size based using a lower bound on the number of symptomatic individuals'}, {'text': 'To address this, rather than considering I τ as the total number of people who have developed symptoms by time τ , we can considerĨ τ as the number of people who have presented with symptoms by time τ . We do no know the true value of I τ , but we know that it cannot be belowĨ τ . We assume that the probability ofĨ τ being i τ for a given value of i τ is uniform at 1 iτ +1 . We can then use the same methods as above to infer a distribution for P . P Ĩ τ =ĩ τ I τ = i τ = 1 iτ +1 , if 0 ≤ĩ τ ≤ i τ 0, otherwise B.1 Inferring a distribution for P givenĨ τ =ĩ τ', 'cite_spans': [], 'ref_spans': [], 'section': 'Appendix B Estimating the generation size based using a lower bound on the number of symptomatic individuals'}, {'text': 'We again assume an uninformative uniform prior distribution for P and as a result the posterior distribution for P is proportional to the likelihood thatĨ τ =ĩ τ given P = p:', 'cite_spans': [], 'ref_spans': [], 'section': 'Appendix B Estimating the generation size based using a lower bound on the number of symptomatic individuals'}, {'text': 'n−ĩτ 2 F 1 1,ĩ τ − n,ĩ τ + 2, −pF (τ ) 1 − pF (τ )', 'cite_spans': [], 'ref_spans': [], 'section': 'Appendix B Estimating the generation size based using a lower bound on the number of symptomatic individuals'}, {'text': 'where c is a normalising constant such that', 'cite_spans': [], 'ref_spans': [], 'section': 'Appendix B Estimating the generation size based using a lower bound on the number of symptomatic individuals'}, {'text': 'n−ĩτ 2 F 1 1,ĩ τ − n,ĩ τ + 2, −pF (τ ) 1 − pF (τ ) dp However, this integral cannot be found analytically and instead must be calculated numerically.', 'cite_spans': [], 'ref_spans': [], 'section': 'Appendix B Estimating the generation size based using a lower bound on the number of symptomatic individuals'}, {'text': 'B.2 Inferring a distribution for E 0 givenĨ τ =ĩ τ', 'cite_spans': [], 'ref_spans': [], 'section': 'Appendix B Estimating the generation size based using a lower bound on the number of symptomatic individuals'}, {'text': 'We can use the same probability analysis in Section 2.4 to write the following probability formula:', 'cite_spans': [], 'ref_spans': [], 'section': 'Appendix B Estimating the generation size based using a lower bound on the number of symptomatic individuals'}, {'text': '(1 − pF (τ )) n−ĩτ 2 F 1 1,ĩ τ − n,ĩ τ + 2, −pF (τ ) 1−pF (τ ) ×pĩ τ (1 − pF (τ )) n−ĩτ 2 F 1 1,ĩ τ − n,ĩ τ + 2, ', 'cite_spans': [], 'ref_spans': [], 'section': 'Appendix B Estimating the generation size based using a lower bound on the number of symptomatic individuals'}, {'text': \"To capture the essential features of household quarantine, we need to add states to the dynamical variables. Let Q n,s,e,p,i,f (t) be the proportion of households in the population at time t of size n, with s susceptibles, e exposed, p prodromal, and i symptomatic infectious individuals, and with vector of 'flags' f representing implementation of more complex interventions.\", 'cite_spans': [], 'ref_spans': [], 'section': 'Appendix B Estimating the generation size based using a lower bound on the number of symptomatic individuals'}, {'text': 'We now suppose that a fraction α S of households start to isolate when there is at least one symptomatic case in the household, and stop isolation 14 days after the absence of symptoms in the household. This is modelled by having a flag f = 0 if the household is not isolating and f = 1 if it is. The dynamics become d dt Q n,s,e,p,i,f = − 1 {f =0} sr s→e (t, Q) + er e→p + pr p→i + ir i→∅ + n −η spτ p + n −η siτ i + 1 {f =1} σ Q n,s,e,p,i,f + 1 {f =0} (s + 1)r s→e (t, Q)Q n,s+1,e−1,p,i,f + (e + 1)r e→p Q n,s,e+1,p−1,i,f + (1 − 1 {f =1&i=1&s+e+p=n−1} α S )(p + 1)r p→i Q n,s,e,p+1,i−1,f + (i + 1)r i→∅ Q n,s,e,p,i+1,f + n −η (s + 1)pτ p Q n,s+1,e−1,p,i,f + n −η (s + 1)iτ i Q n,s+1,e−1,p,i,f + 1 {f =0} σQ n,s,e,p,i,1 + 1 {f =1&i=1&s+e+p=n−1} α S (p + 1)r p→i Q n,s,e,p+1,i−1,0 , Cumulative infection pressure', 'cite_spans': [], 'ref_spans': [], 'section': 'Appendix B Estimating the generation size based using a lower bound on the number of symptomatic individuals'}, {'text': \"No control 7d Individual Isolation 14d Household Isolation (7, 14) d Mixed Isolation 14d Household Quarantine Figure S1 : Accumulated infection pressure in the simulation presented in Figure 8 for different control policies. Horizontal dotted lines represent individuals' resilience thresholds. As time progresses, the accumulated infection pressures (coloured lines) increase and when they cross the resilience thresholds, the corresponding individual acquires infection. Notice that: in the absence of control, one individual escapes infection; with household isolation only, the infection pressure reaches a relatively low endpoint because of the last symptomatic individual slipping through and not transmitting much in the household; with mixed isolation, infection pressure is higher due to combined adherence; and with household quarantine, the infection pressure builds up more slowly at the beginning due to lower adherence.\", 'cite_spans': [{'start': 59, 'end': 62, 'text': '(7,', 'ref_id': 'BIBREF6'}, {'start': 63, 'end': 66, 'text': '14)', 'ref_id': 'BIBREF13'}], 'ref_spans': [{'start': 110, 'end': 119, 'text': 'Figure S1', 'ref_id': 'FIGREF1'}, {'start': 184, 'end': 192, 'text': 'Figure 8', 'ref_id': 'FIGREF14'}], 'section': 'Accumulated infection pressure and individual resilience thresholds (dotted)'}, {'text': 'with between-household term r s→e (t, Q) = Λ(t) + ', 'cite_spans': [], 'ref_spans': [], 'section': 'Accumulated infection pressure and individual resilience thresholds (dotted)'}, {'text': 'The non-Markovian household model of Section 3.3.2 relies on the Sellke construction. In relation to Figure 8 , we report in Figure S1 the infection pressure that accumulates for the different control policies. ', 'cite_spans': [], 'ref_spans': [{'start': 101, 'end': 109, 'text': 'Figure 8', 'ref_id': 'FIGREF14'}, {'start': 125, 'end': 134, 'text': 'Figure S1', 'ref_id': 'FIGREF1'}], 'section': 'Appendix D Non-Markovian household model of within-household transmission'}, {'text': 'CO and HS compiled the manuscript. All authors were involved in the research and revising of the manuscript.', 'cite_spans': [], 'ref_spans': [], 'section': 'Author contributions'}], 'paragraphs': [{'text': 'During an outbreak, many parameters depend on delay distributions (the length of time between two events), such as the time from infection to symptoms onset (the incubation period). If an individual can be followed indefinitely, it is easy to determine the length of these events. However, in reality only events that occur before a given date are observed. Therefore, the data is subject to censoring and truncation issues. In the incubation period, for example, censoring comes into play since, if we have observed an infection but the individual has not yet developed symptoms, we only have a lower bound on how long it will take them to develop symptoms. To account for this, we can instead condition on observing symptom onset before the cut-off date. However, this leads to a truncation issue, since individuals who were infected close to the cut-off date will only be observed if they have a short incubation period, which leads to an overexpression of short delays.', 'id': '00015'}], 'title': 'Using statistics and mathematical modelling to understand infectious disease outbreaks: COVID-19 as an example', 'sha': '023f34678b688f43348eabe8a71364850d3996b3', 'doi': '', 'score': 9.54279899597168}, {'rank': 74, 'paragraphs': [{'text': 'The copyright holder for this preprint this version posted June 16, 2020. . https://doi.org/10.1101/2020.06.14.20131177 doi: medRxiv preprint can have a large impact on the MLE for f X . To mitigate the effect of this noise, we introduce a stopping rule in which an estimate f n is \"accepted\" once the MLE changes by less than 5% over a 3-day period. We use this day-by-day MLE approach to estimate the value of f 1 in the period following the introduction of distancing recommendations in British Columbia, using daily case counts reported between March 1st and April 22nd, 2020. We find a credible band by simulating the SEIR model with 50 different values of the baseline R 0 parameter, normally distributed about the mean R 0 = 2.57 estimated by a pre-distancing model fit, and with standard deviation 0.05.', 'id': '00021'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.52750015258789}, {'rank': 75, 'paragraphs': [{'text': 'Given the continuing global spread of COVID-19, a further investigation of viral incubation by a systematic review and meta-analysis could provide urgently needed support to improve the understanding of COVID-19 transmission potential and aid prediction and decision-making.', 'id': '00005'}], 'title': 'A systematic review and meta-analysis reveals long and dispersive incubation period of COVID-19', 'sha': '3d921c03dc23ee850655ddfd9d5ebe3956d70bf8', 'doi': '10.1101/2020.06.20.20134387', 'score': 9.527199745178223}, {'rank': 76, 'paragraphs': [{'text': 'The copyright holder for this preprint this version posted June 22, 2020. . https://doi.org/10.1101/2020.06. 20.20134387 doi: medRxiv preprint In addition, age is likely to have a positive relationship with the length of incubation period. Notably, the result in our study indicated that older adults had a longer incubation period than younger adults, which was consistent with the findings of previous studies (28, 29) . Older adults tend to have more health complications such as respiratory issues and chronic diseases; thus, pre-existing symptoms may mask the onset of COVID-19 symptoms, which could bias the measurement of incubation period.', 'id': '00032'}], 'title': 'A systematic review and meta-analysis reveals long and dispersive incubation period of COVID-19', 'sha': '3d921c03dc23ee850655ddfd9d5ebe3956d70bf8', 'doi': '10.1101/2020.06.20.20134387', 'score': 9.524999618530273}, {'rank': 77, 'paragraphs': [{'text': \"The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.24.20027474 doi: medRxiv preprint their incubation periods groups using different visualization tools. It indicates that the younger group tends to have a shorter incubation period. The variance of their incubation period also seems to be smaller. The Mann-Whitney rank test shows that there's a statistically significant difference between the incubation of age<40 and age>=40 groups with the null hypothesis: the medians of incubation period between two groups are the same. The p-values for corresponding alternative hypotheses: the age<40 group has a smaller incubation median is 0.00474. It suggests that the age<40 group has a shorter incubation period than the age>=40 group. Similarly, the Siegal-Tukey test indicates the younger group's incubation variance is less than the older group's by p-value: 0.0083. It may suggest COVID-19 has a faster but relatively constant spread speed among people <40 years old than people >=40 years old. Figure 4 illustrates biplots of the dataset by removing two cases with missing items by using PCA (principal component analysis), sparse PCA (sparse principal component analysis, t-SNE (t-distributed stochastic neighbor embedding), and LLE (locally linear embedding) [16] [17] [18] [19] . Data is partitioned as the age>=40 and age<40 groups in visualization. t-SNE shows that only one case in the age>=40 group falls in the cluster of the age < 40 group. But PCA, sparse PCA and LLE all show that the incubation of two groups is linearly separable, which means there exists an obvious linear boundary to separate them, in the subspaces generated by PCA, SPCA, t-SNE, or LLE. Such machine learning results indicate that the two groups are actually independent clusters spatially. However, we also find that the incubation data will no longer demonstrate the linear separability property when we partition it as age>=50 and age<50 groups or age>=55 All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\", 'id': '00026'}], 'title': 'Estimate the incubation period of coronavirus 2019 (COVID-19)', 'sha': 'c1ae608c7ffb926a0f50a6a34c0780983274ea74', 'doi': '10.1101/2020.02.24.20027474', 'score': 9.524399757385254}, {'rank': 78, 'paragraphs': [{'text': 'Here w c represents the distribution of delay between the onset of symptoms and a positive test being reported and ψ t is the fraction of eligible cases on day t that will be tested and reported. The upper limit on the integral, M , represents the maximum possible delay in days between onset and reporting, for which we use twice the mean delay. We seek to determine how soon changes in distancing measures will cause substantial changes in the number of prevalent cases. And then, determine how well we may be able to estimate the strength of these measures given the delay and noise in the reported case counts and uncertainty about the model parameters. Given a set of daily case count data {C t }, where C t is the number of cases identified on day t, we use a negative binomial likelihood with mean µ t and dispersion parameter φ, NB(C t | µ t , φ) as in [2] , to write the likelihood of the data given the model parameters accounting for dispersion:', 'id': '00010'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.520099639892578}, {'rank': 79, 'paragraphs': [{'text': 'COVID-19 studies that described either individual-level incubation period data or summarized statistics for central tendency and dispersion measures of incubation period were recruited. Studies were excluded if they met either of the below criteria: (1) irrelevant subject to incubation period; (2) no individual-level incubation period or insufficient summarized statistics for incubation period (central tendency and dispersion measures are required); (3) non-human studies; (4) sample size for incubation analysis less than 5; (5) studies of insufficient quality; (6) ambiguous definition of incubation period. Figure 1 describes the literature searching steps. Duplicate studies and studies irrelevant to incubation period were deleted, and studies identified via reference list searches were added. Two reviewers then selected 10% of the retrieved articles at random and independently reviewed the title and abstract according to the predefined set of exclusion criteria. Exclusion criteria were applied consistently, indicating high concordance. In case of uncertainty about inclusion or exclusion, the reviewers consulted together.', 'id': '00007'}], 'title': 'A systematic review and meta-analysis reveals long and dispersive incubation period of COVID-19', 'sha': '3d921c03dc23ee850655ddfd9d5ebe3956d70bf8', 'doi': '10.1101/2020.06.20.20134387', 'score': 9.508299827575684}, {'rank': 80, 'paragraphs': [{'text': 'In addition to estimating the first time at which we would expect a substantial difference in incidence to occur after a change in physical distancing, we also aim to understand the time it would take to observe such a difference in reported case counts. We use a likelihood-based approach to estimate if a change in distancing measures is yet having an effect on observed case counts, and to estimate the size of this effect.', 'id': '00019'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.493300437927246}, {'rank': 81, 'paragraphs': [{'text': 'The copyright holder for this preprint this version posted June 16, 2020. . https://doi.org/10.1101/2020.06.14.20131177 doi: medRxiv preprint 12 . CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'id': '00031'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.49329948425293}, {'rank': 82, 'paragraphs': [{'text': 'The copyright holder for this preprint this version posted June 16, 2020. Results are coloured grey before the MLE is accepted. Simulated daily case counts are also provided for reference (top panels). 14 . CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'id': '00036'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.493298530578613}, {'rank': 83, 'paragraphs': [{'text': 'The above results indicate that uncertainty about the model parameters substantially increases the number of days until definitive statements about the effects of relaxing social distancing can be made, with the total range of the number of days until effects are seen varying between 20 and 70.', 'id': '00050'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.493297576904297}, {'rank': 84, 'paragraphs': [{'text': 'Average age and male proportion were extracted from 24 of 56 studies. Meta-regression incorporating two moderators simultaneously was used to explore the impact of individual characteristics on length of incubation period. A linear relationship was identified between age and log-scaled median of incubation period. Average age per 10-year increments resulted in a 16% increment in median incubation period with adjustment for male proportion (incubation period ratio, 1.16, 95%CI 1.01-1.32; P = 0.0250) (Supplement Figure 6A) . No evidence indicated an association between sex and median of incubation period (P = 0.1315) (Supplement Figure 6B ).', 'id': '00024'}], 'title': 'A systematic review and meta-analysis reveals long and dispersive incubation period of COVID-19', 'sha': '3d921c03dc23ee850655ddfd9d5ebe3956d70bf8', 'doi': '10.1101/2020.06.20.20134387', 'score': 9.490300178527832}, {'rank': 85, 'paragraphs': [{'text': 'Having COVID \"How long does it take to recover?\" \"How COVID attacks the body?\" \"How long is the incubation period for COVID?\"', 'id': '00047'}], 'title': 'What Are People Asking About COVID-19? A Question Classification Dataset', 'sha': '09dda196ad420d29bc5993025d5a9426db1d867a', 'doi': '', 'score': 9.486900329589844}, {'rank': 86, 'abstract': '', 'body_text': [{'text': 'Governments will not be able to minimise both deaths from coronavirus disease 2019 (COVID-19) and the economic impact of viral spread. Keeping mortality as low as possible will be the highest priority for individuals; hence governments must put in place measures to ameliorate the inevitable economic downturn. In our view, COVID-19 has developed into a pandemic, with small chains of transmission in many countries and large chains resulting in extensive spread in a few countries, such as Italy, Iran, South Korea, and Japan.1 Most countries are likely to have spread of COVID-19, at least in the early stages, before any mitigation measures have an impact.', 'cite_spans': [{'start': 527, 'end': 528, 'mention': '1', 'ref_id': 'BIBREF0'}], 'section': '', 'ref_spans': []}, {'text': 'What has happened in China shows that quarantine, social distancing, and isolation of infected populations can contain the epidemic.1 This impact of the COVID-19 response in China is encouraging for the many countries where COVID-19 is beginning to spread. However, it is unclear whether other countries can implement the stringent measures China eventually adopted. Singapore and Hong Kong, both of which had severe acute respiratory syndrome (SARS) epidemics in 2002–03, provide hope and many lessons to other countries. In both places, COVID-19 has been managed well to date, despite early cases, by early government action and through social distancing measures taken by individuals.', 'cite_spans': [{'start': 132, 'end': 133, 'mention': '1', 'ref_id': 'BIBREF0'}], 'section': '', 'ref_spans': []}, {'text': 'The course of an epidemic is defined by a series of key factors, some of which are poorly understood at present for COVID-19. The basic reproduction number (R0), which defines the mean number of secondary cases generated by one primary case when the population is largely susceptible to infection, determines the overall number of people who are likely to be infected, or more precisely the area under the epidemic curve. For an epidemic to take hold, the value of R0 must be greater than unity in value. A simple calculation gives the fraction likely to be infected without mitigation. This fraction is roughly 1–1/R0. With R0 values for COVID-19 in China around 2·5 in the early stages of the epidemic,2 we calculate that approximately 60% of the population would become infected. This is a very worst-case scenario for a number of reasons. We are uncertain about transmission in children, some communities are remote and unlikely to be exposed, voluntary social distancing by individuals and communities will have an impact, and mitigation efforts, such as the measures put in place in China, greatly reduce transmission. As an epidemic progresses, the effective reproduction number (R) declines until it falls below unity in value when the epidemic peaks and then decays, either due to the exhaustion of people susceptible to infection or the impact of control measures.', 'cite_spans': [{'start': 704, 'end': 705, 'mention': '2', 'ref_id': 'BIBREF11'}], 'section': '', 'ref_spans': []}, {'text': 'The speed of the initial spread of the epidemic, its doubling time, or the related serial interval (the mean time it takes for an infected person to pass on the infection to others), and the likely duration of the epidemic are determined by factors such as the length of time from infection to when a person is infectious to others and the mean duration of infectiousness. For the 2009 influenza A H1N1 pandemic, in most infected people these epidemiological quantities were short with a day or so to infectiousness and a few days of peak infectiousness to others.3 By contrast, for COVID-19, the serial interval is estimated at 4·4–7·5 days, which is more similar to SARS.4\\n', 'cite_spans': [{'start': 564, 'end': 565, 'mention': '3', 'ref_id': 'BIBREF12'}, {'start': 673, 'end': 674, 'mention': '4', 'ref_id': 'BIBREF13'}], 'section': '', 'ref_spans': []}, {'text': 'First among the important unknowns about COVID-19 is the case fatality rate (CFR), which requires information on the denominator that defines the number infected. We are unaware of any completed large-scale serology surveys to detect specific antibodies to COVID-19. Best estimates suggest a CFR for COVID-19 of about 0·3–1%,4 which is higher than the order of 0·1% CFR for a moderate influenza A season.5\\n', 'cite_spans': [{'start': 325, 'end': 326, 'mention': '4', 'ref_id': 'BIBREF13'}, {'start': 404, 'end': 405, 'mention': '5', 'ref_id': 'BIBREF14'}], 'section': '', 'ref_spans': []}, {'text': 'The second unknown is the whether infectiousness starts before onset of symptoms. The incubation period for COVID-19 is about 5–6 days.4, 6 Combining this time with a similar length serial interval suggests there might be considerable presymptomatic infectiousness (appendix 1). For reference, influenza A has a presymptomatic infectiousness of about 1–2 days, whereas SARS had little or no presymptomatic infectiousness.7 There have been few clinical studies to measure COVID-19 viraemia and how it changes over time in individuals. In one study of 17 patients with COVID-19, peak viraemia seems to be at the end of the incubation period,8 pointing to the possibility that viraemia might be high enough to trigger transmission for 1–2 days before onset of symptoms. If these patterns are verified by more extensive clinical virological studies, COVID-19 would be expected to be more like influenza A than SARS. For SARS, peak infectiousness took place many days after first symptoms, hence the success of quarantine of patients with SARS soon after symptoms started7 and the lack of success for this measure for influenza A and possibly for COVID-19.', 'cite_spans': [{'start': 135, 'end': 136, 'mention': '4', 'ref_id': 'BIBREF13'}, {'start': 138, 'end': 139, 'mention': '6', 'ref_id': 'BIBREF15'}, {'start': 421, 'end': 422, 'mention': '7', 'ref_id': 'BIBREF16'}, {'start': 639, 'end': 640, 'mention': '8', 'ref_id': 'BIBREF17'}, {'start': 1066, 'end': 1067, 'mention': '7', 'ref_id': 'BIBREF16'}], 'section': '', 'ref_spans': []}, {'text': 'The third uncertainty is whether there are a large number of asymptomatic cases of COVID-19. Estimates suggest that about 80% of people with COVID-19 have mild or asymptomatic disease, 14% have severe disease, and 6% are critically ill,9 implying that symptom-based control is unlikely to be sufficient unless these cases are only lightly infectious.', 'cite_spans': [{'start': 236, 'end': 237, 'mention': '9', 'ref_id': 'BIBREF18'}], 'section': '', 'ref_spans': []}, {'text': 'The fourth uncertainty is the duration of the infectious period for COVID-19. The infectious period is typically short for influenza A, but it seems long for COVID-19 on the basis of the few available clinical virological studies, perhaps lasting for 10 days or more after the incubation period.8 The reports of a few super-spreading events are a routine feature of all infectious diseases and should not be overinterpreted.10\\n', 'cite_spans': [{'start': 295, 'end': 296, 'mention': '8', 'ref_id': 'BIBREF17'}, {'start': 424, 'end': 426, 'mention': '10', 'ref_id': 'BIBREF1'}], 'section': '', 'ref_spans': []}, {'text': 'What do these comparisons with influenza A and SARS imply for the COVID-19 epidemic and its control? First, we think that the epidemic in any given country will initially spread more slowly than is typical for a new influenza A strain. COVID-19 had a doubling time in China of about 4–5 days in the early phases.3 Second, the COVID-19 epidemic could be more drawn out than seasonal influenza A, which has relevance for its potential economic impact. Third, the effect of seasons on transmission of COVID-19 is unknown;11 however, with an R0 of 2–3, the warm months of summer in the northern hemisphere might not necessarily reduce transmission below the value of unity as they do for influenza A, which typically has an R0 of around 1·1–1·5.12 Closely linked to these factors and their epidemiological determinants is the impact of different mitigation policies on the course of the COVID-19 epidemic.', 'cite_spans': [{'start': 312, 'end': 313, 'mention': '3', 'ref_id': 'BIBREF12'}, {'start': 518, 'end': 520, 'mention': '11', 'ref_id': 'BIBREF2'}, {'start': 741, 'end': 743, 'mention': '12', 'ref_id': 'BIBREF3'}], 'section': '', 'ref_spans': []}, {'text': 'A key issue for epidemiologists is helping policy makers decide the main objectives of mitigation—eg, minimising morbidity and associated mortality, avoiding an epidemic peak that overwhelms health-care services, keeping the effects on the economy within manageable levels, and flattening the epidemic curve to wait for vaccine development and manufacture on scale and antiviral drug therapies. Such mitigation objectives are difficult to achieve by the same interventions, so choices must be made about priorities.13 For COVID-19, the potential economic impact of self-isolation or mandated quarantine could be substantial, as occurred in China.', 'cite_spans': [{'start': 515, 'end': 517, 'mention': '13', 'ref_id': 'BIBREF4'}], 'section': '', 'ref_spans': []}, {'text': 'No vaccine or effective antiviral drug is likely to be available soon. Vaccine development is underway, but the key issues are not if a vaccine can be developed but where phase 3 trials will be done and who will manufacture vaccine at scale. The number of cases of COVID-19 are falling quickly in China,4 but a site for phase 3 vaccine trials needs to be in a location where there is ongoing transmission of the disease. Manufacturing at scale requires one or more of the big vaccine manufacturers to take up the challenge and work closely with the biotechnology companies who are developing vaccine candidates. This process will take time and we are probably a least 1 year to 18 months away from substantial vaccine production.', 'cite_spans': [{'start': 303, 'end': 304, 'mention': '4', 'ref_id': 'BIBREF13'}], 'section': '', 'ref_spans': []}, {'text': 'So what is left at present for mitigation is voluntary plus mandated quarantine, stopping mass gatherings, closure of educational institutes or places of work where infection has been identified, and isolation of households, towns, or cities. Some of the lessons from analyses of influenza A apply for COVID-19, but there are also differences. Social distancing measures reduce the value of the effective reproduction number R. With an early epidemic value of R0 of 2·5, social distancing would have to reduce transmission by about 60% or less, if the intrinsic transmission potential declines in the warm summer months in the northern hemisphere. This reduction is a big ask, but it did happen in China.', 'cite_spans': [], 'section': '', 'ref_spans': []}, {'text': 'School closure, a major pillar of the response to pandemic influenza A,14 is unlikely to be effective given the apparent low rate of infection among children, although data are scarce. Avoiding large gatherings of people will reduce the number of super-spreading events; however, if prolonged contact is required for transmission, this measure might only reduce a small proportion of transmissions. Therefore, broader-scale social distancing is likely to be needed, as was put in place in China. This measure prevents transmission from symptomatic and non-symptomatic cases, hence flattening the epidemic and pushing the peak further into the future. Broader-scale social distancing provides time for the health services to treat cases and increase capacity, and, in the longer term, for vaccines and treatments to be developed. Containment could be targeted to particular areas, schools, or mass gatherings. This approach underway in northern Italy will provide valuable data on the effectiveness of such measures. The greater the reduction in transmission, the longer and flatter the epidemic curve (figure\\n), with the risk of resurgence when interventions are lifted perhaps to mitigate economic impact.', 'cite_spans': [{'start': 71, 'end': 73, 'mention': '14', 'ref_id': 'BIBREF5'}], 'section': '', 'ref_spans': [{'start': 1102, 'end': 1108, 'mention': 'figure', 'ref_id': 'FIGREF0'}]}, {'text': 'The key epidemiological issues that determine the impact of social distancing measures are what proportion of infected individuals have mild symptoms and whether these individuals will self-isolate and to what effectiveness; how quickly symptomatic individuals take to isolate themselves after the onset of symptoms; and the duration of any non-symptomatic infectious period before clear symptoms occur with the linked issue of how transmissible COVID-19 is during this phase.', 'cite_spans': [], 'section': '', 'ref_spans': []}, {'text': 'Individual behaviour will be crucial to control the spread of COVID-19. Personal, rather than government action, in western democracies might be the most important issue. Early self-isolation, seeking medical advice remotely unless symptoms are severe, and social distancing are key. Government actions to ban mass gatherings are important, as are good diagnostic facilities and remotely accessed health advice, together with specialised treatment for people with severe disease. Isolating towns or even cities is not yet part of the UK Government action plan.15 This plan is light on detail, given the early stages of the COVID-19 epidemic and the many uncertainties, but it outlines four phases of action entitled contain, delay, research, and mitigate.15 The UK has just moved from contain to delay, which aims to flatten the epidemic and lower peak morbidity and mortality. If measures are relaxed after a few months to avoid severe economic impact, a further peak is likely to occur in the autumn (figure). Italy, South Korea, Japan, and Iran are at the mitigate phase and trying to provide the best care possible for a rapidly growing number of people with COVID-19.', 'cite_spans': [{'start': 560, 'end': 562, 'mention': '15', 'ref_id': 'BIBREF6'}, {'start': 755, 'end': 757, 'mention': '15', 'ref_id': 'BIBREF6'}], 'section': '', 'ref_spans': [{'start': 1003, 'end': 1009, 'mention': 'figure', 'ref_id': 'FIGREF0'}]}, {'text': 'The known epidemiological characteristics of COVID-19 point to urgent priorities. Shortening the time from symptom onset to isolation is vital as it will reduce transmission and is likely to slow the epidemic (appendices 2, 3) However, strategies are also needed for reducing household transmission, supporting home treatment and diagnosis, and dealing with the economic consequences of absence from work. Peak demand for health services could still be high and the extent and duration of presymptomatic or asymptomatic transmission—if this turns out to be a feature of COVID-19 infection—will determine the success of this strategy.16\\n', 'cite_spans': [{'start': 633, 'end': 635, 'mention': '16', 'ref_id': 'BIBREF7'}], 'section': '', 'ref_spans': []}, {'text': 'Contact tracing is of high importance in the early stages to contain spread, and model-based estimates suggest, with an R0 value of 2·5, that about 70% of contacts will have to be successfully traced to control early spread.17 Analysis of individual contact patterns suggests that contact tracing can be a successful strategy in the early stages of an outbreak, but that the logistics of timely tracing on average 36 contacts per case will be challenging.17 Super-spreading events are inevitable, and could overwhelm the contact tracing system, leading to the need for broader-scale social distancing interventions.', 'cite_spans': [{'start': 224, 'end': 226, 'mention': '17', 'ref_id': 'BIBREF8'}, {'start': 455, 'end': 457, 'mention': '17', 'ref_id': 'BIBREF8'}], 'section': '', 'ref_spans': []}, {'text': 'Data from China, South Korea, Italy, and Iran suggest that the CFR increases sharply with age and is higher in people with COVID-19 and underlying comorbidities.18 Targeted social distancing for these groups could be the most effective way to reduce morbidity and concomitant mortality. During the outbreak of Ebola virus disease in west Africa in 2014–16, deaths from other causes increased because of a saturated health-care system and deaths of health-care workers.19 These events underline the importance of enhanced support for health-care infrastructure and effective procedures for protecting staff from infection.', 'cite_spans': [{'start': 161, 'end': 163, 'mention': '18', 'ref_id': 'BIBREF9'}, {'start': 468, 'end': 470, 'mention': '19', 'ref_id': 'BIBREF10'}], 'section': '', 'ref_spans': []}, {'text': 'In northern countries, there is speculation that changing contact patterns and warmer weather might slow the spread of the virus in the summer.11 With an R0 of 2·5 or higher, reductions in transmission by social distancing would have to be large; and much of the changes in transmission of pandemic influenza in the summer of 2009 within Europe were thought to be due to school closures, but children are not thought to be driving transmission of COVID-19. Data from the southern hemisphere will assist in evaluating how much seasonality will influence COVID-19 transmission.', 'cite_spans': [{'start': 143, 'end': 145, 'mention': '11', 'ref_id': 'BIBREF2'}], 'section': '', 'ref_spans': []}, {'text': 'Model-based predictions can help policy makers make the right decisions in a timely way, even with the uncertainties about COVID-19. Indicating what level of transmission reduction is required for social distancing interventions to mitigate the epidemic is a key activity (figure). However, it is easy to suggest a 60% reduction in transmission will do it or quarantining within 1 day from symptom onset will control transmission, but it is unclear what communication strategies or social distancing actions individuals and governments must put in place to achieve these desired outcomes. A degree of pragmatism will be needed for the implementation of social distancing and quarantine measures. Ongoing data collection and epidemiological analysis are therefore essential parts of assessing the impacts of mitigation strategies, alongside clinical research on how to best manage seriously ill patients with COVID-19.', 'cite_spans': [], 'section': '', 'ref_spans': [{'start': 273, 'end': 279, 'mention': 'figure', 'ref_id': 'FIGREF0'}]}, {'text': 'There are difficult decisions ahead for governments. How individuals respond to advice on how best to prevent transmission will be as important as government actions, if not more important. Government communication strategies to keep the public informed of how best to avoid infection are vital, as is extra support to manage the economic downturn.', 'cite_spans': [], 'section': '', 'ref_spans': []}], 'paragraphs': [{'text': '', 'id': '00008'}], 'title': 'How will country-based mitigation measures influence the course of the COVID-19 epidemic?', 'sha': '1f7a4b8bbccbd085b8f7f45509f049523db6feb5', 'doi': '10.1016/s0140-6736(20)30567-5', 'score': 9.48169994354248}, {'rank': 87, 'paragraphs': [{'text': 'Interestingly, preprint studies appear to report longer incubation period than do journal publications, indicating potential publication bias. However, large variation of estimates was observed among studies, which indicates a non-negligible heterogeneity in COVID-19 patients. Evolution of SARS-CoV-2 might partially address this heterogeneity (25) .', 'id': '00029'}], 'title': 'A systematic review and meta-analysis reveals long and dispersive incubation period of COVID-19', 'sha': '3d921c03dc23ee850655ddfd9d5ebe3956d70bf8', 'doi': '10.1101/2020.06.20.20134387', 'score': 9.480899810791016}, {'rank': 88, 'paragraphs': [{'text': 'The sum t tot = t 1 + t 2 is the total average time from exposure to the appearance of symptoms severe enough to cause an individual to isolate. We call this the extended incubation period.', 'id': '00010'}], 'title': 'Effects of latency on estimates of the COVID-19 replication number', 'sha': 'a017dd5e8dfa5dfd6385d407813e7c26d98becb9', 'doi': '10.1101/2020.04.07.20056960', 'score': 9.47819995880127}, {'rank': 89, 'paragraphs': [{'text': 'While we focus on data from British Columbia in this work, our methods may be applied to any region or country with case reporting to determine the relevant time lags and interpret reported cases accordingly. In addition, SEIR-type models are used to forecast COVID-19 in all 15 . CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'id': '00040'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.468899726867676}, {'rank': 90, 'abstract': 'As an emerging infectious disease, the 2019 coronavirus disease (COVID-19) has developed into a global pandemic. During the initial spreading of the virus in China, we demonstrated the ensemble Kalman filter performed well as a short-term predictor of the daily cases reported in Wuhan City. Second, we used an individual-level network-based model to reconstruct the epidemic dynamics in Hubei Province at its early stage and examine the effectiveness of non-pharmaceutical interventions on the epidemic spreading with various scenarios. Our simulation results show that without continued control measures, the epidemic in Hubei Province could have become persistent. Only by continuing to decrease the infection rate through 1) protective measures and 2) social distancing can the actual epidemic trajectory that happened in Hubei Province be reconstructed in simulation. Finally, we simulate the COVID-19 transmission with non-Markovian processes and show how these models produce different epidemic trajectories, compared to those obtained with Markov processes. Since recent studies show that COVID-19 epidemiological parameters do not follow exponential distributions leading to Markov processes, future works need to focus on non-Markovian models to better capture the COVID-19 spreading trajectories. In addition, shortening the infectious period via early case identification and isolation can slow the epidemic spreading significantly.', 'body_text': [{'text': 'The 2019 novel coronavirus disease (COVID-19) emerged in Wuhan City, Hubei Province of China, in the late fall of 2019 (Chen et al., 2020) . The highly contagious virus outbreak during the Chinese Spring festival period made containment an unprecedented challenge. The risks of human-to-human transmission, long persistence on surfaces, and asymptomatic infections increased spread and made case detection difficult (Chan et al., 2020; Rothe et al., 2020) . Increasing case reports saturated hospitals in Hubei Province, overwhelming medical resources. Despite the high case reports, the true infection rate was likely underreported, as infected individuals with only mild symptoms might have recovered without being detected Sanche et al., 2020) .', 'cite_spans': [{'start': 119, 'end': 138, 'text': '(Chen et al., 2020)', 'ref_id': 'BIBREF2'}, {'start': 416, 'end': 435, 'text': '(Chan et al., 2020;', 'ref_id': 'BIBREF1'}, {'start': 436, 'end': 455, 'text': 'Rothe et al., 2020)', 'ref_id': 'BIBREF24'}, {'start': 726, 'end': 746, 'text': 'Sanche et al., 2020)', 'ref_id': 'BIBREF27'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'To stop the viral spread, a series of strict control measures have been implemented in mainland China, including the enforcement of the tourism ban, suspension of the school, and extension of the Spring Festival holiday. From January 23 rd to April 8 th , 2020, the Chinese government implemented a metropolitan-wide quarantine of Wuhan and nearby cities, suspending all public transportation and advising residents to stay at home (Du et al., n.d.; J. T. Wu, Leung, & Leung, 2020) . Since February 8 th , a door-to-door check was started in Wuhan to take all confirmed and suspected people into medical care, which ended at around February 19 th (X. Li, Zhao, & Sun, 2020) .', 'cite_spans': [{'start': 432, 'end': 449, 'text': '(Du et al., n.d.;', 'ref_id': None}, {'start': 450, 'end': 481, 'text': 'J. T. Wu, Leung, & Leung, 2020)', 'ref_id': 'BIBREF29'}, {'start': 651, 'end': 673, 'text': 'Li, Zhao, & Sun, 2020)', 'ref_id': 'BIBREF18'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'Understanding the impact of non-pharmaceutical interventions on the epidemic dynamics is crucial for decision-makers to take proper actions to contain the COVID-19. Since the beginning of the outbreak, scientists have been actively retrieving the epidemiological characteristics such as the time from infection to illness onset (incubation period) and the time from illness onset to hospital admission or isolation (infectious period) (Chen et al., 2020; Guan et al., 2020; Huang et al., 2020; Linton et al., 2020) . Researchers have used the reported cases during the early stage of the epidemic and predicted the epidemic trajectory, the basic reproductive number 0 from which ranges from 2.2 to 6.6 (Read, Bridgen, Cummings, Ho, & Jewell, 2020; Sanche et al., 2020; Zhao et al., 2020) . Several studies explored the effects of the quarantine of Wuhan (X. Shen, Peng, Guo, Xiao, & Zhang, 2020) , travel restrictions (Chinazzi et al., 2020) , and other non-pharmaceutical intervention strategies (Lai et al., 2020; Zhu et al., 2020) on the future transmission dynamics at different geographical scales.', 'cite_spans': [{'start': 435, 'end': 454, 'text': '(Chen et al., 2020;', 'ref_id': 'BIBREF2'}, {'start': 455, 'end': 473, 'text': 'Guan et al., 2020;', 'ref_id': 'BIBREF8'}, {'start': 474, 'end': 493, 'text': 'Huang et al., 2020;', 'ref_id': 'BIBREF10'}, {'start': 494, 'end': 514, 'text': 'Linton et al., 2020)', 'ref_id': 'BIBREF20'}, {'start': 702, 'end': 747, 'text': '(Read, Bridgen, Cummings, Ho, & Jewell, 2020;', 'ref_id': 'BIBREF23'}, {'start': 748, 'end': 768, 'text': 'Sanche et al., 2020;', 'ref_id': 'BIBREF27'}, {'start': 769, 'end': 787, 'text': 'Zhao et al., 2020)', 'ref_id': 'BIBREF33'}, {'start': 858, 'end': 895, 'text': 'Shen, Peng, Guo, Xiao, & Zhang, 2020)', 'ref_id': 'BIBREF28'}, {'start': 918, 'end': 941, 'text': '(Chinazzi et al., 2020)', 'ref_id': 'BIBREF3'}, {'start': 997, 'end': 1015, 'text': '(Lai et al., 2020;', 'ref_id': 'BIBREF16'}, {'start': 1016, 'end': 1033, 'text': 'Zhu et al., 2020)', 'ref_id': 'BIBREF35'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': \"China's aggressive and prompt measures have led to a dramatic decrease in the number of new cases. Those measures saved many lives, although at a high economic cost. As of March 18 th , 2020, the cumulative number of confirmed cases in China was 81,186. Compared to the previous day, the number of new cases was only 70, most of which were imported from outside the country (T. Wu, Ge, Yu, & Hu, 2020) . In the meantime, COVID-19 has further spread around the world, and countries like Italy, South Korea, the United States, and Iran are struggling to contain the epidemic.\", 'cite_spans': [{'start': 378, 'end': 401, 'text': 'Wu, Ge, Yu, & Hu, 2020)', 'ref_id': 'BIBREF30'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'In this work, we aim to assess the capability of the ensemble Kalman filter as a good short-term predictor, and test the effectiveness of non-pharmaceutical interventions on the epidemic spreading. First, we provide real-time assessments and forecasts of the case reported in Wuhan City based on the ensemble Kalman filter. Second, we build an individual-level based network model and perform stochastic simulations to reconstruct the epidemics in Hubei Province at its early stage and examine the epidemic dynamics under different scenarios. We consider four scenarios: keeping the early-stage trend without any mitigation, reducing the average node degree within a city through social distancing, reducing the infection rate by adopting protective measures, and decreasing the infection rate through both social distancing and protective measures. Third, we simulate the epidemic spreading in Wuhan City incorporating non-Markovian processes, i.e., non-exponential parameter distributions, and compare the predicted epidemic trajectories obtained from Markov processes.', 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'Mathematical modeling can forecast the size and timing of epidemics, so resource allocations and decision-making can be optimized to maximize intervention strategies. Since most current network-based models are based on Markov processes, this work is probably one of the first to simulate the COVID-19 spreading based on non-Markovian processes. The findings of this work can be a reference for policy recommendations regarding COVID-19 in other countries.', 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'In this section, we first introduce the ensemble Kalman filtering method used in the short-term analysis. Then we illustrate epidemic models based on Markov processes and non-Markovian processes built in the long-term analysis.', 'cite_spans': [], 'ref_spans': [], 'section': 'Model'}, {'text': 'We aim to provide short-term prediction for the COVID-19 spreading in Wuhan city based on the Kalman filtering, which consists of model state estimation at each observation time based only on the observation up to the present. Considering that there is an unknown time-varying parameter β in the SIR model which cannot be dealt with by widely used classic Kalman filter (Kalman, 1960) and extended . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 370, 'end': 384, 'text': '(Kalman, 1960)', 'ref_id': 'BIBREF14'}], 'ref_spans': [], 'section': 'Short-term analysis'}, {'text': 'The copyright holder for this preprint this version posted March 30, 2020 . . https://doi.org/10.1101 Kalman filter (Jazwinski, 2007) , a dual state-parameter estimation filtering method-ensemble Kalman filter (EnKF) is used for our study (Jensen, 2007; Moradkhani, Sorooshian, Gupta, & Houser, 2005) . EnKF is based on Monte Carlo where the approximation of forecast (a priori) state error covariance matrix is made by propagating an ensemble of model states using the updated states (ensemble members) from the previous time step.', 'cite_spans': [{'start': 59, 'end': 73, 'text': 'March 30, 2020', 'ref_id': None}, {'start': 74, 'end': 101, 'text': '. . https://doi.org/10.1101', 'ref_id': None}, {'start': 116, 'end': 133, 'text': '(Jazwinski, 2007)', 'ref_id': 'BIBREF12'}, {'start': 239, 'end': 253, 'text': '(Jensen, 2007;', 'ref_id': 'BIBREF13'}, {'start': 254, 'end': 300, 'text': 'Moradkhani, Sorooshian, Gupta, & Houser, 2005)', 'ref_id': 'BIBREF21'}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'It is quite difficult to have a real specific contact network between each person in Wuhan. Therefore, a generalized network is used, which means that every individual in this area is connected with a possibility. To describe the epidemiology of COVID-19 pneumonia, we used the discrete-time S-I-R model which categorizes the hosts within a population as susceptible (S), infectious (I) and removed (R) (Keeling & Rohani, 2011) . At any discrete time, every individual has to be in one of these 3 compartments. The susceptible are individuals unexposed to the pathogen but can be infected by contact with the infectious population. Infectious individuals can clear the infection by hospitalization, death and quarantine measures and become removed. The equations for the state transitions are shown as follows.', 'cite_spans': [{'start': 403, 'end': 427, 'text': '(Keeling & Rohani, 2011)', 'ref_id': 'BIBREF15'}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'Where β denotes the infection rate and γ denotes the removing rate. In the SIR model, we assume that the infection process of each infectious individual is Poisson and statistically independent and that the removing process for every infected individual is also Poisson. In this case, the infection rate β quantifies the probability that each susceptible individual can be infected by each infectious individual in a time unit, and the recovering rate γ quantifies the average portion of the infectious population that will clear infection in a time unit. Considering that the population flow, prevention measures (like wearing N95 mask) and selfquarantine could affect the infection rate, we take the infection rate β as a time-varying parameter. The removing rate is set as a constant 1/7 based on the estimation from Huang et al. (2020) .', 'cite_spans': [{'start': 820, 'end': 839, 'text': 'Huang et al. (2020)', 'ref_id': 'BIBREF10'}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'Let the state variables (SIR) defined as ∈ 3 , the time-varying parameter ∈ . Consider the following problem', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'where and are model and observation noise which are assumed to be normally distributed and with zero mean and the covariance known.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'Because the true states are generally unknown, we can use the ensemble mean as the best estimate of the true state. The error covariance matrices for the predicted and filtered ensemble are given by', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'where − is the error covariance matrix associated with the forecasted estimate (prior), is the one associated with the updated estimate (posterior), ̂− is the ensemble vector of the forecasted estimate from model equations.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted March 30, 2020. . https://doi.org/10. 1101 In the ensemble presentation, the model states and the time-varying parameter can be put together in a matrix ∈ 4× , holding N ensemble member at time k.', 'cite_spans': [{'start': 97, 'end': 101, 'text': '1101', 'ref_id': None}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The ensemble covariance then can be presented as', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The measurement can also be formulated in an ensemble presentation', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'where 1 = + 1 , is the true measurement and is the perturbation term. The measurement covariance matrix can be estimated as', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'Specify to the situation we are dealing with,', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'and it is the same with , , and . The measurement is the total number of confirmed infected cases reported by the National Health Commission of China, which can be taken as the sum of the infectious and removed. Therefore,', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'In the long-term analysis, several epidemic models are built based on the Susceptible-Exposed-Infectious-Removed (SEIR) compartmental model, which has been frequently used in the study of the COVID-19 (X. Read et al., 2020; J. T. Wu et al., 2020; Zhou et al., 2020) . Each node is either in one of the compartments susceptible, exposed, infectious, or removed. Susceptible nodes refer to people free of the virus and can become infected due to contact with infectious nodes. Exposed nodes have been infected by the disease, but have not yet shown symptoms or become infectious. Removed nodes represent those individuals who are recovered, isolated, hospitalized, or dead due to the disease.', 'cite_spans': [{'start': 205, 'end': 223, 'text': 'Read et al., 2020;', 'ref_id': 'BIBREF23'}, {'start': 224, 'end': 246, 'text': 'J. T. Wu et al., 2020;', 'ref_id': 'BIBREF29'}, {'start': 247, 'end': 265, 'text': 'Zhou et al., 2020)', 'ref_id': 'BIBREF34'}], 'ref_spans': [], 'section': 'Long-term analysis'}, {'text': 'In this section, we apply the generalized modeling framework (GEMF) proposed by Sahneh, Scoglio, & Van Mieghem (2013) to build an individual-level network model. In GEMF, each node stays in a different state, and the joint state of all nodes follow a Markov process. The node-level description of this Markov process can be expressed as follows:', 'cite_spans': [{'start': 80, 'end': 117, 'text': 'Sahneh, Scoglio, & Van Mieghem (2013)', 'ref_id': 'BIBREF25'}], 'ref_spans': [], 'section': 'Network-based approach with Markovian processes'}, {'text': '[ ( + Δ ) = 3| ( ) = 2, ( )] = Δ + (Δ )', 'cite_spans': [], 'ref_spans': [], 'section': 'Network-based approach with Markovian processes'}, {'text': 'Where ( ) is the joint state of all individual nodes at time ; ( ) = 0, 1, 2,3 means that the state of node at time is in susceptible, exposed, infectious and removed compartment, respectively; is the number . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Network-based approach with Markovian processes'}, {'text': 'The copyright holder for this preprint this version posted March 30, 2020. . https://doi.org/10. 1101 of infectious neighbors of node ; ′ ( ) is the rate for one node transitions from susceptible state to the exposed state; is the rate from the exposed state to the infectious state, and is the removal rate from infectious state to the removed state, which is the inverse of the infectious period.', 'cite_spans': [{'start': 97, 'end': 101, 'text': '1101', 'ref_id': None}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'Right after the COVID-19 outbreak in China, many studies have estimated the epidemiological parameters, which are essential to calibrate the epidemic models. However, we noticed that there exists a variety regarding the epidemiological parameters. Based on 1099 patients in 31 provincial municipalities through January 29 th , 2020, Guan et al. (2020) estimated that the median incubation period was 3.0 days, ranging between 0 to 24.0 days. This median incubation period was shorter than the previous studies based on smaller sample sizes, in which the estimated time from infection to illness onset was 5-5.2 days (Gardner, 2020; Q. Li et al., 2020) . Du et al. (2020) considered the delay from symptom onset to case detection as 4-5 days. Considering the possibility of asymptomatic infections, we adopt a relatively short mean incubation period as 3 days and mean infectious period as 4 days .', 'cite_spans': [{'start': 333, 'end': 351, 'text': 'Guan et al. (2020)', 'ref_id': 'BIBREF8'}, {'start': 616, 'end': 631, 'text': '(Gardner, 2020;', 'ref_id': 'BIBREF7'}, {'start': 632, 'end': 651, 'text': 'Q. Li et al., 2020)', 'ref_id': 'BIBREF17'}, {'start': 654, 'end': 670, 'text': 'Du et al. (2020)', 'ref_id': 'BIBREF4'}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'Hubei Province has 17 prefecture-level cities, and the population size of each city is obtained from the Hubei Statistical Yearbook (Hubei Provincial Bureau of Statistics, 2020). Based on the Baidu locationbased service data (https://qianxi.baidu.com/) and the fact that around 5 million people have moved out of Wuhan before the city lockdown on January 23 rd (Zhu et al., 2020) , the number of people that traveled between the cities in Hubei Province from January 1 st to January 29 th , 2020 is obtained.', 'cite_spans': [{'start': 361, 'end': 379, 'text': '(Zhu et al., 2020)', 'ref_id': 'BIBREF35'}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'Since data about the actual people contact network inside a city are unavailable, we created contact networks among people within each city following with the homogenous network topology, namely the Erdős-Rényi model (Erdős & Rényi, 1960) . More specifically, nodes inside a city are randomly connected with a predefined probability such that each node has around 15 links (the average node degree is 15). Intercity links are created to simulate the interaction among the cities according to the number of people that traveled between the cities. For example, if the daily population flow from city to city is number of people, number of directed links will be randomly generated between people in city to people in city . Considering the implementation of city lockdown on January 23 rd , we alter the intercity network topology once during the simulation. In the first network topology, links among cities are created based on the average daily population movement between January 1 st -22 nd , while the second network reflects the population movement from January 23 rd -29 th .', 'cite_spans': [{'start': 217, 'end': 238, 'text': '(Erdős & Rényi, 1960)', 'ref_id': 'BIBREF6'}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'To fit the model, we consider the COVID-19 epidemic outbreak starting in Wuhan City from January 19 th , because all confirmed cases reported in Hubei Province were in Wuhan City before January 19 th , 2020. To reduce the computational cost, the population of each city and population flows among the cities are scaled by 198, which is the number of cumulative confirmed cases in Hubei Province up to January 19 th . After scaling, the total number of nodes in the network equals 298,838. In addition, as the criteria for counting the confirmed cases are changed on February 12 th , 2020, there is a jump in the reported number of confirmed cases in Hubei Province on February 13 th , 2020. Accordingly, by keeping = 0.33 day -1 and = 0.25 day -1 (average) unchanged, we fitted the transmission model to the cumulative number of confirmed cases in Hubei Province (Health Commission of Hubei Province, 2020), from January 19 th to February 12 th by tuning the infection rate ′ ( ). We then examine the future course of the epidemic under different scenarios.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'GEMF is available in MATLAB, R, Python, and C programming language (Sahneh, Vajdi, Shakeri, Fan, & Scoglio, 2017) . We performed extensive stochastic simulations by adapting the GEMF toolbox with MATLAB during the experiment. The number of simulation runs is 500 in all scenarios. We track the number of nodes in each compartment for 365 days and demonstrate the simulation results with confidence intervals.', 'cite_spans': [{'start': 67, 'end': 113, 'text': '(Sahneh, Vajdi, Shakeri, Fan, & Scoglio, 2017)', 'ref_id': 'BIBREF26'}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted March 30, 2020. ', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': \"Since the outbreak of COVID-19, many studies have emerged to learn how this disease spreads. However, an overwhelming majority of the epidemic models have considered Markov processes, in which the transition between compartments follows an exponential distribution. To the best of the authors' knowledge, very few works have been done incorporating non-Markovian processes to epidemic models with respect to COVID-19. In this section, we take Wuhan City as a study area and try to demonstrate how distributions of key epidemiological factors can influence the predicted trajectory of the disease spreading.\", 'cite_spans': [], 'ref_spans': [], 'section': 'Network-based approach with non-Markov processes'}, {'text': 'More specifically, we incorporate non-exponential interevent time distributions to the SEIR model using the modified Gillespie algorithm proposed by Boguñá, Lafuerza, Toral, & Serrano (2014) . Transitions from the susceptible state to the exposed state is a Poisson process that follows the exponential distribution. Following lognormal distributions, the mean and median of the incubation period are estimated at 5.6 days and 5.0 days, respectively, while the infectious period is of mean 3.9 days and median 1.5 days , which is in line with Sanche et al. (2020) . The number of simulation runs is 80.', 'cite_spans': [{'start': 149, 'end': 190, 'text': 'Boguñá, Lafuerza, Toral, & Serrano (2014)', 'ref_id': 'BIBREF0'}, {'start': 543, 'end': 563, 'text': 'Sanche et al. (2020)', 'ref_id': 'BIBREF27'}], 'ref_spans': [], 'section': 'Network-based approach with non-Markov processes'}, {'text': 'In this section, we provide results regarding the short-term analysis with the Kalman filter. From the long-term perspective, we first present results based on GEMF stochastic simulations and then compare the results based on non-Markovian processes and Markov processes.', 'cite_spans': [], 'ref_spans': [], 'section': 'Results and discussion'}, {'text': 'The predictions of COVID-19 cases in Wuhan city from February 22 nd to March 1 st were conducted. Every predictive simulation was run 250 iterations. The removing rate we used was 1/7 and the infection rate was estimated by EnKF. The model noise and measurement noise was tuned by state of art. Fig. 1 shows the distributions of the predicted numbers of COVID-19 cases based on data 1, 2 and 3 days before. The results of the EnKF prediction give two rangesthe weak one from minimum to maximum (the solid black line) which can include all the reported real data (the green circle); the stronger one from the lower quartile to the upper quartile (the blue box) which also covers most of the real numbers. The red line inside the box indicates the median of the samples. We can see that most medians are very close to the corresponding real numbers. Taking the median as our predicted number gives the relative error shown in Fig. 1(d) . The mean absolute percentage error (MAPE) for 1-day-ahead prediction is 0.19%; the MAPE for 2-day-ahead prediction is 0.32%; the one for 3-day-ahead is 0.62%. The performance of 1-day-ahead prediction is the best and the one of 3-day-ahead is the worst. When n-day-ahead prediction is conducted, the results from (n-1)-day-ahead is used, so the errors will accumulate.', 'cite_spans': [], 'ref_spans': [{'start': 295, 'end': 301, 'text': 'Fig. 1', 'ref_id': None}, {'start': 924, 'end': 933, 'text': 'Fig. 1(d)', 'ref_id': None}], 'section': 'Short-term analysis with the Kalman filter'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Short-term analysis with the Kalman filter'}, {'text': 'The copyright holder for this preprint this version posted March 30, 2020 . . https://doi.org/10.1101 The estimation of infection rate β from January 15 th to February 27 th is presented as the blue line in Fig.3 . The orange line in Fig. 3 presents the removing rate. From January 15 th to January 27 th , the infection rate β is increasing. After January 27th, β starts to decrease. The self-quarantine policy was carried out on January 23 rd in Wuhan and a series of administrative actions were taken to prevent the disease from spreading. Considering that there are several days delay from the infection to detection, the estimated β matches with the real situation. On February 12 th , because Wuhan authorities changed the case definition to include symptomatic patients in addition to tested ones, there was a huge increase in cases, which results in a huge increase in estimated infection rate β as well. Therefore, the estimated β on February 12 th is inaccurate. After February 14 th , the infection rate β is less than the removing rate γ, which means the epidemic had peaked in Wuhan.', 'cite_spans': [{'start': 59, 'end': 73, 'text': 'March 30, 2020', 'ref_id': None}, {'start': 74, 'end': 101, 'text': '. . https://doi.org/10.1101', 'ref_id': None}], 'ref_spans': [{'start': 207, 'end': 212, 'text': 'Fig.3', 'ref_id': 'FIGREF1'}, {'start': 234, 'end': 240, 'text': 'Fig. 3', 'ref_id': 'FIGREF1'}], 'section': '(which was not certified by peer review)'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted March 30, 2020 . . https://doi.org/10.1101 ', 'cite_spans': [{'start': 59, 'end': 73, 'text': 'March 30, 2020', 'ref_id': None}, {'start': 74, 'end': 101, 'text': '. . https://doi.org/10.1101', 'ref_id': None}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'To reconstruct the epidemics in Hubei province at its early stage, we continuously tune the infection rate parameter ′ to match the number of nodes in the removed compartment with the number of cumulative confirmed cases from January 19 th to February 12 th , as shown in Fig. 4.   Fig. 4 Fitting the model to the cumulative confirmed cases from January 19 th to February 12 th .', 'cite_spans': [], 'ref_spans': [{'start': 272, 'end': 288, 'text': 'Fig. 4.   Fig. 4', 'ref_id': None}], 'section': 'Long-term analysis with the Markovian process'}, {'text': 'By performing GEMF stochastic simulations, we obtain that ′ is equal to 0.12 for day 0-7, and then becomes 0.027 for day 8-10, and 0.024 for day 11-18. Starting from day 19, ′ = 0.02. The infection rate dramatically drops, indicating that public health interventions implemented in Hubei Province have considerably controlled the epidemic development since the nascent stage of the outbreak.', 'cite_spans': [], 'ref_spans': [], 'section': 'Long-term analysis with the Markovian process'}, {'text': 'The predicted epidemic dynamics from GEMF stochastic simulations are shown in Fig. 5 . We consider four different scenarios: continuing the current trend (Scenario1); decreasing 2 by 25% (Scenario 2); reducing the average node degree inside a city from 15 to 10 (Scenario 3); reducing 2 by 25% and average node degree from 15 to 10 (Scenario 4). Preventive measures such as wearing masks can help reduce the infection rate. On the other hand, the average node degree refers to the possible contacts a person may have in daily life, and this can be reduced by contact reduction and social distancing.', 'cite_spans': [], 'ref_spans': [{'start': 78, 'end': 84, 'text': 'Fig. 5', 'ref_id': None}], 'section': 'Long-term analysis with the Markovian process'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Long-term analysis with the Markovian process'}, {'text': 'The copyright holder for this preprint this version posted March 30, 2020 . . https://doi.org/10.1101 In Fig. 5 , if the current trend as of February 12 th continues, the epidemics in Hubei Province could have become endemic and not been contained in one year. When ′ is reduced by 25% in Scenario 2, the number of infectious cases in Hubei Province would reach the peak at day 22 on average (median: 24, and fade out at around day 250 in Fig. 5(c) . Reducing the average node degree inside each city from 15 to 10, the epidemic would reach the peak at around day 21 (median: 24, and fade out at day 150 on average. When a combination of reducing ′ and average node degree is conducted in scenario 4, the number of infectious cases would reach the peak at around day 19 (median: 23, , and the number of removed cases would approximate the actual epidemic trajectory in Hubei Province in the steady-state as shown in Fig. 5 (e) .', 'cite_spans': [{'start': 59, 'end': 73, 'text': 'March 30, 2020', 'ref_id': None}, {'start': 74, 'end': 101, 'text': '. . https://doi.org/10.1101', 'ref_id': None}, {'start': 392, 'end': 404, 'text': '(median: 24,', 'ref_id': None}, {'start': 564, 'end': 579, 'text': '21 (median: 24,', 'ref_id': None}, {'start': 767, 'end': 782, 'text': '19 (median: 23,', 'ref_id': None}], 'ref_spans': [{'start': 105, 'end': 111, 'text': 'Fig. 5', 'ref_id': None}, {'start': 439, 'end': 448, 'text': 'Fig. 5(c)', 'ref_id': None}, {'start': 916, 'end': 926, 'text': 'Fig. 5 (e)', 'ref_id': None}], 'section': '(which was not certified by peer review)'}, {'text': 'Overall, GEMF simulation results suggest that more strict prevention and control measures have been taken since February 12 th to achieve the current cumulative confirmed cases in Hubei Province. Preventive measures and social distancing measures need to be taken in combination by the government to speed up the containment of the epidemic and reduce the peak infection dramatically.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': '(a) Scaled number of susceptible cases (b) Scaled number of exposed cases (c) Scaled number of infectious cases (d) Scaled number of removed cases . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted March 30, 2020 . . https://doi.org/10.1101 (e) Scaled number of removed cases compared to the cumulative confirmed cases up to March 24 th Fig 5. Epidemic dynamics under different scenarios from GEMF stochastic simulations.', 'cite_spans': [{'start': 59, 'end': 73, 'text': 'March 30, 2020', 'ref_id': None}, {'start': 74, 'end': 101, 'text': '. . https://doi.org/10.1101', 'ref_id': None}], 'ref_spans': [{'start': 198, 'end': 204, 'text': 'Fig 5.', 'ref_id': None}], 'section': '(which was not certified by peer review)'}, {'text': 'With mean infectious period (from compartments → ) set as 3.9 days and mean incubation period (from compartments → ) set as 5 days, we first use ordinary differential equations based on Markov processes to estimate the infection rate from compartments → . Assuming that the epidemic starts from December 1 st , is equal to 0.68 to match the number of cumulative confirmed cases in Wuhan city.', 'cite_spans': [], 'ref_spans': [], 'section': 'Long-term analysis with the non-Markovian process'}, {'text': 'Second, we simulate the disease spread in Wuhan city with transmission between compartments incorporating non-Markovian dynamics. The transition from compartment → still evolves according to an exponential distribution with = 0.68, while transitions from compartments → and → follow the lognormal distributions from Sanche et al., 2020) in Fig. 6 (a-b) . Note that the width of the infectious period becomes much shorter (blue curve) after January 18 th , as shown in Fig. 6 (a) , because the government has started to trace back and monitor those people who have contact with confirmed cases and built temporary hospitals to hospitalize both patients with mild and severe symptoms since mid-January. In Fig. 6 (a-b) , we also show curves of different exponential distributions compared with these lognormal distributions for demonstration purposes.', 'cite_spans': [{'start': 316, 'end': 336, 'text': 'Sanche et al., 2020)', 'ref_id': 'BIBREF27'}], 'ref_spans': [{'start': 340, 'end': 352, 'text': 'Fig. 6 (a-b)', 'ref_id': None}, {'start': 468, 'end': 478, 'text': 'Fig. 6 (a)', 'ref_id': None}, {'start': 704, 'end': 716, 'text': 'Fig. 6 (a-b)', 'ref_id': None}], 'section': 'Long-term analysis with the non-Markovian process'}, {'text': 'In Fig. 6 (c-d), simulated results incorporating non-Markovian processes deviate from the results obtained from Markov processes. More specifically, the epidemic curve with non-Markovian processes is flattened with a smaller epidemic peak compared with that from Markov processes. This is because the infectious period used in the Markovian model has an exponential distribution (mean =3.9 days) similar to the shape of the exponential distribution (mean=4.0 days) in Fig. 6 (a) , but shorter than the lognormal distribution (mean=3.9 days, median=1.5 days) used after January 18 th . The reasons above indicate that control measures to reduce the infectious period can effectively mitigate the outbreak.', 'cite_spans': [], 'ref_spans': [{'start': 3, 'end': 9, 'text': 'Fig. 6', 'ref_id': None}, {'start': 468, 'end': 478, 'text': 'Fig. 6 (a)', 'ref_id': None}], 'section': 'Long-term analysis with the non-Markovian process'}, {'text': 'On the other hand, these two methods with different distributions but with the same mean infectious period lead to inconsistent results. Since the actual infectious period and incubation period do not follow exponential distributions, these simulations suggest that future works need to consider distributions of epidemiological characteristics to better capture the COVID-19 spreading trajectory. Instead of taking mean values derived from clinical studies as input to the epidemic models, researchers need to take into account the empirically derived distribution of the parameters.', 'cite_spans': [], 'ref_spans': [], 'section': 'Long-term analysis with the non-Markovian process'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Long-term analysis with the non-Markovian process'}, {'text': 'The copyright holder for this preprint this version posted March 30, 2020. ', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'In this work, the ensemble Kalman filter is used to make short-term predictions of the COVID-19 cases in Wuhan City, which resulted in accurate forecasts of daily case reports. The model is able to predict daily cases and the epidemic peak. Knowing the daily cases from forecasts three days in advance allows for the proper resource allocation. The ensemble Kalman filter also allows parameter estimation, which is extremely useful for modeling purposes.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusion'}, {'text': 'From an epidemiological modeling perspective, we simulate the epidemic spreading in Wuhan city based on non-Markovian processes, and show that different distributions with the same mean infectious period can lead to inconsistency in terms of epidemic dynamics. Since most current network-based approaches are based on Markov processes, in which epidemiological parameters are assumed to follow exponential distributions, non-Markovian process-based models need to be further explored to better predict the COVID-19 spreading. Our results show that reducing the infectious period via measures such as early case identification and isolation can reduce the epidemic size significantly.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusion'}, {'text': 'In the long-term perspective, we test and confirm the effectiveness of non-pharmaceutical interventions on the containment of the epidemic dynamics by performing GEMF stochastic simulations. The daily cases . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusion'}, {'text': 'The copyright holder for this preprint this version posted March 30, 2020 . . https://doi.org/10.1101 of the COVID-19 epidemic in Hubei Province peaked at over 15,000 on February 12 th and became almost zero in mid-March (Xinhua, 2020) . Our simulation results indicate that the containment of this highly contagious disease was achieved by stringent control measures by the Chinese government. Without continued and aggressive control measures, the epidemic in Hubei Province would have become persistent. If the infection rate is reduced through the enhanced protective measures by 25%, the epidemic would reach the peak in mid-February and fade out in late September. Reducing the average node degree inside each city via social distancing measures, the number of infectious cases would peak at mid-February and decrease to zero at mid-June. Only with combined implementation of enhanced protective measures and social distancing measures, the epidemic dynamics would peak at around mid-February and approximate the actual epidemic trajectory in March. This can be an important message for countries going through the exponential growth of the epidemic in the current days.', 'cite_spans': [{'start': 59, 'end': 73, 'text': 'March 30, 2020', 'ref_id': None}, {'start': 74, 'end': 101, 'text': '. . https://doi.org/10.1101', 'ref_id': None}, {'start': 221, 'end': 235, 'text': '(Xinhua, 2020)', 'ref_id': 'BIBREF32'}], 'ref_spans': [], 'section': 'Conclusion'}], 'paragraphs': [{'text': 'Right after the COVID-19 outbreak in China, many studies have estimated the epidemiological parameters, which are essential to calibrate the epidemic models. However, we noticed that there exists a variety regarding the epidemiological parameters. Based on 1099 patients in 31 provincial municipalities through January 29 th , 2020, Guan et al. (2020) estimated that the median incubation period was 3.0 days, ranging between 0 to 24.0 days. This median incubation period was shorter than the previous studies based on smaller sample sizes, in which the estimated time from infection to illness onset was 5-5.2 days (Gardner, 2020; Q. Li et al., 2020) . Du et al. (2020) considered the delay from symptom onset to case detection as 4-5 days. Considering the possibility of asymptomatic infections, we adopt a relatively short mean incubation period as 3 days and mean infectious period as 4 days .', 'id': '00028'}], 'title': 'Short-term forecasts and long-term mitigation evaluations for the COVID-19 epidemic in Hubei Province, China', 'sha': '26a2d4b805cdb48bccb4d640da82fa4c4d15c7c4', 'doi': '10.1101/2020.03.27.20045625', 'score': 9.466899871826172}, {'rank': 91, 'paragraphs': [{'text': 'In December 2019, a novel pneumonia resulting from SARS-CoV2, named coronavirus disease 2019 (COVID-19), emerged in Wuhan [1,2]. Due to the high viral contagiousness and transmission during the pre-symptomatic phase [3,4], COVID-19 progressively spread throughout China and has since become a global health issue [5]. In China, Wuhan city, the capital of Hubei province, became the center of a COVID-19 outbreak [6]. In an effort to prevent a COVID-19 epidemic, Wuhan and its surrounding cities in Hubei province were locked down on January 23rd. However, a tremendous number of patients who were still in their incubation period traveled across China before the lockdown date due to Chinese New Year, resulting in the widespread transmission of COVID-19. Hunan province is adjacent to Hubei province. By March 20th, 2020, a total of 1018 confirmed cases had been reported in Hunan, most of which showed familial clustering.', 'id': '00001'}], 'title': 'COVID-19 in a patient with long-term use of glucocorticoids: A study of a familial cluster', 'sha': '912c00514457672748738d29efb81f975815cdf6', 'doi': '10.1016/j.clim.2020.108413', 'score': 9.457799911499023}, {'rank': 92, 'paragraphs': [{'text': 'In this systematic review and meta-analysis, we searched PubMed, Embase, medRxiv, bioRxiv, and arXiv to identify studies related to COVID-19 published or publicly posted from December 01, 2019 to April 26, 2020 (date of last search) in parallel by two authors (L.W. and Y.L.). Each database was searched using the terms \"(COVID-19) OR (2019-nCoV) OR (novel coronavirus pneumonia)\". The search strategy is detailed in the Supplement Table 1-4. There were no language restrictions on the search.', 'id': '00006'}], 'title': 'A systematic review and meta-analysis reveals long and dispersive incubation period of COVID-19', 'sha': '3d921c03dc23ee850655ddfd9d5ebe3956d70bf8', 'doi': '10.1101/2020.06.20.20134387', 'score': 9.454999923706055}, {'rank': 93, 'paragraphs': [{'text': 'A number of recent studies have examined the effectiveness of government control measures on the spread of COVID-19 [2, 8, 34, 35, 23, 32, 18, 9] . One approach has been to retrospectively compare observed data to baseline model output [34, 8] . Many studies have also focused on estimating changes in the effective reproduction number over time [11, 6, 35] . Similarly, [2] directly estimates the impact of control measures on COVID-19 transmission patterns using a Bayesian model with explicit physical distancing. However, these studies do not focus on reporting the time until a given amount of change is predicted between many simulations of future case counts. In-stead, they compare simulations to a constant baseline, or report quantitative measures on only a few simulations.', 'id': '00002'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.436400413513184}, {'rank': 94, 'paragraphs': [{'text': 'Our analysis assumes an incubation period that is of the order of a single week, specifically in our simulations, we set it at an average of 5 days. The rationale however is more general, and can be adapted to longer or shorter incubation times, simply by tuning the periodicity of the lock-down shifts, keeping them congruent with the natural cycle of the disease.', 'id': '00025'}], 'title': 'Alternating quarantine for sustainable mitigation of COVID-19', 'sha': 'b923a8fc7d406d0f149ecd5293d1a72dec7ee526', 'doi': '', 'score': 9.430500030517578}, {'rank': 95, 'abstract': \"Based on publicly released data for 1212 patients, we investigated the epidemiological characteristics of COVID-19 in Henan of China. The following findings are obtained: 1) COVID-19 patients in Henan show gender (55% vs 45%) and age (81% aged between 21 and 60) preferences, possible causes were explored; 2) Statistical analysis on 483 patients reveals that the estimated average, mode and median incubation periods are 7.4, 4 and 7 days; Incubation periods of 92% patients were no more than 14 days; 3) The epidemic of COVID-19 in Henan has undergone three stages and showed high correlations with the numbers of patients that recently return from Wuhan; 4) Network analysis on the aggregate outbreak phenomena of COVID-19 revealed that 208 cases were clustering infected, and various people's Hospital are the main force in treating patients. The related investigations have potential implications for the prevention and control of COVID-19.\", 'body_text': [{'text': 'Arising from Wuhan South China seafood market at the end of the year 2019, novel coronavirus pneumonia (Named as COVID-19 by WHO since February 11, 2020) has spread for more than one months, and it has directly resulted to more than 74600 confirmed patients, and 2121 people dead till February 20, 2020 [1] - [19] . Various investigations and reports reveal that the COVID-19 is different from the SARS in the year 2003 from the following aspects [1] [2] [3] [4] [5] [6] : 1) Higher Infection rate; 2) With incubation period; 3) Incubation period is infectious; 4) Low lethality rate; 5) High mortality among the elderly patients that are with underlying diseases. These features result the prevention and control of COVID-19 to be a difficult task.', 'cite_spans': [{'start': 303, 'end': 306, 'text': '[1]', 'ref_id': 'BIBREF0'}, {'start': 309, 'end': 313, 'text': '[19]', 'ref_id': 'BIBREF18'}, {'start': 447, 'end': 450, 'text': '[1]', 'ref_id': 'BIBREF0'}, {'start': 451, 'end': 454, 'text': '[2]', 'ref_id': 'BIBREF1'}, {'start': 455, 'end': 458, 'text': '[3]', 'ref_id': 'BIBREF2'}, {'start': 459, 'end': 462, 'text': '[4]', 'ref_id': 'BIBREF3'}, {'start': 463, 'end': 466, 'text': '[5]', 'ref_id': 'BIBREF4'}, {'start': 467, 'end': 470, 'text': '[6]', 'ref_id': 'BIBREF5'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'As a double-side sword, modern traffic and communication technologies greatly shortage the distances among people and provide convenience for human beings, however, they also facilitate the rapidly and widely spreading of epidemic diseases or rumors [12] . Since January 23, Wuhan has taken strict measures to close the city, and many other places have adopted various measures in succession to prevent the spreading of COVID-19, such as blocking traffic and putting suspicious ones in quarantine. After the outbreak of COVID-19, researchers have paid great efforts on the investigations of COVID-19, including its control and prediction [1] - [18] . Initial investigations pointed out that bats are mostly the source of the virus [3] , and Manis pentadactyla is a potential intermediate host of . The related investigations have great implications on the prevention, control and vaccine development [20] - [30] . Some recent publications have reported the epidemiological and clinical features of patients with COVID-19 [1] [2] [3] [4] [5] [6] . To prevent the spreading of COVID-19, the centers for disease control and prevention (CDCs) of many provinces have timely released the data of confirmed cases, which facilitate us to explore the epidemiological and statistical characteristics of patients with COVID-19, and to understand the current situation and characteristics of the epidemic.', 'cite_spans': [{'start': 250, 'end': 254, 'text': '[12]', 'ref_id': 'BIBREF10'}, {'start': 638, 'end': 641, 'text': '[1]', 'ref_id': 'BIBREF0'}, {'start': 644, 'end': 648, 'text': '[18]', 'ref_id': 'BIBREF16'}, {'start': 731, 'end': 734, 'text': '[3]', 'ref_id': 'BIBREF2'}, {'start': 900, 'end': 904, 'text': '[20]', 'ref_id': 'BIBREF19'}, {'start': 907, 'end': 911, 'text': '[30]', 'ref_id': 'BIBREF29'}, {'start': 1021, 'end': 1024, 'text': '[1]', 'ref_id': 'BIBREF0'}, {'start': 1025, 'end': 1028, 'text': '[2]', 'ref_id': 'BIBREF1'}, {'start': 1029, 'end': 1032, 'text': '[3]', 'ref_id': 'BIBREF2'}, {'start': 1033, 'end': 1036, 'text': '[4]', 'ref_id': 'BIBREF3'}, {'start': 1037, 'end': 1040, 'text': '[5]', 'ref_id': 'BIBREF4'}, {'start': 1041, 'end': 1044, 'text': '[6]', 'ref_id': 'BIBREF5'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'As a neighbor of the Hubei province, Henan province is with a large population size, and it is also one of the hardest hit areas of the epidemic. The current total confirmed cases in Henan are only lower than Hubei and Guangdong. The statistical analysis on the patients data in Henan can help us to understand the epidemiological and clinical features of patients [17] . In this paper, based on data collected from CDCs at all levels in Henan, we will perform statistical analysis. Our data includes 1212 patients ranging from January 21 to February 14, 2020, and covering 18 regions of Henan province. We will try to answer the following questions: 1) the current situation and characteristics of the epidemic in the 18 regions of Henan; 2) The sex and age distributions of confirmed cases; 3) The estimation of incubation period of patients with COVID-19; 4) The correlation between the number of patients with Wuhan travel histories and the total number of confirmed cases; 5) Network analysis on aggregate outbreak phenomena. The rest of the paper is organized as follows. Section 2 will introduce our main results and methods; Sections 3 gives some discussions and conclusions of the paper.', 'cite_spans': [{'start': 365, 'end': 369, 'text': '[17]', 'ref_id': 'BIBREF15'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'From January 21 to February 14, 2020, totally 1212 confirmed cases have been released in Henan. The epidemic situation, time evolution of daily increased infected patients and cumulatively confirmed ones are shown in Fig.1 .', 'cite_spans': [], 'ref_spans': [{'start': 217, 'end': 222, 'text': 'Fig.1', 'ref_id': None}], 'section': 'The current epidemic situations and characteristics in Henan'}, {'text': 'Based on Fig.1 , we see that the daily increment of patients reaches a peak value of 109 at February 3, 2020. After that, the daily increment of patients significantly decreases. Among the 18 regions of Henan province, Xinyang, Nanyang, Zhumadian, Zhengzhou, Shangqiu and Zhoukou encompass more confirmed cases than the other regions. Xinyang is still with the highest daily increment as compared with the other regions, which is the severely affected area in Henan. In fact, Xinyang, Nanyang and Zhumadian are all near Hubei-the worst affected province in China. It was reported that about five million people have left Wuhan since January 23. According to the investigation from Xu et al. [7, 8] , before Wuhan was closed at January 23, Xinyang, Zhengzhou, Nanyang, Zhumadian, Zhoukou, Shangqiu were ranked among the top-50 cities in China that received huge amount of Wuhan personnel. The number of virus carrier is supposed to be proportional to the received amount of Wuhan personnel, which may be one of the main reasons that why the mentioned regions in Henan are so severely affected.', 'cite_spans': [{'start': 691, 'end': 694, 'text': '[7,', 'ref_id': 'BIBREF6'}, {'start': 695, 'end': 697, 'text': '8]', 'ref_id': 'BIBREF7'}], 'ref_spans': [{'start': 9, 'end': 14, 'text': 'Fig.1', 'ref_id': None}], 'section': 'The current epidemic situations and characteristics in Henan'}, {'text': 'Among the 1212 patients, we abstract the gender information of 1158 patients (95.54% of total samples) and the age information of 1156 patients (95.38% of total samples). From the perspective of both the whole province and its 18 regions, we performed detailed statistical analysis, which are summarized in Fig.2 .', 'cite_spans': [], 'ref_spans': [{'start': 307, 'end': 312, 'text': 'Fig.2', 'ref_id': None}], 'section': 'Gender and age distribution'}, {'text': 'Statistical results reveal that 637 out of the 1158 patients are male, which accounts for 55% and apparently higher than females (Fig.2 A) , although there are slightly differences among different regions . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [{'start': 129, 'end': 138, 'text': '(Fig.2 A)', 'ref_id': None}], 'section': 'Gender and age distribution'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Gender and age distribution'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02.21.20026112 doi: medRxiv preprint ( Fig.2 B) . We guess that there are three possible reasons that may result to such gender difference. Firstly, males may be more active and with wider social activities than females, which increases their risk to infect COVID-19. Secondly and may be the most reasonable, an existing medical investigation reported that the expression and distribution of angiotension converting enzyme 2 (ACE2) was wider in male patients than those in females [5] . Similarly to SARS-CoV, it is reported that COVID-19 invades human body through the ACE2 receptor. Based on single cell RNA sequencing technology, researchers have investigated the expression profiles of ACE2 in two male and six female patients with COVID-19 from single cell resolution, and they found that the expression of ACE2 was correlated with gender. The proportion of ACE2 expressed cells is higher in males than in females (1.66% vs. 0.41%) [5] . Besides, the distribution of ACE2 was also wider in males than that in females. Zhao et al. [5] reported that there were at least five different types of cells in the lungs of males with the expression of the ACE2 receptors, while the number was about 2 ∼ 4 in female patients. This may be one of the deep reasons that why male patients were higher than the female ones. Thirdly, one may doubt that the distribution difference may be caused by population structure. To verify whether the male to female ratio (MFR) affects the finding, based on the statistical yearbook report of Henan province in the year 2019, the MFR in the whole province was 1.0126 : 1, which has great difference with the male to female patients ratio (MFPR) (637 : 521 = 1.2226 : 1). Moreover, we perform correlation analysis for the two ratios in the 18 regions, the two ratios show very weak correlation, with Pearson correlation coefficient r = 0.1555 ( Fig.2 C) . Kolmogorov-Smirnov (KS) test [31] on the MFR and MFPR in the 18 regions also shows that the two have significant difference (KS = 0.6111, p = 0.0018 < 0.05). Thus, we speculate that the MFR has no apparent effect on the MFPR. In conclusion, the former two reasons may be the driven force that there was gender difference in confirmed patients.', 'cite_spans': [{'start': 588, 'end': 591, 'text': '[5]', 'ref_id': 'BIBREF4'}, {'start': 1044, 'end': 1047, 'text': '[5]', 'ref_id': 'BIBREF4'}, {'start': 1142, 'end': 1145, 'text': '[5]', 'ref_id': 'BIBREF4'}, {'start': 2021, 'end': 2025, 'text': '[31]', 'ref_id': 'BIBREF30'}], 'ref_spans': [{'start': 146, 'end': 154, 'text': 'Fig.2 B)', 'ref_id': None}, {'start': 1981, 'end': 1989, 'text': 'Fig.2 C)', 'ref_id': None}], 'section': 'Gender and age distribution'}, {'text': 'The age distribution of patients in the whole province or each region all roughly follows normal distribution ( Fig.2 D, E). Patients aged between 21 and 60 years old take up more than 81% (938 out of 1156). The main reason may be that people aged between 21 and 60 years old are the social main labor forces, there are many migrant workers in this age interval. Moreover, they may have wider social circles than the others. Therefore, people aged between 21 and 60 years old are key crowd with high risks of COVID-19.', 'cite_spans': [], 'ref_spans': [{'start': 112, 'end': 117, 'text': 'Fig.2', 'ref_id': None}], 'section': 'Gender and age distribution'}, {'text': 'Traditionally, incubation period is defined as the period between the infection of an individual by a pathogen and the manifestation of the illness or disease it causes [32, 33] . Different infectious diseases have different incubation periods. However, for a certain infectious disease, its incubation period is relatively fixed. Since the number of pathogens entering the body, virulence and reproductive capacity, as well as power of resistance are different for different people, thus, it is reported that the incubation periods obtained from patients for a certain disease should follow logarithm normal distribution [16, 33] . Generally, incubation period can be measured by physiological observations and biological experiments [33] . The determination of incubation period has great implications for disease transmission control and policy making.', 'cite_spans': [{'start': 169, 'end': 173, 'text': '[32,', 'ref_id': None}, {'start': 174, 'end': 177, 'text': '33]', 'ref_id': 'BIBREF31'}, {'start': 622, 'end': 626, 'text': '[16,', 'ref_id': 'BIBREF13'}, {'start': 627, 'end': 630, 'text': '33]', 'ref_id': 'BIBREF31'}, {'start': 735, 'end': 739, 'text': '[33]', 'ref_id': 'BIBREF31'}], 'ref_spans': [], 'section': 'Definition of incubation period and statistical results'}, {'text': \"The estimation of the incubation period of COVID-19 is a very difficult task. The main difficulty is in that it is difficult to determine the infected time. Based on patients' information, some works have reported the incubation period of COVID-19. For example, Yang et al. [4] reported that the median incubation period of COVID-19 is 4.75 days; the interquartile range is 3.0 − 7.2 days. Hereinafter, we try to estimate the incubation period of patients in Henan. The collected data are with varied quality, many descriptions of patients only told us when the person left Wuhan, or When she/he had suspicious contacts with persons from Wuhan or suspicious persons (Following, we call they are exposed). To facilitate us to estimate the incubation period, we define estimated incubation period as the period from the date of exposure to the date of appearing clinical symptoms or making a definite diagnosis. Apparently, such definition overestimates the actual incubation period. For example, the information from a patient . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\", 'cite_spans': [{'start': 274, 'end': 277, 'text': '[4]', 'ref_id': 'BIBREF3'}], 'ref_spans': [], 'section': 'Definition of incubation period and statistical results'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02. 21.20026112 doi: medRxiv preprint 0 20 40 60 80 100 120 5 -J a n 7 -J a n 9 -J a n 1 1 -J a n 1 3 -J a n 1 5 -J a n 1 7 -J a n 1 9 -J a n 2 1 -J a n 2 3 -J a n 2 5 -J a n 2 7 -J a n 2 9 -J a n 3 1 -J a n 2', 'cite_spans': [{'start': 111, 'end': 122, 'text': '21.20026112', 'ref_id': None}], 'ref_spans': [], 'section': 'Definition of incubation period and statistical results'}, {'text': 'Indegree Outdegree Nodes at the two ends of an edge correspond to the exposed date and the date with clinical symptoms or confirmed infected. Edges represent the transfer from the two dates, and thicknesses of edges are proportional to the numbers of patients. Self-loops mean that the dates for exposure and appearing clinical symptoms or confirmed infected were the same. D. Weighted indegree and outdegree distributions of the directed graph that was shown in C. Weighted indegree denotes the total number of patients with clinical symptoms or confirmed infected; while weighted outdegree represents the total number of exposed persons that will be confirmed to be infected later.', 'cite_spans': [], 'ref_spans': [], 'section': 'Definition of incubation period and statistical results'}, {'text': '(Mr Zhou, No.2 patients in Nanyang) in Nanyang says that his son left Wuhan and arrived at home at January 6, he and his son were all disease-free for a long period. Until February 3, he became sick and was quickly confirmed with infection of COVID-19. His son was disease-free till now. From our definition of incubation period, his incubation period will be 28 days, which is surprisingly high. We note that, our definition of estimated incubation period may be a little higher than the actually case, since we can not exclude the cases that the person has contacted some other people that carried the COVID-19. That is, we cannot know who was the intermediate spreader (unknown person B), and when she/he transmitted the virus from person A to person C.', 'cite_spans': [], 'ref_spans': [], 'section': 'Definition of incubation period and statistical results'}, {'text': 'Among the collected data, the incubation periods of 483 confirmed patients can be estimated. Statistical results are shown in Fig.3 and Tab.1. The estimated incubation periods for the 483 patients roughly follow the logarithm normal distribution, with a long right tail. Specifically, the 483 confirmed patients were with average estimated incubation periodτ D = 7.4 days, the mode was 4 days (50 out of the 483 cases) and the median was 7 days (48 out of the 483 cases). About 55% patients were with incubation periods lower than 7 days, while more than 92% patients were lower than 14 days. About 7.45% patients were overestimated with more than 14 days incubation periods.', 'cite_spans': [], 'ref_spans': [{'start': 126, 'end': 131, 'text': 'Fig.3', 'ref_id': 'FIGREF2'}], 'section': 'Definition of incubation period and statistical results'}, {'text': 'Due to the definition, the estimated incubation periods in this paper may be longer than the real cases. However, the distribution has some implications. It indicates that a few patients may be with very long incubation period, which increase the difficulty to control and prevent the COVID-19.', 'cite_spans': [], 'ref_spans': [], 'section': 'Definition of incubation period and statistical results'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Definition of incubation period and statistical results'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02.21.20026112 doi: medRxiv preprint Table 1 . Statistical results of estimated as well as fitted incubation period (Fig.3 A) ', 'cite_spans': [], 'ref_spans': [{'start': 144, 'end': 151, 'text': 'Table 1', 'ref_id': None}, {'start': 223, 'end': 232, 'text': '(Fig.3 A)', 'ref_id': 'FIGREF2'}], 'section': 'Definition of incubation period and statistical results'}, {'text': 'To theoretically model the incubation period, we suppose the incubation period τ follows the logarithm normal distribution [16, 33] :', 'cite_spans': [{'start': 123, 'end': 127, 'text': '[16,', 'ref_id': 'BIBREF13'}, {'start': 128, 'end': 131, 'text': '33]', 'ref_id': 'BIBREF31'}], 'ref_spans': [], 'section': 'Modeling the distribution of the incubation period'}, {'text': 'That is, the probability density function (PDF) of τ can be written as', 'cite_spans': [], 'ref_spans': [], 'section': 'Modeling the distribution of the incubation period'}, {'text': 'Here, µ, σ 2 are two parameters that can be estimated from data. We consider three different approaches to estimate the parameters µ and σ 2 , including the moment estimation (ME), the ordinary least square (OLS) estimation and the maximum likelihood estimation (MLE) [34] . As to the ME, based on the PDF of incubation period τ , we can easily obtain E(τ ) = exp µ + σ 2 /2 , D(τ ) = (e σ 2 − 1)exp 2µ + σ 2 . Using the fsolve function (which is based on the OLS method) in Matlab, we obtainμ m = 1.8239, σ 2 m = 0.3629. The fitted PDF curve can be found in Fig.3 A, and some theoretical estimations of the incubation period based on the ME method are summarized in Tab.1.', 'cite_spans': [{'start': 268, 'end': 272, 'text': '[34]', 'ref_id': 'BIBREF32'}], 'ref_spans': [{'start': 559, 'end': 564, 'text': 'Fig.3', 'ref_id': 'FIGREF2'}], 'section': 'Modeling the distribution of the incubation period'}, {'text': 'Similarly, for the OLS estimation, µ and σ 2 can be estimated from the following optimization objective function:', 'cite_spans': [], 'ref_spans': [], 'section': 'Modeling the distribution of the incubation period'}, {'text': 'Here, p i = n i /n denotes the ratio of patients with incubation period i(i = 0, 1, ..., m) (totally m+1 unique incubation periods), which can be easily computed from the 483 patients (n=483). f (t i ) corresponds to the PDF with incubation period t i , which is a function of the unknown parameters µ and σ 2 . Since τ = 0 in Eq.(2.2), we omit the case τ = 0 (only 29 cases were with τ = 0). By solving the optimization problem (2.5), we obtainμ o = 2.0723, σ 2 o = 0.5929. For the MLE, the estimation of µ and σ 2 can be obtained from minimizing the following negative logarithmic likelihood function.', 'cite_spans': [], 'ref_spans': [], 'section': 'Modeling the distribution of the incubation period'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Modeling the distribution of the incubation period'}, {'text': \"is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02. 21.20026112 doi: medRxiv preprint Here, t j (j = 1, 2, ..., n) denotes the incubation period for the j'th patient. Similarly to OLS, the case with τ = 0 will be not considered. Finally, we obtain\", 'cite_spans': [{'start': 111, 'end': 144, 'text': '21.20026112 doi: medRxiv preprint', 'ref_id': None}], 'ref_spans': [], 'section': 'Modeling the distribution of the incubation period'}, {'text': 'The fitted curves from the three methods are shown in Fig.3 A. Comparing among the three methods, it seems that the fitted curve from the ME method has the smallest variance, the variances of the OLS and MLE approaches are both very high. High variance indicates higher proportions of patients with long incubation periods. The ME method may well mimic the actual long right tail of the incubation period, while the OLS estimation could well describe the cases with middle incubation periods (incubation periods 2 ∼ 14). The MLE approach seems in between the ME and OLS approaches. The estimated interquartile range from the ME method is [4.1495, 9.3523] , which closes to the result from the 483 patients ( [4, 11] ).', 'cite_spans': [{'start': 638, 'end': 654, 'text': '[4.1495, 9.3523]', 'ref_id': None}, {'start': 708, 'end': 711, 'text': '[4,', 'ref_id': 'BIBREF3'}, {'start': 712, 'end': 715, 'text': '11]', 'ref_id': None}], 'ref_spans': [{'start': 54, 'end': 59, 'text': 'Fig.3', 'ref_id': 'FIGREF2'}], 'section': 'Modeling the distribution of the incubation period'}, {'text': 'The theoretical modeling of incubation period further illustrates that the incubation period may follow the logarithmic normal distribution. Moreover, it can be used to estimate the actual statics of incubation periods, and provide valuable references for decision-making.', 'cite_spans': [], 'ref_spans': [], 'section': 'Modeling the distribution of the incubation period'}, {'text': 'The epidemic of COVID-19 in Henan has undergone three stages (Fig.3 C and D) . For the first stage that before January 13, both exposed and confirmed patients were not so much, the number of exposed (outdegree) ones were more than the confirmed ones (indegree). While for the second stage that between January 13 and January 23, the COVID-19 was in outbreak period, both exposed and confirmed ones were considerable, and the number of exposed ones are far more than the confirmed ones. During this period, many college students and migrant workers in Wuhan return their hometown, the large number of people flow from Wuhan increased the risks of exposure and infection. After January 23, the epidemic entered the third stage, the number of daily confirmed cases were apparently higher than exposed ones, and the confirmed cases roughly decreased with time. This may be attributed to the adoption of prevention and control measures. With the closing of the Wuhan city at January 23, many regions of Henan quickly adopt very strict measures to prevent the spreading of COVID-19. Thus, after January 23, the number of exposed ones became fewer and fewer (Fig.3 D) , which is a good phenomena, indicating that the prevention and control measures become to gradually at work.', 'cite_spans': [], 'ref_spans': [{'start': 61, 'end': 76, 'text': '(Fig.3 C and D)', 'ref_id': 'FIGREF2'}, {'start': 1151, 'end': 1160, 'text': '(Fig.3 D)', 'ref_id': 'FIGREF2'}], 'section': 'Transfer diagram from exposure to infection'}, {'text': 'For the 18 regions of Henan province, we also summarized the numbers of currently confirmed patients with Wuhan travel histories (denoted as vector X 1 ), the numbers of patients without Wuhan travel histories (X 2 ), the numbers of patients that have contacts with persons from Wuhan (X 3 ), as well as the numbers of patients that are without Wuhan travel histories, but who have contacts with suspicious persons (X 4 ). The barplots for the numbers of the four categories of patients in the 18 regions are shown in Fig.4 A (In total of 1149 patients were known with or without Wuhan travel histories). Among patients with Wuhan travel histories, the released information of 250 patients contained transportation information. We found that 46% of the 250 patients have recently traveled by train or by bus, these patients may transmit the COVID-19 to a lot of people that in the same train or bus.', 'cite_spans': [], 'ref_spans': [{'start': 518, 'end': 523, 'text': 'Fig.4', 'ref_id': 'FIGREF5'}], 'section': 'Correlation with Wuhan travel histories'}, {'text': 'For the 18 regions, we investigate the correlation between the currently total numbers of confirmed patients (Y ) with X 1 − X 4 respectively. The heatmap of the correlation matrix is shown in Fig.4C . It reveals that Y has the highest correlation with X 1 , the Pearson correlation coefficient between the two is 0.9643, the fitted regression equation [35] is Y = 1.8146X 1 + 9.0920.', 'cite_spans': [{'start': 353, 'end': 357, 'text': '[35]', 'ref_id': 'BIBREF33'}], 'ref_spans': [{'start': 193, 'end': 199, 'text': 'Fig.4C', 'ref_id': 'FIGREF5'}], 'section': 'Correlation with Wuhan travel histories'}, {'text': '(2.8)', 'cite_spans': [], 'ref_spans': [], 'section': 'Correlation with Wuhan travel histories'}, {'text': 'Such linear relationship statistically holds (F = 212.31, p < 0.0001, R 2 = 0.9299). However, though there are certain correlation between Y and X 2 − X 4 , their strength of correlations are all lower than . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Correlation with Wuhan travel histories'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint that between Y and X 1 (Fig.4 D-G) . This indicates that persons with Wuhan travel histories have high risks of infection with COVID-19, and regions with more such persons tend to encompass more patients infected with COVID-19.', 'cite_spans': [], 'ref_spans': [{'start': 99, 'end': 110, 'text': '(Fig.4 D-G)', 'ref_id': 'FIGREF5'}], 'section': 'Correlation with Wuhan travel histories'}, {'text': \"From the 1212 patients, we constructed a heterogeneous network and performed some analysis on such network (Fig.5 ). The network contains 1105 patients that have been treated in 248 hospitals, 123 interhospital transfer relationships that involved 206 patients, and 208 patients that were clustering infected. Nodes in the network include patients and hospitals. Edges among patients represent relationship between relatives, friends or colleagues; Edges between patients and hospitals denote the patients were treated in the related hospitals; while edges among hospitals indicate the inter-hospital transfer treatments. Sizes of nodes were proportional to nodes' degree. Thickness of edges was proportional to the numbers of patients. From Fig.5 , we see that a few hospitals encompass a large number of patients in treatments, and the aggregate outbreak phenomena were ubiquitous. For example, the five patients that are treated in Zhengzhou Jinshui General Hospital are relatives to each other, they were successively infected. Similarly, 12 patients in Kaifeng are also relatives to each other. In fact, totally 208 patients were clustering infected, and Zhengzhou, Shangqiu, Zhoukou, Xuchang and Luoyang encompassed the most aggregative outbreak patients. Among the designated hospitals in Henan province, people's hospitals in different cities encompass a great number of patients. Xinyang Fifth People's Hospital received the most patients in Henan, followed by the Zhengzhou Sixth People's Hospital and the Luoshan County People's Hospital. Network analysis can provide us important information about patients, hospitals and their relationships; it can also provide valuable guidance for the distribution of epidemic prevention materials.\", 'cite_spans': [], 'ref_spans': [{'start': 107, 'end': 113, 'text': '(Fig.5', 'ref_id': 'FIGREF7'}, {'start': 737, 'end': 747, 'text': 'From Fig.5', 'ref_id': 'FIGREF7'}], 'section': 'Network analysis on aggregate outbreak phenomena'}, {'text': 'Through collection of publicly released patients information by various CDCs in Henan province during January 21 to February 14 of 2020, we performed statistical analysis on 1212 confirmed patients infected with COVID-19. The number of newly confirmed cases reached its peak at February 3, and it started . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussions and conclusions'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02. 21.20026112 doi: medRxiv preprint to fall after that day. Among the 18 regions of Henan, Xinyang is with the heaviest epidemic, and it is currently also with the highest daily increment. We found that 55% patients in Henan were male, which is apparently higher than the females. We found that the gender difference of patients in Henan is 1.2226 : 1, while from Yang et al. [4] , they reported that the gender difference for 4021 patients from over 30 provinces is 0.31 : 0.27 = 1.1481 : 1. Our result is close to the existing ones. We reported that the MFRs of the whole province and its 18 regions are not the main factor that results to the gender difference in patients. Two possible main factors that result to such gender difference of patients include 1) the expression and distribution of ACE2s were wider in male patients than those in females, and 2) males are more active and with wider social activities than females in many regions of Henan. The ages of patients generally follow normal distribution, and with more than 81% patients are aged between 21 and 60 years old. N a n y a n g C e n tr a l H o s p . Incubation period is very important but very difficult to be estimated, since it is difficult to know when the patients were infected. We define the period from firstly contacting with suspicious persons or recently return from Wuhan to firstly appearing clinical symptoms (fever, cough etc) as incubation period, and we estimate its distribution. Statistical analysis on 483 patients reveals that the average estimated incubation period was 7.4 days, the mode was 4 days. About 55% patients were with incubation periods lower than 7 days, while more than 92% patients were lower than 14 days. About 7.45% patients were overestimated with more than 14 days incubation periods. Based on ME, OLS and MLE approaches, we modeled the incubation period as logarithmic normal distributions, and we found the ME method can well mimic the long right tails of the PDF for the incubation periods. Based on the ME approach, interquartile range of theoretically estimated incubation period is between 4.1 and 9.4 days. It is noted that, due to . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 111, 'end': 144, 'text': '21.20026112 doi: medRxiv preprint', 'ref_id': None}, {'start': 485, 'end': 488, 'text': '[4]', 'ref_id': 'BIBREF3'}], 'ref_spans': [], 'section': 'Discussions and conclusions'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02. 21.20026112 doi: medRxiv preprint our definition of incubation period, the estimated values are unavoidably higher than the actual cases, however, the distribution and statistics from the estimated incubation periods are undoubtedly meaningful for real-world decision-making.', 'cite_spans': [{'start': 111, 'end': 122, 'text': '21.20026112', 'ref_id': None}], 'ref_spans': [], 'section': 'Discussions and conclusions'}, {'text': \"We also find that the numbers of patients with recent Wuhan travel histories are highly correlated with the total numbers of confirmed cases in the 18 regions of Henan. During our analysis, we consider the linear relationship between Y with each X i , which mainly for the purpose of avoiding multicollinearity [34] among X i (i = 1, 2, 3, 4) . Furthermore, we perform network analysis on aggregate outbreak phenomena of COVID-19 in Henan, we reported that 208 cases were clustering infected ones. Various people's Hospital in Henan are the main forces in treating patients. The related investigations may provide valuable suggestions and guidance for the prevention and control of COVID- 19 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\", 'cite_spans': [{'start': 311, 'end': 315, 'text': '[34]', 'ref_id': 'BIBREF32'}, {'start': 689, 'end': 691, 'text': '19', 'ref_id': 'BIBREF18'}], 'ref_spans': [{'start': 329, 'end': 342, 'text': '= 1, 2, 3, 4)', 'ref_id': 'FIGREF2'}], 'section': 'Discussions and conclusions'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02. 21.20026112 doi: medRxiv preprint ', 'cite_spans': [{'start': 111, 'end': 144, 'text': '21.20026112 doi: medRxiv preprint', 'ref_id': None}], 'ref_spans': [], 'section': 'Discussions and conclusions'}], 'paragraphs': [{'text': 'The fitted curves from the three methods are shown in Fig.3 A. Comparing among the three methods, it seems that the fitted curve from the ME method has the smallest variance, the variances of the OLS and MLE approaches are both very high. High variance indicates higher proportions of patients with long incubation periods. The ME method may well mimic the actual long right tail of the incubation period, while the OLS estimation could well describe the cases with middle incubation periods (incubation periods 2 ∼ 14). The MLE approach seems in between the ME and OLS approaches. The estimated interquartile range from the ME method is [4.1495, 9.3523] , which closes to the result from the 483 patients ( [4, 11] ).', 'id': '00027'}], 'title': 'Epidemiological characteristics of 1212 COVID-19 patients in Henan, China', 'sha': '091a8e9a61e19e88caeb039f0e3888d111b20439', 'doi': '10.1101/2020.02.21.20026112', 'score': 9.429300308227539}, {'rank': 96, 'paragraphs': [{'text': 'Five studies investigated the incubation period among young-age infections (age < 20 years), which were labeled as \" Young-age Infections\".', 'id': '00012'}], 'title': 'A systematic review and meta-analysis reveals long and dispersive incubation period of COVID-19', 'sha': '3d921c03dc23ee850655ddfd9d5ebe3956d70bf8', 'doi': '10.1101/2020.06.20.20134387', 'score': 9.428299903869629}, {'rank': 97, 'paragraphs': [{'text': 'Incubation period was assumed to follow a log-normal distribution (16) . Parameters of the log-normal distribution including mean and variation were calculated for each study (Supplement Table 6 -12).', 'id': '00013'}], 'title': 'A systematic review and meta-analysis reveals long and dispersive incubation period of COVID-19', 'sha': '3d921c03dc23ee850655ddfd9d5ebe3956d70bf8', 'doi': '10.1101/2020.06.20.20134387', 'score': 9.428298950195312}, {'rank': 98, 'paragraphs': [{'text': 'Last, meta-regression was used to explore the association between the age, sex, and median incubation period. All statistical analyses were performed using R software version 3.6.1 (Foundation for Statistical Computing).', 'id': '00018'}], 'title': 'A systematic review and meta-analysis reveals long and dispersive incubation period of COVID-19', 'sha': '3d921c03dc23ee850655ddfd9d5ebe3956d70bf8', 'doi': '10.1101/2020.06.20.20134387', 'score': 9.428297996520996}, {'rank': 99, 'paragraphs': [{'text': 'When both incubation times are fixed,', 'id': '00018'}], 'title': 'Effects of latency on estimates of the COVID-19 replication number', 'sha': 'a017dd5e8dfa5dfd6385d407813e7c26d98becb9', 'doi': '10.1101/2020.04.07.20056960', 'score': 9.420499801635742}, {'rank': 100, 'paragraphs': [{'text': 'However, the underlying mechanism is unclear and warrants further investigation.', 'id': '00033'}], 'title': 'A systematic review and meta-analysis reveals long and dispersive incubation period of COVID-19', 'sha': '3d921c03dc23ee850655ddfd9d5ebe3956d70bf8', 'doi': '10.1101/2020.06.20.20134387', 'score': 9.419999122619629}, {'rank': 101, 'paragraphs': [{'text': 'The copyright holder for this preprint this version posted June 22, 2020. ', 'id': '00044'}], 'title': 'A systematic review and meta-analysis reveals long and dispersive incubation period of COVID-19', 'sha': '3d921c03dc23ee850655ddfd9d5ebe3956d70bf8', 'doi': '10.1101/2020.06.20.20134387', 'score': 9.419998168945312}, {'rank': 102, 'paragraphs': [{'text': 'in the absence of distancing (i.e., when f = 1). We use uncertainty in R 0 to capture uncertainty in transmission and duration of infection simultaneously: given information about the duration, the transmission parameter β can simply be scaled in order to match the exponential growth phase of a dataset. The ODEs can be simulated numerically, drawing R 0 from the prior and keeping other parameters fixed. This procedure induces a distribution over the case counts seen on each day.', 'id': '00015'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.418000221252441}, {'rank': 103, 'paragraphs': [{'text': 'The copyright holder for this preprint this version posted June 16, 2020. . https://doi.org/10.1101/2020.06.14.20131177 doi: medRxiv preprint Figure S6 : (a) Days until active cases in the relaxed model exceeds the baseline model (y-axis), with varying levels of the standard deviation for R 0 (x-axis). There are four relaxation levels (all starting relaxation on May 17th). Parameter settings are identical to other figures aside from sd.', 'id': '00046'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.417999267578125}, {'rank': 104, 'paragraphs': [{'text': 'In another experiment, we consider variation of the ODE parameters D, k 1 , k 2 , q, u d and u r . We vary each of these parameters around their default values (given in Table S1 ) by X%, with X varied in the set {10, 20, 30}. For each X (uncertainty) level, we consider simulations in which the 22 . CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'id': '00048'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.417998313903809}, {'rank': 105, 'abstract': 'Early assessments of the spreading rate of COVID-19 were subject to significant uncertainty, as expected with limited data and difficulties in case ascertainment, but more reliable inferences can now be made. Here, we estimate from European data that COVID-19 cases are expected to double initially every three days, until social distancing interventions slow this growth, and that the impact of such measures is typically only seen nine days - i.e. three doubling times - after their implementation. We argue that such temporal patterns are more critical than precise estimates of the basic reproduction number for initiating interventions. This observation has particular implications for the low-and middle-income countries currently in the early stages of their local epidemics.', 'body_text': [{'text': \"In December 2019, a cluster of unexplained pneumonia cases in Wuhan, the capital of Hubei province in the People's Republic of China, rapidly progressed into a large-scale outbreak, and a global pandemic by 11 March 2020, as declared by the World Health Organisation [1]. The disease caused by this highly contagious infection has since been named COVID-19, and is caused by a single-stranded RNA coronavirus (SARS-CoV-2) similar to the pathogen responsible for severe acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS) [2] . As of 29 March 2020, 657,140 confirmed cases and 29,957 deaths have been reported in nearly 200 countries and territories globally [3] .\", 'cite_spans': [{'start': 207, 'end': 209, 'text': '11', 'ref_id': 'BIBREF10'}, {'start': 547, 'end': 550, 'text': '[2]', 'ref_id': 'BIBREF1'}, {'start': 684, 'end': 687, 'text': '[3]', 'ref_id': 'BIBREF2'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'Various control measures have been implemented worldwide, including isolation of confirmed and suspected cases, contact tracing, and physical distancing. However, only the most aggressive measures have resulted in epidemic suppression. In Hubei, a regional lockdown was implemented on 23-24 January 2020, with a peak in reported cases occurring approximately two weeks later [4] . In Italy, a national lockdown was implemented on 9 March 2020 once 7,375 confirmed cases and 366 deaths had been recorded; only in the last few days (as of 29 March) the epidemic seems to be slowing down [5] . In comparison, India declared a nationwide lockdown on 24 March 2020, with only 434 confirmed cases and 0 deaths [6, 7] . Similarly, South Africa began a 21-day lockdown on 27 th March, with 927 known cases and 0 deaths [8, 9] . The implementation of such early and aggressive control measures in India and South Africa may substantially increase their chances of successful containment, although the social and personal cost could be substantial [10] .', 'cite_spans': [{'start': 375, 'end': 378, 'text': '[4]', 'ref_id': 'BIBREF3'}, {'start': 585, 'end': 588, 'text': '[5]', 'ref_id': None}, {'start': 704, 'end': 707, 'text': '[6,', 'ref_id': 'BIBREF5'}, {'start': 708, 'end': 710, 'text': '7]', 'ref_id': 'BIBREF6'}, {'start': 811, 'end': 814, 'text': '[8,', 'ref_id': 'BIBREF7'}, {'start': 815, 'end': 817, 'text': '9]', 'ref_id': 'BIBREF8'}, {'start': 1038, 'end': 1042, 'text': '[10]', 'ref_id': 'BIBREF9'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'Planning of interventions usually relies on estimates of the basic reproduction number R 0 , defined as the average number of new infections generated by a single infected person within a susceptible population. Reported estimates of COVID-19 R 0 are highly variable, ranging from 1.4 to 6.49 [2, 11] , with the differences attributable to the variety of methods, model structures and parameter values (in particular, the estimated or assumed amount of presymptomatic transmission), as well as the data sources used. Most official sources settle in the range of 2-3 [3, [12] [13] [14] [15] , but these estimates mostly derive from early studies of the epidemic in Wuhan [16] [17] [18] , or the Diamond Princess Cruise ship [19] , and so are subject to important limitations: these include small amounts of data, uncertain or biased reporting of early cases in Wuhan, and the uniqueness of the specific settings in which they occurred.', 'cite_spans': [{'start': 293, 'end': 296, 'text': '[2,', 'ref_id': 'BIBREF1'}, {'start': 297, 'end': 300, 'text': '11]', 'ref_id': 'BIBREF10'}, {'start': 566, 'end': 569, 'text': '[3,', 'ref_id': 'BIBREF2'}, {'start': 570, 'end': 574, 'text': '[12]', 'ref_id': 'BIBREF11'}, {'start': 575, 'end': 579, 'text': '[13]', 'ref_id': 'BIBREF12'}, {'start': 580, 'end': 584, 'text': '[14]', 'ref_id': None}, {'start': 585, 'end': 589, 'text': '[15]', 'ref_id': 'BIBREF13'}, {'start': 670, 'end': 674, 'text': '[16]', 'ref_id': 'BIBREF14'}, {'start': 675, 'end': 679, 'text': '[17]', 'ref_id': 'BIBREF15'}, {'start': 680, 'end': 684, 'text': '[18]', 'ref_id': 'BIBREF16'}, {'start': 723, 'end': 727, 'text': '[19]', 'ref_id': 'BIBREF17'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'We argue official ranges of R 0 should be continuously updated with more recent estimates coming not only from China [20, 21] , but also from the many different outbreaks observed worldwide [22] [23] [24] [25] . Point estimates might not change, but the task remains imperative both because available data is now more numerous and reliable, and because estimates of R 0 in one population do not necessarily translate to another.', 'cite_spans': [{'start': 117, 'end': 121, 'text': '[20,', 'ref_id': 'BIBREF18'}, {'start': 122, 'end': 125, 'text': '21]', 'ref_id': 'BIBREF19'}, {'start': 190, 'end': 194, 'text': '[22]', 'ref_id': 'BIBREF20'}, {'start': 195, 'end': 199, 'text': '[23]', 'ref_id': 'BIBREF21'}, {'start': 200, 'end': 204, 'text': '[24]', 'ref_id': 'BIBREF22'}, {'start': 205, 'end': 209, 'text': '[25]', 'ref_id': 'BIBREF23'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'However, we emphasise that the speed of epidemic growth and the delay between infection and case detection are more relevant for COVID-19 control implementation than estimates of R 0 [26, 27] . Since doubling times can be estimated directly from data and estimates of delays are relatively consistent, sophisticated models are not required to infer when action is urgent.', 'cite_spans': [{'start': 183, 'end': 187, 'text': '[26,', 'ref_id': 'BIBREF24'}, {'start': 188, 'end': 191, 'text': '27]', 'ref_id': 'BIBREF25'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'To support our argument, we first estimate the growth rate in multiple countries and with different methods. We then estimate the incubation period and the distribution of times from symptoms onset to hospitalisation in different settings and compare it with other results from the literature. Observation of the outbreak in the UK and Italy provides further support to the highlighted delays from infection to detection. Finally, we discuss the limitations of relying on R 0 in this context.', 'cite_spans': [], 'ref_spans': [], 'section': 'Results'}, {'text': 'For the estimation of the growth rate, we focus on the number of confirmed cases in European countries that have experienced large local epidemics ( Figure 1A) , as reported by the WHO [28] . To avoid relying only on case confirmation, which could be affected by numerous biases, we also estimate the growth rate in hospitalisations, intensive care unit (ICU) admissions and deaths in Italy ( Figure 1B ; [5] ). To ensure generalisability of results, we performed another analysis on a larger set of European countries ( Figure 2 ). With the exception of a few countries with a small number of cases and potentially unreliable data, we consistently find doubling times of about 3 days or less, that appear to be sustained before mitigation interventions are put in place ( Figure 2 ). These are significantly shorter than early estimates from China [16, 29] . For robustness, we have used two different methods: semiparametric and generalised linear (details in Materials and Methods). Unsurprisingly, the results differ in terms of their confidence intervals, but the conclusions are similar, and are in agreement with the common exploratory analysis based on visually inspecting data plotted on a logarithmic scale ( Figure S3 ).', 'cite_spans': [{'start': 185, 'end': 189, 'text': '[28]', 'ref_id': 'BIBREF26'}, {'start': 405, 'end': 408, 'text': '[5]', 'ref_id': None}, {'start': 849, 'end': 853, 'text': '[16,', 'ref_id': 'BIBREF14'}, {'start': 854, 'end': 857, 'text': '29]', 'ref_id': 'BIBREF27'}], 'ref_spans': [{'start': 149, 'end': 159, 'text': 'Figure 1A)', 'ref_id': 'FIGREF0'}, {'start': 393, 'end': 402, 'text': 'Figure 1B', 'ref_id': 'FIGREF0'}, {'start': 521, 'end': 529, 'text': 'Figure 2', 'ref_id': None}, {'start': 773, 'end': 781, 'text': 'Figure 2', 'ref_id': None}, {'start': 1219, 'end': 1228, 'text': 'Figure S3', 'ref_id': 'FIGREF1'}], 'section': 'Results'}, {'text': 'Although our results are robust to the method used, they might still be misleading if there are biases in the data, such as errors in reporting, changes in case definition or testing regime, and so on. This issue is particularly critical as lags in reporting of cases can create discrepancies between national and international official sources [5, 28, 30] for case counts. However, these are unlikely to affect our conclusions owing to the following considerations:', 'cite_spans': [{'start': 345, 'end': 348, 'text': '[5,', 'ref_id': None}, {'start': 349, 'end': 352, 'text': '28,', 'ref_id': 'BIBREF26'}, {'start': 353, 'end': 356, 'text': '30]', 'ref_id': 'BIBREF28'}], 'ref_spans': [], 'section': 'Results'}, {'text': '\\uf0b7 The fast growth and high numbers likely make small biases negligible;', 'cite_spans': [], 'ref_spans': [], 'section': 'Results'}, {'text': '\\uf0b7 Any multiplicative correction, such as constant underreporting, will not affect the observed trend;', 'cite_spans': [], 'ref_spans': [], 'section': 'Results'}, {'text': '\\uf0b7 It is relatively easy for exponential growth to appear slower than reality, for example if reporting rates decline over time, but difficult for growth to appear consistently faster: aggressive swabbing of asymptomatic individuals (e.g. early on in the Italian locked-down towns; [31] ), as well as changes in case definition [32] , might explain such a bias in the data, but these factors are unlikely to affect observations for longer than a few days or consistently across different countries;', 'cite_spans': [{'start': 281, 'end': 285, 'text': '[31]', 'ref_id': 'BIBREF29'}, {'start': 327, 'end': 331, 'text': '[32]', 'ref_id': 'BIBREF30'}], 'ref_spans': [], 'section': 'Results'}, {'text': '\\uf0b7 Hospitalisation and ICU admission, which in the Italian data appear to grow at similar rates as the number of confirmed cases ( Figure 1B ), should be much less affected by reporting issues. The even faster increase in death rates we observe, instead, may be explained by rapid outbreaks among vulnerable groups, such as those in care homes, coupled with quicker progression to death among these groups, or possibly local hospital saturation.', 'cite_spans': [], 'ref_spans': [{'start': 130, 'end': 139, 'text': 'Figure 1B', 'ref_id': 'FIGREF0'}], 'section': 'Results'}, {'text': 'We conclude that, although existing data has its limitations, the evidence for fast exponential growth in the absence of intervention is overwhelming.', 'cite_spans': [], 'ref_spans': [], 'section': 'Results'}, {'text': 'The delay between infection and case detection is crucial in determining how long cases have been growing unobserved. Early detection of cases during the incubation period, before individuals have become symptomatic, is typically not possible once containment has failed since it relies on full contact tracing and testing of asymptomatic individuals. Detecting cases at symptom onset is potentially more feasible, but in practice depends on the case-finding strategy. For example, in the UK, symptomatic individuals are instructed to self-isolate at home, and are only tested if they subsequently require hospitalisation. Thus the delay between infection and case detection includes the incubation period, the time between symptom onset and hospitalisation, and the time it takes to receive a positive test result. Similar effects will be visible in other countries where case counts are dominated by hospitalisations.', 'cite_spans': [], 'ref_spans': [], 'section': 'Results'}, {'text': 'We report published estimates of the incubation period and the delay between symptom onset and hospitalisation (Table 1 ). Since none of these estimates simultaneously account for truncated observations and exponential growth in the number of infected cases, we also include our own estimates (see Materials and Methods) obtained by analysing UK line-list data provided by Public Health England (unfortunately not publicly available) and a publicly available line-list which collates worldwide data [33] . Although more robust, our estimates are consistent with the existing literature, and highlight geographical heterogeneity, such as shorter onset-to-hospitalisation intervals in Hong Kong and Singapore compared to the UK. With the exception of Singapore, the sum of the mean incubation period and mean onset-tohospitalisation interval is never shorter than 9 days, which corresponds to approximately 3 doubling times in an unconstrained epidemic like those observed in Figures 1 and 2 .', 'cite_spans': [{'start': 499, 'end': 503, 'text': '[33]', 'ref_id': 'BIBREF31'}], 'ref_spans': [{'start': 111, 'end': 119, 'text': '(Table 1', 'ref_id': None}, {'start': 974, 'end': 989, 'text': 'Figures 1 and 2', 'ref_id': 'FIGREF0'}], 'section': 'Results'}, {'text': 'Our estimate of the delay between infection and detection is consistent with observations of the UK and Italian epidemics ( Figure 3 ). For both countries, we plot the numbers of new cases and notice a visible drop occurring 8-9 days after the first, relatively soft, control measures were implemented (in the UK, recommended self-isolation if symptomatic on 13 March, and in Italy, lockdown of infected towns and school and university closure in Northern Italy, on 22-23 February). After the first control measure in the UK ( Figure 3A ), cases continued to increase exponentially with an estimated growth rate of ~0.22/day (corresponding to a doubling time of just over 3 days) for 9 days. During this period numbers of daily confirmed cases rose approximately 8-fold. Subsequently, the number of new infections started to tail off. A similar pattern is observed in Italy ( Figure 3B ). Because nonpharmaceutical interventions, with unknown compliance, cannot be evaluated until their effects emerge in the data, a pattern of introduction of increasingly strong measures has repeated across Europe, with long delays to control. Even with immediate hard interventions halting all community transmission, within-household transmission will continue to occur, creating an additional delay between the beginning of the intervention and its effect. This is consistent with the approximate 2-week delay from lockdown to peak in new cases observed in Hubei [4] . Further delays in case-confirmations, hospitalisations, potential ICU admissions and deaths mean the latter figures keep on growing well after transmission control is achieved.', 'cite_spans': [{'start': 1452, 'end': 1455, 'text': '[4]', 'ref_id': 'BIBREF3'}], 'ref_spans': [{'start': 124, 'end': 132, 'text': 'Figure 3', 'ref_id': 'FIGREF1'}, {'start': 527, 'end': 536, 'text': 'Figure 3A', 'ref_id': 'FIGREF1'}, {'start': 876, 'end': 885, 'text': 'Figure 3B', 'ref_id': 'FIGREF1'}], 'section': 'Results'}, {'text': 'Since R 0 remains the mainstay of most epidemiological analyses, we explored values of R 0 consistent with a range of growth rates and modelling assumptions. For a growth rate of 0.25/ day and our delay estimates (Table 1) , we obtain values ranging from 2 to 4 or larger (Table  S1A) , owing to the extreme sensitivity to assumptions, in particular the extent of presymptomatic transmission, for which estimates in the literature vary widely [34] [35] [36] [37] .', 'cite_spans': [{'start': 443, 'end': 447, 'text': '[34]', 'ref_id': 'BIBREF32'}, {'start': 448, 'end': 452, 'text': '[35]', 'ref_id': 'BIBREF33'}, {'start': 453, 'end': 457, 'text': '[36]', 'ref_id': 'BIBREF34'}, {'start': 458, 'end': 462, 'text': '[37]', 'ref_id': 'BIBREF35'}], 'ref_spans': [{'start': 213, 'end': 222, 'text': '(Table 1)', 'ref_id': None}, {'start': 272, 'end': 284, 'text': '(Table  S1A)', 'ref_id': None}], 'section': 'Results'}, {'text': 'Although R 0 is commonly used to determine the reduction in person-to-person transmission needed to achieve control, this brings limited insight into how to control the COVID-19 pandemic. First, there is sufficient evidence worldwide that interventions must be draconian, particularly in countries where case numbers are high. Second, given the variability in R 0 estimates obtained from the same growth rate appear predominantly due to the assumed amount of pre-symptomatic transmission, the exact value of R 0 is only poorly correlated with the required aggressiveness of the intervention [38] . For example, if R 0 is four and most of the transmission occurs after symptoms onset, 75% of the transmission needs to be stopped and self-isolation when symptomatic can easily achieve this. Conversely, if R 0 is two but most of the transmission is pre-symptomatic, only 50% of the transmission needs to be prevented, but in practice this can only be achieved through interventions, like quarantining apparently healthy individuals, that are highly socially disruptive and likely enforced rather than spontaneous. Finally, R 0 informs how aggressive interventions should be, but not how quickly they should be implemented.', 'cite_spans': [{'start': 591, 'end': 595, 'text': '[38]', 'ref_id': 'BIBREF36'}], 'ref_spans': [], 'section': 'Results'}, {'text': 'The highlighted risks of underestimating the combination of short doubling times and long delays between infection and case detection are consistent with the now-common pattern of countries misjudging the initial small number of observed cases, only to realise the storm has already arrived. At unconstrained growth, even the immense effort of doubling local hospital capacity only buys 3 days of reprieve before bed capacity is breached. Being blind to the extent of an epidemic and the true number of infections at any one time results in intervention strategies based on the number of observed cases and measurements of R 0 . For COVID-19, these are insufficient and dangerously underestimate the true degree of intervention required to slow down and bring the epidemic under control. We advocate stronger action from national and international health care communities, with a particular focus on supporting lowand middle-income countries where numbers of cases, at the time of writing, appear to be relatively low. In settings where health care capacity is low and intergenerational mixing common, swift action will save numerous lives. Note that the Italian Hospitalised and ICU cases are daily prevalence rather than daily incidence.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '12'}, {'text': 'The copyright holder for this preprint . https://doi.org/10.1101/2020.04.12.20059972 doi: medRxiv preprint Fig. 2 . Log daily confirmed cases and growth rate estimation for numerous European countries. Curves (black lines) are fit using the Generalised Linear Model methodology (see Materials and Methods) to the first 9 days non-zero data after a cumulative incidence of 25 is reached (period shown as a blue rectangle), which for most countries coincided with the start of sustained local transmission, with the exception of the UK and Romania where fitting started an additional 9 days after this criterion was met to reflect the local situation. Daily new cases are shown as red dots.', 'cite_spans': [], 'ref_spans': [{'start': 107, 'end': 113, 'text': 'Fig. 2', 'ref_id': None}], 'section': '(which was not peer-reviewed)'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '13'}, {'text': 'The copyright holder for this preprint . https://doi.org/10.1101/2020.04.12.20059972 doi: medRxiv preprint The fit is performed on circles; crosses are data added after the prediction is made. The exponential trend appears reasonably accurate for two more days, after which the effect of the first major intervention becomes visible.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': '14 . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'The copyright holder for this preprint ', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '15'}, {'text': 'The copyright holder for this preprint . https://doi.org/10.1101/2020.04.12.20059972 doi: medRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'We consider four sources for our epidemiological data: the WHO [28 ] , line-list data provided by Public Health England (PHE), line-list data from [33 ] and the Italian Istituto Superiore di Sanità [5 ] . Of these data sets, three are publicly available. The line-list from PHE is unfortunately not publicly available. From these sources, we extract epidemiological data concerning case counts, incidence, hospitalisation, and delays between infection and symptom onset, and onset of symptoms and hospitalisation. These data sources and the code used to carry out our data analysis can be found at: https://github.com/thomasallanhouse/covid19-growth.', 'cite_spans': [{'start': 63, 'end': 68, 'text': '[28 ]', 'ref_id': 'BIBREF26'}, {'start': 147, 'end': 152, 'text': '[33 ]', 'ref_id': 'BIBREF31'}, {'start': 198, 'end': 202, 'text': '[5 ]', 'ref_id': None}], 'ref_spans': [], 'section': 'Data Sources'}, {'text': 'Fitting the growth rate Typically, an infection spread from person to person will grow exponentially in the early phase of an epidemic. This exponential growth can be measured through the real time growth rate r so that, loosely speaking, the prevalence of infection is', 'cite_spans': [], 'ref_spans': [], 'section': 'Supplementary Text'}, {'text': 'A natural mathematical model to derive the estimate of r is a Poisson family generalised linear model (GLM) with a log link. Given the over-dispersed noise inherent in both disease dynamics and surveillance data, a quasi-Poisson family is considered here. The growth rate r is more intuitively reported as a doubling time (the time taken to double case numbers) and so t D = ln(2)/r. The log-linear analysis from a Poisson GLM defined formally below is restricted to datasets (or time windows) with clear exponential growth, or when additional explanatory variables, which are rarely available in real time, exist.', 'cite_spans': [], 'ref_spans': [], 'section': 'Supplementary Text'}, {'text': 'To allow, in semi-parametric manner, time variation in growth rates we adapt a generalised additive model (GAM) where I ∝ e s(t) for some smoother s(t). In particular, we use a quasi-Poisson family with canonical link and a thin-plate spline as implemented in the R package mgcv [45, 46 ] . The instantaneous local growth rate is then the time derivative of the smootherṡ(t) and an instantaneous doubling time calculated as t D = ln(2)/ṡ(t). Potential issues with the GAM approach include that extrapolation outside of the data range (and hence forecasting epidemic trend) is not sensible, and that there may be boundary effects from the choice of smoother. However, this approach has the major advantage that it allows for time-varying estimates of doubling time and thereby implicitly allows for missing explanatory information.', 'cite_spans': [{'start': 279, 'end': 283, 'text': '[45,', 'ref_id': 'BIBREF43'}, {'start': 284, 'end': 288, 'text': '46 ]', 'ref_id': 'BIBREF44'}], 'ref_spans': [], 'section': 'Supplementary Text'}, {'text': 'As well as the semi-parametric GAM approach, we take a parametric approach based on direct estimation of the exponential growth rate. This lacks the ability to capture time variation, but allows for extrapolation and epidemiological interpretation. To capture over-dispersion we use a quasi-Poisson family for the noise model. Explicitly, the Negative Binomial probability mass function (pmf) is', 'cite_spans': [], 'ref_spans': [], 'section': 'Supplementary Text'}, {'text': 'We will work in the parameterisation where the mean is µ and the variance is θµ, i.e.', 'cite_spans': [], 'ref_spans': [], 'section': 'Supplementary Text'}, {'text': 'Let the number of new cases on day t be y(t). We assume that this is generated by an exponentially growing mean,', 'cite_spans': [], 'ref_spans': [], 'section': 'Supplementary Text'}, {'text': 'which is then combined with the negative binomial pmf to give a likelihood function for the observations over a set of times T of L(y|y 0 , r, θ) = t∈T NB(y(t)|n(y 0 exp(rt), θ), p(θ)) ,', 'cite_spans': [], 'ref_spans': [], 'section': 'Supplementary Text'}, {'text': 'where y = (y(t)) t∈T . This can then be viewed as a generalised linear model (GLM) with time as a continuous covariate, intercept ln(y 0 ), slope r, exponential link function and negative binomial noise model [47 ] . We can perform inference through numerical maximum likelihood estimation (MLE) and calculate confidence intervals using the Laplace approximation [48 ] .', 'cite_spans': [{'start': 209, 'end': 214, 'text': '[47 ]', 'ref_id': 'BIBREF45'}, {'start': 363, 'end': 368, 'text': '[48 ]', 'ref_id': 'BIBREF46'}], 'ref_spans': [], 'section': 'Supplementary Text'}, {'text': 'Each page of Figure S1 shows the GAM compared to a simple GLM with θ = 1 and T taken to be all of the data range, in contrast to the results in the main paper and Figure S2 , where we fit θ and let T correspond to the first nine days of the local epidemic after the cumulative number of cases has reached 25 (the only exceptions are the UK and Romania, where fitting started an additional 9 days later to reflect the local situation). While the simple GLM method is clearly inadequate if the fit is not restricted to a window where exponential growth appears reasonable, it is shown for comparison in these plots. The left panel shows the output of the model fit and the data, the middle panel the instantaneous growth rate from GAM (black) and the growth rate from GLM (red) with 95% CI, and the right most panel shows these growth rates converted to doubling times. Of the fifteen countries, two (Belgium and Romania) have equivalent fits from GAM as from GLM and show constant growth rates over the time period. The Czech Republic, Greece, Ireland and Poland have the central GLM result within the 95% CI of the GAM suggesting a constant growth rate is a plausible explanation of the data reported. Austria, France, Italy, Portugal, Spain, Switzerland show a fairly smooth transition from short to longer doubling times. Germany, Netherlands and UK show more oscillatory behaviour in doubling times.', 'cite_spans': [], 'ref_spans': [{'start': 13, 'end': 22, 'text': 'Figure S1', 'ref_id': 'FIGREF0'}, {'start': 163, 'end': 172, 'text': 'Figure S2', 'ref_id': 'FIGREF3'}], 'section': 'Supplementary Text'}, {'text': \"As a further test for the robustness of our results, we simply visually assess the growth rates of the epidemic in a set of countries by plotting data on a log scale and compare the observed slopes with pure exponential trends ( Figure S3 ). To avoid relying on confirmed cases only, we plot a 'mixed bag' of cumulative cases, daily new confirmations, hospitalisations and deaths. We also plot exponential trends with a lower growth rate (r = 0.18 and r = 0.13) to aid the visual distinction in slopes between different exponential growths and the visual assessment of the potential effects of interventions in Italy.\", 'cite_spans': [], 'ref_spans': [{'start': 229, 'end': 238, 'text': 'Figure S3', 'ref_id': 'FIGREF1'}], 'section': 'Supplementary Text'}, {'text': 'Delay distributions describe the time delay between two events. To understand how long until the impact of an intervention may be observed, we need to understand the delay between infection and symptom onset (the incubation period), and the delay from onset to hospitalisation. A difficulty with estimating delay distributions during an outbreak is that events are only observed if they occur before the final sampling date. Since delay distributions depend on the time between two events, if the first event occurs near to the end of the sampling window, it will only be observed if the delay to the second event is short. This causes an over-expression of short delays towards the end of the sampling window, which is exacerbated by the exponential growth of the epidemic. Therefore, we need to account for this growth and truncation within our model.', 'cite_spans': [], 'ref_spans': [], 'section': 'Estimating delay distributions'}, {'text': 'To fit the data we use maximum likelihood estimation. However, we do not observe the delay directly, instead observing the timing of the two events. Therefore, we need to construct a likelihood function for observing these events. Following [49 ] , we construct the conditional density function for observing the second event given the time of the first event and given that the second event occurs before date T . That is, we are interested in the conditional density function', 'cite_spans': [{'start': 241, 'end': 246, 'text': '[49 ]', 'ref_id': 'BIBREF47'}], 'ref_spans': [], 'section': 'Estimating delay distributions'}, {'text': 'where x 1 and x 2 can be exactly observed or interval censored. The delay from onset to hospitalisation for the UK is estimated using FF100 data provided by Public Health England, which contains data on the first few hundred infected individuals in the UK. This data incorporated the time of symptom onset and time of hospitalisation. There were some cases who were hospitalised before their onset date. These cases have been removed from the data set, since they do not provide insight into the delay. Additionally, some cases have no symptom onset, so these have also been removed from the data. For cases where symptom onset and hospitalisation occur on the same day, we add half a day to the hospitalisation day, since the delay is unlikely to be instantaneous. After tidying the data, this left 106 cases from which to infer the onset to hospitalisation delay. The dates in the line list are recorded exactly, so the likelihood function becomes', 'cite_spans': [], 'ref_spans': [], 'section': 'Estimating delay distributions'}, {'text': 'where f is the density of the onset to hospitalisation delay and g is the density of the onset time. Using this truncation corrected method and a gamma distribution to fit the delay distribution, we get a mean delay of 5.14 with standard deviation 4.20. Unfortunately, we cannot share the FF100 data. To compare different regions, we also use data from Hong Kong and Singapore to estimate the local onset to hospitalisation delays. This data is taken from an open access line-list [33 ] , and the filtered data sets used are provided in the supplementary material. Using the method above, for Hong Kong the mean delay is 4.41 days, with standard deviation 4.63, and for Singapore the mean delay is 2.62 with standard deviation 2.38. For the incubation period, we use data from Wuhan during the early stages of the outbreak. This data was extracted from an open access line-list [33 ] , containing dates when individuals were in Wuhan and when they developed symptoms (among other information). Since this data is from the early stages of the epidemic, the majority of cases were in Wuhan. Therefore, it is likely that these individuals were infected in Wuhan, so the time spent in Wuhan provides a potential exposure window during which infection occurred. For individuals with symptom onset date before leaving Wuhan or the same day they left Wuhan, the upper bound on the exposure window was adjusted to half a day before symptom onset. Using the data as of 21/02/2020, we have 162 cases from which to infer the incubation period. This infection date is interval censored, so we obtain the likelihood function', 'cite_spans': [{'start': 481, 'end': 486, 'text': '[33 ]', 'ref_id': 'BIBREF31'}, {'start': 878, 'end': 883, 'text': '[33 ]', 'ref_id': 'BIBREF31'}], 'ref_spans': [], 'section': 'Estimating delay distributions'}, {'text': 'where f is the incubation period density function and g is the density of the infection date. We assume g is proportional to force of infection of the outbreak, which is assumed to follow exponential growth with rate parameter 0.25 day −1 . Using a gamma distribution to describe the incubation period, we get a mean incubation of 4.84 days with standard deviation 2.79. The data for the incubation period is also included as supplementary material, along with MATLAB code to perform the maximum likelihood estimation.', 'cite_spans': [], 'ref_spans': [], 'section': 'Estimating delay distributions'}, {'text': 'The relationship between the growth rate r and the basic reproduction number R 0 in a simple homogeneously mixing model is provided by the Lotka-Euler equation:', 'cite_spans': [], 'ref_spans': [], 'section': 'Estimation of R 0'}, {'text': 'where τ represents the time since the infection of an individual and ω(τ ) is the infectious contact interval distribution, defined as the probability density function (pdf) of the times (since infection) at which an infectious contact is made. An infectious contact is a contact that results in an infection if the contactee is susceptible, and early on in the epidemic any randomly selected contactee is almost surely susceptible. Equation (S9) assumes all individuals have the same infectious contact interval distribution. However, if we assume random variability between individuals, there will be a set S of curves Ω(τ ). However, equation (S9) still applies, with ω(τ ) being the time-point average of all curves in S [50 ] ; see Figure S4 .', 'cite_spans': [{'start': 725, 'end': 730, 'text': '[50 ]', 'ref_id': 'BIBREF48'}], 'ref_spans': [{'start': 737, 'end': 746, 'text': 'Figure S4', 'ref_id': 'FIGREF6'}], 'section': 'Estimation of R 0'}, {'text': 'The generation time T g is defined as the mean of the infectious contact interval distribution ω:', 'cite_spans': [], 'ref_spans': [], 'section': 'Estimation of R 0'}, {'text': 'The same definition extends to a random infectivity profiles of which ω is the time-point average.', 'cite_spans': [], 'ref_spans': [], 'section': 'Estimation of R 0'}, {'text': 'For the incubation period we use our estimates from Table 1 (mean 4.84, standard deviation 2.79), which are anyway similar to those estimated by others. However, information about any form of pre-symptomatic transmission is hard to obtain but crucial for R 0 estimates [34] [35] [36] [37] . Furthermore, there is also limited information concerning how infectivity changes over time. Therefore, Table S1 reports the estimates we obtain assuming the infectious period starts at the onset of symptoms, one, two or three days earlier, and assuming a Gamma-shaped infectivity with mean 2 or 3 days. In both cases, the standard deviation is assumed to be 1.5 and the infectivity is truncated after 7 days (see Figure S4 ).', 'cite_spans': [{'start': 269, 'end': 273, 'text': '[34]', 'ref_id': 'BIBREF32'}, {'start': 274, 'end': 278, 'text': '[35]', 'ref_id': 'BIBREF33'}, {'start': 279, 'end': 283, 'text': '[36]', 'ref_id': 'BIBREF34'}, {'start': 284, 'end': 288, 'text': '[37]', 'ref_id': 'BIBREF35'}], 'ref_spans': [{'start': 395, 'end': 403, 'text': 'Table S1', 'ref_id': None}, {'start': 705, 'end': 714, 'text': 'Figure S4', 'ref_id': 'FIGREF6'}], 'section': 'Estimation of R 0'}, {'text': 'We conclude that the estimates of R 0 are highly sensitive to small variations in quantities that are poorly supported by available data, but that for a growth rate of 0.25 day −1 , close to what is observed in Italy and the UK, are also generally larger, and possibly much larger, than official estimates [3, [12] [13] [14] [15] . Smaller values in this range are associated with significant amounts of pre-symptomatic transmission [34 ] , leading to a generation time for example compatible with some of the shortest estimates of the serial interval seen in the literature [51 ] , and with a front-loaded infectivity curve (mean 2, rather than 3).', 'cite_spans': [{'start': 306, 'end': 309, 'text': '[3,', 'ref_id': 'BIBREF2'}, {'start': 310, 'end': 314, 'text': '[12]', 'ref_id': 'BIBREF11'}, {'start': 315, 'end': 319, 'text': '[13]', 'ref_id': 'BIBREF12'}, {'start': 320, 'end': 324, 'text': '[14]', 'ref_id': None}, {'start': 325, 'end': 329, 'text': '[15]', 'ref_id': 'BIBREF13'}, {'start': 433, 'end': 438, 'text': '[34 ]', 'ref_id': 'BIBREF32'}, {'start': 575, 'end': 580, 'text': '[51 ]', 'ref_id': 'BIBREF49'}], 'ref_spans': [], 'section': 'Estimation of R 0'}, {'text': 'We tested further assumptions. A simple SEIR model, with exponentially distributed incubation and infectious periods (with the same means as above but constant infectivity) leads to much smaller values of R 0 than our estimates, as it favours really short incubation periods (Table S1B, left) . Estimates, instead, do not change significantly if high variability in total infectiousness between individuals, in line with what observed for SARS, is assumed (Table S1B , right) or if 50% of cases are assumed to be fully asymptomatic and transmit at half the rate as those with symptoms (not shown).', 'cite_spans': [], 'ref_spans': [{'start': 275, 'end': 292, 'text': '(Table S1B, left)', 'ref_id': None}, {'start': 456, 'end': 466, 'text': '(Table S1B', 'ref_id': None}], 'section': 'Estimation of R 0'}, {'text': 'These simple estimates are obtained under the assumption of mass-action mixing. The explicit presence of a social structure (e.g. age-stratification, household/network structure, etc.), which in principle could affect them, is likely negligible in such a high R 0 and growth rate regime [52 ] . The effect of the social structure on transmission is expected to grow in importance (especially the household structure, since isolation and quarantine facilitate within-household transmission) the closer R 0 is to 1. 5 . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 6 . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 287, 'end': 292, 'text': '[52 ]', 'ref_id': None}], 'ref_spans': [], 'section': 'Estimation of R 0'}, {'text': 'The copyright holder for this preprint . https://doi.org/10.1101/2020.04.12.20059972 doi: medRxiv preprint 7 . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': '(which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.04.12.20059972 doi: medRxiv preprint 8 . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': '(which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.04. 12 Generation time:', 'cite_spans': [{'start': 104, 'end': 106, 'text': '12', 'ref_id': 'BIBREF11'}], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'Generation time: Table S1 : Values of R 0 derived from different growth rates and different modelling assumptions. A) Gammadistributed latent period with estimates from Table 1 , and Gamma-shaped infectivity profile with mean 2 (left) and 3 (right) and standard deviation 1.5; B) SEIR model (left) and same model as in A) but assuming total infectiousness is randomly drawn from a Gamma-distribution with mean 1 and standard deviation 1/ √ k, with k = 0.25.', 'cite_spans': [], 'ref_spans': [{'start': 17, 'end': 25, 'text': 'Table S1', 'ref_id': None}, {'start': 169, 'end': 176, 'text': 'Table 1', 'ref_id': None}], 'section': '(which was not peer-reviewed)'}], 'paragraphs': [{'text': 'The delay between infection and case detection is crucial in determining how long cases have been growing unobserved. Early detection of cases during the incubation period, before individuals have become symptomatic, is typically not possible once containment has failed since it relies on full contact tracing and testing of asymptomatic individuals. Detecting cases at symptom onset is potentially more feasible, but in practice depends on the case-finding strategy. For example, in the UK, symptomatic individuals are instructed to self-isolate at home, and are only tested if they subsequently require hospitalisation. Thus the delay between infection and case detection includes the incubation period, the time between symptom onset and hospitalisation, and the time it takes to receive a positive test result. Similar effects will be visible in other countries where case counts are dominated by hospitalisations.', 'id': '00014'}], 'title': 'Challenges in control of Covid-19: short doubling time and long delay to effect of interventions', 'sha': '16f7c659cd73992378cff884358b75594bc66a0d', 'doi': '10.1101/2020.04.12.20059972', 'score': 9.41100025177002}, {'rank': 106, 'paragraphs': [{'text': 'The delay between infection and case detection is crucial in determining how long cases have been growing unobserved. Early detection of cases during the incubation period, before individuals have become symptomatic, is typically not possible once containment has failed since it relies on full contact tracing and testing of asymptomatic individuals. Detecting cases at symptom onset is potentially more feasible, but in practice depends on the case-finding strategy. For example, in the UK, symptomatic individuals are instructed to self-isolate at home, and are only tested if they subsequently require hospitalisation. Thus the delay between infection and case detection includes the incubation period, the time between symptom onset and hospitalisation, and the time it takes to receive a positive test result. Similar effects will be visible in other countries where case counts are dominated by hospitalisations.', 'id': '00012'}], 'title': 'Challenges in control of Covid-19: short doubling time and long delay to effect of interventions', 'sha': '7e7dd4bbf8f6d4f6d8d292311a15e7d024571642', 'doi': '', 'score': 9.410999298095703}, {'rank': 107, 'paragraphs': [{'text': 'The copyright holder for this preprint this version posted June 22, 2020. . https://doi.org/10.1101/2020.06.20.20134387 doi: medRxiv preprint 7 / 16 11 studies with sample size less than 5, 1 study of insufficient quality, and 1 with ambiguous definition of incubation period were ineligible. Meanwhile, 2 additional studies were identified via reference list searches. Finally, there were 56 studies that met the inclusion criteria, including 34 published studies, 22 preprint studies, and 4 095 COVID-19 infections in total. Literature quality was evaluated for each included study according to AHRQ guidelines (Supplement Table 5 ). Summarized statistics of incubation periods and population characteristics extracted from each study were shown in Supplement Table 6 .', 'id': '00021'}], 'title': 'A systematic review and meta-analysis reveals long and dispersive incubation period of COVID-19', 'sha': '3d921c03dc23ee850655ddfd9d5ebe3956d70bf8', 'doi': '10.1101/2020.06.20.20134387', 'score': 9.40939998626709}, {'rank': 108, 'paragraphs': [{'text': 'We have conceptually separated the time until there is any substantial difference in model trajectories and the time until we would be able to estimate the impact of a change in distancing, using reported case counts under conditions in British Columbia. These times may differ as a result of reporting delay, but also in some circumstances it may be sufficient to observe signal of a change in distancing under a lower probability or a lower case threshold. Figure 4 shows the daily maximum likelihood estimate of the physical distancing parameter f over time. We find that it took approximately 26 days to accept the value of f 1 (according to the methods described in Section 2.3). We estimate f 1 to be 0.22 on the day of acceptance (April 13th), but this estimate increases to 0.36 by the end of the observation period at 35 days after distancing: April 22nd. Several larger cluster outbreaks had begun to be observed in British Columbia by this time [4] , which may have contributed to this increase in the estimated distancing parameter.', 'id': '00028'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.40410041809082}, {'rank': 109, 'paragraphs': [{'text': 'In this study, it is reasonable to assume (A1) is satisfied between January 19, 2020, and January 23, 2020, because there are over eleven million residents in the Wuhan metropolitan area and nearby neighborhoods and the daily travel volume in and out of Wuhan exceeded million before January 23, 2020. We justify the use of data between January 19 and January 23 below. With adequate long run, the renewal process would reach the equilibrium status. The assumptions (A2) to (A4) are standard. In fact, we may assume that the incubation period is a continuous variable with range (0, M) for some finite number M. It is well known that the first moment exists for a bounded random variable. The justification for assumption (A5) is below. Therefore, the probability renewal process theory can be applied with confidence, and thus we can avoid the challenging mission of ascertaining the backward time.', 'id': '00008'}], 'title': 'Estimation of incubation period distribution of COVID-19 using disease onset forward time: a novel cross-sectional and forward follow-up study', 'sha': 'c61be9fc4562d250e2dec241ef75bd22d61f61d1; 6d3b3f4ab80a61c45f82c61c6c756cfc6ddf4bf2', 'doi': '10.1101/2020.03.06.20032417', 'score': 9.399399757385254}, {'rank': 110, 'paragraphs': [{'text': 'Further, the distribution of the incubation period was simulated for the general transmissions, asymptomatic transmissions, and young-age infections, respectively. The 1 000 posterior means of the log-scaled distribution of incubation period were generated using the Bayesian model that produces probability distribution for each parameter, within R package bayesmeta (18) . In addition, the 1 000 standard deviations of the log-scaled distribution were generated by Bootstrap sampling. The 1 000 means and standard deviations of the log-scaled distribution were used to simulate the distribution of incubation period. The proportion of infections developing symptoms after a certain length of quarantine were estimated; the risk of symptomatic SARS-CoV-2 infections being undetected after a certain length of active monitoring among the active-monitoring population was estimated as well (19, 20) .', 'id': '00017'}], 'title': 'A systematic review and meta-analysis reveals long and dispersive incubation period of COVID-19', 'sha': '3d921c03dc23ee850655ddfd9d5ebe3956d70bf8', 'doi': '10.1101/2020.06.20.20134387', 'score': 9.397700309753418}, {'rank': 111, 'abstract': 'Abstract In December 2019, the outbreak of the novel coronavirus disease (COVID-19) in China spread worldwide, becoming an emergency of major international concern. SARS-CoV-2 infection causes clusters of severe respiratory illness similar to severe acute respiratory syndrome coronavirus. Human-to-human transmission via droplets, contaminated hands or surfaces has been described, with incubation times of 2-14 days. Early diagnosis, quarantine, and supportive treatments are essential to cure patients. This paper reviews the literature on all available information about the epidemiology, diagnosis, isolation and treatments of COVID-19. Treatments, including antiviral agents, chloroquine and hydroxychloroquine, corticosteroids, antibodies, convalescent plasma transfusion and vaccines, are discussed in this article. In addition, registered trials investigating treatment options for COVID-19 infection are listed.', 'body_text': [{'text': 'There is a current worldwide outbreak of a new type of coronavirus (COVID-19), which originated from Wuhan, China and has now spread to 140 other countries, including Japan, Korea and Italy. The World Health Organization (WHO) declared that COVID-19 has become a global health concern, causing severe respiratory tract infections in humans. Current evidence indicates that SARS-CoV-2 spread to humans via transmission from wild animals illegally sold in the Huanan Seafood Wholesale Market. Phylogenetic analysis shows that SARS-CoV-2 is a new member of the Coronaviridae family but is distinct from SARS-CoV (identity of approximately 79%) and MERS-CoV (identity of approximately 50%) [1,2]. Knowing the origin of such a pathogen is critical to developing the means to block further transmission and vaccines [3]. Notably, SARS-CoV-2 shares a high level of genetic similarity (96.3%) with the bat coronavirus RaTG13, which was obtained from bats in Yunnan in 2013; however, bats are not the immediate source of SARS-CoV-2 [4].', 'cite_spans': [{'start': 686, 'end': 688, 'mention': '[1', 'ref_id': 'BIBREF0'}, {'start': 689, 'end': 691, 'mention': '2]', 'ref_id': 'BIBREF1'}, {'start': 810, 'end': 813, 'mention': '[3]', 'ref_id': 'BIBREF2'}, {'start': 1023, 'end': 1026, 'mention': '[4]', 'ref_id': 'BIBREF3'}], 'section': 'Introduction', 'ref_spans': []}, {'text': 'The typical symptoms of COVID-19 are fever, sore throat, fatigue, cough or dyspnea coupled with recent exposure. As of March 16, 2020, the outbreak of COVID-19 generated 168 826 confirmed cases, including 6503 deaths worldwide. In China during the outbreak of the pandemic, 42 000 doctors and nurses from all over the country supported Wuhan. Moreover, the government shared the updated genome sequence of COVID-19 to the public, and scientists from China and overseas are working closely and efficiently on this public health emergency [5,6]. Due to interventions and control measures from the government (shutting down public transportation and implementing a treatment strategy) and the change in personal behaviors (wearing masks and reducing contact with others), the number of confirmed and suspected cases in China has started to decrease.', 'cite_spans': [{'start': 537, 'end': 539, 'mention': '[5', 'ref_id': 'BIBREF4'}, {'start': 540, 'end': 542, 'mention': '6]', 'ref_id': 'BIBREF5'}], 'section': 'Introduction', 'ref_spans': []}, {'text': 'However, the transmission of pneumonia associated with SARS-CoV-2 has not yet been eliminated. The COVID-19 outbreak is still a major challenge for clinicians. The aim of this article is to describe the epidemiology, diagnosis, isolation, and treatment of COVID-19.', 'cite_spans': [], 'section': 'Introduction', 'ref_spans': []}, {'text': 'A study of early transmission dynamics of COVID-19 revealed that the mean incubation period was 5.2 days (95% confidence interval [CI], 4.1-7.0), with the 95th percentile of the distribution at 12.5 days [7]. A later study using the travel history and symptom onset of 88 confirmed cases showed a similar mean incubation period of 6.4 days (95% CI, 5.6-7.7) [8]. An unusual case was also reported in which the incubation period was as long as 19 days [9]. Notably, a long incubation time means adjustments in screening and control policies [10]. The 19-day incubation period is a low probability event, and experts suggest 14 days for quarantine.', 'cite_spans': [{'start': 204, 'end': 207, 'mention': '[7]', 'ref_id': 'BIBREF6'}, {'start': 358, 'end': 361, 'mention': '[8]', 'ref_id': 'BIBREF7'}, {'start': 451, 'end': 454, 'mention': '[9]', 'ref_id': 'BIBREF8'}, {'start': 540, 'end': 544, 'mention': '[10]', 'ref_id': 'BIBREF9'}], 'section': 'Incubation period ::: Epidemiology', 'ref_spans': []}, {'text': 'The basic reproduction number is model-based, largely depends on the epidemiological setting, and is the most important parameter to determine intrinsic transmissibility. The early outbreak data largely follow exponential growth. Different models based on the clinical progression of the disease were devised to estimate the basic reproduction number. In the early stages of COVID-19, the pandemic doubled in size every 7.4 days, and the basic reproduction number was estimated to be 2.2 [7]. Another study estimated the basic reproduction number as ranging from 2.24 to 3.58 [11]. However, a deterministic compartmental model based on the likelihood and a model analysis showed that the control reproduction number may be as high as 6.47 [12]. The authors noted that this basic reproduction number was higher because the estimate accounts for 3-4 generations of viral transmission and intensive social contacts. The basic reproduction number estimated by the majority of studies ranges from 2.24 to 3.58 [13], which is slightly higher than that of SARS.', 'cite_spans': [{'start': 488, 'end': 491, 'mention': '[7]', 'ref_id': 'BIBREF6'}, {'start': 576, 'end': 580, 'mention': '[11]', 'ref_id': 'BIBREF10'}, {'start': 739, 'end': 743, 'mention': '[12]', 'ref_id': 'BIBREF11'}, {'start': 1005, 'end': 1009, 'mention': '[13]', 'ref_id': 'BIBREF12'}], 'section': 'Basic reproduction number ::: Epidemiology', 'ref_spans': []}, {'text': 'Rapid and accurate detection of COVID-19 is crucial to control outbreaks in the community and in hospitals [14]. Current diagnostic tests for coronavirus include reverse-transcription polymerase chain reaction (RT-PCR), real‐time RT-PCR (rRT‐PCR), and reverse transcription loop‐mediated isothermal amplification (RT‐LAMP) [15,16]. RT-LAMP has similar sensitivity to rRT-PCR, is highly specific and is used to detect MERS-CoV [17,18]. According to current diagnostic criteria founded by the China National Health Commission, laboratory examinations, including nasopharyngeal and oropharyngeal swab tests, have become a standard assessment for diagnosis of COVID-19 infection. To identify patients earlier, two one-step quantitative RT-PCR (qRT-PCR) assays were developed to detect two different regions (ORF1b and N) of the SARS-CoV-2 genome [19]. Three novel RT-PCR assays targeting the RNA-dependent RNA polymerase (RdRp)/helicase (Hel), spike (S), and nucleocapsid (N) genes of SARS-CoV-2 were developed. Among the three novel assays, the COVID-19-RdRp/Hel assay had the lowest limit of detection in vitro; highly sensitive and specific assays may help to improve the laboratory diagnosis of COVID-19 [20]. The SARS-CoV E gene assay was more sensitive than the RdRp gene assay combined with the one-step RT-PCR system [21]. The E gene PCR was sufficient to diagnose a SARS-CoV-2 infection but the RdRp protocol was recommended to confirm a positive result [22,23]. The overall positive rate of RT-PCR detection of SARS-CoV-2 infection in 4880 cases from one hospital in Wuhan was 38% [24]. The positive rate of PCR for oropharyngeal swabs is not very high: only 53.3% of COVID-19-confirmed patients had positive oral swabs tests [25]. In a series of 51 patients with confirmed COVID-19 infection, 71% patients were RT-PCR positive at the first time of testing of throat swab or sputum samples [26]. The RT-PCR results usually become positive after several days (2-8 days) [27]. Automated solutions for molecular diagnostics can handle large numbers of samples and can be scaled to keep pace with fluctuating demand [28], [29], [30]. The good analytical performance of a molecular assay for the detection of SARS-CoV-2 on a high-throughput platform, the cobas 6800, was observed with minimal hands-on time, while offering fast and reliable results [31,32].', 'cite_spans': [{'start': 107, 'end': 111, 'mention': '[14]', 'ref_id': 'BIBREF13'}, {'start': 323, 'end': 326, 'mention': '[15', 'ref_id': 'BIBREF14'}, {'start': 327, 'end': 330, 'mention': '16]', 'ref_id': 'BIBREF15'}, {'start': 426, 'end': 429, 'mention': '[17', 'ref_id': 'BIBREF16'}, {'start': 430, 'end': 433, 'mention': '18]', 'ref_id': 'BIBREF17'}, {'start': 842, 'end': 846, 'mention': '[19]', 'ref_id': 'BIBREF18'}, {'start': 1204, 'end': 1208, 'mention': '[20]', 'ref_id': 'BIBREF19'}, {'start': 1321, 'end': 1325, 'mention': '[21]', 'ref_id': 'BIBREF20'}, {'start': 1459, 'end': 1462, 'mention': '[22', 'ref_id': 'BIBREF21'}, {'start': 1463, 'end': 1466, 'mention': '23]', 'ref_id': 'BIBREF22'}, {'start': 1587, 'end': 1591, 'mention': '[24]', 'ref_id': 'BIBREF23'}, {'start': 1732, 'end': 1736, 'mention': '[25]', 'ref_id': 'BIBREF24'}, {'start': 1896, 'end': 1900, 'mention': '[26]', 'ref_id': 'BIBREF25'}, {'start': 1975, 'end': 1979, 'mention': '[27]', 'ref_id': 'BIBREF26'}, {'start': 2118, 'end': 2122, 'mention': '[28]', 'ref_id': 'BIBREF27'}, {'start': 2124, 'end': 2128, 'mention': '[29]', 'ref_id': 'BIBREF28'}, {'start': 2130, 'end': 2134, 'mention': '[30]', 'ref_id': 'BIBREF29'}, {'start': 2350, 'end': 2353, 'mention': '[31', 'ref_id': 'BIBREF30'}, {'start': 2354, 'end': 2357, 'mention': '32]', 'ref_id': 'BIBREF31'}], 'section': 'Diagnosis', 'ref_spans': []}, {'text': 'The current laboratory test is time-consuming, and a shortage of commercial kits delays diagnosis. For patients suffering from fever, sore throat, fatigue, coughing or dyspnea that is coupled with recent exposure, COVID-19 infection should be diagnosed with typical chest computerized tomography (CT) characteristics despite negative RT-PCR results [33]. Of 1014 patients, 59% had positive RT-PCR results, and 88% had positive chest CT scans [34]. COVID-19 belongs to the Coronaviridae family; therefore, it is not surprising that COVID-19 has imaging findings that are similar to those for SARS-CoV and MERS-CoV [35]. Typical CT findings included bilateral pulmonary parenchymal ground-glass and consolidative pulmonary opacities, sometimes with a rounded morphology and peripheral lung distribution [33]. Eighty-six percent of patients showed ground-glass opacities or consolidation, and more than one lobe (71%) with bilateral involvement (76%) was affected in the 21 initial chest CT scans [36]. Notably, lung cavitation, discrete pulmonary nodules, pleural effusions, and lymphadenopathy were absent [36]. Lung abnormalities on chest CT scan were most severe approximately 10 days after the initial onset of symptoms [37]. Chest CT scans can be used to assess the severity of COVID-19. COVID-19 also manifests with chest CT imaging abnormalities in asymptomatic patients, with rapid evolution from focal unilateral to diffuse bilateral ground-glass opacities that progressed to or co-existed with consolidations within 1-3 weeks. Combining assessment of imaging features with clinical and laboratory findings could facilitate early diagnosis of COVID-19 pneumonia [38], [39], [40]. As the diagnostic criteria expanded from laboratory examination to chest CT imaging, more than 14 000 patients were diagnosed on February 12, 2020.', 'cite_spans': [{'start': 349, 'end': 353, 'mention': '[33]', 'ref_id': 'BIBREF32'}, {'start': 442, 'end': 446, 'mention': '[34]', 'ref_id': 'BIBREF33'}, {'start': 613, 'end': 617, 'mention': '[35]', 'ref_id': 'BIBREF34'}, {'start': 801, 'end': 805, 'mention': '[33]', 'ref_id': 'BIBREF32'}, {'start': 994, 'end': 998, 'mention': '[36]', 'ref_id': 'BIBREF35'}, {'start': 1105, 'end': 1109, 'mention': '[36]', 'ref_id': 'BIBREF35'}, {'start': 1222, 'end': 1226, 'mention': '[37]', 'ref_id': 'BIBREF36'}, {'start': 1669, 'end': 1673, 'mention': '[38]', 'ref_id': 'BIBREF37'}, {'start': 1675, 'end': 1679, 'mention': '[39]', 'ref_id': 'BIBREF38'}, {'start': 1681, 'end': 1685, 'mention': '[40]', 'ref_id': 'BIBREF39'}], 'section': 'Diagnosis', 'ref_spans': []}, {'text': 'Classical public health measures, including isolation, quarantine, social distancing and community containment, can be used to curb the pandemic of this respiratory disease [41]. China has been preparing since 2003 to contain future pandemics by applying lessons learned from SARS [42]. In the COVID-19 pandemic, China issued the largest quarantine in history. All the residents living in mainland China were locked-in, and city public transportation, including buses, trains, ferries, and airports, were shutdown. Given the trajectory of this outbreak, the Chinese government scaled up such efforts to keep pace with the rapid increase in cases and geographical spread. The Wuhan government made full use of the gym and two convention centers and transformed them into makeshift hospitals with 3400 beds in only one night to isolate COVID-19 patients from healthy controls. More makeshift hospitals are under construction. Isolation beds were quickly expanded from only 137 at the beginning of the outbreak of COVID-19 to 56 000 to separate infected patients from non-infected individuals. The swift and decisive response of China contributed to reducing the control reproduction number and transmission risk. Due to the powerful and effective isolation measures taken by the Chinese government, the increase in COVID-19 began to slow down on February 14, 2020, according to the data released by the China National Health Commission.', 'cite_spans': [{'start': 173, 'end': 177, 'mention': '[41]', 'ref_id': 'BIBREF40'}, {'start': 281, 'end': 285, 'mention': '[42]', 'ref_id': 'BIBREF41'}], 'section': 'Isolation', 'ref_spans': []}, {'text': 'There is no current evidence from randomized controlled trials (RCTs) to recommend any specific anti-SARS-CoV-2 treatment for patients with a suspected or confirmed COVID-19 infection. Lopinavir (LPV) inhibits the protease activity of coronavirus in vitro and in animal studies. A retrospective, matched-cohort study including 1052 SARS patients showed that LPV/ritonavir as initial treatment was associated with a reduced death rate (2.3% vs. 11.0%) [43]. The protease inhibitor LPV is an effective treatment based on the experience accumulated from the SARS and MERS outbreaks, indicating it is a potential treatment option for COVID‐19 [44]. Ribavirin, a guanosine analogue, is an antiviral compound used to treat several virus infections, including respiratory syncytial virus, hepatitis C virus, and some viral hemorrhagic fevers. Promising results were obtained with ribavirin in a MERS-CoV rhesus macaque model [45]. In addition, SARS-CoV-2 RNA-dependent RNA polymerase (RdRp) model is targeted by ribavirin after sequence analysis, modeling, and docking to build the model. This feature increases its potential as an antiviral against SARS-CoV-2 [46].', 'cite_spans': [{'start': 451, 'end': 455, 'mention': '[43]', 'ref_id': 'BIBREF42'}, {'start': 639, 'end': 643, 'mention': '[44]', 'ref_id': 'BIBREF43'}, {'start': 918, 'end': 922, 'mention': '[45]', 'ref_id': 'BIBREF44'}, {'start': 1154, 'end': 1158, 'mention': '[46]', 'ref_id': 'BIBREF45'}], 'section': 'Antiviral agents ::: Treatments', 'ref_spans': []}, {'text': \"The antiviral agent, remdesivir was designed for the Ebola virus disease [47]. Remdesivir shows broad-spectrum antiviral activity against several RNA viruses, and it may compete for RdRp [48]. Remdesivir and IFNb have superior antiviral activity to LPV and ritonavir in vitro [49]. In a mouse model of SARS-CoV pathogenesis, both prophylactic and therapeutic remdesivir improved pulmonary function and reduced lung viral loads and severe lung pathology [50]. In a rhesus macaque model of MERS-CoV infection, prophylactic remdesivir treatment was initiated 24 h prior to inoculation, and MERS-CoV did not induce clinical disease and did not replicate in respiratory tissues, thus preventing the formation of lung lesions [51]. In cell-based assays, the triphosphate form of remdesivir incorporated at position i, and RNA chain termination was delayed, which explained the high potency of remdesivir against RNA [52]. Remdesivir was used to treat the first case of COVID-19 infection in the United States: the patient's clinical condition improved after only one day of remdesivir treatment [53]. A phase II clinical trial of remdesivir was performed by the University of Nebraska Medical Center, and a phase III clinical trial was performed by the China-Japan Friendship Hospital. The results of these clinical trials will be revealed in April 2020. Remdesivir improved pulmonary function, reduced lung viral loads, and ameliorated severe lung pathology. In contrast, prophylactic LPV/RTV-IFNb only slightly reduced viral loads and did not impact other disease parameters, and therapeutic LPV/RTV-IFNb improved pulmonary function, but did not reduce virus replication or severe lung pathology [49]. Overall, these results indicated that remdesivir showed more potential than LPV/RTV-IFNb [54]. In a case report, lopinavir/ritonavir (Kaletra®) and arbidol were associated with significant improvements in COVID-19 patients [55]. The efficacy and safety of these antiviral agents for COVID-19 will be assessed in further clinical trials. Thirty-four trials of antiviral agents in patients with COVID‐19 have been registered up to March 15, 2020 (Table 1\\n).\", 'cite_spans': [{'start': 73, 'end': 77, 'mention': '[47]', 'ref_id': 'BIBREF46'}, {'start': 187, 'end': 191, 'mention': '[48]', 'ref_id': 'BIBREF47'}, {'start': 276, 'end': 280, 'mention': '[49]', 'ref_id': 'BIBREF48'}, {'start': 453, 'end': 457, 'mention': '[50]', 'ref_id': 'BIBREF49'}, {'start': 720, 'end': 724, 'mention': '[51]', 'ref_id': 'BIBREF50'}, {'start': 910, 'end': 914, 'mention': '[52]', 'ref_id': 'BIBREF51'}, {'start': 1089, 'end': 1093, 'mention': '[53]', 'ref_id': 'BIBREF52'}, {'start': 1787, 'end': 1791, 'mention': '[54]', 'ref_id': 'BIBREF53'}, {'start': 1921, 'end': 1925, 'mention': '[55]', 'ref_id': 'BIBREF54'}], 'section': 'Antiviral agents ::: Treatments', 'ref_spans': [{'start': 2143, 'end': 2150, 'mention': 'Table 1', 'ref_id': 'TABREF0'}]}, {'text': 'Chloroquine is a widely-used antimalarial and autoimmune disease drug that has been reported to be a potential broad-spectrum antiviral drug [56], [57], [58]. Chloroquine is known to block virus infection by increasing endosomal pH required for virus/cell fusion, as well as interfering with the glycosylation of cellular receptors of SARS-CoV [59]. The first results obtained from more than 100 patients showed the apparent efficacy of chloroquine in terms of reduction of exacerbation of pneumonia, duration of symptoms and delay of viral clearance, all in the absence of severe side effects [60]. Chloroquine was included in the recommendations for the prevention and treatment of COVID-19 pneumonia [60,61]. The optimal dosage of chloroquine for SARS-CoV-2 will need to be assessed in future trials [62].', 'cite_spans': [{'start': 141, 'end': 145, 'mention': '[56]', 'ref_id': 'BIBREF55'}, {'start': 147, 'end': 151, 'mention': '[57]', 'ref_id': 'BIBREF56'}, {'start': 153, 'end': 157, 'mention': '[58]', 'ref_id': 'BIBREF57'}, {'start': 344, 'end': 348, 'mention': '[59]', 'ref_id': 'BIBREF58'}, {'start': 594, 'end': 598, 'mention': '[60]', 'ref_id': 'BIBREF59'}, {'start': 703, 'end': 706, 'mention': '[60', 'ref_id': 'BIBREF59'}, {'start': 707, 'end': 710, 'mention': '61]', 'ref_id': 'BIBREF60'}, {'start': 803, 'end': 807, 'mention': '[62]', 'ref_id': 'BIBREF61'}], 'section': 'Chloroquine and hydroxychloroquine ::: Treatments', 'ref_spans': []}, {'text': 'Hydroxychloroquine is a chloroquine analog for which there are fewer concerns about drug-drug interactions [63]. In the previous SARS outbreak, hydroxychloroquine was reported to have anti-SARS-CoV activity in vitro [64]. Using physiologically-based pharmacokinetic (PBPK) models, hydroxychloroquine was found to be more potent than chloroquine in SARS-CoV-2-infected Vero cells [65]. Cytokines IL-6 and IL-10 have been reported to be increased in response to SARS-CoV-2 infection [66,67]. This may progress to a cytokine storm, followed by multiorgan failure and death. Both chloroquine and hydroxychloroquine have immunomodulatory effects and can suppress the immune response [68,69]. Therefore, 21 clinical studies were launched by Chinese hospitals and the University of Oxford to evaluate the efficacy of these agents in COVID-19 infection (Table 2\\n). It is also necessary to determine whether the benefit of chloroquine therapy depends on the age of the patient and the clinical presentation or stage of the disease [70]. If clinical data confirm the biological results, chloroquine and hydroxychloroquine may be used in prophylaxis as well as curative treatment for individuals exposed to SARS-CoV-2 [71].', 'cite_spans': [{'start': 107, 'end': 111, 'mention': '[63]', 'ref_id': 'BIBREF62'}, {'start': 216, 'end': 220, 'mention': '[64]', 'ref_id': 'BIBREF63'}, {'start': 379, 'end': 383, 'mention': '[65]', 'ref_id': 'BIBREF64'}, {'start': 481, 'end': 484, 'mention': '[66', 'ref_id': 'BIBREF65'}, {'start': 485, 'end': 488, 'mention': '67]', 'ref_id': 'BIBREF66'}, {'start': 678, 'end': 681, 'mention': '[68', 'ref_id': 'BIBREF67'}, {'start': 682, 'end': 685, 'mention': '69]', 'ref_id': 'BIBREF68'}, {'start': 1022, 'end': 1026, 'mention': '[70]', 'ref_id': 'BIBREF69'}, {'start': 1207, 'end': 1211, 'mention': '[71]', 'ref_id': 'BIBREF70'}], 'section': 'Chloroquine and hydroxychloroquine ::: Treatments', 'ref_spans': [{'start': 846, 'end': 853, 'mention': 'Table 2', 'ref_id': 'TABREF1'}]}, {'text': 'In a study of 41 COVID-19 patients, 21% received corticosteroids, which could suppress lung inflammation [66]. The administered dose of methylprednisolone varied depending on disease severity. Current interim guidance from the WHO on the clinical management of severe acute respiratory infection when SARS-CoV-2 infection is suspected (released January 28, 2020) advises against the use of corticosteroids unless indicated for another reason. The clinical outcomes of coronavirus and similar outbreaks do not support the use of corticosteroids. In a retrospective observational study of 309 adults who were critically ill with MERS, patients who were given corticosteroids were more likely to require mechanical ventilation, vasopressors, and renal replacement therapy [72]. For the management of SARS, corticosteroid treatment was more associated with psychosis, diabetes and avascular necrosis [73,74]. Overall, there is no unique reason to expect that patients with COVID-19 infection will benefit from corticosteroids, and such treatment may be harmful [75]. However, according to our clinical experience, corticosteroids could be prescribed at the right time for the right patients. The clinical trials involving corticosteroids are shown in Table 3\\n.', 'cite_spans': [{'start': 105, 'end': 109, 'mention': '[66]', 'ref_id': 'BIBREF65'}, {'start': 769, 'end': 773, 'mention': '[72]', 'ref_id': 'BIBREF71'}, {'start': 896, 'end': 899, 'mention': '[73', 'ref_id': 'BIBREF72'}, {'start': 900, 'end': 903, 'mention': '74]', 'ref_id': 'BIBREF73'}, {'start': 1057, 'end': 1061, 'mention': '[75]', 'ref_id': 'BIBREF74'}], 'section': 'Corticosteroids ::: Treatments', 'ref_spans': [{'start': 1247, 'end': 1254, 'mention': 'Table 3', 'ref_id': 'TABREF2'}]}, {'text': 'The development of vaccines and therapeutic antibodies against COVID-19 has important implications. Considering the relatively high identity of the receptor-binding domain (RBD) in SARS-CoV-2 and SARS-CoV, the cross-reactivity of anti-SARS-CoV antibodies with the COVID-19 spike protein was assessed. The spike protein is the major inducer of neutralizing antibodies. Fortunately, the SARS-CoV-specific human monoclonal antibody CR3022 binds potently with the COVID-19 RBD [76]. However, other SARS-CoV RBD-directed antibodies 230, m396 and 80R cannot bind to the COVID-19 RBD [77]. CR3022 may be a potential therapeutic candidate, alone or in combination with other neutralizing antibodies, for the prevention and treatment of COVID-19 infections. Antibodies MAb114 and REGN-EB3 were designed for treatment of Ebola virus infection and significantly reduce mortality from Ebola virus disease [47]. Monoclonal antibodies can only recognize a single antigen epitope, which limits the use of MAb114 and REGN-EB3 in the treatment of COVID-19. However, the development of COVID-19-specific antibodies requires a long time. It is not easy to apply monoclonal antibodies for new pathogens to clinical practice in a short time.', 'cite_spans': [{'start': 473, 'end': 477, 'mention': '[76]', 'ref_id': 'BIBREF75'}, {'start': 577, 'end': 581, 'mention': '[77]', 'ref_id': 'BIBREF76'}, {'start': 893, 'end': 897, 'mention': '[47]', 'ref_id': 'BIBREF46'}], 'section': 'Antibodies ::: Treatments', 'ref_spans': []}, {'text': 'Convalescent plasma was administered early after symptom onset in the treatment of SARS, and the pooled odds of mortality following treatment was reduced compared with placebo or no therapy (odds ratio, 0.25) [78]. However, in Ebola virus disease, the transfusion of up to 500 mL of convalescent plasma in 84 patients was not associated with a significant improvement in survival [79]. In a laboratory test, the COVID-19 virus was isolated from the bronchoalveolar lavage fluid of a critically ill patient, and it could be neutralized by sera from several patients [80]. Current clinical trials involving convalescent plasma transfusion are shown in Table 4\\n. The National Health Commission of China appealed to convalescent patients to donate blood for the treatment of COVID-19 infection. Convalescent plasma should be collected within two weeks after recovery to ensure a high neutralization antibody titer. The difficulty in obtaining plasma during convalescence limits its clinical application. Well-designed clinical trials are needed to further evaluate the efficacy and safety of convalescent plasma therapy in patients with COVID-19 infection.', 'cite_spans': [{'start': 209, 'end': 213, 'mention': '[78]', 'ref_id': 'BIBREF77'}, {'start': 380, 'end': 384, 'mention': '[79]', 'ref_id': 'BIBREF78'}, {'start': 565, 'end': 569, 'mention': '[80]', 'ref_id': 'BIBREF79'}], 'section': 'Convalescent plasma transfusion ::: Treatments', 'ref_spans': [{'start': 650, 'end': 657, 'mention': 'Table 4', 'ref_id': 'TABREF3'}]}, {'text': \"The structure of SARS-CoV-2 S protein has been revealed, and this should enable the rapid development and evaluation of medical countermeasures to address the ongoing public health crisis [77]. These findings provide the basis for further studies to optimize vaccination strategies for this emerging infection. The majority of the vaccines being developed for coronaviruses target the spike glycoprotein or S protein [81]. Vaccine development is a long process, and no vaccines are available at the time of a pandemic outbreak. For example, the Ebola epidemic outbreak occurred in 2013, and three years later, the rVSV Ebola Vaccine was selected for phase I clinical trials for its safety and immunogenicity in Africa and Europe [82]. In November 2019, the European Commission granted marketing authorization to Merck Sharp and Dohme B.V. in Europe for their Ebola vaccine, Ervebo. Fortunately, Moderna company announced on February 24, 2020 that the company's experimental mRNA COVID-19 vaccine, known as mRNA-1273, is ready for human testing. It is a remarkably fast development cycle to develop an initial vaccine just weeks after identifying the SARS-CoV-2 genetic sequence. The clinical trial of safety and immunogenicity of mRNA-1273 in the treatment of COVID-19 is under investigation (ClinicalTrials.gov Identifier: NCT04283461). Moreover, a new oral SARS-CoV-2 vaccine has been successfully developed at Tianjin University, which uses food-grade safe Saccharomyces cerevisiae as a carrier and targets the S protein. There are 18 biotechnology companies and universities in China working on SARS-CoV-2 vaccines. Vaccines for SARS-CoV-2 have been developed much faster than those for Ebola because of the collaborative efforts of scientists around the world and the fast-track approval of SARS-CoV-2 vaccine development efforts by the Chinese health organizations.\", 'cite_spans': [{'start': 188, 'end': 192, 'mention': '[77]', 'ref_id': 'BIBREF76'}, {'start': 417, 'end': 421, 'mention': '[81]', 'ref_id': 'BIBREF80'}, {'start': 729, 'end': 733, 'mention': '[82]', 'ref_id': 'BIBREF81'}], 'section': 'Vaccines ::: Treatments', 'ref_spans': []}, {'text': 'Bats have been recognized as a natural reservoir and vectors of a variety of coronaviruses, and these viruses have crossed species barriers to infect humans and many different kinds of animals, including avians, rodents, and chiropters [83,84]. While the origin of COVID-19 is still being investigated, COVID-19 has features typical of the Coronaviridae family and was classified in the beta-coronavirus 2b lineage. COVID-19 can be transmitted between humans. Interventions, including intensive contact tracing followed by quarantine and isolation, can effectively reduce the spread of COVID-19, with the effect of travel restrictions. Wearing masks, washing hands and disinfecting surfaces contribute to reducing the risk of infection. Human coronaviruses can be efficiently inactivated within 1 min using surface disinfection procedures with 62-71% ethanol, 0.5% hydrogen peroxide or 0.1% sodium hypochlorite [85].', 'cite_spans': [{'start': 236, 'end': 239, 'mention': '[83', 'ref_id': 'BIBREF82'}, {'start': 240, 'end': 243, 'mention': '84]', 'ref_id': 'BIBREF83'}, {'start': 911, 'end': 915, 'mention': '[85]', 'ref_id': 'BIBREF84'}], 'section': 'Conclusions', 'ref_spans': []}, {'text': 'Identification of the causative viral pathogens of respiratory tract viral infections is important to select an appropriate treatment, control the pandemic, and reduce the economic impact of COVID-19 on China and the world. In acute respiratory infection, RT-PCR is routinely used to detect causative viruses from respiratory secretions. The positive rate of PCR from oropharyngeal swabs is not very high. In this situation, more swab testing is needed to clarify diagnosis. Typical CT findings can help early screening of suspected cases and diagnosis of COVID-19.', 'cite_spans': [], 'section': 'Conclusions', 'ref_spans': []}, {'text': 'The COVID-19 infection has a clustering onset and is more likely to affect older males (average age 51 years) with comorbidities [86]. No evidence supports adverse birth outcomes, intrauterine infection, or vertical transmission of COVID-19 [87]. However, viral infections can be acquired when the infant passes through the birth canal during vaginal delivery or through postpartum breastfeeding [88]. The most common symptoms were fever, cough, expectoration, headache, myalgia or fatigue, diarrhea, and hemoptysis [89]. Some people may experience severe acute respiratory distress syndrome. Histological examination of lung biopsy samples showed bilateral diffuse alveolar damage with cellular fibromyxoid exudates [90]. Other organs are also susceptible to COVID-19. The single-cell RNA-seq data was used to analyse receptor ACE2 expression to reveal the potential risk of different human organs to COVID-19 infection [91]. COVID-19 uses the same cell entry receptor as SARS-CoV, ACE2, which regulates both cross-species and human-to-human transmissions [80]. Proximal tubular cells also express higher levels of the ACE2 receptor, which leads to susceptibility to COVID-19 [91] and induces kidney injury. Data from 33 patients with a complete clinical course were analysed, and the levels of blood urea and creatinine were higher in non-survivors than in survivors [92].', 'cite_spans': [{'start': 129, 'end': 133, 'mention': '[86]', 'ref_id': 'BIBREF85'}, {'start': 241, 'end': 245, 'mention': '[87]', 'ref_id': 'BIBREF86'}, {'start': 396, 'end': 400, 'mention': '[88]', 'ref_id': 'BIBREF87'}, {'start': 516, 'end': 520, 'mention': '[89]', 'ref_id': 'BIBREF88'}, {'start': 717, 'end': 721, 'mention': '[90]', 'ref_id': 'BIBREF89'}, {'start': 921, 'end': 925, 'mention': '[91]', 'ref_id': 'BIBREF90'}, {'start': 1057, 'end': 1061, 'mention': '[80]', 'ref_id': 'BIBREF79'}, {'start': 1177, 'end': 1181, 'mention': '[91]', 'ref_id': 'BIBREF90'}, {'start': 1369, 'end': 1373, 'mention': '[92]', 'ref_id': 'BIBREF91'}], 'section': 'Conclusions', 'ref_spans': []}, {'text': 'All patients with COVID-19-infected pneumonia received antibacterial agents, 90% received antiviral therapy, and 45% received methylprednisolone [92]. Clinical trials are underway to investigate the efficacy of new antiviral drugs, convalescent plasma transfusion, and vaccines. Most of the trials were initiated by investigators and the study period is 1 to 11 months. Although the final results of these studies will take a long time to complete, the interim research data may provide some help for the current urgent demand for therapy [93].', 'cite_spans': [{'start': 145, 'end': 149, 'mention': '[92]', 'ref_id': 'BIBREF91'}, {'start': 539, 'end': 543, 'mention': '[93]', 'ref_id': 'BIBREF92'}], 'section': 'Conclusions', 'ref_spans': []}, {'text': 'The COVID-19 pandemic is a public health emergency of international concern, and all countries need a coordinated international effort to fight COVID-19. The transmission of pneumonia associated with SARS-CoV-2 has not yet been eliminated. In the absence of vaccines and antivirals, isolation and quarantine are achieving remarkable results. It is necessary to strengthen the monitoring of COVID-19 and to develop drugs and vaccines against the COVID-19 infection as soon as possible.', 'cite_spans': [], 'section': 'Conclusions', 'ref_spans': []}], 'paragraphs': [{'text': 'A study of early transmission dynamics of COVID-19 revealed that the mean incubation period was 5.2 days (95% confidence interval [CI], 4.1-7.0), with the 95th percentile of the distribution at 12.5 days [7]. A later study using the travel history and symptom onset of 88 confirmed cases showed a similar mean incubation period of 6.4 days (95% CI, 5.6-7.7) [8]. An unusual case was also reported in which the incubation period was as long as 19 days [9]. Notably, a long incubation time means adjustments in screening and control policies [10]. The 19-day incubation period is a low probability event, and experts suggest 14 days for quarantine.', 'id': '00004'}], 'title': 'The epidemiology, diagnosis and treatment of COVID-19', 'sha': '0c9d951acb01cb541671b3065b882bbcb61f9523', 'doi': '10.1016/j.ijantimicag.2020.105955', 'score': 9.397000312805176}, {'rank': 112, 'paragraphs': [{'text': 'Increasing evidence supports the transmission potential of SARS-CoV-2 during the latent period (7-9).', 'id': '00026'}], 'title': 'A systematic review and meta-analysis reveals long and dispersive incubation period of COVID-19', 'sha': '3d921c03dc23ee850655ddfd9d5ebe3956d70bf8', 'doi': '10.1101/2020.06.20.20134387', 'score': 9.396900177001953}, {'rank': 113, 'paragraphs': [{'text': 'The copyright holder for this preprint this version posted June 22, 2020. . https://doi.org/10.1101/2020.06.20.20134387 doi: medRxiv preprint 10 / 16 low viral load that requires a longer incubation to develop symptoms (23, 24) . On the other hand, young-age infections probably have a strong immune status that results in a longer incubation.', 'id': '00028'}], 'title': 'A systematic review and meta-analysis reveals long and dispersive incubation period of COVID-19', 'sha': '3d921c03dc23ee850655ddfd9d5ebe3956d70bf8', 'doi': '10.1101/2020.06.20.20134387', 'score': 9.391300201416016}, {'rank': 114, 'paragraphs': [{'text': 'Funnel plots and Egger tests were used to show the potential publication bias and study heterogeneity. The expected mean of log-scaled incubation period was summarized by meta-analysis followed by exponential calculation to obtain the median of incubation period and the corresponding 95% confidence interval (95%CI). Random-effects meta-analysis was used if P value for heterogeneity test ≤ 0.05; otherwise, fixed-effects meta-analysis would be used.', 'id': '00014'}], 'title': 'A systematic review and meta-analysis reveals long and dispersive incubation period of COVID-19', 'sha': '3d921c03dc23ee850655ddfd9d5ebe3956d70bf8', 'doi': '10.1101/2020.06.20.20134387', 'score': 9.389800071716309}, {'rank': 115, 'paragraphs': [{'text': 'Our search retrieved 13 711 records, of which 13 403 were irrelevant to incubation period and were excluded during screening of titles and abstracts ( Figure 1 ). Fifty-three duplicate studies and 1 study that had no access to full text were removed. By careful screening the full text of the remaining 254 studies, we found that 130 with no summarized statistics for incubation period, 57 non-human studies, . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'id': '00020'}], 'title': 'A systematic review and meta-analysis reveals long and dispersive incubation period of COVID-19', 'sha': '3d921c03dc23ee850655ddfd9d5ebe3956d70bf8', 'doi': '10.1101/2020.06.20.20134387', 'score': 9.389799118041992}, {'rank': 116, 'paragraphs': [{'text': 'In addition, three publications reported characteristics of incubation among cases infected by asymptomatic or presymptomatic carriers, and two studies had a subset of cases infected by carriers in the asymptomatic period (14, 15) ; these five studies were labeled as \"Asymptomatic Transmissions\".', 'id': '00011'}], 'title': 'A systematic review and meta-analysis reveals long and dispersive incubation period of COVID-19', 'sha': '3d921c03dc23ee850655ddfd9d5ebe3956d70bf8', 'doi': '10.1101/2020.06.20.20134387', 'score': 9.38599967956543}, {'rank': 117, 'paragraphs': [{'text': 'The copyright holder for this preprint this version posted June 22, 2020. . https://doi.org/10.1101/2020.06.20.20134387 doi: medRxiv preprint 6 / 16 incubation period of the recruited studies were assumed to follow the inverse gamma distribution to estimate the expectation value. Bootstrap was used to estimate the corresponding 95%CI of the dispersion.', 'id': '00016'}], 'title': 'A systematic review and meta-analysis reveals long and dispersive incubation period of COVID-19', 'sha': '3d921c03dc23ee850655ddfd9d5ebe3956d70bf8', 'doi': '10.1101/2020.06.20.20134387', 'score': 9.385998725891113}, {'rank': 118, 'paragraphs': [{'text': 'In the study of Guan et al on behalf of China Medical Treatment Expert Group for COVID-19, the incubation period had a reported median of 4 days, the first quartile of 2 days and the third quartile of 7 days.4 By fitting a commonly used Weibull distribution to such quartiles, we can obtain α^=1.24 and λ^=0.186 defined in Equation (2). As a consequence, the estimated 90%, 95% and 99% percentiles are, respectively, 10·54, 13·04 and 18·45 days, which indicates that some patients may have extended incubation periods.One particular case reported by Yinbin municipal health commissions in China stated that a 64-year-old female was diagnosed with COVID-19 on February 11, 2020 at Yinbin, Sichuan province 20 days after returning from Wuhan. This patient was under self-quarantine at home with the family for 18 days, from January 23 to February 9. On February 8, the patient developed mild symptoms of cough with sputum production.21', 'id': '00023'}], 'title': 'Estimation of incubation period distribution of COVID-19 using disease onset forward time: a novel cross-sectional and forward follow-up study', 'sha': 'c61be9fc4562d250e2dec241ef75bd22d61f61d1; 6d3b3f4ab80a61c45f82c61c6c756cfc6ddf4bf2', 'doi': '10.1101/2020.03.06.20032417', 'score': 9.384400367736816}, {'rank': 119, 'abstract': '', 'body_text': [{'text': 'With over 3.3 million confirmed cases worldwide and over 230 000 confirmed deaths, the COVID-19 pandemic is, without doubt, the most important event of the 21st century so far. The COVID-19 pandemic has upended all areas of life - and sports is no exception. Unsurprisingly, this pandemic has also stopped the sporting calendar, with professional leagues everywhere suspending their activities to limit the spread of the virus. Guidelines for COVID-19 diagnosis change on a day-to-day basis, as we progress with primary studies and gain more. The guidelines encompass clinical criteria, epidemiological patient history, and molecular diagnostics. Football coaches are keen on perspective and realize that the challenges they face now and in the months ahead hardly compare to the hardships the world confronts during a pandemic, but the return to pre-pandemic activities requires medical guidelines.', 'cite_spans': [], 'section': '', 'ref_spans': []}, {'text': 'Croatian Football Federation, with its Medical Committee is launching a new model of pre-season systematic examination of football players with a particular emphasis on diagnosing COVID-19. Studies suggest that the proportion of asymptomatic cases ranges from 17.9% onboard the Diamond Princess cruise ship, up to 78% detected in newly reported Chinese data [1,2]. In our opinion, there is little doubt that COVID-19 is far more widely distributed than some may believe, knowing the data implies that an asymptomatic person can spread the infection, particularly during the incubation period [3-5]. Identifying the asymptomatic carriers of the disease has become crucial in preventing further spread of the epidemic.', 'cite_spans': [{'start': 359, 'end': 360, 'mention': '1', 'ref_id': 'BIBREF0'}, {'start': 361, 'end': 362, 'mention': '2', 'ref_id': 'BIBREF7'}, {'start': 593, 'end': 594, 'mention': '3', 'ref_id': 'BIBREF8'}, {'start': 595, 'end': 596, 'mention': '5', 'ref_id': 'BIBREF10'}], 'section': '', 'ref_spans': []}, {'text': 'It is expected that majority of people who recover from COVID-19 will not have long-term consequences. However, COVID-19 is a multi-organ disease commonly affecting the lung, heart, kidney, digestive tract, and nervous system, with unclear situation regarding long-term consequences [6,7]. Survivors of the severe COVID-19 disease were reported to have changes in their lungs, similar to those observed in SARS, marked by the diminished pulmonary capacity [8]. Important lessons on long term outcomes for patients who contracted COVID-19 are still to be learned, but in order to adapt to the global situation, we believe it is paramount to draw parallels with the epidemics of SARS (severe acute respiratory syndrome) and MERS (Middle East Respiratory Syndrome). A recent study showed that approximately 20% of COVID-19 patients suffered from cardiac injuries [9]. Studies of SARS and MERS reported high occurrence of hypertension, persistent tachycardia and myocarditis in convalescent patients. A study on 121 patients who were infected with SARS showed that hypertension occurred in over half of all the patients, while 71.9% of them developed persistent tachycardia [10]. One the other hand, it has been shown that MERS causes myocarditis, most likely by the direct viral infection; the same was implied for COVID-19 [11]. Acute kidney injury and proteinuria were also reported in COVID-19 patients, suggesting direct cellular damage of the kidney tissue [7]. By comparison, COVID-19 patients who were treated in the ICU, particularly those who were mechanically ventilated, were reported to suffer from “post-intensive care syndrome”, most likely because the lack of oxygen in blood [12]. All these reports suggest that COVID-19 infection might be regarded as higher risk than it was initially believed. This is of paramount importance in football activities, knowing that career development requires a lot of time and effort invested into prevention of injury, disease, disability and even death [13]. The sport community finds itself facing uncharted territories in both the wake and the aftermath of this pandemic. Therefore, we firmly believe that coordinated, well-communicated and transparent action is of utmost importance if we want the return to regular activities be safe for all stakeholders.', 'cite_spans': [{'start': 284, 'end': 285, 'mention': '6', 'ref_id': 'BIBREF11'}, {'start': 286, 'end': 287, 'mention': '7', 'ref_id': 'BIBREF12'}, {'start': 457, 'end': 458, 'mention': '8', 'ref_id': 'BIBREF13'}, {'start': 861, 'end': 862, 'mention': '9', 'ref_id': 'BIBREF14'}, {'start': 1171, 'end': 1173, 'mention': '10', 'ref_id': 'BIBREF1'}, {'start': 1322, 'end': 1324, 'mention': '11', 'ref_id': 'BIBREF2'}, {'start': 1460, 'end': 1461, 'mention': '7', 'ref_id': 'BIBREF12'}, {'start': 1689, 'end': 1691, 'mention': '12', 'ref_id': 'BIBREF3'}, {'start': 2003, 'end': 2005, 'mention': '13', 'ref_id': 'BIBREF4'}], 'section': '', 'ref_spans': []}, {'text': 'Although professional football players are generally a healthy population without chronic respiratory, cardiac, renal diseases, as well as other chronic conditions, we must take COVID-19 seriously and act accordingly before returning to football pitches. Underlying genetic factors must also be taken into account as they might be aggravated by COVID-19 causing their clinical manifestation. Together with the list of medical examinations that players must undergo in order to be eligible to participate in UEFA competitions, we hereby propose a model for screening professional football players returning to the field after the lift of the ban on all sport activities because of the COVID-19 pandemic. Besides pre-season physical examination (primarily 12-lead ECG, ECG, spirometry with bronchodilatation test, diffusing capacity of the lung for carbon monoxide (DLCO) test and fractional exhaled nitric oxide (FeNO) test) and medical examination defined by the UEFA Medical Regulations (for the next season), we are proposing that every football player from the Croatian first National League must have negative consecutive two RT-qPCR COVID-19 pharyngeal swabs over a 5-day interval. Such tests are targeting two regions of the viral nucleocapsid gene (N1 and N2) or RNA-dependent RNA polymerase (RdRP) and envelope (E) genes [14]. This is of special interest due to the long virus incubation – median incubation period for COVID-19 was estimated to approximately five days [15]. However, it has been noted that time from exposure to onset of infectiousness (latent period) may be shorter than the incubation period [5]. Therefore, it is essential to do two subsequent tests during the proposed period. In addition to the detection of viral genetic material, we will target the immune response of the athlete being screened, looking specifically for antibodies (IgM and IgG) against the virus or viral antigens. Those tests are less complex than molecular tests but since antibody responses to infection take days to weeks to be detectable, serologic tests will not be reliable among those with recent exposure to virus. However, antibodies detected by this test indicate that a person had an immune response to COVID-19, implying the infection was subclinical if the person was asymptomatic. Serologic tests could play an important role in establishing diagnosis, if the COVID-19 patient with late complications of disease is examined since RT-qPCR could produce false-negative results, presumably because of the low viral load [14]. In addition to limiting the potential of viral spread with the start of regular sport activities, the results of this screening protocol will allow us to estimate how many football players have been infected nationally. The results will also provide information on the percentage of Croatian football players who have not had COVID-19 and are still at risk of being infected.', 'cite_spans': [{'start': 1330, 'end': 1332, 'mention': '14', 'ref_id': 'BIBREF5'}, {'start': 1478, 'end': 1480, 'mention': '15', 'ref_id': 'BIBREF6'}, {'start': 1620, 'end': 1621, 'mention': '5', 'ref_id': 'BIBREF10'}, {'start': 2533, 'end': 2535, 'mention': '14', 'ref_id': 'BIBREF5'}], 'section': '', 'ref_spans': []}, {'text': 'We propose that football players need a gradual return to physical activities during four separate phases. The first phase includes training in a small group, while the second phase comprises training of the entire team. Consequently, players will start with the national leagues’ competition (phase three), while in phase four, the clubs will be joining international competitions. Ideally, the club’s prior international competition should provide for all registered players, certificates issued by the accredited laboratories that the players are negative for COVID-19. In addition to all these procedures mentioned above, special preventive recommendations will be given to the football players and other team members in addition to the above described screening program prior to the continuation of competitive matches. Those include the following: 1) Trainings must be performed outdoors. Entry into club rooms and other closed spaces are prohibited. 2) No more than 5 players can participate in training sessions at the same time. The personal distance must be met at all times with at least 5 m separating the players. Players must use own lanes for running and sprints, if this requirement can’t be met, the same lane may be used by more players, but they must keep a distance between each other of at least 40 m when sprinting. Headers are not allowed in training sessions due to close contact of the ball with orifices of the body. 3) The number of coaches in training sessions should be kept to a minimum. Apart from the coaches only a physio or a team doctor may be present in the session. Protective equipment is mandatory for the medical team when they are getting in close contact to a player. Protective equipment includes face masks (N95, FFP2 or FFP3), protective gloves and face shields. 4) Original, sealed plastic water bottles must be used and properly discarded after training. They may not be shared between players. 5) Players and coaches must come to the training grounds alone, in their own cars, wearing appropriate clothes. Changing and shower rooms will not be available for the athletes nor coaches. 6) After the training session both players and coaches must go to their homes using the same transport they came with. This should be done orderly whilst respecting the social distancing measures. We propose to implement these measures first in the Croatian First League. They will also be recommended to lower leagues when they start with training and competitions and will be adhered to until a broader relaxation of preventive measures is advised by the local authorities. We presented our model to the leadership of FIFA and UEFA, with the goal of sharing our knowledge and ideas, as well as synchronising the actions of all the members during the COVID-19 pandemic. We believe that adherence to the recommendations and testing of players will drastically reduce the risk of them being exposed to SARS-CoV-2 and other pathogens. In turn, it should allow a steady return to football we all know and love.', 'cite_spans': [], 'section': '', 'ref_spans': []}], 'paragraphs': [{'text': 'Although professional football players are generally a healthy population without chronic respiratory, cardiac, renal diseases, as well as other chronic conditions, we must take COVID-19 seriously and act accordingly before returning to football pitches. Underlying genetic factors must also be taken into account as they might be aggravated by COVID-19 causing their clinical manifestation. Together with the list of medical examinations that players must undergo in order to be eligible to participate in UEFA competitions, we hereby propose a model for screening professional football players returning to the field after the lift of the ban on all sport activities because of the COVID-19 pandemic. Besides pre-season physical examination (primarily 12-lead ECG, ECG, spirometry with bronchodilatation test, diffusing capacity of the lung for carbon monoxide (DLCO) test and fractional exhaled nitric oxide (FeNO) test) and medical examination defined by the UEFA Medical Regulations (for the next season), we are proposing that every football player from the Croatian first National League must have negative consecutive two RT-qPCR COVID-19 pharyngeal swabs over a 5-day interval. Such tests are targeting two regions of the viral nucleocapsid gene (N1 and N2) or RNA-dependent RNA polymerase (RdRP) and envelope (E) genes [14]. This is of special interest due to the long virus incubation – median incubation period for COVID-19 was estimated to approximately five days [15]. However, it has been noted that time from exposure to onset of infectiousness (latent period) may be shorter than the incubation period [5]. Therefore, it is essential to do two subsequent tests during the proposed period. In addition to the detection of viral genetic material, we will target the immune response of the athlete being screened, looking specifically for antibodies (IgM and IgG) against the virus or viral antigens. Those tests are less complex than molecular tests but since antibody responses to infection take days to weeks to be detectable, serologic tests will not be reliable among those with recent exposure to virus. However, antibodies detected by this test indicate that a person had an immune response to COVID-19, implying the infection was subclinical if the person was asymptomatic. Serologic tests could play an important role in establishing diagnosis, if the COVID-19 patient with late complications of disease is examined since RT-qPCR could produce false-negative results, presumably because of the low viral load [14]. In addition to limiting the potential of viral spread with the start of regular sport activities, the results of this screening protocol will allow us to estimate how many football players have been infected nationally. The results will also provide information on the percentage of Croatian football players who have not had COVID-19 and are still at risk of being infected.', 'id': '00004'}], 'title': 'Pre-season football preparation in the era of COVID-19: Croatian Football Association Model', 'sha': '0d8f3a31a7fabaa7e4df53296d09c9dcf7b71f78', 'doi': '10.7189/jogh.10.010352', 'score': 9.376500129699707}, {'rank': 120, 'paragraphs': [{'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'id': '00003'}], 'title': 'A systematic review and meta-analysis reveals long and dispersive incubation period of COVID-19', 'sha': '3d921c03dc23ee850655ddfd9d5ebe3956d70bf8', 'doi': '10.1101/2020.06.20.20134387', 'score': 9.375900268554688}, {'rank': 121, 'paragraphs': [{'text': 'The copyright holder for this preprint this version posted June 22, 2020. . https://doi.org/10.1101/2020.06.20.20134387 doi: medRxiv preprint', 'id': '00004'}], 'title': 'A systematic review and meta-analysis reveals long and dispersive incubation period of COVID-19', 'sha': '3d921c03dc23ee850655ddfd9d5ebe3956d70bf8', 'doi': '10.1101/2020.06.20.20134387', 'score': 9.375899314880371}, {'rank': 122, 'paragraphs': [{'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'id': '00008'}], 'title': 'A systematic review and meta-analysis reveals long and dispersive incubation period of COVID-19', 'sha': '3d921c03dc23ee850655ddfd9d5ebe3956d70bf8', 'doi': '10.1101/2020.06.20.20134387', 'score': 9.375898361206055}, {'rank': 123, 'paragraphs': [{'text': 'The copyright holder for this preprint this version posted June 22, 2020. . https://doi.org/10.1101/2020.06.20.20134387 doi: medRxiv preprint', 'id': '00009'}], 'title': 'A systematic review and meta-analysis reveals long and dispersive incubation period of COVID-19', 'sha': '3d921c03dc23ee850655ddfd9d5ebe3956d70bf8', 'doi': '10.1101/2020.06.20.20134387', 'score': 9.375897407531738}, {'rank': 124, 'paragraphs': [{'text': 'This study was funded by the National Natural Science Foundation of China (82041024 to F.C., 82041026 to H.S., 81973142 to Y.W.). Sponsors had no role in design of the study, collection and analysis of data, or preparation of the manuscript.', 'id': '00019'}], 'title': 'A systematic review and meta-analysis reveals long and dispersive incubation period of COVID-19', 'sha': '3d921c03dc23ee850655ddfd9d5ebe3956d70bf8', 'doi': '10.1101/2020.06.20.20134387', 'score': 9.375896453857422}, {'rank': 125, 'paragraphs': [{'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'id': '00031'}], 'title': 'A systematic review and meta-analysis reveals long and dispersive incubation period of COVID-19', 'sha': '3d921c03dc23ee850655ddfd9d5ebe3956d70bf8', 'doi': '10.1101/2020.06.20.20134387', 'score': 9.375895500183105}, {'rank': 126, 'paragraphs': [{'text': 'We acknowledge some limitations of this study. First, the sample sizes for asymptomatic transmissions and young-age infections are small, and the results for these subgroups may be less representative.', 'id': '00034'}], 'title': 'A systematic review and meta-analysis reveals long and dispersive incubation period of COVID-19', 'sha': '3d921c03dc23ee850655ddfd9d5ebe3956d70bf8', 'doi': '10.1101/2020.06.20.20134387', 'score': 9.375894546508789}, {'rank': 127, 'paragraphs': [{'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. ', 'id': '00037'}], 'title': 'A systematic review and meta-analysis reveals long and dispersive incubation period of COVID-19', 'sha': '3d921c03dc23ee850655ddfd9d5ebe3956d70bf8', 'doi': '10.1101/2020.06.20.20134387', 'score': 9.375893592834473}, {'rank': 128, 'paragraphs': [{'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'id': '00039'}], 'title': 'A systematic review and meta-analysis reveals long and dispersive incubation period of COVID-19', 'sha': '3d921c03dc23ee850655ddfd9d5ebe3956d70bf8', 'doi': '10.1101/2020.06.20.20134387', 'score': 9.375892639160156}, {'rank': 129, 'paragraphs': [{'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'id': '00043'}], 'title': 'A systematic review and meta-analysis reveals long and dispersive incubation period of COVID-19', 'sha': '3d921c03dc23ee850655ddfd9d5ebe3956d70bf8', 'doi': '10.1101/2020.06.20.20134387', 'score': 9.37589168548584}, {'rank': 130, 'paragraphs': [{'text': 'The copyright holder for this preprint this version posted June 22, 2020. . https://doi.org/10.1101/2020.06.20.20134387 doi: medRxiv preprint', 'id': '00048'}], 'title': 'A systematic review and meta-analysis reveals long and dispersive incubation period of COVID-19', 'sha': '3d921c03dc23ee850655ddfd9d5ebe3956d70bf8', 'doi': '10.1101/2020.06.20.20134387', 'score': 9.375890731811523}, {'rank': 131, 'abstract': 'On March 11, 2020, the World Health Organization declared the coronavirus disease 2019, COVID-19, a global pandemic. In an unprecedented collective effort, massive amounts of data are now being collected worldwide to estimate the immediate and long-term impact of this pandemic on the health system and the global economy. However, the precise timeline of the disease, its transmissibility, and the effect of mitigation strategies remain incompletely understood. Here we integrate a global network model with a local epidemic SEIR model to quantify the outbreak dynamics of COVID-19 in China and the United States. For the outbreak in China, in [Formula: see text] provinces, we found a latent period of 2.56 ± 0.72 days, a contact period of 1.47 ± 0.32 days, and an infectious period of 17.82 ± 2.95 days. We postulate that the latent and infectious periods are disease-specific, whereas the contact period is behavior-specific and can vary between different provinces, states, or countries. For the early stages of the outbreak in the United States, in [Formula: see text] states, we adopted the disease-specific values from China and found a contact period of 3.38 ± 0.69 days. Our network model predicts that—without the massive political mitigation strategies that are in place today—the United States would have faced a basic reproduction number of 5.30 ± 0.95 and a nationwide peak of the outbreak on May 10, 2020 with 3 million infections. Our results demonstrate how mathematical modeling can help estimate outbreak dynamics and provide decision guidelines for successful outbreak control. We anticipate that our model will become a valuable tool to estimate the potential of vaccination and quantify the effect of relaxing political measures including total lockdown, shelter in place, and travel restrictions for low-risk subgroups of the population or for the population as a whole.', 'body_text': [{'text': 'In December 2019, a local outbreak of pneumonia of initially unknown cause was detected in Wuhan, a city of 11 million people in central China (Li et al. 2020). The cause of the disease was identified as the novel severe acute respiratory syndrome coronavirus 2, SARS-CoV-2 (Gorbalenya et al. 2020). Infection with the virus can be asymptomatic or can result in a mild to severe symptomatic disease, coronavirus disease 2019 or COVID-19. The majority of COVID-19 cases result in mild symptoms including fever, cough, shortness of breath, and respiratory distress (Hu et al. 2019). Severe complications arise when the disease progresses to viral pneumonia and multi-organ failure. The SARS-CoV-2 virus can spread quickly, mainly during close contact, but also through small droplets from coughing or sneezing (World Health Organization 2020b). After the first four cases were reported on December 29, the outbreak quickly spread from Wuhan across all provinces of mainland China, and, in the following two months, across the entire world. On March 11, 2020, the World Health Organization acknowledged the alarming levels of spread and severity, and characterized the COVID-19 situation as a pandemic (World Health Organization 2020a). As of today, April 4, 2020, COVID-19 has affected 203 countries with a total of 1,201,483 reported cases, 64,690 deaths, and 264,467 recovered cases (Coronavirus 2020).', 'cite_spans': [{'start': 154, 'end': 158, 'mention': '2020', 'ref_id': 'BIBREF15'}, {'start': 293, 'end': 297, 'mention': '2020', 'ref_id': 'BIBREF6'}, {'start': 574, 'end': 578, 'mention': '2019', 'ref_id': 'BIBREF11'}, {'start': 835, 'end': 840, 'mention': '2020b', 'ref_id': 'BIBREF22'}, {'start': 1226, 'end': 1231, 'mention': '2020a', 'ref_id': 'BIBREF21'}, {'start': 1396, 'end': 1400, 'mention': '2020', 'ref_id': 'BIBREF29'}], 'section': 'Motivation', 'ref_spans': []}, {'text': 'Figure 1 illustrates a typical timeline of COVID-19 in a single person and shows how this timeline maps onto an entire population. For this example, at day 0, a number of susceptible individuals are exposed to the virus and transition from the susceptible to the exposed state. Around at day 3, the exposed individuals become infectious. During this time, they can infect others, while not showing any symptoms themselves. The infectious period lasts for approximately 10 days. Around day 5, infectious individuals become symptomatic. This implies that they have potentially spread the disease for two days without knowing it. In the majority of \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$(1-\\\\nu _{\\\\mathrm{h}})$$\\\\end{document}(1-νh) of the population, the symptomatic period lasts for approximately 9 days. Around day 9, a severely affected population of \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$\\\\nu _{\\\\mathrm{h}}$$\\\\end{document}νh are hospitalized and their hospitalization lasts for approximately 14 days. Around day 10, \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$\\\\nu _{\\\\mathrm{c}}$$\\\\end{document}νc of the hospitalized population experiences critical conditions that last for approximately 10 days and end in \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$(1-\\\\nu _{\\\\mathrm{d}})$$\\\\end{document}(1-νd) of recovery and \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$\\\\nu _{\\\\mathrm{d}}$$\\\\end{document}νd of death. For a hospitalization fraction of \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$\\\\nu _{\\\\mathrm{h}}=0.045$$\\\\end{document}νh=0.045, a critical conditions fraction of \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$\\\\nu _{\\\\mathrm{c}}=0.25$$\\\\end{document}νc=0.25, and a death fraction of \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$\\\\nu _{\\\\mathrm{d}}=0.50$$\\\\end{document}νd=0.50, 99.44% of the population recover and 0.56% die (Heiden and Buchholz 2020).', 'cite_spans': [{'start': 3646, 'end': 3650, 'mention': '2020', 'ref_id': 'BIBREF7'}], 'section': 'Motivation', 'ref_spans': [{'start': 7, 'end': 8, 'mention': '1', 'ref_id': 'FIGREF0'}]}, {'text': 'The first mathematical models for infectious diseases date back to a smallpox model by Daniel Bernoulli (Bernoulli1760). Since the 1920s, compartment models have become the most common approach to model the epidemiology of infectious diseases (Kermack and McKendrick 1927). One of the simplest compartment models is the SEIR model that represents the timeline of a disease through four compartments, the susceptible, exposed, infectious, and recovered populations (Aron and Schwartz 1984). The temporal evolution of these compartments is governed by a set of ordinary differential equations parameterized in terms of the transition rates between them (Hethcote 2000). The transition rates \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$\\\\alpha$$\\\\end{document}α from the exposed to the infectious state and \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$\\\\gamma$$\\\\end{document}γ from the infectious to the recovered state are disease-specific parameters. In fact, they are the inverses of the latent period \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$A=1/\\\\alpha$$\\\\end{document}A=1/α, the time during which an individual is exposed but not yet infectious, and the infectious period \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$C=1/\\\\gamma$$\\\\end{document}C=1/γ, the time during which an individual can infect others. This suggests that these two parameters are relatively independent of country, region, or city. In the example of Fig. 1, the latent and infectious periods are \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$A=3$$\\\\end{document}A=3 days and \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$C=10$$\\\\end{document}C=10 days (Heiden and Buchholz 2020). The most critical feature of the model is the transition from the susceptible to the exposed state. This transition is typically assumed to scale with the susceptible population S, the infectious population I, and the contact rate \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$\\\\beta$$\\\\end{document}β, the inverse of the contact period \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$B=1/\\\\beta$$\\\\end{document}B=1/β, between them (Li and Muldowney 1984).', 'cite_spans': [{'start': 114, 'end': 118, 'mention': '1760', 'ref_id': 'BIBREF25'}, {'start': 267, 'end': 271, 'mention': '1927', 'ref_id': 'BIBREF12'}, {'start': 483, 'end': 487, 'mention': '1984', 'ref_id': 'BIBREF10'}, {'start': 661, 'end': 665, 'mention': '2000', 'ref_id': 'BIBREF8'}, {'start': 2985, 'end': 2989, 'mention': '2020', 'ref_id': 'BIBREF7'}, {'start': 3882, 'end': 3886, 'mention': '1984', 'ref_id': 'BIBREF14'}], 'section': 'Motivation', 'ref_spans': [{'start': 2323, 'end': 2324, 'mention': '1', 'ref_id': 'FIGREF0'}]}, {'text': 'The product of the contact rate and the infectious period defines the basic reproduction number, \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_0 = \\\\beta \\\\, C = C / B$$\\\\end{document}R0=βC=C/B, the number of individuals that are infected by a single one individual in an otherwise uninfected, susceptible population (Dietz 1993). The basic reproduction number is a measure of the contagiousness or transmissibility of an infectious agent and it can vary considerably between different infectious diseases (Delamater et al. 2019). Typical basic reproduction numbers are on the order of 18 for measles, 9 for chickenpox, 7 for mumps, 7 for rubella, and 5 for poliomyelitis (Anderson and May 1982). When the basic reproduction number is larger than one, \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_0 > 1.0$$\\\\end{document}R0>1.0, the infectious period C is larger than the contact period B (Li and Muldowney 1984). This implies that at onset of an epidemic outbreak, when the entire population is susceptible, an infected individual will infect more than one other individual. In agreement with Fig. 1, the infectious population first increases, then reaches a peak, and decreases toward zero (Kermack and McKendrick 1927). As more and more individuals transition from the susceptible through the exposed and infectious states into the recovered state, the susceptible population decreases. Once a large enough fraction of a population has become immune—either through recovery from the infection or through vaccination—this group provides a measure of protection for the susceptible population and the epidemic dies out (Dietz 1993). This indirect protection is called herd immunity (Fine 1993). The concept of herd immunity implies that the converged susceptible population at endemic equilibrium is always larger than zero, \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$S_{\\\\infty }>0$$\\\\end{document}S∞>0, and its value depends on the basic reproduction number \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_0$$\\\\end{document}R0. For a given basic reproduction number \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_0$$\\\\end{document}R0, herd immunity occurs at an immune fraction of \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$(1-1/R_0)$$\\\\end{document}(1-1/R0). Knowing the basic reproduction number is therefore critical to estimate the immune fraction of the population that is required to eradicate an infectious disease, for example, 94.4% for measles and 80.0% for poliomyelitis (Hethcote 2000).', 'cite_spans': [{'start': 546, 'end': 550, 'mention': '1993', 'ref_id': 'BIBREF2'}, {'start': 746, 'end': 750, 'mention': '2019', 'ref_id': 'BIBREF1'}, {'start': 912, 'end': 916, 'mention': '1982', 'ref_id': 'BIBREF0'}, {'start': 1354, 'end': 1358, 'mention': '1984', 'ref_id': 'BIBREF14'}, {'start': 1663, 'end': 1667, 'mention': '1927', 'ref_id': 'BIBREF12'}, {'start': 2074, 'end': 2078, 'mention': '1993', 'ref_id': 'BIBREF2'}, {'start': 2136, 'end': 2140, 'mention': '1993', 'ref_id': 'BIBREF4'}, {'start': 3836, 'end': 3840, 'mention': '2000', 'ref_id': 'BIBREF8'}], 'section': 'Motivation', 'ref_spans': [{'start': 1546, 'end': 1547, 'mention': '1', 'ref_id': 'FIGREF0'}]}, {'text': 'Restrictive measures like medical isolation or quarantine reduce the effective infectious period C and mitigation strategies like contact tracing, physical distancing, or travel restrictions increase the contact period B. Especially during the early stages of an outbreak, passenger air travel can play a critical role in spreading a disease (Balcan et al. 2009), since traveling individuals naturally have a disproportionally high contact rate (Pastor-Satorras et al. 2015). Border control can play a pivotal role in mitigating epidemics and prevent the spreading between cities, states, or countries (Zlojutro et al. 2019). In an attempt to mitigate the COVID-19 outbreak, many countries have implemented travel restrictions and mandatory quarantines, closed borders, and prohibited non-citizens from entry. This has stimulated an ongoing debate about how strong these restrictions should be and when it would be safe to lift them. The basic reproduction number is \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_0$$\\\\end{document}R0 provides guidelines about the required strength of political countermeasures (Hethcote 2000). However, empirically finding the basic reproduction number requires careful contact tracing and is a lot of work, especially once the number of infectious individuals has grown beyond an overseeable size (Li et al. 2020). Network modeling of travel-induced spreading can play an important role in estimating the value of \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_0$$\\\\end{document}R0 (Colizza et al. 2006) and interpreting the impact of travel restrictions and border control (Hsu 2020).', 'cite_spans': [{'start': 357, 'end': 361, 'mention': '2009', 'ref_id': 'BIBREF19'}, {'start': 469, 'end': 473, 'mention': '2015', 'ref_id': 'BIBREF20'}, {'start': 619, 'end': 623, 'mention': '2019', 'ref_id': 'BIBREF24'}, {'start': 1345, 'end': 1349, 'mention': '2000', 'ref_id': 'BIBREF8'}, {'start': 1567, 'end': 1571, 'mention': '2020', 'ref_id': 'BIBREF15'}, {'start': 1980, 'end': 1984, 'mention': '2006', 'ref_id': 'BIBREF28'}, {'start': 2061, 'end': 2065, 'mention': '2020', 'ref_id': 'BIBREF9'}], 'section': 'Motivation', 'ref_spans': []}, {'text': 'We model the epidemiology of the COVID-19 outbreak using an SEIR model with four compartments, the susceptible, exposed, infectious, and recovered populations, governed by a set of ordinary differential equations (Hethcote 2000),\\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$\\\\begin{aligned} {\\\\dot{S}}&= -\\\\, \\\\beta \\\\, S \\\\, I \\\\\\\\ {\\\\dot{E}}&= +\\\\,\\\\beta \\\\, S \\\\, I - \\\\alpha \\\\, E \\\\\\\\ {\\\\dot{I}}&= + \\\\alpha \\\\, E - \\\\gamma \\\\, I \\\\\\\\ {\\\\dot{R}}&= + \\\\gamma \\\\, I . \\\\end{aligned}$$\\\\end{document}S˙=-βSIE˙=+βSI-αEI˙=+αE-γIR˙=+γI.The transition rates between the four compartments, \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$\\\\beta$$\\\\end{document}β, \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$\\\\alpha$$\\\\end{document}α, and \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$\\\\gamma$$\\\\end{document}γ are inverses of the contact period \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$B=1/\\\\beta$$\\\\end{document}B=1/β, the latent period \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$A=1/\\\\alpha$$\\\\end{document}A=1/α, and the infectious period \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$C=1/\\\\gamma$$\\\\end{document}C=1/γ. We interpret the latent and infectious periods A and C as disease-specific, and the contact period B as behavior specific. We discretize the SEIR model in time using an implicit Euler backward scheme and adopt a Newton–Raphson method to solve for the daily increments in each compartment.', 'cite_spans': [{'start': 223, 'end': 227, 'mention': '2000', 'ref_id': 'BIBREF8'}], 'section': 'Epidemiology modeling ::: Methods', 'ref_spans': []}, {'text': 'We model the spreading of COVID-19 across a country through a network of passenger air travel, which we represent as a weighted undirected graph \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$${{\\\\mathcal {G}}}$$\\\\end{document}G with N nodes and E edges. The nodes represent the individual states, the edges the connections between them. We weight the edges by the estimated annual incoming and outgoing passenger air travel as reported by the Bureau of Transportation Statistics (Bureau of Transportation Statistics 2020). We summarize the connectivity of the graph \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$${{\\\\mathcal {G}}}$$\\\\end{document}G in terms of the adjacency matrix \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$A_{IJ}$$\\\\end{document}AIJ, the frequency of travel between two states I and J, and the degree matrix \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$D_{II} = \\\\text{ diag } \\\\, \\\\sum _{J=1,J \\\\ne I}^{N}{} A_{IJ}$$\\\\end{document}DII=diag∑J=1,J≠INAIJ, the number of incoming and outgoing connections of state I.', 'cite_spans': [{'start': 736, 'end': 740, 'mention': '2020', 'ref_id': 'BIBREF27'}], 'section': 'Network modeling ::: Methods', 'ref_spans': []}, {'text': 'The difference between the degree matrix \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$D_{IJ}$$\\\\end{document}DIJ and the adjacency matrix \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$A_{IJ}$$\\\\end{document}AIJ defines the weighted graph Laplacian \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$L_{IJ}$$\\\\end{document}LIJ,\\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$\\\\begin{aligned} L_{IJ} = D_{IJ} - A_{IJ}. \\\\end{aligned}$$\\\\end{document}LIJ=DIJ-AIJ.Figure 2 illustrates the discrete graph \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$${{\\\\mathcal {G}}}$$\\\\end{document}G of the United States with \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$N=50$$\\\\end{document}N=50 nodes and the \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$E=200$$\\\\end{document}E=200 most travelled edges. The size and color of the nodes represent the degree \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$D_{II}$$\\\\end{document}DII, the thickness of the edges represents the adjacency \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$A_{IJ}$$\\\\end{document}AIJ. For our passenger travel-weighted graph, the degree ranges from 100 million in California to less than 1 million in Delaware, Vermont, West Virginia, and Wyoming, with a mean degree of \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$${\\\\bar{D}}_{II}=16$$\\\\end{document}D¯II=16 million per node. We assume that the Laplacian \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$L_{IJ}$$\\\\end{document}LIJ, normalized to one and scaled by the travel coefficient \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$\\\\vartheta$$\\\\end{document}ϑ, characterizes the global spreading of COVID-19 and discretize our SEIR model on our weighted graph \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$${{\\\\mathcal {G}}}$$\\\\end{document}G. Specifically, we introduce the susceptible, exposed, infectious, and recovered populations \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$S_I$$\\\\end{document}SI, \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$E_I$$\\\\end{document}EI, \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$I_I$$\\\\end{document}II, and \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_I$$\\\\end{document}RI as global unknowns at the \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$I=1,\\\\ldots ,N$$\\\\end{document}I=1,…,N nodes of the graph \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$${{\\\\mathcal {G}}}$$\\\\end{document}G. This results in the spatial discretization of the set of equations with \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$4\\\\,N$$\\\\end{document}4N unknowns,\\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$\\\\begin{aligned} {\\\\dot{S}}_I &= - \\\\sum{}_{J=1}^{N} \\\\vartheta \\\\,{L}_{IJ} \\\\, {S}_J - \\\\beta \\\\, S \\\\, I \\\\\\\\ {\\\\dot{E}}_I &= - \\\\sum{}_{J=1}^{N} \\\\vartheta \\\\,{L}_{IJ} \\\\, {E}_J + \\\\beta \\\\, S \\\\, I - \\\\alpha \\\\, E \\\\\\\\ {\\\\dot{I}}_I &= - \\\\sum{}_{J=1}^{N} \\\\vartheta \\\\,{L}_{IJ} \\\\, {I}_J + \\\\alpha \\\\, E - \\\\gamma \\\\, I \\\\\\\\ {\\\\dot{R}}_I &= - \\\\sum{}_{J=1}^{N} \\\\vartheta \\\\,{L}_{IJ} \\\\, {R}_J + \\\\gamma \\\\, I .\\\\end{aligned}$$\\\\end{document}S˙I=-∑J=1NϑLIJSJ-βSIE˙I=-∑J=1NϑLIJEJ+βSI-αEI˙I=-∑J=1NϑLIJIJ+αE-γIR˙I=-∑J=1NϑLIJRJ+γI.We discretize our SEIR network model in time using an implicit Euler backward scheme and adopt a Newton Raphson method to solve for the daily increments in each compartment in each state (Fornari et al. 2019).', 'cite_spans': [{'start': 7875, 'end': 7879, 'mention': '2019', 'ref_id': 'BIBREF5'}], 'section': 'Network modeling ::: Methods', 'ref_spans': [{'start': 1347, 'end': 1348, 'mention': '2', 'ref_id': 'FIGREF8'}]}, {'text': 'Unlike many other countries, China has already seen a peak of the COVID-19 outbreak and is currently not seeing a significant number of new cases. The COVID-19 outbreak data of the Chinese provinces capture all three phases, increase, peak, and decrease in the infectious population and are currently the richest dataset available to date. This dataset describes the temporal evolution of confirmed, recovered, active, and death cases starting January 22, 2020 (Coronavirus 2020). As of April 4, there were 81,639 confirmed cases, 76,755 recovered, 1558 active, and 3326 deaths. From these data, we map out the temporal evolution of the infectious group I as the difference between the confirmed cases minus the recovered and deaths, and the recovered group R as the sum of the recovered and deaths in each Chinese province. To simulate the province-specific epidemiology of COVID19 with the SEIR model, we use these data to identify the latent period \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$A = 1/\\\\alpha$$\\\\end{document}A=1/α, the infectious period \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$C = 1/\\\\gamma$$\\\\end{document}C=1/γ, and the contact period \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$B = 1/\\\\beta$$\\\\end{document}B=1/β as a direct measure of the basic reproduction number \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_0 = B/C$$\\\\end{document}R0=B/C. As our sensitivity analysis in Fig. 3 shows, the dynamics of the SEIR model depend critically on the initial conditions, the number of susceptible \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$S_0$$\\\\end{document}S0, exposed \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$E_0$$\\\\end{document}E0, infectious \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$I_0$$\\\\end{document}I0, and recovered \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_0$$\\\\end{document}R0 individuals on the day the very first infectious case is reported, \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$I_0 \\\\ge 1$$\\\\end{document}I0≥1. Naturally, on this day, the recovered population is \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_0=0$$\\\\end{document}R0=0. Since the exposed population is asymptomatic, its initial value \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$E_0$$\\\\end{document}E0 is unknown. To quantify the initial exposed population \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$E_0$$\\\\end{document}E0, we introduce a parameter \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$\\\\rho = E_0 / I_0$$\\\\end{document}ρ=E0/I0, the initial latent population (Maier and Brockmann 2020). It defines the fraction of exposed versus infectious individuals at day 0 and is a measure of initial hidden community spreading. The fraction of the initial susceptible population, \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$S_0 = 1 - E_0 - I_0 - R_0$$\\\\end{document}S0=1-E0-I0-R0, ensures that the total population sums up to one. To map the total population of one onto the absolute number of cases for each province, we introduce the normalization parameter \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$\\\\eta = N^*/N$$\\\\end{document}η=N∗/N, the affected population. It defines the fraction of the province-specific epidemic subpopulation \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$N^*$$\\\\end{document}N∗ relative to the province population N (National Bureau of Statistics of China 2020). Altogether we identify five parameters for each province, the exposed period \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$A = 1/\\\\alpha$$\\\\end{document}A=1/α, the infectious period \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$C = 1/\\\\gamma$$\\\\end{document}C=1/γ, the contact period \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$B = 1/\\\\beta$$\\\\end{document}B=1/β or the basic reproduction number \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R=C/B$$\\\\end{document}R=C/B, the initial latent population \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$\\\\rho = E_0 / I_0$$\\\\end{document}ρ=E0/I0, and the affected population \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$\\\\eta = N^*/N$$\\\\end{document}η=N∗/N. We performed the parameter identification using the Levenberg–Marquardt method of least squares. In this identification process, we ignored data from secondary outbreaks (Coronavirus 2020).', 'cite_spans': [{'start': 474, 'end': 478, 'mention': '2020', 'ref_id': 'BIBREF29'}, {'start': 5412, 'end': 5416, 'mention': '2020', 'ref_id': 'BIBREF17'}, {'start': 6876, 'end': 6880, 'mention': '2020', 'ref_id': 'BIBREF18'}, {'start': 9097, 'end': 9101, 'mention': '2020', 'ref_id': 'BIBREF29'}], 'section': 'COVID-19 outbreak dynamics in China ::: Parameter identification ::: Methods', 'ref_spans': [{'start': 2298, 'end': 2299, 'mention': '3', 'ref_id': 'FIGREF9'}]}, {'text': 'Unlike China, the United States are at the early stage of the COVID-19 outbreak and all states are still seeing an increase in the number of new cases every day. The available dataset describes the temporal evolution of confirmed, recovered, active, and death cases starting January 21, 2020, the first day of the outbreak in the United States (Coronavirus 2020). As of April 4, there were 311,357 confirmed cases, 14,825 recovered, 288,081 active, and 8451 deaths. Similar to the Chinese data, we map out the temporal evolution of the infectious group I as the difference between the confirmed cases minus the recovered and deaths in each state of the United States. To simulate the state-specific epidemiology of COVID19 with the SEIR model, we use these data to identify the contact time \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$B=1/\\\\beta$$\\\\end{document}B=1/β, while fixing the disease-specific latent and infections periods \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$A = 1/\\\\alpha$$\\\\end{document}A=1/α and \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$C = 1/\\\\gamma$$\\\\end{document}C=1/γ at their mean values of the SEIR dynamics fit for the Chinese provinces, and indirectly fitting the basic reproduction number \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_0 = C/B$$\\\\end{document}R0=C/B. For each state, we set the first day of reported infections \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$I_0 \\\\ge 1$$\\\\end{document}I0≥1 to day zero, at which the recovered population is \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_0=0$$\\\\end{document}R0=0, the unknown exposed population is \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$E_0 = \\\\rho \\\\, I_0$$\\\\end{document}E0=ρI0 (Maier and Brockmann 2020), and the susceptible population is \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$S_0 = N - E_0 - I_0 - R_0$$\\\\end{document}S0=N-E0-I0-R0, where N is the state-specific population (World Population Review 2020). We identify two parameters for each state, the contact period \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$B = 1/\\\\beta$$\\\\end{document}B=1/β and the initial latent population \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$\\\\rho = E_0 / I_0$$\\\\end{document}ρ=E0/I0, while we use the exposed period \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$A = 1/\\\\alpha$$\\\\end{document}A=1/α and the infectious period \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$C = 1/\\\\gamma$$\\\\end{document}C=1/γ from the parameter identification for the Chinese provinces and back-calculate the basic reproduction number \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R=C/B$$\\\\end{document}R=C/B. We perform the parameter identification using the Levenberg–Marquardt method of least squares.', 'cite_spans': [{'start': 357, 'end': 361, 'mention': '2020', 'ref_id': 'BIBREF29'}, {'start': 3263, 'end': 3267, 'mention': '2020', 'ref_id': 'BIBREF17'}, {'start': 3695, 'end': 3699, 'mention': '2020', 'ref_id': 'BIBREF23'}], 'section': 'COVID-19 outbreak dynamics in the\\xa0United States ::: Parameter identification ::: Methods', 'ref_spans': []}, {'text': 'The dynamics of the SEIR model are determined by three parameters, the latent period \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$A=1/\\\\alpha$$\\\\end{document}A=1/α, and the infectious period \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$C=1/\\\\gamma$$\\\\end{document}C=1/γ, and the contact period \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$B = 1/\\\\beta$$\\\\end{document}B=1/β, or, alternatively, the basic reproduction number \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_0 = C/B$$\\\\end{document}R0=C/B. Before identifying these parameters for the outbreaks in China and in the United States, we will illustrate their effects by systematically varying each parameter while keeping the other values fixed. Specifically, unless stated otherwise, we choose a latent period of \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$A = 5$$\\\\end{document}A=5 days, an infectious period of \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$C = 20$$\\\\end{document}C=20 days, a basic reproduction number of \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_0 = C/B = 2.0$$\\\\end{document}R0=C/B=2.0, and an initial exposed population \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$E_0 = 0.010$$\\\\end{document}E0=0.010.', 'cite_spans': [], 'section': 'Outbreak dynamics ::: Results', 'ref_spans': []}, {'text': 'Figure 3 illustrates the sensitivity of the SEIR model with respect to the size of the initial exposed population \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$E_0$$\\\\end{document}E0. Decreasing the initial exposed population from \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$E_0 = 10^{-2}, 10^{-3}, 10^{-4}, 10^{-5}, 10^{-6}, 10^{-7}, 10^{-8}$$\\\\end{document}E0=10-2,10-3,10-4,10-5,10-6,10-7,10-8 delays the onset of the outbreak while the dynamics of the susceptible, exposed, infectious, and recovered populations remain the same. For all seven cases, the susceptible and recovered populations converge to the same endemic equilibrium with \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$S_\\\\infty = 0.202$$\\\\end{document}S∞=0.202 and \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_\\\\infty = 0.798$$\\\\end{document}R∞=0.798. The infectious population increases gradually, reaches its maximum at \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$I_{\\\\mathrm{max}} = 0.121$$\\\\end{document}Imax=0.121, and then decreases. For the largest initial exposed population of \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$E_0 = 0.01$$\\\\end{document}E0=0.01 this maximum occurs after 125 days. Decreasing the initial exposed population by a factor 10 delays the maximum by 65 days. This highlights the exponential nature of the model, which causes a constant delay for a logarithmic decrease in the exponential population, while the overall outbreak dynamics remain the same. In view of the COVID-19 outbreak, this supports the general notion that even a single individual can cause an outbreak. If multiple individuals trigger the outbreak in a province, state, or country, the overall outbreak dynamics will remain the same, but the peak of the outbreak will happen earlier.', 'cite_spans': [], 'section': 'Outbreak dynamics ::: Results', 'ref_spans': [{'start': 7, 'end': 8, 'mention': '3', 'ref_id': 'FIGREF9'}]}, {'text': 'Figure 4 illustrates the sensitivity of the SEIR model with respect to the latent period A. Increasing the latent period from \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$A = 0, 5 ,10, 15, 20, 25$$\\\\end{document}A=0,5,10,15,20,25 days increases the exposed population and decreases the infectious population. The susceptible and recovered populations converge to the same endemic equilibrium at \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$S_\\\\infty = 0.202$$\\\\end{document}S∞=0.202 and \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_\\\\infty = 0.798$$\\\\end{document}R∞=0.798. Convergence is slower for increased latent periods A. The steepest susceptible, infectious, and recovery curves correspond to the special case of the SIR model without a separate exposed population E, for which \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$A = 0$$\\\\end{document}A=0 days. This model does not have a separate exposed population. It reaches its peak infectious population of \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$I_{\\\\mathrm{max}} = 0.157$$\\\\end{document}Imax=0.157 after 86 days. In view of the COVID-19 outbreak, this implies that knowledge of the latent period is important to correctly estimate the timing and peak of the infectious population, which ultimately determines the absolute number of hospital beds and ventilator units required to insure appropriate medical care.', 'cite_spans': [], 'section': 'Outbreak dynamics ::: Results', 'ref_spans': [{'start': 7, 'end': 8, 'mention': '4', 'ref_id': 'FIGREF10'}]}, {'text': 'Figure 5 illustrates the sensitivity of the SEIR model with respect to the infectious period C. Increasing the infectious period at a constant basic reproduction number flattens the exposed population and increases the infectious population. The susceptible and recovered populations converge to the same endemic equilibrium at \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$S_\\\\infty = 0.202$$\\\\end{document}S∞=0.202 and \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_\\\\infty = 0.798$$\\\\end{document}R∞=0.798, however, slower. The flattest susceptible, infectious, and recovery curves correspond to the longest infectious period of \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$C = 30$$\\\\end{document}C=30 days and a contact period of \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$B=15$$\\\\end{document}B=15 days with the maximum infectious population of \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$I_{\\\\mathrm{max}} = 0.135$$\\\\end{document}Imax=0.135 after 169 days. In view of the COVID-19 outbreak, knowing the infectious time is important to correctly estimate the timing and peak of the infectious population, and with it the number of required hospital beds and ventilator units.', 'cite_spans': [], 'section': 'Outbreak dynamics ::: Results', 'ref_spans': [{'start': 7, 'end': 8, 'mention': '5', 'ref_id': 'FIGREF11'}]}, {'text': 'Figure 6 illustrates the sensitivity of the SEIR model with respect to the basic reproduction number \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_0$$\\\\end{document}R0. Decreasing the basic reproduction number decreases the exposed and infectious populations. The susceptible and recovered populations converge to larger and smaller endemic equilibrium values, and converges is slower. The steepest susceptible, exposed, infectious, and recovery curves correspond to the largest basic reproduction number of \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_0=10.0$$\\\\end{document}R0=10.0 with the maximum infectious population of \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$I_{\\\\mathrm{max}} = 0.488$$\\\\end{document}Imax=0.488 after 35 days and converge to an endemic equilibrium at \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$S_\\\\infty = 0.0001$$\\\\end{document}S∞=0.0001 and \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_\\\\infty = 0.9999$$\\\\end{document}R∞=0.9999. In view of the COVID-19 outbreak, the basic reproduction number is the parameter that we can influence by political countermeasures. Reducing the basic reproduction number beyond its natural value by decreasing the contact time B through physical distancing or total lockdown allows us to reduce the maximum infectious population and delay the outbreak, a measure that is commonly referred to in the public media as “flatting the curve”.', 'cite_spans': [], 'section': 'Outbreak dynamics ::: Results', 'ref_spans': [{'start': 7, 'end': 8, 'mention': '6', 'ref_id': 'FIGREF12'}]}, {'text': 'The sensitivity study suggests that an epidemic outbreak is most sensitive to the basic reproduction number \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_0$$\\\\end{document}R0. While the latent period A and the infectious period C are disease-specific, community mitigation and political action can modulate the basic reproduction number \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_0$$\\\\end{document}R0 through a variety of measures including active contact tracing, isolation of infectious individuals, quarantine of close contacts, travel restrictions, physical distancing, or total lockdown.', 'cite_spans': [], 'section': 'Outbreak control ::: Results', 'ref_spans': []}, {'text': 'Figure 7 illustrates the effect of the basic reproduction number \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_0$$\\\\end{document}R0 on the maximum exposed and infectious populations \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$E_{\\\\mathrm{max}}$$\\\\end{document}Emax and \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$I_{\\\\mathrm{max}}$$\\\\end{document}Imax and on the converged susceptible and recovered populations \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$S_\\\\infty$$\\\\end{document}S∞ and \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_\\\\infty$$\\\\end{document}R∞ at endemic equilibrium. Increasing the basic reproduction number beyond one increases the maximum exposed and infectious populations. The converged susceptible and recovered populations decrease towards zero and increase towards one. For the chosen latent and infectious periods of \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$A = 5$$\\\\end{document}A=5 days and \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$C = 20$$\\\\end{document}C=20 days, the time to reach the maximum infectious population reaches its maximum of 213 days at a basic reproduction number \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_0=1.22$$\\\\end{document}R0=1.22 and decreases for increasing basic reproduction numbers. In view of the COVID-19 outbreak, Fig. 7 suggests strategies to modulate the timeline of the epidemic by reducing the basic reproduction number \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_0$$\\\\end{document}R0. For example, if we have access to a certain number of intensive care unit beds and ventilators, and we know rates of the infectious population that have to be hospitalized and require intensive care, we need to limit the maximum size of the population that becomes infectious. To limit the infectious fraction to 20% of the total population, i.e., \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$I_{\\\\mathrm{max}}=0.200$$\\\\end{document}Imax=0.200, we would have to reduce the basic reproduction number to \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_0=2.69$$\\\\end{document}R0=2.69. The gray line indicates that this maximum would occur after 0.25 years or 93 days.', 'cite_spans': [], 'section': 'Outbreak control ::: Results', 'ref_spans': [{'start': 7, 'end': 8, 'mention': '7', 'ref_id': 'FIGREF13'}, {'start': 3076, 'end': 3077, 'mention': '7', 'ref_id': 'FIGREF13'}]}, {'text': 'Figure 8 illustrates the effect of constraining the outbreak by increasing the basic reproduction number R(t) such that the infectious population always remains below a tolerated infectious population, \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$I<I_{\\\\mathrm{tol}}$$\\\\end{document}I<Itol. Decreasing the tolerated infectious population, \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$I_{\\\\mathrm{tol}} = 0.15, 0.10, 0.08, 0.06, 0.05, 0.04, 0.03, 0.02 0.02$$\\\\end{document}Itol=0.15,0.10,0.08,0.06,0.05,0.04,0.03,0.020.02, increases the required level of containment and decreases the relative basic reproduction number, \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_0(t)/R_0 = 1.000, 0.742, 0.661, 0.603, 0.580, 0.541, 0.535, 0.524$$\\\\end{document}R0(t)/R0=1.000,0.742,0.661,0.603,0.580,0.541,0.535,0.524. This has the desired effect of decreasing the exposed and infectious populations. The susceptible population converges to progressively larger endemic equilibrium values \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$S_\\\\infty = 0.202, 0.225, 0.248, 0.274, 0.290, 0.309, 0.331, 0.358$$\\\\end{document}S∞=0.202,0.225,0.248,0.274,0.290,0.309,0.331,0.358. The recovered population converges to progressively smaller endemic equilibrium values \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_\\\\infty = 0.798, 0.775, 0.752, 0.726, 0.710, 0.691, 0.669, 0.642$$\\\\end{document}R∞=0.798,0.775,0.752,0.726,0.710,0.691,0.669,0.642. Convergence is slower under constrained outbreak. The lowest exposed and infectious curves and the flattest susceptible and recovery curves correspond to the most constrained infectious population of \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$I_{\\\\mathrm{tol}} = 0.02$$\\\\end{document}Itol=0.02 with a required level of containment of \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_0(t)/R_0 = 0.524$$\\\\end{document}R0(t)/R0=0.524. The highest exposed and infectious curves and the steepest susceptible and recovery curves correspond to an unconstrained infectious population \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$I_{\\\\mathrm{tol}} = 0.150 >I_{\\\\mathrm{max}}=0.121$$\\\\end{document}Itol=0.150>Imax=0.121 with peak infection after 125 days. In view of the COVID-19 outbreak, the gray line tells us how drastic political countermeasures need to be. A required level of containment of \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_0(t)/R_0 = 0.524$$\\\\end{document}R0(t)/R0=0.524 implies that we need to reduce the number of infections of a single individual by about one half. However, reducing the maximum infectious population comes at a socioeconomic price: The graphs teach us that it is possible to reach an endemic equilibrium at a smaller total number of individuals that have had the disease; yet, this endemic equilibrium would occur much later in time, for this example, after two or three years.', 'cite_spans': [], 'section': 'Outbreak control ::: Results', 'ref_spans': [{'start': 7, 'end': 8, 'mention': '8', 'ref_id': 'FIGREF14'}]}, {'text': 'Figure 9 summarizes the dynamics of the COVID-19 outbreak in 30 Chinese provinces. The dots indicate the reported infectious and recovered populations, the lines highlight the simulated susceptible, exposed, infectious, and recovered populations. The simulations are based on a province-specific parameter identification of the latent period A, the contact period B, the infectious period C, and from both, the basic reproduction number \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_0 = C/B$$\\\\end{document}R0=C/B, the fraction of the initial latent population \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$\\\\rho = E_0/I_0$$\\\\end{document}ρ=E0/I0, and the fraction of the affected population \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$\\\\eta =N^*/N$$\\\\end{document}η=N∗/N for each province. These five province-specific values are reported in each graph. Notably, the province of Hubei, where the outbreak started, has seen the most significant impact with more than 60,000 cases. Naturally, in Hubei, where the first cases were reported, the fraction of the initial latent population \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$\\\\rho$$\\\\end{document}ρ is zero. Small values of \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$\\\\rho$$\\\\end{document}ρ indicate a close monitoring of the COVID-19 outbreak, with very few undetected cases at the reporting of the first infectious case. The largest value of \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$\\\\rho =26.4$$\\\\end{document}ρ=26.4 suggests that, at the onset of the outbreak, a relatively large number of cases in the province of Shandong was undetected. The fraction of the affected population \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$\\\\eta =N^*/N$$\\\\end{document}η=N∗/N is a province-specific measure for the containment of the outbreak. Naturally, this number is largest in the province of Hubei, with \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$\\\\eta =1.3 \\\\cdot 10^{-3}$$\\\\end{document}η=1.3·10-3, and, because of strict containment, much smaller in all other provinces.', 'cite_spans': [], 'section': 'COVID-19 outbreak dynamics in China ::: Results', 'ref_spans': [{'start': 7, 'end': 8, 'mention': '9', 'ref_id': 'FIGREF15'}]}, {'text': 'Table 1 summarizes the parameters for the COVID-19 outbreak in China. Averaged over all Chinese provinces, we found a latent period of \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$A = 2.56 \\\\pm 0.72$$\\\\end{document}A=2.56±0.72 days, a contact period of \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$B = 1.47 \\\\pm 0.32$$\\\\end{document}B=1.47±0.32 days, an infectious period of \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$C = 17.82 \\\\pm 2.95$$\\\\end{document}C=17.82±2.95 days, a basic reproduction number of \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_0 = C/B = 12.58 \\\\pm 3.17$$\\\\end{document}R0=C/B=12.58±3.17, a fraction of the initial latent population of \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$\\\\rho =E_0/I_0=3.19 \\\\pm 5.44$$\\\\end{document}ρ=E0/I0=3.19±5.44, and fraction of the affected population of \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$\\\\eta = N^*/N$$\\\\end{document}η=N∗/N = 5.19\\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$\\\\cdot 10^{-5} \\\\pm$$\\\\end{document}·10-5± 2.23\\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$\\\\cdot 10^{-4}$$\\\\end{document}·10-4.', 'cite_spans': [], 'section': 'COVID-19 outbreak dynamics in China ::: Results', 'ref_spans': [{'start': 6, 'end': 7, 'mention': '1', 'ref_id': 'TABREF0'}]}, {'text': 'Figure 10 shows the dynamics of the early stages of the COVID-19 outbreak in the 50 states of the United States, the District of Columbia, and the territories of Guam, Puerto Rico, and the Virgin Islands. The dots indicate the reported cases and death, the lines highlight the simulated susceptible, exposed, infectious, and recovered populations. The simulations are based on a state-specific parameter identification of the contact period B that defines the basic reproduction number \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_0 = C/B$$\\\\end{document}R0=C/B and of the fraction of the initial latent population \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$\\\\rho = E_0/I_0$$\\\\end{document}ρ=E0/I0 at a given outbreak delay \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$${\\\\rm{d}}_0$$\\\\end{document}d0 for each state. These three state-specific values are reported in each graph. Since the outbreak is currently still in its early stages, we do not attempt to identify the latent and infectious periods, but rather adopt the mean latent and infectious periods \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$A = 2.56$$\\\\end{document}A=2.56 and \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$C = 17.82$$\\\\end{document}C=17.82 from the Chinese outbreak in Table 1. Notably, the state of New York is currently seeing the most significant impact with more than 100,000 cases. Naturally, in Washington, Illinois, California, and Arizona where the first cases were reported, the fraction of the initial latent population \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$\\\\rho$$\\\\end{document}ρ is small. Largest \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$\\\\rho$$\\\\end{document}ρ values occur in New York, New Jersey, Michigan, and Louisiana. The largest basic reproduction numbers \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_0$$\\\\end{document}R0 are identified in Idaho, Puerto Rico, Pennsylvania, and Indiana.', 'cite_spans': [], 'section': 'COVID-19 outbreak dynamics in the United States ::: Results', 'ref_spans': [{'start': 7, 'end': 9, 'mention': '10', 'ref_id': 'FIGREF1'}, {'start': 2370, 'end': 2371, 'mention': '1', 'ref_id': 'TABREF0'}]}, {'text': 'Table 2 summarizes the parameters for the early stages of the COVID-19 outbreak in the United States. Averaged over all states, we found a contact period of \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$B = 3.38 \\\\pm 0.69$$\\\\end{document}B=3.38±0.69 days resulting in a basic reproduction number of \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_0 = C/B = 5.30 \\\\pm 0.95$$\\\\end{document}R0=C/B=5.30±0.95, a fraction of the initial latent population of \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$\\\\rho =E_0/I_0=43.75 \\\\pm 126.34$$\\\\end{document}ρ=E0/I0=43.75±126.34 and an outbreak delay of \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$${\\\\rm{d}}_0 = 41.28 \\\\pm 13.78$$\\\\end{document}d0=41.28±13.78 days.', 'cite_spans': [], 'section': 'COVID-19 outbreak dynamics in the United States ::: Results', 'ref_spans': [{'start': 6, 'end': 7, 'mention': '2', 'ref_id': 'TABREF1'}]}, {'text': 'Figure 11 illustrates the exposed, infectious, and recovered fractions of the affected population for each state. Using the parameter values from Table 2, these curves predict the later stages of the outbreak based on the early stages of the outbreak in Fig. 10 under the assumption that no additional countermeasures are implemented. The simulation uses latent, contact, and infectious periods of \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$A = 2.56$$\\\\end{document}A=2.56 days, \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$B = 3.38 \\\\pm 0.69$$\\\\end{document}B=3.38±0.69 days, and \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$C = 17.82$$\\\\end{document}C=17.82 days from Table 1 and a fraction of the initial latent population of \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$\\\\rho = E_0/I_0= 43.75$$\\\\end{document}ρ=E0/I0=43.75 from Table 2. The orange curve suggests, that the individual states will see a peak of the infectious population at a mean of 39 days after the first infectious case has been reported. The 95% confidence interval suggests that this peak will occur between 4 and 6 weeks after the first reported case provided no additional countermeasures are implemented.', 'cite_spans': [], 'section': 'COVID-19 outbreak dynamics in the United States ::: Results', 'ref_spans': [{'start': 7, 'end': 9, 'mention': '11', 'ref_id': 'FIGREF2'}, {'start': 259, 'end': 261, 'mention': '10', 'ref_id': 'FIGREF1'}, {'start': 152, 'end': 153, 'mention': '2', 'ref_id': 'TABREF1'}, {'start': 1346, 'end': 1347, 'mention': '1', 'ref_id': 'TABREF0'}, {'start': 1730, 'end': 1731, 'mention': '2', 'ref_id': 'TABREF1'}]}, {'text': 'Figure 12 illustrates the outbreak delay \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$${\\\\rm{d}}_0$$\\\\end{document}d0 across the United States. The first reported case was in the state of Washington on January 21, 2020, followed by cases in Illinois with a delay of \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$${\\\\rm{d}}_0=3$$\\\\end{document}d0=3, California with \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$${\\\\rm{d}}_0=4$$\\\\end{document}d0=4, and Arizona with \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$${\\\\rm{d}}_0=5$$\\\\end{document}d0=5, shown in blue. The final states to see an outbreak were Alabama, Idaho, Montana with \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$${\\\\rm{d}}_0=52$$\\\\end{document}d0=52 and West Virginia with \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$${\\\\rm{d}}_0=56$$\\\\end{document}d0=56, shown in red. This illustrates that there was a significant time delay in the outbreak with many of the earlier affected states located on the west coast.', 'cite_spans': [], 'section': 'COVID-19 outbreak dynamics in the United States ::: Results', 'ref_spans': [{'start': 7, 'end': 9, 'mention': '12', 'ref_id': 'FIGREF3'}]}, {'text': 'Figure 13 illustrates the undetected population at the onset of the outbreak across the United States. The \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$\\\\rho = E_0/I_0$$\\\\end{document}ρ=E0/I0 value is small in the first states where the outbreak was reported, Washington, Illinois, California, and Arizona, suggesting that the reported cases were truly the first cases in those states. In states where the first cases occurred later, the \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$\\\\rho$$\\\\end{document}ρ value increases. Notably, Louisiana, Michigan, New Jersey, and New York have the highest \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$\\\\rho$$\\\\end{document}ρ values of 122.8, 136.1, 197.1, and 1000 suggesting that both had an exceptionally high number of exposed individuals or individuals that were infected but unreported.', 'cite_spans': [], 'section': 'COVID-19 outbreak dynamics in the United States ::: Results', 'ref_spans': [{'start': 7, 'end': 9, 'mention': '13', 'ref_id': 'FIGREF4'}]}, {'text': 'Figure 14 illustrates the basic reproduction number for the early stages of the outbreak across the United States. The basic reproduction number \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_0 = C / B$$\\\\end{document}R0=C/B, the number of individuals infected by a single infectious individual, varies from minimum values of 2.5 and 3.6 in Nebraska and Arizona to maximum values of 7.2 and 7.9 in Puerto Rico and Idaho.', 'cite_spans': [], 'section': 'COVID-19 outbreak dynamics in the United States ::: Results', 'ref_spans': [{'start': 7, 'end': 9, 'mention': '14', 'ref_id': 'FIGREF5'}]}, {'text': 'Figure 15 shows the nation-wide exposed, infectious, and recovered cases for the United States. The circles highlight the reported cases, the lines the predictions of the SEIR network model using data from the early stages of the outbreak with parameters from Tables 1 and 2 and a travel coefficient of \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$\\\\vartheta =0.43$$\\\\end{document}ϑ=0.43. The graphs start on \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$${\\\\rm{d}}_0$$\\\\end{document}d0, the day at which the last state reported its first case \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$${\\\\rm{d}}_0 = \\\\hbox {March 17, 2020}$$\\\\end{document}d0=March 17, 2020. Compared to the outbreak characteristics for the individual states in Fig. 11 with a peak of the infectious population at 39 days after the first infectious case has been reported, the nation-wide outbreak peaks 54 days after the last state has seen an outbreak, on May 10, 2020. This difference is a manifestation of both the state-specific outbreak delay \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$${\\\\rm{d}}_0$$\\\\end{document}d0 and the travel of individuals between the different states represented through the network model.', 'cite_spans': [], 'section': 'COVID-19 outbreak dynamics in the United States ::: Results', 'ref_spans': [{'start': 7, 'end': 9, 'mention': '15', 'ref_id': 'FIGREF6'}, {'start': 1400, 'end': 1402, 'mention': '11', 'ref_id': 'FIGREF2'}, {'start': 267, 'end': 268, 'mention': '1', 'ref_id': 'TABREF0'}, {'start': 273, 'end': 274, 'mention': '2', 'ref_id': 'TABREF1'}]}, {'text': 'Figure 16 illustrates the spatiotemporal evolution of the infectious population across the United States as predicted by the SEIR network model. The simulation uses data from the early stages of the outbreak in Fig. 10 summarized in Table 2. As such, the simulation is based on data from the early stages of the outbreak and assumes that no additional countermeasures have been implemented. Days 10 and 20 illustrate the slow growth of the infectious population during the early stages of the outbreak. The state of New York sees the outbreak first, followed by New Jersey and Louisiana. Days 30 and 40 illustrate how the outbreak spreads across the country. With no additional countermeasures, the SEIR network model predicts a nation-wide peak of the outbreak on day 54, on May 10, 2020. Day 50 illustrates that the earlier affected states, New York, New Jersey, and Louisiana already see a decrease in the infected population. Nebraska, West Virginia, and Wisconsin are still far from reaching the peak. Compared to Figs. 12, 13 and 14, these maps account for both, the outbreak delay and the travel of individuals between the different states represented through the network model. This model would allow us to probe the effect of travel restrictions to and from a specific state by locally reducing its travel coefficients or by globally reducing the nation-wide transport coefficient across the United States.', 'cite_spans': [], 'section': 'COVID-19 outbreak dynamics in the United States ::: Results', 'ref_spans': [{'start': 7, 'end': 9, 'mention': '16', 'ref_id': 'FIGREF7'}, {'start': 216, 'end': 218, 'mention': '10', 'ref_id': 'FIGREF1'}, {'start': 1025, 'end': 1027, 'mention': '12', 'ref_id': 'FIGREF3'}, {'start': 1029, 'end': 1031, 'mention': '13', 'ref_id': 'FIGREF4'}, {'start': 1036, 'end': 1038, 'mention': '14', 'ref_id': 'FIGREF5'}, {'start': 239, 'end': 240, 'mention': '2', 'ref_id': 'TABREF1'}]}, {'text': 'Our sensitivity analysis in Figs. 4 and 5 shows the impact of the latent and infectious periods A and C. Both affect the peak of the infectious population both in time and in magnitude. The robust data for the infectious and recovered populations of all 30 Chinese provinces in Fig. 9 suggest that the latent period lasts for 2.5 days, followed by the infectious period of 17.8 days. A study of 391 confirmed COVID-19 cases with 1268 close contacts in Shenzhen found a median incubation period of 4.8 days until the onset of symptoms, a mean time to isolation after the onset of symptoms of 2.7 days or 4.6 days with or without active contact tracing, and a median time to recovery of 20.8 days after the onset of symptoms (Bi et al. 2020). These values agree with the reported incubation period of 5.1 days found in 181 confirmed COVID-19 cases outside Wuhan (Lauer et al. 2020) and 5.2 days for the first 425 cases in Wuhan (Li et al. 2020). The total duration from exposure to recovery, \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$(A+C)$$\\\\end{document}(A+C) of our SEIR model, is 20.3 days, 5.3 days shorted than the reported value of 25.6 for the 391 Shenzhen cases (Bi et al. 2020). In our model, the reported 4.8 to 5.2 day incubation periods maps onto the latent period A of 2.5 days plus 2.3 to 2.7 days within the infectious period C during which the individuals are infectious but still asymptomatic. This period is critical since individuals can spread the disease without knowing it. The contact tracing study postulates that the infectious period C begins on day 4.8 with the onset of symptoms, 2.3 days later than in our model, and ends on day 7.3 or 9.4 with or without active contract tracing with the beginning of isolation, 13.0 or 10.9 days earlier than in our model. This implies that the infectious period C of our SEIR model is 6.6 and 3.9 times larger than the infectious period of the traced and untraced early isolated population in Shenzhen (Bi et al. 2020). This comparison suggests that it is critical to understand how the infectious period is reported, either as a disease-specific parameter or as a medically modulated exposure time.', 'cite_spans': [{'start': 734, 'end': 738, 'mention': '2020', 'ref_id': 'BIBREF26'}, {'start': 874, 'end': 878, 'mention': '2020', 'ref_id': 'BIBREF13'}, {'start': 937, 'end': 941, 'mention': '2020', 'ref_id': 'BIBREF15'}, {'start': 1406, 'end': 1410, 'mention': '2020', 'ref_id': 'BIBREF26'}, {'start': 2203, 'end': 2207, 'mention': '2020', 'ref_id': 'BIBREF26'}], 'section': 'The latent and infectious periods A and C characterize the timeline of the disease ::: Discussion', 'ref_spans': [{'start': 34, 'end': 35, 'mention': '4', 'ref_id': 'FIGREF10'}, {'start': 40, 'end': 41, 'mention': '5', 'ref_id': 'FIGREF11'}, {'start': 283, 'end': 284, 'mention': '9', 'ref_id': 'FIGREF15'}]}, {'text': 'Our sensitivity analysis in Figs. 6, 7 and 8 shows the impact of the contact period B or, more intuitively, the basic reproduction number \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_0$$\\\\end{document}R0. The basic reproduction number significantly affects the peak of the infectious population both in time and magnitude. The early outbreak data for the infectious populations of all 50 states in Fig. 10 suggest that the contact period is for 3.4 days, resulting in a basic reproduction number of 5.3. For the first 425 cases in Wuhan, the basic reproduction number was estimated to 2.2 (Li et al. 2020) and for the 391 cases in Shenzhen, it was 2.6 (Bi et al. 2020). A review of the reported basic reproduction numbers for COVID-19 found ranges from 1.40 to 6.49 with a mean of 3.28, values that are larger than those reported for the SARS coronavirus (Liu et al. 2020). Huge variations of \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_0$$\\\\end{document}R0 values are not uncommon (Dietz 1993); even for simple diseases like the measles, reported \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_0$$\\\\end{document}R0 values vary between 3.7 and 203.3 (Delamater et al. 2019). Community mitigation and political action can modulate the basic reproduction number \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_0$$\\\\end{document}R0 by a variety of measures including active contact tracing, isolation of infectious individuals, quarantine of close contacts, travel restrictions, physical distancing, or total lockdown (Fang et al. 2020). Importantly, many of the reported values already include the effect of isolation (Li et al. 2020) and active contact tracing and quarantine (Bi et al. 2020). If we correct our identified basic reproduction number for China in Fig. 9 and Table 1 by reducing our identified infectious period of 17.8 days to the time prior to isolation using the correction factors of 6.6 and 3.9 with and without contact tracing, our \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_0$$\\\\end{document}R0 values for China would be 1.91 and 3.23 and fall well within the reported range (Liu et al. 2020). Our \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_0$$\\\\end{document}R0 value for the United States of 5.30 agrees well with the range of values reported for mathematical model ranging from 1.50 to 6.49 with a mean of 4.20 (Liu et al. 2020). Understanding the natural value of \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_0$$\\\\end{document}R0—without any mitigation strategy—is critical to predict the endemic equilibrium, interpret herd immunity, and the estimate the fraction of the population that requires vaccination (Hethcote 2000).', 'cite_spans': [{'start': 825, 'end': 829, 'mention': '2020', 'ref_id': 'BIBREF15'}, {'start': 888, 'end': 892, 'mention': '2020', 'ref_id': 'BIBREF26'}, {'start': 1092, 'end': 1096, 'mention': '2020', 'ref_id': 'BIBREF16'}, {'start': 1440, 'end': 1444, 'mention': '1993', 'ref_id': 'BIBREF2'}, {'start': 1842, 'end': 1846, 'mention': '2019', 'ref_id': 'BIBREF1'}, {'start': 2424, 'end': 2428, 'mention': '2020', 'ref_id': 'BIBREF3'}, {'start': 2523, 'end': 2527, 'mention': '2020', 'ref_id': 'BIBREF15'}, {'start': 2582, 'end': 2586, 'mention': '2020', 'ref_id': 'BIBREF26'}, {'start': 3230, 'end': 3234, 'mention': '2020', 'ref_id': 'BIBREF16'}, {'start': 3695, 'end': 3699, 'mention': '2020', 'ref_id': 'BIBREF16'}, {'start': 4217, 'end': 4221, 'mention': '2000', 'ref_id': 'BIBREF8'}], 'section': 'The contact period B and basic reproduction number \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_0$$\\\\end{document}R0 characterize social and political behavior ::: Discussion', 'ref_spans': [{'start': 34, 'end': 35, 'mention': '6', 'ref_id': 'FIGREF12'}, {'start': 37, 'end': 38, 'mention': '7', 'ref_id': 'FIGREF13'}, {'start': 43, 'end': 44, 'mention': '8', 'ref_id': 'FIGREF14'}, {'start': 628, 'end': 630, 'mention': '10', 'ref_id': 'FIGREF1'}, {'start': 2662, 'end': 2663, 'mention': '9', 'ref_id': 'FIGREF15'}, {'start': 2674, 'end': 2675, 'mention': '1', 'ref_id': 'TABREF0'}]}, {'text': 'Current mitigation strategies have the goal to “flatten the curve”, which translates into reducing the number of new infections. As we can see in Figs. 6, 7 and 8, we can achieve this goal by reducing the basic reproduction number \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_0=C/B$$\\\\end{document}R0=C/B, which is a direct signature of effective containment measures and drastic behavioral changes that affect a substantial fraction of the susceptible population (Fang et al. 2020). By isolating infectious individuals, active contact tracing, and quarantining close contacts, we can reduce the effective infectious period C; and by implementing travel restrictions, mandating physical distancing, or enforcing total lockdown, we can increase the contact period B (Maier and Brockmann 2020). Figure 9 demonstrates that combinations of these measures have successfully flattened the curves in the 30 provinces of China (Li et al. 2020). But the million-dollar questions remains: What’s next? In the very near future, our model has the potential to predict the timeline of the outbreak, specifically, the timing and peak of the infectious population in individual states and countries. This will help us optimize planning and distribute medical resources where needed (Heiden and Buchholz 2020). In the short term, we could enhance our model to study the effect of different subgroups of the population (Bi et al. 2020). This could provide scientific guidelines to gradually relax political measures, for example by releasing different subgroups of the population before others. In the long term, we will need accurate values of the basic reproduction number to estimate the effect of vaccination. This will be critical to design rigorous vaccination programs and prioritize which subgroups of the population to vaccinate first (Hethcote 2000). Naturally, as more data become available, we can train our models more reliably and make more accurate predictions.', 'cite_spans': [{'start': 702, 'end': 706, 'mention': '2020', 'ref_id': 'BIBREF3'}, {'start': 1011, 'end': 1015, 'mention': '2020', 'ref_id': 'BIBREF17'}, {'start': 1155, 'end': 1159, 'mention': '2020', 'ref_id': 'BIBREF15'}, {'start': 1513, 'end': 1517, 'mention': '2020', 'ref_id': 'BIBREF7'}, {'start': 1638, 'end': 1642, 'mention': '2020', 'ref_id': 'BIBREF26'}, {'start': 2062, 'end': 2066, 'mention': '2000', 'ref_id': 'BIBREF8'}], 'section': 'What’s next? ::: Discussion', 'ref_spans': [{'start': 152, 'end': 153, 'mention': '6', 'ref_id': 'FIGREF12'}, {'start': 155, 'end': 156, 'mention': '7', 'ref_id': 'FIGREF13'}, {'start': 161, 'end': 162, 'mention': '8', 'ref_id': 'FIGREF14'}, {'start': 1025, 'end': 1026, 'mention': '9', 'ref_id': 'FIGREF15'}]}, {'text': 'This study proposes a new strategy to characterize the timeline of COVID-19. While this allows us to estimate the peaks of the outbreak in space and time, we need to be aware that this study uses a simple model to characterize a complex infectious disease about which we still know very little to this day. Importantly, we have to be cautious not to overstate the results. Specifically, our study has several limitations: First, our mathematical model does not account for asymptomatic cases. Little is known about the fraction of asymptomatic or mildly symptomatic individuals but early studies suggest that up to 25% of individuals have gone from susceptible to recovered without having ever been reported as infectious. Second, the classical SEIR model does not distinguish between asymptomatic infectious in the first days of the disease and symptomatic infectious in the later days. Knowing more about this group and modeling appropriately is critical to accurately estimate the impact of community spreading and mitigation strategies to reduce it. Third, while the initial infectious group \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$I_0$$\\\\end{document}I0 can be reasonably well approximated from the reported active cases and the initial recovered group \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_0$$\\\\end{document}R0 is likely zero, the initial exposed group \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$E_0$$\\\\end{document}E0 is really unknown and can hugely effect the outbreak dynamics as the sensitivity study in Fig. 3 and the data for China and the United States in Figs. 9 and 10 show. We decided to include this effect through the initial latent population \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$\\\\rho$$\\\\end{document}ρ to highlight this effect, but more data are needed to better estimate the size of this group. Fourth and probably most importantly, the major variable we can influence through social and political measures is the basic reproduction number \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_0$$\\\\end{document}R0, or rather the interplay of the contact period B and infectious period C. Obviously, we do not know the true \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_0$$\\\\end{document}R0, nor can we measure it at this stage of the outbreak, where every state, province, or country has implemented different measures to modulate the local outbreak dynamics. Nonetheless, our study shows that estimating \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$R_0$$\\\\end{document}R0 is important to quantify if and how different political countermeasures work and to predict the timeline of the infectious population under no, moderate, and massive political action. Finally, our network model only provides rough mobility estimates from air travel statistics. To more accurately simulate the spreading of COVID-19, we could gradually refine our network and include more granular mobility patterns, for example from cell phone data.', 'cite_spans': [], 'section': 'Limitations ::: Discussion', 'ref_spans': [{'start': 2205, 'end': 2206, 'mention': '3', 'ref_id': 'FIGREF9'}, {'start': 2261, 'end': 2262, 'mention': '9', 'ref_id': 'FIGREF15'}, {'start': 2267, 'end': 2269, 'mention': '10', 'ref_id': 'FIGREF1'}]}, {'text': 'The precise timeline of COVID-19, its basic reproduction number, and the effect of different mitigation strategies remain incompletely understood. Here we combined data from the outbreak in China with data from the early stages of the outbreak in the United States to identify the latent, contact, and infectious periods and the basic reproduction number of COVID-19. To quantify the outbreak dynamics, we integrated a global network model with a local epidemic SEIR model and solved the resulting set of coupled nonlinear equations using a Newton-Raphson scheme. For the outbreak in China, in \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$n=30$$\\\\end{document}n=30 provinces, we found a latent period of 2.6 days, a contact period of 1.5 days, and an infectious period of 17.8 days. For the early stages of the outbreak in the United States, in \\\\documentclass[12pt]{minimal}\\n\\t\\t\\t\\t\\\\usepackage{amsmath}\\n\\t\\t\\t\\t\\\\usepackage{wasysym} \\n\\t\\t\\t\\t\\\\usepackage{amsfonts} \\n\\t\\t\\t\\t\\\\usepackage{amssymb} \\n\\t\\t\\t\\t\\\\usepackage{amsbsy}\\n\\t\\t\\t\\t\\\\usepackage{mathrsfs}\\n\\t\\t\\t\\t\\\\usepackage{upgreek}\\n\\t\\t\\t\\t\\\\setlength{\\\\oddsidemargin}{-69pt}\\n\\t\\t\\t\\t\\\\begin{document}$$n=50$$\\\\end{document}n=50 states, we found a contact period of 3.4 days and a travel coefficient of 0.42. Our network model predicts that—without the massive political mitigation strategies that are in place today—the United States would have faced a basic reproduction number of 5.30 ± 0.95 and a nationwide peak of the outbreak on May 10, 2020 with 3 million infections. Our results suggest that mathematical modeling can help estimate outbreak dynamics and provide decision guidelines for successful outbreak control. Our model has the potential to quantify the impact of community measures and predict the effect of relaxing total lockdown, shelter in place, and travel restrictions for low-risk subgroups of the population or for the population as a whole.', 'cite_spans': [], 'section': 'Conclusion', 'ref_spans': []}], 'paragraphs': [{'text': '\\t\\t\\t\\t\\\\begin{document}$$(A+C)$$\\\\end{document}(A+C) of our SEIR model, is 20.3 days, 5.3 days shorted than the reported value of 25.6 for the 391 Shenzhen cases (Bi et al. 2020). In our model, the reported 4.8 to 5.2 day incubation periods maps onto the latent period A of 2.5 days plus 2.3 to 2.7 days within the infectious period C during which the individuals are infectious but still asymptomatic. This period is critical since individuals can spread the disease without knowing it. The contact tracing study postulates that the infectious period C begins on day 4.8 with the onset of symptoms, 2.3 days later than in our model, and ends on day 7.3 or 9.4 with or without active contract tracing with the beginning of isolation, 13.0 or 10.9 days earlier than in our model. This implies that the infectious period C of our SEIR model is 6.6 and 3.9 times larger than the infectious period of the traced and untraced early isolated population in Shenzhen (Bi et al. 2020). This comparison suggests that it is critical to understand how the infectious period is reported, either as a disease-specific parameter or as a medically modulated exposure time.', 'id': '00029'}], 'title': 'Outbreak dynamics of COVID-19 in China and the United States', 'sha': 'ec65dd386a77991f9db2c668838bd2073aaa83c3', 'doi': '10.1007/s10237-020-01332-5', 'score': 9.37399959564209}, {'rank': 132, 'paragraphs': [{'text': 'This strategy of alternating lock-downs limits social mixing, while providing an outlet for people to sustain their economic and social routines. At the same time it treats one of the main obstacles for COVID-19 mitigation -the 5 day incubation period. Indeed, isolating people for, e.g., an entire week after they have been socially active, ensures that the majority of the exposed individuals are confined to their homes. To understand this consider an individual in Group 1 who was active during week 1, and therefore might have been infected. According to the proposed routine, this individual will anyway enter lock-down in week 2, and hence will be isolated precisely during their suspected incubation period. If, at the end of week 2 they remain asymptomatic, most chances are that they are, in fact, healthy, and can, therefore, resume activity at week 3 according to the planned routine. Conversely, if they do develop symptoms during their lock-down week, they must remain in isolation, similar to all symptomatic individuals. Hence, the weekly succession is in resonance with the natural COVID-19 disease cycle, and in practice, leads to isolation of the majority of invisible spreaders.', 'id': '00003'}], 'title': 'Alternating quarantine for sustainable mitigation of COVID-19', 'sha': 'b923a8fc7d406d0f149ecd5293d1a72dec7ee526', 'doi': '', 'score': 9.372200012207031}, {'rank': 133, 'paragraphs': [{'text': 'Thus, length of incubation period is a crucial parameter to determine the risks for close contacts and guide contact tracing and quarantine policies. The estimated median incubation period in this study was 5.8 days for general transmissions; the estimated mean incubation was 6.9 days which is 33% longer than the previously frequently adopted value-5.2 mean days as reported by Li (16) . Notably, asymptomatic transmissions and young-age infections appear to have an almost two-day longer incubation period than general transmissions. Infections contacted with latent infections may have a . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'id': '00027'}], 'title': 'A systematic review and meta-analysis reveals long and dispersive incubation period of COVID-19', 'sha': '3d921c03dc23ee850655ddfd9d5ebe3956d70bf8', 'doi': '10.1101/2020.06.20.20134387', 'score': 9.370800018310547}, {'rank': 134, 'paragraphs': [{'text': ' the author proceeded to analyze the incubation period of COVID‐19 in older adults compared with younger adults.', 'id': '00013'}], 'title': 'Longer incubation period of coronavirus disease 2019 (COVID‐19) in older adults', 'sha': '8f3c0c4095c08ef8213ada5642ba4a36216dbd0d; 19d6cf627e5afd4d56919357e64d211003665028', 'doi': '10.1002/agm2.12114', 'score': 9.360400199890137}, {'rank': 135, 'paragraphs': [{'text': 'As jurisdictions begin to ease physical distancing measures, we must understand how long it may take to observe a statistically significant difference in reported case counts. We find that it generally takes between 10-60 days before changes in distancing have a substantial, detectable impact on the reported case counts. In certain cases, we find that these methods are unable to differentiate between the relaxed and unrelaxed scenarios; for example, if the deviation in R 0 is high and the relative change in physical distancing is low. Through computing daily estimates of the parameter controlling strength of physical distancing in our model, we find that, at least under public health systems in British Columbia, the time taken to detect changes in distancing is on the order 3-4 weeks. Halving the case count variance or the mean onset-to-reporting delay in British Columbia during March and April could have reduced the time taken to understand the strength of social distancing by approximately 20% and 38%, respectively. This highlights the high potential benefit of improved, consistent surveillance systems, and perhaps contact-tracing apps, which minimise delays and e.g. weekly patterns or discrepancies in case reporting.', 'id': '00034'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.357799530029297}, {'rank': 136, 'paragraphs': [{'text': \"In December 2019, a cluster of pneumonia cases with unclear pathogenesis was reported in Wuhan, Hubei Province, China. This virus was named by World Health Organization (WHO) as the severe acute respiratory syndrome coronavirus (SARS-CoV-2) and the disease it caused was named as coronavirus disease 2019 (COVID- 19) on February 11, 2020 (1). Consequently, COVID-19 was urgently classified as a Class B communicable disease and managed as a Class A communicable disease in accordance with the Law of the People's Republic of China on the Prevention and Treatment of Infectious Disease (2) . Meanwhile, the COVID-19 epidemic continued to spread around the globe, with rapid increases in case numbers in European and American countries, and a looming threat in resource-limited settings across Africa (3) . The World Health Organization (WHO) declared a global pandemic on March 11, 2020. As of May 25, the pandemic had spread to 188 countries on six continents, with a total of over 5 million diagnosed cases worldwide (4) .\", 'id': '00001'}], 'title': 'A systematic review and meta-analysis reveals long and dispersive incubation period of COVID-19', 'sha': '3d921c03dc23ee850655ddfd9d5ebe3956d70bf8', 'doi': '10.1101/2020.06.20.20134387', 'score': 9.3548002243042}, {'rank': 137, 'paragraphs': [{'text': 'Further, assuming the young age infections account for 1.6% of general transmissions, the probability of having young age infections among the general quarantine population was set at 6.15%×1.6%; the risk of having infections develop symptoms after 14-day quarantine period among the quarantine population was estimated (H). Similarly, assuming that individuals who contacted latent infections during the asymptomatic or presymptomatic period, which account for 9.3% of the general quarantine population, and 4.11% probability to be infected, the risk of observed asymptomatic transmissions developing symptoms after 14-day quarantine period is shown in Panel (I).', 'id': '00042'}], 'title': 'A systematic review and meta-analysis reveals long and dispersive incubation period of COVID-19', 'sha': '3d921c03dc23ee850655ddfd9d5ebe3956d70bf8', 'doi': '10.1101/2020.06.20.20134387', 'score': 9.345100402832031}, {'rank': 138, 'paragraphs': [{'text': 'In these equations, the strength of physical distancing is represented by the parameter 0 ≤ f (t) ≤ 1, with f (t) = 1 indicating no physical distancing. This parameter modulates transmission within the distancing compartments. In addition, individuals in the distancing compartments are less likely to be encountered by others, and so contribute a reduced portion of the force of infection as well. The value of f may be allowed to vary in time as physical distancing is strengthened or relaxed in the population. In particular, for the purpose of this work we define: We take the mean number of new cases reported on day t, µ t , to be a weighted sum of those 4 . CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'id': '00007'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.344200134277344}, {'rank': 139, 'paragraphs': [{'text': 'The copyright holder for this preprint this version posted June 16, 2020. . and θ = (u d , u r , q, D, k 1 , k 2 , ψ t ) are the remaining model parameters, which are assumed to be fixed in all of our experiments except for the simulations in Figure S6 . In this parameterization of the negative binomial distribution, the variance scales with the mean according to the dispersion parameter φ: Var[{C t }] = µ + µ 2 /φ. Values of the model parameters for our analysis using British Columbia case count data are provided in Table S1 .', 'id': '00013'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.344199180603027}, {'rank': 140, 'paragraphs': [{'text': 'In Section 2.3, we provide some evidence that the physical distancing undertaken by people in British Columbia corresponds to a modeled value for f 1 of 0.36. We consider a baseline model in which the value of f remains at 0.36 for the duration of the time. We consider alternative relaxation models in which the value of f changes from f 1 = 0.36 to either f 2 = 0.5 (weak relaxation), f 2 = 0.65 (medium relaxation) or f 2 = 0.9 (strong relaxation) on May 17th. For each condition, we again consider 100 replicates each with a value of R 0 drawn from the prior.', 'id': '00018'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.344198226928711}, {'rank': 141, 'paragraphs': [{'text': 'Although, as of the time of writing, we have only started to see the effects of relaxing physical distancing in British Columbia, we simulate the effects of relaxing distancing to 65% of the normal (unrelaxed) level, under the assumption that the noise and delay in observing case counts remains as during March/April in British Columbia (φ = 5, delay shape parameter 9.85, delay scale parameter 1.73 as estimated in [2] ). We also consider other levels of distancing relaxation (50%, 90%) and record the resulting impact on the number of days n to accept the MLE f n . For the credible bands, we increased the standard deviation to 0.15 to obtain a similar level of variation in the model fit as compared to the introduction-of-distancing estimation (the majority of individuals distancing causes the overall reproduction number to be lower).', 'id': '00022'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.344197273254395}, {'rank': 142, 'paragraphs': [{'text': 'We find that it takes 21-23 days (March 18th until April 9th or 11th) before there is a substantial difference between the baseline trajectory (the exponential growth phase of the SEIR-type model) and a trajectory with distancing measures of f 1 = 0.4, 0.7. And it takes 26 days to estimate the strength of distancing using reported data. The situation is similar for relaxing measures: it * http://github.com/carolinecolijn/ObserveEffectsDistancing 8 . CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'id': '00025'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.344196319580078}, {'rank': 143, 'paragraphs': [{'text': 'The copyright holder for this preprint this version posted June 16, 2020. . case counts and the delay between symptom onset and reporting. We explore the effects of this in Figure 5 . As the noise in the daily case counts is reduced (Figure 5a) , so is the time to detect the strength of distancing. Under a level of noise realistic for observed case counts in BC (φ = 5), we estimated 26 days to accept f n . If φ is doubled (corresponding to halving the variance, approximately, if on the order of 50 cases are observed per day), this reduces to 21 days. Under the most optimistic scenario, where there is no noise (but when there is still an average 8.7 day delay), the time-to-accept is 15 days.', 'id': '00032'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.344195365905762}, {'rank': 144, 'paragraphs': [{'text': 'As would be expected, a longer delay between symptom onset and case reporting also results in a longer time taken to estimate the strength of distancing (Figure 5b) . With a Weibull distributed delay of shape 1.73 and scale 9.85 (mean 8.78, variance 27.4) and φ = 5, as estimated for British Columbia in [2] , we estimate 32 days to accept the MLE of f 1 . If the delay is reduced to shape 2 and scale 5 (mean 4.43, variance 5.37), the time to accept is 20 days. Lastly, with a delay shape 10 and scale 1 (mean 0.95, variance 0.013), the time to accept is 15 days.', 'id': '00033'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.344194412231445}, {'rank': 145, 'paragraphs': [{'text': \"The question of determining when two curves begin to differ has been discussed in general in other literature. Such lines of work include Simultaneous Equations Models (SEM), a field within econometrics [25] and growth rates in bacterial culture [16] . These prior works require strong loglinear and normality assumptions and cannot be easily adapted to SEIR-type models. Approaches accounting for the issue of delayed case reporting in epidemiology are often referred to as 'nowcasting' [21, 33] . These are distinct from the methodology we introduce in this work in that that they focus on inference of the true number of cases at a particular time from the observed num-13 . CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\", 'id': '00035'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.344193458557129}, {'rank': 146, 'paragraphs': [{'text': 'We consider two experiments wherein the uncertainty about the ODE parameters are modulated, and record the time to detect a threshold of excess in the active cases above baseline models. In our first experiment we focus on R 0 . In the main text, we consider ODEs in which the R 0 parameter is initialized to a draw from a normal distribution with mean 2.5 and standard deviation 0.15. This reflects our broad understanding about the uncertainty in R 0 . In all conditions, the baseline indicates a situation in which physical distancing is not relaxed, and remains in effect.', 'id': '00043'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.344192504882812}, {'rank': 147, 'paragraphs': [{'text': 'The missing values indicate that our methods could not differentiate between the models. (b) The variables D, k 1 , k 2 , q, u d and u r are all varied between 10% and 30% around their fitted values (with the three levels described in the legend). The value that f is relaxed to is shown in the x-axis (before relaxation, f 1 is set to 0.36). So, R 0 is sampled at ODE initialization, but so too now are the 6 variables mentioned (with independent draws from the uniform distribution). For each f 2 level, 100 iterations are considered each with 100 replicates.', 'id': '00047'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.344191551208496}, {'rank': 148, 'paragraphs': [{'text': 'For comparison with the daily MLE method applied to British Columbia case count data in Figure 4 , we also perform estimation of R t using EpiEstim package [7] in R. Results are shown in Figure S8 . Note that EpiEstim assumes that all cases from the first time step are imported cases, leading to inflated R t estimates during the first week. To account for this, when finding the the earliest time that R t is less than/greater than 1.0 at the 95% level, we ignore this first week of estimation.', 'id': '00055'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.34419059753418}, {'rank': 149, 'paragraphs': [{'text': 'The copyright holder for this preprint this version posted June 16, 2020. Figure S8 : Median estimates of the time-dependent reproduction number R t in British Columbia, using a weekly sliding window and assumed serial interval of 5 (sd 1) days. Top panel: after implementing distancing, using true observed case counts (14 days until R t < 1 at the 95% level). Bottom panel: after relaxing distancing, using simulated data (18, 25 days until R t > 1 at the 95% level for f 2 = 0.9, 0.65, respectively), under the assumption that f 1 = 0.36 and observation noise and delay remain as pre-relaxation during March/April in BC. Coloured bands correspond to 95% quantiles.', 'id': '00057'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.344188690185547}, {'rank': 150, 'paragraphs': [{'text': 'On the other hand, the dispersion for the incubation period was estimated by e σ in which the σ is the estimate of standard deviation of the corresponding log-scaled distribution (17) . Variances of log-scaled . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'id': '00015'}], 'title': 'A systematic review and meta-analysis reveals long and dispersive incubation period of COVID-19', 'sha': '3d921c03dc23ee850655ddfd9d5ebe3956d70bf8', 'doi': '10.1101/2020.06.20.20134387', 'score': 9.344087600708008}, {'rank': 151, 'abstract': 'Coronavirus disease (COVID-19) outbreak started at Wuhan, China, and it has rapidly spread across China and many other countries. Italy is one of the European countries most affected by the COVID-19 disease, and it has registered high COVID-19 death rates and the death toll. In this article, we analyzed different Italian COVID-19 data at the regional level for the period February 24 to March 29, 2020. The analysis pipeline includes the following steps. After individuating groups of similar or dissimilar regions with respect to the ten types of available COVID-19 data using statistical test, we built several similarity matrices (reported in Supplementary file). Then, we mapped those similarity matrices into networks where nodes represent Italian regions and edges represent similarity relationships (edge length is inversely proportional to similarity). Then, network-based analysis was performed mainly discovering communities of regions that show similar behaviour. Then, network-based analysis was performed by running several community detection algorithms on those networks and by underlying communities of regions that show similar behaviour. The network-based analysis of Italian COVID-19 data is able to elegantly show how regions form communities, i.e. how they join and leave them, along time and how community consistency changes along time and with respect to the different available data.', 'body_text': [{'text': 'Coronavirus disease, called COVID-19, emerged in the city of Wuhan, in China, in November 2019 [9] .', 'cite_spans': [{'start': 95, 'end': 98, 'text': '[9]', 'ref_id': 'BIBREF8'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'The disease is caused by the novel coronavirus Sars-CoV-2 [10] and its clinical manifestations include fever, cough, fatigue, chest distress, diarrhoea, nausea, vomiting [2] and also acute respiratory distress syndrome in severe cases [8] .', 'cite_spans': [{'start': 58, 'end': 62, 'text': '[10]', 'ref_id': 'BIBREF9'}, {'start': 170, 'end': 173, 'text': '[2]', 'ref_id': 'BIBREF1'}, {'start': 235, 'end': 238, 'text': '[8]', 'ref_id': 'BIBREF7'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'COVID-19 is featured by a long incubation period, high infectivity, and different transmission ways [4] . The contagion happens mainly through respiratory and blood contact with the coronavirus.', 'cite_spans': [{'start': 100, 'end': 103, 'text': '[4]', 'ref_id': 'BIBREF3'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'In a few months, COVID-19 epidemic quickly spread to Asian countries and it reached more than 200 countries in the world, causing about tens of thousands of deaths.', 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'On March 11, 2020, COVID-19 disease was declared a pandemic by the World Health Organization.', 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'In Italy, COVID-19 has been identified on January 2020 [5] and the outbreak started in Lombardia and Veneto at the end of February 2020. From the northern regions of Italy, the disease spread very quickly to the nearest regions and then to the rest ones. Italy has considered one of the main epicentres of the pandemic recording a number of infected equal to 97689 and a number of deaths equal to 10799 up to 29rd of March. In this paper we want to underline difference at regional level and to evidence how the disease has spread along time. To do this, both statistical and network-based analysis on Italian COVID-19 data has been performed. The data are daily provided by the Italian Civil Protection, and they refer to the period February 24 to March 29, 2020. In particular, we analyzed the Italian COVID-19 data at the regional level in order to evaluate which regions show a similar trend. Then, for each type of data, we built the related networks where the nodes represent the Italian region, and the edges represent a level of similarity among them. Finally, we evaluate how the networks evolve over the weeks up to the end of March 2020. The main contribution of the the paper is a network-based representation of COVID-19 diffusion similarity among regions and graphbased visualization to underline similar diffusion regions. The rest of the paper is organized as follows: Section 2 discusses the Italian COVID-19 data, the statistical and network-based analysis, Section 3 discusses the results, Finally, Section 4 concludes the paper.', 'cite_spans': [{'start': 55, 'end': 58, 'text': '[5]', 'ref_id': 'BIBREF4'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'In this section, we investigate the evolution of the coronavirus pandemic in Italy from a statistical and network-based perspective, The analysis uses data at different temporal zoom e.g. by analysing the period from February 24 to March 29, 2020 and by focusing on single week as well as the entire observation. Moreover analysis comparing data at regional level is also provided. The analysis pipeline includes the following steps:', 'cite_spans': [], 'ref_spans': [], 'section': 'Material and Methods'}, {'text': '1. Building of ten similarity matrices (one matrix for each data measure. The (i,j) value of the matrix for data k (e.g. Swab data) represents the p-value of the Wilcoxon statistical test obtained by performing the test on the swab measures of region i with respect to region j. Lower p-value means that regions are more dissimilar with respect to that measure. Higher p-value means that regions are more similar with respect to that measure. We used the usual significance threshold of 0.05, thus matrices report only p − vales >= 0.05, while p − values < 0.05 are mapped to zero. In such a way the p-value is used as a similarity measure.', 'cite_spans': [], 'ref_spans': [], 'section': 'Material and Methods'}, {'text': '4. Community detection. For each network we extracted subgroups of regions that form a community on the basis of similarity point of view. The identification of community is performed on the networks related to the observation period and for all single week. The, we analized the evolution of the communities at different time points, i.e. at the end of the first week, after tree weeks and after five weeks (the observation period).', 'cite_spans': [], 'ref_spans': [], 'section': 'Material and Methods'}, {'text': 'The present analysis was carried on the dataset on COVID1-9 updated in https://github.com/pcm-dpc/COVID-19 database, provided by the Italian Civil Protection. The dataset consist of:', 'cite_spans': [], 'ref_spans': [], 'section': 'Input DataSet'}, {'text': '• Hospitalised with Symptoms, the numbers of hospitalised patients that present COVID-19 symptoms;', 'cite_spans': [], 'ref_spans': [], 'section': 'Input DataSet'}, {'text': '• Intensive Care, the numbers of hospitalised patients in Intensive Care Units;', 'cite_spans': [], 'ref_spans': [], 'section': 'Input DataSet'}, {'text': '• Total Hospitalised, the total numbers of hospitalised patients;', 'cite_spans': [], 'ref_spans': [], 'section': 'Input DataSet'}, {'text': '• Home Isolation, the numbers of subjects that are in isolation at home;', 'cite_spans': [], 'ref_spans': [], 'section': 'Input DataSet'}, {'text': '• Total Currently Positive, the numbers of subjects that are coronavirus positive;', 'cite_spans': [], 'ref_spans': [], 'section': 'Input DataSet'}, {'text': '• New Currently Positive, the numbers of subjects that are daily coronavirus positive;', 'cite_spans': [], 'ref_spans': [], 'section': 'Input DataSet'}, {'text': '• Discharged/ Healed the numbers of subjects that are healed from the disease;', 'cite_spans': [], 'ref_spans': [], 'section': 'Input DataSet'}, {'text': '• Deceased, the numbers of dead patients;', 'cite_spans': [], 'ref_spans': [], 'section': 'Input DataSet'}, {'text': '• Total Cases, the numbers of subjects affected by COVID-19;', 'cite_spans': [], 'ref_spans': [], 'section': 'Input DataSet'}, {'text': '• Swabs, the numbers of test swab carried on positive subjects and on subjects with suspected positivity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Input DataSet'}, {'text': 'The data are daily provided for each Italian region. The data occupies 47.6 Mbytes of memory.', 'cite_spans': [], 'ref_spans': [], 'section': 'Input DataSet'}, {'text': \"We analyzed the trend of each type of data for the period February 24 to March 29, 2020. We computed the main descriptive statistics for all regions in the study period. All analysis are performed by using R software [6] . The Figure 1 conveys the evolution of all dataset over days. As a preliminary test, we applied Pearson's chi-square test. Since p-value was less than 0.05 for each distribution, we decided to use nonparametric test for the following comparison. As initial step, we used the Wilcoxon Sum Rank test to carry out an analysis within the same type of data for all weeks and then, for each single week. The Wilcoxon test is a non parametric test designed to evaluate the difference between two treatments or conditions where the samples are correlated. The Wilcoxon test performs a pair-wise comparison among regions with the goal to evidence which ones show different trend. For this reason, we built a similarity matrix for each couple of regions, for each of the available COVID-19 data. Table 1 reports the similarity matrix related to Hospitalised with Symptoms network in the observation period. We reported the all similarity matrices computed for Italian COVID-19 data in the observation period e in the sigle week in Supplementary file, for the lack of space.\", 'cite_spans': [{'start': 217, 'end': 220, 'text': '[6]', 'ref_id': 'BIBREF5'}], 'ref_spans': [{'start': 227, 'end': 235, 'text': 'Figure 1', 'ref_id': 'FIGREF0'}, {'start': 1008, 'end': 1015, 'text': 'Table 1', 'ref_id': 'TABREF1'}], 'section': 'Statistical analysis'}, {'text': 'Results show that according the type of data, a significant difference exists (p-value less than 0.05) among some regions while for others it is Finally, we used Kruskal-Wallis test performing an analysis on the same type of data for all regions (i.e. carrying out multiple comparisons) for the observation period and then, for each single week. Kruskal-Wallis test is a non-parametric method for analysis of variance used to determine if more samples originate from the same distribution. The results confirmed a significant difference considering all regions on the same type of data for the observation period for each single week.', 'cite_spans': [], 'ref_spans': [], 'section': 'Statistical analysis'}, {'text': 'In order to evaluate the evolution of Italian COVID-19 data and evidence which regions show similar trend, we built networks of each data [1] starting from the result of Wilcoxon test. The nodes of the networks are the Italian regions and the edges link two regions (nodes) with similar trend according to significance level (p-value > 0.05) obtained from Wilcoxon test, otherwise (p-value < 0.05) there is not connection among nodes. The length of the edges is inversely proportional to nodes similarity. The network analysis is performed using the igraph libraries [3] .', 'cite_spans': [{'start': 138, 'end': 141, 'text': '[1]', 'ref_id': 'BIBREF0'}, {'start': 567, 'end': 570, 'text': '[3]', 'ref_id': 'BIBREF2'}], 'ref_spans': [], 'section': 'Network-based Analysis'}, {'text': 'At first, we built ten networks, one for each data (Hospitalised with Symptoms, Intensive Care data, Total Hospitalised, Home Isolation, Total Currently Positive, New Currently Positive, Discharged/ Healed, Deceased, Total Cases, Swab) by considering the period February 24 to March 29. Then, we build the same networks by considering single weeks. The ten networks for the observation period and for the five weeks are reported in Figures 2-11 .', 'cite_spans': [], 'ref_spans': [{'start': 432, 'end': 444, 'text': 'Figures 2-11', 'ref_id': 'FIGREF0'}], 'section': 'Network-based Analysis'}, {'text': 'Then, starting from the ten networks related to all five weeks, we wanted to identify which regions form a community from the similarity point of view. For this, we applied Walktrap community finding algorithm [7] that identifies densely connected subgraphs, i.e communities, in a graph via random walks. The idea is that short random walks tend to stay in the same community.', 'cite_spans': [{'start': 210, 'end': 213, 'text': '[7]', 'ref_id': 'BIBREF6'}], 'ref_spans': [], 'section': 'Network-based Analysis'}, {'text': 'The extracted communities from all Italian COVID-19 networks are reported in Figure 12 .', 'cite_spans': [], 'ref_spans': [{'start': 77, 'end': 86, 'text': 'Figure 12', 'ref_id': 'FIGREF0'}], 'section': 'Network-based Analysis'}, {'text': 'Finally, we analize the evolution of the extracted communities over a period of time. We want to evaluate how different data present different All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Network-based Analysis'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Network-based Analysis'}, {'text': 'The copyright holder for this preprint this version posted April 22, 2020. . https://doi.org/10.1101/2020.04.17.20068916 doi: medRxiv preprint communities and how the communities are different considering different temporal interval on the same data. We analized the evolution of the communities at different time points, i.e. at the end of the first week, after tree weeks and after five weeks (the observation period). The Figure  13 -22 report the evolution of the communities related to different data.', 'cite_spans': [], 'ref_spans': [{'start': 425, 'end': 435, 'text': 'Figure  13', 'ref_id': 'FIGREF0'}], 'section': 'Network-based Analysis'}, {'text': \"In this section, we analyze the evolution of the Italian COVID-19 networks data. Figure 2 shows the evolution of Hospitalised with Symptoms network over five weeks. Figure 2 (a) reports the network that represents the behaviour of regions respect to number of hospitalised patients up to March 29, whereas Figure 2 (b), (c), (d), (e), (f) represent the networks on each single week. It is to notice, the network structure changes according to the analyzed time interval. At the end of 35th day, the network has all nodes connected with exception of a single node that represent the Basilicata region. Furthermore, it is possibile to highlight different community structures consisting of groups of regions with similar trends. Figure 12 (a) reports five communities: the first one consists of Basilicata; the second one is composed by Piemonte, Marche, Emilia, Lombardia, Veneto; the third one is composed by Liguria, Lazio, Toscana; the fourth one is formed by Campania, Puglia, Sicilia, Abruzzo, Valle d' Aosta, Friuli, Trento, Bolzano; the last one is composed by Umbria, Sardegna, Calabria, Molise. Figure 3 represents the evolution of Intensive Care network. It is possible to notice that Lombardia and Veneto, that are the most affected region by Coronavirus disease with high number hospitalised in Intensive Care Units, are disconnected in the first week. In the second week, Lombardia and Veneto are connected by an edge that represents a level of similarity, whereas in the fifth week Veneto is linked with Emilia and Lombardia and this group of regions becomes a disconnected component among with other regions. Thus, while initially Lombardia and Veneto showed a similar trend of Intensive Care data, after Veneto moved far from Lombardia trend. Furthermore, by analyzing the communities in the Intensive Care network (Figure 12 (b)), it is possible to evidence four subgraphs formed by (i) Lombardia and Veneto, (ii) Umbria and Lazio, (iii) Marche, Emilia, Piemonte, Toscana and (iv) a large module formed by Campania, Sicilia, Sardegna, Abruzzo, Umbria, Calabria, Basilicata, Bolzano, Valle d'Aosta, Friuli, Trento, Molise. Figure 4 show the Total Hospitalised network evolution. Starting form the first week, the structure of the network has two disconnected nodes representing Lombardia and Veneto, whereas in the second week the network evolves by presenting a single disconnected node that represent Emilia, and two connected nodes, Lombardia and Veneto, which in turn are disconnected from dense subgraph. In the third and fourth week the network structure presents all connected components, and high number of nodes result disconnected in the fifth week. Finally, all re-6 All rights reserved. No reuse allowed without permission.\", 'cite_spans': [], 'ref_spans': [{'start': 81, 'end': 89, 'text': 'Figure 2', 'ref_id': None}, {'start': 165, 'end': 173, 'text': 'Figure 2', 'ref_id': None}, {'start': 306, 'end': 314, 'text': 'Figure 2', 'ref_id': None}, {'start': 727, 'end': 736, 'text': 'Figure 12', 'ref_id': 'FIGREF0'}, {'start': 1103, 'end': 1111, 'text': 'Figure 3', 'ref_id': None}, {'start': 1830, 'end': 1840, 'text': '(Figure 12', 'ref_id': 'FIGREF0'}, {'start': 2138, 'end': 2146, 'text': 'Figure 4', 'ref_id': None}], 'section': 'Temporal Evolution of Networks'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Temporal Evolution of Networks'}, {'text': 'The copyright holder for this preprint this version posted April 22, 2020. . https://doi.org/10.1101/2020.04.17.20068916 doi: medRxiv preprint gions are connected in the final network. By analyzing the communities detected in Total Hospitalised network (Figure 12 (c)), it is possible to notice a similarity with those extracted by Hospitalised with Symptoms networks. In fact, there is a correspondence among three communities: (i) Basilicata and Piemonte, (ii) Marche, Emilia, Lombardia, (iii) Veneto and Liguria, Lazio, Toscana. This means that those regions that form the tree communities present the same behaviour according to the number of the hospitalised patients with symptoms and the total number of hospitalised patients. Figure 5 shows the evolution of Home Isolation network over five weeks. In first week, Lombardia, Veneto and Emilia show a different trend of the number of subjects in isolation at home, both them and also compared to other regions. Next, in the network resulting by considering the all weeks, Lombardia, Veneto, Emilia and Marche formed a subgraph disconnected by a different dense subgraph composed by the rest of the regions. However, Veneto represents a single community in Figure 12 (d). This means that the behaviour of Veneto presents a low similarity respect to Lombardia, Emilia and Marche despite forming a module. It is possible to notice that: Lombardia, Veneto, Emilia form a community in both Total Currently Positive network and New Currently Positive network, Piemonte represent in sigle community in Total Currently Positive network and while Piemonte is associated with Marche and Toscana in New Currently Positive network. Figure 8 represents the Discharged/ Healed network over the five weeks. In the first week, the network structure present all nodes connected with exception of Veneto. In the second week, Discharged/ Healed network is formed by three subgraphs; in the third and fourth weeks the network results very dense; in the fifth week is caracterized by different disconnected components, and finally, at the end of 35 days the network is composed by a subgraph composed by Lombardia and Veneto and another subgraph highly connected. This means that Lombardia and Veneto have a similar behaviour that it is different by the rest of the Italian regions. Also Lombardia and Veneto represents one of the five communities extracted by Discharged/ Healed network. The extracted communities are reported in Figure 12 (g). Figure 9 shows the evolution of Deceased network. The evolution of this network is different by other Italian COVID-19 networks. In fact, in first week all nodes are disconnected, so all Italian regions present 7 All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [{'start': 253, 'end': 263, 'text': '(Figure 12', 'ref_id': 'FIGREF0'}, {'start': 734, 'end': 742, 'text': 'Figure 5', 'ref_id': None}, {'start': 1212, 'end': 1221, 'text': 'Figure 12', 'ref_id': 'FIGREF0'}, {'start': 1676, 'end': 1684, 'text': 'Figure 8', 'ref_id': 'FIGREF8'}, {'start': 2466, 'end': 2475, 'text': 'Figure 12', 'ref_id': 'FIGREF0'}, {'start': 2481, 'end': 2489, 'text': 'Figure 9', 'ref_id': 'FIGREF9'}], 'section': 'Temporal Evolution of Networks'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Temporal Evolution of Networks'}, {'text': \"The copyright holder for this preprint this version posted April 22, 2020. . https://doi.org/10.1101/2020.04.17.20068916 doi: medRxiv preprint different trend. In the second week, it is possible to notice that Emilia and Marche nodes are disconnected and there is a subgraph composed by Lombardia and Veneto and then there is a large subgraph formed by other regions. In the third week all nodes present connections. In the fourth and and fifth week the Deceased network presents different disconnected components, then, the final network shows a single disconnected node that represents Basilicata. Also the Basilicata represents a single community of Deceased network, see Figure 12 (h). The others extracted communities are: (i) Piemonte, Toscana Liguria, Lazio, Friuli, Puglia, Valle d' Aosta, (ii) Lombardia, Veneto, Emilia, Marche, (iii) Sicilia, Molise, Abruzzo, Umbria, Campania, Trento, Bolzano, Calabria, Sardegna. Figure 10 represents the Total Cases network over the five weeks. The final network evidences that the Italian regions present a significant level of similarity respect to the number of total Coronavirus cases because all nodes are connected. Figure 12 (i) show the communities identified in Total Cases. The first community is composed by Lombardia, Veneto, Emilia, Marche; the second community is composed by Piemonte; the third community is composed by Basilicata, Molise; the fourth community is composed by Toscana Liguria, Lazio, Campania, Friuli, Sicilia Puglia, Valle d' Aosta formed, whereas Abruzzo, Umbria, Trento, Bolzano, Calabria, Sardegna formed the fifth community. Figure 11 shows the evolution of Swab Network that represents the number of performed swab tests. The network, in the first week, shows Lombardia and Veneto nodes disconnected by other regions. In fact they are the Italian regions that initially performed high number of test swabs. Also, Veneto region has not connections in the final network and this reflects the policy of the Veneto to carry out swab tests on asymptomatic subjects, i.e. it is an outlier with respect to other regions. Figure 12 (j) shows the extracted communities in the Swab network. The first community is composed by Veneto; the second community is composed by Lombardia, Emilia; the third community is composed by Basilicata, Molise; the fourth community is formed by Marche, Toscana, Lazio, Piemonte, Friuli, Valle d'Aosta; the fifth community is formed by Sicilia, Campania, Liguria, Puglia; while Abruzzo, Umbria, Trento, Bolzano, Calabria, Sardegna formed the sixth community.\", 'cite_spans': [], 'ref_spans': [{'start': 675, 'end': 684, 'text': 'Figure 12', 'ref_id': 'FIGREF0'}, {'start': 925, 'end': 934, 'text': 'Figure 10', 'ref_id': 'FIGREF0'}, {'start': 1168, 'end': 1177, 'text': 'Figure 12', 'ref_id': 'FIGREF0'}, {'start': 1607, 'end': 1616, 'text': 'Figure 11', 'ref_id': 'FIGREF0'}, {'start': 2097, 'end': 2106, 'text': 'Figure 12', 'ref_id': 'FIGREF0'}], 'section': 'Temporal Evolution of Networks'}, {'text': 'In this section, we analize the evolution of communities over a period of time. We consider three different temporal zoom: at the end of the fist week, after three weeks and in the observation period. Figure 13 reports the evolution of Hospitalised with Symptoms Network Communities. At the end of three weeks, Venezia that represents a community in the previous week, moves in another community, whereas Emilia leaves the community with Marche and becomes a single community, also some re-8 All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [{'start': 201, 'end': 210, 'text': 'Figure 13', 'ref_id': 'FIGREF0'}], 'section': 'Analysis of Communities'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Analysis of Communities'}, {'text': \"The copyright holder for this preprint this version posted April 22, 2020. . https://doi.org/10.1101/2020.04.17.20068916 doi: medRxiv preprint gions migrate from fifth and sixth communities to other communities. So, Figure 13 (b) reports the five extracted communities after three weeks: (i) Lombardia, (ii)Emilia, (iii)Veneto, Marche, Piemonte, Liguria, Toscana, Lazio, (iv) Trento, Bolzano, Abruzzo, Friuli, Sicilia, Puglia, (v)Campania, Umbria, Sardegna, Calabria, Molise, Valle d'Aosta, Basilicata. Finally, Figure 13 (c)reports five communities in the observation period: the first one consists of Basilicata that leaves the previous community and becomes a single one; the second one is composed by Piemonte, Marche, Emilia, Lombardia, Veneto; the third one is composed by Liguria, Lazio, Toscana; the fourth one is formed by Campania, Puglia, Sicilia, Abruzzo, Valle d'Aosta, Friuli, Trento, Bolzano; the last one is composed by Umbria, Sardegna, Calabria, Molise. After three weeks the number of extracted communities increases, see Figure 14 Figure 15 reports the evolution of Total Hospitalised Network Communities. Figure 15 (a) shows the five mined communities. The fist community is composed by Liguria, Toscana, Piemonte, the second one is formed Sardegna, Umbria, Calabria, Basilicata, Valle d'Aosta, Friuli, Trento, Molise; the third module comprises Lazio, Campania, Sicilia, Abruzzo, Bolzano, Puglia; the forth community is represented by Marche, Emilia; the fifth is represented by Lombardia and the last one consists of Veneto.\", 'cite_spans': [], 'ref_spans': [{'start': 216, 'end': 225, 'text': 'Figure 13', 'ref_id': 'FIGREF0'}, {'start': 512, 'end': 521, 'text': 'Figure 13', 'ref_id': 'FIGREF0'}, {'start': 1041, 'end': 1050, 'text': 'Figure 14', 'ref_id': 'FIGREF0'}, {'start': 1051, 'end': 1060, 'text': 'Figure 15', 'ref_id': 'FIGREF0'}, {'start': 1126, 'end': 1135, 'text': 'Figure 15', 'ref_id': 'FIGREF0'}], 'section': 'Analysis of Communities'}, {'text': \"After three weeks (see Figure 15 (b)) the regions moves,, with the exception of Lombardia that continues to represent a single community and the communities becomes eight. In fact, Emilia becomes a single community; Veneto becomes a community among with Piemonte and Marche; Toscana moves in the community with Liguria; Lazio and Campania forms a new community, as well as, Basilicata and Valle d'Aosta; another community is formed by: Abruzzo, Puglia, Sicilia and the last one is formed by Friuli, Bolzano, Trento, Umbria, Sardegna, Calabria, Molise.\", 'cite_spans': [], 'ref_spans': [{'start': 23, 'end': 32, 'text': 'Figure 15', 'ref_id': 'FIGREF0'}], 'section': 'Analysis of Communities'}, {'text': 'At the end of the observation period the five communities, reported in Figure 15(b) , are formed. The first one consists of Basilicata that 9 All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [{'start': 71, 'end': 83, 'text': 'Figure 15(b)', 'ref_id': 'FIGREF0'}], 'section': 'Analysis of Communities'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Analysis of Communities'}, {'text': \"The copyright holder for this preprint this version posted April 22, 2020. . https://doi.org/10.1101/2020.04.17.20068916 doi: medRxiv preprint leaves the previous community and becomes a single one; the second one is composed by Toscana, Liguria and Lazio; Sardegna, Calabria, Molise leaves the previous large community and form a smaller one; by the fourth one is formed by Piemonte, Marche, Emilia, Lombardia, Veneto; the last one is composed by Umbria, Puglia, Sicilia, Abruzzo, Valle d'Aosta, Friuli, Trento, Bolzano; Figure 15 reports the evolution of Home Isolation Network Communities. Figure 16( Figure  16 (b)). Figure 16 (c) shows the communities topology in the observation period. Veneto leaves the community among with Lombardia and Emilia and it becomes a single one, whereas, Lombardia, Emilia forms a new module among with Marche. Basilicata and Molise move together to form a unique community. Sardegna, Calabria, Abruzzo, Bolzano form a forth community. The fifth community is composed by Puglia, Trento, Lazio, Umbria, Sicilia, and the sixth one is represented by Toscana, Piemonte, Valle d'Aosta, Friuli, Liguria, Campania. Figure 17 reports the evolution of Total Currently Positive Communities. At the end of first week the mined communities are eight and they are reported in Figure 17(a) . The fist community is composed by Umbria, Sardegna, Basilicata, Molise, Friuli Toscana, Calabria, Valle d'Aosta, Trento,; the second one is formed by Bolzano, Lazio, Abruzzo, Puglia; the third module comprises Campania, Sicilia Liguria; Piemonte and Lazio represent the forth community; the fifth one consists of Marche; the sixth community is represented by Emilia; the seventh is represented by Lombardia and the last one consists of Veneto.\", 'cite_spans': [], 'ref_spans': [{'start': 522, 'end': 531, 'text': 'Figure 15', 'ref_id': 'FIGREF0'}, {'start': 593, 'end': 603, 'text': 'Figure 16(', 'ref_id': 'FIGREF0'}, {'start': 604, 'end': 614, 'text': 'Figure  16', 'ref_id': 'FIGREF0'}, {'start': 621, 'end': 630, 'text': 'Figure 16', 'ref_id': 'FIGREF0'}, {'start': 1144, 'end': 1153, 'text': 'Figure 17', 'ref_id': 'FIGREF0'}, {'start': 1299, 'end': 1311, 'text': 'Figure 17(a)', 'ref_id': 'FIGREF0'}], 'section': 'Analysis of Communities'}, {'text': \"After three weeks, the number of communities (see Figure 17 (b)) decreases. In fact, it is possible to notice five subgraphs. Emilia joints with Veneto and Lombardia remains single community. Lazio, Sicilia, Friuli, Puglia form a new community; Piemonte, Toscana, Campania, Marche, Liguria, represent a forth community; Trento, Abruzzo, Umbria, Calabria, Sardegna, Basilicata, Molise, Bolzano, Campania, Valle d'Aosta, form a fifth community;\", 'cite_spans': [], 'ref_spans': [{'start': 50, 'end': 59, 'text': 'Figure 17', 'ref_id': 'FIGREF0'}], 'section': 'Analysis of Communities'}, {'text': 'In the observation period, the number of extracted communities further decreases, see Figure 17 (c) The first community is composed by Veneto, Lombardia, Emilia, Marche; the second community is composed by Piemonte; the third community is composed by Basilicata, Molise, Calabria, Sardegna; the fourth community is formed by Toscana, Lazio, 10 All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [{'start': 86, 'end': 95, 'text': 'Figure 17', 'ref_id': 'FIGREF0'}], 'section': 'Analysis of Communities'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Analysis of Communities'}, {'text': \"The copyright holder for this preprint this version posted April 22, 2020. . https://doi.org/10.1101/2020.04.17.20068916 doi: medRxiv preprint Friuli, Valle d' Aosta, Sicilia, Campania, Liguria Puglia, Abruzzo, Umbria, Trento, Bolzano. Figure 18 reports the evolution of New Currently Positive Communities. Figure 18(a) reports the mined communities at the end of first week. It is possible to notice that Lombardia, Veneto and Emilia form single communities, and then there are two large communities: the first one is represented by Marche, Piemonte, Liguria, Campania, Abruzzo, Toscana; the second one is composed by Puglia, Valle d'Aosta, Umbria, Calabria, Sicilia, Campania, Trento, Lazio, Sardegna, Bolzano, Basilicata and Molise, Friuli.\", 'cite_spans': [], 'ref_spans': [{'start': 236, 'end': 245, 'text': 'Figure 18', 'ref_id': 'FIGREF0'}, {'start': 307, 'end': 319, 'text': 'Figure 18(a)', 'ref_id': 'FIGREF0'}], 'section': 'Analysis of Communities'}, {'text': \"After three weeks, the extracted communities remain five but the regions that form them vary, see Figure 18 (b). The first community is composed by Lombardia; the second community is composed by Veneto and Emilia; the third community is composed by Basilicata, Molise, Valle d' Aosta, Sardegna, Campania, Bolzano; the fourth community is formed by Marche, Toscana, Piemonte; the fifth community is formed by Sicilia, Liguria,Puglia, Abruzzo, Umbria, Trento,, Calabria, Lazio, Friuli.\", 'cite_spans': [], 'ref_spans': [{'start': 98, 'end': 107, 'text': 'Figure 18', 'ref_id': 'FIGREF0'}], 'section': 'Analysis of Communities'}, {'text': \"In the observation period, the number of extracted communities further decreases, see Figure 18 Figure 19 reports the evolution of Discharged/ Healed Network Communities. It is possible to notice that in the first week, Figure 19(a) , there are: a large community formed by Umbria, Piemonte, Toscana Campania, Sicilia, Sardegna, Abruzzo, Umbria, Calabria, Basilicata, Bolzano, Emilia, Valle d'Aosta, Friuli, Trento, Molise; a small community formed by Lombardia, Marche, Lazio; and single communities formed by Veneto.\", 'cite_spans': [], 'ref_spans': [{'start': 86, 'end': 95, 'text': 'Figure 18', 'ref_id': 'FIGREF0'}, {'start': 96, 'end': 105, 'text': 'Figure 19', 'ref_id': 'FIGREF0'}, {'start': 220, 'end': 232, 'text': 'Figure 19(a)', 'ref_id': 'FIGREF0'}], 'section': 'Analysis of Communities'}, {'text': \"After three weeks the number of extracted communities increases, see Figure 19 (b). In fact, Lombardia leaves the previous community and becomes a single one; Lazio, Emilia, Liguria and Veneto get together to form a second community; the third is composed by Friuli, Sicilia, Toscana; the last one is formed by Sardegna, Valle d'Aosta, Marche and Piemonte, Umbria, Campania, Molise, Abruzzo, Trento, Puglia, Calabria, Bolzano, Basilicata.\", 'cite_spans': [], 'ref_spans': [{'start': 69, 'end': 78, 'text': 'Figure 19', 'ref_id': 'FIGREF0'}], 'section': 'Analysis of Communities'}, {'text': \"Finally, Figure 19 (c) shows the communities in the observation period. It is possible to notice five community. The first one consists of Lombardia and Veneto; the second one is composed by Emilia, Liguria, Lazio; the third one is composed by Friuli, Campania, Toscana, Sicilia; the fourth one is formed by, Puglia, Abruzzo, Trento; the last one is composed by Umbria, Sardegna, Calabria, Piemonte, Marche, Valle d'Aosta, Bolzano, Basilicata, Molise. Figure 20 reports the evolution of Deceased Communities. Figure  20 (a) reports the mined communities at the end of first week. It is possible to notice that there are: a large community formed by Emilia, Piemonte, Liguria, Campania, Abruzzo, Puglia, Valle d'Aosta, Umbria, Calabria, Sicilia, Campania, Trento, Lazio, Sardegna, Bolzano, Basilicata and Molise, Friuli; two single communities represented by Lombardia and Veneto; and a single community composed by Marche and Toscana.\", 'cite_spans': [], 'ref_spans': [{'start': 9, 'end': 18, 'text': 'Figure 19', 'ref_id': 'FIGREF0'}, {'start': 452, 'end': 461, 'text': 'Figure 20', 'ref_id': None}, {'start': 509, 'end': 519, 'text': 'Figure  20', 'ref_id': None}], 'section': 'Analysis of Communities'}, {'text': 'After three weeks, the number of extracted communities increase. In 11 All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Analysis of Communities'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Analysis of Communities'}, {'text': \"The copyright holder for this preprint this version posted April 22, 2020. . https://doi.org/10.1101/2020.04.17.20068916 doi: medRxiv preprint fact, the regions that form them vary by forming new communities, see Figure 20 (b). The first community is composed by Lombardia that remains a single community; the second community is composed by Veneto and the third one is composed by Emilia; the fourth community is formed by Basilicata, Molise, Sardegna, Calabria; the fifth community is formed by Sicilia, Liguria,Puglia, Abruzzo, Umbria, Trento, Lazio, Friuli, Valle d'Aosta, Campania, Bolzano; Marche, Toscana, Piemonte.\", 'cite_spans': [], 'ref_spans': [{'start': 213, 'end': 222, 'text': 'Figure 20', 'ref_id': None}], 'section': 'Analysis of Communities'}, {'text': \"In the observation period, the number of extracted communities decreases, see Figure 20 (c) and there are: (i) Basilicata represents a single community, (ii) Piemonte, Toscana Liguria, Lazio, Friuli, Puglia, Valle d' Aosta, (iii) Lombardia, Veneto, Emilia, Marche, (iv) Sicilia, Molise, Abruzzo, Umbria, Campania, Trento, Bolzano, Calabria, Sardegna. Figure 21 reports the evolution of Total Cases Network Communities. Figure 21 (a) shows the five mined communities. The fist community is composed by Liguria, Toscana Lazio, Piemonte, Campania, Sicilia; the second one is formed Sardegna, Abruzzo, Umbria, Calabria, Basilicata, Bolzano, Valle d'Aosta, Friuli, Trento, Molise, Puglia; the third module comprises Marche, Emilia; the forth one Lombardia and the last one Veneto.\", 'cite_spans': [], 'ref_spans': [{'start': 78, 'end': 87, 'text': 'Figure 20', 'ref_id': None}, {'start': 351, 'end': 360, 'text': 'Figure 21', 'ref_id': 'FIGREF0'}, {'start': 419, 'end': 428, 'text': 'Figure 21', 'ref_id': 'FIGREF0'}], 'section': 'Analysis of Communities'}, {'text': \"After three weeks (see Figure 21 (b)) the communities are six. Veneto becomes a community among with Emilia; Marche moves in the community with Liguria, Toscana Lazio, Piemonte, Campania; and three new modules are formed: the first one is composed by Sicilia, Friuli and Puglia, the second one is composed by Abruzzo, Bolzano, Trento and Umbria, the third one is formed by Basilicata, Sardegna, Valle d'Aosta, Calabria, Molise.\", 'cite_spans': [], 'ref_spans': [{'start': 23, 'end': 32, 'text': 'Figure 21', 'ref_id': 'FIGREF0'}], 'section': 'Analysis of Communities'}, {'text': \"At the end of the observation period the communities extracted are five. Figure 21(c) reports the communities. The first community is composed by Lombardia, Veneto, Emilia, Marche; the second community is composed by Piemonte; the third community is composed by Basilicata, Molise; the fourth community is composed by Toscana Liguria, Lazio, Campania, Friuli, Sicilia Puglia, Valle d' Aosta formed , whereas Abruzzo, Umbria, Trento, Bolzano, Calabria, Sardegna formed the fifth community. Figure 21 reports the evolution of Swab Network Communities. At the end of first week the mined communities are eight and they are reported in Figure 22 (a). The fist community is composed by Umbria, Calabria, Valle d'Aosta, Trento, Sicilia, Abruzzo, Puglia; the second one is formed by Sardegna, Basilicata, Molise, Bolzano; the third module comprises Friuli, Campania, Liguria; Piemonte represents the forth community; the fifth one consists of Toscana and Lazio; the sixth community is represented by Emilia and Marche; the seventh is represented by Lombardia and the last one consists of Veneto.\", 'cite_spans': [], 'ref_spans': [{'start': 73, 'end': 85, 'text': 'Figure 21(c)', 'ref_id': 'FIGREF0'}, {'start': 489, 'end': 498, 'text': 'Figure 21', 'ref_id': 'FIGREF0'}, {'start': 632, 'end': 641, 'text': 'Figure 22', 'ref_id': None}], 'section': 'Analysis of Communities'}, {'text': \"After three weeks, the number of communities (see Figure 22 (b)) decreases. In fact, it is possible to notice six subgraphs. Emilia leaves the previous community and forms a single one. Lombardia and Veneto join together. Toscana and Lazio continue to form a community; Piemonte, Friuli, Campania, Marche, Puglia, Liguria, Sicilia form a forth community; Trento, Abruzzo, Umbria, Calabria, form a fifth community and the last one is composed by Sardegna, Basilicata, Molise, Bolzano, Campania, Valle d'Aosta; 12 All rights reserved. No reuse allowed without permission.\", 'cite_spans': [], 'ref_spans': [{'start': 50, 'end': 59, 'text': 'Figure 22', 'ref_id': None}], 'section': 'Analysis of Communities'}, {'text': \"(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. In the observation period, the extracted communities remain six but the regions that form them vary, see Figure 22 (c) The first community is composed by Veneto; the second community is composed by Lombardia, Emilia; the third community is composed by Basilicata, Molise; the fourth community is formed by Marche, Toscana, Lazio, Piemonte, Friuli, Valle d' Aosta; the fifth community is formed by Sicilia, Campania, Liguria Puglia; while Abruzzo, Umbria, Trento, Bolzano, Calabria, Sardegna formed the sixth community.\", 'cite_spans': [], 'ref_spans': [{'start': 241, 'end': 250, 'text': 'Figure 22', 'ref_id': None}], 'section': 'Analysis of Communities'}, {'text': 'In conclusion we can affirm that the topology of the communities varies, i.e. the regions join and leave them along time and the community consistency changes along time and with respect to the different available data. (which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Analysis of Communities'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Analysis of Communities'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Analysis of Communities'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Analysis of Communities'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Analysis of Communities'}, {'text': 'The copyright holder for this preprint this version posted April 22, 2020. ', 'cite_spans': [], 'ref_spans': [], 'section': 'Analysis of Communities'}, {'text': 'All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': '25'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '25'}, {'text': 'The copyright holder for this preprint this version posted April 22, 2020. ', 'cite_spans': [], 'ref_spans': [], 'section': '25'}, {'text': 'All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': '26'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '26'}, {'text': 'The copyright holder for this preprint this version posted April 22, 2020. ', 'cite_spans': [], 'ref_spans': [], 'section': '26'}, {'text': 'All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': '27'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '27'}, {'text': 'The copyright holder for this preprint this version posted April 22, 2020. ', 'cite_spans': [], 'ref_spans': [], 'section': '27'}, {'text': 'The COVID-19 disease has spread worldwide in a matter of weeks. In Italy, the epidemic of COVID-19 has started in the north of the State and it quickly involved all regions. In this paper we evaluated the evolution of Italian COVID-19 data daily provided by the Italian Civil Protection. The goal this work is the network-based representation of COVID-19 diffusion 28 All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusion'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusion'}, {'text': 'The copyright holder for this preprint this version posted April 22, 2020. . https://doi.org/10.1101/2020.04.17.20068916 doi: medRxiv preprint similarity among regions and graph-based visualization with the aim to underline similar diffusion regions. We identified similar Italian region with respect to the available COVID-19 data and we mapped these ones in different networks. Finally, we performed a network-based analysis to discover communities of regions that show similar behaviour. As future work we plan to extend the study by adding other data such as particulate matter (PM10) data available at regional level.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusion'}], 'paragraphs': [{'text': 'COVID-19 is featured by a long incubation period, high infectivity, and different transmission ways [4] . The contagion happens mainly through respiratory and blood contact with the coronavirus.', 'id': '00003'}], 'title': 'Statistical and network-based analysis of Italian COVID-19 data: communities detection and temporal evolution', 'sha': 'ad32c1c00722c7079553a182e4810ad3d4881d1c', 'doi': '10.1101/2020.04.17.20068916', 'score': 9.333999633789062}, {'rank': 152, 'abstract': 'The coronavirus disease (COVID-19) outbreak started in Wuhan, China, and it has rapidly spread across the world. Italy is one of the European countries most affected by COVID-19, and it has registered high COVID-19 death rates and the death toll. In this article, we analyzed different Italian COVID-19 data at the regional level for the period 24 February to 29 March 2020. The analysis pipeline includes the following steps. After individuating groups of similar or dissimilar regions with respect to the ten types of available COVID-19 data using statistical test, we built several similarity matrices. Then, we mapped those similarity matrices into networks where nodes represent Italian regions and edges represent similarity relationships (edge length is inversely proportional to similarity). Then, network-based analysis was performed mainly discovering communities of regions that show similar behavior. In particular, network-based analysis was performed by running several community detection algorithms on those networks and by underlying communities of regions that show similar behavior. The network-based analysis of Italian COVID-19 data is able to elegantly show how regions form communities, i.e., how they join and leave them, along time and how community consistency changes along time and with respect to the different available data.', 'body_text': [{'text': 'Coronavirus disease, known as COVID-19, emerged in the city of Wuhan, in China, in November 2019 [1].', 'cite_spans': [{'start': 98, 'end': 99, 'mention': '1', 'ref_id': 'BIBREF0'}], 'section': '1. Introduction', 'ref_spans': []}, {'text': 'The disease is caused by the novel coronavirus Sars-CoV-2 [2] and its clinical manifestations include fever, cough, fatigue, chest distress, diarrhea, nausea, vomiting [3] and also acute respiratory distress syndrome in severe cases [4].', 'cite_spans': [{'start': 59, 'end': 60, 'mention': '2', 'ref_id': 'BIBREF9'}, {'start': 169, 'end': 170, 'mention': '3', 'ref_id': 'BIBREF10'}, {'start': 234, 'end': 235, 'mention': '4', 'ref_id': 'BIBREF11'}], 'section': '1. Introduction', 'ref_spans': []}, {'text': 'COVID-19 is characterized by a long incubation period, high infectivity, and different transmission methods [5]. The contagion happens mainly through respiratory and blood contact with the coronavirus.', 'cite_spans': [{'start': 109, 'end': 110, 'mention': '5', 'ref_id': 'BIBREF12'}], 'section': '1. Introduction', 'ref_spans': []}, {'text': 'In a few months, COVID-19 epidemic quickly spread to Asian countries and it reached more than 200 countries in the world, causing tens of thousands of deaths.', 'cite_spans': [], 'section': '1. Introduction', 'ref_spans': []}, {'text': 'On 11 March 2020, COVID-19 disease was declared a pandemic by the World Health Organization.', 'cite_spans': [], 'section': '1. Introduction', 'ref_spans': []}, {'text': 'In Italy, COVID-19 was identified in January 2020 [6] and the outbreak started in Lombardi and Veneto at the end of February 2020. From the northern regions of Italy, the disease spread very quickly to the nearest regions and then to the rest ones. Italy was considered one of the main epicenters of the pandemic, with 97,689 infections and 10,799 deaths up to 29th of March. The aim of this study is to provide a graph-based representation of daily data provided by Italian Civil Protection that enables evaluation of which regions show similar behavior and discovery of communities. The data refers to the period 24 February to 29 March 2020. To do this, we designed an analysis pipeline to model Italian COVID-19 data as networks and to perform network-based analysis. At first, for each type of data, we evaluated the similarity among a pair of regions by using statistical tests, and accordingly, we built ten similarity matrices (one for each Italian COVID-19 datum). After that, we mapped the similarity matrices into networks where the nodes represent the Italian region, and the edges connect statistically similar regions. Finally, we evaluated how the networks evolved over the weeks by analyzing the networks at different time points: (i) over the period 24 February to 29 March 2020 (study period); and (ii) in single weeks. Then, network-based analysis was performed mainly to discover communities of regions that show similar behavior. The main contribution of the paper is a network-based representation of COVID-19 diffusion similarity among regions and graph-based visualization to underline similar diffusion regions.', 'cite_spans': [{'start': 51, 'end': 52, 'mention': '6', 'ref_id': 'BIBREF13'}], 'section': '1. Introduction', 'ref_spans': []}, {'text': 'The rest of the paper is organized as follows: Section 1 presents the pipeline to analyze Italian COVID-19 data, Section 2 presents the application of our methodology on Italian COVID-19 data, and Section 3 discusses the results. Finally, Section 4 concludes the paper.', 'cite_spans': [], 'section': '1. Introduction', 'ref_spans': []}, {'text': 'We designed an analysis pipeline with the goal of investigating similarity among Italian regions with respect to data provided by Italian Civil Protection and to identify clusters of regions with similar behavior.', 'cite_spans': [], 'section': '2. Analysis\\xa0Pipeline', 'ref_spans': []}, {'text': 'The analysis pipeline includes the following steps:Building of a similarity matrix. The first step consists of the building of a similarity matrix that records the similarity among a pair of regions with respect to an Italian COVID-19 data measure. The similarity is computed by applying a statistical test. We decided to use the Wilcoxon Sum Rank Test. Therefore, the (i, j) value of the matrix for data k (e.g., swab data) represents the p-value of the Wilcoxon statistical test obtained by performing the test on the swab measures of region i with respect to region j. Lower p-value means that regions are more dissimilar with respect to that measure. Higher p-value means that regions are more similar with respect to that measure. We used the usual significance threshold of 0.05, thus matrices report only p-vales ≥ 0.05, while p-values < 0.05 are mapped to zero.Mapping similarity matrices to networks. The second step consists of the building networks starting from the similarity matrices. We map each matrix M(i, j) to a network N, where nodes represent the Italian regions and an edge connects two regions (i, j) if the p-value in the similarity matrix is greater than the significance threshold of 0.05. edges are weighted with the p-value.Temporal analysis of networks. The third step consists of the building of the network at different time intervals. Assuming that the analyzed data presents a temporal evolution, for each one, the corresponding networks at different time points (i.e., at the end of week 1,2, …, 5) and for an study period are built.Community detection. The fourth step consists of the extraction of community on the network by applying an appropriate community detection algorithm. For each network, we extracted subgroups of regions that form a community based on similarity of point of view. The identification of community is performed on the networks related to the study period and for all single week. Then, we extract the communities at different time points, i.e., at the end of the first week, after three weeks, and after five weeks (the study period).', 'cite_spans': [], 'section': '2. Analysis\\xa0Pipeline', 'ref_spans': []}, {'text': 'The present analysis was carried out on the dataset of COVID-19 updated at the https://github.com/pcm-dpc/COVID-19 database, provided by Italian Civil Protection. The dataset consists of:Hospitalized with Symptoms, the numbers of hospitalized patients that present COVID-19 symptoms;Intensive Care, the numbers of hospitalized patients in Intensive Care Units;Total Hospitalized, the total numbers of hospitalized patients;Home Isolation, the numbers of subjects that are in isolation at home;Total Currently Positive, the numbers of subjects that are coronavirus positive;New Currently Positive, the numbers of subjects that are daily coronavirus positive;Discharged/Healed the numbers of subjects that are healed from the disease;Deceased, the numbers of dead patients;Total Cases, the numbers of subjects affected by COVID-19;Swabs, the numbers of test swab carried on positive subjects and on subjects with suspected positivity.', 'cite_spans': [], 'section': '3.1. Input\\xa0DataSet ::: 3. Results', 'ref_spans': []}, {'text': 'The data is daily provided for each Italian region. The data occupies 47.6 Mbytes of memory.', 'cite_spans': [], 'section': '3.1. Input\\xa0DataSet ::: 3. Results', 'ref_spans': []}, {'text': 'As an initial step, we used the Wilcoxon Sum Rank test to carry out an analysis within the same type of data for all weeks and then, for each single week. The Wilcoxon test is a non-parametric test designed to evaluate the difference between two treatments or conditions where the samples are correlated. We applied the Wilcoxon test to perform a pair-wise comparison among regions with the goal of highlighting statistically similar distributions among them. For this reason, we built a similarity matrix for each couple of regions, for each of the available COVID-19 data. Figure 2 reports the heat map of similarity value related to Hospitalized with Symptoms network for all regions in the study period. We reported the heat maps for Italian COVID-19 data in the study period in Appendix A, for the lack of space. In addition, we report the Tables of the similarity values computed for Italian COVID-19 data in the study period and in the single weeks in Appendix A.', 'cite_spans': [], 'section': '3.2.1. Wilcoxon Sum Rank Test ::: 3.2. Building of Similarity\\xa0Matrices ::: 3. Results', 'ref_spans': [{'start': 575, 'end': 583, 'mention': 'Figure 2', 'ref_id': 'FIGREF1'}]}, {'text': 'Results show that according to the type of data, a significant difference exists (p-value less than 0.05) among some regions while for others, it is possible to highlight statistically similar distributions. Also, the significance varies by performing the analysis on whole selected time interval and on single week.', 'cite_spans': [], 'section': '3.2.1. Wilcoxon Sum Rank Test ::: 3.2. Building of Similarity\\xa0Matrices ::: 3. Results', 'ref_spans': []}, {'text': 'After that, we used the Kruskal–Wallis test performing an analysis on the same type of data for all regions (i.e., carrying out multiple comparisons) for the study period and then, for each single week. The Kruskal–Wallis test is a non-parametric method for analysis of variance used to determine if more samples originate from the same distribution. The results confirmed a significant difference considering all regions on the same type of data for the study period for every single week.', 'cite_spans': [], 'section': '3.2.2. Kruskal–Wallis Sum Rank Test ::: 3.2. Building of Similarity\\xa0Matrices ::: 3. Results', 'ref_spans': []}, {'text': 'Furthermore, we performed multiple linear regression by considering nine indicators: Hospitalized with Symptoms, Intensive Care, Home Isolation, Total Currently Positive, Discharged/Healed, Deceased, Total Cases, Swabs and two geographic factors: population density and number of intensive care beds for regions. We perform a standardization of variables as a preprocessing step. Data related to population density and intensive care beds for regions are reported in Table 2. According to multiple linear regression, we built nine models for each piece of Italian COVID-19 data in order to evaluate an outcome of each indicator on the basis of multiple distinct predictor variables i.e., Population Density and Intensive Care Beds. Table 3 reports the p-values associated with the Population Density and Intensive Care Beds and the Multiple R-squared. It is possible to notice that the intensive care beds variable is significantly related to Hospitalized with Symptoms, Intensive Care Home Isolation, Total Currently Positive, Discharged/Healed, Deceased, Total Cases variables with Multiple R-squared greater than 0.5. Instead, population density is significantly related to the Swabs variable. In this case, Multiple R-squared is equal to 0.318 (i.e., 32% of the data is explained by the explanatory variable). These results demonstrate that the population density does not influence Hospitalized with Symptoms, Intensive Care Home Isolation, Total Currently Positive, Discharged/Healed, Deceased, Total Cases variables which can be affected by other factors such as smog or climate as reported in [8,9].', 'cite_spans': [{'start': 1602, 'end': 1603, 'mention': '8', 'ref_id': 'BIBREF15'}, {'start': 1604, 'end': 1605, 'mention': '9', 'ref_id': 'BIBREF16'}], 'section': '3.2.3. Multiple Linear Regression ::: 3.2. Building of Similarity\\xa0Matrices ::: 3. Results', 'ref_spans': [{'start': 467, 'end': 474, 'mention': 'Table 2', 'ref_id': 'TABREF1'}, {'start': 732, 'end': 739, 'mention': 'Table 3', 'ref_id': 'TABREF2'}]}, {'text': 'To evaluate the evolution of Italian COVID-19 data and evidence which regions show similar behavior, we built networks of each piece of data [10] starting from the result of Wilcoxon test. The nodes of the networks are the Italian regions and the edges link two regions (nodes) with similar trend according to significance level (p-value > 0.05) obtained from the Wilcoxon test, otherwise (p-value < 0.05) there is not connection among nodes. The network analysis is performed using the igraph libraries [11].', 'cite_spans': [{'start': 142, 'end': 144, 'mention': '10', 'ref_id': 'BIBREF1'}, {'start': 505, 'end': 507, 'mention': '11', 'ref_id': 'BIBREF2'}], 'section': '3.3. Mapping Similarity Matrices to\\xa0Networks ::: 3. Results', 'ref_spans': []}, {'text': 'At first, we built ten networks, one for each data (Hospitalized with Symptoms, Intensive Care data, Total Hospitalized, Home Isolation, Total Currently Positive, New Currently Positive, Discharged/Healed, Deceased, Total Cases, Swab) by considering the period 24 February to 29 March. Then, we built the same networks by considering single weeks. The ten networks for the study period and for five weeks are reported in Figure 3, Figure 4, Figure 5, Figure 6, Figure 7, Figure 8, Figure 9, Figure 10, Figure 11 and Figure 12.', 'cite_spans': [], 'section': '3.3. Mapping Similarity Matrices to\\xa0Networks ::: 3. Results', 'ref_spans': [{'start': 421, 'end': 429, 'mention': 'Figure 3', 'ref_id': 'FIGREF2'}, {'start': 431, 'end': 439, 'mention': 'Figure 4', 'ref_id': 'FIGREF3'}, {'start': 441, 'end': 449, 'mention': 'Figure 5', 'ref_id': 'FIGREF4'}, {'start': 451, 'end': 459, 'mention': 'Figure 6', 'ref_id': 'FIGREF5'}, {'start': 461, 'end': 469, 'mention': 'Figure 7', 'ref_id': 'FIGREF6'}, {'start': 471, 'end': 479, 'mention': 'Figure 8', 'ref_id': 'FIGREF7'}, {'start': 481, 'end': 489, 'mention': 'Figure 9', 'ref_id': 'FIGREF8'}, {'start': 491, 'end': 500, 'mention': 'Figure 10', 'ref_id': 'FIGREF9'}, {'start': 502, 'end': 511, 'mention': 'Figure 11', 'ref_id': 'FIGREF10'}, {'start': 516, 'end': 525, 'mention': 'Figure 12', 'ref_id': 'FIGREF11'}]}, {'text': 'Starting from the ten networks related to all five weeks, we wanted to identify which regions form a community from the similarity point of view. For this, we applied the Walktrap community-finding algorithm [12] that identifies densely connected subgraphs, i.e., communities, in a graph via random walks. The idea is that short random walks tend to stay in the same community.', 'cite_spans': [{'start': 209, 'end': 211, 'mention': '12', 'ref_id': 'BIBREF3'}], 'section': '3.4. Community\\xa0Detection ::: 3. Results', 'ref_spans': []}, {'text': 'The extracted communities from all Italian COVID-19 networks in the study period are reported in Figure 13.', 'cite_spans': [], 'section': '3.4. Community\\xa0Detection ::: 3. Results', 'ref_spans': [{'start': 97, 'end': 106, 'mention': 'Figure 13', 'ref_id': 'FIGREF12'}]}, {'text': 'Figure 3 shows the evolution of Hospitalized with Symptoms network over five weeks. Figure a reports the network that represents the behavior of regions respect to number of hospitalized patients up to 29 March, whereas Figure 3b–f represent the networks on each single week. It is possible to notice that the network structure changes according to the analyzed time interval. At the end of the 35th day, the network has all nodes connected with exception of a single node that represents the Basilicata region. Furthermore, it is possible to highlight different community structures consisting of groups of regions with similar trends. Figure 13a reports five communities: the first one consists of Basilicata; the second one is composed by Piemonte, Marche, Emilia, Lombardi, Veneto; the third one is composed by Liguria, Lazio, Toscana; the fourth one is formed by Campania, Puglia, Sicilia, Abruzzo, Valle d’Aosta, Friuli, Trento, Bolzano; the last one is composed by Umbria, Sardegna, Calabria, Molise.', 'cite_spans': [], 'section': '4. Discussion', 'ref_spans': [{'start': 0, 'end': 8, 'mention': 'Figure 3', 'ref_id': 'FIGREF2'}, {'start': 220, 'end': 228, 'mention': 'Figure 3', 'ref_id': 'FIGREF2'}, {'start': 637, 'end': 646, 'mention': 'Figure 13', 'ref_id': 'FIGREF12'}]}, {'text': 'Figure 4 represents the evolution of the Intensive Care network. It is possible to notice that Lombardi and Veneto, which are the regions most affected by coronavirus disease with the highest numbers hospitalized in Intensive Care Units, are disconnected in the first week. In the second week, Lombardi and Veneto are connected by an edge that represents a level of similarity, whereas in the fifth week Veneto is linked with Emilia and Lombardi and this group of regions becomes a disconnected component among other regions. Thus, while initially Lombardi and Veneto showed a similar trend of Intensive Care data, after Veneto moved far from Lombardi trend. Furthermore, by analyzing the communities in the Intensive Care network (Figure 13b), it is possible to demonstrate four subgraphs formed by (i) Lombardi and Veneto, (ii) Umbria and Lazio, (iii) Marche, Emilia, Piemonte, Toscana and (iv) a large module formed by Campania, Sicilia, Sardegna, Abruzzo, Umbria, Calabria, Basilicata, Bolzano, Valle d’Aosta, Friuli, Trento, Molise.', 'cite_spans': [], 'section': '4. Discussion', 'ref_spans': [{'start': 0, 'end': 8, 'mention': 'Figure 4', 'ref_id': 'FIGREF3'}, {'start': 732, 'end': 741, 'mention': 'Figure 13', 'ref_id': 'FIGREF12'}]}, {'text': 'Figure 5 shows the Total Hospitalized network evolution. Starting form the first week, the structure of the network has two disconnected nodes representing Lombardi and Veneto, whereas, in the second week, the network evolves by presenting a single disconnected node that represent Emilia, and two connected nodes, Lombardi and Veneto, which in turn are disconnected from dense subgraph. In the third and fourth weeks, the network structure presents all connected components, and a high number of nodes are disconnected in the fifth week. Finally, all regions are connected in the final network. By analyzing the communities detected in Total Hospitalized network (Figure 13c), it is possible to notice a similarity with respect to those extracted by Hospitalized with Symptoms networks. In fact, there is a correspondence among three communities: (i) Basilicata and Piemonte, (ii) Marche, Emilia, Lombardi, (iii) Veneto and Liguria, Lazio, Toscana. This means that those regions that form the three communities present the same behavior according to the number of the hospitalized patients with symptoms and the total number of hospitalized patients.', 'cite_spans': [], 'section': '4. Discussion', 'ref_spans': [{'start': 0, 'end': 8, 'mention': 'Figure 5', 'ref_id': 'FIGREF4'}, {'start': 665, 'end': 674, 'mention': 'Figure 13', 'ref_id': 'FIGREF12'}]}, {'text': 'Figure 6 shows the evolution of the Home Isolation network over five weeks. In the first week, Lombardi, Veneto and Emilia each shows a different trend of the number of subjects in isolation at home, both between them and compared to other regions. Next, in the network formed by considering the all weeks, Lombardi, Veneto, Emilia and Marche formed a subgraph disconnected by a different dense subgraph composed by the rest of the regions. However, Veneto represents a single community in Figure 13d. This means that the behavior of Veneto presents a low similarity with respect to Lombardi, Emilia and Marche despite forming a module.', 'cite_spans': [], 'section': '4. Discussion', 'ref_spans': [{'start': 0, 'end': 8, 'mention': 'Figure 6', 'ref_id': 'FIGREF5'}, {'start': 490, 'end': 499, 'mention': 'Figure 13', 'ref_id': 'FIGREF12'}]}, {'text': 'Figure 7 and Figure 8 show the Total Currently Positive network and New Currently Positive network. Both networks evolve over five weeks by forming a structure in which all nodes are connected. According to the extracted communities, four modules are identified in Total Currently Positive network (see Figure 13e) and they are formed by: (i) Piemonte; (ii) Lombardi, Veneto, Marche, Emilia, (iii) Basilicata, Molise, Calabria, Sardegna, (iv) Puglia, Friuli, Valle d’Aosta, Toscana, Lazio, Abruzzo, Umbria, Campania, Trento, Liguria Bolzano, Sicilia. The communities identified in New Currently Positive network in Figure 13f are: (i) Piemonte, Marche, Toscana; (ii) Lombardi, Veneto, Emilia, (iii) Basilicata, Molise, (iv) Puglia, Friuli, Valle d’Aosta, Lazio, Abruzzo, Umbria, Campania, Trento, Liguria Bolzano, Calabria, Sardegna, Sicilia.', 'cite_spans': [], 'section': '4. Discussion', 'ref_spans': [{'start': 0, 'end': 8, 'mention': 'Figure 7', 'ref_id': 'FIGREF6'}, {'start': 13, 'end': 21, 'mention': 'Figure 8', 'ref_id': 'FIGREF7'}, {'start': 303, 'end': 312, 'mention': 'Figure 13', 'ref_id': 'FIGREF12'}, {'start': 615, 'end': 624, 'mention': 'Figure 13', 'ref_id': 'FIGREF12'}]}, {'text': 'It is possible to notice that Lombardi, Veneto, Emilia form a community in both Total Currently Positive network and New Currently Positive network, Piemonte represents a single community in the Total Currently Positive network, while Piemonte is associated with Marche and Toscana in the New Currently Positive network.', 'cite_spans': [], 'section': '4. Discussion', 'ref_spans': []}, {'text': 'Figure 9 represents the Discharged/Healed network over five weeks. In the first week, the network structure presents all nodes connected except for Veneto. In the second week, the Discharged/Healed network is formed by three subgraphs; in the third and fourth weeks the network is very dense; in the fifth week, the network structure is characterized by different disconnected components, and finally, at the end of 35 days the network is composed by a subgraph composed by Lombardi and Veneto and another subgraph highly connected. This means that Lombardi and Veneto have a similar behavior that is different from the rest of the Italian regions. Also, Lombardi and Veneto represent one of the five communities extracted by Discharged/Healed network. The extracted communities are reported in Figure 13g.', 'cite_spans': [], 'section': '4. Discussion', 'ref_spans': [{'start': 0, 'end': 8, 'mention': 'Figure 9', 'ref_id': 'FIGREF8'}, {'start': 795, 'end': 804, 'mention': 'Figure 13', 'ref_id': 'FIGREF12'}]}, {'text': 'Figure 10 shows the evolution of Deceased network. The evolution of this network is different from other Italian COVID-19 networks. In fact, in the first week all nodes are disconnected, so all Italian regions present different trends. In the second week, it is possible to notice that Emilia and Marche nodes are disconnected and there is a subgraph composed by Lombardi and Veneto and then there is a large subgraph formed by other regions. In the third week, all nodes present connections. In the fourth and fifth week the Deceased network presents different disconnected components; then, the final network shows a single disconnected node that represents Basilicata. Also, the Basilicata represents a single community of Deceased network; see Figure 13h. The other extracted communities are: (i) Piemonte, Toscana Liguria, Lazio, Friuli, Puglia, Valle d’Aosta, (ii) Lombardi, Veneto, Emilia, Marche, (iii) Sicilia, Molise, Abruzzo, Umbria, Campania, Trento, Bolzano, Calabria, Sardegna.', 'cite_spans': [], 'section': '4. Discussion', 'ref_spans': [{'start': 0, 'end': 9, 'mention': 'Figure 10', 'ref_id': 'FIGREF9'}, {'start': 748, 'end': 757, 'mention': 'Figure 13', 'ref_id': 'FIGREF12'}]}, {'text': 'Figure 11 represents the Total Cases network over five weeks. The final network demonstrates that the Italian regions present a significant level of similarity respect to the number of total coronavirus cases because all nodes are connected. Figure 13i shows the communities identified in Total Cases. The first community is composed by Lombardi, Veneto, Emilia, Marche; the second community is composed by Piemonte; the third community is composed by Basilicata, Molise; the fourth community is composed by Toscana Liguria, Lazio, Campania, Friuli, Sicilia Puglia, Valle d’Aosta formed, whereas Abruzzo, Umbria, Trento, Bolzano, Calabria, Sardegna formed the fifth community.', 'cite_spans': [], 'section': '4. Discussion', 'ref_spans': [{'start': 0, 'end': 9, 'mention': 'Figure 11', 'ref_id': 'FIGREF10'}, {'start': 242, 'end': 251, 'mention': 'Figure 13', 'ref_id': 'FIGREF12'}]}, {'text': 'Figure 12 shows the evolution of the Swab Network that represents the number of performed swab tests. The network, in the first week, shows Lombardi and Veneto nodes disconnected by other regions. In fact, these ones are the Italian regions that initially performed high number of test swabs. Also, the Veneto region has no connections in the final network and this reflects the policy of Veneto to carry out swab tests on asymptomatic subjects, i.e., it is an outlier with respect to other regions. Figure 13j shows the extracted communities in the Swab network. The first community is composed by Veneto; the second community is composed by Lombardi, Emilia; the third community is composed by Basilicata, Molise; the fourth community is formed by Marche, Toscana, Lazio, Piemonte, Friuli, Valle d’Aosta; the fifth community is formed by Sicilia, Campania, Liguria, Puglia; while Abruzzo, Umbria, Trento, Bolzano, Calabria, Sardegna formed the sixth community.', 'cite_spans': [], 'section': '4. Discussion', 'ref_spans': [{'start': 0, 'end': 9, 'mention': 'Figure 12', 'ref_id': 'FIGREF11'}, {'start': 500, 'end': 509, 'mention': 'Figure 13', 'ref_id': 'FIGREF12'}]}, {'text': 'We want to evaluate: (1) if different data present similar or dissimilar communities and (2) if the communities are similar or dissimilar considering different temporal interval on the same data. The Figure 14, Figure 15, Figure 16, Figure 17, Figure 18, Figure 19, Figure 20, Figure 21, Figure 22 and Figure 23 report the evolution of the communities related to different data.', 'cite_spans': [], 'section': '4. Discussion', 'ref_spans': [{'start': 200, 'end': 209, 'mention': 'Figure 14', 'ref_id': 'FIGREF13'}, {'start': 211, 'end': 220, 'mention': 'Figure 15', 'ref_id': 'FIGREF14'}, {'start': 222, 'end': 231, 'mention': 'Figure 16', 'ref_id': 'FIGREF15'}, {'start': 233, 'end': 242, 'mention': 'Figure 17', 'ref_id': 'FIGREF16'}, {'start': 244, 'end': 253, 'mention': 'Figure 18', 'ref_id': 'FIGREF17'}, {'start': 255, 'end': 264, 'mention': 'Figure 19', 'ref_id': 'FIGREF18'}, {'start': 266, 'end': 275, 'mention': 'Figure 20', 'ref_id': 'FIGREF19'}, {'start': 277, 'end': 286, 'mention': 'Figure 21', 'ref_id': 'FIGREF20'}, {'start': 288, 'end': 297, 'mention': 'Figure 22', 'ref_id': 'FIGREF21'}, {'start': 302, 'end': 311, 'mention': 'Figure 23', 'ref_id': 'FIGREF22'}]}, {'text': 'Figure 14 reports the evolution of Hospitalized with Symptoms Network Communities.', 'cite_spans': [], 'section': '4. Discussion', 'ref_spans': [{'start': 0, 'end': 9, 'mention': 'Figure 14', 'ref_id': 'FIGREF13'}]}, {'text': 'Figure 14a reports six communities extracted in the first week: (i) Lombardi, (ii) Veneto, (iii) Emilia, Marche, (iv) Liguria, Toscana, Piemonte, (v) Puglia, Lazio, Campania, Abruzzo, Bolzano, Sicilia, (vi) Umbria, Sardegna, Calabria, Molise, Valle d’Aosta, Friuli, Basilicata, Trento.', 'cite_spans': [], 'section': '4. Discussion', 'ref_spans': [{'start': 0, 'end': 9, 'mention': 'Figure 14', 'ref_id': 'FIGREF13'}]}, {'text': 'At the end of three weeks, Veneto, after representing a community in the previous week, moves in another community, whereas Emilia leaves the community with Marche and becomes a single community. Also, some regions migrate from fifth and sixth communities to other communities. Therefore, Figure 14b reports the five extracted communities after three weeks: (i) Lombardi, (ii) Emilia, (iii)Veneto, Marche, Piemonte, Liguria, Toscana, Lazio, (iv) Trento, Bolzano, Abruzzo, Friuli, Sicilia, Puglia, (v) Campania, Umbria, Sardegna, Calabria, Molise, Valle d’Aosta, Basilicata. Finally, Figure 14c reports five communities in the study period: the first one consists of Basilicata, which leaves the previous community and becomes a single one; the second one is composed by Piemonte, Marche, Emilia, Lombardi, Veneto; the third one is composed by Liguria, Lazio, Toscana; the fourth one is formed by Campania, Puglia, Sicilia, Abruzzo, Valle d’Aosta, Friuli, Trento, Bolzano; the last one is composed by Umbria, Sardegna, Calabria, Molise.', 'cite_spans': [], 'section': '4. Discussion', 'ref_spans': [{'start': 289, 'end': 298, 'mention': 'Figure 14', 'ref_id': 'FIGREF13'}, {'start': 583, 'end': 592, 'mention': 'Figure 14', 'ref_id': 'FIGREF13'}]}, {'text': 'Figure 15 reports the evolution of Intensive Care Network Communities. It is possible to notice that in the first week (Figure 15a), there is: one large community formed by Umbria, Lazio, Piemonte, Toscana Campania, Sicilia, Sardegna, Abruzzo, Umbria, Calabria, Basilicata, Bolzano, Valle d’Aosta, Friuli, Trento, Molise; two single communities formed by Lombardi and Emilia; and a small community formed by Emilia and Marche.', 'cite_spans': [], 'section': '4. Discussion', 'ref_spans': [{'start': 0, 'end': 9, 'mention': 'Figure 15', 'ref_id': 'FIGREF14'}, {'start': 120, 'end': 129, 'mention': 'Figure 15', 'ref_id': 'FIGREF14'}]}, {'text': 'After three weeks the number of extracted communities increases; see Figure 15b. In fact, Lombardi, Sardegna, Valle d’Aosta represent three single communities, Veneto and Emilia form a community, as well as, Marche and Piemonte. Then, Liguria, Lazio and Toscana form a six community, and the last two are composed by (i) Umbria, Campania, Molise, Abruzzo, Friuli, Trento, Puglia and (ii) Calabria, Sicilia, Bolzano, Basilicata.', 'cite_spans': [], 'section': '4. Discussion', 'ref_spans': [{'start': 69, 'end': 78, 'mention': 'Figure 15', 'ref_id': 'FIGREF14'}]}, {'text': 'Finally, in the study period, five communities are mined (see Figure 15c), formed by (i) Lombardi and Veneto, (ii) Liguria and Lazio, (iii) Marche, Emilia, Piemonte, Toscana and (iv) a large module formed by Campania, Sicilia, Sardegna, Abruzzo, Umbria, Calabria, Basilicata, Bolzano, Valle d’Aosta, Friuli, Trento, Molise.', 'cite_spans': [], 'section': '4. Discussion', 'ref_spans': [{'start': 62, 'end': 71, 'mention': 'Figure 15', 'ref_id': 'FIGREF14'}]}, {'text': 'Figure 16 reports the evolution of Total Hospitalized Network Communities.', 'cite_spans': [], 'section': '4. Discussion', 'ref_spans': [{'start': 0, 'end': 9, 'mention': 'Figure 16', 'ref_id': 'FIGREF15'}]}, {'text': 'Figure 16a shows the six mined communities. The first community is composed by Liguria, Toscana, Piemonte, the second one is formed Sardegna, Umbria, Calabria, Basilicata, Valle d’Aosta, Friuli, Trento, Molise; the third module comprises Lazio, Campania, Sicilia, Abruzzo, Bolzano, Puglia; the fourth community is represented by Marche, Emilia; the fifth is represented by Lombardi and the last one consists of Veneto.', 'cite_spans': [], 'section': '4. Discussion', 'ref_spans': [{'start': 0, 'end': 9, 'mention': 'Figure 16', 'ref_id': 'FIGREF15'}]}, {'text': 'After three weeks (see Figure 16b) the regions move, with the exception of Lombardi, which continues to represent a single community and the communities become eight. In fact, Emilia becomes a single community; Veneto becomes a community among with Piemonte and Marche; Toscana moves in the community with Liguria; Lazio and Campania forms a new community, as well as, Basilicata and Valle d’Aosta; another community is formed by: Abruzzo, Puglia, Sicilia and the last one is formed by Friuli, Bolzano, Trento, Umbria, Sardegna, Calabria, Molise.', 'cite_spans': [], 'section': '4. Discussion', 'ref_spans': [{'start': 23, 'end': 32, 'mention': 'Figure 16', 'ref_id': 'FIGREF15'}]}, {'text': 'At the end of the study period, the five communities reported in Figure 16b are formed. The first one consists of Basilicata that leaves the previous community and becomes a single one; the second one is composed by Toscana, Liguria and Lazio; Sardegna, Calabria, Molise leaves the previous large community and form a smaller one; by the fourth one is formed by Piemonte, Marche, Emilia, Lombardi, Veneto; the last one is composed by Umbria, Puglia, Sicilia, Abruzzo, Valle d’Aosta, Friuli, Trento, Bolzano.', 'cite_spans': [], 'section': '4. Discussion', 'ref_spans': [{'start': 65, 'end': 74, 'mention': 'Figure 16', 'ref_id': 'FIGREF15'}]}, {'text': 'Figure 17 reports the evolution of Home Isolation Network Communities.', 'cite_spans': [], 'section': '4. Discussion', 'ref_spans': [{'start': 0, 'end': 9, 'mention': 'Figure 17', 'ref_id': 'FIGREF16'}]}, {'text': 'Figure 17a reports the mined communities at the end of first week. It is possible to notice that Lombardi, Veneto and Emilia form single communities, and then there are three large communities: the first one is represented by Piemonte, Liguria, Marche, Sicilia, Campania; the second one is composed by Puglia, Valle d’Aosta, Toscana, Umbria, Calabria; the third one Trento, Lazio, Abruzzo, Sardegna, Bolzano, Basilicata and Molise, Friuli, Campania.', 'cite_spans': [], 'section': '4. Discussion', 'ref_spans': [{'start': 0, 'end': 9, 'mention': 'Figure 17', 'ref_id': 'FIGREF16'}]}, {'text': 'At the end of three week, Lombardi, Veneto and Emilia move together to form a unique community, whereas, the other regions form new communities, such as (i) Puglia, Trento, Lazio, Umbria, (ii) Calabria, Abruzzo, Valle d’Aosta, Sardegna, Bolzano, Basilicata and Molise, (iii) Sicilia, Toscana, Friuli, Piemonte, Liguria, Marche, Campania (see Figure 17b).', 'cite_spans': [], 'section': '4. Discussion', 'ref_spans': [{'start': 342, 'end': 351, 'mention': 'Figure 17', 'ref_id': 'FIGREF16'}]}, {'text': 'Figure 17c shows the community topology in the study period. Veneto leaves the community among with Lombardi and Emilia and it becomes a single one, whereas, Lombardi, Emilia forms a new module among with Marche. Basilicata and Molise move together to form a unique community. Sardegna, Calabria, Abruzzo, Bolzano form a fourth community. The fifth community is composed by Puglia, Trento, Lazio, Umbria, Sicilia, and the sixth one is represented by Toscana, Piemonte, Valle d’Aosta, Friuli, Liguria, Campania.', 'cite_spans': [], 'section': '4. Discussion', 'ref_spans': [{'start': 0, 'end': 9, 'mention': 'Figure 17', 'ref_id': 'FIGREF16'}]}, {'text': 'Figure 18 reports the evolution of Total Currently Positive Communities. At the end of first week, there are eight mined communities, and they are reported in Figure 18a. The first community is composed by Umbria, Sardegna, Basilicata, Molise, Friuli Toscana, Calabria, Valle d’Aosta, Trento; the second one is formed by Bolzano, Lazio, Abruzzo, Puglia; the third module comprises Campania, Sicilia Liguria; Piemonte and Lazio represent the fourth community; the fifth one consists of Marche; the sixth community is represented by Emilia; the seventh is represented by Lombardi and the last one consists of Veneto.', 'cite_spans': [], 'section': '4. Discussion', 'ref_spans': [{'start': 0, 'end': 9, 'mention': 'Figure 18', 'ref_id': 'FIGREF17'}, {'start': 159, 'end': 168, 'mention': 'Figure 18', 'ref_id': 'FIGREF17'}]}, {'text': 'After three weeks, the number of communities (see Figure 18b) decreases. In fact, it is possible to notice five subgraphs. Emilia joints with Veneto and Lombardi remains single community. Lazio, Sicilia, Friuli, Puglia form a new community; Piemonte, Toscana, Campania, Marche, Liguria, represent a fourth community; Trento, Abruzzo, Umbria, Calabria, Sardegna, Basilicata, Molise, Bolzano, Campania, Valle d’Aosta, form a fifth community;', 'cite_spans': [], 'section': '4. Discussion', 'ref_spans': [{'start': 50, 'end': 59, 'mention': 'Figure 18', 'ref_id': 'FIGREF17'}]}, {'text': 'In the study period, the number of extracted communities further decreases, see Figure 18c The first community is composed by Veneto, Lombardi, Emilia, Marche; the second community is composed by Piemonte; the third community is composed by Basilicata, Molise, Calabria, Sardegna; the fourth community is formed by Toscana, Lazio, Friuli, Valle d’Aosta, Sicilia, Campania, Liguria Puglia, Abruzzo, Umbria, Trento, Bolzano.', 'cite_spans': [], 'section': '4. Discussion', 'ref_spans': [{'start': 80, 'end': 89, 'mention': 'Figure 18', 'ref_id': 'FIGREF17'}]}, {'text': 'Figure 19 reports the evolution of New Currently Positive Communities. Figure 19a reports the mined communities at the end of first week. It is possible to notice that Lombardi, Veneto and Emilia form single communities, and then there are two large communities: the first one is represented by Marche, Piemonte, Liguria, Campania, Abruzzo, Toscana; the second one is composed by Puglia, Valle d’Aosta, Umbria, Calabria, Sicilia, Campania, Trento, Lazio, Sardegna, Bolzano, Basilicata and Molise, Friuli.', 'cite_spans': [], 'section': '4. Discussion', 'ref_spans': [{'start': 0, 'end': 9, 'mention': 'Figure 19', 'ref_id': 'FIGREF18'}, {'start': 71, 'end': 80, 'mention': 'Figure 19', 'ref_id': 'FIGREF18'}]}, {'text': 'After three weeks, there remain five extracted communities but the regions that form them vary; see Figure 19b. The first community is composed by Lombardi; the second community is composed by Veneto and Emilia; the third community is composed by Basilicata, Molise, Valle d’Aosta, Sardegna, Campania, Bolzano; the fourth community is formed by Marche, Toscana, Piemonte; the fifth community is formed by Sicilia, Liguria, Puglia, Abruzzo, Umbria, Trento, Calabria, Lazio, Friuli.', 'cite_spans': [], 'section': '4. Discussion', 'ref_spans': [{'start': 100, 'end': 109, 'mention': 'Figure 19', 'ref_id': 'FIGREF18'}]}, {'text': 'In the study period, the number of extracted communities further decreases, see Figure 19c and there are: (i) Piemonte, Marche, Toscana; (ii) Lombardi, Veneto, Emilia, (iii) Basilicata, Molise, (iv) Puglia, Friuli, Valle d’Aosta, Lazio, Abruzzo, Umbria, Campania, Trento, Liguria Bolzano, Calabria, Sardegna, Sicilia.', 'cite_spans': [], 'section': '4. Discussion', 'ref_spans': [{'start': 80, 'end': 89, 'mention': 'Figure 19', 'ref_id': 'FIGREF18'}]}, {'text': 'Figure 20 reports the evolution of Discharged/Healed Network Communities. It is possible to notice that in the first week (Figure 20a), there are: a large community formed by Umbria, Piemonte, Toscana Campania, Sicilia, Sardegna, Abruzzo, Umbria, Calabria, Basilicata, Bolzano, Emilia, Valle d’Aosta, Friuli, Trento, Molise; a small community formed by Lombardi, Marche, Lazio; and single communities formed by Veneto.', 'cite_spans': [], 'section': '4. Discussion', 'ref_spans': [{'start': 0, 'end': 9, 'mention': 'Figure 20', 'ref_id': 'FIGREF19'}, {'start': 123, 'end': 132, 'mention': 'Figure 20', 'ref_id': 'FIGREF19'}]}, {'text': 'After three weeks the number of extracted communities increases; see Figure 20b. In fact, Lombardi leaves the previous community and becomes a single one; Lazio, Emilia, Liguria and Veneto get together to form a second community; the third is composed by Friuli, Sicilia, Toscana; the last one is formed by Sardegna, Valle d’Aosta, Marche and Piemonte, Umbria, Campania, Molise, Abruzzo, Trento, Puglia, Calabria, Bolzano, Basilicata.', 'cite_spans': [], 'section': '4. Discussion', 'ref_spans': [{'start': 69, 'end': 78, 'mention': 'Figure 20', 'ref_id': 'FIGREF19'}]}, {'text': 'Finally, Figure 20c shows the communities in the study period. It is possible to notice five communities. The first one consists of Lombardi and Veneto; the second one is composed by Emilia, Liguria, Lazio; the third one is composed by Friuli, Campania, Toscana, Sicilia; the fourth one is formed by, Puglia, Abruzzo, Trento; the last one is composed by Umbria, Sardegna, Calabria, Piemonte, Marche, Valle d’Aosta, Bolzano, Basilicata, Molise.', 'cite_spans': [], 'section': '4. Discussion', 'ref_spans': [{'start': 9, 'end': 18, 'mention': 'Figure 20', 'ref_id': 'FIGREF19'}]}, {'text': 'Figure 21 reports the evolution of Deceased Communities. Figure 21a reports the mined communities at the end of first week. It is possible to notice that there is: one large community formed by Emilia, Piemonte, Liguria, Campania, Abruzzo, Puglia, Valle d’Aosta, Umbria, Calabria, Sicilia, Campania, Trento, Lazio, Sardegna, Bolzano, Basilicata and Molise, Friuli; two single communities represented by Lombardi and Veneto; and a single community composed by Marche and Toscana.', 'cite_spans': [], 'section': '4. Discussion', 'ref_spans': [{'start': 0, 'end': 9, 'mention': 'Figure 21', 'ref_id': 'FIGREF20'}, {'start': 57, 'end': 66, 'mention': 'Figure 21', 'ref_id': 'FIGREF20'}]}, {'text': 'After three weeks, the number of extracted communities increases. In fact, the regions that form them vary by forming new communities; see Figure 21b. The first community is composed by Lombardi that remains a single community; the second community is composed by Veneto and the third one is composed by Emilia; the fourth community is formed by Basilicata, Molise, Sardegna, Calabria; the fifth community is formed by Sicilia, Liguria, Puglia, Abruzzo, Umbria, Trento, Lazio, Friuli, Valle d’Aosta, Campania, Bolzano; Marche, Toscana, Piemonte.', 'cite_spans': [], 'section': '4. Discussion', 'ref_spans': [{'start': 139, 'end': 148, 'mention': 'Figure 21', 'ref_id': 'FIGREF20'}]}, {'text': 'In the study period, the number of extracted communities decreases (see Figure 21c) and there are: (i) Basilicata represents a single community, (ii) Piemonte, Toscana Liguria, Lazio, Friuli, Puglia, Valle d’Aosta, (iii) Lombardi, Veneto, Emilia, Marche, (iv) Sicilia, Molise, Abruzzo, Umbria, Campania, Trento, Bolzano, Calabria, Sardegna.', 'cite_spans': [], 'section': '4. Discussion', 'ref_spans': [{'start': 72, 'end': 81, 'mention': 'Figure 21', 'ref_id': 'FIGREF20'}]}, {'text': 'Figure 22 reports the evolution of Total Cases Network Communities.', 'cite_spans': [], 'section': '4. Discussion', 'ref_spans': [{'start': 0, 'end': 9, 'mention': 'Figure 22', 'ref_id': 'FIGREF21'}]}, {'text': 'Figure 22a shows the five mined communities. The first community is composed by Liguria, Toscana Lazio, Piemonte, Campania, Sicilia; the second one is formed Sardegna, Abruzzo, Umbria, Calabria, Basilicata, Bolzano, Valle d’Aosta, Friuli, Trento, Molise, Puglia; the third module comprises Marche, Emilia; the forth one Lombardi and the last one Veneto.', 'cite_spans': [], 'section': '4. Discussion', 'ref_spans': [{'start': 0, 'end': 9, 'mention': 'Figure 22', 'ref_id': 'FIGREF21'}]}, {'text': 'After three weeks (see Figure 22b) there are six communities. Veneto becomes a community among with Emilia; Marche moves in the community with Liguria, Toscana Lazio, Piemonte, Campania; and three new modules are formed: the first one is composed by Sicilia, Friuli and Puglia, the second one is composed by Abruzzo, Bolzano, Trento and Umbria, the third one is formed by Basilicata, Sardegna, Valle d’Aosta, Calabria, Molise.', 'cite_spans': [], 'section': '4. Discussion', 'ref_spans': [{'start': 23, 'end': 32, 'mention': 'Figure 22', 'ref_id': 'FIGREF21'}]}, {'text': 'At the end of the study period, there are five communities extracted. Figure 22c reports the communities. The first community is composed by Lombardi, Veneto, Emilia, Marche; the second community is composed by Piemonte; the third community is composed by Basilicata, Molise; the fourth community is composed by Toscana Liguria, Lazio, Campania, Friuli, Sicilia Puglia, Valle d’Aosta formed a community, whereas Abruzzo, Umbria, Trento, Bolzano, Calabria, Sardegna formed the fifth community.', 'cite_spans': [], 'section': '4. Discussion', 'ref_spans': [{'start': 70, 'end': 79, 'mention': 'Figure 22', 'ref_id': 'FIGREF21'}]}, {'text': 'Figure 23 reports the evolution of the Swab Network Communities. At the end of first week, there are eight mined communities, and they are reported in Figure 23a. The first community is composed by Umbria, Calabria, Valle d’Aosta, Trento, Sicilia, Abruzzo, Puglia; the second one is formed by Sardegna, Basilicata, Molise, Bolzano; the third module comprises Friuli, Campania, Liguria; Piemonte represents the fourth community; the fifth one consists of Toscana and Lazio; the sixth community is represented by Emilia and Marche; the seventh is represented by Lombardi and the last one consists of Veneto.', 'cite_spans': [], 'section': '4. Discussion', 'ref_spans': [{'start': 0, 'end': 9, 'mention': 'Figure 23', 'ref_id': 'FIGREF22'}, {'start': 151, 'end': 160, 'mention': 'Figure 23', 'ref_id': 'FIGREF22'}]}, {'text': 'After three weeks, the number of communities (see Figure 23b) decreases. In fact, it is possible to notice six subgraphs. Emilia leaves the previous community and forms a single one. Lombardi and Veneto join together. Toscana and Lazio continue to form a community; Piemonte, Friuli, Campania, Marche, Puglia, Liguria, Sicilia form a fourth community; Trento, Abruzzo, Umbria, Calabria, form a fifth community and the last one is composed by Sardegna, Basilicata, Molise, Bolzano, Campania, Valle d’Aosta.', 'cite_spans': [], 'section': '4. Discussion', 'ref_spans': [{'start': 50, 'end': 59, 'mention': 'Figure 23', 'ref_id': 'FIGREF22'}]}, {'text': 'In the study period, there remain six extracted communities but the regions that form them vary; see Figure 23c. The first community is composed by Veneto; the second community is composed by Lombardi, Emilia; the third community is composed by Basilicata, Molise; the fourth community is formed by Marche, Toscana, Lazio, Piemonte, Friuli, Valle d’Aosta; the fifth community is formed by Sicilia, Campania, Liguria Puglia; while Abruzzo, Umbria, Trento, Bolzano, Calabria, Sardegna formed the sixth community.', 'cite_spans': [], 'section': '4. Discussion', 'ref_spans': [{'start': 101, 'end': 110, 'mention': 'Figure 23', 'ref_id': 'FIGREF22'}]}, {'text': 'By analyzing the results, it is possible to demonstrate that the topology of the communities varies, i.e., the regions join and leave them along time and the community consistency changes along time on the same data. For the communities related to the different available data, it is possible to notice that after the first week, the extracted communities are different. This changes, after analyzing the communities after three weeks. In fact, the Total Currently Positive Network Communities and New Currently Positive Network show similar communities as well as Deceased Network and Total Cases Network. Finally, after five weeks, the topology of communities is different for all Italian COVID-19 networks except for the Hospitalized with Symptoms Network and Total Hospitalized Network, which show similar extracted communities.', 'cite_spans': [], 'section': '4. Discussion', 'ref_spans': []}, {'text': 'In the literature, there are different works that apply graph theory to analyze the COVID-19 pandemic spread. For example, Reich et al. [13] modeled the COVID-19 spread by using a SEIR (Susceptible–Exposed–Infectious–Recovered–Susceptible) agent-based model on a graph, which takes into account several important real-life attributes of COVID-19: super-spreaders, realistic epidemiological parameters of the disease, testing, and quarantine policies. The agent is represented as a node in a graph, and infection between contacts is represented by graph edges. Then, the authors have applied the SEIR model to analyze the disease progression. Herrmann et al. [14] modeled the human interaction according to three different networks, i.e., Scale-free, Mitigation Hub, and Mitigation Random, and they applied the SIS (Susceptible–Infected–Susceptible) model. The authors demonstrated that network topology could improve the predictive power of SIR model of COVID-19 by providing novel insights into the potential strategies and policies for mitigating and suppressing the spread of the virus.', 'cite_spans': [{'start': 137, 'end': 139, 'mention': '13', 'ref_id': 'BIBREF4'}, {'start': 659, 'end': 661, 'mention': '14', 'ref_id': 'BIBREF5'}], 'section': '4. Discussion', 'ref_spans': []}, {'text': 'Kuzdeuov et al. [15] implemented a network-based stochastic epidemic simulator that models the movement of a disease through the SEIR states of a population. The nodes of the networks represent an administrative unit of the country, such as a city or region, and the edges between nodes represent transit links of roads railways, and air travel routes to model the mobility of inhabitants among cities. In [16], Kumar presents a network-based model for predicting the spread of COVID-19, incorporating human mobility through knowledge of migration and air transport.', 'cite_spans': [{'start': 17, 'end': 19, 'mention': '15', 'ref_id': 'BIBREF6'}, {'start': 407, 'end': 409, 'mention': '16', 'ref_id': 'BIBREF7'}], 'section': '4. Discussion', 'ref_spans': []}, {'text': 'The work of Wang et al. [17] applied statistical and network analysis on heterogeneous network containing patients and hospitals as nodes and relationships between relatives, friends or colleagues as edges. Network analysis provided important information about patients, hospitals and their relationships and it was able to provide a guidance for the distribution of epidemic prevention materials.', 'cite_spans': [{'start': 25, 'end': 27, 'mention': '17', 'ref_id': 'BIBREF8'}], 'section': '4. Discussion', 'ref_spans': []}, {'text': 'In summary, different works rely on network-based representation for the application of predictive models, whereas only Wang et al. [17] uses statistical and network-based analysis to evaluate an infected cluster of people in different hospitals. To the best of our knowledge, our work is the first study that provides a network-based representation and visualization of COVID-19 data at the regional level and applies network-based analysis to discover communities of regions that show similar behavior.', 'cite_spans': [{'start': 133, 'end': 135, 'mention': '17', 'ref_id': 'BIBREF8'}], 'section': '4. Discussion', 'ref_spans': []}, {'text': 'In conclusion, with this study, we wanted to give a graph-based representation of the COVID-19 measures considering how the regions behaved differently with respect to ten different datasets provided by Italian Civil Protection. It emerged that the regions where the epidemic had a greater impact, such as the Lombardi, Veneto, Piemonte and Emilia, had a different behavior with respect to other regions. This is evident in the community detection in which the regions most affected by the epidemic form individual communities or they are part of the same community. In addition, this study also led to identifying similar behaviors of regions that are geographically distant but that together form community. An example is represented by Calabria, Sardegna, and Molise that represent a cluster in Hospitalized with Symptoms Network, Total Hospitalized Network, Total Currently Positive Network, Discharged/Healed Network, Total Cases, Deceased Network, Intensive Care Network. This can lead the search for indicators that unite the regions such as factors, age structure, health care facilities, and socioeconomic status. Moreover, our visual representation of data can lead the search for indicators that are responsible for community formation i.e., factors common to regions such as age structure, health care facilities, and socioeconomic status. Furthermore, starting from the regions that form communities, it could be possible to plan common interventions such as the increase in intensive care units or the increase in swab tests.', 'cite_spans': [], 'section': '4. Discussion', 'ref_spans': []}, {'text': 'The COVID-19 disease has spread worldwide in a matter of weeks. In Italy, the epidemic of COVID-19 started in the north and quickly involved all regions. In this paper, we evaluated the evolution of Italian COVID-19 data provided daily by Italian Civil Protection. The main goal of this work is the network-based representation of COVID-19 diffusion similarity among regions and graph-based visualization with the aim of underlining similar diffusion regions. We identified similar Italian regions with respect to the available COVID-19 data and we mapped these in different networks. Finally, we performed a network-based analysis to discover communities of regions that show similar behavior. For future work, we plan to extend the study by considering the evolution of the communities at greater time intervals to demonstrate a new pattern of regions with respect to COVID-19 data.', 'cite_spans': [], 'section': '5. Conclusions', 'ref_spans': []}], 'paragraphs': [{'text': 'COVID-19 is characterized by a long incubation period, high infectivity, and different transmission methods [5]. The contagion happens mainly through respiratory and blood contact with the coronavirus.', 'id': '00003'}], 'title': 'Statistical and Network-Based Analysis of Italian COVID-19 Data: Communities Detection and Temporal Evolution', 'sha': 'ac6e8cb623609a60c72a6404f2a43ac799709a4e', 'doi': '10.3390/ijerph17124182', 'score': 9.333998680114746}, {'rank': 153, 'paragraphs': [{'text': 'Finally, an interactive real-time risk assessment application was developed to provide real-time updates of the risk assessment for symptomatic SARS-CoV-2 infections that would be undetected during active monitoring among active monitoring population with close contacts by setting several crucial parameters (Supplement Figure 10 ).', 'id': '00025'}], 'title': 'A systematic review and meta-analysis reveals long and dispersive incubation period of COVID-19', 'sha': '3d921c03dc23ee850655ddfd9d5ebe3956d70bf8', 'doi': '10.1101/2020.06.20.20134387', 'score': 9.33240032196045}, {'rank': 154, 'paragraphs': [{'text': 'The copyright holder for this preprint this version posted June 22, 2020. In addition, assuming a 6.15% risk of being infected among the general quarantine population, the risk of having infections develop symptoms after 14-day active monitoring or quarantine was estimated (G).', 'id': '00041'}], 'title': 'A systematic review and meta-analysis reveals long and dispersive incubation period of COVID-19', 'sha': '3d921c03dc23ee850655ddfd9d5ebe3956d70bf8', 'doi': '10.1101/2020.06.20.20134387', 'score': 9.332399368286133}, {'rank': 155, 'paragraphs': [{'text': 'Retained after screening of titles and abstracts . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'id': '00045'}], 'title': 'A systematic review and meta-analysis reveals long and dispersive incubation period of COVID-19', 'sha': '3d921c03dc23ee850655ddfd9d5ebe3956d70bf8', 'doi': '10.1101/2020.06.20.20134387', 'score': 9.332398414611816}, {'rank': 156, 'paragraphs': [{'text': 'The copyright holder for this preprint this version posted June 22, 2020. . https://doi.org/10.1101/2020.06.20.20134387 doi: medRxiv preprint CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'id': '00046'}], 'title': 'A systematic review and meta-analysis reveals long and dispersive incubation period of COVID-19', 'sha': '3d921c03dc23ee850655ddfd9d5ebe3956d70bf8', 'doi': '10.1101/2020.06.20.20134387', 'score': 9.3323974609375}, {'rank': 157, 'paragraphs': [{'text': 'The copyright holder for this preprint this version posted June 22, 2020. . https://doi.org/10.1101/2020.06.20.20134387 doi: medRxiv preprint is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'id': '00047'}], 'title': 'A systematic review and meta-analysis reveals long and dispersive incubation period of COVID-19', 'sha': '3d921c03dc23ee850655ddfd9d5ebe3956d70bf8', 'doi': '10.1101/2020.06.20.20134387', 'score': 9.332396507263184}, {'rank': 158, 'paragraphs': [{'text': 'Weekly alternating lock-down. Our mitigation begins at time t = t 0 , at which point the state of the system is given by S(t 0 ), E(t 0 ), I(t 0 ) and R(t 0 ). With this initial condition, we partition the population randomly into two equal groups 1 and 2, which are instructed to alternate in succession between a complete lock-down and regular activity. Individuals at the I state are instructed to always remain isolated, regardless of whether their group is in the active phase or on lock-down. The lock-down/activity periods are set to τ = 7 (days), i.e. weekly shifts between the groups. This time scale is selected in order to approximately resonate with the disease average incubation period of ∼ 5 days. Hence, if incubation is longer, one must set longer shifts. In our simulations we also allow for variability in incubation period, as captured by the distribution g(t), and quantified by the variance σ 2 .', 'id': '00016'}], 'title': 'Alternating quarantine for sustainable mitigation of COVID-19', 'sha': 'b923a8fc7d406d0f149ecd5293d1a72dec7ee526', 'doi': '', 'score': 9.327400207519531}, {'rank': 159, 'paragraphs': [{'text': 'All the information used in this article was approved by the patients.', 'id': '00004'}], 'title': 'COVID-19 in a patient with long-term use of glucocorticoids: A study of a familial cluster', 'sha': '912c00514457672748738d29efb81f975815cdf6', 'doi': '10.1016/j.clim.2020.108413', 'score': 9.32040023803711}, {'rank': 160, 'paragraphs': [{'text': 'We declare no competing interests.', 'id': '00015'}], 'title': 'COVID-19 in a patient with long-term use of glucocorticoids: A study of a familial cluster', 'sha': '912c00514457672748738d29efb81f975815cdf6', 'doi': '10.1016/j.clim.2020.108413', 'score': 9.320399284362793}, {'rank': 161, 'paragraphs': [{'text': 'Second, most studies obtained data from public resources, and the raw data were not provided; there is a possibility that some data were used repeatedly. Third, 50 of 56 studies were from mainland China; studies from other regions and countries are needed to explore the impact of viral evolution on variation of incubation period and other epidemiological characteristics. Fourth, precisely estimating the exposure window and time of symptom onset related to SARS-CoV-2 infection could be difficult in practice. Studies used different methods to quantify the uncertainty of incubation period for each individual, which may partially explain the non-negligible intra-study heterogeneity. Last, knowledge of the risk of being infected among close contacts is limited and may vary due to different definition of close contacts.', 'id': '00035'}], 'title': 'A systematic review and meta-analysis reveals long and dispersive incubation period of COVID-19', 'sha': '3d921c03dc23ee850655ddfd9d5ebe3956d70bf8', 'doi': '10.1101/2020.06.20.20134387', 'score': 9.309599876403809}, {'rank': 162, 'paragraphs': [{'text': 'This work was supported by the Emergency Project of Prevention and Control for COVID-19 of Central South University (Grant 16026000 to Jie Meng), and COVID-19 Project of Changsha Science and Technology Bureau (Grant 38065 to Jie Meng).', 'id': '00014'}], 'title': 'COVID-19 in a patient with long-term use of glucocorticoids: A study of a familial cluster', 'sha': '912c00514457672748738d29efb81f975815cdf6', 'doi': '10.1016/j.clim.2020.108413', 'score': 9.304200172424316}, {'rank': 163, 'paragraphs': [{'text': 'In summary, it is important to recognize potential patients who might have no or only mild symptoms. Therefore, we highly recommend that all people with an exposure history be strictly quarantined, and for immunocompromised people with long-term use of glucocorticoids, this time should not be limited to 14 days.', 'id': '00013'}], 'title': 'COVID-19 in a patient with long-term use of glucocorticoids: A study of a familial cluster', 'sha': '912c00514457672748738d29efb81f975815cdf6', 'doi': '10.1016/j.clim.2020.108413', 'score': 9.302300453186035}, {'rank': 164, 'paragraphs': [{'text': 'The collection of forward time depends on the follow-up time, that is, if the follow-up time is not long enough, we would only be able to include those with a shorter incubation period in the Wuhan departure cohort. This may lead to an underestimation of the incubation period. The same limitation also applies to Backer et al and Linton et al.5,6 However, as explained earlier, we only included cases who left Wuhan before January 23 in this study, which leaves an average follow-up time of 25 days. Hence it is less likely we missed those patients with longer incubation periods based on the largest incubation period of 24 days reported in Guan et al.4 Note that the 24-day incubation period was reported as an outlier in Guan et al.4We assume that the individuals included in our cohort were either infected in Wuhan or on the way to their destination from Wuhan, violation of such assumption lead to an overestimation of incubation period. The same limitation also applies to Backer et al and Linton et al.5,6 However, with a carefully selected cohort justified in the section of Method, the chance for an individual in the Wuhan departure cohort getting infected outside Wuhan should be relatively small. Nonetheless, we acknowledge this possibility exists, for example, a family member could be uninfected by the time of departing Wuhan but got infected by other family members or outside contacts after leaving Wuhan. A sensitivity analysis was also conducted by removing all cases who left Wuhan with their families in the Wuhan departure cohort, and we found it only resulted in a small change of the estimated distribution of the incubation period.Individuals in our selected cohort were those who got infected in the early days of the outbreak. They were likely the first- or second-generation cases. Our results do not apply to higher generation cases if the virus mutates.', 'id': '00024'}], 'title': 'Estimation of incubation period distribution of COVID-19 using disease onset forward time: a novel cross-sectional and forward follow-up study', 'sha': 'c61be9fc4562d250e2dec241ef75bd22d61f61d1; 6d3b3f4ab80a61c45f82c61c6c756cfc6ddf4bf2', 'doi': '10.1101/2020.03.06.20032417', 'score': 9.291600227355957}, {'rank': 165, 'paragraphs': [{'text': 'The authors have declared that no conflict of interest exists. All authors have completed the ICMJE uniform disclosure form at www.icmje.org/coi_disclosure.pdf and declare: no support from any organization for the submitted work; no financial relationships with any organizations that might have an interest in the submitted work in the previous three years; no other relationships or activities that could appear to have influenced the submitted work.', 'id': '00038'}], 'title': 'A systematic review and meta-analysis reveals long and dispersive incubation period of COVID-19', 'sha': '3d921c03dc23ee850655ddfd9d5ebe3956d70bf8', 'doi': '10.1101/2020.06.20.20134387', 'score': 9.289400100708008}, {'rank': 166, 'paragraphs': [{'text': ' attributed their finding of longer incubation period to missed intermediate exposure events due to pre‐symptomatic transmission and misperceived exposure times.', 'id': '00012'}], 'title': 'Longer incubation period of coronavirus disease 2019 (COVID‐19) in older adults', 'sha': '8f3c0c4095c08ef8213ada5642ba4a36216dbd0d; 19d6cf627e5afd4d56919357e64d211003665028', 'doi': '10.1002/agm2.12114', 'score': 9.287099838256836}, {'rank': 167, 'paragraphs': [{'text': '(2), capturing the transition from E to I, is simulated via a Poissonian process, in which exposed individuals transform into the infected state at a constant rate. This leads to an exponential decay in E(t) that begins upon infection, representing a memory-less process, whose probability is independent of the time since exposure. In reality, however, incubation has a typical time-scale, and therefore, the transition to I occurs preferably around that time after initial exposure. Hence, in our implementation, to model the incubation period more realistically, we assume that incubation times follow an arbitrary distribution g(t), describing the probability density for an individual to remain asymptomatic for a period t ∈ (t, t + dt). The density g(t) captures both the average incubation time, as well as its potential variability across the population. Denoting the fraction of individuals in each state by S(t), E(t), I(t) and R(t), i.e. S(t) + E(t) + I(t) + R(t) = 1, we can model transitions (1) -(3) as shown in Appendix A.', 'id': '00008'}], 'title': 'Alternating quarantine for sustainable mitigation of COVID-19', 'sha': 'b923a8fc7d406d0f149ecd5293d1a72dec7ee526', 'doi': '', 'score': 9.286999702453613}, {'rank': 168, 'abstract': 'Abstract Although not widely studied, behavioral host manipulation by various pathogens has been documented. Host manipulation is the process by which a pathogen evolves adaptations to manipulate the behavior of the host to maximize reproduction (Ro) of the pathogen. The most notable example is rabies. When a host is infected with the rabies virus it gets into the host’s central nervous system and triggers hyper aggression. The virus is also present in the rabid animal’s saliva so being bitten transmits the infection to a new host and the old host is left to eventually die if untreated. Toxoplasmosis is another example. When mice are infected they demonstrate a fearlessness toward cats, thus increasing their chances of being eaten. Toxoplasmosis needs the digestive tract of the feline to survive. Recent studies have shown that exposure to toxoplasmosis in humans (e.g., through cat feces) has also been associated with behavioral changes that are predicted to enhance the spread of the pathogen. Even the common influenza virus has been shown to selectively increase in-person sociality during the 48-hour incubation period, thus producing an obvious vector for transmission. Here we hypothesize that the novel coronavirus, SARS-CoV2, which produces the COVID-19 disease may produce similar host manipulations that maximize its transmission between humans.', 'body_text': [{'text': 'For viruses to spread, being transmissible is essential. Survival is the most critical goal for different pathogens and parasites, especially concerning their chances of reproduction. Past research has shown that behavioral host manipulation can occur to promote the best interest of the pathogen [1], [2]. Host manipulation can be viewed as adaptations that commandeer the behavior and actions of the host in ways that promote the reproductive best interests of the virus. Benefits may result in increased chances of transmission and survival. Host manipulation has been recognized in a number of pathogens such as toxoplasmosis, rabies, and influenza [1], [2], [3], [4], [5], [6]. It is important to mention that certain pathogens are neuro-invasive, and understanding how it targets and manipulates the central nervous system (CNS) can broaden possible questions about host manipulation [7], [8].', 'cite_spans': [{'start': 297, 'end': 300, 'mention': '[1]', 'ref_id': 'BIBREF0'}, {'start': 302, 'end': 305, 'mention': '[2]', 'ref_id': 'BIBREF1'}, {'start': 653, 'end': 656, 'mention': '[1]', 'ref_id': 'BIBREF0'}, {'start': 658, 'end': 661, 'mention': '[2]', 'ref_id': 'BIBREF1'}, {'start': 663, 'end': 666, 'mention': '[3]', 'ref_id': 'BIBREF2'}, {'start': 668, 'end': 671, 'mention': '[4]', 'ref_id': 'BIBREF3'}, {'start': 673, 'end': 676, 'mention': '[5]', 'ref_id': 'BIBREF4'}, {'start': 678, 'end': 681, 'mention': '[6]', 'ref_id': 'BIBREF5'}, {'start': 890, 'end': 893, 'mention': '[7]', 'ref_id': 'BIBREF6'}, {'start': 895, 'end': 898, 'mention': '[8]', 'ref_id': 'BIBREF7'}], 'section': 'Introduction', 'ref_spans': []}, {'text': 'Focusing on how pathogens may impact individuals’ behavior as hosts can also have important implications for the spread of epidemics and pandemics. Seeing increases in stress and various mental illnesses’ may also be likely [5], [9], [10].', 'cite_spans': [{'start': 224, 'end': 227, 'mention': '[5]', 'ref_id': 'BIBREF4'}, {'start': 229, 'end': 232, 'mention': '[9]', 'ref_id': 'BIBREF8'}, {'start': 234, 'end': 238, 'mention': '[10]', 'ref_id': 'BIBREF9'}], 'section': 'Introduction', 'ref_spans': []}, {'text': 'Behavioral manipulation can occur at different times of the pathogen’s infection lifecycle. Here we outline a series of hypotheses that may present behavioral changes before transmission, during incubation, and before symptoms occur.', 'cite_spans': [], 'section': 'Introduction', 'ref_spans': []}, {'text': 'The hypotheses laid out here include, but are not limited to:•Possible behavioral changes in humans that could promote the transmission of SARS-CoV2 prior to showing symptoms•Possible changes in infected children that function to increase the risk of infection in older people such as parents and caretakers•Possible long-term changes in unborn children•Possible mutations that could drive further transmission\\nHypothesis 1Here we outline how the virus might promote social contact among those infected but remain asymptomatic during the incubation period. The current incubation period for SARS-CoV2 has been estimated at 5 days, but has been observed with a range between 0-24 days. It might be the case that incubation period is facultatively responsive to the possibility of transmission rate and R0. In other words, with a high possibility of transmission, incubation to infection may be delayed in order to maximize number of hosts.\\n', 'cite_spans': [], 'section': 'Introduction', 'ref_spans': []}, {'text': 'An alternative means by which a virus could increase sociality is to reduce the negative emotional consequences of social exclusion. Several studies have documented that social exclusion produces the feeling of “pain”, discomfort, and loss of self-esteem in humans [11]. This effect is magnified by the possibility of kin relationships[12]. The neural underpinnings of this social pain/neural alarm system appear to be located in the anterior cingulate cortex (ACC). If a virus could manipulate the response of the ACC to social exclusion (i.e. social distancing) people would feel less upset and might thus seek additional social support or simply gather socially (e.g., parties). In other words, if your primary friend group excludes you, you might find yourself seeking new people to “hang out” with instead of simply wallowing in your own discontent.Hypothesis 2There is some evidence that P.A.N.D.A.S. (pediatric autoimmune neuropsychiatric disorders associated with streptococcal infection) produces increased separation anxiety in young children, along with obsessive compulsive behaviors [13]. These separation anxieties could promote infection of older hosts. Children have been reported to be fearful of being away from their primary caregivers - expressed as separation anxiety about being isolated from them for extended periods of time. If children become asymptomatic carriers of this virus, then they could function as strong vectors for transmission.\\n', 'cite_spans': [{'start': 265, 'end': 269, 'mention': '[11]', 'ref_id': 'BIBREF10'}, {'start': 335, 'end': 339, 'mention': '[12]', 'ref_id': 'BIBREF11'}, {'start': 1096, 'end': 1100, 'mention': '[13]', 'ref_id': 'BIBREF12'}], 'section': 'Introduction', 'ref_spans': []}, {'text': 'Hu et al. recently investigated 71 cases involved with pneumonia developing from the novel SARS-CoV2 virus [14]. It was found that many patients had low levels of serum cholesterol compared to 80 healthy controls. Total cholesterol, LDL-cholesterol, and HDL-Cholesterol were found to be significantly lower. Looking deeper into low cholesterol levels, Huang et al. found that lower serum cholesterol levels might correlate with depressive and anxious patients [15].Hypothesis 3There is mounting evidence in support of the idea that children born to mothers infected with various pathogens are at heightened risk of later complications, both physical and psychiatric. Most notably, children born to mothers that have been infected with the seasonal influenza virus show an increase in the incidence of schizophrenia and affective disorders [16], [17], [18]. The best estimates are that children born to mothers infected with influenza have a 4 times greater risk of developing either schizophrenia or an affective disorder, with schizophrenia being more likely. This line of research is still preliminary and controversial. We will not be able to observe the effects of SARS-CoV2 on unborn children for some time, but the possibility that this virus manipulates fetuses for its own reproductive success is not out of the question. For example, fetal exposure could produce an early immunological response that results in a suppression of symptoms, but not the elimination of infection nor behavioral changes that could later promote reemergence. Thus, these individuals could be very powerful vectors for transmission in the future. This is of course conjecture, but not outside the realm of possibility.\\nHypothesis 4Perhaps the biggest threat associated with this or any virus, is mutation that 1) becomes even more easily transmissible to others and 2) becomes rapidly lethal after the incubation period. Signs of the novel virus SARS-CoV-2 mutating are already being observed [19]. There appears to be a larger divergence between the novel SARS-CoV2 virus and other known coronavirus’. Researchers also found that the virus has branched into at least two types, type L and type S, in which the S is portrayed as the ancestral version of the SARS-CoV-2. Type L is known to be more prevalent and more aggressive, which could lead to a quicker spread throughout the populations.\\n', 'cite_spans': [{'start': 107, 'end': 111, 'mention': '[14]', 'ref_id': 'BIBREF13'}, {'start': 460, 'end': 464, 'mention': '[15]', 'ref_id': 'BIBREF14'}, {'start': 839, 'end': 843, 'mention': '[16]', 'ref_id': 'BIBREF15'}, {'start': 845, 'end': 849, 'mention': '[17]', 'ref_id': 'BIBREF16'}, {'start': 851, 'end': 855, 'mention': '[18]', 'ref_id': 'BIBREF17'}, {'start': 1978, 'end': 1982, 'mention': '[19]', 'ref_id': 'BIBREF18'}], 'section': 'Introduction', 'ref_spans': []}, {'text': 'According to the World Health Organization, approximately 50% of COVID-19 cases are asymptomatic. This in itself can produce an increased opportunity for transmission; as individuals who are told to stay home are less likely to comply. Social isolation can fundamentally make people desire to be around others as well. The longer people are isolated, the more individuals will feel the need to interact with other people. It should also be considered that individuals with severe depression or suicidal tendencies may be at an increased risk of seeking social interaction as a means to execute their suicidal plan.', 'cite_spans': [], 'section': 'Considerations', 'ref_spans': []}, {'text': 'Furthermore, younger individuals who carry the virus but are not showing symptoms are able to transmit the virus more easily because adults are not taking the same precautions they would if the younger individuals were visibly sick.', 'cite_spans': [], 'section': 'Considerations', 'ref_spans': []}, {'text': 'If our hypotheses that this particular virus manipulates hosts during the incubation period to increase transmission, particularly if it makes them more social, then those who chose to go on spring break (or other vacations) should have a higher rate of infection than those who did not. This could seemingly be tested by examining spring break participation by those who test positive for COVID-19 or by looking at differences in viral status by comparing those who traveled somewhere vs. those who did not. There have been reports of young people holding so-called “coronavirus parties” of which at least one college-aged individual has become sick as a consequence. Some countries are making it illegal to purposefully congregate in a manner that would increase the odds of transmission. Making it illegal may actually promote these parties as young, already risky adults may blatantly ignore those laws. Obviously, these data are not ideal and have several confounds associated with them, but might provide initial insight into possible behavioral mechanisms at play.', 'cite_spans': [], 'section': 'Possible Ways to Collect Data to Test these hypotheses', 'ref_spans': []}, {'text': 'Looking at occupational behaviors may be another possible means of data collection. Taking a measure of when people began to stay at home could be used to represent this. Those who are infected would be expected to have continued to work as a possible mechanism to further the spread of the virus.', 'cite_spans': [], 'section': 'Possible Ways to Collect Data to Test these hypotheses', 'ref_spans': []}, {'text': '\\n Furthermore, for epidemiological purposes, Iceland is attempting to test the entire country for COVID-19. The data that comes from this will be very useful in examining the effects of the virus across various samples within the population. It would be ideal if, in addition to virological and biological data, Iceland (and other countries) were to collect behavioral data.', 'cite_spans': [], 'section': 'Possible Ways to Collect Data to Test these hypotheses', 'ref_spans': []}, {'text': 'We want to make it clear that there are currently no known behavioral host-manipulations associated with SARS-CoV2/COVID-19 infection. The hypotheses laid out here were extrapolated from the well-known effects of other infections to change behavior as an adaptation to maximize the pathogens reproductive best interests. Although these are just hypotheses, only time will tell how SARS-CoV2 manipulates its host for its own survival and replication. Although some might presume that these scenarios are unlikely, it’s our position that these hypotheses are worthy of serious attention.', 'cite_spans': [], 'section': 'Conclusion', 'ref_spans': []}, {'text': 'The authors have no conflict of interest associated with this paper.', 'cite_spans': [], 'section': 'Declaration of Competing Interest', 'ref_spans': []}], 'paragraphs': [{'text': '', 'id': '00004'}], 'title': 'Hypothesized behavioral host manipulation by SARS-CoV2 / COVID-19 infection', 'sha': 'aa406561ef608d6c9b732dcac2ac504c5a33f91f', 'doi': '10.1016/j.mehy.2020.109750', 'score': 9.286998748779297}, {'rank': 169, 'paragraphs': [{'text': 'The ongoing outbreak of COVID-19, starting in December 2019, has quickly become a sweeping and unprecedented global challenge. Because COVID-19 can spread from person to person [10,11], a major challenge in controlling the disease is recognizing and quarantining potential infectious sources as soon as possible [12].', 'id': '00011'}], 'title': 'COVID-19 in a patient with long-term use of glucocorticoids: A study of a familial cluster', 'sha': '912c00514457672748738d29efb81f975815cdf6', 'doi': '10.1016/j.clim.2020.108413', 'score': 9.275400161743164}, {'rank': 170, 'abstract': 'On March 11, 2020, the World Health Organization declared the coronavirus disease 2019, COVID-19, a global pandemic. In an unprecedented collective effort, massive amounts of data are now being collected worldwide to estimate the immediate and long-term impact of this pandemic on the health system and the global economy. However, the precise timeline of the disease, its transmissibility, and the effect of mitigation strategies remain incompletely understood. Here we integrate a global network model with a local epidemic SEIR model to quantify the outbreak dynamics of COVID-19 in China and the United States. For the outbreak in China, in n=30 provinces, we found a latent period of 2.56+/-0.72 days, a contact period of 1.47+/-0.32 days, and an infectious period of 17.82+/-2.95 days. We assume that the latent and infectious periods are disease-specific and the contact period is behavior-specific and can vary between different communities, states, or countries. For the early stages of the outbreak in the United States, in n=50 states, we adopted the latent and infectious periods from China, and found a contact period of 3.38+/-0.69 days. Our network model predicts that - without the massive political mitigation strategies that are in place today - the United states would have faced a basic reproduction number of 5.3+/-0.95 and a nationwide peak of the outbreak on May 10, 2020 with 3 million infections. Our results demonstrate how mathematical modeling can help estimate outbreak dynamics and provide decision guidelines for successful outbreak control. We anticipate that our model will become a valuable tool to estimate the potential of vaccination and quantify the effect of relaxing political measures including total lock down, shelter in place, and travel restrictions for low-risk subgroups of the population or for the population as a whole.', 'body_text': [{'text': 'In December 2019, a local outbreak of pneumonia of initially unknown cause was detected in Wuhan, a city of 11 million people in central China [24] . The cause of the disease was identified as the novel severe acute respiratory syndrome coronavirus 2, SARS-CoV-2 [16] . Infection with the virus can be asymptomatic or can result in a mild to severe symptomatic disease, coronavirus disease 2019 or COVID-19. The majority of COVID-19 cases result in mild symptoms including fever, cough, shortness of breath, and respiratory distress [20] . Severe complications arise when the disease progresses to viral pneumonia and multi-organ failure. The SARS-CoV-2 virus can spread quickly, mainly during close contact, but also through small droplets from coughing or sneezing [32] . After the first four cases were reported on December 29, the outbreak quickly spread from Wuhan across all provinces of mainland China, and, in the following two months, across the entire world. On March 11, 2020, the World Health Organization acknowledged the alarming levels of spread and severity, and characterized the COVID-19 situation as a pandemic [31] . As of today, April 4, 2020, COVID-19 has a↵ected At day 0, a fraction of the susceptible population is exposed to the virus. At day 3, exposed individuals become infectious, and the infectious period lasts for 10 days. At day 5, infectious individuals become symptomatic; the majority of the symptomatic population recovers after 9 days. At day 9, a fraction of the symptomatic population is hospitalized; the majority of the hospitalized population recovers after 14 days. At day 10, a fraction of the hospitalized population experiences critical conditions that last for 10 days and end in either recovery or in death. On the population level, the outbreak of COVID-19 can be summarized in eight curves that illustrate the dynamics of the individual subgroups.', 'cite_spans': [{'start': 143, 'end': 147, 'text': '[24]', 'ref_id': None}, {'start': 263, 'end': 267, 'text': '[16]', 'ref_id': 'BIBREF15'}, {'start': 533, 'end': 537, 'text': '[20]', 'ref_id': 'BIBREF19'}, {'start': 767, 'end': 771, 'text': '[32]', 'ref_id': 'BIBREF32'}, {'start': 1130, 'end': 1134, 'text': '[31]', 'ref_id': 'BIBREF31'}], 'ref_spans': [], 'section': 'Motivation'}, {'text': '203 countries with a total of 1,201,483 reported cases, 64,690 death, and 264,467 recovered cases [9] . Figure 1 illustrates a typical timeline of COVID-19 in a single person and shows how this timeline maps onto an entire population. For this example [17] , at day 0, a number of susceptible individuals are exposed to the virus and transition from the susceptible to the exposed state. Around at day 3, the exposed individuals become infectious. During this time, they can infect others, while not showing any symptoms themselves. The infectious period lasts for approximately 10 days. Around day 5, infectious individuals become symptomatic. This implies that they have potentially spread the disease for two days without knowing it. In the majority of (1 ⌫ h ) of the population, the symptomatic period lasts for approximately 9 days. Around day 9, a severely a↵ected population of ⌫ h are hospitalized and their hospitalization lasts for approximately 14 days. Around day 10, ⌫ c of the hospitalized population experiences critical conditions that last for approximately 10 days and end in (1 ⌫ d ) of recovery and ⌫ d of death. For a hospitalization fraction of ⌫ h = 0.045, a critical conditions fraction of ⌫ c = 0.25, and a death fraction of ⌫ d = 0.50, 99.44% of the population recover and 0.56% die [17] .', 'cite_spans': [{'start': 98, 'end': 101, 'text': '[9]', 'ref_id': 'BIBREF8'}, {'start': 252, 'end': 256, 'text': '[17]', 'ref_id': 'BIBREF16'}, {'start': 1310, 'end': 1314, 'text': '[17]', 'ref_id': 'BIBREF16'}], 'ref_spans': [{'start': 104, 'end': 112, 'text': 'Figure 1', 'ref_id': 'FIGREF0'}], 'section': 'Motivation'}, {'text': 'The first mathematical models for infectious diseases date back to a smallpox model by Daniel Bernoulli in 1760 [4] . Since the 1920s, compartment models have become the most common approach to model the epidemiology of infectious diseases [21] . One of the simplest compartment models is the SEIR model that represents the timeline of a disease through four compartments, the susceptible, exposed, infectious, and recovered populations [2] . The temporal evolution of these compartments is governed by a set of ordinary di↵erential equations parameterized in terms of the transition rates between them [18] . The transition rates ↵ from the exposed to the infectious state and from the infectious to the recovered state are disease specific parameters. In fact, they are the inverses of the latent period A = 1/↵, the time during which an individual is exposed but not yet infectious, and the infectious period C = 1/ , the time during which an individual can infect others. This suggests that these two parameters are relatively independent of country, region, or city. In the example of Figure 1 , the latent and infectious periods are A = 3 days and C = 10 days [17] . The most critical feature of the model is the transition from the susceptible to the exposed state. This transition is typically assumed to scale with the susceptible population S, the infectious population I, and the contact rate , the inverse of the contact period B = 1/ , between them [23] .', 'cite_spans': [{'start': 112, 'end': 115, 'text': '[4]', 'ref_id': 'BIBREF3'}, {'start': 240, 'end': 244, 'text': '[21]', 'ref_id': 'BIBREF20'}, {'start': 437, 'end': 440, 'text': '[2]', 'ref_id': 'BIBREF1'}, {'start': 603, 'end': 607, 'text': '[18]', 'ref_id': 'BIBREF17'}, {'start': 1166, 'end': 1170, 'text': '[17]', 'ref_id': 'BIBREF16'}, {'start': 1462, 'end': 1466, 'text': '[23]', 'ref_id': 'BIBREF22'}], 'ref_spans': [{'start': 1090, 'end': 1098, 'text': 'Figure 1', 'ref_id': 'FIGREF0'}], 'section': 'Motivation'}, {'text': 'The product of the contact rate and the infectious period defines the basic reproduction number, R 0 = C = C/B, the number of individuals that are infected by a single one individual in an otherwise uninfected, susceptible population [11] . The basic reproduction number is a measure of the contagiousness or transmissibility of an infectious agent and it can vary considerably between di↵erent infectious diseases [10] . Typical basic reproduction numbers are on the order of 18 for measles, 9 for chickenpox, 7 for mumps, 7 for rubella, and 5 for poliomyelitis [1] . When the basic basic reproduction number is larger than one, R 0 > 1.0, the infectious period C is larger than the contact period B [23] . This implies that at onset of an epidemic outbreak, when the entire population is susceptible, an infected individual will infect more than one other individual. In agreement with Figure 1 , the infectious population first increases, then reaches a peak, and decreases toward zero [21] . As more and more individuals transition from the susceptible through the exposed and infectious states into the recovered state, the susceptible populations decreases. Once a large enough fraction of a population has become immune-either through recovery from the infection or through vaccination-this group provides a measure of protection for the susceptible population and the epidemic dies out [11] . This indirect protection is called herd immunity [14] . The concept of herd immunity implies that the converged susceptible population at endemic equilibrium is always larger than zero, S 1 > 0, and its value depends on the basic reproduction number R 0 . For a given basic reproduction number R 0 , herd immunity occurs at an immune fraction of (1 1/R 0 ). Knowing the basic reproduction number is therefore critical to estimate the immune fraction of the population that is required to eradicate an infectious disease, for example, 94.4% for measles and 80.0% for poliomyelitis [18] .', 'cite_spans': [{'start': 234, 'end': 238, 'text': '[11]', 'ref_id': 'BIBREF10'}, {'start': 415, 'end': 419, 'text': '[10]', 'ref_id': 'BIBREF9'}, {'start': 563, 'end': 566, 'text': '[1]', 'ref_id': 'BIBREF0'}, {'start': 701, 'end': 705, 'text': '[23]', 'ref_id': 'BIBREF22'}, {'start': 989, 'end': 993, 'text': '[21]', 'ref_id': 'BIBREF20'}, {'start': 1394, 'end': 1398, 'text': '[11]', 'ref_id': 'BIBREF10'}, {'start': 1450, 'end': 1454, 'text': '[14]', 'ref_id': 'BIBREF13'}, {'start': 1981, 'end': 1985, 'text': '[18]', 'ref_id': 'BIBREF17'}], 'ref_spans': [{'start': 888, 'end': 896, 'text': 'Figure 1', 'ref_id': 'FIGREF0'}], 'section': 'Motivation'}, {'text': 'Restrictive measures like medical isolation or quarantine reduce the e↵ective infectious period C and mitigation strategies like contact tracing, physical distancing, or travel restrictions increase the contact period B. Especially during the early stages of an outbreak, passenger air travel can play a critical role in spreading a disease [3] , since traveling individuals naturally have a disproportionally high contact rate [30] . Border control can play a pivotal role in mitigating epidemics and prevent the spreading between cities, states, or countries [34] . In an attempt to mitigate the COVID-19 outbreak, many countries have implemented travel restrictions and mandatory quarantines, closed borders, and prohibited non-citizens from entry. This has stimulated an ongoing debate about how strong these restrictions should be and when it would be safe to lift them. The basic reproduction number is R 0 provides guidelines about the required strength of political counter measures [18] . However, empirically finding the basic reproduction number requires careful contact tracing and is a lot of work, especially once the number of infectious individuals has grown beyond an overseeable size [24] . Network modeling of travel-induced spreading can play an important role in estimating the value of R 0 [7] and interpreting the impact of travel restrictions and boarder control [19] .', 'cite_spans': [{'start': 341, 'end': 344, 'text': '[3]', 'ref_id': 'BIBREF2'}, {'start': 428, 'end': 432, 'text': '[30]', 'ref_id': 'BIBREF30'}, {'start': 561, 'end': 565, 'text': '[34]', 'ref_id': 'BIBREF34'}, {'start': 991, 'end': 995, 'text': '[18]', 'ref_id': 'BIBREF17'}, {'start': 1202, 'end': 1206, 'text': '[24]', 'ref_id': None}, {'start': 1312, 'end': 1315, 'text': '[7]', 'ref_id': 'BIBREF6'}, {'start': 1387, 'end': 1391, 'text': '[19]', 'ref_id': 'BIBREF18'}], 'ref_spans': [], 'section': 'Motivation'}, {'text': 'We model the epidemiology of the COVID-19 outbreak using an SEIR model with four compartments, the susceptible, exposed, infectious, and recovered populations, governed by a set of ordinary di↵erential equa- tions [18] ,Ṡ', 'cite_spans': [{'start': 214, 'end': 218, 'text': '[18]', 'ref_id': 'BIBREF17'}], 'ref_spans': [], 'section': 'Epidemiology modeling'}, {'text': 'The transition rates between the four compartments, , ↵, and are inverses of the contact period B = 1/ , the latent period A = 1/↵, and the infectious period C = 1/ . We interpret the latent and infectious periods A and C as disease-specific, and the contact period B as behavior specific. We discretize the SEIR model in time using an implicit Euler backward scheme and adopt a Newton Raphson method to solve for the daily increments in each compartment.', 'cite_spans': [], 'ref_spans': [], 'section': 'Epidemiology modeling'}, {'text': 'We model the spreading of COVID-19 across a country through a network of passenger air travel, which we represent as a weighted undirected graph G with N nodes and E edges. The nodes represent the individual states, the edges the connections between them. We weight the edges by the estimated annual incoming and outgoing passenger air travel as reported by the Bureau of Transportation Statistics [6] . We summarize the connectivity of the graph G in terms of the degree matrix D II , a diagonal matrix that characterizes the degree of each node I, and the weighted adjacency matrix A IJ , the frequency of travel between the nodes,', 'cite_spans': [{'start': 398, 'end': 401, 'text': '[6]', 'ref_id': 'BIBREF5'}], 'ref_spans': [], 'section': 'Network modeling'}, {'text': 'The di↵erence between the degree matrix D IJ and the adjacency matrix A IJ defines the weighted graph Laplacian L IJ , Figure 2 illustrates the discrete graph G of the United States with 50 nodes and the 200 most traveled edges. The size and color of the nodes represent the degree D II , the thickness of the edges represents the adjacency A IJ . For our passenger travel-weighted graph, the degree ranges from 100 million in California to less than 1 million in Delaware, Vermont, West Virginia, and Wyoming, with a mean degree ofD II = 16 million per node. We assume that the Laplacian L IJ , normalized to one and scaled by the travel coe cient #, characterizes the global spreading of COVID-19 and discretize our SEIR model on our weighted graph G. Specifically, we introduce the susceptible, exposed, infectious, and recovered populations S I , E I , I I , and R I as global unknowns at the I = 1, ..., N nodes of the graph G. This results in the spatial discretization of the set of equations with 4 N unknowns,', 'cite_spans': [], 'ref_spans': [{'start': 119, 'end': 127, 'text': 'Figure 2', 'ref_id': None}], 'section': 'Network modeling'}, {'text': 'We discretize our SEIR network model in time using an implicit Euler backward scheme and adopt a Newton Raphson method to solve for the daily increments in each compartment in each state [15] .', 'cite_spans': [{'start': 187, 'end': 191, 'text': '[15]', 'ref_id': 'BIBREF14'}], 'ref_spans': [], 'section': 'Network modeling'}, {'text': 'Unlike many other countries, China has already seen a peak of the COVID-19 outbreak and is currently not seeing a significant number of new cases. The COVID-19 outbreak data of the Chinese provinces capture all three phases, increase, peak, and decrease of the infectious population and are currently the richest dataset available to date. This dataset describes the temporal evolution of confirmed, recovered, active, and death cases starting January 22, 2020 [9] . As of April 4, there were 81,639 confirmed cases, 76,755 recovered, 1,558 active, and 3,326 deaths. From these data, we map out the temporal evolution of the infectious group I as the di↵erence between the confirmed cases minus the recovered and deaths, and the recovered group R as the sum of the recovered and deaths in each Chinese province. To simulate the province-specific epidemiology of COVID19 with the SEIR model, we use these data to identify the latent period A = 1/↵, the infectious period C = 1/ , and the contact period B = 1/ as a direct measure of the basic reproduction number R 0 = B/C. As our sensitivity analysis in Figure 3 shows, the dynamics of the SEIR model depend critically on the initial conditions, the number of susceptible S 0 , exposed E 0 , infectious I 0 , and recovered R 0 individuals on the day the very first infectious case is reported, I 0 1. Naturally, on this day, the recovered population is R 0 = 0. Since the exposed population is asymptomatic, its initial value E 0 is unknown. To quantify the initial exposed population E 0 , we introduce a parameter ⇢ = E 0 /I 0 , the initial latent population [27] . It defines the fraction of exposed versus infectious individuals at day 0 and is a measure of initial hidden community spreading. The fraction of the initial susceptible population, S 0 = 1 E 0 I 0 R 0 , ensures that the total population sums up to one. To map the total population of one onto the absolute number of cases for each province, we introduce the normalization parameter ⌘ = N ⇤ /N , the a↵ected population. It defines the fraction of the province-specific epidemic subpopulation N ⇤ relative to the province population N [29] . Altogether we identify five parameters for each province, the exposed period A = 1/↵, the infectious period C = 1/ , the contact period B = 1/ or the basic reproduction number R = C/B, the initial latent population ⇢ = E 0 /I 0 , and the a↵ected population ⌘ = N ⇤ /N . We performed the parameter identification using the Levenberg-Marquardt method of least squares. In this identification process, we ignored data from secondary outbreaks [9] .', 'cite_spans': [{'start': 461, 'end': 464, 'text': '[9]', 'ref_id': 'BIBREF8'}, {'start': 1611, 'end': 1615, 'text': '[27]', 'ref_id': 'BIBREF27'}, {'start': 2152, 'end': 2156, 'text': '[29]', 'ref_id': 'BIBREF29'}, {'start': 2599, 'end': 2602, 'text': '[9]', 'ref_id': 'BIBREF8'}], 'ref_spans': [{'start': 1104, 'end': 1112, 'text': 'Figure 3', 'ref_id': None}], 'section': 'COVID-19 outbreak dynamics in China'}, {'text': 'Unlike China, the United States are at the early stage of the COVID-19 outbreak and all states are still seeing an increase of the number of new cases every day. The available dataset describes the temporal evolution of confirmed, recovered, active, and death cases starting January 21, 2020, the first day of the outbreak in the United States [9] . As of April 4, there were 311,357 confirmed cases, 14,825 recovered, 288,081 active, and 8,451 deaths. Similar to the Chinese data, we map out the temporal evolution of the infectious group I as the di↵erence between the confirmed cases minus the recovered and deaths in each state state of the United States. To simulate the state-specific epidemiology of COVID19 with the SEIR model, we use these data to . CC-BY 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 344, 'end': 347, 'text': '[9]', 'ref_id': 'BIBREF8'}], 'ref_spans': [], 'section': 'COVID-19 outbreak dynamics in the United States'}, {'text': 'The copyright holder for this preprint Fig. 3 Outbreak dynamics. Sensitivity with respect to the initial exposed population E 0 . Decreasing the initial exposed population delays the onset of the outbreak while the shapes of all four curves remain the same. The susceptible and recovered populations converge to the same endemic equilibrium at S 1 = 0.202 and R 1 = 0.798. For an initial exposed population of E 0 = 0.01, the infectious population reaches its maximum at I max = 0.121 after 125 days. Decreasing the initial exposed population by a factor 10 delays the maximum by 65 days. Latent period A = 5 days, infectious period C = 20 days, basic reproduction number R 0 = C/B = 2.0, and initial exposed population E 0 = 10 2 , 10 3 , 10 4 , 10 5 , 10 6 , 10 7 , 10 8 .', 'cite_spans': [], 'ref_spans': [{'start': 39, 'end': 45, 'text': 'Fig. 3', 'ref_id': None}], 'section': '(which was not peer-reviewed)'}, {'text': 'identify the contact time B = 1/ , while fixing the disease-specific latent and infections periods A = 1/↵ and C = 1/ at their mean values of the SEIR dynamics fit for the Chinese provinces, and indirectly fitting the basic reproduction number R 0 = C/B. For each state, we set the first day of reported infections I 0 1 to day zero, at which the recovered population is R 0 = 0, the unknown exposed population is E 0 = ⇢ I 0 [27] , and the susceptible population is S 0 = N E 0 I 0 R 0 , where N is the state-specific population [33] . We identify two parameters for each state, the contact period B = 1/ and the initial latent population ⇢ = E 0 /I 0 , while we use the exposed period A = 1/↵ and the infectious period C = 1/ from the parameter identification for the Chinese provinces and back-calculate the basic reproduction number R = C/B. We perform the parameter identification using the Levenberg-Marquardt method of least squares.', 'cite_spans': [{'start': 426, 'end': 430, 'text': '[27]', 'ref_id': 'BIBREF27'}, {'start': 530, 'end': 534, 'text': '[33]', 'ref_id': 'BIBREF33'}], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'The dynamics of the SEIR model are determined by three parameters, the latent period A = 1/↵, and the infectious period C = 1/ , and the contact period Increasing the latent period increases the exposed population and decreases the infectious population. The susceptible and recovered populations converge to the same endemic equilibrium at S 1 = 0.202 and R 1 = 0.798, however, slower. The steepest susceptible, infectious, and recovery curves correspond to the SIR model without separate exposed population with A = 0 days with the maximum infectious population of I max = 0.157 after 86 days. Latent period A = 0, 5, 10, 15, 20, 25 days, infectious period C = 20 days, basic reproduction number R 0 = C/B = 2.0, and initial exposed fraction E 0 = 0.010. B = 1/ , or, alternatively, the basic reproduction number R 0 = C/B. Before identifying these parameters for the outbreaks in China and in the United States, we will illustrate their e↵ects by systematically varying each parameter while keeping the other values fixed. Specifically, unless stated otherwise, we choose a latent period of A = 5 days, an infectious period of C = 20 days, a basic reproduction number of R 0 = C/B = 2.0, and an initial exposed population E 0 = 0.010. Figure 3 illustrates the sensitivity of the SEIR model with respect to the size of the initial exposed population E 0 . Decreasing the initial exposed population from E 0 = 10 2 , 10 3 , 10 4 , 10 5 , 10 6 , 10 7 , 10 8 delays the onset of the outbreak while the dynamics of the susceptible, exposed, infectious, and recovered populations remain the same. For all seven cases, the susceptible and recovered populations converge to the same endemic equilibrium with S 1 = 0.202 and R 1 = 0.798. The infectious population increases gradually, reaches its maximum at I max = 0.121, and then decreases. For the largest initial exposed population of E 0 = 0.01 this maximum occurs after 125 days. Decreasing the initial exposed population by a factor 10 delays the maximum by 65 days. This highlights the exponential nature of the model, which causes a constant delay for a logarithmic decrease of the exponential population, while the overall outbreak dynamics remain the same. In view of the COVID-19 outbreak, this supports the general notion that even a single individual can cause an outbreak. If multiple individuals trigger the outbreak in a province, state, or country, the overall outbreak dynamics will remain the same, but the peak of the outbreak will happen earlier. Figure 4 illustrates the sensitivity of the SEIR model with respect to the latent period A. Increasing the latent period from A = 0, 5, 10, 15, 20, 25 days increases the exposed population and decreases the infectious population. The susceptible and recovered populations converge to the same endemic equilibrium at S 1 = 0.202 and R 1 = 0.798. Convergence is slower for increased latent periods A. The steepest susceptible, infectious, and recovery curves correspond to the SIR model with A = 0 days. This model does not have a separate exposed population. It reaches its peak infectious population of I max = 0.157 after 86 days. In view of the COVID-19 outbreak this implies that knowledge of the latent period is important to correctly estimate the timing and peak of the infectious population, which ultimately determines the absolute number of hospital beds and ventilator units required to insure appropriate medical care. Figure 5 illustrates the sensitivity of the SEIR model with respect to the infectious period C. Increasing the infectious period at a constant basic reproduction number flattens the exposed population and increases the infectious population. The susceptible and recovered populations converge to the same endemic equilibrium at S 1 = 0.202 and R 1 = 0.798, however, slower. The flattest susceptible, infectious, and recovery curves correspond to longest infectious period of C = 30 days and a contact period of B = 15 days with the maximum infectious population of I max = 0.135 after 169 days. In view of the COVID-19 outbreak, knowing the infectious time is important to correctly estimate the timing and peak of the infectious population, and with it the number of required hospital beds and ventilator units. Figure 6 illustrates the sensitivity of the SEIR model with respect to the basic reproduction number R 0 . Decreasing the basic reproduction number decreases the exposed and infectious populations. The susceptible and recovered populations converge to larger and smaller endemic equilibrium values, and converges is slower. The steepest susceptible, exposed, infectious, and recovery curves correspond to the largest basic reproduction number of R 0 = 10.0 with the maximum infectious population of I max = 0.488 after 35 days and converge to an endemic equilibrium at S 1 = 0.0001 and R 1 = 0.9999. In view of the COVID-19 outbreak, the basic reproduction number is the parameter that we can influence by political counter measures. Reduc-. CC-BY 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [{'start': 1238, 'end': 1246, 'text': 'Figure 3', 'ref_id': None}, {'start': 2513, 'end': 2521, 'text': 'Figure 4', 'ref_id': 'FIGREF1'}, {'start': 3443, 'end': 3451, 'text': 'Figure 5', 'ref_id': 'FIGREF2'}, {'start': 4256, 'end': 4264, 'text': 'Figure 6', 'ref_id': 'FIGREF3'}], 'section': 'Outbreak dynamics'}, {'text': 'The copyright holder for this preprint Fig. 7 Outbreak control. E↵ect of basic reproduction number R 0 . Increasing the basic reproduction number beyond one increases the maximum exposed and infectious populations E max and I max . The converged susceptible and recovered populations S 1 and R 1 at endemic equilibrium converge towards zero and one. The time to reach the maximum infectious population reaches its maximum of 213 days at a basic reproduction number R 0 = 1.22 and decreases for increasing basic reproduction numbers. Latent period A = 5 days, infectious period C = 20 days, basic reproduction number, and initial exposed fraction E 0 = 0.010.', 'cite_spans': [], 'ref_spans': [{'start': 39, 'end': 45, 'text': 'Fig. 7', 'ref_id': None}], 'section': '(which was not peer-reviewed)'}, {'text': 'ing the basic reproduction number beyond its natural value by decreasing the contact time B through physical distancing or total lock down allows us to reduce the maximum infectious population and delay the outbreak, a measure that is commonly referred to in the public media as \"flatting the curve\".', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'The sensitivity study suggests that an epidemic outbreak is most sensitive to the basic reproduction number R 0 . While the latent period A and the infectious period C are disease specific, community mitigation and political action can modulate the basic reproduction number R 0 through a variety of measures including active contact tracing, isolation of infectious individuals, quarantine of close contacts, travel restrictions, physical distancing, or total lock down. Figure 7 illustrates the e↵ect of the basic reproduction number R 0 on the maximum exposed and infectious populations E max and I max and on the converged susceptible and recovered populations S 1 and R 1 at endemic equilibrium. Increasing the basic reproduction number beyond one increases the maximum exposed and infectious populations. The converged susceptible and recovered populations decrease towards zero and increase towards one. For the chosen latent and infectious periods of A = 5 days and C = 20 days, the time For example, if we we have access to a certain number of intensive care unit beds and ventilators, and we know rates of the infectious population that have to be hospitalized and require intensive care, we need to limit the maximum size of the population that becomes infectious. To limit the infectious fraction to 20% of the total population, i.e., I max = 0.200, we would have to reduce the basic reproduction number to R 0 = 2.69. The gray line indicates that this maximum would occur after 0.25 years or 93 days. is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [{'start': 472, 'end': 480, 'text': 'Figure 7', 'ref_id': None}], 'section': 'Outbreak control'}, {'text': 'The copyright holder for this preprint . https://doi.org/10.1101/2020.04.06.20055863 doi: medRxiv preprint 0.358. The recovered population converges to progressively smaller endemic equilibrium values R 1 =0.798, 0.775, 0.752, 0.726, 0.710, 0,691, 0.669, 0.642. Convergens is slower under constrained outbreak. The lowest exposed and infectious curves and the flattest susceptible and recovery curves correspond to the most constrained infectious population of I tol = 0.02 with a required level of containment of R 0 (t)/R 0 = 0.524. The highest exposed and infectious curves and the steepest susceptible and recovery curves correspond to an unconstrained infectious population I tol = 0.150 > I max = 0.121 with peak infection after 125 days. In view of the COVID-19 outbreak, the gray line tells us how drastic political counter measures need to be. A required level of containment of R 0 (t)/R 0 = 0.524 implies that we need to reduce the number of infections of a single individual by about one half. However, reducing the maximum infectious population comes at a socioeconomic price: The graphs teach us that it is possible to reach an endemic equilibrium at a smaller total number of individuals that have had the disease; yet, this endemic equilibrium would occur much later in time, for this example, after two or three years. Figure 9 summarizes the dynamics of the COVID-19 outbreak in 30 Chinese provinces. The dots indicate the reported infectious and recovered populations, the lines highlight the simulated susceptible, exposed, infectious, and recovered populations. The simulations are based on a province-specific parameter identification of the latent period A, the contact period B, the infectious period C, and from both, the basic reproduction number R 0 = C/B, the fraction of the initial latent population ⇢ = E 0 /I 0 , and the fraction of the a↵ected population ⌘ = N ⇤ /N for each province. These five province-specific values are reported in each graph. Notably, the province of Hubei, where the outbreak started, has seen the most significant impact with more than 60,000 cases. Naturally, in Hubei, where the first cases were reported, the fraction of the initial latent population ⇢ is zero. Small values of ⇢ indicate a close monitoring of the COVID-19 outbreak, with very few undetected cases at the reporting of the first infectious case. The largest value of ⇢ = 26.4 suggests that, at the onset of the outbreak, a relatively number of cases in the province of Shandong was undetected. The fraction of the a↵ected population ⌘ = N ⇤ /N is a province-specific measure for the containment of the outbreak. Naturally, this number is largest in the province of Hubei, with ⌘ = 1.3 · 10 3 , and, because of strict containment, much smaller in all other provinces. Figure 10 shows the dynamics of the early stages of the COVID-19 outbreak in the 50 states of the United States, the District of Columbia, and the territories of Guam, Puerto Rico, and the Virgin Islands. The dots indicate the reported cases and death, the lines highlight the simulated susceptible, exposed, infectious, and recovered populations. The simulations are based on a state-specific parameter identification of the contact period B that defines the basic reproduction number R 0 = C/B and of the fraction of the initial latent population ⇢ = E 0 /I 0 at a given outbreak delay d 0 for each state. These three state-specific values are reported in each graph. Since the outbreak is currently still in its early stages, we do not attempt to identify the latent and infectious periods, but rather adopt the mean latent and infectious periods A = 2.56 and C = 17.82 from the Chinese outbreak in Table 1 . Notably, the state of New York is currently seeing the most significant impact with more than 100,000 cases. Naturally, in Washington, Illinois, California, and Arizona where the first cases were reported, the fraction of the initial latent population ⇢ is small. Largest ⇢ values occur in New York, New Jersey, Michigan, and Louisiana. The largest basic reproduction numbers R 0 are identified in Idaho, Puerto Rico, Pennsylvania, and Indiana. Table 2 summarizes the parameters for the early stages of the COVID-19 outbreak in the United States. Averaged over all states, we found a contact period of . CC-BY 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [{'start': 1336, 'end': 1344, 'text': 'Figure 9', 'ref_id': 'FIGREF6'}, {'start': 2794, 'end': 2803, 'text': 'Figure 10', 'ref_id': 'FIGREF0'}, {'start': 3696, 'end': 3703, 'text': 'Table 1', 'ref_id': 'TABREF3'}, {'start': 4151, 'end': 4158, 'text': 'Table 2', 'ref_id': 'TABREF4'}], 'section': '(which was not peer-reviewed)'}, {'text': 'The copyright holder for this preprint . https://doi.org/10.1101/2020.04.06.20055863 doi: medRxiv preprint Figure 11 illustrates the exposed, infectious, and recovered fractions of the a↵ected population for each state.', 'cite_spans': [], 'ref_spans': [{'start': 107, 'end': 116, 'text': 'Figure 11', 'ref_id': 'FIGREF0'}], 'section': '(which was not peer-reviewed)'}, {'text': 'Using the parameter values from Table 2 , these curves predict the later stages of the outbreak based on the early states of the outbreak in Figure 10 under the assumption that no additional counter measures are implemented. The simulation uses latent, contact, and infectious periods of A = 2.56 days, B = 3.38±0.69 days, and C = 17.82 days from Table 1 and a fraction of the initial latent population of ⇢ = E 0 /I 0 = 43.75 from Table 2. The orange curve suggests, that the individual states will see a peak of the infectious population at a mean of 39 days after the first infectious case has been reported. The 95% confidence interval suggests that this peak will occur between 4 and 6 weeks after the first reported case provided no additional counter measures are implemented. Figure 12 illustrates the outbreak delay d 0 across the United States. The first reported case was in the state of Washington on January 21, 2020, followed by cases in Illinois with a delay of d 0 = 3, California with d 0 = 4, and Arizona with d 0 = 5, shown in blue. The final states . CC-BY 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [{'start': 32, 'end': 39, 'text': 'Table 2', 'ref_id': 'TABREF4'}, {'start': 141, 'end': 150, 'text': 'Figure 10', 'ref_id': 'FIGREF0'}, {'start': 347, 'end': 354, 'text': 'Table 1', 'ref_id': 'TABREF3'}, {'start': 784, 'end': 793, 'text': 'Figure 12', 'ref_id': 'FIGREF0'}], 'section': '(which was not peer-reviewed)'}, {'text': 'The copyright holder for this preprint . https://doi.org/10.1101/2020.04.06.20055863 doi: medRxiv preprint to see an outbreak were Alabama, Idaho, Montana with d 0 = 52 and West Virginia with d 0 = 56, shown in red. This illustrates that there was a significant time delay in the outbreak with many of the earlier a↵ected states located on the west coast. Figure 13 illustrates the undetected population at the onset of the outbreak across the United States. The ⇢ = E 0 /I 0 value is small in the first states where the outbreak was reported, Washington, Illinois, California, and Arizona, suggesting that the reported cases were truly the first cases in those states. In states where the first cases occurred later, the ⇢ value increases. Notably, Louisiana, Michigan, New Jersey, and New York have the highest ⇢ values of 122.8, 136.1, 197.1, and 1,000 suggesting that both had an exceptionally high number of exposed individuals or individuals that were infected . CC-BY 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [{'start': 356, 'end': 365, 'text': 'Figure 13', 'ref_id': 'FIGREF0'}], 'section': '(which was not peer-reviewed)'}, {'text': 'The copyright holder for this preprint . https://doi.org/10.1101/2020.04.06.20055863 doi: medRxiv preprint but unreported. Figure 14 illustrates the basic reproduction number for the early stages of the outbreak across the United States. The basic reproduction number R 0 = C/B, the number of individuals infected by a single infectious individual, varies from minimum values of 2.5 and 3.6 in Nebraska and Arizona to maximum values of 7.2 and 7.9 in Puerto Rico and Idaho. Figure 15 shows the nation-wide exposed, infectious, and recovered cases for the United States. The dots highlight the reported cases, the lines the predictions of the SEIR network model using data from the early stages of the outbreak with parameters from Tables 1 and 2 and a travel coe cient of # = 0.43. The graphs starts on d 0 , the day at which the last state reported its first case d 0 = March 17, 2020. Compared to the outbreak characteristics for the individual states in Figure  11 with a peak of the infectious population at 39 days after the first infectious case has been reported, the nation-wide outbreak peaks 54 days after the last state has seen an outbreak, on May 10, 2020. This di↵erence is a manifestation of both the state-specific outbreak delay d 0 and the travel of individuals between the different states represented through the network model. Figure 16 illustrates the spatio-temporal evolution of the infectious population across the United States as predicted by the SEIR network model. The simulation uses data from the early stages of the outbreak in ure 10 summarized in Table 2 . As such, the simulation is based on data from the early stages of the outbreak and assumes that no additional counter measures have been implemented. Days ', 'cite_spans': [{'start': 1741, 'end': 1745, 'text': 'Days', 'ref_id': None}], 'ref_spans': [{'start': 123, 'end': 132, 'text': 'Figure 14', 'ref_id': 'FIGREF0'}, {'start': 474, 'end': 483, 'text': 'Figure 15', 'ref_id': 'FIGREF0'}, {'start': 957, 'end': 967, 'text': 'Figure  11', 'ref_id': 'FIGREF0'}, {'start': 1348, 'end': 1357, 'text': 'Figure 16', 'ref_id': 'FIGREF0'}, {'start': 1581, 'end': 1588, 'text': 'Table 2', 'ref_id': 'TABREF4'}], 'section': '(which was not peer-reviewed)'}, {'text': 'We have established a simulation tool that can estimate the dynamics of the COVID-19 outbreak, both locally for individual provinces or states and globally for an entire country. Our simulations suggest that-despite the social, regional, demographical, geographical, and socio-economical heterogeneities in di↵erent regionsthe outbreak of COVID-19 follows a universal model with a few relatively robust parameters. Specifically, our simulation integrates a global network model with a local epidemic SEIR model at each network node. It uses six epidemiologically meaningful parameters, the latent and infectious periods A and C to characterize COVID-19 itself, the contact period B to characterize the behavior of the population, the initial latent population ⇢ = E 0 /I 0 to characterize undetected community spreading at the onset of the outbreak, the a↵ected population ⌘ = N ⇤ /N to characterize containment, and the travel coe cient # to characterize spreading through passenger air travel.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'A and C characterize the timeline of the disease.', 'cite_spans': [], 'ref_spans': [], 'section': 'The latent and infectious periods'}, {'text': 'Our sensitivity analysis in Figures 4 and 5 shows the impact of the latent and infectious periods A and C.', 'cite_spans': [], 'ref_spans': [{'start': 28, 'end': 43, 'text': 'Figures 4 and 5', 'ref_id': 'FIGREF1'}], 'section': 'The latent and infectious periods'}, {'text': 'Both a↵ect the peak of the infectious population both in time and magnitude. The robust data for the infectious and recovered populations of all 30 Chinese provinces in Figure 9 suggest that the latent period lasts for 2.5 days, followed by the infectious period of 17.8 days.', 'cite_spans': [], 'ref_spans': [{'start': 169, 'end': 177, 'text': 'Figure 9', 'ref_id': 'FIGREF6'}], 'section': 'The latent and infectious periods'}, {'text': 'A study of 391 confirmed COVID-19 cases with 1,268 close contacts in Shenzhen found a median incubation period of 4.8 days until the onset of symptoms, a mean time to isolation after the onset of symptoms of 2.7 days or 4.6 days with or without active contact tracing, and a median time to recovery of 20.8 days after the onset of symptoms [5] . These values agree with the reported incubation period of 5.1 days found in 181 confirmed COVID-19 cases outside Wuhan [22] and 5.2 days for the first 425 cases in Wuhan [24] . The total duration from exposure to recovery, (A + C) of our SEIR model, is 20.3 days, 5.3 days shorted than the reported value of 25.6 for the 391 Shenzhen cases [5] . In our model, the reported 4.8 to 5.2 day incubation periods maps onto the latent period A of 2.5 days plus 2.3 to 2.7 days within the infectious period C during which the individuals are infectious but still asymptomatic. This period is critical since individuals can spread the disease without knowing it. The contact tracing study postulates that the infectious period C begins on day 4.8 with the onset of symptoms, 2.3 days later than in our model, and ends on day 7.3 or 9.4 with or without active contract tracing with the beginning of isolation, 13.0 or 10.9 days earlier than in our model. This implies that the infectious period C of our SEIR model is 6.6 and 3.9 times larger . CC-BY 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 340, 'end': 343, 'text': '[5]', 'ref_id': 'BIBREF4'}, {'start': 465, 'end': 469, 'text': '[22]', 'ref_id': 'BIBREF21'}, {'start': 516, 'end': 520, 'text': '[24]', 'ref_id': None}, {'start': 686, 'end': 689, 'text': '[5]', 'ref_id': 'BIBREF4'}], 'ref_spans': [], 'section': 'The latent and infectious periods'}, {'text': 'The copyright holder for this preprint than the infectious period of the traced and untraced early isolated population in Shenzhen [5] . This comparison suggests that it is critical to understand how the infectious period is reported, either as a disease-specific parameter or as a medically-modulated exposure time.', 'cite_spans': [{'start': 131, 'end': 134, 'text': '[5]', 'ref_id': 'BIBREF4'}], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'The contact period B and basic reproduction number R 0 characterize social behavior.', 'cite_spans': [], 'ref_spans': [], 'section': '4.2'}, {'text': 'Our sensitivity analysis in Figures 6, 7 , and 8 shows the impact of the contact period B or, more intuitively, the basic reproduction number R 0 . The basic reproduction number significantly a↵ects the peak of the infectious population both in time and magnitude. The early outbreak data for the infectious populations of all 50 United States in Figure 10 suggest that the contact period is for 3.4 days, resulting in a basic reproduction number of 5.3. For the first 425 cases in Wuhan, the basic reproduction number was estimated to 2.2 [24] and for the 391 cases in Shenzhen, it was 2.6 [5] . A review of the reported basic reproduction numbers for COVID-19 found ranges from 1.40 to 6.49 with a mean of 3.28, values that are larger than those reported for the SARS coronavirus [25] . Huge variations of R 0 values are not uncommon [11] ; even for simple diseases like the measles, reported R 0 value vary between 3.7 and 203.3 [10] . Community mitigation and political action can modulate the basic reproduction number R 0 by a variety of measures including active contact tracing, isolation of infectious individuals, quarantine of close contacts, travel restrictions, physical distancing, or total lock down [13] . Importantly, many of the reported values already include the e↵ect of isolation [24] and active contact tracing and quarantine [5] . If we correct our identified basic reproduction number for China in Figure 9 and Table 1 by reducing our identified infectious period of 17.8 days to the time prior to isolation using the correction factors of 6.6 and 3.9 with and without contact tracing, our R 0 values for China would be 1.91 and 3.23 and fall well within the reported range [25] . Our R 0 value for the United States of 5.30 agrees well with the range of values reported for mathematical model ranging from 1.50 to 6.49 with a mean of 4.20 [25] . Understanding the natural value of R 0 -without any mitigation strategy-is critical to predict the endemic equilibrium, interpret herd immunity, and the estimate the fraction of the population that requires vaccination [18] .', 'cite_spans': [{'start': 540, 'end': 544, 'text': '[24]', 'ref_id': None}, {'start': 591, 'end': 594, 'text': '[5]', 'ref_id': 'BIBREF4'}, {'start': 782, 'end': 786, 'text': '[25]', 'ref_id': 'BIBREF25'}, {'start': 836, 'end': 840, 'text': '[11]', 'ref_id': 'BIBREF10'}, {'start': 932, 'end': 936, 'text': '[10]', 'ref_id': 'BIBREF9'}, {'start': 1215, 'end': 1219, 'text': '[13]', 'ref_id': 'BIBREF12'}, {'start': 1302, 'end': 1306, 'text': '[24]', 'ref_id': None}, {'start': 1349, 'end': 1352, 'text': '[5]', 'ref_id': 'BIBREF4'}, {'start': 1699, 'end': 1703, 'text': '[25]', 'ref_id': 'BIBREF25'}, {'start': 1865, 'end': 1869, 'text': '[25]', 'ref_id': 'BIBREF25'}, {'start': 2091, 'end': 2095, 'text': '[18]', 'ref_id': 'BIBREF17'}], 'ref_spans': [{'start': 28, 'end': 40, 'text': 'Figures 6, 7', 'ref_id': 'FIGREF3'}, {'start': 347, 'end': 356, 'text': 'Figure 10', 'ref_id': 'FIGREF0'}, {'start': 1423, 'end': 1431, 'text': 'Figure 9', 'ref_id': 'FIGREF6'}, {'start': 1436, 'end': 1443, 'text': 'Table 1', 'ref_id': 'TABREF3'}], 'section': '4.2'}, {'text': 'Current mitigation strategies have the goal to \"flatten the curve\", i.e., reduce the peak of the infectious population. As we can see in Figures 6, 7 , and 8, we can achieve this goal by reducing the basic reproduction number R 0 = C/B, which is a direct signature of e↵ective containment measures and drastic behavioral changes that a↵ect a substantial fraction of the susceptible population [13] . By isolating infectious individuals, active contact tracing, and quarantining close contacts, we can reduce the e↵ective infectious period C; and by implementing travel restrictions, mandating physical distancing, or enforcing total lock down, we can increase the contact period B [27] . Figure 9 demonstrates that combinations of these measures have successfully flattened the curves in the 30 provinces of China [24] . But the million-dollar questions remains: What\\'s next?', 'cite_spans': [{'start': 393, 'end': 397, 'text': '[13]', 'ref_id': 'BIBREF12'}, {'start': 681, 'end': 685, 'text': '[27]', 'ref_id': 'BIBREF27'}, {'start': 814, 'end': 818, 'text': '[24]', 'ref_id': None}], 'ref_spans': [{'start': 137, 'end': 149, 'text': 'Figures 6, 7', 'ref_id': 'FIGREF3'}, {'start': 688, 'end': 696, 'text': 'Figure 9', 'ref_id': 'FIGREF6'}], 'section': \"What's next?\"}, {'text': 'In the very near future, our model has the potential to predict the timeline of the outbreak, specifically, the timing and peak of the infectious population in individual states and countries. This will help us optimize planning and distribute medical resources where needed [17] . In the short term, we could enhance our model to study the e↵ect of di↵erent subgroups of the population [5] . This could provide scientific guidelines to gradually relax political measures, for example by releasing different subgroups of the population before others. In the long term, we will need accurate values of the basic reproduction number to estimate the e↵ect of vaccination. This will be critical to design rigorous vaccination programs and prioritize which subgroups of the population to vaccinate first [18] . Naturally, as more data become available, we can train our models more reliably and make more accurate predictions.', 'cite_spans': [{'start': 275, 'end': 279, 'text': '[17]', 'ref_id': 'BIBREF16'}, {'start': 387, 'end': 390, 'text': '[5]', 'ref_id': 'BIBREF4'}, {'start': 799, 'end': 803, 'text': '[18]', 'ref_id': 'BIBREF17'}], 'ref_spans': [], 'section': \"What's next?\"}, {'text': 'This study proposes a new strategy to characterize the timeline of COVID-19. While this allows us to estimate the peaks of the outbreak in space and time, we need to be aware that this study uses a simple model to characterize a complex infectious disease about which we still know very little to this day. Importantly, we have to be cautious not to overstate the results. Specifically, our study has several limitations: First, our mathematical model does not account for asymptomatic cases. Little is known about the fraction of asymptomatic or mildly symptomatic individuals but early studies suggest that up to 25% of individuals have gone from susceptible to recovered without having ever been reported as infectious. Second, the classical SEIR model does not distinguish between asymptomatic infectious in the first days of the disease and symptomatic infectious in the later days. Knowing more about this group and modeling appropriately is critical to accurately estimate the impact of community spreading and mitigation strategies to reduce it. Third, while the initial infectious group I 0 can be reasonably well approximated from the reported active cases and the initial recovered group R 0 is likely zero, the initial exposed group E 0 is really unknown and can hugely e↵ect the outbreak dynamics as the sensitivity study in Figure 3 and the data for China and the United States in Figures 9 and 10 show. We decided to include this e↵ect through the initial latent population ⇢ to highlight this e↵ect, but more data are needed to better estimate the size of this group. Fourth and probably most importantly, the major variable we can influence through social and political measures is the basic reproduction number R 0 , or rather the interplay of the contact period B and infectious period C. Obviously, we do not know the true R 0 , nor can we measure it at this stage of the outbreak, were every state, province, or country has implemented di↵erent measures to modulate the local outbreak. Yet, this study shows that estimating R 0 is important to illustrate how di↵erent political counter measures work and predict the timeline of the infectious population under current conditions. Finally, our network model only roughly estimates the travel matrix from air travel supplemented by rationalized ground transportation across small neighboring states. A more detailed travel information would allow us to truly simulate travel restrictions and estimate the impact of state-specific travel constraints.', 'cite_spans': [], 'ref_spans': [{'start': 1338, 'end': 1346, 'text': 'Figure 3', 'ref_id': None}, {'start': 1395, 'end': 1411, 'text': 'Figures 9 and 10', 'ref_id': 'FIGREF0'}], 'section': 'Limitations'}, {'text': 'The precise timeline of COVID-19, its basic reproduction number, and the e↵ect of di↵erent mitigation strategies remain incompletely understood. Here we combined data from the outbreak in China with data from the early stages of the outbreak in the United States to identify the latent, contact, and infectious periods and the basic reproduction number of COVID-19. To quantify the outbreak dynamics, we integrated a global network model with a local epidemic SEIR model and solved the resulting set of coupled nonlinear equations using a Newton-Raphson scheme. For the outbreak in China, in n = 30 provinces, we found a latent period of 2.6 days, a contact period of 1.5 days, and an infectious period of 17.8 days. For the early stages of the outbreak in the United States, in n = 50 states, we . CC-BY 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusion'}, {'text': '(which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.04.06.20055863 doi: medRxiv preprint found a contact period of 3.4 days and a travel coecient of 0.42. With no additional mitigation strategies, our epidemic network model predicts a basic reproduction number of 5.3 and a nationwide peak of the outbreak on May 10, 2020 with 3 million infections across the United States. Our results suggest that mathematical modeling can help estimate outbreak dynamics and provide decision guidelines for mitigating the COVID-19 pandemic. Our model has the potential to quantify the impact of community measures and predict the effect of relaxing total lock down, shelter in place, and travel restrictions for low-risk subgroups of the population or for the population as a whole.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusion'}], 'paragraphs': [{'text': 'A study of 391 confirmed COVID-19 cases with 1,268 close contacts in Shenzhen found a median incubation period of 4.8 days until the onset of symptoms, a mean time to isolation after the onset of symptoms of 2.7 days or 4.6 days with or without active contact tracing, and a median time to recovery of 20.8 days after the onset of symptoms [5] . These values agree with the reported incubation period of 5.1 days found in 181 confirmed COVID-19 cases outside Wuhan [22] and 5.2 days for the first 425 cases in Wuhan [24] . The total duration from exposure to recovery, (A + C) of our SEIR model, is 20.3 days, 5.3 days shorted than the reported value of 25.6 for the 391 Shenzhen cases [5] . In our model, the reported 4.8 to 5.2 day incubation periods maps onto the latent period A of 2.5 days plus 2.3 to 2.7 days within the infectious period C during which the individuals are infectious but still asymptomatic. This period is critical since individuals can spread the disease without knowing it. The contact tracing study postulates that the infectious period C begins on day 4.8 with the onset of symptoms, 2.3 days later than in our model, and ends on day 7.3 or 9.4 with or without active contract tracing with the beginning of isolation, 13.0 or 10.9 days earlier than in our model. This implies that the infectious period C of our SEIR model is 6.6 and 3.9 times larger . CC-BY 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'id': '00028'}], 'title': 'Outbreak dynamics of COVID-19 in China and the United States', 'sha': 'b1edab37b21f38120b9933f7205803befab9debf', 'doi': '10.1101/2020.04.06.20055863', 'score': 9.274900436401367}, {'rank': 171, 'paragraphs': [{'text': 'Case 4 and Case 5 were the other two family members but were excluded from COVID-19. Remarkably, Case 4, who developed fever, was diagnosed with malignant lymphoma according to the result of her bone marrow puncture.', 'id': '00010'}], 'title': 'COVID-19 in a patient with long-term use of glucocorticoids: A study of a familial cluster', 'sha': '912c00514457672748738d29efb81f975815cdf6', 'doi': '10.1016/j.clim.2020.108413', 'score': 9.271699905395508}, {'rank': 172, 'paragraphs': [{'text': 'In the case of the usual seasonal flu the main parameters of the spread of the viruses are the transmission rate which reflects the power of the transmission of the virus from infected people to susceptibles, the recovery rate which is reciprocle to the recovery period (which is the sum of recovery to health + isolated sick people + mortality due to the sickness ), and the parameter which is the reciprocle to the incubation period. Due to the long incubation period and large number of asymptomatic or mild-symptomatic cases, COVID-19 has proved to be very insidious and requires intensive emergency measures from the authorities to reduce the transmission rate and to increase the recovery rate For comparison, in the case of the seasonal flu no intensive measures are necessary to be undertaken by the authorities.', 'id': '00008'}], 'title': 'The TVBG-SEIR spline model for analysis of COVID-19 spread, and a Tool for prediction scenarios', 'sha': 'de3c922efb6cd2e9d1a6cc9ab5e586a63eab3b4f', 'doi': '', 'score': 9.266200065612793}, {'rank': 173, 'paragraphs': [{'text': 'The literature quality assessment was evaluated in parallel by two researchers (L.W., Y.L.) according to Agency for Healthcare Research and Quality (AHRQ) guidelines (Supplement Table 5 ). Disagreement between the two researchers was resolved by consensus and the resolution was confirmed by two senior authors (F.C., Y.W.). From each recruited study, either individual-level incubation period data or summarized statistics for central tendency (mean or median) and dispersion (variance, standard deviation, interquatiles, or range) measures were extracted, as well as population characteristics including sample size, average age, and male proportion (Supplement Table 6 ). Data were extracted by two independent research coordinators from each publication (L.W., Y.L.); inconsistent inputs were verified and justified by a third author to ensure correctness of data extraction (Y.W.). All literature included in the meta-analysis was labeled as \"General Transmissions\".', 'id': '00010'}], 'title': 'A systematic review and meta-analysis reveals long and dispersive incubation period of COVID-19', 'sha': '3d921c03dc23ee850655ddfd9d5ebe3956d70bf8', 'doi': '10.1101/2020.06.20.20134387', 'score': 9.261899948120117}, {'rank': 174, 'paragraphs': [{'text': 'The copyright holder for this preprint this version posted June 22, 2020. . https://doi.org/10.1101/2020.06.20.20134387 doi: medRxiv preprint . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. Studies were ordered by date of post online. Significant heterogeneity was observed among studies (I 2 = 96.1%, P < 0.0001). The random-effects meta-analysis using restricted maximum likelihood (REML) was used to summarize the median incubation period (days) and the corresponding 95% confidence interval (95%CI). CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'id': '00040'}], 'title': 'A systematic review and meta-analysis reveals long and dispersive incubation period of COVID-19', 'sha': '3d921c03dc23ee850655ddfd9d5ebe3956d70bf8', 'doi': '10.1101/2020.06.20.20134387', 'score': 9.2618989944458}, {'rank': 175, 'paragraphs': [{'text': \"Our study demonstrates that SARS-CoV-2 has a considerably longer incubation period than most types of viral respiratory infections. Notably, a significant positive association between length of incubation period and magnitude of R 0 was observed. This finding indicates viral respiratory infections beyond SARS-CoV-2 may have transmission potential during their incubation period. In addition, SARS-CoV-2 has a high dispersion of incubation period, which increases the difficulty in tracing and controlling for contacts. These unique epidemiologic characteristics partially contribute to today's global spread of COVID-19. The 14-day quarantine period has been adopted in mainland China and suggested to the international community by WHO (26) . We estimated that 7 out of 100 general infections would develop symptoms after 14 days. Further, nearly 15 out of 100 infected people under age 20 years will develop symptoms after 14 days. However, considering the probability of being infected among contacts of the population in active monitoring or quarantine, about 20 per 10 000 contacts would develop symptoms after 14 days in active monitoring or quarantine; the risk of observing young-age infections or asymptomatic transmissions developing symptoms after 14 days is below 1 in 10 000. Assuming the risks of being infected for the active monitoring population having contacts with symptomatic infections and latent infections were 1% in equal, about 3 in 10 000 contacts would develop symptoms after 14 days, which is more than 1.0 in 10 000 reported in the previous study. 19 Thus, the 14-day quarantine policy is sufficient to trace and identify infections among an active monitoring population. However, precise understanding of the crucial epidemiological parameters related to transmission probability in active monitoring population could aid in further refining the appropriate length of quarantine (27) .\", 'id': '00030'}], 'title': 'A systematic review and meta-analysis reveals long and dispersive incubation period of COVID-19', 'sha': '3d921c03dc23ee850655ddfd9d5ebe3956d70bf8', 'doi': '10.1101/2020.06.20.20134387', 'score': 9.257699966430664}, {'rank': 176, 'paragraphs': [{'text': 'The results are shown in Figure 1 , which gives the computed value of R, for all three models, as a function of the latency t 1 . The relevant formulas are as follows: When both incubation times are treated as exponential random variables,', 'id': '00017'}], 'title': 'Effects of latency on estimates of the COVID-19 replication number', 'sha': 'a017dd5e8dfa5dfd6385d407813e7c26d98becb9', 'doi': '10.1101/2020.04.07.20056960', 'score': 9.25529956817627}, {'rank': 177, 'paragraphs': [{'text': 'In the model with fixed incubation times, Exposed become Infected at rate RI(t − t 1 )/t 2 , while Infected are Removed at rate RI(t − t 1 − t 2 )/t 2 , so', 'id': '00060'}], 'title': 'Effects of latency on estimates of the COVID-19 replication number', 'sha': 'a017dd5e8dfa5dfd6385d407813e7c26d98becb9', 'doi': '10.1101/2020.04.07.20056960', 'score': 9.255298614501953}, {'rank': 178, 'paragraphs': [{'text': 'such that upon infection susceptible individuals enter an incubation period, in which they are at the exposed state E.', 'id': '00006'}], 'title': 'Alternating quarantine for sustainable mitigation of COVID-19', 'sha': 'b923a8fc7d406d0f149ecd5293d1a72dec7ee526', 'doi': '', 'score': 9.253399848937988}, {'rank': 179, 'paragraphs': [{'text': 'The copyright holder for this preprint this version posted June 22, 2020. . https://doi.org/10.1101/2020.06.20.20134387 doi: medRxiv preprint 9 / 16 monitoring was 0.2 in 10 000 (95%CI: 0.0-3.1 in 10 000) ( Figure 4I ). Sensitivity analyses considering various settings for the risk of being infected among an active monitoring population were performed as well (Supplement Table 14 ). Overall, in the active monitoring population, the risk of developing symptoms after a 14-day quarantine period was about 20 in 10 000, while the risk of young age infections and asymptomatic transmissions developing symptoms after 14-day quarantine period was below 1 in 10 000.', 'id': '00023'}], 'title': 'A systematic review and meta-analysis reveals long and dispersive incubation period of COVID-19', 'sha': '3d921c03dc23ee850655ddfd9d5ebe3956d70bf8', 'doi': '10.1101/2020.06.20.20134387', 'score': 9.246800422668457}, {'rank': 180, 'paragraphs': [{'text': \"The home quarantine involved in our article was completely implemented according to the announcement released by the National Health Commission of the People's Republic of China (http://www.gov.cn/zhengce/zhengceku/2020-02/05/content_5474688.htm.accessed 26 March 2020).\", 'id': '00005'}], 'title': 'COVID-19 in a patient with long-term use of glucocorticoids: A study of a familial cluster', 'sha': '912c00514457672748738d29efb81f975815cdf6', 'doi': '10.1016/j.clim.2020.108413', 'score': 9.229700088500977}, {'rank': 181, 'abstract': 'BACKGROUND: As of April 2, 2020, the global reported number of COVID-19 cases has crossed over 1 million with more than 55,000 deaths. The household transmissibility of SARS-CoV-2, the causative pathogen, remains elusive. METHODS: Based on a comprehensive contact-tracing dataset from Guangzhou, we estimated both the population-level effective reproductive number and individual-level secondary attack rate (SAR) in the household setting. We assessed age effects on transmissibility and the infectivity of COVID-19 cases during their incubation period. RESULTS: A total of 195 unrelated clusters with 212 primary cases, 137 nonprimary (secondary or tertiary) cases and 1938 uninfected close contacts were traced. We estimated the household SAR to be 13.8% (95% CI: 11.1–17.0%) if household contacts are defined as all close relatives and 19.3% (95% CI: 15.5–23.9%) if household contacts only include those at the same residential address as the cases, assuming a mean incubation period of 4 days and a maximum infectious period of 13 days. The odds of infection among children (<20 years old) was only 0.26 (95% CI: 0.13–0.54) times of that among the elderly (≥60 years old). There was no gender difference in the risk of infection. COVID-19 cases were at least as infectious during their incubation period as during their illness. On average, a COVID-19 case infected 0.48 (95% CI: 0.39–0.58) close contacts. Had isolation not been implemented, this number increases to 0.62 (95% CI: 0.51–0.75). The effective reproductive number in Guangzhou dropped from above 1 to below 0.5 in about 1 week. CONCLUSION: SARS-CoV-2 is more transmissible in households than SARS-CoV and MERS-CoV, and the elderly ≥60 years old are the most vulnerable to household transmission. Case finding and isolation alone may be inadequate to contain the pandemic and need to be used in conjunction with heightened restriction of human movement as implemented in Guangzhou.', 'body_text': [{'text': 'The ongoing pandemic of the coronavirus disease 2019 (COVID-19), caused by the novel coronavirus SARS-CoV-2, has now affected 181 countries worldwide. As of April 2, 2020, there were more than 1 million reported cases and over 55,000 deaths.1 The elderly and individuals with chronic conditions such as diabetes and cardiopulmonary disease are most vulnerable to severe disease and death. Efficient transmission via droplets and fomites is potentially supplemented by other transmission routes such as aerosol and fecal contamination.2,3 Evidence is emerging that pre-symptomatic or asymptomatic carriers can transmit the virus.5,6 It is suspected that within-household transmission might have contributed substantially to the continued rise in cases in China even after the introduction of nationally enforced restrictions on human movement.7,8 Home isolation/quarantine of people with an exposure history or mild symptoms is frequently recommended as a disease control measure in countries with COVID-19 outbreaks, but such restrictions likely have limited or no effect on family transmission.', 'cite_spans': [{'start': 241, 'end': 242, 'mention': '1', 'ref_id': 'BIBREF0'}, {'start': 534, 'end': 535, 'mention': '2', 'ref_id': 'BIBREF11'}, {'start': 536, 'end': 537, 'mention': '3', 'ref_id': 'BIBREF22'}, {'start': 628, 'end': 629, 'mention': '5', 'ref_id': 'BIBREF25'}, {'start': 630, 'end': 631, 'mention': '6', 'ref_id': 'BIBREF26'}, {'start': 842, 'end': 843, 'mention': '7', 'ref_id': 'BIBREF27'}, {'start': 844, 'end': 845, 'mention': '8', 'ref_id': 'BIBREF28'}], 'section': 'Introduction', 'ref_spans': []}, {'text': 'Thus far, transmissibility of the disease has primarily been assessed either at the population level using mathematical models or at the individual level in synthetic populations using agent-based models.9–11 Transmissibility within households or through other types of close contact remains under-investigated, despite the importance of these social interactions in shaping the overall dynamics of disease spread and in determining the effectiveness of mitigation strategies.12 Obtained via tracing, contact data provide the most accurate information about human-to-human transmissibility of any infectious pathogen, because transmissibility can be assessed conditioning on exposure. Currently available estimates for the secondary attack rate (SAR) of SARS-CoV-2 were based on contact-tracing data of hundreds of cases in Shenzhen and Guangzhou, Guangdong Province in southern China as well as 10 cases in the United States.13–15 These estimate are proportions of confirmed infections among all traced contacts, not accounting for heterogeneity in exposure level, possibility of transmission among contacts themselves and infection risks from untraced contact or fomites. We refer to these proportions as the non-primary attack rate rather than SAR, as the infections among contacts are not necessarily secondary.', 'cite_spans': [{'start': 204, 'end': 205, 'mention': '9', 'ref_id': 'BIBREF29'}, {'start': 206, 'end': 208, 'mention': '11', 'ref_id': 'BIBREF2'}, {'start': 476, 'end': 478, 'mention': '12', 'ref_id': 'BIBREF3'}, {'start': 926, 'end': 928, 'mention': '13', 'ref_id': 'BIBREF4'}, {'start': 929, 'end': 931, 'mention': '15', 'ref_id': 'BIBREF6'}], 'section': 'Introduction', 'ref_spans': []}, {'text': 'Applying a statistical transmission model to the contact-tracing data from Guangzhou, we estimated the SAR of SARS-CoV-2 among household and non-household close contacts. We assessed the effects of age and gender on the infectiousness of COVID-19 cases and susceptibility of their close contacts. In addition, we evaluated the relative infectiousness of COVID-19 cases during the incubation period compared to the period of illness.', 'cite_spans': [], 'section': 'Introduction', 'ref_spans': []}, {'text': 'Case definitions (Appendix Sec. 1.1) are in accordance with the Control and Prevention Plan for the Novel Coronavirus Pneumonia Disease (v5) issued by the Chinese Center for Disease Control and Prevention (China CDC). Briefly, a suspected COVID-19 case is defined as a patient meeting ≥1 epidemiological criteria and ≥2 clinical criteria as outlined in the appendix. A confirmed case is defined as a suspected case with positive detection of SARS-CoV-2 nucleic acid by real-time RT-PCR or viral genes that are highly homologous to SARS-CoV-2 by sequencing using respiratory or blood specimens. An asymptomatic infection is an individual with laboratory confirmation but without clinical signs, mainly found by outbreak investigation and contact tracing.', 'cite_spans': [], 'section': 'Case definition ::: Methods', 'ref_spans': []}, {'text': 'Epidemiological investigations were conducted by county-level CDC offices within 24 hours after a suspected or confirmed case or an asymptomatic infection was reported. For each suspected or confirmed case, demographic, clinical, diagnostic and occupational data, baseline health conditions, clinical samples and laboratory test results, and exposure history in the 14 days prior to symptom onset were recorded using a standardized investigation form. Typically, a suspected case would be changed to a confirmed case or removed from the surveillance system when laboratory test results became available. A close contact is defined as an individual who had unprotected close contact (within 1 meter) with a confirmed case or an asymptomatic infection within 2 days before their symptom onset or sample collection, including but not limited to household members, care givers and individuals in the same workplace, classroom, ward or transportation vehicle. Close contacts were quarantined at an observatory place or at home and followed for 14 days, according to the Temporary Guidelines for Investigation and Management of Close Contacts of Novel Coronavirus Pneumonia Patients issued by China CDC.', 'cite_spans': [], 'section': 'Epidemiological investigation and contact-tracing ::: Methods', 'ref_spans': []}, {'text': 'In our analyses, individuals who are linked by contact tracing are considered as a cluster. Given a cluster, we define cases with symptom onset on either the earliest onset day or the following day as primary cases. If a cluster had only asymptomatic infections, primary cases are determined by specimen collection dates. Individuals who are either family members, or close relatives, such as parents and parents-in-law, or live in the same residential address are considered as household contacts of each other. In sensitivity analyses, we also restrict household contacts to individuals living at the same address.', 'cite_spans': [], 'section': 'Epidemiological investigation and contact-tracing ::: Methods', 'ref_spans': []}, {'text': 'Standard nonparametric tests such as Fisher’s exact test were used to compare characteristics between demographic groups in R.16 The spatial distribution of clusters was mapped at the community level using ArcGIS (Environmental Systems Research Institute, Redlands, California), with a directed graph indicating potential transmission chains. We estimated effective reproductive numbers (Rt) based on the contact tracing data (Appendix Sec. 1.2). Due to the uncertainty in the transmission relationship, we explore three scenarios defined by the following assumptions: (1) all imported cases (with travel or living history in Wuhan or Hubei Province 14 days before symptom onset) were primary cases, and all nonprimary cases were infected by primary cases in the same cluster; (2) in addition to (1), local primary cases might have been infected by earlier primary cases in other clusters; and (3) in addition to (2), imported non-primary cases were not considered as primary cases.', 'cite_spans': [{'start': 126, 'end': 128, 'mention': '16', 'ref_id': 'BIBREF7'}], 'section': 'Statistical Analysis ::: Methods', 'ref_spans': []}, {'text': 'A chain-binomial statistical model was used to estimate SAR and local reproductive number (Appendix Sec. 1.3).17 Only confirmed infections (symptomatic or asymptomatic) and their contacts were included in the analysis. the Expectation and maximization (EM) approach was used to account for uncertainty in the infection time of asymptomatic infections (Appendix Sec. 1.3). Possible distributions of the incubation period and the infectious period were derived from the literature and our previous work on a separate contact-tracing database (Appendix Sec. 1.4).18–20', 'cite_spans': [{'start': 110, 'end': 112, 'mention': '17', 'ref_id': 'BIBREF8'}, {'start': 560, 'end': 562, 'mention': '18', 'ref_id': 'BIBREF9'}, {'start': 563, 'end': 565, 'mention': '20', 'ref_id': 'BIBREF12'}], 'section': 'Statistical Analysis ::: Methods', 'ref_spans': []}, {'text': 'Briefly, this model estimates the probabilities, denoted by p1 and p2, of viral transmission from an infectious household contact and from an infectious non-household contact (e.g., friends, co-workers, passengers, etc.), respectively, to a susceptible person per daily contact. In addition, each susceptible person is subject to a constant daily probability,b, of being infected by an unspecified external source, which accounts for untraced contacts and fomites. Suppose that the infectivity of a COVID-19 case differs between the incubation period and the post-illness-onset period (referred to as the illness period hereinafter). We use (Dmin, Dmax) to represent the whole infectious period with the symptom onset day set as day 0. We then model the effective daily transmission probability as pk*(l)=pk on days Dmin ≤ l < 0 (incubation period) and pk*(l)1−pk*(l)=pk1−pk×OR on days 0 ≤ l < Dmax (illness period), where OR is the odds ratio measuring the relative infectivity of the illness period vs. the incubation period. The SAR is defined as\\nSARk=1−∏l=DminDmax[1−pk*(l)ϕ*(l)],k=1,2,\\nwhere ϕ*(l) specifies the relative infectivity level on the lth day of the infectious period based on prior studies,20 peaking near the time of symptom onset. We define the local reproductive number as the average number of infections a symptomatic case can generate via household and non-household contacts,\\nR=∑k=12∑l=DminDmax{nk(l)pk*(l)ϕ*(l)∏m=Dminl−1[1−pk*(l)ϕ*(l)]},\\nwhere n1(l) and n2(l) are the average numbers of household and non-household contacts per primary case on day l (Appendix Sec. 1.5).', 'cite_spans': [{'start': 1207, 'end': 1209, 'mention': '20', 'ref_id': 'BIBREF12'}], 'section': 'Statistical Analysis ::: Methods', 'ref_spans': []}, {'text': 'We assessed the effect of age and gender on susceptibility or infectivity by regressing transmission probabilities on these demographic characteristics. Finally, we evaluated the goodness-of-fit of the model under different settings of the natural history of disease (Appendix Sec. 1.6).', 'cite_spans': [], 'section': 'Statistical Analysis ::: Methods', 'ref_spans': []}, {'text': 'Data were collected as part of an continuing public health response required by the national Health Commission of China, and hence informed consent was waived. Analysis of deidentified data was considered not to be human subjects research after consultation with the University of Florida IRB. All analyses of personally identifiable data were conducted at the Guangzhou CDC.', 'cite_spans': [], 'section': 'Ethics statement ::: Methods', 'ref_spans': []}, {'text': 'The funder of the study had no role in study design, data collection, data analyses, result interpretation, or writing of the manuscript. The corresponding authors had full access to all the data and had final responsibility for the decision to submit for publication.', 'cite_spans': [], 'section': 'Role of the funding source ::: Methods', 'ref_spans': []}, {'text': 'By February 17, 2020, a total of 349 lab-confirmed cases were reported to Guangzhou Municipal CDC, forming 195 unrelated clusters with 212 primary cases and 137 nonprimary (secondary or tertiary) cases. The size of clusters ranges from 1 to 274, with a median size of 6 and an inter-quartile range (IQR) of (4, 10). The majority (65%) of clusters had no secondary cases.', 'cite_spans': [], 'section': 'Results', 'ref_spans': []}, {'text': ' The majority of cases were adults aged 20–59 years (Table 1). The majority of primary cases (171, 81%) and more than half of non-primary cases (77, 56%) had recent travel or living history outside Guangzhou (referred to as imported cases hereinafter), and 89% of these imported cases were related to Hubei Province. Among the 155 imported primary cases with known arrival dates, 123 (79%) arrived at Guangzhou on or before the complete lockdown of Wuhan (Jan. 24). The overall non-primary attack rates are 12.6% and 3.06% among household and non-household contacts, respectively. Within households, the non-primary attack rate was much lower in contacts <20 years group, 5.26% (95% CI 2.43, 9.76%), as compared to 13.72% (95% CI 10.68, 17.24%) and 17.69% (95% CI: 11.89, 24.83%) in 20–59 year olds and ≥60 year olds, respectively (p-values <0.005). A similar age trend was observed among non-household contacts, but the differences were not of statistical significance. There was no gender difference in the non-primary attack rates, for both inside and outside the household.', 'cite_spans': [], 'section': 'Results', 'ref_spans': [{'start': 53, 'end': 60, 'mention': 'Table 1', 'ref_id': 'TABREF0'}]}, {'text': 'Most cases were found in the densely populated districts (56% of the total population in Guangzhou) including Yuexiu, Liwan, Haizhu, Tianhe and Baiyun (Figure 1). Three clusters with 5 or more secondary cases (not counting tertiary or further generations) were observed, one in each of Yuexiu, Haizhu and Baiyun districts, and all primary cases in these clusters were imported (Figure 1A–B). The longest transmission chain had three subsequent generations from the primary case, which occurred in Panyu district (Figure 1C). Five other clusters had two subsequent generations (Figure 1A–B). The reported residential locations of primary and non-primary cases within the same clusters are mostly identical, but non-household non-primary cases might live far from the primary cases. Most transmissions occurred between household members (Appendix Figure S1).', 'cite_spans': [], 'section': 'Results', 'ref_spans': [{'start': 152, 'end': 160, 'mention': 'Figure 1', 'ref_id': 'FIGREF0'}, {'start': 378, 'end': 387, 'mention': 'Figure 1A', 'ref_id': 'FIGREF0'}, {'start': 388, 'end': 389, 'mention': 'B', 'ref_id': 'FIGREF0'}, {'start': 513, 'end': 522, 'mention': 'Figure 1C', 'ref_id': 'FIGREF0'}, {'start': 577, 'end': 586, 'mention': 'Figure 1A', 'ref_id': 'FIGREF0'}, {'start': 587, 'end': 588, 'mention': 'B', 'ref_id': 'FIGREF0'}]}, {'text': 'Figure 2 shows the temporal trend of the COVID-19 epidemic in Guangzhou. The first imported primary case had symptom onset on Jan. 7, 2020, and this case arrived in Guangzhou on Jan. 13. The earliest local primary case had symptom onset on Jan. 16, by which time there were already at least 2 imported cases in Guangzhou. The number of imported cases peaked together with the epidemic in Guangzhou around Jan. 27, 4 days after the lockdown in Wuhan. After that, both case importation and the epidemic itself waned fast, and only sporadic cases were reported by the middle of February. The Rt started relatively high at the early phase, reaching 1.3 in scenario (3), barely 1.1 in scenarios (2) and only 0.7 in scenario (1). Rt quickly declined after the peak to below 0.5 by Jan. 27 for all scenarios, likely reflecting the tightening of control measures in Guangzhou.', 'cite_spans': [], 'section': 'Results', 'ref_spans': [{'start': 0, 'end': 8, 'mention': 'Figure 2', 'ref_id': 'FIGREF1'}]}, {'text': 'After excluding 12 clusters with only primary cases but no contact, we included in the transmission analysis 183 clusters with a total of 335 cases (329 symptomatic and 6 asymptomatic) and 1938 noninfected contacts. We estimated the household and non-household SARs for the combinations of mean incubation period of 4–7 days and maximum infectious period of 13, 16 and 19 days. Assessment of the goodness-of-fit for these settings suggests that all models fits the data reasonably well, with model predicted infection numbers falling largely in line with the observed values. A mean incubation period of 4 days and a maximum infectious period of 13 days yielded the smallest difference between observed and model-fitted numbers (Appendix Figure S3). Consequently, we reported the estimates associate with this setting as the primary results.', 'cite_spans': [], 'section': 'Results', 'ref_spans': []}, {'text': 'When household contact is defined based on close relatives, we estimate the household SAR as 13.8% (95% CI: 11.1–17.0%) and the non-household SAR as 7.1% (95% CI: 4.7–10.6%), had there been no case isolation. A longer incubation periods is associated with a slightly lower, whereas a longer infectious period is associated with a slightly higher, SAR estimate (Table 2; Appendix Table S3). The household SAR varies from 12% to 17%, and the non-household SAR varied from 6% to 9%. The local R based on observed contact frequencies of primary cases was estimated to be 0.48 (0.39, 0.58), which is insensitive to the setting for the incubation and infectious periods. In other words, a typical case infected 0.48 individuals on average in Guangzhou, implying an inefficient transmission of the disease under the control measures. The projected local R, had there been no quarantine of cases, was estimated as 0.62 (0.51–0.75). High estimates of projected local R were associated with short incubation and longer infectious periods (Appendix Table S3). The daily transmission probability during the incubation period tends to be similar to that during the illness period (Table 3), with an estimated odds ratio (OR) of 1.13 (0.59–2.18); however, the difference is much wider when longer incubation periods were assumed (Table3; Appendix Table S4–S5).', 'cite_spans': [], 'section': 'Results', 'ref_spans': [{'start': 361, 'end': 368, 'mention': 'Table 2', 'ref_id': 'TABREF1'}, {'start': 1168, 'end': 1175, 'mention': 'Table 3', 'ref_id': 'TABREF2'}, {'start': 1316, 'end': 1322, 'mention': 'Table3', 'ref_id': 'TABREF2'}]}, {'text': 'Individuals ≥60 years old were the most susceptible group to COVID-19 (Table 4). In comparison to the elderly group (≥60 years old), the odds of infection is 0.27 (0.13–0.55) in the youngest group (<20 years old) and 0.8 (95% CI: 0.54–1.20) in the younger adult group (20–59 years old). These age effects on susceptibility are insensitive to the assumptions about the natural history of disease. Cases <60 years old were slightly less infectious than those ≥60 years old, with ORs varying in the range of 0.73–0.79 but not significantly different from 1 (Table 4; Appendix Table S6). The ORs for the relative infectivity of the incubation vs. illness periods are similar to the model not adjusted for age group. The estimated daily transmission probabilities in the age-adjusted model are for the elderly and therefore higher than the estimates unadjusted for age (Appendix Table S4).', 'cite_spans': [], 'section': 'Results', 'ref_spans': [{'start': 71, 'end': 78, 'mention': 'Table 4', 'ref_id': 'TABREF3'}, {'start': 555, 'end': 562, 'mention': 'Table 4', 'ref_id': 'TABREF3'}]}, {'text': 'Restricting household contacts to those who were living at the same address as the primary case led to higher SAR estimates among household contacts, 16.4–23.4%, but lower SAR estimates among the non-household contacts under the various settings of the natural history of disease (Table 2; Appendix Table S3). Under the setting of the shortest incubation and infectious periods, the SAR was estimated as 19.3% (95% CI: 15.51–23.86%). The age effects and the relative infectivity of the illness period vs. the incubation period remained qualitatively similar (Table 4; Appendix Table S6).', 'cite_spans': [], 'section': 'Results', 'ref_spans': [{'start': 281, 'end': 288, 'mention': 'Table 2', 'ref_id': 'TABREF1'}, {'start': 559, 'end': 566, 'mention': 'Table 4', 'ref_id': 'TABREF3'}]}, {'text': 'We characterized the spatial and temporal epidemiology of the COVID-19 epidemic in Guangzhou, the most populated city in southern China. We assessed several aspects of the transmissibility of the disease, such as the time-varying effective reproductive number at the population level, household and non-household SARs, and local reproductive number among close contacts. In addition, we assessed the relative infectivity of the incubation period in comparison to the illness period of COVID-19 cases, and explored the role of age in susceptibility and infectivity. We found that COVID-19 patients in their incubation period were at least as infectious as during their illness period, and that that elderly people are the most susceptible to infection by the SARS-CoV-2 virus, not just severe disease.', 'cite_spans': [], 'section': 'Discussion', 'ref_spans': []}, {'text': 'When household contact is defined by living at the same address, our model-based household SAR estimate for the Guangzhou contact-tracing data is 19.3%, which is higher than the non-model-based estimates 14.9% for the Shenzhen and 10.2% for Guangzhou.13,14 These non-model-based estimates are non-primary attack rates rather than the actual household SARs. In general, a non-primary attack rate is higher than an SAR as the former does not exclude tertiary transmission and untraced exposure. However, these non-primary attack rate estimates reflect transmissibility under control measures such as case isolation, whereas our SAR estimate assumes exposure of a susceptible to the full infectious period of an infector, which is more epidemiologically relevant and generalizable.', 'cite_spans': [{'start': 251, 'end': 253, 'mention': '13', 'ref_id': 'BIBREF4'}, {'start': 254, 'end': 256, 'mention': '14', 'ref_id': 'BIBREF5'}], 'section': 'Discussion', 'ref_spans': []}, {'text': 'Model-based household SAR estimates for the severe acute respiratory syndrome-related coronavirus (SARS-CoV) or the Middle East respiratory syndrome-related coronavirus (MERS-CoV) are not available, but a few studies have reported non-primary attack rates in the household or comparable settings. For SARS-CoV, the non-primary attack rate was estimated to be 4.6% (95%CI: 2.3–8.9%) in Beijing, 8% (95% CI: 6.9–9.1%) in Hong Kong, 6.2% (95% CI: 3.9–8.6%) in Singapore, and 10.2% (95%CI: 6.7–23.5%) in Toronto.21–23 Information on household transmissibility of MERS-CoV is less clear. A multi-city household study in Saudi Arabia found the non-primary attack rate to be 4% (95% CI:2–7%).24 In an outbreak among female workers who lived in expatriate dormitory consisting of 24 villas in Riyadh, Saudi Arabia, 19 out of 828 were infected. If each villa (24–50 inhabitants) is considered a household, we estimated the non-primary attack rate to be 5.1%.25 We conclude that SARS-CoV-2 is more transmissible than both SARS-CoV and MERS-CoV in households.', 'cite_spans': [{'start': 508, 'end': 510, 'mention': '21', 'ref_id': 'BIBREF13'}, {'start': 511, 'end': 513, 'mention': '23', 'ref_id': 'BIBREF15'}, {'start': 685, 'end': 687, 'mention': '24', 'ref_id': 'BIBREF16'}, {'start': 949, 'end': 951, 'mention': '25', 'ref_id': 'BIBREF17'}], 'section': 'Discussion', 'ref_spans': []}, {'text': 'We are the first to quantify the infectivity of COVID-19 patients during their incubation period using contact-tracing data, but qualitative evidence has been noted before. Transmission to secondary cases during the incubation period of the primary case has been noted in both Germany and China.6, 26 In our unpublished analysis of a separate dataset of case clusters in China, the mean serial interval (time between symptom onsets of a primary and a secondary case) was shorter than the mean incubation period, suggesting that infectivity during the incubation period is not trivial.27 In contrast, shedding of SARS-CoV peaks 6–11 days after illness onset, and asymptomatic and mild MERS-CoV cases were thought to transmit inefficiently, indicating the key role of symptoms in the transmission of these two coronaviruses.28–30 This finding has a profound implication on prevention and control strategies, e.g., the necessity of testing asymptomatic close contacts of COVID-19 cases, and the importance of wearing facial mask regardless of symptoms in order to curb the pandemic.', 'cite_spans': [{'start': 295, 'end': 296, 'mention': '6', 'ref_id': 'BIBREF26'}, {'start': 298, 'end': 300, 'mention': '26', 'ref_id': 'BIBREF18'}, {'start': 584, 'end': 586, 'mention': '27', 'ref_id': 'BIBREF19'}, {'start': 822, 'end': 824, 'mention': '28', 'ref_id': 'BIBREF20'}, {'start': 825, 'end': 827, 'mention': '30', 'ref_id': 'BIBREF23'}], 'section': 'Discussion', 'ref_spans': []}, {'text': 'We estimated the local reproductive number to be relatively low, around 0.48 (Table 3), which is consistent with the average level of Rt (Figure 2). We projected that, without isolation of cases and their contacts, the local reproductive number would have been about 30% higher to reach 0.62 in our primary analysis setting. This moderate effect of isolation is partly due to high infectivity during the incubation period, that is, many secondary cases had been infected before or shortly after the symptom onset of the primary cases before the primary cases were identified and isolated. This observation also implies that the low reproductive number in Guangzhou possibly resulted more from policies restricting human movement, limiting the number of contacts of COVID-19 cases during their entire infectious period including the incubation period, than from the detection and isolation of cases and close contacts.', 'cite_spans': [], 'section': 'Discussion', 'ref_spans': [{'start': 138, 'end': 146, 'mention': 'Figure 2', 'ref_id': 'FIGREF1'}, {'start': 78, 'end': 85, 'mention': 'Table 3', 'ref_id': 'TABREF2'}]}, {'text': 'Our analysis has several limitations. When the mean incubation period is assumed relatively long, the model suggests that the incubation period could be much more infectious than the illness period, which differs from our experience with most respiratory pathogens. This effect could be due to the fact that exposure of close contacts to cases was mostly truncated soon after symptom onset of cases, a consequence of efficient case-finding and isolation of both cases and close contacts all over China after Wuhan’s lockdown. When few transmissions occurred during the illness period, a large bias could happen by chance, as a long incubation period in the model would attribute more transmissions to the incubation period. In addition, we were not able to reliably quantify the infectivity of asymptomatic infections given the limited number of asymptomatic infections, 0.9% and 2.9% among primary and non-primary cases, and our assumption that asymptomatic infections share the same infectivity as the symptomatic cases during their incubation period may not be realistic.', 'cite_spans': [], 'section': 'Discussion', 'ref_spans': []}, {'text': 'The infectiousness of COVID-19 patients during their incubation period is alarming. This potential of “silent” transmission substantially increased the difficulty in curbing the ongoing pandemic. More and more countries now have realized that active case finding, testing and isolation alone may be inadequate and should be used in conjunction with general human movement restrictions such as stay-at-home policies. On the other hand, the efficient household transmissibility of SARS-CoV-2 should be given full consideration when stay-at-home is ordered. It is crucial to advocate self-monitoring of symptoms and timely self-isolation from other household members when either symptoms emerge or potential exposure occurs. Providing comfortable facilities for exposed contacts to quarantine or for mild cases to isolate away from their families could be a valuable strategy for limiting onward transmission within households.', 'cite_spans': [], 'section': 'Discussion', 'ref_spans': []}], 'paragraphs': [{'text': 'Our analysis has several limitations. When the mean incubation period is assumed relatively long, the model suggests that the incubation period could be much more infectious than the illness period, which differs from our experience with most respiratory pathogens. This effect could be due to the fact that exposure of close contacts to cases was mostly truncated soon after symptom onset of cases, a consequence of efficient case-finding and isolation of both cases and close contacts all over China after Wuhan’s lockdown. When few transmissions occurred during the illness period, a large bias could happen by chance, as a long incubation period in the model would attribute more transmissions to the incubation period. In addition, we were not able to reliably quantify the infectivity of asymptomatic infections given the limited number of asymptomatic infections, 0.9% and 2.9% among primary and non-primary cases, and our assumption that asymptomatic infections share the same infectivity as the symptomatic cases during their incubation period may not be realistic.', 'id': '00026'}], 'title': 'Household Secondary Attack Rate of COVID-19 and Associated Determinants', 'sha': '', 'doi': '10.1101/2020.04.11.20056010', 'score': 9.225299835205078}, {'rank': 182, 'abstract': 'Air travel has a decisive role in the spread of infectious diseases at the global level. We present a methodology applied during the early stages of the COVID-19 pandemic that uses detailed aviation data at the final destination level in order to measure the risk of the disease spreading outside China. The approach proved to be successful in terms of identifying countries with a high risk of infected travellers and as a tool to monitor the evolution of the pandemic in different countries. The high number of undetected or asymptomatic cases of COVID-19, however, limits the capacity of the approach to model the full dynamics. As a result, the risk for countries with a low number of passengers from Hubei province appeared as low. Globalization and international aviation connectivity allow travel times that are much shorter than the incubation period of infectious diseases, a fact that raises the question of how to react in a potential new pandemic.', 'body_text': [{'text': 'The recent pandemic caused by the novel coronavirus COVID-19 has once again brought epidemiological modeling into the limelight. Especially during the early stages of the virus propagation—when data were limited and highly uncertain—model projections were a major input for understanding the magnitude and potential risks of the outbreak. Apart from predicting the dynamics of transmission within a specific population, particular emphasis was put on quantifying the risk of the disease spreading beyond the area of its original focus in Wuhan, China.', 'cite_spans': [], 'section': '1. Introduction', 'ref_spans': []}, {'text': 'The contribution of transport—and aviation in particular—has been shown to be significant in the spread of airborne diseases such as SARS, MERS [1,2] and the influenza A H1N1 [3], as well as other infectious such as Ebola [4] or even vector-borne diseases such as Dengue [5] or Malaria [6].', 'cite_spans': [{'start': 145, 'end': 146, 'mention': '1', 'ref_id': 'BIBREF0'}, {'start': 147, 'end': 148, 'mention': '2', 'ref_id': 'BIBREF11'}, {'start': 176, 'end': 177, 'mention': '3', 'ref_id': 'BIBREF22'}, {'start': 223, 'end': 224, 'mention': '4', 'ref_id': 'BIBREF29'}, {'start': 272, 'end': 273, 'mention': '5', 'ref_id': 'BIBREF30'}, {'start': 287, 'end': 288, 'mention': '6', 'ref_id': 'BIBREF31'}], 'section': '1. Introduction', 'ref_spans': []}, {'text': 'Spatio-temporal dynamics also have a major role. Social interactions and personal contact patterns at either the origin of the disease or areas connected to it affect the speed and range of prevalence across different geographic zones [7]. Budd et al. [8] made a historical review of journey times and the incubation period of selected infectious diseases. Travel times have decreased drastically and even intercontinental trips have a duration that is a fraction of the incubation period of several diseases. Coming to the case of the COVID-19 outbreak, no intercontinental flights from Wuhan were possible before 2012, but in 2020 direct flights were available to London, Paris, Rome, Moscow, New York City, San Francisco, Istanbul, Dubai, Sydney, and numerous other international destinations. As an example, a direct flight from Wuhan to Rome, Italy takes 12 h 15 m and is available three times weekly.', 'cite_spans': [{'start': 236, 'end': 237, 'mention': '7', 'ref_id': 'BIBREF32'}, {'start': 253, 'end': 254, 'mention': '8', 'ref_id': 'BIBREF33'}], 'section': '1. Introduction', 'ref_spans': []}, {'text': 'Several studies have utilized air travel data for the estimation of disease importation risks. The Vector-Borne Disease Airline Importation Risk (VBD-AIR) tool allows the combination of disease information and air travel networks to produce a user-defined global map of risk distribution [9]. A generic tool that allows the estimation of the median early disease arrival time from around the world using air transport schedules was proposed by [10]. Air travel patterns were also employed by Bajardi et al. [11] in order to theoretically model the potential impacts of travel restrictions on the spread of an epidemic. Nevertheless, while travel and trade have been shown as relevant for the international spread of the 2009 A/H1N1 influenza, the slow deployment of control measures in countries with lower healthcare capacities led to spatial imbalances [12]. Air transport data have been used indirectly to measure the effective distance and the relative arrival time of a disease outbreak, with promising results when tested on historical data [13]. Based on the same premises, producing near real-time or now-casting predictions on the international spread of COVID-19 appeared to be based on scientifically sound principles [14].', 'cite_spans': [{'start': 289, 'end': 290, 'mention': '9', 'ref_id': 'BIBREF34'}, {'start': 445, 'end': 447, 'mention': '10', 'ref_id': 'BIBREF1'}, {'start': 508, 'end': 510, 'mention': '11', 'ref_id': 'BIBREF2'}, {'start': 856, 'end': 858, 'mention': '12', 'ref_id': 'BIBREF3'}, {'start': 1048, 'end': 1050, 'mention': '13', 'ref_id': 'BIBREF4'}, {'start': 1230, 'end': 1232, 'mention': '14', 'ref_id': 'BIBREF5'}], 'section': '1. Introduction', 'ref_spans': []}, {'text': 'However, most expectations about the ability to predict the global spread of COVID-19 proved misleading or of limited use.', 'cite_spans': [], 'section': '1. Introduction', 'ref_spans': []}, {'text': 'This includes our own work, which we present here in order to discuss the reasons why most projections missed important aspects of the disease propagation dynamics. In our opinion, and as a part of the scientific process, it is important to publish and discuss even the negative results of research activities. The lessons learned from unsuccessful applications can be valuable for future work, either by improving current practices or by re-examining risk assessment from a new perspective. Especially in the case of predictive modeling, the wide body of literature that presents successful ex-post examples of possible value may give the false impression that the spatial aspects of disease propagation are more easily predictable than what is actually possible in reality.', 'cite_spans': [], 'section': '1. Introduction', 'ref_spans': []}, {'text': 'The question this paper is trying to answer is whether air passenger traffic alone can provide early information on the potential distribution of contagious diseases at the global level. Based on the experience from the SARS, MERS and Ebola epidemics—which suggests that travel patterns are a good predictor of spatial evolution of an epidemic—we mapped the expected distribution of COVID-19 at the global level using now-casted origin-destination estimates of aviation passengers and near real-time data on the spatial distribution of diagnosed infections. The number of reported cases for each possible destination outside the initial outburst area was compared to the estimated traffic intensity between Wuhan and the specific destination. The resulting ratio was used as an indicator of the expected rate of infection of aviation passengers and as a means of comparison of the evolution of the pandemic worldwide. The approach gave promising results during the initial phases of the pandemic, but underestimated the risks associated with local infections at several distant destinations and the development of numerous secondary foci of the disease.', 'cite_spans': [], 'section': '1. Introduction', 'ref_spans': []}, {'text': 'The structure of the paper is as follows: Section 2 describes the main elements of the modeling approach followed. Section 3 discusses the results and Section 4 analyzes the factors limiting the predictive capacity. Finally, Section 5 summarizes the conclusions of this work.', 'cite_spans': [], 'section': '1. Introduction', 'ref_spans': []}, {'text': 'The first step of the approach was the estimation of the number of passengers leaving Hubei province by plane and reaching a specific final destination, including possible connections. Wuhan is an aviation hub in central China [15]. Wuhan and Hubei province are well connected with neighboring provinces by rail and coach [16]. We used monthly data from the SABRE database [17,18] to track the number of passengers to all possible destinations using historical data from 2016 to November 2019, the last month for which data were available at the time this paper was written. Almost 1 million passengers flew out of Wuhan every month. While China represents 89.9% of the final destinations, indirect connections linked Hubei province to 23,532 distinct airports across the world. Air transport activity from Wuhan, as across the aviation sector in general, is quite volatile with important fluctuations over time and a high seasonal variation [19]. In addition, January and early February traffic is greatly affected by the Chinese New Year’s holiday, both in terms of reduced economic activity and increased tourism.', 'cite_spans': [{'start': 228, 'end': 230, 'mention': '15', 'ref_id': 'BIBREF6'}, {'start': 323, 'end': 325, 'mention': '16', 'ref_id': 'BIBREF7'}, {'start': 374, 'end': 376, 'mention': '17', 'ref_id': 'BIBREF8'}, {'start': 377, 'end': 379, 'mention': '18', 'ref_id': 'BIBREF9'}, {'start': 943, 'end': 945, 'mention': '19', 'ref_id': 'BIBREF10'}], 'section': '2.1. Analyzing Air Transport Patterns ::: 2. Modeling the Risk of Coronavirus Spread Using Aviation Data', 'ref_spans': []}, {'text': 'In order to create a more realistic picture of the number of passengers who left Wuhan during January 2020, we developed a now-casting model to estimate the expected numbers of passengers based on the trends revealed by data until November 2019. We applied an ARIMA (Autoregressive Integrated Moving Average) model for each of the 23,532 possible destinations. The model was estimated using the forecast package in R [20]. The decomposition of the monthly time series from January 2016 to November 2019 allows the estimation of the contribution to the observed number of passengers for each destination of the underlying trend in activity, the seasonal variation and the random effects. The examples in Figure 1 correspond to three characteristic destinations: Beijing (the top destination from Wuhan), Singapore (the top destination outside China) and Paris (the top destination outside Asia). It is evident that the three examples follow different patterns as regards both the trend in demand and the monthly variation, due to the different factors that affect international air transport demand [21].', 'cite_spans': [{'start': 418, 'end': 420, 'mention': '20', 'ref_id': 'BIBREF12'}, {'start': 1099, 'end': 1101, 'mention': '21', 'ref_id': 'BIBREF13'}], 'section': '2.1. Analyzing Air Transport Patterns ::: 2. Modeling the Risk of Coronavirus Spread Using Aviation Data', 'ref_spans': [{'start': 703, 'end': 711, 'mention': 'Figure 1', 'ref_id': 'FIGREF0'}]}, {'text': 'The model we applied uses a rolling window of 36 months of historical data and a prediction step of 2 months ahead. The results were validated by comparing the model estimates with the ground truth for the last 11 months for which data were available (January 2019 to November 2019). The accuracy of the model at the individual destination level can be considered high, with an R2 of 0.799, MAE of 159.04 and RMSE of 906.54. A number of graphs that show how the estimates match data at the country or continent level are provided in the Appendix A.', 'cite_spans': [], 'section': '2.1. Analyzing Air Transport Patterns ::: 2. Modeling the Risk of Coronavirus Spread Using Aviation Data', 'ref_spans': []}, {'text': 'Figure 2 and Table 1 summarize the results of the model at the aggregate level. For illustration purposes, China is split into three different groups based on distance from Wuhan. The distance bands (below 700 km, 700 to 1000 km and over 1000 km) correspond to the ranges where aviation has a low, moderate and high modal share [22]. The share of Chinese destinations in January 2019 was higher than their average share during the 2016–2019 period (90.5% and 89.9%, respectively), but according to the model estimates, in January 2020 it was expected to decrease to 89.4%. This would be the result of switching between various destinations, especially the increase in the number of passengers to Southeast and Far East Asia. The estimates, based on the underlying trends visible in the data available until November 2019, suggest that the share of the trips with a final destination outside Asia would remain low in January 2020, at around 1.8% of the total.', 'cite_spans': [{'start': 329, 'end': 331, 'mention': '22', 'ref_id': 'BIBREF14'}], 'section': '2.1. Analyzing Air Transport Patterns ::: 2. Modeling the Risk of Coronavirus Spread Using Aviation Data', 'ref_spans': [{'start': 0, 'end': 8, 'mention': 'Figure 2', 'ref_id': 'FIGREF1'}, {'start': 13, 'end': 20, 'mention': 'Table 1', 'ref_id': 'TABREF0'}]}, {'text': 'The second building block of the approach is the comparison with the numbers of diagnosed cases of COVID-19 at each potential destination. This indicator, as opposed to the number of deaths, hospitalized or recovered patients, was at the moment the only one with presumably reliable enough data in real time to derive early warning indicators on the international spread of the disease. The data we used were available, on a daily basis, through the Johns Hopkins Resource Center [23] and are summarized in Table 2, Table 3, Table 4 and Table 5.', 'cite_spans': [{'start': 481, 'end': 483, 'mention': '23', 'ref_id': 'BIBREF15'}], 'section': '2.2. Combining Disease Data with Air Transport Activity ::: 2. Modeling the Risk of Coronavirus Spread Using Aviation Data', 'ref_spans': [{'start': 507, 'end': 514, 'mention': 'Table 2', 'ref_id': 'TABREF1'}, {'start': 516, 'end': 523, 'mention': 'Table 3', 'ref_id': 'TABREF2'}, {'start': 525, 'end': 532, 'mention': 'Table 4', 'ref_id': 'TABREF3'}, {'start': 537, 'end': 544, 'mention': 'Table 5', 'ref_id': 'TABREF4'}]}, {'text': 'The dynamics of the spread of a contagious disease are usually analyzed using either a compartmental or a phenomenological model. In the case of compartmental models, the population is divided into various groups depending on each individual’s state of infection. Such models include the SIR model (Susceptible, Infected, Recovered) and its variations, and consist of a set of differential equations that describe the change over time of the number of individuals in each compartment. In the case of the basic SIR model, the rate of change of the number of infections is expected to be:dI/dt = β I S/N − γ I(1)\\nwhere N is the total population, I is the number of infected individuals, S is the population still susceptible to infection (total minus infected and recovered), while β and γ are parameters directly linked to the basic reproductive number of the disease R0 = β/γ.', 'cite_spans': [], 'section': '2.3. A Brief Review of Epidemiological Models ::: 2. Modeling the Risk of Coronavirus Spread Using Aviation Data', 'ref_spans': []}, {'text': 'Phenomenological models, on the other hand, attempt to simulate the disease spread by fitting data and assumptions into standard curves. A frequently used approach is the logistic equation:dI/dt = r It−1 (1 − It−1/K).(2)', 'cite_spans': [], 'section': '2.3. A Brief Review of Epidemiological Models ::: 2. Modeling the Risk of Coronavirus Spread Using Aviation Data', 'ref_spans': []}, {'text': 'Such models rely on two parameters: the intrinsic infection rate, r, and the final epidemic size, K [24].', 'cite_spans': [{'start': 101, 'end': 103, 'mention': '24', 'ref_id': 'BIBREF16'}], 'section': '2.3. A Brief Review of Epidemiological Models ::: 2. Modeling the Risk of Coronavirus Spread Using Aviation Data', 'ref_spans': []}, {'text': 'A more recent approach, the Richards curves [25], improves the matching with actual epidemiological curves, especially the early slower growth, but requires the estimation of two additional parameters:dI/dt = r Ipt−1 (1 − It−1)/Kα)(3)\\nwhere p indicates the similarity to the exponential growth curve (ranging from 0 for constant to 1 for exponential), and α is a parameter that shifts the timing of the peak.', 'cite_spans': [{'start': 45, 'end': 47, 'mention': '25', 'ref_id': 'BIBREF17'}], 'section': '2.3. A Brief Review of Epidemiological Models ::: 2. Modeling the Risk of Coronavirus Spread Using Aviation Data', 'ref_spans': []}, {'text': 'During the early stages of an epidemic, when the population dynamics in terms of deaths and recoveries are not yet in play, the compartmental and the phenomenological approaches are in practice equivalent and the cumulative number of infections over time can be approximated by:(4)limt→∞  I(t)=(1−γβ) N', 'cite_spans': [], 'section': '2.3. A Brief Review of Epidemiological Models ::: 2. Modeling the Risk of Coronavirus Spread Using Aviation Data', 'ref_spans': []}, {'text': 'Estimating the value of R0 (and thus the values of β and γ), or even the population N that should be used as a reference, is extremely challenging—if not outright futile—at the early stages of the spread of a new disease. The uncertainty and questionable accuracy of the scarce data available in the beginning of an outburst would lead to epidemiological models that are practically useless for any prediction.', 'cite_spans': [], 'section': '2.3. A Brief Review of Epidemiological Models ::: 2. Modeling the Risk of Coronavirus Spread Using Aviation Data', 'ref_spans': []}, {'text': 'Nevertheless, monitoring and comparing the spread of the disease in different locations can still provide value to public health authorities. In the approach we present here, we calculated the ratio of infected individuals to the estimated number of travellers from Hubei province to each destination. Assuming that all travellers leaving Wuhan have the same probability of being infected regardless of their destination, the cumulative number of infected individuals at time t reaching destination j would be expected to be:I(t)j = nj(t) q(5)\\nwhere nj(t) is the total number of passengers from the origin of the infection (Hubei province in this case) to each destination j, and q is the apparent ratio of infected passengers to total passengers:(6)q=I(t)0 ∑jn0(t)=I(t)0v0N0\\nwhere I(t)0 is the cumulative number of infected at the origin, sum(n0(t)) is the total number of passengers from the origin, N0 is the population at the origin and v0 is the ratio of travellers to population at the origin.', 'cite_spans': [], 'section': '2.4. Modelling the Spread of the Disease Based of Air Trip Patterns ::: 2. Modeling the Risk of Coronavirus Spread Using Aviation Data', 'ref_spans': []}, {'text': 'While neither formulation of Equation (6) can be directly estimated, it is sufficient to indicate that q is the same regardless of the destination. We should therefore expect, transforming Equation (5), that the ratio:Q = I(t)j/nj(t)(7)\\nshould be constant for all destinations and across time.', 'cite_spans': [], 'section': '2.4. Modelling the Spread of the Disease Based of Air Trip Patterns ::: 2. Modeling the Risk of Coronavirus Spread Using Aviation Data', 'ref_spans': []}, {'text': 'We estimated q, the ratio of infected passengers to total passengers, at the province and territory level within China, and at the country level for the rest of the world. As described in Section 2, we used the estimated number of aviation passengers for January 2020 and transformed them to an equivalent daily volume. The q indicator was calculated for four different points in time at weekly intervals: on the 3rd, 10th, 17th and 24th of February 2020, corresponding to t values of 34, 41, 48 and 54 days, respectively (assuming t = 0 as 31/12/2019, the date of the first reported case in the Johns Hopkins database). The number of aviation passengers to the provinces within less than 700 km from Wuhan is virtually null (Hubei province itself, as well as Anhui, Hebei, Henan, Hunan and Jiangxi), therefore the approach is not applicable in those cases.', 'cite_spans': [], 'section': '3. Discussion of the Results', 'ref_spans': []}, {'text': 'Figure 3 shows the results for Chinese provinces between 700 and 1000 km from Wuhan. The q indicator ranges from 4 to 10 infected cases per one thousand passengers in most cases, except Zhejiang and especially Jiangsu (of which Nanjing, a major city, is only 3 h from Wuhan by train). Apart from the obvious possible differences in diagnosing and reporting, the main reason for discrepancies in the case of those provinces close to Wuhan and Hubei is their high share of train, bus and car travel to and from Wuhan.', 'cite_spans': [], 'section': '3. Discussion of the Results', 'ref_spans': [{'start': 0, 'end': 8, 'mention': 'Figure 3', 'ref_id': 'FIGREF2'}]}, {'text': 'On the other hand, the picture is more homogeneous for Chinese provinces and territories over 1000 km from Wuhan (Figure 4). For such distances, air travel is practically the only option for passengers and the q indicator has a range of between 2 and 6 cases per thousand passengers. The two outliers are Tibet (with a single case in the period, but also a very low number of estimated passengers) and Heilongjiang, where the number of cases was proportionally higher already on February 3, possibly as a result of the province having entered the local transmission phase early.', 'cite_spans': [], 'section': '3. Discussion of the Results', 'ref_spans': [{'start': 114, 'end': 122, 'mention': 'Figure 4', 'ref_id': 'FIGREF3'}]}, {'text': 'As regards other countries apart from China (Figure 5), the indicator had a range of between 0 and 6 cases per thousand passengers until 17 February, with Germany being an outlier (but with only 16 reported cases). Up to that point, the comparison of the indicators for distant Chinese destinations (more than 1000 km from Wuhan) and countries outside China confirmed the hypothesis of diagnosed cases being proportional to the number of passengers arriving from Wuhan. Even though the absolute number of cases was very low and the ratios were unstable, the global distribution patterns seemed to follow the expected dynamics and to correspond to the air transport patterns.', 'cite_spans': [], 'section': '3. Discussion of the Results', 'ref_spans': [{'start': 45, 'end': 53, 'mention': 'Figure 5', 'ref_id': 'FIGREF4'}]}, {'text': 'Given that the majority of final destinations of passengers from Wuhan were in Mainland China, the results implied that the appearance of infected cases would follow the same pattern as that of aviation trips (Figure 1), with the risk for countries outside Asia being low. Additionally, since China had implemented an extended quarantine since the last week of January, these patterns suggested that the risk for additional cases in the rest of the world would be limited, as infections in other Chinese provinces and territories were kept under control. An additional observation that could be made at that moment was that the reporting quality in China was comparable to that of most neighboring countries, with indicator levels being at similar levels. However, surprisingly a number of countries—including Indonesia and the Philippines—report a very low number of cases compared to the expected number of passengers from Hubei. It should be noted, though, that the number of tests performed in several countries was very low and the actual number of symptomatic or asymptomatic cases may actually be much higher.', 'cite_spans': [], 'section': '3. Discussion of the Results', 'ref_spans': [{'start': 210, 'end': 218, 'mention': 'Figure 1', 'ref_id': 'FIGREF0'}]}, {'text': 'A dramatic change can be seen in the data for the 24th of February in a number of countries outside China (Figure 5). Suddenly, the number of cases in South Korea, Japan and Iran presented a pronounced spike. While South Korea and Japan were among the countries where the risk had already been identified, Iran was a case that the travel pattern analysis had completely missed. The model estimates on the number of travellers from Hubei to Iran were between 33 and 236 per month, a value too low to raise any alarm. These three countries were only the beginning of the pandemic at the worldwide level that in the course of the following 8 weeks infected more than 3 million people and caused more than 200 thousand deaths (as of 27 April 2020).', 'cite_spans': [], 'section': '3. Discussion of the Results', 'ref_spans': [{'start': 107, 'end': 115, 'mention': 'Figure 5', 'ref_id': 'FIGREF4'}]}, {'text': 'It is now known that the main reason for the rapid expansion in countries outside Asia was the appearance of local foci that were not detected initially, and the subsequent spread of the virus from them both locally and internationally. This evolution of the pandemic revealed a curious pattern: the countries that had high levels of interaction with China realized the level of risk early and applied strict control measures, resulting in a comparatively limited spread of the disease. Whereas countries that appeared to have a lower risk—since they were not in such direct contact with the area of the primary focus of the pandemic—eventually reached disease levels that exceeded those in the original focus in China (especially the USA, Spain and Italy). In that sense, our approach appears as sufficient to capture the true positives, but—as all approaches that we are aware of—may underestimate risk and lead to false negatives with a high impact. This is a general weakness of any risk assessment model that we try to explain and discuss in the following sections.', 'cite_spans': [], 'section': '3. Discussion of the Results', 'ref_spans': []}, {'text': 'The capacity of aviation activity to predict the appearance of infected cases can be affected by various factors that may influence the ratio of infected passengers reaching a specific destination. In the approach presented here, we assumed—for lack of data—that the socio-economic and demographic profile of travellers to all destinations is the same. The assumed profile includes the ratio of local to visiting travellers and their age distribution, as well as their income or professional activity distribution.', 'cite_spans': [], 'section': '4. Limitations to Predictive Capacity', 'ref_spans': []}, {'text': 'Most probably, the infection rates of the local population in Hubei province are different than the one for visitors, mainly because of the difference in terms of the total time spent in the origin of the pandemic and—as a result—the period of exposure. The analysis of exportations of symptomatic cases with MERS-CoV in the period 2013–2015 [26] differentiated between locals and visitors to the Middle East, the center of the MERS outbreak. The exposure time of visitors was estimated using tourism statistics, while the exposure of locals was assumed to be equal to the upper bound of the disease incubation period (10.2 days). A model-based study of frequent flyers suggests that the probability that an asymptomatic infected person will make an international flight is higher for high-frequency travellers than for low-frequency ones [27]. One would therefore expect differences in the infection rates between destinations that serve mainly visitors to Wuhan (e.g., flights to Europe) and destinations that serve a higher proportion of locals (e.g., flights to destinations in China with employees returning to work).', 'cite_spans': [{'start': 343, 'end': 345, 'mention': '26', 'ref_id': 'BIBREF18'}, {'start': 840, 'end': 842, 'mention': '27', 'ref_id': 'BIBREF19'}], 'section': '4. Limitations to Predictive Capacity', 'ref_spans': []}, {'text': 'The socio-economic profile of the travellers can also affect the probability of super-spreaders, individuals who due to the type or intensity of their social interactions can transmit the disease to a disproportionate number of others. A higher frequency of infections in health care settings has been observed for SARS and MERS [28]. To what concerns travel, frequent flyers have been shown to have a higher probability of being super-spreaders [1]. Visitors to Wuhan, especially ones for business purposes, probably had a higher number of social interactions at the origin than the average local inhabitant (as well as at the destination, increasing even more the probabilities of spreading the disease). A similar pattern may have been in place in the propagation from the secondary focus of Milan, where a large number of foreign visitors to commercial exhibitions or sport events may have caused the spread to other countries.', 'cite_spans': [{'start': 330, 'end': 332, 'mention': '28', 'ref_id': 'BIBREF20'}, {'start': 447, 'end': 448, 'mention': '1', 'ref_id': 'BIBREF0'}], 'section': '4. Limitations to Predictive Capacity', 'ref_spans': []}, {'text': 'Infection during flights has been found to have a significant probability. Even though aircraft are equipped with High-Efficiency Particulate Air (HEPA) filters, which reduce the risk of transmission through the air circulation system, several studies have demonstrated that infection rates can be as high as 4.3% in a 9-h flight, and that the risk does not depend on the relative seating of contagious passengers and passengers infected in-flight [29]. Mangili and Gendreau already discussed the need for additional work on in-flight transmissions [30], but insufficient data have been made available in the meanwhile to allow a more robust conclusion.', 'cite_spans': [{'start': 449, 'end': 451, 'mention': '29', 'ref_id': 'BIBREF21'}, {'start': 550, 'end': 552, 'mention': '30', 'ref_id': 'BIBREF23'}], 'section': '4. Limitations to Predictive Capacity', 'ref_spans': []}, {'text': 'The diagnostic capacity and protocols applied in airport controls have been discussed using the experience from SARS [31,32]. Grout et al. [33] identified several inconsistencies in infection control measures and major differences in how national and international legislation is enforced by airlines and airports. The differences in the diagnosing quality may affect the number of cases reported. However, while focusing on controls at the point of entry may improve the identification of travellers with symptoms, in the case of COVID-19 this strategy may have given a false sense of security.', 'cite_spans': [{'start': 118, 'end': 120, 'mention': '31', 'ref_id': 'BIBREF24'}, {'start': 121, 'end': 123, 'mention': '32', 'ref_id': 'BIBREF25'}, {'start': 140, 'end': 142, 'mention': '33', 'ref_id': 'BIBREF26'}], 'section': '4. Limitations to Predictive Capacity', 'ref_spans': []}, {'text': 'Climate and local environmental conditions may also affect the speed of local transmission and can lead to deviations from the expected number of infections. Such an association has been suggested for SARS and MERS [34], but has not yet been demonstrated in the case of COVID-19. It is still not clear how the virus reacts to temperature or air pollution, especially particulates. Both aspects will probably be researched extensively in the future.', 'cite_spans': [{'start': 216, 'end': 218, 'mention': '34', 'ref_id': 'BIBREF27'}], 'section': '4. Limitations to Predictive Capacity', 'ref_spans': []}, {'text': 'The low level of risks suggested by both aviation data and airport controls can distract attention from what—in hindsight—proved to be the main reason the pandemic reached such levels: asymptomatic cases. The number and transmission capacity of asymptomatic cases was a “known unknown” from the beginning of the outbreak. Since COVID-19 has a long incubation period (up to or even exceeding 14 days) and tests were not yet available, the number of asymptomatic carriers could not be measured or estimated even in later stages of the pandemic.', 'cite_spans': [], 'section': '4. Limitations to Predictive Capacity', 'ref_spans': []}, {'text': 'The frequently cited work by Colizza et al. (2007) explored the predictability of stochastic dynamic models on global disease distribution and concluded that:', 'cite_spans': [], 'section': '5. Conclusions', 'ref_spans': []}, {'text': '“… stochastic fluctuations are less important than the constraints imposed by the transport network structure that imposes an overall pattern to the epidemic evolution. In this respect, global properties provide insights on the general features of the epidemic spreading in relation with the underlying network but do not provide adequate information on the predictability of the resulting pattern” [35].', 'cite_spans': [{'start': 400, 'end': 402, 'mention': '35', 'ref_id': 'BIBREF28'}], 'section': '5. Conclusions', 'ref_spans': []}, {'text': 'Our empirical results from modeling the spread of COVID-19 suggest the opposite, at least as regards the first part of the statement. The stochastic fluctuations of the factors that affect the air transport-related spread of the disease introduce high levels of uncertainty and consequently limit the predictability of the resulting spatial pattern.', 'cite_spans': [], 'section': '5. Conclusions', 'ref_spans': []}, {'text': 'Aviation activity analyzed with the methodology we presented here can provide an early signal for risk. Such information is important for destinations with a high number of passengers from the origin of the pandemic, since it can correctly identify potential risks. The indicator used is also a convenient measure to compare the evolution across countries and identify changes in the direction of the trends.', 'cite_spans': [], 'section': '5. Conclusions', 'ref_spans': []}, {'text': 'Soon, however, if a large enough number of disease carriers has travelled undetected, the easiness of air travel can convert a local outburst into a global pandemic. In that case, the value of aviation-based distribution models is limited, due to their systematic underestimation of risk for destinations with low passenger volume from the origin or interactions with a secondary focus of infection.', 'cite_spans': [], 'section': '5. Conclusions', 'ref_spans': []}, {'text': 'Strict measures (quarantine, flight cancelations, travel restrictions) can reduce risks significantly but come with the cost of a large social, economic and political risk. Airport controls, especially in the case of novel diseases with unknown symptoms or high asymptomatic shares, can prove inadequate and may even give a false sense of security. International collaboration and exchange of information are important elements in managing risk. While the random effects of disease propagation cannot be controlled, monitoring the situation at the global level using reliable data can significantly improve the reaction of public authorities.', 'cite_spans': [], 'section': '5. Conclusions', 'ref_spans': []}, {'text': 'As with any risk assessment approach, the evaluation of the actual risk and the costs of the possible risk mitigation strategies can pose delicate trade-offs. However, keeping in mind that modern aviation permits connecting any two points on the planet in less than 48 h, the chances for new virulent diseases spreading around the world are high and the need for better predictive tools becomes even more urgent.', 'cite_spans': [], 'section': '5. Conclusions', 'ref_spans': []}], 'paragraphs': [{'text': 'The low level of risks suggested by both aviation data and airport controls can distract attention from what—in hindsight—proved to be the main reason the pandemic reached such levels: asymptomatic cases. The number and transmission capacity of asymptomatic cases was a “known unknown” from the beginning of the outbreak. Since COVID-19 has a long incubation period (up to or even exceeding 14 days) and tests were not yet available, the number of asymptomatic carriers could not be measured or estimated even in later stages of the pandemic.', 'id': '00035'}], 'title': 'The Predictive Capacity of Air Travel Patterns during the Global Spread of the COVID-19 Pandemic: Risk, Uncertainty and Randomness', 'sha': '9ed380101fbf5651a76737abdc0038ceaf7a822f', 'doi': '10.3390/ijerph17103356', 'score': 9.218000411987305}, {'rank': 183, 'abstract': 'Summary Background Adolescents and young adults might play a key role in the worldwide spread of Coronavirus Disease 2019 (COVID-19) because they are more likely to be involved in overseas study, business, work, and travel. However, the epidemiological and clinical characteristics remain unknown. Methods We collected demographic, epidemiological, and clinical data from 46 confirmed COVID-19 patients aged 10 to 35 years from the Chongqing Three Gorges Central Hospital. Several key epidemiological parameters, asymptomatic cases, transmission to family members, and clinical characteristics at admission and during treatment were summarized. Results Of 46 confirmed patients, 14 patients (30.4%) were aged between 10 and 24 years, and 24 (52.2%) patients were male. The estimated mean incubation period was 6.6 days (95% confidence interval [CI] 4.4–9.6). The median serial interval was 1.9 days (95% CI 0.4–6.2). Three of the asymptomatic cases showed transmission to their family members. Only one patient was identified as a severe case at admission. The common symptoms at admission were dry cough (34, 81.0%) and fever (29, 69.1%). Nearly 60% of the patients showed ground-glass opacity on chest computed tomography. Three patients developed acute kidney injury during treatment. Most of the patients (78.3%) recovered and were discharged by the end of the follow-up. Conclusions This single-center study with a relatively small sample size showed that adolescent and young adult patients with COVID-19 had a long incubation period and a short serial interval. The transmission occurred from asymptomatic cases to family members. Fewer patients developed complications during treatment.', 'body_text': [{'text': 'The coronavirus disease COVID-19, a newly emerging infectious pneumonia with unknown causes, was firstly reported in Wuhan, Hubei Province, China in December 2019. The incidences of COVID-19 were rapidly reported across China because it occurred around the time of the Chinese spring festival. 1 Early epidemiological and clinical studies reported that the majority of patients were middle-aged or elderly individuals, a mean incubation period of 5.2 days (range 0-14 days), and a serial interval of 7.5 days (95% confidence interval [CI] 2-17 days). [2] [3] [4] The most common symptoms were fever, cough, and fatigue. [3] [4] [5] Most of the patients presented the abnormal chest computed tomography (CT) findings such as ground-glass opacity and bilateral patchy shadowing. [4] [5] [6] Patients older than 65 years were more likely to be severe cases and developed severe acute complications such as pneumonia, acute respiratory distress syndrome (ARDS), shock, and acute cardiac injury during treatment. [3] [4] [5] These studies provided essential evidence to guide the early medical screen, diagnosis of COVID-19 cases, isolation of suspected cases, and clinical treatments. However, with the rapid progress of COVID-19 many new characteristics have emerged, which need to augment the evidence. One new feature is that increasing numbers of younger patients were confirmed across China. One study from the Chinese Center for Disease Control and Prevention indicated that 4,168 (9%) of the patients through February 11, 2020 were younger than 30 years. 7 In addition, outbreaks of COVID-19 were reported worldwide. Several countries, such as South Korea, Japan, and Italy, are experiencing a sharp increase in confirmed cases of incident COVID-19. Younger individuals were more likely to be carriers of COVID-19 across countries, since they were more likely to be involved in overseas study, business, work, and travel. For example, in South Korea, 178 out of 431 confirmed cases, which were publicly announced on the website of the Ministry of Health and Welfare, were aged %35 years through March 1, 2020. 8 However, as far as we know no study has been specifically conducted to explore the epidemiological and clinical characteristics in younger patients with In this study, based on retrospective case series data, we aimed to estimate the key epidemiological characteristics and describe the clinical symptoms, treatments, and hospital outcomes for adolescents and young adults with COVID-19.', 'cite_spans': [{'start': 294, 'end': 295, 'text': '1', 'ref_id': 'BIBREF0'}, {'start': 551, 'end': 554, 'text': '[2]', 'ref_id': 'BIBREF1'}, {'start': 555, 'end': 558, 'text': '[3]', 'ref_id': 'BIBREF3'}, {'start': 559, 'end': 562, 'text': '[4]', 'ref_id': 'BIBREF4'}, {'start': 620, 'end': 623, 'text': '[3]', 'ref_id': 'BIBREF3'}, {'start': 624, 'end': 627, 'text': '[4]', 'ref_id': 'BIBREF4'}, {'start': 628, 'end': 631, 'text': '[5]', 'ref_id': 'BIBREF5'}, {'start': 777, 'end': 780, 'text': '[4]', 'ref_id': 'BIBREF4'}, {'start': 781, 'end': 784, 'text': '[5]', 'ref_id': 'BIBREF5'}, {'start': 785, 'end': 788, 'text': '[6]', 'ref_id': 'BIBREF6'}, {'start': 1008, 'end': 1011, 'text': '[3]', 'ref_id': 'BIBREF3'}, {'start': 1012, 'end': 1015, 'text': '[4]', 'ref_id': 'BIBREF4'}, {'start': 1016, 'end': 1019, 'text': '[5]', 'ref_id': 'BIBREF5'}, {'start': 1558, 'end': 1559, 'text': '7', 'ref_id': 'BIBREF7'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'Of 248 patients registered at the study hospital during the study period, 51 (20.6%) patients were aged from 10 to 35 years. Five patients who did not provide sufficient data at the epidemiological outbreak interview were further excluded.The social-economic demographics and symptoms of the study patients are summarized in Table 1 .The majority were young adults (n = 32) and the rest were adolescents (n = 14). The main types of exposure were ', 'cite_spans': [], 'ref_spans': [{'start': 325, 'end': 332, 'text': 'Table 1', 'ref_id': 'TABREF0'}], 'section': 'Results'}, {'text': 'The Innovation contact with other confirmed cases (22, 47.8%) or residing in Wuhan (19, 41 .3%). Half of the patients were men (24, 52.2%), had a normal body mass index (BMI) (24, 52.2%), and never conducted physical activity (23, 50.0%). Fewer patients had histories of one or more medical disease (6, 13.0%), including obesity (n = 1), diabetes (n = 1), chronic obstructive pulmonary disease (n = 1), hyperthyroidism (n = 1), kidney stones (n = 1), and arthrolithiasis (n = 1). Only one (2.2%) patient was identified as a severe case.The individual characteristics of this case are summarized in Table S1 . Four patients were identified as asymptomatic on admission. The most common symptoms at admission were dry cough (34, 81.0%), fever (29, 69.1%), and expectoration (16, 38.1%). Less common symptoms included headache, fatigue, pharyngalgia, chest pain, anorexia, myalgia, dizziness, diarrhea, nausea, and shortness of breath. The common pathological changes seen on chest CT were ground-glass opacity (29, 63.0%) and bilateral patchy shadowing (12, 26.1%). No severe cases were identified among adolescent patients. Compared with young adults, adolescent patients had a lower probability to be asymptomatic (6.3% versus 14.3%). Fewer adolescent patients reported fever, headache, and fatigue. Only 7 (50.0%) adolescent patients showed ground-glass opacity on chest CT compared with 22 (68.8%) in young adults. However, none of these differences reached levels of statistical significance. The typical patterns of chest CT for adolescents and young adults are shown in Figure 1 .', 'cite_spans': [{'start': 83, 'end': 87, 'text': '(19,', 'ref_id': 'BIBREF17'}, {'start': 88, 'end': 90, 'text': '41', 'ref_id': None}], 'ref_spans': [{'start': 598, 'end': 606, 'text': 'Table S1', 'ref_id': 'TABREF0'}, {'start': 1575, 'end': 1583, 'text': 'Figure 1', 'ref_id': 'FIGREF0'}], 'section': 'Report'}, {'text': 'We recorded family-clustered events from six symptomatic index patients on admission ( Figure 2 ). According to the definition of the serial interval, we only included 12 secondary cases out of these family-cluster data to estimate the distribution of serial interval (Figure 3A) . The estimated median serial interval was ', 'cite_spans': [], 'ref_spans': [{'start': 87, 'end': 95, 'text': 'Figure 2', 'ref_id': None}, {'start': 268, 'end': 279, 'text': '(Figure 3A)', 'ref_id': 'FIGREF1'}], 'section': 'Report'}, {'text': 'The Innovation 1.9 days (95% CI 0.4-6.2). The estimated 95 th percentile of serial interval could reach as long as 28.6 days (95% CI 10.6-76.9). Among 14 patients who provided the exact date of traveling to Wuhan or contacting other confirmed cases, the estimated median incubation period was 8.3 days (95% CI 5.0-13.4) ( Figure 3B ). The estimated 95 th percentile of the incubation period could reach as long as 24.8 days (95% CI 14.9-47.6). After excluding three asymptomatic cases, the estimated median incubation period decreased to 6.6 days (95% CI 4.4-9.6) ( Figure 3C ), and the estimated 95 th percentile of the incubation period decreased to 14.8 days (95% CI 10.4-22.0). Based on 42 symptomatic cases, we estimated the median interval from symptom onset to the first medical visit to be 1.4 days (95% CI 0.8-2.4) ( Figure 3D ). The estimated 95 th percentile of the period from symptom onset to the first medical visit was 13.2 days (95% CI 8.3-20.9).', 'cite_spans': [], 'ref_spans': [{'start': 322, 'end': 331, 'text': 'Figure 3B', 'ref_id': 'FIGREF1'}, {'start': 566, 'end': 575, 'text': 'Figure 3C', 'ref_id': 'FIGREF1'}, {'start': 826, 'end': 835, 'text': 'Figure 3D', 'ref_id': 'FIGREF1'}], 'section': 'Report'}, {'text': 'Numbers in boxes are calendar dates. Data from the 12 secondary cases (close contacts were defined as those who had clear exposure to only one index case and had no other potential source of infection) were used to estimate the distribution of serial interval.', 'cite_spans': [], 'ref_spans': [], 'section': 'Figure 2 Information on Exposures and Dates of Illness Onset in Six Symptomatic Cases and Their Family Close Contacts'}, {'text': 'The Innovation', 'cite_spans': [], 'ref_spans': [], 'section': 'Report'}, {'text': 'On admission, 10 (21.7%) patients had leukopenia (white blood cell count <4 3 10 9 /L) and 29 (63.0%) had lymphopenia (Table 2 ). Ten patients (21.7%) had decreased levels of platelets (<150 3 10 9 /L). The other elevated laboratory indicators in study patients were lactate dehydrogenase (9, 19.6%), C-reactive protein (9, 19 .6%), D-dimer (7, 15.2%), alanine aminotransferase (7, 15.2%), and total bilirubin (7, 15.2%). Several differences were observed in laboratory findings between adolescent and young adult patients. For example, 8 (25.0%) young adult patients presented elevated levels of C-reactive protein while only one adolescent patient showed a similar pattern.', 'cite_spans': [{'start': 320, 'end': 323, 'text': '(9,', 'ref_id': 'BIBREF8'}, {'start': 324, 'end': 326, 'text': '19', 'ref_id': 'BIBREF17'}], 'ref_spans': [{'start': 118, 'end': 126, 'text': '(Table 2', 'ref_id': 'TABREF3'}], 'section': 'Report'}, {'text': 'The treatments and prognosis outcomes are summarized in Table 3 February 25, 2020, 36 patients had recovered and been discharged, 10 patients were still hospitalized, and no patients died. Compared with young adults, adolescent patients received less oxygen inhalation therapy.', 'cite_spans': [], 'ref_spans': [{'start': 56, 'end': 63, 'text': 'Table 3', 'ref_id': 'TABREF5'}], 'section': 'Report'}, {'text': 'We observed four asymptomatic cases at admission, all of whom were consistently confirmed as asymptomatic cases by our face-to-face or telephone interviews. The disease progress of the asymptomatic patients during the treatment period is shown in Figure 4 . Two asymptomatic patients (cases 2 and 3) still did not show any symptoms until February 23, 2020. Asymptomatic case 1 developed symptoms of shortness of breath, difficulty breathing, and chest tightness in the 17 days after admission. Asymptomatic case 4 developed symptoms of dry cough, phlegm, and nausea in the 6 days after admission, and his COVID-19 nucleic acid transformed into negative in the 14 days after admission. We detected family-clustered events from three asymptomatic cases, which indicated that the transmission during their asymptomatic periods occurred between them and their family close contacts. For example, two relatives of asymptomatic case 1 who lived with him and did not report other potential infection sources developed the illness on January 17, 2020. ', 'cite_spans': [], 'ref_spans': [{'start': 247, 'end': 255, 'text': 'Figure 4', 'ref_id': 'FIGREF4'}], 'section': 'Report'}, {'text': 'The Innovation', 'cite_spans': [], 'ref_spans': [], 'section': 'Report'}, {'text': 'This study, to the best of our knowledge, is the first to assess the epidemiological and clinical characteristics of COVID-19 in adolescent and young adult patients. We added new knowledge to help understand the characteristics of COVID-19 in the younger population. We detected four asymptomatic cases out of 46 patients admitted. We reported a mean incubation period of 7.2 days in symptomatic cases, which could reach as long as 10 days allowing for the truncated time to event data for asymptomatic cases. We estimated a median serial interval of 1.9 days from the date of illness onset in index patients to the date of developing illness in their family close contacts. We found that the most common symptoms were dry cough, fever, and expectoration. Only 29 (63.0%) of the patients showed the ground-glass opacity on chest CT scan. The typical changes of laboratory indicators were decreased white blood cell count, decreased lymphocyte count, decreased platelet count, increased lactate dehydrogenase, and increased C-reactive protein. During the treatment three patients developed acute kidney injury, but no other medical complications were documented. Nearly 80% of the patients recovered and were discharged at the end of follow-up.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'Our study suggests that the incubation period of COVID-19 in adolescents and young adults might be longer than that in older patients. A retrospective study reported a mean incubation period of 5.2 days (95% CI 4.1-7.0) and 95 th percentile of the incubation period of 12.5 days based on early COVID-19 patients from ', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'The Innovation', 'cite_spans': [], 'ref_spans': [], 'section': 'Report'}, {'text': \"Wuhan. 2 A later study, which used the data of travelers from Wuhan, estimated the mean incubation period as 6.4 days (95% CI 5.6-7.7), ranging from 2.2 to 11.1 days. 9 Similar studies reported a shorter incubation period (median 4 days) for patients outside Wuhan. 6 Our study suggests that person-to-person transmission of COVID-19 can occur rapidly from infected adolescent and young adults to their family contacts. We recorded six family-clustered events of COVID-19 in asymptomatic patients. We estimated the mean serial interval to be 6.5 days (95% CI 2.5-17.4), which is shorter than that (7.5 days, 95% CI 5.3-19.0) estimated from early patients in Wuhan. 2 Most importantly, we estimated the median serial interval to be 1.9 days (95% CI 0.4-6.2), which was still lower than that (4.0 days, 95% CI 3.1-4.9) estimated in a recent modeling study. 11 We provided evidence supporting the transmission of COVID-19 from adolescent and young adult asymptomatic patients to their family or close contacts. In this study, of the four out of 46 patients identified as asymptomatic cases, three were identified as the index patients in their families. The two asymptomatic primary cases had neither symptoms nor chest CT findings during the treatment. One asymptomatic primary case suffered from difficulty breathing, shortness of breath, and chest tightness in the 17 days during treatment. Most importantly, all of their family contacts developed symptoms before the admission date of the asymptomatic index patients. Our findings were consistent with the existing evidence. Rothe et al. first reported that an asymptomatic Chinese woman might be the transmission source for her two German business partners. 12 Tong et al. reported a two-family cluster of COVID-19 patients in Zhejiang Province after each family's primary case contacted an asymptomatic case of COVID-19 from Wuhan. 13 Recently, a similar study has identified a 20-year-old \", 'cite_spans': [{'start': 167, 'end': 168, 'text': '9', 'ref_id': 'BIBREF8'}, {'start': 266, 'end': 267, 'text': '6', 'ref_id': 'BIBREF6'}, {'start': 665, 'end': 666, 'text': '2', 'ref_id': 'BIBREF1'}, {'start': 855, 'end': 857, 'text': '11', 'ref_id': None}, {'start': 1710, 'end': 1712, 'text': '12', 'ref_id': 'BIBREF11'}, {'start': 1885, 'end': 1887, 'text': '13', 'ref_id': 'BIBREF12'}], 'ref_spans': [], 'section': 'Report'}, {'text': 'Chinese woman as an asymptomatic carrier who has infected five individuals in her family. 14 Compared with the early evidence from Wuhan patients, the adolescent and young adult patients with COVID-19 presented different patterns of symptoms and fewer abnormalities of laboratory indicators on admission. The most common symptoms were fever (83%), cough (82%), and shortness of breath (31%) in early elderly patients from Wuhan. 3 Later studies with more case series reported other common symptoms including fatigue, gastrointestinal symptoms, upper ', 'cite_spans': [{'start': 429, 'end': 430, 'text': '3', 'ref_id': 'BIBREF3'}], 'ref_spans': [], 'section': 'Report The Innovation'}, {'text': 'The Innovation airway congestion, myalgia, and headache. 4, 6, 15 The results of chest CT indicated that nearly 80% of the early patients showed bilateral pneumonia and ground-glass opacity. 3, 4, 15, 16 Laboratory examinations indicated that over 70% of the patients had lymphocytopenia, elevated lactate dehydrogenase, and elevated C-reactive protein. 3, 4, 17 In this study, the most common symptoms on admission were dry cough (81.0%), fever (69.1%), and expectoration (38.1%). Only one patient reported shortness of breath on admission. The proportion of those with reported fever at admission decreased to 58.3% in adolescent patients. Nearly 60% of the patients showed ground-glass opacity changes on chest CT, which decreased to 50% in adolescent patients. Only 26.09% and 13.04% of all patients showed bilateral patchy shadowing or consolidation on chest CT. In terms of laboratory examinations, 63.0% of the patients had lymphocytopenia, which agrees closely with the existing evidence. However, fewer patients had elevated levels of lactate dehydrogenase (19.6%) and C-reactive protein (19.6%). Both of these abnormalities of laboratory findings were less pronounced in adolescent patients.', 'cite_spans': [{'start': 57, 'end': 59, 'text': '4,', 'ref_id': 'BIBREF4'}, {'start': 60, 'end': 62, 'text': '6,', 'ref_id': 'BIBREF6'}, {'start': 63, 'end': 65, 'text': '15', 'ref_id': 'BIBREF13'}, {'start': 191, 'end': 193, 'text': '3,', 'ref_id': 'BIBREF3'}, {'start': 194, 'end': 196, 'text': '4,', 'ref_id': 'BIBREF4'}, {'start': 197, 'end': 200, 'text': '15,', 'ref_id': 'BIBREF13'}, {'start': 201, 'end': 203, 'text': '16', 'ref_id': 'BIBREF14'}, {'start': 354, 'end': 356, 'text': '3,', 'ref_id': 'BIBREF3'}, {'start': 357, 'end': 359, 'text': '4,', 'ref_id': 'BIBREF4'}, {'start': 360, 'end': 362, 'text': '17', 'ref_id': 'BIBREF15'}], 'ref_spans': [], 'section': 'Report'}, {'text': \"Our study indicated that younger patients have a better prognosis during treatment. Early studies reported that nearly 40% of the patients have at least one medical chronic disease at admission, and common complications during treatment included ARDS, shock, acute cardiac injury, arrhythmia, kidney injury, and liver dysfunction. 3, 4, 6, 18 Most of the patients received antiviral therapy and oxygen inhalation while some received glucocorticoid therapy or antifungal treatment. Nearly 20% of the patients were identified as severe cases and received mechanical ventilation and extracorporeal membrane oxygenation. In our study, only one (2.2%) patient was identified as a severe case on admission. After undergoing treatment with antiviral therapy, interferon-a inhalation, and oxygen inhalation, nearly 80% of the patients recovered and were discharged at the end of follow-up. Three patients developed severe kidney injury during treatment. Although a significant difference was observed in the treatment with oxygen inhalation for adolescent and young adult patients, this was largely due to clinician's personal preference. Therefore, this difference has no clinical significance.\", 'cite_spans': [{'start': 331, 'end': 333, 'text': '3,', 'ref_id': 'BIBREF3'}, {'start': 334, 'end': 336, 'text': '4,', 'ref_id': 'BIBREF4'}, {'start': 337, 'end': 339, 'text': '6,', 'ref_id': 'BIBREF6'}, {'start': 340, 'end': 342, 'text': '18', 'ref_id': 'BIBREF16'}], 'ref_spans': [], 'section': 'Report'}, {'text': 'This study provided initial evidence for the epidemiological and clinical characteristics of COVID-19 in adolescents and young adults. Compared with early evidence from middle-aged or elderly patients, the adolescent and young adult patients had a longer incubation period, indicating that a longer period for medical observation or isolation is needed for these patients. The shorter serial interval indicated that transmission could occur rapidly from younger patients to their close contacts. Compared with older patients, younger counterparts had fewer typical signs and symptoms and fewer abnormalities in laboratory findings, and fewer of them developed severe complications during treatment. Our results suggest that adolescents and young adults might be the key subpopulation at a later stage for preventing the worldwide spread of COVID-19.', 'cite_spans': [], 'ref_spans': [], 'section': 'Report'}, {'text': 'This study has some limitations. First, we conducted this study based on only 46 patients. The relatively small sample size limited us in obtaining sound evidence concerning differences in most characteristics between subgroups, so the study findings should interpreted with caution. Second, at the end date of this study nearly 20% of the patients were still hospitalized, which limited us in fully illuminating the outcomes of the study patients. Finally, our conclusions should be extended to the general population with caution because the study patients are from a single hospital in Chongqing City.', 'cite_spans': [], 'ref_spans': [], 'section': 'Report'}, {'text': 'This single-center study with a relatively small sample size suggested that adolescent and young adult COVID-19 patients had a longer incubation period, a shorter serial interval, and a higher probability of being asymptomatic in comparison with older patients reported thus far in the literature. Transmission to family contacts occurred in several asymptomatic cases. Fewer patients developed complications during treatment.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusions'}, {'text': 'The Innovation', 'cite_spans': [], 'ref_spans': [], 'section': 'Report'}, {'text': 'In this study, we defined the adolescents as 10- An RT-PCR assay with a cycle threshold value (Ct-value) of less than 37 was defined as positive. Asymptomatic cases were defined as those who presented positive results by conducting the nucleic acid test of COVID-19 and had no elevated measured temperature or self-reported fever, and no gastrointestinal or respiratory symptoms such as cough and sore throat reported by physicians on admission. To confirm the validity, we further conducted a face-to-face or telephone interview with each asymptomatic patient to collect information on symptoms before 2 weeks of admission. For each patient the laboratory tests, which included routine blood tests, serum biochemistry, and coagulation function, were performed on admission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Study Design and Participants'}, {'text': 'We described the differences in demographic factors, symptoms at admission, comorbidities, and chest CT findings across age groups.', 'cite_spans': [], 'ref_spans': [], 'section': 'Statistical Analysis'}, {'text': 'We summarized the distribution of laboratory findings for cases using median and IQR among total cases, those aged 10-24 years, and those aged 25-35 years.We defined the incubation period as the time interval from the date of exposure to the date of symptom onset. We included two types of patients who could recall the exact date of traveling to Wuhan or contacting other confirmed cases.To ensure accuracy, those who reported more than one exposure source (n = 1) or contacting period over 3 days (n = 7) were excluded. For patients who reported exposure time on 1 day, the exposure day was defined as the exposure date.', 'cite_spans': [], 'ref_spans': [], 'section': 'Statistical Analysis'}, {'text': 'The Innovation', 'cite_spans': [], 'ref_spans': [], 'section': 'Report'}, {'text': \"The first day was defined as the exposure date for those who reported exposure time within 2 days. The middle day was defined as the exposure date for those who reported exposure time within 3 days.We used a parametric survival analysis model with Weibull distribution to estimate the distribution of the incubation period. Since the asymptomatic cases at the first medical visit could develop symptoms during the follow-up,we first treated the incubation period for asymptomatic cases as right-censored data (from the date of exposure to the date of the first medical visit) and performed the estimations. We then excluded the asymptomatic cases and repeated the estimations. We defined familyclustered events as patients who were the first to develop symptoms in their family (index patient) and whose family members (secondary cases) had a clear contact history with the index patient and had no other potential sources of infection.We used the date of symptom onset to measure the date of illness onset.We defined the serial interval as the time interval from the date of illness onset for the index patient to the date of illness onset for the secondary cases.We used a parametric survival analysis model with gamma distribution to estimate the distribution of serial interval. We further calculated the time interval from the date of symptom onset to the date of the first medical visit using a parametric survival model with Weibull distribution. We finally compared the differences of treatments, days of persistent fever during treatment, days of transformation to negative results by COVID-19 nucleic acid tests during treatment, and prognosis outcomes across different age groups. We compared proportions of categorical variables using the chi-square test and used Fisher's exact test when the minimum expected values for the variables were less than 1. We used independent group t tests to compare means of continuous variables following a normal distribution; otherwise, Mann-Whitney rank tests were used.\", 'cite_spans': [], 'ref_spans': [], 'section': 'Report'}, {'text': 'We performed summaries and significance tests using SAS 9.4, and parametric survival analyses were conducted by R 3.1.1. The statistical significance level was defined as 0.05 with a two-sided test.', 'cite_spans': [], 'ref_spans': [], 'section': 'Report'}, {'text': 'Data collection and analyses of cases were approved by the institutional ethics board of Three Gorges Hospital affiliated with Chongqing University (No.2020-7(论)).', 'cite_spans': [], 'ref_spans': [], 'section': 'Ethical Approval'}, {'text': 'Since the epidemiological interview for the study case is part of a continuing public health outbreak investigation, individual consent was considered exempt.', 'cite_spans': [], 'ref_spans': [], 'section': 'Ethical Approval'}, {'text': 'All authors declare no competing interests.', 'cite_spans': [], 'ref_spans': [], 'section': 'Declaration of Interests'}, {'text': 'Supplemental Information can be found online at https://doi.org/10.1016/j.xinn.2020.04.001.', 'cite_spans': [], 'ref_spans': [], 'section': 'Supplemental Information'}], 'paragraphs': [{'text': 'The Innovation 1.9 days (95% CI 0.4-6.2). The estimated 95 th percentile of serial interval could reach as long as 28.6 days (95% CI 10.6-76.9). Among 14 patients who provided the exact date of traveling to Wuhan or contacting other confirmed cases, the estimated median incubation period was 8.3 days (95% CI 5.0-13.4) ( Figure 3B ). The estimated 95 th percentile of the incubation period could reach as long as 24.8 days (95% CI 14.9-47.6). After excluding three asymptomatic cases, the estimated median incubation period decreased to 6.6 days (95% CI 4.4-9.6) ( Figure 3C ), and the estimated 95 th percentile of the incubation period decreased to 14.8 days (95% CI 10.4-22.0). Based on 42 symptomatic cases, we estimated the median interval from symptom onset to the first medical visit to be 1.4 days (95% CI 0.8-2.4) ( Figure 3D ). The estimated 95 th percentile of the period from symptom onset to the first medical visit was 13.2 days (95% CI 8.3-20.9).', 'id': '00005'}], 'title': 'Epidemiological and Clinical Characteristics of COVID-19 in Adolescents and Young Adults', 'sha': '03b9ed59e65b1eb754eb9b24232a1e6ed2d85f9c; 8b19549de3e1e9bc6eddb055de2584a3b7060cdd', 'doi': '10.1016/j.xinn.2020.04.001', 'score': 9.203100204467773}, {'rank': 184, 'paragraphs': [{'text': '', 'id': '00016'}], 'title': 'Longer incubation period of coronavirus disease 2019 (COVID‐19) in older adults', 'sha': '8f3c0c4095c08ef8213ada5642ba4a36216dbd0d; 19d6cf627e5afd4d56919357e64d211003665028', 'doi': '10.1002/agm2.12114', 'score': 9.201700210571289}, {'rank': 185, 'paragraphs': [{'text': 'The copyright holder for this preprint this version posted June 16, 2020. . https://doi.org/10.1101/2020.06.14.20131177 doi: medRxiv preprint Figure 1 : Schematic of the epidemiological model from [2] . Compartments are: susceptible to the virus (S); exposed (E 1 ); exposed, pre-symptomatic, and infectious (E 2 ); symptomatic and infectious (I); quarantined (Q); and removed (recovered or deceased; R). There are analogous states for individuals practising physical distancing (bottom row). An individual in state X can begin distancing and move to the corresponding distanced state X d at rate u d . The reverse transition occurs at rate u r . The model quickly settles on a fraction e = u d /(u d + u r ) participating in distancing, and dynamics depend on this fraction, rather than on the rates u d and u r . The red and tinted red squares represent the compartments that we identify as active cases. The orange/orange tinted and red/red tinted compartments indicate infectious cases.', 'id': '00008'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.200499534606934}, {'rank': 186, 'paragraphs': [{'text': 'The likelihood expression, as described in Section 2.1, relates predicted case counts from the SEIR-type model to reported case counts, over time and corrected for the delay between symptomonset and reporting which is assumed to have a Weibull distribution (w c ). This likelihood, written as a function of a given strength of physical distancing f X , (e.g., f 1 , the strength of distancing when it is first implemented or f 2 , the strength of distancing after relaxation) may be maximised to find f n , the maximum likelihood estimate (MLE) of f X on day n. Estimation of f n uses data from the start of the outbreak up to and including day n. Since case count data can be quite noisy, there can also be considerable variation in the day-by-day MLEs. For example, in the first few days after a change in physical distancing, a single day with a low number of identified cases 7 . CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'id': '00020'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.200498580932617}, {'rank': 187, 'paragraphs': [{'text': 'We also explore the impact of changing the noise/delay of case count reporting on the time to detect a change in the strength of physical distancing. We simulate outbreak trajectories for strengthening distancing under varying levels of the dispersion parameter φ. In this investigation, we control the variance in the number of daily reported cases: note that this is not the same as the dispersion parameter about R 0 , commonly referred to as k. We also control the shape and scale parameters of the onset-to-reporting Weibull distribution. We then perform the same daily MLE estimation procedure on these simulated time series. Note that this back-estimation requires the assumption that the true noise level and delay are known. If we did not have estimates of these quantities in reality, we would need to co-estimate them along with the strength of distancing, which may introduce identifiability issues or a longer time to acceptance.', 'id': '00023'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.2004976272583}, {'rank': 188, 'paragraphs': [{'text': 'The copyright holder for this preprint this version posted June 16, 2020. . takes approximately 45 days before a substantial difference arises due to relaxing distancing from f 1 = 0.36 to f 2 = 0.65, and 23 days to detect such a difference using observed cases. Figure 2 shows rising case trajectories for which distancing is introduced, and declining ones in which distancing is relaxed, and illustrates the dependence of the timing on the severity of the change. If the change is weak (e.g., a weak relaxation from f 1 = 0.36 to f 2 = 0.5) there is no discernible difference between the trajectories before July 1 (47 days). The time until a substantial difference arises naturally depends on what is considered a substantial difference, and on the underlying uncertainty. We explore the impact of the threshold choice 9 . CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'id': '00026'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.200496673583984}, {'rank': 189, 'paragraphs': [{'text': 'If the standard deviation is varied away from 0.15, then the days until the threshold is reached (i.e., a significant difference between the baseline and a relaxed condition is seen) is affected. The results of varying the standard deviation between 0.01 and 0.4 (considering the values in the set {0.01, 0.11, 0.21, 0.31, 0.41}) are shown in Figure S6a . In that Figure, we see that the days until a significant increase in the active cases can vary from 10 to 50 days. Missing values in the lines in this plot indicate that our methods cannot determine whether or not the condition will ever differ from the baseline: if the standard deviation of R 0 is > 0.11, we do not predict that a weak relaxation of physical distancing to f 2 = 0.6 will result in a substantial increase in active cases. However, if the standard deviation of R 0 is < 0.11 a relaxation of physical distancing to f 2 = 0.6 could lead to a significant increase in active case count after 20 or 40 days.', 'id': '00044'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.200495719909668}, {'rank': 190, 'paragraphs': [{'text': ' reports on a long COVID‐19 incubation period (mean, 8.62 days; median, 8.13 days) that is close to the present figures, but their sample was younger (median age 40 years; 13.2% aged over 60 years) and a different methodology was used as discussed above.', 'id': '00015'}], 'title': 'Longer incubation period of coronavirus disease 2019 (COVID‐19) in older adults', 'sha': '8f3c0c4095c08ef8213ada5642ba4a36216dbd0d; 19d6cf627e5afd4d56919357e64d211003665028', 'doi': '10.1002/agm2.12114', 'score': 9.188400268554688}, {'rank': 191, 'paragraphs': [{'text': 'Our approach to determining when the effect of modifying measures is observable relies on using case count data as the indicator for increased community-based transmission. However, public health officials may find outbreaks-even where they do not contribute to statistically higher case counts-by noting epidemiological links among cases (e.g., links through workplace, family, healthcare or gatherings). Changes in these smaller outbreaks may be detected much faster than our 3 week estimates, but it remains the case that measures directed towards the general population are the main intervention for COVID-19. Our results focus on estimating impacts on this broader population level, and this has long time scales. Sentinel surveillance systems, contact tracing and outbreak detection are among the tools used by public health agencies to gather rapid and more detailed information than case counts during a disease outbreak. These form multi-faceted surveillance networks-including hospitals, primary care, symptom trackers-which may often be faster than confirmed and lab-tested cases. However, these networks also have complex limitations, and greatly vary by jurisdiction. Their data are not always consistently or widely published. Confirmed case counts remain a widely used source for population-level understanding of COVID-19 control, particular for those seeking a broad assessment of COVID-19 in multiple jurisdictions to support policy on borders and travel or to compare effectiveness of control measures.', 'id': '00039'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.163100242614746}, {'rank': 192, 'paragraphs': [{'text': 'To overcome the aforementioned problems, we propose a novel method to estimate the incubation period of COVID-19 by using the well-known renewal theory in probability.18 Such a method enhances the accuracy of estimation by reducing recall bias and utilizing abundance of the readily available forward time with a large sample size of 1211. To the best of our knowledge, this paper is a study of the distribution of the incubation period involving the largest number of samples to date. We find the estimated median of the incubation period is 8·13 days (95% CI: 7·37–8·91), and mean is 8·62 days (95% CI: 8·02–9·28), the 90th percentile is 14·65 days (95% CI: 14·00–15·26), and the 99th percentile is 20·59 days (95% CI: 19·47–21·62). Our estimated incubation period of COVID-19 is longer than the those given by previous researches on SARS, MERS, and COVID-19 in Table 1.', 'id': '00004'}], 'title': 'Estimation of incubation period distribution of COVID-19 using disease onset forward time: a novel cross-sectional and forward follow-up study', 'sha': 'c61be9fc4562d250e2dec241ef75bd22d61f61d1; 6d3b3f4ab80a61c45f82c61c6c756cfc6ddf4bf2', 'doi': '10.1101/2020.03.06.20032417', 'score': 9.162500381469727}, {'rank': 193, 'abstract': 'The coronavirus disease‐2019 (COVID‐19) has been found to be caused by the severe acute respiratory syndrome coronavirus 2 (SARS‐CoV‐2). However, comprehensive knowledge of COVID‐19 remains incomplete and many important features are still unknown. This manuscript conducts a meta‐analysis and a sensitivity study to answer the questions: What is the basic reproduction number? How long is the incubation time of the disease on average? What portion of infections are asymptomatic? And ultimately, what is the case fatality rate? Our studies estimate the basic reproduction number to be 3.15 with the 95% CI (2.41‐3.90), the average incubation time to be 5.08 days with the 95% CI (4.77‐5.39) (in day), the asymptomatic infection rate to be 46% with the 95% CI (18.48%‐73.60%), and the case fatality rate to be 2.72% with 95% CI (1.29%‐4.16%) where asymptomatic infections are accounted for.', 'body_text': [{'text': 'Since the first case of the coronavirus disease‐2019 (COVID‐19) was found in Wuhan, China in December 2019, the disease has rapidly spread in the city of Wuhan, then to Hubei Province, China, and subsequently, across the world.\\n1\\n On 11 March 2020, the World Health Organization (WHO) declared COVID‐19 to be a pandemic. The swift spread of the virus is largely attributed to its stealthy transmissions for which infected patients may be asymptomatic or exhibit only flu‐like symptoms in the early stage. Undetected transmissions present a remarkable challenge for the containment of the virus and pose an appalling threat to the public health. To understand the drastically negative impacts of COVID‐19 on the public health, it is urgent to investigate key features pertinent to the disease: How severe is the transmission? How long is the incubation time of the disease on average? How many infections are asymptomatic? And ultimately, what is the case fatality rate?', 'cite_spans': [], 'section': 'INTRODUCTION', 'ref_spans': []}, {'text': 'To evaluate the severity of the virus spread, it is useful to estimate the basic reproduction number (denoted R\\n0), defined as the average number of cases generated by an infected individual in a population where everyone is susceptible to infection. If the basic reproduction number R\\n0 is larger than 1, the outbreak is regarded as self‐sustaining unless control measures are implemented to mitigate the transmission.\\n2\\n Defined as the time from the moment of exposure to the virus until signs and symptoms of COVID‐19 appear, the incubation time of a COVID‐19 infected patient provides a useful measure for the disease development. Knowing the average incubation time of the COVID‐19 patients is important for disease surveillance. To determine how deadly the COVID‐19 is, it is fundamental to evaluate the case fatality rate which is calculated as the ratio of the number of deaths from COVID‐19 to the number of infected cases.', 'cite_spans': [], 'section': 'INTRODUCTION', 'ref_spans': []}, {'text': 'Since the outbreak of the disease, a large body of research on COVID‐19 has been done and many articles have been published in scientific journals or shared on platforms such as bioRxir and medRxir. Simulations of the epidemic have been published under various assumptions to delineate hidden transmissions of the virus.\\n2\\n While estimates of those important quantities have been reported in the literature, those results are quite different and vary considerably from study to study. There has been a lack of consensus of those estimates because of serious concerns on the heterogeneity among the studies. Different studies have been carried out on different patients under different conditions, and different authors may make different model assumptions. Interpreting the available findings must be coupled with the associated features of the studies.', 'cite_spans': [], 'section': 'INTRODUCTION', 'ref_spans': []}, {'text': 'Moreover, COVID‐19 data contain substantial errors in that the number of confirmed cases is considerably under‐reported, which is attributed to two primary reasons. Insufficient test kits do not allow every potential patient with COVID‐19‐like symptoms to be tested, and there has been a good portion of asymptomatic COVID‐19 carriers who would never be tested and counted as confirmed cases. It is useful to understand the asymptomatic infection rate, defined as the ratio of the number of asymptomatic infections to the number of all infected cases.', 'cite_spans': [], 'section': 'INTRODUCTION', 'ref_spans': []}, {'text': 'To address these issues, we carry out a meta‐analysis to synthesize the reported estimates of the basic reproduction number, the average incubation time, and the case fatality rate as well as the asymptomatic rate in a rigorous way by factoring out the variabilities associated with the relevant studies. To accommodate the effects of missing asymptomatic infections on calculating the case fatality rate, we further perform a sensitivity analysis for the estimation of the case fatality rate. Our study provides a comprehensive evaluation of key measures of COVID‐19 by taking into account of the heterogeneity and measurement error effects which are intrinsically associated with COVID‐19 data. Our results offer sensible estimates of the clinical features of COVID‐19 to enhance the understanding of the disease.', 'cite_spans': [], 'section': 'INTRODUCTION', 'ref_spans': []}, {'text': 'The third author (YZ) conducted a literature screening for the articles published between 24 January 2020 and 31 March 2020 by using online databases, including PubMed, Web of Science, Google Scholar, and the official websites of core scientific and biomedical journals including Science, Nature, The Lancet, The New England Journal of Medicine, and The Journal of American Medical Association, as well as some preprint platforms such as BioRxiv and MedRxir, with search terms specified as COVID‐19, SARS‐CoV‐2, 2019‐nCov, and novel coronavirus. Forty‐three articles were found with the theme on the basic reproduction number, the incubation period, the percentage of asymptomatic cases, and the case fatality rate. Among those articles, 20 articles, described in Table 1, were identified by the first author (WH) to be included in the analysis, together with Serra\\n3\\n and Day,\\n4\\n which were found on April 2. The inclusion criteria are the availability of both point estimates and 95% confidence intervals (95% CIs) (or equivalently, standard deviations) for the basic transmission number, the average incubation time, the asymptomatic rate, or the case fatality rate.', 'cite_spans': [], 'section': 'Search strategy and selection criteria ::: METHOD OF DATA COLLECTION', 'ref_spans': [{'start': 770, 'end': 771, 'mention': '1', 'ref_id': 'TABREF0'}]}, {'text': 'Table 1 presents the summary information of the selected articles together with the descriptions of the data used in those articles. We extract the results for the basic reproduction number from\\n2\\n, \\n5\\n, \\n8\\n, \\n10\\n, \\n11\\n and the results for the average incubation time from.\\n5\\n, \\n6\\n, \\n9\\n, \\n13\\n, \\n16\\n The results from\\n3\\n, \\n4\\n, \\n20\\n, \\n21\\n, \\n23\\n are extracted for estimation of the asymptomatic infection rate. The estimates for the case fatality rate together with their 95% CIs are taken from.\\n11\\n, \\n12\\n, \\n14\\n, \\n15\\n, \\n17\\n, \\n19\\n In the articles,\\n6\\n, \\n7\\n, \\n9\\n the reported 95% CIs were asymmetric which we suspect were caused by employing a transformation (such as the exponential transformation) to the initial CIs for the reparameterized effective size; for example, some authors may apply the logarithm to reparameterize the basic reproduction number or the average incubation time before performing the analysis. Using the inverse transformation, we convert the reported asymmetric CIs and work out the associated standard deviations which are used in determining the weights for the meta‐analysis.', 'cite_spans': [], 'section': 'Data extraction and analysis ::: METHOD OF DATA COLLECTION', 'ref_spans': [{'start': 6, 'end': 7, 'mention': '1', 'ref_id': 'TABREF0'}]}, {'text': 'As shown in the top panel of Figures 1, 2, 3, 4, estimates of the basic reproduction number, the mean incubation time, the asymptomatic infection rate, and the case fatality rate are quite different from study to study. To obtain synthetic results, we perform a meta‐analysis to aggregate the information from multiple studies with the same estimand (or effect size of interest) yet different features including the differences in the data collection, the sample size, and the conditions. Suppose K studies report an estimate and the associate standard deviation for an effect size of interest. For the ith study with i = 1, …, K, let Y\\ni denote the effect size of interest and let σ\\ni\\n2 represent its associated variance estimate. In our analysis here, Y\\ni is taken as the basic reproduction number, the average incubation time, the asymptomatic infection rate, and the case fatality rate, respectively. We calculate a weighted average of the results from those K studies under either the fixed effect model or the random effects model.\\n24\\n\\n', 'cite_spans': [], 'section': 'Method ::: META‐ANALYSIS', 'ref_spans': [{'start': 37, 'end': 38, 'mention': '1', 'ref_id': 'FIGREF0'}, {'start': 40, 'end': 41, 'mention': '2', 'ref_id': 'FIGREF1'}, {'start': 43, 'end': 44, 'mention': '3', 'ref_id': 'FIGREF2'}, {'start': 46, 'end': 47, 'mention': '4', 'ref_id': 'FIGREF3'}]}, {'text': 'Under the fixed effect model, the meta mean effect size is given by\\n(1)Ymeta,F=∑i=1KwiYi∑i=1Kwi,and the associated standard deviation is\\n(2)sd(Ymeta,F)=1∑i=1Kwi,where w\\ni = 1/σ\\ni\\n2 is the weight for the ith study.', 'cite_spans': [], 'section': 'Method ::: META‐ANALYSIS', 'ref_spans': []}, {'text': 'With the random effects model, the meta mean effect size, denoted Y\\nmeta,R, and its standard deviation, denoted sd(Y\\nmeta,R), are determined by the same expression as Equations (1) and (2) except for replacing the weight w\\ni with a new weight wi⁎=1/(σi2+T2), where\\nT2=Q−K+1W\\u2062with\\u2062Q=∑i=1KwiYi2−(∑i=1KwiYi)2∑i=1Kwi\\u2062and\\u2062W=∑i=1Kwi−(∑i=1Kwi)2∑i=1Kwi.\\n', 'cite_spans': [], 'section': 'Method ::: META‐ANALYSIS', 'ref_spans': []}, {'text': 'To determine whether the fixed effect model or the random effects model is suitable for the meta‐analysis, we calculate the I\\n2 index,\\n25\\n defined as\\nI2=(Q−(K−1)Q)100/%.\\n', 'cite_spans': [], 'section': 'Method ::: META‐ANALYSIS', 'ref_spans': []}, {'text': 'Consistent with,\\n18\\n we take the fixed effect model if I\\n2 < 50%, and the random effects model otherwise. In displaying the meta‐analysis results, we use the R package forestplot.\\n26\\n\\n', 'cite_spans': [], 'section': 'Method ::: META‐ANALYSIS', 'ref_spans': []}, {'text': 'The top panel of Figure 1 shows the results for the basic production number reported in the seven studies. The I\\n2 index for those studies is 97.8%, suggesting that the random effect model should be considered in conducting the meta‐analysis. This result agrees with the perception that the basic reproduction number is time‐dependent and varies from place to place.', 'cite_spans': [], 'section': 'Basic reproduction number ::: META‐ANALYSIS', 'ref_spans': [{'start': 24, 'end': 25, 'mention': '1', 'ref_id': 'FIGREF0'}]}, {'text': 'The bottom panel of Figure 1 includes the meta‐analysis results. The meta estimate of the basic reproduction number is 3.15, suggesting that a virus carrier may infect at least three individuals on average if preventive measures such as social distancing or quarantine are not applied to the public. This estimate is obtained from the data before 29 February 2020 where most data are collected from Wuhan city or other places in China. As more studies on the basic reproduction number become available for different places at different time periods, we can apply the same meta‐analysis procedure to estimate the basic reproduction number to reflect its changes with the implementation of various measures to curb the virus spread in different regions.', 'cite_spans': [], 'section': 'Basic reproduction number ::: META‐ANALYSIS', 'ref_spans': [{'start': 27, 'end': 28, 'mention': '1', 'ref_id': 'FIGREF0'}]}, {'text': 'To estimate the average of the incubation time for infections, we study the results reported in the five articles summarized in the top panel of Figure 2. The I\\n2 index is 28%, showing that the fixed effect model is suitable when conducting the meta‐analysis. While the incubation time differs from patient to patient, varying between 1 and 14 days, as reported by European Centre for Disease Prevention and Control (ECDC),\\n27\\n it is feasible to take their average time to be a fixed quantity.', 'cite_spans': [], 'section': 'Average incubation time ::: META‐ANALYSIS', 'ref_spans': [{'start': 152, 'end': 153, 'mention': '2', 'ref_id': 'FIGREF1'}]}, {'text': 'The bottom panel of Figure 2 reports the meta‐analysis results for the average incubation time of COVID‐19. It shows that the mean incubation time is 5.08 days, with 95% CIs being about 4.77 to 5.39 days. This estimated average incubation time is about 2 days shorter than the mean incubation time of 7 days announced by ECDC.\\n27\\n We point out that our estimate is obtained by combining the information from those studies before 24 February 2020 with study subjects in Wuhan city or other places in China.', 'cite_spans': [], 'section': 'Average incubation time ::: META‐ANALYSIS', 'ref_spans': [{'start': 27, 'end': 28, 'mention': '2', 'ref_id': 'FIGREF1'}]}, {'text': 'In the top panel of Figure 3, we display the estimates of the asymptomatic infection rate reported by.\\n3\\n, \\n4\\n, \\n20\\n, \\n21\\n, \\n23\\n It is clear that those studies provided very different estimates of the asymptomatic infection rate, varying from 17.9% to 78.3%. Such a heterogeneity of the studies is confirmed by the I\\n2 index which is 98%. Thus, we take the random effects model when conducting the meta‐analysis. We report the results at the bottom panel of Figure 3. Our analysis suggests that the combined asymptomatic infection rate is 46% with the 95% CI ranging from 18.4% to 73.6%. This estimate is derived from the information up to 2 April 2020 reported by the six studies which include worldwide individuals.', 'cite_spans': [], 'section': 'Asymptomatic infection rate ::: META‐ANALYSIS', 'ref_spans': [{'start': 27, 'end': 28, 'mention': '3', 'ref_id': 'FIGREF2'}, {'start': 465, 'end': 466, 'mention': '3', 'ref_id': 'FIGREF2'}]}, {'text': 'Finally, we are interested in estimating the case fatality rate which measures how deadly COVID‐19 is for the infected people. The meta‐analysis results derived from seven studies available in the literature, shown in the top panel of Figure 4, are reported at the bottom panel of Figure 4, where we assume the random effect models because the I\\n2 index is 99.5%. The estimated case fatality rate is 3.34%, slightly smaller than 3.4%, the estimate reported on 3 March 2020 by the WHO.\\n28\\n The 95% CI suggests that the average case fatality rate can be as small as 2.18% and as large as 4.49%. These results are obtained from the data up to 21 March 2020 which contain five studies with subjects in China and two studies with worldwide subjects.', 'cite_spans': [], 'section': 'Case fatality rate ::: META‐ANALYSIS', 'ref_spans': [{'start': 242, 'end': 243, 'mention': '4', 'ref_id': 'FIGREF3'}, {'start': 288, 'end': 289, 'mention': '4', 'ref_id': 'FIGREF3'}]}, {'text': 'We comment that the true average case fatality rate is expected to be smaller than the estimate here, because the reported estimates of the case fatality rate in the literature were merely calculated as the ratio of the number of deaths from COVID‐19 to the number of reported confirmed infected cases, where the number of reported confirmed infected cases is typically under‐reported due to limited testing capacity and the exclusion of asymptomatic infections.', 'cite_spans': [], 'section': 'Case fatality rate ::: META‐ANALYSIS', 'ref_spans': []}, {'text': 'To better understand what the true case fatality rate may be, we further conduct two sensitivity studies. In the first study, we repeat the meta‐analysis of the case fatality rate in Section 3.5 by further including the results calculated from the data of the Princess Diamond cruise.\\n29\\n This analysis is driven by the consideration that the case fatality rate derived from the cohort of the cruise passengers is highly likely to be accurate, because the number of confirmed cases from the cruise is very likely to be close to the true number of infections. The bottom of Figure 5 reports the meta‐analysis results obtained from the random effects model. With the inclusion of the results for the data of the Princess Diamond cruise, the weights of other seven studies become smaller than those in Figure 4, as shown by the size of the squares in Figures 4 and 5; and the estimate of the case fatality rate becomes smaller. The case fatality rate is estimated as 2.72% with the 95% CI (1.29%‐4.16%).', 'cite_spans': [], 'section': 'SENSITIVITY ANALYSIS', 'ref_spans': [{'start': 580, 'end': 581, 'mention': '5', 'ref_id': 'FIGREF4'}, {'start': 806, 'end': 807, 'mention': '4', 'ref_id': 'FIGREF3'}, {'start': 856, 'end': 857, 'mention': '4', 'ref_id': 'FIGREF3'}, {'start': 862, 'end': 863, 'mention': '5', 'ref_id': 'FIGREF4'}]}, {'text': 'In our second sensitivity analysis, we revise the results in Figure 4 by incorporating the information of asymptomatic cases. To see the adjustment, we let D represent the number of deaths caused from COVID‐19. Let C\\nR denote the number of reported infected cases of COVID‐19, let C\\nA stand for the number of the SARS‐Cov‐2 carriers who are asymptomatic, and let C be the total number of infected cases with the virus. Let r\\nA = C\\nA/C be the ratio of asymptomatic infections to the true number of infections.', 'cite_spans': [], 'section': 'SENSITIVITY ANALYSIS', 'ref_spans': [{'start': 68, 'end': 69, 'mention': '4', 'ref_id': 'FIGREF3'}]}, {'text': 'Let p\\nR = D/C\\nR be the reported case fatality rate and let p\\nT = D/C be the true case fatality rate. If we assume that C = C\\nR + C\\nA, then the reported case fatality rate and the true case fatality differ by the factor 1 − r\\nA:\\n(3)pT=(1−rA)pR.\\n', 'cite_spans': [], 'section': 'SENSITIVITY ANALYSIS', 'ref_spans': []}, {'text': 'Estimates of the case fatality rate that have been reported in the current literature are merely directed to p\\nR rather than p\\nT.', 'cite_spans': [], 'section': 'SENSITIVITY ANALYSIS', 'ref_spans': []}, {'text': 'To sensibly estimate the true case fatality, we use Equation (3) to adjust the reported results of the seven studies listed at the top panel of Figure 4. Specifically, we may multiply the factor 1 − r\\nA with an estimate for the reported rate p\\nR as well as its standard deviation for each study and then run a meta‐analysis. However, the exact value of the asymptomatic infection rate is unavailable, and we only have its estimates from various studies displayed at the top panel of Figure 3. To assess how the uncertainty of not knowing the true value of r\\nA, we use two ways to set a value for r\\nA to modify the reported fatality rates for the studies listed at the top panel of Figure 4 for running a new meta‐analysis.', 'cite_spans': [], 'section': 'SENSITIVITY ANALYSIS', 'ref_spans': [{'start': 151, 'end': 152, 'mention': '4', 'ref_id': 'FIGREF3'}, {'start': 490, 'end': 491, 'mention': '3', 'ref_id': 'FIGREF2'}, {'start': 688, 'end': 689, 'mention': '4', 'ref_id': 'FIGREF3'}]}, {'text': 'First, taking r\\nA as one of seven reported estimates listed at the top panel of Figure 3, we modify the reported results provided by each study listed at the top panel of Figure 4 using Equation (3), and report the meta‐analysis results at the top panels of Figure 6. In the second analysis, we take r\\nA as the synthesized estimate reported in Figure 3, that is, r\\nA is set as 46%, and then run the meta‐analysis for these adjusted case fatality rates under the random effects model. We report the results at the bottom panel of Figure 6, which shows that the estimate of the case fatality rate is 1.8% with the 95% CI ranging from 1.18% to 2.43%.', 'cite_spans': [], 'section': 'SENSITIVITY ANALYSIS', 'ref_spans': [{'start': 87, 'end': 88, 'mention': '3', 'ref_id': 'FIGREF2'}, {'start': 178, 'end': 179, 'mention': '4', 'ref_id': 'FIGREF3'}, {'start': 265, 'end': 266, 'mention': '6', 'ref_id': 'FIGREF5'}, {'start': 351, 'end': 352, 'mention': '3', 'ref_id': 'FIGREF2'}, {'start': 536, 'end': 537, 'mention': '6', 'ref_id': 'FIGREF5'}]}, {'text': 'We carry out a meta‐analysis and sensitivity study for estimating the basic reproduction number, the average incubation time, the asymptomatic infection rate, and the case fatality rate for COVID‐19. Examining the published results between 24 January 2020 and 31 March 2020, together with two studies found on 2 April 2020, our study aggregates different results reported in the literature and provides synthetic estimates by addressing the heterogeneity present in different studies conducted at different times for different regions. Our study shows that the basic reproduction number is estimated to be 3.15 with the 95% CI (2.41‐3.90) and the average incubation time is 5.08 days with the 95% CI ranging from 4.77 days to 5.39 days. The asymptomatic infection rate is estimated to be 46% with the 95% CI (18.48%‐73.60%). While multiple studies reported estimates of the case fatality rate, those estimates are typically higher than the true case fatality rate under the same conditions, which is attributed to the fact that a good portion of asymptomatic infections are not counted when estimating the case fatality rate. Our sensitivity study addresses this important issue and makes an adjustment to provide a sensible estimate of the case fatality rate. Compared with the estimated 3.34% case fatality rate obtained from the meta‐analysis, our sensitivity study estimates the case fatality rate to be 1.8% with 95% CI (1.18%‐2.43%) where asymptomatic infections are accounted for.', 'cite_spans': [], 'section': 'DISCUSSION AND CONCLUSIONS', 'ref_spans': []}, {'text': 'Our studies reveal sensible estimates for the important quantities of COVID‐19 by accommodating discrepancy effects associated with different studies such as the variability of the data collected from different populations at different time periods. With the evolution of the pandemic, the basic production number can greatly reduce as a result of the implementation of active measures to mitigate the virus spread. The estimation of the case fatality rate may be closer to the true case fatality rate because of the increase of the test capacity; more infected cases may be detected so the reported number of infections would be closer to the true number of COVID‐19 carriers. Our results are useful in enhancing the knowledge of COVID‐19. Though we focus on evaluating the basic reproduction number, the average incubation time, the asymptomatic infection rate, and the case fatality rate, other features, such as the time from symptom onset to hospitalization or to death and the morbidity rate of the disease, are also important and they are worth being estimated in a sensible way.', 'cite_spans': [], 'section': 'DISCUSSION AND CONCLUSIONS', 'ref_spans': []}, {'text': \"The present investigations have limitations. Not all published results for the four measures are included in our study; we do not include those manuscripts which reported merely a point estimate without the associated standard deviation or a 95% CI, because they do not allow us to decide a proper weight for the inclusion of the result. While reporting a single estimate of the average incubation time and the case fatality rate gives us an easy way to assess the impact of COVID‐19, such measures marginalize the effects from the associated factors such as the disease severity, the patient's medical conditions, and age. With more studies available for categorizing the case fatality rate or the incubation time, it is useful to apply the meta‐analysis to estimate those measures by stratifying the population based on the demographic and clinical characteristics. When data at the individual level are available, better estimates of key features for COVID‐19 can be obtained and the pandemic trend can be more reasonably projected using statistical regression models. Finally, we emphasize that care should be taken when interpreting the estimates from the meta‐analysis and the sensitivity analyses here. A meaningful interpretation should be coupled with the associated features of the individual studies such as the time window, the study subjects, the study design, and different measures of controlling the virus spread by different countries.\", 'cite_spans': [], 'section': 'DISCUSSION AND CONCLUSIONS', 'ref_spans': []}, {'text': 'The authors declare that there are no conflict of interests.', 'cite_spans': [], 'section': 'CONFLICT OF INTERESTS', 'ref_spans': []}, {'text': 'WH identified the articles that were screened by the third author, extracted the results from individual studies, conducted all the data analyses, and prepared the initial draft. GY discussed the analysis methods with WH and wrote the manuscript. YZ searched the literature, provided the candidate articles to WH for further examination.', 'cite_spans': [], 'section': 'AUTHOR CONTRIBUTIONS', 'ref_spans': []}], 'paragraphs': [{'text': 'Our studies reveal sensible estimates for the important quantities of COVID‐19 by accommodating discrepancy effects associated with different studies such as the variability of the data collected from different populations at different time periods. With the evolution of the pandemic, the basic production number can greatly reduce as a result of the implementation of active measures to mitigate the virus spread. The estimation of the case fatality rate may be closer to the true case fatality rate because of the increase of the test capacity; more infected cases may be detected so the reported number of infections would be closer to the true number of COVID‐19 carriers. Our results are useful in enhancing the knowledge of COVID‐19. Though we focus on evaluating the basic reproduction number, the average incubation time, the asymptomatic infection rate, and the case fatality rate, other features, such as the time from symptom onset to hospitalization or to death and the morbidity rate of the disease, are also important and they are worth being estimated in a sensible way.', 'id': '00027'}], 'title': 'Estimation of the basic reproduction number, average incubation time, asymptomatic infection rate, and case fatality rate for COVID‐19: Meta‐analysis and sensitivity analysis', 'sha': '2c63ae442219d870167e691f91cd32e4ab9a7505', 'doi': '10.1002/jmv.26041', 'score': 9.160599708557129}, {'rank': 194, 'paragraphs': [{'text': \"On February 28th, a new COVID-19 case was confirmed in the Fourth People's Hospital of Yiyang. In following the patient's epidemiological history, it emerged that this was a familial cluster. Afterward, we further collected the clinical data of the patient's family members, including epidemiological history, laboratory examinations, lung CTs, pathogen results, treatment course, and recovery. All the clinical data were collected from the Fourth People's Hospital of Yiyang, and all the epidemiological histories of the patients were gathered from the patients themselves; these data were further confirmed by the local Centers for Disease Control.\", 'id': '00003'}], 'title': 'COVID-19 in a patient with long-term use of glucocorticoids: A study of a familial cluster', 'sha': '912c00514457672748738d29efb81f975815cdf6', 'doi': '10.1016/j.clim.2020.108413', 'score': 9.156200408935547}, {'rank': 195, 'paragraphs': [{'text': '(2)f(y)=αλ(yλ)α−1exp{−(yλ)α},y≥0.', 'id': '00014'}], 'title': 'Estimation of incubation period distribution of COVID-19 using disease onset forward time: a novel cross-sectional and forward follow-up study', 'sha': 'c61be9fc4562d250e2dec241ef75bd22d61f61d1; 6d3b3f4ab80a61c45f82c61c6c756cfc6ddf4bf2', 'doi': '10.1101/2020.03.06.20032417', 'score': 9.15250015258789}, {'rank': 196, 'abstract': 'If the knowledge of the incubation time is helpful in designing non-pharmaceutical interventions such as quarantine measures, can one use the number of cases arising after a lockdown to check (a) if our assumptions of the incubation time were correct and (b) if the quarantine measures were as successful as they could theoretically be. These are the two questions we raise by studying the number of new cases arising after lockdowns in a few European countries. The analysis which purely relies on the publicly available data of the numbers of new infections, rather than extensive contact tracing of individual patients, suggests a memory of the infections in the community with a median of 13.3 days. This distribution of the memory of infections which may even be considered as a surrogate of the incubation time in a perfect lockdown, suggests that even a perfect quarantine of 30 days is only 90% complete.', 'body_text': [{'text': 'In designing non-pharmaceutical interventions such as quarantine measures, it is important to know how long an individual can potentially remain infected and infectious. [1] The incubation time of an infectious disease is the duration between the exposure to an infected individual and the appearance of the symptoms. [2, 3] The incubation times for several acute respiratory infections have been measured [4] assuming their distributions. [5] These incubation times range from a couple of days to about 2 weeks. [4] In the studies of the COVID-19 pandemic, incubations lasting 2 to 14 days [6] , and reports suggesting up to 19 days [7] or even 24 days [8] have been observed with median incubation times of around 5.1 days [9] to 5.8 days [10] . At the heart of the estimation of the incubation period is an efficient contact tracing. The symptomatic individuals history is traced back to the possible day of exposure to infected individuals. As rigorous as this procedure is, by its nature it is not easily scalable as the collecting the exposure histories of hundreds of individuals may not be easy. This problem is even more complicated in the COVID-19 infections, as a very large fraction of the infections have been passed on by asymptomatic individuals. Further, the number of patients studied is also limited, which makes the likelihood of the errors in the \"tail regions\" of the distribution much higher.', 'cite_spans': [{'start': 170, 'end': 173, 'text': '[1]', 'ref_id': 'BIBREF0'}, {'start': 318, 'end': 321, 'text': '[2,', 'ref_id': 'BIBREF1'}, {'start': 322, 'end': 324, 'text': '3]', 'ref_id': 'BIBREF2'}, {'start': 406, 'end': 409, 'text': '[4]', 'ref_id': 'BIBREF3'}, {'start': 440, 'end': 443, 'text': '[5]', 'ref_id': 'BIBREF4'}, {'start': 513, 'end': 516, 'text': '[4]', 'ref_id': 'BIBREF3'}, {'start': 591, 'end': 594, 'text': '[6]', 'ref_id': 'BIBREF5'}, {'start': 634, 'end': 637, 'text': '[7]', 'ref_id': 'BIBREF6'}, {'start': 654, 'end': 657, 'text': '[8]', 'ref_id': 'BIBREF7'}, {'start': 725, 'end': 728, 'text': '[9]', 'ref_id': None}, {'start': 741, 'end': 745, 'text': '[10]', 'ref_id': 'BIBREF11'}], 'ref_spans': [], 'section': 'Introduction.'}, {'text': 'Almost all countries went into a lockdown for several weeks. However, the new cases of infection continue to arise, despite the best quarantine practices many democratic countries could adopt. This raises several question -are estimates of the incubation period based on a few hundred cases reliable, or is there an alternative way to estimate this incubation period even if one did not have the detailed exposure histories? Or is there an alternative estimation one needs to do at the level of the community with the best quarantine practices possible? Are the new cases in a lockdown scenario arising from incubation or because of other factors? We address some of these questions by proposing a new way of estimating the incubation time.', 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction.'}, {'text': 'We propose a new way of estimating the incubation time only from the publicly available epidemiological data of the number of infections. The two advantages of this approach are that the exposure history is not required and in a pandemic one can work with data tens of thousands of infections. In the publicly available data on the number of new infections from different countries, one sees that the number of daily new cases (I(t)) actually increase for a few days (t) after the lockdown (t=0), before they decline. In an ideal scenario, the new cases after lockdown come from the individuals who were infected in the preceding days and after a certain incubation are becoming symptomatic. Thus the number of new infections on day 1, depends on the asymptomatic ones on day 0 (A(0)) and an incubation of 1 day, as well as the ones infected on day -1 and with an incubation of 2 days, and so on. Mathematically, the predicted number of infections I*(t) on day t after the lockdown is', 'cite_spans': [], 'ref_spans': [], 'section': 'Results and Discussion. A new surrogate for incubation time.'}, {'text': 'Eq (1) where µ(t+t; l,k) represents the distribution of the incubation period which we assume to be a Weibull distribution with parameters l and k, without loss of generality. Convoluting two functions such as in Eq. (1) is a standard approach in the linear response theory. We further assume that on any day prior to the lockdown the number of asymptomatic infections is proportional to the number of reported infections (A(-t) = a.I(-t) ). We performed a linear regression analysis between the infections observed after lockdown I(t) and the predicted ones I*(t) for various choices of l and k ( Figures 1A,B,C) . The values of l and k for which the regression coefficient was the highest was considered the maximum likelihood estimate of the incubation period ( Figure 1D ).', 'cite_spans': [], 'ref_spans': [{'start': 598, 'end': 613, 'text': 'Figures 1A,B,C)', 'ref_id': None}, {'start': 765, 'end': 774, 'text': 'Figure 1D', 'ref_id': None}], 'section': 'Results and Discussion. A new surrogate for incubation time.'}, {'text': 'We find that the incubation time follows a Weibull distribution of µ(t+t; l,k) with parameters l=16, k=2 best helps the infections before lockdown from Austria, Spain and Germany ( Figure  1D ) to predict the ones after. In working with the epidemiological data, we developed an estimation of the incubation period of the virus in the community, which may in principle be the same as that for the individuals in a perfectly implemented quarantine.', 'cite_spans': [], 'ref_spans': [{'start': 181, 'end': 191, 'text': 'Figure  1D', 'ref_id': None}], 'section': 'Results and Discussion. A new surrogate for incubation time.'}, {'text': 'Clearly this incubation is not sufficient to explain the new cases in all countries. If one predicts the new infections post lockdown in Italy, one sees there are more infections than what may be expected purely from a median incubation of 13.3 days (=l.(ln2) 1/k ) for the said distribution (Supplementary Figure 1) . However, since there is no reason to believe that the nature of the infections in Italy are different from its neighbor Switzerland (which also has similar incubation time as the countries discussed here), it appears that one should look beyond the incubation to understand the persistent infections.', 'cite_spans': [], 'ref_spans': [{'start': 307, 'end': 316, 'text': 'Figure 1)', 'ref_id': None}], 'section': 'New cases beyond the incubation.'}, {'text': 'A Weibull distribution with parameters l=16, k=2 has a cumulative probability that reaches around 0.9 after 30 days. This suggests that the new estimate of the incubation proposed in this work will require a lockdown longer than 30 days for a complete elimination of the asymptomatic cases. The longer incubation time may be a consequence of either an imperfect implementation of the lockdown or a genuinely longer incubation time in individuals that could not be estimated by contact tracing a few hundred individuals. We believe in the latter hypothesis, although in principle the nature of the conclusions do not change regardless of the causative factor. Practically, even if the former is the case, the approach we take is still relevant because several countries have implemented the most stringent measures. In that sense, although physiologically it may slightly differ from the incubation of the individual, this incubation at the community level with the lowest possible societal interactions has its own meaning. In this sense, it is imperative that one uses the incubation period of the infections in the community level into the public health models.', 'cite_spans': [], 'ref_spans': [], 'section': 'Why is the incubation longer?'}, {'text': 'In summary, we introduced a new way of estimating the incubation time by studying the new infection that arise after measures that restrict the interactions between the individuals in the community to a minimal or non-existent level. The advantages are that it is easy estimate. The estimates we made for three European countries suggest an incubation time with a median of 13.3 days. Models for public health should consider estimates for incubation time in the community, such as what we introduce here, for a better understanding of the differences between new cases arising after the incubation and the genuinely newer ones. Figure 1 . The daily infections observed after lockdown [11] and the predictions made using the infections before lockdown for: A. Austria (lockdown, 16 March 2020) B. Spain (lockdown, 14 March 2020) C. Germany (lockdown, 21 March 2020) are compared. The data until 13 April 2020 was used. The predictions were made as described with l=16, k=2. D. Similar graphs like A, B, C were generated for l between 5 to 20 to identify the value that best suits the data. l=16 optimizes the Weibull distribution for the data studied. k=2 was used in this analysis. It was independently optimized in the range k=1 to 3.', 'cite_spans': [{'start': 685, 'end': 689, 'text': '[11]', 'ref_id': 'BIBREF12'}], 'ref_spans': [{'start': 629, 'end': 637, 'text': 'Figure 1', 'ref_id': None}], 'section': 'Conclusions.'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Figures'}, {'text': '(which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.04.15.20067058 doi: medRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': 'Figures'}], 'paragraphs': [{'text': 'In designing non-pharmaceutical interventions such as quarantine measures, it is important to know how long an individual can potentially remain infected and infectious. [1] The incubation time of an infectious disease is the duration between the exposure to an infected individual and the appearance of the symptoms. [2, 3] The incubation times for several acute respiratory infections have been measured [4] assuming their distributions. [5] These incubation times range from a couple of days to about 2 weeks. [4] In the studies of the COVID-19 pandemic, incubations lasting 2 to 14 days [6] , and reports suggesting up to 19 days [7] or even 24 days [8] have been observed with median incubation times of around 5.1 days [9] to 5.8 days [10] . At the heart of the estimation of the incubation period is an efficient contact tracing. The symptomatic individuals history is traced back to the possible day of exposure to infected individuals. As rigorous as this procedure is, by its nature it is not easily scalable as the collecting the exposure histories of hundreds of individuals may not be easy. This problem is even more complicated in the COVID-19 infections, as a very large fraction of the infections have been passed on by asymptomatic individuals. Further, the number of patients studied is also limited, which makes the likelihood of the errors in the \"tail regions\" of the distribution much higher.', 'id': '00001'}], 'title': 'Community Memory of COVID-19 Infections Post Lockdown as a Surrogate for Incubation Time', 'sha': 'a4b68af0652c852faaf8212ebe0ae02d7bbc5c8f', 'doi': '10.1101/2020.04.15.20067058', 'score': 9.147000312805176}, {'rank': 197, 'abstract': 'SARS-CoV-2 has been marked as a highly pathogenic coronavirus of COVID-19 disease into the human population, causing over 5.5 million confirmed cases worldwide. As COVID-19 has posed a global threat with significant human casualties and severe economic losses, there is a pressing demand to further understand the current situation and develop rational strategies to contain the drastic spread of the virus. Although there are no specific antiviral therapies that have proven effective in randomized clinical trials, currently, the rapid detection technology along with several promising therapeutics for COVID-19 have mitigated its drastic transmission. Besides, global institutions and corporations have commenced to parse out effective vaccines for the prevention of COVID-19. Herein, the present review will give exhaustive details of extensive researches concerning the drug discovery and therapeutic options for COVID-19 as well as some insightful discussions of the status of COVID-19.', 'body_text': [{'text': 'This century has witnessed the worldwide spread of several hitherto unknown coronaviruses. The rapid alteration of ecology and urbanization along with vulnerable public health systems have facilitated more frequent emerging of epidemics, which have become more and more intractable for us to prevent and contain. In December 2019, a new coronavirus (CoV) correlated with human respiratory disease was firstly reported causing pneumonia and death [1]. Soon afterward, the disease cases continued to expand and soared dramatically worldwide. The causative virus, named as severe acute respiratory syndrome CoV-2 (SARS-CoV-2), was identified as the pathogen leading to the CoV disease COVID-19. Infections with SARS-CoV-2 are now swift and violent, and as of 28 May 2020, over 5.5 million cases have been confirmed in more than 215 countries, with over 353,334 deaths. To date, SARS-CoV-2 has most closest relation to SARS and relevant viruses that circulate in bats, as evidenced by viral genome analysis as well as the research probing into the proximal origin of such virus [2].', 'cite_spans': [{'start': 446, 'end': 449, 'mention': '[1]', 'ref_id': 'BIBREF0'}, {'start': 1074, 'end': 1077, 'mention': '[2]', 'ref_id': 'BIBREF1'}], 'section': 'Introduction', 'ref_spans': []}, {'text': 'In general, CoVs, subfamily coronavirinae, are a cluster of highly diversified, enveloped, positive-sense and single-stranded viruses ((+)ssRNA virus) that can induce enteric, respiratory, hepatic as well as neurological disorders of discrepant severity in a wide range of animal species, encompassing humans [3,4]. As the largest RNA viruses ever discovered, CoVs can be categorized into α-, β-, δ- and γ-CoVs. Among these genera, the β group can be subdivided into A, B, C and D lineages [5]. In the past 17 years, three neoteric β-CoVs, SARS-CoV, Middle East respiratory syndrome CoV (MERS-CoV), along with SARS-CoV-2 have emerged, engendering severe human diseases. Although the origin of SARS-CoV-2 outbreak is not yet clear, recent studies have deduced that it might be transmitted through bats because it is highly similar to the bat SARS-CoV-like coronaviruses [6]. Later on, genomic and evolutionary proofs of the occurrence of Pangolin-CoV indicated that pangolin species might be the potential intermediate host for SARS-CoV-2 [7,8]. Unlike human CoVs, zoonotic viruses hold the capacity of infecting both animals and humans, leading to severe respiratory diseases (i.e., acute respiratory distress syndrome (ARDS) and pneumonia) [9,10]. Clinical data revealed that the COVID-19 symptoms are far more severe among the elders with comorbidities, while asthma, allergic illnesses, as well as chronic obstructive pulmonary disease are also risk factors [11,12]. Despite the continuous improvement of the prevention strategy and the disease surveillance system, the lack of efficacious drug treatment and correlated high morbidity cases of the SARS-CoV-2 along with its potentiality to induce pandemics, highlighting the urgent demand for neoteric drug discovery. In this review, we will first briefly describe the status of epidemiology, mechanism and diagnosis of COVID-19. In addition to the discussion of current management strategies for COVID-19, the emphasis has been given to the treatment options and drug discovery of COVID-19. We will refer to the drug screening progress and the development of vaccines for the COVID-19 therapy. Ultimately, we will delineate the overarching challenges in the clinical invention of new anti-coronavirus agents and offer some insights toward the prevention of such disease based on the understanding of its epidemic dynamics in real-time.', 'cite_spans': [{'start': 310, 'end': 311, 'mention': '3', 'ref_id': 'BIBREF2'}, {'start': 312, 'end': 313, 'mention': '4', 'ref_id': 'BIBREF3'}, {'start': 490, 'end': 493, 'mention': '[5]', 'ref_id': 'BIBREF4'}, {'start': 869, 'end': 872, 'mention': '[6]', 'ref_id': 'BIBREF5'}, {'start': 1039, 'end': 1040, 'mention': '7', 'ref_id': 'BIBREF6'}, {'start': 1041, 'end': 1042, 'mention': '8', 'ref_id': 'BIBREF7'}, {'start': 1242, 'end': 1243, 'mention': '9', 'ref_id': 'BIBREF8'}, {'start': 1244, 'end': 1246, 'mention': '10', 'ref_id': 'BIBREF9'}, {'start': 1462, 'end': 1464, 'mention': '11', 'ref_id': 'BIBREF10'}, {'start': 1465, 'end': 1467, 'mention': '12', 'ref_id': 'BIBREF11'}], 'section': 'Introduction', 'ref_spans': []}, {'text': 'Spread mainly through respiratory droplets or close contact, SARS-CoV-2 induced diseases has been growing dramatically in accordance with the published data from the World Health Organization (WHO) [13]. Although the number of confirmed cases in China has decreased a lot from late February 2020, and there is no report of COVID-19 deaths on 6th April, the confirmed cases of CoVs worldwide are still expanding with a vengeance. The spectrum of illness presentation or severity profile also affects triage and diagnostic decision-making, along with the therapeutic options and prognostic expectations [14]. Till now, the exact source of the current outbreak of COVID-19 remains unclear, but the dynamic model is similar to the classic zoonotic emergence to human-to-human transmission [15]. The mortality rate of SARS-CoV-2 (∼3.8%) is lower compared to that of MERS-CoV (37.1%) or SARS-CoV (10%), but the number of infections is more than ten times higher [16]. With respect to the lack of evidence that companion animals might be a source of infection, patients with COVID-19 are the prime source of infection, and those with severe conditions are more infectious than those with mild conditions. Intriguingly, asymptomatically infected persons or patients in incubation has also been demonstrated to shed the infectious virus, serving as a potential infection source to drive the transmission of the COVID-19 [17]. In addition, researches focused on the follow-up of recovered patients revealed that the tested samples of rehabilitees continuously showed a positive RT-PCR result, implicating that asymptomatic infection during incubation or recovery from COVID-19 may pose a daunting challenge to disease control and prevention [18].', 'cite_spans': [{'start': 198, 'end': 202, 'mention': '[13]', 'ref_id': 'BIBREF12'}, {'start': 601, 'end': 605, 'mention': '[14]', 'ref_id': 'BIBREF13'}, {'start': 785, 'end': 789, 'mention': '[15]', 'ref_id': 'BIBREF14'}, {'start': 956, 'end': 960, 'mention': '[16]', 'ref_id': 'BIBREF15'}, {'start': 1411, 'end': 1415, 'mention': '[17]', 'ref_id': 'BIBREF16'}, {'start': 1731, 'end': 1735, 'mention': '[18]', 'ref_id': 'BIBREF17'}], 'section': 'The epidemiology of COVID-19', 'ref_spans': []}, {'text': 'The incubation period refers to the time between exposure to the virus and initial symptoms. A research report on the early propagation dynamics of COVID-19 unveiled that the average incubation period of COVID-19 was around 5 d, and its 95% distribution was 12.5 d [19]. Another study analyzing the travel history and symptoms in 88 confirmed cases revealed a similar average incubation period of around 6 days [20]. In addition, there was an unusual case with an incubation period of up to 19 d [21]. Although such a long incubation period may be a low probability event (the condition of 14 d was suggested by experts for quarantine), the longer incubation time indicates the adjustment of screening and control policies [22].', 'cite_spans': [{'start': 265, 'end': 269, 'mention': '[19]', 'ref_id': 'BIBREF18'}, {'start': 411, 'end': 415, 'mention': '[20]', 'ref_id': 'BIBREF19'}, {'start': 496, 'end': 500, 'mention': '[21]', 'ref_id': 'BIBREF20'}, {'start': 723, 'end': 727, 'mention': '[22]', 'ref_id': 'BIBREF21'}], 'section': 'The epidemiology of COVID-19', 'ref_spans': []}, {'text': 'The early outbreak data of COVID-19 largely follow exponential growth. Disparate models based upon the clinical progression of the disease had been proposed to assess the basic reproductive ratio R0. A retrospective analysis of the first 425 identified cases demonstrated that in the early stages of COVID-19, the R0 was assessed to be 2.2 [19]. Nevertheless, deterministic compartmental models based upon the likelihood and a model analysis revealed that the control reproduction number Rc might be as high as 6.5 due to the estimation of four generations of viral transmission and serried social contacts [23]. In this regard, it is noteworthy that R0 estimates may vary in the light of numerous biologics, social behavior, and environmental factors [24]. In general, the basic R0 assessed by the majority of researches ranges between 2 to 4 [25]. According to the WHO data updated on May 28, 2020, more than 215 countries have reported 5,593,631 confirmed cases, including 353,334 deaths (Fig. 1\\n). The grand total case fatality rate of global cases outside China is 1.31% [26]. The US had also recorded the largest number of coronavirus deaths in a day, with more than 1810 deaths reported on April 7th, according to data from Johns Hopkins University. Given the condition that the population of all races and ages is generally susceptible, there is an urgent need to further implementing the timely diagnosis, along with efficient isolation of patients, to cut down the R0 of SARS-CoV-2 and control the epidemic outbreak.', 'cite_spans': [{'start': 340, 'end': 344, 'mention': '[19]', 'ref_id': 'BIBREF18'}, {'start': 607, 'end': 611, 'mention': '[23]', 'ref_id': 'BIBREF22'}, {'start': 752, 'end': 756, 'mention': '[24]', 'ref_id': 'BIBREF23'}, {'start': 844, 'end': 848, 'mention': '[25]', 'ref_id': 'BIBREF24'}, {'start': 1076, 'end': 1080, 'mention': '[26]', 'ref_id': 'BIBREF25'}], 'section': 'The epidemiology of COVID-19', 'ref_spans': [{'start': 992, 'end': 998, 'mention': 'Fig. 1', 'ref_id': 'FIGREF0'}]}, {'text': 'As previously mentioned, SARS-CoV-2, belonging to the coronaviridae family, is the causative pathogen of COVID-19. Closely resembled other β-CoVs, the SARS-CoV-2 virion with a genome size of ∼3 kb has a nucleocapsid consisted of genomic RNA and phosphorylated nucleocapsid (N) protein [27]. Of note, the nucleocapsid is embedded in a phospholipid bilayer and is encased by two disparate types of spike proteins: spike glycoprotein trimmer present in all CoVs and the hemagglutinin-esterase shared merely in some CoVs. The spike (S) protein plays a pivotal role in binding to receptors and is the key to determine host tropism and transmission capacity (Fig. 2\\n). The matrix protein (M) along with the viral envelop (E) are also located in the viral envelope [28]. Genome analysis showed that the SARS-CoV-2 possesses 5’ and 3’ terminal sequences, with a gene order 5’ -replicase open reading frame (ORF) 1ab-S-envelope(E)-membrane(M)-N-3’ [29]. The virus particles are 60−100 nm in diameter, and are round or oval [30]. It can be inactivated by ultraviolet light or heated at 56°C for 30 min and is sensitive to most disinfectants (i.e., ether, 75% ethanol, peracetic acid, chlorine, as well as chloroform) [31].', 'cite_spans': [{'start': 285, 'end': 289, 'mention': '[27]', 'ref_id': 'BIBREF26'}, {'start': 758, 'end': 762, 'mention': '[28]', 'ref_id': 'BIBREF27'}, {'start': 939, 'end': 943, 'mention': '[29]', 'ref_id': 'BIBREF28'}, {'start': 1014, 'end': 1018, 'mention': '[30]', 'ref_id': 'BIBREF29'}, {'start': 1207, 'end': 1211, 'mention': '[31]', 'ref_id': 'BIBREF30'}], 'section': 'Virological characteristics of SARS-CoV-2', 'ref_spans': [{'start': 653, 'end': 659, 'mention': 'Fig. 2', 'ref_id': 'FIGREF1'}]}, {'text': 'Accumulating evidence has revealed that SARS-CoV-2 and SARS-CoV have the same human cell receptor, the angiotensin-converting enzyme 2 (ACE2), while MERS-CoV hinges on dipeptidyl peptidase 4 for host cell entry [35]. ACE2 is a type I membrane protein expressed in the lungs, hearts, kidneys, and intestines, mainly associated with cardiovascular diseases [35]. A recent study analyzed the cryogenic electron microscopy structure of the SARS-CoV-2 S protein unveiled that it exhibits around 10 to 20-fold higher binding affinity to ACE2 in comparison with SARS-CoV [36]. The overall replication cycle of SARS-CoV-2 is depicted in Fig. 3\\n.', 'cite_spans': [{'start': 211, 'end': 215, 'mention': '[35]', 'ref_id': 'BIBREF34'}, {'start': 355, 'end': 359, 'mention': '[35]', 'ref_id': 'BIBREF34'}, {'start': 564, 'end': 568, 'mention': '[36]', 'ref_id': 'BIBREF35'}], 'section': 'Virological characteristics of SARS-CoV-2', 'ref_spans': [{'start': 629, 'end': 635, 'mention': 'Fig. 3', 'ref_id': 'FIGREF2'}]}, {'text': 'As for the phylogenetic network analysis of SARS-CoV-2 genomes, after analysis of the sample from across the world, researchers have found three central variants differentiated through amino acid alterations, denoted A, B, and C, with A being the ancestral type in line with the bat outgroup CoV. Intriguingly, types A and C account for a considerable proportion outside East Asia (Europe and the United States) [37]. By comparison, B is the most common type in East Asia, and its ancestors ’genomes do not appear to spread outside East Asia without first mutating into a derived type B. Hence, SARS-CoV-2 genomes were found to be closely correlated, and evolutionary selection took place in human hosts, sometimes with parallel evolutionary events, where the same viral mutation occurs in two disparate human hosts [37]. Owing to the erratic nature of RNA viruses as well as its high contagiousness, the continuous monitoring of SARS-CoV-2 from humans or animal species is of extreme significance for pandemic control.', 'cite_spans': [{'start': 412, 'end': 416, 'mention': '[37]', 'ref_id': 'BIBREF36'}, {'start': 816, 'end': 820, 'mention': '[37]', 'ref_id': 'BIBREF36'}], 'section': 'Virological characteristics of SARS-CoV-2', 'ref_spans': []}, {'text': 'Rapid and accurate diagnosis of COVID-19 is of considerable significance for controlling outbreaks in the communities and hospitals [38]. Technologies such as polymerase chain reaction (PCR), reverse-transcription polymerase chain reaction (RT-PCR), real-time RT-PCR (rRT-PCR), and reverse transcription loop-mediated isothermal amplification (RT-LAMP) have been leveraged as ideal diagnostic tests for CoVs [39,40]. To date, the frontline reaction to the SARS-CoV-2 outbreak has been PCR testing. As the gold standard for diagnosing the source of infection, PCR holds the preponderance that the primers required for such assays can be generated relatively quickly once the viral sequence is identified (Figure 4\\n) [41]. Soon after the virus was identified, the first quantitative RT-PCR assays to detect SARS-CoV-2 were inaugurated and distributed in January 2020 by WHO. Nevertheless, this test protocol was complicated and high-priced, and is primarily applicable for large centralized diagnostic laboratories. As for the diagnostic criteria currently formulated by the China National Health Commission, nasopharyngeal cancer and oropharyngeal swab tests have ripened into the standard evaluation for the diagnosis of COVID-19 infection. So far, three new RT-PCR tests targeting the RNA-dependent RNA polymerase (RdRp)/helicase (Hel), nucleocapsid, and spike genes of SARS-CoV-2 had been inaugurated, with extremely lower detection limit in vitro\\n[42]. The SARS-CoV E gene detection was superior to the RdRp gene test combined with the one-step RT-PCR system. The E gene PCR was adequate for diagnosing SARS-CoV-2 infection, but the RdRp protocol was endorsed to verify positive results [43]. Remarkably, a new FDA-authorized COVID-19 test using the Abbott ID NOW diagnostics platform has been developed, which can produce results in just 5 minutes, cutting down on wait times both in terms of getting tested and receiving a diagnosis. As gene detection of SARS-CoV-2 might provide false negative results, it can be complemented by antibody detection, especially to better screen asymptomatic patients.', 'cite_spans': [{'start': 132, 'end': 136, 'mention': '[38]', 'ref_id': 'BIBREF37'}, {'start': 409, 'end': 411, 'mention': '39', 'ref_id': 'BIBREF38'}, {'start': 412, 'end': 414, 'mention': '40', 'ref_id': 'BIBREF39'}, {'start': 715, 'end': 719, 'mention': '[41]', 'ref_id': 'BIBREF40'}, {'start': 1450, 'end': 1454, 'mention': '[42]', 'ref_id': 'BIBREF41'}, {'start': 1690, 'end': 1694, 'mention': '[43]', 'ref_id': 'BIBREF42'}], 'section': 'Diagnostic testing for COVID-19 ::: Diagnosis and pathogenesis of SARS-CoV-2', 'ref_spans': [{'start': 704, 'end': 712, 'mention': 'Figure 4', 'ref_id': 'FIGREF3'}]}, {'text': 'Clinically, for those who are recently suffering from fever, fatigue, sore throat, cough or dyspnea due to exposure, the diagnosis of COVID-19 infection should be conducted with typical chest computerized tomography (CT) characteristics regardless of negative RT-PCR outcomes [44].', 'cite_spans': [{'start': 276, 'end': 280, 'mention': '[44]', 'ref_id': 'BIBREF43'}], 'section': 'Diagnostic testing for COVID-19 ::: Diagnosis and pathogenesis of SARS-CoV-2', 'ref_spans': []}, {'text': 'Most of the COVID-19 cases shared similar characteristics on CT images, presenting bilateral distribution of patchy shadows and ground-glass opacity, sometimes presenting a circular shape and peripheral lung distribution [45]. Some of the data published from China showed that in 21 primal chest CT scans, a large proportion of patients (86%) developed frosted glass opacity, affecting more than one lung lobe (71%) (bilateral involvement) [46]. It is also worth noting that lung cavitation, pleural effusions, discrete pulmonary nodules, along with lymphadenopathy were absent [46]. In addition to imaging technology, a recent study displayed that the Cas13-based SHERLOCK (specific high-sensitivity enzymatic reporter unlocking) platform can be harnessed for diagnosis of SARS-CoV-2 [47]. In this system, RNA-targeting Cas13 enzyme is deployed to identify specific genetic targets. The Cas13 can cleave nearby RNAs, a ‘collateral’ feature useful for amplifying a reporter signal in the diagnostic test. However, such a system needs to be further verified in clinical tests. Overall, combined with immunochromatography, colloidal gold, and other biotechnologies, associative detection strategies have been progressed swiftly.', 'cite_spans': [{'start': 221, 'end': 225, 'mention': '[45]', 'ref_id': 'BIBREF44'}, {'start': 440, 'end': 444, 'mention': '[46]', 'ref_id': 'BIBREF45'}, {'start': 578, 'end': 582, 'mention': '[46]', 'ref_id': 'BIBREF45'}, {'start': 785, 'end': 789, 'mention': '[47]', 'ref_id': 'BIBREF46'}], 'section': 'Diagnostic testing for COVID-19 ::: Diagnosis and pathogenesis of SARS-CoV-2', 'ref_spans': []}, {'text': 'With regard to the transmission of SARS-CoV-2, primary viral replication is assumed to occur in the mucosal epithelium of the upper respiratory tract and further multiplicated in the lower respiratory tract and gastrointestinal mucosa, inducing a mild viremia [48]. At this point, very few infections are under control and remain asymptomatic. Some patients may also suffer non-respiratory symptoms (i.e., acute liver and heart injury, renal failure, diarrhea), suggesting multiple organ involvement [49,50]. Since ACE2 is extensively expressed in the nasal mucosa, bronchus, lung, heart, and kidney, etc., many human organs are vulnerable to SARS-CoV-2 [51]. Specifically, the S protein plays a critical role in determining the cell tropism and hence interspecies transmission of SARS-CoV-2 since it hitches the virus to a cellular receptor and subsequently prompts virus entry via membrane fusion [32]. After binding to the receptor, the spike protein can catalyze the viral fusion process, allowing the viral genome to enter the cytoplasm. A prerequisite for this procedure is the division of S to subunits, a process known as priming (Fig. 3). The work of Hoffmann et al. unraveled that SARS-CoV-2 utilizes the ACE2 receptor for entry and the serine protease TMPRSS2 for S protein priming [52]. Hence, TMPRSS2 inhibitors approved for clinical use may block the entrance and may give rise to an underlying treatment option. Notably, the ability that S can readily obtain new protease cleavage sites and the fact that miscellaneous proteases can perform the same task suggests that this virus can easily adapt to the proliferation in several cell types [33]. Further, a panel of murine monoclonal antibodies (mAbs) and polyclonal antibodies (pAbs) against SARS-CoV-S1/receptor-binding domain (RBD) had been reported to be unable to interplay with S protein, implicating conspicuous discrepancies in antigenicity between SARS-CoV-2 and SARS-CoV [34].', 'cite_spans': [{'start': 260, 'end': 264, 'mention': '[48]', 'ref_id': 'BIBREF47'}, {'start': 501, 'end': 503, 'mention': '49', 'ref_id': 'BIBREF48'}, {'start': 504, 'end': 506, 'mention': '50', 'ref_id': 'BIBREF49'}, {'start': 654, 'end': 658, 'mention': '[51]', 'ref_id': 'BIBREF50'}, {'start': 899, 'end': 903, 'mention': '[32]', 'ref_id': 'BIBREF31'}, {'start': 1293, 'end': 1297, 'mention': '[52]', 'ref_id': 'BIBREF51'}, {'start': 1655, 'end': 1659, 'mention': '[33]', 'ref_id': 'BIBREF32'}, {'start': 1946, 'end': 1950, 'mention': '[34]', 'ref_id': 'BIBREF33'}], 'section': 'Pathogenesis of SARS-CoV-2 ::: Diagnosis and pathogenesis of SARS-CoV-2', 'ref_spans': [{'start': 1139, 'end': 1145, 'mention': 'Fig. 3', 'ref_id': 'FIGREF2'}]}, {'text': \"According to the pathological findings, the first report on the pathological results of severe COVID-19 revealed that diffuse alveolar injury on both sides of the lung was accompanied by cellular fibromyxoid exudates [53]. The right lung displayed significant lung cell shedding and hyaline membrane formation, suggesting ARDS. Moreover, the left lung tissue showed pulmonary edema and hyaline membrane formation, implying early ARDS. Interstitial mononuclear inflammatory infiltrates, dominated by lymphocytes, were found in both lungs. Another study reported that acute kidney injury and proteinuria might also occur during the progression of COVID-19 disease. ACE2 was seen to be upregulated in COVID-19 patients, and immunostaining with the SARS-CoV nucleoprotein antibody was positive in tubules [54]. Additionally, only a few interstitial mononuclear inflammatory infiltrates were found in the heart tissue, meaning that this virus may not directly induce heart impairment [53]. Aside from the acute respiratory distress syndrome, exuberant inflammatory responses during the infection process were also observed in clinical, giving rise to unrestrained pulmonary inflammation. Of note, the virus-induced ACE2 downregulation, rapid virus replication and cell damage, and antibody-dependent enhancement may lead to aggressive inflammation aroused by SARS-CoV-2 [55]. The initial stage of rapid viral replication would induce a large number of epithelial and endothelial cell death, thereby facilitating the generation of raging pro-inflammatory cytokines and chemokines (Fig. 5\\n) [56]. Intriguingly, a recent research compared the transcriptional responses of SARS-CoV-2 with other respiratory viruses to identify the transcriptional features that might form the biological basis of COVID-19 [57]. Their work demonstrated that the overall transcriptional induction of SARS-CoV-2 is abnormal. Despite viral replication, the host's response to SARS-CoV-2 failed to initiate powerful responses of type I and III interferons (IFN-I and -III), and at the same time induced high levels of chemokines required to recruit effector cells. In other words, this unique inflammatory response was defined by low levels of IFN-I and -III juxtaposed to elevated chemokines and high expression of IL-6. The reduced innate antiviral defenses coupled with exuberant inflammatory cytokine production may be the defining and driving features of COVID-19 [57]. Moreover, because the weakened immune response will further enable sustained viral replication, this critical finding may explain why severe cases of COVID-19 are more frequently observed in patients with comorbidities [57].\", 'cite_spans': [{'start': 217, 'end': 221, 'mention': '[53]', 'ref_id': 'BIBREF52'}, {'start': 801, 'end': 805, 'mention': '[54]', 'ref_id': 'BIBREF53'}, {'start': 979, 'end': 983, 'mention': '[53]', 'ref_id': 'BIBREF52'}, {'start': 1365, 'end': 1369, 'mention': '[55]', 'ref_id': 'BIBREF54'}, {'start': 1584, 'end': 1588, 'mention': '[56]', 'ref_id': 'BIBREF55'}, {'start': 1796, 'end': 1800, 'mention': '[57]', 'ref_id': 'BIBREF56'}, {'start': 2438, 'end': 2442, 'mention': '[57]', 'ref_id': 'BIBREF56'}, {'start': 2663, 'end': 2667, 'mention': '[57]', 'ref_id': 'BIBREF56'}], 'section': 'Pathogenesis of SARS-CoV-2 ::: Diagnosis and pathogenesis of SARS-CoV-2', 'ref_spans': [{'start': 1575, 'end': 1581, 'mention': 'Fig. 5', 'ref_id': 'FIGREF4'}]}, {'text': 'In addition to the cytokine storm, several experimentations had unraveled that lymphopenia is a customary characteristic of COVID-19, which may also accounts for severity and mortality [49,57].', 'cite_spans': [{'start': 186, 'end': 188, 'mention': '49', 'ref_id': 'BIBREF48'}, {'start': 189, 'end': 191, 'mention': '57', 'ref_id': 'BIBREF56'}], 'section': 'Pathogenesis of SARS-CoV-2 ::: Diagnosis and pathogenesis of SARS-CoV-2', 'ref_spans': []}, {'text': 'The preceding overview of the virology of SARS-CoV-2, as well as the sundry potential mechanisms of damage to the host, lay the foundation for developing specific targeted treatment and prevention. A general idea of pivotal targets for drug discovery is shown in Fig. 6\\n. In consideration of the role of the surface structural S in the virus-cell receptor interplay, it is of particular interest for the antiviral development. mAbs against the S1 subunit RBD and fusion inhibitors targeting the S2 subunit possess potential anti-SARS-CoV-2 capacity in vitro or in vivo\\n[59]. Besides, since ACE2 is the key functional host receptor of SARS-CoV-2 to determine the pathogenicity, mAbs, or molecules targeting the host receptor are effective anti- SARS-CoV-2 drugs, as long as they do not elicit immunopathological effects in animal models [4]. A recent study also probed into the COVID-19 S protein-binding site to the cell-surface receptor (known as Glucose Regulated Protein 78 (GRP78)). Their outcomes unveiled that the binding between regions III and IV of the S protein model and GRP78 was more favorable. In this regard, region IV is the major tractive force for GRP78 binding, and these 9 residues can be leveraged to design therapeutics specific against this disease [60]. Of note, although inhibitors of the proteases that prime S for fusion possess antiviral activity, multiple inhibitors are needed because S can utilize a variety of proteases for priming [61]. In addition, agents directly targeting the highly conserved S2 subunit may be potential treatment candidates.', 'cite_spans': [{'start': 569, 'end': 573, 'mention': '[59]', 'ref_id': 'BIBREF58'}, {'start': 836, 'end': 839, 'mention': '[4]', 'ref_id': 'BIBREF3'}, {'start': 1272, 'end': 1276, 'mention': '[60]', 'ref_id': 'BIBREF59'}, {'start': 1464, 'end': 1468, 'mention': '[61]', 'ref_id': 'BIBREF60'}], 'section': 'Crucial SARS-CoV-2 targets for novel drug development ::: Treatment strategy of COVID-19', 'ref_spans': [{'start': 263, 'end': 269, 'mention': 'Fig. 6', 'ref_id': 'FIGREF5'}]}, {'text': 'For SARS-CoV-2, the large replicase polyprotein 1a (pp1a) and pp1ab encoded by the ORF1a/b are subjected to two viral proteases, papain-like protease (PLpro) and 3C cleavage-like protease (3CLpro) (also known as Mpro), for producing non-structural proteins (i.e., RdRp, helicases) which are correlated to viral transcription and replication (Fig. 3) [4]. Therefore, enzyme inhibitors targeting these proteins may exhibit anti-SARS-CoV-2 activity in vitro. A recent study has uncovered that the Mpro of SARS-CoV-2, which is the translated polyproteins of ORF 1a/b, is a crucial enzyme that mediates viral replication and transcription [62]. Specifically, a Gln residue almost always requires a substrate at P1 (an amino acid in substrates). There is currently no human homologous for Mpro, which makes it a promising antiviral target (Fig. 6) [63]. Dai and coworkers had conducted the structure-based design of antiviral agents targeting this protease by parsing out the substrate-binding pocket of Mpro\\n[62]. In this regard, further study targeting such a protease may give rise to certain antiviral drug candidates.', 'cite_spans': [{'start': 350, 'end': 353, 'mention': '[4]', 'ref_id': 'BIBREF3'}, {'start': 634, 'end': 638, 'mention': '[62]', 'ref_id': 'BIBREF61'}, {'start': 842, 'end': 846, 'mention': '[63]', 'ref_id': 'BIBREF62'}, {'start': 1003, 'end': 1007, 'mention': '[62]', 'ref_id': 'BIBREF61'}], 'section': 'Crucial SARS-CoV-2 targets for novel drug development ::: Treatment strategy of COVID-19', 'ref_spans': [{'start': 342, 'end': 348, 'mention': 'Fig. 3', 'ref_id': 'FIGREF2'}, {'start': 834, 'end': 840, 'mention': 'Fig. 6', 'ref_id': 'FIGREF5'}]}, {'text': 'Another notable drug target may be the cellular enzymes that attach fatty acids to a cluster of cysteines in the cytoplasmic tail of S due to the fact that fatty acids are essential for the fusion of host cell and assembly of the virus, like the description of other S proteins, such as hemagglutinin for influenza viruses. The enzyme that connects the acyl chain to S has not yet been discovered, but the cellular protein will undergo acetylation by the members from the ZDHHC family with unique, only partially overlapping substrate specificities. If a few of them may be acetylated in the airway cells of the lungs, their blockage may suppress the viral replication, and the acylation of cellular proteins will rarely be damaged. In this regard, targeting acyltransferases may be promising, because the cysteine group is existed in all CoV genus S, in spite of their source [64]. However, in consideration of the palmitoylation of crucial proteins in the innate immunity, if the proteins of the innate immune response are modified with the same enzymes as viral proteins, the acylation inhibitor may be limited.', 'cite_spans': [{'start': 877, 'end': 881, 'mention': '[64]', 'ref_id': 'BIBREF63'}], 'section': 'Crucial SARS-CoV-2 targets for novel drug development ::: Treatment strategy of COVID-19', 'ref_spans': []}, {'text': 'In addition, Bojkova et al. recently identified the host cell pathways modulated by SARS-CoV-2 infection and revealed that suppression of these pathways may prevent viral replication in human cells [65]. Of note, SARS-CoV-2 infection profile was determined by translatome3 and proteome proteomics at different times after infection, suggesting that this virus could reshape central cellular pathways (e.g., translation, splicing, carbon metabolism and nucleic acid metabolism) [65]. Accordingly, spliceosome inhibitors and glycolysis inhibitors may also be the potential targets.', 'cite_spans': [{'start': 198, 'end': 202, 'mention': '[65]', 'ref_id': 'BIBREF64'}, {'start': 477, 'end': 481, 'mention': '[65]', 'ref_id': 'BIBREF64'}], 'section': 'Crucial SARS-CoV-2 targets for novel drug development ::: Treatment strategy of COVID-19', 'ref_spans': []}, {'text': 'The last point of view is the application of small interfering RNAs (siRNAs). SARS-CoV-2 can disintegrate in host cells, releasing the nucleocapsid and viral RNA into the cytoplasm, and then translate ORF1a/b into pp1a and pp1ab for genomic RNA replication [66]. Hence, siRNAs targeting structural genes may play a role in the SARS-CoV-2 infections, and further optimizing the delivery of siRNAs in vivo may make them clinically valuable.', 'cite_spans': [{'start': 257, 'end': 261, 'mention': '[66]', 'ref_id': 'BIBREF65'}], 'section': 'Crucial SARS-CoV-2 targets for novel drug development ::: Treatment strategy of COVID-19', 'ref_spans': []}, {'text': 'The drug discovery strategy bears the brunt of the COVID-19 outbreak is the test of existing broad-spectrum antiviral agents that have been harnessed to cure other viral infections via utilizing standard tests for measuring the effects of drugs on the cytopathy, virus production along with plaque generation in live and pseudotyped CoVs. Drug discovery utilizing this method encompass interferon α, interferon β, interferon γ, arbidol, ribavirin, along with cyclophilin inhibitors [65,66]. These drugs possess the distinct preponderance of easy access to known pharmacokinetic and pharmacodynamic features, dose regimens, and adverse effects [69]. Nevertheless, they have no specific anti-SARS-CoV-2 effect and may be correlated with severe untoward effects.', 'cite_spans': [{'start': 483, 'end': 485, 'mention': '65', 'ref_id': 'BIBREF64'}, {'start': 486, 'end': 488, 'mention': '66', 'ref_id': 'BIBREF65'}, {'start': 643, 'end': 647, 'mention': '[69]', 'ref_id': 'BIBREF68'}], 'section': 'Drug discovery approaches toward anti-SARS-CoV-2 drug screening ::: Treatment strategy of COVID-19', 'ref_spans': []}, {'text': 'In addition to the test of existing broad-spectrum antiviral agents, another anti-SARS-CoV-2 drug discovery approach might be the chemical libraries screening, which involves a good deal of existing compounds or databases with the information on transcriptional signatures in disparate cell lines [68,69]. Such a method holds the potential of offering a prompt, high-throughput screening of many off-the-shelf composites, which can thereby be further assessed via antiviral detection test. Importantly, a variety of different types of agents have been discovered among these agent repurposing programs, incorporating many that with significant physiological and immune effects (i.e., those affecting the lipid or sterol metabolism, regulation of neurotransmitters, kinase signaling, estrogen receptors, and DNA synthesis/repair) [70,71]. It is noteworthy that this method has the main drawback that most agents are not clinically useful in virtue of their underlying immunosuppressive effects or anti-SARS-CoV-2 half-maximal effective concentration values that significantly surpass the peak serum concentration levels that are obtainable at the therapeutic dose [74]. An interesting exception is the anti-HIV protease inhibitor lopinavir-ritonavir, which has been reported to be effective for SARS-CoV in both non-human primate models and in non-randomized clinical trials [75]. In addition, nelfinavir, a selective inhibitor of HIV protease, which has been demonstrated to possess a sound suppresion of SARS-CoV, suggestive of a potential drug candidate for COVID-19 [76].', 'cite_spans': [{'start': 298, 'end': 300, 'mention': '68', 'ref_id': 'BIBREF67'}, {'start': 301, 'end': 303, 'mention': '69', 'ref_id': 'BIBREF68'}, {'start': 830, 'end': 832, 'mention': '70', 'ref_id': 'BIBREF69'}, {'start': 833, 'end': 835, 'mention': '71', 'ref_id': 'BIBREF70'}, {'start': 1163, 'end': 1167, 'mention': '[74]', 'ref_id': 'BIBREF73'}, {'start': 1374, 'end': 1378, 'mention': '[75]', 'ref_id': 'BIBREF74'}, {'start': 1569, 'end': 1573, 'mention': '[76]', 'ref_id': 'BIBREF75'}], 'section': 'Drug discovery approaches toward anti-SARS-CoV-2 drug screening ::: Treatment strategy of COVID-19', 'ref_spans': []}, {'text': 'The most critical method for anti-SARS-CoV-2 drug discovery includes the de novo development of new, specific drugs based upon the genomic and biophysical understanding of this virus. For instance, the determination of key SARS-CoV-2 targets may bring about the production of siRNA molecules or inhibitors against specific viral enzymes correlated with viral replication. Additionally, mAb targeting host receptors, inhibitors of host cell proteases, host cell endocytosis viruses, along with humanized mAb targeting the RBD, and antiviral peptides targeting the S2 subunit offer various methodology and options for the design and development of possible therapeutics. With the emerging outbreak of the COVID-19 pandemic, the above-mentioned methods are critical for identifying candidate drug composites that can be widely categorized into virus-based and host-based therapy options.', 'cite_spans': [], 'section': 'Drug discovery approaches toward anti-SARS-CoV-2 drug screening ::: Treatment strategy of COVID-19', 'ref_spans': []}, {'text': '\\n(I)Nucleoside Analogs\\n', 'cite_spans': [], 'section': 'Antiviral agents ::: Therapeutics options for SARS-CoV-2 in clinical ::: Treatment strategy of COVID-19', 'ref_spans': []}, {'text': 'Nucleoside analogs can interfere with cellular nucleotide synthesis pathways and terminate viral genome replication through accumulating mutations and cutting off the entry of natural nucleotides [83]. Since nucleosides and nucleotides are the basic components of viral nucleic acids, nucleoside analogs serve as viral RNA synthesis inhibitors in a wide spectrum of RNA viruses. With the view of the targeting ability toward RdRp, nucleoside analogs are responsible for viral RNA replication [84].', 'cite_spans': [{'start': 196, 'end': 200, 'mention': '[83]', 'ref_id': 'BIBREF82'}, {'start': 492, 'end': 496, 'mention': '[84]', 'ref_id': 'BIBREF83'}], 'section': 'Antiviral agents ::: Therapeutics options for SARS-CoV-2 in clinical ::: Treatment strategy of COVID-19', 'ref_spans': []}, {'text': 'Ribavirin is a guanine analog among approved nucleoside analogs used to treat hepatitis c virus and respiratory syncytial virus infections and has been harnessed to cure patients with SARS [85]. Ribavirin was extensively leveraged for patients with or without concomitant usage of steroids during the SARS outbreak in 2003 [86]. When combined with IFN-β, it can exert the synergistical inhibition effect on SARS-associated CoV replication in vitro\\n[87]. However, the efficacy and safety of this agent remain uncertain, and it may arouse adverse reactions like anemia in high doses [77]. In COVID-19 therapy, ribavirin was utilized with pegylated interferon for stimulating the innate antiviral reaction at a relatively lower dose to curtail side effects.', 'cite_spans': [{'start': 189, 'end': 193, 'mention': '[85]', 'ref_id': 'BIBREF84'}, {'start': 323, 'end': 327, 'mention': '[86]', 'ref_id': 'BIBREF85'}, {'start': 448, 'end': 452, 'mention': '[87]', 'ref_id': 'BIBREF86'}, {'start': 581, 'end': 585, 'mention': '[77]', 'ref_id': 'BIBREF76'}], 'section': 'Antiviral agents ::: Therapeutics options for SARS-CoV-2 in clinical ::: Treatment strategy of COVID-19', 'ref_spans': []}, {'text': 'In addition, another promising guanine analog is favipiravir (T-705), which has been approved in Japan for the therapy of influenza virus infections and has also been demonstrated to suppress the replication of Ebola, yellow fever, enterovirus and norovirus [78]. Recently, Wang and coworkers suggested that favipiravir may also be a potential candidate for COVID-19 therapy, which showed effective antiviral activity in Vero E6 cells with an EC50 value of around 61 μM. To improve the condition of COVID-19 patients, favipiravir was utilized with other antiviral drugs like baloxavir marboxil [88].', 'cite_spans': [{'start': 258, 'end': 262, 'mention': '[78]', 'ref_id': 'BIBREF77'}, {'start': 594, 'end': 598, 'mention': '[88]', 'ref_id': 'BIBREF87'}], 'section': 'Antiviral agents ::: Therapeutics options for SARS-CoV-2 in clinical ::: Treatment strategy of COVID-19', 'ref_spans': []}, {'text': \"As an adenine analogue with a similar chemical structure to the approved HIV reverse transcriptase inhibitor tenofovir alafenamide, remdesivir (GS-5734) exhibits broad-spectrum antiviral activity against several RNA viruses and has the capacity of competing with RdRp (Fig. 7\\n) [89]. It also possesses outstanding in vitro antiviral activity compared with lopinavir and ritonavir [90]. In the United States, the first case of SARS-CoV-2 infection was reported, and remdesivir was administered. The patient's clinical condition improved after only one day of remdesivir treatment [79]. A newly released research offered remdesivir for COVID-19 inpatients on the basis of sympathy. In the cohort of patients admitted for treatment of severe COVID-19, patients treated with sympathetic remdesivir achieved relatively good clinical improvement (36 of 53 patients (68%)) [91]. However, recently, leaked data from a crucial remdesivir investigation suggests this potent CoV agent may not be effective. Accordingly, the chief medical officer of Gilead Sciences said that the summary post online might include inappropriate characterizations of the study and it was terminated early due to low enrollment [92]. Paradoxically, the drug's maker announced that in its own trial, more than half of 400 participants with severe COVID-19 had recovered from their illness within two weeks of receiving treatment [93]. However, this study lacked a placebo-controlled arm. Moreover, the outcomes from the National Institute of Allergy and Infectious Diseases (NIAID) further demonstrated that this drug can stop some patients from becoming critically ill [93]. In this regard, these findings suggest that differences in races and differences in clinical trials may lead to disparate experimental outcomes. The real anti-SARS-CoV-2 activity of remdesivir need to be further studied in the long term.(I)Type I interferons\\n\", 'cite_spans': [{'start': 278, 'end': 282, 'mention': '[89]', 'ref_id': 'BIBREF88'}, {'start': 380, 'end': 384, 'mention': '[90]', 'ref_id': 'BIBREF89'}, {'start': 579, 'end': 583, 'mention': '[79]', 'ref_id': 'BIBREF78'}, {'start': 866, 'end': 870, 'mention': '[91]', 'ref_id': 'BIBREF90'}, {'start': 1197, 'end': 1201, 'mention': '[92]', 'ref_id': 'BIBREF91'}, {'start': 1397, 'end': 1401, 'mention': '[93]', 'ref_id': 'BIBREF92'}, {'start': 1638, 'end': 1642, 'mention': '[93]', 'ref_id': 'BIBREF92'}], 'section': 'Antiviral agents ::: Therapeutics options for SARS-CoV-2 in clinical ::: Treatment strategy of COVID-19', 'ref_spans': [{'start': 269, 'end': 275, 'mention': 'Fig. 7', 'ref_id': 'FIGREF6'}]}, {'text': 'Type I interferons designate a cluster of antiviral cytokines consisting of the omnipresent α and β subtypes, along with the ω, ε and κ subtypes, inducing large numbers of proteins that can undermine viral replication in host cells [94]. Former researches have revealed that IFN-β was superior against SARS-CoV in comparison with IFN-α [95]. As for the clinical trial of type I interferons, in China, therapy guidelines for COVID-19 recommend the management of 5 million U of IFN-α via vapor inhalation in patients twice daily with ribavirin as combined therapy [96]. In the detailed study of Sallard et al., they reported that IFN-β1 might constitute a safe and easy to upscale therapy against COVID-19 in the early stages of the disease. Besides, in vitro data indicated that this virus might be significantly more sensitive to IFN-I than other CoVs [80].(I)Protease inhibitors\\n', 'cite_spans': [{'start': 232, 'end': 236, 'mention': '[94]', 'ref_id': 'BIBREF93'}, {'start': 336, 'end': 340, 'mention': '[95]', 'ref_id': 'BIBREF94'}, {'start': 562, 'end': 566, 'mention': '[96]', 'ref_id': 'BIBREF95'}, {'start': 852, 'end': 856, 'mention': '[80]', 'ref_id': 'BIBREF79'}], 'section': 'Antiviral agents ::: Therapeutics options for SARS-CoV-2 in clinical ::: Treatment strategy of COVID-19', 'ref_spans': []}, {'text': 'Protease inhibitorsare promising candidates for antiviral agents, and can block the replication of viral genes via binding to enzymes that are responsible for proteolysis [97]. Lopinavir and ritonavir, both protease inhibitors, have been approved as HIV medicines and have been found to possess antiviral activity against SARS and MERS. For the sake of destroying the SARS-CoV-2, clinical trials had begun to parse out the antiviral property of HIV protease inhibitors among patients. Notwithstanding, the antiviral effect of such inhibitors in coronavirus proteases remains controversial. Notably, a study regarding the comparison of the efficacy of prophylactic remdesivir and therapeutic remdesivir in combination with lopinavir, ritonavir, and interferon β against MERS-CoV unveiled that remdesivir was more effective than the combination therapy in reducing viral load and improving the degree of pathological changes in lung tissue. Aside from the gastrointestinal adverse reactions aroused by lopinavir/ritonavir, it is worth noting that lopinavir/ritonavir treatment alone may fail to offer benefits in comparison with standard care alone. The median time for clinical improvement was 16 d, and the reduction in viral RNA load among patients with severe SARS-CoV-2 did not appear to differ in both cases [98]. Despite the disheartening outcomes, a marginally lower number of deaths were observed among the patients with lopinavir/ritonavir treatment in the late stage of this disease in contrast to the standard-care group. Further, Baden et al. suggested that the concentration of lopinavir/ritonavir required to inhibit the replication of SARS-CoV-2 in the lungs may be higher than the serum level [99]. Besides, nelfinavir, which is a selective inhibitor of HIV protease, has also been displayed to possess a robust suppression of SARS-CoV, indicating an alternative therapeutic option for COVID-19 [76].', 'cite_spans': [{'start': 171, 'end': 175, 'mention': '[97]', 'ref_id': 'BIBREF96'}, {'start': 1312, 'end': 1316, 'mention': '[98]', 'ref_id': 'BIBREF97'}, {'start': 1708, 'end': 1712, 'mention': '[99]', 'ref_id': 'BIBREF98'}, {'start': 1910, 'end': 1914, 'mention': '[76]', 'ref_id': 'BIBREF75'}], 'section': 'Antiviral agents ::: Therapeutics options for SARS-CoV-2 in clinical ::: Treatment strategy of COVID-19', 'ref_spans': []}, {'text': 'Chloroquine, as a drug extensively utilized in anti-malaria and autoimmune diseases, has been found to be a potential broad-spectrum antiviral agent [100]. It can prevent viral infections via elevating the endosomal pH needed for virus-cell fusion and disturbing the glycosylation of SARS-CoV cell receptors [101]. Gao et al. revealed that chloroquine was effective in the therapy of COVID-19-associated pneumonia [102]. Wang and coworkers also conducted in vitro study, and they found that it is an ideal candidate antiviral drug against SARS-CoV-2 infection in Vero E6 cells with EC50 value of around 1 μM [88]. Although several trials had verified that chloroquine suppresses the exacerbation of COVID-19, the optimal dosage of chloroquine will require to be evaluated in future trials [102].', 'cite_spans': [{'start': 149, 'end': 154, 'mention': '[100]', 'ref_id': 'BIBREF99'}, {'start': 308, 'end': 313, 'mention': '[101]', 'ref_id': 'BIBREF100'}, {'start': 414, 'end': 419, 'mention': '[102]', 'ref_id': 'BIBREF101'}, {'start': 608, 'end': 612, 'mention': '[88]', 'ref_id': 'BIBREF87'}, {'start': 789, 'end': 794, 'mention': '[102]', 'ref_id': 'BIBREF101'}], 'section': 'Chloroquine and hydroxychloroquine ::: Therapeutics options for SARS-CoV-2 in clinical ::: Treatment strategy of COVID-19', 'ref_spans': []}, {'text': 'Hydroxychloroquine is an analog of chloroquine, and there are few studies on its interaction [103]. In previous SARS outbreaks, hydroxychloroquine was found to possess anti-SARS-CoV capacity in vitro\\n[104]. In line with the research of Yao et al. by applying a physiologically based pharmacokinetic model, they found that hydroxychloroquine is more effective than chloroquine in Vero cells infected with SARS-CoV-2 [105]. Of note, it has been revealed that cytokines IL-6 and IL-10 are elevated in response to SARS-CoV-2 infection, which may induce the cytokine storm (Fig. 5), which in turn cause multiple organ failure and death [49]. Both chloroquine and hydroxychloroquine possess immunomodulatory effects and can inhibit such immune responses [106]. Hence, Chinese hospitals and Oxford University had initiated 21 clinical studies to evaluate the efficacy of these drugs in COVID-19 infection. The further crucial study may relate with the determination of whether the benefit of chloroquine treatment is hinged on the age of the patients as well as the clinical manifestations [107]. Recently, a multinational registry analysis of the application of hydroxychloroquine or chloroquine unraveled that there might be an increased frequency of ventricular arrhythmias during the treatment of COVID-19 diseases [108]. The WHO has also suspended hydroxychloroquine treatment in COVID-19 solidarity trial in terms of the outcomes of in-hospital mortality and de-novo ventricular arrhythmias [109].', 'cite_spans': [{'start': 93, 'end': 98, 'mention': '[103]', 'ref_id': 'BIBREF102'}, {'start': 200, 'end': 205, 'mention': '[104]', 'ref_id': 'BIBREF103'}, {'start': 415, 'end': 420, 'mention': '[105]', 'ref_id': 'BIBREF104'}, {'start': 631, 'end': 635, 'mention': '[49]', 'ref_id': 'BIBREF48'}, {'start': 748, 'end': 753, 'mention': '[106]', 'ref_id': 'BIBREF105'}, {'start': 1083, 'end': 1088, 'mention': '[107]', 'ref_id': 'BIBREF106'}, {'start': 1312, 'end': 1317, 'mention': '[108]', 'ref_id': 'BIBREF107'}, {'start': 1490, 'end': 1495, 'mention': '[109]', 'ref_id': 'BIBREF108'}], 'section': 'Chloroquine and hydroxychloroquine ::: Therapeutics options for SARS-CoV-2 in clinical ::: Treatment strategy of COVID-19', 'ref_spans': [{'start': 569, 'end': 575, 'mention': 'Fig. 5', 'ref_id': 'FIGREF4'}]}, {'text': 'Monoclonal or polyclonal antibodies have been recommended as tools for preventing and treating viral infections. Considering the relatively high RBD in SARS-CoV-2, the cross-reactivity of anti-SARS-CoV antibodies with the COVID-19 S protein was evaluated. Tian et al. determined the potent binding of S protein via a SARS CoV-specific human mAb CR3022 [82]. Unfortunately, other SARS-CoV RBD directed mAbs (i.e., 230, m396, and 80R) cannot bind to the COVID-19 RBD [32]. In this respect, CR3022 may be a promising therapeutic candidate, either alone or in combination with other neutralizing mAbs, for the therapy of COVID-19 disease. In addition, tocilizumab is a monoclonal antibody for the therapy of RA exacerbation. It was designed to suppress the binding of IL-6 to its receptors, hence mitigating cytokine release syndrome. At present, it is also being trailed for the COVID-19 therapy [110]. Interestingly, Shi et al. reported a human neutralizing antibody targets the receptor binding site of SARS-CoV-2 [111]. In their work, they successfully isolated 2 specific human mAbs (named as CA1 and CB6) from a convalescent COVID-19 patient. CA1 and CB6 demonstrated potent SARS-CoV2-specifc neutralization activity in vitro against SARS-CoV-2. Moreover, CB6 could suppress SARS-CoV-2 infection in rhesus monkeys at both prophylactic and treatment settings. The following structural studies unraveled that CB6 could recognize an epitope that overlaps with ACE2-binding sites in SARS-CoV-2 receptor binding domain, thus interfering with the virus/receptor interactions by both steric hindrance and direct interface-residue competition [111]. Another recent study revealed the isolation and characterization of 206 RBD-specific mAbs derived from single B cells of eight SARS-CoV-2 infected individuals [112]. The researchers identified antibodies with potent anti-SARS-CoV-2 neutralization capacity that correlated with their competitive capacity with ACE2 for RBD binding. Of note, crystal structure analysis of RBD-bound antibody uncovered steric hindrance that suppressed viral engagement with ACE2 and thereby blocked viral entry [112]. These findings implicate that anti-RBD antibodies may be viral species-specific inhibitors.', 'cite_spans': [{'start': 352, 'end': 356, 'mention': '[82]', 'ref_id': 'BIBREF81'}, {'start': 465, 'end': 469, 'mention': '[32]', 'ref_id': 'BIBREF31'}, {'start': 893, 'end': 898, 'mention': '[110]', 'ref_id': 'BIBREF109'}, {'start': 1013, 'end': 1018, 'mention': '[111]', 'ref_id': 'BIBREF110'}, {'start': 1637, 'end': 1642, 'mention': '[111]', 'ref_id': 'BIBREF110'}, {'start': 1803, 'end': 1808, 'mention': '[112]', 'ref_id': 'BIBREF111'}, {'start': 2135, 'end': 2140, 'mention': '[112]', 'ref_id': 'BIBREF111'}], 'section': 'Monoclonal or polyclonal antibodies ::: Therapeutics options for SARS-CoV-2 in clinical ::: Treatment strategy of COVID-19', 'ref_spans': []}, {'text': 'Besides, most patients with severe COVID-19 suffered a lot from the cytokine storm (Figure 5) [53]. So, neutralizing antibodies against other pro-inflammatory cytokines may be another promising strategy to dampen the inflammatory responses, thus responding well to treat the COVID-19. In a clinical trial conducted in Anhui, China, IL-6 receptor-targeted mAb tocilizumab was utilized to treat 21 patients with severe COVID-19. Clinical data showed quick fever control and improved respiratory function [113]. Overall, the development of COVID-19-specific antibodies takes a long time, meaning the difficulty in applying antibodies for neoteric pathogens to clinical practice in a brief period.', 'cite_spans': [{'start': 94, 'end': 98, 'mention': '[53]', 'ref_id': 'BIBREF52'}, {'start': 502, 'end': 507, 'mention': '[113]', 'ref_id': 'BIBREF112'}], 'section': 'Monoclonal or polyclonal antibodies ::: Therapeutics options for SARS-CoV-2 in clinical ::: Treatment strategy of COVID-19', 'ref_spans': [{'start': 84, 'end': 92, 'mention': 'Figure 5', 'ref_id': 'FIGREF4'}]}, {'text': 'As a class of drug that lowers inflammation in the body, corticosteroids have been utilized in several serious viral respiratory infections such as SARS-CoV and MERS-CoV with limited benefits. However, in some cases, there is proof of delayed viral clearance and increased rates of secondary infection and mortality [114]. In a study of 41 COVID-19 patients, 22% were given corticosteroids, which inhibit inflammation in the lungs [49]. However, according to the current WHO interim guidance, glucocorticoids are not recommended for routine treatment unless otherwise indicated, as other coronaviruses and influenza studies have identified possible injury and increased risk of death from glucocorticoid therapy [115]. Despite the potential antiviral activity toward COVID-19, corticosteroids should not be given principally, and corticosteroid pulse therapy should be conducted with caution. In addition, corticosteroid therapy for the SARS therapy yielded adverse effects such as psychosis, diabetes, and avascular necrosis [116]. In general, the treatment with corticosteroids may be harmful, but such agents can be prescribed to the right patient at the proper time.', 'cite_spans': [{'start': 316, 'end': 321, 'mention': '[114]', 'ref_id': 'BIBREF113'}, {'start': 431, 'end': 435, 'mention': '[49]', 'ref_id': 'BIBREF48'}, {'start': 712, 'end': 717, 'mention': '[115]', 'ref_id': 'BIBREF114'}, {'start': 1026, 'end': 1031, 'mention': '[116]', 'ref_id': 'BIBREF115'}], 'section': 'Corticosteroids ::: Therapeutics options for SARS-CoV-2 in clinical ::: Treatment strategy of COVID-19', 'ref_spans': []}, {'text': 'Convalescent plasma has also been utilized as a last resort to improve the survival in patients with miscellaneous viral infections (i.e., SARS, H5N1, H1N1, and Ebola virus infection) [117]. The theoretical basis for the therapeutic effect of plasma in the recovery phase is that immunoglobulin Abs in the plasma of patients recovering from the viral infection may inhibit viremia. To date, in the previous SARS therapy, convalescent plasma given early after the onset of symptoms reduced overall mortality after treatment compared to placebo or no treatment [118]. For the test of convalescent plasma effect on COVID-19 infection, Zhou et al. found that the SARS-CoV-2 isolated from bronchoalveolar lavage fluid in a severe patient could be neutralized by the serum of several patients [6]. Additionally, another study of Shen et al. showed that improvements in clinical conditions were observed following plasma transfusions, improvements included normalization of body temperature within three days, a decrease in Sequential Organ Failure Assessment score, resolution of ARDS, and decline in viral loads [119]. During the emerging of COVID-19 infection in China, the National Health Commission of China also called on convalescent patients to donate blood for COVID – 19 therapy. Despite the difficulty in collecting the plasma, convalescent plasma therapy showed great therapeutic potential for the COVID-19 therapy.', 'cite_spans': [{'start': 184, 'end': 189, 'mention': '[117]', 'ref_id': 'BIBREF116'}, {'start': 559, 'end': 564, 'mention': '[118]', 'ref_id': 'BIBREF117'}, {'start': 787, 'end': 790, 'mention': '[6]', 'ref_id': 'BIBREF5'}, {'start': 1107, 'end': 1112, 'mention': '[119]', 'ref_id': 'BIBREF118'}], 'section': 'Convalescent plasma transfusion ::: Therapeutics options for SARS-CoV-2 in clinical ::: Treatment strategy of COVID-19', 'ref_spans': []}, {'text': 'Based on data mining, the Traditional Chinese Medicine was also regarded as an alternative method for the treatment of COVID-19 disease in high-risk populations. During the COVID-19 outbreak in China, Chinese herbal drugs such as Astragali Radix, Glycyrrhizae Radix Et Rhizoma, Atractylodis Macrocephalae Rhizoma, Saposhnikoviae Radix, Lonicerae Japonicae Flos and Fructus forsythia were commonly harnessed for improving the treatment effect of COVID-19 infections. Herbal medications can offer a more effective and personalized treatment by adjusting the specific medicine for each patient based on the disparate syndromes [120]. Previous studies also indicated that the patients with SARS-CoV infection have benefited from TCM treatment [121], involving amelioration of adverse effects of tranditional therapeutics. Some TCM such as Lianhuaqingwen capsules exert anti-viral replication and anti-inflammatory activity against SARS-CoV-2 in vitro\\n[122]. However, the experimental study on the efficacy of herbal medicines lags behind the clinical application of traditional Chinese medicine in the COVID-19 treatment [123]. Hence, rigorous clinical trials on large populations should be carried out to identify the potential therapeutic efficacy of herbal medications.', 'cite_spans': [{'start': 624, 'end': 629, 'mention': '[120]', 'ref_id': 'BIBREF119'}, {'start': 739, 'end': 744, 'mention': '[121]', 'ref_id': 'BIBREF120'}, {'start': 947, 'end': 952, 'mention': '[122]', 'ref_id': 'BIBREF121'}, {'start': 1117, 'end': 1122, 'mention': '[123]', 'ref_id': 'BIBREF122'}], 'section': 'Herbal medications ::: Therapeutics options for SARS-CoV-2 in clinical ::: Treatment strategy of COVID-19', 'ref_spans': []}, {'text': 'Stem cell therapy is also making its way into COVID-19 disease treatment. Recently, some researches have demonstrated that the intravenous transplantation of mesenchymal stem cells (MSCs) was safe and efficient for COVID-19 pneumonia, especially for critically ill patients [124]. As mentioned, COVID-19 infection may induce uncontrolled inflammatory innate reactions along with undermined adaptive immune reactions, subsequently leading to detrimental tissue damage. MSC-based immunomodulation treatment is able to counteract the cytokine storm aroused by the immune system and foster endogenous repair via reparative attributes of the stem cells [125]. As it is unveiled by the recent study of China, seven COVID-19 patients (1 critically ill patient, 4 critically ill patients, and 2 patients with mild symptoms) received intravenous injection of bone marrow MSCs. In all cases, the patients were cured, while the 3 patients in the placebo control group all suffered from severe illness, 1 died, 1 developed ARDS, and 1 was in stable condition [126]. This study uncovered that MSCs could reclaim the lung microenvironment, protect the alveolar epithelial cells, block pulmonary fibrosis, and treat pulmonary dysfunction [124]. Notably, the FDA has also opened the way to the compassionate use of MSCs intravenous infusions in patients with COVID-19 ARDS and very dismal prognosis [127]. Although such a kind of treatment is an ideal option for COVID-19 therapy, it has limitations regarding the supply of clinical-grade MSCs and the rate of preparation for subsequent clinical use.', 'cite_spans': [{'start': 274, 'end': 279, 'mention': '[124]', 'ref_id': 'BIBREF123'}, {'start': 648, 'end': 653, 'mention': '[125]', 'ref_id': 'BIBREF124'}, {'start': 1047, 'end': 1052, 'mention': '[126]', 'ref_id': 'BIBREF125'}, {'start': 1223, 'end': 1228, 'mention': '[124]', 'ref_id': 'BIBREF123'}, {'start': 1383, 'end': 1388, 'mention': '[127]', 'ref_id': 'BIBREF126'}], 'section': 'Mesenchymal stem cell therapy ::: Therapeutics options for SARS-CoV-2 in clinical ::: Treatment strategy of COVID-19', 'ref_spans': []}, {'text': 'Hydrogen peroxide (H2O2) appears to be another potential therapeutic option for COVID-19. Of note, health experts have said that this compound could help prevent the virus from spreading across the body and from causing damage [128]. A recent study illustrated that even just 0.5% of H2O2 could kill human CoVs, such as those that caused SARS and MERS [129]. Inhaling the vapor with a nebulizer has been the most convenient way to receive H2O2 to fight viral infections. The microscopic mist can easily penetrate the nostrils, sinuses and lungs, which are commonly affected by respiratory diseases like COVID-19. Besides, molecular hydrogen has been verified to favorably modulate the generation of both O2\\n− and NO through influencing NADPH oxidase, and the NOS enzymes [130]. According to the National Health Commission of China, the conditional use of mixed inhalation of hydrogen and oxygen (H2/O2: 66.6%/33.3%) treatment may improve the symptoms [131].', 'cite_spans': [{'start': 227, 'end': 232, 'mention': '[128]', 'ref_id': 'BIBREF127'}, {'start': 352, 'end': 357, 'mention': '[129]', 'ref_id': 'BIBREF128'}, {'start': 771, 'end': 776, 'mention': '[130]', 'ref_id': 'BIBREF129'}, {'start': 951, 'end': 956, 'mention': '[131]', 'ref_id': 'BIBREF130'}], 'section': 'Other therapies ::: Therapeutics options for SARS-CoV-2 in clinical ::: Treatment strategy of COVID-19', 'ref_spans': []}, {'text': 'Lung transplantation can be performed in advanced patients with respiratory failure owing to COVID-19-related pulmonary fibrosis. As it is reported in a clinical study, lung transplantation may offer the ultimate treatment option for severe patients to avoid certain deaths, while at the same time protecting transplantation doctors and medical staff appropriately [132].', 'cite_spans': [{'start': 365, 'end': 370, 'mention': '[132]', 'ref_id': 'BIBREF131'}], 'section': 'Other therapies ::: Therapeutics options for SARS-CoV-2 in clinical ::: Treatment strategy of COVID-19', 'ref_spans': []}, {'text': 'DNA vaccines provide a precise and flexible tactic to deliver antigens to the immune system and can contain additional sequences of coding molecules to manipulate the results [147]. So far, a variety of DNA vaccine platforms have been exploited to enhance the vaccine efficacy through electroporation to deliver plasmids and addition of adjuvants, yielding improved the immune responses [148]. Inovio Pharmaceuticals have begun pre-clinical trials of a DNA vaccine (named as INO-4800) against COVID-19 [149]. This vaccine can induce T cell activation via transferring DNA plasmids that express the SARS-CoV-2 S proteins [150]. This vaccine platform holds the preponderance of producing therapeutic antibodies and activating immune cells to deliver the vaccine to patients through the skin. Recently, Yu and colleagues developed a series of DNA vaccine candidates expressing different forms of SARS-CoV-2 S protein and assessed them in 35 rhesus macaques [151]. Vaccinated animals developed humoral and cellular immune responses, including neutralizing antibody titers in comparison with those found in convalescent humans and macaques infected with SARS-CoV-2. Surprisingly, all animals were challenged with SARS-CoV-2 after vaccination, and the vaccine encoding the full-length S protein resulted in over 3.1 and over 3.7 log10 reductions in median viral loads in bronchoalveolar lavage and nasal mucosa, respectively, as compared with sham controls [151]. Their data indicated vaccine protection against SARS-CoV-2 in nonhuman primates [152]. Hence, in the near future, with the support of CEPI, these findings will boost the vaccine development and the corporation is making provision for the first phase of trials in the United States and China [153].', 'cite_spans': [{'start': 175, 'end': 180, 'mention': '[147]', 'ref_id': 'BIBREF146'}, {'start': 387, 'end': 392, 'mention': '[148]', 'ref_id': 'BIBREF147'}, {'start': 502, 'end': 507, 'mention': '[149]', 'ref_id': 'BIBREF148'}, {'start': 620, 'end': 625, 'mention': '[150]', 'ref_id': 'BIBREF149'}, {'start': 954, 'end': 959, 'mention': '[151]', 'ref_id': 'BIBREF150'}, {'start': 1451, 'end': 1456, 'mention': '[151]', 'ref_id': 'BIBREF150'}, {'start': 1538, 'end': 1543, 'mention': '[152]', 'ref_id': 'BIBREF151'}, {'start': 1749, 'end': 1754, 'mention': '[153]', 'ref_id': 'BIBREF152'}], 'section': 'DNA-based Vaccines ::: Development of SARS-CoV-2 vaccines', 'ref_spans': []}, {'text': 'The mRNA vaccine is a hopeful alternative to traditional vaccine methods in virtue of its high efficiency, rapid development capabilities, and the potential for low-cost manufacturing [154]. Recently, Moderna, Inc. has launched phase I clinical trials for mRNA-1273, which encodes S protein of SARS-CoV-2. This mRNA-vaccine was fabricated with the cooperation of the National Institute of Allergy and Infectious Diseases [155]. mRNA-1273, which encodes a prefusion-stabilized form of the SARS-CoV-2 spike, is under test for a broad dosing range (25—250 micrograms) during its phase 1 submission [143]. The firm further announced that although commercial vaccines are unlikely to be marketed in at least 12 to 18 months, in an emergency, possibly in the fall of 2020, some people, including medical professionals, may obtain the vaccine [146,147].', 'cite_spans': [{'start': 184, 'end': 189, 'mention': '[154]', 'ref_id': 'BIBREF153'}, {'start': 421, 'end': 426, 'mention': '[155]', 'ref_id': 'BIBREF154'}, {'start': 595, 'end': 600, 'mention': '[143]', 'ref_id': 'BIBREF142'}, {'start': 837, 'end': 840, 'mention': '146', 'ref_id': 'BIBREF145'}, {'start': 841, 'end': 844, 'mention': '147', 'ref_id': 'BIBREF146'}], 'section': 'mRNA-based Vaccines ::: Development of SARS-CoV-2 vaccines', 'ref_spans': []}, {'text': \"Subunit vaccines are superior to other types of vaccines since they are highly safe and possess fewer adverse effects via eliciting the immune system without drawing into any infectious viruses [9]. It has also been reported that the enhancement of T cell responses and generation of high titer neutralizing Abs were observed in vivo during the vaccine development researches [157]. Clover Biopharmaceuticals was pre-clinically testing a recombinant subunit vaccine in the light of the S-Trimer of the SARS-CoV-2 [158]. The researchers detected the antigen-specific neutralizing Abs in the sera of fully recovered patients [158]. Besides, GlaxoSmithKline (GSK) disclosed a vaccine which can elicit a protective immune response against SARS. The vaccine contains an S protein immunogen, which was combined with the emulsion adjuvant, GSK2, yielding elevated level of anti-SARS-CoV IgG2a/IgG2b Ab responses. Recently, GSK and Clover Biopharmaceuticals announced a partnership to enhance immune response via introducing GSK's adjuvant system to S-Trimer [159]. In addition, the team of the University of Queensland is also designing subunit vaccines via the transformative technology, known as “molecular clamp” [160]. Molecular clamps are peptides that stabilize surface proteins, improve the recognition of the correct antigens, and lead to a more robust immune response. This vaccine platform can be easily applied to a variety of enveloped viruses and can rapidly manufacture their products [160].\", 'cite_spans': [{'start': 194, 'end': 197, 'mention': '[9]', 'ref_id': 'BIBREF8'}, {'start': 376, 'end': 381, 'mention': '[157]', 'ref_id': 'BIBREF156'}, {'start': 513, 'end': 518, 'mention': '[158]', 'ref_id': 'BIBREF157'}, {'start': 623, 'end': 628, 'mention': '[158]', 'ref_id': 'BIBREF157'}, {'start': 1051, 'end': 1056, 'mention': '[159]', 'ref_id': 'BIBREF158'}, {'start': 1209, 'end': 1214, 'mention': '[160]', 'ref_id': 'BIBREF159'}, {'start': 1492, 'end': 1497, 'mention': '[160]', 'ref_id': 'BIBREF159'}], 'section': 'Recombinant Subunit Vaccines ::: Development of SARS-CoV-2 vaccines', 'ref_spans': []}, {'text': 'Further, patent application US20060002947 disclosed the fabrication of hybrid peptides comprised of three elements: (1) an invariant chain (Ii) essential peptide; (2) a chemical structure linking the Ii to the antigenic epitope; (3) an antigen epitope binding MHC II molecule. This theory was harnessed to produce Ii-Key/MHC II SARS hybrids. In this respect, Generex reported that they will employ its Ii-Key immune system activation biotechnology to generate a COVID-19 viral peptide vaccine for human clinical practice [161].', 'cite_spans': [{'start': 521, 'end': 526, 'mention': '[161]', 'ref_id': 'BIBREF160'}], 'section': 'Recombinant Subunit Vaccines ::: Development of SARS-CoV-2 vaccines', 'ref_spans': []}, {'text': 'Aside from the aforementioned approaches, Genexine Inc. is exploring a new vaccine via utilizing the Hyleukin-7 platform technology [162]. Such a platform may improve the immune responses fusing IL-7 with hyFc, aiming to hybridize IgD and IgG4 for the long-term effect of Fc fusion proteins. Specifically, IgD possesses a flexible hinge structure and can maximize the biological activity [153,154]. IgG4 possesses an unexposed junction site that mitigates immunogenicity via preventing antibody-dependent cellular cytotoxicity [155,156]. This corporation reported enhanced vaccine efficacy, lung T cell accumulation, and plasmacytoid dendritic cell growth after Fc-fused IL-7 treatment in the virus infection models [166].', 'cite_spans': [{'start': 132, 'end': 137, 'mention': '[162]', 'ref_id': 'BIBREF161'}, {'start': 389, 'end': 392, 'mention': '153', 'ref_id': 'BIBREF152'}, {'start': 393, 'end': 396, 'mention': '154', 'ref_id': 'BIBREF153'}, {'start': 528, 'end': 531, 'mention': '155', 'ref_id': 'BIBREF154'}, {'start': 532, 'end': 535, 'mention': '156', 'ref_id': 'BIBREF155'}, {'start': 716, 'end': 721, 'mention': '[166]', 'ref_id': 'BIBREF165'}], 'section': 'Other Vaccine Approaches ::: Development of SARS-CoV-2 vaccines', 'ref_spans': []}, {'text': 'Despite prodigious global efforts made to contain SARS-CoV-2, the current COVID-19 epidemic has expanded into a full-blown pandemic, yielding outright panic and economic slowdown. Health-care systems in many regions are overburdened and under-resourced, impelling medical workers and governors to make formerly unthinkable judgments concerning the allotment of medical care. In the wake of such a severe situation, a number of researches have trailed strategies to tackling the direct impact of COVID-19, either by modeling studies of the viral activity or via parsing out potential therapeutic options and finding a vaccine to end the coronavirus pandemic. Although the development of therapeutics and vaccines for the COVID-19 therapy is still in its middle stage, some marked advances have been made from complete genome sequencing of SARS-CoV-2 to the clinical practice of COVID-19 vaccines.', 'cite_spans': [], 'section': 'Conclusions and inspirations', 'ref_spans': []}, {'text': 'Rapid genome-wide association of SARS-CoV-2, along with international sharing of information, enabled us to generate rapider and more proper diagnostic tools. As set forth, the major fashioned diagnostic tools are qRT-PCR-based approaches, requiring a long time for specimen preparation and analysis, thus putting off the imperative actions for COVID-19 infections. Of note,', 'cite_spans': [], 'section': 'Conclusions and inspirations', 'ref_spans': []}, {'text': 'In acute respiratory infections, RT-PCR is commonly utilized for detecting pathogenic viruses in respiratory secretions. The positive rate of PCR from oropharyngeal swabs is not very high, implicating the requirement of more swab testing are need to confirm the diagnosis. With the continuous spread of SARS-CoV-2, there is a desperate demand to develop rapid diagnostic methods that can be tested more efficiently.', 'cite_spans': [], 'section': 'Conclusions and inspirations', 'ref_spans': []}, {'text': 'According to reports, certain drugs are known to effectively treat patients with COVID-19. Nevertheless, the lack of clinical data may render the clinical prognosis difficult to predict. In addition to the judicious design of novel therapeutics that target viral replication or immunopathology, currently, rapid screening of therapeutic agents to repurpose FDA-approved and well-characterized agents might be a more workable method. Considering the severity of the recent zoonotic coronavirus outbreaks, therapeutic drugs for pancoronavirus should be carried out to cope with future outbreaks.', 'cite_spans': [], 'section': 'Conclusions and inspirations', 'ref_spans': []}, {'text': 'With respect to the detailed exploitation of COVID-19 vaccines, several pharmaceutical corporations and institutions have also launched the project for vaccine development. Notwithstanding, the commercial market for vaccines, especially vaccines for emerging infectious diseases, is confined by the high cost and time required for vaccine development along with the uncertainty of profitability. Accordingly, CEPI is combining efforts to encourage the progress of vaccines against COVID-19. Besides, in the near future, the production of a lot of vaccines may bring about the challenge in scaling up manufacturing quickly, because the infrastructure needed will differ in virtue of the vaccine type. Another concern regarding the vaccine development may be the antibody-dependent enhancement issue.', 'cite_spans': [], 'section': 'Conclusions and inspirations', 'ref_spans': []}, {'text': 'In addition, this global epidemic crisis indicated that host-species expansion or interspecies transmission of neoteric coronavirus to humans might be unavoidable. Clinicians and researchers should combine efforts to swiftly evolve our perception of all facets of SARS-CoV-2 infections and fill in the gaps concerning the emergence of this virus. Ultimately, we can also learn from this epidemic that we need to improve our disease monitoring and surveillance system to prevent such a serious disease outbreak. Digital technologies such as big-data analytics, artificial intelligence, and blockchain technology may also need to be fully exploited and developed, thereby enabling real-time data collection at scale and modeling risk associations immediately to contain the pathophoresis.', 'cite_spans': [], 'section': 'Conclusions and inspirations', 'ref_spans': []}, {'text': 'Overall, the challenges posed by 21st century epidemics are real and changing: future epidemics will be fueled by various internal and external causes. In our response, we must view pandemics as interconnected cycles, not isolated events, and while we cannot forecast specific outbreaks, we can make provision for them.', 'cite_spans': [], 'section': 'Conclusions and inspirations', 'ref_spans': []}, {'text': '[58, 67, 72, 73, 142, 144, 145, [163], [164], [165]]', 'cite_spans': [{'start': 1, 'end': 3, 'mention': '58', 'ref_id': 'BIBREF57'}, {'start': 5, 'end': 7, 'mention': '67', 'ref_id': 'BIBREF66'}, {'start': 9, 'end': 11, 'mention': '72', 'ref_id': 'BIBREF71'}, {'start': 13, 'end': 15, 'mention': '73', 'ref_id': 'BIBREF72'}, {'start': 17, 'end': 20, 'mention': '142', 'ref_id': 'BIBREF141'}, {'start': 22, 'end': 25, 'mention': '144', 'ref_id': 'BIBREF143'}, {'start': 27, 'end': 30, 'mention': '145', 'ref_id': 'BIBREF144'}, {'start': 32, 'end': 37, 'mention': '[163]', 'ref_id': 'BIBREF162'}, {'start': 39, 'end': 44, 'mention': '[164]', 'ref_id': 'BIBREF163'}, {'start': 46, 'end': 51, 'mention': '[165]', 'ref_id': 'BIBREF164'}], 'section': 'Uncited References:', 'ref_spans': []}, {'text': 'The authors assert that there is no conflict of interests concerning the publication of this review.', 'cite_spans': [], 'section': 'Declaration of Competing Interest', 'ref_spans': []}], 'paragraphs': [{'text': 'The incubation period refers to the time between exposure to the virus and initial symptoms. A research report on the early propagation dynamics of COVID-19 unveiled that the average incubation period of COVID-19 was around 5 d, and its 95% distribution was 12.5 d [19]. Another study analyzing the travel history and symptoms in 88 confirmed cases revealed a similar average incubation period of around 6 days [20]. In addition, there was an unusual case with an incubation period of up to 19 d [21]. Although such a long incubation period may be a low probability event (the condition of 14 d was suggested by experts for quarantine), the longer incubation time indicates the adjustment of screening and control policies [22].', 'id': '00004'}], 'title': 'Recent insights for the emerging COVID-19: drug discovery, therapeutic options and vaccine development', 'sha': '5b804f88f68fa6a81bf634c6740ff79253ba471a', 'doi': '10.1016/j.ajps.2020.06.001', 'score': 9.138199806213379}, {'rank': 198, 'abstract': 'Since December 2019, China has experienced a widespread outbreak of COVID-19. However, at the early stage of outbreak, investigations revealed a variety of patterns resulting in the transmission of COVID-19. Thus, it is essential to understand the transmission types and the potential for sustained human-to-human transmission. Moreover, the information regarding the characteristics of transmission helps in coordinating the current screening programme, and controlling and containing measures, and also, helps in deciding the appropriate quarantine duration. Thus, this investigation reports an outbreak of COVID-19 in a family residing in Wenzhou, Zhejiang, China during the month of January−February 2020.', 'body_text': [{'text': 'On 31 December 2019, a novel coronavirus was first suspected and reported in Wuhan, China. Subsequently, on 7 January 2020, this coronavirus, termed as SARS-CoV-2, was identified as the cause of an outbreak of viral pneumonia (COVID-19) in Wuhan [1, 2]. The outbreak has resulted in a global epidemic with a number of confirmed cases of COVID-19 being reported in China and abroad, and new cases are being diagnosed daily [3–7]. On 31 January 2020, the World Health Organization (WHO) declared the outbreak of COVID-19 as a public health emergency of international concern (PHEIC) [8]. In China, during the initial stage of the outbreak, human-to-human transmission of COVID-19 was observed and it included family clusters and healthcare settings [9–13]. This human-to-human transmission led to accelerated spread of COVID-19, ultimately resulting in intercity spread [12]. During the spring festival, a patient with COVID-19, from city X, visited her family living in village A, located in the southeast of Zhejiang province, China. This resulted in a familial outbreak of COVID-19. This paper reports the epidemiological features of an outbreak of COVID-19 in a family of one asymptomatic and eight symptomatic individuals residing in village A, Wenzhou, China.', 'cite_spans': [{'start': 247, 'end': 248, 'mention': '1', 'ref_id': 'BIBREF0'}, {'start': 250, 'end': 251, 'mention': '2', 'ref_id': 'BIBREF11'}, {'start': 423, 'end': 424, 'mention': '3', 'ref_id': 'BIBREF16'}, {'start': 425, 'end': 426, 'mention': '7', 'ref_id': 'BIBREF20'}, {'start': 582, 'end': 583, 'mention': '8', 'ref_id': 'BIBREF21'}, {'start': 748, 'end': 749, 'mention': '9', 'ref_id': 'BIBREF22'}, {'start': 750, 'end': 752, 'mention': '13', 'ref_id': 'BIBREF4'}, {'start': 869, 'end': 871, 'mention': '12', 'ref_id': 'BIBREF3'}], 'section': 'Introduction', 'ref_spans': []}, {'text': 'Definitions of suspected case and confirmed case: patients with any two of the following clinical features and any epidemiological risk were the suspected case: (1) clinical features: fever, imaging features of pneumonia, normal or reduced white blood cell count, or reduced lymphocyte count in the early stages of the disease onset. (2) Epidemiologic risk: a history of travel to or residence in Wuhan city, China or other cities with continuous transmission of local cases in the last 14 days before symptom onset; contact with patients with fever or respiratory symptoms from Wuhan city, China or other cities with continuous transmission of local cases in the last 14 days before symptom onset; or epidemiologically connected to SARS-CoV-2 infections or clustered onsets. Suspected case with positive evidence for the SARS-CoV-2 by the real-time PCR test for nucleic acid in respiratory or blood samples is the confirmed case [14].', 'cite_spans': [{'start': 931, 'end': 933, 'mention': '14', 'ref_id': 'BIBREF5'}], 'section': 'Case definition ::: Methods', 'ref_spans': []}, {'text': 'Definitions of close contacts: those who have one of the following contacts after the onset of confirmed cases in the absence of effective protection [14]: (1) those who live, study, work or have close contact with the confirmed cases, or other close contacts such as working closely with or sharing the same classroom or living in the same house with the confirmed case. (2) Medical and nursing staff and their family members living with them, who treated, nursed or visited the confirmed case, or other personnel who have similar close contact with the case, such as providing direct treatment or care of the case, visiting the case or staying in a closed environment where the cases are located; other patients or caregivers in the same room with case. (3) People who have close contact with the patients in the same transport vehicle, including those who had taken care of the patients on the vehicle; the person who had accompanied the patients (family members, colleagues, friends etc.); other passengers and traffic staff considered having likely close contact with the patients by investigation and evaluation.', 'cite_spans': [{'start': 151, 'end': 153, 'mention': '14', 'ref_id': 'BIBREF5'}], 'section': 'Case definition ::: Methods', 'ref_spans': []}, {'text': 'Throat swabs for patients and contacts were collected. SARS-CoV-2 nucleic acid detection was carried out by the real-time PCR method.', 'cite_spans': [], 'section': 'Laboratory testing ::: Methods', 'ref_spans': []}, {'text': 'As this COVID-19 family cluster was an emergency public health issue, an ethics review was not carried out.', 'cite_spans': [], 'section': 'Laboratory testing ::: Methods', 'ref_spans': []}, {'text': 'On 29 January 2020, a 46-year-old female, the index case, developed fever with chills, dry cough, stuffy and runny nose and headache. On the next day, she visited a doctor in clinic and was prescribed anti-cold medication. However, the symptoms did not improve. On the evening of 31 January, she was sent to the emergency department of the county hospital for further treatment. On admission, her body temperature was 38.7 °C. Routine blood examination revealed a white blood cell count of 10.16 × 109/l, neutrocyte ratio of 78.2%, neutrocyte count of 7.94 × 109/l, lymphocyte ratio of 10.3%, and lymphocyte count of 1.05 × 109/l. Her C-reactive protein (CRP) was found to be 33.66/l.', 'cite_spans': [], 'section': 'Index case finding ::: Results', 'ref_spans': []}, {'text': 'Chest radiography on CT revealed an infectious focus involving the lower lobes of both the lungs, especially the right lung. There was a transmittance of light from the lungs, and increased intensity of patchy nodular density shadow with fuzzy borders of lower lobes of both the lungs. The abnormal density shadows were observed in the other areas of the lungs. The lymph nodes in mediastinum were found to be enlarged. The cardiac silhouette was not found to be obviously abnormal. Moreover, there was no presence of pleural effusions.', 'cite_spans': [], 'section': 'Index case finding ::: Results', 'ref_spans': []}, {'text': 'She was engaged in a wholesale indoor business in city X, China. She had no history of comorbidities such as hypertension, diabetes or chronic hepatopathy. Neither she had a history of travel to Wuhan, nor attended a dinner party. However, on 22 January 2020, she had a train journey travelling for >24 h from city X to Wenzhou, with her husband. On enquiring, she recalled that during the journey, an unrecognised man occupying the lower berth of the same coach was coughing persistently. The man looked weak and was coughing without putting a mask. She stayed with this unrecognised man for 16 h during travel, while an epidemic of COVID-19 was spreading in many provinces of China. On 24 January, she arrived at Wenzhou and then travelled further to village A by an earliest available bus.', 'cite_spans': [], 'section': 'Index case finding ::: Results', 'ref_spans': []}, {'text': 'On 31 January 2020, at around 14:00 h, the county hospital reported a suspected patient, the index case, with COVID-19 to the local Center for Disease Control and Prevention and collected throat swabs that were sent to local CDC for SARS-CoV-2 nucleic acid testing, i.e. polymerase chain reaction (PCR). On the next day, at 18:50 h, the throat swabs were found to be positive.', 'cite_spans': [], 'section': 'Index case finding ::: Results', 'ref_spans': []}, {'text': 'As the patient had high fever, and her symptoms did not improve, on 3 February, at around 13:00 h, she was referred to the First Affiliated Hospital of Wenzhou Medical University for further treatment.', 'cite_spans': [], 'section': 'Index case finding ::: Results', 'ref_spans': []}, {'text': 'After the index case was reported, an epidemiological investigation involving the index case, her family members and their contacts was carried out.', 'cite_spans': [], 'section': 'Outbreak detection and investigation ::: Results', 'ref_spans': []}, {'text': 'After going home to village A on 24 January and before the onset of symptoms of COVID-19, the index patient lived with her family. The family consisted of 14 members, including her husband, parents, elder sister, sister-in-law, brother-in-law, aunt, uncle, younger brother, son, niece and two nephews, which is the same size and structure as most families in the area. Further, she denied having gone outside, and was living and dining with family members. During Spring Festival or Lunar New Year, they had a joyous gathering in village A.', 'cite_spans': [], 'section': 'Outbreak detection and investigation ::: Results', 'ref_spans': []}, {'text': 'Search for source of infection, screening of the contacts, and medical observation of the contacts were carried out by medical workers, in order to find as many patients as possible. The throat swabs of contacts were collected and PCR for SARS-CoV-2 nucleic acid was carried out, for all family members, by skilled laboratory workers in the local CDC. All cases were laboratory confirmed, following the case definition by National Health Commission of China. Thirteen family members, 15 close contacts and 10 bus contacts (six contacts in Wenzhou and four in village A) of the index case were traced and found. The onset of symptoms in contacts was one after the other. The time series of a family outbreak of COVID-19 is shown in Figure 1. The investigation confirmed one asymptomatic and seven symptomatic cases with COVID-19 and all of them were family members of the index case. The relations of cases with COVID-19 in a family outbreak are shown in Figure 2. The date of onset and clinical presentation of cases with COVID-19, and the relation of family members with the index case is given in Table 1.\\n\\n\\n', 'cite_spans': [], 'section': 'Outbreak detection and investigation ::: Results', 'ref_spans': [{'start': 731, 'end': 739, 'mention': 'Figure 1', 'ref_id': 'FIGREF0'}, {'start': 954, 'end': 962, 'mention': 'Figure 2', 'ref_id': 'FIGREF1'}, {'start': 1099, 'end': 1106, 'mention': 'Table 1', 'ref_id': 'TABREF0'}]}, {'text': 'However, after 14 days of medical observation, fever and the other related symptoms of COVID-19 were not observed in 15 close contact and 10 bus contacts isolated in individual rooms.', 'cite_spans': [], 'section': 'Outbreak detection and investigation ::: Results', 'ref_spans': []}, {'text': \"\\nAll the patients were admitted in the Department of Infectious Disease of the county hospital for treatment and isolation. The index case and other eight cases were reported to local CDC and public health authority via the web, immediately after being diagnosed as a case of COVID-19.All the close contacts of nine cases were isolated for 14 days, and observed for fever and respiratory symptoms by the healthcare workers in a designated hotel. These workers were instructed to report to a doctor immediately, as soon as the symptoms of discomfort occurred, and take the throat swabs for PCR for SARS-CoV-2 nucleic acid.A collaborative inspection system was initiated to screen the exposed travellers by train from city X on 22–24 January and by buses from Wenzhou and village A on 24 January. All the exposed travellers (six and four exposed individuals in the same bus in Wenzhou and village A, respectively; and no close contacts were found amongst traced individuals in train) were advised self-isolation and they stayed at home for 14 days.Thorough screening and tracing activity resulted in finding 185 contacts of the index cases, in village A. Out of these 185 contacts, 58 and 127 contacts were isolated in a shift quarantine camps and at home, respectively. On 4 February, 1230 individuals living in 366 houses located in the residential district of the index case were subjected to medical observation at home. These contacts were followed up for the entire period of 14 days. Despite active follow-up, no secondary transmission event were detected.The agriculture trade market neighbouring the patient's house was closed, so as to decrease the flow and the contact of people.Individuals coming in contact with COVID-19 cases and their close contacts were asked to frequently wash the hands, open the windows so as to increase the ventilation and put on masks.The terminal disinfection of the focus, the building, and sites of living and activities of the patients were carried out with chlorine-containing disinfectants.\", 'cite_spans': [], 'section': 'Outbreak control measures ::: Results', 'ref_spans': []}, {'text': 'The ongoing COVID-19 outbreak has rapidly evolved and spread globally [15]. Epidemiological studies suggest that SARS-CoV-2 has an intrinsic capacity to cause epidemic spread [16]. SARS-CoV-2 appears to be highly transmissible from human-to-human resulting in a wide spectrum of clinical manifestations in patients with COVID-19 [15]. However, similar to our investigation, increasing evidence suggests that mild clinical symptoms could be more frequent in cases of COVID-19. In our investigation, amongst nine cases, one had severe symptoms, seven had mild symptoms and one was asymptomatic.', 'cite_spans': [{'start': 71, 'end': 73, 'mention': '15', 'ref_id': 'BIBREF6'}, {'start': 176, 'end': 178, 'mention': '16', 'ref_id': 'BIBREF7'}, {'start': 330, 'end': 332, 'mention': '15', 'ref_id': 'BIBREF6'}], 'section': 'Discussion', 'ref_spans': []}, {'text': 'Although the Huanan market in Wuhan was highly relevant to the emergence and spread of COVID-19, and the SARS-CoV-2 has been detected in environmental samples obtained from the market, the origin of this virus has been determined conclusively [17]. Currently, there is no confirmed animal reservoir of the virus, but the available evidence strongly supports that the SARS-CoV-2 was derived from bats [7, 18]. However, it is not yet clear whether SARS-CoV-2 infected humans via direct transmission from a bat or through an intermediate host [17]. It is believed that clarifying the source of the virus will help determine zoonotic transmission patterns. Notably, cases have occurred in patients who had not travelled to Wuhan, thereby, indicative of human-to-human transmission [19]. Moreover, various investigations have resulted in a positive evidence supporting human-to-human transmission of SARS-CoV-2.', 'cite_spans': [{'start': 244, 'end': 246, 'mention': '17', 'ref_id': 'BIBREF8'}, {'start': 401, 'end': 402, 'mention': '7', 'ref_id': 'BIBREF20'}, {'start': 404, 'end': 406, 'mention': '18', 'ref_id': 'BIBREF9'}, {'start': 541, 'end': 543, 'mention': '17', 'ref_id': 'BIBREF8'}, {'start': 778, 'end': 780, 'mention': '19', 'ref_id': 'BIBREF10'}], 'section': 'Discussion', 'ref_spans': []}, {'text': 'Similar human-to-human transmission was observed in our investigation. The index patient had the history of contact with the suspected patient and the obvious clinical symptoms of COVID-19. Moreover, the test results of SARS-CoV-2 nucleic acid of throat swabs were positive. Therefore, the index case was diagnosed as COVID-19, following the case definition by National Health Commission of China [14].', 'cite_spans': [{'start': 398, 'end': 400, 'mention': '14', 'ref_id': 'BIBREF5'}], 'section': 'Discussion', 'ref_spans': []}, {'text': 'The investigation revealed that the index case had 14 family members. The index case did not contact other people with similar symptoms before the onset of symptoms, except the suspected patient occupying the lower berth of the same coach of the train. The suspected patient had continuous cough and they travelled together for 16 h. In addition, transmission of COVID-19 occurs easily in closed coaches. Thus, it is recognised that the source of infection for the index case is this suspected patient.', 'cite_spans': [], 'section': 'Discussion', 'ref_spans': []}, {'text': 'After arriving at village A, before the onset of symptoms (i.e. from 24–29 January), the index case had been dwelling with her family members. They had meals and some activities together, in a four-floor building with six bedrooms. All the family members, including the index case had not gone outside, before the onset of symptoms in the index case. During this period, they did not have any visitors. Our investigation revealed that they did not come in contact with people from Wuhan or individuals with similar symptoms. However, all seven symptomatic and one asymptomatic cases had developed a contact with the index case. Thus, we believed that the source of infection for these eight cases was the index case. Moreover, this direct human-to-human transmission of COVID-19 amongst the family members was due to close contact with the index case and they developed the symptoms one after another. Fifteen close contacts except the family members did not develop COVID-19. Thus, over-crowding as a risk factor can also play a significant role in the transmission of COVID-19. The high infectivity of COVID-19 was discovered from the investigation of this family outbreak leading to development of symptoms in eight patients and one infected asymptomatic individual.', 'cite_spans': [], 'section': 'Discussion', 'ref_spans': []}, {'text': 'The range of incubation period is required for the purpose of epidemiological case definitions, and is essential to determine the appropriate duration of quarantine [20]. Moreover, the knowledge of the incubation period helps in assessing the effectiveness of entry screening and contact tracing programme. The distribution of incubation period is also used in estimating the size of the epidemic [21, 22] and the transmission potential [23]. During the early outbreak phase, the mean incubation period, in Wuhan, was 5.2 days (95% confidence interval (CI), 4.1–7.0), with the 95th percentile of the distribution at 12.5 days [13]. In our investigation, however, the median incubation period of COVID-19 was 7.5 days, with the longest and shortest incubation periods being 21 and 5 days, respectively. The patient with the longest incubation period was the 18-year-old son of the index case. On or after 24 January, he developed a contact with the index case, who was asymptomatic before 29 January. After 21 days, he presented with fever (38.2 °C), cough, myalgia and arthralgia. Chest radiography on CT revealed an infectious focus involving the upper lobe of the left lung. The throat swabs of the patient were collected and examined, and were found to be positive for SARS-CoV-2 nucleic acid. Moreover, he had been isolated when the index case was diagnosed with COVID-19. It is important to note that such a long incubation period of COVID-19 is very rare in literature.', 'cite_spans': [{'start': 166, 'end': 168, 'mention': '20', 'ref_id': 'BIBREF12'}, {'start': 398, 'end': 400, 'mention': '21', 'ref_id': 'BIBREF13'}, {'start': 402, 'end': 404, 'mention': '22', 'ref_id': 'BIBREF14'}, {'start': 438, 'end': 440, 'mention': '23', 'ref_id': 'BIBREF15'}, {'start': 627, 'end': 629, 'mention': '13', 'ref_id': 'BIBREF4'}], 'section': 'Discussion', 'ref_spans': []}, {'text': 'Our investigation indicates that the cases are highly infectious even during the incubation period. Moreover, it has been demonstrated that the familial outbreak of COVID-19 has a potential for human-to-human transmission, even during the incubation period. This is supported by our investigation, where the index case during the incubation period transmitted COVID-19 to one asymptomatic and seven symptomatic cases. Thus, it is prudent to consider this fact while carrying out an epidemiological investigation and taking control measures against COVID-19, in future, including a quarantine and observation period for the suspected cases. Moreover, it is judicious to consider an incubation period, as long as 21 days, as highlighted by our investigation.', 'cite_spans': [], 'section': 'Discussion', 'ref_spans': []}], 'paragraphs': [{'text': 'The range of incubation period is required for the purpose of epidemiological case definitions, and is essential to determine the appropriate duration of quarantine [20]. Moreover, the knowledge of the incubation period helps in assessing the effectiveness of entry screening and contact tracing programme. The distribution of incubation period is also used in estimating the size of the epidemic [21, 22] and the transmission potential [23]. During the early outbreak phase, the mean incubation period, in Wuhan, was 5.2 days (95% confidence interval (CI), 4.1–7.0), with the 95th percentile of the distribution at 12.5 days [13]. In our investigation, however, the median incubation period of COVID-19 was 7.5 days, with the longest and shortest incubation periods being 21 and 5 days, respectively. The patient with the longest incubation period was the 18-year-old son of the index case. On or after 24 January, he developed a contact with the index case, who was asymptomatic before 29 January. After 21 days, he presented with fever (38.2 °C), cough, myalgia and arthralgia. Chest radiography on CT revealed an infectious focus involving the upper lobe of the left lung. The throat swabs of the patient were collected and examined, and were found to be positive for SARS-CoV-2 nucleic acid. Moreover, he had been isolated when the index case was diagnosed with COVID-19. It is important to note that such a long incubation period of COVID-19 is very rare in literature.', 'id': '00021'}], 'title': 'Outbreak of COVID-19 in a family, Wenzhou, China', 'sha': '4fabdcf18948e7fd575dee30654f1bb01ac0b6ee', 'doi': '10.1017/s0950268820001089', 'score': 9.13700008392334}, {'rank': 199, 'abstract': 'The COVID-19 pandemic has become a crucial public health issue in many countries including the United States. In the absence of the right vaccine strain and sufficient antiviral stockpiles on hand, nonpharmaceutical interventions have become valuable public health tools at the early stages of the pandemic and they are employed by many countries across the globe. These interventions are designed to increase social distancing between individuals to reduce the transmission of the virus and eventually dampen the burden on the healthcare system. The virus transmissibility is a function of the average number of contacts individuals have in their communities and it is highly dependent on population density and daily mobility patterns, along with other social factors. These show significant variation across the United States. In this article, we study the effectiveness of social distancing measures in communities with different population density. Specifically, first we show how the empirical estimation of reproduction number differs for two completely different states, thus the experience of the COVID-19 outbreak is drastically different, suggesting different outbreak growth rates in practice. Second, we develop an age-structured compartmental model for simulating the disease spread to demonstrate the variation in the observed outbreak characteristics. We find that early trigger and late trigger options present a trade-off between the peak magnitude and the overall death toll of the outbreak which may also vary across different populations.', 'body_text': [{'text': 'distancing measures depends on the population density and underlying mobility patterns of a region.', 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': 'In the United States (US), the states show significant variation in population density and demographic configurations. For example, when Nebraska is compared to New York the population density and the mobility patterns are drastically different. Nebraska has low population density and mobility patterns with low-use of public transportation. New York, on the other had, especially New York City exhibit high population density and high-use of public transportation.', 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': 'In this article, we study the practical effectiveness of both social distancing measures for outbreak in regions with different population density and mobility patterns as well as reopening strategies. Specifically, first we show that the evolution of the empirical fit for reproduction number (i.e. a measure for transmissibility) for two different states in the US, i.e., Nebraska and New York, for the COVID-19 outbreak are drastically different, suggesting different outbreak transmission rates in practice, emulating different states. Second, we develop an age-structured compartmental model to simulate the disease spread for different growth rates and show that the observed outbreak characteristics are different for these states.', 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': 'Third, employing computational experiments with the simulation model we show that social distancing measures with different decision parameters and paths (e.g., various triggers to start and different lengths of social distancing) can result in effective management of the outbreak in dissimilar states. We find that early trigger and late trigger options present a trade-off between the peak magnitude and the overall death toll of the outbreak. Finally, we analyze reopening strategies with a two-phased reopening scenario. We show that for outbreaks with small transmission rate a short length of social distancing and immediate reopening scenario may be feasible. Whereas, for outbreaks with medium or high transmission rate the surveillance data and findings that provided details about case fatality rates and transmissibility. An early study on the transmission dynamics of COVID-19 analyzed data of the first 425 confirmed cases in Wuhan, China and found that the mean incubation period was 5.2 days. However, the 95 th percentile of the incubation period was 12.5 days in this sample (Li et al. 2020a ). In addition, this study estimated the basic reproductive number (R 0 ), which is the average number of secondary cases generated from a single infectious case in a completely susceptible population, as anywhere between 1.5 and 3.5 (Li et al. 2020a ). These epidemiological estimates show that the virus strain is highly transmissible and can infect mass populations across the globe in a very short amount of time. Another study on news reports and press releases about COVID-19 outside Wuhan estimated the mean incubation period and 97.5 th percentile as 5.1 days and 11.5 days, respectively (Lauer et al. 2020) . Linton et al. (2020) also estimated the expected incubation time around 5 days with relatively higher range of 2 to 14 days. Given relatively high estimates of the basic reproductive number R 0 , and relatively long incubation period, there is an exponential growth of confirmed COVID-19 cases in several countries and developing control measure for the epidemic is quite challenging. More importantly, a significant proportion of cases are reported to be asymptomatic but infectious cases (Li et al. 2020a) . This characteristic of the virus is found one of the main drivers of the spread of COVID-19 across populations. Therefore, social distancing measures are crucial for gaining time for healthcare systems to meet the demand for care and ultimately mitigating the impact of the pandemic.', 'cite_spans': [{'start': 1093, 'end': 1109, 'text': '(Li et al. 2020a', 'ref_id': None}, {'start': 1344, 'end': 1360, 'text': '(Li et al. 2020a', 'ref_id': None}, {'start': 1706, 'end': 1725, 'text': '(Lauer et al. 2020)', 'ref_id': None}, {'start': 1728, 'end': 1748, 'text': 'Linton et al. (2020)', 'ref_id': None}, {'start': 2218, 'end': 2235, 'text': '(Li et al. 2020a)', 'ref_id': None}], 'ref_spans': [], 'section': ''}, {'text': 'By mid-March, there has been a significant impact on the economy of several countries as more restrictions to mitigate the risk are being implemented, such as travel bans, cancellation of social events (concerts, sports, etc.) , closure of non-essential businesses, \"stay at home\" orders. While social distancing interventions have been implemented in various states of the US and also in other countries worldwide they have been very controversial policy since they can have significant impact on the economy. Because it is usually hard to accurately estimate the transmissibility and severity of infections caused by a newly emerged virus at the early stages of the epidemic, there will still be many uncertainties about disease progression dynamics in the communities. Therefore, the public health decision makers will have to make important decisions on using school closures and other social distancing measures as community mitiga-4 tion strategies until the right strain of a vaccine is developed and distributed. However, these strategies may vary based on the social and demographic characteristics of different communities. In this article, we present the timely evolution of the estimates of the reproductive number in two different states, i.e., New York where we observe high transmission rates; and Nebraska where we observe relatively lower community transmission rates, along with a baseline scenario based on more global estimates. We evaluate social distancing measures to analyze whether one unified policy would be effective in completely different states in the United States.', 'cite_spans': [{'start': 202, 'end': 226, 'text': '(concerts, sports, etc.)', 'ref_id': None}], 'ref_spans': [], 'section': ''}, {'text': 'Social distancing measures have significant impact on the spread of infectious diseases in populations, this is also observed during the COVID-19 pandemic (Courtemanche et al. 2020) While a pandemic possibility continuously posed global risks to public health systems and business continuity (Araz et al. 2020) , federal and state health departments developed public health policies for mitigation and response (Ramirez-Nafarrate et al. 2019) . While modeling and simulation studies are used for optimal pharmaceutical intervention design, which include vaccination policies (Duijzer et al. 2018b,a) , non-pharmaceutical interventions are also modeled with disease progression dynamics (Griffiths et al. 2000 , Teytelman & Larson 2012 . In developing social distancing policies, researchers and public health officials have used models and quantitative analyses to evaluate their effectiveness and costs under possible pandemic scenarios. Some studies demonstrate school closures can have a significant impact on 5 the effective reproduction number and on the overall spread of disease (Lee et al. 2010) . One advocates, for example, 26 weeks of school closure in conjunction with other policies (Sander et al. 2010) . Since the H1N1 influenza pandemic in 2009 many studies showed the cost effectiveness and epidemiological impact of a large set of school closure policies. Different than the H1N1 pandemic in 2009, COVID-19 pandemic has raised more political, social and economical challenges as it turned to be a more severe pandemic. Therefore, in this article we evaluate several more comprehensive social distancing policies coupled with different strategies to reopen. The outcomes measures considered for the policies are health care utilization and fatalities via a model that is calibrated based on state specific transmission data and socio-demographic decomposition. This study contributes to the literature by presenting a data driven epidemiological modeling with decision analytics framework to inform social distancing policies for different states. In addition the analysis show that multiple waves of infections should be expected based on the social distancing policies, which would vary in peak magnitude and timing depending on the policies.', 'cite_spans': [{'start': 155, 'end': 181, 'text': '(Courtemanche et al. 2020)', 'ref_id': 'BIBREF7'}, {'start': 292, 'end': 310, 'text': '(Araz et al. 2020)', 'ref_id': 'BIBREF2'}, {'start': 411, 'end': 442, 'text': '(Ramirez-Nafarrate et al. 2019)', 'ref_id': 'BIBREF12'}, {'start': 575, 'end': 599, 'text': '(Duijzer et al. 2018b,a)', 'ref_id': None}, {'start': 686, 'end': 708, 'text': '(Griffiths et al. 2000', 'ref_id': None}, {'start': 709, 'end': 734, 'text': ', Teytelman & Larson 2012', 'ref_id': 'BIBREF14'}, {'start': 1086, 'end': 1103, 'text': '(Lee et al. 2010)', 'ref_id': None}, {'start': 1196, 'end': 1216, 'text': '(Sander et al. 2010)', 'ref_id': 'BIBREF13'}], 'ref_spans': [], 'section': ''}, {'text': 'Finally, the study highlights how time gained by using social distancing interventions can vary across states with different population configurations until mass vaccination or antiviral dispensing can become available.', 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': 'In this paper, we present a decision analytic approach for social distancing during a pandemic. We develop and simulate an age structured mass action model integrated with a decision analytical approach to evaluate the impact of social distancing measures and reopening strategies. The impact of these strategies are evaluated based on the computational results obtained for cumulative attack rate, projected peak value of infections, i.e., peak prevalence (%), timing of the peak, peak hospitalizations, and cumulative deaths.', 'cite_spans': [], 'ref_spans': [], 'section': 'Model'}, {'text': 'In our decision analytic approach, the social distancing measures are triggered with the prevalence in the community similar to Araz et al. (2012) , as most of the social distancing policies are implemented by monitoring the % of cases as triggers. All the considered reopening strategies that are evaluated are modeled after a fixed duration of social distancing policies and with a gradual removal strategy implemented based on multiple phases. We assume the social distancing policies are implemented based on 6 All rights reserved. No reuse allowed without permission.', 'cite_spans': [{'start': 128, 'end': 146, 'text': 'Araz et al. (2012)', 'ref_id': 'BIBREF1'}], 'ref_spans': [], 'section': 'Model'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Model'}, {'text': 'The copyright holder for this preprint this version posted June 26, 2020. . the prevalence of COVID-19 infections in the community and a series of reopening decisions are considered after a fix duration of social distancing. The considered closure durations are 1, 2, 3, 4, 8, 16 and 24 weeks which are coupled with the prevalence based triggers, i.e., 0.001%, 0.05%, 0.1%, 0.2%, 0.3%, 0.4%, 0.5% and 1%. See Figure 2 for all the scenarios considered in our analyses. The removal of the policies are modeled in phases, following the guidelines released from CDC (2020b). Next, we first present the differential equation based disease spread model constructed to simulate disease progression in the communities, then we explain the triggering strategies used for social distancing policies and phase-based reopening.', 'cite_spans': [{'start': 262, 'end': 264, 'text': '1,', 'ref_id': None}, {'start': 265, 'end': 267, 'text': '2,', 'ref_id': None}, {'start': 268, 'end': 270, 'text': '3,', 'ref_id': None}, {'start': 271, 'end': 273, 'text': '4,', 'ref_id': None}, {'start': 274, 'end': 276, 'text': '8,', 'ref_id': None}, {'start': 277, 'end': 279, 'text': '16', 'ref_id': None}], 'ref_spans': [{'start': 409, 'end': 417, 'text': 'Figure 2', 'ref_id': 'FIGREF2'}], 'section': 'Model'}, {'text': 'Compartmental disease spread models are widely utilized in computational and mathematical epidemiology (Anderson & May 1991) . In these models all individuals act similarly but separately from each other in a homogeneously mixed population (Dimitrov et al. 2009 ). Here, in this study we use an age-structured, continuous time compartmental model using a population specific data and considering uncertainty on several input parameters. The equations (1)-(7) model the transmission dynamics of COVID-19 disease in a given population and flow of individuals moving from one disease state to another as given in Figure 1. They represent the disease progression for individuals who are first susceptible then exposed to the disease, and then become asymptomatic or symptomatically infectious. Asymptomatic infectious individuals either develop symptoms and become symptomatically infectious or recover from the disease without any symptoms. The symptomatic infectious individuals can either recover themselves or be hospitalized. The hospitalized individuals can either recover or die from the disease. S i (t) represents the number of susceptible individuals in the community at time t and for age group i, E i (t) represents the exposed individuals, while I A i (t) is used for asymptomatic infectious individual and I S i (t) is used for the symptomatic infectious cases. R i (t) is used for the recovered cases in age group i and H i (t) is the hospitalized cases at time t for the age group i. Finally D i (t) represents the number of deaths in each age group at time t. The force of infection for each age group is represented with λ i , and it is computed as', 'cite_spans': [{'start': 103, 'end': 124, 'text': '(Anderson & May 1991)', 'ref_id': 'BIBREF0'}, {'start': 240, 'end': 261, 'text': '(Dimitrov et al. 2009', 'ref_id': 'BIBREF8'}], 'ref_spans': [], 'section': 'Disease Spread Model'}, {'text': '. 7 All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Disease Spread Model'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Disease Spread Model'}, {'text': 'The copyright holder for this preprint this version posted June 26, 2020. .', 'cite_spans': [], 'ref_spans': [], 'section': 'Disease Spread Model'}, {'text': 'Figure 1: Natural progression of COVID-19 with hospitalization', 'cite_spans': [], 'ref_spans': [], 'section': 'Disease Spread Model'}, {'text': 'Using the model presented above we have derived the basic reproductive number R 0 of the system, which is age specific, since the model is an age structured one. Using the next generation operator the theoretical expression for R 0 is derived as given below. See the Appendix B for the details of the derivation process.', 'cite_spans': [], 'ref_spans': [], 'section': 'Disease Spread Model'}, {'text': '. Now, the time dependent reproductive number i.e., R i (t) for age group i can be stated as following:', 'cite_spans': [], 'ref_spans': [], 'section': 'Disease Spread Model'}, {'text': '.', 'cite_spans': [], 'ref_spans': [], 'section': 'Disease Spread Model'}, {'text': 'The epidemic control can be achieved if R i (t) ≤ 1 can be satisfied with social distancing policies for all age groups.', 'cite_spans': [], 'ref_spans': [], 'section': 'Proposition 1'}, {'text': 'We calibrate the model based on the daily cases reported for considered communities to achieve the observed disease propagation and using the result stated in proposition 1 we derive community specific analysis for evaluating the social distancing policies.', 'cite_spans': [], 'ref_spans': [], 'section': 'Proposition 1'}, {'text': 'We evaluate a range of social distancing and reopening strategies for different scenarios for the transmissibility and estimated ranges of severity of the pandemic. Each social distancing policy alternative consists of a prevalence-based trigger and a fixed duration until a reopening decision is made as shown in Figure 2 .', 'cite_spans': [], 'ref_spans': [{'start': 314, 'end': 322, 'text': 'Figure 2', 'ref_id': 'FIGREF2'}], 'section': 'Modeling Closure and Reopening Decisions'}, {'text': 'Now we explain the modeling process of closure and reopening decisions. As defined earlier I A i (t) represents the asymptomatic infections for age group i at time t and I S i (t) represents the symptomatic infections for age group i at time t. In our model, age groups i ∈ {k, a, e} are defined as k for \"kids\", a for \"adults\" and e for \"elderly\". Let the function f (t) be the cumulative number of individuals infected at time t which is computed as following:', 'cite_spans': [], 'ref_spans': [], 'section': 'Modeling Closure and Reopening Decisions'}, {'text': 'We can mathematically define trigger time for social distancing policy to take place as follows:', 'cite_spans': [], 'ref_spans': [], 'section': 'Modeling Closure and Reopening Decisions'}, {'text': '9 All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Modeling Closure and Reopening Decisions'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Modeling Closure and Reopening Decisions'}, {'text': 'The copyright holder for this preprint this version posted June 26, 2020. . (which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Modeling Closure and Reopening Decisions'}, {'text': 'The copyright holder for this preprint this version posted June 26, 2020. . https://doi.org/10. 1101 Given the policy trigger time, t * , below we show how contact rates c ij (t) for each age group changes at time t as the closure and reopening decisions are implemented. As the effects of policies would take some time to be observed, let δ be the implementation time after the trigger is hit.', 'cite_spans': [{'start': 96, 'end': 100, 'text': '1101', 'ref_id': None}], 'ref_spans': [], 'section': 'SOCIAL'}, {'text': 'The early published estimates of the basic reproduction number (R 0 ) of the COVID-19 were slightly higher than 2 (Li et al. 2020a). However, depending on the source of the data used in these estimation studies, there were some variation in these estimates. Beside the basic reproductive number, other estimates of the key parameters used in the model are presented in Table 1 which are compiled from the literature and include the latency period, proportion of the asymptotically infected individuals, age dependent hospitalization and mortality rates. Using the cumulative cases data from each state we considered in this study, i.e., New York and Nebraska we have estimated respective reproduction numbers. Then, using the theoretical formula for R(t) we have calibrated contact rates to achieve the observed reproduction of the cases estimated for each state. See Appendix Table 8 for the values used for the model calibration.', 'cite_spans': [], 'ref_spans': [{'start': 369, 'end': 376, 'text': 'Table 1', 'ref_id': 'TABREF0'}, {'start': 877, 'end': 884, 'text': 'Table 8', 'ref_id': 'TABREF12'}], 'section': 'Data'}, {'text': 'In this article, we use daily number of cases data published by the CDC from the January 3rd, 2020 to April 23rd, 2020 for the states of New York and Nebraska and calibrated the contact rate parameter value to achieve the reproductive number estimated for each state. The cumulative number of case over time for each state are presented in Figure 3 , with respective time dependent reproduction number estimates.', 'cite_spans': [], 'ref_spans': [{'start': 340, 'end': 348, 'text': 'Figure 3', 'ref_id': 'FIGREF3'}], 'section': 'Data'}, {'text': '11 All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Data'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Data'}, {'text': 'The copyright holder for this preprint this version posted June 26, 2020. . 12 All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Data'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Data'}, {'text': 'The copyright holder for this preprint this version posted June 26, 2020. . Table 9 CDC, 2020 Age specific recovery rate ϕ 1 A. Table 9 − CDC, 2020 Age specific mortality rate ϕ 2 =1-ϕ 1 A. Table 9 − CDC, 2020', 'cite_spans': [], 'ref_spans': [{'start': 76, 'end': 83, 'text': 'Table 9', 'ref_id': None}, {'start': 128, 'end': 135, 'text': 'Table 9', 'ref_id': None}, {'start': 190, 'end': 197, 'text': 'Table 9', 'ref_id': None}], 'section': 'Data'}, {'text': 'Using our compartmental model, we evaluate different social distancing strategies by varying two key parameters, namely the threshold prevalence to start the social distancing policies and the length of social distancing. We also evaluate these strategies for different R 0 values (i.e., epidemic growth rate), as empirically we find different growth rate values for different states, in addition to a base case of globally estimated R 0 value of 2.1.', 'cite_spans': [], 'ref_spans': [], 'section': 'Results'}, {'text': 'For example, the case reproduction dynamics in New York is different than the one in Nebraska.', 'cite_spans': [], 'ref_spans': [], 'section': 'Results'}, {'text': 'New York has higher transmission rate and Nebraska has lower transmission rate than the base scenario Figure 4a shows the temporal dynamics of outbreak for the three different growth rates with R 0 values of 1.57, 2.1 and 3.15, when no social distancing policy is executed. i.e., labeled as \"no intervention\".', 'cite_spans': [], 'ref_spans': [{'start': 102, 'end': 111, 'text': 'Figure 4a', 'ref_id': 'FIGREF4'}], 'section': 'Results'}, {'text': 'We find that practical growth rate of the outbreak in a region directly affects the dynamics of the outbreak. Regions with high transmission rate should expect to experience their peak to be observed much earlier than regions with lower transmission rate. Similarly, the magnitude (i.e. peak height) of the outbreak is significantly higher for those regions with high growth rate than those with small growth rate, 13 All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Results'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Results'}, {'text': 'The copyright holder for this preprint this version posted June 26, 2020. . Furthermore, when same social distancing policies are employed for these different states, we observed different effects of the interventions. For example, Figure 4b compares one specific social distancing policy, i.e., triggering when the cumulative outbreak reaches 0.5% infections under different basic reproductive number scenarios. For states with higher growth rate, the outbreak results in higher and earlier first peak, and an earlier second peak. For regions with lower reproduction number, the social distancing trigger threshold is reached later and the observed peak would be smaller. These simulations suggest that the same social distancing policy can have different effects on the evolution of the epidemic in different populations. Thus, we evaluate each state separately. In the following sections, we evaluate base scenario with (R 0 = 2.1), high transmission rate scenario with (R 0 = 3.15) and low transmission scenario with (R 0 = 1.57) in order to reflect the time dependent reproduction number estimates using COVID-19 cumulative cases data from each state.', 'cite_spans': [], 'ref_spans': [{'start': 232, 'end': 241, 'text': 'Figure 4b', 'ref_id': 'FIGREF4'}], 'section': 'Results'}, {'text': 'Early estimates for the characteristics of the COVID-19 virus suggest that basic reproductive number would be greater than 2, however wide uncertainty rages are also reported (Li et al. 2020b). Therefore we use a base case scenario for the transmission dynamics with an R 0 value of 2.1. One critical question 14 All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Base Scenario (R 0 = 2.1)'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Base Scenario (R 0 = 2.1)'}, {'text': 'The copyright holder for this preprint this version posted June 26, 2020. . https://doi.org/10. 1101 in mitigation efforts is to decide when to start the social distancing. Figure 5 shows how applying two distinct trigger thresholds, i.e., 0.5% and 1%, can change the dynamics of the outbreak with a fixed duration of social distancing under the base case scenario with 8 weeks of social distancing. Specifically, the graph compares early trigger (0.5%) and late trigger (1%) options with no intervention. When the social distancing policy is triggered early (0.5%), the outbreak has a smaller first peak in number of cases, and a much higher second peak. Conversely, when the social distancing policy is triggered late (1%), the outbreak has a higher first peak, and a smaller second peak. The early trigger results in an earlier second peak than when a late trigger is used. In both cases, the peaks are considerably dampened compared to the peak without any social distancing implemented. For an outbreak with baseline transmission rate our systematic and extensive simulations suggest the main benefit of social distancing is to buy time by delaying the peak as shown in Table 2 . The length of social distancing phase has minimal to no effect on the magnitude of the second wave. However, it directly changes the timing of the second wave. This time until the second wave could help public health officials to prepare for a second wave. The threshold used to trigger the social distancing policy has significant implications on the epidemic dynamics. Specifically, early triggers result in a small first wave 15 All rights reserved. No reuse allowed without permission.', 'cite_spans': [{'start': 96, 'end': 100, 'text': '1101', 'ref_id': None}], 'ref_spans': [{'start': 173, 'end': 181, 'text': 'Figure 5', 'ref_id': 'FIGREF5'}, {'start': 1175, 'end': 1182, 'text': 'Table 2', 'ref_id': 'TABREF2'}], 'section': 'Base Scenario (R 0 = 2.1)'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Base Scenario (R 0 = 2.1)'}, {'text': 'The copyright holder for this preprint this version posted June 26, 2020. . https://doi.org/10.1101/2020.06.24.20139329 doi: medRxiv preprint and a larger second wave. Conversely, later triggers result in a relatively bigger first wave, and smaller second wave when compared to the second waves of other policy triggering options. This result suggests that depending on the hospital bed and ICU capacity, early triggers or late triggers may be used to balance the magnitudes of the waves for mitigation and to manage the health care system capacity. The number of hospitalizations also follow a similar trend as preented in Figure 6 for social distancing. We compare hospitalization volumes for early and late trigger with 8 weeks and 16 weeks of social distancing. Using an earlier trigger results in a smaller initial peak in hospitalizations than triggering that later. Conversely, the second peak is observed higher when an early trigger is used than the later trigger.', 'cite_spans': [], 'ref_spans': [{'start': 624, 'end': 632, 'text': 'Figure 6', 'ref_id': None}], 'section': 'Base Scenario (R 0 = 2.1)'}, {'text': 'The length of the social distancing policy merely delays the second peak, both for early and late triggers, with no material impact to the size of the second peak. Using a smaller prevalence value for triggering the policy results in a smaller first peak than the later trigger, however, the second peak for early trigger is larger than that of late trigger. The length of the social distancing again affects the timing of the second peak.', 'cite_spans': [], 'ref_spans': [], 'section': 'Base Scenario (R 0 = 2.1)'}, {'text': '16 All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Base Scenario (R 0 = 2.1)'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Base Scenario (R 0 = 2.1)'}, {'text': 'The copyright holder for this preprint this version posted June 26, 2020. . https://doi.org/10.1101/2020.06.24.20139329 doi: medRxiv preprint Figure 6 : Comparison of hospitalization volumes for early and late trigger (i.e., columns) for 8 weeks and 16 weeks closures (i.e., rows). While some highly dense US states observed faster growth in the number of cases, e.g., New York and California, some states with lower population density observed steep and later increase in the cases. Therefore, we simulate a scenario with higher transmission rate which is achieved with R 0 = 3.15. Similar to base case scenario, simulations suggest the main benefit of social distancing is to buy time by delaying the peak as shown in Table 3 . The length of social distancing phase has minimal to no effect on the magnitude, i.e., height of the peak of the second wave. However, it directly changes the timing of the second wave. The magnitudes of the second peaks for high transmission scenario, for all trigger options, is more than 2 times larger than those of for the baseline transmission scenario.', 'cite_spans': [], 'ref_spans': [{'start': 142, 'end': 150, 'text': 'Figure 6', 'ref_id': None}, {'start': 720, 'end': 727, 'text': 'Table 3', 'ref_id': 'TABREF4'}], 'section': 'Base Scenario (R 0 = 2.1)'}, {'text': 'Depending on the length of the social distancing implemented, public health officials gain time until the second wave ranging from one month, when the least aggressive scenario is employed, to 9 months, when the most aggressive scenario is employed. Such delay in the second peak could enable increasing hospital capacity, explorations for vaccinations and improved public health awareness.', 'cite_spans': [], 'ref_spans': [], 'section': 'Base Scenario (R 0 = 2.1)'}, {'text': '17 All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Base Scenario (R 0 = 2.1)'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. ', 'cite_spans': [], 'ref_spans': [], 'section': 'Base Scenario (R 0 = 2.1)'}, {'text': 'For an outbreak with low transmission rate our simulations suggest the main benefit of social distancing, similar to baseline scenario and high transmission scenario, is to buy time by delaying the peak as shown in Table 4 . However, for the low transmission scenario the magnitude of the peak, even with no intervention, is small compared to the high and baseline transmission rates. Furthermore, the timing of the peak for low transmission is much later than that of baseline transmission (i.e., 432nd day versus 226th day). In fact, the hospital capacity might be already enough or could be increased until the peak time to cover the 0.39% peak magnitude. Overall, the closure length should depend on the extra capacity needed to handle the patients during the peak.', 'cite_spans': [], 'ref_spans': [{'start': 215, 'end': 222, 'text': 'Table 4', 'ref_id': 'TABREF6'}], 'section': 'Lower Transmission Scenario (R 0 = 1.57)'}, {'text': 'Since our compartmental model is age structured, it allows us to look at each age group separately in addition to overall population. This feature enables us to explore the vulnerability of high risk population.', 'cite_spans': [], 'ref_spans': [], 'section': 'Evaluating Closure/Reopening Decisions'}, {'text': 'Therefore, here we compare cumulative attack rate (CAR), % deaths and peak hospitalizations in each 18 All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Evaluating Closure/Reopening Decisions'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Evaluating Closure/Reopening Decisions'}, {'text': 'The copyright holder for this preprint this version posted June 26, 2020. age group separately for all transmission scenarios. In Figure 7 , we present % deaths for each age group for different transmission scenario considered. We find that the burden of the pandemic on elderly population is the worse, in all potential transmission scenario. Specifically, as the transmission rate increases, cumulative death % for elderly population increases drastically. This suggests that for dense regions, high risk groups should be given more attention.', 'cite_spans': [], 'ref_spans': [{'start': 130, 'end': 138, 'text': 'Figure 7', 'ref_id': None}], 'section': 'Evaluating Closure/Reopening Decisions'}, {'text': 'On the other hand, early trigger and late trigger would result in different death tolls. We compare different closure strategies for all levels of social distancing along with different trigger levels (See Appendix C and Table 5 ). Early trigger results in a relatively lower death rate in low transmission rate while late trigger results in a larger death rate as shown in Table 5 . This result is presented for 16 weeks of social distancing. However, for the base scenario and higher transmission rate scenario the expected deaths would be lower when later trigger levels are used. Appendix C shows that till the 16 week of social distancing (1 week, 2 weeks, 3 weeks, 4 weeks and 8 weeks) in all transmission rates death % decreases as we increase the trigger level. (See Appendix C and Tables 10, 11 , 12, 13, 14, 15).', 'cite_spans': [], 'ref_spans': [{'start': 221, 'end': 228, 'text': 'Table 5', 'ref_id': 'TABREF8'}, {'start': 374, 'end': 381, 'text': 'Table 5', 'ref_id': 'TABREF8'}, {'start': 770, 'end': 803, 'text': '(See Appendix C and Tables 10, 11', 'ref_id': 'TABREF0'}], 'section': 'Evaluating Closure/Reopening Decisions'}, {'text': 'To control the rapidly growing outbreak across the US, public health officials and governments issued shelter-in-place orders. Although social distancing and shelter-in-place orders help mitigate the outbreak, it has significant economic and social costs. For example, in the US during the social distancing for 19 All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Evaluating Closure/Reopening Decisions'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Evaluating Closure/Reopening Decisions'}, {'text': 'The copyright holder for this preprint this version posted June 26, 2020. . Figure 7 : Cumulative Deaths (%) for different R 0 values for each age group; social distancing is triggered when cumulative prevalence is reached 0.5% for 8 weeks. COVID-19, the unemployment rates have spiked. To identify possible reopening strategies, we evaluate a two-phase reopening strategy. We compare this strategy to two basic alternatives: (1) No closures (i.e., 20 All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [{'start': 76, 'end': 84, 'text': 'Figure 7', 'ref_id': None}], 'section': 'Evaluating Closure/Reopening Decisions'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Evaluating Closure/Reopening Decisions'}, {'text': 'The copyright holder for this preprint this version posted June 26, 2020 . . https://doi.org/10.1101 no social distancing) , (2) Immediate reopening (i.e., no phases).', 'cite_spans': [{'start': 59, 'end': 72, 'text': 'June 26, 2020', 'ref_id': None}, {'start': 73, 'end': 100, 'text': '. . https://doi.org/10.1101', 'ref_id': None}], 'ref_spans': [], 'section': 'Evaluating Closure/Reopening Decisions'}, {'text': 'For example, Figure 8 shows the three different scenarios: (1) no-closure, (2) Immediate reopening, and (3) Reopening with Phases for an outbreak with medium growth rate. The no-closure strategy is without any social distancing implemented, indicated by the red line. The immediate reopening strategy is returning back to original contact rates for all age groups immediately after 8 weeks of closure shown by the yellow line. The reopening with phases strategy, indicated by the blue line in Figure 8a , corresponds to reopening with phases of length 8-weeks, i.e., after 8 weeks of social distancing an 8-week long phase 1 starts with increases the contact rate with certain rate. Phase 1 is followed by another 8-week long phase 2, where contact rates are even more increased. Finally, after phase 2, original contact rates are restored (See Appendix Table 8 for the corresponding contact rates). In Figure 8a , we observe that phased reopening results in smaller outbreak sizes (i.e., smaller peak magnitude) than both the no-closure strategy and the immediate reopening strategy. We have also evaluated reopening strategies with 24-week length phases. Figure 8b shows that the burden of the pandemic is further mitigated using longer term phases. Figure 8 shows the different scenarios for the base transmission scenario and two possible lengths of reopening phases. We also show outbreaks with different growth rate and with other possible lengths of reopening phases. For example, Figure 9 shows different possible lengths of reopening phases for an outbreak with baseline transmission rate. For all lengths considered, the longer the phase duration is, the more dampened the outbreak burden, as well as the further in time the peak is observed. For example, the peak is reduced more than half with 24-week long phases instead of 4-week long phases. This presents a trade-off between hospital capacity especially how much it could be enhanced with the delay of the peak by social distancing and the size of the second peak as a result of the length of social distancing. Similarly, Figure 10 shows different possible lengths of reopening phases for an outbreak with higher transmission case. Again, for all lengths of social distancing considered, the longer the phase windows the more the reduction in the epidemic size is achieved. Though, surprisingly, the observed peak time for different reopening phases shifts marginally. Finally, Figure 11 shows different possible duration of reopening phases for the pandemic with smaller growth rate. Unsurprisingly, the overall outbreak is much smaller. Specific to this scenario, reopening 21 All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [{'start': 13, 'end': 21, 'text': 'Figure 8', 'ref_id': 'FIGREF7'}, {'start': 493, 'end': 502, 'text': 'Figure 8a', 'ref_id': 'FIGREF7'}, {'start': 845, 'end': 861, 'text': 'Appendix Table 8', 'ref_id': 'TABREF12'}, {'start': 903, 'end': 912, 'text': 'Figure 8a', 'ref_id': 'FIGREF7'}, {'start': 1157, 'end': 1166, 'text': 'Figure 8b', 'ref_id': 'FIGREF7'}, {'start': 1252, 'end': 1260, 'text': 'Figure 8', 'ref_id': 'FIGREF7'}, {'start': 1488, 'end': 1496, 'text': 'Figure 9', 'ref_id': 'FIGREF8'}, {'start': 2089, 'end': 2098, 'text': 'Figure 10', 'ref_id': 'FIGREF9'}, {'start': 2445, 'end': 2454, 'text': 'Figure 11', 'ref_id': 'FIGREF10'}], 'section': 'Evaluating Closure/Reopening Decisions'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Evaluating Closure/Reopening Decisions'}, {'text': 'The copyright holder for this preprint this version posted June 26, 2020 . . https://doi.org/10.1101 phases with 16-week and 24-week length of closure delays the peak timing even beyond the next 600 days. Arguably, the healthcare system could absorb a reopening with the minimum length considered, i.e., 4-weeks. In other words, we note that for regions with low transmission rate with enough hospital capacity, reopening relatively sooner might be a feasible strategy. Table 6 summarizes possible scenarios that minimizes expected Cumulative Attack Rate (CAR) and expected total death rate under different transmission cases considered. For lower transmission rate, an 8-week closure results in a significantly bigger reduction in CAR as well as total death toll than 4-week closure. For medium transmission rate, i.e., the baseline, we observe that a 24-week closure results in a significantly higher reduction in CAR as well as total death toll. For higher transmission rate, although a 24-week closure results in highest reduction in CAR as well as total death toll, the overall reduction magnitudes are very small. Furthermore, the marginal reduction due to additional weeks to closure appear to be negligible. To achieve reductions similar to other transmission rates longer closure scenarios should be considered.', 'cite_spans': [{'start': 59, 'end': 72, 'text': 'June 26, 2020', 'ref_id': None}, {'start': 73, 'end': 100, 'text': '. . https://doi.org/10.1101', 'ref_id': None}], 'ref_spans': [{'start': 470, 'end': 477, 'text': 'Table 6', 'ref_id': None}], 'section': 'Evaluating Closure/Reopening Decisions'}, {'text': 'As for reopening, we also show that reopening with phases might be more crucial in dense populations like New York where higher transmission is estimated. Especially, long time window reopening can significantly reduce the burden of the outbreak; which makes it easier for the healthcare system. However, in places where lower transmission rates are estimated, e.g., in Nebraska, reopening with phases will also be useful but as critical as in New York. Even the shortest time-window reopening strategies considered results with cases that the healthcare system can absorb. To summarize, one size fits all reopening policies treating entire population as one is vulnerable to result in inefficiencies. Premature reopening strategies may result in excessive overload in hospitalizations and ICU capacity creating a deadlock in healthcare system. Reopening strategies with phases might provide a feasible middle ground to manage the trade-off between improving the economy and giving more time to health officials to improve healthcare capacity and develop vaccines.', 'cite_spans': [], 'ref_spans': [], 'section': 'Evaluating Closure/Reopening Decisions'}, {'text': 'Early reopening might result in increased number of infections and death tolls in the dense areas and late reopening might result in economic loss in comparatively less dense areas. Each region should consider its own hospital/ICU capacity and other limitations while easing social distancing. Our model 22 All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Evaluating Closure/Reopening Decisions'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Evaluating Closure/Reopening Decisions'}, {'text': 'The copyright holder for this preprint this version posted June 26, 2020. . (which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Evaluating Closure/Reopening Decisions'}, {'text': 'The copyright holder for this preprint this version posted June 26, 2020. . https://doi.org/10.1101/2020.06.24.20139329 doi: medRxiv preprint 24 All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Evaluating Closure/Reopening Decisions'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Evaluating Closure/Reopening Decisions'}, {'text': 'The copyright holder for this preprint this version posted June 26, 2020. . https://doi.org/10.1101/2020.06.24.20139329 doi: medRxiv preprint 25 All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Evaluating Closure/Reopening Decisions'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Evaluating Closure/Reopening Decisions'}, {'text': 'The copyright holder for this preprint this version posted June 26, 2020. . https://doi.org/10.1101/2020.06.24.20139329 doi: medRxiv preprint 26 All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Evaluating Closure/Reopening Decisions'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Evaluating Closure/Reopening Decisions'}, {'text': 'The copyright holder for this preprint this version posted June 26, 2020 . . https://doi.org/10.1101 Table 6 : Highlight of different scenarios (i.e., different closure triggers and lengths) for the three transmission rates that minimizes both the cumulative attack rate (CAR) and total death toll, separately. The bold rows indicate the scenarios for each transmission rate, that minimizes these objectives. Note that independently minimizing CAR and minimizing total deaths result in the same scenarios.', 'cite_spans': [{'start': 59, 'end': 72, 'text': 'June 26, 2020', 'ref_id': None}, {'start': 73, 'end': 100, 'text': '. . https://doi.org/10.1101', 'ref_id': None}], 'ref_spans': [{'start': 101, 'end': 108, 'text': 'Table 6', 'ref_id': None}], 'section': 'Evaluating Closure/Reopening Decisions'}, {'text': 'provides a framework to consider the various trade-offs (e.g., the length of social distancing vs the number of hospitalizations) at a local level.', 'cite_spans': [], 'ref_spans': [], 'section': 'Evaluating Closure/Reopening Decisions'}, {'text': 'In this paper, we present a compartmental model for COVID-19 outbreak dynamics with three age groups, (i.e. kids, adults and elderly). We perform systematic and extensive simulations to evaluate social distancing and reopening scenarios for regions with different disease dynamics. Specifically, we find that the effective growth rate of the disease is different for different regions in the US. For example, New York, i.e., high density urban population, has a higher transmission rate whereas Nebraska, i.e., low density more rural population, has a lower transmission rate than the base case considered. Our simulations suggest that depending on the transmission rate estimated, the magnitude and the timing of the peak cases would vary across regions; thus the optimal mitigation policies may vary.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusions'}, {'text': 'Furthermore, we present a trade-off between overall death toll of the outbreak and the enhanced hospital capacity in a region by preparations during the delayed time of the outbreak peak. Specifically, we 27 All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusions'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusions'}, {'text': 'The copyright holder for this preprint this version posted June 26, 2020 . . https://doi.org/10.1101 consider triggering social distancing early (0.001%), late (0.5%), or very late (1%). We find that early trigger social distancing strategies result in small death tolls, however relatively larger second waves.', 'cite_spans': [{'start': 59, 'end': 72, 'text': 'June 26, 2020', 'ref_id': None}, {'start': 73, 'end': 100, 'text': '. . https://doi.org/10.1101', 'ref_id': None}], 'ref_spans': [], 'section': 'Conclusions'}, {'text': 'Conversely, late trigger social distancing strategies result in higher initial death tolls but relatively small second waves This study show that policy makers should expect multiple waves of cases as a result of the social distancing policies implemented when there are no vaccines available for mass immunization and appropriate antiviral treatments. Social distancing policies only provide time for managing the cases in the population until these pharmaceutical interventions become available. We have shown that with higher transmission scenario it is better to use relatively later trigger with longer closure duration. Also our results show that social distancing is comparatively more effective when there is lower transmission in terms of the time gained for pharmaceutical interventions mentioned above.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusions'}, {'text': 'Finally, we note that there is so much unknown about COVID-19, and our models include several assumptions regarding parameter values as summarized in Table 1 . Although, we expect the relative comparisons of different strategies should still apply, the results of this work should be considered within the limitations of the model parameters. Our results are based on parameters obtained mainly from CDC reports and early estimates of the disease. As the parameters are refined, our projections will be improved.', 'cite_spans': [], 'ref_spans': [{'start': 150, 'end': 157, 'text': 'Table 1', 'ref_id': 'TABREF0'}], 'section': 'Conclusions'}, {'text': 'The number of hospitalizations and deaths will also depend on how effectively we protect our high risk populations. In the reopening scenarios, phase 1 and phase 2 are solely based on contact rate relaxations in social distancing. Depending on the current situation, the restrictions and contact rates may have been tightened to prevent hospitalizations for overwhelming local capacities. As a future work, we are planning on controlling the spread proactively so that we could control the hospital capacity and actively use the risk-based guidelines to manage the pandemic speed.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusions'}, {'text': 'All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusions'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. Table 9 : Age specific hospitalization rate, recovery rate and mortality rate.', 'cite_spans': [], 'ref_spans': [{'start': 136, 'end': 143, 'text': 'Table 9', 'ref_id': None}], 'section': 'Conclusions'}, {'text': '32 All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusions'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusions'}, {'text': 'The copyright holder for this preprint this version posted June 26, 2020. . https://doi.org/10.1101/2020.06.24.20139329 doi: medRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusions'}, {'text': 'The basic reproductive number (R 0i ) is define as the number of secondary cases an infectious individual on a particular age group i generates during the time period he/she is infectious on the susceptible population of their age group at the beginning of an epidemic. The individuals that can potentially infect the population in our model are infected individuals either symptomatic or asymptomatic in this case. To compute the R 0i we use the next generation operator Van den Driessche & Watmough (2002) . Let the vector F be the rate of new infections flowing to the latent compartment and the vector V to be the rate of transfer of individuals out of the compartment that are able to transmit the disease. Then using our system of equations we define:', 'cite_spans': [{'start': 472, 'end': 507, 'text': 'Van den Driessche & Watmough (2002)', 'ref_id': 'BIBREF15'}], 'ref_spans': [], 'section': 'B Basic Reproduction Number R 0i'}, {'text': 'In order to compute R 0i let the gradient of F be define as F = ∂F', 'cite_spans': [], 'ref_spans': [], 'section': 'B Basic Reproduction Number R 0i'}, {'text': '∂F ∂IS i (t) and let the gradient of V be define as V = ∂V', 'cite_spans': [], 'ref_spans': [], 'section': 'B Basic Reproduction Number R 0i'}, {'text': '∂V ∂IS i (t) then we get:', 'cite_spans': [], 'ref_spans': [], 'section': 'B Basic Reproduction Number R 0i'}, {'text': 'Then R 0i is the spectral radius of the second generation operator ρ(FV −1 ) also know as the dominant eigenvalue of the matrix FV −1 . Hence,', 'cite_spans': [], 'ref_spans': [], 'section': 'B Basic Reproduction Number R 0i'}, {'text': 'All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'B Basic Reproduction Number R 0i'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'B Basic Reproduction Number R 0i'}, {'text': 'The copyright holder for this preprint this version posted June 26, 2020. .', 'cite_spans': [], 'ref_spans': [], 'section': 'B Basic Reproduction Number R 0i'}, {'text': 'Then the dominant eigenvalue of FV −1 defined as ρ(FV −1 ) is the R 0i which means that the basic reproductive number for each age group i is: 34 All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'B Basic Reproduction Number R 0i'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'B Basic Reproduction Number R 0i'}, {'text': 'The copyright holder for this preprint this version posted June 26, 2020. (which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'B Basic Reproduction Number R 0i'}, {'text': 'The copyright holder for this preprint this version posted June 26, 2020. (which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'B Basic Reproduction Number R 0i'}, {'text': 'The copyright holder for this preprint this version posted June 26, 2020. 37 All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'B Basic Reproduction Number R 0i'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'B Basic Reproduction Number R 0i'}, {'text': 'The copyright holder for this preprint this version posted June 26, 2020. . https://doi.org/10. 1101 ', 'cite_spans': [{'start': 96, 'end': 100, 'text': '1101', 'ref_id': None}], 'ref_spans': [], 'section': 'B Basic Reproduction Number R 0i'}], 'paragraphs': [{'text': 'Third, employing computational experiments with the simulation model we show that social distancing measures with different decision parameters and paths (e.g., various triggers to start and different lengths of social distancing) can result in effective management of the outbreak in dissimilar states. We find that early trigger and late trigger options present a trade-off between the peak magnitude and the overall death toll of the outbreak. Finally, we analyze reopening strategies with a two-phased reopening scenario. We show that for outbreaks with small transmission rate a short length of social distancing and immediate reopening scenario may be feasible. Whereas, for outbreaks with medium or high transmission rate the surveillance data and findings that provided details about case fatality rates and transmissibility. An early study on the transmission dynamics of COVID-19 analyzed data of the first 425 confirmed cases in Wuhan, China and found that the mean incubation period was 5.2 days. However, the 95 th percentile of the incubation period was 12.5 days in this sample (Li et al. 2020a ). In addition, this study estimated the basic reproductive number (R 0 ), which is the average number of secondary cases generated from a single infectious case in a completely susceptible population, as anywhere between 1.5 and 3.5 (Li et al. 2020a ). These epidemiological estimates show that the virus strain is highly transmissible and can infect mass populations across the globe in a very short amount of time. Another study on news reports and press releases about COVID-19 outside Wuhan estimated the mean incubation period and 97.5 th percentile as 5.1 days and 11.5 days, respectively (Lauer et al. 2020) . Linton et al. (2020) also estimated the expected incubation time around 5 days with relatively higher range of 2 to 14 days. Given relatively high estimates of the basic reproductive number R 0 , and relatively long incubation period, there is an exponential growth of confirmed COVID-19 cases in several countries and developing control measure for the epidemic is quite challenging. More importantly, a significant proportion of cases are reported to be asymptomatic but infectious cases (Li et al. 2020a) . This characteristic of the virus is found one of the main drivers of the spread of COVID-19 across populations. Therefore, social distancing measures are crucial for gaining time for healthcare systems to meet the demand for care and ultimately mitigating the impact of the pandemic.', 'id': '00004'}], 'title': 'A Decision Analytic Approach for Social Distancing Policies During the COVID-19 Pandemic', 'sha': 'd7ce8394d0f174f2d2ad2999f1f0e8a570e05251', 'doi': '10.1101/2020.06.24.20139329', 'score': 9.136500358581543}, {'rank': 200, 'abstract': 'The novel coronavirus disease (COVID-19) arises from the virus SARS-CoV-2 which is similar to the original SARS virus. The most common symptoms of the COVID-19 infection are fever, coughing and shortness of breath. According to the current data, the primary mode of transmission for the COVID-19 virus is between people through respiratory droplets and contact routes. The virus may lead to worse respiratory complications, including pneumonia, especially in older patients and patients with pre-existing illnesses, such as cancer. Cancer patients are at a significantly higher risk of getting infected with COVID-19 since their immune system can be compromised and that reality has to do with both that they have cancer and that they are on therapy for their cancer. COVID-19 crisis has impacted every aspect of the practice, including outpatient, elective, wards, emergency care, conferences, teaching and research. We should make sure cancer patients on active treatment are treated appropriately. In this review, we tried to explain how to prevent the negative effects of the COVID-19 pandemic on the diagnosis and treatment of breast cancer patients.', 'body_text': [{'text': 'Following extensive discussions, the consortium of organizations and their representatives divided groups of patients with breast cancer according to three priorities as follows:[4]', 'cite_spans': [{'start': 179, 'end': 180, 'mention': '4', 'ref_id': 'BIBREF8'}], 'section': 'Classifications', 'ref_spans': []}, {'text': '\\nPriority A: Patients in this group are patients who need to be treated immediately and have a disease that may threaten their life or progressively disturbing symptoms if treatment is not performed.Priority B: Patients in this group are patients who do not need immediate treatment, but patients whose treatment cannot be postponed until the end of the long-term pandemic. The priority B is divided into subgroups as follows: B1 (higher priority), B2 (mid-level priority), B3 (lower priority).Priority C: Patients in this group are patients whose treatment can be delayed until the pandemic is over and patients who will not experience a negative result as a result of this delay.\\n', 'cite_spans': [], 'section': 'Classifications', 'ref_spans': []}, {'text': 'Approaches and treatment suggestions (outpatient visit, diagnostics and imaging of breast disease, surgical oncology, radiation oncology and medical oncology) have been classified by the European Society for Medical Oncology (ESMO) as high, medium and low priority.[5]', 'cite_spans': [{'start': 266, 'end': 267, 'mention': '5', 'ref_id': 'BIBREF9'}], 'section': 'Classifications', 'ref_spans': []}, {'text': '\\nHigh-priority: Recommendations are applied to patients whose condition is either clinically unstable or who have life-threatening cancer burden and requires immediate hospital treatment.Medium-priority: Recommendations are applied to patients for whom delaying care beyond six weeks. Risk of delays caused by the COVID-19 pandemic are assessed, there should not be a concern that they may negatively affect this situation.Low-priority: Recommendations are applied to patients for whom services can be delayed until the end of the COVID-19 pandemic.\\n', 'cite_spans': [], 'section': 'Classifications', 'ref_spans': []}, {'text': 'American College of Surgeons (ACS), evaluated health system and hospitals in three phases by the community adequacy of resources, health workers:', 'cite_spans': [], 'section': 'Classifications', 'ref_spans': []}, {'text': '\\nPhase 1: The number of COVID-19 patients in hospitals is low and resources are sufficient. Neoadjuvant patients finishing treatment, clinical stage T2 or N1 ER +/PR +/HER2-tumors, triple-negative or HER2 + patients, discordant biopsies likely to be malignant, excision of malignant recurrence are the cases need to be done as soon as possible.Phase 2: The number of COVID-19 patients has increased; hospitals have limited resources. Incision and drainage of breast abscess, evacuation of a hematoma, revision of an ischemic mastectomy flap, and revision of an autologous tissue flap are the cases that need to be made as soon as possible.Phase 3: Hospitals are full of COVID-19 patients; hospital resources are stated to be insufficient.\\n', 'cite_spans': [], 'section': 'Classifications', 'ref_spans': []}, {'text': 'Incision and drainage of breast abscess, evacuation of a hematoma, revision of an ischemic mastectomy flap, and revision of an autologous tissue flap are the cases that need to be made as soon as possible.', 'cite_spans': [], 'section': 'Classifications', 'ref_spans': []}, {'text': 'Neoadjuvant chemotherapy may be considered in appropriate cases in phase 2 and phase 3 conditions. If institutional resources are sufficient in phase 1 conditions, it may be preferable to proceed with surgery instead of subjecting a patient to an immunocompromised state with neoadjuvant chemotherapy.[6]', 'cite_spans': [{'start': 302, 'end': 303, 'mention': '6', 'ref_id': 'BIBREF10'}], 'section': 'Classifications', 'ref_spans': []}, {'text': 'All imaging studies, including mammography, ultrasound, and MRI, should be postponed until the COVID-19 period ends because there is no evidence of delaying screening mammography for the proposed short time will affect mortality.[7] Diagnostic imaging for an abnormal mammogram or for suspicious symptoms of breast cancer, biopsies for BI-RADS 4 or 5 lesions, and breast MRI for the extent of disease evaluation or pre-chemotherapy assessment are still being performed in hospitals having sufficient resources because pandemic may be long-lasting and this may threaten patients’ lives in the long run.[4]', 'cite_spans': [{'start': 230, 'end': 231, 'mention': '7', 'ref_id': 'BIBREF11'}, {'start': 602, 'end': 603, 'mention': '4', 'ref_id': 'BIBREF8'}], 'section': 'Imaging Studies', 'ref_spans': []}, {'text': 'Although we specifically talk about breast cancer treatment management, we would like to draw attention to some invariable rules in all surgical operations. All patients are questioned for signs of COVID-19 at admission, and patients with fever and respiratory symptoms are referred to the relevant departments during admission. In COVID-19 patients, operations are delayed if not urgent. Before surgery, all resources like devices, healthcare workers, intensive care beds, should be recorded against the possibility of being used at any time.', 'cite_spans': [], 'section': 'Surgical Procedures', 'ref_spans': []}, {'text': 'In the operating rooms, only the minimum number of staff must be employed alternately. Periodic training is given to the operating room staff about COVID-19 symptoms and prevention methods by digital methods. Personal protective equipment, including N95/FFP2 mask, sterile surgical gowns, disposable sterile gloves, goggles/protective visors, disposable bone, alcohol-based hand antiseptic, is provided to the healthcare staff in the operating room. Special stretcher, operating room, and elevator are reserved for patients who are infected with COVID-19 to be operated. Necessary precautions are taken before the operation, patient transfer, during and after the operation.', 'cite_spans': [], 'section': 'Surgical Procedures', 'ref_spans': []}, {'text': 'WHO, China’s National Health Commission (NHC), The United States’ CDC reported an incubation period for COVID-19 between 2-10, 10-14, 2-14 days, respectively.[8–10] As a result, although the incubation period is reported on average between 2-14 days, there is still no exact interval. Thus, we should assume everyone could be a carrier. Twenty-four hours before elective surgeries, PCR (Polymerase Chain Reaction) tests should be performed; patients should be isolated and visitors should not be taken to their rooms. If the PCR test result is positive, the operation of the patient should be postponed unless it is urgent; COVID-19 patients should be treated by following appropriate protocols. However, if the test result is negative, the test should be renewed every week as long as the patient stays in the hospital after the operation and PCR test should be performed before discharge.[11]', 'cite_spans': [{'start': 159, 'end': 160, 'mention': '8', 'ref_id': 'BIBREF12'}, {'start': 161, 'end': 163, 'mention': '10', 'ref_id': 'BIBREF1'}, {'start': 891, 'end': 893, 'mention': '11', 'ref_id': 'BIBREF2'}], 'section': 'Surgical Procedures', 'ref_spans': []}, {'text': 'Many guidelines have common points in approaching breast cancer treatment, but patients are recommended to be evaluated on a case-by-case basis. Unless a complication, such as an abscess, infection, requires urgent treatment, surgery should be postponed for at least three months due to atypia, prophylactic/risk-reducing surgery, and benign breast disease. ER expressing types of atypical hyperplasia and lobular carcinoma in situ can be treated with tamoxifen or aromatase inhibitors. Although the main treatment of DCIS is breast-conserving surgery with or without whole breast radiation therapy, or alternatively, mastectomy, treatment management has changed due to pandemics. In this pandemic period, it is important to specify the hormone receptor status in DCIS-compliant core biopsies taken in the treatment of DCIS. ER + DCIS patients should be treated with endocrine therapy primarily and they should be checked via telehealth periodically. ER- DCIS patients with low volume and low risk of invasive disease can be followed periodically without treatment. If ER- DCIS is high grade, large volume or palpable, reassessments of patients using telemedicine should be more frequent. In DCIS with microinvasion, the treatment approach, such as ER + invasive cancers is shown in ER + DCIS patients, while patients with ER- are treated like ER- DCIS patients.[12]', 'cite_spans': [{'start': 1363, 'end': 1365, 'mention': '12', 'ref_id': 'BIBREF3'}], 'section': 'Treatment Approach', 'ref_spans': []}, {'text': 'Generally, patients with early-stage breast cancer undergo primary surgery (breast-conserving surgery or mastectomy) with or without radiotherapy. Following local treatment, adjuvant systemic therapy may be recommended based on primary tumor characteristics, such as tumor size, stage, lymph node involvement, ER/PR status, and expression of the HER2 receptor. However, in the pandemic period, some priorities have also changed in the treatment of invasive breast cancer. ER+ early breast cancer patients should have neoadjuvant endocrine therapy before surgery during the COVID-19 pandemic. The decision of selection between Tamoxifen and aromatase inhibitors depends on patients’ premenopausal or postmenopausal situation and hormonal status. If the patient is not postmenopausal, sequential evaluation of hormonal status is recommended to consider an alternative endocrine agent.[12]', 'cite_spans': [{'start': 883, 'end': 885, 'mention': '12', 'ref_id': 'BIBREF3'}], 'section': 'Treatment Approach', 'ref_spans': []}, {'text': 'The rate of pathologic complete response (pCR) is higher in HER2 overexpressing (HER2+) and triple-negative (TN) breast cancers, and less frequent in the more common hormone- receptor-positive cancers. As a result, TN and Her2 + breast cancer patients with T2 or larger tumor or LN involvement are candidates for neoadjuvant therapy, and there was no significant difference in the approach in these patients during the pandemic process. Patients with T1N0M0 disease also encouraged to have NAC before surgery during the COVID-19 pandemic.[12]', 'cite_spans': [{'start': 539, 'end': 541, 'mention': '12', 'ref_id': 'BIBREF3'}], 'section': 'Treatment Approach', 'ref_spans': []}, {'text': 'Patients who are having NAC are at increased risk for COVID-19 infection because of immune deficiency. For the most part, symptoms of COVID-19 are the same in cancer patients as the general population. Medications in breast cancer patients receiving NAC can suppress the development of a fever or other symptoms, and COVID-19 disease may remain asymptomatic until they develop sepsis. If the patient tests positive for COVID-19 while receiving cancer treatment, we need to stop treatment until the patient recovers.[13]', 'cite_spans': [{'start': 516, 'end': 518, 'mention': '13', 'ref_id': 'BIBREF4'}], 'section': 'Treatment Approach', 'ref_spans': []}, {'text': 'Reconstructive surgery procedures, such as delayed breast reconstruction and planned secondary or revision breast reconstruction, are elective processes and should be delayed until the pandemic is over. Immediate reconstruction after lumpectomy or mastectomy should be made by risk balancing due to the possibility of complications, prolonging of the hospitalization period and the need for additional surgeries. Immediate autologous flap reconstruction for breast reconstruction is elective and should be delayed, but decisions regarding immediate tissue expander or direct implant reconstruction should be made according to the hospital’s resources and patient characteristics.[14]', 'cite_spans': [{'start': 680, 'end': 682, 'mention': '14', 'ref_id': 'BIBREF5'}], 'section': 'Reconstruction', 'ref_spans': []}, {'text': 'In this review, we have explained some changes in the approach to breast cancer above, but the tailored therapy must be our first goal individually, depending on the healthcare workers, the materials and devices to be used, intensive care unit bed capacity and other resources. Breast cancer treatment requires a multidisciplinary approach. Dynamic changing of local conditions may affect the priority levels and treatment options. Regional ministry of health departments must have all hospital capacity for outpatient care. Patient treatment decisions should be made by thinking both during and after the pandemic management of healthcare systems. Social distancing and precautionary measures are still in effect. More recently, regions with low rates of infection are planning to gradually reopen their economies and begin lifting some of these measures. Unfortunately, remaining at home may be difficult for patients among us with severe medical conditions that require urgent attention, but COVID-19 impacts everything, including how we manage breast cancer.', 'cite_spans': [], 'section': 'Conclusion', 'ref_spans': []}], 'paragraphs': [{'text': 'WHO, China’s National Health Commission (NHC), The United States’ CDC reported an incubation period for COVID-19 between 2-10, 10-14, 2-14 days, respectively.[8–10] As a result, although the incubation period is reported on average between 2-14 days, there is still no exact interval. Thus, we should assume everyone could be a carrier. Twenty-four hours before elective surgeries, PCR (Polymerase Chain Reaction) tests should be performed; patients should be isolated and visitors should not be taken to their rooms. If the PCR test result is positive, the operation of the patient should be postponed unless it is urgent; COVID-19 patients should be treated by following appropriate protocols. However, if the test result is negative, the test should be renewed every week as long as the patient stays in the hospital after the operation and PCR test should be performed before discharge.[11]', 'id': '00012'}], 'title': 'Management of Breast Cancer during the COVID-19 Pandemic', 'sha': '4bb317e880d612fa2e73fb54bf3b92ad59c8afa4', 'doi': '10.14744/semb.2020.23326', 'score': 9.128899574279785}, {'rank': 201, 'abstract': 'We used contact tracing to document how COVID‐19 was transmitted across 5 generations involving 10 cases, starting with an individual who became ill on January 27. We calculated the incubation period of the cases as the interval between infection and development of symptoms. The median incubation period was 6.0 days (interquartile range, 3.5‐9.5 days). The last two generations were infected in public places, 3 and 4 days prior to the onset of illness in their infectors. Both had certain underlying conditions and comorbidity. Further identification of how individuals transmit prior to being symptomatic will have important consequences.', 'body_text': [{'text': 'Novel coronavirus disease 2019 (COVID‐19) was declared a pandemic on March 11, 2020.\\n1\\n Transmission chains have been difficult to identify in routine investigations. Notable cases in Italy\\n2\\n and the United States\\n3\\n had no known source of infection. These cases have raised the concern that there is transmission of the novel coronavirus (SARS‐CoV‐2) in asymptomatic individuals (who never express symptoms) or in pre‐symptomatic individuals (prior to any symptoms).', 'cite_spans': [], 'section': 'INTRODUCTION', 'ref_spans': []}, {'text': 'Many public actions so far, including quarantine measures, body temperature measurement, and fever symptom surveillance, have prioritized identification of possible infected cases. However, these all depend on active expression of symptoms and are not able to identify asymptomatic transmission or pre‐symptomatic transmission.\\n4\\n\\n', 'cite_spans': [], 'section': 'INTRODUCTION', 'ref_spans': []}, {'text': 'Better understanding of transmission parameters, including the serial interval and incubation period, can also help determine the possible progression of the outbreak. The serial interval is defined as the interval between a primary case of COVID‐19 developing symptoms and a secondary case developing symptoms, whereas the incubation period is the time lag between infection and the start of symptoms. The Chinese Center for Disease Control and Prevention (CDC) has estimated that the mean serial interval for COVID‐19 is 7.5 days, which is slightly longer than the estimated incubation period of 5.2 days.\\n5\\n However, the serial interval has varied in other studies, such as a median of 4.6 days,\\n6\\n which is shorter than the incubation period.', 'cite_spans': [], 'section': 'INTRODUCTION', 'ref_spans': []}, {'text': 'The challenge for controlling COVID‐19 is to determine at what point an individual becomes infectious, which can have implications for contact tracing and other epidemiological investigations. In addition, one study in China showed that 12.1% of transmission was likely to be pre‐symptomatic.\\n7\\n On February 24, 2020, the China CDC revised the definition of the start date for close contact from “at the illness onset of a confirmed case” to “2 days before the illness onset,”\\n8\\n as increased evidence suggests pre‐symptomatic transmission might be plausible.', 'cite_spans': [], 'section': 'INTRODUCTION', 'ref_spans': []}, {'text': 'Many previous studies of the transmission dynamics of COVID‐19 have relied on publicly available data or large surveillance datasets. It is difficult to discover asymptomatic or pre‐symptomatic cases in these datasets, and overall there is limited information on how to determine when an individual becomes infectious. This study describes a transmission chain of 5 generations involving 10 COVID‐19 cases, and tracks whether infection occurred from asymptomatic or pre‐symptomatic individuals.', 'cite_spans': [], 'section': 'INTRODUCTION', 'ref_spans': []}, {'text': 'An index case (ie, the last case) was first identified in our study who had no obvious previous contact with a symptomatic case of COVID‐19. For the index case, we identified 3 close contacts in the 14 days prior to disease onset, none of which had COVID‐19 at the time of contact. We worked backward to identify several generations of transmission. We used contact tracing to identify possible people who could have exposed the case. Close contacts were defined as people who live, study, work, or otherwise have close contact with the case; medical personnel, family members, or other people who have similarly had close contact with the case and who did not take effective protective measures; other patients and their accompanying staff in the same ward of the case; persons in the same vehicle as the case and who had close contact with the case; and other persons who were deemed close contacts by the field investigator.', 'cite_spans': [], 'section': 'METHODS', 'ref_spans': []}, {'text': 'We retrospectively summarized the journey and visited places of the cases within 14 days before the illness onset in each generation. In this way, we identified possible source cases and we worked backward to identify cases, which were epidemiologically linked with each other, and which constituted a complete transmission chain. Information about symptoms and date of illness onset was provided directly by the cases. We acknowledge recall bias: Cases may not have been able to remember every contact they had.', 'cite_spans': [], 'section': 'METHODS', 'ref_spans': []}, {'text': \"All of the investigations were conducted by experienced investigators at Centers for Disease Control and Prevention in the Shanghai Municipality and Zhejiang Province. This study involved the use of existing, routinely collected data from a public health outbreak investigation, under the National Health Commission of the People's Republic of China. Thus, this study is exempt from ethical review and informed consent was not obtained.\", 'cite_spans': [], 'section': 'METHODS', 'ref_spans': []}, {'text': 'The index case was a 74‐year‐old female who developed symptoms on January 27. She lived alone and had very limited contact with foreign people. She was found to have met with a 75‐year‐old male, in a real estate trade center for 3 hours on January 21, 6 days prior to the onset of her symptoms. The male was recognized as an existing confirmed case (case 1) of COVID‐19, who developed symptoms only on January 25. Subsequently, case 2 was identified, who was a friend of case 1 and had met him for 2.5 hours at a gym on January 19. Case 2 developed symptoms on January 22. The timeline of possible infection and illness onset is illustrated in Figure 1.', 'cite_spans': [], 'section': 'Initiation of investigation ::: RESULTS', 'ref_spans': [{'start': 651, 'end': 652, 'mention': '1', 'ref_id': 'FIGREF0'}]}, {'text': 'We determined the epidemiological linkages between index case, case 1, and case 2, after excluding all of their close contacts who remained healthy throughout a 14‐day medical observation. It is noted that the infectees (index case and case 1) were possibly infected with SARS‐CoV‐2 by infectors (case 1 and case 2) during the incubation periods of the infectors (Figure 1).', 'cite_spans': [], 'section': 'Identification of two occasions of pre‐symptomatic transmission ::: RESULTS', 'ref_spans': [{'start': 371, 'end': 372, 'mention': '1', 'ref_id': 'FIGREF0'}]}, {'text': \"To confirm the finding, we further checked whether there was previous contact between these cases. Cases 1 and 2 had not met for about one month, and did not meet after January 19. Thus, the transmission was likely to occur on January 19 in the gym. The index case and case 1 had also met on January 18. However, case 1 was not infected on that day and thus was likely to have occurred on January 21. Through an examination of cases 1 and 2's clinical symptoms, we inferred both transmitted infection while pre‐symptomatic.\", 'cite_spans': [], 'section': 'Identification of two occasions of pre‐symptomatic transmission ::: RESULTS', 'ref_spans': []}, {'text': \"We worked backward to further investigate the chain of transmission. Case 2 and case 3 had not met before; however, they possibly became infected by a common friend (case 4) at dinners on January 13 and 14. Similarly, cases 4‐7 became infected at dinners with a common friend (case 8). During the dinners, case 8's daughter (case 9) could have been the common source of infection, and she was infected by her co‐worker who had traveled back from the city of Wuhan, Hubei Province. Across these cases, we did not recognize any evidence of pre‐symptomatic transmission.\", 'cite_spans': [], 'section': 'Source of transmission chain ::: RESULTS', 'ref_spans': []}, {'text': \"In this study, all the cases had an incubation period less than 14 days, with a median of 6.0 days (interquartile range, 3.5‐9.5 days). Comparatively, we calculated the median serial interval to be 5.0 days (interquartile range, 3.0‐5.0 days). For the two instances of pre‐symptomatic transmission, the infectees' contact with the infectors occurred 3 or 4 days prior to the infectors' illness onset, which suggests a shorter latent period than an incubation period.\", 'cite_spans': [], 'section': 'Incubation period and serial interval ::: RESULTS', 'ref_spans': []}, {'text': 'To identify differences between pre‐symptomatic transmission and symptomatic transmission, we examined the health conditions of all cases. The index case underwent a surgery for stomach cancer and had been receiving chemotherapy for 1.5 years. Case 1 suffered from pneumonia and recovered late December 2019. No other cases had underlying conditions and comorbidity.', 'cite_spans': [], 'section': 'Co‐morbidities ::: RESULTS', 'ref_spans': []}, {'text': 'In this study, we traced the spread of coronavirus across five generations in Shanghai, China, back to a case who had previously traveled to Wuhan, the original source of the outbreak. The serial interval was estimated to be a little shorter than the incubation period, and there were no known symptoms among suggested possible pre‐symptomatic transmission. However, two infector‐infectee pairs with pre‐symptomatic transmission were identified out of 9 pairs. The index case and case 1 may have been more susceptible to pre‐symptomatic infection due to their health conditions, compared to other cases.', 'cite_spans': [], 'section': 'DISCUSSION', 'ref_spans': []}, {'text': \"That the latent period may be shorter than the incubation period could affect recommendations for contact tracing and case definitions. We found evidence of pre‐symptomatic transmission 3‐4 days prior to the infectors' illness onset. Thus, we suggest advancing the upper time limit of close contact to 4 days prior to illness onset of a COVID‐19 case, which is greater than the China CDC and WHO guidelines of 2 days.\\n8\\n, \\n9\\n We note that guidelines in Beijing use a time limit of 4 days as of May 18, 2020.\\n10\\n One another study in China reported only 2‐day lead time between infectee's contact with the infector and infector's illness onset,\\n11\\n and we have not found any information that this could be longer. We recognize that increasing the duration of a possible time of infection can add to the workload of routine epidemiological investigations during the epidemic of COVID‐19. Consequently, we recommend it should be applied to cases with an unknown source of infection, such as the index case in our study. Contact tracing guidelines should also rely on the capacity of contact tracing and on updated information on when pre‐symptomatic transmission can occur. For example, a study of COVID‐19 cases in Taiwan also started investigations of COVID‐19 up to 4 days before symptom onset, although this was not consistently done.\\n12\\n Evidence from more chains of transmission can better delineate the borderline between the latent period and infectious period.\", 'cite_spans': [], 'section': 'DISCUSSION', 'ref_spans': []}, {'text': 'Asymptomatic transmission has been documented among returned Japanese citizens from China.\\n13\\n However, it might be misunderstood due to short duration of observation, that is to say, if we extend the duration of observation, we might observe the occurrence of symptoms. Consequently, pre‐symptomatic transmission may be misunderstood as asymptomatic transmission. So far, asymptomatic transmission and pre‐symptomatic transmission have not been well documented and further study of natural history of SARS‐CoV‐2 infection is urgently warranted.', 'cite_spans': [], 'section': 'DISCUSSION', 'ref_spans': []}, {'text': 'All authors: No reported conflicts of interest.', 'cite_spans': [], 'section': 'CONFLICTS OF INTEREST', 'ref_spans': []}, {'text': '\\nDechuan Kong: Conceptualization (equal); Investigation (equal); Visualization (supporting); Writing‐original draft (supporting). Yang Zheng: Conceptualization (equal); Investigation (equal); Writing‐review & editing (supporting). Huanyu Wu: Conceptualization (equal); Funding acquisition (equal); Investigation (equal); Writing‐review & editing (supporting). Hao Pan: Conceptualization (equal); Funding acquisition (equal); Investigation (equal); Writing‐original draft (supporting). Abram L. Wagner: Writing‐review & editing (supporting). Yaxu Zheng: Investigation (supporting); Writing‐review & editing (supporting). Xiaohuan Gong: Investigation (supporting); Writing‐review & editing (supporting). Yiyi Zhu: Investigation (supporting); Writing‐review & editing (supporting). Bihong Jin: Investigation (supporting); Writing‐review & editing (supporting). Wenjia Xiao: Investigation (supporting); Writing‐review & editing (supporting). Shenghua Mao: Investigation (supporting); Writing‐review & editing (supporting). Sheng Lin: Investigation (supporting); Writing‐review & editing (supporting). Ruobing Han: Investigation (supporting); Writing‐review & editing (supporting). Xiao Yu: Investigation (supporting); Writing‐review & editing (supporting). Peng Cui: Investigation (supporting); Writing‐review & editing (supporting). Chenyan Jiang: Investigation (supporting); Writing‐review & editing (supporting). Qiwen Fang: Investigation (supporting); Writing‐review & editing (supporting). Yihan Lu: Investigation (supporting); Visualization (lead); Writing‐original draft (lead). Chen Fu: Conceptualization (equal); Funding acquisition (equal); Investigation (equal); Writing‐review & editing (supporting).', 'cite_spans': [], 'section': 'AUTHOR CONTRIBUTION', 'ref_spans': []}], 'paragraphs': [{'text': ' which is shorter than the incubation period.', 'id': '00003'}], 'title': 'Pre‐symptomatic transmission of novel coronavirus in community settings', 'sha': '33d4618670f943618b03739410e3a7b023c7d9d9', 'doi': '10.1111/irv.12773', 'score': 9.119400024414062}, {'rank': 202, 'paragraphs': [{'text': 'In response to the COVID-19 global pandemic, many countries have implemented large-scale physical (\"social\") distancing measures. The details of physical distancing vary significantly between and within countries. While government-mandated lockdowns have been enforced in many parts of the world, certain regions have introduced less strict distancing measures. Such differences in the severity and timeliness of the response, as well as variation in patterns of social contact within a community and public compliance, all modulate the effect of distancing measures on a region\\'s epidemic trajectory. The relaxation of physical distancing must be done gradually and with care. Accordingly, countries experiencing increasing case counts have considered, and will continue to need to consider, what degree of distancing measures to implement in order to reduce disease spread whilst maintaining public compliance and minimising negative effects of distancing (such as adverse health and economic impacts). Understanding the possible trajectories which may arise from changing physical distancing measures is crucial to ensuring that sufficient measures are still taken after relaxation begins. Quantifying the effect and time scale of such relaxations will contribute to health care capacity forecasting, and delivery of timely advice and instructions to communities.', 'id': '00001'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.117799758911133}, {'rank': 203, 'paragraphs': [{'text': 'In this work, we use a likelihood-based approach to determine when we may expect to see the effects of implementing or relaxing physical distancing measures using case count data. This is in contrast to the work described above which focuses on quantifying the effect of such measures. We further use this method to estimate the changes in growth rate that would be expected after beginning to strengthen or relax physical distancing. We use the SEIR-type model developed in [2] to simulate scenarios over a broad range of simulation parameters and strengths of distancing. We consider the impact on the time to detect a change in distancing of data collection delays and inconsistencies, and apply our methods to publicly available case count data from British Columbia, Canada. While we focus here on the \"first\" implementation and relaxation of physical distancing for COVID-19, these methods can also be used to detect the first time we would expect to see a change in reported case counts in response to modifications of any non-pharmaceutical intervention (NPI). For example, we could use this model to explore the effects of introducing digital contact tracing or improved testing.', 'id': '00003'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.117798805236816}, {'rank': 204, 'paragraphs': [{'text': 'Our analysis has a number of limitations. Although policy changes happen at defined times, distancing behaviour and other COVID-19 control practices do not change instantaneously, and the time frame to detect changes may therefore be longer than we have estimated. We have not explored a wide range of growth rates or baseline prevalence levels and these may affect the results. The transmission model used is a deterministic SEIR variant, and does not include stochastic effects (except in the observation model), or age or risk structure. The model includes a fraction of the population practising physical distancing, thereby reducing their contact rates, but does not otherwise include heterogeneity in contact patterns. In particular, we focus on the case onset-toreporting delay, but certain high-risk groups such as healthcare workers may be more likely to get tested and have expedited testing available. Our methodology could readily be extended to structured models, but this requires age-stratified counts and knowledge of the mixing terms. Indeed, any disease model will include exponential growth and decay, and this work is somewhat modelagnostic in that, whatever level of detail goes into producing this exponential behaviour, we can still perform the same eventual inference.', 'id': '00038'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.1177978515625}, {'rank': 205, 'paragraphs': [{'text': 'In conclusion, this study obtained robust estimates of several key epidemiological parameters of COVID-19. It provides additional evidence on the mean incubation period of COVID-19, which supports current practice of 14-day quarantine of persons with potential exposure but also suggests that longer monitoring periods might be needed for selected groups. The estimates of serial interval, time point of exposure and latent period provide consistent evidence on pre-symptomatic transmission. Asymptomatic transmission was also observed. This together with asymptomatic transmission and the generally longer incubation and serial interval of less severe cases suggest a high risk of long-term epidemic in the absence of appropriate control measures.', 'id': '00047'}], 'title': 'Epidemiological parameters of coronavirus disease 2019: a pooled analysis of publicly reported individual data of 1155 cases from seven countries', 'sha': '3ece7750f80832bf9cc368c5138df12fd19183b9', 'doi': '10.1101/2020.03.21.20040329', 'score': 9.110300064086914}, {'rank': 206, 'paragraphs': [{'text': 'The incubation period of an infectious disease is the time elapsed between infection and appearance of the first symptoms and signs. Precise knowledge of the incubation period would help to provide an optimal length of quarantine period for disease control purpose, and also is essential in the investigation of the mechanism of transmission and development of treatment. For example, the distribution of the incubation period is used to estimate the reproductive number, that is, the average number of secondary infections produced by a primary case. The reproductive number is a key quantity that impacts the potential size of an epidemic. Despite the importance of the incubation period, it is often poorly estimated based on limited data.', 'id': '00002'}], 'title': 'Estimation of incubation period distribution of COVID-19 using disease onset forward time: a novel cross-sectional and forward follow-up study', 'sha': 'c61be9fc4562d250e2dec241ef75bd22d61f61d1; 6d3b3f4ab80a61c45f82c61c6c756cfc6ddf4bf2', 'doi': '10.1101/2020.03.06.20032417', 'score': 9.107999801635742}, {'rank': 207, 'abstract': 'Background Timely diagnosis of SARS-CoV-2 infection is the prerequisite for treatment and preventive quarantine. The serology characteristics and complement diagnosis value of antibody test to RNA test needs to be demonstrated. Method A patient cohort study was conducted at the first affiliated hospital of Zhejiang University, China. Serial sera of COVID-19 patients were collected and total antibody (Ab), IgM and IgG antibody against SARS-CoV-2 were detected. The antibody dynamics during the infection were described. Results The seroconversion rate for Ab, IgM and IgG in COVID-19 patients was 98.8% (79/80), 93.8% (75/80) and 93.8% (75/80), respectively. The first detectible serology marker is total antibody and followed by IgM and IgG, with a median seroconversion time of 15, 18 and 20 day post exposure (d.p.e) or 9, 10 and 12 days post onset, separately. The antibody levels increased rapidly since 6 d.p.o and accompanied with the decline of viral load. For patients in the early stage of illness (0-7d.p.o),Ab showed the highest sensitivity (64.1%) compared to the IgM and IgG (33.3% for both, p<0.001). The sensitivities of Ab, IgM and IgG detection increased to 100%, 96.7% and 93.3% two weeks later, respectively. Conclusions Typical acute antibody response is induced during the SARS-CoV-2 infection. The serology testing provides important complementation to RNA test for pathogenic specific diagnosis and helpful information to evaluate the adapted immunity status of patient. It should be strongly recommended to apply well-validated antibody tests in the clinical management and public health practice to improve the control of COVID-19 infection.', 'body_text': [{'text': 'In the early December 2019, a novel coronavirus (SARS-CoV-2) was first reported to cause lethality pneumonia in human and person-to-person transmission had been demonstrated soon in Wuhan, the capital city of Hubei province, China. [1] The virus rapidly spread through China and then many other countries globally. Up to March 17, 2020, the virus had resulted in over 190,000 laboratory-confirmed cases of coronavirus disease 2019 (Covid-19) and more than 7800 deaths in 163 countries. [2] The World Health Organization (WHO) has declared Covid-19 a public health emergency of international concern and given a \"very high\" risk assessment on global level. [3] A recent report from China showed that the median incubation period of Covid-19 infection was 4 days (interquartile range, 2 to 7). [4] Fever, cough and fatigue are the most common symptoms. [1] Severe cases could rapidly progress to acute respiratory distress syndrome (ARDS) and septic shock. The abnormalities on chest computed tomography, particularly the ground-glass opacity and bilateral patchy shadowing, were found in over 80% of patients. [5] Over 80% of patients had lymphopenia, and about 60% of patients had elevated C-reactive protein. Currently, the diagnosis of SARS-CoV-2 infection almost solely depends on the All rights reserved. No reuse allowed without permission.', 'cite_spans': [{'start': 232, 'end': 235, 'text': '[1]', 'ref_id': None}, {'start': 486, 'end': 489, 'text': '[2]', 'ref_id': 'BIBREF1'}, {'start': 656, 'end': 659, 'text': '[3]', 'ref_id': 'BIBREF2'}, {'start': 792, 'end': 795, 'text': '[4]', 'ref_id': 'BIBREF3'}, {'start': 851, 'end': 854, 'text': '[1]', 'ref_id': None}, {'start': 1109, 'end': 1112, 'text': '[5]', 'ref_id': None}], 'ref_spans': [], 'section': 'Introduction'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'The copyright holder for this preprint this version posted March 27, 2020. . https://doi.org/10.1101/2020.03.23.20041707 doi: medRxiv preprint detection of viral RNA using polymerase chain reaction (PCR) based technics. [8] Unfortunately, the sensitivity of RNA test in the real-world is not satisfied, particularly when sample collected from upper respiratory tract is used. [9] [10] [11] [12] In Wuhan, the overall positive rate of RNA testing is estimated to be around 30-50% in Covid-19 patients when they come to hospital.', 'cite_spans': [{'start': 220, 'end': 223, 'text': '[8]', 'ref_id': None}, {'start': 376, 'end': 379, 'text': '[9]', 'ref_id': None}, {'start': 380, 'end': 384, 'text': '[10]', 'ref_id': None}, {'start': 385, 'end': 389, 'text': '[11]', 'ref_id': None}, {'start': 390, 'end': 394, 'text': '[12]', 'ref_id': None}], 'ref_spans': [], 'section': 'Introduction'}, {'text': '[13] Furthermore, the overall throughput of available RNA tests are highly limited by their nature of requiring high workload, needing skillful operators for testing and sample collection, and needing costly instrument and special operate places. [14] As a result, a convenient serological detection is expected to be helpful. However, current knowledge of the antibody response to SAR-CoV-2 infection is very limited. The diagnosis value of antibody test remains to be clearly demonstrated. How many patients would raise antibody response and how long will the antibody convert to be positive since the exposure?', 'cite_spans': [{'start': 247, 'end': 251, 'text': '[14]', 'ref_id': None}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'Are there any meaningful differences between patients with short and long incubation period? What are the sensitivities of antibody detection for patients in different illness stages? Is there any temporal association between the antibody response and the decline of viral load? In order to answer some of the questions, we investigated the characteristics of antibody responses in 80 Covid-19 patients during their hospitalization periods, through detecting total antibody, IgM and IgG using immunoassays.', 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Study design and participants'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. fever and/or respiratory symptoms; 2) abnormal lung imaging findings; and 3) positive result of the nucleic acid of SARS-CoV-2. The degree of severity of patient was categorized as critical case if any of the bellowed clinical scenes appeared: 1) with ARDS or oxygen saturation < 93% and needing mechanical ventilation either invasively or non-invasively; 2) shock; and 3) complication of organ functional failure and need intensive care unit support. A Covid-19 patient did not meet the above criteria was defined as non-critical case.', 'cite_spans': [], 'ref_spans': [], 'section': 'Study design and participants'}, {'text': 'This study enrolls a total of 80 cases of COVID-19, where all patients were admitted to the hospital between Jan 19 and Feb 9, 2020, and were willing to donate their blood samples. All enrolled cases were confirmed to be infected by SARS-CoV-2 through real-time RT-PCR (rRT-PCR) testing. The date of illness onset, clinical classification, RNA testing results during the hospitalization period, and the personal demographic information were obtained from the clinical records. A total of 300 healthy people were enrolled from the local community. This study was reviewed and approved by the Medical Ethical Committee of the first affiliated hospital of Zhejiang University (approval number 2020-IIT-47). Written informed consent was obtained from each enrolled subject.', 'cite_spans': [], 'ref_spans': [], 'section': 'Study design and participants'}, {'text': 'The total antibody (Ab), IgM antibody and IgG antibody against SARS-CoV-2 in All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Antibody measurement'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Antibody measurement'}, {'text': 'The copyright holder for this preprint this version posted March 27, 2020. . https://doi.org/10.1101/2020.03.23.20041707 doi: medRxiv preprint performed, 10 μ L of sample was pipetted onto the sample receiving zone followed by adding two drops of sample buffer. For LFIA-IgG, sample was 1000-fold diluted with sample buffer, then 80 μ L of dilution was added onto the sample receiving area.', 'cite_spans': [], 'ref_spans': [], 'section': 'Antibody measurement'}, {'text': 'Fifteen minutes later after sample was added, the result of LFIA was observed by eyes and recorded.', 'cite_spans': [], 'ref_spans': [], 'section': 'Antibody measurement'}, {'text': 'Incubation period was defined as interval between earliest date of SARS-Cov-2 exposure and earliest date of symptom onset. Non-normal distribution continue data was described as median with interquartile range (IQR) and compared by Wilcoxon test. Categorical data was summarized as counts and percentages. The 95% confidence intervals of sensitivity and specificity were estimated by binomial exact test. Categorical data was compared using ', 'cite_spans': [], 'ref_spans': [], 'section': 'Statistical analysis'}, {'text': 'No patients were involved in setting the research question or the outcome measures, nor were they involved in the design and implementation of the study.', 'cite_spans': [], 'ref_spans': [], 'section': 'Patient and public involvement'}, {'text': 'Dissemination to these groups is not possible/applicable.', 'cite_spans': [], 'ref_spans': [], 'section': 'Dissemination declaration'}, {'text': 'All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Results'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Results'}, {'text': 'The copyright holder for this preprint this version posted March 27, 2020. .', 'cite_spans': [], 'ref_spans': [], 'section': 'Results'}, {'text': 'Among totally 81 cases of COVID-19 patients admitted in the hospital (before Feb 9, 2020), 80 (99%) patients were enrolled in the study ( Table 1 ). The median age of patients was 55 years (IQR, 45-64 years) and 38.7% were females. The critical patients were significantly older than non-critical patients (p<0.001). The time of SARS-CoV-2 exposure before onset in 45 patients (15 were critical cases) had been determined according to the unambiguous close contact with a confirmed Covid-19 patients through the epidemiological inspection when admission to hospital. The incubation period was ranged from 0 to 23 days with a median of 5 days (IQR, 2 -10 days). By February 15, a total of 32 patients (40%, all were non-critical cases) were recovered and discharged from hospital and none died.', 'cite_spans': [], 'ref_spans': [{'start': 138, 'end': 145, 'text': 'Table 1', 'ref_id': None}], 'section': 'Characteristics of enrolled COVID-19 patients'}, {'text': 'A total of 80 Covid-19 patients and 100 to 300 healthy people were tested for antibodies against SARS-CoV-2 using different immunoassays. The seroconversion rate for Ab, IgM and IgG in patients was 98.8% (79/80), 93.8% (75/80) and 93.8%', 'cite_spans': [], 'ref_spans': [], 'section': 'The performance of 2019-nCoV antibody detection kits'}, {'text': '(75/80), respectively ( Table 2 ). The last blood sample of the only patient who had not seroconverted was collected on the 7 days post onset (d.p.o). For Ab, IgM and IgG test, the performance of ELISAs seems the best, although the differences are generally not significant. Therefore, the following serological analyses were all based on the results of ELISAs unless specifically noted.', 'cite_spans': [], 'ref_spans': [{'start': 24, 'end': 31, 'text': 'Table 2', 'ref_id': 'TABREF8'}], 'section': 'The performance of 2019-nCoV antibody detection kits'}, {'text': 'The seroconversion was sequential appeared for Ab, IgM and then IgG, with a median All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'The performance of 2019-nCoV antibody detection kits'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'The performance of 2019-nCoV antibody detection kits'}, {'text': 'The copyright holder for this preprint this version posted March 27, 2020. . (which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'The performance of 2019-nCoV antibody detection kits'}, {'text': 'The copyright holder for this preprint this version posted March 27, 2020. . https://doi.org/10.1101/2020.03.23.20041707 doi: medRxiv preprint exposure were assigned into the short incubation period group and the remaining patients were assigned into the long incubation period group (Table 4 ). There was no significant difference on age, viral load in the early stage of illness and the risk of critical status between the groups. However, the median seroconversion time was shorter for the short incubation period group than for the long incubation period group (13d.p.e v.s. 21d.p.e, p<0.001). In contrast, the median seroconversion day post onset was longer for the short incubation period group than the long incubation period', 'cite_spans': [], 'ref_spans': [{'start': 284, 'end': 292, 'text': '(Table 4', 'ref_id': 'TABREF11'}], 'section': 'The performance of 2019-nCoV antibody detection kits'}, {'text': 'The present study showed that near all (98.8%, 79/80) Covid-19 patients converted to be seropositive during the illness course. Seroconversion was first observed on 7d.p.e (2d.p.o). The first detectible antibody is total antibody and followed by IgM and However, it had been documented that some close contactors presented symptoms and causing transmission after de-isolation [18] . It is believed that antibody test might improve the sensitivity to find infections, but the evidence had not been shown before. All rights reserved. No reuse allowed without permission.', 'cite_spans': [{'start': 376, 'end': 380, 'text': '[18]', 'ref_id': None}], 'ref_spans': [], 'section': 'Discussion'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. Recently, Guan et al [4] reported that the median time from onset of symptoms to need mechanical ventilator support is 9.3 days. Therefore, timely diagnosis and admit All rights reserved. No reuse allowed without permission.', 'cite_spans': [{'start': 157, 'end': 160, 'text': '[4]', 'ref_id': 'BIBREF3'}], 'ref_spans': [], 'section': 'Discussion'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. [19] reported that the overall RNA positive rate was lower than 70% in patients on the first week post symptoms onset and fall to 50% on the next week.', 'cite_spans': [{'start': 136, 'end': 140, 'text': '[19]', 'ref_id': None}], 'ref_spans': [], 'section': 'Discussion'}, {'text': \"Many reports indicated that lots of patients were finally diagnosed by RNA testing through day and day's repeat sampling and testing, and many cases that were strongly epidemiologically linked to SARS-CoV-2 exposure and with typical lung radiological findings remained RNA negative.\", 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': '[9] In our study, the RNA positive rate is 100% when admitted in hospital, but it cannot be excluded that some patients missed diagnosis due to their undetectable viral RNA. Another reason for the relatively high RNA positive rate in our study is that we used the deep sputum sample for RNA test, in contrast to the more convenient and popular throat/nasal swabs in many other hospitals. It suggests that the lower respiratory samples such as deep sputum and bronchoalveolar lavage might be more reliable for SARS-CoV-2 RNA detection.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'Similar as the RNA test, the negative antibody finding in the early stage of illness could not exclude the possibility of infection. Our study showed that the seroconversion rate on 7d.p.o reached 64.1% and then increased to over 90% in the next week. The relative long window period of seroconversion indicates that, for searching previously exposed patients or subclinical carriers, the specific serology should not replace RNA detection but could be an important complement.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'Furthermore, the fact that near all patients will convert to antibody positive and the titer increased rapidly underlines the usage of antibody test in confirming or excluding diagnosis of SARS-CoV-2 infection if the convalescent sample were tested. All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. The present data showed that the sensitivity of total antibody detection was higher than that of IgM and IgG (p<0.001) while the specificities are overall comparable when the same testing technic (ELISA, CLMA or LFIA) is used. The detection of total antibody is based on double-antigen sandwich methodology. It can detect any types of antibodies including IgM, IgG and IgA in principle. Besides, it needs the two Fab arms of the same antibody molecular binding to the coated antigen and the enzyme conjugated antigen, which guarantees the specificity of the test and then allows high concentration of antigens be used for coating and second binding to increase the sensitivity of the assay. Therefore, it is not unexpected that the sensitivity of total antibody detection superiors to that of IgM and IgG detection in our study. Usually, IgM is considered as a marker of current or recent infection, while IgG be a marker of post or recent infection. The implications of total antibody are not so straight and then less being used in clinical practice. However, total antibody test outweighs IgM and IgG if the sensitivity is of the top priority, and has been frequently used in blood transfusion products screening such as human immunodeficiency virus and hepatitis C virus. As for current urgently needs of sensitive diagnosis for SARS-CoV-2 infection to contain the spread, Ab test might be a better choice than IgM or IgG, particularly considered the fact that the virus had invaded human society for less than 4 months then the prevalence of antibody induced by post infection is nearly zero.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'In our study, antibody tests based on ELISA, CMIA and LFIA were validated and showed good performance overall. CMIA takes the advantages of automatic-operation, rapid and high-throughput, objective and quantitative, but All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'The copyright holder for this preprint this version posted March 27, 2020. . (which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. Project of Xiamen (3502Z2020YJ01). The funders had no role in study design, data collection, data analysis, data interpretation, or writing of the report. No support from any organisation for the submitted work.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'Conflict of interest：All authors have completed the ICMJE uniform disclosure form at www.icmje.org/coi_disclosure.pdf and have nothing to disclose. (which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'The copyright holder for this preprint this version posted March 27, 2020. (which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. LFIA, lateral flow immunoassay. The combined sensitivities were calculated based on positive findings by any of the assays, and the combined specificities were calculated based on negative findings for all assays.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. Day post onset when the antibody converted, median (IQR) 10(9, 12) 7(6, 10) 0.013 * The viral load was measured using the CT value of real time RT-PCR when detect the first available sputum samples collected after the onset.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'The copyright holder for this preprint this version posted March 27, 2020. . The antibody levels were surrogated expressed using the relative binding signals compared to the cutoff value of each assay (S/CO). Four parameter logistic fitting curves were used to mimic the trends of antibody levels.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'The copyright holder for this preprint this version posted March 27, 2020. . https://doi.org/10.1101/2020.03.23.20041707 doi: medRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}], 'paragraphs': [{'text': 'The copyright holder for this preprint this version posted March 27, 2020. . https://doi.org/10.1101/2020.03.23.20041707 doi: medRxiv preprint exposure were assigned into the short incubation period group and the remaining patients were assigned into the long incubation period group (Table 4 ). There was no significant difference on age, viral load in the early stage of illness and the risk of critical status between the groups. However, the median seroconversion time was shorter for the short incubation period group than for the long incubation period group (13d.p.e v.s. 21d.p.e, p<0.001). In contrast, the median seroconversion day post onset was longer for the short incubation period group than the long incubation period', 'id': '00025'}], 'title': 'Serology characteristics of SARS-CoV-2 infection since the exposure and post symptoms onset', 'sha': 'c509c214b517166ebe3dc963e478a9033517a83b', 'doi': '10.1101/2020.03.23.20041707', 'score': 9.10420036315918}, {'rank': 208, 'abstract': 'Recently, the cholera outbreak in Haiti demonstrated just how unprepared the country is to rapidly isolate an outbreak of this magnitude, and its vulnerability to the COVID-19 pandemic. This communication briefly examines the health system in Haiti and its vulnerability toward the COVID-19 outbreak.', 'body_text': [{'text': 'The coronavirus (COVID-19) pandemic is a major, worldwide public health challenge that originated in Wuhan, China in December 2019. While health officials in China were unable to contain COVID-19 within their country over the course of a few months, it was rapidly transmitted based on human to human contact to more than 105 countries and territories [1]. This is concerning for countries with weak public health systems (such as Haiti and others) and the spread of the virus within these countries is a major concern expressed by the World Health Organization (WHO). Recently, the cholera outbreak in Haiti demonstrated just how unprepared the country is to rapidly deal with an outbreak of this magnitude, and its vulnerability to the COVID-19 outbreak. This communication briefly examines the health system in Haiti and its vulnerability toward the COVID-19 outbreak.', 'cite_spans': [{'start': 353, 'end': 354, 'mention': '1', 'ref_id': 'BIBREF0'}], 'section': 'Introduction', 'ref_spans': []}, {'text': 'In response to the spread of COVID-19, the Haitian government, with the help of WHO and the U.S. Centers for Disease Control and Prevention (CDC), took proactive steps and quickly prepared health professionals to monitor and detect infected travelers at the two international airports. Additionally, Haitian authorities also shut down the border with the Dominican Republic. These efforts were undertaken to prevent the rapid spread of COVID-19 in Haiti by minimizing movement to and from the country and by identifying any possibly infected persons before entry. But, how really prepared is Haiti to deal with another outbreak of a highly contagious disease?', 'cite_spans': [], 'section': 'Coronavirus in Haiti', 'ref_spans': []}, {'text': 'The population in Haiti is about 11 million with 35% of the people residing in the capital, Port-au-Prince [2]. The health care system is problematic. Only 900 health institutions are available to the 11 million inhabitants, among which 38% are public institutions, 42% private, and 20% mixed. Nearly half of these institutions are located in the capital [2, 3]. On average, the public health care spending in Haiti is 13 U.S. Dollars per capita a year, which is below the average (15 U.S. Dollars) for low-income countries. By comparison, neighboring countries such as Dominican Republic and Cuba—with the same population as Haiti, respectively spend 180 and 781 U.S. Dollars per capita a year [4]. Over the years, the Haitian government has systematically reduced its national health budget from 16.6 to 4.4% since 2004. Surprisingly, the Haitian Parliament continue to receive more funding than its Ministry of Public Health and Population (Ministère de la Santé Public et de la Population, MSPP). For the 2017–2018 national budget, the Parliament received 7.2 billion Haitian gourdes (7.2 million U.S. Dollars) compared to the 6.1 billion Haitian gourdes (6.1 million U.S. Dollars) allocated to MSPP [5]. The MSPP reported that 64% of Haiti’s health budget is provided by international assistance through many Non-Government Organizations (NGOs).', 'cite_spans': [{'start': 108, 'end': 109, 'mention': '2', 'ref_id': 'BIBREF1'}, {'start': 356, 'end': 357, 'mention': '2', 'ref_id': 'BIBREF1'}, {'start': 359, 'end': 360, 'mention': '3', 'ref_id': 'BIBREF2'}, {'start': 696, 'end': 697, 'mention': '4', 'ref_id': 'BIBREF3'}, {'start': 1205, 'end': 1206, 'mention': '5', 'ref_id': 'BIBREF4'}], 'section': 'Coronavirus in Haiti', 'ref_spans': []}, {'text': 'With limited medical supplies, infrastructures, and health professionals, Haiti is currently very vulnerable against the COVID-19 virus. The highly pathogenic Coronavirus is highly transmittable, causes fatal pneumonia, and infects lower respiratory airways, resulting in pulmonary failure and death. The virus is transmitted from human to human by contact from respiratory droplets, coughing, and sneezing. As a result, it can live on various surfaces for a relatively long period of time. COVID-19 has an incubation period of 5–14 days. Infected individuals and asymptomatic individuals are the primary sources of transmission. At the time of this writing, the virus has already produced a mortality rate of about 5.6% and affected large numbers of people, causing many deaths [1].', 'cite_spans': [{'start': 780, 'end': 781, 'mention': '1', 'ref_id': 'BIBREF0'}], 'section': 'Coronavirus in Haiti', 'ref_spans': []}, {'text': 'While the virus has spread in a number of countries in the American continent, Haiti is one the latest countries to have had confirmed cases of the virus. On 19 March 2020, Haitian authorities reported the first two cases of the deadly COVID-19 disease in Haiti [6]. Based on previous experiences, including the 7.0 magnitude earthquake in 2010 and the cholera outbreak, the public health system in Haiti is not equipped to battle a pandemic of this magnitude. An immediate plan of action is urgently needed. While the public hospitals in Haiti receive a large volume of patients daily, they are operating with low funding and inadequate medical equipment. They are currently in an unstable position and are not ready to diagnose individuals with COVID-19 as they do not have the available testing to determine if an individual is infected with the virus. To date, of the 11 million inhabitants, only 218 people have been tested for COVID-19 with 20 confirmed positive cases and one reported death. The limited number of tests is a great disadvantage when compared to the neighboring country, Dominican Republic where more than 4000 tests were conducted with 1488 positive cases and 68 reported deaths [1]. Analyses showed failures to rapidly test individuals for the COVID-19 pandemic resulted in major catastrophe. This is the case of the Unites States of America (USA) which had a slow response to the pandemic.', 'cite_spans': [{'start': 263, 'end': 264, 'mention': '6', 'ref_id': 'BIBREF5'}, {'start': 1203, 'end': 1204, 'mention': '1', 'ref_id': 'BIBREF0'}], 'section': 'Coronavirus in Haiti', 'ref_spans': []}, {'text': 'The lack of information and understanding about the virus in Haiti is another obstacle. Many believe that the virus is the result of sin and it is God’s punishment on humans for their sinful ways. Others believe that the world’s elite manufactured the virus in order to kill minorities and annihilate those in poor countries. A selected number of Haitians have the accurate information about the deadly disease and understand that the virus can infect anyone regardless of religious background or ethnicity. As they understand the risks and preventative measures to combat the virus, the majority of the population still does not have the necessary tools and equipment to respond. The challenge is due to the fact that only 36% of the population has access to electricity, nearly 56% has access to clean water, and 28% has access to basic sanitation [7, 8]. Altogether, the Haitian government and public health officials need to continuously engage in circulating information, providing guidance, and clarifying misinformation to the public. This nationwide campaign can be facilitated through press briefings, announcements on radio and television stations, and through text messages using WhatsApp—which is a well-established form of conversation among Haitians living in Haiti and abroad. In doing so, this campaign will consequently help reduce the risk of transmission and infection in Haiti.', 'cite_spans': [{'start': 851, 'end': 852, 'mention': '7', 'ref_id': 'BIBREF6'}, {'start': 854, 'end': 855, 'mention': '8', 'ref_id': 'BIBREF7'}], 'section': 'Public Reaction and Engagement', 'ref_spans': []}], 'paragraphs': [{'text': 'With limited medical supplies, infrastructures, and health professionals, Haiti is currently very vulnerable against the COVID-19 virus. The highly pathogenic Coronavirus is highly transmittable, causes fatal pneumonia, and infects lower respiratory airways, resulting in pulmonary failure and death. The virus is transmitted from human to human by contact from respiratory droplets, coughing, and sneezing. As a result, it can live on various surfaces for a relatively long period of time. COVID-19 has an incubation period of 5–14 days. Infected individuals and asymptomatic individuals are the primary sources of transmission. At the time of this writing, the virus has already produced a mortality rate of about 5.6% and affected large numbers of people, causing many deaths [1].', 'id': '00004'}], 'title': 'Coronavirus (COVID-19) in Haiti: A Call for Action', 'sha': '8bf38a4e9fa1e5b3ecd62ab09b6a7161770dbd60', 'doi': '10.1007/s10900-020-00825-9', 'score': 9.092399597167969}, {'rank': 209, 'paragraphs': [{'text': '(A1). The renewal process has reached its equilibrium status;(A2). The distribution of the incubation period is continuous;(A3). The distribution of the incubation period has a finite first moment;(A4). The incubation period for each case is independent and identically distributed;(A5). The cases included in the analysis were infected at Wuhan and developed their symptoms outside Wuhan.', 'id': '00007'}], 'title': 'Estimation of incubation period distribution of COVID-19 using disease onset forward time: a novel cross-sectional and forward follow-up study', 'sha': 'c61be9fc4562d250e2dec241ef75bd22d61f61d1; 6d3b3f4ab80a61c45f82c61c6c756cfc6ddf4bf2', 'doi': '10.1101/2020.03.06.20032417', 'score': 9.07919979095459}, {'rank': 210, 'paragraphs': [{'text': 'The mean and percentiles of the incubation period can be calculated from the parametric Weibull distribution. The confidence intervals in this study are obtained using bootstrap method with B = 1000 resamples.', 'id': '00017'}], 'title': 'Estimation of incubation period distribution of COVID-19 using disease onset forward time: a novel cross-sectional and forward follow-up study', 'sha': 'c61be9fc4562d250e2dec241ef75bd22d61f61d1; 6d3b3f4ab80a61c45f82c61c6c756cfc6ddf4bf2', 'doi': '10.1101/2020.03.06.20032417', 'score': 9.071999549865723}, {'rank': 211, 'paragraphs': [{'text': 'Among the collected data, the incubation periods of 483 confirmed patients can be estimated. Statistical results are shown in Fig.3 and Tab.1. The estimated incubation periods for the 483 patients roughly follow the logarithm normal distribution, with a long right tail. Specifically, the 483 confirmed patients were with average estimated incubation periodτ D = 7.4 days, the mode was 4 days (50 out of the 483 cases) and the median was 7 days (48 out of the 483 cases). About 55% patients were with incubation periods lower than 7 days, while more than 92% patients were lower than 14 days. About 7.45% patients were overestimated with more than 14 days incubation periods.', 'id': '00016'}], 'title': 'Epidemiological characteristics of 1212 COVID-19 patients in Henan, China', 'sha': '091a8e9a61e19e88caeb039f0e3888d111b20439', 'doi': '10.1101/2020.02.21.20026112', 'score': 9.066699981689453}, {'rank': 212, 'paragraphs': [{'text': 'The copyright holder for this preprint this version posted June 16, 2020. . In predictive investigations of the effects of relaxing physical distancing, we simulate the SEIRtype model displayed in Section 2.1 forwards in time, starting from May 1st and ending on July 1st. We investigate situations in which the amount of physical distancing is modulated on May 17th (the strategy for restarting British Columbia through relaxation of physical distancing recommendations began in the middle of May [28] ) through a forcing of the ODE parameter f 2 , and in which many of the ODE parameters are modulated through grid-searches. We also consider a retrospective analysis of the introduction of physical distancing in British Columbia, assuming that physical distancing is introduced in mid-March (March 18th) by modulation of parameter f from f 0 = 1 to f 1 = 0.4 and f 1 = 0.7, and compare to a baseline model in which f remains at 1. We investigate 25% to 50% confidence intervals around the number of active case in early/mid-April, by considering 100 replicates in the ODE simulations, each with a value of R 0 drawn independently from the prior.', 'id': '00017'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 9.062000274658203}, {'rank': 213, 'abstract': 'The coronavirus disease 2019 (COVID-19) has been found to be caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). However, comprehensive knowledge of COVID-19 remains incomplete and many important features are still unknown. This manuscripts conduct a meta-analysis and a sensitivity study to answer the questions: What is the basic reproduction number? How long is the incubation time of the disease on average? What portion of infections are asymptomatic? And ultimately, what is the case fatality rate? Our studies estimate the basic reproduction number to be 3.15 with the 95% interval (2.41, 3.90), the average incubation time to be 5.08 days with the 95% confidence interval (4.77, 5.39) (in day), the asymptomatic infection rate to be 46% with the 95% confidence interval (18.48%, 73.60%), and the case fatality rate to be 2.72% with 95% confidence interval (1.29%, 4.16%) where asymptomatic infections are accounted for.', 'body_text': [{'text': 'Since the first case of the coronavirus disease 2019 (COVID-19) was found in Wuhan, P. R. China in December 2019, the disease has rapidly spread in the city of Wuhan, then to Hubei Province, China, and subsequently, across the world [1]. On March 11, 2020 , the World Health Organization (WHO) declared COVID-19 to be a pandemic. The swift spread of the virus is largely attributed to its stealth transmissions for which infected patients may be asymptomatic or exhibit only flu-like symptoms in the early stage. Undetected transmissions present a remarkable challenge for the containment of the virus and pose an appalling threat to the public health. To understand the drastically negative impacts of COVID-19 on the public health, it is urgent to investigate key features pertinent to the disease: How severe is the transmission? How long is the incubation time of the disease on average? How many infections are asymptomatic? And ultimately, what is the case fatality rate?', 'cite_spans': [{'start': 241, 'end': 255, 'text': 'March 11, 2020', 'ref_id': None}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'To evaluate the severity of the virus spread, it is useful to estimate the basic reproduction number (denoted R 0 ), defined as the average number of cases generated by an infected individual in a population where everyone is susceptible to infection. If the basic reproduction number R 0 is larger than 1, the outbreak is regarded as self-sustaining unless control measures are implemented to mitigate the transmission [5] . Defined as the time from the moment of exposure to the virus until signs and symptoms of COVID-19 appear, the incubation time of a COVID-19 infected patient provides a useful measure for the disease development. Knowing the average incubation time of the COVID-19 patients is important for disease surveillance.', 'cite_spans': [{'start': 420, 'end': 423, 'text': '[5]', 'ref_id': 'BIBREF3'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'To determine how deadly the COVID-19 is, it is fundamental to evaluate the case fatality rate which is calculated as the ratio of the number of deaths from COVID-19 to the number of infected cases.', 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'Since the outbreak of the disease, a large body of research on COVID-19 has been done and many articles have been published in scientific journals or shared on platforms such as bioRxir and medRxir. Simulations of the epidemic have been published under various assumptions to delineate hidden transmissions of the virus [5] . While estimates of those important quantities have been reported in the literature, those results are quite different and vary considerably from study to study. There has been a lack of consensus of those estimates because of serious concerns on the heterogeneity among the studies. Different studies have been carried out on different patients under different conditions, and different authors may make different model assumptions. Interpreting the available findings must be coupled with the associated features of the studies.', 'cite_spans': [{'start': 320, 'end': 323, 'text': '[5]', 'ref_id': 'BIBREF3'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'Moreover, COVID-19 data contain substantial errors in that the number of confirmed cases is considerably under-reported, which is attributed to two primary reasons. Insufficient test kits do not allow every potential patient with COVID-19-like symptoms to be tested, and there has been a good portion of asymptomatic COVID-19 carriers who would never be tested and counted as confirmed cases. It is useful to understand the asymptomatic infection rate, defined as the ratio of the number of asymptomatic infections to the number of all infected cases.', 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'To address these issues, we carry out a meta-analysis to synthesize the reported estimates of the basic reproduction number, the average incubation time, and the case fatality rate as well as the asymptomatic rate in a rigorous way by factoring out the variabilities associated with the relevant studies. To accommodate the effects of missing asymptomatic infections on calculating the case fatality rate, we further perform a sensitivity analysis for the estimation of the case fatality rate. Our study provides a comprehensive evaluation of key measures of COVID-19 by taking into account of the heterogeneity and measurement error effects which are intrinsically associated with COVID-19 data. Our results offer sensible estimates of the clinical features of COVID-19 to enhance the understanding of the disease.', 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'The third author (Y.Z.) conducted a literature screening for the articles published between January 24, 2020 and March 31, 2020 by using online databases, including PubMed, Web of Science, Google Scholar and the official websites of core scientific and biomedical journals including Science, Nature, The Lancet, The New England Journal of Medicine, and The Journal of American Medical Association, as well as some preprint platforms such as BioRxiv and MedRxir, with search terms specified as COVID-19, SARS-CoV-2, 2019-nCov, and novel coronavirus. Forty-three articles were found with the theme on the basic reproduction number, the incubation period, the percentage of asymptomatic cases, and the case fatality rate. Among those articles, 20 articles, described in Table 1 , were identified by the first author (W.H.) to be included in the analysis, together with [24, 25] which were found on April 2. The inclusion criteria are the availability of both point estimates and 95% confidence intervals (or equivalently, standard deviations) for the basic transmission number, the average incubation time, the asymptomatic rate, or the case fatality rate. Table 1 presents the summary information of the selected articles together with the descriptions of the data used in those articles. We extract the results for the basic reproduction number from [3, 4, 5, 6, 7, 9, 10] and the results for the average incubation time from [3, 4, 8, 12, 15] . The results from [20, 21, 22, 23, 24, 25] are extracted for estimation of the asymptomatic infection rate. The estimates for the case fatality rate together with their 95% confidence intervals are taken from [10, 11, 13, 14, 16, 17, 18] . In the articles [4, 6, 8] , the reported 95% confidence intervals were asymmetric which we suspect were caused by employing a transformation (such as the exponential transformation) to the initial confidence intervals for the reparameterized effective size; for example, some authors may apply the logarithm to reparameterize the basic reproduction number or the average incubation time before performing the analysis. Using the inverse transformation, we convert the reported asymmetric confidence intervals and work out the associated standard deviations which are used in determining the weights for the meta-analysis.', 'cite_spans': [{'start': 866, 'end': 870, 'text': '[24,', 'ref_id': 'BIBREF22'}, {'start': 871, 'end': 874, 'text': '25]', 'ref_id': 'BIBREF23'}, {'start': 1349, 'end': 1352, 'text': '[3,', 'ref_id': 'BIBREF1'}, {'start': 1353, 'end': 1355, 'text': '4,', 'ref_id': 'BIBREF2'}, {'start': 1356, 'end': 1358, 'text': '5,', 'ref_id': 'BIBREF3'}, {'start': 1359, 'end': 1361, 'text': '6,', 'ref_id': 'BIBREF4'}, {'start': 1362, 'end': 1364, 'text': '7,', 'ref_id': 'BIBREF5'}, {'start': 1365, 'end': 1367, 'text': '9,', 'ref_id': 'BIBREF7'}, {'start': 1368, 'end': 1371, 'text': '10]', 'ref_id': 'BIBREF8'}, {'start': 1425, 'end': 1428, 'text': '[3,', 'ref_id': 'BIBREF1'}, {'start': 1429, 'end': 1431, 'text': '4,', 'ref_id': 'BIBREF2'}, {'start': 1432, 'end': 1434, 'text': '8,', 'ref_id': 'BIBREF6'}, {'start': 1435, 'end': 1438, 'text': '12,', 'ref_id': 'BIBREF10'}, {'start': 1439, 'end': 1442, 'text': '15]', 'ref_id': 'BIBREF13'}, {'start': 1462, 'end': 1466, 'text': '[20,', 'ref_id': 'BIBREF18'}, {'start': 1467, 'end': 1470, 'text': '21,', 'ref_id': 'BIBREF19'}, {'start': 1471, 'end': 1474, 'text': '22,', 'ref_id': 'BIBREF20'}, {'start': 1475, 'end': 1478, 'text': '23,', 'ref_id': 'BIBREF21'}, {'start': 1479, 'end': 1482, 'text': '24,', 'ref_id': 'BIBREF22'}, {'start': 1483, 'end': 1486, 'text': '25]', 'ref_id': 'BIBREF23'}, {'start': 1653, 'end': 1657, 'text': '[10,', 'ref_id': 'BIBREF8'}, {'start': 1658, 'end': 1661, 'text': '11,', 'ref_id': 'BIBREF9'}, {'start': 1662, 'end': 1665, 'text': '13,', 'ref_id': 'BIBREF11'}, {'start': 1666, 'end': 1669, 'text': '14,', 'ref_id': 'BIBREF12'}, {'start': 1670, 'end': 1673, 'text': '16,', 'ref_id': 'BIBREF14'}, {'start': 1674, 'end': 1677, 'text': '17,', 'ref_id': 'BIBREF15'}, {'start': 1678, 'end': 1681, 'text': '18]', 'ref_id': 'BIBREF16'}, {'start': 1700, 'end': 1703, 'text': '[4,', 'ref_id': 'BIBREF2'}, {'start': 1704, 'end': 1706, 'text': '6,', 'ref_id': 'BIBREF4'}, {'start': 1707, 'end': 1709, 'text': '8]', 'ref_id': 'BIBREF6'}], 'ref_spans': [{'start': 767, 'end': 774, 'text': 'Table 1', 'ref_id': 'TABREF1'}, {'start': 1154, 'end': 1161, 'text': 'Table 1', 'ref_id': 'TABREF1'}], 'section': 'Search Strategy and Selection Criteria'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'cite_spans': [], 'ref_spans': [], 'section': '3'}, {'text': 'The copyright holder for this preprint this version posted May 5, 2020. Imai et al. [5] up to Jan 18, 2020 4000 total cases in Wuhan Read et al. [6] up to Jan. 21, 2020 worldwide Liu et al. [7] up to Feb 7, 2020 Chinawide Backer et al. [8] Jan 20 -Jan 28, 2020 88 Wuhan travellers Shen et al. [9] up to Jan 22, 2020 Chinawide Wu et al. [10] up to Feb 29, 2020 Wuhan, China', 'cite_spans': [{'start': 84, 'end': 87, 'text': '[5]', 'ref_id': 'BIBREF3'}, {'start': 145, 'end': 148, 'text': '[6]', 'ref_id': 'BIBREF4'}, {'start': 190, 'end': 193, 'text': '[7]', 'ref_id': 'BIBREF5'}, {'start': 236, 'end': 239, 'text': '[8]', 'ref_id': 'BIBREF6'}, {'start': 293, 'end': 296, 'text': '[9]', 'ref_id': 'BIBREF7'}, {'start': 336, 'end': 340, 'text': '[10]', 'ref_id': 'BIBREF8'}, {'start': 347, 'end': 359, 'text': 'Feb 29, 2020', 'ref_id': None}], 'ref_spans': [], 'section': '3'}, {'text': 'Baud et al. [11] up to Mar 1, 2020 worldwide Jiang et al. [12] up to Feb 8, 2020', 'cite_spans': [{'start': 12, 'end': 16, 'text': '[11]', 'ref_id': 'BIBREF9'}, {'start': 58, 'end': 62, 'text': '[12]', 'ref_id': 'BIBREF10'}], 'ref_spans': [], 'section': '3'}, {'text': 'Ruan [13] up to Mar 21, 2020 Chinawide Verity et al. [14] Jan 4 -Feb 24, 2020 outside Hubei province, China', 'cite_spans': [{'start': 5, 'end': 9, 'text': '[13]', 'ref_id': 'BIBREF11'}, {'start': 53, 'end': 57, 'text': '[14]', 'ref_id': 'BIBREF12'}], 'ref_spans': [], 'section': 'confirmed cases in Wuhan'}, {'text': 'Lauer et al. [15] up to Feb 24, 2020 mainly about China Sun et al. [16] a meta analysis with ten studies 50466 total cases in China', 'cite_spans': [{'start': 13, 'end': 17, 'text': '[15]', 'ref_id': 'BIBREF13'}, {'start': 24, 'end': 36, 'text': 'Feb 24, 2020', 'ref_id': None}, {'start': 67, 'end': 71, 'text': '[16]', 'ref_id': 'BIBREF14'}], 'ref_spans': [], 'section': 'confirmed cases in Wuhan'}, {'text': 'Li et al. [17] Dec 2019 -Feb 2020 China Wang et al. [18] up to Feb 27, 2020 worldwide Nishiura et al. [20] up to Feb 6 565 Japanese nationals evacuated from Wuhan Kimball et al. [21] Mar 13-20, 2020 13 long-term care residents in King County, Washington', 'cite_spans': [{'start': 10, 'end': 14, 'text': '[17]', 'ref_id': 'BIBREF15'}, {'start': 52, 'end': 56, 'text': '[18]', 'ref_id': 'BIBREF16'}, {'start': 102, 'end': 106, 'text': '[20]', 'ref_id': 'BIBREF18'}, {'start': 178, 'end': 182, 'text': '[21]', 'ref_id': 'BIBREF19'}], 'ref_spans': [], 'section': 'confirmed cases in Wuhan'}, {'text': 'Song et al. [22] followed up until Mar 6, 2020 retrospective single-centre study in Daofu county, Sichuan', 'cite_spans': [{'start': 12, 'end': 16, 'text': '[22]', 'ref_id': 'BIBREF20'}], 'ref_spans': [], 'section': 'confirmed cases in Wuhan'}, {'text': 'Mizumoto et al. [23] up to Feb 21, 2020 3,711 people on board the Diamond Princess cruise ship Serra [24] April 2, 2020 Northern Italy, 60 volunteer blood donors Day [25] April 1, 2020', 'cite_spans': [{'start': 16, 'end': 20, 'text': '[23]', 'ref_id': 'BIBREF21'}, {'start': 101, 'end': 105, 'text': '[24]', 'ref_id': 'BIBREF22'}, {'start': 166, 'end': 170, 'text': '[25]', 'ref_id': 'BIBREF23'}], 'ref_spans': [], 'section': 'confirmed cases in Wuhan'}, {'text': 'CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'confirmed cases in Wuhan'}, {'text': 'The copyright holder for this preprint this version posted May 5, 2020. .', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'As shown in the top panel of Figures 1-4 , estimates of the basic reproduction number, the mean incubation time, the asymptomatic infection rate, and the case fatality rate are quite different from study to study. To obtain synthetic results, we perform a meta-analysis to aggregate the information from multiple studies with the same estimand (or effect size of interest) yet different features including the differences in the data collection, the sample size, and the conditions. Suppose K studies report an estimate and the associate standard deviation for an effect size of interest. For the ith study with i = 1, ..., K, let Y i denote the effect size of interest and let σ 2 i represent its associated variance estimate. In our analysis here, Y i is taken as the basic reproduction number, the average incubation time, the asymptomatic infection rate, and the case fatality rate, respectively. We calculate a weighted average of the results from those K studies under either the fixed effect model or the random effects model [26] .', 'cite_spans': [{'start': 1033, 'end': 1037, 'text': '[26]', 'ref_id': 'BIBREF24'}], 'ref_spans': [{'start': 29, 'end': 40, 'text': 'Figures 1-4', 'ref_id': 'FIGREF0'}], 'section': 'Method'}, {'text': 'Under the fixed effect model, the meta mean effect size is given by', 'cite_spans': [], 'ref_spans': [], 'section': 'Method'}, {'text': 'and the associated standard deviation is', 'cite_spans': [], 'ref_spans': [], 'section': 'Method'}, {'text': 'where w i = 1/σ 2 i is the weight for the ith study. With the random effects model, the meta mean effect size, denoted Y meta,R , and its standard deviation, denoted sd(Y meta,R ), are determined by the same expression as (1) and (2) except for replacing the weight w i with a new weight', 'cite_spans': [], 'ref_spans': [], 'section': 'Method'}, {'text': 'To determine whether the fixed effect model or the random effects model is suitable for the meta-analysis, we calculate the I 2 index [28] , defined as', 'cite_spans': [{'start': 134, 'end': 138, 'text': '[28]', 'ref_id': 'BIBREF26'}], 'ref_spans': [], 'section': 'Method'}, {'text': 'Consistent with [17] , we take the fixed effect model if I 2 < 50%, and the random effects model otherwise. In displaying the meta-analysis results, we use the R package forestplot [30].', 'cite_spans': [{'start': 16, 'end': 20, 'text': '[17]', 'ref_id': 'BIBREF15'}], 'ref_spans': [], 'section': 'Method'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '5'}, {'text': 'The copyright holder for this preprint this version posted May 5, 2020. . https://doi.org/10.1101/2020.04.28.20083758 doi: medRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The top panel of Figure 1 shows the results for the basic production number reported in the seven studies. The I 2 index for those studies is 97.8%, suggesting that the random effect model should be considered in conducting the meta-analysis. This result agrees with the perception that the basic reproduction number is time-dependent and varies from place to place.', 'cite_spans': [], 'ref_spans': [{'start': 17, 'end': 25, 'text': 'Figure 1', 'ref_id': 'FIGREF0'}], 'section': 'Basic Reproduction Number'}, {'text': 'The bottom panel of Figure 1 includes the meta-analysis results. The meta estimate of the basic reproduction number is 3.15, suggesting that a virus carrier may infect at least three individuals on average if preventive measures such as social distancing or quarantine are not applied to the public. It is noted that except for [3] and [7] , other five studies listed in Figure 1 were based on the data for the earlier period of the outbreak where the lock down of Wuhan city has not been in effect yet. As more studies on the basic reproduction number become available for different places at different time periods, we can apply the same metaanalysis procedure to estimate the basic reproduction number to reflect its changes with the implementation of various measures to curb the virus spread in different regions.', 'cite_spans': [{'start': 328, 'end': 331, 'text': '[3]', 'ref_id': 'BIBREF1'}, {'start': 336, 'end': 339, 'text': '[7]', 'ref_id': 'BIBREF5'}], 'ref_spans': [{'start': 20, 'end': 28, 'text': 'Figure 1', 'ref_id': 'FIGREF0'}, {'start': 371, 'end': 379, 'text': 'Figure 1', 'ref_id': 'FIGREF0'}], 'section': 'Basic Reproduction Number'}, {'text': 'Tian et al. [3] Li et al. [4] Imai et al. [5] Read et al. [6] Liu et al. [7] Shen et al. [9] Wu et al. [10] ', 'cite_spans': [{'start': 12, 'end': 15, 'text': '[3]', 'ref_id': 'BIBREF1'}, {'start': 26, 'end': 29, 'text': '[4]', 'ref_id': 'BIBREF2'}, {'start': 42, 'end': 45, 'text': '[5]', 'ref_id': 'BIBREF3'}, {'start': 58, 'end': 61, 'text': '[6]', 'ref_id': 'BIBREF4'}, {'start': 73, 'end': 76, 'text': '[7]', 'ref_id': 'BIBREF5'}, {'start': 89, 'end': 92, 'text': '[9]', 'ref_id': 'BIBREF7'}, {'start': 103, 'end': 107, 'text': '[10]', 'ref_id': 'BIBREF8'}], 'ref_spans': [], 'section': 'Study'}, {'text': 'To estimate the average of the incubation time for infections, we study the results reported in the five articles summarized in the top panel of Figure 2 . The I 2 index is 28%, showing that the fixed effect model is suitable when conducting the meta-analysis. While the incubation 6 . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [{'start': 145, 'end': 153, 'text': 'Figure 2', 'ref_id': 'FIGREF1'}], 'section': 'Average Incubation Time'}, {'text': 'The copyright holder for this preprint this version posted May 5, 2020. . time differs from patient to patient, varying between 1 and 14 days, as reported in [19] , it is feasible to take their average time to be a fixed quantity.', 'cite_spans': [{'start': 158, 'end': 162, 'text': '[19]', 'ref_id': 'BIBREF17'}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The bottom panel of Figure 2 reports the meta-analysis results for the average incubation time of COVID-19. It shows that the mean incubation time is 5.08 days, with 95% confidence intervals being about 4.77 to 5.39 days. This estimated average incubation time is about 2 days shorter than the mean incubation time of 7 days announced by [19] .', 'cite_spans': [{'start': 338, 'end': 342, 'text': '[19]', 'ref_id': 'BIBREF17'}], 'ref_spans': [{'start': 20, 'end': 28, 'text': 'Figure 2', 'ref_id': 'FIGREF1'}], 'section': '(which was not certified by peer review)'}, {'text': 'Tian et al. [3] Li et al. [4] Backer et al. [8] Jiang et al. [12] Lauer et al. [15] ', 'cite_spans': [{'start': 12, 'end': 15, 'text': '[3]', 'ref_id': 'BIBREF1'}, {'start': 26, 'end': 29, 'text': '[4]', 'ref_id': 'BIBREF2'}, {'start': 44, 'end': 47, 'text': '[8]', 'ref_id': 'BIBREF6'}, {'start': 61, 'end': 65, 'text': '[12]', 'ref_id': 'BIBREF10'}, {'start': 79, 'end': 83, 'text': '[15]', 'ref_id': 'BIBREF13'}], 'ref_spans': [], 'section': 'Study'}, {'text': 'In the top panel of Figure 3 we display the estimates of the asymptomatic infection rate reported by [20, 21, 22, 23, 24, 25] . It is clear that those studies provided very different estimates of the asymptomatic infection rate, varying from 17.9% to 78.3%. Such a heterogeneity of the studies is confirmed by the I 2 index which is 98%. Thus, we take the random effects model when conducting the meta-analysis. We report the results at the bottom panel of Figure 3 . Our analysis suggests that the combined asymptomatic infection rate is 46% with the 95% confidence interval ranging from 18.4% to 73.6%.', 'cite_spans': [{'start': 101, 'end': 105, 'text': '[20,', 'ref_id': 'BIBREF18'}, {'start': 106, 'end': 109, 'text': '21,', 'ref_id': 'BIBREF19'}, {'start': 110, 'end': 113, 'text': '22,', 'ref_id': 'BIBREF20'}, {'start': 114, 'end': 117, 'text': '23,', 'ref_id': 'BIBREF21'}, {'start': 118, 'end': 121, 'text': '24,', 'ref_id': 'BIBREF22'}, {'start': 122, 'end': 125, 'text': '25]', 'ref_id': 'BIBREF23'}], 'ref_spans': [{'start': 20, 'end': 28, 'text': 'Figure 3', 'ref_id': 'FIGREF3'}, {'start': 457, 'end': 465, 'text': 'Figure 3', 'ref_id': 'FIGREF3'}], 'section': 'Asymptomatic Infection Rate'}, {'text': 'Finally, we are interested in estimating the case fatality rate which measures how deadly COVID-19 is for the infected people. The meta-analysis results derived from seven studies available in the literature, shown in the top panel of Figure 4 , are reported at the bottom 7 . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [{'start': 235, 'end': 243, 'text': 'Figure 4', 'ref_id': 'FIGREF4'}], 'section': 'Case Fatality Rate'}, {'text': 'The copyright holder for this preprint this version posted May 5, 2020. . https://doi.org/10.1101/2020.04. 28.20083758 doi: medRxiv preprint Study Nishiura et al. [20] Kimball et al. [21] Song et al. [22] Mizunoto et al. [23] Serra [24] Day [25] Summary (Random) Figure 4 , where we assume the random effect models because the I 2 index is 99.5%. The estimated case fatality rate is 3.34%, slightly smaller than 3.4%, the estimate reported on March 3, 2020 by the WHO [2] . The 95% confidence interval suggests that the average case fatality rate can be as small as 2.18% and as large as 4.49%.', 'cite_spans': [{'start': 107, 'end': 118, 'text': '28.20083758', 'ref_id': None}, {'start': 163, 'end': 167, 'text': '[20]', 'ref_id': 'BIBREF18'}, {'start': 183, 'end': 187, 'text': '[21]', 'ref_id': 'BIBREF19'}, {'start': 200, 'end': 204, 'text': '[22]', 'ref_id': 'BIBREF20'}, {'start': 221, 'end': 225, 'text': '[23]', 'ref_id': 'BIBREF21'}, {'start': 232, 'end': 236, 'text': '[24]', 'ref_id': 'BIBREF22'}, {'start': 241, 'end': 245, 'text': '[25]', 'ref_id': 'BIBREF23'}, {'start': 468, 'end': 471, 'text': '[2]', 'ref_id': 'BIBREF0'}], 'ref_spans': [{'start': 263, 'end': 271, 'text': 'Figure 4', 'ref_id': 'FIGREF4'}], 'section': '(which was not certified by peer review)'}, {'text': 'We comment that the true average case fatality rate is expected to be smaller than the estimate here, because the reported estimates of the case fatality rate in the literature were merely calculated as the ratio of the number of deaths from COVID-19 to the number of reported confirmed infected cases, where the number of reported confirmed infected cases is typically under-reported due to limited testing capacity and the exclusion of asymptomatic infections.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'To better understand what the true case fatality rate may be, we further conduct two sensitivity studies. In the first study, we repeat the meta-analysis of the case fatality rate in Section 3.5 by further including the results calculated from the data of the Princess Diamond cruise [29] . This analysis is driven by the consideration that the case fatality rate derived from the cohort of the cruise passengers is highly likely to be accurate, because the number of confirmed cases from the cruise is very likely to be close to the true number of infections. The bottom of Figure 5 reports the meta-analysis results obtained from the random effects model. With the inclusion of the results for the data of the Princess Diamond cruise, the 8 . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 284, 'end': 288, 'text': '[29]', 'ref_id': 'BIBREF27'}], 'ref_spans': [{'start': 575, 'end': 583, 'text': 'Figure 5', 'ref_id': 'FIGREF5'}], 'section': 'Sensitivity Analysis'}, {'text': 'The copyright holder for this preprint this version posted May 5, 2020. . https://doi.org/10.1101/2020.04. 28.20083758 doi: medRxiv preprint', 'cite_spans': [{'start': 107, 'end': 118, 'text': '28.20083758', 'ref_id': None}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'Baud et al. [11] Ruan [13] Verity et al. [14] Wu et al. [10] Sun et al. [16] Li et al. [17] Wang et al. [18] Summary (Random) Effect Size The case fatality rate is estimated as 2.72% with the 95% confidence interval (1.29%, 4.16%). In our second sensitivity analysis, we revise the results in Figure 4 by incorporating the information of asymptomatic cases. To see the adjustment, we let D represent the number of deaths caused from COVID-19. Let C R denote the number of reported infected cases of COVID-19, let C A stand for the number of the SARS-Cov-2 carriers who are asymptomatic, and let C be the total number of infected cases with the virus. Let r A = C A /C be the ratio of asymptomatic infections to the true number of infections.', 'cite_spans': [{'start': 12, 'end': 16, 'text': '[11]', 'ref_id': 'BIBREF9'}, {'start': 22, 'end': 26, 'text': '[13]', 'ref_id': 'BIBREF11'}, {'start': 41, 'end': 45, 'text': '[14]', 'ref_id': 'BIBREF12'}, {'start': 56, 'end': 60, 'text': '[10]', 'ref_id': 'BIBREF8'}, {'start': 72, 'end': 76, 'text': '[16]', 'ref_id': 'BIBREF14'}, {'start': 87, 'end': 91, 'text': '[17]', 'ref_id': 'BIBREF15'}, {'start': 104, 'end': 108, 'text': '[18]', 'ref_id': 'BIBREF16'}], 'ref_spans': [{'start': 293, 'end': 301, 'text': 'Figure 4', 'ref_id': 'FIGREF4'}], 'section': 'Study'}, {'text': 'Let p R = D/C R be the reported case fatality rate and let p T = D/C be the true case fatality rate. If we assume that C = C R + C A , then the reported case fatality rate and the true case fatality differ by the factor 1 − r A :', 'cite_spans': [], 'ref_spans': [], 'section': 'Study'}, {'text': 'Estimates of the case fatality rate that have been reported in the current literature are merely directed to p R rather than p T . To sensibly estimate the true case fatality, we use (3) to adjust the reported results of the seven studies listed at the top panel of Figure 4 . Specifically, we may multiply the factor 1 − r A with an estimate for the reported rate p R as well as its standard deviation for each study and then run a meta-analysis. However, the exact value of the asymptomatic infection rate is unavailable, and we only have its estimates from various studies displayed at the top panel of Figure 3 . To assess how the uncertainty of not knowing the true value of r A , we use two ways to set a value for r A to modify the reported fatality rates for the studies listed at 9 . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [{'start': 266, 'end': 274, 'text': 'Figure 4', 'ref_id': 'FIGREF4'}, {'start': 606, 'end': 614, 'text': 'Figure 3', 'ref_id': 'FIGREF3'}], 'section': 'Study'}, {'text': 'The copyright holder for this preprint this version posted May 5, 2020. . https://doi.org/10.1101/2020.04. 28.20083758 doi: medRxiv preprint Study Baud et al.[11] Ruan [13] Verity et al. [14] Wu et al. [10] Sun et al. [16] Li et al. [17] Wang et al. [18] Diamond Cruise [29] Summary (Random) Effect Size Figure 4 for running a new meta-analysis. First, taking r A as one of seven reported estimates listed at the top panel of Figure 3 , we modify the reported results provided by each study listed at the top panel of Figure 4 using (3), and report the meta analysis results at the top panels of Figure 6 . In the second analysis, we take r A as the synthesized estimate reported in Figure 3 , i.e, r A is set as 46%, and then run the meta-analysis for these adjusted case fatality rates under the random effects model. We report the results at the bottom panel of Figure 6 , which shows that the estimate of the case fatality rate is 1.8% with the 95% confidence interval ranging from 1.18% to 2.43%.', 'cite_spans': [{'start': 107, 'end': 118, 'text': '28.20083758', 'ref_id': None}, {'start': 147, 'end': 162, 'text': 'Baud et al.[11]', 'ref_id': None}, {'start': 168, 'end': 172, 'text': '[13]', 'ref_id': 'BIBREF11'}, {'start': 187, 'end': 191, 'text': '[14]', 'ref_id': 'BIBREF12'}, {'start': 202, 'end': 206, 'text': '[10]', 'ref_id': 'BIBREF8'}, {'start': 218, 'end': 222, 'text': '[16]', 'ref_id': 'BIBREF14'}, {'start': 233, 'end': 237, 'text': '[17]', 'ref_id': 'BIBREF15'}, {'start': 250, 'end': 254, 'text': '[18]', 'ref_id': 'BIBREF16'}, {'start': 270, 'end': 274, 'text': '[29]', 'ref_id': 'BIBREF27'}], 'ref_spans': [{'start': 304, 'end': 312, 'text': 'Figure 4', 'ref_id': 'FIGREF4'}, {'start': 426, 'end': 434, 'text': 'Figure 3', 'ref_id': 'FIGREF3'}, {'start': 518, 'end': 526, 'text': 'Figure 4', 'ref_id': 'FIGREF4'}, {'start': 596, 'end': 604, 'text': 'Figure 6', 'ref_id': 'FIGREF6'}, {'start': 683, 'end': 691, 'text': 'Figure 3', 'ref_id': 'FIGREF3'}, {'start': 865, 'end': 873, 'text': 'Figure 6', 'ref_id': 'FIGREF6'}], 'section': '(which was not certified by peer review)'}, {'text': 'We carry out a meta-analysis and sensitivity study for estimating the basic reproduction number, the average incubation time, the asymptomatic infection rate, and the case fatality rate for COVID-19. Examining the published results between January 24, 2020 and March 31, 2020, our study aggregates different results reported in the literature and provides synthetic estimates by addressing the heterogeneity present in different studies. Our study shows that the basic reproduction number is estimated to be 3.15 with the 95% confidence interval (2.41, 3.90) and the average incubation time is 5.08 days with the 95% confidence interval ranging from 4.77 days to 5.39 days. The asymptomatic infection rate is estimated to be 46% with the 95% confidence interval (18.48%, 73.60%). While multiple studies reported estimates of the case fatality rate, those estimates are typically higher than the true case fatality rate under the same conditions, which is attributed to the fact that a good 10 . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusions and Discussion'}, {'text': 'The copyright holder for this preprint this version posted May 5, 2020. . https://doi.org/10.1101/2020.04. 28.20083758 doi: medRxiv preprint', 'cite_spans': [{'start': 107, 'end': 118, 'text': '28.20083758', 'ref_id': None}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'Nishiura et al. [20] Kimball et al. [21] Song et al. [22] Mizunoto et al. [23] Serra [24] Day [25] Meta−Random portion of asymptomatic infections are not counted when estimating the case fatality rate. Our sensitivity study addresses this important issue and makes an adjustment to provide a sensible estimate of the case fatality rate. Compared to the estimated 3.34% case fatality rate obtained from the meta-analysis, our sensitivity study estimates the case fatality rate to be 1.8% with 95% confidence interval (1.18%, 2.43%) where asymptomatic infections are accounted for. Our studies reveal sensible estimates for the important quantities of COVID-19 by accommodating discrepancy effects associated with different studies such as the variability of the data collected from different populations at different time periods. With the evolution of the pandemic, the basic production number can greatly reduce as a result of the implementation of active measures to mitigate the virus spread. The estimation of the case fatality rate may be closer to the true case fatality rate because of the increase of the test capacity; more infected cases may be detected so the reported number of infections would be closer to the true number of COVID-19 carriers. Our results are useful in enhancing the knowledge of COVID-19. Though we focus on evaluating the basic reproduction number, the average incubation time, the asymptomatic infection rate, and the case fatality rate, other features, such as the time from symptom onset to hospitalization or to death and the morbidity rate of the disease, are also important and they are worth being estimated in a sensible way.', 'cite_spans': [{'start': 16, 'end': 20, 'text': '[20]', 'ref_id': 'BIBREF18'}, {'start': 36, 'end': 40, 'text': '[21]', 'ref_id': 'BIBREF19'}, {'start': 53, 'end': 57, 'text': '[22]', 'ref_id': 'BIBREF20'}, {'start': 74, 'end': 78, 'text': '[23]', 'ref_id': 'BIBREF21'}, {'start': 85, 'end': 89, 'text': '[24]', 'ref_id': 'BIBREF22'}, {'start': 94, 'end': 98, 'text': '[25]', 'ref_id': 'BIBREF23'}], 'ref_spans': [], 'section': 'Asym. Study'}, {'text': 'The present investigations have limitations. Not all published results for the four measures are included in our study; we do not include those manuscripts which reported merely a point estimate without the associated standard deviation or a 95% confidence intervals, because they do not allow us to decide a proper weight for the inclusion of the result. While reporting a single estimate of the average incubation time and the case fatality rate gives us 11 . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'cite_spans': [], 'ref_spans': [], 'section': 'Effect Size'}, {'text': \"The copyright holder for this preprint this version posted May 5, 2020. . https://doi.org/10.1101/2020.04. 28.20083758 doi: medRxiv preprint an easy way to assess the impact of COVID-19, such measures marginalize the effects from the associated factors such as the disease severity, the patient's medical conditions, and age. With more studies available for categorizing the case fatality rate or the incubation time, it is useful to apply the meta-analysis to estimate those measures by stratifying the population based on the demographic and clinical characteristics. When data at the individual level are available, better estimates of key features for COVID-19 can be obtained and the pandemic trend can be more reasonably projected using statistical regression models.\", 'cite_spans': [{'start': 107, 'end': 118, 'text': '28.20083758', 'ref_id': None}], 'ref_spans': [], 'section': 'Effect Size'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'cite_spans': [], 'ref_spans': [], 'section': 'Effect Size'}, {'text': 'The copyright holder for this preprint this version posted May 5, 2020. . https://doi.org/10.1101/2020.04. 28.20083758 doi: medRxiv preprint ', 'cite_spans': [{'start': 107, 'end': 140, 'text': '28.20083758 doi: medRxiv preprint', 'ref_id': None}], 'ref_spans': [], 'section': 'Effect Size'}], 'paragraphs': [{'text': 'The bottom panel of Figure 1 includes the meta-analysis results. The meta estimate of the basic reproduction number is 3.15, suggesting that a virus carrier may infect at least three individuals on average if preventive measures such as social distancing or quarantine are not applied to the public. It is noted that except for [3] and [7] , other five studies listed in Figure 1 were based on the data for the earlier period of the outbreak where the lock down of Wuhan city has not been in effect yet. As more studies on the basic reproduction number become available for different places at different time periods, we can apply the same metaanalysis procedure to estimate the basic reproduction number to reflect its changes with the implementation of various measures to curb the virus spread in different regions.', 'id': '00027'}], 'title': 'Estimation of the basic reproduction number, average incubation time, asymptomatic infection rate, and case fatality rate forCOVID-19: Meta-analysis and sensitivity analysis', 'sha': '4bc132d5407496a57dabc2d8db6e46536f0a57f3', 'doi': '10.1101/2020.04.28.20083758', 'score': 9.061699867248535}, {'rank': 214, 'abstract': 'Abstract Aims We aimed to briefly review the general characteristics of the novel coronavirus (SARS-CoV-2) and provide a better understanding of the coronavirus disease (COVID-19) in people with diabetes, and its management. Methods We searched for articles in PubMed and Google Scholar databases till 02 April 2020, with the following keywords: “SARS-CoV-2”, “COVID-19”, “infection”, “pathogenesis”, “incubation period”, “transmission”, “clinical features”, “diagnosis”, “treatment”, “diabetes”, with interposition of the Boolean operator “AND”. Results The clinical spectrum of COVID-19 is heterogeneous, ranging from mild flu-like symptoms to acute respiratory distress syndrome, multiple organ failure and death. Older age, diabetes and other comorbidities are reported as significant predictors of morbidity and mortality. Chronic inflammation, increased coagulation activity, immune response impairment, and potential direct pancreatic damage by SARS-CoV-2 might be among the underlying mechanisms of the association between diabetes and COVID-19. No conclusive evidence exists to support the discontinuation of angiotensin-converting enzyme inhibitors (ACEI) or angiotensin receptor blockers because of COVID-19 in people with diabetes. Caution should be taken to potential hypoglycemic events with the use of chloroquine in these subjects. Patient tailored therapeutic strategies, rigorous glucose monitoring and careful consideration of drug interactions might reduce adverse outcomes. Conclusions Suggestions are made on the possible pathological mechanisms of the relationship between diabetes and COVID-19, and its management. No definite conclusions can be made based on current limited evidence. Further research regarding this relationship and its clinical management is warranted.', 'body_text': [{'text': 'Coronaviruses are enveloped, positive single-stranded RNA viruses widely distributed in humans and animals worldwide [1]. Although most human coronavirus infections are mild, major outbreaks of two betacoronaviruses, severe acute respiratory syndrome coronavirus (SARS-CoV) in 2002-2003 and Middle East respiratory syndrome coronavirus (MERS-CoV) in 2012, have caused deadly pneumonia, with mortality rates of 10% for SARS-CoV and 36% for MERS-CoV [2].', 'cite_spans': [{'start': 117, 'end': 120, 'mention': '[1]', 'ref_id': 'BIBREF0'}, {'start': 448, 'end': 451, 'mention': '[2]', 'ref_id': 'BIBREF1'}], 'section': 'Introduction', 'ref_spans': []}, {'text': 'In December 2019, clusters of pneumonia cases of unknown etiology emerged in Wuhan, Hubei Province, China. Deep sequencing analysis from lower respiratory tract samples indicated a novel coronavirus as the causative agent, which was named Severe Acute Respiratory Syndrome-Coronavirus-2 (SARS-CoV-2), and the disease it causes called COVID-19 [1], [3]. Although SARS-CoV-2 has shown phylogenetic and clinical similarities with SARS-CoV, the novel coronavirus appears to have a higher transmissibility and lower case fatality rates [4]. On 30 January 2020, the World Health Organization (WHO) declared the COVID-19 outbreak a Public Health Emergency of International Concern, and on March 11, the epidemic was upgraded to pandemic [5]. As of today (02.04.2020), 827,419 confirmed cases are officially reported in more than 200 countries or territories with 40,777 deaths [6].', 'cite_spans': [{'start': 343, 'end': 346, 'mention': '[1]', 'ref_id': 'BIBREF0'}, {'start': 348, 'end': 351, 'mention': '[3]', 'ref_id': 'BIBREF2'}, {'start': 531, 'end': 534, 'mention': '[4]', 'ref_id': 'BIBREF3'}, {'start': 730, 'end': 733, 'mention': '[5]', 'ref_id': 'BIBREF4'}, {'start': 870, 'end': 873, 'mention': '[6]', 'ref_id': 'BIBREF5'}], 'section': 'Introduction', 'ref_spans': []}, {'text': 'We conducted a scoping review to provide a brief summary of the general characteristics of COVID-19, as well as a more detailed description and critical assessment of the association between this new infectious disease and diabetes. We hope this review can provide meaningful information for future research and ultimately contribute to better clinical management of patients with COVID-19 and diabetes.', 'cite_spans': [], 'section': 'Introduction', 'ref_spans': []}, {'text': 'A literature review was carried out to search for articles in PubMed and Google Scholar databases till 02 April 2020. The following keywords were used: “SARS-CoV-2”, “COVID-19”, “infection”, “pathogenesis”, “incubation period”, “transmission”, “clinical features”, “diagnosis”, “treatment”, “diabetes”, with interposition of the Boolean operator “AND”. We also retrieved the full text of the relevant cross references from the search results. Furthermore, we accessed the currently available scientific literature and recommendations in the WHO and United States Centers for Disease Control and Prevention (CDC) websites.', 'cite_spans': [], 'section': 'Methods', 'ref_spans': []}, {'text': 'The incubation period is described as the time from infection to illness onset. In a study of 1099 patients from China with laboratory-confirmed symptomatic COVID-19, the median incubation period was four days (interquartile range, 2 to 7) [7]. Another study including 181 confirmed cases reported a median incubation period of approximately 5 days, and that symptoms would develop in 97.5% of infected individuals within 12 days [8]. Based on the incubation period of SARS-CoV and MERS-CoV, as well as observational data, the United States CDC has estimated that symptoms of COVID-19 will usually develop within 2-14 days after exposure. Therefore, fourteen days has been the time applied internationally for monitoring and restricting the movement of healthy individuals (quarantine period) [9].', 'cite_spans': [{'start': 240, 'end': 243, 'mention': '[7]', 'ref_id': 'BIBREF6'}, {'start': 430, 'end': 433, 'mention': '[8]', 'ref_id': 'BIBREF7'}, {'start': 793, 'end': 796, 'mention': '[9]', 'ref_id': 'BIBREF8'}], 'section': 'Incubation Period ::: General Characteristics of COVID-19', 'ref_spans': []}, {'text': 'Most initial COVID-19 patients had a direct contact history with a local Chinese seafood and wildlife market, suggesting a common-source zoonotic exposure as the main mode of transmission [10]. Findings from virus genome sequencing analysis have pointed out that SARS-CoV-2 and bat coronavirus (bat CoV) might share the same ancestor, although bats are not for sale in this seafood market [11]. Later cases were reported among health care workers and others without exposure history of wildlife or visiting Wuhan, which indicated human-to-human transmission [10]. Currently, it is considered that the virus can be mainly transmitted through droplets, direct contact and aerosols. Droplets transmission may occur when respiratory droplets, produced when an infected person coughs or sneezes, are ingested or inhaled by individuals nearby (within about 6 feet). A subject can also get infected by touching a surface or object contaminated with the virus and subsequently touching his/her mouth, nose, or eyes [12]. Additionally, it has been shown experimentally that the virus can remain viable in aerosols for at least 3 hours [13], and can be transmitted in closed environments if inhaled into the lungs [12]. Therefore, airborne transmission is a possibility during aerosol generating procedures, e.g., endotracheal intubation, bronchoscopy, non-invasive positive-pressure ventilation, tracheostomy, cardiopulmonary resuscitation, etc [14]. Although viable virus has been identified in fecal swabs, the fecal-oral route does not appear to be a driver of COVID-19 transmission [11].', 'cite_spans': [{'start': 188, 'end': 192, 'mention': '[10]', 'ref_id': 'BIBREF9'}, {'start': 389, 'end': 393, 'mention': '[11]', 'ref_id': 'BIBREF10'}, {'start': 558, 'end': 562, 'mention': '[10]', 'ref_id': 'BIBREF9'}, {'start': 1007, 'end': 1011, 'mention': '[12]', 'ref_id': 'BIBREF11'}, {'start': 1126, 'end': 1130, 'mention': '[13]', 'ref_id': 'BIBREF12'}, {'start': 1204, 'end': 1208, 'mention': '[12]', 'ref_id': 'BIBREF11'}, {'start': 1436, 'end': 1440, 'mention': '[14]', 'ref_id': 'BIBREF13'}, {'start': 1577, 'end': 1581, 'mention': '[11]', 'ref_id': 'BIBREF10'}], 'section': 'Modes of Transmission ::: General Characteristics of COVID-19', 'ref_spans': []}, {'text': 'It is uncertain how long an individual with COVID-19 remains infectious. The period of infectivity is often assessed indirectly by detection of viral RNA from respiratory specimens. However, viral RNA does not necessarily confirm the presence of infectious virus. Higher viral loads have been detected soon after symptom onset, suggesting that transmission may be more likely to occur in the earlier stages of infection [15]. The viral shedding duration seems to vary according to the disease severity. It has been found that around 90% of patients with milder symptoms had a negative viral RNA test on nasopharyngeal swabs by day 10 post-onset, while the test remained positive for a longer time in all severe cases [16]. On the other hand, it has been reported that the viral load detected in asymptomatic patients was similar to that in symptomatic subjects [15]. Indeed, transmission from asymptomatic carriers or individuals within the incubation period has been described [17]. Nevertheless, the extent to which this occurs remains to be determined.', 'cite_spans': [{'start': 420, 'end': 424, 'mention': '[15]', 'ref_id': 'BIBREF14'}, {'start': 717, 'end': 721, 'mention': '[16]', 'ref_id': 'BIBREF15'}, {'start': 861, 'end': 865, 'mention': '[15]', 'ref_id': 'BIBREF14'}, {'start': 978, 'end': 982, 'mention': '[17]', 'ref_id': 'BIBREF16'}], 'section': 'Period of Infectivity ::: General Characteristics of COVID-19', 'ref_spans': []}, {'text': 'Even though all age groups have been affected by COVID-19, the median age appears to be around 47-59 years, and usually higher among severe cases and non-survivors. No specific gender bias seems to exist for the contamination with the virus, but men tend to have a higher propensity of the cases [7], [18], [19]. Fewer cases have been identified among children and infants. In a large Chinese report including 72,314 patients, only 2% of those infected were younger than 20 years old [20].', 'cite_spans': [{'start': 296, 'end': 299, 'mention': '[7]', 'ref_id': 'BIBREF6'}, {'start': 301, 'end': 305, 'mention': '[18]', 'ref_id': 'BIBREF17'}, {'start': 307, 'end': 311, 'mention': '[19]', 'ref_id': 'BIBREF18'}, {'start': 484, 'end': 488, 'mention': '[20]', 'ref_id': 'BIBREF19'}], 'section': 'Demography and Clinical Characteristics ::: General Characteristics of COVID-19', 'ref_spans': []}, {'text': 'The clinical spectrum of COVID-19 can be very heterogeneous. Most adults and children present mild flu-like symptoms, but some may rapidly develop acute respiratory distress syndrome (ARDS), respiratory failure, arrhythmias, acute cardiac injury, shock, multiple organ failure and death [1], [18]. The most commonly reported symptoms are fever, cough, fatigue, sputum production and shortness of breath. However, headache, upper respiratory symptoms (e.g., sore throat and rhinorrhea) and gastrointestinal symptoms (e.g., nausea and diarrhea) occur less often [1], [7], [18]. Although not described in the initial Chinese studies, smell and taste disorders (e.g., anosmia and dysgeusia) have also been found frequently in patients with COVID-19 in Italy [21].', 'cite_spans': [{'start': 287, 'end': 290, 'mention': '[1]', 'ref_id': 'BIBREF0'}, {'start': 292, 'end': 296, 'mention': '[18]', 'ref_id': 'BIBREF17'}, {'start': 560, 'end': 563, 'mention': '[1]', 'ref_id': 'BIBREF0'}, {'start': 565, 'end': 568, 'mention': '[7]', 'ref_id': 'BIBREF6'}, {'start': 570, 'end': 574, 'mention': '[18]', 'ref_id': 'BIBREF17'}, {'start': 754, 'end': 758, 'mention': '[21]', 'ref_id': 'BIBREF20'}], 'section': 'Demography and Clinical Characteristics ::: General Characteristics of COVID-19', 'ref_spans': []}, {'text': 'In laboratory examination results, most patients have normal or decreased white blood cell counts, particularly lymphocytopenia [7]. However, in severe patients, the neutrophil count, inflammatory markers, D-dimer, blood urea, and creatinine levels are generally higher, with further decreased lymphocyte counts [1]. Chest computed tomography (CT) most commonly shows ground-glass opacifications with or without consolidative abnormalities. They are also more likely to be bilateral, have a peripheral distribution and involve the lower lobes [7], [22]. While some confirmed cases may present normal CT images [11], abnormalities have also been identified prior to the development of symptoms in some patients [22].', 'cite_spans': [{'start': 128, 'end': 131, 'mention': '[7]', 'ref_id': 'BIBREF6'}, {'start': 312, 'end': 315, 'mention': '[1]', 'ref_id': 'BIBREF0'}, {'start': 543, 'end': 546, 'mention': '[7]', 'ref_id': 'BIBREF6'}, {'start': 548, 'end': 552, 'mention': '[22]', 'ref_id': 'BIBREF21'}, {'start': 610, 'end': 614, 'mention': '[11]', 'ref_id': 'BIBREF10'}, {'start': 710, 'end': 714, 'mention': '[22]', 'ref_id': 'BIBREF21'}], 'section': 'Demography and Clinical Characteristics ::: General Characteristics of COVID-19', 'ref_spans': []}, {'text': 'Individuals with new onset fever and/or respiratory tract symptoms (e.g., cough, dyspnea) or patients with severe lower respiratory tract illness, without any clear cause and without any history of close contact with a confirmed COVID-19 patient or have travelled to an area of community transmission.', 'cite_spans': [], 'section': 'Preliminary symptoms for observations ::: Criteria for suspicion and testing ::: Diagnosis ::: General Characteristics of COVID-19', 'ref_spans': []}, {'text': 'Subjects with new onset fever and/or respiratory tract symptoms (e.g., cough, dyspnea) or patients with severe lower respiratory tract illness, who have been in contact with COVID-19 patients or travelled within 14 days to a location where community transmission of SARS-CoV-2 is present. Furthermore, patients older than 60 years of age, as well as those with underlying conditions like diabetes, hypertension, cardiovascular diseases (CVDs), chronic renal disease, cancer and immunocompromising conditions, who develop symptoms of fever, cough and/or dyspnea should be prioritized for testing.', 'cite_spans': [], 'section': 'Tests may be performed depending on resources ::: Criteria for suspicion and testing ::: Diagnosis ::: General Characteristics of COVID-19', 'ref_spans': []}, {'text': 'Persons in close contact (within 2 meters) with a SARS-CoV-2 infected individual for a prolonged period or in direct contact with infectious secretions while not wearing personal protective equipment.', 'cite_spans': [], 'section': 'Must be tested ::: Criteria for suspicion and testing ::: Diagnosis ::: General Characteristics of COVID-19', 'ref_spans': []}, {'text': 'Although the pathophysiological mechanisms are still not understood, it has been observed that most severe and fatal cases with COVID-19 have occurred in the elderly or in patients with underlying comorbidities, particularly CVDs, diabetes mellitus, chronic lung and renal disease, hypertension, and cancer [7], [20], [26], [27].', 'cite_spans': [{'start': 307, 'end': 310, 'mention': '[7]', 'ref_id': 'BIBREF6'}, {'start': 312, 'end': 316, 'mention': '[20]', 'ref_id': 'BIBREF19'}, {'start': 318, 'end': 322, 'mention': '[26]', 'ref_id': 'BIBREF25'}, {'start': 324, 'end': 328, 'mention': '[27]', 'ref_id': 'BIBREF26'}], 'section': 'COVID-19 and Comorbidities ::: General Characteristics of COVID-19', 'ref_spans': []}, {'text': 'One Chinese meta-analysis including 1527 patients showed that the most prevalent cardiovascular metabolic comorbidities with COVID-19 were hypertension (17.1%, 95% CI 9.9 - 24.4%) and cardio-cerebrovascular disease (16.4%, 95% CI 6.6 -26.1%), followed by diabetes (9.7%, 95% CI 6.9 - 12.5%). In this report, patients with diabetes or hypertension had a 2-fold increase in risk of severe disease or requiring intensive care unit (ICU) admission, while those with cardio-cerebrovascular disease had a 3-fold increase [28]. In a sub-set of 355 patients with COVID-19 in Italy who died, the mean number of pre-existing underlying conditions was 2.7, and only 3 subjects did not have any comorbidity [29].', 'cite_spans': [{'start': 515, 'end': 519, 'mention': '[28]', 'ref_id': 'BIBREF27'}, {'start': 695, 'end': 699, 'mention': '[29]', 'ref_id': 'BIBREF28'}], 'section': 'COVID-19 and Comorbidities ::: General Characteristics of COVID-19', 'ref_spans': []}, {'text': 'It has been consistently reported that, in addition to pneumonia, SARS-CoV-2 may cause damage to other organs including the heart, liver and kidneys [1], [18]. Therefore, full attention should be paid to the treatment of the original comorbidities, especially in older patients with already severe underlying conditions.', 'cite_spans': [{'start': 149, 'end': 152, 'mention': '[1]', 'ref_id': 'BIBREF0'}, {'start': 154, 'end': 158, 'mention': '[18]', 'ref_id': 'BIBREF17'}], 'section': 'COVID-19 and Comorbidities ::: General Characteristics of COVID-19', 'ref_spans': []}, {'text': 'Diabetes is one of the leading causes of morbidity and mortality throughout the world. The condition is associated with several macrovascular and microvascular complications, that ultimately impact the overall patient’s survival [30]. A relationship between diabetes and infection has long been clinically recognized [31]. Infections, particularly influenza and pneumonia, are often common and more serious in older people with type 2 diabetes mellitus (T2DM) [32], [33]. Nevertheless, the evidence remains controversial regarding whether diabetes itself indeed increases susceptibility and impacts outcomes from infections, or the cardiovascular and renal comorbidities that are frequently associated with diabetes are the main factors involved [34].', 'cite_spans': [{'start': 229, 'end': 233, 'mention': '[30]', 'ref_id': 'BIBREF29'}, {'start': 317, 'end': 321, 'mention': '[31]', 'ref_id': 'BIBREF30'}, {'start': 460, 'end': 464, 'mention': '[32]', 'ref_id': 'BIBREF31'}, {'start': 466, 'end': 470, 'mention': '[33]', 'ref_id': 'BIBREF32'}, {'start': 746, 'end': 750, 'mention': '[34]', 'ref_id': 'BIBREF33'}], 'section': 'Diabetes and Infection: General Considerations and Potential Mechanisms ::: Association between COVID-19 and Diabetes', 'ref_spans': []}, {'text': 'Diabetes and uncontrolled glycaemia were reported as significant predictors of severity and deaths in patients infected with different viruses, including the 2009 pandemic influenza A (H1N1) [35], SARS-CoV [36] and MERS-CoV [37]. In the current SARS-CoV-2 pandemic, some studies did not find a clear association between diabetes and severe disease [19], [38]. However, other reports from China [7], [20] and Italy [29] showed that older patients with chronic diseases, including diabetes, were at higher risk for severe COVID-19 and mortality.', 'cite_spans': [{'start': 191, 'end': 195, 'mention': '[35]', 'ref_id': 'BIBREF34'}, {'start': 206, 'end': 210, 'mention': '[36]', 'ref_id': 'BIBREF35'}, {'start': 224, 'end': 228, 'mention': '[37]', 'ref_id': 'BIBREF36'}, {'start': 348, 'end': 352, 'mention': '[19]', 'ref_id': 'BIBREF18'}, {'start': 354, 'end': 358, 'mention': '[38]', 'ref_id': 'BIBREF37'}, {'start': 394, 'end': 397, 'mention': '[7]', 'ref_id': 'BIBREF6'}, {'start': 399, 'end': 403, 'mention': '[20]', 'ref_id': 'BIBREF19'}, {'start': 414, 'end': 418, 'mention': '[29]', 'ref_id': 'BIBREF28'}], 'section': 'Diabetes and Infection: General Considerations and Potential Mechanisms ::: Association between COVID-19 and Diabetes', 'ref_spans': []}, {'text': 'Scarce data exist regarding glucose metabolism and development of acute complications of diabetes (e.g., ketoacidosis) in patients with COVID-19. Infection of SARS-CoV-2 in those with diabetes possibly triggers higher stress conditions, with greater release of hyperglycemic hormones, e.g., glucocorticoids and catecholamines, leading to increased blood glucose levels and abnormal glucose variability [39]. On the other hand, a retrospective study from Wuhan reported that around 10% of the patients with T2DM and COVID-19 suffered at least one episode of hypoglycemia (<3.9 mmol/L) [40]. Hypoglycemia has been shown to mobilize pro-inflammatory monocytes and increase platelet reactivity, contributing to a higher cardiovascular mortality in patients with diabetes [41]. Yet it remains largely unknown how exactly the inflammatory and immune response occurs in these patients, as well as whether hyper- or hypoglycemia may alter the SARS-CoV-2 virulence, or the virus itself interferes with insulin secretion or glycemic control. Furthermore, the impact of usual diabetes drug treatment on COVID-19 outcomes, as well as therapeutic approaches for COVID-19 on glucose regulation remains unspecified.', 'cite_spans': [{'start': 402, 'end': 406, 'mention': '[39]', 'ref_id': 'BIBREF38'}, {'start': 584, 'end': 588, 'mention': '[40]', 'ref_id': 'BIBREF39'}, {'start': 767, 'end': 771, 'mention': '[41]', 'ref_id': 'BIBREF40'}], 'section': 'Diabetes and Infection: General Considerations and Potential Mechanisms ::: Association between COVID-19 and Diabetes', 'ref_spans': []}, {'text': 'Diabetes is a chronic inflammatory condition characterized by multiple metabolic and vascular abnormalities that can affect our response to pathogens [34]. Hyperglycemia and insulin resistance promote increased synthesis of glycosylation end products (AGEs) and pro-inflammatory cytokines, oxidative stress, in addition to stimulating the production of adhesion molecules that mediate tissue inflammation [34], [42]. This inflammatory process may compose the underlying mechanism that leads to a higher propensity to infections, with worse outcomes thereof in patients with diabetes [34].', 'cite_spans': [{'start': 150, 'end': 154, 'mention': '[34]', 'ref_id': 'BIBREF33'}, {'start': 405, 'end': 409, 'mention': '[34]', 'ref_id': 'BIBREF33'}, {'start': 411, 'end': 415, 'mention': '[42]', 'ref_id': 'BIBREF41'}, {'start': 583, 'end': 587, 'mention': '[34]', 'ref_id': 'BIBREF33'}], 'section': 'Diabetes and Infection: General Considerations and Potential Mechanisms ::: Association between COVID-19 and Diabetes', 'ref_spans': []}, {'text': 'Several defects in immunity have been associated with hyperglycemia, even though the clinical relevance of some in vitro disturbances are still not fully understood [43]. Poorly controlled diabetes has been linked to inhibited lymphocyte proliferative response to different kinds of stimuli [44], as well as impaired monocyte/macrophage and neutrophil functions [34]. Abnormal delayed type hypersensitivity reaction [43] and complement activation dysfunction [45] have also been described in patients with diabetes. In vitro studies have shown that pulmonary epithelial cells exposure to high glucose concentrations significantly increases influenza virus infection and replication, indicating that hyperglycemia may enhance viral replication in vivo\\n[46]. In animal models, structural lung changes have been related to diabetes, such as augmented vasculature permeability and collapsed alveolar epithelium [47]. On the other hand, patients with diabetes generally present a significant reduction in forced vital capacity (FVC) and forced expiratory volume in one second (FEV1), which is associated with raised plasma glucose levels [48].', 'cite_spans': [{'start': 165, 'end': 169, 'mention': '[43]', 'ref_id': 'BIBREF42'}, {'start': 291, 'end': 295, 'mention': '[44]', 'ref_id': 'BIBREF43'}, {'start': 362, 'end': 366, 'mention': '[34]', 'ref_id': 'BIBREF33'}, {'start': 416, 'end': 420, 'mention': '[43]', 'ref_id': 'BIBREF42'}, {'start': 459, 'end': 463, 'mention': '[45]', 'ref_id': 'BIBREF44'}, {'start': 751, 'end': 755, 'mention': '[46]', 'ref_id': 'BIBREF45'}, {'start': 907, 'end': 911, 'mention': '[47]', 'ref_id': 'BIBREF46'}, {'start': 1133, 'end': 1137, 'mention': '[48]', 'ref_id': 'BIBREF47'}], 'section': 'Diabetes and Infection: General Considerations and Potential Mechanisms ::: Association between COVID-19 and Diabetes', 'ref_spans': []}, {'text': 'Patients with COVID-19 commonly show on admission lymphocytopenia, and to a lesser extent thrombocytopenia and leukopenia, which are more prominent among those with severe disease [7]. Further, elevated levels of pro-inflammatory cytokines, including interleukin-6 (IL-6) and C-reactive protein, as well as increased coagulation activity, marked by higher d-dimer concentrations, were also associated with severity [7], [26]. In T2DM, besides the marked inflammatory process previously discussed, an imbalance between coagulation and fibrinolysis takes place, with increased levels of clotting factors and relative inhibition of the fibrinolytic system. Both insulin resistance and T2DM are associated with endothelial dysfunction, and enhanced platelet aggregation and activation. These abnormalities favor the development of a hypercoagulable pro-thrombotic state [49]. Additionally, atherosclerosis, vascular inflammation and endothelial dysfunction are also part of the pathogenesis of other chronic conditions, e.g., hypertension and CVDs [42]. Animal studies involving SARS-CoV reported that older age was related to defects in T-cell and B-cell function and excess inflammation markers. Thus, T2DM alone or in association with older age, hypertension and/or CVDs might contribute to a deficient control of SARS-CoV-2 replication and more prolonged proinflammatory response, potentially leading to poor outcomes [26].', 'cite_spans': [{'start': 180, 'end': 183, 'mention': '[7]', 'ref_id': 'BIBREF6'}, {'start': 415, 'end': 418, 'mention': '[7]', 'ref_id': 'BIBREF6'}, {'start': 420, 'end': 424, 'mention': '[26]', 'ref_id': 'BIBREF25'}, {'start': 866, 'end': 870, 'mention': '[49]', 'ref_id': 'BIBREF48'}, {'start': 1044, 'end': 1048, 'mention': '[42]', 'ref_id': 'BIBREF41'}, {'start': 1418, 'end': 1422, 'mention': '[26]', 'ref_id': 'BIBREF25'}], 'section': 'Aspects of SARS-CoV-2 Pathogenesis and Potential Implications for Clinical Management of Patients with COVID-19 and Diabetes ::: Association between COVID-19 and Diabetes', 'ref_spans': []}, {'text': 'Viral entry into the host cells is a fundamental component of cross-species transmission, particularly for the coronaviruses (CoVs). Upon exposure of the host to the virus, all CoVs, through a Spike protein, bind to cells that express specific receptors. After binding to the target cells, the host-cell protease cleaves the spike, which allows the virus to enter and replicate [50]. The angiotensin-converting enzyme 2 (ACE2) has been identified as one of the main receptors for both SARS-CoV [51] and SARS-CoV-2 [50]. ACE2 is widely expressed on the respiratory tract, heart, kidneys, intestines, cerebral neurons, endothelium of arteries and veins, immune cells and pancreas [2]. A Chinese study compared 39 SARS-CoV patients without previous diabetes, who did not receive steroid treatment, with 39 matched healthy siblings and showed that 20 of the 39 SARS-CoV patients developed diabetes during hospitalization. Since immunostaining for ACE2 was strong in the pancreatic islets, it was suggested that SARS-CoV might have damaged islets and caused acute insulin dependent diabetes mellitus [52]. Therefore, although further evidence is needed, pancreatic damage may also be present in COVID-19 patients, possibly contributing to worse outcomes in subjects with diabetes.', 'cite_spans': [{'start': 378, 'end': 382, 'mention': '[50]', 'ref_id': 'BIBREF49'}, {'start': 494, 'end': 498, 'mention': '[51]', 'ref_id': 'BIBREF50'}, {'start': 514, 'end': 518, 'mention': '[50]', 'ref_id': 'BIBREF49'}, {'start': 678, 'end': 681, 'mention': '[2]', 'ref_id': 'BIBREF1'}, {'start': 1095, 'end': 1099, 'mention': '[52]', 'ref_id': 'BIBREF51'}], 'section': 'Aspects of SARS-CoV-2 Pathogenesis and Potential Implications for Clinical Management of Patients with COVID-19 and Diabetes ::: Association between COVID-19 and Diabetes', 'ref_spans': []}, {'text': 'Previous studies have reported decreased mortality and endotracheal intubation in patients with viral pneumonia who were in continued use of ACE inhibitors (ACEIs) and angiotensin receptor blockers (ARBs) [53], [54]. These medications are postulated to have significant immunomodulatory effects [55] and reduce pulmonary and systemic inflammatory response by decreasing cytokines [53], [54]. They are commonly used by those with diabetes and hypertension [56], therefore, their impact on the clinical course of COVID-19 has been widely debated. Considering that ACE2 is a functional receptor for SARS-CoV-2 and its levels can be increased by ACE inhibitors and ARBs, it has been argued that these drugs might affect negatively the outcome of COVID-19 patients [57]. On the contrary, some have advocated that ACEIs and ARBs might rather be beneficial [58]. SARS-CoV infection and the virus Spike protein reduce ACE2 expression. Mice injected with SARS-CoV Spike presented worsened acute lung failure, which could be attenuated by blocking the renin-angiotensin pathway [59]. Nevertheless, a retrospective analysis performed on 112 patients with COVID-19 and CVD did not show a significant difference in the proportion of ACEI/ARB medication between survivors and non-survivors [60]. Similar to ACEIs and ARBs, ibuprofen [61] and thiazolidinediones [62] can also result in increased levels of ACE2, thus generating questions regarding the safety of these drugs in patients with COVID-19.', 'cite_spans': [{'start': 205, 'end': 209, 'mention': '[53]', 'ref_id': 'BIBREF52'}, {'start': 211, 'end': 215, 'mention': '[54]', 'ref_id': 'BIBREF53'}, {'start': 295, 'end': 299, 'mention': '[55]', 'ref_id': 'BIBREF54'}, {'start': 380, 'end': 384, 'mention': '[53]', 'ref_id': 'BIBREF52'}, {'start': 386, 'end': 390, 'mention': '[54]', 'ref_id': 'BIBREF53'}, {'start': 455, 'end': 459, 'mention': '[56]', 'ref_id': 'BIBREF55'}, {'start': 760, 'end': 764, 'mention': '[57]', 'ref_id': 'BIBREF56'}, {'start': 850, 'end': 854, 'mention': '[58]', 'ref_id': 'BIBREF57'}, {'start': 1068, 'end': 1072, 'mention': '[59]', 'ref_id': 'BIBREF58'}, {'start': 1276, 'end': 1280, 'mention': '[60]', 'ref_id': 'BIBREF59'}, {'start': 1319, 'end': 1323, 'mention': '[61]', 'ref_id': 'BIBREF60'}, {'start': 1347, 'end': 1351, 'mention': '[62]', 'ref_id': 'BIBREF61'}], 'section': 'Aspects of SARS-CoV-2 Pathogenesis and Potential Implications for Clinical Management of Patients with COVID-19 and Diabetes ::: Association between COVID-19 and Diabetes', 'ref_spans': []}, {'text': 'Although diabetes has been associated with worse outcomes in COVID-19 patients, the susceptibility to SARS-CoV-2 infection may not be higher in people with diabetes. According to several studies, the prevalence of diabetes in people infected with the virus is about the same as in the general population, even slightly lower [28], [63]. A meta-analysis of 12 studies describing data from 2,108 Chinese patients with COVID-19 reported a diabetes prevalence of 10.3% [63], which was similar to the national prevalence of 10.9% reported in 2013 [64]. An Italian study conducted among 146 patients with confirmed SARS-CoV-2 infection at the University Hospital of Padova found an equivalent pattern. The prevalence of diabetes in these patients was 8.9% (mean age 65.3 years), while it was 11.0% among people aged 55-75 years (mean age 65 years) from the same region in 2018 [65]. Although underreporting may be an issue to consider, potential biological mechanisms should not be disregarded. It has been shown that dipeptidyl peptidase-4 (DPP-4) is the primary receptor of MERS-CoV [2]. Since DPP-4 inhibitors are commonly applied in the treatment of diabetes worldwide, future research should explore whether DPP-4 may also act as receptor for SARS-CoV-2, thus contributing to a potential protective effect of these drugs against COVID-19 [63], [66].', 'cite_spans': [{'start': 325, 'end': 329, 'mention': '[28]', 'ref_id': 'BIBREF27'}, {'start': 331, 'end': 335, 'mention': '[63]', 'ref_id': 'BIBREF62'}, {'start': 465, 'end': 469, 'mention': '[63]', 'ref_id': 'BIBREF62'}, {'start': 542, 'end': 546, 'mention': '[64]', 'ref_id': 'BIBREF63'}, {'start': 871, 'end': 875, 'mention': '[65]', 'ref_id': 'BIBREF64'}, {'start': 1079, 'end': 1082, 'mention': '[2]', 'ref_id': 'BIBREF1'}, {'start': 1337, 'end': 1341, 'mention': '[63]', 'ref_id': 'BIBREF62'}, {'start': 1343, 'end': 1347, 'mention': '[66]', 'ref_id': 'BIBREF65'}], 'section': 'Aspects of SARS-CoV-2 Pathogenesis and Potential Implications for Clinical Management of Patients with COVID-19 and Diabetes ::: Association between COVID-19 and Diabetes', 'ref_spans': []}, {'text': 'So far, no anti-SARS-CoV-2 drug or vaccine has been officially approved for COVID-19 treatment [67]. Several clinical trials are in progress to assess the safety and efficacy of potential treatment alternatives, including remdesivir, tocilizumab, lopinavir/ritonavir, ribavirin, interferon, chloroquine phosphate, arbidol, among others [68]. One promising pharmacological option of relevance for patients with diabetes is chloroquine and its hydroxy-analogue hydroxychloroquine. Widely used for malaria and autoimmune diseases, chloroquine has also been reported as a potential broad-spectrum antiviral drug. Although the efficacy and safety of chloroquine for COVID-19 treatment remain unclear, a recent study showed that the drug was highly effective in controlling SARS-CoV-2 infection in vitro. In addition to its immunomodulant and anti-inflammatory effect, chloroquine increases endosomal pH and interferes with the glycosylation of cellular receptors of SARS-CoV, thereby blocking viral infection [69]. Preliminary results from more than 100 patients included in a Chinese clinical trial showed that chloroquine was superior to the control group in shortening the disease course, inhibiting pneumonia exacerbation, promoting a virus negative conversion and radiological improvement without severe side effects [70]. On the other hand, several studies have reported that hydroxychloroquine improves glycemic control in decompensated, treatment-refractory patients with diabetes [71], [72]. It has even been approved to treat T2DM in India as an add-on therapy for patients who do not achieve glycemic targets with two other oral glucose-lowering drugs [73]. Although inflammation is associated with impaired glucose control, the underlying mechanism of hydroxychloroquine’s hypoglycemic effect remains unclear [71]. It has been described that chloroquine increases the C peptide response, potentially reflecting an improved pancreatic β-cell function [72]. Reduced intracellular insulin degradation and increased insulin accumulation have also been identified as possible effects of hydroxychloroquine in animals models [74]. Given the previously reported impact of chloroquine/hydroxychloroquine on glucose metabolism, caution should be taken when the drug is administered to patients with diabetes and COVID-19. A dose adjustment of the oral antidiabetic drugs and/or insulin might be necessary in order to prevent potential hypoglycemic events.', 'cite_spans': [{'start': 95, 'end': 99, 'mention': '[67]', 'ref_id': 'BIBREF66'}, {'start': 336, 'end': 340, 'mention': '[68]', 'ref_id': 'BIBREF67'}, {'start': 1004, 'end': 1008, 'mention': '[69]', 'ref_id': 'BIBREF68'}, {'start': 1317, 'end': 1321, 'mention': '[70]', 'ref_id': 'BIBREF69'}, {'start': 1484, 'end': 1488, 'mention': '[71]', 'ref_id': 'BIBREF70'}, {'start': 1490, 'end': 1494, 'mention': '[72]', 'ref_id': 'BIBREF71'}, {'start': 1658, 'end': 1662, 'mention': '[73]', 'ref_id': 'BIBREF72'}, {'start': 1816, 'end': 1820, 'mention': '[71]', 'ref_id': 'BIBREF70'}, {'start': 1957, 'end': 1961, 'mention': '[72]', 'ref_id': 'BIBREF71'}, {'start': 2126, 'end': 2130, 'mention': '[74]', 'ref_id': 'BIBREF73'}], 'section': 'Aspects of SARS-CoV-2 Pathogenesis and Potential Implications for Clinical Management of Patients with COVID-19 and Diabetes ::: Association between COVID-19 and Diabetes', 'ref_spans': []}, {'text': 'The effect of corticosteroids on COVID-19 is also under investigation [68]. Acute lung damage and ARDS are partly due to the host immune response. While corticosteroids suppress lung inflammation, they also inhibit immunity and pathogen clearance [75]. In SARS-CoV and MERS-CoV infections, pulmonary histology showed inflammation and diffuse alveolar damage [76]. Therefore, corticosteroids were broadly applied [77], [78]. However, evidence did not show benefits, rather it was reported delayed viral RNA clearance or increased mortality and rate of complications, including diabetes, psychosis, and avascular necrosis [75]. The interim guidance from the WHO on clinical management of severe acute respiratory infection when SARS-CoV-2 infection is suspected advises against the use of corticosteroids outside clinical trials [79]. Considering the hyperglycemic effect [80] and the impact of these drugs on the immune response [75], special caution should be taken in patients with diabetes included in trials assessing the safety and efficacy of corticosteroids for COVID-19 [80].', 'cite_spans': [{'start': 70, 'end': 74, 'mention': '[68]', 'ref_id': 'BIBREF67'}, {'start': 247, 'end': 251, 'mention': '[75]', 'ref_id': 'BIBREF74'}, {'start': 358, 'end': 362, 'mention': '[76]', 'ref_id': 'BIBREF75'}, {'start': 412, 'end': 416, 'mention': '[77]', 'ref_id': 'BIBREF76'}, {'start': 418, 'end': 422, 'mention': '[78]', 'ref_id': 'BIBREF77'}, {'start': 620, 'end': 624, 'mention': '[75]', 'ref_id': 'BIBREF74'}, {'start': 827, 'end': 831, 'mention': '[79]', 'ref_id': 'BIBREF78'}, {'start': 870, 'end': 874, 'mention': '[80]', 'ref_id': 'BIBREF79'}, {'start': 928, 'end': 932, 'mention': '[75]', 'ref_id': 'BIBREF74'}, {'start': 1077, 'end': 1081, 'mention': '[80]', 'ref_id': 'BIBREF79'}], 'section': 'Aspects of SARS-CoV-2 Pathogenesis and Potential Implications for Clinical Management of Patients with COVID-19 and Diabetes ::: Association between COVID-19 and Diabetes', 'ref_spans': []}, {'text': 'No data is available regarding the most appropriate management of patients with diabetes infected by SARS-CoV-2, as well as patients with COVID-19 who develop glycemic decompensation. Rigorous glucose monitoring and careful consideration of drug interactions might attenuate worsening of symptoms and adverse outcomes. Although hyperglycemia is usually the main concern in this context, one should not disregard the possibility of hypoglycemic episodes as a result of the interplay between drug treatment, viral pathogenesis and typical metabolic disturbances of diabetes. Patient tailored therapeutic strategies and optimal glucose control goals should be formulated based on disease severity, presence of comorbidities and diabetes-related complications, age and other factors. A multidisciplinary team approach, including infectologists, endocrinologists, pulmonologists, psychologists, nutritionists and exercise rehabilitation specialists may be necessary during the prolonged hospitalization periods and recovery. Special attention should be paid to those with diabetic nephropathy, or diabetes-related heart complications, since they are also at higher risk for severe COVID-19 and death [7], [19]. Finally, increased vigilance and testing in outpatient diabetes clinics for COVID-19, as well as lower thresholds for hospitalization of these patients may have a positive impact on their outcomes.', 'cite_spans': [{'start': 1195, 'end': 1198, 'mention': '[7]', 'ref_id': 'BIBREF6'}, {'start': 1200, 'end': 1204, 'mention': '[19]', 'ref_id': 'BIBREF18'}], 'section': 'Aspects of SARS-CoV-2 Pathogenesis and Potential Implications for Clinical Management of Patients with COVID-19 and Diabetes ::: Association between COVID-19 and Diabetes', 'ref_spans': []}, {'text': 'COVID-9 has rapidly spread since its initial identification in Wuhan and has shown a broad spectrum of severity. Early isolation, early diagnosis, and early management might collectively contribute to a better control of the disease and outcomes. Diabetes and other comorbidities are significant predictors of morbidity and mortality in patients with COVID-19. Future research is urgently needed to provide a better understanding regarding potential differences in genetic predispositions across populations, underlying pathophysiological mechanisms of the association between COVID-19 and diabetes, and its clinical management.', 'cite_spans': [], 'section': 'Conclusions', 'ref_spans': []}, {'text': '\\n6. Declaration of competing interest: None.', 'cite_spans': [], 'section': 'Conclusions', 'ref_spans': []}, {'text': '\\n7. Funding: None.', 'cite_spans': [], 'section': 'Conclusions', 'ref_spans': []}, {'text': '\\n8. Author Contributions: A.H., N.C.d.V.M. and B.B. conceptualized and wrote the paper; A.H. and N.C.d.V.M. revised the text; and all approved the final manuscript.', 'cite_spans': [], 'section': 'Conclusions', 'ref_spans': []}, {'text': '\\n9. Authorship: All authors meet the International Committee of Medical Journal Editors (ICMJE) criteria for authorship and take responsibility for the integrity of the work. They confirm that this paper will not be published elsewhere in the same form, in English or in any other language, including electronically.', 'cite_spans': [], 'section': 'Conclusions', 'ref_spans': []}], 'paragraphs': [{'text': 'It is uncertain how long an individual with COVID-19 remains infectious. The period of infectivity is often assessed indirectly by detection of viral RNA from respiratory specimens. However, viral RNA does not necessarily confirm the presence of infectious virus. Higher viral loads have been detected soon after symptom onset, suggesting that transmission may be more likely to occur in the earlier stages of infection [15]. The viral shedding duration seems to vary according to the disease severity. It has been found that around 90% of patients with milder symptoms had a negative viral RNA test on nasopharyngeal swabs by day 10 post-onset, while the test remained positive for a longer time in all severe cases [16]. On the other hand, it has been reported that the viral load detected in asymptomatic patients was similar to that in symptomatic subjects [15]. Indeed, transmission from asymptomatic carriers or individuals within the incubation period has been described [17]. Nevertheless, the extent to which this occurs remains to be determined.', 'id': '00007'}], 'title': 'COVID-19 and Diabetes: Knowledge in Progress', 'sha': '62c692fa56138dbf3dbd03b6244a70d797ec83fb', 'doi': '10.1016/j.diabres.2020.108142', 'score': 9.060799598693848}, {'rank': 215, 'paragraphs': [{'text': \"Influenza A and B viruses and respiratory syncytial virus were tested by the Xpert Xpress Flu/RSV assay (GeneXpert System, Cepheid, Sunnyvale, CA, USA) according to the manufacturer's instructions [9]. The IgG/IgM of Chlamydophila pneumoniae and Mycoplasma pneumoniae were also tested using the SeroMP kit (Savyon Diagnostics, Ashdod, Israel). The SARS-COV2 nucleic acid test was conducted via real-time RT-PCR according to the protocol of the nucleic acid kit (Kangwei Century Biotechnology Company, China). The SARS-CoV2 antibody kit was used to test for specific IgM and IgG antibodies (Guangzhou Wonfo Biological Technology Co, Ltd., China) via colloidal gold immunochromatography.\", 'id': '00006'}], 'title': 'COVID-19 in a patient with long-term use of glucocorticoids: A study of a familial cluster', 'sha': '912c00514457672748738d29efb81f975815cdf6', 'doi': '10.1016/j.clim.2020.108413', 'score': 9.054200172424316}, {'rank': 216, 'abstract': 'An epidemic of extreme respiratory deterrence, pneumonia and shortness of breath, the SARS-CoV-2 viral infection began in Wuhan, Hubei Province, China in December 2019, and rapidly spread across China and beyond, with human to human transmission. On February 12, 2020, World Health Organization officially named the new coronavirus disease as coronavirus disease 19 (COVID-19). Most COVID-19 patients were diagnosed with pneumonia and many were treated using Chinese medicines and other secondary therapies. As of April 22, 2020, the total figure of infected patients has crossed 2.6 million people worldwide with over 180,000 deaths and 700,000 patients that have recovered. Preliminary reports suggest that certain drugs, such as chloroquine and antiviral nucleotide analogues such as remdesivir, which inhibit viral replication, can target the new coronavirus, although their usefulness in the clinic is still under debate. An expert US committee developed the US NIH guidelines for COVID-19 treatment, which was just released and will be regularly updated. This manuscript reviews the epidemiology, etiology, mortality, COVID-19 clinical symptoms, and potential therapeutic drugs, while highlighting the seriousness and damage-induced by SARS-CoV-2.', 'paragraphs': [{'text': \"In a report released on February 9, 2020, the incubation time was observed as long as 24 days (range from 0-24 days; average of 3 days)30. The WHO stated on February 10, 2020, at its press conference, that a very long duration of incubation can represent double exposure, 24 days was a specified point to be regarded in the sense of the study's key results31. Nevertheless, more recently, in a JAMA report released on February 21, 2020, a 19-day incubation period in infected patient has been identified30, Hubei Province registered another case with an incubation time of 27 days, on February 22, 202032. The incubation time period has averaged 5.2 days. However, it greatly differs between the patients, according to a Chinese article reported by New England Journal of Medicine on January 30, 202028. An analysis sponsored by the Holland Ministry of Health and released by Eurosurveillance examined data for 88 identified travelers to and from Wuhan, which were identified as COVID-19 infected patients between January 20 and 28, 2020. It was calculated that the mean incubation period was 6.4 days. The incubation duration varied from 2.1 to 11.1 days. The 11.1-day maximum limit may be known as conservative28. A comparison with other viruses can be found in Table 2.\", 'id': '00020'}], 'title': 'COVID-19: Review of Epidemiology and Potential Treatments Against 2019 Novel Coronavirus', 'sha': 'c540b5c7f8fda5460d0581c2b4fbf5dfd97a3141', 'doi': '10.15190/d.2020.5', 'score': 9.04800033569336}, {'rank': 217, 'paragraphs': [{'text': 'As described in the previous section, the distribution of the incubation period in most of the literature is either described through a parametric model or its empirical distribution based on the observed incubation period from the contact-tracing data. However, the contact-tracing data are challenging and expensive to obtain, and their accuracy can be highly influenced by recall bias. Hence, a low-cost and high-accuracy method to estimate the incubation distribution is needed.', 'id': '00005'}], 'title': 'Estimation of incubation period distribution of COVID-19 using disease onset forward time: a novel cross-sectional and forward follow-up study', 'sha': 'c61be9fc4562d250e2dec241ef75bd22d61f61d1; 6d3b3f4ab80a61c45f82c61c6c756cfc6ddf4bf2', 'doi': '10.1101/2020.03.06.20032417', 'score': 9.04580020904541}, {'rank': 218, 'paragraphs': [{'text': ' which were found on April 2. The inclusion criteria are the availability of both point estimates and 95% confidence intervals (95% CIs) (or equivalently, standard deviations) for the basic transmission number, the average incubation time, the asymptomatic rate, or the case fatality rate.', 'id': '00006'}], 'title': 'Estimation of the basic reproduction number, average incubation time, asymptomatic infection rate, and case fatality rate for COVID‐19: Meta‐analysis and sensitivity analysis', 'sha': '2c63ae442219d870167e691f91cd32e4ab9a7505', 'doi': '10.1002/jmv.26041', 'score': 9.037400245666504}, {'rank': 219, 'abstract': 'Abstract Background The rapid spread of the coronavirus disease 2019 (COVID-19), caused by a zoonotic beta-coronavirus entitled 2019 novel coronavirus (2019-nCoV), has become a global threat. Awareness of the biological features of 2019-nCoV should be updated in time and needs to be comprehensively summarized to help optimize control measures and make therapeutic decisions. Methods Based on recently published literatures, official documents and selected up-to-date preprint studies, we reviewed the virology and origin, epidemiology, clinical manifestations, pathology and treatment of 2019-nCoV infection, in comparison with severe acute respiratory syndrome coronavirus (SARS-CoV) and middle east respiratory syndrome coronavirus (MERS-CoV) infection. Results The genome of 2019-nCoV partially resembled SARS-CoV and MERS-CoV, and indicating a bat origin. The COVID-19 generally had a high reproductive number, a long incubation period, a short serial interval and a low case fatality rate (much higher in patients with comorbidities) than SARS and MERS. Clinical presentation and pathology of COVID-19 greatly resembled SARS and MERS, with less upper respiratory and gastrointestinal symptoms, and more exudative lesions in post-mortems. Potential treatments included remdesivir, chloroquine, tocilizumab, convalescent plasma and vaccine immunization (when possible). Conclusion The initial experience from the current pandemic and lessons from the previous two pandemics can help improve future preparedness plans and combat disease progression.', 'body_text': [{'text': 'In late December 2019, a pneumonia outbreak of unknown etiology took place in Wuhan, Hubei province, China, and spread quickly nationwide. Chinese Center for Disease Control and Prevention (CCDC) identified a novel beta-coronavirus called 2019-nCoV, now officially known as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) (Gorbalenya et al., 2020), that responsible for the pandemic. This was the third zoonotic coronavirus breakout in the first two decades of 21st century that allowing human-to-human transmission and raising global health concerns. Chinese government had taken immediate, transparent and extraordinary measures, and reached initial achievements to control the outbreak. As of 11 March 2020, the pandemic caused accumulated 80955 confirmed cases and 3162 deaths in China, 37364 confirmed cases and 1130 deaths in 113 other countries worldwide. World Health Organization (WHO) deeply concerned the unprecedented swift global spread and severity of the outbreak, and by ignorance and inaction of some countries. Therefore, WHO announced the COVID-19 can be characterized as a pandemic (WHO, 2020a).', 'cite_spans': [{'start': 336, 'end': 359, 'mention': 'Gorbalenya et al., 2020', 'ref_id': 'BIBREF12'}, {'start': 1116, 'end': 1125, 'mention': 'WHO, 2020', 'ref_id': 'BIBREF43'}], 'section': 'Introduction', 'ref_spans': []}, {'text': 'Biological features of 2019-nCoV and experiences combating COVID-19 should be updated in time and needs to be comprehensively summarized to help optimize control measures and make therapeutic decisions. What’s more, the 2019-nCoV demonstrated partial resemblance with SARS-CoV and MERS-CoV, in phylogenetic analysis, clinical manifestations and pathological findings. Scientific advances from the SARS and MERS outbreaks can provide valuable insight into rapid understanding and control measures of the current pandemic. We searched literatures and guidelines in Pubmed, Web of Science, Embase, CNKI, Wanfang, VIP, preprint bioRxiv and medRxiv databases from the earliest available date to 11 March, 2020. Initial search terms were \"2019-nCoV\" OR “2019 novel coronavirus\" OR \"SARS-CoV-2\" OR \"COVID-19\" OR \"corona virus disease 2019\" OR \"NCP\" OR \"Novel coronavirus pneumonia\". Further search words were above keywords, \"SARS\" OR \"SARS-CoV\" OR \"severe acute respiratory syndrome\", \"MERS\" OR \"MERS-CoV\" OR \"middle east respiratory syndrome\", in combinations of with \"spike protein\" OR \"genome\" OR \"reproductive number\" OR \"incubation period\" OR \"serial interval\" OR \"fatality rate\" OR \"clinical characteristics\" OR \"pathology\" OR \"autopsy\" OR \"treatment\". Moreover, official documents and news released by National Health Commission of P.R. China, CCDC, CDC(USA) and WHO were accessed for up-to-date information on COVID-19. Only the articles in English or Chinese were considered.', 'cite_spans': [], 'section': 'Introduction', 'ref_spans': []}, {'text': 'In this review, we highlight the pandemic potential and pathological indications of emerging coronavirus, comprehensively and systematically summarize the up-to-date knowledge of the biological characteristics of 2019-nCoV, including virology and origin, epidemiology, clinical manifestations, pathology and treatment. Because of its natural structures and biological features to bind receptors on host cells, the spike protein of 2019-nCoV may played an essential role in disease spreading. We summarized all of the four available pathology studies of COVID-19 biopsy and autopsy, and compared the results with previous two deadly coronavirus diseases. New therapeutic measures are emerging one after another. Potential effective treatments were remdesivir, chloroquine, tocilizumab, convalescent plasma and vaccine immunization (when possible). Evidence-based medicine should always be advocated to guide our clinical decision.', 'cite_spans': [], 'section': 'Introduction', 'ref_spans': []}, {'text': 'The pandemic escalated exponentially at the beginning of 2020, which might only be the tip of the iceberg due to delayed case reporting and deficiency in testing kits(Li et al., 2020). The onset of first cluster cases were reported an exposure history to the Huanan seafood(wild animal) wholesale market in Wuhan. However, phyloepidemiologic analyses suggested that Huanan market was not the origin of 2019-nCoV. The virus was imported from elsewhere and boosted in the crowded market(Yu et al., 2020). The proportion of infected cases without an exposure history and in health care workers gradually increased. All of the evidence indicated the human-to-human transmission ability of 2019-nCoV, which may already be spread silently between people in Wuhan before the cluster of cases from Huanan market was discovered in late December. Person-to-person transmission may occur mainly through droplet or contact transmission. According to Guan’s latest pilot study, 2019-nCoV was detected positive in the gastrointestinal tract specimens (stool and rectal swabs) as well as in saliva and urine, and even in esophageal erosion and bleeding site of severe peptic ulcer patients(Guan et al., 2020). Four important epidemiological parameters of 2019-nCoV were reviewed in comparison with those of SARS-CoV and MERS-CoV(shown in Table 1\\n).', 'cite_spans': [{'start': 167, 'end': 182, 'mention': 'Li et al., 2020', 'ref_id': 'BIBREF22'}, {'start': 485, 'end': 500, 'mention': 'Yu et al., 2020', 'ref_id': 'BIBREF50'}, {'start': 1175, 'end': 1192, 'mention': 'Guan et al., 2020', 'ref_id': 'BIBREF13'}], 'section': 'Epidemiology ::: Virology and origin', 'ref_spans': [{'start': 1323, 'end': 1330, 'mention': 'Table 1', 'ref_id': 'TABREF0'}]}, {'text': 'Reproductive number is an indication of the transmissibility of a virus, representing the average number of new infections generated by an infectious person in a totally naïve population. For R0<1, the number of infected is likely to increase; for R0 <1, transmission is likely to decline and die out. The reproductive number updated along with the development of the outbreak and interventions. R0 was estimated to be around 3 for SARS(Bauch et al., 2005) and <1 for MERS(Bauch and Oraby, 2013). The preliminary R0 of 2019-nCoV was reported as 2.24-3.58(Liu et al., 2020). Several research groups reported estimated R0 of the outbreak depending on distinct estimation methods and the validity of underlying assumptions. Liu et al. (2020) reviewed all of the 12 references of an estimated R0 ranged from 1.4 to 6.49, with a mean of 3.28 and a median of 2.79. In clinical studies, a 425-case study by 22 January 2020, reported an R0 of approximately 2.2(95%CI, 1.4-3.9)(Li et al., 2020), while another 4021-case study by 26 January 2020, estimated 3.77(95%CI, 3.51-4.05) (Yang et al., 2020). The discrepancy may be due to sample number and different stages of the pandemic.', 'cite_spans': [{'start': 437, 'end': 455, 'mention': 'Bauch et al., 2005', 'ref_id': 'BIBREF3'}, {'start': 473, 'end': 494, 'mention': 'Bauch and Oraby, 2013', 'ref_id': 'BIBREF4'}, {'start': 555, 'end': 571, 'mention': 'Liu et al., 2020', 'ref_id': 'BIBREF24'}, {'start': 721, 'end': 738, 'mention': 'Liu et al. (2020)', 'ref_id': 'BIBREF24'}, {'start': 969, 'end': 984, 'mention': 'Li et al., 2020', 'ref_id': 'BIBREF22'}, {'start': 1071, 'end': 1088, 'mention': 'Yang et al., 2020', 'ref_id': 'BIBREF49'}], 'section': 'Epidemiology ::: Virology and origin', 'ref_spans': []}, {'text': 'Incubation period is defined as the interval from initial exposure to an infectious agent to onset of any symptoms or signs it causes. A long incubation period may lead to a high rate of asymptomatic and subclinical infection. The first prediction of mean incubation period was 5.2 days (95%CI, 4.1-7.0 days), with the 95th percentile of the distribution at 12.5 days, based on 2019-nCoV exposure histories of the first 425 cases in Wuhan(Li et al., 2020). A 4021-case study reported 4.75 days (interquartile range: 3.0-7.2 days) (Yang et al., 2020). Another 88-exported-case study calculated the mean incubation period to be 6.4 days (95%CI, 5.6–7.7 days), using known travel histories to and from Wuhan and symptom onset dates(Backer et al., 2020). All these literatures lay the foundation to set 14 days as the medical observation period if any exposure occurred. A latest study collected 1099 cases from 552 hospitals in 31 provinces in China and declared a median incubation period of 3.0 days, ranging from 0 to surprisingly 24.0 days. An adjustments in screening and control policies may be needed. The 2019-nCoV generally has a longer incubation time than SARS-CoV (4.0 days, 95% CI 3.6-4.4 days) (Lessler et al., 2009) and MERS-CoV (range 4.5-5.2 days) (Park et al., 2018).', 'cite_spans': [{'start': 439, 'end': 454, 'mention': 'Li et al., 2020', 'ref_id': 'BIBREF22'}, {'start': 531, 'end': 548, 'mention': 'Yang et al., 2020', 'ref_id': 'BIBREF49'}, {'start': 729, 'end': 748, 'mention': 'Backer et al., 2020', 'ref_id': 'BIBREF1'}, {'start': 1206, 'end': 1226, 'mention': 'Lessler et al., 2009', 'ref_id': 'BIBREF21'}, {'start': 1263, 'end': 1280, 'mention': 'Park et al., 2018', 'ref_id': 'BIBREF31'}], 'section': 'Epidemiology ::: Virology and origin', 'ref_spans': []}, {'text': 'Serial interval is the interval from illness onset in a primary case to illness onset in the secondary case. The mean serial interval was estimated at 7.5 days(95% CI, 5.3-19days) using contact tracing data from early Wuhan cases in 2019-nCoV pandemic, which was shorter than the 8.4-day mean serial interval reported for SARS (Lipsitch et al., 2003) and 12.6-day for MERS(Cowling et al., 2015). Another estimation of the mean serial interval from 26 infector-infectee pairs was surprisingly 2.6 days, which was shorter than the median incubation period, suggesting a substantial proportion of secondary transmission before illness onset(Nishiura et al., 2020).', 'cite_spans': [{'start': 328, 'end': 349, 'mention': 'Lipsitch et al., 2003', 'ref_id': 'BIBREF23'}, {'start': 373, 'end': 393, 'mention': 'Cowling et al., 2015', 'ref_id': 'BIBREF10'}, {'start': 638, 'end': 659, 'mention': 'Nishiura et al., 2020', 'ref_id': 'BIBREF30'}], 'section': 'Epidemiology ::: Virology and origin', 'ref_spans': []}, {'text': 'The CFR in early studies of COVID-19 involving relatively small samples of confirmed cases in Wuhan, varied from 4.3% to 14.6%(Chen et al., 2020a, Huang et al., 2020, Wang et al., 2020a), but that may not be able to reflect the truth. The CFR in Wuhan was undoubtedly higher than CFR outside of Wuhan. The reported CFR ranged 1.4%-3.06% in large nationwide case studies(Guan et al., 2020, Yang et al., 2020). Prognosis factors such as male, elderly patients aged≥ 60 years, underlying disease, severe pneumonia at baseline and a delay from onset to diagnosis >5 days substantially elevated the CFRs(Yang et al., 2020). CFRs in patients with cardiovascular disease, diabetes, hypertension and respiratory disorders were as high as 10.5%, 7.3%, 6.0% and 6.3%, respectively. According to WHO announcement, SARS accounted for 8096 cases and 774 death, with a CFR of 9.6%(WHO, 2004). MERS-CoV infection were responsible for 2229 cases and 791 death, with a crude CFR of 35.5%(WHO, 2018). A study of 44672 cases conducted by CCDC reported a CFR of 2.3% nationwide, with a CFR of 2.9% in Hubei province compared to 0.4% in other parts of China(Wu and McGoogan, 2020). Despite a gradually stable CFR around 3, the prognosis and outcome of COVID-19 was also extraordinarily destructive and causing more death because of its enormous infected population. We can see a family close relative clustering phenomenon of severe and death cases. Genome sequencing and linkage analysis of ACE2 gene polymorphism may help determine individual susceptibility to 2019-nCoV and provide early-warning detection mode of severe COVID-19 in clinical practice.', 'cite_spans': [{'start': 127, 'end': 145, 'mention': 'Chen et al., 2020a', 'ref_id': 'BIBREF7'}, {'start': 147, 'end': 165, 'mention': 'Huang et al., 2020', 'ref_id': 'BIBREF16'}, {'start': 167, 'end': 185, 'mention': 'Wang et al., 2020a', 'ref_id': 'BIBREF38'}, {'start': 370, 'end': 387, 'mention': 'Guan et al., 2020', 'ref_id': 'BIBREF13'}, {'start': 389, 'end': 406, 'mention': 'Yang et al., 2020', 'ref_id': 'BIBREF49'}, {'start': 599, 'end': 616, 'mention': 'Yang et al., 2020', 'ref_id': 'BIBREF49'}, {'start': 971, 'end': 980, 'mention': 'WHO, 2018', 'ref_id': 'BIBREF42'}, {'start': 1137, 'end': 1158, 'mention': 'Wu and McGoogan, 2020', 'ref_id': 'BIBREF46'}], 'section': 'Epidemiology ::: Virology and origin', 'ref_spans': []}, {'text': 'Clinical presentation of COVID-19 greatly resembled viral pneumonia such as SARS and MERS. Most cases are mild cases(81%), whose symptoms were usually self-limiting and recovery in two weeks(Wu and McGoogan, 2020). Severe patients progressed rapidly with acute respiratory distress syndrome (ARDS) and septic shock, eventually ended in multiple organ failure.', 'cite_spans': [{'start': 191, 'end': 212, 'mention': 'Wu and McGoogan, 2020', 'ref_id': 'BIBREF46'}], 'section': 'Clinical manifestations ::: Virology and origin', 'ref_spans': []}, {'text': 'General information of four inpatient case studies with relatively comprehensive data were summarized in supplementary Table 1. The 2019-nCoV was more likely to infect elderly men with comorbidities. Males were more susceptible to 2019-nCoV infection, same as SARS-CoV and MERS-CoV studies(Badawi and Ryoo, 2016), due to X chromosome and sex hormones’ role on innate and adaptive immunity(Jaillon et al., 2019). Chronic underlying diseases (mainly hypertension, cardio-cerebrovascular diseases and diabetes) may increase the risk of 2019-nCoV infection(Guan et al., 2020), which is similar to MERS-CoV infection(Badawi and Ryoo, 2016). Smoking may be a negative prognostic indicator for COVID-19(Chen et al., 2020a, Guan et al., 2020).', 'cite_spans': [{'start': 290, 'end': 311, 'mention': 'Badawi and Ryoo, 2016', 'ref_id': 'BIBREF2'}, {'start': 389, 'end': 409, 'mention': 'Jaillon et al., 2019', 'ref_id': 'BIBREF17'}, {'start': 553, 'end': 570, 'mention': 'Guan et al., 2020', 'ref_id': 'BIBREF13'}, {'start': 612, 'end': 633, 'mention': 'Badawi and Ryoo, 2016', 'ref_id': 'BIBREF2'}, {'start': 696, 'end': 714, 'mention': 'Chen et al., 2020a', 'ref_id': 'BIBREF7'}, {'start': 716, 'end': 733, 'mention': 'Guan et al., 2020', 'ref_id': 'BIBREF13'}], 'section': 'Clinical manifestations ::: Virology and origin', 'ref_spans': [{'start': 119, 'end': 126, 'mention': 'Table 1', 'ref_id': 'TABREF0'}]}, {'text': 'Clinical information of the above four selected inpatient case studies were summarized in supplementary table 2. Onset of symptoms were usually mild and nonspecific, presenting by fever, dry cough and shortness of breath. Very few COVID-19 patients had prominent upper respiratory tract and gastrointestinal symptoms (eg, diarrhea) (Guan et al., 2020, Huang et al., 2020), compared to 20–25% of patients with MERS-CoV or SARS-CoV infection developed diarrhea(Assiri et al., 2013). However, only 43.8% of COVID-19 patients had an initial presentation of fever, and developed to 87.9% following hospitalization(Guan et al., 2020), compared to as high as 99% and 98% frequent in SARS-CoV and MERS-CoV infection(Badawi and Ryoo, 2016). Those patients without fever or even asymptomatic may be left un-quarantined as silent infection source, if the surveillance methods focused heavily on fever detection. Moreover, the onset of symptoms may help physicians identifying patients with poor prognosis. Patients admitted to the ICU were more likely to report pharyngeal pain, dyspnea, dizziness, abdominal pain and anorexia(Wang et al., 2020a).', 'cite_spans': [{'start': 333, 'end': 350, 'mention': 'Guan et al., 2020', 'ref_id': 'BIBREF13'}, {'start': 352, 'end': 370, 'mention': 'Huang et al., 2020', 'ref_id': 'BIBREF16'}, {'start': 459, 'end': 478, 'mention': 'Assiri et al., 2013', 'ref_id': 'BIBREF0'}, {'start': 609, 'end': 626, 'mention': 'Guan et al., 2020', 'ref_id': 'BIBREF13'}, {'start': 708, 'end': 729, 'mention': 'Badawi and Ryoo, 2016', 'ref_id': 'BIBREF2'}, {'start': 1116, 'end': 1134, 'mention': 'Wang et al., 2020a', 'ref_id': 'BIBREF38'}], 'section': 'Clinical manifestations ::: Virology and origin', 'ref_spans': []}, {'text': 'In terms of laboratory findings, a substantial decrease in the total number of lymphocytes could be used as an index in the diagnosis of 2019-nCoV infection, indicating a consumption of immune cells and an impairment to cellular immune function(Chen et al., 2020a). Non-survivors developed more severe lymphopenia over time(Wang et al., 2020a). Initial proinflammatory plasma cytokine concentrations were higher in COVID-19 patients than in healthy adults. ICU patients had even higher plasma levels of IL2, IL7, IL10, GSCF, IP10, MCP1, MIP1A, and TNFα compared to non-ICU patients(Huang et al., 2020). There were numerous differences in laboratory findings between patients admitted to the ICU and those not, including higher white blood cell and neutrophil counts, higher levels of D-dimer, creatine kinase, and creatine in ICU patients(Wang et al., 2020a).', 'cite_spans': [{'start': 245, 'end': 263, 'mention': 'Chen et al., 2020a', 'ref_id': 'BIBREF7'}, {'start': 324, 'end': 342, 'mention': 'Wang et al., 2020a', 'ref_id': 'BIBREF38'}, {'start': 582, 'end': 600, 'mention': 'Huang et al., 2020', 'ref_id': 'BIBREF16'}, {'start': 839, 'end': 857, 'mention': 'Wang et al., 2020a', 'ref_id': 'BIBREF38'}], 'section': 'Clinical manifestations ::: Virology and origin', 'ref_spans': []}, {'text': 'Typical chest CT manifestation of COVID-19 pneumonia were initially small subpleural ground glass opacities that grew larger with crazy-paving pattern and consolidation. After two weeks of growth, the lesions were gradually absorbed leaving extensive opacities and subpleural parenchymal bands in recovery patients. However, Guan et al. ( 2020) demonstrated that patients with normal radiologic findings on initial presentation consisted of 23.9% and 5.2% of severe and non-severe cases respectively, which add the complexity to disease control.', 'cite_spans': [{'start': 325, 'end': 343, 'mention': 'Guan et al. ( 2020', 'ref_id': 'BIBREF13'}], 'section': 'Clinical manifestations ::: Virology and origin', 'ref_spans': []}, {'text': 'Autopsy or biopsy studies would always be the key to understand the biological features of 2019-nCoV. Histological examinations of two patients underwent lung lobectomies for adenocarcinoma, revealed edema, proteinaceous exudate, focal hyperplasia of pneumocytes with only patchy inflammatory cellular infiltration without prominent hyaline membranes. Since both patients didn’t develop symptoms of COVID-19 pneumonia at the time of surgery, these changes likely represent an early phase of the lung pathology of COVID-19 pneumonia. Qian et al. (2020) first reported the pathological characteristics of a patient who died from COVID-19. General observation from raw eyes showed less fibrosis and consolidation, instead more exudative lesions in COVID-19 than SARS. Microscopic examination showed bilateral diffuse alveolar damage with cellular fibromyxoid exudates, indicating ARDS. Interstitial mononuclear inflammatory infiltrates were dominated by lymphocytes. Multinucleated syncytial cells with atypical enlarged pneumocytes showed viral cytopathic-like changes, without obvious intranuclear or intracytoplasmic viral inclusions. Results from flow cytometric analysis demonstrated that the counts of peripheral CD4+ and CD8 + T cells were substantially reduced, while their status were hyper-activated. It indicated the severe immune injury in later stage of COVID-19, but not by virus direct destruction(Xu et al., 2020b). Based on the public database and single-cell RNA-Seq technique, pathological studies revealed that male donors had a higher ACE2-expressing cell ratio than their female counterparts. The only Asian male specimen has five more times as much as ACE2 expressing on the white and African American donors. This might explain why the 2019-nCov and previous SARS-CoV pandemic were concentrated in the Asian population and a heightened susceptibility of male patients, although more evidence was needed to draw such conclusion(Yu et al., 2020).', 'cite_spans': [{'start': 533, 'end': 551, 'mention': 'Qian et al. (2020)', 'ref_id': 'BIBREF33'}, {'start': 1410, 'end': 1426, 'mention': 'Xu et al., 2020b', 'ref_id': 'BIBREF48'}, {'start': 1948, 'end': 1963, 'mention': 'Yu et al., 2020', 'ref_id': 'BIBREF50'}], 'section': 'Pathology ::: Virology and origin', 'ref_spans': []}, {'text': 'Pathological manifestations of SARS and MERS infected patients may shed lights to control current 2019-nCoV pandemic. Histology examination demonstrated a considerably higher viral load of SARS-CoV RNA in lung and small bowels than other organs of the body, suggesting an reason for manifestation of pneumonia and diarrhea in SARS patients. Living 2019-nCoV was also detected positive in stool specimens and rectal swabs of infected patients, indicating possible transmission route of oral-faecal transmission. Proper handling with the infected corpse and disposal of human excreta of infected patients were of great importance(Nicholls et al., 2003). Thrombi were seen in all six autopsies of SARS-CoV infected patients, with even huge thrombus formation in part of pulmonary vessels. Coagulation function disorders were reported in most of the severe COVID-19 patients, by elevated levels of D-Dimer and prolonged prothrombin time, some of whom ended in disseminated intravascular coagulation(Chen et al., 2020a, Huang et al., 2020, Wang et al., 2020a). This may explain some sudden death of clinical recovery patients and serve as an indication for disease severity. In an autopsy study, the only one patient without usage of corticosteroids demonstrated increased CD3+ lymphocyte than five other specimen treated by corticosteroids (Pei and Gao, 2005). It suggested an inhibition of immune system and a careful usage of corticosteroids in COVID-19 treatment. Much is to be discovered in more 2019-nCoV autopsies.', 'cite_spans': [{'start': 628, 'end': 649, 'mention': 'Nicholls et al., 2003', 'ref_id': 'BIBREF29'}, {'start': 995, 'end': 1013, 'mention': 'Chen et al., 2020a', 'ref_id': 'BIBREF7'}, {'start': 1015, 'end': 1033, 'mention': 'Huang et al., 2020', 'ref_id': 'BIBREF16'}, {'start': 1035, 'end': 1053, 'mention': 'Wang et al., 2020a', 'ref_id': 'BIBREF38'}, {'start': 1337, 'end': 1354, 'mention': 'Pei and Gao, 2005', 'ref_id': 'BIBREF32'}], 'section': 'Pathology ::: Virology and origin', 'ref_spans': []}, {'text': 'There are currently no vaccine or specific effective antiviral therapies for COVID-19 in general. Thus there is an urgent need for global surveillance of COVID-19 patients. New therapeutic drugs are emerging one after another. However, double-blinded randomized controlled trials with larger sample size are needed to determine the safety and efficacy of these new drugs and guide clinical decision. Medical interventions can be divided into four major categories: general treatment, coronavirus specific treatments, antiviral treatments and others.', 'cite_spans': [], 'section': 'Treatment ::: Virology and origin', 'ref_spans': []}, {'text': 'General treatments included nutritional interventions, immuno-enhancers and Chinese medicine. Interferon, intravenous gamma-globulin and thymosin were believed to boost our immune system to fight with SARS-CoV and MERS-CoV as well as 2019-nCoV. Chloroquine, an old Chinese medicine for treatment of malaria and autoimmune disease, had demonstrated remarkable inhibition in the spread of SARS-CoV by interfering with ACE2 in Vero E6 cell lines(Vincent et al., 2005). Wang et al. (2020b) demonstrated that chloroquine functioned at both entry and post-entry stages of the 2019-nCoV infection in Vero E6 cells, as well as an immune-modulating activity that enhanced antiviral effect in vivo. Recent multicenter clinical trails conducted in China have also reported obvious efficacy and acceptable safety in COVID-19 patients by reducing exacerbation of pneumonia, improving radiological findings, promoting a virus negative conversion, and shortening the disease course(Gao et al., 2020).', 'cite_spans': [{'start': 443, 'end': 463, 'mention': 'Vincent et al., 2005', 'ref_id': 'BIBREF37'}, {'start': 466, 'end': 485, 'mention': 'Wang et al. (2020b)', 'ref_id': 'BIBREF39'}, {'start': 967, 'end': 983, 'mention': 'Gao et al., 2020', 'ref_id': 'BIBREF11'}], 'section': 'Treatment ::: Virology and origin', 'ref_spans': []}, {'text': 'Due to the indispensable role of the S-protein in coronavirus, therapies and vaccine exploration targeting S-protein-ACE2 interaction may be very promising. Previous therapies targeting SARS-CoV and MERS-CoV may accelerate the development of treatment of COVID-19 because of their structure resemblance and genome similarities. The human monoclonal antibody could efficiently neutralize SARS-CoV and inhibit syncytia formation between S-protein and ACE2 expressing cells(Sui et al., 2004). Appropriate modification of the monoclonal antibody may be effective for treatment of COVID-19. What’s more, potential therapies targeting the renin-angiotensin system, to increase ACE2 expression and inhibit ACE may be developed to treat COVID-19 in the future. Hoffmann et al. (2020) reported a cellular protease TMPRSS2 for 2019-nCoV priming upon entrance into cells and viral spread in the infected host cells. The serine protease inhibitor camostat mesylate against TMPRSS2, can efficiently blocked 2019-nCoV-S-protein-driven cell entry, which could be a promising treatment for 2019-nCoV infection.', 'cite_spans': [{'start': 471, 'end': 487, 'mention': 'Sui et al., 2004', 'ref_id': 'BIBREF35'}, {'start': 753, 'end': 775, 'mention': 'Hoffmann et al. (2020)', 'ref_id': 'BIBREF14'}], 'section': 'Treatment ::: Virology and origin', 'ref_spans': []}, {'text': 'There are no effective antiviral treatment for coronavirus infection, even the strong candidates as lopinavir/ritonavir and abidol exhibited no remarkable effect on clinical improvement, day 28 mortality or virus clearance(Chen et al., 2020). Expectation and attention were shifted to “remdesivir” which may be the most potential wide-spectrum drug for antiviral treatment of 2019-nCoV. Remdesivir is an adenosine analogue, which incorporates into novel viral RNA chains and results in pre-mature termination. It is currently under clinical development for the treatment of Ebola virus infection(Mulangu et al., 2019). Wang et al. (2020b) revealed that remdesivir were highly effective and safe in the control of 2019-nCoV infection in Vero E6 cells and Huh-7 cells. A successful appliance of remdesivir on the first 2019-nCoV infected case in the United States when the his clinical status was getting worsen, were recently released(Holshue et al., 2020). Animal experiments also showed superiority of remdesivir over lopinavir/ritonavir combined with interferon-β, by reducing MERS-CoV titers of infected mice and improving the lung tissue damage(Sheahan et al., 2020). The effectiveness and safety of remdesivir can be expected by the clinical trial lead by Dr Bin Cao.', 'cite_spans': [{'start': 596, 'end': 616, 'mention': 'Mulangu et al., 2019', 'ref_id': 'BIBREF27'}, {'start': 619, 'end': 638, 'mention': 'Wang et al. (2020b)', 'ref_id': 'BIBREF39'}, {'start': 934, 'end': 954, 'mention': 'Holshue et al., 2020', 'ref_id': 'BIBREF15'}, {'start': 1149, 'end': 1169, 'mention': 'Sheahan et al., 2020', 'ref_id': 'BIBREF34'}], 'section': 'Treatment ::: Virology and origin', 'ref_spans': []}, {'text': 'The 2019-nCoV infection is associated with a cytokine storm triggered by over-activated immune system(Huang et al., 2020, Xu et al., 2020b), similar to SARS and MERS. The aberrant and excessive immune responses lead to a long-term lung function and structure damage in patients survived from ICU. Ongoing trials of IL-6 antagonist tocilizumab, which shown effective against cytokine release syndrome resulting from CAR-T cell infusion against B cell acute lymphoblastic leukemia, may be expanded to restore T cell counts and treat severe 2019-nCoV infection(Le et al., 2018). The available observational studies and meta-analysis of corticosteroid treatment suggested impaired antibody response, increased mortality and secondary infection rates in influenza, increased viraemia and impaired virus clearance of SARS-CoV and MERS-CoV, and complications of corticosteroid therapy in survivors(Zumla et al., 2020). Therefore, corticosteroid should not be recommended for treatment of 2019-nCoV, or use on severe patient with special caution. A review on convalescent plasma for treatment of SARS-CoV and severe influenza infection, suggested a reduction in hospital stay and mortality rate, especially when administered early after symptom onset (Mair-Jenkins et al., 2015). However, another study demonstrated no significant improvement of convalescent plasma transfusion on survival of Ebola virus infected patients. Possible reasons may be the unknown levels of neutralizing antibodies in convalescent plasma and transfusion timing(van Griensven et al., 2016). In terms of vaccine, if any cross-reactive epitopes were identified between 2019-nCoV and SARS-CoV, previous vaccine for SARS-CoV might be re-utilized to facilitate 2019-nCoV vaccine development. We recommend an influenza and Streptococcus pneumoniae vaccination for prophylaxis, especially in elderly adults(Chen et al., 2020b). Both pandemic viruses result in similar respiratory symptoms and hard to distinguish. The 2019-nCoV pandemic initiated in flu season which easy to develop a combination infection of 2019-nCoV and influenza or Streptococcus infection. Vaccination against influenza and Streptococcus pneumonia in vulnerable elderly people with comorbidities are highly cost-effective, which is demonstrated to associate with reductions in the risk of hospitalization and death from all causes during influenza seasons(Nichol et al., 2003).', 'cite_spans': [{'start': 102, 'end': 120, 'mention': 'Huang et al., 2020', 'ref_id': 'BIBREF16'}, {'start': 122, 'end': 138, 'mention': 'Xu et al., 2020b', 'ref_id': 'BIBREF48'}, {'start': 558, 'end': 573, 'mention': 'Le et al., 2018', 'ref_id': 'BIBREF20'}, {'start': 891, 'end': 909, 'mention': 'Zumla et al., 2020', 'ref_id': 'BIBREF52'}, {'start': 1244, 'end': 1269, 'mention': 'Mair-Jenkins et al., 2015', 'ref_id': 'BIBREF26'}, {'start': 1532, 'end': 1558, 'mention': 'van Griensven et al., 2016', 'ref_id': 'BIBREF36'}, {'start': 1870, 'end': 1888, 'mention': 'Chen et al., 2020b', 'ref_id': 'BIBREF8'}, {'start': 2391, 'end': 2410, 'mention': 'Nichol et al., 2003', 'ref_id': 'BIBREF28'}], 'section': 'Treatment ::: Virology and origin', 'ref_spans': []}, {'text': 'In conclusion, it still remains a challenging task to fight the 2019-nCoV of unknown origin and mysterious biological features, and to control an outbreak of COVID-19 with such a high R0, a long incubation period and a short serial interval, by limited treatment and prevention measures. Lessons learned from the MERS and SARS outbreaks can provide valuable insight into how to handle the current pandemic. The successful public health outbreak response tactics of Chinese government, such as hand hygiene, wearing masks, isolation, quarantine, social distancing, and community containment, can be copied by other countries according to their national situation. As the pandemic is still ongoing and expanding, experiences and research literatures from China and other countries will increase. The 2019-nCoV should be monitored of any possible gene variation of antigenic drift or antigenic conversion, to avoid another round of outbreak. Another lessons from this pandemic will be awe for nature and love for life.', 'cite_spans': [], 'section': 'Conclusion', 'ref_spans': []}, {'text': 'Author Mingxuan Xie is responsible for conceptualization, literature reviewing, manuscript drafting and submitting. Author Qiong Chen is responsible for supervision, literature reviewing, manuscript drafting and revising. All authors have read and agreed to the published version of the manuscript.', 'cite_spans': [], 'section': 'Author Contributions', 'ref_spans': []}, {'text': 'This research did not receive any specific grant from funding agencies in the public, commercial, or not-for-profit sectors.', 'cite_spans': [], 'section': 'Funding Source', 'ref_spans': []}, {'text': 'The ethical approval or individual consent was not applicable.', 'cite_spans': [], 'section': 'Ethical Approval', 'ref_spans': []}, {'text': 'All authors declare no conflict of interest. All authors don’t have any financial and personal relationships with other people or organizations that could influence our work.', 'cite_spans': [], 'section': 'Conflicts of Interest', 'ref_spans': []}, {'text': '\\nChendd et al., 2020, Wen-Bin et al., 2020 and WHO (2003).', 'cite_spans': [{'start': 1, 'end': 20, 'mention': 'Chendd et al., 2020', 'ref_id': 'BIBREF6'}, {'start': 22, 'end': 42, 'mention': 'Wen-Bin et al., 2020', 'ref_id': 'BIBREF40'}, {'start': 47, 'end': 57, 'mention': 'WHO (2003)', 'ref_id': 'BIBREF41'}], 'section': 'Uncited references', 'ref_spans': []}], 'paragraphs': [{'text': 'In conclusion, it still remains a challenging task to fight the 2019-nCoV of unknown origin and mysterious biological features, and to control an outbreak of COVID-19 with such a high R0, a long incubation period and a short serial interval, by limited treatment and prevention measures. Lessons learned from the MERS and SARS outbreaks can provide valuable insight into how to handle the current pandemic. The successful public health outbreak response tactics of Chinese government, such as hand hygiene, wearing masks, isolation, quarantine, social distancing, and community containment, can be copied by other countries according to their national situation. As the pandemic is still ongoing and expanding, experiences and research literatures from China and other countries will increase. The 2019-nCoV should be monitored of any possible gene variation of antigenic drift or antigenic conversion, to avoid another round of outbreak. Another lessons from this pandemic will be awe for nature and love for life.', 'id': '00021'}], 'title': 'Insight into 2019 novel coronavirus — an updated intrim review and lessons from SARS-CoV and MERS-CoV', 'sha': '4bc41aaa2b754b3ec526eb40c384a18ab38b8d51', 'doi': '10.1016/j.ijid.2020.03.071', 'score': 9.035699844360352}, {'rank': 220, 'abstract': 'Background: Africa is the last major region to capitulate to the SARS-CoV-2 (COVID-19) pandemic. The first confirmed COVID-19 case in the region was reported on February 14, but what lies ahead in terms of the course and magnitude of infection remains speculative. To the best of our knowledge, no study, using a robust methodology, provides the immediate and long-term trajectory of COVID-19 for the entire region or accounts for its local context. This paper is the first systematic attempt to provide estimates on how many people would contract the virus and how many would die in the coming few months across Africa. Methods: The forecasts on caseloads and incidences are from a co-variate-based instrumental variable regression model. Fatality rates from Italy and China were further applied to generate mortality estimates after adjustments were made for differences in age-structure, health service quality, and living standards between each of the African countries and those of the reference population. We cover all countries that reported a confirmed case as of March 31, 2020. Results: By the end of June, 16,283,085 people will contract COVID-19 (95% CI 718,403 to 98, 358, 799). With a cumulative caseload of 5,413,4517 (95% CI 1,332,953 to 8,489,940) and 906,625 (95% CI 173, 821 to 4,742,917) Northern and Eastern Africa will respectively be the most and least affected sub-regions in the continent. Cumulative COVID-19 cases on June 30, 2020 are expected to reach 2,912, 864 (95% CI 465,028 to 18,286,358) in Southern Africa, 2,787, 913 (95% CI 517, 489 to 15,056,314) in Western Africa, and 1,185,742 (95% CI 229, 111 to 6,138,692) in Central Africa. New infections (incidence) for the month of April are expected to be the highest in Djibouti, 32.8 per 1000 (95% CI 6.25 to 171.77), while Morocco 1045 (95% CI 167 to 6,547) will register the highest number of deaths. Conclusion: Our study shows that countries that are least urbanized and have a low level of socio-economic development, hence least connected to the outside world, are likely to register lower and slower transmissions, at least at the early stage of the epidemic. However, the same set of enabling factors that worked for their benefit are likely to go against them in implementing interventions that have lessened the impact of the disease elsewhere.', 'body_text': [{'text': 'The novel human coronavirus (SARS-CoV-2 or COVID-19) outbreak initially emerged in Wuhan, China, in late 2019. Since then, it has spread to 198 countries and territories around the world and has been declared a pandemic (1) . As of March 31, 2020, there have been reported 750,874 covid-19 cases, 36,045 deaths, and 117,603 recoveries (1) . The first confirmed COVID-19 case in Africa was reported in Egypt on February 14, 2020 and since then the number of confirmed infections in the region kept increasing and has surpassed over 5000 cases by March 31, 2020 (2, 3) . Comoros, Lesotho, Malawi, and South Sudan are the only African countries that had not reported a confirmed case by March 31, 2020.', 'cite_spans': [{'start': 220, 'end': 223, 'text': '(1)', 'ref_id': 'BIBREF0'}, {'start': 335, 'end': 338, 'text': '(1)', 'ref_id': 'BIBREF0'}, {'start': 560, 'end': 563, 'text': '(2,', 'ref_id': None}, {'start': 564, 'end': 566, 'text': '3)', 'ref_id': None}], 'ref_spans': [], 'section': 'INTRODUCTION'}, {'text': \"To date, no vaccine is available for COVID-19. Therefore, the ability to limit the devastating consequences of the disease on people's lives and livelihoods would have to rely on the implementation of effective preventative non-pharmaceutical interventions (NPIs). NPIs are multiple public health measures designed to reduce viral transmission rates in a population by reducing the reproduction number (R0); the average number of secondary cases each case generates (4) (5) (6) .\", 'cite_spans': [{'start': 466, 'end': 469, 'text': '(4)', 'ref_id': 'BIBREF3'}, {'start': 470, 'end': 473, 'text': '(5)', 'ref_id': 'BIBREF4'}, {'start': 474, 'end': 477, 'text': '(6)', 'ref_id': 'BIBREF5'}], 'ref_spans': [], 'section': 'INTRODUCTION'}, {'text': 'NPIs directly influence the course of the COVID-19 pandemic, including the rate of spread, and the expected duration of the pandemic. However, several characteristics of the virus such as incubation period (the time between infection and symptom onset), serial interval (the time between symptom onset of a primary and secondary case) the extent of asymptomatic cases, the possibility of presymptomatic infectiousness, and the case fatality rate (CFR), and also the possible role of weather in transmission, all remain ambiguous or mostly unknown (7, 8) . Estimates for COVID-19 CFR range from 0.3-1% (9) . Asymptomatic or mild presentation comprises the bulk of the reported cases, which is an estimated at 80% (10) . Longitudinal viremia measurements from a small study (sample size of 16) , suggests that there are high enough viral loads to trigger pre-symptomatic infectiousness for 1-2 days before the onset of symptoms (11) . With the lack of clinical studies measuring viremia, the infectious period also remains largely unknown, with estimates ranging from few days to 10 days or more after the incubation period (11) .', 'cite_spans': [{'start': 547, 'end': 550, 'text': '(7,', 'ref_id': 'BIBREF7'}, {'start': 551, 'end': 553, 'text': '8)', 'ref_id': 'BIBREF8'}, {'start': 601, 'end': 604, 'text': '(9)', 'ref_id': 'BIBREF9'}, {'start': 712, 'end': 716, 'text': '(10)', 'ref_id': 'BIBREF10'}, {'start': 788, 'end': 791, 'text': '16)', 'ref_id': 'BIBREF17'}, {'start': 926, 'end': 930, 'text': '(11)', 'ref_id': 'BIBREF11'}, {'start': 1122, 'end': 1126, 'text': '(11)', 'ref_id': 'BIBREF11'}], 'ref_spans': [], 'section': 'INTRODUCTION'}, {'text': \"The complexity of the infection and recovery process, therefore, means that proper understanding of the epidemiological dynamics of COVID-19 within the local context is fundamental to combat the pandemic. Specifically, studies illustrating future trajectories of the disease can help nations and communities to develop early warning systems, avoid overwhelming healthcare services, minimize morbidity and mortality from the disease, evaluate the effects of interventions and the long-term consequences of the virus on peoples' livelihood. This is particularly true in Africa, where livelihoods are fragile, and previous epidemics, such as HIV/AIDS and, more recently, Ebola, has been known to exert enormous socioeconomic consequences (12) (13) (14) . In addition, in the face of a new pandemic in the region, the already overstretched healthcare systems that are struggling to deliver essential healthcare services such as immunization and HIV/AIDS treatment would be in further jeopardy and risk losing the gains achieved so far in disease control efforts.\", 'cite_spans': [{'start': 735, 'end': 739, 'text': '(12)', 'ref_id': 'BIBREF12'}, {'start': 740, 'end': 744, 'text': '(13)', 'ref_id': 'BIBREF14'}, {'start': 745, 'end': 749, 'text': '(14)', 'ref_id': 'BIBREF15'}], 'ref_spans': [], 'section': 'INTRODUCTION'}, {'text': 'This underscores the need for decision-makers to be equipped with actionable information that brings visibility to contemporary and future epidemiological trends to guide interventions. This study, using a robust methodology, provides spatial and temporal trajectories of COVID-19 for the entire Africa region. To the best of our knowledge, this is the first such attempt and accounts for the local context and characteristics relevant to the epidemiology of the region.', 'cite_spans': [], 'ref_spans': [], 'section': 'INTRODUCTION'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'cite_spans': [], 'ref_spans': [], 'section': 'INTRODUCTION'}, {'text': 'The copyright holder for this preprint this version posted April 14, 2020. The remaining part of this paper is divided into three sections. In section 2, we describe our covariate based predictive model and the input data that has been used in the modeling exercise. We also describe the methodology used for estimating deaths from COVID-19. Section 3 provides the results on the projected new infections, cumulated cases, and deaths due to COVID-19 across countries, covering the five sub regions. Finally, we discuss the key findings and implications of the study in section 4 and provide recommendations for action.', 'cite_spans': [], 'ref_spans': [], 'section': 'INTRODUCTION'}, {'text': 'The analysis is based on a linear instrumental variable (IV) regression framework. The basic model consists of two equations which can be expressed as follows (15) (16) (17) :', 'cite_spans': [{'start': 159, 'end': 163, 'text': '(15)', 'ref_id': 'BIBREF16'}, {'start': 164, 'end': 168, 'text': '(16)', 'ref_id': 'BIBREF17'}, {'start': 169, 'end': 173, 'text': '(17)', 'ref_id': 'BIBREF18'}], 'ref_spans': [], 'section': 'METHODS'}, {'text': 'Here, Y is the outcome variable of interest, and in our case represents the number of confirmed cases of COVID-2019 as of March 31, 2020. K is the rate of infection between week 1 and week 2 and is an endogenous variable. T and X represent a set of instrumental and exogenous variables, respectively. ui and zi are error terms and both are assumed to have a zero mean and a nonzero-correlation (18-20).', 'cite_spans': [], 'ref_spans': [], 'section': 'METHODS'}, {'text': 'Based on our understanding of the epidemiology of COVID-19 and the scanty literature, we have identified eleven regressors that are deemed associated with the trajectory of COVID-19 in the Africa region (4, 6) . These are urbanicity, socio-demographic index, average household size, the age structure of the population, health care access, health quality index, adherence to international health regulations, and prevalence of HIV and asthma. In addition, we have controlled for time since the first reported case and the rate of expansion of the disease between week one and week two of the epidemic in the respective countries. This was modeled using air traffic as its instrument variable.', 'cite_spans': [{'start': 203, 'end': 206, 'text': '(4,', 'ref_id': 'BIBREF3'}, {'start': 207, 'end': 209, 'text': '6)', 'ref_id': 'BIBREF5'}], 'ref_spans': [], 'section': 'METHODS'}, {'text': 'The data for analysis were assembled from various sources. The data on prevalence of HIV and asthma, as well as on socio-demographic index and the index of health care access and quality index, were from (21-23). The data on urbanization were obtained from the Population Reference Bureau (24), while average household size, age profile, and data on the total population used for the calculation of rates were all from the UN Population Division (25). The index of adherence to international health regulations was accessed from the WHO, while the number of confirmed cases was from the John Hopkins database of COVID-19 (26, 27).', 'cite_spans': [], 'ref_spans': [], 'section': 'METHODS'}, {'text': \"The 11 variables selected for analyses were chosen out of an initial set of over 20 variables that included the proportion of households with at least one member aged 60 years or over, the proportion of households with at least one member aged 65 years or over, the World Bank's governance index, as well as the prevalence of TB and diabetes. STATA's Lasso software for model selection was used to narrow down the list of covariates (28). Furthermore, out of the 193 countries captured in our data, for a few countries, data were unavailable for selected variables, namely air traffic (25 countries), adherence to international health regulation (20 countries), and household size (24 countries). To . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)\", 'cite_spans': [], 'ref_spans': [], 'section': 'METHODS'}, {'text': \"The copyright holder for this preprint this version posted April 14, 2020. . https://doi.org/10.1101/2020.04.09.20059154 doi: medRxiv preprint bridge the gap, we performed multiple imputation procedures, particularly by running 1000 imputations and 99 000 iterations using STATA's multiple imputation routine (29, 30). The resulting median values from the imputation exercise were subsequently used to estimate the prediction model specified earlier. We transformed all the regressors as well as the dependent variables to a log-scale before they were put into the model.\", 'cite_spans': [], 'ref_spans': [], 'section': 'METHODS'}, {'text': 'The goodness-of-fit for the predictive model was evaluated using k-fold cross-validation techniques (31, 32). The advantage of a k-fold cross-validation technique over other statistical validation procedures is that all observations are used for both training and validation, and each observation is used for validation exactly once (31).', 'cite_spans': [], 'ref_spans': [], 'section': 'METHODS'}, {'text': 'Estimation of COVID-19 deaths was made using indirect standardization technique, with additional adjustments for differences in health system readiness and living standards. In the absence of agespecific infection data the proposed approach can be expressed in a general form as follows:', 'cite_spans': [], 'ref_spans': [], 'section': 'METHODS'}, {'text': \"Here, is the estimated number of deaths for country y (y= 1, 2…47) using country k's (China and Italy) COVID-19 fatality rate, general health system readiness and socio-economic standards as a reference.\", 'cite_spans': [], 'ref_spans': [], 'section': 'METHODS'}, {'text': 'is the standard or reference age-specific fatality rate for age group i in standard country k, in our case obtained from two sources, China and Italy.', 'cite_spans': [], 'ref_spans': [], 'section': 'METHODS'}, {'text': 'is the population at age i in country y; is population level COVID-19 prevalence rate and is an estimate we generated using the instrumental variable regression expressed in equation 1. The quantity in the square bracket represents age-standardized estimated number of deaths in country y, if age-specific fatality rates were to follow those of the standard populations. However, our study population differs from the two countries not just with respect to age structure but also living standard and health system readiness. Hence, to adjust the estimated deaths further we developed a correction factor that links the socio-economic status and health system readiness of each country with those two populations, which is represented by . In this case, is a geometric-mean of socio-demographic index (S) and health access and quality index (Q), and the relevant data for each country was sourced from IHME (21, 22). Analysis was performed using STATA version 16.0 (33).', 'cite_spans': [], 'ref_spans': [], 'section': 'METHODS'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'cite_spans': [], 'ref_spans': [], 'section': 'METHODS'}, {'text': 'The copyright holder for this preprint this version posted April 14, 2020. . https://doi.org/10.1101/2020.04.09.20059154 doi: medRxiv preprint Figure A1 in the Appendix shows a graphical comparison of observed and predicated COVID-19 confirmed cases (in log scale). Table A1 presents results from the cross-fold robustness test for the predictive model. The statistical diagnostic tests as well the patterns from the graphical analysis suggest that the predicative model sufficiently captures the observed data. As can be seen from the table, not only the RMSE, the MAE and the Pseudo-R 2 are consistent across samples, the error margins are also small. Hence, we have used the resulting model to forecast cumulative COVID-19 cases, estimate new infections and deaths from COVID-19 for each of the 47 African countries which have reported at least one case as of March 31, 2020.', 'cite_spans': [], 'ref_spans': [{'start': 143, 'end': 152, 'text': 'Figure A1', 'ref_id': 'FIGREF1'}, {'start': 266, 'end': 274, 'text': 'Table A1', 'ref_id': 'TABREF1'}], 'section': 'METHODS'}, {'text': 'Cumulative COVID-19 cases and point prevalence rates Table 1 presents cumulative COVID-19 cases and prevalence rates for Africa and its five sub-regions for the period April 1 to June 30 2020. By the end of April cumulated infection across the region will reach 2,459,328 cases (CI 95% 424,155 to 14,380,080), and will further increase to 16, 283, 084 (CI 95% 2,718,402 to 98, 358, 798) by June 30, 2020. During the same period, cumulative cases in the hardest hit sub-region in the African continent, Northern Africa, is predicted to increase from 1.3 million at the end of April (CI 95% 207982 to 7,914,479) to over 7.5 million by June 30, 2020. This is followed by Southern, Western, and Central Africa sub-regions. Throughout the forecast period, Eastern Africa is predicated to experience the lowest case load and prevalence rate in the continent throughout the forecast period. . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [{'start': 53, 'end': 60, 'text': 'Table 1', 'ref_id': 'TABREF1'}], 'section': 'RESULTS'}, {'text': \"The copyright holder for this preprint this version posted April 14, 2020. . https://doi.org/10.1101/2020.04.09.20059154 doi: medRxiv preprint Figure 1A shows spatial patterns of cumulative COVID-19 cases by June 30, 2020 (see also Appendix 2A for country estimates). In Northern Africa, the leading contributor to the burden of COVID-19 is Morocco. By the end of June 2020, Morocco will have 4,459,877 (CI 95% 689,547 to 28,845,720) cumulative cases of COVID-19 and this is almost double the estimated number for Algeria, a country with the next highest burden, 2,804,674 (CI 95% 438,629 to 17,935, 144 ) by the end of June 2020. In Southern Africa, South Africa and Swaziland are the leading contributors to the pandemic. By the end of June 2020, the countries are expected to have 2,581366 (CI 95% 404,835 to 16,459, 651) and 254,403 (CI 95% 45,730 to 1,415,274)] cumulative cases, respectively. In the Western Africa subregion, cumulative cases of infection are dominated by Cote d'Ivoire and Ghana, despite Nigeria having a larger population than both countries combined.\", 'cite_spans': [], 'ref_spans': [{'start': 143, 'end': 152, 'text': 'Figure 1A', 'ref_id': 'FIGREF1'}], 'section': '(which was not certified by peer review)'}, {'text': 'To remove the effect of size, we also estimated prevalence rates shown in Panel B of Table 1 by combining the forecasts on cumulative cases with population estimates which we generated for each month from the UN annual population estimates. This suggests that population level prevalence of COVID-19 in Africa is expected to remain under 5 percent (CI 95% 0.6 to 15.9) throughout the prediction period, but there are wide inter-regional differences. In the Northern and Southern Africa sub-regions, cumulative infection rates are expected to be over 15 percent, with the rate in the respective subregions expected to hit as high as 34 percent (CI 95% 0.53 to 21.36) and 19 percent (CI 95% 0.34 to 10.13) by the end of June 30, 2020.', 'cite_spans': [], 'ref_spans': [{'start': 85, 'end': 92, 'text': 'Table 1', 'ref_id': 'TABREF1'}], 'section': '(which was not certified by peer review)'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted April 14, 2020. Sub-regional and country level differences are expected to deepen as the pandemic becomes more established in the region. In the coming three months incidence is expected to increase faster in Southern Africa, followed by Northern and Eastern sub-region. In each sub-region, we expect some countries to be affected more than others and become hotbeds of new infections (see Figure 2B ). Table 3A and Figure 1B ).', 'cite_spans': [], 'ref_spans': [{'start': 449, 'end': 458, 'text': 'Figure 2B', 'ref_id': 'FIGREF2'}, {'start': 462, 'end': 470, 'text': 'Table 3A', 'ref_id': 'TABREF5'}, {'start': 475, 'end': 484, 'text': 'Figure 1B', 'ref_id': 'FIGREF1'}], 'section': '(which was not certified by peer review)'}, {'text': 'Deaths for COVID-19 are estimated for each country, using indirect standardisation techniques after making relevant adjustments to the fatality rates from China and Italy as described in the methods section. As can be seen in Figure 2 , the adjustment shows relatively higher expected mortality using Italy as a standard compared to China, but generally are closer to each other. Hence, our final country level estimates presented in Appendix Table A4 , Figure 1C and the regional summaries presented in . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [{'start': 226, 'end': 234, 'text': 'Figure 2', 'ref_id': 'FIGREF2'}, {'start': 443, 'end': 451, 'text': 'Table A4', 'ref_id': None}, {'start': 454, 'end': 463, 'text': 'Figure 1C', 'ref_id': 'FIGREF1'}], 'section': 'Estimated COVID-19 mortality'}, {'text': \"The copyright holder for this preprint this version posted April 14, 2020. . https://doi.org/10.1101/2020.04.09.20059154 doi: medRxiv preprint Table 3 , were generated using the arithmetic mean of death numbers generated using the two standard populations. The estimates are for the month of April 2020. The first COVID-19 case in Africa was reported on February 14,2020 (2, 3). Our estimates suggest that at the end of April, after two months since the first COVID-19 cases were reported in the region, the number of deaths is expected to reach over 3 thousand deaths (CI 95% 537 to 18482). Most deaths in the region will come from the Northern Africa sub-region, which, as reported earlier, also had the largest number of COVID-19 cases and the highest prevalence rates of all sub-regions in the content. Southern and Western Africa are expected to experience comparable death toll, while deaths in the Eastern Africa sub-region are expected to be the lowest of the sub-regions in Africa. Some countries such as Algeria, Morocco and South Africa are expected to experience the highest number of causalities since the beginning of the pandemic, in excess of 400 deaths. Deaths in the three countries will comprise of almost two-thirds of all expected deaths in the region. In others, such as the Democratic Republic of Congo, Tunisia and Cote d I'vorie death toll will reach between 150 and 300, . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\", 'cite_spans': [], 'ref_spans': [{'start': 143, 'end': 150, 'text': 'Table 3', 'ref_id': 'TABREF5'}], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted April 14, 2020. . https://doi.org/10.1101/2020.04.09.20059154 doi: medRxiv preprint and contribute 20 % of the regional total, while half of the continent (about 29 countries) will experience less than 10 deaths each. The same trend continues in May and June, but with much higher number of expected deaths.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The COVID-19 pandemic has been spreading rapidly across different parts of the world and the African continent has not been spared. Given the global trajectory, what lies ahead in terms of the course and magnitude of infection in Africa in the days and months to come remains speculative. While there have been attempts elsewhere in capturing the trajectory of the COVID-19 pandemic using statistical models, this has not been the case for the African region as whole (4, 6, 8) . To the best of our knowledge, no study, using a robust methodology, provides estimates on future trend of COVID-19 for the entire region or accounts for its local context.', 'cite_spans': [{'start': 468, 'end': 471, 'text': '(4,', 'ref_id': 'BIBREF3'}, {'start': 472, 'end': 474, 'text': '6,', 'ref_id': 'BIBREF5'}, {'start': 475, 'end': 477, 'text': '8)', 'ref_id': 'BIBREF8'}], 'ref_spans': [], 'section': 'DISCUSSION'}, {'text': \"Modeling and predicting the epidemiology and trajectory of a disease such as COVID-19 is a challenging exercise. Firstly, COVID-19 is a newly identified pathogen and knowledge about the characteristics of the virus, including how the disease is spread and the time from exposure to onset of symptoms are still evolving. Secondly, at the time of writing, less than a handful of countries had reached a plateau in their infection rate, and this makes modeling future trends an open-ended exercise (1) . The fact that some people only experience mild symptoms and that even the best health system can only detect and treat those presenting to facilities also means that the available data on 'confirmed' cases represents only a fraction of the true picture of the pandemic.\", 'cite_spans': [{'start': 495, 'end': 498, 'text': '(1)', 'ref_id': 'BIBREF0'}], 'ref_spans': [], 'section': 'DISCUSSION'}, {'text': 'Thirdly, literature on the association between disease prevalence and population level characteristics that are known to be the staple of social and descriptive epidemiology, and the backbone of predictive modeling exercises are yet to emerge for COVID-19 (4) (5) (6) 8) . Further, at the individual level, knowledge on the determinants of COVID-19 morbidity and mortality is limited to selected characteristics such as age and pre-existing conditions, and even then, they are drawn from scanty data. The association with community and population level characteristics is even more limited, hence largely speculative. In this context, the task of developing a multi-country covariate-based predictive model for COVID-19 in any country or region of the world, let alone Africa, is likely to be guided by what is feasible, and to the extent, the available data permits it. It also means that there is a need for constant revision of predictive models as new data becomes available. In our case, the work is further complicated because in Africa data on key covariates are either lacking or when they exist tend to be biased or derived from other global covariate-based modeling exercises (21, 22).', 'cite_spans': [{'start': 256, 'end': 259, 'text': '(4)', 'ref_id': 'BIBREF3'}, {'start': 260, 'end': 263, 'text': '(5)', 'ref_id': 'BIBREF4'}, {'start': 264, 'end': 267, 'text': '(6)', 'ref_id': 'BIBREF5'}, {'start': 268, 'end': 270, 'text': '8)', 'ref_id': 'BIBREF8'}], 'ref_spans': [], 'section': 'DISCUSSION'}, {'text': 'Despite this, we have made efforts to mitigate these limitations, by curating data from various credible sources around the world and assessing them for consistence. In developing our model, we have used data sources from 193 countries globally and used statistical relationships to fill data gaps on covariates in the Africa region. Additionally, our model performance was assessed through rigorous out of sample calibration using k-fold cross-validation techniques and obtaining robust results. We have also restricted our forecast to the first few months (April-June) and refrained from long term exercise. This is because first we anticipate new and additional data that is more informative and will lead to better and improved estimates. Second, in the face of sparse data any long-term projection is more likely to be detached from reality and easily become a wild guess, and less useful to policy actions. Third, if the countries of the region will not have been able to put effective strategies to contain the disease before the end of June, the last month of our projection period, the extent and consequence of COVID-19 on the continent would be more than any predictive model can anticipate.', 'cite_spans': [], 'ref_spans': [], 'section': 'DISCUSSION'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'DISCUSSION'}, {'text': 'The copyright holder for this preprint this version posted April 14, 2020. . https://doi.org/10.1101/2020.04.09.20059154 doi: medRxiv preprint Within these caveats, our results show a heterogeneous picture of projected new infections, cumulative cases, and deaths across the African continent. Our study substantially contributes to the growing literature on understanding the trajectory of the COVID-19 pandemic, in addition to conveying an African perspective. This perspective is particularly important at a time of controlling a pandemic given that Africa is not only the second largest continent but also home to 17% of the total world population, with the current population at 1,332,805,306 (34).', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'Among our key findings are that African countries that are relatively more urbanized and have a higher socio-demographic index (SDI) are projected to experience a faster growth of the epidemic, at least during the initial periods. To illustrate this trend, Algeria, Morocco, and Tunisia, in the Northern African sub-region, which have a predominantly urban population, and are in close proximity to the high burden European countries, are projected to experience a higher incidence of COVID-19 infections in comparison to their neighbors. In the Southern Africa sub-region, South Africa, which has a proportionately higher urban population and greater international connectivity, is also projected to lead the rest of the pack in terms of new infections rates. In the Eastern Africa sub-region, Kenya, which is relatively more urbanized has a higher per capita incidence rate in comparison to other countries within the sub-region. However, it is to be noted that some of these countries also have higher living standards and better health system than most countries in the region. Hence the high prevalence rates and estimated deaths in these countries may in part reflect their greater ability to detect and confirm COVI-19 cases.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'On the other hand, countries such as Angola, Botswana and Mozambique, with a lower international connectivity are projected to have lower rates of new infections at the initial periods. Similar trends are observed in Burundi and Tanzania in the Eastern Africa region. However, within a short time, as the epidemic grows many of these countries are projected to have an increased caseload of cumulative infections that are likely to overwhelm their health system capacity. Overall, these trends are consistent with other regions of the world, where since the onset of this pandemic, it has been clear that higher connectivity, particularly through air traffic, and higher population densities are key drivers in the introduction and growth of the epidemic (35-39).', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The impact of population density is clearly illustrated by the projected new infections of COVID-19 in places like Djibouti and Rwanda. To compound this, close to 90% of urban settlements in Africa are comprised of informal settlements, most of which are overcrowded and lack basic amenities such as clean water and sanitation that are critical prevention measures for COVID-19 (40) (41) (42) . Moreover, 60% of jobs are in urban areas are in the informal economy (43) , and if African countries were to implement stringent lockdowns these livelihoods would be largely affected and could lead to significant socioeconomic consequences. For instance, those that lose their jobs are likely to move back to their homes in rural areas, where many of the elderly population reside, putting them at risk. Similarly, as the economy slows down in several countries due to the COVID-19 pandemic, some sections of migrant workers from African countries might be forced to return to their home countries further complicating the dynamics of the pandemic.', 'cite_spans': [{'start': 378, 'end': 382, 'text': '(40)', 'ref_id': 'BIBREF19'}, {'start': 383, 'end': 387, 'text': '(41)', 'ref_id': None}, {'start': 388, 'end': 392, 'text': '(42)', 'ref_id': 'BIBREF21'}, {'start': 464, 'end': 468, 'text': '(43)', 'ref_id': None}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'Therefore, large scale measures that are aimed at limiting population movement, across countries as well as increasing the social distance among populations, are not enough, or at best may proof to be impracticable to address the pandemic in the African context. Comprehensive response measures should be contextualized and seek to address some of the underlying individual and structural factors that are likely to complicate the epidemic within these environments. It is crucial to balance interventions geared towards preventing the spread of the epidemic with the need to maintaining . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted April 14, 2020. . https://doi.org/10.1101/2020.04.09.20059154 doi: medRxiv preprint livelihoods and social cohesion. Measures such as appropriate messaging, provision of adequate water and sanitation subsidies, food as well as targeted restriction of movement (e.g. from urban to rural) would go a long way to mitigate the spread.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'Still, even with all these NPIs measures, a number of countries with weak healthcare delivery systems are expected to have a significant number of infections and fatalities. Therefore, the already overstretched healthcare systems in Africa need to anticipate and prepare to handle an increased number of patients as a consequence of the pandemic, on top of the other common disease conditions that are prevalent in the region. Unfortunately, this is a lesson Africa has already had to learn from the 2014 Ebola outbreak in West Africa. It is estimated there were an additional 11,00 deaths from malaria, HIV/AIDS and TB across Sierra-Leone, Guinea and Liberia (44) , 60% decrease in the number of children treated for diarrhea and acute respiratory infections (45) and 38% increase in maternal mortality in Guinea and another 111% in Liberia (46, 47) .', 'cite_spans': [{'start': 660, 'end': 664, 'text': '(44)', 'ref_id': 'BIBREF24'}, {'start': 760, 'end': 764, 'text': '(45)', 'ref_id': 'BIBREF25'}, {'start': 842, 'end': 846, 'text': '(46,', 'ref_id': 'BIBREF26'}, {'start': 847, 'end': 850, 'text': '47)', 'ref_id': 'BIBREF27'}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'Understanding the factors that accelerate and those that mitigate the spread and mortality related to COVID-19, while accounting for local realities is fundamental for sound public health measures to tackle the pandemic. In fact, lessons from HIV/AIDS programming have taught us that highly effective health interventions fail if the local context is not recognized. Therefore, decision makers in Africa must realize that one size does not fit all and could lead to disastrous outcomes.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'Effective detection and response to epidemics is premised on decision makers having the right epidemiological information. Unfortunately, many countries in Africa have weak health information systems that are not able to collect and process data rapidly to facilitate timely and targeted action. For example, Lesotho, despite being in the same region and with closer proximity to South Africa and Swaziland, countries that are at the epicentre of the COVID-19 pandemic in the Southern Africa subregion, is yet to register a confirmed case. This is a clear example of the gap in diagnostic capacity and health information system than of the absence of infection in the country. Hence, the fight to stop the pandemic in Africa should be focused on both strengthening clinical capacity as well reporting systems with the health care system. Innovations around collecting data through digital platforms make it easier to combine different data streams, thereby helping decision-makers gain a fuller understanding of epidemics and the impact of the interventions (48) (49) (50) (51) (52) . With close to 747 million SIM connections in Sub-Saharan Africa, representing 74% of the population on the continent (53), the response to COVID-19 pandemic provides an opportunity for countries to retool their data collection systems to meet their surveillance needs.', 'cite_spans': [{'start': 1058, 'end': 1062, 'text': '(48)', 'ref_id': 'BIBREF28'}, {'start': 1063, 'end': 1067, 'text': '(49)', 'ref_id': 'BIBREF29'}, {'start': 1068, 'end': 1072, 'text': '(50)', 'ref_id': 'BIBREF30'}, {'start': 1073, 'end': 1077, 'text': '(51)', 'ref_id': 'BIBREF31'}, {'start': 1078, 'end': 1082, 'text': '(52)', 'ref_id': 'BIBREF32'}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': \"In addition, the mobile digital technologies could enable confidential self-reporting of symptoms to healthcare providers' catalyzing further investigation and epidemic control measures before spread beyond defined population subgroups. As an information and health communication tool, mobile platforms could also be leveraged to deliver appropriate health messages to the population relieving direct consultation pressure from health system providers Efficient resource allocation and use is only feasible where there is improved visibility across the various components of the health system through high quality health information systems (54). Ideally, this should cover key areas such as human resources for health, infrastructure and equipment, medicines and other health commodities, and financial resources available. This is particularly needed in epidemic situations, where there are often huge distortions in demand for resources, that if not well addressed could lead to wastage and suboptimal population health outcomes.\", 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted April 14, 2020. . https://doi.org/10.1101/2020.04.09.20059154 doi: medRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': \"Lastly, but equally important in the African context, the global health community and governments addressing the COVID-19 pandemic should be inclusive in their efforts because the efficacy of a response to any emergency is only as good as its weakest link. It is, therefore, important that governments include marginalized communities, such as refugees, migrants and internally displaced populations (IDPs) in their policies and actions. Across Africa, approximately 12.3 million people remain forcibly displaced, including 8.1 million internally displaced people and 4.2 million refugees (55) . Moreover, Sub-Saharan Africa alone hosts more than 26% of the world's refugee population (56, 57) . These populations are likely to face a different set of constraints that restricts their ability to prevent themselves from getting infected and accessing health care. The fact that they also live in under difficult circumstances means that governments across the region should maintain critical supply corridors for humanitarian assistance and address the needs of the most vulnerable within their borders. In the longer term this pandemic should also be a call for African governments to further strengthen engagement with international humanitarian actors in order to foster strong partnership building on local capacities.\", 'cite_spans': [{'start': 589, 'end': 593, 'text': '(55)', 'ref_id': None}, {'start': 685, 'end': 689, 'text': '(56,', 'ref_id': None}, {'start': 690, 'end': 693, 'text': '57)', 'ref_id': None}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'It is true that some of the most successful responses to global health threats including recent ones like Ebola and HIV/AIDS have been characterized by multi-stakeholder partnerships. African health systems have been in the forefront in the implementation of such programs and would be best served to leverage that experience and resources in responding to COVID-19. Bringing all stakeholders together to ensure effective coordination, pooling of resources and delivery of evidence-based interventions is imperative for any sustainable response to such pandemic situations.', 'cite_spans': [], 'ref_spans': [], 'section': 'CONCLUSIONS'}, {'text': 'Within, the African context policy makers should also consider (a) impact of strategies that could potentially deepen health inequalities and (b) continuously use data driven approaches to identifying their unique most at risk/ vulnerable groups e.g. IDP, HIV positive, food insecure to support them in an equitable manner and bring them into the COVID-19 prevention framework. At a time of a pandemic no one community can be marginalized.', 'cite_spans': [], 'ref_spans': [], 'section': 'CONCLUSIONS'}, {'text': 'Finally, in as much as health systems are dealing with an emergence situation, measurement should not be relegated to a nice to have, but as a fundamental tool to guide strategic response actions. The measurement framework should be agile yet comprehensive, rather than being piecemeal. It should seek to combine data from multiple sources within the healthcare system into a joint assessment framework such that a consistent and informative narrative emerges to tackle the epidemic. The ageold call for accurate health metrics to address population health still remains valid. Our best estimates presented in the paper are forecast, and their ability to capture reality is as good as the underlying data. It is thus imperative to review the model and the results as new data becomes available, and update the estimates accordingly.', 'cite_spans': [], 'ref_spans': [], 'section': 'CONCLUSIONS'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'CONCLUSIONS'}, {'text': 'The copyright holder for this preprint this version posted April 14, 2020. ', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'Yohannes Kinfu (YK), Uzma Alam (UA) and Tom Achoki (TA) report no conflicts of interest.', 'cite_spans': [], 'ref_spans': [], 'section': 'CONFLICTS OF INTEREST'}, {'text': 'YK conceptualised the study, designed methods and approach; undertook statistical analysis and drafted the manuscript. UA synthesised the literature, contributed to conceptualisation and drafted the manuscript. TA accessed data, contributed to methods and design, undertook the statistical analysis and drafted the manuscript. All authors contributed to the discussion, read and approved the final draft.', 'cite_spans': [], 'ref_spans': [], 'section': 'AUTHORS CONTRIBUTIONS'}, {'text': 'This study was finalised when YK was in a government designated quarantine centre for COVID-19 at Sydney InterContinental Hotel, upon his return to Australia. YK would like to acknowledge the Commonwealth Government of Australia and the New South Wales Government for supporting his stay at the hotel free of charge and creating an environment that enabled him to continue with his research and teaching activities, while doing their part to combat COVID-19.', 'cite_spans': [], 'ref_spans': [], 'section': 'ACKNOWLEDGMENTS'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'ACKNOWLEDGMENTS'}, {'text': 'The copyright holder for this preprint this version posted April 14, 2020. . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted April 14, 2020. . https://doi.org/10.1101/2020.04.09.20059154 doi: medRxiv preprint . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted April 14, 2020. . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted April 14, 2020. . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted April 14, 2020. . https://doi.org/10.1101/2020.04.09.20059154 doi: medRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}], 'paragraphs': [{'text': 'NPIs directly influence the course of the COVID-19 pandemic, including the rate of spread, and the expected duration of the pandemic. However, several characteristics of the virus such as incubation period (the time between infection and symptom onset), serial interval (the time between symptom onset of a primary and secondary case) the extent of asymptomatic cases, the possibility of presymptomatic infectiousness, and the case fatality rate (CFR), and also the possible role of weather in transmission, all remain ambiguous or mostly unknown (7, 8) . Estimates for COVID-19 CFR range from 0.3-1% (9) . Asymptomatic or mild presentation comprises the bulk of the reported cases, which is an estimated at 80% (10) . Longitudinal viremia measurements from a small study (sample size of 16) , suggests that there are high enough viral loads to trigger pre-symptomatic infectiousness for 1-2 days before the onset of symptoms (11) . With the lack of clinical studies measuring viremia, the infectious period also remains largely unknown, with estimates ranging from few days to 10 days or more after the incubation period (11) .', 'id': '00003'}], 'title': 'COVID-19 pandemic in the African continent: forecasts of cumulative cases, new infections, and mortality', 'sha': 'aca748230b163c9105c9a9b2f3ca97a096badb9b', 'doi': '10.1101/2020.04.09.20059154', 'score': 9.03439998626709}, {'rank': 221, 'abstract': 'The COVID-19 pandemic has resulted in a major shift in how breast services are being utilised and managed. The guidelines relating to this have been published by recognised medical associations from the UK, Europe and the USA, addressing many aspects of breast cancer care. There is an awareness that there may be limitations in resources during this time, and therefore, prioritisation strategies have been identified to ensure that breast cancer patients are appropriately treated, whilst reducing the risk of infection from COVID-19 to both patients and staff. Equally important is the consideration of how breast cancer services can be safely re-introduced, as infection rates from COVID-19 decline. However, until there is a definite management strategy for COVID-19, such as a vaccine being developed, it is likely that there will still be a significant impact from COVID-19 on breast cancer care. This paper, therefore, aims to highlight the current guidance and evidence regarding breast cancer management in the era of COVID-19, and also aims to look at future management strategies in this period of uncertainty.', 'paragraphs': [{'text': 'The first documented cases of COVID-19 were identified in the Wuhan province, China, in December 2019, and there are now currently over 5 million cases confirmed worldwide [1–3]. COVID-19 is caused by the severe acute respiratory coronavirus 2 (SARS-CoV-2), and may present with a constellation of symptoms, including pyrexia, cough and respiratory distress [2, 4]. It is approximately twice as infective as the seasonal influenza virus with an R0 of 2.2 (average number of patients infected from index patient) [2, 5, 6]. This is felt to be due to the long incubation period of COVID-19, where individuals can be asymptomatic for up to 14 days, yet still have potential to transmit the disease to others. The SARS-CoV-2 virus is primarily spread in large respiratory droplets but has also been detected in both blood and stool [5, 7–9].', 'id': '00001'}], 'title': 'A practical approach to the management of breast cancer in the COVID-19 era and beyond', 'sha': '76ddc5e3d5454d86b55b433953dcb1a296914abd', 'doi': '10.3332/ecancer.2020.1059', 'score': 9.03380012512207}, {'rank': 222, 'paragraphs': [{'text': 'Taken together, we find that alternating lock-down is highly efficient, thanks to its resonance with the natural COVID-19 disease cycle. Under ideal conditions, i.e. f, σ → 0 is eliminates the disease within a single two week cycle, guaranteeing that all invisible spreaders remain locked down. Our results, however, indicate that its efficiency extends beyond that. The method is found to be robust both against imperfect implementation (large f ) and against individual variability in incubation times (large σ). It should therefore be considered as a leading strategy, tailored for the mitigation of COVID-19.', 'id': '00022'}], 'title': 'Alternating quarantine for sustainable mitigation of COVID-19', 'sha': 'b923a8fc7d406d0f149ecd5293d1a72dec7ee526', 'doi': '', 'score': 9.032699584960938}, {'rank': 223, 'abstract': 'The coronavirus disease-19 (COVID-19) is caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). The long incubation period of this new virus, which is mostly asymptomatic yet contagious, is a key reason for its rapid spread across the world. Currently, there is no worldwide-approved treatment for COVID-19. Therefore, the clinical and scientific communities have joint efforts to reduce the severe impact of the outbreak. Research on previous emerging infectious diseases have created valuable knowledge that is being exploited for drug repurposing and accelerated vaccine development. Nevertheless, it is important to generate knowledge on SARS-CoV-2 mechanisms of infection and its impact on host immunity, to guide the design of COVID-19 specific therapeutics and vaccines suitable for mass immunization. Nanoscale delivery systems are expected to play a paramount role in the success of these prophylactic and therapeutic approaches. This Review provides an overview of SARS-CoV-2 pathogenesis and examines immune-mediated approaches currently explored for COVID-19 treatments, with an emphasis on nanotechnological tools.', 'body_text': [{'text': 'The coronavirus disease-19 (COVID-19) pandemic caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), was first reported in Wuhan, China in December 2019. Since then, it has spread globally, already infecting millions of people worldwide. As of 30 June 2020, 213 countries have reported COVID-19 cases, with a total number that reached above 10.3 million, the most being in the USA (2.6 million), Brazil (1.4 million), Russia (640 thousand), India (548 thousand) and UK (314 thousand). USA has the highest number of deaths (126 thousand) followed by Brazil (58 thousand), UK (44 thousand) and Italy (35 thousand). The worldwide case fatality rate across all communities is 4.9%.', 'cite_spans': [], 'section': 'Main', 'ref_spans': []}, {'text': 'Coronaviruses (CoVs) are enveloped viruses entrapping non-segmented, positive-sense and single-stranded ribonucleic acid (ssRNA). Their genome size ranges from 26 to 32 kb, being the largest known RNA virus. SARS-CoV-2 3’ terminus encodes structural proteins, including spike (S) glycoproteins1,2, membrane (M) glycoproteins3, as well as envelope (E)4 and nucleocapsid (N) proteins2,5 (Fig. 1). In addition to the genes encoding structural proteins, there are specific genomic regions encoding for viral proteins required for replication6, in addition to other non-structural proteins, such as the papain-like protease (PLpro)7 and coronavirus main protease (3CLpro)8.', 'cite_spans': [{'start': 293, 'end': 294, 'mention': '1', 'ref_id': 'BIBREF0'}, {'start': 295, 'end': 296, 'mention': '2', 'ref_id': 'BIBREF111'}, {'start': 324, 'end': 325, 'mention': '3', 'ref_id': 'BIBREF128'}, {'start': 350, 'end': 351, 'mention': '4', 'ref_id': 'BIBREF139'}, {'start': 381, 'end': 382, 'mention': '2', 'ref_id': 'BIBREF111'}, {'start': 383, 'end': 384, 'mention': '5', 'ref_id': 'BIBREF150'}, {'start': 537, 'end': 538, 'mention': '6', 'ref_id': 'BIBREF161'}, {'start': 626, 'end': 627, 'mention': '7', 'ref_id': 'BIBREF172'}, {'start': 666, 'end': 667, 'mention': '8', 'ref_id': 'BIBREF183'}], 'section': 'Main', 'ref_spans': [{'start': 391, 'end': 392, 'mention': '1', 'ref_id': 'FIGREF0'}]}, {'text': 'According to the Center for Disease Control and Prevention (CDC), the incubation period following infection is 2–14 days, with an estimated median of 5.1 days9,10. However, cases with longer incubation of 24 days have been reported11. The long incubation period is the primary reason for the massive infection, as it is mostly asymptomatic yet contagious10. Although the estimated patients’ age average is ~70, all age groups are susceptible to this virus. However, the elder population (>60) and people with comorbidities are more likely to develop severe symptoms upon infection12.', 'cite_spans': [{'start': 158, 'end': 159, 'mention': '9', 'ref_id': 'BIBREF194'}, {'start': 160, 'end': 162, 'mention': '10', 'ref_id': 'BIBREF1'}, {'start': 231, 'end': 233, 'mention': '11', 'ref_id': 'BIBREF12'}, {'start': 354, 'end': 356, 'mention': '10', 'ref_id': 'BIBREF1'}, {'start': 580, 'end': 582, 'mention': '12', 'ref_id': 'BIBREF23'}], 'section': 'Main', 'ref_spans': []}, {'text': 'Much like previous CoVs, severe acute respiratory syndrome (SARS) and Middle East respiratory ryndrome (MERS), SARS-CoV-2 is predominantly infecting the lower airways, ranging from mild respiratory illness to severe acute respiratory syndrome and septic shock in advanced stages6. The most commonly reported symptoms are fever, dry cough, dyspnea, fatigue and myalgia, which are early characteristics of the most frequent manifestation of SARS-CoV-2 infection, pneumonia13–15. Physicians and pathologists are also reporting devastating damage to the cardiovascular system, gut, kidneys and brain16,17. Of importance is the recently observed tendency of COVID-19 patients´ blood to clot, which leads to vessel constriction and ultimately can result in pulmonary embolism or large-vessel ischemic stroke, in addition to ischemia in fingers and toes16,18.', 'cite_spans': [{'start': 278, 'end': 279, 'mention': '6', 'ref_id': 'BIBREF161'}, {'start': 470, 'end': 472, 'mention': '13', 'ref_id': 'BIBREF34'}, {'start': 473, 'end': 475, 'mention': '15', 'ref_id': 'BIBREF56'}, {'start': 595, 'end': 597, 'mention': '16', 'ref_id': 'BIBREF67'}, {'start': 598, 'end': 600, 'mention': '17', 'ref_id': 'BIBREF78'}, {'start': 846, 'end': 848, 'mention': '16', 'ref_id': 'BIBREF67'}, {'start': 849, 'end': 851, 'mention': '18', 'ref_id': 'BIBREF89'}], 'section': 'Main', 'ref_spans': []}, {'text': 'On 7 May, remdesivir was approved for COVID-19 in Japan19. In the rest of the world, there are no specific treatments or vaccines available for COVID-19. To the best of our knowledge, the only available options are the condition marketing authorization to remdesivir recommended by the European Medicines Agency (EMA) on 25 June, as well as the emergency use authorizations (EUA) provided by the US Food and Drug Administration (FDA) on 1 May for remdesivir20, on 30 March for the decades-old malaria drugs hydroxychloroquine sulfate and chloroquine phosphate21, and on 24 March for the infusion of plasma of COVID-19 convalescent patients22. The latter was supported by data obtained on only 5 critically ill COVID-19 patients23, which had high levels of neutralizing antibodies, raising the realistic hope of producing virus-specific enriched immunoglobulins, similarly to the ones used for tetanus, hepatitis A/B, cytomegalovirus, varicella and measles. On 2 June, a clinical trial designed to evaluate the safety and tolerability of the antibody LY-CoV555 was posted. This IgG1 antibody is the first potential preventive and therapeutic medicine developed to specifically target the spike protein on the SARS-CoV-224.', 'cite_spans': [{'start': 55, 'end': 57, 'mention': '19', 'ref_id': 'BIBREF100'}, {'start': 457, 'end': 459, 'mention': '20', 'ref_id': 'BIBREF112'}, {'start': 559, 'end': 561, 'mention': '21', 'ref_id': 'BIBREF119'}, {'start': 639, 'end': 641, 'mention': '22', 'ref_id': 'BIBREF120'}, {'start': 727, 'end': 729, 'mention': '23', 'ref_id': 'BIBREF121'}, {'start': 1218, 'end': 1220, 'mention': '24', 'ref_id': 'BIBREF122'}], 'section': 'Main', 'ref_spans': []}, {'text': 'The rapid spread of SARS-CoV-2 requires accelerated development timelines for COVID-19 vaccines and therapeutic candidates to enter expeditiously into phase 1 clinical trials. Hence, the type and extent of preclinical and preliminary clinical data needed to inform these clinical development programs must be weighed against the overall risk–benefit assessment of this unmet medical need. In fact, so far, most published clinical trials on prophylactic and therapeutic strategies against COVID-19 have countless weaknesses, such as not being randomized nor controlled, studying small populations, and/or lacking a placebo group, making it difficult to draw conclusive outcomes.', 'cite_spans': [], 'section': 'Main', 'ref_spans': []}, {'text': 'As of 30 June, 2,351 clinical trials are listed in ClinicalTrials.gov, using COVID-19, 2019-nCOV, SARS-CoV-2 or 2019 novel coronavirus as search terms (Fig. 2). Most of trial locations are reported in Europe (709), North America (441), Asia (200) and the Middle East (94).', 'cite_spans': [], 'section': 'Main', 'ref_spans': [{'start': 157, 'end': 158, 'mention': '2', 'ref_id': 'FIGREF1'}]}, {'text': 'The strong commitment of the scientific community in addressing COVID-19 disease and global social and economic impact led already to the publication of 27,255 manuscripts (as of 30 June, using keywords COVID-19 or SARS-CoV-2) and submission of 5,891 preprints (4,701 medRxiv, 1,190 bioRxiv). Thus, the large amount of information generated and reported in the last months raises the need for a consensus analysis.', 'cite_spans': [], 'section': 'Main', 'ref_spans': []}, {'text': 'The similarities found between SARS-CoV-2 and SARS-CoV, MERS-CoV, or human immunodeficiency viruses (HIV) can enhance the development of potential therapeutic approaches and advance the understanding of the virus mechanism of action25–28. Currently, major efforts are being invested around the world by commercial vaccine manufacturers, pharmaceutical companies, biotech entities and laboratories at academic institutions to accelerate the discovery and development of prophylactic and therapeutic solutions against SARS-CoV-2 infection using technologies and platforms mostly based on therapeutic modalities developed against these previous infectious diseases. These include repurposing of antivirals (for example, viral polymerase and protease inhibitors29,30), anti-inflammatory drugs31,32, monoclonal antibodies (mAb)27, antibiotics and immune modulators (for example, anti-interleukin-6 (IL-6) agents33,34, PEGylated interferon (IFN)-alpha and beta (PEG-IFN-α/β)35,36), activators of toll-like receptors (TLR)37 and vaccines (for example, based on whole virus38, messenger RNA (mRNA)39, DNA40,41, recombinant proteins42 and peptides43, including when packaged in diverse delivery systems, as well as viral-vectored vaccines44,45 and virus-like particles (VLP)46).', 'cite_spans': [{'start': 232, 'end': 234, 'mention': '25', 'ref_id': 'BIBREF123'}, {'start': 235, 'end': 237, 'mention': '28', 'ref_id': 'BIBREF126'}, {'start': 757, 'end': 759, 'mention': '29', 'ref_id': 'BIBREF127'}, {'start': 760, 'end': 762, 'mention': '30', 'ref_id': 'BIBREF129'}, {'start': 788, 'end': 790, 'mention': '31', 'ref_id': 'BIBREF130'}, {'start': 791, 'end': 793, 'mention': '32', 'ref_id': 'BIBREF131'}, {'start': 822, 'end': 824, 'mention': '27', 'ref_id': 'BIBREF125'}, {'start': 906, 'end': 908, 'mention': '33', 'ref_id': 'BIBREF132'}, {'start': 909, 'end': 911, 'mention': '34', 'ref_id': 'BIBREF133'}, {'start': 968, 'end': 970, 'mention': '35', 'ref_id': 'BIBREF134'}, {'start': 971, 'end': 973, 'mention': '36', 'ref_id': 'BIBREF135'}, {'start': 1015, 'end': 1017, 'mention': '37', 'ref_id': 'BIBREF136'}, {'start': 1065, 'end': 1067, 'mention': '38', 'ref_id': 'BIBREF137'}, {'start': 1089, 'end': 1091, 'mention': '39', 'ref_id': 'BIBREF138'}, {'start': 1096, 'end': 1098, 'mention': '40', 'ref_id': 'BIBREF140'}, {'start': 1099, 'end': 1101, 'mention': '41', 'ref_id': 'BIBREF141'}, {'start': 1123, 'end': 1125, 'mention': '42', 'ref_id': 'BIBREF142'}, {'start': 1138, 'end': 1140, 'mention': '43', 'ref_id': 'BIBREF143'}, {'start': 1229, 'end': 1231, 'mention': '44', 'ref_id': 'BIBREF144'}, {'start': 1232, 'end': 1234, 'mention': '45', 'ref_id': 'BIBREF145'}, {'start': 1265, 'end': 1267, 'mention': '46', 'ref_id': 'BIBREF146'}], 'section': 'Potential repurposed treatments', 'ref_spans': []}, {'text': 'Hydroxychloroquine sulfate and chloroquine phosphate are anti-malaria drugs shown to change the acidic conditions of organelles in mammalian cell culture studies47, as well as to inhibit the terminal glycosylation of ACE2 in vitro against SARS-CoV48, indicating its possible role in preventing the fusion of the virus with the cell membrane and thus blocking SARS-CoV-2 infection. Hydroxychloroquine sulfate was shown to affect immune cell activation and phenotype against lupus erythematosus49, which indicates its possible impact on the modulation of host immune response during SARS-CoV-2 infection50–52. Hydroxychloroquine sulfate is also being tested in combination with an array of drugs, amongst them the antibiotic azithromycin, to treat COVID-19 patients53.', 'cite_spans': [{'start': 161, 'end': 163, 'mention': '47', 'ref_id': 'BIBREF147'}, {'start': 247, 'end': 249, 'mention': '48', 'ref_id': 'BIBREF148'}, {'start': 492, 'end': 494, 'mention': '49', 'ref_id': 'BIBREF149'}, {'start': 601, 'end': 603, 'mention': '50', 'ref_id': 'BIBREF151'}, {'start': 604, 'end': 606, 'mention': '52', 'ref_id': 'BIBREF153'}, {'start': 763, 'end': 765, 'mention': '53', 'ref_id': 'BIBREF154'}], 'section': 'Potential repurposed treatments', 'ref_spans': []}, {'text': 'Viruses commonly use specific enzymes that act in mechanisms distinct from human cell biology, hence they may be used as specific targets for the treatment of viral infections. In vitro studies have reported that umifenovir inhibits SARS entry into target cells by disturbing the angiotensin-converting enzyme 2 (ACE2)/S protein interaction, further inhibiting fusion of viral envelope54, also being effective in vivo against H1N155 and influenza A viruses56. Umifenovir is now being considered as a COVID-19 treatment in combination with protease inhibitors14,57.', 'cite_spans': [{'start': 385, 'end': 387, 'mention': '54', 'ref_id': 'BIBREF155'}, {'start': 430, 'end': 432, 'mention': '55', 'ref_id': 'BIBREF156'}, {'start': 456, 'end': 458, 'mention': '56', 'ref_id': 'BIBREF157'}, {'start': 558, 'end': 560, 'mention': '14', 'ref_id': 'BIBREF45'}, {'start': 561, 'end': 563, 'mention': '57', 'ref_id': 'BIBREF158'}], 'section': 'Potential repurposed treatments', 'ref_spans': []}, {'text': 'Lopinavir and ritonavir (LPV/RTV) are protease inhibitors used for the treatment of HIV infections by inhibiting the HIV protease activity, which is essential for virus maturation. Both are effective against MERS-CoV in mice58. Cobicistat is a cytochrome P450 (CYP) 3A inhibitor used as a pharmacokinetic enhancer in combination with ritonavir for HIV treatment29,30. Other protease inhibitors are under investigation, such as ASC09 and darunavir59. Ribavirin is a broad-spectrum antiviral drug currently used as a standard of care against hepatitis C virus, in combination with PEG-IFN-α60. It is a ribonucleic analogue that prevents viral RNA replication61.', 'cite_spans': [{'start': 224, 'end': 226, 'mention': '58', 'ref_id': 'BIBREF159'}, {'start': 361, 'end': 363, 'mention': '29', 'ref_id': 'BIBREF127'}, {'start': 364, 'end': 366, 'mention': '30', 'ref_id': 'BIBREF129'}, {'start': 446, 'end': 448, 'mention': '59', 'ref_id': 'BIBREF160'}, {'start': 588, 'end': 590, 'mention': '60', 'ref_id': 'BIBREF162'}, {'start': 656, 'end': 658, 'mention': '61', 'ref_id': 'BIBREF163'}], 'section': 'Potential repurposed treatments', 'ref_spans': []}, {'text': 'One of the most investigated drugs at the moment is remdesivir, an adenosine nucleotide analogue that interferes with the viral RNA-polymerase activity, recently approved for COVID-19 treatment in Japan. Remdesivir was shown to be effective against the SARS-CoV virus in vitro and is under evaluation in clinical trials62. Preliminary data on a randomized, placebo-controlled trial of remdesivir enrolling 1063 patients have been released on 22 May, showing that the drug was overall safe and effective for COVID-19 patients, leading to a shorter time to recovery than that obtained in the placebo group63. Similarly, favipiravir is a guanine analogue that also acts as an RNA polymerase inhibitor, currently used for influenza treatment, and 38 clinical trials were recently initiated to assess its effect against COVID-1964,65. Other antivirals currently being considered in COVID-19 clinical trials include the neuraminidase inhibitor oseltamivir and the viral endonuclease inhibitor baloxavir marboxil, both used against influenza infection66. In addition, azvudine is a reverse transcriptase inhibitor developed for HIV treatment that may also interfere with the viral replication67.', 'cite_spans': [{'start': 319, 'end': 321, 'mention': '62', 'ref_id': 'BIBREF164'}, {'start': 603, 'end': 605, 'mention': '63', 'ref_id': 'BIBREF165'}, {'start': 823, 'end': 825, 'mention': '64', 'ref_id': 'BIBREF166'}, {'start': 826, 'end': 828, 'mention': '65', 'ref_id': 'BIBREF167'}, {'start': 1044, 'end': 1046, 'mention': '66', 'ref_id': 'BIBREF168'}, {'start': 1185, 'end': 1187, 'mention': '67', 'ref_id': 'BIBREF169'}], 'section': 'Potential repurposed treatments', 'ref_spans': []}, {'text': 'Additional clinical trials are exploring the use of anti-inflammatory agents to prevent COVID-19 patients from developing severe lung inflammation. Those include corticosteroids and non-steroid immune suppressive agents, such as indomethacin and barcitinib68, in addition to mAb targeting inflammatory cytokines, such as IL-6 and complement protein 5 (C5). On 16 June, the RECOVERY study announced the first findings of this randomized and controlled clinical trial, which show that the drug dexamethasone reduced the death of COVID-19 ventilated patients by one-third, supporting its potential use as standard of care in these patients69.', 'cite_spans': [{'start': 256, 'end': 258, 'mention': '68', 'ref_id': 'BIBREF170'}, {'start': 636, 'end': 638, 'mention': '69', 'ref_id': 'BIBREF171'}], 'section': 'Potential repurposed treatments', 'ref_spans': []}, {'text': 'Interestingly, angiotensin-receptor blockers (ARB) and angiotensin-converting enzyme (ACE) inhibitors were shown to increase the expression of ACE2, thus potentially enhancing cell infection by SARS-CoV-2. However, a large population-based study, carried out on 6272 COVID-19 patients, did not find a correlation between the use of ACE inhibitors or ARB and SARS-CoV-2 infection70. Therefore, besides the limited data available, currently, there is no clear evidence that supports the discontinuation of these drugs, considering the potential risk that this action could bring to patients with a history of hypertension, heart failure, post-myocardial infarction state and chronic kidney disease.', 'cite_spans': [{'start': 378, 'end': 380, 'mention': '70', 'ref_id': 'BIBREF173'}], 'section': 'Potential repurposed treatments', 'ref_spans': []}, {'text': 'Cell infection by pathogenic agents may trigger host humoral and cellular immunities essential to eliminate the viral infection. However, an uncontrolled or insufficient immune response may lead to immunopathology and cause severe damage to patients71. A deeper understanding of the immune response induced by SARS-CoV-2 infection may lead to new immunotherapies while reducing the potential risk of inflammation.', 'cite_spans': [{'start': 249, 'end': 251, 'mention': '71', 'ref_id': 'BIBREF174'}], 'section': 'SARS-CoV-2 mechanisms of infection and immunity', 'ref_spans': []}, {'text': 'The best known cellular infection mechanism of SARS-CoV-2 is mediated by the cell surface receptor ACE2 (Fig. 3), similarly to SARS-CoV65,72. Several studies have shown that the SARS-CoV-2 was able to infect cells that were genetically modified to express solely the ACE2 receptor30,73. As this ACE2 receptor is predominantly found in the human epithelia of lung and small intestine, the SARS-CoV-2 is more likely to infect the respiratory and gastrointestinal tracts72,74. The preprint posted by Hikmet et al. indicates that the expression of ACE2 was also found in glandular cells of the seminal vesicle, renal proximal tubules, cardiomyocytes, testicular Sertoli cells, Leydig cells and gallbladder epithelium75. Moreover, it has been suggested that the brain may also be infected by this virus, as COVID-19 patients present neurological signs, such as the hyposmia at early stages of infection, but also nausea, vomiting, headache and cerebral damage14 at severe situations. SARS-CoV was indeed found in the brain of animals and patients76. It has been suggested that SARS-CoV-2 enters into the brain through the cerebral circulation, due to the presence of the ACE2 receptor on the endothelium, or from the cribriform plate close to the olfactory bulb, which can justify at least in part the altered sense of smell. In fact, glial cells and neurons are known to express ACE277, being, therefore, potential targets for SARS-CoV-2.', 'cite_spans': [{'start': 135, 'end': 137, 'mention': '65', 'ref_id': 'BIBREF167'}, {'start': 138, 'end': 140, 'mention': '72', 'ref_id': 'BIBREF175'}, {'start': 280, 'end': 282, 'mention': '30', 'ref_id': 'BIBREF129'}, {'start': 283, 'end': 285, 'mention': '73', 'ref_id': 'BIBREF176'}, {'start': 467, 'end': 469, 'mention': '72', 'ref_id': 'BIBREF175'}, {'start': 470, 'end': 472, 'mention': '74', 'ref_id': 'BIBREF177'}, {'start': 712, 'end': 714, 'mention': '75', 'ref_id': 'BIBREF178'}, {'start': 954, 'end': 956, 'mention': '14', 'ref_id': 'BIBREF45'}, {'start': 1041, 'end': 1043, 'mention': '76', 'ref_id': 'BIBREF179'}, {'start': 1379, 'end': 1381, 'mention': '77', 'ref_id': 'BIBREF180'}], 'section': 'SARS-CoV-2 mechanisms of infection and immunity', 'ref_spans': [{'start': 110, 'end': 111, 'mention': '3', 'ref_id': 'FIGREF2'}]}, {'text': 'L-SIGN (CD209L) is a less explored potential SARS-CoV-2 receptor. It is a type II transmembrane glycoprotein of the C-type lectin family, mainly expressed in human lung alveolar epithelial type II cells and endothelial cells. This receptor was shown to mediate SARS-CoV entry into host cells, but at a lesser extent than the ACE2 receptor78,79. Therefore, CD209L is likely to mediate SARS-CoV-2 entry as well.', 'cite_spans': [{'start': 338, 'end': 340, 'mention': '78', 'ref_id': 'BIBREF181'}, {'start': 341, 'end': 343, 'mention': '79', 'ref_id': 'BIBREF182'}], 'section': 'SARS-CoV-2 mechanisms of infection and immunity', 'ref_spans': []}, {'text': 'The surface-exposed S glycoprotein is the first viral component that interacts with host cells, being encoded by the S gene, the most variable region of the SARS-CoV genome30. The S protein is a major component accounting for viral binding, fusion and further entry into target cells. It comprises two main functional units. The SARS-CoV-2 S1 subunit contains the receptor-binding domain (RBD), which affinity for binding to host receptor varies among different CoVs, therefore affecting the infection and thereby dictating disease severity6,80. The SARS-CoV-2 S2 subunit is involved in the virus fusion with the host-cell membrane6.', 'cite_spans': [{'start': 172, 'end': 174, 'mention': '30', 'ref_id': 'BIBREF129'}, {'start': 540, 'end': 541, 'mention': '6', 'ref_id': 'BIBREF161'}, {'start': 542, 'end': 544, 'mention': '80', 'ref_id': 'BIBREF184'}, {'start': 631, 'end': 632, 'mention': '6', 'ref_id': 'BIBREF161'}], 'section': 'SARS-CoV-2 mechanisms of infection and immunity', 'ref_spans': []}, {'text': 'The binding and/or entering mechanisms suggested for SARS-CoV-2 include several steps similar to those identified for SARS-CoV and MERS-CoV infection due to the RBD genomic similarity26. The virus entry is a complex process that starts with the recognition of the RBD 394 glutamine residue by the lysine 31 residue on the human ACE2, further adopting specific conformations that will allow a fast dissociation of the S1 and S2 subunits80,81 (Fig. 3). This proteolytic priming of the S protein by host proteases is also crucial to release the fusion peptide, enabling the viral and host cell membrane fusion, and subsequent release of SARS-CoV-2 RNA into the cytoplasm28,82,83.', 'cite_spans': [{'start': 183, 'end': 185, 'mention': '26', 'ref_id': 'BIBREF124'}, {'start': 435, 'end': 437, 'mention': '80', 'ref_id': 'BIBREF184'}, {'start': 438, 'end': 440, 'mention': '81', 'ref_id': 'BIBREF185'}, {'start': 667, 'end': 669, 'mention': '28', 'ref_id': 'BIBREF126'}, {'start': 670, 'end': 672, 'mention': '82', 'ref_id': 'BIBREF186'}, {'start': 673, 'end': 675, 'mention': '83', 'ref_id': 'BIBREF187'}], 'section': 'SARS-CoV-2 mechanisms of infection and immunity', 'ref_spans': [{'start': 447, 'end': 448, 'mention': '3', 'ref_id': 'FIGREF2'}]}, {'text': 'It has been shown that the inhibition or depletion of host proteases blocked the entry of the virus in vitro, while protease treatment of target-cell-associated virus led to the opposite effect82,84–86. The transmembrane protease serine 2 (TMPRSS2) expressed in human lung cells is one of these proteases. The knockout of TMPRSS2 decreased the viral entry in cell cultures86, but it was not enough to prevent the SARS-CoV-2 infection. An additional study showed that the complete inhibition of viral entry was achieved when the camostat mesylate, a clinically approved serine protease inhibitor, was combined with cathepsin B/L inhibitors. Previous research on SARS-CoV infection exploited other proteases, such as cathepsin L and B, for cell entry87. This study showed that the inhibition of the pH-sensitive endosomal protease cathepsin L activated SARS-CoV membrane fusion in vitro, indicating that the proteolysis of this cathepsin within the endosomes may be required for viral fusion and entry. Therefore, combinations of host cell protease inhibitors constitute potential therapeutics against COVID-19, becoming an active area of research.', 'cite_spans': [{'start': 193, 'end': 195, 'mention': '82', 'ref_id': 'BIBREF186'}, {'start': 196, 'end': 198, 'mention': '84', 'ref_id': 'BIBREF188'}, {'start': 199, 'end': 201, 'mention': '86', 'ref_id': 'BIBREF190'}, {'start': 372, 'end': 374, 'mention': '86', 'ref_id': 'BIBREF190'}, {'start': 748, 'end': 750, 'mention': '87', 'ref_id': 'BIBREF191'}], 'section': 'SARS-CoV-2 mechanisms of infection and immunity', 'ref_spans': []}, {'text': 'Upon virus entry into host cells, genome RNA serves as mRNA for the first open reading frame (ORF1), being thus translated into viral replicase polyproteins that are later cleaved into small products by viral proteinases. These assemble on double-membrane vesicles that become sites for RNA viral synthesis, which has two stages. The first one constitutes the genome replication, while the second one includes the subgenomic RNA transcription and further translation into structural and accessory proteins from additional ORF78,88. These structural proteins are crucial for RNA synthesis by RNA-dependent RNA polymerases (RdRP) in order to replicate the genomic RNA that will be subsequently released upon fusion with plasma membrane89. The double-membranes enable the viral evasion of host immune responses by lacking the pattern recognition receptors (PRR), the activation of which is crucial for triggering host innate immunity against these viral invading pathogens90.', 'cite_spans': [{'start': 525, 'end': 527, 'mention': '78', 'ref_id': 'BIBREF181'}, {'start': 528, 'end': 530, 'mention': '88', 'ref_id': 'BIBREF192'}, {'start': 733, 'end': 735, 'mention': '89', 'ref_id': 'BIBREF193'}, {'start': 969, 'end': 971, 'mention': '90', 'ref_id': 'BIBREF195'}], 'section': 'SARS-CoV-2 mechanisms of infection and immunity', 'ref_spans': []}, {'text': 'The first line of defence mounted by the host at the entry site, encompassing the induced expression of type I IFN (IFN-I) and other pro-inflammatory cytokines, was reported to be suppressed by SARS-CoV and MERS-CoV36, being related to disease severity.', 'cite_spans': [{'start': 215, 'end': 217, 'mention': '36', 'ref_id': 'BIBREF135'}], 'section': 'SARS-CoV-2 mechanisms of infection and immunity', 'ref_spans': []}, {'text': 'The modulation of host IFN-I response involves the interference with the ubiquitination and degradation of RNA sensor molecules and the nuclear translocation of the IFN regulatory factor 3 (IRF3), as well as with the reduction of the signal transducer and activator of transcription 1 (SATA1) phosphorylation91,92. It has been already reported that high levels of the cytokine IL-6 were correlated with SARS-CoV-2 viral load in the blood of critically ill COVID-19 patients80.', 'cite_spans': [{'start': 308, 'end': 310, 'mention': '91', 'ref_id': 'BIBREF196'}, {'start': 311, 'end': 313, 'mention': '92', 'ref_id': 'BIBREF197'}, {'start': 473, 'end': 475, 'mention': '80', 'ref_id': 'BIBREF184'}], 'section': 'SARS-CoV-2 mechanisms of infection and immunity', 'ref_spans': []}, {'text': 'In addition, the inhibition of cellular components of host immunity has also been reported for SARS-CoV-2 and MERS-CoV infections. The latter led to the downregulation of gene expression related to antigen presentation93, while increased levels of exhausted CD8+ T cells and loss of CD4+ T cell function were found in the peripheral blood of patients infected with SARS-CoV-294.', 'cite_spans': [{'start': 218, 'end': 220, 'mention': '93', 'ref_id': 'BIBREF198'}, {'start': 375, 'end': 377, 'mention': '94', 'ref_id': 'BIBREF199'}], 'section': 'SARS-CoV-2 mechanisms of infection and immunity', 'ref_spans': []}, {'text': 'Therefore, the induction of a balanced host immune response against pathogens in general, and SARS-CoV-2 in particular, is crucial to control and eliminate infection, employing adaptive and innate immune responses, as well as events mediated by the complement system (Supplementary Table 1). On the one hand, an uncontrolled immunity may result in pulmonary tissue damage, functional impairment and reduced lung capacity71. On the other hand, immune insufficiency or misdirection may increase viral replication and cause tissue damage90.', 'cite_spans': [{'start': 420, 'end': 422, 'mention': '71', 'ref_id': 'BIBREF174'}, {'start': 534, 'end': 536, 'mention': '90', 'ref_id': 'BIBREF195'}], 'section': 'SARS-CoV-2 mechanisms of infection and immunity', 'ref_spans': []}, {'text': 'Considering the infection rate of SARS-CoV-2, the prolonged incubation time and the associated high fatality rates, especially in those vulnerable high-risk populations, currently the only possible option to control COVID-19 pandemic is by quarantine, physical distancing and face masks. These measures have been adopted worldwide, but it is becoming an immense economic burden. Historically, vaccines were shown to be effective tools to abolish pandemics, and protect the population, especially high-risk groups, against a variety of viral infections95. Individuals that recover from SARS-CoV-2 infection may develop a protective immunity, considering the immune response triggered by other CoVs and recent findings reported in a small study performed in rhesus macaques, posted as a preprint to bioRxiv96. Even though it is currently unknown if humans previously exposed to the virus are protected against SARS-CoV-2, and there is no sufficient information on the impact of this infection on host immunity, the kinetics of the immune response detected in a patient with mild-to-moderate COVID-19 was recently reported97. The blood of this patient contained increased levels of activated CD4+ and CD8+ T cells, follicular T-helper cells, antibody-secreting cells and IgM/IgG SARS CoV-2-binding antibodies. However, it is not known if those titres are high enough and will remain for a period required to confer protection against re-infection. The antibody response against SARS-CoV was detected at 10–20 days98 and lasts for 2–3 years80.', 'cite_spans': [{'start': 551, 'end': 553, 'mention': '95', 'ref_id': 'BIBREF200'}, {'start': 804, 'end': 806, 'mention': '96', 'ref_id': 'BIBREF201'}, {'start': 1119, 'end': 1121, 'mention': '97', 'ref_id': 'BIBREF202'}, {'start': 1510, 'end': 1512, 'mention': '98', 'ref_id': 'BIBREF203'}, {'start': 1536, 'end': 1538, 'mention': '80', 'ref_id': 'BIBREF184'}], 'section': 'COVID-19 vaccine—a real unmet need', 'ref_spans': []}, {'text': 'Importantly, the clinical condition of five patients infected with SARS-CoV-2 already presenting acute respiratory distress syndrome (ARDS), the most severe acute lung injury, was improved following the administration of plasma containing neutralizing antibodies collected from patients that recovered from this disease99. Caution should be taken while drawing conclusions from a clinical study covering such a small number of individuals, however, these observations support the enthusiasm associated with the development of COVID-19 vaccines.', 'cite_spans': [{'start': 319, 'end': 321, 'mention': '99', 'ref_id': 'BIBREF204'}], 'section': 'COVID-19 vaccine—a real unmet need', 'ref_spans': []}, {'text': 'Comparison of a consensus SARS-CoV-2 protein sequence to sequences of SARS-CoV, bat-SARS-like CoV, and MERS-CoV revealed a high degree of similarity between SARS-CoV and bat-SARS-like CoV. This is in contrast to the low similarity to MERS-CoV (Supplementary Table 2)30,63,100.', 'cite_spans': [{'start': 266, 'end': 268, 'mention': '30', 'ref_id': 'BIBREF129'}, {'start': 269, 'end': 271, 'mention': '63', 'ref_id': 'BIBREF165'}, {'start': 272, 'end': 275, 'mention': '100', 'ref_id': 'BIBREF2'}], 'section': 'Elucidation of SARS-CoV-2 potential epitopes', 'ref_spans': []}, {'text': 'Taken together, the limited information regarding SARS-CoV-2 and the similarity between the two viruses (SARS-CoV and SARS-CoV-2), led to the rational development of vaccines for SARS-CoV-2 based on the already available knowledge on the Betacoronavirus comparisons.', 'cite_spans': [], 'section': 'Elucidation of SARS-CoV-2 potential epitopes', 'ref_spans': []}, {'text': 'Among the SARS-CoV-2 proteins, multiple studies suggest that the S and N structural proteins are the most promising targets for vaccine design1,2,26,63,80,101. Since S protein mediates the virus entry into host cells, it is the main target of neutralizing antibodies upon infection1,2. The N protein is a highly immunogenic and abundantly expressed protein during infection of SARS-CoV1,2. In addition, T cell responses against the S, M and N proteins of the SARS-CoV-2 have been reported to be among the most dominant and long-lasting100.', 'cite_spans': [{'start': 142, 'end': 143, 'mention': '1', 'ref_id': 'BIBREF0'}, {'start': 144, 'end': 145, 'mention': '2', 'ref_id': 'BIBREF111'}, {'start': 146, 'end': 148, 'mention': '26', 'ref_id': 'BIBREF124'}, {'start': 149, 'end': 151, 'mention': '63', 'ref_id': 'BIBREF165'}, {'start': 152, 'end': 154, 'mention': '80', 'ref_id': 'BIBREF184'}, {'start': 155, 'end': 158, 'mention': '101', 'ref_id': 'BIBREF3'}, {'start': 281, 'end': 282, 'mention': '1', 'ref_id': 'BIBREF0'}, {'start': 283, 'end': 284, 'mention': '2', 'ref_id': 'BIBREF111'}, {'start': 385, 'end': 386, 'mention': '1', 'ref_id': 'BIBREF0'}, {'start': 387, 'end': 388, 'mention': '2', 'ref_id': 'BIBREF111'}, {'start': 535, 'end': 538, 'mention': '100', 'ref_id': 'BIBREF2'}], 'section': 'Elucidation of SARS-CoV-2 potential epitopes', 'ref_spans': []}, {'text': 'Diverse SARS-CoV epitope sequences have been defined and compared to the SARS-CoV-2 sequence, most of them being derived from S and N proteins. In addition to the characterization of SARS-CoV-2 genome sequences, vaccine design requires the evaluation of their inherent antigenicity to identify T cell and B cell epitope candidates (Fig. 4). To this end, it is fundamental to understand if these sequences bind to major histocompatibility complex (MHC) class I or class II molecules, and further confirm their recognition and subsequent anti-SARS-CoV-2 immune response induction following the activation of human T cell and B cells.', 'cite_spans': [], 'section': 'Elucidation of SARS-CoV-2 potential epitopes', 'ref_spans': [{'start': 337, 'end': 338, 'mention': '4', 'ref_id': 'FIGREF3'}]}, {'text': 'Across the world, many research groups addressed several B and T cell epitopes, mainly using bioinformatic tools (Virus Pathogen Database and Analysis Resource (ViPR) or The Immune Epitope Database (IEDB)). Although this is a topic under active investigation, the first genome sequences became available on 11 January 202026,100 and those constitute a valuable source to support the development of COVID-19 vaccine candidates.', 'cite_spans': [{'start': 322, 'end': 324, 'mention': '26', 'ref_id': 'BIBREF124'}, {'start': 325, 'end': 328, 'mention': '100', 'ref_id': 'BIBREF2'}], 'section': 'Elucidation of SARS-CoV-2 potential epitopes', 'ref_spans': []}, {'text': 'The large amount of information generated and reported in the last months suggests the need for a consensus analysis. We summarized the most described B and T cell epitopes that may have the potential to elicit a cross-reactive and effective response against SARS-CoV-2 (Supplementary Table 3).', 'cite_spans': [], 'section': 'Elucidation of SARS-CoV-2 potential epitopes', 'ref_spans': []}, {'text': 'The homology of MERS-CoV, SARS-CoV, and the newly emerging SARS-CoV-2, as well as the similar clinical development of these diseases that starts by a pulmonary inflammatory state followed by fibrosis and subsequent compromise of lung function, are driving the attention of the scientific and clinical communities to the use of immune-modulators, which has already achieved remarkable success149. The immune-mediated targets vary between the different types of viruses, as in some infections this pathogen has a major role in host tissue/organ damage, while in others it is the systemic and local immune activation mounted upon infection that impairs host repair function due to hyperstimulation149.', 'cite_spans': [{'start': 391, 'end': 394, 'mention': '149', 'ref_id': 'BIBREF55'}, {'start': 694, 'end': 697, 'mention': '149', 'ref_id': 'BIBREF55'}], 'section': 'Host immune-modulation beyond vaccines', 'ref_spans': []}, {'text': 'The scientific and clinical communities have been exploring mostly two approaches to achieve balanced immunity to reduce viral load, while preventing end-stage failure of the infected organ90. The first one encompasses the administration of immune suppressors150 to reduce the hyperinflammation induced by the viral infection, that in turn can provoke damage to the inflamed area90. Corticosteroids are widely used to suppress lung inflammation and reduce the risk for pulmonary damage in critically ill patients affected by respiratory viral infections, such as those induced by MERS-CoV32,151 and SARS-CoV152. However, the real risk-benefit underlying the use of corticosteroids is yet unclear. In fact, corticosteroids also limit immune responses and thereby might counteract viral clearance, as it was observed in some individuals affected by MERS-CoV infection31,153. Taking this into account, together with ARDS developed in patients affected by SARS-CoV-2 infection, the recommendation of corticosteroid treatment to these patients is still controversial, being currently addressed in ongoing clinical trials (NCT04244591)13,154. Nevertheless, the RECOVERY trial results point toward the potential use of dexamethasone to reduce the death rate of patients that require respiratory support69.', 'cite_spans': [{'start': 189, 'end': 191, 'mention': '90', 'ref_id': 'BIBREF195'}, {'start': 259, 'end': 262, 'mention': '150', 'ref_id': 'BIBREF57'}, {'start': 379, 'end': 381, 'mention': '90', 'ref_id': 'BIBREF195'}, {'start': 588, 'end': 590, 'mention': '32', 'ref_id': 'BIBREF131'}, {'start': 591, 'end': 594, 'mention': '151', 'ref_id': 'BIBREF58'}, {'start': 607, 'end': 610, 'mention': '152', 'ref_id': 'BIBREF59'}, {'start': 865, 'end': 867, 'mention': '31', 'ref_id': 'BIBREF130'}, {'start': 868, 'end': 871, 'mention': '153', 'ref_id': 'BIBREF60'}, {'start': 1129, 'end': 1131, 'mention': '13', 'ref_id': 'BIBREF34'}, {'start': 1132, 'end': 1135, 'mention': '154', 'ref_id': 'BIBREF61'}, {'start': 1295, 'end': 1297, 'mention': '69', 'ref_id': 'BIBREF171'}], 'section': 'Host immune-modulation beyond vaccines', 'ref_spans': []}, {'text': 'The second therapeutic approach is based on the use of immune potentiators, such as cytokines, immune checkpoint inhibitors, signalling proteins, antimicrobial peptides and PRR ligands. The main goal is to eradicate the virus following the stimulation of host innate and adaptive immune responses against the virus.', 'cite_spans': [], 'section': 'Host immune-modulation beyond vaccines', 'ref_spans': []}, {'text': 'Patients with severe COVID-19 develop viral sepsis following the excessive inflammatory response syndrome, as well as inflammation-lung injury, which often leads to septic shock and ARDS, the leading cause of death of these individuals155. Viral sepsis induces the dysfunction of several physiological responses, including both innate and adaptive immune responses, that reduces host ability to eradicate the viral infection156. However, treatments addressing this excessive inflammation against other microbial infections did not improve patient prognosis157.', 'cite_spans': [{'start': 235, 'end': 238, 'mention': '155', 'ref_id': 'BIBREF62'}, {'start': 424, 'end': 427, 'mention': '156', 'ref_id': 'BIBREF63'}, {'start': 556, 'end': 559, 'mention': '157', 'ref_id': 'BIBREF64'}], 'section': 'Host immune-modulation beyond vaccines', 'ref_spans': []}, {'text': 'The immunosuppression, mainly lymphocytopenia, is recognized as a leading cause of increased morbidity and mortality during sepsis158, enhancing patient vulnerability to bacterial or viral infections159. One of the main hypotheses for this immunosuppression occurred during sepsis is the increased expression of the immune regulatory checkpoints156, such as programmed death-1 (PD-1) and programmed death ligand-1 (PD-L1), which have a crucial role in T-cell depletion in septic patients160. Preclinical and clinical studies have shown that the inhibition of those immune checkpoint molecules can reverse the sepsis-induced immunosuppression, overcoming lymphocytopenia and thereby improving host resistance to infection161. These effects suggest that PD-1/PDL-1 inhibition constitutes a potential treatment against SARS-CoV-2- induced sepsis, and anti-PD-1 blocking antibodies are currently being investigated, as single agents or in combination with thymosin, that has also been shown to reduce the mortality in these patients (NCT04268537). In addition, anti-PD-L1-treated DC induced T cell priming and proliferation162. Therefore, the blockage of PD-1 associated with a vaccine may also be considered in future endeavours.', 'cite_spans': [{'start': 130, 'end': 133, 'mention': '158', 'ref_id': 'BIBREF65'}, {'start': 199, 'end': 202, 'mention': '159', 'ref_id': 'BIBREF66'}, {'start': 345, 'end': 348, 'mention': '156', 'ref_id': 'BIBREF63'}, {'start': 487, 'end': 490, 'mention': '160', 'ref_id': 'BIBREF68'}, {'start': 720, 'end': 723, 'mention': '161', 'ref_id': 'BIBREF69'}, {'start': 1119, 'end': 1122, 'mention': '162', 'ref_id': 'BIBREF70'}], 'section': 'Host immune-modulation beyond vaccines', 'ref_spans': []}, {'text': 'Additional approaches focus on the reduction of cytokine secretion. Cytokines are protein mediators that provide signals crucial for key biological processes including immunity, cell proliferation and inflammation, wound healing and repair, cell migration, fibrosis and angiogenesis163. High levels of transforming growth factor beta-1 (TGF-β1)26,27 have been detected in SARS patients, being related to the development of pulmonary fibrosis and to the reduction in apoptosis of SARS-CoV-infected cells164. Similar consequences of high levels of TGF-β1 were found in SARS-CoV-2-infected patients. This suggests that targeting pro-inflammatory cytokines that induce pulmonary fibrosis may be useful. One such therapeutic option is pirfenidone, for which different mechanisms have been suggested for its activity. Specifically, it reduces the levels of fibrosis-associated proteins and cytokines, as well as the extracellular matrix accumulation in response to TGF-β1 and platelet-derived growth factor (PDGF)165. The effect of pirfenidone is currently being studied in a clinical trial (NCT04282902) taking into account encouraging results previously reported against idiopathic pulmonary fibrosis, in addition to its pronounced anti-inflammatory and anti-oxidant effects. Furthermore, the impact of the suppression of the TGF-β pathway on overall T-cell function should also be considered in these patients, as it may limit the regulatory T cell (Treg)-mediated immunosuppression166.', 'cite_spans': [{'start': 282, 'end': 285, 'mention': '163', 'ref_id': 'BIBREF71'}, {'start': 344, 'end': 346, 'mention': '26', 'ref_id': 'BIBREF124'}, {'start': 347, 'end': 349, 'mention': '27', 'ref_id': 'BIBREF125'}, {'start': 502, 'end': 505, 'mention': '164', 'ref_id': 'BIBREF72'}, {'start': 1007, 'end': 1010, 'mention': '165', 'ref_id': 'BIBREF73'}, {'start': 1479, 'end': 1482, 'mention': '166', 'ref_id': 'BIBREF74'}], 'section': 'Host immune-modulation beyond vaccines', 'ref_spans': []}, {'text': 'In contrast to what was observed regarding the overexpression of TGF-β1 in SARS-CoV and MERS-CoV infected patients, the analysis of the expression profiles of immune-related genes indicated the downregulation of IFN-α/β genes35,36. The SARS-CoV virus does not indeed trigger the activation of the IFN-α/β pathway97, which has a relevant role on reducing viral replication and dissemination at an early stage167, in addition to the overall stimulatory effect on host immune system168. Therefore, several ongoing clinical trials are exploring the efficacy and safety of different IFN subtypes for COVID-19 patients as therapeutic and prophylactic agents (NCT04254874, NCT04293887, NCT04315948).', 'cite_spans': [{'start': 225, 'end': 227, 'mention': '35', 'ref_id': 'BIBREF134'}, {'start': 228, 'end': 230, 'mention': '36', 'ref_id': 'BIBREF135'}, {'start': 312, 'end': 314, 'mention': '97', 'ref_id': 'BIBREF202'}, {'start': 407, 'end': 410, 'mention': '167', 'ref_id': 'BIBREF75'}, {'start': 479, 'end': 482, 'mention': '168', 'ref_id': 'BIBREF76'}], 'section': 'Host immune-modulation beyond vaccines', 'ref_spans': []}, {'text': 'In addition to this passive administration of IFN, alternative approaches for inducing the stimulation of TLR3 and TLR7 are particularly promising. These are transmembrane upstream regulatory factors that upon recognition of a specific pattern, such as a single and double stranded RNA viruses, will induce the transcription of pro-inflammatory cytokines, including IFN91, increasing host ability to eliminate the pathogen169.', 'cite_spans': [{'start': 369, 'end': 371, 'mention': '91', 'ref_id': 'BIBREF196'}, {'start': 422, 'end': 425, 'mention': '169', 'ref_id': 'BIBREF77'}], 'section': 'Host immune-modulation beyond vaccines', 'ref_spans': []}, {'text': 'TLR bridge innate and adaptive immune responses as they do not only activate the innate immune system but can also determine the nature of the adaptive immune response by upregulating MHC on DC, in addition to the secretion of costimulatory molecules and proinflammatory cytokine release. This process leads to the differentiation of CD4+ T cells into Th1 cells, which in turn produce IFN-γ, and lead to class switch from IgM to IgG2 antibodies by B cells170. Therefore, an ongoing clinical trial in China (ChiCTR2000029776) is evaluating the therapeutic effect of triggering TLR signalling pathways in COVID-19 patients by an analogue of double-stranded RNA [Poly(I:C)].', 'cite_spans': [{'start': 455, 'end': 458, 'mention': '170', 'ref_id': 'BIBREF79'}], 'section': 'Host immune-modulation beyond vaccines', 'ref_spans': []}, {'text': 'Another immunological approach for treating COVID-19 is based on the use of endogenous molecules of the innate immune system such as defensins. Defensins are a family of endogenous antibiotic peptides with a wide antimicrobial and antiviral spectrum activity, being important for the function of the innate defence system171. It was previously observed that the treatment of SARS-CoV-infected mice with defensin caused alterations in the cytokine profile at the lung parenchyma, but it did not seem to affect lung pathology, inhibit proliferation or eradicate the SARS‐CoV virus. However, the intranasal administration of defensin significantly protected mice against SARS-CoV infection172, being therefore potentially beneficial for high-risk groups.', 'cite_spans': [{'start': 321, 'end': 324, 'mention': '171', 'ref_id': 'BIBREF80'}, {'start': 686, 'end': 689, 'mention': '172', 'ref_id': 'BIBREF81'}], 'section': 'Host immune-modulation beyond vaccines', 'ref_spans': []}, {'text': 'Since the end of 2019, multiple strategies are being adopted for the SARS-CoV-2 vaccine development74,173. The majority of these approaches target the surface-exposed S glycoprotein or the S protein (full-length or specific subunits) to induce a potent neutralizing effect by eliciting specific T-cell responses and neutralizing antibodies174,175. As of 8 April 2020, there were 115 vaccine candidates, of which 37 have not been confirmed as active against SARS-CoV-2 infection, and 73 out of the 78 reported as active, are in very early stage phases with no preclinical studies yet97.', 'cite_spans': [{'start': 99, 'end': 101, 'mention': '74', 'ref_id': 'BIBREF177'}, {'start': 102, 'end': 105, 'mention': '173', 'ref_id': 'BIBREF82'}, {'start': 339, 'end': 342, 'mention': '174', 'ref_id': 'BIBREF83'}, {'start': 343, 'end': 346, 'mention': '175', 'ref_id': 'BIBREF84'}, {'start': 582, 'end': 584, 'mention': '97', 'ref_id': 'BIBREF202'}], 'section': 'COVID-19 preclinical prophylactic immune-based approaches', 'ref_spans': []}, {'text': 'Supplementary Tables 4a,b summarize diverse vaccine candidates developed to overcome SARS-CoV-2 infection, of which efficacy and safety are being evaluated at a preclinical stage. It is reasonable to anticipate that even those candidates that will successfully demonstrate a safety profile and considerable ability to modulate host immunity against this virus, it will still take about a year to start phase I clinical trials.', 'cite_spans': [], 'section': 'COVID-19 preclinical prophylactic immune-based approaches', 'ref_spans': []}, {'text': 'Therapeutic options that could be used against SARS-CoV-2 include virus-binding molecules, inhibitors that target specific enzymes involved in viral replication and transcription, small-molecule inhibitors targeting helicase, proteases or other proteins crucial for the survival of the virus, host cell protease and endocytosis inhibitors, siRNA, anti-sense RNA and ribozyme, neutralizing antibodies, mAb against host receptor or S1 RBD, anti-viral peptide targeting S2 and natural products173,176,177. For now, remdesivir, lopinavir/ritonavir alone or in combination with IFN-β, mAb or convalescent plasma are among the most investigated therapeutic options against the SARS-CoV-2173,178. In addition, the characterization of the effects induced by mAb in COVID-19 patients may also advance the development of vaccines, and increasingly specific diagnostics179. However, every single one of these tools needs to be evaluated regarding clinical efficacy and safety, before being used to treat infected patients. Supplementary Tables 5a,d list the advanced therapeutic approaches under preclinical development for SARS-CoV-2.', 'cite_spans': [{'start': 490, 'end': 493, 'mention': '173', 'ref_id': 'BIBREF82'}, {'start': 494, 'end': 497, 'mention': '176', 'ref_id': 'BIBREF85'}, {'start': 498, 'end': 501, 'mention': '177', 'ref_id': 'BIBREF86'}, {'start': 681, 'end': 684, 'mention': '173', 'ref_id': 'BIBREF82'}, {'start': 685, 'end': 688, 'mention': '178', 'ref_id': 'BIBREF87'}, {'start': 858, 'end': 861, 'mention': '179', 'ref_id': 'BIBREF88'}], 'section': 'COVID-19 preclinical therapeutic immune-based approaches', 'ref_spans': []}, {'text': 'Prevention of SARS-CoV-2 new infections may be the most effective approach, not only to prevent COVID-19 but also to block the spreading of the virus worldwide. Since SARS-CoV-2 virus was only recently identified, large efforts dedicated to vaccine development are being pursued. As of 30 June, 48 clinical trials are already evaluating the efficacy of COVID-19 vaccines (Supplementary Table 6a). One approach, developed by ModernaTX, Inc. uses lipid nanoparticles (LNP) encapsulating mRNA-1273 that encodes full-length, SARS-CoV-2 S protein (NCT04283461). Cells expressing this viral protein will be able to present T cell-recognized SARS-CoV-2 antigen and induce an immune response against the virus. This may be an effective and safe approach since it does not use viral particles, but rather delivers mRNA that can be expressed by immune cells and non-immune cells, leading to both MCH class I and MHC class II antigen presentation. Furthermore, encapsulating viral mRNA in LNP will protect the mRNA from degradation and enhance delivery efficiency180. ModernaTX’s press release181 published on 18 May announced the interim data of this ongoing phase I trial for mRNA-1273 vaccine, which indicate that the vaccine was generally well-tolerated and safe, and led to the production of neutralizing antibodies in eight initial patients enrolled in this study, two weeks after receiving the second dose of this COVID-19 vaccine candidate.', 'cite_spans': [{'start': 1052, 'end': 1055, 'mention': '180', 'ref_id': 'BIBREF90'}, {'start': 1082, 'end': 1085, 'mention': '181', 'ref_id': 'BIBREF91'}], 'section': 'COVID-19 prophylactic approaches in clinical development', 'ref_spans': []}, {'text': 'Recombinant novel CoV vaccine using adenovirus type 5 vector, also called Ad5-nCoV (NCT04313127) was proposed by Tianjin-based CanSino Biologics Inc. This product uses a genetically-modified adenovirus type 5 that is replication-defective and expresses the SARS-CoV-2 S protein. The ChAdOx1 nCoV-19 (NCT04324606) developed by the University of Oxford is also an adenovirus-vectored vaccine in clinical development, which has been explored as a vaccine against other infectious diseases, such as influenza, tuberculosis, Chikungunya virus (CHIKV), Zika, meningitis B and the plague.', 'cite_spans': [], 'section': 'COVID-19 prophylactic approaches in clinical development', 'ref_spans': []}, {'text': 'There are other clinical trials exploring the use of genetically modified, activated immune cells, which recognize SARS-CoV-2 antigens (Supplementary Table 6a). Shenzhen Geno-Immune Medical Institute engineered minigenes (exon gene fragments) based on multiple viral genes, which are introduced into artificial antigen presenting cells (aAPC) using a lentiviral vector (NCT04299724). When injected into the patient, the aAPC will activate T cells and generate an immune response against the virus. In addition, LV-SMENP-DC, also from Shenzhen Geno-Immune Medical Institute, is a modified DC that expresses viral antigens (NCT04276896). These DC are used to activate cytotoxic T cells (CTL) ex vivo. The modified DC and activated CTL will be both injected into the patients. On 6 April 2020, INOVIO Pharmaceuticals announced the initiation of phase 1 clinical trial of its INO-4800 DNA plasmid vaccine encoding S protein delivered by electroporation (NCT04336410), based on their technology previously exploited for Lassa, Nipah, HIV, filovirus, human papillomavirus, cancer indications, Zika and hepatitis B. An additional prophylactic approach to prevent COVID-19 disease in adults exposed to the virus, encompasses the inhalation of the PUL-042 solution, which is a TLR 2/6/9 agonist182. By binding to these TLR, PUL-042 can activate immune cells, such as natural killer (NK) cells, macrophages and DC, and stimulate lung epithelial cells to produce diverse factors against photogenic infection183.', 'cite_spans': [{'start': 1285, 'end': 1288, 'mention': '182', 'ref_id': 'BIBREF92'}, {'start': 1496, 'end': 1499, 'mention': '183', 'ref_id': 'BIBREF93'}], 'section': 'COVID-19 prophylactic approaches in clinical development', 'ref_spans': []}, {'text': 'Although promising, these clinical trials are now recruiting for phase I or phase II, and efficacy and safety in human patients have not yet been confirmed.', 'cite_spans': [], 'section': 'COVID-19 prophylactic approaches in clinical development', 'ref_spans': []}, {'text': 'Most of the clinical studies test current immunotherapies or re-purposing of existing drugs for the treatment of COVID-19 patients (Supplementary Tables 6a–d). As of 30 June, there are 217 registered clinical trials using immune modulators against COVID-19, while over 476 investigate the added-value of repurposing drugs.', 'cite_spans': [], 'section': 'COVID-19 therapeutic approaches in clinical development', 'ref_spans': []}, {'text': 'The injection of immunoglobulins from convalescent patients has been tested to induce a specific immune response against SARS-COV-2 (NCT04264858). Other trials are trying to induce a broad immune response through the non-specific activation of the immune system. For instance, a combination of a standard treatment with anti-PD-1 antibody is under clinical investigation (NCT04268537). Another example is the recombinant IFN-α2β, an immune-modulator approved for the treatment of viral infections, such as hepatitis B and C, and for the treatment of different types of cancer, such as metastatic melanoma (NCT04293887)184,185.', 'cite_spans': [{'start': 618, 'end': 621, 'mention': '184', 'ref_id': 'BIBREF94'}, {'start': 622, 'end': 625, 'mention': '185', 'ref_id': 'BIBREF95'}], 'section': 'COVID-19 therapeutic approaches in clinical development', 'ref_spans': []}, {'text': 'Other clinical studies are testing different anti-inflammatory agents to reduce lung inflammation (pneumonia), the leading cause of death for COVID-19 patients. These include antibodies targeting inflammatory factors, such as IL-6 and complement protein C5, or the CD24Fc conjugate that blocks TLR activation. Two clinical studies are using the anti-angiogenic drug bevacizumab (anti-VEGF mAb) to reduce lung oedema. Another antibody in clinical development is the meplazumab, which blocks the binding of SARS-CoV-2 S protein to CD147 molecule on human cells, thus reducing the virus infection ability34. Other immunosuppressive agents are also being tested, such as the JAK1/JAK2 inhibitor baricitinib and the anti-malaria drug hydroxychloroquine sulfate. Although optimal treatment regimens are still under investigation, different dosing and schedules are being reported by clinicians.', 'cite_spans': [{'start': 601, 'end': 603, 'mention': '34', 'ref_id': 'BIBREF133'}], 'section': 'COVID-19 therapeutic approaches in clinical development', 'ref_spans': []}, {'text': 'Few clinical studies are exploring cell therapies, such as NK cells to support and strengthen the immune response (NCT04280224), while others use mesenchymal stem cells, which can facilitate tissue regeneration and immune suppression186.', 'cite_spans': [{'start': 233, 'end': 236, 'mention': '186', 'ref_id': 'BIBREF96'}], 'section': 'COVID-19 therapeutic approaches in clinical development', 'ref_spans': []}, {'text': 'Facing the tragic numbers that are arriving every day from all over the world, on 18 March 2020, the United Nations Director-General announced an unprecedented action where international cooperation made possible the design of the Solidarity Trial. The World Health Organization (WHO)187 announced on 27 March 2020 that 45 countries were already contributing, and many others had demonstrated interest in joining this historic trial, where the major aim is to rapidly understand the safety and effectiveness of remdesivir, lopinavir/ritonavir, lopinavir/ritonavir with IFN-β1a and chloroquine or hydroxychloroquine188. The contribution of data obtained by this multi-country trial in a concerted manner enables the direct comparison between those drugs, which will drastically cut the time needed to provide clear and robust data to demonstrate which drugs will be actually effective against COVID-19.', 'cite_spans': [{'start': 284, 'end': 287, 'mention': '187', 'ref_id': 'BIBREF97'}, {'start': 614, 'end': 617, 'mention': '188', 'ref_id': 'BIBREF98'}], 'section': 'COVID-19 therapeutic approaches in clinical development', 'ref_spans': []}, {'text': 'The COVID-19 pandemic has spread to 213 countries and territories, and the number of cases reported as of 30 June 2020 surpasses 8.5 million worldwide. More than 5.6 million people recovered from SARS-CoV-2 infection, but 506 thousand people were defeated by this virus. The number of new cases continuously rises.', 'cite_spans': [], 'section': 'Conclusions and perspectives', 'ref_spans': []}, {'text': 'The severity of the situation has already re-shaped our society. Reports are arriving from most countries testifying the tremendous commitment of the governmental, scientific and clinical communities, in making concerted efforts to help local populations to deal with the pandemic.', 'cite_spans': [], 'section': 'Conclusions and perspectives', 'ref_spans': []}, {'text': 'Reorientation of research at many academic and industrial institutions is also currently in place, taking advantage of already existing knowledge and experience in fighting previous infectious diseases that may bring new solutions to the COVID-19 pipeline. History also shows that these crises create unique opportunities for the development of new technologies or the flexible use of existing ones.', 'cite_spans': [], 'section': 'Conclusions and perspectives', 'ref_spans': []}, {'text': 'The SARS-CoV-2 is indeed a new strain of coronavirus, and even if the scientific community is rapidly gaining tremendous knowledge about this virus, the world population has not yet acquired immunity. The development of an effective vaccine will be particularly important to protect high-risk patients, taking into account their decaying immune function. Therefore, the development of a COVID-19 vaccine suitable for mass immunization is urgent and companies and research institutes have already reported the development of more than 115 vaccine candidates. Researchers worldwide are exploiting platform technologies previously developed to control CoV infections, as well as other diseases, such as HIV, influenza, Zika, Ebola, plague, tuberculosis and meningitis. The development of these vaccine candidates is expected to be particularly fast as their safety and efficacy in modulating host immune response have already been attested, even if against other agents that may not exactly present the same pathogenesis but that modulate physiological functions in a similar manner. However, despite the diverse platform technologies available to produce these vaccines, all developers agree that it is highly unlikely that a vaccine will be available for worldwide use in less than 12 months, in particular taking into account that the development of an Ebola vaccine took 5 years and a mumps vaccine took 4 years in a record time.', 'cite_spans': [], 'section': 'Conclusions and perspectives', 'ref_spans': []}, {'text': 'The tremendous advances in molecular engineering and biotechnology for the last years have led to engineered biotech compounds, such as peptide and protein antigens, multiple copies of antigen-encoding mRNA, in addition to gene regulators of immune cell function, such as siRNA to suppress the expression of immunosuppression-related genes.', 'cite_spans': [], 'section': 'Conclusions and perspectives', 'ref_spans': []}, {'text': 'However, as candidates move towards clinical investigation, it becomes clear that their biological effects depend on the development of a tool able to attain their transport across biological barriers. In fact, low response rate of patients has been related to the limited intrinsic immunogenicity of antigens, and therefore, the added value of adjuvants, such as TLR ligands, are commonly considered to overcome sub-optimal efficacy. However, oligonucleotides, such as the TLR ligands CpG or Poly (I:C), or siRNA and mRNA candidates, are limited by low stability under physiological conditions, off-target effects and limited cellular uptake189. The systemic effect of oligonucleotides demands the design of safe and effective delivery systems to target and cross plasma membranes, but also to escape from endosomal compartments into the cytoplasm. The key to unlock their potential has pointed nanomedicines as an approach able to guarantee the target selectivity required for their efficacy, while ensuring patients’ safety. Accordingly, nanotechnology-based systems incorporating combinations of antigen epitopes and adjuvants are being developed to improve vaccine delivery. These nano-based vaccines orchestrate a broad immunity by modulating B cell-mediated responses towards an increased and fast production of high-affinity neutralizing antibodies upon re-encountering cognate antigens, but also by allowing an adequate activation and expansion of T cell-function directed, for example, to the destruction of virus-infected cells. Nanotechnology-based strategies also constitute a potentially useful tool to enhance the stability and improve the pharmacokinetics of therapeutic antibodies. The entrapment of antibody derivatives has been explored to improve their delivery and endosomal escape, which will result in improved intracellular antigen recognition190. It has also been shown that the entrapment of immune checkpoint monoclonal antibody modulators by polymeric nanoparticles improved the immune-mediated responses to these immunotherapies, such as to the agonist anti-OX40191 and to the anti-CTLA4 blocker192. Nanoscale delivery systems improve the delivery of active compounds to specific cells and tissues, and decrease adverse effects cause by systemic administration, for example of FDA-approved cytokines, such as TNF-α193 and IL-2194.', 'cite_spans': [{'start': 642, 'end': 645, 'mention': '189', 'ref_id': 'BIBREF99'}, {'start': 1867, 'end': 1870, 'mention': '190', 'ref_id': 'BIBREF101'}, {'start': 2091, 'end': 2094, 'mention': '191', 'ref_id': 'BIBREF102'}, {'start': 2124, 'end': 2127, 'mention': '192', 'ref_id': 'BIBREF103'}, {'start': 2343, 'end': 2346, 'mention': '193', 'ref_id': 'BIBREF104'}, {'start': 2355, 'end': 2358, 'mention': '194', 'ref_id': 'BIBREF105'}], 'section': 'Conclusions and perspectives', 'ref_spans': []}, {'text': 'The similarity between the infection mechanisms of SARS-CoV-2 and other CoVs and HIV also indicates that drugs already clinically-approved to target different steps of the infection pathway constitute promising potential solutions against COVID-19. Accordingly, ritonavir and lopinavir are currently under consideration. However, caution should be taken due to distinct HIV and SARS proteases, which can limit the specificity of their therapeutic activity. Chloroquine phosphate and hydroxychloroquine sulfate, as well as remdesivir may lead to better clinical outcomes as these will probably have a broad-spectrum activity by eventually inhibiting SARS-CoV-2 cell entry and the RNA polymerase, respectively195. Time is also crucial; vaccines are intended for administration before the infection (or as intervention in case of a therapeutic vaccine), antivirals need to be administered as soon as possible after infection, whereas immunostimulants or -inhibitors could be given later.', 'cite_spans': [{'start': 707, 'end': 710, 'mention': '195', 'ref_id': 'BIBREF106'}], 'section': 'Conclusions and perspectives', 'ref_spans': []}, {'text': 'Despite lack of medical evidence, the severe respiratory symptoms presented by an 80-year old critically ill COVID-19 patient were rapidly attenuated upon the administration of recombinant human erythropoietin (EPO)196. Previous preclinical studies reported the effect of EPO on septic cases197,198, and on the inhibition of pro-inflammatory cytokine expression and increase of B cells, as well as CD8+ and CD4+ T cells in circulation199. Even though this patient also received hydroxychloroquine, oseltamivir and lopinavir/ritonavir, these effects were observed following EPO and red blood cell transfusion, which led to an increase in the levels of haemoglobin. Clinical studies on the use of EPO in COVID-19 patients are required, but its use should be carefully considered, for example in patients with anaemia associated with renal chronic disease due to already known side effects such as thrombotic events.', 'cite_spans': [{'start': 215, 'end': 218, 'mention': '196', 'ref_id': 'BIBREF107'}, {'start': 291, 'end': 294, 'mention': '197', 'ref_id': 'BIBREF108'}, {'start': 295, 'end': 298, 'mention': '198', 'ref_id': 'BIBREF109'}, {'start': 434, 'end': 437, 'mention': '199', 'ref_id': 'BIBREF110'}], 'section': 'Conclusions and perspectives', 'ref_spans': []}, {'text': 'It is important to bear in mind that unfortunately, we are still far from having a full picture of the pathophysiology of this dangerous disease, including its long-term implications on individuals that did not experience the severe form of this infection. What the clinical and scientific communities first thought of as being an infectious disease that, similarly to other CoV-related infections, would mostly affect the respiratory tract, soon became a multifaceted pathology, which major features include the massive accumulation of blood clots. It is now clear that the SARS-CoV-2 infection causes heart attacks, coronary-related kidney damage, severe stroke, including in young people with no previous history of cardiovascular disease, being all related to extensive clot formation. In addition, several countries, such as Italy200, UK201, Canada202 and USA202, reported the hospitalization of children and young adults presenting severe Kawasaki-like disease related to COVID-19, showing inflammation in medium-size blood vessels, rash, fever and, in some cases, shock. Children are accepted to be the least affected by this disease, but this emerging link between SARS-CoV-2 infection and this rare inflammatory condition associated with systemic blood vessel damage in COVID-19 positive young patients, is raising awareness towards the urgent need for collecting data covering clinical outcomes and presentations of this emerging phenomenon. Clinicians and scientist worldwide are driving concerted efforts to correlate the behaviour of the immune system components with SARS-CoV-2 infection, as well as to decipher genes related to high-risk of disease severity, fundamental to point out children who are at high-risk and help on clarifying the chaotic/uncontrolled immunity found in adults.', 'cite_spans': [{'start': 835, 'end': 838, 'mention': '200', 'ref_id': 'BIBREF113'}, {'start': 842, 'end': 845, 'mention': '201', 'ref_id': 'BIBREF114'}, {'start': 853, 'end': 856, 'mention': '202', 'ref_id': 'BIBREF115'}, {'start': 864, 'end': 867, 'mention': '202', 'ref_id': 'BIBREF115'}], 'section': 'Conclusions and perspectives', 'ref_spans': []}, {'text': 'The search for life-saving solutions to treat COVID-19 patients or achieve the control of SARS-CoV-2 transmission is crucial to win the race against the rapid transmission of this virus, and regulatory bodies worldwide have joined efforts to respond to this outbreak by providing new mechanisms to support a faster development and approval of the most promising therapeutics and vaccine candidates. However, it is fundamental to simultaneously ensure that adequate measures are being taken to thoroughly demonstrate their safety and efficacy. For example, the European Medicines Agency (EMA) released on 27 March an updated version of the Guidance on the management of clinical trials during COVID-19203, namely addressing communication with the governmental authorities, informed consent and distribution channels of investigational medicines. The guidance further includes information regarding the distribution of diagnosis tests, safety reporting and auditing.', 'cite_spans': [{'start': 700, 'end': 703, 'mention': '203', 'ref_id': 'BIBREF116'}], 'section': 'Conclusions and perspectives', 'ref_spans': []}, {'text': 'The COVID-19 pandemic reminds us of the devastating effect that previous emerging diseases had in our history until the development of effective vaccines that greatly reduced the burden caused by those infections. As recognized by the WHO204, only access to clean water has a greater impact on disease and death of populations. Anti-vaccine movements have raised their voices against vaccination programmes worldwide, which has already increased the incidence of diseases that were already rare in developed countries. Therefore, the measures now being taken by society to limit the severe economic, social and cultural consequences of the COVID-19 pandemic, has the potential to make the national vaccination programmes strong again. Sadly, however, physical distancing measures and lockdown have already heavily compromised the vaccination of children against other important infectious diseases, namely measles, polio, diphtheria, tetanus and pertussis205. WHO indicates that 100 million children are expected to miss measles vaccination due to COVID-19 lockdown. As a result, as the pandemic starts to reach a containment phase in several countries, it is important to analyse data gathered for the last 5 months and the expected impact of a prolonged lockdown, looking also at potential effect on social and economic levels. It is also important to investigate medical care procedures and the treatment of patients affected by other diseases that have been abruptly interrupted due to the COVID-19 pandemic. For example, cancelled procedures that limit disease diagnosis and cancer treatment schedules that have been affected, such as surgeries, chemotherapy or radiotherapy. Taking these facts into account, several countries are already taking careful measures to relax COVID-19 lockdown, to simultaneously control SARS-CoV-2 spread among the population.', 'cite_spans': [{'start': 238, 'end': 241, 'mention': '204', 'ref_id': 'BIBREF117'}, {'start': 955, 'end': 958, 'mention': '205', 'ref_id': 'BIBREF118'}], 'section': 'Conclusions and perspectives', 'ref_spans': []}, {'text': 'The management of highly contagious and potentially fatal infectious diseases has shaped human history over the centuries, and the COVID-19 pandemic will not be an exception. However, we are now in an era in which highly advanced technological resources are available, which can be our weapon to change the course of this pandemic. Therefore, it is important to invest in health systems and in basic and innovative science to find diagnostic and therapeutic solutions. It is heart-warming to see the collaborative efforts amongst all sectors involved in the global healthtech ecosystem. We only hope that these insights will still resonate after this pandemic is behind us.', 'cite_spans': [], 'section': 'Conclusions and perspectives', 'ref_spans': []}], 'paragraphs': [{'text': 'According to the Center for Disease Control and Prevention (CDC), the incubation period following infection is 2–14 days, with an estimated median of 5.1 days9,10. However, cases with longer incubation of 24 days have been reported11. The long incubation period is the primary reason for the massive infection, as it is mostly asymptomatic yet contagious10. Although the estimated patients’ age average is ~70, all age groups are susceptible to this virus. However, the elder population (>60) and people with comorbidities are more likely to develop severe symptoms upon infection12.', 'id': '00003'}], 'title': 'Immune-mediated approaches against COVID-19', 'sha': '5032da4ed35eb64b2b376b854ad64ab343fced3e; 246eb0ed05487b824c56db347785920048f6b3d1', 'doi': '10.1038/s41565-020-0732-3', 'score': 9.032599449157715}, {'rank': 224, 'paragraphs': [{'text': 'If α ≠ 1, it is possible to identify all underlying parameters. We explore the sensitivity of estimates of, incubation period by assuming a range of π, that is π = 0,0.005, 0.1, and 0.2, where π = 0 is the reference, and estimate α and λ by maximizing the product of likelihoods, ∏i=1Ih(vi), with respect to α and λ.', 'id': '00019'}], 'title': 'Estimation of incubation period distribution of COVID-19 using disease onset forward time: a novel cross-sectional and forward follow-up study', 'sha': 'c61be9fc4562d250e2dec241ef75bd22d61f61d1; 6d3b3f4ab80a61c45f82c61c6c756cfc6ddf4bf2', 'doi': '10.1101/2020.03.06.20032417', 'score': 9.02910041809082}, {'rank': 225, 'paragraphs': [{'text': 'Table 3 summaries the estimates of parameters, and the mean and percentiles of incubation period. We can see that the estimates for mean and percentiles decrease as the proportion of people who got infected at the event of departure, π, increases. However, the results are not shifted significantly. Variation of the results from π = 0 to 0.2 is only about one day.', 'id': '00022'}], 'title': 'Estimation of incubation period distribution of COVID-19 using disease onset forward time: a novel cross-sectional and forward follow-up study', 'sha': 'c61be9fc4562d250e2dec241ef75bd22d61f61d1; 6d3b3f4ab80a61c45f82c61c6c756cfc6ddf4bf2', 'doi': '10.1101/2020.03.06.20032417', 'score': 9.029099464416504}, {'rank': 226, 'paragraphs': [{'text': 'The Center for Disease Control and Prevention (CDC) of China and World Health Organization (WHO) are closely monitoring the current outbreak of coronavirus disease 2019 (COVID-19). It was first identified in Wuhan, Hubei province, China, and has quickly spread across countries and become a global crisis. As of February 22, 2020, the National Health Commission (NHC) of China had confirmed a total of 76 936 cases of COVID-19 in mainland China, including 2442 fatalities and 22 888 recoveries.1 Various containment measures, including travel restrictions, isolation, and quarantine have been implemented in China with the aim to minimize virus transmission via human-to-human contact.2 Quarantine of individuals with exposure to infectious pathogens has always been an effective approach for containing contagious diseases in the past. One of the critical factors to determine the optimal quarantine of asymptomatic individuals is a good understanding of the incubation period, and this has been lacking for COVID-19.', 'id': '00001'}], 'title': 'Estimation of incubation period distribution of COVID-19 using disease onset forward time: a novel cross-sectional and forward follow-up study', 'sha': 'c61be9fc4562d250e2dec241ef75bd22d61f61d1; 6d3b3f4ab80a61c45f82c61c6c756cfc6ddf4bf2', 'doi': '10.1101/2020.03.06.20032417', 'score': 9.028300285339355}, {'rank': 227, 'abstract': 'A unified 14-day quarantine may not effectively contain the spread of COVID-19. With the current recommendation of a uniform 14-day quarantine, up to 9.2% of patients 42 or younger and 18.6% of patients older than 42 years could develop symptoms after the end of the period.', 'body_text': [{'text': 'The coronavirus disease 2019 (COVID-43 19) was first reported in Wuhan, China, in December 2019; an outbreak rapidly spread worldwide. The novel virus infection can be asymptomatic or unapparent during a certain period and asymptomatic persons could spread the virus unknowingly. Among patients who develop symptoms, the incubation period is defined as the elapsed time between infection and appearance of the first symptom. Knowledge of the incubation period is essential for disease prevention, facilitating an optimal quarantine guideline to confine the spread.', 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'The incubation period of COVID-19 was investigated in several reports. [1, 2, 3, 4] Despite its importance, it remains unclear how the incubation distribution could vary by gender and age and whether the current 14-day quarantine period ignoring the patient demographic factors would be sufficient for the containment of COVID-19. A more accurate estimation of the incubation period using these factors can optimize COVID-19 surveillance guidelines. We estimate distribution of the incubation periods of COVID-19, and its association with gender and age, using the best available information to model uncertain dates of infection and symptom onset.', 'cite_spans': [{'start': 71, 'end': 74, 'text': '[1,', 'ref_id': 'BIBREF0'}, {'start': 75, 'end': 77, 'text': '2,', 'ref_id': 'BIBREF1'}, {'start': 78, 'end': 80, 'text': '3,', 'ref_id': 'BIBREF2'}, {'start': 81, 'end': 83, 'text': '4]', 'ref_id': 'BIBREF3'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'Our analysis was based on publicly reported, clinically confirmed cases with symptoms from two sources, available as of March 30, 2020 [5, 6] . To obtain the period of potential exposure, we included the following patients in the analytic dataset: 1) non-residents of Wuhan who visited Wuhan since December 1, 2019 and for whom the exposure period was the time between the earliest possible arrival to and the latest possible departure from Wuhan; 2) people with travel history of visiting the coronavirus-affected nations known at the time or taking a Diamond Princess cruise; 3) non-travel related cases with an exposure history based on contact with infected persons. Among these, 312 cases had recorded exposure periods, or at least an exposure ending date, and dates of symptom onset or hospitalization, as well as gender and age; this number reflects removal of 34 potentially duplicated cases (Supplement S1).', 'cite_spans': [{'start': 135, 'end': 138, 'text': '[5,', 'ref_id': 'BIBREF4'}, {'start': 139, 'end': 141, 'text': '6]', 'ref_id': 'BIBREF5'}], 'ref_spans': [], 'section': 'Methods'}, {'text': 'There were 111 cases with complete information on the exposure period (Supplement S2). For cases without an exact starting date of exposure interval, the initial date was set to have a maximum of thirty days of exposure 2 . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Methods'}, {'text': 'The copyright holder for this preprint this version posted June 28, 2020. . or December 1, 2019, whichever one was later. The ending date of exposure was set to precede the known dates of symptom onset and hospitalization. If the exact date of symptom onset was unknown, it was assumed to have occurred before hospitalization (i.e., left-censored).', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'In a total of 312 patients, the median age was 42 (interquartile range 33-55) years and 126 (40.4%) were women. Age was dichotomized at its median prior to the analysis. The effects of gender and age on the incubation period were estimated under the generalized odds-rate class of regression models, which consists of general parametric models including the log-logistic proportional odds models and the Weibull proportional hazards models, as special cases. [7] We estimated the distribution of incubation period by modelling the interval-censored exposure duration and the possible left-censored symptom onset time (Supplement S3). Figure 1 depicts four types of observed patient cases used for modelling.', 'cite_spans': [{'start': 459, 'end': 462, 'text': '[7]', 'ref_id': 'BIBREF6'}], 'ref_spans': [{'start': 634, 'end': 642, 'text': 'Figure 1', 'ref_id': 'FIGREF0'}], 'section': 'Results'}, {'text': 'Patients older than 42 years of age have, on average, longer incubation periods, compared to 42-year-old or younger patients (p=0.036), whereas gender has no effect on the incubation period (p=0.417). The median incubation period is 6.3 (95% CI 5.4-7.2) days overall, being 5.5 (95% CI 4.5-6.5) days and 7.2 (95% CI 5.8-8.6) days for younger and older patients, respectively. The mean incubation period is 6.7 (95% CI 5.5-7.9) days for the younger patients, and 8.7 (95% CI 7.2-10.2) days for the older patients (Supplement S4). Table 1 shows the estimated differences of days between the quantiles of incubation distribution from two age groups. The incubation period difference between the two age groups was not obvious at the 25th quantile or lower; however, it becomes conspicuous after the medians, leading to 6-day differences between the two age groups at the 97.5th quantile. ', 'cite_spans': [], 'ref_spans': [{'start': 529, 'end': 536, 'text': 'Table 1', 'ref_id': 'TABREF0'}], 'section': 'Results'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '3'}, {'text': 'The copyright holder for this preprint this version posted June 28, 2020. ', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The finding that persons older than 42 years have longer incubation periods than those who are younger has important implications for enacting age-specific quarantine policies. This result agrees with previous studies on Severe Acute Respiratory Syndrome that showed a relationship between age 4 . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': \"The copyright holder for this preprint this version posted June 28, 2020. . and the incubation period. [8] Using the uniform 14-day quarantine policy recommended by the World Health Organization and the US Centers for Disease Control and Prevention, [9, 10] our estimators imply that 9.2% (95% CI 3.5-14.8%) of COVID-19 patients younger than 42, and 18.6% (95% CI 11.3-25.9%) of older patients may pose a risk of infection to others before onset of their symptoms in the worst-case scenario. Using a 21-day quarantine, these percentages reduced to 1.7% (95% CI 0-3.7%) and 5.6% (95% CI 1.5-9.7%), respectively. To ensure that at least 90% of cases' symptoms being manifested during quarantine periods, the required durations are estimated to be 14 (95% CI 11-16) days for patients 42 years of age or younger and 18 (95% CI 15-21) days for patients older than 42 years. As such, a unified quarantine policy could be inefficient during a viral outbreak. These estimates were derived from the conservative assumption that the quarantine periods started immediately after infection.\", 'cite_spans': [{'start': 103, 'end': 106, 'text': '[8]', 'ref_id': 'BIBREF7'}, {'start': 250, 'end': 253, 'text': '[9,', 'ref_id': 'BIBREF8'}, {'start': 254, 'end': 257, 'text': '10]', 'ref_id': 'BIBREF9'}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'We used a larger data cohort consisting of more confirmed COVID-19 cases than other published papers in this field; the cases may not have the exact symptom onset time (left censored) and include an interval-censored incubation period. Our choice of a parametric model for the incubation period has more flexibility than other commonly used distributions. This flexible model was reduced to the Weibull distribution, a special case as reported in Backer et al [1] and Linton et al [4] (Supplement S5). We also used a descriptive method to check the model fitting by comparing a nonparametric estimation of the incubation distribution for interval-censored data with the model-based estimation under the Weibull distribution by age groups. As shown in Supplement S6, the Weibull distribution fits the data adequately. To examine how sensitive the estimates were to the assumption of 30 days of maximum exposure for cases with missing starting date of exposure interval, we performed the analyses by changing this maximum exposure duration to 20 days or by shifting the lower bound back and forth within 15 days. The changed assumptions had little effect on analysis results (Supplement S7). This analysis has some notable limitations. Our inferences relied on publicly reported confirmed cases that might over represent more severely symptomatic patients. Moreover, the definition of COVID-19 symptoms and hospitalization criteria could differ by country, especially during the initial outbreak. We combined the data sets from two different sources, and the potential variation in source criteria for tracing infected cases may lead to different exposure records. However, we obtained similar findings when fitting the model to each dataset separately, although one indicated no signif-5 . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'cite_spans': [{'start': 460, 'end': 463, 'text': '[1]', 'ref_id': 'BIBREF0'}, {'start': 481, 'end': 484, 'text': '[4]', 'ref_id': 'BIBREF3'}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted June 28, 2020. . icant differences but the same trend in the incubation period between the age groups (Supplement S8). An optimal differentiating cut-off age for the incubation period was not identified here due to the limited available data, which is worth pursuing to reduce risks to public health most effectively.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted June 28, 2020. . https://doi.org/10.1101/2020.06.27.20141002 doi: medRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}], 'paragraphs': [{'text': 'The incubation period of COVID-19 was investigated in several reports. [1, 2, 3, 4] Despite its importance, it remains unclear how the incubation distribution could vary by gender and age and whether the current 14-day quarantine period ignoring the patient demographic factors would be sufficient for the containment of COVID-19. A more accurate estimation of the incubation period using these factors can optimize COVID-19 surveillance guidelines. We estimate distribution of the incubation periods of COVID-19, and its association with gender and age, using the best available information to model uncertain dates of infection and symptom onset.', 'id': '00002'}], 'title': 'Age Associated Coronavirus Disease 2019 Incubation Period: Impact on Quarantine Policy', 'sha': '2ac8a8ae6a5040b664b39a4262ac1771af3d4b1b', 'doi': '10.1101/2020.06.27.20141002', 'score': 9.026200294494629}, {'rank': 228, 'paragraphs': [{'text': '(3)g(v)=αλexp{−(vλ)α}Γ(1/α),v≥0.', 'id': '00015'}], 'title': 'Estimation of incubation period distribution of COVID-19 using disease onset forward time: a novel cross-sectional and forward follow-up study', 'sha': 'c61be9fc4562d250e2dec241ef75bd22d61f61d1; 6d3b3f4ab80a61c45f82c61c6c756cfc6ddf4bf2', 'doi': '10.1101/2020.03.06.20032417', 'score': 9.024299621582031}, {'rank': 229, 'paragraphs': [{'text': ' On 11 March 2020, the World Health Organization (WHO) declared COVID‐19 to be a pandemic. The swift spread of the virus is largely attributed to its stealthy transmissions for which infected patients may be asymptomatic or exhibit only flu‐like symptoms in the early stage. Undetected transmissions present a remarkable challenge for the containment of the virus and pose an appalling threat to the public health. To understand the drastically negative impacts of COVID‐19 on the public health, it is urgent to investigate key features pertinent to the disease: How severe is the transmission? How long is the incubation time of the disease on average? How many infections are asymptomatic? And ultimately, what is the case fatality rate?', 'id': '00001'}], 'title': 'Estimation of the basic reproduction number, average incubation time, asymptomatic infection rate, and case fatality rate for COVID‐19: Meta‐analysis and sensitivity analysis', 'sha': '2c63ae442219d870167e691f91cd32e4ab9a7505', 'doi': '10.1002/jmv.26041', 'score': 9.020500183105469}, {'rank': 230, 'paragraphs': [{'text': 'Among the 1209 cases with gender information in the study, 533 (44·09%) are female. The mean age of patients was 41·31 and the median age was 40. Over 80% of the cases were between 20 and 60. The youngest confirmed case in our cohort was six months-old while the oldest was 86 years-old. Table 2 shows the demographic characteristics of patients with COVID-19 in the Wuhan departure cohort and the entire data collected as of February 15, 2020. We can see that there was no significant difference, which means the selected cases present the population well.', 'id': '00012'}], 'title': 'Estimation of incubation period distribution of COVID-19 using disease onset forward time: a novel cross-sectional and forward follow-up study', 'sha': 'c61be9fc4562d250e2dec241ef75bd22d61f61d1; 6d3b3f4ab80a61c45f82c61c6c756cfc6ddf4bf2', 'doi': '10.1101/2020.03.06.20032417', 'score': 9.0201997756958}, {'rank': 231, 'abstract': 'The ongoing COVID-19 pandemic spread to the UK in early 2020 with the first few cases being identified in late January. A rapid increase in confirmed cases started in March, and the number of infected people is however unknown, largely due to the rather limited testing scale. A number of reports published so far reveal that the COVID-19 has long incubation period, high fatality ratio and non-specific symptoms, making this novel coronavirus far different from common seasonal influenza. In this note, we present a modified SEIR model which takes into account the time lag effect and probability distribution of model states. Based on the proposed model, it is estimated that the actual total number of infected people by 1 April in the UK might have already exceeded 610,000. Average fatality rates under different assumptions at the beginning of April 2020 are also estimated. Our model also reveals that the R0 value is between 7.5-9 which is much larger than most of the previously reported values. The proposed model has a potential to be used for assessing future epidemic situations under different intervention strategies.', 'body_text': [{'text': 'Authors in [9] analysed the probability distribution of onset to death and onset to recover time, and the proportion of all infections that would lead to hospitalisation according the data in China. These statistic data are used in this paper to formulate the time delay of COVID-19 status transition with the probability distribution over time. Figure 2 illustrates the basic structure of the SEIR model, a SIR derivative, where S(t) represents the susceptible cases at day t, E(t) is for the exposed cases at day t, I(t) stands for the infectious cases at day t and I Σ (t) denotes the total infected population at day t. R(t) and D(t) represent the cumulative recovered cases and cumulative deaths till day t respectively. The total of cases that may need hospital treatment at day t is denoted as H(t), while Q(t) represents the quarantined cases at day t. The time variable is t = 1, 2, · · · , K and K is the prediction horizon.', 'cite_spans': [{'start': 11, 'end': 14, 'text': '[9]', 'ref_id': 'BIBREF8'}], 'ref_spans': [{'start': 346, 'end': 354, 'text': 'Figure 2', 'ref_id': 'FIGREF0'}], 'section': ''}, {'text': 'The model dynamics are formulated as follows:', 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': '∆I Σ (t) = I Σ (t + 1) − I Σ (t) (4) where β N (t) is the contact ratio and N (t) is the current population in the whole country. In (1), the unquarantined people can contact and spread the virus to the some of the susceptible people ∆S(t). The imported case E import (t) and ∆S(t) add up to the increment of E(t). At each time step a certain proportion of E(t) will be converted to I(t) and the increment of total infected people I Σ (t) is subtracted from E(t) as shown in (2).', 'cite_spans': [{'start': 33, 'end': 36, 'text': '(4)', 'ref_id': 'BIBREF3'}], 'ref_spans': [], 'section': ''}, {'text': 'However once a person is exposed to the virus and get infected, the virus starts to incubate in his/her body and that person can then become infectious. Therefore a cumulative distribution function of the incubation time in [5] introduced and is shifted two days 1 before the symptoms onset to the new cases of E(t) to I Σ (t) as shown in Figure 2 ,', 'cite_spans': [{'start': 224, 'end': 227, 'text': '[5]', 'ref_id': 'BIBREF4'}], 'ref_spans': [{'start': 339, 'end': 347, 'text': 'Figure 2', 'ref_id': 'FIGREF0'}], 'section': ''}, {'text': 'where P E2I is a row vector of dimension K representing the cumulative probability.', 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': 'Similarly the number of recovered and death cases can also be derived by introducing two separate cumulative probability vectors P I2R and P I2D as shown in (6) and (7),', 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': 'where κ is the average infection fatality rate (IFR).', 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': 'The state Q(t) is estimated three days 2 after onset of non-mild symptoms, i.e. after people develop fever/dry cough symptoms they need either be under quarantine in hospital or start self-isolation at home. State H(t) is used to represent the accumulation of people that need hospital treatment.', 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': 'where τ is the time period from becoming infectious to having developed non-mild symptoms 3 that need either admission to hospital or self-isolation at home, λ Q and λ H are ratio of non-mild symptoms and ratio of people needing hospital admission.', 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': 'Finally, the infectious people i.e. state I at day t can be updated by (10) .', 'cite_spans': [{'start': 71, 'end': 75, 'text': '(10)', 'ref_id': 'BIBREF9'}], 'ref_spans': [], 'section': ''}, {'text': 'Apparently at day t, N (t) = S(t) + E(t) + I(t) + R(t) + D(t) is automatically satisfied.', 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': 'The real epidemiological dynamics is extremely complex and we made several assumptions to set parameters and to simplify the model:', 'cite_spans': [], 'ref_spans': [], 'section': 'Modelling Assumptions and Parameter Setting'}, {'text': '1. The genetic difference in human race is not considered. The cumulative probability functions used in the model are acquired from the analysis of the COVID-19 data from China, and are applied to the UK situation under the assumption that this virus spreading mechanism remains unchanged. 2. About 20% of the infected people will develop symptoms and can be classified as quarantined cases. 3. Infected people can become infectious two days before symptom onset. 4. The virus tests can be conducted to each patient in severe situation on the third day after COVID-19 symptom onset. 5. The officially published death case number is accurate [7] . 1 According to WHO report people can shed COVID-19 virus 24-48 hours prior to symptom onset.', 'cite_spans': [{'start': 641, 'end': 644, 'text': '[7]', 'ref_id': 'BIBREF6'}, {'start': 647, 'end': 648, 'text': '1', 'ref_id': 'BIBREF0'}], 'ref_spans': [], 'section': 'Modelling Assumptions and Parameter Setting'}, {'text': 'https: //www.who.int/docs/default-source/coronaviruse/situation-reports/20200306-sitrep-46-covid-19.pdf? sfvrsn=96b04adf_4 2 In average, patients may need hospital admission on the 3rd day after symptom onset. https://patient.info/ news-and-features/coronavirus-how-quickly-do-covid-19-symptoms-develop-and-how-long-do-they-last 3 According to the WHO report, 80% of the patients experienced mild illness https://www.who.int/docs/default-source/ coronaviruse/situation-reports/20200301-sitrep-41-covid-19.pdf?sfvrsn=6768306d_2', 'cite_spans': [{'start': 123, 'end': 124, 'text': '2', 'ref_id': 'BIBREF1'}, {'start': 329, 'end': 330, 'text': '3', 'ref_id': 'BIBREF2'}], 'ref_spans': [], 'section': 'Modelling Assumptions and Parameter Setting'}, {'text': '3 . CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Modelling Assumptions and Parameter Setting'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.04.12.20062588 doi: medRxiv preprint Infection fatality ratio 0.5-3% τ Time period from being infectious to having non-mild symptom 5 days 6. The total confirmed case number is larger than the number of cases that require hospital treatment (severe cases). 7. The earliest imported cases occurred on 15 Jan 2020 [8] and the imported cases kept growing at an exponential rate till international flights were reduced.', 'cite_spans': [{'start': 419, 'end': 422, 'text': '[8]', 'ref_id': 'BIBREF7'}], 'ref_spans': [], 'section': 'Modelling Assumptions and Parameter Setting'}, {'text': 'Based on published results, official statistics and announcements, the parameters used to build the UK pandemic model are illustrated in Fig. 4 and Fig. 5 .', 'cite_spans': [], 'ref_spans': [{'start': 137, 'end': 143, 'text': 'Fig. 4', 'ref_id': None}, {'start': 148, 'end': 154, 'text': 'Fig. 5', 'ref_id': None}], 'section': 'Modelling Assumptions and Parameter Setting'}, {'text': 'By the end of March 2020 international flights are greatly reduced from the normal capacity, we assume the number of imported cases grew exponentially before 16 March. The international flight capacity was decreased linearly afterwards and by the end of March most of the international flights are grounded. The number of imported cases is used as an adjustable parameter and is also used as an initial condition for future projection.', 'cite_spans': [], 'ref_spans': [], 'section': 'Modelling Assumptions and Parameter Setting'}, {'text': 'As shown in Figure 3 , from 23 March, pubs and schools are closed and people dispersed to their home to work and study. So we simply assume the infection rate 4 as:', 'cite_spans': [], 'ref_spans': [{'start': 12, 'end': 20, 'text': 'Figure 3', 'ref_id': 'FIGREF1'}], 'section': 'Modelling Assumptions and Parameter Setting'}, {'text': 'The actual infection fatality ratio κ and the severe case ratio λ H can not be accurately calculated, while the reported total case number might be way below the actual infected cases, therefore we have adopted the following procedures to fit the model to the UK death data (1 Feb to 10 Apr):', 'cite_spans': [], 'ref_spans': [], 'section': 'Modelling Assumptions and Parameter Setting'}, {'text': '1. Select a set of λ H and κ satisfying λ H > κ.', 'cite_spans': [], 'ref_spans': [], 'section': 'Modelling Assumptions and Parameter Setting'}, {'text': '2. Search feasible combination of ∆E max import and β 0 such that the following conditions (a)-(c) are met: (a) The average of daily reported cases must be greater than the average of estimated H(t) over the whole data fitting period, and the number of reported cases should be greater than the model estimated H(t) in the last day of the whole period, i.e. 10 April; 4 This model is used to fit the death case curve, the variation on β(t) after lockdown does not affect death number from February to the early April. So for convenience, 1 50 β is used.', 'cite_spans': [{'start': 368, 'end': 369, 'text': '4', 'ref_id': 'BIBREF3'}], 'ref_spans': [], 'section': 'Modelling Assumptions and Parameter Setting'}, {'text': '. CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '4'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.04.12.20062588 doi: medRxiv preprint ', 'cite_spans': [], 'ref_spans': [], 'section': '4'}, {'text': 'By performing the aforementioned modeling procedures, estimates of the number of actual infected people with different infection fatality ratio κ and different severe case ratio λ H are plotted in Figure 6 . The figure indicates that although the actual IFR κ cannot be calculated accurately, based on our modeling results shown in Figure 6 , the total number of infected cases could have been greater than 0.61 million by 1 April 2020. If the IFR κ was lower than 2%, the total case number could be even greater than 1 million. Several reports indicate that IFR is below 1% [2] , but based on our model, the lower IFR κ implies lower severe case ratio λ H and larger infection number. Hence, the total number of infections would exceed 2 million if IFR κ is below 1%. However, according to the UK demographic data, about 5% infected individuals may need hospital treatment, therefore a higher IFR (κ > 2%) is more likely to be true in the early stage of this virus outbreak.', 'cite_spans': [{'start': 575, 'end': 578, 'text': '[2]', 'ref_id': 'BIBREF1'}], 'ref_spans': [{'start': 197, 'end': 205, 'text': 'Figure 6', 'ref_id': 'FIGREF3'}, {'start': 332, 'end': 340, 'text': 'Figure 6', 'ref_id': 'FIGREF3'}], 'section': 'Epidemic prevalence estimation on 1st April 2020'}, {'text': 'By curve-fitting the results illustrated in Figure 6 , the estimate of the infection rate β is around 0.41. The basic reproduction ratio R 0 can be estimated by calculating S(∞) − S(1) while setting E(1) = 1, ∆E(t) = 0. In our estimate R 0 ≈ 8.8 which is much larger than the results obtained in other published papers. However if we assume anyone who is infected by the virus will be under quarantine 7 days after s/he becomes infectious, then our model estimate for R 0 ≈ 2.8-3.6 which is comparable to the current popular estimates of R 0 reported in the literature [10] . The reason for our estimated R 0 being much larger than the R 0 estimates reported in the literature might be due to that there is a large proportion of people who only got mild symptoms or even were asymptomatic. They can spread the virus before symptom onset till recovery over a long period of time before the lockdown measure was put in place.', 'cite_spans': [{'start': 569, 'end': 573, 'text': '[10]', 'ref_id': 'BIBREF9'}], 'ref_spans': [{'start': 44, 'end': 52, 'text': 'Figure 6', 'ref_id': 'FIGREF3'}], 'section': 'Epidemic prevalence estimation on 1st April 2020'}, {'text': 'As presented in the previous section, although the severe case ratio λ H and the infection fatality ratio κ cannot be accurately obtained, our model produces some consistent estimates on the total death toll if community spread is greatly suppressed after 23 March. If the UK government continues the current lockdown strategy to the end of June, the estimated total death number is about 21000 (if the reproduction ratio is suppressed to 0.2). Figure 7 and Figure 8 illustrate the total number of deaths and infections to the end of April if the reproduction ratio is controlled to be less than 0.2 after the lockdown, and assume the severe case ratio λ H = 4%, infection fatality ratio (IFR) κ = 2%, and the 5 https://www.hsj.co.uk/coronavirus/covid-19-hospital-admissions-flattening/7027364.article 5 . CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. Figure 10 : Daily increase in hospital treatment maximum imported cases E max import = 2100. These figures suggest that about 1 million people would be infected, and 81.2% of infected people will be recovered by the end of April in this scenario.', 'cite_spans': [], 'ref_spans': [{'start': 445, 'end': 453, 'text': 'Figure 7', 'ref_id': None}, {'start': 458, 'end': 466, 'text': 'Figure 8', 'ref_id': None}, {'start': 958, 'end': 967, 'text': 'Figure 10', 'ref_id': None}], 'section': 'Assessment of future epidemic situation'}, {'text': 'However the death toll might be higher if the lockdown rules are not strictly obeyed by people unless improved medical treatment is available in the forthcoming weeks. On the other hand, a long period of lockdown may damage the economy to an unacceptable extent, and the wishes to lift the lockdown measure as early as possible may grow with time. Figure 9 and Figure 10 show different curves representing possible daily increase in death toll and daily increase in hospital treatment requirements under different reproduction rates. Clearly if the reproduction rate is suppressed below 1, both numbers will keep decreasing. However even the viral transmission is managed to the minimum, the daily death toll will still likely keep at a high level (>500) for two weeks. Based on the current data 6 in the UK, our model also projects that the inflection point of current pandemic wave in the UK is likely to occur in the following week (between 12-15 April 2020) if the lockdown rules are strictly obeyed by people.', 'cite_spans': [], 'ref_spans': [{'start': 348, 'end': 356, 'text': 'Figure 9', 'ref_id': None}, {'start': 361, 'end': 370, 'text': 'Figure 10', 'ref_id': None}], 'section': 'Assessment of future epidemic situation'}, {'text': 'The model shows that in the ideal situation where everyone stays at home until the end of April, then the daily death number can be suppressed to less than 200 or even less afterwards. Similarly the curve of the number of infected people who need hospital treatment can also be flattened by mid of April.', 'cite_spans': [], 'ref_spans': [], 'section': 'Assessment of future epidemic situation'}, {'text': 'According to Figure 11 7 , if people start to return to work in May, then the reproduction ratio has to be suppressed below 2 in order that the NHS resources are sufficient to cope with the medical treatment requirements in the next two months (May and June). 6 up to 10 Apr 2020 7 Because the hospitalised number is based on Chinese data. It can be different from the UK scenario where the hospital admission procedures are different. In Figure 11 , λH is set to 4%.', 'cite_spans': [{'start': 260, 'end': 261, 'text': '6', 'ref_id': 'BIBREF5'}, {'start': 280, 'end': 281, 'text': '7', 'ref_id': 'BIBREF6'}], 'ref_spans': [{'start': 13, 'end': 24, 'text': 'Figure 11 7', 'ref_id': 'FIGREF5'}, {'start': 439, 'end': 448, 'text': 'Figure 11', 'ref_id': 'FIGREF5'}], 'section': 'Assessment of future epidemic situation'}, {'text': '6 . CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Assessment of future epidemic situation'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.04.12.20062588 doi: medRxiv preprint However if the reproduction ratio can not be suppressed well enough while the lockdown measure is lifted in May, one feasible solution to slow down the transmission might be to allow people work every other day. This kind of intervention measures (e.g. only allow people to go outside every other day) has a similar pattern (r=4,0.2) 8 as the curve with a halved reproduction ratio (r=2).', 'cite_spans': [], 'ref_spans': [], 'section': 'Assessment of future epidemic situation'}, {'text': 'In this paper, a modified SEIR model has been proposed to allow the state transitions to be governed by the cumulative probability vector rather than by an averaged value, so the COVID-19 data can be utilised to reflect the distribution characteristics over a long period of time. Time variant parameters such as the number of imported cases and infection rate are also introduced and designed to reflect the changes introduced by adopting intervention measures such as reducing the number of international flights and the lockdown measures.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusion and Discussion'}, {'text': 'The model is fitted using the officially published data. Because the number of total reported cases may not precisely reflect the reality, the death toll is used instead to assess the pandemic progression in the UK. The results reveal that more than 610,000 people might have been infected by 1 April 2020 and the average death rate in last month is very likely to be greater than 1% and may be even higher than 2%. Also according to the proposed model, the value of R 0 is much greater than the previously published values in the literature, if no intervention measure is implemented. The estimated range of R 0 is between 7.5-9 due to large proportion of patients with mild symptoms. However, if people could keep self-isolated after developing COVID-19 symptoms, then the reproduction ratio is about 2-3, which is comparable to the values published in the literature.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusion and Discussion'}, {'text': \"Our model reveals that if people fully comply the 'stay at home' order to the end of April, the total death toll is likely to be less than 21,000. To lift the lockdown measure from May, people will need to keep the strict social distance rule and apply other personal protection measures to ensure that the reproduction ratio is suppressed below 2 in order that the NHS has the sufficient capacity to cope with the medical treatment demand from people having severe COVID-19 symptoms. However, if this is not achievable then interventions measures such as people stay at home every other/third day may equally reduce the reproduction ratio in the near future.\", 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusion and Discussion'}, {'text': '. CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusion and Discussion'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.04.12.20062588 doi: medRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusion and Discussion'}], 'paragraphs': [{'text': 'https: //www.who.int/docs/default-source/coronaviruse/situation-reports/20200306-sitrep-46-covid-19.pdf? sfvrsn=96b04adf_4 2 In average, patients may need hospital admission on the 3rd day after symptom onset. https://patient.info/ news-and-features/coronavirus-how-quickly-do-covid-19-symptoms-develop-and-how-long-do-they-last 3 According to the WHO report, 80% of the patients experienced mild illness https://www.who.int/docs/default-source/ coronaviruse/situation-reports/20200301-sitrep-41-covid-19.pdf?sfvrsn=6768306d_2', 'id': '00014'}], 'title': 'An SEIR Model for Assessment of Current COVID-19 Pandemic Situation in the UK', 'sha': '82340a413835a89bf51ae36292506e9ad53d99c3', 'doi': '10.1101/2020.04.12.20062588', 'score': 9.018099784851074}, {'rank': 232, 'paragraphs': [{'text': 'Our investigation indicates that the cases are highly infectious even during the incubation period. Moreover, it has been demonstrated that the familial outbreak of COVID-19 has a potential for human-to-human transmission, even during the incubation period. This is supported by our investigation, where the index case during the incubation period transmitted COVID-19 to one asymptomatic and seven symptomatic cases. Thus, it is prudent to consider this fact while carrying out an epidemiological investigation and taking control measures against COVID-19, in future, including a quarantine and observation period for the suspected cases. Moreover, it is judicious to consider an incubation period, as long as 21 days, as highlighted by our investigation.', 'id': '00022'}], 'title': 'Outbreak of COVID-19 in a family, Wenzhou, China', 'sha': '4fabdcf18948e7fd575dee30654f1bb01ac0b6ee', 'doi': '10.1017/s0950268820001089', 'score': 9.015000343322754}, {'rank': 233, 'paragraphs': [{'text': 'Parameters for log-normal distribution of incubation period were derived for each study (Supplement For comparison with other viral respiratory infections, the summarized statistics of incubation periods for 9 viral respiratory infections were obtained from a previously published systematic review, including measles, adenovirus, respiratory syncytial virus, SARS-CoV, human coronavirus, parainfluenza, rhinovirus, influenza A, and influenza B (17) . In addition, meta-analysis was performed . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. Table 14 ) (21), assuming a 6.15% risk of symptomatic infections among the overall active monitoring population with close contacts, the estimated probability of symptomatic SARS-CoV-2 infections that would be undetected after 14-day active monitoring or quarantine was 20.5 (95%CI,5.2-50.4) per 10 000 monitored individuals ( Figure 4G ). Similarly, one study summarized the proportion of young-age infections at 1.6% among general infections (22) ; thus, among the overall active monitoring population, the estimated probability of symptomatic SARS-CoV-2 infections that would be undetected after 14-day active monitoring was 0.9 in 10 000 (95%CI: 0.2-2.2 in 10 000) ( Figure 4H ). In addition, assuming a 4.11% risk of symptomatic infections among contacts with latent infections and a proportion of latent infections at 9.3% in the overall active monitoring population, the estimated probability of undetected SARS-CoV-2 infections with no sympotom within 14-day active . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'id': '00022'}], 'title': 'A systematic review and meta-analysis reveals long and dispersive incubation period of COVID-19', 'sha': '3d921c03dc23ee850655ddfd9d5ebe3956d70bf8', 'doi': '10.1101/2020.06.20.20134387', 'score': 9.006500244140625}, {'rank': 234, 'abstract': 'A novel coronavirus strain 2019-nCoV has caused a rapid global pandemic-COVID-19. Scientists have taken onto the task of characterizing this new virus and understanding how this virus has transmitted to humans. All preliminary studies have found some striking similarities between this new virus and the SARS-CoV that caused a similar kind of epidemic in 2002–2003. Through bioinformatics tools, a great deal of information has been gathered about the origin, evolution and zoonosis of this virus. We, in this review, report the symptoms, mode of transmission and available and putative treatments to tackle 2019-nCoV infections. We also comprehensively summarize all the information so far made available regarding the genome, evolution and zoonosis of this virus.', 'body_text': [{'text': 'Since December 21, 2019, China has been under the grip of a severe epidemic with cases even being reported at a rapid rate from neighboring countries like Japan, Thailand, South Korea, Nepal and India as well as far flung countries like the USA, the UK, Italy, Germany and France (https://www.who.int>docs). The cause of this epidemic is a novel strain of coronavirus earlier temporarily designated by World Health Organization (WHO) as 2019-novel coronavirus (2019-nCoV)1, and the infection due to 2019-nCoV has been designated as COVID-19 (CO—corona, VI—virus, D—disease and 19 stands for 2019).', 'cite_spans': [], 'section': 'Introduction', 'ref_spans': []}, {'text': 'The previous related outbreaks like the 2002–03 global severe acute respiratory syndrome coronavirus (SARS-CoV) outbreak that originated in the Guangdong province in China or the Middle East Respiratory Syndrome coronavirus (MERS CoV) outbreak that originated in Saudi Arabia had a mortality rate of 9.6% and 37% respectively (Drosten et al. 2003; Fouchier et al. 2003; Zhong et al. 2003; Zaki et al. 2012; de Wit et al. 2016). The 2019-nCoV is the seventh coronavirus known to infect humans. Of these, four coronavirus strains HCoV-229E, HCoV-OC43, HCoV-NL6 infect infants, the upper respiratory tract of the immunocompetent and lower respiratory tract of immunocompromised individuals causing mild cold and flu-like symptoms. In contrast, the highly pathogenic SARS CoV, MERS-CoV and 2019-nCoV infect the lower respiratory tract (bronchial epithelial cells) of immunocompetent individuals causing chronic respiratory ailments.', 'cite_spans': [{'start': 342, 'end': 346, 'mention': '2003', 'ref_id': 'BIBREF2'}, {'start': 364, 'end': 368, 'mention': '2003', 'ref_id': 'BIBREF4'}, {'start': 383, 'end': 387, 'mention': '2003', 'ref_id': 'BIBREF34'}, {'start': 401, 'end': 405, 'mention': '2012', 'ref_id': 'BIBREF29'}, {'start': 421, 'end': 425, 'mention': '2016', 'ref_id': 'BIBREF1'}], 'section': 'Introduction', 'ref_spans': []}, {'text': 'Almost all the index patients of the 2019-nCoV outbreak were either sellers or visitors at the Huanan seafood market in Wuhan where besides aquatic animals even mammals and snakes were sold. It is speculated that the zoonosis of the virus happened from an intermediate host animal being sold at the market. However, no bats were being sold. It is similar to how Himalayan civets (Paguma larvata) acted as the intermediate host for the zoonosis of SARS CoV in 2002–03 although horseshoe bats (belonging to genus Rhinolophus) were primary hosts coronaviruses are positive-sense single-stranded RNA viruses enclosed in a solar corona shaped envelope (hence the name). They belong to the sub-family Coronavirinae of the family Coronaviridae of the order Neovirales. They are further divided into four genera—Alphacoronavirus, Betacoronavirus, Gammacoronavirus and Deltacoronavirus. Alphacoronavirus and Betacoronavirus infect only mammals, whereas the Gammacoronavirus and Deltacoronavirus mostly infect birds. The 2019-nCoV along with the SARS and MERS CoV belongs to the genus Betacoronavirus. Data from the phylogenetic analysis show that 2019-nCoV is distantly related to SARS CoV (with less than 79% sequence identity at genome level) or MERS CoV (less than 50% sequence identity). This proves that the 2019-nCoV is novel and distinct from the previously reported pathogenic coronaviruses. The 2019-nCoV is further classified under the sub-genera Sarbecoviruses along with two closely related (about 88% identity) bat-derived SARS-related coronaviruses (SARSr-CoV) designated as SL-CoVZC45 and SL-CoVZXC21. This throws light on the fact that 2019-nCoV like all previously reported pathogenic coronaviruses (SARS and MERS CoV) has its ancestry to coronaviruses with bats as the primary host. However, the ancestry of 2019-nCoV is not directly linked to the two bat-derived coronaviruses and homologous recombination events in an intermediate host have played a key role in the evolution of this novel strain.', 'cite_spans': [], 'section': 'Introduction', 'ref_spans': []}, {'text': 'Symptoms exhibited by COVID-19 patients include pneumonia (which includes fever, dyspnoea, malaise, dry cough and respiratory distress), leucopenia and lymphopenia (a low lymphocyte count below a threshold has been suggested as a method for initial clinical diagnosis) (Hui et al. 2020; Chen et al. 2020a; Huang et al. 2020). Less common symptoms include headache, diarrhea, sputum production and hemoptysis (Zhang et al. 2020). Moreover, chest radiographs exhibit multifocal ground glass lung opacities. The incubation period of the virus is relatively long of about 5–14 days (Bouadma et al. 2020; Jiang et al. 2020b). Many patients that go asymptomatic during incubation period become major cause of the spread of this virus.', 'cite_spans': [{'start': 281, 'end': 285, 'mention': '2020', 'ref_id': 'BIBREF10'}, {'start': 299, 'end': 304, 'mention': '2020a', 'ref_id': 'BIBREF38'}, {'start': 319, 'end': 323, 'mention': '2020', 'ref_id': 'BIBREF9'}, {'start': 422, 'end': 426, 'mention': '2020', 'ref_id': 'BIBREF31'}, {'start': 594, 'end': 598, 'mention': '2020', 'ref_id': 'BIBREF0'}, {'start': 613, 'end': 618, 'mention': '2020b', 'ref_id': 'BIBREF14'}], 'section': 'Infection symptoms, host immunological response and transmission', 'ref_spans': []}, {'text': 'In a study on nine index patients (out of which eight had visited the Huanan seafood market in Wuhan), there was a sequence similarity of greater than 99.9% among the isolated and cultured 2019-nCoV virus particles. Thus, all the patients were infected from the same animal source in a short span of time. All the patients had almost simultaneously developed the symptoms quite quickly.', 'cite_spans': [], 'section': 'Infection symptoms, host immunological response and transmission', 'ref_spans': []}, {'text': 'Most of the severely ill patients were reported to be in their old age, had some previously underlying conditions like diabetes or cardiovascular or cerebrovascular diseases or were co-infected with infectious bacteria or fungi like A. baumannii and C. albicans. The number of males infected with 2019-nCoV is higher than the number of females. This trend was observed even at the time of the SARS-CoV outbreak. A reason behind this trend can be that X chromosome and sex hormones positively regulate innate and adaptive immunity enhancing anti-viral immunity in females (Channappanavar et al. 2017). Moreover, the 2019-nCoV like the previous SARS and MERS CoV infections elicits strong cytokine responses in the host causing increased pulmonary inflammation and lung damage. Elevated levels of IgA and IgM in the lungs of an infected patient had been detected. The viral infection leads to the production of IL1B, IFNγ, IP10 and MCP1 which further lead to T-Helper 1 cell production (pro-inflammatory). However, a striking observation is that unlike the previous SARS CoV the 2019-nCoV elicited the production of T-Helper 2 cell-related cytokines like IL-10 and IL-4 as well (which are associated with anti-inflammatory signature). This can be one of the reasons for relatively less severe symptoms in many patients.', 'cite_spans': [{'start': 594, 'end': 598, 'mention': '2017', 'ref_id': 'BIBREF37'}], 'section': 'Infection symptoms, host immunological response and transmission', 'ref_spans': []}, {'text': 'The ninth patient from the above-mentioned study had, however, never visited the Huanan seafood market but stayed in a hotel near the market. It is speculated that this patient must have been infected either through an unknown source or through droplet transmission. Moreover, spread of infection in family clusters from these patients establishes the possibility of human to human transmission of infection (Chan et al. 2020b). All the early patients diagnosed with novel coronavirus infection outside China like in Japan, Thailand or India had visited Wuhan just days before the outbreak (Zhao et al. 2020). Given the cases where health workers or other patients and visitors to hospitals across Wuhan have contracted the infection, airborne droplets due to coughing, touching and using personal hygiene items (like towels and cosmetics) of the infected and nosocomial transmission seem the most probable modes of further spread of infection. However, further investigations are required to understand the exact mode of transmission of this virus among humans.', 'cite_spans': [{'start': 421, 'end': 426, 'mention': '2020b', 'ref_id': 'BIBREF33'}, {'start': 603, 'end': 607, 'mention': '2020', 'ref_id': 'BIBREF32'}], 'section': 'Infection symptoms, host immunological response and transmission', 'ref_spans': []}, {'text': 'All the earliest patients of the outbreak in Wuhan were diagnosed with pneumonia from an unknown cause. Bronchoalveolar lavage fluid (BALF) was collected from patients for the isolation of nucleic acid samples which was tested for 22 pathogens (18 viruses and 4 bacteria) using PCR. All the tests resulted in negative.', 'cite_spans': [], 'section': 'Isolation and characterization of the novel coronavirus', 'ref_spans': []}, {'text': 'BALF along with throat swab from one patient was then cultured on human airway epithelial cells (HAL) (Zhu et al. 2020). Viral particles from HAL showing cytopathic effects were collected for RNA isolation. The isolated RNA samples were then cloned as cDNA and sequenced.', 'cite_spans': [{'start': 114, 'end': 118, 'mention': '2020', 'ref_id': 'BIBREF36'}], 'section': 'Isolation and characterization of the novel coronavirus', 'ref_spans': []}, {'text': 'A BLASTn of the sequenced cDNAs (query sequences) was performed for which no matching sequence was found in the GenBank database. However, the query sequences had high similarity with the Betacoronavirus sequences pertaining to lineage B (which include SARS CoV and SARSr-CoV). The highly similar sequences were then used as references for analysis and gene annotation of the query sequences. Further electron microscopic analysis of the viral particles in the inclusion bodies in the cytoplasm of the cytopathic HAL revealed spiked envelope with a solar corona-like shape, confirming that the viral particles belonged to the family Coronaviridae. Phylogenetic analysis of the novel query sequences along with other Betacoronavirus genome sequences revealed that the closest relatives of the 2019-nCoV are bat-derived SL-CoVZC45 and SL-CoVZXC21 (forming one clade), whereas the SARS-CoV is distantly related forming a separate clade (Fig. 1). However, the 2019-nCoV formed a distinct monophyletic cluster within the clade with a long branch separating away the two bat-derived SARSr-CoV (Fig. 2).', 'cite_spans': [], 'section': 'Isolation and characterization of the novel coronavirus', 'ref_spans': [{'start': 939, 'end': 940, 'mention': '1', 'ref_id': 'FIGREF0'}, {'start': 1093, 'end': 1094, 'mention': '2', 'ref_id': 'FIGREF1'}]}, {'text': 'The RNA-dependent RNA polymerase (RdRp) gene in the ORF1a/b region of the Betacoronavirus genomes is highly conserved across the genus (Lu et al. 2020). The RdRp gene thus formed the basis for real-time-PCR (RT-PCR)-based laboratory diagnosis of the 2019-nCoV infection in early index patients. The cycle threshold or ct values of the patient-derived samples ranged from 22.85 to 32.41. These low ct values may be indicative of a high viral nucleic acid abundance in patient samples due to a very high rate of virus replication resulting in enhanced severity of the infection. The S gene in the 2019-nCoV genome that codes for the spike glycoprotein has also formed the basis for RT-PCR-based diagnosis of infection. The spike protein on the outer surface of coronaviruses is responsible for host cell receptor binding and invasion.', 'cite_spans': [{'start': 146, 'end': 150, 'mention': '2020', 'ref_id': 'BIBREF19'}], 'section': 'Isolation and characterization of the novel coronavirus', 'ref_spans': []}, {'text': 'Gene annotation studies have successfully deciphered the genome of the novel coronavirus in great detail (Chan et al. 2020a). The single-stranded RNA of the 2019-nCoV consists of 29,881 nucleotides coding for 9860 amino acids. The either ends of the RNA genome are flanked by 5′ and 3′ untranslated region (UTR) which is similar to that of other Betacoronavirus. Starting from the 5′ end, the genome consists of ORF1 a/b that codes for 16 non-structural proteins (nsp) including RdRp followed by other ORFs coding for accessory and structural protein genes—spike (S), envelope (E), membrane (M) and nucleocapsid (N). There is no major difference between the functions of various nsp of 2019-nCoV and SARS-CoV.', 'cite_spans': [{'start': 118, 'end': 123, 'mention': '2020a', 'ref_id': 'BIBREF22'}], 'section': 'Genomic structure and replication cycle of the 2019-nCoV', 'ref_spans': []}, {'text': 'Coronaviruses produce a set of nested sub-genomic (SG) mRNAs of varying sizes which are then translated to produce various structural and accessory proteins (Sztuba-Solińska et al. 2011). These SG mRNAs are synthesized by a ‘discontinuous template synthesis’ method in which a 5′ leader sequence is joined to an ORF(s) via the transcription regulatory sequence (TRS). TRS is a conserved 5–7 nucleotide sequences upstream of ORFs in coronavirus genomes. Bioinformatics studies have revealed that in 2019-nCoV around 12 ORFs produce a set of 9 nested SG mRNAs. Inside a host cell when the virus makes its copies, the orf1a/b is directly translated to produce two polypeptides pp1a and pp1ab which are cleaved to form 16 nsp including RdRp. These nsp together form a replication transcription complex (RTC). RTC then replicates the virus genome and produces various structural and accessory protein SG mRNAs inside a double membrane vesicle.', 'cite_spans': [{'start': 181, 'end': 185, 'mention': '2011', 'ref_id': 'BIBREF24'}], 'section': 'Genomic structure and replication cycle of the 2019-nCoV', 'ref_spans': []}, {'text': 'The 2002–03 SARS-CoV binds with the angiotensin-converting enzyme 2 (ACE 2) receptor present on bronchial epithelial cells and type II pneumocytes in the lower respiratory tract via the receptor binding domain (RBD) of the spike protein. The spike protein is divided into two functional subunits—S1 domain and S2 domain. The S1 domain is responsible for receptor binding, whereas the S2 domain is responsible for fusion of virus with the host cell membrane. The SARS-CoV forms a clover-shaped trimer with three S1 heads and a trimeric S2 stalk (Cui et al. 2019). The S1 domain is further sub-divided into an amino terminal domain (S1-NTD) and a cysteine-rich carboxy terminal domain (S1-CTD). The S1-CTD at the tip of S1 domain acts as the RDB. The RBD is further sub-divided into a core region and a receptor binding motif (RBM). The compatibility of spike protein with the host cell receptor is largely dependent on the amino acid residues in the RBM. Any changes in the RBM or even core region amino acid residues can decrease or increase the affinity of spike –receptor binding. Changes in amino acid residues in RBM can even alter the virus’ tropism (Song et al. 2005).', 'cite_spans': [{'start': 556, 'end': 560, 'mention': '2019', 'ref_id': 'BIBREF40'}, {'start': 1168, 'end': 1172, 'mention': '2005', 'ref_id': 'BIBREF23'}], 'section': 'Receptor binding and animal to human transmission of the 2019-nCoV', 'ref_spans': []}, {'text': 'Residues 442, 472, 479, 480 and 487 in the SARS-CoV RBM contribute to host receptor specificity and in determining host range (Wu et al. 2012). The residues at each of these positions form a salt bridge with specific ACE-2 receptor residues. Asparagine or arginine at residue 479 of RBM increases binding affinity with human and civet (intermediate host of SARS-CoV) ACE-2, whereas lysine at the same position decreases this binding affinity. Serine at residue 487 is required for RBM binding with civet ACE-2, but it fails to bind efficiently with human ACE-2. On the other hand, mutations in SARS-CoV leading to threonine at residue 487 lead to very efficient RBM and human ACE-2 binding. The early phase SARS-CoV strain-hcGd03 (Ser at residue487) that infected humans from civets had low infectivity and failed to transmit between humans. However, middle- and late-phase SARS-CoV strains especially hTor02 had a high infectivity and rapidly transmitted between humans as it had Thr at residue 487. Therefore, it can be inferred that RBD of spike protein in Betacoronavirus is the cornerstone of infectivity and animal to human transmissions.', 'cite_spans': [{'start': 137, 'end': 141, 'mention': '2012', 'ref_id': 'BIBREF26'}], 'section': 'Receptor binding and animal to human transmission of the 2019-nCoV', 'ref_spans': []}, {'text': 'The overall phylogenetic analysis of spike protein of 2019-nCoV with various reference genomes of Betacoronavirus is more or less similar to the full-length genome phylogenetic analysis. In fact, the S2 domain of 2019-nCoV has a high degree of similarity with its two bat-derived ancestors—SL-CoVZC45 and SL-CoVZXC21, but interestingly, on the other hand, the S1 domain shares very little similarity. Rather the S1 domain of 2019-nCoV is very similar to that of SARS-CoV with around 50 conserved amino acid residues despite the fact the two falls in different clades. Consistent with this, the homology modeling of RBD of 2019-nCoV with SARS-CoV RBD as template revealed that the RBM of the 2019-nCoV is very similar in structure to that of SARS-CoV. These results led to the conclusion that 2019-nCoV may also bind to the human ACE-2 receptor. Another study using HeLa cells expressing ACE-2 also reached the same conclusion that 2019-nCoV may bind with the ACE-2 receptors (Zhou et al. 2020). However, many residues that are crucial for binding of SARS-CoV RBD with ACE-2 vary in 2019-nCoV. The significance of these variations on ACE-2 binding needs to be further investigated.', 'cite_spans': [{'start': 988, 'end': 992, 'mention': '2020', 'ref_id': 'BIBREF35'}], 'section': 'Receptor binding and animal to human transmission of the 2019-nCoV', 'ref_spans': []}, {'text': 'The rate of 2019-nCoV spread and resultant deaths have dramatically risen with time in Wuhan despite proper quarantine measures in place. As more strains from the later phases (after the onset) of the 2019-nCoV outbreak are isolated and sequenced, mutations in the S gene leading to the accelerated transmission and enhanced infectivity will be elucidated. These further investigations will also throw light on the recombination events that led to animal–human transmission of this virus.', 'cite_spans': [], 'section': 'Receptor binding and animal to human transmission of the 2019-nCoV', 'ref_spans': []}, {'text': 'Recombination events leading to zoonotic 2019-nCoV from bat-derived SL-CoVZC45 and SL-CoVZXC21 must have taken place inside bats or in some unknown intermediate host being sold at Huanan seafood market. A study based on comparative relative synonymous codon usage bias (RSCU) analysis claimed that two species of snakes—Bungarus multicinctus and Naja atra, acted as intermediate host for 2019-nCoV in which the recombination events happened about 2 years ago (Ji et al. 2020). The principal behind RSCU analysis is that viruses tend to evolve codon usage bias similar to that of their hosts. The study compared the synonymous codon usage bias in 2019-nCoV with that of all possible intermediate hosts—birds, marmots, hedgehog, bats, snakes and humans, and the results were expressed as squared Euclidean distance. The two species of snakes mentioned above had the least squared Euclidean distance value compared to other animals. The RSCU results were further supported by claims that the two species of snakes are found in the central Hubei province and were even being sold at Huanan sea food market. The study has, however, been discredited by a large number of scientists on the grounds that it is highly unlikely that any coronavirus strain would ever infect a reptile which are so distant from mammals.', 'cite_spans': [{'start': 470, 'end': 474, 'mention': '2020', 'ref_id': 'BIBREF12'}], 'section': 'Receptor binding and animal to human transmission of the 2019-nCoV', 'ref_spans': []}, {'text': 'According to recent reports (Cyranoski 2020), researchers at the South China Agricultural University in Guangzhou have claimed that the endangered pangolins are most likely the intermediate host of 2019-nCoV before it spread to humans. This claim is based on the fact that genome sequences of viruses isolated from pangolins are 99% similar to the genome sequence of 2019-nCoV isolated from patients and that coronavirus use receptors with similar molecular structure in pangolins as in humans for infection. Further supporting this claim is a study that reported coronavirus infection is one of the causes of deaths in pangolins (Liu et al. 2019). This claim has not been published so far and requires further investigations to precisely ascertain how this zoonosis from pangolins to humans took place.', 'cite_spans': [{'start': 39, 'end': 43, 'mention': '2020', 'ref_id': 'BIBREF41'}, {'start': 642, 'end': 646, 'mention': '2019', 'ref_id': 'BIBREF18'}], 'section': 'Receptor binding and animal to human transmission of the 2019-nCoV', 'ref_spans': []}, {'text': 'Coronaviruses have an unusually large RNA genome (26–30 kb) which makes them prone to errors during replication. The average error rate of RdRp is 10–3–10–5 mutations per nucleotide per replication cycle. However, a set of RNA proofreading and editing enzymes like Exonuclease N synthesized by coronaviruses enhance the RdRp fidelity by around tenfold. It has been observed that during ecological stresses and positive selection pressures to switch between hosts the RdRp fidelity decreases presumably due to decrease in Exonuclease N activity. This in turn makes the coronavirus prone to higher rates of mutations and may lead to altered tropism. Studies have in fact shown when the selection pressures are over and the virus has successfully switched to new host, RdRp fidelity is enhanced again.', 'cite_spans': [], 'section': 'Polymerase error rate ::: Molecular evolution of 2019-nCoV from bat-derived ancestors', 'ref_spans': []}, {'text': 'It is considered to be the most probable mechanism by which coronaviruses evolve. Homologous recombination between coronavirus genomes may take place during co-infection inside a host cell. Coronaviruses take advantage of TRS to carry on homologous recombination using a model similar to copy-choice recombination model. The RdRp while attached to the nascent nucleic acid molecule may switch to an acceptor template from a donor template where the acceptor and donor templates are from different closely related coronaviruses. The 2019-nCoV genome may have undergone a similar kind of homologous recombination with a closely related donor coronavirus during co-infection in an intermediate host.', 'cite_spans': [], 'section': 'Homologous recombination ::: Molecular evolution of 2019-nCoV from bat-derived ancestors', 'ref_spans': []}, {'text': 'The ability of viruses to establish and sustain infections in host cells is known as viral persistence. In case of downregulation of the expression of host receptor through which the coronavirus binds, the virus may evolve to either be able to bind to a different receptor or may bind to the original receptor with even greater affinity. As mentioned above, the SARS-CoV strains isolated from middle- and late-phase patients of the outbreak were more virulent and had a higher affinity for ACE-2 receptors compared to that of early phase. It is speculated that this happened due to accumulation of mutations in the S gene of middle- and late-phase strains resulting in greater affinity of spike protein for ACE-2. The steep rise in number of cases as well as deaths in a short span of time due to 2019-nCoV infection can be attributed to viral persistence inside host cell and during human–human transmission.', 'cite_spans': [], 'section': 'Viral persistence ::: Molecular evolution of 2019-nCoV from bat-derived ancestors', 'ref_spans': []}, {'text': 'A very limited dosage of methylprednisolone (1–2 mg/kg/day) has been suggested to alleviate the symptoms like acute respiratory distress and lung inflammation (Huang et al. 2020). Although corticosteroids were found to reduce inflammation in lungs due to SARS-CoV studies had also reported delay in viral clearance due to corticosteroids. Recombinant IFN along with Ribavirin is also being administered to 2019-CoV patients, but the treatment is not very effective (Chen et al. 2020b).', 'cite_spans': [{'start': 173, 'end': 177, 'mention': '2020', 'ref_id': 'BIBREF9'}, {'start': 478, 'end': 483, 'mention': '2020b', 'ref_id': 'BIBREF39'}], 'section': 'Medical interventions to combat 2019-nCoV', 'ref_spans': []}, {'text': 'Hydroxychloroquine (an anti-malarial drug) administration has emerged as a popular and effective strategy to treat COVID-19 patients worldwide. The drug alleviates inflammation and resultant respiratory distress by neutralizing the pro-inflammatory cytokine (such as IL-6) and macrophage response (Brufsky 2020). The drug may also act as anti-viral by inhibiting virus replication inside host cells and by decreasing ACE2 affinity for spike glycoprotein (Sinha and Balayla 2020).', 'cite_spans': [{'start': 306, 'end': 310, 'mention': '2020', 'ref_id': 'BIBREF11'}, {'start': 473, 'end': 477, 'mention': '2020', 'ref_id': 'BIBREF21'}], 'section': 'Medical interventions to combat 2019-nCoV', 'ref_spans': []}, {'text': 'Combined lopinavir and ritonavir is an effective regime in controlling HIV infection. The administration of this combination was even found effective for alleviation of clinical symptoms due to SARS-CoV infection (Huang et al. 2020). This combination has also been successfully tried on 2019-nCoV patients admitted in hospitals and can be used as treatment till more effective drugs are found. Transfer of plasma and antibodies from convalescent patients to severely affected patients is also being proposed.', 'cite_spans': [{'start': 227, 'end': 231, 'mention': '2020', 'ref_id': 'BIBREF9'}], 'section': 'Medical interventions to combat 2019-nCoV', 'ref_spans': []}, {'text': 'The S2 domain is highly conserved between 2019-nCoV, its two bat-derived ancestors and SARS-CoV. Development of anti-viral peptides targeting the S2 domain has been also been proposed to combat a wide range of SARS-related infections (Du et al. 2009). A study has found that dimerization of nsp9 and its subsequent binding with nucleic acid is essential for nucleic acid replication in coronaviruses (Zeng et al. 2018). The dimerization of nsp9 is brought about by N-finger motif in nsp9. Drugs targeting nsp9 dimerization can also be an effective treatment in curbing 2019-nCoV infection.', 'cite_spans': [{'start': 245, 'end': 249, 'mention': '2009', 'ref_id': 'BIBREF3'}, {'start': 413, 'end': 417, 'mention': '2018', 'ref_id': 'BIBREF30'}], 'section': 'Medical interventions to combat 2019-nCoV', 'ref_spans': []}, {'text': 'As of now, most of the patients have to rely on their immune system to fight 2019-nCoV infection as no vaccine or effective drug is available. It may take several months if not years before one such vaccine is even developed and commercialized. Possible vaccine may include attenuated 2019-nCoV strains, substructures like membrane or envelope proteins, etc.', 'cite_spans': [], 'section': 'Medical interventions to combat 2019-nCoV', 'ref_spans': []}, {'text': 'A recent study has found that coronaviruses can persist on commonly touched inanimate objects like door knobs for about 9 days and for up to a month under low temperatures (about 4–5 °C). Steps as simple as regular disinfection procedures using 70% ethanol or 0.1% sodium hypochlorite are highly effective in eradicating coronaviruses (Kampf et al. 2020). Therefore, use of these simple disinfection procedures in houses, hospitals and various public places will significantly reduce 2019-nCoV spread.', 'cite_spans': [{'start': 349, 'end': 353, 'mention': '2020', 'ref_id': 'BIBREF15'}], 'section': 'Medical interventions to combat 2019-nCoV', 'ref_spans': []}, {'text': 'Bats are known to be the reservoirs of many deadly viruses like the Nipa virus and Ebola beside coronaviruses. A reason for this is the inability of bats’ innate immunity to sense viral DNAs in the cytosol. Bats have a very high rate of metabolism and increased rates of oxidative phosphorylation cause self-DNA damage leading to its escape from the nucleus and mitochondria into the cytosol. Bats have with time evolved to ignore this self-DNA in the cytosol by dampening of STING mechanism of DNA sensing and downregulating the production of appropriate IFN response. This in turn leads to even viral DNA in bat cytosol to go undetected leading them to host a wide range of viruses (Xie et al. 2018).', 'cite_spans': [{'start': 696, 'end': 700, 'mention': '2018', 'ref_id': 'BIBREF27'}], 'section': 'Conclusion', 'ref_spans': []}, {'text': 'But bats are not solely to be blamed for spread of viral infections to humans. Rather breach of animal–human boundaries due to unchecked urbanization and expansion of agriculture are responsible for zoonosis of viruses from bats and other animals to humans. We need to find solutions to stop this encroachment on habitats of other animals and develop a more vigilant and holistic outlook towards environment. We also need to become more conscious of the source, biology and habitat of the animals that become part of our diet.', 'cite_spans': [], 'section': 'Conclusion', 'ref_spans': []}, {'text': 'Many scientists had predicted the possibility of re-emergence of a 2002–03 SARS-CoV like coronavirus outbreak in future. We still fell short of the requisite preparedness to combat the current outbreak. We need to learn many lessons from this current novel coronavirus outbreak and other epidemics in the past from different geographical regions under the ‘One Human–Animal–Environment’ approach. If an effective treatment to eradicate novel coronavirus infections is not found quickly the pandemic may turn into a recurring endemic in certain regions which will be even harder to tackle. Biomedical engineers, scientists, chemists and doctors need to come together to find innovative solution. A lot is known about coronaviruses, but still a lot needs to be known about their evolution, transmission and functions of different regions of their genome. Research focusing on development of broad range drugs and vaccines against coronaviruses based on conserved protein structures needs attention to stay ready for similar outbreaks in future. Newer technologies like artificial intelligence for prediction of disease epidemiology and development of more sensitive and affordable biosensors must also be given attention in this context.', 'cite_spans': [], 'section': 'Conclusion', 'ref_spans': []}, {'text': 'In this time of grave crisis, all the countries need to stand in solidarity with each other. We must also value the sacrifices and unflinching dedication of all the health-workers throughout the world in combating this pandemic.', 'cite_spans': [], 'section': 'Conclusion', 'ref_spans': []}], 'paragraphs': [{'text': 'Symptoms exhibited by COVID-19 patients include pneumonia (which includes fever, dyspnoea, malaise, dry cough and respiratory distress), leucopenia and lymphopenia (a low lymphocyte count below a threshold has been suggested as a method for initial clinical diagnosis) (Hui et al. 2020; Chen et al. 2020a; Huang et al. 2020). Less common symptoms include headache, diarrhea, sputum production and hemoptysis (Zhang et al. 2020). Moreover, chest radiographs exhibit multifocal ground glass lung opacities. The incubation period of the virus is relatively long of about 5–14 days (Bouadma et al. 2020; Jiang et al. 2020b). Many patients that go asymptomatic during incubation period become major cause of the spread of this virus.', 'id': '00004'}], 'title': 'The novel coronavirus 2019-nCoV: Its evolution and transmission into humans causing global COVID-19 pandemic', 'sha': '71ee917626ee37754db85feb5358e06b757c2dfe', 'doi': '10.1007/s13762-020-02781-2', 'score': 8.99370002746582}, {'rank': 235, 'abstract': 'Our understanding of the Coronavirus disease 2019 (COVID-19) continues to evolve and there are many unknowns about its epidemiology. This study aims to synthesise case fatality rate (CFR) among confirmed COVID-19 patients, incubation period and time from onset of COVID-19 symptoms to first medical visit, intensive care unit (ICU) admission, recovery, and death. We searched MEDLINE, Embase, Google Scholar, and bibliographies of relevant articles from 01 December 2019 to 11 March 2020 without any language restrictions. Quantitative studies that recruited people with confirmed COVID-19 diagnosis were included. Two independent reviewers extracted the data. Out of 1675 non-duplicate studies, 43 were included in the meta-analysis. The pooled mean incubation period was 5.68 (99% confidence interval [CI]: 4.78, 6.59) days. The pooled mean number of days from the onset of COVID-19 symptoms to first clinical visit was 4.92 (95% CI: 3.95, 5.90), ICU admission was 9.84 (95% CI: 8.78, 10.90), recovery was 18.55 (95% CI: 13.69, 23.41), and death was 15.93 (95% CI: 13.07, 18.79). Pooled CFR among confirmed COVID-19 patients was 0.02 (95% CI: 0.02, 0.03). We found that the incubation period and lag between the onset of symptoms and first clinical visit for COVID-19 are longer than other respiratory viral infections including Middle East respiratory syndrome and severe acute respiratory syndrome; however, the current policy of 14 days of mandatory quarantine for everyone potentially exposed to severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) might be too conservative. Longer quarantine periods might be more justified for extreme cases.', 'body_text': [{'text': 'The severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) was first identified in a few unusual pneumonia patients linked to the Wuhan seafood wholesale market in China in December 2019 [1]. However, it soon grew out of China and the Coronavirus disease 2019 (COVID-19) was declared a pandemic on 11 March 2020, and has been reported in 216 countries, areas, or territories [2]. While the epidemic has slowed down in China due to the strict quarantine and preventive regulations, the numbers of COVID-19 patients (i.e. 10021401 as of 28 June, 2020) and confirmed deaths (i.e. 499913 as of 28 June, 2020) are rapidly increasing [2] and have surpassed that of other viruses in the coronavirus family with similar genomes to SARS-CoV-2. For example, SARS which emerged in 2003, infected 8098 patients and caused 774 deaths across 29 countries. The Middle East respiratory syndrome (MERS) which appeared in 2012, led to 2494 patients and 858 deaths across 27 countries [3–6]. The healthcare systems in many countries such as the USA, Spain, Italy, France, UK, Turkey and Iran have been overwhelmed and struggling with the soaring number of patients [7].', 'cite_spans': [{'start': 194, 'end': 195, 'mention': '1', 'ref_id': 'BIBREF0'}, {'start': 382, 'end': 383, 'mention': '2', 'ref_id': 'BIBREF11'}, {'start': 635, 'end': 636, 'mention': '2', 'ref_id': 'BIBREF11'}, {'start': 973, 'end': 974, 'mention': '3', 'ref_id': 'BIBREF22'}, {'start': 975, 'end': 976, 'mention': '6', 'ref_id': 'BIBREF55'}, {'start': 1153, 'end': 1154, 'mention': '7', 'ref_id': 'BIBREF66'}], 'section': 'Introduction', 'ref_spans': []}, {'text': \"Although our understanding of COVID-19's epidemiology is evolving, it is assumed that SARS-CoV-2 is mainly transmitted via droplets and close contacts with people carrying the virus [2]. However, recent reports have also proposed the possibility of the virus being contracted via various surfaces, gastrointestinal transmission [8] and potentially airborne exposures [2, 9]. Based on the existing evidence, elderly population, those with suppressed immune systems and underlying metabolic, cardiovascular or respiratory diseases are at an increased risk for adverse outcomes; however, recent reports from outside China, point to a considerable risk of severe outcomes among the general adult population (i.e. <65 years old) [10, 11].\", 'cite_spans': [{'start': 183, 'end': 184, 'mention': '2', 'ref_id': 'BIBREF11'}, {'start': 329, 'end': 330, 'mention': '8', 'ref_id': 'BIBREF77'}, {'start': 368, 'end': 369, 'mention': '2', 'ref_id': 'BIBREF11'}, {'start': 371, 'end': 372, 'mention': '9', 'ref_id': 'BIBREF88'}, {'start': 725, 'end': 727, 'mention': '10', 'ref_id': 'BIBREF1'}, {'start': 729, 'end': 731, 'mention': '11', 'ref_id': 'BIBREF2'}], 'section': 'Introduction', 'ref_spans': []}, {'text': 'As we continue to learn more about COVID-19 and its characteristics, there are many unknowns about its epidemiology such as hospitalisation and recovery-related outcomes that are critical for healthcare system preparedness [12, 13]. For example, the mean number of incubation days for COVID-19 varies greatly across the existing literature ranging from 2.5 [14] to >20 days [15, 16]. Our understanding of time from contracting the disease to recovery or death is even more limited. In this systematic review and meta-analysis, we tried to identify the studies that recruited patients diagnosed with COVID-19 and calculate pooled estimates for several epidemiological and clinical outcomes to help provide an overall picture of the epidemiological characteristics of COVID-19. Findings of this study could help inform the ongoing public health and public policy practices across the world.', 'cite_spans': [{'start': 224, 'end': 226, 'mention': '12', 'ref_id': 'BIBREF3'}, {'start': 228, 'end': 230, 'mention': '13', 'ref_id': 'BIBREF4'}, {'start': 358, 'end': 360, 'mention': '14', 'ref_id': 'BIBREF5'}, {'start': 375, 'end': 377, 'mention': '15', 'ref_id': 'BIBREF6'}, {'start': 379, 'end': 381, 'mention': '16', 'ref_id': 'BIBREF7'}], 'section': 'Introduction', 'ref_spans': []}, {'text': 'Following the Systematic Reviews and Meta-Analyses (PRISMA) checklist (see Supplementary Table S1) and the Peer Review of Electronic Search Strategies (PRESS) guideline [17, 18], we searched PubMed, Embase and Google Scholar from 1 December 2019 to 11 March 2020 for studies that measured and reported several characteristics of COVID-19 (e.g. incubation period, hospitalisation, death). Search terms were combined using appropriate Boolean operators and included subject heading terms/keywords relevant to COVID-19 (e.g. novel coronavirus, sars-cov2, coronavirus disease). Please see Supplementary Table S2 for our sample search strategy.', 'cite_spans': [{'start': 170, 'end': 172, 'mention': '17', 'ref_id': 'BIBREF8'}, {'start': 174, 'end': 176, 'mention': '18', 'ref_id': 'BIBREF9'}], 'section': 'Literature search ::: Methods', 'ref_spans': []}, {'text': \"Quantitative studies were included in the review if they reported incubation period of of SARS-CoV-2 as well as time from onset of the symptoms to first medical visit, intensive care unit (ICU) admission, recovery (as defined by studies' authors) or death. Studies were also included if they reported the number of deaths among patients with a confirmed COVID-19 diagnosis. Studies were included in the meta-analysis if they provided data on the above-mentioned outcomes along with their standard error and sample size. Case-reports with a sample size of one were removed from the meta-analysis as they did not provide any dispersion estimate. Studies were not excluded based on language, location or measurement method. Given that this study used secondary data and involved no interaction with humans, no ethics approval was required.\", 'cite_spans': [], 'section': 'Inclusion criteria ::: Methods', 'ref_spans': []}, {'text': 'Two authors (SJ and NN) completed the abstract and full-text screening, independently. The full-texts of citations that met our inclusion criteria or were unclear were screened by two independent reviewers (SJ and NN). Disagreements over the inclusion of studies were resolved through discussion or by arbitration with the senior author (HS). Duplicate records were excluded.', 'cite_spans': [], 'section': 'Study selection ::: Methods', 'ref_spans': []}, {'text': \"Data were extracted independently by the two authors (SJ and NN) and discrepancies were resolved through discussion or by arbitration with the senior author (HS). Data were extracted on publication date, study type (e.g. cross-sectional, case-series, cohort), location, sample size, as well as patients' age and sex. We also extracted data on exposure history, X-ray and computed tomography scan findings, symptoms and underlying conditions in addition to the main outcomes of interests including the number of deaths among confirmed COVID-19 patients (i.e. case fatality rate [CFR]), incubation period and time from onset of COVID-19 symptoms to first medical visit, ICU admission, recovery, and death.\", 'cite_spans': [], 'section': 'Data extraction ::: Methods', 'ref_spans': []}, {'text': \"Meta-analysis was performed using STATA's (V.15.1) metan (for numerical variables) and metaprop (for binary variables) commands. The 95% confidence intervals (CI) for binary variables were computed using the exact binomial method. Heterogeneity between the studies was assessed using both the I2 statistic with a cut-off of 50% and the χ2 test with P-value <0.10 [19]. As all results turned out to be significantly heterogeneous, we used random-effects models to calculate the pooled point estimate and 95% CI for the CFR, mean time from onset of COVID-19 symptoms to first medical visit, ICU admission, recovery, and death. For the mean incubation period, we estimated 99% CI. We also conducted a random-effects meta-regression using STATA's metareg command to identify the sources of heterogeneity and explored the effect of study-level covariates where data were available (Supplementary Table S3). Meta-regression was considered when there were at least 10 studies included in the meta-analysis [20]. A two-sided P-value <0.05 was considered as statistically significant.\", 'cite_spans': [{'start': 364, 'end': 366, 'mention': '19', 'ref_id': 'BIBREF10'}, {'start': 1000, 'end': 1002, 'mention': '20', 'ref_id': 'BIBREF12'}], 'section': 'Statistical analysis ::: Methods', 'ref_spans': []}, {'text': 'We found a total of 1675 non-duplicate studies, 57 of which were included in the qualitative synthesis and 43 were included in the meta-analysis (Fig. 1). A description of the main characteristics of the included studies is provided in Table 1. The 57 studies included 27 cross-sectional, one case-control, one retrospective cohort and 28 case series/case report studies with sample sizes ranging greatly from one in case-reports to 58 182 for a study in the Hubei province [21]. Inclusion criteria varied greatly across the studies but most study participants were hospitalised patients living or travelling from various provinces in China. Median (range) age of the participants was 46.2 (range: 17 days to 78.5 years) and about 60% were male. Most studies were conducted between January and February 2020. Clinical and epidemiological characteristics of the patients included in the study are presented in Table 2. Among studies that reported exposure history among their participants, most patients were directly or indirectly traced back to the city Wuhan (e.g. lived in Wuhan or had recently travelled to Wuhan) and the Huanan seafood market in Hubei province, China. Several cases of contracting SARS-CoV-2 through close contacts with family members were also reported across the studies. Frequent CT or X-ray findings included thickened texture of the lungs, bilateral focal consolidation, lobar consolidation, ground-glass opacity, patchy consolidation, and unilateral/bilateral pneumonia. Common symptoms reported across the studies included fever, cough, shortness of breath, and fatigue/weakness. Only 15 studies reported some information about the pre-existing conditions of the patients; most of whom had metabolic and cardiovascular underlying conditions.\\n\\n\\n', 'cite_spans': [{'start': 475, 'end': 477, 'mention': '21', 'ref_id': 'BIBREF13'}], 'section': 'Participants and study characteristics ::: Results', 'ref_spans': [{'start': 146, 'end': 152, 'mention': 'Fig. 1', 'ref_id': 'FIGREF0'}, {'start': 236, 'end': 243, 'mention': 'Table 1', 'ref_id': 'TABREF0'}, {'start': 909, 'end': 916, 'mention': 'Table 2', 'ref_id': 'TABREF1'}]}, {'text': 'The estimated mean incubation period obtained from the included studies and the pooled mean are presented in Figure 2. Out of the 18 studies included in the meta-analysis, 15 were conducted in China. The pooled mean incubation period was 5.68 (99% CI: 4.78, 6.59) days. Heterogeneity testing (I2 = 98.4%) revealed notable differences among the included studies in the meta-analysis. Multivariate meta-regression results showed no significant differences in incubation period time by country (China vs. others, Adjusted β = 1.76; P-value = 0.375), age (Adjusted β = −1.16; P-value = 0.151) or male percentage of the participants (Adjusted β = −12.35; P-value = 0.058).\\n', 'cite_spans': [], 'section': 'Mean incubation period ::: Results', 'ref_spans': [{'start': 109, 'end': 117, 'mention': 'Figure 2', 'ref_id': 'FIGREF1'}]}, {'text': 'The estimated mean number of days from the onset of COVID-19 symptoms to first clinical visit was 4.92 (95% CI: 3.95, 5.90). As shown in Figure 3, out of the 24 studies included in the meta-analysis, only six were conducted outside China. Heterogeneity testing (I2 = 98.3%) revealed notable differences among the included studies in the meta-analysis. Multivariate meta-regression results showed no significant differences in time from onset of symptoms to first clinical visit by country (China vs. others, Adjusted β = 1.51; P-value = 0.411), age (Adjusted β = 0.92; P-value = 0.153) or male percentage of the participants (Adjusted β = −2.60; P-value = 0.626).\\n', 'cite_spans': [], 'section': 'Mean time from onset of symptoms to first clinical visit ::: Results', 'ref_spans': [{'start': 137, 'end': 145, 'mention': 'Figure 3', 'ref_id': 'FIGREF2'}]}, {'text': 'The estimated mean number of days from the onset of COVID-19 symptoms to ICU admission was 9.84 (95% CI: 8.78, 10.90), an estimate that was derived from one study in Singapore and two studies in Wuhan, China (Fig. 4).\\n', 'cite_spans': [], 'section': 'Mean time from onset of symptoms to ICU admission ::: Results', 'ref_spans': [{'start': 209, 'end': 215, 'mention': 'Fig. 4', 'ref_id': 'FIGREF3'}]}, {'text': 'The estimated mean number of days from the onset of symptoms to recovery was reported in seven studies and the resulting pooled mean was 18.55 (95% CI: 13.69, 23.41). Only two studies were conducted in China and the rest were completed in France, South Korea, the UK, Singapore and Japan (Fig. 5).\\n', 'cite_spans': [], 'section': 'Mean time from onset of symptoms to recovery ::: Results', 'ref_spans': [{'start': 289, 'end': 295, 'mention': 'Fig. 5', 'ref_id': 'FIGREF4'}]}, {'text': 'The estimated mean number of days from the onset of symptoms to death was reported in three studies with a pooled mean of 15.93 (95% CI: 13.07, 18.79). All of the studies were conducted in China (Fig. 6).\\n', 'cite_spans': [], 'section': 'Mean time from onset of symptoms to death ::: Results', 'ref_spans': [{'start': 196, 'end': 202, 'mention': 'Fig. 6', 'ref_id': 'FIGREF5'}]}, {'text': 'The estimated CFR among COVID-19 patients was reported in 23 studies; most of which included hospitalised patients, three included ICU patients [37, 45, 60], and none included outpatients. The pooled CFR was estimated as 0.02 (95% CI: 0.02, 0.03) (Fig. 7). Heterogeneity testing (I2 = 97.6%) revealed notable differences among the included studies in the meta-analysis. Multivariate meta-regression results showed a significant difference in CFR by age (Adjusted β = 0.056; P-value = 0.003).\\n', 'cite_spans': [{'start': 145, 'end': 147, 'mention': '37', 'ref_id': 'BIBREF30'}, {'start': 149, 'end': 151, 'mention': '45', 'ref_id': 'BIBREF39'}, {'start': 153, 'end': 155, 'mention': '60', 'ref_id': 'BIBREF56'}], 'section': 'Case fatality rate ::: Results', 'ref_spans': [{'start': 248, 'end': 254, 'mention': 'Fig. 7', 'ref_id': 'FIGREF6'}]}, {'text': 'We conducted a systematic review and meta-analysis to provide an overview of the epidemiological characteristics of COVID-19 based on the existing evidence as of 11 March 2020. Our findings suggest that COVID-19 has an average incubation period of 5.68 days and there is a lag of 4.92 days from onset of symptoms to the first clinical visit. On average, the symptoms of the patients lasted less than 20 days (18.55 days) before recovery was achieved and the CFR among confirmed COVID-19 patients was 2%, which significantly increased by age. Similar to previous studies [77], fever, dry cough, shortness of breath and fatigue were common symptoms among the patients in the included studies. As expected, history of direct or indirect exposure to Wuhan was frequently reported. The most common radiologic findings were bilateral consolidation and pneumonia [78, 79].', 'cite_spans': [{'start': 571, 'end': 573, 'mention': '77', 'ref_id': 'BIBREF74'}, {'start': 857, 'end': 859, 'mention': '78', 'ref_id': 'BIBREF75'}, {'start': 861, 'end': 863, 'mention': '79', 'ref_id': 'BIBREF76'}], 'section': 'Discussion', 'ref_spans': []}, {'text': 'We found the average incubation period of COVID-19 infection to be less than 6 days which is broadly consistent with previously reported estimates [23, 29, 80, 81]. The right tail of the 99% CI of the incubation period for COVID-19 was less than 7 days (6.59). This finding is of particular interest as there are many uncertainties about the incubation period of COVID-19. For example, both the World Health Organization and Centers for Disease Control and Prevention in the USA suggest an incubation period of 2–14 days. However, single outlier cases as long as 19 [47], 24 [15] or 27 days [16] have been reported; estimates that are possibly reflecting a double exposure. Our findings are of particular importance for quarantine-related policies and planning and suggest that the current 14-day quarantine period might be rather conservative. Indeed, we found that except for one small study from China in Anyang city on a cluster of six patients [47], all other studies reported incubation periods less than 10.6 days; therefore, a shorter period of 14 days would suffice and almost all people exposed to SARS-CoV-2 would show symptoms within 11 days of their initial exposure. All in all, decisions to modify or keep the existing policies need to weigh the costs of extending active quarantine against the potential or costs of missing a few patients with delayed-onset symptoms.', 'cite_spans': [{'start': 148, 'end': 150, 'mention': '23', 'ref_id': 'BIBREF15'}, {'start': 152, 'end': 154, 'mention': '29', 'ref_id': 'BIBREF21'}, {'start': 156, 'end': 158, 'mention': '80', 'ref_id': 'BIBREF78'}, {'start': 160, 'end': 162, 'mention': '81', 'ref_id': 'BIBREF79'}, {'start': 567, 'end': 569, 'mention': '47', 'ref_id': 'BIBREF41'}, {'start': 576, 'end': 578, 'mention': '15', 'ref_id': 'BIBREF6'}, {'start': 592, 'end': 594, 'mention': '16', 'ref_id': 'BIBREF7'}, {'start': 950, 'end': 952, 'mention': '47', 'ref_id': 'BIBREF41'}], 'section': 'Discussion', 'ref_spans': []}, {'text': 'COVID-19 seems to have a longer incubation period than that of other acute respiratory viral infections such as human coronavirus (3.2 days), influenza A (1.43–1.64 days), parainfluenza (2.6 days), respiratory syncytial virus (4.4 days) and rhinovirus (1.4 days) [82, 83]. Furthermore, the median incubation period for SARS has been estimated as 4.0 days in 2009 [82], which is considerably lower than what we observed for COVID-19. The longer incubation period of the COVID-19 may be one of the major factors that helps explain its rapid spread in comparison with previous respiratory infection viruses. Other factors contributing to the spread of COVID-19 are the lag between the onset of symptoms and first clinical visit (i.e. 4.92 days) and the high number of asymptomatic COVID-19 patients. These findings suggest that MERS and SARS patients may progress to severe symptoms and respiratory failures [84] much faster than most COVID-19 patients [85].', 'cite_spans': [{'start': 264, 'end': 266, 'mention': '82', 'ref_id': 'BIBREF80'}, {'start': 268, 'end': 270, 'mention': '83', 'ref_id': 'BIBREF81'}, {'start': 364, 'end': 366, 'mention': '82', 'ref_id': 'BIBREF80'}, {'start': 906, 'end': 908, 'mention': '84', 'ref_id': 'BIBREF82'}, {'start': 951, 'end': 953, 'mention': '85', 'ref_id': 'BIBREF83'}], 'section': 'Discussion', 'ref_spans': []}, {'text': 'In comparison to MERS with a fatality rate of 35.67% [86] and SARS with a fatality rate of 11% [87], we found COVID-19 to have a much lower CFR (2%) that significantly increased by age (5.6% increase for every 10-year increase). Although this estimate is comparable with previous studies [40, 88], it is important to recognise the limitations of calculating fatality rates of COVID-19 while the epidemic is still growing. As most COVID-19 patients remain asymptomatic and may recover without seeking medical care, it is likely that the true CFR among people infected with SARS-CoV-2 could be even lower. On the other hand, the estimated fatality rates reported in most studies need to be interpreted with caution as they are often based on the cumulative number of deaths relative to the number of confirmed cases, while patients who die on a given day have been infected at a much earlier date and this would bias the denominator used to calculate the fatality rate [89].', 'cite_spans': [{'start': 54, 'end': 56, 'mention': '86', 'ref_id': 'BIBREF84'}, {'start': 96, 'end': 98, 'mention': '87', 'ref_id': 'BIBREF85'}, {'start': 289, 'end': 291, 'mention': '40', 'ref_id': 'BIBREF34'}, {'start': 293, 'end': 295, 'mention': '88', 'ref_id': 'BIBREF86'}, {'start': 968, 'end': 970, 'mention': '89', 'ref_id': 'BIBREF87'}], 'section': 'Discussion', 'ref_spans': []}, {'text': 'We acknowledge four main limitations of our systematic review. First, our findings are mainly based on studies that recruited patient from clinics and hospitals and therefore, may be biased towards more severe cases. Moreover, our data might be skewed towards early reporting from provinces in China and outcomes might be different in other countries in the Western context. We are, therefore, aiming to update the review as more data become available in the next 12 months to provide more accurate estimates. Second, many studies did not report the study outcomes by subgroups such as age or sex and we could not report group-specific outcomes. Third, we used the mean and the standard error of the incubation period assuming a normal distribution which may have led to the underestimation of the right tail of the distribution. Lastly, given the urgency of the topic and the heterogeneity of the studies included in the review, we did not conduct the risk of bias or quality assessment of the studies. Given the emerging nature of COVID-19 and the observational study design of most of the available evidence, most studies in the review are at a high risk of bias and the quality of existing evidence is relatively low. Nonetheless, our systematic review of literature provides an insightful picture of the epidemiological characteristics of COVID-19 which could inform ongoing public health and public policy decision makings.', 'cite_spans': [], 'section': 'Discussion', 'ref_spans': []}, {'text': \"Our findings of the epidemiological characteristics of COVID-19 provide important insight into healthcare systems' prevention and planning efforts. The incubation period (i.e. <11 days in most studies) and the lag between the onset of symptoms and diagnosis (i.e. ~5 days) are longer for COVID-19 compared to other respiratory viral infections including MERS and SARS. Current policies of 14 days of mandatory quarantine for everyone potentially exposed to SARS-CoV-2 might be too conservative and longer quarantine periods might be more justified for extreme cases. As effective vaccination or treatment for COVID-19 are yet to be developed, practising the fundamentals of public health and prevention science such as physical distancing and personal hygiene are critical and need to be emphasised and enforced further to reduce the risk of SARS-CoV-2 transmission.\", 'cite_spans': [], 'section': 'Conclusions', 'ref_spans': []}], 'paragraphs': [{'text': 'We found the average incubation period of COVID-19 infection to be less than 6 days which is broadly consistent with previously reported estimates [23, 29, 80, 81]. The right tail of the 99% CI of the incubation period for COVID-19 was less than 7 days (6.59). This finding is of particular interest as there are many uncertainties about the incubation period of COVID-19. For example, both the World Health Organization and Centers for Disease Control and Prevention in the USA suggest an incubation period of 2–14 days. However, single outlier cases as long as 19 [47], 24 [15] or 27 days [16] have been reported; estimates that are possibly reflecting a double exposure. Our findings are of particular importance for quarantine-related policies and planning and suggest that the current 14-day quarantine period might be rather conservative. Indeed, we found that except for one small study from China in Anyang city on a cluster of six patients [47], all other studies reported incubation periods less than 10.6 days; therefore, a shorter period of 14 days would suffice and almost all people exposed to SARS-CoV-2 would show symptoms within 11 days of their initial exposure. All in all, decisions to modify or keep the existing policies need to weigh the costs of extending active quarantine against the potential or costs of missing a few patients with delayed-onset symptoms.', 'id': '00017'}], 'title': 'Epidemiological characteristics of COVID-19: a systematic review and meta-analysis', 'sha': 'cd00ea659ea6a6574181a03ce63a88e07a8ad94c', 'doi': '10.1017/s0950268820001430', 'score': 8.987799644470215}, {'rank': 236, 'paragraphs': [{'text': 'Since the first case of the coronavirus disease 2019 (COVID-19) was found in Wuhan, P. R. China in December 2019, the disease has rapidly spread in the city of Wuhan, then to Hubei Province, China, and subsequently, across the world [1]. On March 11, 2020 , the World Health Organization (WHO) declared COVID-19 to be a pandemic. The swift spread of the virus is largely attributed to its stealth transmissions for which infected patients may be asymptomatic or exhibit only flu-like symptoms in the early stage. Undetected transmissions present a remarkable challenge for the containment of the virus and pose an appalling threat to the public health. To understand the drastically negative impacts of COVID-19 on the public health, it is urgent to investigate key features pertinent to the disease: How severe is the transmission? How long is the incubation time of the disease on average? How many infections are asymptomatic? And ultimately, what is the case fatality rate?', 'id': '00001'}], 'title': 'Estimation of the basic reproduction number, average incubation time, asymptomatic infection rate, and case fatality rate forCOVID-19: Meta-analysis and sensitivity analysis', 'sha': '4bc132d5407496a57dabc2d8db6e46536f0a57f3', 'doi': '10.1101/2020.04.28.20083758', 'score': 8.987199783325195}, {'rank': 237, 'paragraphs': [{'text': '(4)L(α,λ)=∏i=1Iαλexp{−(λvi)α}Γ(1/α),α>0,λ>0.', 'id': '00016'}], 'title': 'Estimation of incubation period distribution of COVID-19 using disease onset forward time: a novel cross-sectional and forward follow-up study', 'sha': 'c61be9fc4562d250e2dec241ef75bd22d61f61d1; 6d3b3f4ab80a61c45f82c61c6c756cfc6ddf4bf2', 'doi': '10.1101/2020.03.06.20032417', 'score': 8.980199813842773}, {'rank': 238, 'paragraphs': [{'text': 'The funder of the study had no role in study design, data collection, modelling analysis, results interpretation, or writing of this article. The corresponding authors had full access to all the data in the study and had final responsibility for the decision to submit for publication.', 'id': '00020'}], 'title': 'Estimation of incubation period distribution of COVID-19 using disease onset forward time: a novel cross-sectional and forward follow-up study', 'sha': 'c61be9fc4562d250e2dec241ef75bd22d61f61d1; 6d3b3f4ab80a61c45f82c61c6c756cfc6ddf4bf2', 'doi': '10.1101/2020.03.06.20032417', 'score': 8.980198860168457}, {'rank': 239, 'abstract': 'Background: Our understanding of the corona virus disease 2019 (COVID-19) continues to evolve. However, there are many unknowns about its epidemiology. Purpose: To synthesize the number of deaths from confirmed COVID-19 cases, incubation period, as well as time from onset of COVID-19 symptoms to first medical visit, ICU admission, recovery and death of COVID-19. Data Sources: MEDLINE, Embase, and Google Scholar from December 01, 2019 through to March 11, 2020 without language restrictions as well as bibliographies of relevant articles. Study Selection: Quantitative studies that recruited people living with or died due to COVID-19. Data Extraction: Two independent reviewers extracted the data. Conflicts were resolved through discussion with a senior author. Data Synthesis: Out of 1675 non-duplicate studies identified, 57 were included. Pooled mean incubation period was 5.84 (99% CI: 4.83, 6.85) days. Pooled mean number of days from the onset of COVID-19 symptoms to first clinical visit was 4.82 (95% CI: 3.48, 6.15), ICU admission was 10.48 (95% CI: 9.80, 11.16), recovery was 17.76 (95% CI: 12.64, 22.87), and until death was 15.93 (95% CI: 13.07, 18.79). Pooled probability of COVID-19-related death was 0.02 (95% CI: 0.02, 0.03). Limitations: Studies are observational and findings are mainly based on studies that recruited patient from clinics and hospitals and so may be biased toward more severe cases. Conclusion: We found that the incubation period and lag between the onset of symptoms and diagnosis of COVID-19 is longer than other respiratory viral infections including MERS and SARS; however, the current policy of 14 days of mandatory quarantine for everyone might be too conservative. Longer quarantine periods might be more justified for extreme cases.', 'body_text': [{'text': 'related outcomes which are indeed of utmost importance when it comes to health system 84 preparedness (12, 13) . For example, mean number of days of incubation for COVID-19 varies 85 greatly across the existing literature ranging from 2.5 (14) to 12.1 (15) days. Our knowledge 86 of time from contracting the disease to recovery or death are even more limited. In this 87 systematic review, we tried to identify the studies that recruited patients diagnosed with 88 COVID-19 and calculate polled estimates for several epidemiological and clinical outcomes to 89 help provide an overall picture of the characteristics of COVID-19. Findings of this study could 90 help inform the ongoing public health and public policy practices across the world. Data were extracted independently by the two authors (SJ and NN), and discrepancies were 130 resolved through discussion or by arbitration with a senior co-author (AM). Data were 131 extracted on study type (e.g., descriptive, case-series, mathematical modeling), publication 132', 'cite_spans': [{'start': 102, 'end': 106, 'text': '(12,', 'ref_id': None}, {'start': 107, 'end': 110, 'text': '13)', 'ref_id': None}], 'ref_spans': [], 'section': ''}, {'text': \"year, location, sample size, patients' age, gender, exposure history, X-ray and computed 133 tomography (18) scan findings, symptoms, and underlying conditions if reported. We also 134 extracted data on the main outcomes of interests including the number of deaths from 135 confirmed COVID-19 cases, incubation period, as well as time from onset of COVID-19 136 symptoms to first medical visit, to ICU admission, to recovery and to death. 137 138\", 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': \"For studies that did not provided enough data to be included in the meta-analysis, we 140 performed a qualitative data synthesis. Case-reports with a sample size of one were also 141 removed from the meta-analysis as they did not provide any dispersion estimate. Meta-142 analysis was performed using STATA's (V.15.1) metan (for numerical variables) and metaprop 143 (for binary variables) commands. The 95% confidence intervals (CI) for binary variables were 144 computed using the exact binomial method. Heterogeneity between studies was assessed 145 using both the I 2 statistic with a cutoff of ≥ 50% and the Chi-square test with P-value <0.10 to ICU admission, recovery and death. For mean incubation period, we estimated 99% CI. We 150 also conducted a random-effects meta-regression using STATA's metareg command to 151 identify the sources of heterogeneity and explore the effect of study-level covariates where 152 data were available. A two-sided P-value <0.05 was considered as statistically significant 153 effect. 154\", 'cite_spans': [], 'ref_spans': [], 'section': 'Statistical analysis 139'}, {'text': 'Participants and study characteristics 157', 'cite_spans': [], 'ref_spans': [], 'section': 'Results 156'}, {'text': 'We found a total of 1675 non-duplicate studies, 57 of which were included in our qualitative 158 synthesis and 43 were considered for meta-analysis ( Figure 1) . A description of the main 159 characteristics of the included studies is provided in Table 1 . The 57 studies included 27 cross-160 sectional, one case-control, one retrospective cohort, and 28 case series/case report studies 161 with sample sizes of observational studies ranging greatly from one to 58182 for a study in 162 the Hubei province (20) . Inclusion criteria varied greatly across the studies but most 163 participants were hospitalized patients living or traveling from various provinces in China. 164', 'cite_spans': [{'start': 507, 'end': 511, 'text': '(20)', 'ref_id': None}], 'ref_spans': [{'start': 150, 'end': 159, 'text': 'Figure 1)', 'ref_id': None}, {'start': 247, 'end': 254, 'text': 'Table 1', 'ref_id': 'TABREF6'}], 'section': 'Results 156'}, {'text': 'Median (range) age of the participants was 46.2 (17 days to 78.5 years) and about 60% were 165 men. Most studies were conducted between January and February 2020. Clinical and 166 epidemiological characteristics of the patients included in the study are presented in Table 2 . 167', 'cite_spans': [], 'ref_spans': [{'start': 267, 'end': 274, 'text': 'Table 2', 'ref_id': 'TABREF7'}], 'section': 'Results 156'}, {'text': 'Among studies that reported exposure history among their participants, most patients were 168 directly or indirectly traced back to the city Wuhan (e.g., lived in Wuhan or had recently 169 travelled to Wuhan) and the Huanan seafood market in Hubei province, China. Several cases 170 of contracting COVID-19 through close contacts with family members were also reported unilateral/bilateral pneumonia. Common symptoms reported across the studies include 174 fever, cough, shortness of breath, and fatigue/weakness. Only 15 studies reported some 175 information about the pre-existing conditions of the patients; most of whom had metabolic 176 and cardiovascular underlying conditions. 177', 'cite_spans': [], 'ref_spans': [], 'section': 'Results 156'}, {'text': '178', 'cite_spans': [], 'ref_spans': [], 'section': 'Results 156'}, {'text': 'The estimated mean incubation period obtained from the included studies as well as resulting 180 pooled mean are presented in Fig. 2 . Out of the 14 studies included in the meta-analysis, 12 181', 'cite_spans': [], 'ref_spans': [{'start': 126, 'end': 132, 'text': 'Fig. 2', 'ref_id': None}], 'section': 'Mean incubation period 179'}, {'text': 'were conducted in China. The findings of meta-analysis showed that the mean incubation 182 period was 5.84 (95% CI: 4.83, 6.85) days. Heterogeneity testing (I 2 =94.7%) revealed notable 183 differences among the included studies in the meta-analysis. Multivariate meta-regression 184 results ( Table 2) showed no significant differences in incubation period time by country (China 185 vs. others, Adjusted Beta = 1.70; P-value = 0.484), age (Adjusted Beta = -1.16; P-value = 0.239) 186 or percent of male (Adjusted Beta = -13.07; P-value = 0.09) participants. 187 188', 'cite_spans': [], 'ref_spans': [{'start': 294, 'end': 302, 'text': 'Table 2)', 'ref_id': 'TABREF7'}], 'section': 'Mean incubation period 179'}, {'text': 'The estimated mean number of days from the onset of COVID-19 symptoms to first clinical 190 visit was 4.82 (95% CI: 3.48, 6.15). As shown in Fig. 3 . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [{'start': 141, 'end': 147, 'text': 'Fig. 3', 'ref_id': None}], 'section': 'Mean time from onset of symptoms to first clinical visit 189'}, {'text': '(which was not peer-reviewed) The copyright holder for this preprint . https: //doi.org/10.1101 //doi.org/10. /2020 Mean time from onset of symptoms to ICU admission 199 The estimated mean number of days from the onset of COVID-19 symptoms to ICU admission 200 was 10.48 (95% CI: 9.80, 11.16), an estimate that was derived from one study in Singapore 201 and one in Wuhan, China (Fig. 4) . The estimated mean number of days from the onset of symptoms to death was reported in 215 three studies with a pooled mean of 15.93 (95% CI: 13.07, 18.79). All of the studies were 216 conducted in China (Fig. 6) . 217 notable differences among the included studies in the meta-analysis. Multivariate meta-222 regression results (Supplement 2) showed a significant difference in death probability by age 223 (Adjusted Beta = 0.058; P-value = 0.016). 224 225', 'cite_spans': [{'start': 78, 'end': 95, 'text': '//doi.org/10.1101', 'ref_id': None}, {'start': 96, 'end': 115, 'text': '//doi.org/10. /2020', 'ref_id': None}, {'start': 152, 'end': 169, 'text': 'ICU admission 199', 'ref_id': None}], 'ref_spans': [{'start': 379, 'end': 387, 'text': '(Fig. 4)', 'ref_id': None}, {'start': 593, 'end': 601, 'text': '(Fig. 6)', 'ref_id': None}], 'section': 'Mean time from onset of symptoms to first clinical visit 189'}, {'text': 'We conducted a systematic review and meta-analysis to provide an overview of the 227 epidemiological characteristics of COVID-19. Our findings suggested that the COVID-19 has an 228 incubation time of more than five days (5.84 days) and a lag of less than five days (4.82 days) 229 from onset of symptoms to first clinical visit. On average, the symptoms lasted more than 230 twenty days (17.76 days) before recovery was achieved and the mortality risk among 231 confirmed cases was 2%, which significantly increased by age. Similar to previous studies (21), 232 fever, dry cough, shortness of breath, and fatigue were common symptoms among the 233 patients in the included studies. As expected, history of direct or indirect exposure to Wuhan 234 was frequent and the most common radiologic findings were bilateral consolidation and 235 pneumonia (22, 23). 236', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion 226'}, {'text': \"We found that the average incubation period of COVID-19 infection to be less than 6 days 237 which is broadly consistent with previously reported estimates (24-27). The right tail of the 238 incubation period for COVID-19 even for 99% CI was less than seven days (6.85). This finding 239 is of particular interest as uncertainties continue to exist about the incubation period of 240 COVID-19. For example, while both WHO and United States' CDC suggest an incubation period 241 of 2-14 days, single outlier cases as long as 19 (15), 24 (28) or 27 days (29) have been reported; 242 estimates that are most likely reflecting a double exposure. Our meta-analytic findings are of 243 particular importance for quarantine-related policies and planning and suggest that the 244 current 14-day quarantine period might be rather conservative. Indeed, we found that except 245 . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\", 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion 226'}, {'text': 'The copyright holder for this preprint . https://doi.org/10. 1101 /2020 for one small study from China in Anyang city on a cluster of six patients (Bai 2020 (15)), all 246 other studies reported incubation periods less than 11 days; therefore, a shorter period of 14 247 days would most likely suffice and it is expected that almost all infected people will show 248 symptoms within 11 days of initial exposure. Nonetheless, decisions to modify or keep the 249 existing policies need to weigh the costs of extending active quarantine against the potential 250 or costs of missing a symptomatic case. 251 COVID-19 seems to have a longer incubation period than that of other acute respiratory viral 252 infections (30) such as human coronavirus (3.2 days), influenza A (1.43-1.64 days (31)), 253 parainfluenza (2.6 days), respiratory syncytial virus (4.4 days), and rhinovirus (1.4 days). 254', 'cite_spans': [{'start': 61, 'end': 65, 'text': '1101', 'ref_id': None}, {'start': 66, 'end': 71, 'text': '/2020', 'ref_id': None}], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'Furthermore, the median incubation period for SARS has been reported estimated as 4.0 days 255 in 2009 (30), which is considerably lower than what we observed for COVID-19. The longer 256 incubation period of the COVID-19 may be one of the major factors that helps explain its rapid 257 spread in comparison previous respiratory infection viruses. Another factor that contributes 258 to spread of COVID-19 is the lag between the onset of symptoms and first clinical visit and the 259 high number of asymptomatic cases of COVID-19. Our results indicated on average it is less 260 than five days (4.82 days), and particularly increases by age (1.94 days increase per each 10 261 years increase in age). This finding suggests that MERS and SARS progress rapidly to sever 262 symptoms and respiratory failures (32) than most cases infected by . 263', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'In compare to MERS-COV with a mortality rate of 35.67% (34) and SARS with a mortality rate 264 of 11% (35), we found that COVID-19 has a much lower mortality rate (2%) which significantly 265 increase by age (48% increase for every 1% increase in male participants). Although this 266 estimate is comparable with previous studies (36, 37), it is important to recognize the 267 limitations of calculating mortality rates of COVID-19 while the epidemic is still growing. As 268 most cases of CVOID-19 remain asymptomatic and may recover without seeking medical care, 269 . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'The copyright holder for this preprint . https://doi.org/10. 1101 /2020 it is likely that the true rate of death among people infected with COVID-19 could be even 270 lower. On the other hand, the estimated mortality rates reported in most studies need to be 271 interpreted with caution as they are often based on the cumulative number of deaths relative 272 to the number of confirmed cases, while patients who die on a given day have been infected 273 at a much earlier date and this would bias the denominator of the mortality rate (38). 274', 'cite_spans': [{'start': 61, 'end': 65, 'text': '1101', 'ref_id': None}, {'start': 66, 'end': 71, 'text': '/2020', 'ref_id': None}], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'We acknowledge four main limitations of our systematic review. First, our findings are mainly 275 based on studies that recruited patient from clinics and hospitals and so may be biased 276 toward more severe cases. Second, many studies did not report the study outcomes by 277 subgroups such as age or gender and so we could report group-specific outcomes. Third, we 278 used the mean and the standard error of the incubation period assuming a normal 279 distribution which may have led to underestimate the right tail of the distribution. Lastly, 280', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': \"given the urgency of topic and the heterogeneity of the studies included in the review, we did 281 not conduct risk of bias and quality assessment of the studies. Inevitably, given the novelty of for healthcare systems' prevention and planning efforts. The incubation period (i.e., <11 days 291 in most studies) and the lag between the onset of symptoms and diagnosis (i.e., <5 days) are 292 longer for COVID-19 compared to other respiratory viral infections including MERS and SARS. 293 . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\", 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'The copyright holder for this preprint . https://doi.org/10. 1101 /2020 Nonetheless, current policies of 14 days of mandatory quarantine for everyone, might be too 294 conservative and longer quarantine periods might be more justified for extreme cases. As 295 effective vaccination or treatment for COVID-19 are yet to be developed, practicing the 296 fundamentals of public health and prevention efforts such as social distancing and personal 297 hygiene are critical and need to be emphasized and enforced further to reduce the 298 transmission risk of COVID-19 to vulnerable populations. 299 300 301 302 303 304 305', 'cite_spans': [{'start': 61, 'end': 65, 'text': '1101', 'ref_id': None}, {'start': 66, 'end': 71, 'text': '/2020', 'ref_id': None}], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'The copyright holder for this preprint . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'The copyright holder for this preprint . https: //doi.org/10.1101 //doi.org/10. /2020 a Studies with a sample size less than or equal to 4 patients were labeled as case-reports (88); b A 7-year-old-boy and his parents; c Studies are in press and will be published in future issues of the respective journal ', 'cite_spans': [{'start': 48, 'end': 65, 'text': '//doi.org/10.1101', 'ref_id': None}, {'start': 66, 'end': 85, 'text': '//doi.org/10. /2020', 'ref_id': None}], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}], 'paragraphs': [{'text': \"We found that the average incubation period of COVID-19 infection to be less than 6 days 237 which is broadly consistent with previously reported estimates (24-27). The right tail of the 238 incubation period for COVID-19 even for 99% CI was less than seven days (6.85). This finding 239 is of particular interest as uncertainties continue to exist about the incubation period of 240 COVID-19. For example, while both WHO and United States' CDC suggest an incubation period 241 of 2-14 days, single outlier cases as long as 19 (15), 24 (28) or 27 days (29) have been reported; 242 estimates that are most likely reflecting a double exposure. Our meta-analytic findings are of 243 particular importance for quarantine-related policies and planning and suggest that the 244 current 14-day quarantine period might be rather conservative. Indeed, we found that except 245 . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\", 'id': '00014'}], 'title': 'Epidemiological Characteristics of COVID-19: A Systemic Review and Meta-Analysis', 'sha': '4a2cf4eeb236cbaa2885c9940554afc57cb4c9d5', 'doi': '10.1101/2020.04.01.20050138', 'score': 8.97920036315918}, {'rank': 240, 'paragraphs': [{'text': 'Another challenge to our strategy is the potential variability in the incubation period. This allows a fraction of people, whose asymptomatic phase is longer than average, to remain in the E state throughout their lock-down week, then continue to infect others as they resume activity. Our model accounts for such variability through g(t) and its variance σ 2 . In Fig. 2h -j we examine the impact σ on our mitigation. Quite strikingly, we find that even extreme levels of heterogeneity in incubation time, with σ as high as 5 or 10 (namely, two times the mean), still have little impact on the performance of our strategy. In fact, under all values of σ the peak infection remains at a level of less than 1%. The only exception is when t 0 = 50, a late response in which I(t) was already above 1%, and hence I Peak reached, at its maximum level 3% (Fig. 2i, red) . For the duration T we continue to observe a similar trend, where T is low under small σ, but becomes higher as σ is increased, and the I(t) curve is flattened rather than eliminated. Overall, however, we find that our strategy is, by and large, insensitive to incubation time variability.', 'id': '00021'}], 'title': 'Alternating quarantine for sustainable mitigation of COVID-19', 'sha': 'b923a8fc7d406d0f149ecd5293d1a72dec7ee526', 'doi': '', 'score': 8.974499702453613}, {'rank': 241, 'paragraphs': [{'text': 'Figure 1 shows the design of the cross-sectional and forward follow-up study. The dot on the left end of each segment is a date of infection, while the square on the right end is a date of symptoms onset. The date of departure from Wuhan cuts the line segment in between. Note that only solid lines were followed in our cohort, while dashed lines are not in the cohort because the date of departure from Wuhan is not between January 19, 2020, and January 23, 2020.', 'id': '00011'}], 'title': 'Estimation of incubation period distribution of COVID-19 using disease onset forward time: a novel cross-sectional and forward follow-up study', 'sha': 'c61be9fc4562d250e2dec241ef75bd22d61f61d1; 6d3b3f4ab80a61c45f82c61c6c756cfc6ddf4bf2', 'doi': '10.1101/2020.03.06.20032417', 'score': 8.969799995422363}, {'rank': 242, 'paragraphs': [{'text': '• What the literature reports about range of incubation periods for COVID-19 in humans? • How the range of incubation periods for COVID-19 varies across human health status? • How the range of incubation periods for COVID-19 varies across human age?', 'id': '00027'}], 'title': 'CAiRE-COVID: A Question Answering and Multi-Document Summarization System for COVID-19 Research', 'sha': '956cc65a5fa297d9e38c5d1da910fa6de02bdf0c', 'doi': '', 'score': 8.968899726867676}, {'rank': 243, 'paragraphs': [{'text': '(Mr Zhou, No.2 patients in Nanyang) in Nanyang says that his son left Wuhan and arrived at home at January 6, he and his son were all disease-free for a long period. Until February 3, he became sick and was quickly confirmed with infection of COVID-19. His son was disease-free till now. From our definition of incubation period, his incubation period will be 28 days, which is surprisingly high. We note that, our definition of estimated incubation period may be a little higher than the actually case, since we can not exclude the cases that the person has contacted some other people that carried the COVID-19. That is, we cannot know who was the intermediate spreader (unknown person B), and when she/he transmitted the virus from person A to person C.', 'id': '00015'}], 'title': 'Epidemiological characteristics of 1212 COVID-19 patients in Henan, China', 'sha': '091a8e9a61e19e88caeb039f0e3888d111b20439', 'doi': '10.1101/2020.02.21.20026112', 'score': 8.96399974822998}, {'rank': 244, 'abstract': 'Background: Long-term care facilities (LTCFs) are particularly vulnerable to nosocomial outbreaks of coronavirus disease 2019 (COVID-19), with high rates of transmission and mortality. Timely epidemiological surveillance is essential to detect and respond to outbreaks, but testing resources are highly limited in the current pandemic context. Methods: We used an individual-based transmission model to simulate COVID-19 spread along inter-individual contact networks in the LTCF setting. A range of surveillance strategies were evaluated for their ability to detect simulated outbreaks, assuming limited availability of standard RT-PCR tests. Various epidemiological scenarios were considered, including COVID-19 importation from patient transfers or staff members infected in the community. Findings: We estimated a median delay of 7 (95% uncertainty interval: 2-15) days from importation of an asymptomatic COVID-19-infected patient to first presentation of COVID-19 symptoms among any patients or staff, at which point an additional 7 (0-25) individuals were infected but did not (yet) show symptoms. Across a range of scenarios, the reference surveillance strategy (testing individuals with COVID-like symptoms with signs of severity) took a median 11-21 days to detect an outbreak. Group testing (pooling specimens from multiple individuals for a single RT-PCR test) patients and staff with any COVID-like symptoms was both the most timely and efficient strategy, detecting outbreaks up to twice as quickly as the reference, and more quickly than other considered strategies while using fewer tests. Maximizing use of available tests via testing cascades was more effective than group testing only when substantial testing resources were available (on the order of 1 test/20 beds/day). Including not merely those with symptoms but also newly admitted patients in group tests and testing cascades reduced delays in outbreak detection for LTCFs actively admitting patients potentially already infected with COVID-19. Interpretation: Improving COVID-19 surveillance can alert healthcare institutions to emerging outbreaks before they escalate, informing a need for urgent public health intervention in settings with ongoing nosocomial transmission.', 'body_text': [{'text': 'The ongoing coronavirus disease 2019 (COVID-19) pandemic, caused by novel human coronavirus SARS-CoV-2, is a public health emergency of international concern. COVID-19 is highly transmissible with a wide clinical spectrum, ranging from asymptomatic respiratory tract infection to progressive . CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'INTRODUCTION'}, {'text': 'The copyright holder for this preprint this version posted April 22, 2020. . https://doi.org/10.1101/2020.04. 19.20071639 doi: medRxiv preprint pneumonia and death. 1 Older patients and those with comorbidities are at greatest risk of severe outcomes, 2 making healthcare settings such as long-term care facilities (LTCFs) particularly vulnerable to COVID-19. 3 From rehabilitation hospitals to nursing homes, a growing number of LTCFs worldwide are reporting catastrophic outbreaks, with high rates of infection and mortality among patients and healthcare workers (HCWs) alike. [3] [4] [5] Timely detection of nosocomial COVID-19 outbreaks is essential to protect patients and HCWs and to slow contagion. Yet COVID-19 surveillance is limited by available testing resources. The current standard RT-PCR (reverse transcriptase polymerase chain reaction) test is laborious and expensive, and is in some settings subject to specific guidelines. For instance, in LTCFs in countries such as France, the Netherlands, the UK and the USA, tests have been limited only to individuals presenting with characteristic COVID-19 symptoms or signs of severity. 3, [6] [7] [8] Yet symptomatic cases represent just the tip of the iceberg: infectious individuals with no, mild, or as-yet undeveloped symptoms are common in COVID-19 outbreaks, and can easily go undetected by syndromic surveillance systems while nonetheless seeding chains of transmission. [9] [10] [11] Recent studies have indicated that surveillance strategies such as group testing may represent effective and resource-efficient alternatives to standard syndromic surveillance, [12] [13] [14] [15] but few settings have adopted such practices.', 'cite_spans': [{'start': 110, 'end': 121, 'text': '19.20071639', 'ref_id': None}, {'start': 165, 'end': 166, 'text': '1', 'ref_id': 'BIBREF0'}, {'start': 360, 'end': 361, 'text': '3', 'ref_id': 'BIBREF2'}, {'start': 579, 'end': 582, 'text': '[3]', 'ref_id': 'BIBREF2'}, {'start': 583, 'end': 586, 'text': '[4]', 'ref_id': 'BIBREF3'}, {'start': 587, 'end': 590, 'text': '[5]', 'ref_id': 'BIBREF4'}, {'start': 1146, 'end': 1148, 'text': '3,', 'ref_id': 'BIBREF2'}, {'start': 1149, 'end': 1152, 'text': '[6]', 'ref_id': 'BIBREF5'}, {'start': 1153, 'end': 1156, 'text': '[7]', 'ref_id': 'BIBREF6'}, {'start': 1157, 'end': 1160, 'text': '[8]', 'ref_id': 'BIBREF7'}, {'start': 1438, 'end': 1441, 'text': '[9]', 'ref_id': 'BIBREF8'}, {'start': 1442, 'end': 1446, 'text': '[10]', 'ref_id': 'BIBREF9'}, {'start': 1447, 'end': 1451, 'text': '[11]', 'ref_id': 'BIBREF10'}, {'start': 1629, 'end': 1633, 'text': '[12]', 'ref_id': 'BIBREF11'}, {'start': 1634, 'end': 1638, 'text': '[13]', 'ref_id': 'BIBREF12'}, {'start': 1639, 'end': 1643, 'text': '[14]', 'ref_id': 'BIBREF13'}, {'start': 1644, 'end': 1648, 'text': '[15]', 'ref_id': 'BIBREF14'}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'In order to mitigate and prevent further nosocomial outbreaks, there is an urgent need to improve COVID-19 surveillance, taking into account both limited testing resources and the unique transmission dynamics and clinical characteristics of COVID-19. Here, we investigated the efficacy and timeliness of a range of COVID-19 surveillance strategies using simulations from a dynamic COVID-19 transmission model among patients and HCWs in the LTCF setting.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'We used simulated epidemics (described below) to evaluate a range of surveillance strategies for detecting introductions of COVID-19 into the LTCF setting (Table 1) . Surveillance strategies varied according to who received conventional RT-PCR tests and with what priority, assuming limited testing resources fixed at a maximum of 1, 2, 4 or 8 tests per day (for the 170-bed LTCF simulated in the main analysis, this corresponds to roughly one test per day per 170, 85, 43 or 21 beds). We assumed a one-day lag between a test and its result, a low test sensitivity upon initial infection, and subsequently a high sensitivity upon symptom onset (see all parameter values in Table S1 ). Individuals were selected for testing according to three clinical indications: presentation of COVIDlike symptoms (mild or severe), hospital admission, or belonging to a particular demographic (e.g., patients, HCWs). Symptom-based tests were administered on the first day that symptoms appeared.', 'cite_spans': [], 'ref_spans': [{'start': 155, 'end': 164, 'text': '(Table 1)', 'ref_id': 'TABREF0'}, {'start': 673, 'end': 681, 'text': 'Table S1', 'ref_id': 'TABREF0'}], 'section': 'Surveillance strategies for COVID-19 in long-term care'}, {'text': 'COVID-19 may clinically resemble other acute respiratory infections, 1 so individuals with COVID-like or actual COVID-19 symptoms were both indicated for symptom-based testing. Average daily incidence of influenza-like illness among older adults in French emergency departments (50-99 years, 2008-2017) was used as a proxy attack rate for patients and staff presenting with COVID-like symptoms of other aetiologies, 20% of whom were assumed to develop signs of severity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Surveillance strategies for COVID-19 in long-term care'}, {'text': 'The reference strategy in the current resource-limited pandemic context was taken as testing upon presentation of COVID-like symptoms with signs of severity. Three groups of alternative strategies were also evaluated: (i) testing for other indications (any COVID-like symptoms, hospital admission), (ii) maximizing daily testing capacity via testing cascades, or (iii) group testing. To reach daily testing capacity, cascades of priority apportioned tests first to those with severe symptoms and then other indications (e.g. mild symptoms, admission) until all tests were used. For group testing, clinical specimens from multiple individuals were pooled and tested as one (up to, but not necessarily reaching a maximum of 2, 4, 8 or 16 specimens per test). Although such a group testing procedure can not reveal which individual(s) is (are) potentially infected, it can save cost and time and is widely used in screening for rare diseases, quality control in manufacturing, and computational biology.', 'cite_spans': [], 'ref_spans': [], 'section': 'Surveillance strategies for COVID-19 in long-term care'}, {'text': 'COVID-19 group testing also comes at the cost of reduced test sensitivity, 13 which was assumed to decrease linearly with each additional true negative sample as', 'cite_spans': [{'start': 75, 'end': 77, 'text': '13', 'ref_id': 'BIBREF12'}], 'ref_spans': [], 'section': 'Surveillance strategies for COVID-19 in long-term care'}, {'text': 'where sg and si are group-and individual-level test sensitivities, r is the discounting rate for each additional negative sample, S is the number of specimens, and P is the number of true positive specimens. As with standard RT-PCR testing, group testing for COVID-19 is very highly specific, 15 so perfect specificity was assumed. For all group testing strategies, we assumed that tests were first reserved for individuals presenting with severe symptoms, and that a group test was then conducted only if any tests remained.', 'cite_spans': [], 'ref_spans': [], 'section': 'Surveillance strategies for COVID-19 in long-term care'}, {'text': '. CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'cite_spans': [], 'ref_spans': [], 'section': 'Surveillance strategies for COVID-19 in long-term care'}, {'text': 'The copyright holder for this preprint this version posted April 22, 2020. . . CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'cite_spans': [], 'ref_spans': [], 'section': 'Surveillance strategies for COVID-19 in long-term care'}, {'text': 'The copyright holder for this preprint this version posted April 22, 2020. . https://doi.org/10.1101/2020.04. 19.20071639 doi: medRxiv preprint Simulating nosocomial COVID-19 spread COVID-19 epidemics were simulated using a dynamic, stochastic, individual-based transmission model, describing dynamic inter-individual contacts among and between hospital patients and personnel in a five-ward, 170-bed LTCF. 16 There were on average 154 patients and 239 members of staff present in the hospital per day, the latter partitioned across 13 distinct categories (e.g. nursing, administrative or operations staff). Both patients and staff could potentially become infected with COVID-19 and/or experience COVID-like symptoms. Hospital structure, demographics, and dynamic contact networks were estimated from close-proximity interaction data, measured via sensors worn by all patients and personnel over a 12-week period in a five-ward rehabilitation hospital in northern France (described elsewhere) 17 . Distinct contact patterns among and between individuals reflect both inherent differences in behaviour as well as different amounts of time spent in the hospital (e.g., patients are admitted and discharged but typically spend 24 hours per day in the hospital, whereas staff are present according to their standard working hours).', 'cite_spans': [{'start': 110, 'end': 143, 'text': '19.20071639 doi: medRxiv preprint', 'ref_id': None}, {'start': 407, 'end': 409, 'text': '16', 'ref_id': 'BIBREF15'}, {'start': 994, 'end': 996, 'text': '17', 'ref_id': 'BIBREF16'}], 'ref_spans': [], 'section': 'Surveillance strategies for COVID-19 in long-term care'}, {'text': 'Clinical progression of COVID-19 was characterized by: (i) a non-infectious incubation period of 2-5 days, (ii) an infectious pre-symptomatic period of 1-3 days, (iii) an on-average 7-day symptomatic period, and (iv) eventual recovery with full immunity. For (iii), we estimated that 70% of COVID-19 patients develop clinical symptoms, 18, 19 and that 20% of symptomatic patients develop severe (including critical) symptoms. 20 To reflect rapid pulmonary deterioration characteristic of severe COVID-19, we assumed no difference in average time to symptom onset across cases. As surveillance strategies were evaluated only for detection of outbreaks, death and potential long-term clinical outcomes were not explicitly simulated. All clinical and epidemiological parameters are presented in supplementary table S1.', 'cite_spans': [{'start': 336, 'end': 339, 'text': '18,', 'ref_id': 'BIBREF17'}, {'start': 340, 'end': 342, 'text': '19', 'ref_id': 'BIBREF18'}, {'start': 426, 'end': 428, 'text': '20', 'ref_id': 'BIBREF19'}], 'ref_spans': [], 'section': 'Surveillance strategies for COVID-19 in long-term care'}, {'text': 'All simulations began with an asymptomatic index case of COVID-19 imported into the LTCF on day one (t1), but five distinct epidemiological scenarios were considered to account for variable outbreak onset: (i) one infected patient admitted at t1, (ii) one infected patient admitted at t1 and subsequently once weekly , (iii) one staff member infected in the community at t1, (iv) one staff member infected in the community at t1 and subsequently once weekly, and (v) either one infected patient or one staff member introduced at t1 and subsequently once weekly (assuming 50% probability of patient or staff). Index cases, whether at t1 or later, had equal probabilities of being in the incubating, pre-symptomatic or asymptomatic stages of infection. Visitors were excluded from the model to represent common practices in the current context of pandemic COVID-19, though no additional COVID-specific interventions were implemented. For each outbreak simulation (100 per . CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Surveillance strategies for COVID-19 in long-term care'}, {'text': 'The copyright holder for this preprint this version posted April 22, 2020. . https://doi.org/10.1101/2020.04. 19.20071639 doi: medRxiv preprint importation scenario), the stochastic surveillance algorithm was run 100 times (=10,000 simulations ´ 5 scenarios) for each surveillance strategy. Resulting simulation uncertainty in model outcomes is expressed as 95% uncertainty intervals (UI).', 'cite_spans': [{'start': 110, 'end': 121, 'text': '19.20071639', 'ref_id': None}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The per-contact transmission rate of COVID-19 was estimated assuming a basic reproduction number of R0 = 3. We assumed that', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'where p is the per-minute probability of transmission between susceptible and infectious individuals in contact with one another; n is the average number of daily contacts per individual; d is the average duration of these contacts; and is the duration of the infectious period. Based on data from a contact survey in the French general community (n = 8 contacts/days, d = 30 minutes) and assuming an infectious period of = 9 days, 21, 22 we calculated a transmission probability of p = 0.14% per minute spent in contact. We further set a saturation threshold at one hour of contact, such that the per-contact transmission probability was at most 8.3% per contact between any two individuals.', 'cite_spans': [{'start': 432, 'end': 435, 'text': '21,', 'ref_id': 'BIBREF20'}, {'start': 436, 'end': 438, 'text': '22', 'ref_id': 'BIBREF21'}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'Sensitivity analyses were conducted to account for model and parameter uncertainty. First, we simulated a smaller, single-ward, 30-bed geriatric LTCF, with on average 26 patients and 80 staff per week, and fewer and longer contacts (but a similar total duration of contact) between patients.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'Second, outbreaks were simulated assuming alternative transmission rates (p=0.07%/min of contact, p=0.28%/min of contact; derived from R0 = 1.5, R0 = 6).', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'In our simulations, COVID-19 spread quickly but with a great degree of stochasticity upon its introduction to the LTCF and in the absence of surveillance or control measures ( Figure 1 ). Transfer of a single asymptomatic infected patient always resulted in an outbreak (100% of simulations).', 'cite_spans': [], 'ref_spans': [{'start': 176, 'end': 184, 'text': 'Figure 1', 'ref_id': 'FIGREF0'}], 'section': 'COVID-19 spreads quickly, but symptoms lag'}, {'text': 'Outbreaks were characterized by a median time lag of 7 (2) (3) (4) (5) (6) (7) (8) (9) (10) (11) (12) (13) (14) (15) days between the index case entering the facility and the first appearance of any COVID-19 symptoms among any infected patients or staff ( Figure 1 ). By this time, 7 (0-25) additional patients and/or staff had acquired COVID-19 but did not . CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 55, 'end': 58, 'text': '(2)', 'ref_id': 'BIBREF1'}, {'start': 59, 'end': 62, 'text': '(3)', 'ref_id': 'BIBREF2'}, {'start': 63, 'end': 66, 'text': '(4)', 'ref_id': 'BIBREF3'}, {'start': 67, 'end': 70, 'text': '(5)', 'ref_id': 'BIBREF4'}, {'start': 71, 'end': 74, 'text': '(6)', 'ref_id': 'BIBREF5'}, {'start': 75, 'end': 78, 'text': '(7)', 'ref_id': 'BIBREF6'}, {'start': 79, 'end': 82, 'text': '(8)', 'ref_id': 'BIBREF7'}, {'start': 83, 'end': 86, 'text': '(9)', 'ref_id': 'BIBREF8'}, {'start': 87, 'end': 91, 'text': '(10)', 'ref_id': 'BIBREF9'}, {'start': 92, 'end': 96, 'text': '(11)', 'ref_id': 'BIBREF10'}, {'start': 97, 'end': 101, 'text': '(12)', 'ref_id': 'BIBREF11'}, {'start': 102, 'end': 106, 'text': '(13)', 'ref_id': 'BIBREF12'}, {'start': 107, 'end': 111, 'text': '(14)', 'ref_id': 'BIBREF13'}, {'start': 112, 'end': 116, 'text': '(15)', 'ref_id': 'BIBREF14'}], 'ref_spans': [{'start': 256, 'end': 264, 'text': 'Figure 1', 'ref_id': 'FIGREF0'}], 'section': 'COVID-19 spreads quickly, but symptoms lag'}, {'text': 'The copyright holder for this preprint this version posted April 22, 2020. . https://doi.org/10.1101/2020.04. 19.20071639 doi: medRxiv preprint (yet) show symptoms. The lag until first onset of severe COVID-19 symptoms was greater (12 days from index case, [4] [5] [6] [7] [8] [9] [10] [11] [12] [13] [14] [15] [16] [17] [18] [19] [20] [21] and coincided with a greater number of undetected infections (23, 2-73).', 'cite_spans': [{'start': 110, 'end': 121, 'text': '19.20071639', 'ref_id': None}, {'start': 257, 'end': 260, 'text': '[4]', 'ref_id': 'BIBREF3'}, {'start': 261, 'end': 264, 'text': '[5]', 'ref_id': 'BIBREF4'}, {'start': 265, 'end': 268, 'text': '[6]', 'ref_id': 'BIBREF5'}, {'start': 269, 'end': 272, 'text': '[7]', 'ref_id': 'BIBREF6'}, {'start': 273, 'end': 276, 'text': '[8]', 'ref_id': 'BIBREF7'}, {'start': 277, 'end': 280, 'text': '[9]', 'ref_id': 'BIBREF8'}, {'start': 281, 'end': 285, 'text': '[10]', 'ref_id': 'BIBREF9'}, {'start': 286, 'end': 290, 'text': '[11]', 'ref_id': 'BIBREF10'}, {'start': 291, 'end': 295, 'text': '[12]', 'ref_id': 'BIBREF11'}, {'start': 296, 'end': 300, 'text': '[13]', 'ref_id': 'BIBREF12'}, {'start': 301, 'end': 305, 'text': '[14]', 'ref_id': 'BIBREF13'}, {'start': 306, 'end': 310, 'text': '[15]', 'ref_id': 'BIBREF14'}, {'start': 311, 'end': 315, 'text': '[16]', 'ref_id': 'BIBREF15'}, {'start': 316, 'end': 320, 'text': '[17]', 'ref_id': 'BIBREF16'}, {'start': 321, 'end': 325, 'text': '[18]', 'ref_id': 'BIBREF17'}, {'start': 326, 'end': 330, 'text': '[19]', 'ref_id': 'BIBREF18'}, {'start': 331, 'end': 335, 'text': '[20]', 'ref_id': 'BIBREF19'}, {'start': 336, 'end': 340, 'text': '[21]', 'ref_id': 'BIBREF20'}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'When COVID-19 was introduced by an asymptomatic infected staff member, outbreaks were less likely to occur than when patients introduced the virus (64% of simulations). Outbreaks caused by staff were further characterized by fewer transmission events on average and a greater lag between the index case and symptom onset among anyone infected, but similar mixes of patients and staff became infected regardless of who initiated the outbreak ( Figure S1 ).', 'cite_spans': [], 'ref_spans': [{'start': 443, 'end': 452, 'text': 'Figure S1', 'ref_id': 'FIGREF0'}], 'section': '(which was not certified by peer review)'}, {'text': 'The reference strategy is never the best one, whatever the number of tests Across five scenarios of COVID-19 importation into the LTCF (Figure 2A ), surveillance strategies varied greatly in their ability to detect COVID-19 outbreaks ( Figure 2B ). We first examined indicationbased testing strategies individually, prior to testing cascades or group tests. Across scenarios and testing capacities, the reference strategy (severe symptoms) was always outperformed, with lags in onset of severe symptoms entailing lags in outbreak detection. When COVID-19 was imported by patient transfer, testing patients upon hospital admission facilitated rapid outbreak detection, but at the cost of failing to detect ongoing outbreaks already underway in the facility. Conversely, testing patients and staff with any COVID-like symptoms was the strategy most likely to detect ongoing outbreaks, but like the reference strategy was hindered by lags between outbreak onset and the appearance of symptoms.', 'cite_spans': [], 'ref_spans': [{'start': 135, 'end': 145, 'text': '(Figure 2A', 'ref_id': 'FIGREF1'}, {'start': 236, 'end': 245, 'text': 'Figure 2B', 'ref_id': 'FIGREF1'}], 'section': '(which was not certified by peer review)'}, {'text': 'Increasing testing capacity generally led to more rapid outbreak detection, but some strategies benefited more from increased capacity than others ( Figure S2 ). The reference strategy benefited the least: owing to a low average daily incidence of severe COVID-like symptoms (1·0, 0·6-1·8 for scenario five), increasing capacity above 2 tests/day rarely resulted in more tests being used and had a negligible impact on outbreak detection. A similar threshold was observed for testing upon admission, owing to a limited number of new patients admitted per day (2·0, 0·0-7·0). Relative efficacies of different surveillance strategies thus depended not only on daily testing capacity but on the number of individuals indicated for testing, and in turn the number of tests actually used each day.', 'cite_spans': [], 'ref_spans': [{'start': 149, 'end': 158, 'text': 'Figure S2', 'ref_id': 'FIGREF1'}], 'section': 'Increase testing capacity to improve outbreak detection'}, {'text': '. CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Cascades and group tests detect outbreaks sooner'}, {'text': 'The copyright holder for this preprint this version posted April 22, 2020. . https://doi.org/10.1101/2020.04. 19.20071639 doi: medRxiv preprint Testing cascades and group testing detected outbreaks more quickly than traditional indicationbased testing, but for different reasons. For cascades, increasing testing capacity always resulted in more individuals being tested and hence a greater probability of outbreak detection. The relative efficacies of different cascades depended on the order of priority given to different indications ( Figure S3 ). As with individual-based tests, prioritizing admissions was advantageous for outbreaks only caused by patient transfer, whereas prioritizing symptoms was advantageous for outbreaks only caused by staff ( Figure 3 ).', 'cite_spans': [{'start': 110, 'end': 143, 'text': '19.20071639 doi: medRxiv preprint', 'ref_id': None}], 'ref_spans': [{'start': 539, 'end': 548, 'text': 'Figure S3', 'ref_id': None}, {'start': 756, 'end': 764, 'text': 'Figure 3', 'ref_id': None}], 'section': '(which was not certified by peer review)'}, {'text': 'Group testing strategies detected outbreaks more quickly than cascades when testing capacity was limited ( Figure 3 ). By pooling multiple specimens per test, group testing resulted in a greater probability of testing an infected patient or member of staff and ultimately of detecting the presence of COVID-19 in the facility, despite reduced test sensitivity. Outbreak detection was improved by increasing the maximum allowable number of specimens per group test, up to a threshold set by the number of individuals potentially indicated for inclusion, e.g. in our simulations a daily average incidence of 5·4 (3·9-7·6) individuals presenting with any COVID-like symptoms over the first three weeks of the outbreak ( Figure S4 ). Again, group testing of admitted patients was preferable when outbreaks were only caused by patient transfers, and of individuals presenting with any COVID-like symptoms when only caused by staff (Figure 3 ). Group testing strategies required on average just two tests per day, explaining minimal benefit to further increased capacity.', 'cite_spans': [], 'ref_spans': [{'start': 107, 'end': 115, 'text': 'Figure 3', 'ref_id': None}, {'start': 717, 'end': 726, 'text': 'Figure S4', 'ref_id': 'FIGREF2'}, {'start': 926, 'end': 935, 'text': '(Figure 3', 'ref_id': None}], 'section': '(which was not certified by peer review)'}, {'text': 'Group testing was more efficient than other surveillance strategies, requiring fewer tests ( Figure 4) and, in some instances, fewer specimens than other strategies ( Figure S5 ) to detect outbreaks on average up to twice as quickly as the reference strategy. In particular, group testing patients and staff with any COVID-like symptoms was the most effective and efficient strategy under limited testing capacity for outbreaks potentially caused by either patients or staff (scenario five). In this scenario, cascades required a high daily testing capacity and on average up to three times as many tests to outperform group testing. The number of specimens per group test was ultimately limited by the number of individuals indicated for inclusion, explaining variability in the number of specimens collected across different group testing strategies.', 'cite_spans': [], 'ref_spans': [{'start': 93, 'end': 102, 'text': 'Figure 4)', 'ref_id': 'FIGREF2'}, {'start': 167, 'end': 176, 'text': 'Figure S5', 'ref_id': None}], 'section': 'Group testing is the most resource-efficient strategy'}, {'text': '. CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Detection lags entail larger outbreaks'}, {'text': 'The copyright holder for this preprint this version posted April 22, 2020. . https://doi.org/10.1101/2020.04. 19.20071639 doi: medRxiv preprint The longer outbreaks went undetected, the more undocumented infections accumulated. Due to the exponential nature of emerging outbreaks, delays of only one or two days sometimes resulted in tens more infections. By detecting outbreaks more quickly, group testing and testing cascades coincided with fewer infections upon outbreak detection than simple indication-based strategies, across a range of testing capacities ( Figure S6 ).', 'cite_spans': [{'start': 110, 'end': 143, 'text': '19.20071639 doi: medRxiv preprint', 'ref_id': None}], 'ref_spans': [{'start': 564, 'end': 573, 'text': 'Figure S6', 'ref_id': None}], 'section': '(which was not certified by peer review)'}, {'text': 'When assuming lower or higher transmission rates, the sizes and durations of outbreaks varied, entailing overall longer and shorter lags until outbreak detection, respectively. However, across transmission rates, group testing patients and staff with any COVID-like symptoms remained the fastest and most efficient means of detecting an outbreak when testing capacity was limited ( Figures   S7, S8) .', 'cite_spans': [], 'ref_spans': [{'start': 382, 'end': 399, 'text': 'Figures   S7, S8)', 'ref_id': None}], 'section': 'Sensitivity analyses'}, {'text': 'In a smaller LTCF with only one ward and 30 beds, COVID-19 introductions were less likely to result in outbreaks (96% vs 100% of simulations when introduced by a newly admitted patient, and 24% vs 64% when introduced by a member of staff). For similar per-bed testing capacities, efficacies of surveillance strategies were similar to the larger LTCF, with group testing strategies proving both the most effective and most efficient under limited testing resources (Fig S9) . However, unlike in the larger LTCF, group testing random patients generally led to more rapid outbreak detection than symptomatic individuals due to the small ward size.', 'cite_spans': [], 'ref_spans': [{'start': 464, 'end': 472, 'text': '(Fig S9)', 'ref_id': None}], 'section': 'Sensitivity analyses'}, {'text': '. CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Sensitivity analyses'}, {'text': 'The copyright holder for this preprint this version posted April 22, 2020. . . CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted April 22, 2020. .', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'Comparing group tests and testing cascades in different outbreak settings. Here, detection lag is a function of daily testing capacity, comparing selected group tests and cascades to the reference strategy across three scenarios of COVID-19 importation into a LTCF (panels). For group testing, individual tests were first given to anyone presenting with severe COVID-like symptoms, and subsequently to the indicated group if any tests remained (here, to a maximum of eight individual specimens per group test). For cascades: SS=severe symptoms, MS=mild symptoms, A=admission, R=random (patients). Circles represent median, error bars represent 95% uncertainty intervals. is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Figure 3:'}, {'text': 'The copyright holder for this preprint this version posted April 22, 2020. . error bars represent 95% uncertainty intervals. For group testing strategies, individual tests were first apportioned to any individuals presenting with severe COVID-like symptoms. Overlapping circles were shifted along the x-axis by up to 5 units (tests conducted) and can be identified by reduced size of error bar whiskers. For cascades: SS=severe symptoms, MS=mild symptoms, A=admission, R=random (patients).', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'COVID-19 outbreaks in LTCFs can be catastrophic, with rapid transmission and high rates of mortality among particularly frail and elderly patients. [3] [4] [5] This motivates a need for timely and effective surveillance strategies that optimize limited testing resources to detect outbreaks as quickly as possible. In the current pandemic context, the standard-of-care in many settings is to only test individuals presenting with COVID-19 symptoms with signs of severity. [6] [7] [8] Yet active syndromic surveillance is hampered by reportedly large proportions of mild and asymptomatic infections, and by delays between infection and symptom onset. 9,10,18, 19 We used mathematical modelling to compare a range of strategies for how to distribute limited RT-PCR tests to optimize COVID-19 outbreak detection in a LTCF under a range of epidemiological scenarios. Our work suggests that the reference standard-of-care is a comparatively poor strategy. Under limited testing capacity, group testing symptomatic patients and staff was both the timeliest and most efficient strategy considered, in most scenarios requiring fewer tests and in some cases fewer clinical specimens to detect outbreaks more quickly than other strategies. Maximizing use of available tests via testing cascades was inefficient, but even more effective than group testing for timely outbreak detection when large numbers of tests were available (on the order of 1 test/20 beds/day). In addition to those with symptoms, including newly admitted patients in group tests and testing cascades substantially reduced delays in outbreak detection, but only for LTCFs actively admitting patients potentially already infected with COVID-19.', 'cite_spans': [{'start': 148, 'end': 151, 'text': '[3]', 'ref_id': 'BIBREF2'}, {'start': 152, 'end': 155, 'text': '[4]', 'ref_id': 'BIBREF3'}, {'start': 156, 'end': 159, 'text': '[5]', 'ref_id': 'BIBREF4'}, {'start': 472, 'end': 475, 'text': '[6]', 'ref_id': 'BIBREF5'}, {'start': 476, 'end': 479, 'text': '[7]', 'ref_id': 'BIBREF6'}, {'start': 480, 'end': 483, 'text': '[8]', 'ref_id': 'BIBREF7'}, {'start': 659, 'end': 661, 'text': '19', 'ref_id': 'BIBREF18'}], 'ref_spans': [], 'section': 'DISCUSSION'}, {'text': 'Our simulations predicted large nosocomial outbreaks of COVID-19 in the absence of specific control strategies. This is consistent with large outbreaks recently observed in LTCFs worldwide, including an infamous outbreak in King County, Washington that resulted in 167 confirmed infections within three weeks of the first reported case. 4 We further predicted larger and more rapid outbreaks when COVID-19 was introduced through admission of an infected patient, rather than through an infected member of staff. This is probably due to the nature of human interactions in LTCFs (patient-patient contacts are particularly long and numerous) 17 and highlights both (i) a need to screen incoming patients potentially exposed to or infected with COVID-19, 23 and (ii) the importance of interventions to limit contact between patients (e.g. isolation of retirement home residents), as already recommended for affected facilities in the current pandemic context. 3 . CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 337, 'end': 338, 'text': '4', 'ref_id': 'BIBREF3'}, {'start': 640, 'end': 642, 'text': '17', 'ref_id': 'BIBREF16'}, {'start': 752, 'end': 754, 'text': '23', 'ref_id': 'BIBREF22'}, {'start': 957, 'end': 958, 'text': '3', 'ref_id': 'BIBREF2'}], 'ref_spans': [], 'section': 'DISCUSSION'}, {'text': 'The copyright holder for this preprint this version posted April 22, 2020. . https://doi.org/10.1101/2020.04. 19.20071639 doi: medRxiv preprint Our simulations were further characterized by delays between COVID-19 introductions and symptom onset, during which time new infections not (yet) showing symptoms accumulated. This is consistent with (i) a reportedly substantial proportion of asymptomatic cases, and (ii) an important role for presymptomatic transmission. 24 Recent modelling studies estimate that 44% of secondary infections among hospital transmission-pairs resulted from pre-symptomatic transmission, 22 and that, early on in COVID-19 outbreaks, transmission events occur on average two to three days prior to symptom onset. 11 These findings highlight epidemiological challenges associated with detecting emerging outbreaks using symptoms alone. We further found that testing patients and healthcare workers with any, and not only severe COVID-like symptoms can substantially improve outbreak detection, supporting recent recommendations to expand testing criteria in LTCFs to include individuals with atypical signs and symptoms of COVID-19, such as muscle aches, sore throat and chest pain. 23 Resource limitations for COVID-19 surveillance can in part be overcome by pooling specimens from multiple individuals together in the form of group tests, which in our analysis was the most efficient surveillance strategy. Recent mathematical modeling results have also suggested that group testing could be highly cost-effective for COVID-19 screening in large populations. 12 However, our analysis was limited to one-stage group testing, in which infected individuals were not specifically identified.', 'cite_spans': [{'start': 110, 'end': 143, 'text': '19.20071639 doi: medRxiv preprint', 'ref_id': None}, {'start': 467, 'end': 469, 'text': '24', 'ref_id': 'BIBREF23'}, {'start': 615, 'end': 617, 'text': '22', 'ref_id': 'BIBREF21'}, {'start': 739, 'end': 741, 'text': '11', 'ref_id': 'BIBREF10'}, {'start': 1208, 'end': 1210, 'text': '23', 'ref_id': 'BIBREF22'}, {'start': 1586, 'end': 1588, 'text': '12', 'ref_id': 'BIBREF11'}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'This strategy is pertinent for outbreak detection but is problematic for subsequent infection control (e.g., who to isolate). In two-stage group testing, initially proposed by Dorfman in 1943 for syphilis screening among soldiers, 25 individual samples are re-tested after a positive group result to determine who is infected. Other strategies have also been proposed that involve split samples and simultaneous multi-pool samples. 26 Such strategies may be considered by facilities conducting group tests to facilitate subsequent case identification. Under the assumptions made in our analysis, including a one-day delay between identifying individuals for testing and receiving the test result, group testing is still likely to be more timely than other strategies when accounting for additional delays in obtaining second-stage test results. Decision-makers should consider the trade-off between conducting fewer tests and the potential consequences of delays in knowing which patient(s) from the group sample are positive. This work has several limitations. First, substantial uncertainties remain regarding epidemiological characteristics of COVID-19, including its transmissibility in particular settings such as LTCFs, and a potential role for environmental acquisition, 27,28 which was not included in our model. Furthermore, many LTCFs have already implemented control measures, such as interruption of social activities and provisioning of personal protective equipment, that should act to reduce transmission from baseline.', 'cite_spans': [{'start': 231, 'end': 233, 'text': '25', 'ref_id': 'BIBREF24'}, {'start': 432, 'end': 434, 'text': '26', 'ref_id': 'BIBREF25'}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'We conducted sensitivity analyses to consider unusually high and low transmission rates to reflect . CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted April 22, 2020. . https://doi.org/10.1101/2020.04. 19.20071639 doi: medRxiv preprint these uncertainties. Although outbreaks spread more or less quickly, the relative efficacies of surveillance strategies were largely unchanged in these analyses, resulting in identical conclusions for optimizing use of limited testing resources, regardless of non-pharmaceutical interventions potentially already put in place or a role for environmentally-mediated transmission.', 'cite_spans': [{'start': 110, 'end': 121, 'text': '19.20071639', 'ref_id': None}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'Second, LTCFs represent a diverse range of healthcare institutions, each with unique specializations, patient populations and living conditions, and the generalizability of our findings across these settings is not clear. Although our main analysis was informed using data from a large, five-ward rehabilitation hospital in France, we conducted a scenario analysis for a smaller, single-ward geriatric facility, and reached similar conclusions. However, we found that conducting group tests among randomly selected individuals may be preferable to symptomatic or newly admitted individuals in small facilities. This is because a large proportion of individuals could potentially be included in a daily group test that randomly samples among all individuals in the facility, improving the probability of including infected individuals in tests without a need for exceedingly large numbers of clinical specimens.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'Finally, the testing landscape for COVID-19 is due to shift quickly, with increased testing capacity and alternative testing technologies, such as rapid diagnostic tests, likely to become available in the coming months. However, uptake of new technologies is certain to be heterogeneous, and testing resources may remain limited for the foreseeable future. Although we explicitly modelled standard RT-PCR testing, our findings may be broadly generalizable to other COVID-19 testing technologies with limited capacity. Findings for group testing, however, necessarily assume that pooling samples from multiple individuals is both logistically feasible and retains sufficient test sensitivity, as demonstrated for RT-PCR. 13 Further, even in settings with abundant testing capacity, limiting the number of tests necessary to detect an outbreak may remain a priority given health-economic concerns.', 'cite_spans': [{'start': 720, 'end': 722, 'text': '13', 'ref_id': 'BIBREF12'}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'In conclusion, our results have potentially important implications for COVID-19 surveillance in the long-term care setting. With limited resources available to detect and manage burgeoning outbreaks, LTCFs may continue to be hardest hit by COVID-19, even as non-pharmaceutical interventions such as lockdowns and quarantines come to slow transmission in the community. Broadening testing guidelines to include both (i) newly admitted patients, and (ii) patients and staff presenting with any COVID-like symptoms could reduce lags in outbreak detection. Furthermore, group testing in particular could allow LTCFs to detect outbreaks days, if not weeks earlier than traditional surveillance practices, all while preserving precious testing resources.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': '. CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. ', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'None declared. Table S1 . Model parameter estimates. is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [{'start': 15, 'end': 23, 'text': 'Table S1', 'ref_id': 'TABREF0'}], 'section': 'Declarations of interest'}, {'text': 'The copyright holder for this preprint this version posted April 22, 2020 10, 29 Duration of presymptomatic period 1-3 days (uniform) 10, 29 Duration of symptomatic period (whether asymptomatic, mild symptomatic or severe symptomatic) 7 days (log-normal, σ² = 7) 30 Proportion of COVID-19 infections presenting any symptoms 0.7 18, 19 Proportion of symptomatic . CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 59, 'end': 73, 'text': 'April 22, 2020', 'ref_id': None}, {'start': 74, 'end': 77, 'text': '10,', 'ref_id': 'BIBREF9'}, {'start': 78, 'end': 80, 'text': '29', 'ref_id': 'BIBREF28'}, {'start': 134, 'end': 137, 'text': '10,', 'ref_id': 'BIBREF9'}, {'start': 138, 'end': 140, 'text': '29', 'ref_id': 'BIBREF28'}, {'start': 263, 'end': 265, 'text': '30', 'ref_id': 'BIBREF29'}, {'start': 328, 'end': 331, 'text': '18,', 'ref_id': 'BIBREF17'}, {'start': 332, 'end': 334, 'text': '19', 'ref_id': 'BIBREF18'}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted April 22, 2020. . Figure S1 . Epidemiological characteristics of COVID-19 epidemics simulated over a 12-week period in the LTCF setting, in the absence of any surveillance, control measures or interventions, and comparing different importation scenarios in each column. A) The cumulative number of infections among patients and staff in the hospital over the 12-week simulation period. Coloured bands represent 95% uncertainty intervals from stochastic simulations. B) Histograms of the final epidemic size at 12 weeks (NB: data are naturally censured by the 12-week simulation period). C) Distributions of cumulative infection totals at 12 weeks among the fourteen different categories of individuals present in the hospital.', 'cite_spans': [], 'ref_spans': [{'start': 77, 'end': 86, 'text': 'Figure S1', 'ref_id': 'FIGREF0'}], 'section': '(which was not certified by peer review)'}, {'text': '. CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted April 22, 2020. . Figure S2 . Probability of detecting COVID-19 over time for each importation scenario, here demonstrating the impact of daily testing capacity on detection probability for individual indication-based strategies. Each colour and shape represents a different testing capacity. Detection probability was computed over 10,000 simulations per scenario.', 'cite_spans': [], 'ref_spans': [{'start': 77, 'end': 86, 'text': 'Figure S2', 'ref_id': 'FIGREF1'}], 'section': '(which was not certified by peer review)'}, {'text': '. CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted April 22, 2020. . https://doi.org/10.1101/2020.04. 19.20071639 doi: medRxiv preprint Figure S3 . Probability of detecting COVID-19 outbreaks using different surveillance strategies over time for each importation scenario, here comparing the reference strategy (red crosses) to selected cascade (triangles) and group testing (squares) strategies. Detection probability was computed over 10,000 simulations per scenario. Figure S4 . Efficacy of group-based testing depends on the targets of group testing (here, newly admitted patients, random individuals, individuals with any COVID-like symptoms), daily testing capacity (x-axis), and the maximum number of specimens included per group test (colours). Here, estimates are for importation scenario five. Circles represent medians and error bars represent 95% uncertainty intervals.', 'cite_spans': [{'start': 110, 'end': 121, 'text': '19.20071639', 'ref_id': None}], 'ref_spans': [{'start': 144, 'end': 153, 'text': 'Figure S3', 'ref_id': None}, {'start': 478, 'end': 487, 'text': 'Figure S4', 'ref_id': 'FIGREF2'}], 'section': '(which was not certified by peer review)'}, {'text': '. CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted April 22, 2020. . https://doi.org/10.1101/2020.04. 19.20071639 doi: medRxiv preprint Figure S5 . For selected surveillance strategies, comparison of surveillance efficacy (y-axis) and efficiency (xaxis), here in terms of the number of clinical specimens (swabs) collected until the outbreak was detected, for COVID-19 importation scenario five (random importation by both patients and staff). Circles represent medians and error bars represent 95% uncertainty intervals. For group tests, individual tests were first apportioned to any individuals presenting with severe COVID-like symptoms. Overlapping circles were shifted along the x-axis by up to 5 units and can be identified by reduced size of error bar whiskers. For cascades: SS=severe symptoms, MS=mild symptoms, A=admission, R=random (patients).', 'cite_spans': [{'start': 110, 'end': 121, 'text': '19.20071639', 'ref_id': None}], 'ref_spans': [{'start': 144, 'end': 153, 'text': 'Figure S5', 'ref_id': None}], 'section': '(which was not certified by peer review)'}, {'text': '. CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted April 22, 2020. . Figure S6 . Relationship between detection lag (x-axis) and the number of undetected cases upon outbreak detection (y-axis), for selected surveillance strategies (colours), importation scenarios (columns) and testing capacities (rows). Circles represent medians and error bars represent 95% credible intervals. Note that some circles overlap.', 'cite_spans': [], 'ref_spans': [{'start': 77, 'end': 86, 'text': 'Figure S6', 'ref_id': None}], 'section': '(which was not certified by peer review)'}, {'text': '. CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted April 22, 2020. . https://doi.org/10.1101/2020.04. 19.20071639 doi: medRxiv preprint Figure S7 . For a sensitivity analysis with a low transmission rate (estimated from R0 = 1.5), comparison of surveillance efficacy (y-axis) and efficiency (x-axis) for COVID-19 importation scenario five (random importation by both patients and staff). Circles represent medians and error bars represent 95% uncertainty intervals. For group tests, individual tests were first apportioned to any individuals presenting with severe COVID-like symptoms. Overlapping circles were shifted along the x-axis by up to 5 units and can be identified by reduced size of error bar whiskers. For cascades: SS=severe symptoms, MS=mild symptoms, A=admission, R=random (patients). Figure S8 . For a sensitivity analysis with a high transmission rate (estimated from R0 = 6), comparison of surveillance efficacy (y-axis) and efficiency (x-axis) for COVID-19 importation scenario five (random importation by both patients and staff). Circles represent medians and error bars represent 95% uncertainty intervals. For group tests, individual tests were first apportioned to any individuals presenting with severe COVID-like symptoms. Overlapping circles were shifted along the x-axis by up to 5 units and can be identified by reduced size of error bar whiskers. For cascades: SS=severe symptoms, MS=mild symptoms, A=admission, R=random (patients).', 'cite_spans': [{'start': 110, 'end': 121, 'text': '19.20071639', 'ref_id': None}], 'ref_spans': [{'start': 144, 'end': 153, 'text': 'Figure S7', 'ref_id': None}, {'start': 808, 'end': 817, 'text': 'Figure S8', 'ref_id': None}], 'section': '(which was not certified by peer review)'}, {'text': '. CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted April 22, 2020. . Figure S9 . For a sensitivity analysis in a smaller LTCF (one ward, 30 beds), comparison of surveillance efficacy (y-axis) and efficiency (x-axis) for COVID-19 importation scenario five (random importation by both patients and staff). Circles represent medians and error bars represent 95% uncertainty intervals. For group tests, individual tests were first apportioned to any individuals presenting with severe COVID-like symptoms. Overlapping circles were shifted along the x-axis by up to 5 units and can be identified by reduced size of error bar whiskers. For cascades: SS=severe symptoms, MS=mild symptoms, A=admission, R=random (patients).', 'cite_spans': [], 'ref_spans': [{'start': 77, 'end': 86, 'text': 'Figure S9', 'ref_id': None}], 'section': '(which was not certified by peer review)'}, {'text': '. CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted April 22, 2020 . CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'cite_spans': [{'start': 59, 'end': 73, 'text': 'April 22, 2020', 'ref_id': None}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted April 22, 2020. . https://doi.org/10.1101/2020.04. 19.20071639 doi: medRxiv preprint ', 'cite_spans': [{'start': 110, 'end': 143, 'text': '19.20071639 doi: medRxiv preprint', 'ref_id': None}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}], 'paragraphs': [{'text': 'Clinical progression of COVID-19 was characterized by: (i) a non-infectious incubation period of 2-5 days, (ii) an infectious pre-symptomatic period of 1-3 days, (iii) an on-average 7-day symptomatic period, and (iv) eventual recovery with full immunity. For (iii), we estimated that 70% of COVID-19 patients develop clinical symptoms, 18, 19 and that 20% of symptomatic patients develop severe (including critical) symptoms. 20 To reflect rapid pulmonary deterioration characteristic of severe COVID-19, we assumed no difference in average time to symptom onset across cases. As surveillance strategies were evaluated only for detection of outbreaks, death and potential long-term clinical outcomes were not explicitly simulated. All clinical and epidemiological parameters are presented in supplementary table S1.', 'id': '00012'}], 'title': 'How best to use limited tests? Improving COVID-19 surveillance in long-term care', 'sha': '688cb01929e3fda511d7a94af0741520a6644ec1', 'doi': '10.1101/2020.04.19.20071639', 'score': 8.955100059509277}, {'rank': 245, 'paragraphs': [{'text': 'Due to the definition, the estimated incubation periods in this paper may be longer than the real cases. However, the distribution has some implications. It indicates that a few patients may be with very long incubation period, which increase the difficulty to control and prevent the COVID-19.', 'id': '00017'}], 'title': 'Epidemiological characteristics of 1212 COVID-19 patients in Henan, China', 'sha': '091a8e9a61e19e88caeb039f0e3888d111b20439', 'doi': '10.1101/2020.02.21.20026112', 'score': 8.947600364685059}, {'rank': 246, 'paragraphs': [{'text': 'Shaoqing et al [23] undertook research investigating the clinical characteristics and outcomes for patients submitting to surgery in the COVID-19 incubation period. The researchers undertook analysis of clinical data for 34 subjects submitting to elective surgery during the COVID-19 incubation period at four Chinese hospitals (Renmin, Tongji, Zhongnan, and Central) in Wuhan between January 1 and February 5, 2020. The patients had a median age of 55, with 20 of them (58.8%) being female. Every patient exhibited COVID-19 pneumonia symptoms within a short time of surgery, with abnormalities showing on chest CT scans. Symptoms exhibited by these patients encompassed fever (31 (91.2%)), fatigue (25 (73.5%)) and a dry cough (18 (52.9%)). 15 of the patients (44.1%) had to be admitted to the intensive care unit (ICU) as the disease progressed, and seven of these died (20.5%). Every patient examined in this research had being directly exposed to the environment in Wuhan before being admitted to hospital; no patient had exhibited any symptoms of COVID-19 prior to surgery. It was notable how swiftly the COVID-19 symptoms appeared once surgery had been completed, with the infection being confirmed by laboratory within a short period. The time gap between hospital admission and surgery (median time 2.5 days) is less than the median incubation period of 5.2 days found in patients with laboratory-confirmed infections in Wuhan [15] ; it is also less than the general incubation time in hospitals in China (median time 4.0 days, from research into infected patients from 552 Chinese hospitals [19] ). Taken together, this evidence is confirmation of the hypothesis that patients within this research were incubating COVID-19 prior to submitting to surgery.', 'id': '00017'}], 'title': 'The estimations of the COVID-19 incubation period: a systematic review of the literature', 'sha': '09ff6011e4e95bff0b9221c89ffdb91d958e4ef1', 'doi': '10.1101/2020.05.20.20108340', 'score': 8.946900367736816}, {'rank': 247, 'paragraphs': [{'text': 'where F¯(⋅) is the survival function corresponding to f(·). It is worth noting that A + V is a length-biased, version of Y, as with a longer observed Y, the corresponding V is easier to be observed. Hence the mean value E(A + V) is longer than the mean incubation period μ.', 'id': '00013'}], 'title': 'Estimation of incubation period distribution of COVID-19 using disease onset forward time: a novel cross-sectional and forward follow-up study', 'sha': 'c61be9fc4562d250e2dec241ef75bd22d61f61d1; 6d3b3f4ab80a61c45f82c61c6c756cfc6ddf4bf2', 'doi': '10.1101/2020.03.06.20032417', 'score': 8.941399574279785}, {'rank': 248, 'abstract': '', 'body_text': [{'text': 'Dear Editor,', 'cite_spans': [], 'section': '', 'ref_spans': []}, {'text': 'Initial identification of novel coronavirus disease (COVID-19) in Wuhan was alerted by the ophthalmologist by name Li Wenliang and after the enquiry of reporting unusual cases of pneumonia by Wuhan Municipal Health Commission geared up the panic in public. Though it was believed to be linked with Wuhan Huanan seafood wholesale market initially, now it is spread to more than 166 countries in the world, and it was declared as pandemic by WHO. Whole landscape of the disease has changed from epidemic to pandemic in just 45 days. Worldwide, 191,127 confirmed cases and 7807 deaths were reported as on March 18, 2020, indicating serious global public health concern (https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200318-sitrep-58-covid-19.pdf?sfvrsn=20876712_2). Situational awareness in all levels for risk management is badly required to contain the spread of this deadly disease.', 'cite_spans': [], 'section': '', 'ref_spans': []}, {'text': 'Based on SARS and MERS coronaviruses, it was believed 2–14 days as incubation period for COVID-19. The Chinese researchers found that the incubation period could be 24 days and asymptomatic carriers were evidenced to transmit the COVID-19 also. The incubation period may be as long as 0–24 days in some cases (https://www.pharmaceutical-technology.com/news/coronavirus-study-incubation-period/). The viral genomes, isolated from both bronchoalveolar lavage fluid and cell cultures, were sequenced using next-generation sequencing, and found COVID-19 virus was similar to betacoronavirus ‘bat-SL-CoVZC45’. However, the remarkable exception was the spike protein, which is only 80% similar to bat coronaviruses. The envelope spike protein plays major role in binding to receptor, membrane fusion, and transmission capacity. Interestingly, it was more similar to SARS-CoV with respect to receptor-binding domain [1]. Despite there are some variations in COVID-19 receptor-binding domain, it was suggested that COVID-19 might use angiotensin-converting enzyme 2 (ACE2) as cell membrane receptor similar to SARS-CoV [1]. ACE2 is expressed by epithelial cells of the lung, intestine, kidney, and blood vessels.', 'cite_spans': [{'start': 910, 'end': 911, 'mention': '1', 'ref_id': 'BIBREF0'}, {'start': 1112, 'end': 1113, 'mention': '1', 'ref_id': 'BIBREF0'}], 'section': '', 'ref_spans': []}, {'text': 'At present, no specific antiviral treatment and no efficient vaccines are available for COVID-19 in humans. Recently, two familiar broad-spectrum antiviral drugs remdesivir and favipiravir have been tested against clinical isolate of COVID-2019 in vitro. They found that EC90 of a remdesivir, an adenosine analogue, as 1.76 μM which can be achieved in non-human primate studies. Moreover, their preliminary data showed that it inhibits COVID-19 virus infection also in human liver cancer (Huh-7) cells.', 'cite_spans': [], 'section': '', 'ref_spans': []}, {'text': 'Some clinical trials with remdesivir, ritonavir-boosted lopinavir monotherapy, have been launched (ChiCTR2000029308, NCT04257656). Xia et al. developed a pan-CoV fusion inhibitor, EK1 peptide, to inhibit the infection of five human coronaviruses including SARS-CoV. The same group showed that the EK1 peptide and the peptide derived from HR2 domain in spike protein of COVID-19 virus could effectively inhibit COVID-19 pseudo virus infection and domain S2 mediated cell fusion [2]. It has been recently shown by using probative assays that SARS-CoV receptor-binding domain–specific monoclonal antibody, CR3022, could potentially bind with COVID-19 viral receptor-binding domain.', 'cite_spans': [{'start': 478, 'end': 479, 'mention': '2', 'ref_id': 'BIBREF1'}], 'section': '', 'ref_spans': []}, {'text': 'There are some medical conditions, which are being treated by ACE inhibitors and angiotensin II type-I receptor blockers, such as type-1 and type-2 diabetes and hypertension that result in the increased expression of ACE2 [3]. It was hypothesised that treatment with ACE2-stimulating drugs may facilitate the infection with COVID-19 virus and increase the risk of developing severe fatal COVID-19 viral infection [4]. The ACE2 gene polymorphisms might play important role in the binding of viral spike protein to cell membrane in turn COVID-19 viral infection. It suggests that the recovery of an individual is based on both therapy and ACE2 polymorphism.', 'cite_spans': [{'start': 223, 'end': 224, 'mention': '3', 'ref_id': 'BIBREF2'}, {'start': 414, 'end': 415, 'mention': '4', 'ref_id': 'BIBREF3'}], 'section': '', 'ref_spans': []}, {'text': 'Chloroquine, an antimalarial drug, has been reported as potential broad-spectrum antiviral drug, and it also has shown to inhibit viral infection of SARS-CoV. In a recent study, it was shown that chloroquine efficiently inhibits the COVID-19 viral infection in Vero E6 cells at entry and post-entry stages [5]. As chloroquine is cheap and safe drug and is being used for last 70 years, it can be clinically used against COVID-19. Further studies are warranted to better understand its efficacy.', 'cite_spans': [{'start': 307, 'end': 308, 'mention': '5', 'ref_id': 'BIBREF4'}], 'section': '', 'ref_spans': []}, {'text': 'Effective cure in the form of vaccine or therapeutic drug is the need of hour to control the COVID-19 from infecting more people. Many vaccine candidates for COVID-19 are in preclinical stage (https://www.who.int/blueprint/priority-diseases/key-action/novel-coronavirus-landscape-ncov.pdf?ua=1). The vaccine candidate developed against SARS-CoV should be evaluated against COVID-19 virus for its efficacy in inducing neutralizing antibodies as there is similarity with respect to spike protein.', 'cite_spans': [], 'section': '', 'ref_spans': []}], 'paragraphs': [{'text': 'Based on SARS and MERS coronaviruses, it was believed 2–14 days as incubation period for COVID-19. The Chinese researchers found that the incubation period could be 24 days and asymptomatic carriers were evidenced to transmit the COVID-19 also. The incubation period may be as long as 0–24 days in some cases (https://www.pharmaceutical-technology.com/news/coronavirus-study-incubation-period/). The viral genomes, isolated from both bronchoalveolar lavage fluid and cell cultures, were sequenced using next-generation sequencing, and found COVID-19 virus was similar to betacoronavirus ‘bat-SL-CoVZC45’. However, the remarkable exception was the spike protein, which is only 80% similar to bat coronaviruses. The envelope spike protein plays major role in binding to receptor, membrane fusion, and transmission capacity. Interestingly, it was more similar to SARS-CoV with respect to receptor-binding domain [1]. Despite there are some variations in COVID-19 receptor-binding domain, it was suggested that COVID-19 might use angiotensin-converting enzyme 2 (ACE2) as cell membrane receptor similar to SARS-CoV [1]. ACE2 is expressed by epithelial cells of the lung, intestine, kidney, and blood vessels.', 'id': '00003'}], 'title': 'COVID-19 Outbreak: an Update on Therapeutic Options', 'sha': '9d1c23b1c95341b37fcba06bb16a682448b370d2', 'doi': '10.1007/s42399-020-00264-6', 'score': 8.935700416564941}, {'rank': 249, 'abstract': '', 'body_text': [{'text': '\\nTo the Editor—A large global outbreak of coronavirus disease 2019 (COVID-19), caused by SARS-CoV-2, has become a critical public health issue since December 2019.1,2 SARS-CoV-2 spreads by human-to-human transmission mainly via droplets or direct contact, and it has been estimated to have mean incubation period of 6.4 days and a basic reproduction number of 2.24–3.58.3,4 An understanding of the incubation period is essential to detect epidemiological cases and helpful to determine the quarantine and medical observation period of intimate contacts. Whether SARS-CoV-2 can be transmitted during the incubation period remains controversial.5-7 Here, we report the case of a patient who had a long incubation period (38 days) and infected 1 close contact with SARS-CoV-2 during the incubation period.', 'cite_spans': [{'start': 163, 'end': 164, 'mention': '1', 'ref_id': None}, {'start': 165, 'end': 166, 'mention': '2', 'ref_id': None}, {'start': 370, 'end': 371, 'mention': '3', 'ref_id': None}, {'start': 372, 'end': 373, 'mention': '4', 'ref_id': None}, {'start': 643, 'end': 644, 'mention': '5', 'ref_id': None}, {'start': 645, 'end': 646, 'mention': '7', 'ref_id': None}], 'section': '', 'ref_spans': []}, {'text': 'A 50-year-old man (person 1), who worked in Hankou District, Wuhan, China, returned home on January 21, 2020, to a small village in the rural area of Wuhan City. First, he took the city subway, then he boarded an intercity train to a rural town, and finally his brother-in-law (person 2) drove him home by car. After returning home, person 1 lived with his elder brother (person 3), who was a single man with mild deafness who lived alone during ordinary (nonoutbreak) times. From then on, both men did not leave their home. Person 3 lived in an independent house in the small village and had neither gone to other places nor had contacted other people since January 1, 2020. Person 2 sent some food to the house of person 3 on January 17, but at that time, person 3 was not at home because he was doing farm work in the field. Persons 1 and 3 were at home together beginning January 21, 2020, and during this period they met nobody.', 'cite_spans': [], 'section': '', 'ref_spans': []}, {'text': 'Person 3 developed symptoms of cough and wheezing on February 5, 2020. On February 6, he underwent a chest computed tomography (CT) examination and was suspected of COVID-19 pneumonia. Person 3 was admitted to the hospital for treatment on February 7, and the next day his throat swab test for SARS-CoV-2 was positive by quantitative RT-PCR (qRT-PCR) analysis. During the hospitalization, person 3 was in stable condition without severe complications such as acute respiratory distress syndrome or shock. After treatment, person 3 was discharged on February 23 and isolated himself.', 'cite_spans': [], 'section': '', 'ref_spans': []}, {'text': 'Person 1 began a 14-day quarantine on February 7 and ended on February 21. On February 24, he underwent a routine CT examination before returning home. The results showed very slight inflammation of the lung, but at that time, he still did not have any symptoms. On February 27, person 1 started to have a cough, and he underwent another chest CT test, which indicated progressive pulmonary inflammation, so he was admitted to the hospital for quarantine and treatment. His SARS-CoV-2 test was also positive by qRT-PCR analysis. After treatment, he had 2 throat swab test for SARS-CoV-2 a week apart. The results of these tests were negative, and another chest CT image showed that the inflammation was absorbing. Thus, person 1 was discharged on March 2.', 'cite_spans': [], 'section': '', 'ref_spans': []}, {'text': 'Person 2 was the other man who had initial contact with person 1 on January 21. He lived with his family (his wife and son) from then on, and he had a family dinner with his other 2 relatives (his brother and sister-in-law) on January 24. These 5 people were in good health until March 6. Under strict isolation control, the villagers basically had no contact, and no other people were reported to have COVID-19 in the village where person 1 and person 3 lived (Fig. 1).', 'cite_spans': [], 'section': '', 'ref_spans': [{'start': 467, 'end': 468, 'mention': '1', 'ref_id': 'FIGREF0'}]}, {'text': '\\n\\n', 'cite_spans': [], 'section': '', 'ref_spans': []}, {'text': 'Except for person 1, who came from the Hankou District of Wuhan, the epidemic area on January 21, when COVID-19 was rapidly spreading, person 3 had no contact with other people, especially with infected or suspected persons, for more than a month before he was identified of COVID-19 on February 8. Person 3’s infection was suspected to have come from person 1, of whom the incubation period was 16 days.', 'cite_spans': [], 'section': '', 'ref_spans': []}, {'text': 'Based on the exposure history and the onset of symptoms of person 1, we reasonably speculated that the incubation period of person 1 was at least 38 days, which was calculated from January 21 to February 27. By contrast, the incubation period of these 2 patients was much longer than the mean incubation period reported by Backer et al,3 and person 1’s case may represent the longest incubation period of all cases reported up to now.8,9\\n', 'cite_spans': [{'start': 336, 'end': 337, 'mention': '3', 'ref_id': None}, {'start': 434, 'end': 435, 'mention': '8', 'ref_id': None}, {'start': 436, 'end': 437, 'mention': '9', 'ref_id': None}], 'section': '', 'ref_spans': []}, {'text': 'The long incubation period and mild symptoms of these 2 patients is notable and may be due to the weak virulence of the SARS-CoV-2 type to which they were exposed. Tang et al10 divided SARS-CoV-2 into 2 major types based on population genetic analyses of 103 SARS-CoV-2 genomes, 1 of which was less aggressive. The weak virulence of the virus is a double-edged sword. On one hand, it poses less threat to the infected, and on the other hand, people may carry the virus for a long time without realizing it and therefore transmit it to others inadvertently.', 'cite_spans': [{'start': 174, 'end': 176, 'mention': '10', 'ref_id': None}], 'section': '', 'ref_spans': []}, {'text': 'In this report, person 3 was infected by person 1 during the incubation period, which provides evidence that SARS-CoV-2 was contagious during the incubation period. The long incubation period of this COVID-19 case may contribute to our knowledge of SARS-CoV-2 and may imply that a longer observation or isolation period for contacts should be considered in SARS-CoV-2 epidemic areas.', 'cite_spans': [], 'section': '', 'ref_spans': []}], 'paragraphs': [{'text': 'In this report, person 3 was infected by person 1 during the incubation period, which provides evidence that SARS-CoV-2 was contagious during the incubation period. The long incubation period of this COVID-19 case may contribute to our knowledge of SARS-CoV-2 and may imply that a longer observation or isolation period for contacts should be considered in SARS-CoV-2 epidemic areas.', 'id': '00010'}], 'title': 'A case of COVID-19 with an ultralong incubation period', 'sha': '028d213714a327ba66679c8fc46dc9b8cdc6162e', 'doi': '10.1017/ice.2020.221', 'score': 8.920700073242188}, {'rank': 250, 'paragraphs': [{'text': 'To the Editor—A large global outbreak of coronavirus disease 2019 (COVID-19), caused by SARS-CoV-2, has become a critical public health issue since December 2019.1,2 SARS-CoV-2 spreads by human-to-human transmission mainly via droplets or direct contact, and it has been estimated to have mean incubation period of 6.4 days and a basic reproduction number of 2.24–3.58.3,4 An understanding of the incubation period is essential to detect epidemiological cases and helpful to determine the quarantine and medical observation period of intimate contacts. Whether SARS-CoV-2 can be transmitted during the incubation period remains controversial.5-7 Here, we report the case of a patient who had a long incubation period (38 days) and infected 1 close contact with SARS-CoV-2 during the incubation period.', 'id': '00001'}], 'title': 'A case of COVID-19 with an ultralong incubation period', 'sha': '028d213714a327ba66679c8fc46dc9b8cdc6162e', 'doi': '10.1017/ice.2020.221', 'score': 8.9128999710083}, {'rank': 251, 'abstract': '', 'body_text': [{'text': 'Dear Editor,', 'cite_spans': [], 'section': '', 'ref_spans': []}, {'text': 'We report here a case of COVID-19 most likely acquired during a flight from Bangui, Central African Republic to Paris, France.', 'cite_spans': [], 'section': '', 'ref_spans': []}, {'text': 'A patient in his fifties without any past medical history consulted his general practitioner, in the Marseille area on March 6th, 2020, because of fever, headache and cough evolving since February 29th. He had been sent by his company (company X) to the Central Africa Republic (CAR) from February 13th to February 25th where he gave presentations (training in management) for 6 days, to a public of about 30 resource directors of several CAR ministries. Because of persistence of fever and cough at day 9-post onset, he was referred to the emergency unit of a local hospital on March 9th. On clinical examination he had fever (40 °C) and dyspnea with oxygen saturation at 91% in ambient air. Pulmonary auscultation revealed bilateral basal crackling sounds. Malaria was ruled-out by blood smear microscopic examination and influenza was ruled-out by PCR on nasal swab fluid. C-reactive protein was at 103 mg/L and white blood cell count was at 8.8 Giga/L. Naso-pharyngeal swab fluid was tested for SARS-CoV-2 by RT-PCR as previously described [1] and resulted positive. The patient was immediately transferred to our center (IHU Méditerranée Infection) for hospitalization in a highly contagious patient section. A Chest CT-scan showed bilateral interstitial pulmonary infiltrates (Fig. 1\\n). Progressive aggravation led to his temporary transfer to an intensive care unit for five days, before coming back to our ward where he is still hospitalized at the time of writing. The patient reported that his partner (in her fifties) who works for company X and undertook the same business travel, had cough and fever from February 25th to February 29th that resolved thereafter. She had a negative SARS-Cov-2 RT-PCR on a nasopharyngeal sample obtained on March 3rd.', 'cite_spans': [{'start': 1045, 'end': 1046, 'mention': '1', 'ref_id': 'BIBREF0'}], 'section': '', 'ref_spans': [{'start': 1283, 'end': 1289, 'mention': 'Fig. 1', 'ref_id': 'FIGREF0'}]}, {'text': 'In a recently published study, conducted in COVID-19 patients, fewer than 2.5% of infected persons showed symptoms within 2.2 days (CI, 1.8–2.9 days) of exposure, and symptom onset occurred within 11.5 days (CI, 8.2–15.6 days) for 97.5% of infected persons. The estimate of the dispersion parameter was 1.52 (CI, 1.32 to 1.72), and the estimated mean incubation period was 5.5 days [2]. Given the incubation time range of COVID-19, we excluded that our patient acquired COVID-19 in France before leaving to CAR. Furthermore, only 15 documented cases were identified in France before our patient traveled to CAR none of which was documented in Marseille area, where the patient currently lives [3]. Acquisition in France on return, between February 25th and 27th, with a short incubation time was considered possible, but unlikely given that no local circulation was documented in Marseille area during this period of time [3]. The more likely place of exposure was therefore suspected to be in CAR, in a first instance, because the patient had multiple contacts in small closed rooms with several groups of people. However, investigation conducted by telephone in collaboration with the Medical Doctor of the Pasteur Institute of Bangui, where the meeting was organized, and by contacting the three other French collaborators from the company X who participated to the meeting, revealed that none of the other French and African participants presented with respiratory symptoms during the event or soon after. This data provides no strong argument for a potential exposure of our patient during his stay in CAR. Furthermore, the first confirmed case of COVID-19 in CAR was identified on March 8th only, in an Italian patient flying from Milano, Italy to CAR, long after our patient returned to France [4]. In addition, no case of COVID-19 was reported to the GeoSentinel network (https://www.istm.org/geosentinel) in patients returning from CAR, so far. By contrast, the first case of COVID-19 diagnosed in Cameroun was a French national who entered in the country after flying Air France (AF775, on February, 24th) from Paris to Yaoundé with a stopover in Bangui [5]. Our patient and his partner, used the same flight from Bangui to Yaoundé and then to Paris and Marseille in economic class. Therefore, our patient likely get infected in the plane, while traveling with the patient diagnosed eleven days later with COVID-19, in Cameroun. Investigation by Air France medical service is currently ongoing. Of note, of the three other French employees from company X who participated to the meeting, one traveled back on the same flight but in business class and the other two came back in France two days later. Aircraft transmission of COVID-19 has already been reported in China which corroborates our view [6]. This report illustrates how easily SARS-CoV-2 may travel together with their human carriers and spread the virus on board. Travel restrictions clearly make sense in the current context, not only to limit the spread of the disease to still unaffected areas but also to prevent travelers from getting infected on board.', 'cite_spans': [{'start': 383, 'end': 384, 'mention': '2', 'ref_id': 'BIBREF1'}, {'start': 694, 'end': 695, 'mention': '3', 'ref_id': 'BIBREF2'}, {'start': 923, 'end': 924, 'mention': '3', 'ref_id': 'BIBREF2'}, {'start': 1802, 'end': 1803, 'mention': '4', 'ref_id': 'BIBREF3'}, {'start': 2165, 'end': 2166, 'mention': '5', 'ref_id': 'BIBREF4'}, {'start': 2809, 'end': 2810, 'mention': '6', 'ref_id': 'BIBREF5'}], 'section': '', 'ref_spans': []}], 'paragraphs': [{'text': 'In a recently published study, conducted in COVID-19 patients, fewer than 2.5% of infected persons showed symptoms within 2.2 days (CI, 1.8–2.9 days) of exposure, and symptom onset occurred within 11.5 days (CI, 8.2–15.6 days) for 97.5% of infected persons. The estimate of the dispersion parameter was 1.52 (CI, 1.32 to 1.72), and the estimated mean incubation period was 5.5 days [2]. Given the incubation time range of COVID-19, we excluded that our patient acquired COVID-19 in France before leaving to CAR. Furthermore, only 15 documented cases were identified in France before our patient traveled to CAR none of which was documented in Marseille area, where the patient currently lives [3]. Acquisition in France on return, between February 25th and 27th, with a short incubation time was considered possible, but unlikely given that no local circulation was documented in Marseille area during this period of time [3]. The more likely place of exposure was therefore suspected to be in CAR, in a first instance, because the patient had multiple contacts in small closed rooms with several groups of people. However, investigation conducted by telephone in collaboration with the Medical Doctor of the Pasteur Institute of Bangui, where the meeting was organized, and by contacting the three other French collaborators from the company X who participated to the meeting, revealed that none of the other French and African participants presented with respiratory symptoms during the event or soon after. This data provides no strong argument for a potential exposure of our patient during his stay in CAR. Furthermore, the first confirmed case of COVID-19 in CAR was identified on March 8th only, in an Italian patient flying from Milano, Italy to CAR, long after our patient returned to France [4]. In addition, no case of COVID-19 was reported to the GeoSentinel network (https://www.istm.org/geosentinel) in patients returning from CAR, so far. By contrast, the first case of COVID-19 diagnosed in Cameroun was a French national who entered in the country after flying Air France (AF775, on February, 24th) from Paris to Yaoundé with a stopover in Bangui [5]. Our patient and his partner, used the same flight from Bangui to Yaoundé and then to Paris and Marseille in economic class. Therefore, our patient likely get infected in the plane, while traveling with the patient diagnosed eleven days later with COVID-19, in Cameroun. Investigation by Air France medical service is currently ongoing. Of note, of the three other French employees from company X who participated to the meeting, one traveled back on the same flight but in business class and the other two came back in France two days later. Aircraft transmission of COVID-19 has already been reported in China which corroborates our view [6]. This report illustrates how easily SARS-CoV-2 may travel together with their human carriers and spread the virus on board. Travel restrictions clearly make sense in the current context, not only to limit the spread of the disease to still unaffected areas but also to prevent travelers from getting infected on board.', 'id': '00004'}], 'title': 'Probable aircraft transmission of Covid-19 in-flight from the Central African Republic to France', 'sha': '', 'doi': '10.1016/j.tmaid.2020.101643', 'score': 8.912500381469727}, {'rank': 252, 'abstract': 'After the outbreak of COVID-19 (especially in the stage of tourism recovery), the bed and breakfast (B&B) tourism industry faced big challenges in improving its health strategies. B&Bs are very important for the tourism industry in China and many other countries. However, few studies have studied the impact of B&Bs, under COVID-19, on tourism in China. Our paper is among one of the first studies to investigate the impact of COVID-19 on tourist satisfaction with B&Bs in China. The work/travel restrictions started from 20 January 2020, and work/after travel resumed from 20 February 2020 in Zhejiang, China. Data were collected from 588 tourists (who experienced B&Bs in Zhejiang, China) from a WeChat online survey, from 1 March to 15 March 2020. The current study attempted to fill the gap by studying the changing tourist satisfaction levels with B&Bs before/after COVID-19. Moreover, some suggestions are given to the B&B industry for tourism resumption after COVID-19 by an importance–performance analysis (IPA).', 'body_text': [{'text': 'Corona virus disease 2019 (COVID-19) is a highly infectious disease with a long incubation period [1]. It is the latest infectious disease to rapidly develop worldwide [2]. Twenty-seven cases of the unknown virus were reported on 31 December 2019 [3]. An estimated 60 million residents of Wuhan and many other cities in China were subjected to community containment measures from 23 January 2020. These large-scale types of actions have never been used in the past (even for SARS in China) [4].', 'cite_spans': [{'start': 99, 'end': 100, 'mention': '1', 'ref_id': 'BIBREF0'}, {'start': 169, 'end': 170, 'mention': '2', 'ref_id': 'BIBREF31'}, {'start': 248, 'end': 249, 'mention': '3', 'ref_id': 'BIBREF42'}, {'start': 491, 'end': 492, 'mention': '4', 'ref_id': 'BIBREF53'}], 'section': '1. Introduction', 'ref_spans': []}, {'text': 'One of the important goals is to minimize the economic impact of the virus on a global scale [5]. China, as the world’s most populous nation and the world’s second-largest economy, had already battled with an epidemic (SARS); at the time, however, it was 4% of the global total—it is now 17% [6]. Consumption during the first season in China will be greatly reduced: tourism (e.g., bed and breakfast (B&B)), hotels, catering, entertainment, and other traditional living service industries have suffered the most [7]. Work resumption in China was raised step by step from 20 February 2020 [8].', 'cite_spans': [{'start': 94, 'end': 95, 'mention': '5', 'ref_id': 'BIBREF64'}, {'start': 293, 'end': 294, 'mention': '6', 'ref_id': 'BIBREF75'}, {'start': 513, 'end': 514, 'mention': '7', 'ref_id': 'BIBREF86'}, {'start': 589, 'end': 590, 'mention': '8', 'ref_id': 'BIBREF97'}], 'section': '1. Introduction', 'ref_spans': []}, {'text': 'In 2003, a window of opportunity to modify tourism development was opened by the crisis of SARS [8]. Nature-based areas (e.g., B&Bs in the countryside) were likely to be the target destinations [9,10]. New motivations to travel to nature-based areas became evident with SARS [11]. There was a potential marketing emphasis that nature-based tourism types (e.g., nature-based B&Bs) could be invigorated and expanded after the COVID-19 crisis [12].', 'cite_spans': [{'start': 97, 'end': 98, 'mention': '8', 'ref_id': 'BIBREF97'}, {'start': 195, 'end': 196, 'mention': '9', 'ref_id': 'BIBREF108'}, {'start': 197, 'end': 199, 'mention': '10', 'ref_id': 'BIBREF1'}, {'start': 276, 'end': 278, 'mention': '11', 'ref_id': 'BIBREF12'}, {'start': 441, 'end': 443, 'mention': '12', 'ref_id': 'BIBREF23'}], 'section': '1. Introduction', 'ref_spans': []}, {'text': 'Thus, this article focuses on B&Bs in Zhejiang, China. There are two reasons for this: (1) the COVID-19 epidemic improved in Zhejiang. From 23 January to 1 April, 2020, there was only one death in Zhejiang. No medical staff were infected. There were also no new confirmed cases from the residents for more than 14 consecutive days. Zhejiang resumed work gradually from 20 February 2020 [13]. (2) According to a B&B market development report in 2019, the main force of the B&B was from Zhejiang [14].', 'cite_spans': [{'start': 387, 'end': 389, 'mention': '13', 'ref_id': 'BIBREF24'}, {'start': 495, 'end': 497, 'mention': '14', 'ref_id': 'BIBREF25'}], 'section': '1. Introduction', 'ref_spans': []}, {'text': 'The COVID-19 epidemic has been reported in many previous papers. Some researchers have reported the impact of COVID-19 on mental health in China [15,16]. However, few studies have reported the impact of the bed and breakfast (B&B) industry, under COVID-19, on tourism in China, even though it has severely affected China and the rest of the world. B&Bs were very important for the tourism industry in China and many other countries and were especially welcomed by tourists in China, United States, and other countries [17,18]. Our paper is among one of the first studies to investigate the impact of COVID-19 on tourist satisfaction with B&B in China. The time before/after satisfaction was before the work/travel restrictions (from 20 January 2020) until work/after travel resumption (after 20 February 2020). Data were collected from 588 tourists (who have experienced B&Bs in Zhejiang, China) from a WeChat online survey, lasting from 1 March to 15 March 2020. The adjusted importance (after COVID-19)–performance (before COVID-19) analysis (IPA) was used. The current study attempts to fill the research gap by investigating the changes in tourist satisfaction levels with B&Bs before/after COVID-19. Moreover, some suggestions are given to the B&B industry to recover after the COVID-19 crisis by an importance–performance analysis (IPA).', 'cite_spans': [{'start': 146, 'end': 148, 'mention': '15', 'ref_id': 'BIBREF26'}, {'start': 149, 'end': 151, 'mention': '16', 'ref_id': 'BIBREF27'}, {'start': 519, 'end': 521, 'mention': '17', 'ref_id': 'BIBREF28'}, {'start': 522, 'end': 524, 'mention': '18', 'ref_id': 'BIBREF29'}], 'section': '1. Introduction', 'ref_spans': []}, {'text': 'Figure 1 shows the logical model. First, this study was carried out to measure the intervening influence of B&Bs before/after COVID-19 on the correlations with tourist satisfaction levels in Zhejiang, China [19,20]. There were 588 responses that were selected (who have experienced B&B in Zhejiang, China) for the analysis. Second, descriptive statistics and an importance–performance analysis (IPA) were used to measure the impact of B&B before/after COVID-19 on tourist satisfaction levels in Zhejiang. IPA is a business research technique developed as a market tool to examine and suggest management strategies [21]. IPA prioritizes management suggestions regarding the optimal allocations that should improve tourist satisfaction. Thus, it could be a valuable practical tool for management decisions [22]. Third, some suggestions are given to the B&B industry to recover after the COVID-19 crisis by an importance–performance analysis (IPA). Moreover, suggestions of crisis preparedness and disaster–management strategies for future research are given [23]. The purpose of this article was to help the B&B industry to adapt to resumption after the COVID-19 crisis.', 'cite_spans': [{'start': 208, 'end': 210, 'mention': '19', 'ref_id': 'BIBREF30'}, {'start': 211, 'end': 213, 'mention': '20', 'ref_id': 'BIBREF32'}, {'start': 615, 'end': 617, 'mention': '21', 'ref_id': 'BIBREF33'}, {'start': 805, 'end': 807, 'mention': '22', 'ref_id': 'BIBREF34'}, {'start': 1057, 'end': 1059, 'mention': '23', 'ref_id': 'BIBREF35'}], 'section': '1. Introduction', 'ref_spans': [{'start': 0, 'end': 8, 'mention': 'Figure 1', 'ref_id': 'FIGREF0'}]}, {'text': 'This article contains six sections. Section 1 is the introduction. Section 2 contains a literature review. Section 3 contains data collection and research methods. Section 4 contains the results. Section 5 comprises the impacts and limitations. Section 6 contains the conclusions.', 'cite_spans': [], 'section': '1. Introduction', 'ref_spans': []}, {'text': 'First, natural disasters and anthropogenic environmental problems [24,25], as well as their potential to affect the image of destinations, have impacts on travel and tourism on various scales [26,27]. According to world tourism organization (WTO), in 2003, tourism arrivals fell by 1.2% to 694 million (compared to the same period in 2002) in China, and hotel occupancy rates fell by 10% [28].', 'cite_spans': [{'start': 67, 'end': 69, 'mention': '24', 'ref_id': 'BIBREF36'}, {'start': 70, 'end': 72, 'mention': '25', 'ref_id': 'BIBREF37'}, {'start': 193, 'end': 195, 'mention': '26', 'ref_id': 'BIBREF38'}, {'start': 196, 'end': 198, 'mention': '27', 'ref_id': 'BIBREF39'}, {'start': 389, 'end': 391, 'mention': '28', 'ref_id': 'BIBREF40'}], 'section': '2.1. Crisis (e.g., SARS and COVID-19) Impact on Chinese Tourism and B&B ::: 2. Literature Review', 'ref_spans': []}, {'text': 'Second, the number of tourists increased by 9.2% (the first two months of 2003) over the same period in 2002, and tourism revenue increased by 14.0%. After the outbreak of SARS, the number of tourists in March 2003 decreased by 6.5%, as compared to the same period in 2002. See in Figure 2 the first monthly decrease in past decades [12].', 'cite_spans': [{'start': 334, 'end': 336, 'mention': '12', 'ref_id': 'BIBREF23'}], 'section': '2.1. Crisis (e.g., SARS and COVID-19) Impact on Chinese Tourism and B&B ::: 2. Literature Review', 'ref_spans': [{'start': 281, 'end': 289, 'mention': 'Figure 2', 'ref_id': 'FIGREF1'}]}, {'text': 'Third, how long does it take to repair the impacts of an infectious disease on tourism? The development of the crisis’ events can be divided into three periods, according to the impact on tourist flow, including the incubation period, outbreak period, and recession period. The impact time of most crisis events is within one year; the impact period of a few events was around two years. Taking SARS as an example, the peak period of impact was from March to June 2003, and the entire impact period was about 1 year [12]. Taking the accommodation industry as an example, during 2003, single-store revenue of in high-star hotels declined significantly. B&B grew by 15.2% in 2003 and continued to grow, resulting in a 22% growth in 2004. Therefore, the impact of SARS on B&Bs in tourism was basically eliminated about half a year after the end of SARS [28].', 'cite_spans': [{'start': 517, 'end': 519, 'mention': '12', 'ref_id': 'BIBREF23'}, {'start': 851, 'end': 853, 'mention': '28', 'ref_id': 'BIBREF40'}], 'section': '2.1. Crisis (e.g., SARS and COVID-19) Impact on Chinese Tourism and B&B ::: 2. Literature Review', 'ref_spans': []}, {'text': 'First, the destination image is defined as an individual’s mental representation and overall perception of a particular destination [29]. Destination image and tourist satisfaction are also important tools to actively research and manage the perceptions of tourists about the destination [30,31]. The key of the before/after crisis themes that emerged included a lack of disaster-management plans, damage to destination image and reputation. It also included the changes in tourist behavior during the crisis (e.g., COVID-19) [23]. To influence the destination choice decision-making process and to condition the after-decision-making behaviors, including participation [32], satisfaction, and future intention (e.g., sustainable mountain tourism [33,34]) to revisit [35]. The destination image is generally interpreted as impressions based on information processing from various sources over time that results in a mental representation of the attributes and benefits sought in a destination [36].', 'cite_spans': [{'start': 133, 'end': 135, 'mention': '29', 'ref_id': 'BIBREF41'}, {'start': 289, 'end': 291, 'mention': '30', 'ref_id': 'BIBREF43'}, {'start': 292, 'end': 294, 'mention': '31', 'ref_id': 'BIBREF44'}, {'start': 527, 'end': 529, 'mention': '23', 'ref_id': 'BIBREF35'}, {'start': 671, 'end': 673, 'mention': '32', 'ref_id': 'BIBREF45'}, {'start': 748, 'end': 750, 'mention': '33', 'ref_id': 'BIBREF46'}, {'start': 751, 'end': 753, 'mention': '34', 'ref_id': 'BIBREF47'}, {'start': 768, 'end': 770, 'mention': '35', 'ref_id': 'BIBREF48'}, {'start': 994, 'end': 996, 'mention': '36', 'ref_id': 'BIBREF49'}], 'section': '2.2. Tourism Resumption Post-Crisis (e.g., SARS and COVID-19) ::: 2. Literature Review', 'ref_spans': []}, {'text': 'Second, our focus on post-crisis recovery is required because much of the research relates to tourism crisis (e.g., COVID-19) management [37]. The recovery should be taken as more than just an industry or economic approach, and should focus on pre-event levels [38,39,40]. The importance of the relationship between marketing with tourist satisfaction and suggestions to repair destination images was identified [23].', 'cite_spans': [{'start': 138, 'end': 140, 'mention': '37', 'ref_id': 'BIBREF50'}, {'start': 262, 'end': 264, 'mention': '38', 'ref_id': 'BIBREF51'}, {'start': 265, 'end': 267, 'mention': '39', 'ref_id': 'BIBREF52'}, {'start': 268, 'end': 270, 'mention': '40', 'ref_id': 'BIBREF54'}, {'start': 413, 'end': 415, 'mention': '23', 'ref_id': 'BIBREF35'}], 'section': '2.2. Tourism Resumption Post-Crisis (e.g., SARS and COVID-19) ::: 2. Literature Review', 'ref_spans': []}, {'text': 'First of all, apart from hotels and guesthouses, the most common form of accommodation is bed and breakfasts (B&Bs), which is a concept that originated in Europe [41]. These refer to small hotels that provide a non-commercial, home-like environment and only serve breakfast [41]. This also means that visitors or guests pay to stay in a private residence and interact with a local family [42]. B&Bs allow tourists to seek lodging for the night, especially when hotels and inns are unavailable in remote areas [43]. Second, the basic standards are different from other types of hotels are. The differences include B&Bs being small scale, family operated and providing special services [44]. In recent years, the B&B industry has become a unique and rapidly growing industry in the hotel industry [45]. This operation attracts tourists with different standards than hotels [46].', 'cite_spans': [{'start': 163, 'end': 165, 'mention': '41', 'ref_id': 'BIBREF55'}, {'start': 275, 'end': 277, 'mention': '41', 'ref_id': 'BIBREF55'}, {'start': 389, 'end': 391, 'mention': '42', 'ref_id': 'BIBREF56'}, {'start': 510, 'end': 512, 'mention': '43', 'ref_id': 'BIBREF57'}, {'start': 685, 'end': 687, 'mention': '44', 'ref_id': 'BIBREF58'}, {'start': 796, 'end': 798, 'mention': '45', 'ref_id': 'BIBREF59'}, {'start': 872, 'end': 874, 'mention': '46', 'ref_id': 'BIBREF60'}], 'section': '2.3. The Concept of B&B ::: 2. Literature Review', 'ref_spans': []}, {'text': 'This study was carried out in Zhejiang, China. As the most popular B&B rural tourist destination in China, the area receives more than 23.52 million tourists. From 1 January 2015, to 14 December 2019, the Baidu index results showed that the top 10 B&B provinces and cities were Zhejiang, Guangdong, Sichuan, Jiangsu, Beijing, Shanghai, Shandong, Henan, Chongqing, and Hubei. It shows that Zhejiang was the most concerned about B&Bs. Most of these areas are economically developed provinces and cities. According to the B&B Market Development Report in 2019, the main force of the B&B was from Zhejiang Province and accounts for about 60% of tourists, which is consistent with the search results of the area where B&Bs are present [47]. The highest media coverage about B&Bs in China was in Zhejiang from January 2015 to February 2018. The topics of media concern ranged from the rapid development of B&Bs and the reference of B&B experience to the problems arising in the development of B&Bs and lasted until the introduction of B&B standards, which indicates that the development of B&Bs in China entered a stable development stage from the initial stage of rapid growth without supervision [47].', 'cite_spans': [{'start': 731, 'end': 733, 'mention': '47', 'ref_id': 'BIBREF61'}, {'start': 1193, 'end': 1195, 'mention': '47', 'ref_id': 'BIBREF61'}], 'section': '2.4. B&B in Zhejiang ::: 2. Literature Review', 'ref_spans': []}, {'text': 'Various definitions of satisfaction have been proposed in the literature. In the tourism sector, tourist satisfaction (TS) is an essential aspect of the tourist services sector [48]. As services directly impact people [49], some researchers have indicated that services are linked to tourist satisfaction [41]. Tourist satisfaction, as a marketing tool, plays a key role in the construction of strategies in the tourism market [50]. Furthermore, satisfaction is vital for successful destination marketing [51], as well as a service organization [52]. Feelings of pleasure by tourists are a sign of satisfaction [53], while tourists who enjoy visiting are satisfied [54,55]. Therefore, tourist contentment is a considerable factor for tourists in making up their minds to visit again or not [56,57,58].', 'cite_spans': [{'start': 178, 'end': 180, 'mention': '48', 'ref_id': 'BIBREF62'}, {'start': 219, 'end': 221, 'mention': '49', 'ref_id': 'BIBREF63'}, {'start': 306, 'end': 308, 'mention': '41', 'ref_id': 'BIBREF55'}, {'start': 428, 'end': 430, 'mention': '50', 'ref_id': 'BIBREF65'}, {'start': 506, 'end': 508, 'mention': '51', 'ref_id': 'BIBREF66'}, {'start': 546, 'end': 548, 'mention': '52', 'ref_id': 'BIBREF67'}, {'start': 612, 'end': 614, 'mention': '53', 'ref_id': 'BIBREF68'}, {'start': 666, 'end': 668, 'mention': '54', 'ref_id': 'BIBREF69'}, {'start': 669, 'end': 671, 'mention': '55', 'ref_id': 'BIBREF70'}, {'start': 791, 'end': 793, 'mention': '56', 'ref_id': 'BIBREF71'}, {'start': 794, 'end': 796, 'mention': '57', 'ref_id': 'BIBREF72'}, {'start': 797, 'end': 799, 'mention': '58', 'ref_id': 'BIBREF73'}], 'section': '2.5. Tourist Satisfaction ::: 2. Literature Review', 'ref_spans': []}, {'text': 'Enhancing tourist satisfaction is a key strategy that leads to the success of companies in the hotel [59,60], catering [61,62], and tourism industries [63]. The quantitative approach with surveys was extensively adopted by scholars to study the multiple determinants of tourist satisfaction [64]. For instance, Deng et al. [65] surveyed 412 overseas tourists in Taiwan and found tourist complaints and service quality were related to tourist satisfaction. Kim et al. [66] gathered the opinions of 317 tourists from Beijing and discovered that convenience, safety, and technological inclination were the main factors that influenced tourist satisfaction. As a unique style of accommodation, it was inappropriate to employ the factors identified in other contexts directly to B&B during our investigations [67,68,69].', 'cite_spans': [{'start': 102, 'end': 104, 'mention': '59', 'ref_id': 'BIBREF74'}, {'start': 105, 'end': 107, 'mention': '60', 'ref_id': 'BIBREF76'}, {'start': 120, 'end': 122, 'mention': '61', 'ref_id': 'BIBREF77'}, {'start': 123, 'end': 125, 'mention': '62', 'ref_id': 'BIBREF78'}, {'start': 152, 'end': 154, 'mention': '63', 'ref_id': 'BIBREF79'}, {'start': 292, 'end': 294, 'mention': '64', 'ref_id': 'BIBREF80'}, {'start': 324, 'end': 326, 'mention': '65', 'ref_id': 'BIBREF81'}, {'start': 468, 'end': 470, 'mention': '66', 'ref_id': 'BIBREF82'}, {'start': 805, 'end': 807, 'mention': '67', 'ref_id': 'BIBREF83'}, {'start': 808, 'end': 810, 'mention': '68', 'ref_id': 'BIBREF84'}, {'start': 811, 'end': 813, 'mention': '69', 'ref_id': 'BIBREF85'}], 'section': '2.5. Tourist Satisfaction ::: 2. Literature Review', 'ref_spans': []}, {'text': 'This study was carried out to measure the intervening influence of B&Bs before/after COVID-19 on its relationship with tourist satisfaction in Zhejiang, China. We used WeChat (Tencent, Shenzhen, China) for this online survey in Zhejiang, China. We received 1120 answers to the questionnaire. However, there were 588 responses from people who have experienced B&Bs in Zhejiang before the COVID-19 that were selected for the analysis. The responses were collected from 1 March to 15 March 2020.', 'cite_spans': [], 'section': '3.1.1. Explanation of Questionnaire ::: 3.1. Data Collection ::: 3. Materials and Methods', 'ref_spans': []}, {'text': 'The questionnaire consisted of 30 factors, from the expectation of B&Bs before check-in, to the perception of facilities after check-in [70]. Likert’s five-point scale was used to measure tourists’ expectations before check-in, with five optional levels [71]: (1) Importance (After COVID-19): “5 = very important”, “4 = important”, “3 = so–so”, “2 = unimportant”, and “1 = very unimportant”. Appendix A shows the sample questionnaire [72,73]. (2) Performance (Before COVID-19): “5 = very good”, “4 = good”, “3 = so–so”, “2 = not good”, and “1 = bad”.', 'cite_spans': [{'start': 137, 'end': 139, 'mention': '70', 'ref_id': 'BIBREF87'}, {'start': 255, 'end': 257, 'mention': '71', 'ref_id': 'BIBREF88'}, {'start': 435, 'end': 437, 'mention': '72', 'ref_id': 'BIBREF89'}, {'start': 438, 'end': 440, 'mention': '73', 'ref_id': 'BIBREF90'}], 'section': '3.1.1. Explanation of Questionnaire ::: 3.1. Data Collection ::: 3. Materials and Methods', 'ref_spans': []}, {'text': 'In addition to people’s natural awareness and sharing awareness, the development of home-stays is more about providing experiential services for tourists than those provided by basic accommodation services [74]. Some researchers constructed an experiential scale to tap into tourist experiences in the accommodation industry [75,76]. The determinants of consumer satisfaction with B&B establishments were studied and a hierarchical structure of these determinants was built. Thus, with the intention of bridging this gap, we aspired to develop a multiple-item scale to measure tourist opinions about B&Bs before/after COVID-19.', 'cite_spans': [{'start': 207, 'end': 209, 'mention': '74', 'ref_id': 'BIBREF91'}, {'start': 326, 'end': 328, 'mention': '75', 'ref_id': 'BIBREF92'}, {'start': 329, 'end': 331, 'mention': '76', 'ref_id': 'BIBREF93'}], 'section': '3.1.2. Questionnaire Items ::: 3.1. Data Collection ::: 3. Materials and Methods', 'ref_spans': []}, {'text': 'Ten determinants of tourist satisfaction were identified [77]. Based on previous research and B&B industry evaluation standards (BIES) in China (Table 1, Figure 3), a number of factors were generated. All the factors were assessed for content and face validity by a panel of experts from two institutions affiliated with the authors [78,79].', 'cite_spans': [{'start': 58, 'end': 60, 'mention': '77', 'ref_id': 'BIBREF94'}, {'start': 334, 'end': 336, 'mention': '78', 'ref_id': 'BIBREF95'}, {'start': 337, 'end': 339, 'mention': '79', 'ref_id': 'BIBREF96'}], 'section': '3.1.2. Questionnaire Items ::: 3.1. Data Collection ::: 3. Materials and Methods', 'ref_spans': [{'start': 154, 'end': 162, 'mention': 'Figure 3', 'ref_id': 'FIGREF2'}, {'start': 145, 'end': 152, 'mention': 'Table 1', 'ref_id': 'TABREF0'}]}, {'text': 'Importance–Performance Analysis (IPA) is a business research technique developed as a marketing tool to review and suggest new management strategies [21]. While it was originally developed for marketing purposes, its applications have expanded to various fields, including tourism [90,91,92], healthcare [93,94], sustainable cities [95], social and economic outcomes [96,97], etc. The main goal of IPA is to diagnose the performance of different product or service attributes while facilitating data interpretation and providing practical recommendations for management [98]. IPA can gain insight into which product or service area managers should be targeted by identifying the most critical attributes, strengths and weaknesses [99]. The IPA technique combines measures of tourists’ perceived performance and importance into a two-dimensional plot to facilitate data interpretation [21]. Thus, each quadrant in the standard IPA chart represents a different strategy that can help managers to identify areas of concern and necessary measures to increase tourist satisfaction [100]. Choosing the right attributes to measure importance and performance is essential for obtaining the best management decisions because these decisions rely on the information revealed from the selected attribute set [101]. Figure 4 shows the IPA model.', 'cite_spans': [{'start': 150, 'end': 152, 'mention': '21', 'ref_id': 'BIBREF33'}, {'start': 282, 'end': 284, 'mention': '90', 'ref_id': 'BIBREF109'}, {'start': 285, 'end': 287, 'mention': '91', 'ref_id': 'BIBREF110'}, {'start': 288, 'end': 290, 'mention': '92', 'ref_id': 'BIBREF111'}, {'start': 305, 'end': 307, 'mention': '93', 'ref_id': 'BIBREF112'}, {'start': 308, 'end': 310, 'mention': '94', 'ref_id': 'BIBREF113'}, {'start': 333, 'end': 335, 'mention': '95', 'ref_id': 'BIBREF114'}, {'start': 368, 'end': 370, 'mention': '96', 'ref_id': 'BIBREF115'}, {'start': 371, 'end': 373, 'mention': '97', 'ref_id': 'BIBREF116'}, {'start': 571, 'end': 573, 'mention': '98', 'ref_id': 'BIBREF117'}, {'start': 731, 'end': 733, 'mention': '99', 'ref_id': 'BIBREF118'}, {'start': 885, 'end': 887, 'mention': '21', 'ref_id': 'BIBREF33'}, {'start': 1077, 'end': 1080, 'mention': '100', 'ref_id': 'BIBREF2'}, {'start': 1298, 'end': 1301, 'mention': '101', 'ref_id': 'BIBREF3'}], 'section': '3.2.1. Concept of IPA ::: 3.2. Importance–Performance Analysis (IPA) ::: 3. Materials and Methods', 'ref_spans': [{'start': 1304, 'end': 1312, 'mention': 'Figure 4', 'ref_id': 'FIGREF3'}]}, {'text': 'The best place to divide the graph into quadrant thresholds is one of the biggest problems in IPA applications [102]. The choice of threshold is almost a matter of judgment [103,104]. However, their subjective positions has led to inconsistencies in existing IPA research results [104].', 'cite_spans': [{'start': 112, 'end': 115, 'mention': '102', 'ref_id': 'BIBREF4'}, {'start': 174, 'end': 177, 'mention': '103', 'ref_id': 'BIBREF5'}, {'start': 178, 'end': 181, 'mention': '104', 'ref_id': 'BIBREF6'}, {'start': 281, 'end': 284, 'mention': '104', 'ref_id': 'BIBREF6'}], 'section': '3.2.2. Location of the Discriminating Thresholds within the IPA Plot ::: 3.2. Importance–Performance Analysis (IPA) ::: 3. Materials and Methods', 'ref_spans': []}, {'text': 'First, the data-centric (DC) method uses the actual data average of the observed importance and performance level as the critical point. Therefore, scholars have proposed another solution [105,106,107]. That is, they set the mean of the experience gained from the data as the intersection [108].', 'cite_spans': [{'start': 189, 'end': 192, 'mention': '105', 'ref_id': 'BIBREF7'}, {'start': 193, 'end': 196, 'mention': '106', 'ref_id': 'BIBREF8'}, {'start': 197, 'end': 200, 'mention': '107', 'ref_id': 'BIBREF9'}, {'start': 290, 'end': 293, 'mention': '108', 'ref_id': 'BIBREF10'}], 'section': '3.2.2. Location of the Discriminating Thresholds within the IPA Plot ::: 3.2. Importance–Performance Analysis (IPA) ::: 3. Materials and Methods', 'ref_spans': []}, {'text': 'Second, some authors suggested that the SC method is more transparent when interpreting research results and usually provides a simpler description than using actual data methods [104]. However, using the scaling method has a serious drawback, that is, in addition to the fact that it is not driven by actual data, it also tends to record the high importance level of all the attributes. The latter means that, regardless of the characteristics of the interviewees, it turns out that this is the determinant of their expectations and opinions [109]. Each survey will have the same discrimination threshold. Incorrect threshold settings may lead to misleading and conflicting management recommendations [22].', 'cite_spans': [{'start': 180, 'end': 183, 'mention': '104', 'ref_id': 'BIBREF6'}, {'start': 544, 'end': 547, 'mention': '109', 'ref_id': 'BIBREF11'}, {'start': 703, 'end': 705, 'mention': '22', 'ref_id': 'BIBREF34'}], 'section': '3.2.2. Location of the Discriminating Thresholds within the IPA Plot ::: 3.2. Importance–Performance Analysis (IPA) ::: 3. Materials and Methods', 'ref_spans': []}, {'text': 'Third, other researchers used diagonal lines (DL) or so-called isolines (IRL) to divide the plot into two separate areas [103]. The point on this 45° upward line indicates an attribute with the same importance and performance level; compared to the subjective threshold selection method, the IRL method can be said to be a more suitable method for identifying the area of interest because it directly focuses on satisfaction and importance grade difference [22]. Rial et al. [110] simplified this method by empirical means and a diagonal line with discrepancies. The difference in attributes (distance from the diagonal) is considered to be a priority in improving the service [108].', 'cite_spans': [{'start': 122, 'end': 125, 'mention': '103', 'ref_id': 'BIBREF5'}, {'start': 458, 'end': 460, 'mention': '22', 'ref_id': 'BIBREF34'}, {'start': 476, 'end': 479, 'mention': '110', 'ref_id': 'BIBREF13'}, {'start': 678, 'end': 681, 'mention': '108', 'ref_id': 'BIBREF10'}], 'section': '3.2.2. Location of the Discriminating Thresholds within the IPA Plot ::: 3.2. Importance–Performance Analysis (IPA) ::: 3. Materials and Methods', 'ref_spans': []}, {'text': 'Linear relationship (or linear association) is a statistical term used to describe the linear relationship between variables and constants. Mathematically speaking, the linear relationship satisfies the equation:(1)y=mx+b.', 'cite_spans': [], 'section': '3.2.2. Location of the Discriminating Thresholds within the IPA Plot ::: 3.2. Importance–Performance Analysis (IPA) ::: 3. Materials and Methods', 'ref_spans': []}, {'text': 'In this equation, “x” and “y” are the two variables associated with parameters “m” and “b”. Graphically, y = mx + b is drawn on the x-y plane with the slope “m” and y-intercept “b”. When x = 0, y-intercept “b” is just the value of “y”. Calculate the slope “m” from any two separate points (x1, y1) and (x2, y2), as follows:(2)m=(x2−x1)/(y2−y1).', 'cite_spans': [], 'section': '3.2.2. Location of the Discriminating Thresholds within the IPA Plot ::: 3.2. Importance–Performance Analysis (IPA) ::: 3. Materials and Methods', 'ref_spans': []}, {'text': 'However, compared to the standard IPA chart (with four quadrants), it produces less information, provides limited identification ability and therefore has limited interpretation ability. Therefore, it limits the usefulness of IPA [22].', 'cite_spans': [{'start': 231, 'end': 233, 'mention': '22', 'ref_id': 'BIBREF34'}], 'section': '3.2.2. Location of the Discriminating Thresholds within the IPA Plot ::: 3.2. Importance–Performance Analysis (IPA) ::: 3. Materials and Methods', 'ref_spans': []}, {'text': 'The actual means of importance and performance are likely to differ in most cases, and therefore, require study-specific adjustments to the scales in order to interpret the importance and performance ratings [111], as well as the relative interpretation of attributes within the importance and performance ratings [22]. Most researchers use DC and average values of the actual importance and performance level when determining the threshold value of tourism research [108]. IRL directly focuses on differences in satisfaction (Before COVID-19) and importance (After COVID-19) ratings. Therefore, this article uses the method of DC+IRL when specifying the thresholds of the impact before/after COVID-19 on tourist satisfaction with B&B (Figure 5).', 'cite_spans': [{'start': 209, 'end': 212, 'mention': '111', 'ref_id': 'BIBREF14'}, {'start': 315, 'end': 317, 'mention': '22', 'ref_id': 'BIBREF34'}, {'start': 468, 'end': 471, 'mention': '108', 'ref_id': 'BIBREF10'}], 'section': '3.2.2. Location of the Discriminating Thresholds within the IPA Plot ::: 3.2. Importance–Performance Analysis (IPA) ::: 3. Materials and Methods', 'ref_spans': [{'start': 736, 'end': 744, 'mention': 'Figure 5', 'ref_id': 'FIGREF4'}]}, {'text': 'Table 1 describes the respondents’ demographic profile [96,97,98]. Among the 588 tourists, 55.78% were women and 44.22% were men. The majority of the participants ranged from 25 to 35, accounting for 45.92% of the samples. Most of the respondents had a bachelor or graduate degree (51.02%, n = 300), followed by graduate degrees (32.31%, n = 190). With regards to monthly income, 45.92% (n = 270) reported that their annual income was between $801 and $1200.', 'cite_spans': [{'start': 56, 'end': 58, 'mention': '96', 'ref_id': 'BIBREF115'}, {'start': 59, 'end': 61, 'mention': '97', 'ref_id': 'BIBREF116'}, {'start': 62, 'end': 64, 'mention': '98', 'ref_id': 'BIBREF117'}], 'section': '4.1.1. Profile of Survey Respondents ::: 4.1. The Descriptive Statistics ::: 4. Results', 'ref_spans': [{'start': 0, 'end': 7, 'mention': 'Table 1', 'ref_id': 'TABREF0'}]}, {'text': 'The statistical software of SPSS 26 (IBM, New York, NY, USA) was used in the questionnaire analysis [112,113]. The calculation of the questionnaire’s reliability was based on the Cronbach’s Alpha coefficient [101,102]. An α larger than 0.7 indicates “highly reliable” and larger than 0.5 “reliable” [114,115,116]. The α for this questionnaire was 0.978, which indicated a relatively high and acceptable reliability [71,90]. The questionnaire also proved satisfying in terms of the content validity, criterion-related validity, and construct validity (Table 2, Table 3).', 'cite_spans': [{'start': 101, 'end': 104, 'mention': '112', 'ref_id': 'BIBREF15'}, {'start': 105, 'end': 108, 'mention': '113', 'ref_id': 'BIBREF16'}, {'start': 209, 'end': 212, 'mention': '101', 'ref_id': 'BIBREF3'}, {'start': 213, 'end': 216, 'mention': '102', 'ref_id': 'BIBREF4'}, {'start': 300, 'end': 303, 'mention': '114', 'ref_id': 'BIBREF17'}, {'start': 304, 'end': 307, 'mention': '115', 'ref_id': 'BIBREF18'}, {'start': 308, 'end': 311, 'mention': '116', 'ref_id': 'BIBREF19'}, {'start': 416, 'end': 418, 'mention': '71', 'ref_id': 'BIBREF88'}, {'start': 419, 'end': 421, 'mention': '90', 'ref_id': 'BIBREF109'}], 'section': '4.1.2. Reliability and Validity Analysis ::: 4.1. The Descriptive Statistics ::: 4. Results', 'ref_spans': [{'start': 551, 'end': 558, 'mention': 'Table 2', 'ref_id': 'TABREF1'}, {'start': 560, 'end': 567, 'mention': 'Table 3', 'ref_id': 'TABREF2'}]}, {'text': 'Cronbach’s alpha is a function of the number of test items and the average inter-correlation among the items. It showed the formula of the Cronbach’s alpha below [114]:(3)α=N∗cˉvˉ+(N−1)∗cˉ.', 'cite_spans': [{'start': 163, 'end': 166, 'mention': '114', 'ref_id': 'BIBREF17'}], 'section': '4.1.2. Reliability and Validity Analysis ::: 4.1. The Descriptive Statistics ::: 4. Results', 'ref_spans': []}, {'text': 'Here, N is equal to the number of items, cˉ is the average covariance between the item-pairs, and vˉ is equal to the average variance. It can be seen from this formula that, if you increase the number of items, you will increase Cronbach’s alpha. In addition, if the correlation between the average items is low, the alpha will be low. As the correlation between the average items increases, Cronbach’s alpha will increase (keeping the number of items unchanged).', 'cite_spans': [], 'section': '4.1.2. Reliability and Validity Analysis ::: 4.1. The Descriptive Statistics ::: 4. Results', 'ref_spans': []}, {'text': 'The mean responses for the importance and performance of the 30 attributes were analyzed in accordance with the IPA framework and are shown in Table 4. Most of the importance and performance means (Table 5.) were found to be significantly different (Sig. 2–tailed) at the <0.01 level (QN. 23/25/28/29/30 <0.05) [117]. Variables in each category were ranked in order by Paired Differences (IA–PB) [118,119].', 'cite_spans': [{'start': 312, 'end': 315, 'mention': '117', 'ref_id': 'BIBREF20'}, {'start': 397, 'end': 400, 'mention': '118', 'ref_id': 'BIBREF21'}, {'start': 401, 'end': 404, 'mention': '119', 'ref_id': 'BIBREF22'}], 'section': '4.1.3. Importance–Performance Scores ::: 4.1. The Descriptive Statistics ::: 4. Results', 'ref_spans': [{'start': 143, 'end': 150, 'mention': 'Table 4', 'ref_id': 'TABREF3'}, {'start': 198, 'end': 205, 'mention': 'Table 5', 'ref_id': 'TABREF4'}]}, {'text': 'Figure 6 shows results of the analysis. High priority area (part of Quadrant 4+3): (Quadrant 4) the “concentrate here” area; (Quadrant 3) the “low priority” area. The attributes in this quadrant were considered to perform poorly and therefore represent the main weakness of the product and a threat to its competitiveness. In terms of investment, these attributes have the highest priority [98]. Rank by paired differences (IA–PB): (1)“The layout of the rooms is scattered”, (2) “Split air conditioners are used in guest rooms”, (3) “Places or items for cleaning and disinfection are provided to tourists”, (4) “Rooms of Single B&B are few and exquisite”, (5) “Contingency plans are developed and can be exercised regularly”, and (6) “Buildings are intelligent (e.g., semi–self–service management)”.', 'cite_spans': [{'start': 391, 'end': 393, 'mention': '98', 'ref_id': 'BIBREF117'}], 'section': '4.2. Importance (after COVID-19)–Performance (before COVID-19) Analysis (IPA) ::: 4. Results', 'ref_spans': [{'start': 0, 'end': 8, 'mention': 'Figure 6', 'ref_id': 'FIGREF5'}]}, {'text': 'Priority area (part of Quadrant 1+3): (Quadrant 1) the “keep up the good work” area; (Quadrant 3) the “low priority” area. It represents the main and potential competitive advantages of a product or service. Attributes in this quadrant are considered to be performing well and investment needs to continue. Rank by paired differences (IA–PB): (7) “Rooms are naturally ventilated”, (8) “The emergency facilities are complete (such as: first aid kit, escape equipment)”, (9) “The rooms are spacious and clean”, (10) “The outdoor space is large and natural (e.g., courtyard, terrace, roof garden)”, and (11) “Green consumption is encouraged and environmental protection measures are implemented”.', 'cite_spans': [], 'section': '4.2. Importance (after COVID-19)–Performance (before COVID-19) Analysis (IPA) ::: 4. Results', 'ref_spans': []}, {'text': 'Medium priority area (part of Quadrant 1): (Quadrant 1) the “only keep up the good work” area. The importance and performance of these factors (e.g., “location and nearby facilities are safe and good”, “other service rooms are clean and tidy”) were good.', 'cite_spans': [], 'section': '4.2. Importance (after COVID-19)–Performance (before COVID-19) Analysis (IPA) ::: 4. Results', 'ref_spans': []}, {'text': 'Low priority area (part of Quadrant 2+3): (Quadrant 2) the “possible overkill” area; (Quadrant 3) the “low priority” area. Their performance was not particularly good, but they were considered relatively unimportant to tourists; therefore, managers should not pay too much attention to these attributes. They represent a slight weakness, and poor performance is not a big problem. These factors are not important in this article (e.g., “the indoor and outdoor transition spaces are natural and beautiful (e.g., gallery frames, awnings, balconies)”, “the shading performance is good (e.g., opaque curtains))”.', 'cite_spans': [], 'section': '4.2. Importance (after COVID-19)–Performance (before COVID-19) Analysis (IPA) ::: 4. Results', 'ref_spans': []}, {'text': 'First, to the best of our knowledge, this study is among the first to uncover the impact of COVID-19 factors influencing tourists’ satisfaction with B&Bs. Second, from the perspective of methodology, DC and IRL were combined with content analysis to sort and guide the complexity of the relationship between variables, which has certain value for future research. Third, some suggestions would be given to the B&B industry to recover after COVID-19 by the importance–performance analysis (IPA). Our study extends this research area from the traditional B&B context and adds knowledge to the post–COVID-19 B&B tourism management area.', 'cite_spans': [], 'section': 'Implications ::: 5.1. Impacts ::: 5. Discussion', 'ref_spans': []}, {'text': 'Suggestions for the rank by paired differences (IA–PB): (1) “The layout of the rooms is scattered”. This shows that, after the COVID-19 epidemic, tourists prefer scattered room layouts. Centralized room layouts need to be reconsidered. (2) “Split air conditioners are used in guest rooms”. At present, central air conditioning has been used by many B&Bs. However, after COVID-19, this is not an ideal choice. (3) “Places or items for cleaning and disinfection are provided to tourists”, and (4) “Rooms of Single B&B are few and exquisite”. At present, there are more and more rooms in many B&Bs (single) and the scale is getting larger. This is obviously inappropriate for B&B tourism after the COVID-19 epidemic and it needs to be changed. (5) “Contingency plans are developed and can be exercised regularly”. These measures were not paid enough attention before the outbreak. It needs to be focused on after the COVID-19 epidemic. (6) “Buildings are intelligent (e.g., semi–self–service management)”. “Intelligentization” will be a trend in the future. It also needs to be focused on in B&Bs.', 'cite_spans': [], 'section': '5.2.1. High Priority Suggestions ::: 5.2. Suggestions ::: 5. Discussion', 'ref_spans': []}, {'text': 'Suggestions for the rank by paired differences (IA–PB): (7) “Rooms are naturally ventilated”, (8) “The emergency facilities are complete (such as first aid kit, escape equipment)”, (9) “The rooms are spacious and clean”, (10) “The outdoor space is large and natural (e.g., courtyard, terrace, roof garden)”, and (11) “Green consumption is encouraged and environmental protection measures are implemented”. The suggestion in this part is that more attention should be paid to nature and green areas in the B&B tourism after COVID-19. Just like after SARS in 2003, people tended to go to places with nature-based areas rather than urban vacations [7,43].', 'cite_spans': [{'start': 646, 'end': 647, 'mention': '7', 'ref_id': 'BIBREF86'}, {'start': 648, 'end': 650, 'mention': '43', 'ref_id': 'BIBREF57'}], 'section': '5.2.2. Priority Suggestions ::: 5.2. Suggestions ::: 5. Discussion', 'ref_spans': []}, {'text': 'The priority suggestions in this paper will be of great help to improve the attraction of B&Bs to tourists after Covid-19. Enough attention was not paid to these measures before the outbreak. They need to be focused on after the COVID-19 epidemic. B&Bs are very important for the tourism industry in many countries, and tourists have especially welcomed them in recent years in China. To the best of our knowledge, our study was among the first to investigate the immediate impact of the COVID-19 pandemic on tourist satisfaction with B&Bs in China. Many previous studies have reported on COVID-19. Some others studied the correlations between COVID-19 and the quality of life in China. However, few studies have reported the impact of B&B under COVID-19 on tourism in China. The adjusted importance (after COVID-19)–performance (before COVID-19) analysis (IPA) was a new attempt. Moreover, some promotion suggestions were given to the B&B industry recovery after COVID-19 by the IPA.', 'cite_spans': [], 'section': '6. Conclusions', 'ref_spans': []}, {'text': 'However, there were some limitations to our study and future research areas. First, the data were collected from tourists in B&Bs in Zhejiang, China. Thus, it was somewhat difficult to apply the suggestions of the impact of COVID-19 to other areas. Future researchers may expand this scope. Second, although we identified the relationships between the determinants of tourist satisfaction and COVID-19, the relative strength of these correlations was unknown. We can test the model and identify the degree of influence of the correlations between these factors to promote the B&B industry in further research. More nuanced research questions should be incorporated. Third, the current paper employed an IPA approach. Even though this method is a widely known method in the tourism industry, it was also a new attempt to use the IPA model with B&Bs. Thus, we suggest that researchers in other parts of China and in other continents work together to produce similar studies, thereby creating a worldwide body of literature that examines the phenomena related to the effects of crises (e.g., COVID-19) and their impact on B&Bs and tourism.', 'cite_spans': [], 'section': '6. Conclusions', 'ref_spans': []}], 'paragraphs': [{'text': 'Third, how long does it take to repair the impacts of an infectious disease on tourism? The development of the crisis’ events can be divided into three periods, according to the impact on tourist flow, including the incubation period, outbreak period, and recession period. The impact time of most crisis events is within one year; the impact period of a few events was around two years. Taking SARS as an example, the peak period of impact was from March to June 2003, and the entire impact period was about 1 year [12]. Taking the accommodation industry as an example, during 2003, single-store revenue of in high-star hotels declined significantly. B&B grew by 15.2% in 2003 and continued to grow, resulting in a 22% growth in 2004. Therefore, the impact of SARS on B&Bs in tourism was basically eliminated about half a year after the end of SARS [28].', 'id': '00010'}], 'title': 'The Impact of COVID-19 on Tourist Satisfaction with B&B in Zhejiang, China: An Importance–Performance Analysis', 'sha': '3195a095d79871b16e19787330234ed388387ed0', 'doi': '10.3390/ijerph17103747', 'score': 8.905400276184082}, {'rank': 253, 'paragraphs': [{'text': 'To the best of our knowledge, there is only a handful of studies estimating the incubation period of COVID-19. Among them are Li et al (2020), Guan et al (2020), Backer et al (2020), and Linton et al (2020).3–6 In Li et al, the first 425 lab-confirmed cases, reported as of January 22, 2020, were included in the study, while only ten cases could be identified with the exact dates of exposure.3 The distribution of the incubation period was subsequently approximated by fitting a lognormal distribution to these ten data points, resulting in a mean incubation period of 5·2 days (95% CI: 4·1–7·0), and the 95th percentile is 12·5 days. However, given the limited sample size, it is challenging to make a solid inference on the distribution of the incubation period. A different result was reported by Guan et al, based on 291 patients who had clear information regarding the specific date of exposure as of January 29, 2020, stating that the median incubation period was 4·0 days (interquartile range, 2 to 7).4 However, such study of the incubation period can be highly influenced by the individuals’ recall bias or interviewers’ judgement on the possible dates of exposure rather than the actual dates of exposure that, in turn, might not be accurately monitored and determined, thus leading to a high percentage of error. In Backer et al, 88 confirmed cases detected outside Wuhan were used to estimate the distribution of the incubation period.5 For each selected case, a censored interval for the incubation period can be obtained by travel history and symptoms onset. The distribution of the incubation period can then be estimated by fitting a Weibull, gamma, or lognormal distribution with censored data. However, this method contained two types of sampling biases: (1) with the longer incubation period, the patients who resided at Wuhan but developed symptoms outside Wuhan were easier to be observed and hence lead to an overestimation; (2) if the follow-up time (from infection to the end of the study) is short, only the shorter incubation period would be observed and hence lead to an underestimation. Linton et al proposed a similar approach to estimate the incubation period of Backer et al, but corrected the aforementioned second sampling bias.5,6 However, the first problem in regard to sampling bias is still an unsolved issue. The estimates of the incubation period from these four studies, together with other results of two other coronavirus disease, SARS and MERS, are listed in Table 1.', 'id': '00003'}], 'title': 'Estimation of incubation period distribution of COVID-19 using disease onset forward time: a novel cross-sectional and forward follow-up study', 'sha': 'c61be9fc4562d250e2dec241ef75bd22d61f61d1; 6d3b3f4ab80a61c45f82c61c6c756cfc6ddf4bf2', 'doi': '10.1101/2020.03.06.20032417', 'score': 8.904399871826172}, {'rank': 254, 'abstract': 'The SARS-CoV-2 driven infectious novel coronavirus disease (COVID-19) has been declared a pandemic by virtue of its brutal impact on the world in terms of loss on human life, health, economy, and other crucial resources. With the aim to explore more about its aspects, we adopted the SEIQRD (Susceptible-Exposed-Infected-Quarantine-Recovered-Death) pandemic spread with a time delay on the heterogeneous population and geography in this work. Focusing on the spatial heterogeneity, the entire population of interest in a region is divided into small distinct geographical sub regions, which interact by means of migration networks across boundaries. Utilizing the estimations of the time delay differential equations based model, we analyzed the spread dynamics of disease in a region and its sub regions. The model based numerical outcomes are validated from real time available data for India. We computed the approximate peak infection in forward time and relative timespan when disease outspread halts. To further evaluate the influence of the delay on the long term system dynamics, the sensitivity analysis is performed on time delay. The most crucial parameter, basic reproduction number R0 and its time-dependent generalization, has been estimated at both regional and sub regional levels. The impact of the most significant lockdown measure that has been implemented in India to contain the pandemic spread has been extensively studied by considering no lockdown scenario. A suggestion based on outcomes, for a bit relaxed lockdown, followed by an extended period of strict social distancing as one of the most effective control measures to manage COVID-19 spread is provided for India.', 'body_text': [{'text': 'The SARS-CoV-2 driven infectious novel coronavirus disease has been declared a pandemic by virtue of its brutal impact on the world in terms of loss on human life, health, economy, and other crucial resources. With the aim to explore more about its aspects, we adopted the SEIQRD (Susceptible-Exposed-Infected-Quarantine-Recovered-Death) pandemic spread with a time delay on the heterogeneous population and geography in this work. Focusing on the spatial heterogeneity, the entire population of interest in a region is divided into small distinct geographical sub regions, which interact by means of migration networks across boundaries. Utilizing the estimations of the time delay differential equations based model, we analyzed the spread dynamics of disease in a region and its sub regions. The model based numerical outcomes are validated from real time available data for India. We computed the approximate peak infection in forward time and relative timespan when disease outspread halts. To further evaluate the influence of the delay on the long term system dynamics, the sensitivity analysis is performed on time delay. The most crucial parameter, basic reproduction number R0 and its time-dependent generalization, has been estimated at both regional and sub regional levels. The impact of the most significant lockdown measure that has been implemented in India to contain the pandemic spread has been extensively studied by considering no lockdown scenario. A suggestion based on outcomes, for a bit relaxed lockdown, followed by an extended period of strict social distancing as one of the most effective control measures to manage COVID-19 spread is provided for India.', 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': 'The novel coronavirus disease, officially known as 2019-nCoV, or SARS-CoV-2 (severe acute respiratory syndrome coronavirus 2), commonly called COVID-19, has provided the world with an unparallel challenge. The first reported case in the COVID-19 outbreak appeared in the Wuhan city of China in December 2019, and since then, it has rapidly spread in 210 countries and territories around the world [1]. As of April 30, 2020, a total of 3,275,475 confirmed cases of the coronavirus and a toll of 231,576 deaths has been reported globally [2] . At this moment of time, there is no definite vaccine or treatment for COVID-19. It has thrown an unpredictable burden on healthcare systems in the majority of the developed as well as developing countries. With its breakneck spread all across, World Health Organization(WHO) has declared it as a pandemic and to contain the upsurge by breaking its person to person transmission chain involving significant human migration, major steps including partial or complete lockdowns, international travel bans, and social distancing has been widely adopted.', 'cite_spans': [{'start': 536, 'end': 539, 'text': '[2]', 'ref_id': 'BIBREF0'}], 'ref_spans': [], 'section': 'I. INTRODUCTION'}, {'text': 'In India, the first case of coronavirus pandemic was identified on January 30, 2020, originating from China. Since this appearance, there has been a continuous rise in the number of infections with the total number of 33,610 cases on April 30, 2020, of which there are 8,373 recoveries and 1,075 deaths [3] . As a preventive measure, India also implemented a strict lockdown along with curfew in certain states, limiting movement of the entire pop- * akgupta@iitrpr.ac.in ulation to minimize social contacts. The efficiency of such measures is geographically sensitive due to different population densities, social contact networks, and health care facilities. Though these measures are important for controlling the virus, they have an extensive burden on the global economy as well. Quantitative projection of the effect of these measures in reducing infection is crucial in outlining social and economic policies. Further, understanding the transmission dynamics of COVID-19 not only provides deep insights into the epidemiological situation to enhance public-health planning, but such investigations can also aid in the layout of alternative outbreak control measures.', 'cite_spans': [{'start': 303, 'end': 306, 'text': '[3]', 'ref_id': 'BIBREF1'}], 'ref_spans': [], 'section': 'I. INTRODUCTION'}, {'text': 'Knowledge of the early spread dynamics of the infection and figuring out the capability and performance of applied control measures is critical for assessing the extensive outspread to occur in new regions. Besides medical and biological research, understanding the urgency to develop a predictable mathematical model for the COVID-19 outbreak, few mathematical studies based on statistical reasoning and simulations have been taken up in past months [4] [5] [6] . Later, accepting the challenges to explore the efficiency of various control measures since the outbreak, fewer studies adopted more appropriate dynamical equations based on mathematical modeling techniques. Compartment models such as SI, SIR, SEIR which exhibits the change in the category of population among the susceptible (S), exposed (E), infected (I) and recovered (R) classes have been developed and studied to understand the spread pattern of COVID-19 in many countries [7] [8] [9] [10] [11] .', 'cite_spans': [{'start': 451, 'end': 454, 'text': '[4]', 'ref_id': None}, {'start': 455, 'end': 458, 'text': '[5]', 'ref_id': None}, {'start': 459, 'end': 462, 'text': '[6]', 'ref_id': None}, {'start': 944, 'end': 947, 'text': '[7]', 'ref_id': None}, {'start': 948, 'end': 951, 'text': '[8]', 'ref_id': None}, {'start': 952, 'end': 955, 'text': '[9]', 'ref_id': 'BIBREF7'}, {'start': 956, 'end': 960, 'text': '[10]', 'ref_id': 'BIBREF8'}, {'start': 961, 'end': 965, 'text': '[11]', 'ref_id': None}], 'ref_spans': [], 'section': 'I. INTRODUCTION'}, {'text': 'Further, as reported, COVID-19 has a latent/incubation period (time from exposure to the development of symptoms), which is estimated to be between 2 and 14 days [12] . This incubation period is very significant as it allows the health authorities to introduce more adequate systems for separating people suspected of carrying the virus, as a way of controlling and preventing the spread of the pandemic [13] . The inclusion of the latent period in compartment models gives rise to COVID-19 time delay models in which the dynamic behavior of the disease at time t depends on the state of the system at a time prior to the delay period [14, 15] . However, these studies assume homogeneity in a large population and ignore numerical variations originating from natural births, deaths, and human migration networks across the regions. To the amount that population and geographic heterogeneities play crucial roles in the infection outbreak process, in order to pick up the vital characteristics of the pandemic, it is advantageous to include them in the time delay model. Capturing the critical impact of the delay period of COVID-19, this study presents a mathematical model to analyze the spread of the novel coronavirus within a heterogeneous population and geography. We propose a more realistic time delay differential equations based SEIQRD model by incorporating quarantined (Q)-confirmed and separated infected population as well as death due to infection class (D) along with the optimal values for model parameters, which reasonably fit the actual infection cases of the pandemic. Spatial and stochastic parameters like heterogeneous population, natural births, deaths, and population migration networks within a geographical state (subpopulation in a country) boundaries have been considered. Predominantly, in the presented model, considering the spatial heterogeneity, the entire population of the broad geographical region of interest is divided into small sub regions-states, which interact by means of migrations across the state boundaries. Moreover, there are many inevitable questions related to the spread of the pandemic. How many people are at risk of infection ? When will be the infection at its peak and when will it end ? Are the existing control measures sufficient to control the outbreak ? With the aim to explore more about the transmission of COVID-19 dynamics and to predict its potential tendency, the COVID-19 pandemic is estimated for forward time. The general points of interest including total infected cases, infection peak, pandemic ending time are investigated. The most vital parameter, basic reproduction number R 0 and its time-dependent generalization that decides if an originating infectious disease can extend in a population, has been estimated at both regional and its sub regional levels. Additionally, the influence of the time delay on the long term pandemic dynamics has been explored. Further, we use the model to examine the impact of the most significant lockdown measure that has been implemented to contain the pandemic spread.', 'cite_spans': [{'start': 162, 'end': 166, 'text': '[12]', 'ref_id': 'BIBREF10'}, {'start': 404, 'end': 408, 'text': '[13]', 'ref_id': None}, {'start': 635, 'end': 639, 'text': '[14,', 'ref_id': 'BIBREF12'}, {'start': 640, 'end': 643, 'text': '15]', 'ref_id': None}], 'ref_spans': [], 'section': 'I. INTRODUCTION'}, {'text': 'A. Generalized SEIQRD model with time delay', 'cite_spans': [], 'ref_spans': [], 'section': 'II. MATERIAL AND METHOD'}, {'text': 'To estimate the trajectories of COVID-19 transmission, in the proposed approach, a vast geographic region with heterogeneous population distribution (country) is partitioned into n smaller sub-regions (states). To take into account the effect of both inter and intrastate infectious population, migration for these states are explicitly incorporated ( Fig. 1(a) ), using population migration adjacency matrix λ (with order n × n). The diagonal coefficients of this matrix represent the migration rate within a state while for any of its j th row, k th element indicates the migration factor from the j th state to the k th state. For any two states which share no boundaries migration rate is 0. Within a given state, the transmission is handled according to a deterministic compartmental model dividing the population into six epidemiological classes S(t), E(t), I(t), Q(t), R(t), D(t) describing at time t the respective proportion of the susceptible cases, exposed cases (infected but not capable of transmitting infection, in a latent period), infected cases (with the infection spreading capacity and not yet separated from those who are not infected), quarantined cases (confirmed and separated infected), recovered cases and deaths owing to virus cases. Fig. 1(b) shows the movement within the classes in model with time delay. Since the latent period (τ ) of the COVID-19 is as long as 2 to 14 days [1], the exposed class (E) originates because of interactions with the amount β within susceptible (S) and infected class (I). Only those members of the exposed class who survive the latency period move to the infected class with a rate of σ. The infected population tested positive for COVID-19 moves to quarantine class (Q) for further treatment with parameter . After a course of quarantine treatment, either a part of its population is discharged with the rate δ from the hospital (R) or encounter death due to underline diseases (D) with an estimate γ. An appendix (A) provides complete details of our mathematical model characterized by a group of ordinary differential equations depicting the outbreak at time t. The primary model parameters, reasonably fitting the actual infected cases till date, are described in Table I in the appendix (C).', 'cite_spans': [], 'ref_spans': [{'start': 352, 'end': 361, 'text': 'Fig. 1(a)', 'ref_id': None}, {'start': 1261, 'end': 1270, 'text': 'Fig. 1(b)', 'ref_id': None}, {'start': 2230, 'end': 2237, 'text': 'Table I', 'ref_id': 'TABREF4'}], 'section': 'II. MATERIAL AND METHOD'}, {'text': 'The epidemic bulletin from the World Health Organization provides us with primary data on epidemiological research. According to the reports by WHO, the incubation period for COVID-19, which is the time between exposure to the virus (becoming infected) and symptom onset, is, on average 7 days; however, it can be up to 14 days [1]. The average mortality rate based on historical data is about 3.4% [2, 11] .', 'cite_spans': [{'start': 399, 'end': 402, 'text': '[2,', 'ref_id': 'BIBREF0'}, {'start': 403, 'end': 406, 'text': '11]', 'ref_id': None}], 'ref_spans': [], 'section': 'B. Data Source'}, {'text': 'To check how optimal is the fitting between the actual official data for the pandemic spread and the model based obtained numerical approximations, we use the statistical Kolmogorov-Smirnov (K-S) test [16, 17] . The K-S goodness-of-fit test is performed on the data to ascertain whether two data series are from the same continuous distribution or not. The null hypothesis implies that data samples are from the same continuous distribution, and the alternative hypothesis states the opposite. The statistic h is 1 if the test rejects the null hypothesis at the 5% level of significance, and 0 otherwise. For the proposed model with retained parameters, the results indicate (with h = 0) that both the actual and the projected number of cases are from the same continuous distribution, as desired, showing optimistic estimations.', 'cite_spans': [{'start': 201, 'end': 205, 'text': '[16,', 'ref_id': None}, {'start': 206, 'end': 209, 'text': '17]', 'ref_id': None}], 'ref_spans': [], 'section': 'B. Data Source'}, {'text': 'In this section, we conduct a more detailed model analysis to depict outspread for the COVID-19 pandemic. We obtained the numerical solution of the mathematical model discussed in the appendix (A) based on the chosen set of parameters for India. Using suitably preferred parameters, the same detailing can be obtained for any geographic region. Moving further, we aim at forecasting the infection cases along with the basic reproduction number. Moreover, we use the model to examine the impact of the most significant lockdown measure that has All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'III. RESULTS'}, {'text': 'was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'III. RESULTS'}, {'text': 'The copyright holder for this preprint (which this version posted May 8, 2020. been implemented in India to contain the pandemic. It is significant to mention that, at present, due to lack of sufficient diagnostic test for COVID-19, the total number of infected cases (I + Q) in any j th state, can be given by (I(t)+Q(t))αN j . Here, N j denotes the population of the j th state, and α is considered approximately as 0.1 in the role of the current testing events [18] . Clearly, the total infected cases in a country with n states can be obtained by the sum of corresponding data from the states. At the onset, we used the proposed model to fit actual available COVID-19 infection cases for India up to April 30, 2020 (Fig. 2) . Notably, since India went into a strict lockdown that commenced on March 25, 2020, officially published data by Ministry of Health and Family Welfare (MoHFW), Government of India from March 16, 2020 to April 30, 2020 are marked in pink spots and are considered as a direct validation source. As expected, we can clearly see that the prediction of the exact number of cases diagnosed in the past time by the model is in reasonable agreement with the real value.', 'cite_spans': [{'start': 464, 'end': 468, 'text': '[18]', 'ref_id': 'BIBREF16'}], 'ref_spans': [{'start': 719, 'end': 727, 'text': '(Fig. 2)', 'ref_id': 'FIGREF1'}], 'section': 'III. RESULTS'}, {'text': 'With an aim to estimate the potential tendency of the COVID-19 pandemic and to examine the impact of the All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'III. RESULTS'}, {'text': 'was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'III. RESULTS'}, {'text': 'The copyright holder for this preprint (which this version posted May 8, 2020. . https://doi.org/10.1101/2020.05.06.20092858 doi: medRxiv preprint lockdown control measure we compared the infection spread in the cases of current strict lockdown and no lockdown in place as under:', 'cite_spans': [], 'ref_spans': [], 'section': 'III. RESULTS'}, {'text': 'Nationwide strict Lockdown: With parameters in hand, we executed the model forward in time to visualize the progress of the pandemic in the present situation of strict lockdown. Fig. 3 shows the trajectory for the total number of infected cases in India (Infectious + Quarantined) in the coming months. Fig. 3(a) displays a five month forecast in the present scenario of nationwide lockdown as a strategic move to control the pandemic spread. With strict lockdown measures currently in place, since all transport services: road, air, and rail were suspended, we set the interstate population migration index to null in the population migration adjacency matrix λ, and the average contacts between susceptible and infected are considered to be proximately three. Based on these criteria, our model featured a single infection peak on June 30, 2020 with the total number of estimated infected to be around 5.2 million. The outbreak is expected to be nearing its end by late August, 2020.', 'cite_spans': [], 'ref_spans': [{'start': 178, 'end': 184, 'text': 'Fig. 3', 'ref_id': None}, {'start': 303, 'end': 312, 'text': 'Fig. 3(a)', 'ref_id': None}], 'section': 'III. RESULTS'}, {'text': 'No Lockdown: To highlight the effectiveness of the ongoing national lockdown, in comparison, Fig.  3(b) shows infection forecast if in case lockdown was not implemented. In this situation, we allowed normal migration within the states and across the boundaries by setting non zero estimates for diagonal and off diagonal entries in the migration adjacency matrix. However, assuming that some form of control measure would continue to be in the system to reduce social contact, we choose average contacts between susceptible and infected to be approximately nine [18] . As anticipated, if interventions would not have been there, India might have experienced 18 times worst the numbers with 40 million infections by mid of October, 2020 with a peak of approximately 90 million cases around April, 2021. There still would have been around 5, 00, 000 infectious cases revealed at the end of August, 2021 as exhibited by Fig. 4(b) .', 'cite_spans': [{'start': 562, 'end': 566, 'text': '[18]', 'ref_id': 'BIBREF16'}], 'ref_spans': [{'start': 93, 'end': 103, 'text': 'Fig.  3(b)', 'ref_id': None}, {'start': 917, 'end': 926, 'text': 'Fig. 4(b)', 'ref_id': None}], 'section': 'III. RESULTS'}, {'text': 'Proposed Scenario: In the third scenario, Fig.  3 (c) panel exhibits a rundown for a proposed protocol with a bit relaxed lockdown, allowing restricted movements within and across the state boundaries beyond May 3, 2020. This relaxation assumes gradual freedom of movement for essential activities only, thereby ensuring the upliftment of lockdown in stages to save the economy. Moreover, considering self awareness within the population, we invoked reduced average contact parameter β between susceptible and infected to be approximately one, through social distancing impacts in this case. This alteration was required as a step to break or lessen human to human transmission chain of the virus. As likely, this consideration brings the total number of infections to comparatively lower values and slows the spread of the disease to manageable levels. Moreover, peak infections in the proposed scenario decrease significantly in comparison to a strict lockdown situation. This observation clearly highlights the fact that social distancing is one of several changes that we have to adapt strictly in the coming weeks. To ensure the distances are well maintained at work or public places, more mature ways of monitoring, including artificial intelligence or machine learning software, can be used. As depicted in Fig. 3(c) , as a suggestion, if social distancing is enforced broadly and is maintained in the coming months, the number of new infections would decrease to a significantly submissive level, and the outbreak could eventually be controlled. Additionally, in order to further evaluate the influence of the delay on the long term disease dynamics, we performed the sensitivity analysis on τ keeping other parameters fixed. We plotted the total infected cases and infectious cases as a function of time for systematically varied values of τ ∈ [6, 12] shown by shaded regions in Fig. 2, Fig. 3 and Fig. 4 . Apparently, a small change in delay has no qualitative impact on the nature of the spread, apart from shifting of the curve, thereby preserving the basic shape of the infection pattern. Shifts in infection peak with varied τ provide us with a possible domain window range for estimation of infection cases owing to regional differences in India on account of health, education, religion, and per capita income.', 'cite_spans': [{'start': 1853, 'end': 1856, 'text': '[6,', 'ref_id': None}, {'start': 1857, 'end': 1860, 'text': '12]', 'ref_id': 'BIBREF10'}], 'ref_spans': [{'start': 42, 'end': 49, 'text': 'Fig.  3', 'ref_id': None}, {'start': 1314, 'end': 1323, 'text': 'Fig. 3(c)', 'ref_id': None}, {'start': 1888, 'end': 1902, 'text': 'Fig. 2, Fig. 3', 'ref_id': 'FIGREF1'}, {'start': 1907, 'end': 1913, 'text': 'Fig. 4', 'ref_id': None}], 'section': 'III. RESULTS'}, {'text': 'In order to capture R 0 for heterogeneous population distribution in India, we computed its value using the proposed model by the relation given by Eq.(7) in the appendix (B). The estimated basic reproduction number for India in the present scenario of nationwide lockdown turns out to be 1.18. Based on this value of R 0 , as expected, from the transmission dynamics of the infection in a strick lockdown situation, shown in Fig. 4(a) , the number first increases before tending to zero, thereby R 0 = 1 acts as a clear edge between the disease widely spreading or dying out. It is worth mentioning here that in case of a relaxed lockdown with average contact rate between susceptible and infected as one, R 0 turns out to be 0.8739, which is much desired for outspread levels to reduce further to a manageable extent. This can be clearly verified from Fig. 4(c) , where the number of infectious decreases monotonically to zero. Additionally, Fig.  5 shows the time-dependent effective basic reproductive number R ef f 0 (t) corresponding to the infection trajectories in Fig. 4(a) . As hoped for, this value is greater than unity before peak infection and smaller than unity beyond its peak, serving as a useful estimate of the local rate of change of infectives at any time. It is noteworthy to mention here that for any state of India, R 0 can be obtained based on the relation given by Eq. (8) in the appendix (B).', 'cite_spans': [{'start': 1395, 'end': 1398, 'text': '(8)', 'ref_id': None}], 'ref_spans': [{'start': 426, 'end': 435, 'text': 'Fig. 4(a)', 'ref_id': None}, {'start': 854, 'end': 863, 'text': 'Fig. 4(c)', 'ref_id': None}, {'start': 944, 'end': 951, 'text': 'Fig.  5', 'ref_id': 'FIGREF3'}, {'start': 1073, 'end': 1082, 'text': 'Fig. 4(a)', 'ref_id': None}], 'section': 'III. RESULTS'}, {'text': 'The model discussed here, could be helpful to health authorities for depicting the total number of infection cases along with the peak infection. The estimated fit values could be made better on a daily base as more data become available.', 'cite_spans': [], 'ref_spans': [], 'section': 'III. RESULTS'}, {'text': 'We have presented a generalized SEIQRD mathematical model with a time delay to analyze the spread of COVID-19 infection in a population. The proposed model incorporates spatial and stochastic parameters like heterogeneous population, natural births, deaths, and population migration networks within geographical state boundaries. Based on a detailed analysis of the available public data, we projected COVID-19 pandemic peaks and possible ending time and total infected cases in India. Overall, the current pandemic situation with nationwide lockdown is expected to end up by August, 2020, which is a much better scenario than no lockdown in place. Furthermore, we applied our mathematical model to interpret the public data on the total number of infected cases from March, 16, 2020 onwards in two sub regions of India, including Maharastra and Uttar Pradesh, as discussed in the appendix (D). With lockdown in place, the peak infection witnessed a significant fall in comparison to the situation if there would have been no lockdown indicating the potential benefit of lockdown in controlling the outspread. Our model suggests a bit relaxed lockdown, followed by an extended period of strict social distancing as one of the most effective control measures to manage COVID-19 spread in days to come.', 'cite_spans': [], 'ref_spans': [], 'section': 'IV. DISCUSSION'}, {'text': 'Further, capturing the impact of the delay on the pan-demic transmission, we carried the sensitivity analysis on time delay by varying its values systematically. No qualitative changes have been observed in the infection pattern apart from shifting of the curve resulting in possible domain range for estimated infection.', 'cite_spans': [], 'ref_spans': [], 'section': 'IV. DISCUSSION'}, {'text': 'Owing to significant spatial variations affecting the basic reproduction number R 0 , we included spatial heterogeneity by dividing the entire population into smaller sub regions. Based on the technique, R 0 and its timedependent generalization has been estimated at both regional and sub regional levels. The approach helped us to capture sensible COVID-19 infection dynamics over local and global levels.', 'cite_spans': [], 'ref_spans': [], 'section': 'IV. DISCUSSION'}, {'text': 'The proposed model is an attempt to provide deep insight to analyze the dynamics of COVID-19, helping the involved agencies to arrange and manage crucial resources and design its control strategies. Additionally, the proposed work necessarily made some assumptions when framing the model. We ignore the impact of infection spread due to exposed class [1]. Moreover, the work is based on acquired data for a limited duration of time to fit and estimate the spread of COVID-19. With the release of more epidemic data for India, owing to the regional differences on account of health, education, religion, and per capita income, the key parameters may undergo momentous changes influencing the spread of pandemic among the masses. We consider a large geographic region with total population N , partitioned into n states labeled by j = 1, 2, ...n. The population within the j th state is partitioned into susceptibles (S j ), exposures (E j ), infectives (I j ), quarantines (Q j ), recoveries (R j ) and deaths (D j ), where N j =S j +E j +I j +Q j +R j +D j and N = n j=1 N j . The proposed time delay SEIQRD system that captures population and geographic heterogeneities is given by:', 'cite_spans': [], 'ref_spans': [], 'section': 'IV. DISCUSSION'}, {'text': 'All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'IV. DISCUSSION'}, {'text': 'was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. ', 'cite_spans': [], 'ref_spans': [], 'section': 'IV. DISCUSSION'}, {'text': 'Here β j > 0 is the average rate of contact between susceptible and infected (people exposed at each time step by infected people) in j th state; λ j,k ∈ [0, 1] denotes the population migration factor for the states j and k. The parameter σ ∈ [0, 1] stands for the portion of exposed individuals, which become infectious per time step and ∈ [0, 1] is the quarantine rate from the infection per time step. We call δ ∈ [0, 1] the proportion of quarantined people who are cured per time step and γ ∈ [0, 1] is the fatality rate due to COVID-19. The parameter τ > 0 is the delay time. Further, without affecting the behavior and general aspects of infection, it is assumed that the population in the j th state is born susceptible with the natural birth rate µ > 0 per one individual per time step and the total population remains constant since the natural death rate is considered to be same as the natural birth rate µ.', 'cite_spans': [], 'ref_spans': [], 'section': 'IV. DISCUSSION'}, {'text': 'The first term on the R.H.S. of Eq.(1) represents the fragment of the susceptible individual of j th state who have been exposed to the disease by the infected individuals of the same state and by the infected individuals of the other states, who have moved to the former main state, taking into account the average contact rate; migration coefficients for the states; and the relation between the populations of the states.', 'cite_spans': [], 'ref_spans': [], 'section': 'IV. DISCUSSION'}, {'text': 'Those who were exposed to the infection at time t − τ and survive the latent period [t − τ, t] with probability e −µτ moves to the infected class at time t [19, 20] . Moreover, the proportion removed by disease independent mortality for each compartment is given by the last term on R.H.S of each equation. From Eqs.(1) -(6), we get the normality condition S j (t)+E j (t)+I j (t)+Q j (t)+ R j (t) + D j (t) = 1 , for every state j at any time t.', 'cite_spans': [{'start': 156, 'end': 160, 'text': '[19,', 'ref_id': None}, {'start': 161, 'end': 164, 'text': '20]', 'ref_id': None}], 'ref_spans': [], 'section': 'IV. DISCUSSION'}, {'text': 'Following the approaches similar to those taken in [13, 21, 22 ] with respect to system in Eqs.(1) - (6) , it can be verified that the system always has the diseasefree equilibrium P 0 = (S 0 1 , 0, 0, 0, 0, 0, ..., S 0 n , 0, 0, 0, 0, 0) where S 0 j = 1 is the equilibrium in the j th state in the absence of infection, that is I 1 = I 2 = ... = I n = 0. Further, let R 0 = ρ(M 0 ) represents the spectral radius of the matrix M 0 with entries m 0 (j, k) give by', 'cite_spans': [{'start': 51, 'end': 55, 'text': '[13,', 'ref_id': None}, {'start': 56, 'end': 59, 'text': '21,', 'ref_id': None}, {'start': 60, 'end': 62, 'text': '22', 'ref_id': None}, {'start': 101, 'end': 104, 'text': '(6)', 'ref_id': None}], 'ref_spans': [], 'section': 'B. Computation of Reproductive number:'}, {'text': 'where 1 ≤ k, j ≤ n, then the parameter R 0 is referred to as the basic reproduction number of the considered geographic region. This number is intended to be an indicator of the transmissibility of COVID-19, the outbreak is expected to continue if R 0 > 1 and to end if R 0 < 1.', 'cite_spans': [], 'ref_spans': [], 'section': 'B. Computation of Reproductive number:'}, {'text': 'Eqs. (7) shows that the basic reproductive number significantly depends on the average rate of contact between susceptible and infected, population migration factor for the states, state population and time delay. Also, the time-dependent effective basic reproductive number, R ef f 0 (t), is taken to be R ef f 0 (t) = 1 δt(µ+ ) log I(t+δt)', 'cite_spans': [], 'ref_spans': [], 'section': 'B. Computation of Reproductive number:'}, {'text': '. Further, for the j th state, the basic reproduction number is given by', 'cite_spans': [], 'ref_spans': [], 'section': 'B. Computation of Reproductive number:'}, {'text': 'C. Model Parameters:', 'cite_spans': [], 'ref_spans': [], 'section': 'B. Computation of Reproductive number:'}, {'text': 'The first case of the 2019 − 20 coronavirus pandemic in India was reported on January 30, 2020. We gathered the dataset for COVID-19 in India from the Ministry of Health and Family Welfare (MoHFW), Government of India from March 16, 2020 to April 30, 2020 including the cumulative number of infected cases, the cumulative number of people in recovery and the cumulative number of deaths due to virus [3] . Simultaneously, we collected the state wise total population data for India (projected 2019) from Unique Identification Authority of India (UIDAI), Government of India and migration data to capture the movement of population in different parts of the country from Ministry of Home Affairs, Government of India [23, 24] . State wise population for India is summarized in Table II in order of states as appeared in the map of India in Fig. 1 . Therefore, the preliminary estimated parameters that reflect the primary situation of the pandemic in India at the present stage can be summarized as: All rights reserved. No reuse allowed without permission.', 'cite_spans': [{'start': 400, 'end': 403, 'text': '[3]', 'ref_id': 'BIBREF1'}, {'start': 716, 'end': 720, 'text': '[23,', 'ref_id': 'BIBREF21'}, {'start': 721, 'end': 724, 'text': '24]', 'ref_id': 'BIBREF22'}], 'ref_spans': [{'start': 776, 'end': 784, 'text': 'Table II', 'ref_id': 'TABREF4'}, {'start': 839, 'end': 845, 'text': 'Fig. 1', 'ref_id': None}], 'section': 'B. Computation of Reproductive number:'}, {'text': 'was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. ) in Maharastra, and Uttar Pradesh, respectively with respect to time (in days) in case of strict (green) and no lockdown (red). Blue marker represents the real data. The parameters for (a) and green curve in (b) are: (intrastate population migration factor, interstate population migration factor, β, τ ) = (0.41, 0, 3.4, 7) while in case of no lockdown (red) the chosen values are (0.8, 0. 6, 9, 7) . In (c) and green curve in (d) these values are (0.393, 0, 3, 8.2) while in case of no lockdown (red) the chosen values are (0.8, 0. 6, 9, 8.2) . µ = 0.0412, δ = 0.22, σ = 1/7, = 0.85, and γ = 0.034 are same in (a-d). In (b) and (d), 29 July, and 5 April belong to 2021, and 2022, respectively.', 'cite_spans': [{'start': 521, 'end': 523, 'text': '6,', 'ref_id': None}, {'start': 524, 'end': 526, 'text': '9,', 'ref_id': 'BIBREF7'}, {'start': 527, 'end': 529, 'text': '7)', 'ref_id': None}, {'start': 664, 'end': 674, 'text': '6, 9, 8.2)', 'ref_id': None}], 'ref_spans': [{'start': 129, 'end': 130, 'text': ')', 'ref_id': None}], 'section': 'B. Computation of Reproductive number:'}, {'text': 'We apply our pre-described mathematical model to interpret the public data on the total number of infected cases from March 16, 2020 onwards in sub regions of India, which are published daily by Ministry of Health and Family Welfare (MoHFW), Government of India. Our analysis includes two different regions (states), that is, Maharastra and Uttar Pradesh. It is significant to mention here that we have chosen Maharashtra since it has the highest number of infected cases as of April 30, 2020. Further, to analyze how accurately our model incorporates population heterogeneity, we have chosen Uttar Pradesh since it is the most populated state of India. In Fig.  6 (a) and Fig. 6(c) , the estimated number of total infec-ted cases is plotted for both the states by solid lines in case of nationwide strict lockdown scenario and actual COVID-19 infection data from March 16, 2020 to April 30, 2020 are marked by circles. Here also the reasonable agreement is observed between actual and forecasted data. We then run the proposed model for chosen parameters for the next 24 months and performed detailed analysis on similar lines for nationwide strict lockdown and no lockdown scenario, as discussed in Section III in Fig. 6(b) and Fig. 6(d) . From the numerically obtained outcomes in forward time, with the implementation of lockdown, the peak infection witnessed a significant fall in comparison to the situation if there would have been no lockdown indicating the potential benefit of lockdown in controlling the outspread. Further, as All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [{'start': 657, 'end': 664, 'text': 'Fig.  6', 'ref_id': 'FIGREF5'}, {'start': 673, 'end': 682, 'text': 'Fig. 6(c)', 'ref_id': 'FIGREF5'}, {'start': 1216, 'end': 1225, 'text': 'Fig. 6(b)', 'ref_id': 'FIGREF5'}, {'start': 1230, 'end': 1239, 'text': 'Fig. 6(d)', 'ref_id': 'FIGREF5'}], 'section': 'D. Impact on Indian States:'}, {'text': 'was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. β 1-13 [18] Entries of λ 0.1 − 0.8 [24] suggested, the situation improves much better if social distancing is enforced strictly along with relaxed lockdown. Using suitably selected parameters, the pandemic transmission patterns can be obtained for any other state of India on similar lines. [23] , in order of states as appeared in map of India in Fig. 1. ', 'cite_spans': [{'start': 136, 'end': 140, 'text': '[18]', 'ref_id': 'BIBREF16'}, {'start': 164, 'end': 168, 'text': '[24]', 'ref_id': 'BIBREF22'}, {'start': 420, 'end': 424, 'text': '[23]', 'ref_id': 'BIBREF21'}], 'ref_spans': [{'start': 477, 'end': 484, 'text': 'Fig. 1.', 'ref_id': None}], 'section': 'D. Impact on Indian States:'}], 'paragraphs': [{'text': 'Further, as reported, COVID-19 has a latent/incubation period (time from exposure to the development of symptoms), which is estimated to be between 2 and 14 days [12] . This incubation period is very significant as it allows the health authorities to introduce more adequate systems for separating people suspected of carrying the virus, as a way of controlling and preventing the spread of the pandemic [13] . The inclusion of the latent period in compartment models gives rise to COVID-19 time delay models in which the dynamic behavior of the disease at time t depends on the state of the system at a time prior to the delay period [14, 15] . However, these studies assume homogeneity in a large population and ignore numerical variations originating from natural births, deaths, and human migration networks across the regions. To the amount that population and geographic heterogeneities play crucial roles in the infection outbreak process, in order to pick up the vital characteristics of the pandemic, it is advantageous to include them in the time delay model. Capturing the critical impact of the delay period of COVID-19, this study presents a mathematical model to analyze the spread of the novel coronavirus within a heterogeneous population and geography. We propose a more realistic time delay differential equations based SEIQRD model by incorporating quarantined (Q)-confirmed and separated infected population as well as death due to infection class (D) along with the optimal values for model parameters, which reasonably fit the actual infection cases of the pandemic. Spatial and stochastic parameters like heterogeneous population, natural births, deaths, and population migration networks within a geographical state (subpopulation in a country) boundaries have been considered. Predominantly, in the presented model, considering the spatial heterogeneity, the entire population of the broad geographical region of interest is divided into small sub regions-states, which interact by means of migrations across the state boundaries. Moreover, there are many inevitable questions related to the spread of the pandemic. How many people are at risk of infection ? When will be the infection at its peak and when will it end ? Are the existing control measures sufficient to control the outbreak ? With the aim to explore more about the transmission of COVID-19 dynamics and to predict its potential tendency, the COVID-19 pandemic is estimated for forward time. The general points of interest including total infected cases, infection peak, pandemic ending time are investigated. The most vital parameter, basic reproduction number R 0 and its time-dependent generalization that decides if an originating infectious disease can extend in a population, has been estimated at both regional and its sub regional levels. Additionally, the influence of the time delay on the long term pandemic dynamics has been explored. Further, we use the model to examine the impact of the most significant lockdown measure that has been implemented to contain the pandemic spread.', 'id': '00005'}], 'title': 'Spatial Network based model forecasting transmission and control of COVID-19', 'sha': 'a3d841a1d812255ca308d2324857ba77ca645b78', 'doi': '10.1101/2020.05.06.20092858', 'score': 8.897000312805176}, {'rank': 255, 'abstract': 'In this work, we assess the global impact of COVID-19 showing how demographic factors, testing policies and herd immunity are key for saving lives. We extend a standard epidemiological SEIR model in order to: (a) identify the role of demographics (population size and population age distribution) on COVID-19 fatality rates; (b) quantify the maximum number of lives that can be saved according to different testing strategies, different levels of herd immunity, and specific population characteristics; and (d) infer from the observed case fatality rates (CFR) what the true fatality rate might be. Different from previous SEIR model extensions, we implement a Bayesian Melding method in our calibration strategy which enables us to account for data limitation on the total number of deaths. We derive a distribution of the set of parameters that best replicate the observed evolution of deaths by using information from both the model and the data.', 'body_text': [{'text': 'As the prospects for the development of a COVID-19/ SARS-CoV-2 vaccine are not in the near future, countries will have to manage the dramatic consequences of long-term lockdowns and quarantine regimes, which are the most employed non-pharmaceutical public health interventions (NPIs) that control the infection (1) (2) (3) . While for SARS-CoV-1 the peak viral shedding occurred 5 after patients were very ill with respiratory symptoms that could easily be traced, identified and isolated (1), viral shedding for COVID-19 is long and observed in asymptomatic individuals (3) making it difficult to control human-to-human transmission, with many countries only realizing the spread of the disease when they were already seriously hit (4) . With the rapid undetected spread and lack of fast response or testing availability, most countries, with few exceptions 1 , relied heavily 10 on widespread social distance measures and lockdowns, while restricting testing to symptomatic individuals. These measures are effective to control the infection, but have important socioeconomic, psychological and health consequences, and unless herd immunity considerably develops, most likely a second wave (or waves) of infection will arise as people slowly return to their activities (5) . In addition, due to the major role of infectious asymptomatic individuals, 15 restricting testing to the symptomatic makes it difficult to estimate the true case fatality rates and appropriately measure the death toll, since the less tests you make, the lower the number of detected persons and the higher the case fatality rates (6) .', 'cite_spans': [{'start': 311, 'end': 314, 'text': '(1)', 'ref_id': 'BIBREF0'}, {'start': 315, 'end': 318, 'text': '(2)', 'ref_id': 'BIBREF15'}, {'start': 319, 'end': 322, 'text': '(3)', 'ref_id': 'BIBREF16'}, {'start': 571, 'end': 574, 'text': '(3)', 'ref_id': 'BIBREF16'}, {'start': 733, 'end': 736, 'text': '(4)', 'ref_id': 'BIBREF17'}, {'start': 1270, 'end': 1273, 'text': '(5)', 'ref_id': 'BIBREF18'}, {'start': 1606, 'end': 1609, 'text': '(6)', 'ref_id': 'BIBREF19'}], 'ref_spans': [], 'section': 'Main Text:'}, {'text': 'These factors make these strategies unsustainable in the long-run and both country endurance to social distance measures and their effect on fatality rates will most likely depend on socioeconomic 20 settings (7), detecting asymptomatic individuals (8) , undocumented cases (9) , demographic characteristics (10) , and the level of herd immunity (11) . We account in a combined fashion for those last three factors by extending a standard epidemiological (Susceptible-Exposed-Infected-Removed) compartment model by (a) accounting for the age structure of the population and age-specific mortality rates, (b) explicitly modeling the mortality rate of the COVID-19 25 epidemics and (c) introducing isolation periods during incubation (E) and the infectious period (I) after testing (see Fig. S1 for a schematic diagram of the SEIR model implemented). Different from previous SEIR model extensions, we implement a Bayesian Melding method in our calibration 1 South Korea, Taiwan, Singapore, Germany, Iceland and Vò employed aggressive testing; the first three Asian countries had health protocols based on previous epidemic experience that enabled them to act quickly, while Germany had extensive testing availability and both Iceland and Vò are small, the latter a 3,000 inhabitants Italian town. 4 reported for pediatric patients (18) and one day shorter than the average incubation period of 5.2 days reported for patients older than 50 years (19) . In addition, the recovery period is on average 11 days (IQR [6] [7] [8] [9] [10] [11] [12] [13] [14] [15] , the time the virus affects individuals is 14.5 days , and the transmission rate 0.432 (IQR 0.325-0.489). Fig. S3 shows the posterior distribution of the basic reproduction number ℛ 0 . Our results show that the most probable ℛ 0 values range between 2.6-5 5.1, with a mean value of 4.51, very close to the basic reproduction number estimated for Wuhan (20) (for a detailed description of the calibration procedures and the epidemic parameters of the model see section 1. 2 and Table S1 in the supp. material). In addition, our calibration shows that 56% of the infected individuals are on average asymptomatic, with the distribution of the infected individuals according to their symptoms shown in Table 1 As the role of the asymptomatic individuals is particularly important for the spread of COVID-19, taking into account that on average 56% of infected individuals are asymptomatic gives a picture of the potential for infection spread. By multiplying the modeled fatality rate by the fraction of symptomatic people among all infectious people, the distribution of the true fatality rate is derived 20 5 ', 'cite_spans': [{'start': 249, 'end': 252, 'text': '(8)', 'ref_id': 'BIBREF21'}, {'start': 274, 'end': 277, 'text': '(9)', 'ref_id': 'BIBREF22'}, {'start': 308, 'end': 312, 'text': '(10)', 'ref_id': None}, {'start': 346, 'end': 350, 'text': '(11)', 'ref_id': None}, {'start': 954, 'end': 955, 'text': '1', 'ref_id': 'BIBREF0'}, {'start': 1295, 'end': 1296, 'text': '4', 'ref_id': 'BIBREF17'}, {'start': 1329, 'end': 1333, 'text': '(18)', 'ref_id': 'BIBREF31'}, {'start': 1443, 'end': 1447, 'text': '(19)', 'ref_id': 'BIBREF32'}, {'start': 1510, 'end': 1513, 'text': '[6]', 'ref_id': 'BIBREF19'}, {'start': 1514, 'end': 1517, 'text': '[7]', 'ref_id': 'BIBREF20'}, {'start': 1518, 'end': 1521, 'text': '[8]', 'ref_id': 'BIBREF21'}, {'start': 1522, 'end': 1525, 'text': '[9]', 'ref_id': 'BIBREF22'}, {'start': 1526, 'end': 1530, 'text': '[10]', 'ref_id': None}, {'start': 1531, 'end': 1535, 'text': '[11]', 'ref_id': None}, {'start': 1536, 'end': 1540, 'text': '[12]', 'ref_id': None}, {'start': 1541, 'end': 1545, 'text': '[13]', 'ref_id': None}, {'start': 1546, 'end': 1550, 'text': '[14]', 'ref_id': None}, {'start': 1551, 'end': 1555, 'text': '[15]', 'ref_id': None}, {'start': 1910, 'end': 1914, 'text': '(20)', 'ref_id': None}, {'start': 2029, 'end': 2030, 'text': '2', 'ref_id': 'BIBREF15'}, {'start': 2663, 'end': 2664, 'text': '5', 'ref_id': 'BIBREF18'}], 'ref_spans': [{'start': 785, 'end': 792, 'text': 'Fig. S1', 'ref_id': 'FIGREF5'}, {'start': 1663, 'end': 1670, 'text': 'Fig. S3', 'ref_id': 'FIGREF8'}, {'start': 2035, 'end': 2043, 'text': 'Table S1', 'ref_id': 'TABREF4'}, {'start': 2256, 'end': 2263, 'text': 'Table 1', 'ref_id': None}], 'section': 'Main Text:'}, {'text': 'We adjust the epidemic curve to each country in the world (see supplementary material Section 2), which allows us to analyze the global spread and evolution of the total number of deaths throughout 365 days. Because we are particularly concerned with the aftermath of the virus, we present all results for day 365, i.e., after one year of outbreak for each country (the daily 5 distribution is available upon request). We first estimate the impact of COVID-19 death toll in the absence of a lockdown policy and no available vaccine, assuming a well-mixed population. For comparability reasons, we apply in all countries the same initial number of imported and infected individuals, which follows a temporal Poisson process Pois(λt), with λ = 10. This implies that countries receive a monthly average of 300 infected individuals. The age of imported cases is 10 assumed to be a random number drawn from a uniform distribution with a minimum age of 18 and a maximum of 65 which corresponds to the age of potential workers (average age of the infected individuals is 41.5 years old, which does not necessarily coincide with the average age of the population analyzed). people infected depends on country size and the fatality rate depends on the age distribution of the population. Fig. 1 shows in panel A an example with extreme cases on how the transmission rate is faster in countries with a small population size like Iceland (over 340 thousand inhabitants) compared to larger countries like Brazil (over 210 million inhabitants) and China (over 1400 million inhabitants). This implies that in a context of no lockdown policy, no available vaccine, 20 and assuming a well-mixed population, small populations get infected faster than large populations. As a consequence, the peak of the mortality rate will be sooner in small populations relative to larger ones (see Table S2 for the average death toll of COVID-19 after one year in the world). Panel B shows the positive relationship between the average fatality rate and the mean age of the population for 200 countries in the world. Younger populations (mean age < 30 years) face 25 a fatality rate of around 0.2%, whereas older populations (mean age > 40 years) face a fatality rate close to 0.9%. When comparing to the grey dots that represent deaths unrelated to COVID-19, it is clear that COVID-19 is not the main cause of death in younger populations, but has the potential to double the number of deaths in ageing populations, if no policies are implemented (see Table   S2 ). CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review) 7 The potential for saving lives: the joint role of testing, herd immunity and demographic factors After estimating the global death toll in a no-policy scenario, we first include testing and isolation as the only interventions adopted to control the outbreak and its effect on saving lives. We consider 5 both people who develop symptoms and those infectious individuals who are asymptomatic.', 'cite_spans': [{'start': 2739, 'end': 2740, 'text': '7', 'ref_id': 'BIBREF20'}], 'ref_spans': [{'start': 1279, 'end': 1285, 'text': 'Fig. 1', 'ref_id': None}, {'start': 1867, 'end': 1875, 'text': 'Table S2', 'ref_id': 'TABREF6'}, {'start': 2522, 'end': 2532, 'text': 'Table   S2', 'ref_id': 'TABREF6'}], 'section': 'Global fatality rates and the role of demographic factors'}, {'text': 'Following evidence, we assume that among the fraction of symptomatic individuals, 80% have mild symptoms, while 20% develop critical and severe symptoms (21) . See Table S3 and eq. S9-S10, section 3 for a detailed description of the alternative testing options that are implemented. Fig. 2 shows that countries with higher mean-age have higher average fatality rates 10 than younger ones and that the higher the level of testing, the lower the fatality rates. Panel B in Fig. 2 shows the average proportion of lives saved after one year under different testing strategies and considering population size. 2 The death toll is reduced by 2% in large countries when severe or critical care individuals are tested and isolated. If the total coverage of testing reaches 22%, the death toll will be reduced by 9.4% in large countries. Only when all the symptomatic individuals 15 are tested and isolated, which corresponds to almost 44% of the total infectious population, the death toll is reduced by 46% in large countries. 2 The proportion of lives saved is defined as the relative difference between the total deaths without testing and the total deaths for a specific level of testing (See Tables S2 and S8 ). We also analyzed the sensitivity of COVID-deaths with respect to the timing of 5 testing by simulating eight alternative intervention days assuming that governments start testing after 45 days from the first imported and infected case. See Table S4 for the impact of the day of intervention and the level of testing on the total number of deaths in US after 365 days.', 'cite_spans': [{'start': 605, 'end': 606, 'text': '2', 'ref_id': 'BIBREF15'}, {'start': 1020, 'end': 1021, 'text': '2', 'ref_id': 'BIBREF15'}], 'ref_spans': [{'start': 164, 'end': 172, 'text': 'Table S3', 'ref_id': 'TABREF8'}, {'start': 283, 'end': 289, 'text': 'Fig. 2', 'ref_id': None}, {'start': 471, 'end': 477, 'text': 'Fig. 2', 'ref_id': None}, {'start': 1189, 'end': 1205, 'text': 'Tables S2 and S8', 'ref_id': 'TABREF6'}, {'start': 1449, 'end': 1457, 'text': 'Table S4', 'ref_id': 'TABREF10'}], 'section': 'Global fatality rates and the role of demographic factors'}, {'text': 'When all the symptomatic plus half of the asymptomatic are tested in large countries, the average 10 reduction of the death toll reaches 98%. This emphasizes the importance of testing asymptomatic individuals but also the role of population size on the potential of saving a higher proportion of lives. It is especially important to start mass testing sooner in countries with smaller populations.', 'cite_spans': [], 'ref_spans': [], 'section': 'Panel A in'}, {'text': 'After testing and isolation are included, we then add herd immunity to estimate the increasing 15 chance of saving lives. Herd immunity and serological survey can increase the amount of lives 9 saved both through decreasing the transmission rate and increasing the number of recovered, since people who have fully recovered from COVID-19 have antibodies in their plasma that can attack the virus and be used for treatment of severe patients (23, 24). Panel A in Fig. 3 shows the relationship between the average COVID-19 fatality rate and the mean age of the population for five different levels of testing and three possible degrees of herd immunity. It may come as a 5 surprise that the fatality rate does not change much across different levels of herd immunity, but this is due to the fact that both the number of deaths and the number of infected individuals decrease, affecting the numerator and the denominator that compose the fatality rate. However, the herd immunity level does change the percentage of lives saved relative to the population size. Fig. 3 panel B shows that, for a given level of testing, the higher is the herd immunity level in a 10 population, the greater is the percentage of lives saved (relative to the initial susceptible population). Moreover, this effect is more intense in large populations and for greater levels of testing. The fact that the fatality rate is rather insensitive to different levels of herd immunity reinforces the importance of looking not only into the fatality rates as a way to assess the impact on lives lost, but also considering the proportion of lives saved. . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'cite_spans': [{'start': 192, 'end': 193, 'text': '9', 'ref_id': 'BIBREF22'}], 'ref_spans': [{'start': 462, 'end': 468, 'text': 'Fig. 3', 'ref_id': None}, {'start': 1058, 'end': 1072, 'text': 'Fig. 3 panel B', 'ref_id': None}], 'section': 'Panel A in'}, {'text': 'The copyright holder for this preprint this version posted May 5, 2020. ', 'cite_spans': [], 'ref_spans': [], 'section': 'Panel A in'}, {'text': 'As a final analysis, we distinguish between the (true) fatality rate that we derive from the model, which was used in previous sections, and the case fatality rate (CFR). We define CFR as the ratio 5 between the total number of deaths and the total infected individuals who are detected. CFRs have been reported by countries based on the number of deaths and infected individuals they can detect.', 'cite_spans': [], 'ref_spans': [], 'section': 'Demographics as the bridge between the (true) fatality rate and the observed CFR'}, {'text': 'In our case, we can derive the CFR by dividing the (true) fatality rate by the fraction of infected individuals who are tested. 3 Fig. 4 infected people is at least ten times higher than those being reported. Rates in Brazil and Niger suggest that less than 9% of the total infected individuals are tested. Thus, in these two countries the total number of infected people is more than ten times higher than those being reported. In the US, the CFR value suggests that around 15% of the infected individuals are tested and therefore the likely total number of infected people is more than six times higher than those reported. In 20 Austria, which is a country that started testing at high proportions, the CFR value suggests that around than 25% of the infected individuals are tested. As a result, the total number of infected individuals is at least more than four times the total infected individuals tested, which coincides with the lower bound estimated by a recent study (25). We extend the analysis to the whole world, with results detailed in the supplementary materials (see Fig. S12 and Table S7 ). Overall, only 25 testing the critical and severe cases implies that CFRs are above 10% in countries with a mean age of the population above 42 years and close to 2% in countries with a mean age of the population below 25 years. Once the level of testing reaches all symptomatic individuals (i.e. 44% of all infected), we observe CFR values that are lower than 2.5% in all countries. This implies that CFRs can be reduced by half by testing at least 72% of the total infected population (relative to the 30 symptomatic individuals). Additionally, intervention day, population size, and herd immunity are 13 factors that influence the total number of people infected, as shown in Table S4 and Figs. 4 and S10. However, conditional on being infected, the probability of dying from the epidemic (fatality rate) just depends on age. Thus, given the demographic characteristic of each population -e.g. the mean age of the population-we can infer through the case fatality rate the total number of people infected in each country, which is a feature that may help us shed light on the number of 5 people infected.', 'cite_spans': [{'start': 1712, 'end': 1714, 'text': '13', 'ref_id': None}], 'ref_spans': [{'start': 130, 'end': 136, 'text': 'Fig. 4', 'ref_id': None}, {'start': 1084, 'end': 1092, 'text': 'Fig. S12', 'ref_id': 'FIGREF5'}, {'start': 1097, 'end': 1105, 'text': 'Table S7', 'ref_id': 'TABREF17'}, {'start': 1787, 'end': 1795, 'text': 'Table S4', 'ref_id': 'TABREF10'}], 'section': 'Demographics as the bridge between the (true) fatality rate and the observed CFR'}, {'text': 'It is fundamental to prepare for the aftermath of this global pandemic with the most flexible and 10 wide range of strategies possible, that envisage short, mid and long-term solutions, since mitigation and suppression strategies will need to be maintained until vaccines or effective treatments become widely available (26). We show that the role of testing, demographics and herd immunity are key for assessing the global impact of the COVID-19 epidemic. CFRs can be reduced by half by testing at least 72% of the total infected population and the number of infected in some countries is at 15 least ten times higher than those being reported. Countries that test only severe and critical COVID-19 cases have CFRs above 10% in countries with a mean age of the population above 42 years and close to 2% in countries with a mean age of the population below 25 years, so younger countries experience lower impacts in terms of fatality rates relative to older ones. Smaller countries are more vulnerable to a faster spread of the virus and herd immunity is important, but 20 affects more the proportion of lives being saved than the fatality rates. We also show how we can use demographic characteristics to infer from the case fatality rate the total number of people infected in each country, which can be a valuable tool for indirectly estimating the number of people infected.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'However, we are not accounting for underlying health conditions and country-specific healthcare 25 system capacity which affects resilience levels of countries coping with the pandemic (27).', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'Nonetheless, because the fatality rates by health characteristics are likely to present important bias at this moment, focusing on age provides more robust results. Gender differences (28) were also not considered and because there is an infection differential by gender and women have higher life expectancies than men, they may feel more stringent effects on savings, income support and loss 30 of spouse or other safety nets (29). Spatial flows were also not accounted for and living 14 arrangements are precluded from the analysis, when multigenerational households might have an important impact on transmission rates (30). Other important consequences of coping with the pandemic such as the increase in suicide rates among medical staff and leaders (42), detrimental psychological and physical effects of isolation, and vulnerable people with preexisting mental disorders (31, 33) are also not the focus of our study. 5', 'cite_spans': [{'start': 487, 'end': 489, 'text': '14', 'ref_id': None}, {'start': 879, 'end': 883, 'text': '(31,', 'ref_id': 'BIBREF14'}, {'start': 884, 'end': 887, 'text': '33)', 'ref_id': None}], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'As a final remark, despite the undebatable importance of testing, the recent strong worldwide demand caused a severe disruption in the production and supply of laboratory reagents, affecting test availability, particularly for low-income countries (26, 34). Given the importance of testing, it is paramount that institutions make a concerted effort at increasing testing availability.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': '1 Material and methods', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'To predict the evolution of the number of deaths caused by the epidemic and highlight the importance of testing we extend a standard epidemiological age-structured SEIR (susceptibleexposed-infected-removed) model by (a) accounting for the age-specific mortality rates, (b) explicitly modeling the mortality rate of the COVID-19 epidemics and (c) introducing isolation periods during the incubation period (E) and the infectious period (I) after testing.', 'cite_spans': [], 'ref_spans': [], 'section': 'Model'}, {'text': 'To account for the differential effect of mortality by age, each state is comprised of 95 (Ω) age-groups. We distinguish vectors and matrices from scalars by using bold letters. The dynamics of our extended age-structured SEIR model are as follows:', 'cite_spans': [], 'ref_spans': [], 'section': 'Model'}, {'text': '] is the infection rate and β is the effective transmission rate. 1 S t denotes the vector of susceptible individuals. {E u t , I u t } denote the vector of exposed and infectious individuals, respectively, who are undetected. {E d t , I d t } denote the vector of exposed and infectious individuals, respectively, who are detected (tested) and isolated. Note, that similar to (8) we also denote the state I d t as the infectious detected people and E d t as the detected exposed people (those having been in close contact with infected people and therefore been isolated). R t is the vector of recovered individuals and D c t is the vector of deaths from the COVID-19 outbreak. M and M c (ε) are, respectively, diagonal matrices with age-specific death rates not caused by COVID-19 and the estimated age-specific fatality rates for the COVID-19. The term ε in M c (ε) denotes the share of infectious and symptomatic individuals among all the infectious individuals (i.e. also including the asymptomatic cases). We assume that the (true) fatality rate, M c (ε), is a fraction ε of the (observed) case fatality rate. The (true) fatality rate is defined as the ratio between the total number of deaths from COVID-19 and the total number of infectious individuals, whereas the (observed) case fatality rate is the ratio between the total number of deaths from COVID-19 and the total infected individuals who are detected (tested). Thus, the lower the share of symptomatic individuals the lower the (true) fatality rate compared to the (observed) case fatality rate. 1 denotes the identity matrix. The set of parameters {β, π E , π I , ν, σ, ε} denotes the effective transmission rate of the disease, the fraction of people isolated out of the exposed individuals, the fraction of people isolated out of the total infectious individuals, the removal rate, the inverse of the incubation period, and the share of infectious and symptomatic individuals.', 'cite_spans': [{'start': 377, 'end': 380, 'text': '(8)', 'ref_id': 'BIBREF21'}], 'ref_spans': [], 'section': 'Model'}, {'text': '1 . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'cite_spans': [], 'ref_spans': [], 'section': 'Model'}, {'text': 'The copyright holder for this preprint this version posted May 5, 2020. The dynamics are presented in the flow diagram Figure S1 . In each time step (day) a susceptible individual may become exposed with probability β t . We assume that a fraction π E of exposed individuals are isolated. An exposed individual spends an average period of σ −1 days in incubation until becoming infectious. We assume that a fraction π I of infected people are isolated. After an average period of ν −1 days, infected individuals can either recover with probability 1 − M c (ε) or die due to COVID-19 with probability M c (ε).', 'cite_spans': [], 'ref_spans': [{'start': 119, 'end': 128, 'text': 'Figure S1', 'ref_id': 'FIGREF5'}], 'section': 'Model'}, {'text': 'The COVID-19 outbreak is characterized by a large uncertainty on the number of people infected. The number of deaths from COVID-19 seems, however, to be more reliable despite the fact that it can also be subject to under-reporting (6) as well as to over-reporting, due to competing causes of death. Under-reporting is occurring because there is no common agreement across countries on how COVID-19 deaths should be counted. Indeed, many countries are only counting as COVID-19 deaths those individuals who were tested positive, despite the fact that many people who died with COVID-19 symptoms were not tested. To avoid over-reporting, we account in the model for all causes of death, that are not COVID-19, by introducing the matrix of age-specific death rates M.', 'cite_spans': [], 'ref_spans': [], 'section': 'Calibration'}, {'text': 'So far, most of the papers on COVID-19 have mainly implemented the infectious disease models using data of reported infected cases and of reported and underreported infections (19) . However, given that mortality data presents less uncertainty than data on infections, we propose to calibrate the epidemiological model using the evolution of the number of deaths caused by COVID-19. There are some key features of the mortality pattern shown by COVID-19. First, early statistics on death by COVID-19 have shown a sizable age gradient (3), which is quite similar to the age gradient observed at old age in standard mortality rates. For this reason, it is important to consider in the model that the fatality rate of COVID-19 is increasing with age. To account for the age pattern, we regressed through an OLS the 2 . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'cite_spans': [{'start': 176, 'end': 180, 'text': '(19)', 'ref_id': 'BIBREF32'}], 'ref_spans': [], 'section': 'Calibration'}, {'text': 'The copyright holder for this preprint this version posted May 5, 2020. . function: log m x = γ 0 + γ 1 x + γ 2 x 2 + v to the log of COVID-19 age-specific fatality rates for ages older than 30 years from (15) . We do not fit the fatality rate data below age 30, given that the share of infected and asymptomatic individuals below age 30 is likely to be underreported. The dotted red line in Figure S2 shows the fit of our regression function to the data (red squares) where γ 0 = −10.5063, γ 1 = 0.1310, and γ 2 = −0.0003. Note that the estimated fatality rate (m c ) fits well the data after age 30. Modeled fatality rate, m c Figure S2 : Fitted fatality rate for COVID-19. Source: \"Natural\" age-specific mortality rates were calculated using data from the UN Population Division. Data on COVID-19 fatality rates by age is taken from (15) .', 'cite_spans': [{'start': 205, 'end': 209, 'text': '(15)', 'ref_id': None}, {'start': 836, 'end': 840, 'text': '(15)', 'ref_id': None}], 'ref_spans': [{'start': 392, 'end': 401, 'text': 'Figure S2', 'ref_id': None}, {'start': 629, 'end': 638, 'text': 'Figure S2', 'ref_id': None}], 'section': 'Calibration'}, {'text': 'To control for the unknown number of infected people who were not developing symptoms (i.e. asymptomatic) and were not tested against the virus, we have introduced in the calibration process the adjustment factor ε ∈ (0, 1). Thus, the fatality rate implemented in the SEIR model is M c (ε) = diag(εm c ) and the death rate excluding COVID-19 is M = diag(m · dt), with dt adjusting the annual rates, shown in Fig. S2 , to daily rates.', 'cite_spans': [], 'ref_spans': [{'start': 408, 'end': 415, 'text': 'Fig. S2', 'ref_id': None}], 'section': 'Calibration'}, {'text': 'The vector of age-specific mortality rates, m, for each country is computed using the estimated deaths (D x ) and exposures (E x ) from (17) . Since both deaths and population counts are grouped into 5-year age intervals (with the exception of ages 0-1) we first ungroup the data using a penalized composite link model (14; 11) . After ungrouping the data we estimate the observed death rates (M x = D x /E x ) and later, following the procedure adopted by the Human Mortality Database (HMD), we substitute them for smoothed observed death rates from ages 80 to 95 by fitting a Kannisto model. This procedure is to deal with erratic behavior and random fluctuations that are common at older ages (for more details see (16) and the https://www.mortality.org/ for their complete protocol). Thus, assuming that D x ∼ Pois(E x µ x+0.5 (a, b)) we estimate the parameters a ≥ 0 and b ≥ 0 by maximizing the log-likelihood function:', 'cite_spans': [{'start': 136, 'end': 140, 'text': '(17)', 'ref_id': 'BIBREF30'}, {'start': 319, 'end': 327, 'text': '(14; 11)', 'ref_id': None}, {'start': 718, 'end': 722, 'text': '(16)', 'ref_id': 'BIBREF29'}], 'ref_spans': [], 'section': 'Calibration'}, {'text': '3 . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'cite_spans': [], 'ref_spans': [], 'section': 'Calibration'}, {'text': 'The copyright holder for this preprint this version posted May 5, 2020. . where the function µ x is given by', 'cite_spans': [{'start': 59, 'end': 73, 'text': 'May 5, 2020. .', 'ref_id': None}], 'ref_spans': [], 'section': 'Calibration'}, {'text': 'and the constant term accounts for all the functions not dependent on (a, b). Substituting Eq. (1) into (2) yields the smoothed death ratesM x ≡ m x . In this way, smoothed rates cannot decline after age 80. 2', 'cite_spans': [], 'ref_spans': [], 'section': 'Calibration'}, {'text': \"Bayesian Melding The aim of our calibration strategy is to fit our SEIR model against the total number of deaths. Since the necessary information to fit the model is likely incomplete, we implement the Bayesian Melding method (12) , which provides an inferential framework that takes into account both model's inputs and outputs. Given that we fit a time-series (evolution of the number of deaths), we implement the Bayesian Melding following (1) . The basic purpose with the Bayesian Melding method is to derive the distribution of the set of parameters that best replicate the observed evolution of deaths by using the information from the model and the data. Let Θ ⊆ R 4 be the set of parameters -inputs-of our SEIR model. For simplicity, we denote by M (Θ) the application of our SEIR model given the inputs. Let a realization of Θ be θ = {σ, ν, β, ε}. 3 Each parameter is considered a random variable with a joint prior distribution q 1 (Θ). Given that the epidemiological characteristics of COVID-19 are unknown, we assume an uninformative prior distribution on the inputs. 4 Let the prior distribution on Θ be \", 'cite_spans': [{'start': 226, 'end': 230, 'text': '(12)', 'ref_id': None}, {'start': 443, 'end': 446, 'text': '(1)', 'ref_id': 'BIBREF0'}, {'start': 857, 'end': 858, 'text': '3', 'ref_id': 'BIBREF16'}, {'start': 1080, 'end': 1081, 'text': '4', 'ref_id': 'BIBREF17'}], 'ref_spans': [], 'section': 'Calibration'}, {'text': \"The domains of parameters σ, ν, and β are taken from the existing literature (13), while we consider any potential value between 0 and 1 for the proportion of symptomatic individuals, i.e. ε ∈ (0, 1). Let Φ be the set of outputs of our SEIR model. Given that M (Θ) = Φ, the outputs Φ are also a random variable with a joint prior distribution q 2 (Φ). Let a realization of Φ be φ = {e 1 , e 2 , . . . , e T }, where the output e s is the difference between the model's total number of deaths (D c s ) and the total observed deaths from COVID-19 (D s ) until time s or, the error of fit until time s,\", 'cite_spans': [], 'ref_spans': [], 'section': 'Calibration'}, {'text': 'We define time s as the number of days since the first observed death and T as the number of days since the first observed death from which the quarantine measures may start slowing the spread of the infection. Since the observed number of deaths until time T is subject to under-and over-reporting, we assume the following uninformative joint prior distribution on outputs', 'cite_spans': [], 'ref_spans': [], 'section': 'Calibration'}, {'text': 'where e max is the maximum discrepancy of the model with respect to the observed number of deaths until time T . Therefore, we assume following (12) that the prior joint distribution of inputs and outputs are independent. Since our goal is to obtain the joint posterior distribution of Θ, which we denote by π [Θ] (Θ), we need to update the joint prior distribution of inputs, q 1 (Θ), using the observed data through a likelihood function, i.e. L(Φ). Because M (Θ) might not be invertible, we calculate the pooled joint prior distribution on outputs, denoted byq [Φ] (Φ), through geometric pooling 5q', 'cite_spans': [{'start': 564, 'end': 567, 'text': '[Φ]', 'ref_id': None}], 'ref_spans': [], 'section': 'Calibration'}, {'text': 'where q * 1 (M (Θ)) is the induced joint prior distribution of the outputs and α is the pooling weight. This is equivalent to finding the region on which both priors have common support (12) . A value of α close to one (resp. zero) will give a low (resp. high) weight of the information provided by the model on the posterior distribution of inputs. Thus, the Bayesian joint posterior distribution of the outputs is defined as', 'cite_spans': [{'start': 186, 'end': 190, 'text': '(12)', 'ref_id': None}], 'ref_spans': [], 'section': 'Calibration'}, {'text': 'Notice that because the COVID-19 is a new virus we assume no likelihood for the inputs. For the application of the calibration, we assume the error e s is distributed according to a Normal distribution with µ = 0 and σ = √ s, i.e., N (0, √ s). 6 Thus, we calculate the likelihood of retaining the set of parameters θ ∈ Θ as', 'cite_spans': [{'start': 244, 'end': 245, 'text': '6', 'ref_id': 'BIBREF19'}], 'ref_spans': [], 'section': 'Calibration'}, {'text': \"where s 0 is the date at which the total number of deaths is above 50. Our analysis and calibration were performed with the use of Julia 1.3 (Julia Lab) and R software (R Foundation for Statistical Computing). We calibrate the model to the evolution of the total number of deaths in the province of Hubei (China). We run our model assuming that the first COVID-19 case appeared at November 17th, 2019 (t = 1), the first death occurred on January 11th, 2020 (s = 1 and t = 56), and the epidemic curve started to flatten on February 12, 2020 (T = 33 and t = 88). The maximum discrepancy e max is set at 150 deaths in order to allow for sufficient output variability. Moreover, we set the pooling weight, α, at 0.5 in order to give a similar importance to the model and the data. Since the age distribution of the population in Hubei (China) resembles that of China, we scaled down the Chinese population to the total population size of Hubei. However, we keep the death tolls reported from official statistics, since the majority of the cases belong to Hubei. Next, we detail the steps of the Bayesian Melding algorithm: 5 Our model M (Θ) might not be invertible because the parameters β and σ are highly correlated. 6 The error distribution of es is the result of assuming that the difference between the model's deaths and the observed deaths at any time t is i.i.d. according to a N (0, 1).\", 'cite_spans': [{'start': 1119, 'end': 1120, 'text': '5', 'ref_id': 'BIBREF18'}, {'start': 1215, 'end': 1216, 'text': '6', 'ref_id': 'BIBREF19'}], 'ref_spans': [], 'section': 'Calibration'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '5'}, {'text': 'The copyright holder for this preprint this version posted May 5, 2020. .', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'We draw a sample of around two million values of θ from the joint prior distribution on inputs q 1 (Θ) 2. For each θ i sampled, we run our SEIR model to obtain the output M (θ i ) = φ i 3. We estimate q * 1 (φ) using a standard gaussian kernel density estimator (kde) 4. We construct the importance sampling weights (ISW)', 'cite_spans': [], 'ref_spans': [], 'section': '1.'}, {'text': '5. We sample 500 quadruplets from the discrete distribution (θ i , w(θ i )) Table S1 summarizes the marginal posterior distributions of the model parameters (Θ). 7 Parameters σ −1 and ν −1 measure the average durations that a representative individual spends in states E and I, respectively. 8 Parameter β measures the transmission rate and 1 − ε accounts for the proportion of asymptomatic individuals among the infected. We obtain that the mean incubation period is slightly higher than the median incubation of 3.0 days reported for pediatric patients (2) and one day shorter than the average incubation period of 5.2 days reported for patients older than 50 years by (7) . Our calibration gives that the recovery period ranges between 6 and 15 days, with an average period of 11 days. By adding the incubation and recovery periods we obtain that the time the virus affects individuals ranges between 7.7 (1st Qu.) and 19 days (3rd Qu.), with an average time of 14.5 days. The transmission rate takes values between 0.325 and 0.489, where the average is 0.432. Figure S3 shows the posterior distribution of the basic reproduction number (R 0 ). This plot shows how frequent an R 0 value is obtained across the 500 combinations of the parameters drawn from the discrete distribution (θ, w(θ)). Our results give that the most probable R 0 values range between 2.6 (1st Qu.) and 5.1 (3rd. Qu.). The mean value of R 0 , which is represented by a vertical red line in Fig. S3 , is 4.51. Thus, the mean R 0 is very close to the basic reproduction number estimated for Wuhan (9) , while the most frequently obtained value is between 2.5 and 3.0. The last row in Table S1 reports the 1st and 3rd quartiles and the median and mean values for the fraction of asymptomatic individuals, 1 − ε. Our calibration suggests that 56 percent of the infected individuals are on average asymptomatic and with 50% probability the fraction of asymptomatic individuals range between 44.65% and 68.31%. Moreover, following (10) and assuming that among the symptomatic individuals roughly 80% are mild, 15% are severe, and 5% are critical, our model suggests that the distribution of the infected individuals according to their symptoms is: Multiplying the modeled fatality rate m c by the fraction of symptomatic people (ε), the distribution of the true fatality rate is derived. Figure S4 shows the inferred COVID-19 fatality rate after the calibration process. We obtain that the average fatality rate exceeds 1% at age 60, 5% at age 80, and 10% at age 90, which is sixty percent lower than the fatality rates reported by (18).', 'cite_spans': [{'start': 671, 'end': 674, 'text': '(7)', 'ref_id': 'BIBREF20'}, {'start': 1571, 'end': 1574, 'text': '(9)', 'ref_id': 'BIBREF22'}, {'start': 2001, 'end': 2005, 'text': '(10)', 'ref_id': None}], 'ref_spans': [{'start': 76, 'end': 84, 'text': 'Table S1', 'ref_id': 'TABREF4'}, {'start': 1064, 'end': 1073, 'text': 'Figure S3', 'ref_id': 'FIGREF8'}, {'start': 1466, 'end': 1473, 'text': 'Fig. S3', 'ref_id': 'FIGREF8'}, {'start': 1658, 'end': 1666, 'text': 'Table S1', 'ref_id': 'TABREF4'}, {'start': 2358, 'end': 2367, 'text': 'Figure S4', 'ref_id': 'FIGREF9'}], 'section': '1.'}, {'text': 'To analyze the spread of the virus and the evolution of the total number of deaths, we need an initial number of imported and infected individuals and the date of the onset of COVID-19 7 . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Mortality without testing'}, {'text': 'The copyright holder for this preprint this version posted May 5, 2020. outbreak. For comparability reasons, we apply in all countries the same initial number of imported cases and, hence, we use the onset of the outbreak to adjust the epidemic curve to each country. Thus, by assuming similar initial characteristics for all countries, this strategy will allow us to understand, in future work, whether the rapid spread of the virus was due to the number of undetected and infected cases or to the number of days since the first undetected and infected case appeared in the country. Let us consider that the number of imported and infected individuals follows a temporal Poisson process, Pois(λt), with λ = 10. This process, which is shown in Figure S5 , implies that each month a country receives on average 300 infected individuals and, therefore, an outbreak can occur at any time in the year. Let us also assume that the age of the infected individuals is a random number drawn from a uniform distribution with a minimum age of 18 and a maximum of 65, U(18, 65), which corresponds to the age of potential workers. This age range implies that the average age of the infected individuals is 41.5 years old, which does not necessarily coincide with the average age of the population of each country we consider. Also, these two assumptions yield that the average period to observe the first death after the first infected cases enter in the country is 18 days. Figure S6 shows that the model is capable of tracking well the evolution of the total number of deaths in eight different countries (United States of America, Italy, Spain, France, United Kingdom, Belgium, Netherlands, Brazil), which have different demographic characteristics. At the end of the period analyzed circles deviate from the model prediction because of the implementation of lockdown measures. These countries are chosen because they had more than 300 deaths by April 3rd, 2020 -the date at which the first data was collected-and because their case fatality rate was greater than 2.5%. 9 These two assumptions imply that neither 9 The case fatality rate calculated as the ratio between the total number of deaths and the total number of 8 . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 2061, 'end': 2062, 'text': '9', 'ref_id': 'BIBREF22'}, {'start': 2104, 'end': 2105, 'text': '9', 'ref_id': 'BIBREF22'}], 'ref_spans': [{'start': 744, 'end': 753, 'text': 'Figure S5', 'ref_id': 'FIGREF10'}, {'start': 1463, 'end': 1472, 'text': 'Figure S6', 'ref_id': 'FIGREF16'}], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted May 5, 2020. massive testing nor isolation measure were initially implemented. In Fig. S6 the average estimate of the model is depicted with a dashed blue line, data is represented with black circles, and gray lines correspond to each simulation drawn from the posterior distribution, (θ, w(θ)), obtained with the Bayesian Melding method. Countries are distributed according to the total number of deaths. To calculate the number of days since the onset of COVID-19 outbreak, we modify the date in which the first infected case is reported. In particular, we assume that in the Netherlands the first imported cases occurred 22 days before the first infected case was detected, in Brazil 15 days, in Italy 13 days, in Spain 9 days, in United Kingdom 0 days, in France -3 days, and in Belgium and US -4 days. A negative number implies that the assumed flow of imported cases (see Fig. S5 ) is higher than what the actual data shows.', 'cite_spans': [], 'ref_spans': [{'start': 141, 'end': 148, 'text': 'Fig. S6', 'ref_id': 'FIGREF16'}, {'start': 937, 'end': 944, 'text': 'Fig. S5', 'ref_id': 'FIGREF10'}], 'section': '(which was not certified by peer review)'}, {'text': 'To investigate the factors explaining the potential difference in fatality rates across countries, we assume that there is no isolation measures and the population is well mixed. Under these two assumptions we obtain that one key factor explaining differences in the fatality rate across countries is the population size. To show the extreme cases, we show in Figure S7 how the transmission rate is faster in countries with a small population size like Iceland (over 340 thousand inhabitants) compared to countries with a large population size like Brazil (over 210 million inhabitants) and China (over 1 400 million inhabitants). This implies that whenever the above mentioned assumptions are satisfied, small populations will get infected faster than large population. As a consequence, the peak of the mortality rate is reached sooner in small populations than in large populations.', 'cite_spans': [], 'ref_spans': [{'start': 360, 'end': 369, 'text': 'Figure S7', 'ref_id': 'FIGREF17'}], 'section': '(which was not certified by peer review)'}, {'text': 'Another key factor explaining differences in fatality rates across countries is the age distribution of the population. Assuming no isolation measures, Figure S8 shows the positive relationship between the average COVID-19 fatality rate and the mean age of the population for two hundred countries after one year since the onset of the outbreak. Gray dots depict the death rate (excluding COVID-19 deaths) in each country according to the mean age of infected and detected cases.', 'cite_spans': [], 'ref_spans': [{'start': 152, 'end': 161, 'text': 'Figure S8', 'ref_id': 'FIGREF18'}], 'section': '(which was not certified by peer review)'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '9'}, {'text': 'The copyright holder for this preprint this version posted May 5, 2020. . https://doi.org/10.1101/2020.04.29.20084400 doi: medRxiv preprint q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q Day United_States_of_America q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q Italy q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q Day Spain q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q Day France q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q Day United_Kingdom q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q 0 3 6 9 12 40 60 80 Day Belgium q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q 0 2 4 6 40 60 80 Day Netherlands q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q the population. We can see in Fig. S8 that younger populations -mean age younger than 30 years-face a fatality rate around 0.2%, whereas older populations -mean age older than 40 years-face a fatality rate close to 0.9%. By comparing the average fatality rates to the death rates (gray dots) we can observe that COVID-19 is not the main caused of death in younger populations. In contrast, since the average fatality rates and the deaths rates are almost similar in aging countries, COVID-19 has the potential to double the number of deaths in aging populations, unless policies to contain the spread of the virus are implemented. All the data used for plotting Fig. S8 is reported in Table S2 . 10 10 Table S2 and Figure S8 only reports average fatality rate values. The distribution of fatality rates for each country is reported in Table S6. 10 . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 2642, 'end': 2644, 'text': '10', 'ref_id': None}], 'ref_spans': [{'start': 1976, 'end': 1983, 'text': 'Fig. S8', 'ref_id': 'FIGREF18'}, {'start': 2608, 'end': 2615, 'text': 'Fig. S8', 'ref_id': 'FIGREF18'}, {'start': 2631, 'end': 2639, 'text': 'Table S2', 'ref_id': 'TABREF6'}, {'start': 2648, 'end': 2656, 'text': 'Table S2', 'ref_id': 'TABREF6'}, {'start': 2661, 'end': 2670, 'text': 'Figure S8', 'ref_id': 'FIGREF18'}, {'start': 2781, 'end': 2790, 'text': 'Table S6.', 'ref_id': 'TABREF13'}], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted May 5, 2020. Table S2. 11 . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [{'start': 72, 'end': 81, 'text': 'Table S2.', 'ref_id': 'TABREF6'}], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted May 5, 2020. . https://doi.org/10.1101/2020.04.29.20084400 doi: medRxiv preprint CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted May 5, 2020. 3 Mortality: Potential factors to reduce the death toll', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'In the previous section we have shown the potential death toll of COVID-19 assuming that (a) no lockdown policies are implemented and (b) the population is well mixed. In this section, we show that testing and isolation may help to control the outbreak and give estimates on how many lives can be saved. Let ε be the fraction of infectious people who develop some symptoms and 1 − ε those infectious individuals who are asymptomatic. Following (10) we assume that among the fraction ε of infectious individuals, 80% develop mild symptoms, while 20% develop critical and severe symptoms. Moreover, we consider that governments do not start isolating individuals until time t I . From time t I onwards, we consider that governments may commit to find those individuals who are infected. Let the proportion of asymptomatic 13 . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'The role of testing and isolation'}, {'text': 'The copyright holder for this preprint this version posted May 5, 2020. . and positively tested people who become isolated be τ 1 . Let the proportion of symptomatic and positively tested people who become isolated be τ 2 . Thus, we define the fraction of people who are exposed and isolated as', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'while we define the fraction of people infected who become isolated as', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'In order to analyze the impact that different degrees of testing and isolation have on controlling the spread of COVID-19 we implement five different testing options, which are summarized in Table S3 . In all the options summarized in Table S3 we assume governments do not start the intervention until one and a half months after the onset of the outbreak (i.e., t I = 45). The impact of alternative intervention days is analyzed in the next subsection. Under the assumption t I = 45, we show in Figure S9 the impact of testing on the fatality rate by mean-age of the population and on lives saved by population size. Panel A in Figure S9 shows that countries with aged populations (i.e. higher mean-age) have higher average fatality rates than countries with younger populations. However, given that the fatality rate only informs about the ratio between total deaths and total infected individuals, Panel A in Fig. S9 does not represents well how many people are saved by various testing strategies. The proportion of lives saved is defined as the relative difference between the total deaths without testing and the total deaths for a specific level of testing. Panel B in Figure S9 shows the average proportion of lives saved by testing and isolating after one year. Comparing across levels of testing Figure S9 shows that by testing and isolating individuals who develop severe or critical symptoms, the death 14 .', 'cite_spans': [], 'ref_spans': [{'start': 191, 'end': 199, 'text': 'Table S3', 'ref_id': 'TABREF8'}, {'start': 235, 'end': 243, 'text': 'Table S3', 'ref_id': 'TABREF8'}, {'start': 496, 'end': 505, 'text': 'Figure S9', 'ref_id': 'FIGREF21'}, {'start': 629, 'end': 638, 'text': 'Figure S9', 'ref_id': 'FIGREF21'}, {'start': 912, 'end': 919, 'text': 'Fig. S9', 'ref_id': 'FIGREF21'}, {'start': 1176, 'end': 1185, 'text': 'Figure S9', 'ref_id': 'FIGREF21'}, {'start': 1306, 'end': 1315, 'text': 'Figure S9', 'ref_id': 'FIGREF21'}], 'section': '(which was not certified by peer review)'}, {'text': 'CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted May 5, 2020. . toll is reduced by 2% in large countries. If the total coverage of testing reaches 22%, the death toll will be reduced by 9.4% in large countries. Only when all the symptomatic individuals are tested and isolated, which correspond to almost 44% of the total infectious population, the death toll is reduced by 46% in large countries. When all the symptomatic plus half of the asymptomatic are tested in large countries, the average reduction of the death toll reaches 98%. These results emphasizes the importance of mass testing. Indeed, Panel B, Figure S9 , clearly shows that it is especially important to start mass testing sooner in countries with smaller populations. 11 A q q q q q q q q q qqq q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q qqq q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q ', 'cite_spans': [{'start': 59, 'end': 73, 'text': 'May 5, 2020. .', 'ref_id': None}], 'ref_spans': [{'start': 621, 'end': 630, 'text': 'Figure S9', 'ref_id': 'FIGREF21'}], 'section': '(which was not certified by peer review)'}, {'text': 'Another alternative for controlling the spread of COVID-19 is to start testing and isolating the infectious as soon as possible. To analyze the sensitivity of COVID-deaths with respect to the timing of testing we simulate eight alternative intervention days t I . The first possible day of intervention (t I ) is set at day 31 and all subsequent dates of intervention, t I , are delayed by one week until t I = 80. Table S4 shows the impact of the day of intervention and the level of testing on the total number of deaths in the US after 365 days. For a given level of testing, the death toll is higher, the later the day of intervention. Similarly, for a given day 11 Given the existence of tests shortages, it is also true that countries with smaller populations may have an advantage over countries with large populations because they do not need to buy as many tests.', 'cite_spans': [{'start': 667, 'end': 669, 'text': '11', 'ref_id': None}], 'ref_spans': [{'start': 415, 'end': 423, 'text': 'Table S4', 'ref_id': 'TABREF10'}], 'section': 'The role of the intervention day'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '15'}, {'text': 'The copyright holder for this preprint this version posted May 5, 2020. . of intervention, the lower is the level of testing, the higher is the death toll. Table S4 also shows that when the level of testing is lower than 45%, the day of intervention does not have a significant effect on the death toll. Therefore, in order to save more lives the best policy is testing all the infected individuals as soon as possible. ', 'cite_spans': [], 'ref_spans': [{'start': 156, 'end': 169, 'text': 'Table S4 also', 'ref_id': 'TABREF10'}], 'section': '(which was not certified by peer review)'}, {'text': 'In this subsection we study the impact of herd immunity on reducing the future death toll. We define herd immunity as the resistance to the spread of the disease due to the existence of recovered individuals who are immune to the virus. Thus, herd immunity will prevent the spread of the virus by reducing the infection rate β t . To study the effect of herd immunity we run the model, for all the countries contained in Tab. S2, setting the intervention day (t I ) at time 0 and assuming three alternative scenarios in which the total proportion of recovered people is 25%, 50%, and 75%. We set t I at 0 because we assume that countries who have attained a specific level of herd immunity will continue testing individuals.', 'cite_spans': [], 'ref_spans': [], 'section': 'The role of herd immunity'}, {'text': 'Panel A in Figure S10 shows the relationship between the average COVID-19 fatality rate and the mean age of the population for five different levels of testing and three possible degrees of herd immunity. It is not surprising that the fatality rate does not change much across different levels of herd immunity, since both the number of deaths and the number of infected individuals decrease. As a consequence, the fatality rate becomes rather insensitive to the herd immunity level. The herd immunity level, however, changes the percentage of lives saved relative to the population size. Panel B in Figure S10 shows that, for a given level of testing, the higher is the herd immunity level in a population, the greater is the percentage of lives saved (relative to the initial susceptible population). Moreover, this effect is more intense in large populations and the greater is the level of testing.', 'cite_spans': [], 'ref_spans': [{'start': 11, 'end': 21, 'text': 'Figure S10', 'ref_id': 'FIGREF5'}, {'start': 600, 'end': 610, 'text': 'Figure S10', 'ref_id': 'FIGREF5'}], 'section': 'The role of herd immunity'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '16'}, {'text': 'The copyright holder for this preprint this version posted May 5, 2020. ', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'A q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q qq q q q q q q q q q q q q q q q q q q q q q q q qq q q qq q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q qq q q q q q q qq q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q qq q q q q q q q q q q q q q q q q q q q q q q q qq q q qq q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q qq q q q q q q qq q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q qq q q q q q q q q q q q q q q q q q q q q q q q qq q q qq q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q qq q q q q q q qq q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q qq q q q q q q q q q q q q q q q q q q q q q q q qq q q qq q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q qq q q q q q q qq q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q qq q q q q q q q q q q q q q q q q q q q q q q q qq q q qq q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q qq q q q q q q qq q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q qq q q q q q q qq q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q qq q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q qq q q q q q q qq q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q qq q q q q q q q q q q q q q q q q q q q q q q q qq q q qq q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q qq q q q q q q qq q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q qq q q q q q q q q q q q q q q q q q q q q q q q qq q q qq q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q qq q q q q q q qq q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q qq q q q q q q q q q q q q q q q q q q q q q q q qq q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q qq q q q q q q qq q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q ', 'cite_spans': [], 'ref_spans': [], 'section': 'in %)'}, {'text': 'In this section we investigate how to infer the number of infected people in each country using the model. To do so, we use the case fatality rate, which differs from the (true) fatality 17', 'cite_spans': [], 'ref_spans': [], 'section': 'Total infected individuals and the case fatality rates'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Total infected individuals and the case fatality rates'}, {'text': 'The copyright holder for this preprint this version posted May 5, 2020. . rate used in previous sections. We define the case fatality rate (CFR) as the ratio between the total number of deaths and the total number of infected individuals that are detected. Note that information about the total number of deaths and the total number of infected and detected individuals are provided in country reports. In our case we derive the CFR dividing the (true) fatality rate by the fraction of infected individuals who are tested. Figure S11 shows the CFR that would be reached for a given level of testing, that is maintained after one year since the onset of the epidemic outbreak, in eight selected countries (Italy, Spain, Austria, USA, China, Brazil, India, and Niger). By using the CFR in day 365, the CFR value reported is the maximum that a country will reach under a specific level of testing. 12 Equivalently, each country curve, depicted in Fig. S11 , also shows the maximum level of testing for a given CFR value. The gray area represents the level of testing for the symptomatic individuals. For each country curve, the CFR values at the left-hand side of the gray area correspond to testing infected individuals who only develop critical and severe symptoms, or 20% of the symptomatic individuals. The CFR values at the right-hand side of the gray area correspond to testing 100% of the symptomatic individuals. While the true fatality rates and the CFRs only coincide when the level of testing is one hundred percent, which corresponds to the far right point in the diagram.', 'cite_spans': [{'start': 895, 'end': 897, 'text': '12', 'ref_id': None}], 'ref_spans': [{'start': 523, 'end': 533, 'text': 'Figure S11', 'ref_id': 'FIGREF5'}, {'start': 944, 'end': 952, 'text': 'Fig. S11', 'ref_id': 'FIGREF5'}], 'section': '(which was not certified by peer review)'}, {'text': 'The CFR values until April 21st, 2020 were 13.3% (=24 114 deaths/181 228 cases) in Italy, 10.4%(=20 852/200 210) in Spain, 3.2% (=470/14 783) in Austria, 5.4% (=42 539/787 752) in US, 5.5% (=4 636/83 849) in China, 6.3% (=2575/40 581) in Brazil, 3.2% (=590/18 600) in India, and 3.1% (=20/655) in Niger. Thus, according to Fig. S11 the CFR values in China, India, Italy, and Spain suggest that these countries are mainly testing individuals who develop critical and severe symptoms (see the crossed circles over each country curve). Hence, the total number of infected people is at least ten times higher than those being reported in China, India, Italy, and Spain. The CFR values in Brazil and Niger suggest that less than 9% of the total infected individuals are tested. Thereby, the total number of infected people is more than ten times higher than those being reported in these two countries. In the US, the CFR value suggests that close to 15% of the infected individuals are tested (see the crossed circle over the orange curve) and therefore it is likely that total number of infected people is more than six times higher than those being detected. In Austria, which is a country that started testing at high proportions, the CFR value suggests that around 25% of the infected individuals are tested (see the crossed circle in third curve starting from the top). As a result, the total number of infected individuals is at least more than four times the total infected individuals tested, which coincides with the lower bound estimated by (5) for Austria. q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q qq q q q q q q qq q q q q q q q q q q q q q q q q q q qq q q q q q q q q q q q q q q q q qqq q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q q We extend the analysis to the whole world by showing in Figure S12 and Table S7 the average CFR across countries by mean age of the population and testing level. When only testing at individuals who develop severe or critical symptoms, Fig. S12 shows that the CFRs will be above 10% in countries with a mean age of the population above 42 years. In contrast,', 'cite_spans': [{'start': 1547, 'end': 1550, 'text': '(5)', 'ref_id': 'BIBREF18'}], 'ref_spans': [{'start': 323, 'end': 331, 'text': 'Fig. S11', 'ref_id': 'FIGREF5'}, {'start': 2227, 'end': 2237, 'text': 'Figure S12', 'ref_id': 'FIGREF5'}, {'start': 2242, 'end': 2250, 'text': 'Table S7', 'ref_id': 'TABREF17'}, {'start': 2407, 'end': 2415, 'text': 'Fig. S12', 'ref_id': 'FIGREF5'}], 'section': '(which was not certified by peer review)'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '19'}, {'text': 'The copyright holder for this preprint this version posted May 5, 2020. . when only testing those individuals with severe and critical symptoms, the CFRs will be close to 2% in countries with a mean age of the population below 25 years. 13 Once that the level of testing reaches all the symptomatic individuals (i.e. 44% of all infected), we observe values for the CFR lower than 2.5% in all countries. Moreover, Fig. S12 shows that the CFRs can be halved with respect to testing all the symptomatic individuals by testing at least 72% of the total infected population.', 'cite_spans': [{'start': 59, 'end': 73, 'text': 'May 5, 2020. .', 'ref_id': None}, {'start': 237, 'end': 239, 'text': '13', 'ref_id': None}], 'ref_spans': [{'start': 413, 'end': 421, 'text': 'Fig. S12', 'ref_id': 'FIGREF5'}], 'section': '(which was not certified by peer review)'}, {'text': 'In sum, despite the uncertainty associated to the COVID-19 outbreak, we have found an important feature that may help us to shed light on the number of people infected. This is because the day of intervention, the population size, and the herd immunity affect on the total number of people infected, as shown in Table S4 and Figs. S9 and S10. However, conditional on being infected the probability of dying from the epidemic (or fatality rate) just depends on age. Thus, given the demographic characteristic of each population -e.g. the mean age of the population-we can infer through the case fatality rate the total number of people infected in each country.', 'cite_spans': [], 'ref_spans': [{'start': 312, 'end': 320, 'text': 'Table S4', 'ref_id': 'TABREF10'}], 'section': '(which was not certified by peer review)'}, {'text': 'We complement the information provided in previous sections with four additional tables. Table S5 shows the death toll of COVID-19 for each region in the world by degree of testing under fully susceptible populations. In Table S6 we report the distribution of the fatality rate of COVID-19 obtained through the Bayesian Melding method for 200 countries. Table S7 reports the average case fatality rates by level of testing after one year in the 200 countries plotted in Fig. S12 . Table S8 shows the average death toll from COVID-19 for 200 countries by level of testing in fully susceptible populations after 365 days. is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [{'start': 89, 'end': 97, 'text': 'Table S5', 'ref_id': 'TABREF12'}, {'start': 221, 'end': 229, 'text': 'Table S6', 'ref_id': 'TABREF13'}, {'start': 354, 'end': 362, 'text': 'Table S7', 'ref_id': 'TABREF17'}, {'start': 470, 'end': 478, 'text': 'Fig. S12', 'ref_id': 'FIGREF5'}, {'start': 481, 'end': 489, 'text': 'Table S8', 'ref_id': None}], 'section': 'The impact of COVID-19 in the World'}, {'text': 'The copyright holder for this preprint this version posted May 5, 2020. . https://doi.org/10.1101/2020.04.29.20084400 doi: medRxiv preprint CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted May 5, 2020. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted May 5, 2020. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted May 5, 2020. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted May 5, 2020. 28 . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted May 5, 2020. 29 . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted May 5, 2020. . https://doi.org/10.1101/2020.04.29.20084400 doi: medRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}], 'paragraphs': [{'text': 'These factors make these strategies unsustainable in the long-run and both country endurance to social distance measures and their effect on fatality rates will most likely depend on socioeconomic 20 settings (7), detecting asymptomatic individuals (8) , undocumented cases (9) , demographic characteristics (10) , and the level of herd immunity (11) . We account in a combined fashion for those last three factors by extending a standard epidemiological (Susceptible-Exposed-Infected-Removed) compartment model by (a) accounting for the age structure of the population and age-specific mortality rates, (b) explicitly modeling the mortality rate of the COVID-19 25 epidemics and (c) introducing isolation periods during incubation (E) and the infectious period (I) after testing (see Fig. S1 for a schematic diagram of the SEIR model implemented). Different from previous SEIR model extensions, we implement a Bayesian Melding method in our calibration 1 South Korea, Taiwan, Singapore, Germany, Iceland and Vò employed aggressive testing; the first three Asian countries had health protocols based on previous epidemic experience that enabled them to act quickly, while Germany had extensive testing availability and both Iceland and Vò are small, the latter a 3,000 inhabitants Italian town. 4 reported for pediatric patients (18) and one day shorter than the average incubation period of 5.2 days reported for patients older than 50 years (19) . In addition, the recovery period is on average 11 days (IQR [6] [7] [8] [9] [10] [11] [12] [13] [14] [15] , the time the virus affects individuals is 14.5 days , and the transmission rate 0.432 (IQR 0.325-0.489). Fig. S3 shows the posterior distribution of the basic reproduction number ℛ 0 . Our results show that the most probable ℛ 0 values range between 2.6-5 5.1, with a mean value of 4.51, very close to the basic reproduction number estimated for Wuhan (20) (for a detailed description of the calibration procedures and the epidemic parameters of the model see section 1. 2 and Table S1 in the supp. material). In addition, our calibration shows that 56% of the infected individuals are on average asymptomatic, with the distribution of the infected individuals according to their symptoms shown in Table 1 As the role of the asymptomatic individuals is particularly important for the spread of COVID-19, taking into account that on average 56% of infected individuals are asymptomatic gives a picture of the potential for infection spread. By multiplying the modeled fatality rate by the fraction of symptomatic people among all infectious people, the distribution of the true fatality rate is derived 20 5 ', 'id': '00002'}], 'title': 'How many lives can be saved? A global view on the impact of testing, herd immunity and demographics on COVID-19 fatality rates', 'sha': '986b152f8392d89bd4f62bc3fceba91a6aa3af8d', 'doi': '10.1101/2020.04.29.20084400', 'score': 8.89109992980957}, {'rank': 256, 'paragraphs': [{'text': 'The bottom panel of Figure 1 includes the meta‐analysis results. The meta estimate of the basic reproduction number is 3.15, suggesting that a virus carrier may infect at least three individuals on average if preventive measures such as social distancing or quarantine are not applied to the public. This estimate is obtained from the data before 29 February 2020 where most data are collected from Wuhan city or other places in China. As more studies on the basic reproduction number become available for different places at different time periods, we can apply the same meta‐analysis procedure to estimate the basic reproduction number to reflect its changes with the implementation of various measures to curb the virus spread in different regions.', 'id': '00014'}], 'title': 'Estimation of the basic reproduction number, average incubation time, asymptomatic infection rate, and case fatality rate for COVID‐19: Meta‐analysis and sensitivity analysis', 'sha': '2c63ae442219d870167e691f91cd32e4ab9a7505', 'doi': '10.1002/jmv.26041', 'score': 8.886500358581543}, {'rank': 257, 'abstract': \"Abstract Background COVID-19 is spreading quickly all over the world. Publicly released data for 1212 COVID-19 patients in Henan of China was analyzed in this paper. Methods Various statistical and network analysis methods were employed. Results We found that COVID-19 patients show gender (55% vs 45%) and age (81% aged between 21 and 60) preferences, possible causes were explored; The estimated average, mode and median incubation periods are 7.4, 4 and 7 days; Incubation periods of 92% patients were no more than 14 days; The epidemic in Henan has undergone three stages and showed high correlations with the numbers of patients recently return from Wuhan; Network analysis revealed that 208 cases were clustering infected and various people's Hospitals are the main force in treating COVID-19. Conclusions The incubation period was statistical estimated and the proposed state transition diagram can well explore the epidemic stages of emerging infectious disease. We suggest that though the quarantine measures are gradually at work, strong measures might be still needed for a period of time, since ∼7.45% patients may have very long incubation periods. Migrant workers or college students are with high risk. State transition diagram can help us to recognize the time-phased nature of epidemic. Our investigations have implications for the prevention and control of COVID-19 in other regions of the world.\", 'title': 'Statistical and network analysis of 1212 COVID-19 patients in Henan, China', 'sha': '', 'doi': '10.1016/j.ijid.2020.04.051', 'score': 8.8843994140625}, {'rank': 258, 'paragraphs': [{'text': 'Based on the COVID-19 daily updates from provincial and municipal health commissions in China, we notice that there is an abundance of cases who asymptomatically left Wuhan, the epicenter of COVID-19, and developed symptoms outside Wuhan. Assuming that these cases were infected before their departure from Wuhan, the time differences between departure and symptoms onset is the censored observations of their incubation periods. Hence, we conducted a cross-sectional and forward follow-up study by assuming to catch those asymptomatic individuals at their departure time and followed them until their symptoms developed. Using the language of renewal processes, we can treat the development of the disease starting from infection by a pathogen as a stochastic process that could be observed from a specific time point in chronological order. In this study, the specific time point refers to the time of departure from Wuhan. For each prevalent case, the complete process from the infection to the onset of symptoms can be considered as a renewal process. As illustrated in Figure 1, the backward recurrence time is hence defined as the time between infection and departure from Wuhan, and the forward recurrence time is the time between departure from Wuhan and symptom onset. Clearly, the forward time is observable and the corresponding observations are with good veracity, while the backward time is either unable to be observed or the corresponding observations are with large uncertainty due to recall bias. Note that for each infected individual, the backward time and forward time do not have to be same. However, when the renewal process reaches its equilibrium status, it becomes reversible, that is, the statistical properties of this process is the same as the one for time-reversed data in a same process. Hence, at equilibrium, the backward time can be treated as the forward time if time periods are reversed.19', 'id': '00006'}], 'title': 'Estimation of incubation period distribution of COVID-19 using disease onset forward time: a novel cross-sectional and forward follow-up study', 'sha': 'c61be9fc4562d250e2dec241ef75bd22d61f61d1; 6d3b3f4ab80a61c45f82c61c6c756cfc6ddf4bf2', 'doi': '10.1101/2020.03.06.20032417', 'score': 8.881999969482422}, {'rank': 259, 'paragraphs': [{'text': 'Case 3 is a 44-year-old female from Yiyang. She also had close contact with her elder sister (Case 1). She developed a fever (37.8 °C) and inappetence on February 13th. Three days later, she was taken to a hospital. Laboratory examination showed a WBC of 2.88 ∗ 10^9/L, C-reactive protein of 17.06 mg/L, and fibrinogen of 4.27 g/L. Influenza A and B were negative. Lung CT (Fig. 2G) showed multiple patchy high-density shadows on both lungs. On February 17th and February 20th, the nasopharyngeal swab was negative for SARS-CoV2. After treatment, the symptoms disappeared. Hence, she was discharged on February 21st and placed under home quarantine. Since her father (Case 2) was diagnosed with COVID-19 on February 28th, she was admitted to the hospital again for further screening. This time, her SARS-CoV2 nucleic acids and specific IgG and IgM antibodies were positive. The ESR was 55.6 mm/h. Antiviral therapy was given with interferon and lopinavir-ritonavir. On March 2nd, her repeat viral nucleic acid test was still positive. The nasopharyngeal swab was negative twice, respectively on March 4th and March 6th, with simultaneously improved lung CT (Fig. 2H).', 'id': '00009'}], 'title': 'COVID-19 in a patient with long-term use of glucocorticoids: A study of a familial cluster', 'sha': '912c00514457672748738d29efb81f975815cdf6', 'doi': '10.1016/j.clim.2020.108413', 'score': 8.881199836730957}, {'rank': 260, 'paragraphs': [{'text': 'Publicly available data were retrieved from provincial and municipal health commissions in China and the ministries of health in other countries, including 12 963 confirmed cases outside Hubei province as of February 15, 2020. Detailed information on confirmed cases includes region, gender, age, date of symptom onset, date of confirmation, history of travel or residency in Wuhan, and date of departure from Wuhan. The date of symptoms onset in these data refers to the date reported by the patient on which the clinical symptoms first appeared, where the clinical symptoms include fever, cough, nausea, vomiting, diarrhea, and others. Among 12 963 confirmed cases, 6345 cases had their dates of symptom onset collected; 3169 cases had histories of travel or residency in Wuhan; 2514 cases had their dates of departure recorded; and 1922 cases had records of both dates of departure from Wuhan and dates of symptoms onset.', 'id': '00009'}], 'title': 'Estimation of incubation period distribution of COVID-19 using disease onset forward time: a novel cross-sectional and forward follow-up study', 'sha': 'c61be9fc4562d250e2dec241ef75bd22d61f61d1; 6d3b3f4ab80a61c45f82c61c6c756cfc6ddf4bf2', 'doi': '10.1101/2020.03.06.20032417', 'score': 8.85200023651123}, {'rank': 261, 'abstract': 'Objective: many potential factors contribute to the outbreak of COVID-19.It aims to explore the effects of various meteorological factors on the incidence of COVID-19. Methods: Taking Hubei province of China as an example, where COVID-19 was first reported and there were the most cases, we collected 53 days of confirmed cases (total 67773 cases) and ten meteorological parameters up to March 10. Correlation analysis and linear regression were used to judge the relationship of meteorological factors and increment of COVID-19 confirmed cases. Results: Under 95% CI, the increment of confirmed cases in Hubei were correlated with four meteorological parameters of average pressure, average temperature, minimum temperature and average water vapor pressure (equivalent to absolute humidity).The average pressure was positively correlated with the increment (r=+0.358).The negative correlations included average temperature (r=-0.306), minimum temperature (r=-0.347), and average water vapor pressure (r=-0.326). The linear regression results show if minimum temperature increases by 1℃, the incremental confirmed cases in Hubei decreases by 72.470 units on average. Conclusion: Statistically, the incidence of COVID-19 was correlated with average pressure, average temperature, minimum temperature and average water vapor pressure. It is positively correlated with the average pressure and negatively correlated with the other three parameters. Compared with relative humidity, 2019-nCov is more sensitive to water vapor pressure. The reason why the epidemic situation in Hubei expanded rapidly is significantly related to the climate characteristics of low temperature and dryness of Hubei in winter.', 'body_text': [{'text': ', swine flu virus (Lowen et al., 2007) 、 SARS-Cov (Chan et al., 2011) , MERS-CoV (van Doremalen et al., 2013) , coronavirus (Casanova et al., 2010; Kim et al., 2007) . The effects of different meteorological parameters (temperature, RH, absolute humidity, atmospheric pressure) and other meteorological factors on viral transmission or viral activity, were studied and some relevant mechanisms were revealed.', 'cite_spans': [{'start': 18, 'end': 38, 'text': '(Lowen et al., 2007)', 'ref_id': None}, {'start': 50, 'end': 69, 'text': '(Chan et al., 2011)', 'ref_id': 'BIBREF12'}, {'start': 72, 'end': 109, 'text': 'MERS-CoV (van Doremalen et al., 2013)', 'ref_id': None}, {'start': 124, 'end': 147, 'text': '(Casanova et al., 2010;', 'ref_id': 'BIBREF8'}, {'start': 148, 'end': 165, 'text': 'Kim et al., 2007)', 'ref_id': None}], 'ref_spans': [], 'section': ''}, {'text': 'However, it still cannot be fully explained. For example, studies have suggested that the survival of influenza virus is affected by the amount of water vapor pressure (equal to absolute humidity, AH) in the surrounding air, however, it is unclear why the stability of an influenza virus encased within a droplet nucleus would be sensitive to atmospheric water vapor pressure conditions. It is presumed that high atmospheric humidity levels will lead to surface inactivation of lipid-containing viruses, such as influenza viruses (Shaman et al. & Kohn, 2009 ) .', 'cite_spans': [{'start': 530, 'end': 559, 'text': '(Shaman et al. & Kohn, 2009 )', 'ref_id': 'BIBREF55'}], 'ref_spans': [], 'section': ''}, {'text': 'A variety of statistical methods were used for mathematical modeling to further reveal the potential laws. Kim et al. believed that the incidence and mortality of diseases followed the Poisson distribution and used Poisson function as the link function to establish a generalized linear model (GLM) (Kim et al., 2016) . Lim et al. used the cross design of Poisson GLM and temperature-matched case. Through case cross design, personal characteristics and long-term and seasonal trends can be controlled (Lim et al., 2012) . GLM was also used in the studies of Onozuka et al. and Souza et al. (Souza et al., 2012; Onozuka et al., 2009) . For seasonal viral infectious diseases, Gurgel et al. and Yusuf et al. used time series analysis to reduce model deviations (Gurgel et al., 2016; Yusury et al., 2007) . Chan et al., du Prel et al., Lin et al., Varela et al. use the autoregressive integral moving average (ARIMA) model to control the autocorrelation of time series data (Chan et al ., 2002; du Prel et al., 2009; Lin et al., 2009; Varela et al., 2004) . Carreras et al. used a generalized additive model (GAM) using a quasi-Poisson distribution as link function.', 'cite_spans': [{'start': 107, 'end': 117, 'text': 'Kim et al.', 'ref_id': None}, {'start': 299, 'end': 317, 'text': '(Kim et al., 2016)', 'ref_id': 'BIBREF34'}, {'start': 320, 'end': 335, 'text': 'Lim et al. used', 'ref_id': None}, {'start': 502, 'end': 520, 'text': '(Lim et al., 2012)', 'ref_id': 'BIBREF40'}, {'start': 578, 'end': 611, 'text': 'Souza et al. (Souza et al., 2012;', 'ref_id': 'BIBREF59'}, {'start': 612, 'end': 633, 'text': 'Onozuka et al., 2009)', 'ref_id': 'BIBREF48'}, {'start': 694, 'end': 711, 'text': 'Yusuf et al. used', 'ref_id': None}, {'start': 760, 'end': 781, 'text': '(Gurgel et al., 2016;', 'ref_id': None}, {'start': 782, 'end': 802, 'text': 'Yusury et al., 2007)', 'ref_id': None}, {'start': 805, 'end': 863, 'text': 'Chan et al., du Prel et al., Lin et al., Varela et al. use', 'ref_id': None}, {'start': 972, 'end': 992, 'text': '(Chan et al ., 2002;', 'ref_id': 'BIBREF11'}, {'start': 993, 'end': 1014, 'text': 'du Prel et al., 2009;', 'ref_id': 'BIBREF24'}, {'start': 1015, 'end': 1032, 'text': 'Lin et al., 2009;', 'ref_id': 'BIBREF41'}, {'start': 1033, 'end': 1053, 'text': 'Varela et al., 2004)', 'ref_id': 'BIBREF69'}, {'start': 1056, 'end': 1067, 'text': 'Carreras et', 'ref_id': None}], 'ref_spans': [], 'section': ''}, {'text': 'In this study, the daily temperature range (DTR) was used as the main predictor, and other control variables included day of the week, RH and PM10. These variables are included in the model as linear terms. In the secondary analysis, the same model was used, but all variables were stratified by season, age group, seasonal pattern and socioeconomic status (Carreras et al., 2015) . In China, another study used GAM and controlled long-term trends. In addition, the average temperature was fitted into the model to explore the logarithmic risk of daily hospitalization (Bai YL, et al.2019 ).', 'cite_spans': [{'start': 357, 'end': 380, 'text': '(Carreras et al., 2015)', 'ref_id': 'BIBREF7'}, {'start': 569, 'end': 588, 'text': '(Bai YL, et al.2019', 'ref_id': None}], 'ref_spans': [], 'section': ''}, {'text': 'However, whether this relationship exists in respiratory infections caused by viruses other than influenza is unclear, furthermore, laboratory studies of novel coronavirus (2019-ncov) related to meteorological factors have not been reported.', 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': 'So far, the research on the COVID-2019 and the climatic characteristics of Hubei province in China has not been reported. Similarly, the relationship of the COVID-19 epidemic and meteorological factors has been unavailable in published professional journals. The purpose of this study is to unfold whether the meteorological factors have influence on the 2019-nCoV transmission and the occurrence and development of epidemic diseases, and to explore in detail the relationship between the explosion of confirmed cases and meteorological factors under subtropical monsoon climate conditions, this is of practical significance for predicting the extinction of the virus. Due to the different characteristics of virus transmission in different latitude regions, therefore, with appropriate spatial resolution, Hubei province is used instead of China or Wuhan City (i.e., at provincial level rather than national level, also not city level) analyzing surveillance data is very important for understanding the global persistence of viruses and formulating local prevention and control measures.', 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': 'This study designed the correlation analysis between the COVID-19 increment cases in Hubei province and 10 meteorological parameters and the regression analysis of four sensitive parameters, combined with the meteorological characteristics of Hubei province, namely cold and dry in winter, hot and rainy in summer with frequent meteorological disasters, in order to find the relevant clues of meteorological factors on the occurrence, development and extinction of COVID-19. altitude: 23.6 (the altitude is calculated from Qingdao tide inspection station in Shandong as the starting point )). The available data period is from December 1, 2019 to March 10, 2020 (100 daily meteorological data). The actual data selected starts on January 1, 2020 and ends on March 10, 2020. Meteorological observatory data include: average air pressure (hPa), average temperature (℃), maximum temperature (℃), minimum temperature (℃), average water vapor pressure (hPa) (equivalent to water vapor pressure), average relative humidity (%), average wind speed (m/s), precipitation (mm), total solar radiation (0.01 megajoules/square meter), maximum solar irradiance (Watt/square meter ). Afterwards, it uses Ave_Pre, Ave_Temp, Max_Temp, Min_Temp, Ave_WVP, Ave_RH, Ave_WS, Prep, Tot_SR and Max_SI to refer to them respectively.', 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': 'According to the suggestion of the Meteorological Data Center of China Meteorological Administration, the average water vapor pressure (hPa) should be introduced as a quantity representing the moisture content in the atmosphere. When there is much water vapor in the atmosphere, the water vapor pressure is large and vice versa.', 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': '(2) Number of confirmed cases Wuhan Health Committee reported that the local epidemic situation in Wuhan was reported on December 27. After tracing to the source, it was confirmed that the first patient was diagnosed in December 12. For a long time, Wuhan Health Committee had been using \"unexplained viral pneumonia\" rather than COVID-2019 before this, so the cases could not guarantee its authenticity.', 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': 'After January 16 th , 2020, especially on 17 th and 18 th , the state has delegated the power of virus testing to the Hubei Center for Disease Control (HCDC), that is, the samples do not need to be sent to Beijing, and can be tested in the local organization, HCDC detection capability is about . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': \"The copyright holder for this preprint this version posted April 3, 2020. . confirmed cases on the 17 th and 18 th increased. This study uses January 17, 2020 as the starting point of time to retrieve data. Case data include Hubei's cumulative confirmed cases (67773 as of March 10), Hubei's cumulative death (3046 as of March 10), and Hubei's incremental cases, namely HB_CumN, HB_CumD, and HB_DeltaN respectively.\", 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': 'As for number of confirmed cases, several issues to be explained: 1) From 0: 00 to 24: 00 on February 12, 2020, the number of newly incremental cases of COVID-19 in Hubei province surged explosively to 14840 (including 13332 clinically diagnosed cases), in which are 13436 cases in Wuhan. The reason was that the General Office of the State Health Commission and the Office of the State Administration of Traditional Chinese Medicine issued the new coronavirus infection pneumonia diagnosis and treatment plan (trial version 5).', 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': 'Type of \"Clinical Diagnosis\" has been added to the classification of case diagnosis in Hubei province, so that patients can receive standardized treatment as early as possible according to confirmed cases and further improve the success rate of treatment. According to this plan, Hubei province carried out investigation on the previous suspected cases and revised the diagnosis results, and diagnosed the new patients according to the new diagnosis classification. In order to be consistent with the classification of case diagnosis released by other provinces in the country, since February 12, 2020, the number of clinically diagnosed cases in Hubei province has been included in the number of confirmed cases for publication.', 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': '2) The incremental confirmed cases in January 19, 2020 were 0.', 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': 'Here, the above two days are outliers and are eliminated in correlation analysis and regression analysis.', 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': '(1) Correlation analysis Pearson correlation coefficient, Spearman rank correlation coefficient and Kendall Rank determination correlation coefficient are commonly used in the correlation analysis for binary variables. The calculation method of Spearman rank correlation coefficient is as follows.', 'cite_spans': [], 'ref_spans': [], 'section': 'Selection of statistical methods'}, {'text': 'The calculation formula is as follows:', 'cite_spans': [], 'ref_spans': [], 'section': 'Selection of statistical methods'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Selection of statistical methods'}, {'text': 'The copyright holder for this preprint this version posted April 3, 2020. . The value range of correlation coefficient r is equal and greater than -1, equal and lower than 1. r > 0 is positive correlation, r <0 is negative correlation; 0.3 <|r|<0.5 is low degree linear correlation; 0.5 <|r|< 0.8 is significant linear correlation; (2) Linear regression', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'On the basis of correlation analysis, we estimate the parameters in the regression using the Ordinary Least Square (OLS) method. Incremental cases and four meteorological parameters are further selected for unary linear regression. All regressions are performed with the code SPSS v15.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The prediction model of linear regression is:', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'In the formula, t X represents the period t independent variable; t Y represents the dependent variable of period t; a and b represent the parameters of a linear regression equation. Obtained by the following formula:', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'For the regression analysis of quantitative data, four prerequisites must be met: the linear relationship between incremental cases (dependent variables) and certain meteorological parameters, and each parameter is independent of each other, the residual of incremental cases satisfies the normality, and the residual of incremental cases satisfies the homogeneity of variance.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted April 3, 2020. . https://doi.org/10. 1101 If linear regression results cannot be obtained, generalized linear regression is considered. Table 1 summarizes the basic statistical results of incremental cases and meteorological parameters. As of March 20, 2020, the minimum number of incremental cases in Hubei province was 13 and the maximum was 4810, which occurred on February 13, 2020. From January 1 st , 2020 to March 10, the average air pressure range is 1006.0-1031.2 hPa, the average is 1021.7hPa, the standard deviation is 5.1 hPa, and the fluctuation range is small. The average wind speed ranges from 0 to 5.0 m/s and the fluctuation range are small too. The maximum rainfall is 38.9mm, which occurs in January 9, 2020 and has no obvious regularity. The average temperature, the maximum temperature and the minimum temperature are 6.9 ℃ , 11.6 ℃ and 3.4 ℃ respectively, the standard deviation of the maximum temperature is 5.0 ℃ > that of the minimum temperature 4.3 ℃ > that of the average temperature 4.0 ℃ . The average water vapor pressure and the average RH are both related to the water loading in the air, and the mean ± standard deviations are 8.3 ± 2.4 (hPa) and 83.0 ± 8.2 (%), respectively. Ultraviolet intensity is not the routine monitoring items in the meteorological station. Therefore, the daily total solar radiation and the maximum solar irradiance are selected as two auxiliary parameters, and their mean values are 757.7 (0.01 MJ/m 2 ) respectively and 456.6 (W/m 2 ), the standard deviations are 587.3 (0.01 MJ/m 2 ) and 274.2 (W/m 2 ), respectively. Obviously, the volatility of the values is very strong. Figure 1 shows the frequency statistics of incremental cases and meteorological parameters, among which the normality of average air pressure, average temperature, minimum temperature, average relative humidity and average wind speed is better. The total solar radiation and the maximum solar irradiance present a saddle-shaped bimodal structure. Figure 2 shows the tendency of incremental cases, average temperature, average water pressure and the maximum solar irradiance changing with time. Obviously, incremental cases-time pair is a single-peak structure. As for average temperature-time pair, it can be seen that there is a fluctuating process after about January 10, 2020. As for average water vapor pressure-time pair, after January 10, 2020, also has a fluctuating process, but the trend is relatively gentle. However, . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 96, 'end': 100, 'text': '1101', 'ref_id': None}], 'ref_spans': [{'start': 195, 'end': 202, 'text': 'Table 1', 'ref_id': None}, {'start': 1696, 'end': 1704, 'text': 'Figure 1', 'ref_id': None}, {'start': 2043, 'end': 2051, 'text': 'Figure 2', 'ref_id': None}], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted April 3, 2020. . there are two parallel tracks for the maximum solar irradiance-time pair, the general trend is to rise, but the trend is gentle. Table 2 summarizes cross-relationship between Hubei delta number with weather parameters.', 'cite_spans': [], 'ref_spans': [{'start': 205, 'end': 212, 'text': 'Table 2', 'ref_id': 'TABREF0'}], 'section': '(which was not certified by peer review)'}, {'text': 'Under the condition of confidence interval 95% (95% CI), incremental cases in Hubei province have significant correlation with four meteorological parameters: average air pressure, average temperature, minimum temperature and average water vapor pressure. among them, there is a positive correlation with the average air pressure (P = 0.010), and the correlation coefficient r = + 0.358, which belongs to low degree linear correlation. Negative correlation indexes include average temperature (P = 0.029), minimum temperature (P = 0.013), average water pressure (P = 0.020), correlation coefficient r =-0.306,-0.347, -0.326. From this, it can be judged that the increment of cases will be inhibited by the increase of temperature and water vapor loading. At the same time, the increase of average air pressure may increase the reproduction rate of the virus. In addition, the maximum temperature, RH and incremental cases are also negatively correlated, but the significance level is insufficient. Figure 3 shows the curve fitting of HB_DeltaN with average pressure, average temperature, minimum temperature and average water vapor pressure. . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [{'start': 998, 'end': 1006, 'text': 'Figure 3', 'ref_id': None}], 'section': 'Correlation Analysis between incremental cases and meteorological parameters'}, {'text': 'The copyright holder for this preprint this version posted April 3, 2020. . Figure 4a shows the normality of the residual of incremental cases satisfies. (2) Time series method, commonly used in seasonal and repeatable viral infectious diseases.', 'cite_spans': [], 'ref_spans': [{'start': 76, 'end': 85, 'text': 'Figure 4a', 'ref_id': None}], 'section': '(which was not certified by peer review)'}, {'text': 'Because the 2019-nCoV is a sudden infectious disease, time series analysis cannot be considered (Gurgel et al., 2016; Yusef et al., 2007) .', 'cite_spans': [{'start': 96, 'end': 117, 'text': '(Gurgel et al., 2016;', 'ref_id': None}, {'start': 118, 'end': 137, 'text': 'Yusef et al., 2007)', 'ref_id': None}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': '(3) Unary linear regression prediction is a method to establish the linear regression equation of X and Y for prediction according to the correlation between independent variable X and dependent variable Y. If linear regression results cannot be obtained, generalized linear regression is considered. The extension of the linear model of the generalized linear model is to establish the relationship between the mathematical expectation value of the response variable and the prediction variable of the linear combination through the link function. Its feature is that it does not forcibly change the natural measurement of data, and the data can have nonlinear and non-constant variance structure, it is a development of linear model when studying non-normal distribution of response value and simple and direct linear transformation of nonlinear model. However, linear regression results are easy to explain natural processes of respiratory infectious diseases caused by viruses and are of great significance. (Kim et al., 2016; Lim et al., 2012; Souza et al., 2012; Onozuka et al., 2009) .', 'cite_spans': [{'start': 1012, 'end': 1030, 'text': '(Kim et al., 2016;', 'ref_id': 'BIBREF34'}, {'start': 1031, 'end': 1048, 'text': 'Lim et al., 2012;', 'ref_id': 'BIBREF40'}, {'start': 1049, 'end': 1068, 'text': 'Souza et al., 2012;', 'ref_id': 'BIBREF59'}, {'start': 1069, 'end': 1090, 'text': 'Onozuka et al., 2009)', 'ref_id': 'BIBREF48'}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'In this study, the correlation analysis between 10 meteorological parameters and case . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted April 3, 2020. . increment was introduced, and finally 4 parameters were found to have strong correlation with case increment. Considering the three-month statistical cycle, the actual valid data is only 53. In the case of limited observations, there can be 4 parameters with obvious correlation. We have every reason to believe that, first, with the increase of effective observations, the statistical correlation between parameters will gradually appear; secondly, case increment is very sensitive to meteorological parameters, and analyzing cases with meteorological parameters is an effective method; thirdly, in addition to the minimum temperature, the other three parameters cannot establish a linear regression equation. Similarly, this study also uses the method of multiple linear regression analysis, but the overall fitting degree is very poor, the ultimate reason, it is not the problem of statistical method itself, but the effective observations for sudden infectious diseases is too little and the law is not obvious. 2019-nCoV evolution to seasonal infectious disease such as influenza virus is possible, if this happens, then periodic and repetitive data can be accumulated; meanwhile, this study finds that meteorological parameters, such as the minimum temperature and the average water vapor pressure, are the specific influencing factors of 2019-nCoV, this will become very meaningful for predicting the 2019-nCoV epidemic trend in the future.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'In addition, if regression analysis is to be carried out between temperature and RH and the number of cases (whether cumulative or incremental), taking into account the correlation between independent variables, either the Durbin-Watson Test (0 <Durbin-Watson <4) must be passed, or the random forest model must be used. The advantage of random forests is that they do not need a pre-specified model to evaluate like regression models, and can process interactive variables with correlation.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': '(4) Although incubation period has influence on correlation analysis and regression equation establishment, 2019-nCoV is a new type of virus. Its incubation period data is limited and there is great uncertainty. According to the data of 10 cases, a study shows that the average incubation period of n2019-nCoV is 5.2 days, CI 95%: 4.1 ~ 7.0. . The incubation period is relatively long. The incubation period based on the data of 10 cases is inaccurate. Therefore, no choice was made to include the incubation period in the study. In addition, without adding incubation period parameters, the relationship between meteorological factors and virus onset will not be affected, and the research results will not be affected.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Relationship between COVID-19 epidemic and climate characteristics of Hubei province'}, {'text': \"The copyright holder for this preprint this version posted April 3, 2020. . We summarize the daily incremental cases and meteorological factors of COVID-19 in Hubei province by using the correlation analysis of Spearman's rank correlation coefficient. It is found that the average temperature, minimum temperature, water vapor pressure and air pressure are significantly correlated with the daily increase of COVID-19 cases .The atmospheric pressure is positive correlation. The other three are negatively correlated with the delta. The linear relationship between the incremental cases and the minimum temperature is regained by using the linear equation. As far as we know, these results are reported for the first time.\", 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'It is generally believed that seasonality is the manifestation of respiratory diseases related to meteorological factors. At present, MERS is considered to be seasonal (Nassar et al., 2018) , and meteorological factors (high temperature, high ultraviolet index, low wind speed and low RH) are also found to be the cause of the increase in MERS-CoV cases (Altamimi et al., 2019 ) . Research on the relationship between meteorological factors and SARS shows that the number of daily cases is negatively correlated with the highest and/or lowest temperatures and the air pressure is positively correlated with SARS transmission (Bi et al.,2007) . Temperature, RH and wind speed are the three key meteorological factors (Cai et al.,2007) that affect the spread of SARS, but seasonal problems cannot be analyzed because SARS occurred only once.', 'cite_spans': [{'start': 168, 'end': 189, 'text': '(Nassar et al., 2018)', 'ref_id': 'BIBREF46'}, {'start': 354, 'end': 378, 'text': '(Altamimi et al., 2019 )', 'ref_id': 'BIBREF0'}, {'start': 625, 'end': 641, 'text': '(Bi et al.,2007)', 'ref_id': 'BIBREF2'}, {'start': 716, 'end': 733, 'text': '(Cai et al.,2007)', 'ref_id': 'BIBREF4'}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'Among the viral respiratory tract infections, the most frequently studies related to meteorological factors are RSV, a common respiratory virus. RSV accounts for 20-25% of respiratory infections in children and 3-7% in adults and the elderly. Previous studies have found that, in tropical, subtropical and temperate regions, RSV infections are related to meteorological factors such as temperature, RH, atmospheric pressure and rainfall Darniot et al.,2018; Gurgel et al., 2016; Tian et al.,2017; Sirimi et al.,2016; Liu et al.,2016; Chen et al.,2014; du Prel et al.,2009) . The increase of RSV cases is negatively correlated with RH (Gurgel et al., 2016; Tian et al., 2017; Liu et al., 2016) . Similarly, due to the high incidence rate of influenza virus, it brings heavy medical burden and has been fully studied. The results show that influenza virus (including A, B, etc.) is obviously related to meteorological parameters (temperature, ultraviolet . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 437, 'end': 457, 'text': 'Darniot et al.,2018;', 'ref_id': 'BIBREF19'}, {'start': 458, 'end': 478, 'text': 'Gurgel et al., 2016;', 'ref_id': None}, {'start': 479, 'end': 496, 'text': 'Tian et al.,2017;', 'ref_id': 'BIBREF66'}, {'start': 497, 'end': 516, 'text': 'Sirimi et al.,2016;', 'ref_id': 'BIBREF56'}, {'start': 517, 'end': 533, 'text': 'Liu et al.,2016;', 'ref_id': None}, {'start': 534, 'end': 551, 'text': 'Chen et al.,2014;', 'ref_id': 'BIBREF44'}, {'start': 552, 'end': 572, 'text': 'du Prel et al.,2009)', 'ref_id': 'BIBREF24'}, {'start': 631, 'end': 655, 'text': 'RH (Gurgel et al., 2016;', 'ref_id': None}, {'start': 656, 'end': 674, 'text': 'Tian et al., 2017;', 'ref_id': 'BIBREF66'}, {'start': 675, 'end': 692, 'text': 'Liu et al., 2016)', 'ref_id': None}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted April 3, 2020. . index, wind speed, RH, etc.) (Yu et al., 2013; Soebiyanto et al., 2010; Tamerius et al., 2019; Chong et al., 2019; Peci et al., 2019; Jaakkola et al., 2014) .', 'cite_spans': [{'start': 105, 'end': 122, 'text': '(Yu et al., 2013;', 'ref_id': 'BIBREF72'}, {'start': 123, 'end': 147, 'text': 'Soebiyanto et al., 2010;', 'ref_id': 'BIBREF58'}, {'start': 148, 'end': 170, 'text': 'Tamerius et al., 2019;', 'ref_id': 'BIBREF64'}, {'start': 171, 'end': 190, 'text': 'Chong et al., 2019;', 'ref_id': None}, {'start': 191, 'end': 209, 'text': 'Peci et al., 2019;', 'ref_id': 'BIBREF51'}, {'start': 210, 'end': 232, 'text': 'Jaakkola et al., 2014)', 'ref_id': 'BIBREF33'}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The geographical location of Hubei province is between NL 29 degrees 05 minutes ~33 degrees 20 minutes and EL 108 degrees 21 minutes -116 degrees 07 minutes, which mainly belongs to subtropical monsoon climate, it is characterized by transition from subtropical to temperate. The average temperature in the whole province is 15-17 ℃ , the coldest in January, the average temperature in most areas is 3-6 ℃ , and the hottest in July, the average temperature is 27-29 ℃ . Due to the huge thermal difference between the sea and land, the cold and high pressure of Siberia-Mongolia in winter and the hot and low pressure of Indian monsoon in summer are formed on the ground barometric field, which makes the prevailing wind direction on the ground generate periodic conversion accordingly. Winter monsoon is cold and dry, resulting in low temperature and less precipitation in winter. In summer, the southward wind blowing from the ocean to the mainland prevails, which is warm and moist, resulting in cold and dry winter, hot and rainy summer and frequent meteorological disasters.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'According to the report of Huang CL, et al, Wuhan Jin Yin-tan Hospital has received a series of pneumonia cases of unknown on December 2019 (Huang et al.,2020) . By January 2020, the whole Hubei province was under the condition of the lowest temperature and humidity throughout the year. Unfortunately, the temperature and water vapour pressure were negatively correlated with the 2019-nCoV increment, which was conducive to the survival and transmission of 2019-nCoV. If the epidemic occurs in July 2019, the most likely result is that the intensity of the epidemic will be greatly weakened and the duration will be shorter. In a sense, the epidemic is a derivative of meteorological disasters.', 'cite_spans': [{'start': 140, 'end': 159, 'text': '(Huang et al.,2020)', 'ref_id': 'BIBREF31'}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'We first observed that 2019-nCoV has significant correlation with the average minimum temperature, water vapor pressure, and atmospheric pressure. The average minimum temperature has a negative interference effect on the stability and transmission of 2019-nCov, which may directly affect the incidence of COVID-2019. The minimum temperature increased by 1 ° C, and the number of newly diagnosed patients in Hubei dropped by an average of 72.470 units. We deem . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Two meteorological parameters related to moisture in air'}, {'text': 'The copyright holder for this preprint this version posted April 3, 2020. . that under subtropical monsoon conditions, water vapor pressure is more sensitive to RH than 2019-nCov.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The research results show that the average water vapor pressure and RH are both negatively correlated with the case increment, but the significance level of RH is insufficient, or in other words, in statistical significance, correlation is not very credible.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'Relative humidity (f) is the ratio of the actual water vapor pressure in the air to the saturated water vapor pressure at the same temperature (expressed as a percentage), that is relative humidity directly reflects the degree of air saturation.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': '100% e f E = × (Eq.6) When f = 100%, the air has reached saturation, when unsaturated, f <100%, when over saturated, f> 100%. the magnitude of RH is not only related to the moisture content in the atmosphere, but also decreases with the increase of temperature. When the water vapor pressure remains unchanged, the temperature rises, the saturated water vapor pressure increases, and the RH decreases. Therefore, there is a correlation between temperature and RH, and the two are inter-dependent variables.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'There has always been opposition to the occurrence and epidemic of virus (influenza) by water vapor pressure and RH (Prussin et al., 2018) . Water vapor pressure describes the mass of water vapor per unit of air (i .e. AH), while RH describes the ratio of actual concentration of water vapor to the maximum possible concentration, this ratio varies with temperature. Studies have shown that the relationship between water vapor pressure and RH is significant (Shaman & Kohn. 2009 ). According to their analysis, water vapor pressure explained 50% and 90% of influenza virus transmission and survival variability respectively, while RH explained 12% and 36% of variability respectively. In the study of establishing influenza-related mortality model, it is found that the change of water vapor pressure is the root cause of seasonal trend, while RH prediction is relatively insufficient (Shaman et al., 2010) .It is also reported that AH(water vapor pressure) is better than RH in virus inactivation. Causal Analysis of Global Influenza incidence data shows that water vapor pressure is a stronger driver than RH (Deyle et al., 2016) .', 'cite_spans': [{'start': 116, 'end': 138, 'text': '(Prussin et al., 2018)', 'ref_id': 'BIBREF52'}, {'start': 459, 'end': 479, 'text': '(Shaman & Kohn. 2009', 'ref_id': 'BIBREF55'}, {'start': 886, 'end': 907, 'text': '(Shaman et al., 2010)', 'ref_id': None}, {'start': 1112, 'end': 1132, 'text': '(Deyle et al., 2016)', 'ref_id': 'BIBREF23'}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'In epidemiological studies, seasonal and metering conditions of influenza were compared, and the results showed lower temperature and water vapor pressure can increase the survival and . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted April 3, 2020 . . https://doi.org/10.1101 transmission of influenza virus in temperate regions (Shaman & Kohn, 2009; Shaman et al., 2010 Shaman et al., & 2011 Kolberg et al., 2019) . Studies from China also prove that influenza virus is negatively correlated with temperature and water vapor pressure (Cao et al., 2010; Sun et al., 2018; Yang et al., 2019; Su et al., 2020) . The average water vapor pressure in sub-cold zone climate is lower than 4 hPa, which may trigger the occurrence of influenza (Kolberg et al., 2019) . Studies in tropical/subtropical regions of China have found that, water vapor pressure is negatively correlated with the incidence of A/dnan1pdm09 (Influenza A) and Yamagata (Influenza B) (Pan et al., 2019) .', 'cite_spans': [{'start': 59, 'end': 72, 'text': 'April 3, 2020', 'ref_id': None}, {'start': 73, 'end': 100, 'text': '. . https://doi.org/10.1101', 'ref_id': None}, {'start': 154, 'end': 175, 'text': '(Shaman & Kohn, 2009;', 'ref_id': 'BIBREF55'}, {'start': 176, 'end': 195, 'text': 'Shaman et al., 2010', 'ref_id': None}, {'start': 196, 'end': 217, 'text': 'Shaman et al., & 2011', 'ref_id': None}, {'start': 218, 'end': 239, 'text': 'Kolberg et al., 2019)', 'ref_id': 'BIBREF37'}, {'start': 360, 'end': 378, 'text': '(Cao et al., 2010;', 'ref_id': 'BIBREF6'}, {'start': 379, 'end': 396, 'text': 'Sun et al., 2018;', 'ref_id': 'BIBREF63'}, {'start': 397, 'end': 415, 'text': 'Yang et al., 2019;', 'ref_id': 'BIBREF49'}, {'start': 416, 'end': 432, 'text': 'Su et al., 2020)', 'ref_id': 'BIBREF61'}, {'start': 560, 'end': 582, 'text': '(Kolberg et al., 2019)', 'ref_id': 'BIBREF37'}, {'start': 773, 'end': 791, 'text': '(Pan et al., 2019)', 'ref_id': 'BIBREF49'}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'This study argues that it is because RH is related to both the water vapor pressure and the temperature, and the annual changes are relatively complex so the changes of time in the past study included one year or many years, and meteorological parameters including different seasons (range is larger). However, in our study because the effective time period was for 51 days, and in the epidemic area, the range of average minimum temperature was 18.10 °C (-3.6 ~ 14.5°C), the regression coefficient with statistical significance cannot be obtained, but the correlation analysis shows that it has positive and negative correlation with the average temperature, water vapor pressure and atmospheric pressure. In other words, the increment of cases is less sensitive to relative humidity. According to the data analysis results of the epidemic situation in Hubei province, under the subtropical monsoon climate conditions, the RH cannot be selected, and the vapor pressure should be selected as the parameter to analyze the influence relationship between the moisture in the air and the activity of the case.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'We demonstrate for the first time the epidemiological correlation between water vapor pressure and respiratory tract infection virus-2019-nCoV other than influenza. In coronavirus(SARS-CoV, MERS-CoV) and RSV have not been reported to be associated with water vapor pressure. According to its related mechanism analysis, Sundell et al., believes that, as for the relationship between climatic factors and viral respiratory epidemiology, in temperate climate, non-coated viruses (HRV and HEV) when causing infection, may be transmitted through large drops or close contact rather than through small atomized particles, so the dependence of transmission factors on meteorological factors is relatively small (Sundell et al., 2016 ) . Previous studies have shown that the envelope (E) protein of coronavirus that causes Severe Acute Respiratory Syndrome is directly related to viral toxicity (DeDiego et al.,2014; CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 320, 'end': 353, 'text': 'Sundell et al., believes that, as', 'ref_id': None}, {'start': 705, 'end': 728, 'text': '(Sundell et al., 2016 )', 'ref_id': 'BIBREF62'}, {'start': 888, 'end': 909, 'text': '(DeDiego et al.,2014;', 'ref_id': None}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted April 3, 2020 . . https://doi.org/10.1101 .04.01.20050526 doi: medRxiv preprint et al.,2018 Schoeman &Fielding,2019) , our speculation that the envelope proteins closely related to 2019-nCov pathogenicity are sensitive to the average minimum temperature and water vapor pressure (negative correlation), and they are more dependent on climatic factors, especially temperature (including average temperature and minimum temperature). This finding should have theoretical significance for viral respiratory infectious diseases with abundant meteorological factors, and of course need more evidence to support it in the future.', 'cite_spans': [{'start': 59, 'end': 72, 'text': 'April 3, 2020', 'ref_id': None}, {'start': 73, 'end': 100, 'text': '. . https://doi.org/10.1101', 'ref_id': None}, {'start': 101, 'end': 150, 'text': '.04.01.20050526 doi: medRxiv preprint et al.,2018', 'ref_id': None}, {'start': 151, 'end': 175, 'text': 'Schoeman &Fielding,2019)', 'ref_id': None}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'Previous studies believe that atmospheric pressure is positively correlated with influenza virus (Li et al., 2015; Soebiyanto et al., 2010) . Atmospheric pressure may be the driver of influenza (Sundell et al.,2016) . RSV activity increases with the increase of atmospheric pressure (Hervás et al.,2012 ). An analysis of the relationship between the meteorological factors of 9 cities with different climate types in the United States and the activity of RSV shows that in Delhi, air pressure is the main related factor of the total amount of RSV, it was associated with 22% of RSV activity (Yusuf et al.,2007) . The cause of influenza may be that nasal mucosa is easy to rupture under dry conditions and high pressure in winter, so it is easy to be infected with virus. However, the detailed mechanism needs further study (Su et al.,2020) .', 'cite_spans': [{'start': 97, 'end': 114, 'text': '(Li et al., 2015;', 'ref_id': 'BIBREF38'}, {'start': 115, 'end': 139, 'text': 'Soebiyanto et al., 2010)', 'ref_id': 'BIBREF58'}, {'start': 194, 'end': 215, 'text': '(Sundell et al.,2016)', 'ref_id': 'BIBREF62'}, {'start': 283, 'end': 302, 'text': '(Hervás et al.,2012', 'ref_id': 'BIBREF29'}, {'start': 591, 'end': 610, 'text': '(Yusuf et al.,2007)', 'ref_id': 'BIBREF73'}, {'start': 823, 'end': 839, 'text': '(Su et al.,2020)', 'ref_id': 'BIBREF61'}], 'ref_spans': [], 'section': 'Relationship between case increment and atmospheric pressure'}, {'text': 'This study has some limitations.', 'cite_spans': [], 'ref_spans': [], 'section': 'Shortcomings'}, {'text': 'First, studies have shown that COVID-19 has a certain incubation period [average incubation period is 5.2 days, 95% CI: 4.1 ~ 7.0], but this estimation is based on 10 cases, more information support must be provided. The COVID-19 in Hubei province belongs to sudden acute infectious disease, and the pathogen has not yet been determined. The relevant information we have obtained does not have the presumed incubation period parameter (lag effect). Before January 17, Wuhan released the number of intermittent cases, but it was not a daily statistical result, which did not meet the requirements of the basic data collection designed by our research. At the same time, our collection range of cases started from January 17, covering 53 days. By March 10, the incidence of cases in Hubei province decreased. The collection date of cases has covered most of the basic process of occurrence and development in Hubei province. This content should be added to the analysis of relevant factors for further summarizing and analyzing the incidence of epidemic cases in Hubei province in order to make the research more accurate and scientific.', 'cite_spans': [], 'ref_spans': [], 'section': 'Shortcomings'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Shortcomings'}, {'text': 'The copyright holder for this preprint this version posted April 3, 2020. . Secondly, we have examined the correlation studies between 10 meteorological indicators and case increment respectively, average atmospheric pressure, Average temperature, average minimum temperature, average water pressure and case increment have significant correlation. We believe that atmospheric pressure, temperature and water vapor pressure are crucial to shaping conditions conducive to the diffusion of 2019-nCoV aerosol, but in the epidemiological model, whether they should be treated as individual factors or as an associated entity remains to be further studied. Unary linear regression analysis was carried out on average atmospheric pressure, average temperature, average minimum temperature, average water pressure and case increment respectively, and a statistically significant regression equation was obtained. However, there is no statistically significant binary linear regression equation describing two factors. We need further research, such as the correlation between the onset of viral respiratory tract infection under the combined action of minimum temperature and water vapor pressure.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'Third, we cannot exclude the possible effects of some mixed factors, such as population flow and air pollution, which may affect the number of diseases and further deviate our results. In addition, we did not consider host factors, such as immunity or vulnerability, which play a role in disease transmission. A large number of studies have shown that viral activity that causes respiratory infectious diseases is sensitive to climate. Climatic factors may affect the survival and transmission of the virus in the environment, host susceptibility and exposure possibility. We found that 2019-nCoV showed significant role in meteorological factors during the outbreak of the COVID-19 in Hubei province, but we are also very clear, as scholars have agreed, meteorological parameters can only explain no more than 30% changes in influenza activity (Monamele et al., 2017) , there are still many problems about 2019-nCoV to be confirmed.', 'cite_spans': [{'start': 845, 'end': 868, 'text': '(Monamele et al., 2017)', 'ref_id': 'BIBREF45'}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'Social distancing measures are essential components of the public health response to COVID-19. The purpose of these mitigation measures is to reduce transmission, thereby delaying the peak of the epidemic, reducing the total number of infected people, and spreading new cases over a longer period of time to relieve the pressure on the health care system and achieve the purpose of controlling the epidemic (Fong et al., 2020) . The corresponding measures taken in Wuhan and Hubei Province have already seen practical results.', 'cite_spans': [{'start': 407, 'end': 426, 'text': '(Fong et al., 2020)', 'ref_id': 'BIBREF25'}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusions'}, {'text': 'The copyright holder for this preprint this version posted April 3, 2020. . is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted April 3, 2020. . *. Correlation is significant at the 0.05 level (2-tailed).', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': '.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'CC-BY-NC-ND 4.0 International license'}, {'text': 'The copyright holder for this preprint this version posted April 3, 2020. is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted April 3, 2020. is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted April 3, 2020. is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted April 3, 2020. . https://doi.org/10.1101/2020.04.01.20050526 doi: medRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': '(1) Meteorological Observatory data', 'cite_spans': [], 'ref_spans': [], 'section': 'Supporting information-data description'}, {'text': 'The available data period provided from the Meteorological Data Center of China Meteorological Administration is from December 1, 2019 to March 10, 2020 (100 daily meteorological data). The actual data selected starts on January 1, 2020 and ends on March 10, 2020. Meteorological observatory data include: average air pressure (hPa), average temperature ( ℃ ), maximum temperature (℃), minimum temperature (℃), average water vapor pressure (hPa) (equivalent to absolute humidity), average relative humidity (%), average wind speed (m/s), precipitation (mm), total solar radiation (0.01 megajoules/square meter), maximum solar irradiance (Watt/square meter ). Afterwards, it uses Ave_Pre, Ave_Temp, Max_Temp, Min_Temp, Ave_WVP, Ave_RH, Ave_WS, Prep, Tot_SR and Max_SI to refer to them respectively.', 'cite_spans': [], 'ref_spans': [], 'section': 'Supporting information-data description'}, {'text': \"(2) Number of confirmed cases This study uses January 17, 2020 as the starting point of time to retrieve data. Case data include Hubei's cumulative confirmed cases, Hubei's cumulative death, and Hubei's incremental cases, namely HB_CumN, HB_CumD, and HB_DeltaN respectively. February 12, 2020 and January 19, 2020 are outliers.\", 'cite_spans': [], 'ref_spans': [], 'section': 'Supporting information-data description'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'cite_spans': [], 'ref_spans': [], 'section': 'Supporting information-data description'}, {'text': 'The copyright holder for this preprint this version posted April 3, 2020. .', 'cite_spans': [], 'ref_spans': [], 'section': 'Supporting information-data description'}], 'paragraphs': [{'text': '(4) Although incubation period has influence on correlation analysis and regression equation establishment, 2019-nCoV is a new type of virus. Its incubation period data is limited and there is great uncertainty. According to the data of 10 cases, a study shows that the average incubation period of n2019-nCoV is 5.2 days, CI 95%: 4.1 ~ 7.0. . The incubation period is relatively long. The incubation period based on the data of 10 cases is inaccurate. Therefore, no choice was made to include the incubation period in the study. In addition, without adding incubation period parameters, the relationship between meteorological factors and virus onset will not be affected, and the research results will not be affected.', 'id': '00034'}], 'title': 'Meteorological factors correlate with transmission of 2019-nCoV: Proof of incidence of novel coronavirus pneumonia in Hubei Province, China', 'sha': '30a471ddef6335b88d487168148d283c0676806c', 'doi': '10.1101/2020.04.01.20050526', 'score': 8.849699974060059}, {'rank': 262, 'abstract': 'BACKGROUND: When a new disease such starts to spread, the commonly asked questions are how deadly is it? and how many people are likely to die of this outbreak? The World Health Organization (WHO) announced in a press conference on January 29, 2020 that the death rate of COVID-19 was 2% on the case fatality rate (CFR). It was underestimated assuming no lag days from symptom onset to deaths while many CFR formulas have been proposed, the estimation on Bays theorem is worthy of interpretation. Hence, it is hypothesized that the over-loaded burdens of treating patients and capacities to contain the outbreak (LSBHRS) may increase the CFR. METHODS: We downloaded COVID-19 outbreak numbers from January 21 to February 14, 2020, in countries/regions on a daily basis from Github that contains information on confirmed cases in >30 Chinese locations and other countries/regions. The pros and cons were compared among the 5 formula of CFR, including [A] deaths/confirmed; [B] deaths/(deaths + recovered); [C] deaths/(cases x days ago); [D] Bayes estimation based on [A] and the outbreak (LSBHRS) in each country/region; and [E] Bayes estimation based on [C] deaths/(cases x days ago). The coefficients of variance (CV = the ratio of the standard deviation to the mean) were applied to measure the relative variability for each CFR. A dashboard was developed for daily display of the CFR across each region. RESULTS: The Bayes based on (A)[D] has the lowest CV (=0.10) followed by the deaths/confirmed (=0.11) [A], deaths/(deaths + recoveries) (=0.42) [B], Bayes based on (C) (=0.49) [E], and deaths/(cases x days ago) (=0.59) [C]. All final CFRs will be equal using the formula (from, A to E). A dashboard was developed for the daily reporting of the CFR. The CFR (3.7%) greater than the prior CFR of 2.2% was evident in LSBHRS, increasing the CFR. A dashboard was created to present the CFRs on COVID-19. CONCLUSION: We suggest examining both trends of the Bayes based on both deaths/(cases 7 days ago) and deaths/confirmed cases as a reference to the final CFR. An app developed for displaying the provisional CFR with the 2 CFR trends can improve the underestimated CFR reported by WHO and media.', 'body_text': [{'text': 'When a new disease (e.g., COVID-19) starts to spread, many questions emerge: how is the virus transmitted, how long is the incubation period, what is the influence of those asymptomatically infected, what is the definite reproductive number (i.e., so-called R0), how long does viral shedding persist after symptoms faded, who is at risk for a severe course, and how high is the case fatality rate (CFR), defined as the total number of deaths known by today divided by the total number of known cases confirmed by today.[7] The CFR is related to the questions: how deadly is this? and how many people will die in this outbreak? SARS, MERS, Ebola, and H1NI yielded a real CFR of 9.6%, 34.4%,73%, and 0.4%, respectively,[5–9] the CFR for COVID-19 is of interest to numerous researchers.[7,10,11]', 'cite_spans': [{'start': 520, 'end': 521, 'mention': '7', 'ref_id': 'BIBREF32'}, {'start': 718, 'end': 719, 'mention': '5', 'ref_id': 'BIBREF30'}, {'start': 720, 'end': 721, 'mention': '9', 'ref_id': 'BIBREF34'}, {'start': 784, 'end': 785, 'mention': '7', 'ref_id': 'BIBREF32'}, {'start': 786, 'end': 788, 'mention': '10', 'ref_id': 'BIBREF1'}, {'start': 789, 'end': 791, 'mention': '11', 'ref_id': 'BIBREF2'}], 'section': 'Concerns on case fatality rate ::: Introduction', 'ref_spans': []}, {'text': 'Due to the underestimated CFR using the formula (=deaths/confirmed),[5] particularly at the fast-spreading period, several other calculations of CFR were proposed,[12–15] such as deaths/(deaths + recoveries) and deaths/(cases x days ago), where x denotes the lag days from symptom onset to death.[3] None of the provisional CFRs can accurately predict the real (final) CFR[7] based on the preliminary COVID-19 data.[6] Ignoring the underline meaning of the CFR and carelessly reporting imprecise information with CFR (=deaths/cases) might be biased; see the next section.', 'cite_spans': [{'start': 69, 'end': 70, 'mention': '5', 'ref_id': 'BIBREF30'}, {'start': 164, 'end': 166, 'mention': '12', 'ref_id': 'BIBREF3'}, {'start': 167, 'end': 169, 'mention': '15', 'ref_id': 'BIBREF6'}, {'start': 297, 'end': 298, 'mention': '3', 'ref_id': 'BIBREF22'}, {'start': 373, 'end': 374, 'mention': '7', 'ref_id': 'BIBREF32'}, {'start': 416, 'end': 417, 'mention': '6', 'ref_id': 'BIBREF31'}], 'section': 'Concerns on case fatality rate ::: Introduction', 'ref_spans': []}, {'text': 'The World Health Organization (WHO), in a press conference on January 29, 2020, announced the death rate of COVID-19 was 2% based on the calculation of CFR (=deaths/cases).[16,17] It was substantially underestimated due to assuming no lag days from symptom onset to death and all currently infected cases totally recovered by computing the CFR, particularly, in the early stages of this COVID-19 epidemic.[11]', 'cite_spans': [{'start': 173, 'end': 175, 'mention': '16', 'ref_id': 'BIBREF7'}, {'start': 176, 'end': 178, 'mention': '17', 'ref_id': 'BIBREF8'}, {'start': 406, 'end': 408, 'mention': '11', 'ref_id': 'BIBREF2'}], 'section': 'CFR of 2% reported by WHO and media ::: Introduction', 'ref_spans': []}, {'text': 'Accordingly, many big media platform released news with such statements as below:“The mortality rate for the new coronavirus is about 2.1%, currently far lower than the 9.6% of SARS.”[18] “With a fatality rate of around 2%, which experts agree appears to be the current level for the virus.”[19] Health experts say they are encouraged by the steady rise in the number of recoveries. They take it as evidence that the treatments meted out have been effective and that the virus does not appear to be as deadly as SARS. SARS had a mortality rate of 9.6 percent, and about 2 percent of those reported to have been infected with the new coronavirus have died.”[20] “The Wuhan coronavirus seems to have a low fatality rate, and most patients make full recoveries.”[21] “The number of confirmed cases and deaths indicates that it is around 2 percent, significantly lower than SARS’ 10 percent.”', 'cite_spans': [{'start': 184, 'end': 186, 'mention': '18', 'ref_id': 'BIBREF9'}, {'start': 292, 'end': 294, 'mention': '19', 'ref_id': 'BIBREF10'}, {'start': 657, 'end': 659, 'mention': '20', 'ref_id': 'BIBREF12'}, {'start': 760, 'end': 762, 'mention': '21', 'ref_id': 'BIBREF13'}], 'section': 'CFR of 2% reported by WHO and media ::: Introduction', 'ref_spans': []}, {'text': \"However, “many epidemiologists and people who are following this outbreak closely are assuming that it's probably quite a bit more widespread than the case counts suggest.”[22] The statements of underestimated CFR that was false and[16] motivated us to ask whether this provisional CFR was truly underestimated with false messages to the public and intrigued us to analyze the data with various CFR calculations in this study.\", 'cite_spans': [{'start': 173, 'end': 175, 'mention': '22', 'ref_id': 'BIBREF14'}, {'start': 233, 'end': 235, 'mention': '16', 'ref_id': 'BIBREF7'}], 'section': 'CFR of 2% reported by WHO and media ::: Introduction', 'ref_spans': []}, {'text': 'Bayes theorem (alternatively Bayes law or Bayes rule) describes the probability of an event, based on prior knowledge of conditions that might be related to the event.[23] For instance, the equation,  is to estimate the probability of (P(A1)) based on the condition B. Someone have cancer (A1, or death in this study) caused by their age (P(B)). The age can be used to more accurately assess the probability of cancer than can be done without the knowledge of the age using Bayes theorem to estimation.', 'cite_spans': [{'start': 168, 'end': 170, 'mention': '23', 'ref_id': 'BIBREF15'}], 'section': 'Bayes theorem used for computing CFR ::: Introduction', 'ref_spans': []}, {'text': 'In the current study, we are motivated to apply Bayes theorem to estimate the CFR probability (P(A1|B)) through the likelihood of the observed events (P(B)), such as the level of staff burnouts and healthcare resource scarcities (LSBHRS) across countries/regions of COVID-19.', 'cite_spans': [], 'section': 'Bayes theorem used for computing CFR ::: Introduction', 'ref_spans': []}, {'text': 'We assumed that supportive treatment is crucial for severe respiratory disease. Differences in CFRs may be caused by differences in medical resources among a large epidemic versus care of individual cases. Hence, it is hypothesized that the over-loaded burdens of treating patients and capacities to contain the outbreak (i.e., LSBHRS) may increase the CFR probability (P(A1|B)), where A1 denotes the death, and B stands for the factor (or cause) of the death.', 'cite_spans': [], 'section': 'Bayes theorem used for computing CFR ::: Introduction', 'ref_spans': []}, {'text': 'The aims of the current study are to compare the differences of the proposed CFR formula and design an app that allows better understandings of the daily CFR on the COVID-19 situations.', 'cite_spans': [], 'section': 'The aims of this study ::: Introduction', 'ref_spans': []}, {'text': 'We downloaded COVID-19 outbreak numbers in countries/regions on February 14, 2020, from the Github website[24] that contains information on confirmed cases in >30 Chinese locations and other countries/regions. All downloaded data were publicly deposited on the website.[24] Ethical approval was not necessary for this study because all the data were obtained from the Github website on the Internet.', 'cite_spans': [{'start': 107, 'end': 109, 'mention': '24', 'ref_id': 'BIBREF16'}, {'start': 270, 'end': 272, 'mention': '24', 'ref_id': 'BIBREF16'}], 'section': 'Data source ::: Methods', 'ref_spans': []}, {'text': 'The prior CFR probability (e.g., 0.02) is denoted by the deaths (A1). The remainder (A2) is defined by (1 − CFR) (e.g., 0.98), see Table 1.', 'cite_spans': [], 'section': 'The prior CFR and 1-CFR ::: Bayes theorem used for estimating CFRs ::: Methods', 'ref_spans': [{'start': 137, 'end': 138, 'mention': '1', 'ref_id': 'TABREF0'}]}, {'text': 'The goal is to compute the P(A1|B) (i.e., post-CFR) in terms of the outbreak (LSBHRS) (=P(B)), see Fig. 1. The LSBHRS is referred to as the next section.', 'cite_spans': [], 'section': 'The prior CFR and 1-CFR ::: Bayes theorem used for estimating CFRs ::: Methods', 'ref_spans': [{'start': 104, 'end': 105, 'mention': '1', 'ref_id': 'FIGREF0'}]}, {'text': 'The shares of the outbreak (LSBHRS) (=P(B)) are proportioned in countries/regions based on currently infected cases (defined as the confirmed cases subtracted by deaths and recoveries), indicating the medical loadings on COVID-19. For instance, the daily currently infected cases are reported by 4 designated regions as 20, 30, 50, and 100, respectively. The outbreak (LSBHRS) proportions for 4 regions (i.e., just exampled in this section) are thus computed as 0.1(=20/[20 + 30 + 50 + 100]), 0.15, 0.25, and 0.5 based on Eq. (3), see the computation in Table 1. ', 'cite_spans': [], 'section': 'The proportion of LSBHRS for each region ::: Bayes theorem used for estimating CFRs ::: Methods', 'ref_spans': [{'start': 560, 'end': 561, 'mention': '1', 'ref_id': 'TABREF0'}]}, {'text': 'where n is the number of regions, Total_LSBHRS means the summation of the outbreak (LSBHRS) (i.e., the currently infected cases) in all regions, and LSBHRSj stands for the outbreak (LSBHRS) in a particular region.', 'cite_spans': [], 'section': 'The proportion of LSBHRS for each region ::: Bayes theorem used for estimating CFRs ::: Methods', 'ref_spans': []}, {'text': 'The summation of proportions of deaths (i.e., A1) in all regions is defined in Eq. (4). ', 'cite_spans': [], 'section': 'Proportions of deaths and recoveries for regions ::: Bayes theorem used for estimating CFRs ::: Methods', 'ref_spans': []}, {'text': 'where n is the number of regions, Total_deaths means the summation of death tolls in all regions, and deaths stand for the number of deaths in a particular region.', 'cite_spans': [], 'section': 'Proportions of deaths and recoveries for regions ::: Bayes theorem used for estimating CFRs ::: Methods', 'ref_spans': []}, {'text': 'Similarly, the summation of all recovered proportions (i.e., A2) in all regions is defined in Eq. (5), see Table 1. ', 'cite_spans': [], 'section': 'Proportions of deaths and recoveries for regions ::: Bayes theorem used for estimating CFRs ::: Methods', 'ref_spans': [{'start': 113, 'end': 114, 'mention': '1', 'ref_id': 'TABREF0'}]}, {'text': 'Both proportions (i.e., shares) for the deaths (i.e., A1) and the recoveries (i.e., A2) in each region are assumed (i.e., just exampled in this section) and allocated as the 2 of {0.1, 0.2, 0.3, 0.4} and {0.15, 0.25, 0.3, 0.3} (Eq. (4)), see Table 1.', 'cite_spans': [], 'section': 'Proportions of deaths and recoveries for regions ::: Bayes theorem used for estimating CFRs ::: Methods', 'ref_spans': [{'start': 248, 'end': 249, 'mention': '1', 'ref_id': 'TABREF0'}]}, {'text': 'From Eq. (1), the CFR can be obtained in Eq. (6), see Table 1. ', 'cite_spans': [], 'section': 'To compute the provisional CFR ::: Bayes theorem used for estimating CFRs ::: Methods', 'ref_spans': [{'start': 60, 'end': 61, 'mention': '1', 'ref_id': 'TABREF0'}]}, {'text': 'Referring to the example in this section, the CFR can be computed by the () where P(B|A1) = (0.1∗0.1 + 0.15∗0.2 + 0.25∗0.3 + 0.5∗0.4)=0.315, P(B|A2) = (0.1∗0.15 + 0.15∗0.25 + 0.25∗0.3 + 0.5∗0.3) = 0.2775, Eq. (3).', 'cite_spans': [], 'section': 'To compute the provisional CFR ::: Bayes theorem used for estimating CFRs ::: Methods', 'ref_spans': []}, {'text': 'We can see that the post probability of CFR is increased from the prior probability of 0.02 to 0.023 due to the unequal weights in regions, see scenario (A) in Table 1. The final CFR equals the prior CFR at the end of the outbreak, see scenario (B) in Table 1.', 'cite_spans': [], 'section': 'To compute the provisional CFR ::: Bayes theorem used for estimating CFRs ::: Methods', 'ref_spans': [{'start': 166, 'end': 167, 'mention': '1', 'ref_id': 'TABREF0'}, {'start': 258, 'end': 259, 'mention': '1', 'ref_id': 'TABREF0'}]}, {'text': 'Similarly, the post-CFR for a region can be obtained by Eq. (7) ', 'cite_spans': [], 'section': 'To compute the provisional CFR ::: Bayes theorem used for estimating CFRs ::: Methods', 'ref_spans': []}, {'text': 'where share_A1 is the product of LSBHRS% and  and share_A2 equals the product of LSBHRS% and  see the example (1) in the next section.', 'cite_spans': [], 'section': 'To compute the provisional CFR ::: Bayes theorem used for estimating CFRs ::: Methods', 'ref_spans': []}, {'text': 'Four scenarios are illustrated in the scenario (C) in Table 1.', 'cite_spans': [], 'section': 'Attributes of the provisional CFR ::: Bayes theorem used for estimating CFRs ::: Methods', 'ref_spans': [{'start': 60, 'end': 61, 'mention': '1', 'ref_id': 'TABREF0'}]}, {'text': '(1)  when the prior CFR = 0.04. Tj', 'cite_spans': [], 'section': 'Attributes of the provisional CFR ::: Bayes theorem used for estimating CFRs ::: Methods', 'ref_spans': []}, {'text': '(2)  the prior CFR when all LSBHRS loadings in regions are zero, see scenario (B) in Table 1, and then let P(B|A1) = (0.25∗0.1 + 0.25∗0.2 + 0.25∗0.3 + 0.25∗0.4) = 0.25 and P(B|A2) = (0.25∗0.15 + 0.25∗0.25 + 0.25∗0.3 + 0.25∗0.3) = 0.25;', 'cite_spans': [], 'section': 'Attributes of the provisional CFR ::: Bayes theorem used for estimating CFRs ::: Methods', 'ref_spans': [{'start': 91, 'end': 92, 'mention': '1', 'ref_id': 'TABREF0'}]}, {'text': '(3) The provisional  the prior CFR when the unweighted deaths and the weighted recoveries are applied (see scenario D in Table 1):', 'cite_spans': [], 'section': 'Attributes of the provisional CFR ::: Bayes theorem used for estimating CFRs ::: Methods', 'ref_spans': [{'start': 127, 'end': 128, 'mention': '1', 'ref_id': 'TABREF0'}]}, {'text': 'P(B|A1) = (0.25∗1.0) = 0.25 and P(B|A2) = (0.25∗1.0) = 0.25;', 'cite_spans': [], 'section': 'Attributes of the provisional CFR ::: Bayes theorem used for estimating CFRs ::: Methods', 'ref_spans': []}, {'text': '(4) The provisional  if the surveyed P(A1) and P(A2) of weighted LSBHRS are substantially different such as P(B|A1) = 0.6 and P(B|A2) = 0.1, where surveyed P(A1) (or P(A2)) means the proportion from a survey (e.g., how many percentages are experienced with the severe outbreak [LSBHRS] [e.g., intensive care unit {ICU}] in deaths or in recoveries).', 'cite_spans': [], 'section': 'Attributes of the provisional CFR ::: Bayes theorem used for estimating CFRs ::: Methods', 'ref_spans': []}, {'text': 'As such, we can apply the formula from (1) to (6) to estimate the post-CFR and verify that unequally weighted LSBHRS loadings cause the CFR probability(P(A1|B)) to increase when  in Eq. (6) or  in Eq. (7).', 'cite_spans': [], 'section': 'Attributes of the provisional CFR ::: Bayes theorem used for estimating CFRs ::: Methods', 'ref_spans': []}, {'text': 'We created HTML pages showing the death counts on a choropleth map. All relevant COVID-19 information on the countries/areas can be linked to dashboards on Google Maps when the color is clicked.[25]', 'cite_spans': [{'start': 195, 'end': 197, 'mention': '25', 'ref_id': 'BIBREF17'}], 'section': 'Task 1: Deaths on a choropleth map ::: Methods', 'ref_spans': []}, {'text': 'The features and limitations were compared among the 5 CFR formulae, including', 'cite_spans': [], 'section': 'Task 2: Using the coefficient of variance (CV) to compare the features of formula on CFRs ::: Methods', 'ref_spans': []}, {'text': 'The coefficients of variance (CV = the ratio of the standard deviation to the mean) were applied to measure the relative variability for each CFR. The lower is the CV; the higher stable is the feature of the CFR computation.', 'cite_spans': [], 'section': 'Task 2: Using the coefficient of variance (CV) to compare the features of formula on CFRs ::: Methods', 'ref_spans': []}, {'text': 'A dashboard was developed for daily display of the CFR across various regions using the Kano model.[26,27] In which 3 features are displayed, including the wow feature (top), one dimension quality (middle), and the essential requirement (bottom).', 'cite_spans': [{'start': 100, 'end': 102, 'mention': '26', 'ref_id': 'BIBREF18'}, {'start': 103, 'end': 105, 'mention': '27', 'ref_id': 'BIBREF19'}], 'section': 'Task 4: A dashboard on Google Maps to present the daily CFR ::: Methods', 'ref_spans': []}, {'text': 'The changes between prior and post-CFRs were compared using the Kano model[26,27] when the Bayes theorem was applied to compute the post-CFR. Details are included in Additional File 1 with an MS-Excel dataset.', 'cite_spans': [{'start': 75, 'end': 77, 'mention': '26', 'ref_id': 'BIBREF18'}, {'start': 78, 'end': 80, 'mention': '27', 'ref_id': 'BIBREF19'}], 'section': 'Task 3: The changes between prior-/post-CFRs ::: Methods', 'ref_spans': []}, {'text': 'A choropleth map about the distribution of the deaths in regions is present in Fig. 2. The darker color means a large number of deaths in the region. We can see that Hubei province in China with a darker red color. When the color is clicked, a box will pop up and shows that infected = 51,986, deaths = 1318 (0.03%), and recoveries = 3900. All data were extracted on February 14, 2020.', 'cite_spans': [], 'section': 'Deaths on a choropleth map ::: Results', 'ref_spans': [{'start': 84, 'end': 85, 'mention': '2', 'ref_id': 'FIGREF1'}]}, {'text': 'Three line graphs regarding the cumulative confirmed cases, the daily confirmed cased in the recent 7 days, and the trend was linked for interested readers to examine the details. For more information about the recent COVID-19 situations, readers are invited to click on the link[28] or scan the QR-code in Fig. 2.[29]', 'cite_spans': [{'start': 280, 'end': 282, 'mention': '28', 'ref_id': 'BIBREF20'}, {'start': 315, 'end': 317, 'mention': '29', 'ref_id': 'BIBREF21'}], 'section': 'Deaths on a choropleth map ::: Results', 'ref_spans': [{'start': 312, 'end': 313, 'mention': '2', 'ref_id': 'FIGREF1'}]}, {'text': 'In Fig. 3 we can see that the Bayes estimation (D) has the lowest CV (=0.10) followed by the deaths/confirmed (=0.11) (A), deaths/(deaths + recoveries) (=0.42) (B), combined Bayes and deaths/(cases 7 days ago) (=0.49) (E), and deaths/(cases x days ago) (=0.59) (C), see Table 2. In which, we explained the equations from A to E in details.', 'cite_spans': [], 'section': 'Using CV to compare formula on CFRs ::: Results', 'ref_spans': [{'start': 8, 'end': 9, 'mention': '3', 'ref_id': 'FIGREF2'}, {'start': 276, 'end': 277, 'mention': '2', 'ref_id': 'TABREF1'}]}, {'text': 'Each method will eventually yield an identical CFR over the course of the outbreak. Some converged from a lower percentage of CFR to a higher one (e.g., (A) and (D)); other three (e.g., (B), (C), and (E)) dropped from a higher CFR to a lower one. From Fig. 3, the combined Bayes and deaths/(cases 7 days ago) was recommended instead of the lower CFR WHO claimed and reported by media.', 'cite_spans': [], 'section': 'Using CV to compare formula on CFRs ::: Results', 'ref_spans': [{'start': 257, 'end': 258, 'mention': '3', 'ref_id': 'FIGREF2'}]}, {'text': \"A dashboard was developed in responding to the daily CFR in comparison to each region. The CFR (=3.7% with purple color) greater than the WHO's 2% was shown on our developed dashboards based on data of February 14, 2020, using Bays estimation on axis y and prior CFR on axis x in Fig. 4. From this, we can compare the daily CFRs for each region. Other CFRs (e.g., deaths/confirmed cases on 7 days ago and deaths/[deaths + recoveries]) are present when the bubble of interest is clicked. Interested readers are invited to scan the QR-code in Fig. 4 and see the details by clicking the bubble of interest.\", 'cite_spans': [], 'section': 'A dashboard to present the daily CFR ::: Results', 'ref_spans': [{'start': 285, 'end': 286, 'mention': '4', 'ref_id': 'FIGREF3'}, {'start': 546, 'end': 547, 'mention': '4', 'ref_id': 'FIGREF3'}]}, {'text': 'From Fig. 4, we can see that all bubbles sized by the post-CFR were within the one-dimensional band using the Kano model. Indicating both prior- and post-CFRs were almost equal to each other. The black bubbles would denote the post-CFRs equal to zero even if the prior-CFRs were significantly higher toward the right-hand side on axis x. The highest prior-CFR (=33%) is the Philippines (on February 14, 2020) because 3 were infected and 1 died. The highest post = CFR was Hubei (China) with post-CFR = 3.66% higher than the prior-CFR = 2.54%, indicating the heavy loadings of the outbreak (LSBHRS) leads to a higher probability of the post-CFR.', 'cite_spans': [], 'section': 'A dashboard to present the daily CFR ::: Results', 'ref_spans': [{'start': 10, 'end': 11, 'mention': '4', 'ref_id': 'FIGREF3'}]}, {'text': 'It is worth noting that the prior-CFR is derived from the deaths/confirmed by today, and the post-CFR is yielded by Eq. (6). The post-CFR (3.7%) greater than the prior CFR of 2.2% on February 14 and equal to 3.7% on March 13 was evident of the outbreak (LSBHRS) increasing the CFR due to weighted scheme applied, see Table 1.', 'cite_spans': [], 'section': 'A dashboard to present the daily CFR ::: Results', 'ref_spans': [{'start': 323, 'end': 324, 'mention': '1', 'ref_id': 'TABREF0'}]}, {'text': 'The recent CFRs were online computed in Fig. 5, where prior CFRs are shown on axis x and post-CFRs on axis y. The highest prior-CFR was located in Suan with 1 died and 1 confirmed case on March 13, 2020, see the right-top corner, Fig. 5. All bubbles are sized by the post-CFRs and colored the areas in the Kano diagram. For instance, bubbles in the one-dimensional area are colored in blue.', 'cite_spans': [], 'section': 'The recent CFRs using Bayes theorem ::: Results', 'ref_spans': [{'start': 45, 'end': 46, 'mention': '5', 'ref_id': 'FIGREF4'}, {'start': 235, 'end': 236, 'mention': '5', 'ref_id': 'FIGREF4'}]}, {'text': 'The global with the big purple bubble had an increase from the prior CFR (=3.71%) to the post CFR (3.27%) in comparison to Hubei (China) having a decrease from the prior CFR = 4.521% to the post CFR = 3.52% on March 13, 2020, due to the outbreak (LSBHRS) in deaths relatively lower than that in recoveries.', 'cite_spans': [], 'section': 'The recent CFRs using Bayes theorem ::: Results', 'ref_spans': []}, {'text': 'Three countries outside the one-dimensional area are Italy, Spain, and France with significantly relative post-CFRs more than the prior-CFRs. The recent CFRs are shown on Google Maps when the QR-code is scanned in Figs. 4 and 5.', 'cite_spans': [], 'section': 'The recent CFRs using Bayes theorem ::: Results', 'ref_spans': [{'start': 220, 'end': 221, 'mention': '4', 'ref_id': 'FIGREF3'}, {'start': 226, 'end': 227, 'mention': '5', 'ref_id': 'FIGREF4'}]}, {'text': 'We observed that the Bayes estimation [D] has the lowest CV (=0.10) followed by the deaths/confirmed (=0.11), deaths/(deaths + recovered) (=0.42) [B], Bayes based on deaths/(cases × days ago) (=0.49) [E], and deaths/(cases 7 days ago) (=0.59) [C]. All final CFRs will be equal. The CFR (3.7%) greater than the prior-CFR of 2.2% was evident in LSBHRS impact on the CFR. A dashboard was created to present the CFRs.', 'cite_spans': [], 'section': 'Principle findings ::: Discussion', 'ref_spans': []}, {'text': 'A dashboard was designed to respond to the daily overall CFR. Each individual CFR can be viewed by clicking the bubble of interest.[30,31]', 'cite_spans': [{'start': 132, 'end': 134, 'mention': '30', 'ref_id': 'BIBREF23'}, {'start': 135, 'end': 137, 'mention': '31', 'ref_id': 'BIBREF24'}], 'section': 'Principle findings ::: Discussion', 'ref_spans': []}, {'text': 'We found that the method (D) has the lowest CV (=0.10), and the (C) has the highest CV (=0.59). The (E), Bayes based on deaths/(cases 7 days ago) (=0.49), is suggested to calculate the provisional CFR to the general public. The reason for use is illustrated in the next section.', 'cite_spans': [], 'section': 'Principle findings ::: Discussion', 'ref_spans': []}, {'text': 'CFR is a measure of the “severity” of a disease and is defined as the proportion of cases of a specified disease or condition which are fatal.[16] We believe that WHO voiced the CFR (=2%)[17] with misinformation (i.e., underestimation) which is only based on confirmed cases, ignoring the lag time between the onset of symptoms and progression to deaths, as well as in mild cases assuming the LSBHRS being equal across countries/regions.', 'cite_spans': [{'start': 143, 'end': 145, 'mention': '16', 'ref_id': 'BIBREF7'}, {'start': 188, 'end': 190, 'mention': '17', 'ref_id': 'BIBREF8'}], 'section': 'What this finding adds to what we already knew ::: Discussion', 'ref_spans': []}, {'text': 'WHO did not explain to the media and public what CFR implicates. The understated CFR has been shared worldwide.[18–22] It is clear that the conventional CFR defined as deaths/cases in the current fast-growing epidemic gives us a totally useless and unreliable figure when it comes to an outbreak like COVID-19 in the past 50 days since February 14, 2020.', 'cite_spans': [{'start': 112, 'end': 114, 'mention': '18', 'ref_id': 'BIBREF9'}, {'start': 115, 'end': 117, 'mention': '22', 'ref_id': 'BIBREF14'}], 'section': 'What this finding adds to what we already knew ::: Discussion', 'ref_spans': []}, {'text': 'The real fatality rate is hard to estimate because we have many currently infected cases yet known the distributions on age, sex, and other factors such as the proportion of severity in intensive care units (ICU) and patients with multiple comorbidities, and the CFR taking into account those clinically diagnosed cases as confirmed cases determined in Hubei since February 14, 2020.[32]', 'cite_spans': [{'start': 384, 'end': 386, 'mention': '32', 'ref_id': 'BIBREF25'}], 'section': 'What this finding adds to what we already knew ::: Discussion', 'ref_spans': []}, {'text': 'What WHO and media did by publishing CFR (=2%) has served to avoid public panic for a period of time by displaying the comparison of data with the true CFRs of SARS (9.6%), MERS (34.4%), Ebola (73%), and H1NI (0.4%).[5–9]', 'cite_spans': [{'start': 217, 'end': 218, 'mention': '5', 'ref_id': 'BIBREF30'}, {'start': 219, 'end': 220, 'mention': '9', 'ref_id': 'BIBREF34'}], 'section': 'What this finding adds to what we already knew ::: Discussion', 'ref_spans': []}, {'text': 'However, true and final CFRs are usually much bigger than the provisional CFRs according to the SARS experience: when WHO first reported daily statistics, the SARS death rate was about 2%. It was 2.4% on March 17 and 1.8% on March 18 (in 2003).[33] In April 2003, CFR was increased to 5.6% instead of an initial 1.8%. The final CFR in the middle of the summer (2003) raised up to almost 10%.[33]', 'cite_spans': [{'start': 245, 'end': 247, 'mention': '33', 'ref_id': 'BIBREF26'}, {'start': 392, 'end': 394, 'mention': '33', 'ref_id': 'BIBREF26'}], 'section': 'What this finding adds to what we already knew ::: Discussion', 'ref_spans': []}, {'text': 'That is the reason why we suggested using the Bayes based on deaths/(cases 7 days ago) in reporting the provisional CFR to the general public. Through this, we can avoid the negative effect of the underestimated CFR and prepare the population, including doctors, authorities, and epidemiologists, etc., to face the real threat of COVID-19.', 'cite_spans': [], 'section': 'What this finding adds to what we already knew ::: Discussion', 'ref_spans': []}, {'text': 'Dr Michael Ryan, Executive Director of WHO, addressed that the COVID-19 fatality rate is 2% in a press conference on January 29, 2020.[16] However, most of the confirmed cases of this novel coronavirus are still hospitalized, many of them in ICUs. The virus does not kill a person immediately, hence creating a lag between infection confirmation and death.[16] For example, only patients that had died by today (February 14, 2020) would have been taken into account in the provisional CFR, but these individuals would have been registered as “confirmed cases” a few days or more than a week ago. The provisional CFR is, therefore, underestimated due to the potential deaths of currently confirmed cases in the next couple of days (a.k.a. lag days).[34] As the number of death increased, the death rate has also steadily risen, leaving the officials worried, based on the CFR trend calculated as deaths/cases, see Fig. 3(upper). Due to the imprecise and carelessness in the calculation, the media has misled the public that “CFR of 2% on COVID-19 is less than the 10% in SARS.”[16–21]', 'cite_spans': [{'start': 135, 'end': 137, 'mention': '16', 'ref_id': 'BIBREF7'}, {'start': 357, 'end': 359, 'mention': '16', 'ref_id': 'BIBREF7'}, {'start': 749, 'end': 751, 'mention': '34', 'ref_id': 'BIBREF27'}, {'start': 1077, 'end': 1079, 'mention': '16', 'ref_id': 'BIBREF7'}, {'start': 1080, 'end': 1082, 'mention': '21', 'ref_id': 'BIBREF13'}], 'section': 'What it implies and what should be changed ::: Discussion', 'ref_spans': [{'start': 918, 'end': 919, 'mention': '3', 'ref_id': 'FIGREF2'}]}, {'text': 'Although we compared the final CFR for the SARS epidemic with current CFR for COVID-19 on the same basis of deaths/cases, what has been forgotten is that CFR was 1.8% for SARS on an early stage in 2003 epidemic calculated using this underestimated method. The current calculation of CFR for a growing epidemic stage is not suitable for representing the final CFR. An adjusted CFR is required in an epidemic to better elucidate the outbreak.', 'cite_spans': [], 'section': 'What it implies and what should be changed ::: Discussion', 'ref_spans': []}, {'text': 'We suggest that in a fast-growing epidemic, CFR cannot be used at all when referring to CV or the slop of the CFR trend (e.g., in Fig. 3). If the true CFR managed to reach over 10% while the provisional CFR was just under 3%, it makes no sense to use such definition in the calculation of CFR (deaths/cases).', 'cite_spans': [], 'section': 'What it implies and what should be changed ::: Discussion', 'ref_spans': [{'start': 135, 'end': 136, 'mention': '3', 'ref_id': 'FIGREF2'}]}, {'text': 'The best way to calculate the provisional CFR is based on cases that already concluded by either death or recovery. However, the epidemic is spreading fast, and there is not enough time to collect data on how many patients of first, say 100, patients recovered and died by any particular day. Examining the interception of the 2 trends (Fig. 6) might be a reference to predict the final CFR, which should be evident in the future. With this change, it is hopeful that the public would not be misinformed again.', 'cite_spans': [], 'section': 'What it implies and what should be changed ::: Discussion', 'ref_spans': [{'start': 342, 'end': 343, 'mention': '6', 'ref_id': 'FIGREF5'}]}, {'text': 'In our suggested CFR calculation of the method of Bayes based on deaths/(cases 7 days ago), 3 pivotal components have been focused in the current study: assuming unequal LSBHRS proportions (based on the daily no-outcome-yet cases) across all countries/regions in COVID-19; using Bayes theorem to adjust the daily provisional CFR and observing whether the CFR trend has been flatted at stationarity phase (i.e., approaching real CFR); considering the lag time between symptom to death as the thread alerted at the fast-growing (i.e., beginning) period.', 'cite_spans': [], 'section': 'Strengths of this study ::: Discussion', 'ref_spans': []}, {'text': 'Although it is hard to accurately estimate the provisional CFR because we do not know the distributions of age, severity, number in ICU, etc., an appropriate death lag time (e.g., 7 days) was designed in this study. The longer this estimated period is, the higher the provisional CFR will be.[3,16]', 'cite_spans': [{'start': 293, 'end': 294, 'mention': '3', 'ref_id': 'BIBREF22'}, {'start': 295, 'end': 297, 'mention': '16', 'ref_id': 'BIBREF7'}], 'section': 'Strengths of this study ::: Discussion', 'ref_spans': []}, {'text': 'As with all forms of Web-based technology, advances in health communication technology are happening every moment.[35] The provisional CFR shown on a dashboard is practical and worth replicating in other fields of a disease outbreak, as we demonstrated in Figs. 4 and 5. Interested readers are recommended to scan the QR codes in Figs. 4–6 and read the details about the corresponding line graph on trends of the outbreak.', 'cite_spans': [{'start': 115, 'end': 117, 'mention': '35', 'ref_id': 'BIBREF28'}], 'section': 'Strengths of this study ::: Discussion', 'ref_spans': [{'start': 262, 'end': 263, 'mention': '4', 'ref_id': 'FIGREF3'}, {'start': 268, 'end': 269, 'mention': '5', 'ref_id': 'FIGREF4'}, {'start': 336, 'end': 337, 'mention': '4', 'ref_id': 'FIGREF3'}, {'start': 338, 'end': 339, 'mention': '6', 'ref_id': 'FIGREF5'}]}, {'text': 'Our study has some limitations. First, although the data were downloaded from Google Sheet on a daily basis, we are concerned with different criteria for determining confirmed cases among regions that affect the CFR. For instance, the confirmed cases are from clinically diagnosed cases in Hubei since February 14, 2020.[32]', 'cite_spans': [{'start': 321, 'end': 323, 'mention': '32', 'ref_id': 'BIBREF25'}], 'section': 'Limitations and future studies ::: Discussion', 'ref_spans': []}, {'text': 'Second, although we applied Bayes theorem to estimate the provisional CFR using Eqs. from (1) to (5), the post-CFR calculated by Bayes theorem (Eq. (6)) is strongly related to the prior = CFR. The result due to LSBHRS among regions drives a little bit of increase compared with the CFR calculated by the deaths over cases assuming all medical resources and staff burnout are equal. The Bayes based on deaths/(cases 7 days ago) suggested in this study is worthy of investigation in the future.', 'cite_spans': [], 'section': 'Limitations and future studies ::: Discussion', 'ref_spans': []}, {'text': 'Third, the study was based on the data from the Github website.[24] The Google team has announced that the service will be terminated and transformed into Github,[24] providing the same information. Due to the download limit, the Google team will soon stop the update of this Google Sheet, which might affect the data being transferred effectively and efficiently on the daily calculated CFR (e.g., in Figs. 4 and 5).', 'cite_spans': [{'start': 64, 'end': 66, 'mention': '24', 'ref_id': 'BIBREF16'}, {'start': 163, 'end': 165, 'mention': '24', 'ref_id': 'BIBREF16'}], 'section': 'Limitations and future studies ::: Discussion', 'ref_spans': [{'start': 408, 'end': 409, 'mention': '4', 'ref_id': 'FIGREF3'}, {'start': 414, 'end': 415, 'mention': '5', 'ref_id': 'FIGREF4'}]}, {'text': 'Fourth, the post-CFRs (calculated by Bayes theorem) in Fig. 4 confused us with the gap from the prior-CFRs (resulted from the deaths/confirmed by today), such as the Philippines (0%: 33%) with confirmed = 3, deaths = 1, and recovered = 1. Owing to the LSBHRS (=[3-1-1]/64,460) substantially relatively less than Hubei (China) (=[51,986-1318-3900]/64,460) (3.66%: 2.54% with confirmed = 51,986, deaths = 1318, recovered = 3900), the expected CFR for the Philippines thus sharply decreases after computation based on Bayes theorem.', 'cite_spans': [], 'section': 'Limitations and future studies ::: Discussion', 'ref_spans': [{'start': 60, 'end': 61, 'mention': '4', 'ref_id': 'FIGREF3'}]}, {'text': 'Finally, we suggest examining the interception of the 2 trends (Fig. 6) as a reference to the final CFR, which should be further verified in the future.', 'cite_spans': [], 'section': 'Limitations and future studies ::: Discussion', 'ref_spans': [{'start': 69, 'end': 70, 'mention': '6', 'ref_id': 'FIGREF5'}]}, {'text': 'We suggest examining the interception of the 2 trends of the Bayes based on both deaths/(cases 7 days ago) and death/confirmed cases as a reference to the final CFR. An app developed for displaying the provisional CFR with the 2 CFR trends is encouraged to ameliorate the underestimated CFR reported by WHO and media.', 'cite_spans': [], 'section': 'Conclusion', 'ref_spans': []}, {'text': 'The authors thank Enago (www.enago.tw) for the English language review of this manuscript.', 'cite_spans': [], 'section': 'Acknowledgments', 'ref_spans': []}, {'text': 'Tsair-Wei Chien developed the study concept and design. Tsair-Wei Chien, Chi-Sheng Chang, Jui-Chung John Lin, and Yu-Tsen Yeh analyzed and interpreted the data. Shu-Chun Kuo monitored the process of this study and helped in responding to the reviewers’ advice and comments. Tsair-Wei Chien drafted the manuscript, and all authors provided critical revisions for important intellectual content. The study was supervised by Shu-Chun Kuo. All authors read and approved the final manuscript.', 'cite_spans': [], 'section': 'Author contributions', 'ref_spans': []}], 'paragraphs': [{'text': 'When a new disease (e.g., COVID-19) starts to spread, many questions emerge: how is the virus transmitted, how long is the incubation period, what is the influence of those asymptomatically infected, what is the definite reproductive number (i.e., so-called R0), how long does viral shedding persist after symptoms faded, who is at risk for a severe course, and how high is the case fatality rate (CFR), defined as the total number of deaths known by today divided by the total number of known cases confirmed by today.[7] The CFR is related to the questions: how deadly is this? and how many people will die in this outbreak? SARS, MERS, Ebola, and H1NI yielded a real CFR of 9.6%, 34.4%,73%, and 0.4%, respectively,[5–9] the CFR for COVID-19 is of interest to numerous researchers.[7,10,11]', 'id': '00001'}], 'title': 'The computation of case fatality rate for novel coronavirus (COVID-19) based on Bayes theorem: An observational study', 'sha': '4d70be0e0845026d0de8e7ac2c78e017d2dc3f47', 'doi': '10.1097/md.0000000000019925', 'score': 8.846799850463867}, {'rank': 263, 'paragraphs': [{'text': 'Yan Bei et al [20] undertook research into cases where it was assumed that the virus had been passed by an asymptomatic carrier. This research looked at a family from Anyang, China, of which five members were suffering COVID-19 pneumonia who had, prior to developing symptoms, been in contact with an asymptomatic member of the family who had travelled to see them from Wuhan, the origin of the pandemic. The timeline they uncovered implies that the coronavirus could have been passed on by this asymptomatic carrier. The first patient to develop symptoms incubated for 19 days, a long period but one which falls within the reported range (0 to 24 days). This patient initially produced a negative return for the RT-PCR test; RT-PCR is a common test for diagnostic virology and does not often return false positives, so her second result from this test was probably not a false positive, and so it was assumed that she was infected with the coronavirus that is responsible for COVID-19.', 'id': '00015'}], 'title': 'The estimations of the COVID-19 incubation period: a systematic review of the literature', 'sha': '09ff6011e4e95bff0b9221c89ffdb91d958e4ef1', 'doi': '10.1101/2020.05.20.20108340', 'score': 8.843400001525879}, {'rank': 264, 'paragraphs': [{'text': 'By pooling individual data from seven countries, we estimated the mean incubation period of COVID-19 to be 7.44 days, latent period 2.52 days, serial interval 6.70 days, time point of All rights reserved. No reuse allowed without permission. perpetuity. preprint (which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in Our estimates of incubation period and serial interval are longer than most of the previous estimates. [2] [3] [4] [5] [6] [7] There are several possible reasons for the difference. First, the sample size is generally small in previous studies, 2, 4, 6, 7, [9] [10] [11] [12] but much larger in the present one. Second, most cases in previous studies had a long or even unclear interval of exposure, making it difficult to determine the exact exposure date 3, 5, 13 and giving rise to error. By contrast, this study followed strict inclusion criteria regarding exposure period and method to determine exposure date to ensure the potential error in the estimates be small (< 0.3 day, according to the sensitivity analysis). Third, the approaches to determination of transmission order within clusters are different. For the cases involving cluster infection, which represent the majority of all cases, a common practice of previous studies was to take the case with earliest date of symptom onset as infector and others as infectee. 9 However, we deemed it inappropriate in view of the varying incubation periods of individuals and potential pre-symptomatic or asymptomatic transmission.', 'id': '00038'}], 'title': 'Epidemiological parameters of coronavirus disease 2019: a pooled analysis of publicly reported individual data of 1155 cases from seven countries', 'sha': '3ece7750f80832bf9cc368c5138df12fd19183b9', 'doi': '10.1101/2020.03.21.20040329', 'score': 8.836299896240234}, {'rank': 265, 'abstract': '', 'body_text': [{'text': 'upholds technocracy and delegates significant authority and broad discretion to health experts in the Executive Branch to steer the country in times of emergency, entrenching an administrative state 8 in this young democracy.', 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': 'In January of this year, when China decided to lock down Wuhan due to COVID-19, Taiwan took an exceptionally precautionary approach based on three major factors 9 : Taiwan\\'s decades-long exclusion from the international health community without World Health Organization (WHO) membership/observership and access to other arms of the United Nations (UN) 10 ; traumatic experiences during the 2003 severe acute respiratory syndrome (SARS) outbreak, where regulatory failure led to a wave of reforms 11 ; and the island\\'s geographical proximity to and geopolitical suspicion of China. Because international exclusion has prevented Taiwan from benefitting through information-sharing and collective efforts underpinned by the multilateral WHO framework, 12 the country has developed a \"self-help\" approach in countering possible public health threats, which has partly resulted in a heavy reliance on 10 D Shapiro, \"Taiwan Shows Its Mettle in Coronavirus Crisis, While the WHO is MIA\" (Brookings, 19 March 2020) <https://www.brookings.edu/blog/order-from-chaos/2020/03/19/taiwan-shows-its-mettle-in-coronavirus-crisis-whilethe-who-is-mia/> (last accessed 21 March 2020); G van der Wees, \"Taiwan and the World Health Assembly\" (The Diplomat, 10 May 2016) <https://thediplomat.com/2016/05/taiwan-and-the-world-health-assembly/> (last accessed 21 March 2020); H Tan, \"Taipei Lashes Out at China for Blocking Taiwan\\'s Access to the World Health Organization\" (CNBC, 6 February 2020) <https://www.cnbc.com/2020/02/06/coronavirus-taiwan-lashes-out-atchina-for-blocking-who-access.html> (last accessed 24 March 2020); \"China\\'s Taiwan Policy Hurts World Health\" (WSJ, 13 February 2020) <https://www.wsj.com/articles/chinas-taiwan-policy-hurts-world-health-11581618021> (last accessed 24 March 2020). 11 K-T Chen et al, \"SARS in Taiwan: An Overview and Lessons Learned\" (2005) 9(2) International Journal of Infectious Diseases 77; \"Record SARS Toll in Taiwan After WHO Alert\" (The Irish Times, 22 May 2003) <https:// www.irishtimes.com/news/record-sars-toll-in-taiwan-after-who-alert-1.478506> (last accessed 21 March 2020); E Graham-Harrison, \"Experience of SARS a Key Factor in Countries\\' Response to Coronavirus\" (The Guardian, 15 Mar 2020) <https://www.theguardian.com/world/2020/mar/15/experience-of-sars-key-factor-in-response-to-coronavirus> (last accessed 21 March 2020); G McGregor, \"SARS Taught Taiwan How to Contain the Coronavirus Outbreak\" (Fortune, 15 March 2020) <https://fortune.com/2020/03/15/coronavirus-taiwan-casesresponse/> (last accessed 21 March 2020). 12 EL-L Chu, \"The World Can Benefit From Taiwan\\'s WHO Participation\" (The Diplomat, 11 February 2020) <https://thediplomat.com/2020/02/the-world-can-benefit-from-taiwans-who-participation/> (last accessed 21 March 2020); N Kassam, \"As Wuhan Virus Spreads, Taiwan Has No Say at WHO\" (Foreign Policy, 22 January 2020) <https://foreignpolicy.com/2020/01/22/china-health-coronavirus-wuhan-virus-spreads-taiwan-no-say-who/> (last accessed 21 March 2020); M Yun, \"How Taiwan Is Containing Coronavirus -Despite Diplomatic Isolation by China\" (The Guardian, 13 March 2020) <https://www.theguardian.com/world/2020/mar/13/how-taiwan-iscontaining-coronavirus-despite-diplomatic-isolation-by-china> (last accessed 21 March 2020); B Blanchard, \"Taiwan Says It Didn\\'t Need China\\'s Permission for WHO Meeting\" (Reuters, 12 February 2020) <https://www. reuters.com/article/us-china-health-taiwan/taiwan-says-it-didnt-need-chinas-permission-for-who-meeting-idUSKBN2060JZ> (last accessed 24 March 2020). technocratic decision-making. 13 This sense of exclusion and the need for \"self-help\" was strengthened by bitter SARS experiences, when the WHO offered limited help. 14 During the SARS outbreak, Taiwan lost 73 lives, had hundreds of its citizens hospitalised and experienced the controversial lockdown of Ho-Ping Hospital in Taipei, 15 and such a history has prompted Taiwan to take a highly precautionary approach. Furthermore, COVID-19 and SARS share the same origin -Chinaand everything related to China seems controversial in Taiwan (even more so after the current administration took power in 2016) because of the complex historical, social, economic and political relationship between the two countries. China earned a track record for concealing information during the SARS outbreak, 16 and it is widely believed in Taiwan that China has done the same regarding COVID-19. Economically, Taiwanese enterprises invest intensively in Wuhan and other provinces in China, which results in frequent travel between the two countries. Socially, a number of Taiwanese marry Chinese citizens, and the Lunar New Year is a time for family reunions, which may have contributed to virus transmission from China to Taiwan. Finally, and most importantly, the current administration in Taiwan is suspicious of China, which leads it to take a sober look at the information and news released by China and to take a precautionary approach in countering COVID-19.', 'cite_spans': [{'start': 2563, 'end': 2565, 'text': '12', 'ref_id': None}, {'start': 3579, 'end': 3581, 'text': '13', 'ref_id': None}, {'start': 4340, 'end': 4342, 'text': '16', 'ref_id': None}], 'ref_spans': [], 'section': ''}, {'text': 'However, Taiwan\\'s proactive and precautionary response to COVID-19 might at the same time pose a threat to this young democracy. The critical issue here is how a constitutional democracy survives a public health emergency with the unprecedented magnitude of COVID-19. To be sure, governments worldwide have adopted a wide range of regulatory measures, 17 such as social distancing and stay-home orders, 18 mass testing, 19 contact-tracing and surveillance, 20 the shuttering of non-essential services 21 13 Taiwan\\'s huge foreign reserve is also reflective of this \"self-help\" approach. and even border closures and national shutdowns. 22 The legality and legitimacy of such regulatory actions may be examined pursuant to different norms and standards underlying a country\\'s respective political system, legal tradition and social context. On the one hand, an emergency calls for stronger and swifter governmental actions to safeguard public health and national security in a manner that may not allow sufficient time for parliamentary deliberation or traditional administrative procedures. 23 The urgency to take effective and efficient measures oftentimes justifies a wide margin of executive discretion. On the other hand, measures adopted in times of emergency tend to linger and become normalised. Emergencies frequently serve to expand the power of executive agencies and side-line legislative or even judicial gatekeepers or lead to irreversible harm to fundamental human rights. 24 In Taiwan, among those measures adopted to combat COVID-19, the Special Act for Prevention, Relief and Revitalization Measures for Severe Pneumonia with Novel Pathogens (COVID-19 Special Act) seems to have triggered controversial questions about the bending and even breaking of fundamental human rights and core legal principles. This paper therefore aims to offer an anatomy of Taiwan\\'s regulatory actions taken in response to the global COVID-19 pandemic, assess their implications for risk regulation and governance in a global context and urge a reimagining of the administrative state in thehopefullypost-COVID-19 world.', 'cite_spans': [{'start': 635, 'end': 637, 'text': '22', 'ref_id': None}, {'start': 1090, 'end': 1092, 'text': '23', 'ref_id': None}, {'start': 1486, 'end': 1488, 'text': '24', 'ref_id': None}], 'ref_spans': [], 'section': ''}, {'text': 'Public health emergencies such as COVID-19 usually compel governments to steer, coordinate and respond with various measures. Taiwan\\'s CDC Act, which was carefully re-evaluated and amended after the SARS outbreak, serves as the primary legal basis. 25 Overall, executive agencies are granted considerable power to adopt various regulatory measures to cope with imminent epidemics and public health threats, and such agencies enjoy a substantial margin of discretion under urgent circumstances. Yet the government has so far treated the COVID-19 outbreak as a 22 See, eg, Cheung, supra, note 20; \"Coronavirus: Travel Restrictions, Border Shutdowns by Country\" (Al-Jazeera 22 March 2020) <https://www.aljazeera.com/news/2020/03/coronavirus-travel-restrictions-border-shutdowns-country-200318091505922.html> (last accessed 22 March 2020). 23 Emergencies, such as disease outbreaks, natural disasters or terrorist attacks, often call for considerable discretion conferred to the administrative agencies, resulting in \"the hour of the executive\" for a legal system. See national security issue and has adopted extraordinary interventions reflecting such a framing. 26 While these interventions have been largely effective and have garnered majority support from society, they might also risk breaking fundamental rule of law principles, causing irreversible harms to human rights and bending the country\\'s constitutional order with lasting and systematic effects. 27 Here, we examine some key measures taken by the government in Taiwan along three essential dimensions of risk regulation, providing an anatomy of Taiwan\\'s actions in its whole-of-society and global approach.', 'cite_spans': [{'start': 249, 'end': 251, 'text': '25', 'ref_id': None}, {'start': 559, 'end': 561, 'text': '22', 'ref_id': None}, {'start': 836, 'end': 838, 'text': '23', 'ref_id': None}, {'start': 1061, 'end': 1064, 'text': 'See', 'ref_id': None}, {'start': 1459, 'end': 1461, 'text': '27', 'ref_id': None}], 'ref_spans': [], 'section': \"II. TAIWAN'S REGULATORY ACTIONS: AN ANATOMY\"}, {'text': 'To address a rapidly changing pandemic of global concern that is uncertain and dynamic in nature, the right timing for regulatory action is always a guesstimate and can only be evaluated in hindsight. Acting too early and taking excessive precautions risk being criticised ex post as paranoid and unduly disrupting the market and society; acting too late may cause irreversible damage and miss the window of opportunity to tackle the problem. Therefore, the key to successful pandemic response is to determine how and when a specific event becomes a public health emergency with limited scientific data. 28 This is by no means an easy task.', 'cite_spans': [], 'ref_spans': [], 'section': 'Advanced but informed risk assessment?'}, {'text': \"Taiwan has learned from its SARS experiences that timely and accurate determinations about a public health emergency may pose a formidable challenge if the roles and responsibilities in the decision-making process are not clearly defined (eg tensions and overlapping jurisdictions between local and central governments) and if scientific evidence is inadequate, complicated or unavailable (eg exclusion from the WHO and failure to receive critical information). 29 To establish a unified authority and decisionmaking process, 30 Article 8 of the CDC Act grants the Ministry of Health and Welfare, particularly the Centers for Disease Control, the exclusive power and responsibility to determine public health emergencies and announce them with a single voice. This further entrenches the technocratic anchor in Taiwan's legal framework for communicable disease control, which relies heavily on the medical profession, as it has long enjoyed prestigious status in society and has generally been regarded as a trusted expert voice in policy-making.\", 'cite_spans': [{'start': 462, 'end': 464, 'text': '29', 'ref_id': None}], 'ref_spans': [], 'section': 'Advanced but informed risk assessment?'}, {'text': 'When the world saw only uncertain signs of mysterious pneumonia cases in Wuhan in December 2019, Taiwan treated them with the utmost urgency. 31 On 31 December 2019, 26 S Hameiri and L Jones, Governing Borderless Threats (Cambridge, Cambridge University Press 2015) pp 25-26. 27 A Neve, \"Human Rights Muse Remain at the Heart of Our Response to Coronavirus Crisis\" (The Star, 19 March 2020) <https://www.thestar.com/opinion/contributors/2020/03/19/human-rights-must-remain-at-the-very-heart-ofour-response-to-the-coronavirus-crisis.html> (last accessed 26 March 2020). 28 when China silenced doctors from disclosing information, the Taiwanese government warned China and the WHO International Health Regulations (IHR) Contact Points of the danger of human-to-human transmission 32 and began to send officials to board all direct flights from Wuhan and inspect passengers for fever or pneumonia symptoms, prepared contact tracking and tracing mechanisms and surveyed the availability of medical supplies. 33 Regardless of the insufficiency of scientific evidence and clinical data at the time, the government spent great effort on assessing whether the disease constituted an emergency. 34 Taiwan sent two experts to Wuhan to obtain more information on the outbreak. 35 On 15 January 2020, Taiwan officially determined COVID-19 to be a Category V Communicable Disease, 36 setting up the Central Epidemics Command Center (CECC) on 20 January to coordinate all control measures across various agencies. 37 Following an advanced risk assessment, and recognising the need to act swiftly, the CECC moved to prevent Wuhan residents from entering the country on 23 January 2020, suspended tours to China on 25 January and, in the end, banned all Chinese visitors on 6 February. 38 Yet, such an \"advanced\" risk assessment was not necessarily science-based or informed by sufficient clinical data, but was rather a multifaceted decision based on a complex weighing and balancing of technical, social, economic and political factors. First, because of China\\'s concealment of information in the early stages of the outbreak and the WHO\\'s failure to take appropriate action, there was very little scientific evidence or international guidance for risk assessment. Second, due to the tensions between Taiwan and China and the former\\'s exclusion from the WHO, the management of risks in public health emergencies is not merely a technical and scientific issue, but also a socioeconomic and even a political task. The government is forced to make decisions in the absence of solid data and to take additional steps ahead of emergencies by using its own judgement. Thus, public health risk assessment goes beyond disease control and extends to holistic social, economic and 32 See Y Lee and B Blanchard, \"Taiwan Says WHO Ignored Its Coronavirus Questions at Start of Outbreak\" (US News, 24 March 2020) <https://www.usnews.com/news/world/articles/2020-03-24/taiwan-says-who-ignoredits-coronavirus-questions-at-start-of-outbreak> (last accessed 24 March 2020); \"Taiwan Says WHO Failed to Act on Coronavirus Transmission Warning\" (Financial Times, 20 March 2020) <https://www.usnews.com/news/world/ articles/2020-03-24/taiwan-says-who-ignored-its-coronavirus-questions-at-start-of-outbreak> (last accessed 26 March 2020 political management. 39 This reflects a classic paradox in risk regulation 40 : if a country adopts a precautionary approach and engages in advanced preparedness, such a decision is thus to be made in the absence of sufficient data and scientific certainty and to run counter to the principle of scientific-based decision-making. Transparency and reasoning requirements are likely to be side-lined in the decision-making process. In times of emergency, such a paradox is exacerbated by the urgency and uncertainty of an event and may only be partly addressed by ex post review and accountability mechanisms, which are also prone to hindsight bias.', 'cite_spans': [{'start': 569, 'end': 571, 'text': '28', 'ref_id': None}, {'start': 1004, 'end': 1006, 'text': '33', 'ref_id': None}, {'start': 1266, 'end': 1268, 'text': '35', 'ref_id': None}, {'start': 1500, 'end': 1502, 'text': '37', 'ref_id': None}, {'start': 1770, 'end': 1772, 'text': '38', 'ref_id': None}, {'start': 2757, 'end': 2759, 'text': '32', 'ref_id': None}, {'start': 3322, 'end': 3324, 'text': '39', 'ref_id': None}], 'ref_spans': [], 'section': 'Advanced but informed risk assessment?'}, {'text': 'Risk communication plays an important role in tackling public health emergencies. Effective and efficient risk communication informs people of potential hazards and ensures public trust, which serves as the very premise of successful control measures.', 'cite_spans': [], 'ref_spans': [], 'section': 'Effective and efficient risk communication'}, {'text': 'In order for complicated technical information to be conveyed to the lay public and subsequently transformed into concrete disease control measures, the Taiwanese government has employed both formal and informal channels (such as social media platforms and messengers) and has also worked with civic technology groups to facilitate two-way communication between the CECC and the public on a daily basis. 41 Information about new imported and indigenous cases, global prevalence, new and revised control policies, clarification of misinformation and even the availability of medical masks and household necessities is provided at least once a day in a national press conference, followed by an open session for questions from the media and the public. 42 The government has also taken measures to counter fake news, or disinformation. Taiwan has fought a long battle with fake news (China is the primary origin) since 2012 and has modernised its legislative and regulatory responses. 43 Citing the CDC Act and the Social Order Maintenance Act (SOM Act), the government has 39 See Yip, supra, note 13. 40 . 43 In order to take charge of disinformation dissemination during epidemics, not only the CDC Act Arts 9 and 63 but also the Social Order Maintenance Act (SOM Act) Article 63(I)(5) can be cited as a legal basis to penalise those who spread fake news with imprisonment or a fine or the mass media companies who refuse to help correct disinformation upon notification by the authority. fined or even arrested individuals for spreading rumours regarding COVID-19 and the shortage of household necessities. 44 These measures may seem well-intentioned and make some sense in theory, but they may lack the scope of sensitivity regarding what is at stake. Some criticise the fairly broad authority and substantial discretion extended by law to competent authorities in determining when and what information should be contained and punished. 45 If the government fails to strike a proper yet difficult balance, 46 the fight against fake news and misinformation may ultimately backfire, resulting in a chilling effect that causes further information withholding, self-censorship and poor communication. Noting this dilemma, the government is gradually shifting to an open communication method that aims to actively inform the public via social media and rectify the marketplace of speech with correct information in a timely manner.', 'cite_spans': [{'start': 751, 'end': 753, 'text': '42', 'ref_id': None}, {'start': 983, 'end': 985, 'text': '43', 'ref_id': None}, {'start': 1072, 'end': 1074, 'text': '39', 'ref_id': None}, {'start': 1100, 'end': 1102, 'text': '40', 'ref_id': None}, {'start': 1105, 'end': 1107, 'text': '43', 'ref_id': None}, {'start': 1939, 'end': 1941, 'text': '45', 'ref_id': None}], 'ref_spans': [], 'section': 'Effective and efficient risk communication'}, {'text': 'Overall, the CECC has played a successful role in establishing and maintaining a credible channel for communicating information to the public in plain language and in a timely and consistent manner. There is also a responsive mechanism to address public enquiries, concerns and criticisms, which promotes public awareness and ensures constructive dialogue without politicising key messages. This open, responsive and credible risk communication helps the government build public trust when imposing necessary restrictive measures, encouraging citizens to self-report their health status and debating risk management options and broader policies.', 'cite_spans': [], 'ref_spans': [], 'section': 'Effective and efficient risk communication'}, {'text': \"Taiwan has taken various proactive and swift risk management measures to control the spread of COVID-19. 47 Here, we examine three dimensions of Taiwan's risk management strategy: optimisation of key medical supplies; border control and travel bans; and big data-powered mechanisms for tracking and tracing the illness.\", 'cite_spans': [{'start': 105, 'end': 107, 'text': '47', 'ref_id': None}], 'ref_spans': [], 'section': 'Proactive risk management'}, {'text': 'a. Optimisation of essential medical resources The outbreak has led to a global shortage of medical masks. Taiwan first banned the export of medical masks on 24 January 2020, and further implemented a nationwide requisition and real-name rationing scheme premised upon the National Health 44 B Hioe and L Wooster, \"Taiwan News Publishes COVID-19 Misinformation as Epidemic Spreads\" (New Bloom, 12 February 2020) <https://newbloommag.net/2020/02/12/coronavirus-taiwan-news> (last accessed 23 March 2020). 45 See, eg, M-H Huang, \"Fake News, Social Media, and the Freedom of Speech in the Internet Era (in Chinese)\" (2019) 292 Taiwan Law Review 5; Taiwan Association for Human Rights, \"Guanjhih Bushih Syunsi, Ying Jiangu Yanlunzihyou (管制不實訊息，應兼顧言論自由) [Fake News Regulations Should Consider the Protection of the Freedom of Expression]\" (TAHR, 13 January 2020) <https://newbloommag.net/2020/02/12/coronavirus-taiwan-news> (last accessed 24 March 2020). 46 But these regulations require a delicate balance because excessive controls of freedom of expression and freedom of the press would also present a threat to democracy, as much as would fake news. 47 Taiwan\\'s risk regulation against the global COVID-19 crisis has so far been one of the successful stories of the crisis; ibid. In addition, among others, Korea is also taking a proactive approach to combating COVID-19. See Insurance (NHI) system after only a few days. 48 From an economic perspective, the government has taken necessary measures to prioritise the acquisition and allocation of key medical supplies. To ensure a steady domestic supply, the government even drew up a budget of US$6.66 million to invest in equipment and 60 production lines by way of public-private partnerships. 49 Taiwan\\'s supplies of medical masks have since risen at a steady pace, growing from daily manufacturing capability at 2.44 million units on 22 January to 13 million units in early March 2020. 50 Such a surge in domestic medical mask production has made possible special allocations to medical personnel, airport control and schools. Taiwan further agreed to provide 100,000 masks per week to the USA in exchange for protective medical clothing when its production capacity is stabilised. 51 In the process of optimising the supply of medical masks, however, a more ethicssensitive framework to address distributional justice issues (eg unequal access to medical mask outlets in practice due to urban-rural differences) was overlooked. This can also be seen in the government\\'s management of medical resources in disaster preparedness planning. In Taiwan, the Communicable Disease Control Medical Network (CDCMN) was established following the SARS outbreak to centralise and optimise medical resources, 52 under which designated infection hospitals would be appointed as centres for the provision of treatment and resource mobilisation during public health emergencies. The CDCMN also facilitated the expansion of negative-pressure isolation rooms (1100 beds) 53 and isolation rooms (21,000 beds). 54 Yet this system and its methodology for prioritisation are predominantly based on biomedical models and medical resources rationing. Accordingly, they may overlook complex socioeconomic dynamics and varying local needs. 55 A framework for healthcare rationing and clinical prioritisation, especially under such exceptional circumstances, is desirable. Understandably, it may not be possible to set up ex ante criteria due to the uncertainty of an emergency. As such, extending broad discretion to physicians seems justifiable. 56 Yet, even so, legal and ethical guidance that strikes a balance between efficiency and fairness, 57 and transparency and due process in different circumstances, is much needed, as it can help guide the process of rationing medical resources 58 and facilitate ex post review and accountability mechanisms. 59 b. Border control and travel bans Because cross-border travel constitutes the primary source of COVID-19 transmission for Taiwan, it proactively imposed entry restrictions for people from China as early as January 2020. 60 Such restrictions were subsequently widened, and travellers from other countries listed on the Travel Notice were required to be quarantined for 14 days, eventually followed by a ban of all inbound foreign visitors on 9 March 2020. 61 In addition to prohibiting inbound travel, outbound travel is also restricted: medical personnel, civil servants and middle-and primary-school teachers, students and staff are prohibited or strongly discouraged from travelling abroad, with a view towards maintaining human medical resources and avoiding transmission among vulnerable groups. 62 These border control measures and travel 55 There has been no research done regarding whether this centralised system and prioritisation methodology would unfairly affect other healthcare priorities (such as those for non-communicable diseases). B Jennings and J Arras, \"Ethical Guidance for Public Emergency Preparedness and Response: Highlighting Ethics and Values in Vital Public Health Service\" (2008) <https://www.cdc.gov/od/science/integrity/phethics/docs/white_paper_final_for_ website_2012_4_6_12_final_for_web_508_compliant.pdf> (last accessed 26 March 2020). 56 Due to the significant uncertainty of a novel virus, it is important to confer more discretion to physicians so that they can adopt timely and effective measures. SA Raza, \"What \\'Distributive Justice\\' Means for Doctors Treating COVID-19\" (3 Quarks Daily, 22 March 2020) <https://www.3quarksdaily.com/3quarksdaily/2020/03/what-distributive-justicemeans-for-doctors-treating-covid-19.html> (last accessed 26 March 2020). 57 For example, the segments of the population upon whom the burden is likely to fall and the impacts of this should be evaluated, and cost-effectiveness analysis should not be the only evaluation tool. S Hoffman, \"Preparing bans are supported by the general public in the name of public interest at the expense of individual liberty. 63 In times of health emergencies, public interest may serve as a justification for the government to bypass the legal imperatives and bring about a downwards cycle. 64 An illustrative example is the CECC\\'s decision to publish the names of quarantined individuals who travel \"unnecessarily\" to these countries and to deprive their eligibility for the quarantine subsidy. 65 Whereas the CDC Act, in Article 58(I)(4), authorises the government to impose home-based quarantine on individuals entering Taiwan from the worst-affected countries, the publication of the names of the quarantined individuals is without legal basis and disproportionately invades individuals\\' privacy. The travel bans also suffer from the same weaknesses. Article 5(I) of the Immigration Act authorises the government to restrict the freedom to travel abroad for individuals involved in national security, yet neither medical professionals nor teachers/students are subject to the Act. 66 c. Big data and surveillance To have a clear picture of the travel and contact history, health records, real-time movement and other relevant information of (potentially) infected persons, the government has resorted to \"big data\" technologies to facilitate the effective implementation of quarantine measures and efficient contact tracing and locating. For instance, the government has actively integrated individuals\\' travel records from the National Immigration Agency with other existing databases, such as the NHI and even police systems. Physicians then have a channel by which to access a patient\\'s travel history through the NHI card reader. The government also used the NHI database, which contains the entire population\\'s medical records, to review all patients\\' health status and to locate suspected patients. Several patients\\' daily routes, explored through surveillance cameras and mobile phone tracking, were disclosed to the public so that people who have visited these places can be on alert. 67 A monitoring system, which can track individuals through GPS using their mobile devices and receive their daily health reports, was built to ensure individuals\\' compliance with the quarantine order. 68 Despite their effectiveness, these measures, with their great potential to infringe upon individuals\\' privacy, were not carefully scrutinised according to rule of law and constitutional principles. Indeed, aggregating databases and mobility data can help fight COVID-19 by mapping travel and contact history, measuring social distancing and ensuring compliance with quarantine orders, but only when \"appropriate legal, organizational, and computational safeguards [are] in place\". 69 The Personal Data Protection Act sets out rules for collecting, processing and using personal data, such as lawfulness, purpose limitation, data minimisation and data security, 70 but it has long been plagued by inflexible legal transplant and legal formalism without taking into account local contexts, and it has failed to provide a healthy regulatory environment. Furthermore, while the CDC Act 71 and the COVID-19 Special Act 72 authorise the government to impose quarantine, isolation care and \"other necessary measures\", it remains debatable whether linking multiple national databases, disclosing names, collecting mobile data and analysing surveillance data qualify as \"necessary measures\" and pass the scrutiny of fundamental rule of law principles. Additionally, there is no notification process, revision mechanism or appeal process for those affected to challenge and seek dispute resolution. 73 Again, these formal requirements merely serve as minimum safeguards and by no means suffice. A stronger mechanism that ensures deliberation, transparency and accountability by dynamic agency interaction and oversight is urgently needed.', 'cite_spans': [{'start': 504, 'end': 506, 'text': '45', 'ref_id': None}, {'start': 950, 'end': 952, 'text': '46', 'ref_id': None}, {'start': 1149, 'end': 1151, 'text': '47', 'ref_id': None}, {'start': 1371, 'end': 1374, 'text': 'See', 'ref_id': None}, {'start': 1421, 'end': 1423, 'text': '48', 'ref_id': None}, {'start': 1940, 'end': 1942, 'text': '50', 'ref_id': 'BIBREF0'}, {'start': 2236, 'end': 2238, 'text': '51', 'ref_id': None}, {'start': 2750, 'end': 2752, 'text': '52', 'ref_id': 'BIBREF2'}, {'start': 3007, 'end': 3009, 'text': '53', 'ref_id': 'BIBREF3'}, {'start': 4730, 'end': 4732, 'text': '55', 'ref_id': None}, {'start': 5258, 'end': 5260, 'text': '56', 'ref_id': None}, {'start': 5680, 'end': 5682, 'text': '57', 'ref_id': None}, {'start': 6015, 'end': 6017, 'text': '63', 'ref_id': None}, {'start': 6181, 'end': 6183, 'text': '64', 'ref_id': 'BIBREF4'}, {'start': 6386, 'end': 6388, 'text': '65', 'ref_id': 'BIBREF5'}, {'start': 6975, 'end': 6977, 'text': '66', 'ref_id': 'BIBREF6'}, {'start': 7987, 'end': 7989, 'text': '67', 'ref_id': 'BIBREF7'}, {'start': 8189, 'end': 8191, 'text': '68', 'ref_id': 'BIBREF8'}, {'start': 8673, 'end': 8675, 'text': '69', 'ref_id': None}], 'ref_spans': [], 'section': 'Proactive risk management'}, {'text': 'In light of this development, the Taiwan Association for Human Rights (TAHR) has raised the question of whether privacy protection mechanisms, such as proportionality, transparency, due process and expiration, are overlooked in the process. 74 In a crisis-driven fashion, mass surveillance techniques have become a convenient tool for the government to employ, with the aim of protecting people from 69 CO Buckee, \"Aggregated Mobility Data Could Help COVID-19\" (2020) <https://science.sciencemag.org/ content/early/2020/03/20/science.abb8021/tab-pdf> (last accessed 26 March 2020). March 2020) <https://medium.com/@tahr1984/protecting-civil-liberties-during-a-public-health-crisis-1de3c6d8e724> (last accessed 24 March 2020). It seems unnecessary and disproportionate to maintain years of travel history in the NHI ID card since COVID-19 has only a two-week incubation period. Furthermore, the lack of transparency when processing personal data also unfairly affects the substance and detail of the legal challenges individuals could have made to these privacy invasive measures. Thus, detailed information regarding how personal data are collected, the retention period for the data, the tools used to process the data and how the data analysis guides public health decisions should be published and made available to the public. Moreover, these measures failed to provide individuals fair opportunity to challenge in a timely manner the limits imposed on them based upon the analysis of personal data. Last but not least, there is no withdrawal mechanism either, which would set out the conditions for when the government should roll back the invasive programmes created in the name of public health. This mechanism is important because the data surveillance measures might long outlive the COVID-19 crisis.', 'cite_spans': [{'start': 241, 'end': 243, 'text': '74', 'ref_id': None}], 'ref_spans': [], 'section': 'Proactive risk management'}, {'text': 'COVID-19. 75 The bottom line is fairly straightforward: while some exceptional measures may be justified by a \"legality of emergency\", 76 ignoring existing rule of law and human rights safeguards seems to be a dangerous move and may ultimately institutionalise exceptionalism and leave a lasting impact on the legal system and culture such as public health and in a time of emergency. 80 Yet, even so, can we harness the administrative state with only traditional safeguards such as due process, proportionality and judicial review? What are the merits of a dynamic deliberation and accountability mechanism? Most importantly, how should we reconfigure the administrative state post-COVID-19?', 'cite_spans': [{'start': 10, 'end': 12, 'text': '75', 'ref_id': None}], 'ref_spans': [], 'section': 'Proactive risk management'}, {'text': \"Albeit marginalised, lawyers' input into the debates and discourses in countering COVID-19 would seem ever more urgent. Haunted by the ghost of martial law, Taiwan strives to maintain a high degree of democratic governance and rule of law. This in part explains why the government has been reluctant to issue an across-theboard lockdown that bears an authoritarian legacy. The same also applies to the Legislature's expeditious enactment of the COVID-19 Special Act as a legal basis for countering and containing the epidemic, with a view to fulfilling minimum democratic legitimacy and rule of law. Similarly, whereas the opposition party proposed that the President should invoke Article 43 of the Constitution to issue emergency decrees, 81 the President has so far declined to do so on the practical ground of the sufficiency of the COVID-19 Special Act, and perhaps also on the ideological ground of past bitter experiences. Nonetheless, constitutional and legal guarantees of human rights and the rule of law extend far beyond a formal legal basis for emergency-response measures. In recognition of broad and unlimited delegation, as endorsed in Article 7 of the COVID-19 Special Act, some members of the Legislative Yuan (Legislature of Taiwan) have proposed an amendment with more stringent rules designed to strengthen democratic oversight, remedy rule of law deficits and prevent systematic failure. 82 Relatedly, Taiwan's dynamic and robust civil society serves as an additional safety net for extreme measures in the name of emergencies. 83 In particular, the TAHR issued a statement emphasising the importance of democratic governance and the rule of law in times of pandemic, calling for resistance to blank-cheque authorisation. 84 Yet sticking to such a formalist understanding of legal safeguards remains inadequate. A broader socio-political mechanism that involves different stakeholders to ensure constructive deliberation and to address challenges seems desirable. 80 Taiwan's experience should also be read together with the dynamics of global health governance. Despite its alleged failure, the WHO continues to play a key role in providing guidance, expertise, technical assistance, a collaborative forum and personnel and practical tools, especially when the entire globe has been affected by COVID-19. 85 From Taiwan's perspective, meaningful participation in the WHO has been of the highest priority in its foreign relations, both for political reasons and for public health concerns. However, there is growing resentment of this organisation due to the sense of exclusion and the belief that is shared by at least some Taiwanese that this organisation uncritically takes positions that are overly favourable to China. Taiwan's scepticism towards the WHO and its relationship with China has driven the country's precautionary approach to coping with COVID-19. For the moment at least, the WHO seems to have lost its legitimacy and authority in global health governance and, subsequently, its charm for the Taiwanese people. Taiwan's experience may further fuel the existing momentum in favour of a global call for WHO reforms.\", 'cite_spans': [{'start': 741, 'end': 743, 'text': '81', 'ref_id': 'BIBREF10'}, {'start': 1410, 'end': 1412, 'text': '82', 'ref_id': 'BIBREF11'}, {'start': 1550, 'end': 1552, 'text': '83', 'ref_id': 'BIBREF12'}, {'start': 1744, 'end': 1746, 'text': '84', 'ref_id': 'BIBREF13'}, {'start': 1986, 'end': 1988, 'text': '80', 'ref_id': None}, {'start': 2328, 'end': 2330, 'text': '85', 'ref_id': 'BIBREF14'}], 'ref_spans': [], 'section': 'Proactive risk management'}, {'text': \"The changing global order represents yet another context by which to expound upon Taiwan's actions in fighting COVID-19 and its pursuit of health diplomacy. The growing influence of China in international organisations is due in part to the US's turn from multilateralism to bilateralism, particularly under the Trump Administration. 86 Because Taiwan is at the forefront in containing the transmission of COVID-19, as well as China's expansionist ambition, a close bilateral US-Taiwan collaboration is natural. On 18 March 2020, the USA and Taiwan issued a joint statement to strengthen their consultation and cooperation in combatting COVID-19. 87 This joint statement is of practical and political importance. Practically, it reflects the two sides' wish to pursue closer collaboration; politically, after the USA switched its diplomatic relations from Taiwan to China, it marks the first time that the Taiwanese Minister of Foreign Affairs signed an agreement or issued a statement with a US authority, albeit still through the American Institute in Taiwan. 88 In addition, the compelling need for cooperation in addressing the global COVID-19 crisis and for sharing experiences and information have similarly been felt by other countries. 89 Leaders in Western countries have voiced their support for Taiwan's participation in the WHO, 90 signalling yet another wave of contestation against international public authority. Is multilateralism in freefall in the face of this global public health crisis?\", 'cite_spans': [{'start': 334, 'end': 336, 'text': '86', 'ref_id': 'BIBREF15'}, {'start': 647, 'end': 649, 'text': '87', 'ref_id': 'BIBREF16'}, {'start': 1062, 'end': 1064, 'text': '88', 'ref_id': None}, {'start': 1244, 'end': 1246, 'text': '89', 'ref_id': 'BIBREF17'}], 'ref_spans': [], 'section': 'Proactive risk management'}, {'text': 'The global COVID-19 crisis has driven governments worldwide to adopt exceptional measures to address imminent threats that may bend or break fundamental legal principles and constitutional guarantees in the process. While Taiwan has done an extraordinary job in response to this grand challenge, it is not immune to this dilemma. Drawing from the lessons of SARS and COVID-19, how do we reinvent the administrative state and ensure an adequate governance framework that safeguards core human rights and legal principles in cases of public health emergencies?', 'cite_spans': [], 'ref_spans': [], 'section': 'IV. CONCLUSION'}, {'text': \"To begin, democratic governance and open society are cornerstones, not only for the state of normalcy, but also for the state of emergency. Congressional oversight and the principle of legal reservation remain the first gatekeepers and the foundation for safeguarding the rule of law and human rights. The COVID-19 Special Act, regardless of its controversial open-ended delegation, reflects this young democracy's attempt to maintain its democratic legitimacy. With a large degree of transparency, the Chief of the CECC communicates daily with the affected population, which builds trust within the community and strengthens the legitimacy of its decisions. In response, human rights advocates and non-governmental organisations continue to point to the danger of human rights violations surrounding these emergency measures.\", 'cite_spans': [], 'ref_spans': [], 'section': 'IV. CONCLUSION'}, {'text': 'Furthermore, law, especially a set of pre-established procedures and substantive rules designed for risk regulation in times of such public health emergencies, is the second gatekeeper and remains the key instrument to guide and discipline Executive power. With its bitter lessons learned from SARS, Taiwan revised the CDC Act and relevant regulations, which provide a well-functioning framework to prepare for and respond to another public health emergency. This legal fabric helps prevent the Executive Branch from exercising its power in an arbitrary, capricious, misinformed or excessively disproportionate manner. A well-designed framework with pre-established procedures and substantive rules also enables effective ex post judicial oversight. Nevertheless, this second gatekeeper should go beyond a formalist understanding of legal safeguards and enlist innovative and dynamic approaches to broader sociopolitical checks and balances.', 'cite_spans': [], 'ref_spans': [], 'section': 'IV. CONCLUSION'}, {'text': 'Finally, pandemics hit unevenly, both within this island country and across the globe. Emergency measures carry significant distributional justice implications, and the vulnerable are most susceptible to the threat of pandemics. Whereas Taiwan has so 90 \"Taiwan: EU Supports Taipei\\'s Bid to Join WHO\" (Stratfor, 31 January 2020) <https://worldview.stratfor.com/ situation-report/taiwan-eu-supports-taipeis-bid-join-who> (last accessed 26 March 2020); B Blanchard and YL Taiwan, \"U.S. Increases Support for Taiwan, China Threatens to Strike Back\" (Reuters, 27 March 2020) <https://www.reuters. com/article/us-taiwan-usa/u-s-increases-support-for-taiwan-in-recognition-battle-with-china-idUSKBN21E0B7> (last accessed 27 March 2020); \"Japan PM Abe Calls for Taiwan\\'s Participation in WHO as Coronavirus Spreads\" (Kyodo News, 30 January 2020) <https://english.kyodonews.net/news/2020/01/cff2af87f289-abe-calls-for-taiwansparticipation-in-who-as-coronavirus-spreads.html> (last accessed 26 March 2020); CW Nelson, \"COVID-19: Time for WHO to Reconsider Its Stance Towards Taiwan\\' (2020) 579 Nature 193. far produced a successful story in countering COVID-19, it has not been without cost: medical professionals are one example, and existing, chronic patients who are crowded out of treatment due to resource mobilisation are another. At the international level, the current health governance regime is by no means global or universal. Apart from Taiwan\\'s exclusion, it does not really reflect the interests of those most in need of assistance. Time will tell as COVID-19 hits the least-developed countries of the world. Therefore, a distributional justice logic deserves to be institutionalised as the normative anchor and the third gatekeeper of national and global health governance.', 'cite_spans': [], 'ref_spans': [], 'section': 'IV. CONCLUSION'}, {'text': \"Taiwan's experience in fighting COVID-19 presents an alternative model to China's myth of authoritarian effectiveness. It also points to the failure of the current international public authority in moving beyond the political boundaries that diseases neither recognise nor respect. In the post-COVID-19 world, we deserve a new system of global health governance that delivers rather than undermines global public goods.\", 'cite_spans': [], 'ref_spans': [], 'section': 'IV. CONCLUSION'}], 'paragraphs': [{'text': 'In light of this development, the Taiwan Association for Human Rights (TAHR) has raised the question of whether privacy protection mechanisms, such as proportionality, transparency, due process and expiration, are overlooked in the process. 74 In a crisis-driven fashion, mass surveillance techniques have become a convenient tool for the government to employ, with the aim of protecting people from 69 CO Buckee, \"Aggregated Mobility Data Could Help COVID-19\" (2020) <https://science.sciencemag.org/ content/early/2020/03/20/science.abb8021/tab-pdf> (last accessed 26 March 2020). March 2020) <https://medium.com/@tahr1984/protecting-civil-liberties-during-a-public-health-crisis-1de3c6d8e724> (last accessed 24 March 2020). It seems unnecessary and disproportionate to maintain years of travel history in the NHI ID card since COVID-19 has only a two-week incubation period. Furthermore, the lack of transparency when processing personal data also unfairly affects the substance and detail of the legal challenges individuals could have made to these privacy invasive measures. Thus, detailed information regarding how personal data are collected, the retention period for the data, the tools used to process the data and how the data analysis guides public health decisions should be published and made available to the public. Moreover, these measures failed to provide individuals fair opportunity to challenge in a timely manner the limits imposed on them based upon the analysis of personal data. Last but not least, there is no withdrawal mechanism either, which would set out the conditions for when the government should roll back the invasive programmes created in the name of public health. This mechanism is important because the data surveillance measures might long outlive the COVID-19 crisis.', 'id': '00013'}], 'title': 'Reimagining the Administrative State in Times of Global Health Crisis: An Anatomy of Taiwan’s Regulatory Actions in Response to the COVID-19 Pandemic', 'sha': '6e842a48bd618d1d4e72983208194db56e04fc92', 'doi': '10.1017/err.2020.25', 'score': 8.833000183105469}, {'rank': 266, 'paragraphs': [{'text': 'On the other hand, these two methods with different distributions but with the same mean infectious period lead to inconsistent results. Since the actual infectious period and incubation period do not follow exponential distributions, these simulations suggest that future works need to consider distributions of epidemiological characteristics to better capture the COVID-19 spreading trajectory. Instead of taking mean values derived from clinical studies as input to the epidemic models, researchers need to take into account the empirically derived distribution of the parameters.', 'id': '00054'}], 'title': 'Short-term forecasts and long-term mitigation evaluations for the COVID-19 epidemic in Hubei Province, China', 'sha': '26a2d4b805cdb48bccb4d640da82fa4c4d15c7c4', 'doi': '10.1101/2020.03.27.20045625', 'score': 8.818400382995605}, {'rank': 267, 'abstract': 'Mathematical models proffer a rational basis to epidemiologists and policy makers on how, where and when to control an infectious disease. Through mathematical models one can explore and provide solutions to phenomena which are difficult to measure in the field. In this paper, a mathematical models has been used to explore the role of government and individuals reaction to the recent outbreak of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). The proposed framework incorporates all the relevant biological factors as well as the effects of individual behavioral reaction and government action such as travel restrictions, social distancing, hospitalization, quarantine and hygiene measures. Understanding the dynamics of this highly contagious SARS-CoV-2, which at present does not have any therapy assist the policy makers on evaluating the effectiveness of the control measures currently being implemented. Moreover, policy makers can have insights on short-and-long term dynamics of the disease. The proposed conceptual framework was combined with data on cases of coronavirus disease (COVID-19) in South Africa, March 2020 to early May 2020. Overall, our work demonstrated optimal conditions necessary for the infection to die out as well as persist.', 'body_text': [{'text': 'In late December 2019, a novel strand of Coronavirus (SARS-CoV-2) was reported in Wuhan, a central and crowded city of China (Li et al., 2020). Subsequently, the World Health Organization (WHO) has since officially termed this pandemic the Corona Virus Disease 2019 (COVID-19) [1]. COVID-19 is a rapidly spreading infectious disease and continues to cause several outbreaks in multiple world countries. As of April 12, 2020 (08:00 GMT+2), COVID-19 had resulted in 3,146,651 confirmed cases, 218,178 deaths, 961,833 recoveries, in 213 nations, areas and territories [2].', 'cite_spans': [{'start': 278, 'end': 279, 'mention': '1', 'ref_id': 'BIBREF0'}, {'start': 566, 'end': 567, 'mention': '2', 'ref_id': 'BIBREF11'}], 'section': 'Introduction', 'ref_spans': []}, {'text': 'The public panic in face of the ongoing COVID-19 outbreak reminds us the history of the 1918 influenza pandemic, which killed approximately 50–100 million people worldwide [3]. Moreover, its characteristics of mild symptoms in most cases and short serial interval (i.e., 4–5 days) are synonymous to that of the 1918 influenza, rather than the two other coronaviruses (severe acure respiratory syndrome coronavirus, SARS-CoV, and Middle East respiratory syndrome coronavirus, MERS-CoV) [4]. Moreover, since there is neither a vaccine nor treatment (as of April 2020) to control the spread of the disease, measures such as case isolation, contact tracing and quarantine, social distancing and hygiene measures-which were used to mitigate the 1918 influenza are currently being implemented.', 'cite_spans': [{'start': 173, 'end': 174, 'mention': '3', 'ref_id': 'BIBREF16'}, {'start': 486, 'end': 487, 'mention': '4', 'ref_id': 'BIBREF17'}], 'section': 'Introduction', 'ref_spans': []}, {'text': 'Considering the social and economic impact the disease has caused within the shortest period of time, it is therefore prudent to evaluate the strength of the aforementioned intervention strategies to curb the spread of the disease. Various techniques and methods can be used to explore the impact of the current ongoing mitigation strategies. There are various tools and techniques available which predict the dynamics of the disease transmission and also suggest suitable control interventions. Among them, mathematical modeling, analysis and simulation has been found to be a very successful guiding tool that could give a sound direction to policy makers and public health administration on how to effectively prevent and control disease and has been extensively used [5]. With the aid of mathematical models one can be able to infer, understand and proffer solutions to phenomena which are difficult to measure in the field.', 'cite_spans': [{'start': 772, 'end': 773, 'mention': '5', 'ref_id': 'BIBREF18'}], 'section': 'Introduction', 'ref_spans': []}, {'text': 'Since the outbreak of COVID-19 a number of mathematical models have been proposed, see, for example [4,[6], [7], [8], [9], [10]], to mention but a few. Gilbert and co-workers developed a conceptual model for COVID-19, with a goal to infer the preparedness and vulnerability of different nations. In Kucharski et al. [12] a stochastic transmission model to assess the early transmission and control of COVID-19 in Wuhan was presented. One of the key results from their study was that the decline in COVID-19 cases late January, 2020 was due to the introduction of travel control measures. In Lin et al. [4] a general mathematical model for the coronavirus disease 2019 (COVID-19) outbreak in Wuhan, China with individual reaction and governmental action was presented. Although Lin and Co-workers did not fit their framework with data, their work demonstrated the role of individual reaction and governmental action on transmission and control of COVID-19. Motivated by the work of Lin and Co-workers, we proposed a mathematical model for transmission and control of COVID-19 in South Africa. As in Lin et al. [4], the proposed framework incorporates individual reaction and governmental action. However, our framework is unique from that of Lin et al. [4] in that we incorporated six infectious classes, latent stage, symptomatic severe patients, symptomatic mild patients, asymptomatic patients and detected patients, whereas in Lin and Co-workers had only three infectious compartments. We argue that these additional classes clearly present the dynamics of COVID-19 in South Africa and the greater part of the world. For example, as more knowledge about COVID-19 continues to emerge, the role of asymptomatic patients on transmission and control of the disease has become one of the important topic. Zou et al. [13] report that the viral load of asymptomatic patients was similar to symptomatic patients indicating a transmission potential of asymptomatic patients. Hence there is need to infer on the role asymptomatic patients play on COVID-19 dynamics.', 'cite_spans': [{'start': 101, 'end': 102, 'mention': '4', 'ref_id': 'BIBREF17'}, {'start': 103, 'end': 106, 'mention': '[6]', 'ref_id': 'BIBREF19'}, {'start': 108, 'end': 111, 'mention': '[7]', 'ref_id': 'BIBREF20'}, {'start': 113, 'end': 116, 'mention': '[8]', 'ref_id': 'BIBREF21'}, {'start': 118, 'end': 121, 'mention': '[9]', 'ref_id': 'BIBREF22'}, {'start': 123, 'end': 127, 'mention': '[10]', 'ref_id': 'BIBREF1'}, {'start': 317, 'end': 319, 'mention': '12', 'ref_id': 'BIBREF3'}, {'start': 603, 'end': 604, 'mention': '4', 'ref_id': 'BIBREF17'}, {'start': 1110, 'end': 1111, 'mention': '4', 'ref_id': 'BIBREF17'}, {'start': 1253, 'end': 1254, 'mention': '4', 'ref_id': 'BIBREF17'}, {'start': 1815, 'end': 1817, 'mention': '13', 'ref_id': 'BIBREF4'}], 'section': 'Introduction', 'ref_spans': []}, {'text': \"In this section we present the conceptual framework for modelling COVID-19 outbreak in South Africa. We constructed an ordinary differential equations model that considers the human population subdivided into compartments based upon infection status. Furthermore, the proposed frame work incorporates the effects of individual reaction as well as the government action. Precisely, the model has been formulated based on the following assumptions:(i)Vital dynamics (birth rate and natural mortality rate) are not essential, since we are interested on investigating the dynamics of the disease over a very short time frame. The total human population at time t, N(t) comprise of: susceptible population S(t)-these are individuals who are yet to contract the disease but can do upon exposure to the infection; exposed/latently infected individuals E(t)-these are individuals who have contracted the infection but are not yet infectious, in other words, they are incubating the disease; undetected asymptomatic patients A(t)-these are individuals who would have completed their incubation period and can now transmit the infection. In general, these individuals cannot be recognized if they are not confirmed by RT-PCR or other laboratory testing [14]. The model also includes undetected clinically infected individuals and this population has been further subdivided into two different classes-mild patients Im(t) and severe patients Is(t). In a recent study by Wu and McGoogan [15], it was noted that approximately 81% of the detected COVID-19 patients were of mild symptom and the remainder (about 20%) were severe. In addition, we have also included detected and quarantined patients (both asymptomatic and symptomatic) Q(t); as well as the deceased and successfully recovered individuals and these are respectively denoted by D(t) and R(t). The population of removed individuals represents individuals who have successfully recovered from infection ‘naturally’ or through ‘treatment’. Thus, the total population is N(t)=S(t)+E(t)+A(t)+Im(t)+Is(t)+Q(t)+R(t)+D(t).\\n(ii)As in He et al. [3] and Lin et al. [4], we assume that the public's perception regarding the number of confirmed cases and deceased influences the dynamics of the disease, hence we have have captured this aspect by including an additional compartment to our framework and this compartment is denoted by P(t). (iii) Susceptible individuals are assumed to acquire infection following effective contact with undetected asymptomatic patients A(t), undetected symptomatic patients with mild symptoms Im(t), undetected symptomatic patients with severe symptoms Is(t) and detected patients Q(t). Since the outbreak of the novel coronavirus disease (COVID-19) in December 2019, one of the issues that has received major attention is the transmissibility of the coronavirus from asymptomatic patients to healthy individuals [13,16]. In recent study of Yin and Jin [16], no difference in the transmission rates of coronavirus between symptomatic and asymptomatic patients was observed. Our study is also unique from a number of recently published works in that we have assumed that detected patients can transmit the disease. This assertion is based on several reports which have highlighted that healthcare workers (HCWs) are being infected by COVID-19 [[17], [18], [19]]. On 24 February 2020, during a press conference of the WHO-China Joint Mission on CoVID-19, National Health Commission of the People's Republic of China (NHCPRC) reported that since the outbreak began in December 2019 till that day about 2055 healthcare workers (community/hospital-acquired not to be defined) had been confirmed infected with COVID-19, with 22 (1.1%) deaths [17]. As of April 24, 2020, the Spanish Health Ministry reported that 35, 295 HCWs were infected and this represented 20% of all registered cases of COVID-19 in Spain by that time [19]. In addition, a cross-sectional study conducted in 2 teaching hospitals in the Netherlands between 12 March 12 2020 and 16 March 2020, revealed that 6% of HCWs at these two institutions were infected with SARS-CoV2 [18]. Thus, the force of infection λ(t), which represents the rate at which susceptible individuals become infected is expressed as:\\n(1)λ(t)=β(t)[Is(t)+A(t)+Im(t)+εqQ(t)N(t)],where the parameter εq is a positive constant and accounts for differential infectivity of the detected and quarantine individuals in relation to the undetected patients. Since these individuals are quarantined and interact with a small susceptible population that mainly constitute of HCWs, we assume that 0<εq<1. As suggested in the studies of [13,16] we will assume no difference in transmission rate among all the undetected patients. The parameter β(t) models disease transmission rate and is mathematically expressed as(2)β(t)=β0[1−α][1−P(t)N(t)]κ.\\n\", 'cite_spans': [{'start': 1244, 'end': 1246, 'mention': '14', 'ref_id': 'BIBREF5'}, {'start': 1476, 'end': 1478, 'mention': '15', 'ref_id': 'BIBREF6'}, {'start': 2085, 'end': 2086, 'mention': '3', 'ref_id': 'BIBREF16'}, {'start': 2104, 'end': 2105, 'mention': '4', 'ref_id': 'BIBREF17'}, {'start': 2884, 'end': 2886, 'mention': '13', 'ref_id': 'BIBREF4'}, {'start': 2887, 'end': 2889, 'mention': '16', 'ref_id': 'BIBREF7'}, {'start': 2924, 'end': 2926, 'mention': '16', 'ref_id': 'BIBREF7'}, {'start': 3313, 'end': 3317, 'mention': '[17]', 'ref_id': 'BIBREF8'}, {'start': 3319, 'end': 3323, 'mention': '[18]', 'ref_id': 'BIBREF9'}, {'start': 3325, 'end': 3329, 'mention': '[19]', 'ref_id': 'BIBREF10'}, {'start': 3707, 'end': 3709, 'mention': '17', 'ref_id': 'BIBREF8'}, {'start': 3887, 'end': 3889, 'mention': '19', 'ref_id': 'BIBREF10'}, {'start': 4107, 'end': 4109, 'mention': '18', 'ref_id': 'BIBREF9'}, {'start': 4628, 'end': 4630, 'mention': '13', 'ref_id': 'BIBREF4'}, {'start': 4631, 'end': 4633, 'mention': '16', 'ref_id': 'BIBREF7'}], 'section': 'Epidemiological model ::: Material and methods', 'ref_spans': []}, {'text': \"Equation (2), incorporates the effects of both individuals' reaction and governmental actions; β0 is the baseline transmission rate; α (0≤α<1) is the efficacy of ‘governmental actions’ (lockdown, encouraging use of sanitizers, face masks, social distancing) to reduce contacts among individuals. A value of α close to one implies high efficacy and the reverse is true for values of α close to zero. The term (1−P/N)κ captures the effects of public perception of the risk to contract the disease based on severe cases reported. Here, κ is a parameter controlling the strength of the response.(iv)Susceptible individuals who contract the disease progress to the exposed/latent stage where they will incubate the disease for an average period of σ−1 days. During this period these individuals will not be capable of transmitting the disease. A couple of recent studies on Wuhan COVID-19 dynamics suggest that the average (median) incubation period could be as short as 4 days [4,20].(v)Upon the completion of the incubation period, we assume that a fraction f of the exposed individuals move to the asymptomatic stage and the remainder (1−f) become symptomatic. Prior studies suggests that of the individuals who become symptomatic, more often there exists two classes, mild patients and severe patients [4,15]. In particular, in a study on Wuhan COVID-19 dynamics by Wu and McGoogan [15] it was observed that approximately 81% of the cases were mild symptoms (without pneumonia or only mild pneumonia), 14% were severe cases with difficulty breathing, and 5% were critical with respiratory failure, septic shock, and/or multiple organ dysfunction or failure. Based on this assertion, we assume that a proportion (1−f)p of exposed individuals progress to infectious stage with mild symptoms and the complementary proportion (1−f)(1−p) will become severe.(vi)A fraction (1−p1) of infectious individuals with mild symptoms are assumed to successfully recover from infection after an average period of γ−1 days and the remainder p1 join the class Q. For severely and critically infected individuals, it is assumed that a fraction p2 enter the class Q after an average duration of δ−1 days and the remainder (1−p2) succumb to disease-related death. Thus, γ−1 and δ−1, represents the mean infectious period of mild and severe patients, respectively. Some prior studies suggests a mean infectious period of 4 days [4].(vii)Through RT-PCR or other laboratory testing, asymptotic patients are assumed to be detected and quarantined at rate ω. Furthermore, we assume that after an average period of ϕ−1 days, asymptomatic patients will begin to display clinical signs of the disease, with a proportion θ showing mild symptoms and the remainder (1−θ) severe. Detected individuals are assumed to exit this class either through successful recovery at rate (1−p3)η or death at rate p3η, η−1 represents the average period one stays in the quarantine stage and p3 denotes a proportion of the quarantined patients who will suffer disease-related death.(viii)The public's perception of risk to COVID-19 increases when the number of confirmed COVID-19 cases increases as well as the increase in deaths for quarantined individuals. It is assumed to decay naturally, implying that the perception of risk diminishes over time in the absence of COVID-19 severe and critical cases and deaths. In the proposed model, λ−1 models the mean duration of impact of COVID-19 severe and critical cases and deaths on public perception.\\n\", 'cite_spans': [{'start': 974, 'end': 975, 'mention': '4', 'ref_id': 'BIBREF17'}, {'start': 976, 'end': 978, 'mention': '20', 'ref_id': 'BIBREF12'}, {'start': 1302, 'end': 1303, 'mention': '4', 'ref_id': 'BIBREF17'}, {'start': 1304, 'end': 1306, 'mention': '15', 'ref_id': 'BIBREF6'}, {'start': 1382, 'end': 1384, 'mention': '15', 'ref_id': 'BIBREF6'}, {'start': 2406, 'end': 2407, 'mention': '4', 'ref_id': 'BIBREF17'}], 'section': 'Epidemiological model ::: Material and methods', 'ref_spans': []}, {'text': \"Based on the aforementioned assumptions we have the following system of nonlinear ordinary differential equations:(3)S'(t)=−λ(t)S(t),E'(t)=λ(t)S(t)−σE(t),A'(t)=σfE(t)−(ϕ+ω)A(t),Im'(t)=σ(1−f)pE(t)+θϕA(t)−γIm(t),Is'(t)=σ(1−f)(1−p)E(t)+(1−θ)ϕA(t)−δIs(t),Q'(t)=p1γIm(t)+p2δIs(t)+ωA(t)−ηQ(t),R'(t)=(1−p1)γIm(t)+(1−p3)ηQ(t),D'(t)=(1−p2)δIs(t)+p3ηQ(t),P'(t)=p1γIm(t)+p2δIs(t)+ωA(t)+p3ηQ(t)−λP(t).}\\n\", 'cite_spans': [], 'section': 'Epidemiological model ::: Material and methods', 'ref_spans': []}, {'text': '\\nFig. 1\\nillustrates the transition of individuals from one epidemiological state to another.', 'cite_spans': [], 'section': 'Epidemiological model ::: Material and methods', 'ref_spans': [{'start': 1, 'end': 7, 'mention': 'Fig. 1', 'ref_id': 'FIGREF0'}]}, {'text': 'The basic reproduction number, often denoted by R0 is an important threshold parameter for epidemiological models. It is defined as the expected number of number of secondary cases generated in a completely susceptible population, by one infectious individual during his/her entire infectious period. For models with forward bifurcation, if this metric is less than unity it implies that the infection will die out in the community. However, if it is greater than unity then the disease persists. Although there are several ways of deriving this parameter, the next-generation matrix technique [21,22] is the most popular. One can easily verify that in the absence of the disease, model system (3) admits a trivial equilibrium point, commonly known as the disease-free equilibrium and is given by S=N,\\nE=A=Is=Im=Q=R=D=P=0. Utilizing the notation in Ref. [21], the nonnegative matrix F that denotes the generation of new infection terms and the non-singular matrix V(t) that denotes the remaining transfer terms are respectively given (at the disease-free equilibrium) by;F=[0β0(1−α)β0(1−α)β0(1−α)β0εq(1−α)00000000000000000000],\\nV=[σ0000−σfω+ϕ000−σfp−σfp−σf(1−p)−θϕγ00−(1−θ)ϕ0δ00−ω−p1γ−p2δη].\\n', 'cite_spans': [{'start': 595, 'end': 597, 'mention': '21', 'ref_id': 'BIBREF13'}, {'start': 598, 'end': 600, 'mention': '22', 'ref_id': 'BIBREF14'}, {'start': 855, 'end': 857, 'mention': '21', 'ref_id': 'BIBREF13'}], 'section': 'The basic reproduction number ::: Material and methods', 'ref_spans': []}, {'text': 'It follows that the spectral radius of model (3) which is the reproduction number of the model is given by(4)Reff=Ra+Rs+Rm+Rq,where Ra, Rs, Rm and Rq, represents the contribution of undetected asymptomatic patients, undetected symptomatic patients with severe symptoms, undetected symptomatic patients mild symptoms and quarantined patients, respectively, on the generation of secondary infections, and they are defined as followsRa=β0(1−α)fϕ+ω,Rs=β0(1−α)fδ[(1−p)+(1−θ)ϕϕ+ω],Rm=β0(1−α)fδ[p+θϕϕ+ω],Rq=β0εq(1−α)fη[p1{p+(ω+p1θϕ)p1(ϕ+ω)}+p2{(1−p)+(1−θ)ϕϕ+ω}].}\\n', 'cite_spans': [], 'section': 'The basic reproduction number ::: Material and methods', 'ref_spans': []}, {'text': 'From the expression of the reproduction number Reff, we can note the following:(i)Increasing the strength of governmental action to high levels (close to 100%), will lead to a huge reduction in the magnitude of the reproduction number, and the weak actions (values of α close to zero) will not be effective to reduce the magnitude of the basic reproduction number.(ii)In the absence of governmental actions (α=p1=p2=ω=εq=0), the number of secondary cases that will be generated by each infected individual will be higher compared to when actions are even weak. The expression for the reproduction number in the absence of intervention strategies is given by\\nR0=β0f[δ+ϕ(2−(p+θ))(1−εm)]ϕδ.\\n(iii)We can also observe that if quarantined individuals do not contribute to the generation of new infections, that is, εq=Rq=0, then the reproduction number will be less.\\n', 'cite_spans': [], 'section': 'The basic reproduction number ::: Material and methods', 'ref_spans': []}, {'text': 'We present an application of our model to a case study of South Africa. We use the COVID-19 data for South Africa released and updated by the South African government authorities available on www.worldometer.com [23]. The dataset has also been included in the appendix. The data presented in the appendix starts from the 5th of March 2020 when South Africa reported its first case of COVID-19 case imported from Italy one of the global hotspots of the pandemic. However, when fitting our model, we considered the data ranging from 26 of March 2020 to 3 May 2020, when the lockdown started in South Africa. This was necessitated by the need to exclude the effects of imported cases. Furthermore, we assume that by this date our framework will be able to capture the effects of local transmission since it was now the time of the lockdown. Overall, the data in the appendix suggests that with increasing testing and screening of possible COVID-19 infected individuals, the reported detected cases continue to rise with time indicating the prevalence of COVID-19 infection in South Africa might be higher than the current reported cases.', 'cite_spans': [{'start': 213, 'end': 215, 'mention': '23', 'ref_id': 'BIBREF15'}], 'section': 'Data acquisition and disease parameter estimation ::: Results and discussions', 'ref_spans': []}, {'text': 'Moreover, the data presented in the appendix reflects the number of confirmed COVID-19 cases during the ongoing lockdowns and will be used to estimate our unknown model parameters not currently available in literature. Much information remains unknown regarding COVID-19 infection and more work is being done by experts from across the multi-disciplinary facets of research to provide answers to some unknown information on COVID-19. However, much effort to unpack these underlying issues of this pandemic is being witnessed to date. Thus, in this work we resort to curve fitting which is a process that allows us to quantitatively estimate the trend of the outcomes of this pandemic. Equations of approximating curves are fit to the raw field data. However, the fitting curves for any given set of data are not unique. Thus, we choose a curve with the minimum possible deviation from all the data points involved. We make use of the least squares curve fit routine (lsqcurvefit) in Matlab with optimization to estimate our unknown model parameters. Estimated model parameters and their 95% confidence intervals are presented in Table 1\\n. Other parameters values which were drawn from literature are presented in Table 2\\n.', 'cite_spans': [], 'section': 'Data acquisition and disease parameter estimation ::: Results and discussions', 'ref_spans': [{'start': 1129, 'end': 1136, 'mention': 'Table 1', 'ref_id': 'TABREF1'}, {'start': 1213, 'end': 1220, 'mention': 'Table 2', 'ref_id': 'TABREF2'}]}, {'text': \"We fitted the model to cumulative daily new infection data presented in the appendix. The cumulative new infections predicted by our model, C(t), are given by the solution (6) of the following equation:(5)C'(t)=p1γIm(t)+p2δIs(t)+ωA(t),\\n\", 'cite_spans': [], 'section': 'Data acquisition and disease parameter estimation ::: Results and discussions', 'ref_spans': []}, {'text': 'Thus, the estimation of confirmed cumulative cases for COVID-19 over a defined time frame tk−1≤t≤tk (where tk−1 and tk marks the beginning and end of the time interval, respectively) from the model output requires to compute:(6)∫tk−1tk[p1γIm(t)+p2δIs(t)+ωA(t)]dt.\\n', 'cite_spans': [], 'section': 'Data acquisition and disease parameter estimation ::: Results and discussions', 'ref_spans': []}, {'text': 'The following initial conditions were determined upon fitting the data, S(0)=N−E(0)−A(0)−Im(0)−Is(0)−Q(0)−R(0) where N=58×106, E(0)=3000, A(0)=100, Im(0)=200, Is(0)=50, Q(0)=915, R(0)=12, D(0)=0, P(0)=20. In this study, the term ‘active cases’ as defined by Worldometer refers to the population of individuals who have tested positive to infection but have neither successfully recovered from it nor succumbed to disease-related death.', 'cite_spans': [], 'section': 'Data acquisition and disease parameter estimation ::: Results and discussions', 'ref_spans': []}, {'text': 'Using baseline values on Table 1, Table 2 we have noted that the basic reproduction number of model (3) in the absence of government action and individual reaction R0 will be 3.54. However, in the presence of governmental action and individual reaction at 55% efficacy the reproduction number will be Reff=1.8. It is worth noting that this is baseline value when effectiveness of aforementioned intervention strategies is fixed at 55% efficacy.', 'cite_spans': [], 'section': 'Data acquisition and disease parameter estimation ::: Results and discussions', 'ref_spans': [{'start': 25, 'end': 32, 'mention': 'Table 1', 'ref_id': 'TABREF1'}, {'start': 34, 'end': 41, 'mention': 'Table 2', 'ref_id': 'TABREF2'}]}, {'text': '\\nFig. 2\\nshows the trends in the cumulative COVID-19 detected cases in South Africa. We observe from Fig. 2, that system (3) fits well with the data from Table 3 (see appendix). Estimated parameter values are shown in Table 1.', 'cite_spans': [], 'section': 'Data acquisition and disease parameter estimation ::: Results and discussions', 'ref_spans': [{'start': 1, 'end': 7, 'mention': 'Fig. 2', 'ref_id': 'FIGREF1'}, {'start': 100, 'end': 106, 'mention': 'Fig. 2', 'ref_id': 'FIGREF1'}, {'start': 153, 'end': 160, 'mention': 'Table 3', 'ref_id': 'TABREF0'}, {'start': 217, 'end': 224, 'mention': 'Table 1', 'ref_id': 'TABREF1'}]}, {'text': 'To explore the impact of individual reaction and governmental action on combating COVID-19 disease in South Africa, we will simulate model (1) using parameter values in Table 1, Table 2 Majority of these parameter values were adopted from the recently published literature on COVID-19 and the remaining unknown parameters were estimated from data fitting. Estimated parameters are within plausible range of values so as to capture the current COVID-19 transmission dynamics in South Africa. The total population of South Africa in this study was assumed to be 58 million. We perform numerical simulations considering the period when the lockdown was instituted in South Africa, on the 26th of March 2020. At that particular time we assume that there were certain individuals who remained undetected and suit to be classified in one of the following compartments: asymptomatic, mildly infected or severely infected. Also, as reported by the South African government, there were certain individuals who had successfully recovered from the disease.', 'cite_spans': [], 'section': 'Simulation results ::: Results and discussions', 'ref_spans': [{'start': 169, 'end': 176, 'mention': 'Table 1', 'ref_id': 'TABREF1'}, {'start': 178, 'end': 185, 'mention': 'Table 2', 'ref_id': 'TABREF2'}]}, {'text': 'We aim to intrinsically investigate the impact of asymptomatic infections on the general dynamics of COVID-19. We determine to what extent the current 35 day lockdown in South Africa has been by comparison with the scenario when there was no lockdown instituted.', 'cite_spans': [], 'section': 'Simulation results ::: Results and discussions', 'ref_spans': []}, {'text': '\\nFig. 3\\nillustrates how the active cases were going to progress in the absence of lockdown. As shown in the graph, the number of cases were going to increase rapidly more or less in an exponential growth scenario. In particular, the number of cases could be around 100000 beginning of June. However, in Fig. 4\\nwe can observe that in the presence of intervention strategies, the number of infections may not exceed 18,000 cases for the entire 450 day period.', 'cite_spans': [], 'section': 'Simulation results ::: Results and discussions', 'ref_spans': [{'start': 1, 'end': 7, 'mention': 'Fig. 3', 'ref_id': 'FIGREF2'}, {'start': 303, 'end': 309, 'mention': 'Fig. 4', 'ref_id': 'FIGREF3'}]}, {'text': '\\nFig. 5\\nillustrates the expected number of active cases for different successive lockdown extensions after the initial 35 day lockdown. We consider extensions of 14 day stages, 21 day stages and 35 day stages. We observe that a 14 day extension will result to approximately 15000 cases, a 21 day extension will result to approximately 13000 and a 35 day extension will result to approximately 11000 cases by the 4th of June 2020.', 'cite_spans': [], 'section': 'Simulation results ::: Results and discussions', 'ref_spans': [{'start': 1, 'end': 7, 'mention': 'Fig. 5', 'ref_id': 'FIGREF4'}]}, {'text': '\\nFig. 6\\nillustrates the dynamics of the disease in the absence and presence of intervention strategies over a 450 day period. As we can observe, the number of active cases in the absence of intervention strategies will reach a peak around mid-June whereas in the presence of intervention strategies the peak will be attained around the 23rd of August. The results suggests that the presence of intervention strategies may be responsible for the delay in attaining the peak. This may give policy makers ample time to prepare for effective disease management.', 'cite_spans': [], 'section': 'Simulation results ::: Results and discussions', 'ref_spans': [{'start': 1, 'end': 7, 'mention': 'Fig. 6', 'ref_id': 'FIGREF5'}]}, {'text': '\\nFig. 7\\nindicates that as the detection rate, modelled by ω, increases then the number of active cases decreases. It is important to note that a value of ω between 0.5 and 1.0 may not result in a significant change in the number of active cases. Thus, we recommend that the detection level of 0.5 and above may lead to a remarkable decline in the number of active cases.', 'cite_spans': [], 'section': 'Simulation results ::: Results and discussions', 'ref_spans': [{'start': 1, 'end': 7, 'mention': 'Fig. 7', 'ref_id': 'FIGREF6'}]}, {'text': 'Numerical illustration in Fig. 8\\ndepicts the effects of different levels of exposed individuals who progress to asymptomatic and infectious stage. As we can observe, an increase in the number of these individuals have a significant impact on short-and long-term dynamics of the disease. In particular, for any value of f greater than 20%, then the number of active cases may be greater than 10,000 in 100 days. However, for any value of f less than 20% the number of active cases may not exceed 10000, for the entire 300 day period.', 'cite_spans': [], 'section': 'Simulation results ::: Results and discussions', 'ref_spans': [{'start': 26, 'end': 32, 'mention': 'Fig. 8', 'ref_id': 'FIGREF7'}]}, {'text': \"The present study aimed to investigate the situation of COVID-19 in South Africa in the presence and absence of mitigation measures. A mathematical model that subdivides the total human population based on one's epidemiological status has been developed. The epidemiological classes considered are susceptible, exposed/latent, asymptomatic, infectious with mild symptoms, infectious with severe and critical symptoms, detected and quarantined, recovered and deceased. The proposed conceptual model has an additional compartment that captures the effects of public perception of risk of infection.\", 'cite_spans': [], 'section': 'Concluding remarks', 'ref_spans': []}, {'text': 'Although we have noted that the presence of intervention strategies may reduce the number of infections per unit time, there are optimal thresholds of intervention strategies that can lead to a significant reduction of cases. For instance, an initial 35 day lockdown followed by successive 14 day lockdowns with relaxation may not be as effective as an initial 35 day lockdown followed by successive 35 day lockdowns with relaxation. Furthermore, we observed that a detection rate of at least 0.5 per day may lead to a significant reduction of the number of active cases. In addition, we also noted that in the absence of intervention strategies the peak number of cases could be attained around mid-June whereas in the presence of intervention strategies the peak will be attained around the 23rd of August. Hence, we can deduce that the presence of intervention strategies may be responsible for the delay in attaining the peak, thereby prompting policy makers ample time to prepare for various and effective ways of managing the disease.', 'cite_spans': [], 'section': 'Concluding remarks', 'ref_spans': []}, {'text': 'The proposed framework could be of significant importance on understanding the transmission and control of COVID-19. However, we acknowledge that there are several aspects of the disease that are yet to be clearly unraveled, for instance, the duration one remains as an asypmtomatic infectious patient is still debatable. In the event that additional information have been found, it can be used to improve the framework.', 'cite_spans': [], 'section': 'Concluding remarks', 'ref_spans': []}, {'text': 'The authors received no specific funding for this work.', 'cite_spans': [], 'section': 'Funding', 'ref_spans': []}, {'text': 'All data have been included in the manuscript.', 'cite_spans': [], 'section': 'Availability of data and materials', 'ref_spans': []}, {'text': 'No personal information is used in this study, thus Not Applicable.', 'cite_spans': [], 'section': 'Consent for publication', 'ref_spans': []}, {'text': 'The authors declare that they have no known competing financial interests or personal relationships that could have appeared to influence the work reported in this paper.', 'cite_spans': [], 'section': 'Declaration of competing interest', 'ref_spans': []}], 'paragraphs': [{'text': '', 'id': '00006'}], 'title': 'On the role of governmental action and individual reaction on COVID-19 dynamics in South Africa: A mathematical modelling study', 'sha': '7a42a5631b784a94a278b0a30c5ac26dd8ba422f', 'doi': '10.1016/j.imu.2020.100387', 'score': 8.815699577331543}, {'rank': 268, 'paragraphs': [{'text': 'Nishiura et al. [20] Kimball et al. [21] Song et al. [22] Mizunoto et al. [23] Serra [24] Day [25] Meta−Random portion of asymptomatic infections are not counted when estimating the case fatality rate. Our sensitivity study addresses this important issue and makes an adjustment to provide a sensible estimate of the case fatality rate. Compared to the estimated 3.34% case fatality rate obtained from the meta-analysis, our sensitivity study estimates the case fatality rate to be 1.8% with 95% confidence interval (1.18%, 2.43%) where asymptomatic infections are accounted for. Our studies reveal sensible estimates for the important quantities of COVID-19 by accommodating discrepancy effects associated with different studies such as the variability of the data collected from different populations at different time periods. With the evolution of the pandemic, the basic production number can greatly reduce as a result of the implementation of active measures to mitigate the virus spread. The estimation of the case fatality rate may be closer to the true case fatality rate because of the increase of the test capacity; more infected cases may be detected so the reported number of infections would be closer to the true number of COVID-19 carriers. Our results are useful in enhancing the knowledge of COVID-19. Though we focus on evaluating the basic reproduction number, the average incubation time, the asymptomatic infection rate, and the case fatality rate, other features, such as the time from symptom onset to hospitalization or to death and the morbidity rate of the disease, are also important and they are worth being estimated in a sensible way.', 'id': '00045'}], 'title': 'Estimation of the basic reproduction number, average incubation time, asymptomatic infection rate, and case fatality rate forCOVID-19: Meta-analysis and sensitivity analysis', 'sha': '4bc132d5407496a57dabc2d8db6e46536f0a57f3', 'doi': '10.1101/2020.04.28.20083758', 'score': 8.814399719238281}, {'rank': 269, 'paragraphs': [{'text': 'Incubation period is defined as the interval from initial exposure to an infectious agent to onset of any symptoms or signs it causes. A long incubation period may lead to a high rate of asymptomatic and subclinical infection. The first prediction of mean incubation period was 5.2 days (95%CI, 4.1-7.0 days), with the 95th percentile of the distribution at 12.5 days, based on 2019-nCoV exposure histories of the first 425 cases in Wuhan(Li et al., 2020). A 4021-case study reported 4.75 days (interquartile range: 3.0-7.2 days) (Yang et al., 2020). Another 88-exported-case study calculated the mean incubation period to be 6.4 days (95%CI, 5.6–7.7 days), using known travel histories to and from Wuhan and symptom onset dates(Backer et al., 2020). All these literatures lay the foundation to set 14 days as the medical observation period if any exposure occurred. A latest study collected 1099 cases from 552 hospitals in 31 provinces in China and declared a median incubation period of 3.0 days, ranging from 0 to surprisingly 24.0 days. An adjustments in screening and control policies may be needed. The 2019-nCoV generally has a longer incubation time than SARS-CoV (4.0 days, 95% CI 3.6-4.4 days) (Lessler et al., 2009) and MERS-CoV (range 4.5-5.2 days) (Park et al., 2018).', 'id': '00006'}], 'title': 'Insight into 2019 novel coronavirus — an updated intrim review and lessons from SARS-CoV and MERS-CoV', 'sha': '4bc41aaa2b754b3ec526eb40c384a18ab38b8d51', 'doi': '10.1016/j.ijid.2020.03.071', 'score': 8.806900024414062}, {'rank': 270, 'paragraphs': [{'text': ', and, at times observed to be as long as two weeks. During this period exposed individuals are asymptomatic, but may still spread the virus -essentially behaving as invisible spreaders that continue to interact. Under such circumstances it is difficult to selectively isolate the sick, as the asymptomatic spreaders continue to infect the population [3] . This, we believe, may be the reason that, despite our seclusion of the infected individuals, we continue to witness a rapid growth in the disease coverage -further strengthening the need for an economically hurtful population wide lock-down.', 'id': '00001'}], 'title': 'Alternating quarantine for sustainable mitigation of COVID-19', 'sha': 'b923a8fc7d406d0f149ecd5293d1a72dec7ee526', 'doi': '', 'score': 8.800700187683105}, {'rank': 271, 'abstract': 'Novel coronavirus (COVID 19) rapidly spread from China to other parts of the world. Knowledge of incubation period and reproduction number is important in controlling any epidemic. The distribution of these parameters helps estimate the epidemic size and transmission potential of the disease. We estimated incubation period and reproduction number of COVID 19 for India utilizing data reported by Ministry of Health and Family Welfare (MoHFW), Government of India (GOI) and data in public domain. The mean incubation period seems to be larger at 6.93 (SD=5.87, 95% CI: 6.11, 7.75). and 95th percentile estimate for best fit normal distribution is 17.8 days. Weibull distribution, the best fit for the reproduction number estimated pre lockdown reproduction number as 2.6 (95% CI=2.34, 2.86) and post lockdown reduced to 1.57 (95% CI=1.3 , 1.84) implying effectiveness of the epidemic response strategies. The herd immunity is estimated between 36 to 61% for R0 of 1.57 and 2.6 respectively.', 'body_text': [{'text': 'While the novel coronavirus (COVID-19) spread rapidly from China to other developed countries, India saw a steady flow of patients mod early March and by 10 May 2020, it had gripped the country with 63,420 confirmed cases and 2109 deaths [1] [2] . The novel coronavirus (SARS-CoV-2) though related is distinct from severe acute respiratory syndrome (SARS) coronavirus and Middle East respiratory syndrome (MERS) coronavirus 3 . Many researchers struggled to estimate the magnitude of the epidemic wherein the epidemiological parameters remained uncertain. Knowledge of key epidemiological parameters including incubation period and reproduction number is important in controlling any epidemic. The distribution of these parameters helps estimate the epidemic size and transmission potential 3-7 of the disease.', 'cite_spans': [{'start': 238, 'end': 241, 'text': '[1]', 'ref_id': 'BIBREF0'}, {'start': 242, 'end': 245, 'text': '[2]', 'ref_id': None}, {'start': 424, 'end': 425, 'text': '3', 'ref_id': 'BIBREF1'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'The incubation period is defined as the time from infection to the onset of illness 8 and is crucial for epidemiological modelling in predicting the transmission dynamics, infectiousness and quarantine period 9 . It is also important for several important public health activities like length of active monitoring, surveillance and control.', 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'The reproduction number (R 0 ) is the most fundamental parameter in infectious disease dynamics describing the contagiousness or transmissibility of infectious agents and is defined as the average number of secondary cases caused by a single infectious individual in a entirely susceptible population 10 . An outbreak is expected to continue if R 0 has a value >1 and to end if R 0 is <1. R 0 fluctuates if the rate of human-to-human or human and vector interactions varies over time or space. There exists scant evidence which supports the applicability of R 0 outside the region for which the value was calculated 11 . Estimation of changes in transmission over time can provide insights into the epidemiological situation and identify whether outbreak control measures are adequate and are having the desired measurable effect, and help in undertaking midcourse corrections.', 'cite_spans': [{'start': 301, 'end': 303, 'text': '10', 'ref_id': None}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'Herd immunity is defined as the resistance to the spread of a contagious disease within a population that results if a sufficiently high proportion of individuals are immune to the disease, especially through vaccination or immunity post natural infection 12 . When a high proportion of the population is immune, it is difficult for All rights reserved. No reuse allowed without permission.', 'cite_spans': [{'start': 256, 'end': 258, 'text': '12', 'ref_id': 'BIBREF10'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'The copyright holder for this preprint this version posted June 29, 2020. . https://doi.org/10.1101/2020.06.27.20141424 doi: medRxiv preprint infectious diseases to spread, because there are not many people who can be infected and the transmission chain gets broken.', 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'Our current understanding of these epidemiological parameters for India is limited.', 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'Hence this study was undertaken to address above issue and estimate incubation period and reproduction number of COVID-19 for India utilizing data reported by Ministry of Health and Family Welfare (MoHFW), Government of India (GOI) and the data available in public domain.', 'cite_spans': [], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'The analysis is based on publicly available data. Data were retrieved from the official website of the MoHFW, GOI 1 16 . Since the analysis is based on publicly available data ethics approval was not required. ', 'cite_spans': [{'start': 114, 'end': 115, 'text': '1', 'ref_id': 'BIBREF0'}, {'start': 116, 'end': 118, 'text': '16', 'ref_id': 'BIBREF12'}], 'ref_spans': [], 'section': 'Data sources'}, {'text': 'The incubation period data post adjustment of delay-time in test results was subjected to best fit model. Besides normal distribution four other commonly used incubation period distribution (Weibull, Log normal, Gamma and Erlang) were fitted.', 'cite_spans': [], 'ref_spans': [], 'section': 'Statistical distributions for Incubation period and reproduction number'}, {'text': 'Estimation of the median incubation period, mean (sd), and quantiles (5 th , 25 th , 75 th , 95 th and 97 th ) was also done.', 'cite_spans': [], 'ref_spans': [], 'section': 'Statistical distributions for Incubation period and reproduction number'}, {'text': 'For reproduction number, the best fit model (Weibull distribution) was used. Besides the best, fit three more distributions based on review of literature for distribution of reproduction number (lognormal, Gamma and Exponential) were considered. The values of R 0 were estimated for pre and post lockdown period to evaluate the impact All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Statistical distributions for Incubation period and reproduction number'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Statistical distributions for Incubation period and reproduction number'}, {'text': 'The copyright holder for this preprint this version posted June 29, 2020. . https://doi.org/10.1101/2020.06.27.20141424 doi: medRxiv preprint of the epidemic control measures instituted by the Government of India. The herd immunity (HI) estimate was based on R 0 value 12 . (HI= (R 0 -1) or R 0 = 1 − 1/R 0 )', 'cite_spans': [], 'ref_spans': [], 'section': 'Statistical distributions for Incubation period and reproduction number'}, {'text': 'The statistical software used were IBM SPSS Statistics for Windows version 23.0 (SPSS Inc., Chicago, Ill., USA) and EASY-FIT, a software system for data fitting in dynamical systems.', 'cite_spans': [], 'ref_spans': [], 'section': 'Statistical distributions for Incubation period and reproduction number'}, {'text': 'Data on 268 lab confirmed cases were extracted, the range of age was from 1.5 years to 89 years with a mean age of 36.45 years (SD=±17.27). Ratio of male to female was 1.5:1 with 60.3% males and 39.7% females.', 'cite_spans': [], 'ref_spans': [], 'section': 'Results'}, {'text': 'Using date of exposure and date of confirmation of disease status/illness onset, the estimates of incubation period were determined. Table 1 gives the incubation period estimates for various distributions. We fitted five distributions to the data: Normal, Weibull, Log normal, Gamma and Erlang. The normal distribution provided best fit for data with median and mean incubation period of 6.93 (SD=±5.87, 95% confidence interval CI: 6.11-7.75). The incubation period ranged from 1 to 19.26 days (5 th percentile to 97 th percentile) for this best fit. The Weibull distribution was the best next fit with mean incubation period to be higher than normal distribution with mean of 8.17 (SD=8.10). Figure 1 show cumulative distribution functions for best fit. The median incubation period for other distributions ranged 3.46 to 6.06. The probability and cumulative distribution functions for various distributions in order of best fit is provided in the supplementary.', 'cite_spans': [{'start': 494, 'end': 499, 'text': '(5 th', 'ref_id': None}], 'ref_spans': [{'start': 133, 'end': 140, 'text': 'Table 1', 'ref_id': 'TABREF5'}, {'start': 693, 'end': 701, 'text': 'Figure 1', 'ref_id': None}], 'section': 'Incubation Period'}, {'text': 'The reproduction number was estimated for days before lockdown and post lockdown to assess the impact of various control measures to include social distancing adopted by India. Weibull distribution was the best fit for the reproduction number followed by Log Normal and Gamma distribution. We estimated the initial reproduction number before lockdown by GOI to be 2.6 (95% CI=2.34 -2.86) and post lockdown the reproduction number reduced to 1.57 (95% CI=1.3 -1.84). The herd immunity is estimated in the range of 61-62%. Table 2 gives the descriptive statistics of reproduction number before and post lock down. All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [{'start': 521, 'end': 528, 'text': 'Table 2', 'ref_id': 'TABREF6'}], 'section': 'Reproduction Number and Herd Immunity'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. ', 'cite_spans': [], 'ref_spans': [], 'section': 'Reproduction Number and Herd Immunity'}, {'text': 'This data driven paper to the best of our knowledge presents the estimates of incubation period and reproduction number of COVID-19 in India for the first time.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'We characterised the distribution of incubation period and reproduction number for COVID-19 for India. We estimated that the incubation period follows the best fit of normal distribution with around 7 days ranging from 1 to 22 day with 95 th percentile of the distribution at 17.8 days. The incubation period seems to be longer for India as compared to 2 to 10 days by WHO 13 and 2-12 days by ECDC 17 Weibull distribution seems to be the best fit for reproduction number distribution in India. The reproduction number for India is estimated to be 2.6 in pre lock down phase and reduced to around 1.6 post lockdown phase with a herd immunity threshold of 61.5%. Though the value of R 0 has reduced from 2.6 in pre lock down period to 1.57 in post lock down period. i.e. 60.38% reduction it is still greater than 1.', 'cite_spans': [{'start': 398, 'end': 400, 'text': '17', 'ref_id': 'BIBREF13'}], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'COVID-19 epidemic will increase as long as R 0 is greater than 1 and control efforts to bring R 0 below 1 needs to be implemented aggressively.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'A report based on the impact of the interventions across 11 European countries estimated a posterior mean of 0.97 [0.14-2.14] for Norway and 2.64 [1.40-4 .18] for Sweden, with an average of 1.43 across the 11 country posterior means, a 64% reduction compared to the pre-intervention values 23 . A study from Wuhan, Hubei All rights reserved. No reuse allowed without permission.', 'cite_spans': [{'start': 146, 'end': 153, 'text': '[1.40-4', 'ref_id': None}, {'start': 290, 'end': 292, 'text': '23', 'ref_id': None}], 'ref_spans': [], 'section': 'Discussion'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (95% CI: 2·47-2·86). The wide range of R 0 from different studies indicates the challenges in estimating R 0 . Additionally, R 0 is dynamic in the sense that any factor having the potential to influence the contact rate, including population density, social distancing, and seasonality will ultimately affect R 0 .', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'Our estimate of herd immunity threshold as 61% (with R 0 =2.6) is staggering as 826,873,494 people need to be infected to achieve herd immunity in India which may result in high death rate. Many countries have implemented aggressive lock downs however some countries like Sweden had a different approach to tackle COVID-19 by taking individual responsibility for social distancing and keeping society functioning with no official lock down by the Sweden government and still successful in keeping the reproduction number in control 26 . Long term lockdowns are not sustainable as its ill effects impacts many other health programmes. As per WHO 27 the weekly detection of new tuberculosis cases in India has gone down by nearly 75 per cent during the COVID-19 lockdown. The malaria modelling analysis by the Global Malaria Programme at WHO 28 and the HIV modelling study, convened by the WHO and UNAID 29 , predicts the collateral damage of COVID will be as devastating as pandemic, with millions succumbing to preventable, treatable illness and disease.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'However, in absence of vaccine and other uncertainties regarding the virus, herd immunity is debatable issue as a preventive measure and achieving herd immunity is likely to be a long-drawn process.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'Limitations of the study: The data on complete information for estimating incubation period as well reproduction number could be extracted for limited number of cases. Also, the time delay of test results, with varying turn around time among States, for estimating incubation period was averaged for India based on the public domain information and media reports. All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. ', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'This study provides the estimates for key epidemiological charachteristics of COVID-19 in India. The mean incubation period seems to be larger at 6.93 and 95 th percentile estimate for best fit normal distribution to be 17.8 days. Best fit for reproduction number follows weibull distribution. Our estimates for pre lockdown reproduction number was 2.6. and post lockdown the reproduction number reduced to 1.57. This implies and shows that the epidemic response strategies adopted by India are effective. However the herd immunity is estimated in the range of 36-61% for R 0 of 1.57 and 2.6 respectively. All rights reserved. No reuse allowed without permission.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusions'}, {'text': '(which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusions'}, {'text': 'The copyright holder for this preprint this version posted June 29, 2020. . https://doi.org/10.1101/2020.06.27.20141424 doi: medRxiv preprint ', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusions'}], 'paragraphs': [{'text': 'COVID-19 epidemic will increase as long as R 0 is greater than 1 and control efforts to bring R 0 below 1 needs to be implemented aggressively.', 'id': '00022'}], 'title': 'Incubation Period and Reproduction Number for novel coronavirus (COVID-19) infections in India', 'sha': '4cc2e02fbf932137fb5dda889927d090206f248e', 'doi': '10.1101/2020.06.27.20141424', 'score': 8.79889965057373}, {'rank': 272, 'paragraphs': [{'text': 'By fitting the observed forward times vi of the 1211 cases in our cohort to the likelihood function (4), we find the maximum likelihood estimates α^=2.04 (95% CI: 1.80−2⋅32) and λ^=0.103 (95% CI: 0⋅096−0⋅111) in our reference scenario. The estimated 5th, 25th, 50th, 75th, 90th, 95th, 99th, and 99·9th percentiles of the incubation period are 2·27 (95% CI: 1·73–2·86), 5·28 (95% CI: 4·53–6·06), 8·13 (95% CI: 7·37–8·91), 11·42 (95% CI: 10·74–12·11), 14·65 (95% CI: 14·00–15·26), 16·67 (95% CI: 15·94–17·32), 20·59 (95% CI: 19·47–21·62), and 25·12 (95% CI: 23·35–26·87) days, respectively. The mean incubation period is 8·62 (95% CI: 8·02–9·28) days. The average time from leaving Wuhan to symptom onset is 5·44 days, the sample median is 5 days, and the maximum is 21 days. Figure 2 visualizes the fitted density function (3) in a solid line onto the histogram of observed forward times, and the dashed line is the Weibull probability density function (2) for incubation period distribution. Note that (3) is a monotonically decreasing function, which fits the observed forward times well, suggesting that our model is reasonable and the results are therefore trustworthy.', 'id': '00021'}], 'title': 'Estimation of incubation period distribution of COVID-19 using disease onset forward time: a novel cross-sectional and forward follow-up study', 'sha': 'c61be9fc4562d250e2dec241ef75bd22d61f61d1; 6d3b3f4ab80a61c45f82c61c6c756cfc6ddf4bf2', 'doi': '10.1101/2020.03.06.20032417', 'score': 8.798700332641602}, {'rank': 273, 'paragraphs': [{'text': 'In addition to these three parameters, we consider several ways to treat the distribution of incubation times for individuals. The most common choice is to assume that the time each individual spends in the exposed state and the infectious state are independent exponential random variables with means t 1 and t 2 , respectively. This choice leads to a simple system of differential equations. However, in this approach some exposed individuals move to the infectious stage almost immediately, which is not realistic. Alternatively, we can assume that the time spent in the exposed state is always exactly t 1 , and the time spent in the infectious state is exactly t 2 , but that ignores the fact that some people get sick much faster than others, while some people stay undetected and infectious for much longer than others. A promising approach is a hybrid, keeping the time in the exposed state constant and making the time in the infectious state random.', 'id': '00013'}], 'title': 'Effects of latency on estimates of the COVID-19 replication number', 'sha': 'a017dd5e8dfa5dfd6385d407813e7c26d98becb9', 'doi': '10.1101/2020.04.07.20056960', 'score': 8.79539966583252}, {'rank': 274, 'paragraphs': [{'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02. 21.20026112 doi: medRxiv preprint to fall after that day. Among the 18 regions of Henan, Xinyang is with the heaviest epidemic, and it is currently also with the highest daily increment. We found that 55% patients in Henan were male, which is apparently higher than the females. We found that the gender difference of patients in Henan is 1.2226 : 1, while from Yang et al. [4] , they reported that the gender difference for 4021 patients from over 30 provinces is 0.31 : 0.27 = 1.1481 : 1. Our result is close to the existing ones. We reported that the MFRs of the whole province and its 18 regions are not the main factor that results to the gender difference in patients. Two possible main factors that result to such gender difference of patients include 1) the expression and distribution of ACE2s were wider in male patients than those in females, and 2) males are more active and with wider social activities than females in many regions of Henan. The ages of patients generally follow normal distribution, and with more than 81% patients are aged between 21 and 60 years old. N a n y a n g C e n tr a l H o s p . Incubation period is very important but very difficult to be estimated, since it is difficult to know when the patients were infected. We define the period from firstly contacting with suspicious persons or recently return from Wuhan to firstly appearing clinical symptoms (fever, cough etc) as incubation period, and we estimate its distribution. Statistical analysis on 483 patients reveals that the average estimated incubation period was 7.4 days, the mode was 4 days. About 55% patients were with incubation periods lower than 7 days, while more than 92% patients were lower than 14 days. About 7.45% patients were overestimated with more than 14 days incubation periods. Based on ME, OLS and MLE approaches, we modeled the incubation period as logarithmic normal distributions, and we found the ME method can well mimic the long right tails of the PDF for the incubation periods. Based on the ME approach, interquartile range of theoretically estimated incubation period is between 4.1 and 9.4 days. It is noted that, due to . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'id': '00037'}], 'title': 'Epidemiological characteristics of 1212 COVID-19 patients in Henan, China', 'sha': '091a8e9a61e19e88caeb039f0e3888d111b20439', 'doi': '10.1101/2020.02.21.20026112', 'score': 8.793399810791016}, {'rank': 275, 'paragraphs': [{'text': 'In Fig. 1a we show the results of the SEIR dynamics, starting from a fully susceptible population penetrated by a small seed of infectious individuals, i.e. S(t = 0) = 1−10 −3 , E(t = 0) = 10 −3 , I(t = 0) = 0 and R(t = 0) = 0. This captures the evolution of the unmitigated epidemic, absent any intervention policy, aside from isolation of the sick, i.e. people in the I state. We set the parameters to match the observed epidemiological data obtained for COVID-19: for the incubation period we set g(t) ∼ N (µ, σ 2 ), a normal distribution with mean µ = 5 days and σ = 0.5 [5, 6] . Assuming an average 11 day recovery from the onset of symptoms [8] , we set α = 0.09 (days −1 ). To evaluate the infection rate β we collected data capturing all diagnosed cases in Israel over a period of 20 days, providing an empirical observation of I(t). We find that the diagnosed cases can be wellapproximated by an exponential inflation of the form I(t) ∼∼ e kt , with k ≈ 0.3 days −1 . Setting β = 0.32 in (1) provides the desired exponential increase that best concurs with the observed data ( Fig. 1a ).', 'id': '00009'}], 'title': 'Alternating quarantine for sustainable mitigation of COVID-19', 'sha': 'b923a8fc7d406d0f149ecd5293d1a72dec7ee526', 'doi': '', 'score': 8.778300285339355}, {'rank': 276, 'paragraphs': [{'text': \"Wei Xia et al [21] undertook research into how the coronavirus was transmitted in incubation periods in 2019. The researchers harvested data on the demographics, possible exposure, and time to symptom onset for confirmed cases published by local Chinese authorities. They assessed the possibilities of transmission in the course of the incubation period for 50 clusters of infection; these included 124 cases outwith Wuhan/Hubei province. Every secondary case examined and been in contact with a first-generation case prior to experiencing symptoms. This research found that the mean incubation period for COVID-19 was 4.9 days (95% CI, 4.4-5.4), with a range of 0.8 to 11.1 days (2.5 th to 97.5 th percentile). The infectious curve demonstrated that 73% of secondary cases became infected prior to symptoms appearing for first-generation cases; this was especially the case in the final three days of incubation. These findings demonstrated that COVID-19 is transmitted between those who are in close contact in the course of incubation, which could indicate a weakness in the quarantine system. Robust workable countermeasures are required for the prevention or mitigation of the virus being transmitted asymptomatically amongst high-risk populations in the course of the incubation period. Juanjuan Z. [22] undertook research into the evolution of COVID-19's transmission dynamics and epidemiology outwith Hubei province. It was hoped that being able to understand the evolution of the transmission dynamics and epidemiology outwith the center of the epidemic would provide useful information that could be used as guidance for intervention policies. The researchers harvested data on individuals whose diagnosis was confirmed by laboratory testing in mainland China (apart from Hubei) that was reported by official public sources between January 19 and February 17, 2020. The date of the fourth time the case definition was revised (January 27) was employed as a means of dividing the epidemic into two phases (December 24-January 27 and January 28-February 17) as dates for symptom onset. Trends were estimated in terms of central time-event periods and subject demographics. This research encompassed 8579 cases, covering 30 provinces. The median age of the subjects was 44 years (range 33 to 56); as the epidemic continued, more cases emerged in the younger age groups and for the elderly (those aged over 64). Mean time between symptom onset and hospitalization fell from 4.4 days (% 95% CI 0.0-14.0) between December 24 and January 27, down to 2.6 days (0.0-9.0) between January 28 and February 17. For the whole period, mean incubation period was calculated at 5.2 days (1. 8-12.4 ) with the mean serial interval being 5.1 days (1.3-11.6).\", 'id': '00016'}], 'title': 'The estimations of the COVID-19 incubation period: a systematic review of the literature', 'sha': '09ff6011e4e95bff0b9221c89ffdb91d958e4ef1', 'doi': '10.1101/2020.05.20.20108340', 'score': 8.776700019836426}, {'rank': 277, 'paragraphs': [{'text': 'Our study suggests that the incubation period of COVID-19 in adolescents and young adults might be longer than that in older patients. A retrospective study reported a mean incubation period of 5.2 days (95% CI 4.1-7.0) and 95 th percentile of the incubation period of 12.5 days based on early COVID-19 patients from ', 'id': '00013'}], 'title': 'Epidemiological and Clinical Characteristics of COVID-19 in Adolescents and Young Adults', 'sha': '03b9ed59e65b1eb754eb9b24232a1e6ed2d85f9c; 8b19549de3e1e9bc6eddb055de2584a3b7060cdd', 'doi': '10.1016/j.xinn.2020.04.001', 'score': 8.774100303649902}, {'rank': 278, 'paragraphs': [{'text': 'flattened. (f) Similar results for I Peak vs. f . (g) Duration T vs. f . For small f the mitigation is rapid (elimination). For large f , however, the overall duration is increased -an inevitable price of flattening the curve. (h) -(j) We measured the three parameters vs. the variability in incubation times σ. We find that our strategy is highly insensitive to σ, however for σ > 5, a rather extreme level of variability, duration becomes highly extended.', 'id': '00030'}], 'title': 'Alternating quarantine for sustainable mitigation of COVID-19', 'sha': 'b923a8fc7d406d0f149ecd5293d1a72dec7ee526', 'doi': '', 'score': 8.771499633789062}, {'rank': 279, 'abstract': 'Background: In the beginning of January 2020, new unknown virus pneumonia cases started to emerge in local hospitals in Wuhan, China. This virus epidemic quickly became a public health emergency of international concern by the WHO. Enormous amount of medical supplies as well as healthcare personals from other provinces were mobilized to support Wuhan. This current work tent to help people understanding how infectious disease spread and the purpose and consequences of various efforts based on simulation model. Method: a simulation model was created using known parameters. R0 set to 3 and mean incubation time to be 7.5days. the epidemic was divided to 3 periods. Simulation would run 50 times to mimic different patient0 status. Personal activity index was used to mimic different level of control measures. 141427709 simulated patients were created. Cumulation number of patients at the end of period 1 (day50) is 2868.7 ± 1739.0. Total infected patients could be 913396.5 ± 559099.9 by the end of period 2 (day70) in free transmission state. And at day90, total patients number is 913396.5 ± 559099.9. Conclusion: COVID-19 is a novel severe respiratory disease. This will put great burden on the shoulder of healthcare workers as well as on medical hardware and supplements. Current strict control measures help to contain disease from spreading. An early detecting, reporting and fast reacting system needs to be setup to prevent future unknown infectious disease.', 'body_text': [{'text': 'In the beginning of January 2020, new unknown virus pneumonia cases started to emerge in local hospitals in Wuhan, China. transmissions between human were seen by the middle of January. And patient number increased sharply. In order to block the virus transmission from entire China and even worldwide. This virus epidemic quickly became a public health emergency of international concern by the World Health Organization.', 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': 'Wuhan municipal government issued an official order to stop all transportation between Wuhan and the rest of China. Public transportations within Wuhan were also stopped. By 9th, February 2020, China central government mobilized enormous amount of medical supplies as well as healthcare personals from other provinces to support Wuhan. Till 11th February，18700 healthcare personals belong to 154 medical teams had arrived in Wuhan and started to work with local hospitals. Wuhan government also declared regulations on residence building control, which include identity check at building entrance and strict infection control measures in buildings with suspect patients or identified patients.', 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': 'With all these intervention measures undertaken, and maybe more on the way, proper tools are need anticipate possible effects on epidemic control and to provide reliable information to support future decisions. Simulation also can help people understand how infectious disease spread and how to understand the purpose and consequences of various efforts.', 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': 'This current work was based on simulation model build on known transmission parameters from published research papers.', 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': \"According to Qun Li et al. [1] , 2019-nCoV's average incubation time is 7.5 days (95% CI 5.3-19.0), 95% percentiles is 12.5 days. In early stage of the disease, patients' number would double every 7.4 days. Basic reproduction number (R0) was presumed to be 2.2 (95% CI 1.4-3.9). In Qun's paper R0's 95% CI was between 1.4-3.9. R0=2, 3 or 4 were used in model preparation. And 2 and 4 were ruled out because of obvious devotion from open reported data. So R0 was set at 3 in this simulation.\", 'cite_spans': [{'start': 27, 'end': 30, 'text': '[1]', 'ref_id': 'BIBREF0'}], 'ref_spans': [], 'section': 'Method'}, {'text': 'Patient incubation time after been transmitted was assigned by python numpy random.normal function, with parameters set as: loc ( mean value of random serie )=7.5, scale ( standard deviation) = 3.4. Values lower than 0 are converted to positive values. In this study, simulation cases can only transmit virus to new patients on the last day of incubation to make calculation simple.', 'cite_spans': [], 'ref_spans': [], 'section': 'Method'}, {'text': \"According to media report and Wuhan government's announcement, the 2019-nCOV epidemic could be divided into 3 periods. . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\", 'cite_spans': [], 'ref_spans': [], 'section': 'Method'}, {'text': \"(which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02.14.20023168 doi: medRxiv preprint PAI: personal activity index, an artificial index created to demonstrate patients' mobility status in this study. PAI=1 means no restriction of social activity, while PAI=0 means totally grounded (include in house isolation and all sorts of hospitalization) .\", 'cite_spans': [], 'ref_spans': [], 'section': 'Method'}, {'text': 'Based on these parameters a simple simulation model was accomplished with python 3.6.', 'cite_spans': [], 'ref_spans': [], 'section': 'Method'}, {'text': 'All calculation was run on a mac pro notebook with 3.1 GHz duel core CPU and 8GB memory. Simulation would run 50 times to mimic different patient0 status.', 'cite_spans': [], 'ref_spans': [], 'section': 'Method'}, {'text': 'Data were shown as mean ± standard deviation.', 'cite_spans': [], 'ref_spans': [], 'section': 'Method'}, {'text': 'Simulations were run by 50 times. 141427709 simulated patients were created with incubation time 7.5 ± 3.4 days. The distribution of incubation time are show in Figure 2 . 913396.5 ± 559099.9 (Table 1) . And among these patients, 5% (more than 45000 patients) would be critically ill [2] .', 'cite_spans': [{'start': 284, 'end': 287, 'text': '[2]', 'ref_id': 'BIBREF1'}], 'ref_spans': [{'start': 161, 'end': 169, 'text': 'Figure 2', 'ref_id': 'FIGREF2'}, {'start': 192, 'end': 201, 'text': '(Table 1)', 'ref_id': None}], 'section': 'Results'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Results'}, {'text': 'The copyright holder for this preprint . https://doi.org/10.1101/2020.02.14.20023168 doi: medRxiv preprint Table 1 . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [{'start': 107, 'end': 114, 'text': 'Table 1', 'ref_id': None}], 'section': '(which was not peer-reviewed)'}, {'text': 'The copyright holder for this preprint . https://doi.org/10.1101/2020.02.14.20023168 doi: medRxiv preprint . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'The copyright holder for this preprint . https://doi.org/10.1101/2020.02.14.20023168 doi: medRxiv preprint . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'The copyright holder for this preprint . https://doi.org/10.1101/2020.02.14.20023168 doi: medRxiv preprint . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'The copyright holder for this preprint . https://doi.org/10.1101/2020.02.14.20023168 doi: medRxiv preprint . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'The copyright holder for this preprint . https://doi.org/10.1101/2020.02.14.20023168 doi: medRxiv preprint . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'The copyright holder for this preprint . https://doi.org/10.1101/2020.02.14.20023168 doi: medRxiv preprint Under multiple measures meant to contaminate this novel virus, diagnosed and suspected cases kept increasing. Among the infected patients, 5% or more were in critical condition and were located in intensive care units. When could the public see ending of this epidemic become a common concern. We used computer model simulation to show effects of . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': \"The copyright holder for this preprint . https://doi.org/10.1101/2020.02.14.20023168 doi: medRxiv preprint different level of efforts in containing this virus from spreading. To make things simple, restrictions of patients' activities (including in-hospital treatment, cabin hospital stay, and in-house surveillance) were represented as personal activity index PAI.\", 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': \"Read, Jonathan's preprint work showed a R0 to be 3.6-4.0 [3] . In our simulation study, data Till 11 th February 2020, more than 154 medical teams, 18700 healthcare personels were sent to Wuhan. A whole population body temperature monitoring program was also undertaken to identify fever patients. And local CDC announced to stop using time consuming RNA test as common diagnosis standard. Local residencies were locked down to minimize inter people transmission.\", 'cite_spans': [{'start': 57, 'end': 60, 'text': '[3]', 'ref_id': 'BIBREF2'}], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'How these measures would affect disease spreading is still not knowing. So we simulated the effects by set PAI from 0.9 to 0. PAI = 0.9 means no change in current people behavior while PAI = 0 means total grounded (include hospital stay and in-house surveillance).', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'With simulated data, we found that: . CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'The copyright holder for this preprint . https://doi.org/10.1101/2020.02.14.20023168 doi: medRxiv preprint At the beginning of stage 3, different PAI were used to calculate future trends in disease epidemic. We found that new daily patients number seem to stay in plateau when PAI =0.3.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': 'When PAI reduced to 0.2, a decline in daily new patient could be seen. If PAI could be restricted to 0.1, an ideal state of zero new daily patient would show up at the end of period 3.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not peer-reviewed)'}, {'text': '1. COVID-19 is a novel severe respiratory disease. It can cause strong inter people transmission. Enormous difficulties could be anticipated if not contained in early stage.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusion'}, {'text': '2. 5% of COVID-19 patients need intensive care. This will put great burden on the shoulder of healthcare workers as well as on medical hardware and supplements.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusion'}, {'text': '3. Current strict control measures help to contain disease from spreading. And long-term effect relates to how well Wuhan fulfill this task.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusion'}, {'text': '4. An early detecting, reporting and fast reacting system needs to be setup to prevent future unknown infectious disease.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusion'}, {'text': 'A respiratory disease spreading involves numerous factors. This simulation model uses only arbitrary data to make the calculation. And also, parameters are simplified to make the calculation faster, the results may be varying from real data. The model should be modified using actual data in order to get more accurate results.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusion'}, {'text': '. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusion'}, {'text': '(which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02.14.20023168 doi: medRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': 'Conclusion'}], 'paragraphs': [{'text': '3. Current strict control measures help to contain disease from spreading. And long-term effect relates to how well Wuhan fulfill this task.', 'id': '00029'}], 'title': 'A model simulation study on effects of intervention measures in Wuhan COVID-19 epidemic', 'sha': '57e01ad2a4961cd5cc6a3733f5f8c013a8946f3c', 'doi': '10.1101/2020.02.14.20023168', 'score': 8.757599830627441}, {'rank': 280, 'abstract': 'Abstract Background The outbreak of 2019 novel coronavirus disease (COVID-19) in Wuhan, China, has spread rapidly worldwide. In the early stage, we encountered a small but meaningful number of patients who were unintentionally scheduled for elective surgeries during the incubation period of COVID-19. We intended to describe their clinical characteristics and outcomes. Methods We retrospectively analyzed the clinical data of 34 patients underwent elective surgeries during the incubation period of COVID-19 at Renmin Hospital, Zhongnan Hospital, Tongji Hospital and Central Hospital in Wuhan, from January 1 to February 5, 2020. Findings Of the 34 operative patients, the median age was 55 years (IQR, 43–63), and 20 (58·8%) patients were women. All patients developed COVID-19 pneumonia shortly after surgery with abnormal findings on chest computed tomographic scans. Common symptoms included fever (31 [91·2%]), fatigue (25 [73·5%]) and dry cough (18 [52·9%]). 15 (44·1%) patients required admission to intensive care unit (ICU) during disease progression, and 7 patients (20·5%) died after admission to ICU. Compared with non-ICU patients, ICU patients were older, were more likely to have underlying comorbidities, underwent more difficult surgeries, as well as more severe laboratory abnormalities (eg, hyperleukocytemia, lymphopenia). The most common complications in non-survivors included ARDS, shock, arrhythmia and acute cardiac injury. Interpretation In this retrospective cohort study of 34 operative patients with confirmed COVID-19, 15 (44·1%) patients needed ICU care, and the mortality rate was 20·5%. Funding National Natural Science Foundation of China.', 'body_text': [{'text': 'In December 2019, an outbreak of the 2019 novel coronavirus disease (COVID-19) caused by the SARS coronavirus 2 (SARS-CoV-2) occurred in Wuhan, China [1,2]. It has spread rapidly to other areas in China and worldwide. [3,4] The most common manifestations of COVID-19 included fever, dry cough, dyspnea, myalgia, fatigue, hypolymphaemia, and radiographic evidence of pneumonia. Complications (eg, acute respiratory distress syndrome [ARDS], arrhythmia, shock, acute cardiac injury, secondary infection, and acute kidney injury) and death may occur in severe cases. [2,[5], [6], [7]] The course of the COVID-19 is long, and COVID-19 is highly contagious even during the incubation period. [8] Furthermore, asymptomatic carrier of SARS-CoV-2, accounting for 1% of the laboratory confirmed cases of SARS-CoV-infection, [9] may potentially transmit the virus during incubation time, [10] which makes the identification and prevention of COVID-19 infection highly challenging. During the early phase of the COVID-19 outbreak, we encountered a small number of asymptomatic patients who underwent elective surgeries during the incubation period of COVID-19 infection, but the clinical manifestations and prognosis of these patients were beyond our expectation. It is our belief that these represent a specific surgical patient population that deserves our attention.', 'cite_spans': [{'start': 151, 'end': 152, 'mention': '1', 'ref_id': 'BIBREF0'}, {'start': 153, 'end': 154, 'mention': '2', 'ref_id': 'BIBREF1'}, {'start': 219, 'end': 220, 'mention': '3', 'ref_id': 'BIBREF2'}, {'start': 221, 'end': 222, 'mention': '4', 'ref_id': 'BIBREF3'}, {'start': 565, 'end': 566, 'mention': '2', 'ref_id': 'BIBREF1'}, {'start': 567, 'end': 570, 'mention': '[5]', 'ref_id': 'BIBREF4'}, {'start': 572, 'end': 575, 'mention': '[6]', 'ref_id': 'BIBREF5'}, {'start': 577, 'end': 580, 'mention': '[7]', 'ref_id': 'BIBREF6'}, {'start': 687, 'end': 690, 'mention': '[8]', 'ref_id': 'BIBREF7'}, {'start': 815, 'end': 818, 'mention': '[9]', 'ref_id': 'BIBREF8'}, {'start': 878, 'end': 882, 'mention': '[10]', 'ref_id': 'BIBREF9'}], 'section': 'Introduction', 'ref_spans': []}, {'text': 'Currently, the data on the clinical characteristics and outcomes of patients with COVID-19 infection undergoing surgeries are rare. A study by Chen and colleagues reported nine pregnant women who had COVID-19 infection and whose babies needed to be delivered via cesarean section surgery. [11] However, this report mainly focused on the general clinical characteristics of pregnant women and whether or not intrauterine vertical transmission may occur. The data on the impact of surgery performed unintentionally during the incubation period of COVID-19 infection on the outcomes of patients are lacking, but are of utmost importance to reduce the morbidity and mortality, given that surgery may cause an immediate impairment of cell-mediated immunity, [12] one of the major mechanisms that bring viral infections under control. [13] In this study, we aim to present the epidemiological, clinical, and laboratory characteristics, treatment, and outcomes of patients undergoing elective surgeries during the incubation period of COVID-19 infection and to compare severe patients who received ICU care during disease progression and those who did not receive ICU care. We hope that our findings of the SARS-CoV-2 associated postoperative morbidity and mortality will benefit the global community in the battle against COVID-19 infection.', 'cite_spans': [{'start': 289, 'end': 293, 'mention': '[11]', 'ref_id': 'BIBREF10'}, {'start': 753, 'end': 757, 'mention': '[12]', 'ref_id': 'BIBREF11'}, {'start': 829, 'end': 833, 'mention': '[13]', 'ref_id': 'BIBREF12'}], 'section': 'Introduction', 'ref_spans': []}, {'text': 'This is a multicenter, retrospective study, which was done at Renmin Hospital, Zhongnan Hospital, Tongji Hospital and Central Hospital in Wuhan, the most serious area of COVID-19 epidemic in China. We retrospectively reviewed patients who had undergone elective surgeries admitted from January 1 to February 5, 2020, the early stage of COVID-19 epidemic in Wuhan, China. 37 asymptomatic patients developed symptoms after operation and were diagnosed with COVID-19 according to WHO interim guidance. [14] Laboratory confirmation of SAR-CoV-2 was done by quantitative RT-PCR on samples from the respiratory tract, which was performed by the local health authority as described. [2,5] Of these patients, 3 patients were visited shortly after surgery by persons who were thereafter confirmed for COVID-19 infection, which made us difficult to judge whether or not the patients’ infections were due to their exposures prior to surgery. Thus, these 3 patients were excluded, and finally 34 patients were included in this study.', 'cite_spans': [{'start': 499, 'end': 503, 'mention': '[14]', 'ref_id': 'BIBREF13'}, {'start': 677, 'end': 678, 'mention': '2', 'ref_id': 'BIBREF1'}, {'start': 679, 'end': 680, 'mention': '5', 'ref_id': 'BIBREF4'}], 'section': 'Study design and participants ::: Methods', 'ref_spans': []}, {'text': 'This study was reviewed and approved by the Medical Ethical Committee of participating institutes (approval number WDRY2020-K067). Oral consent was obtained from the patients. The clinical outcomes of these operative patients were monitored up to March 10, 2020, the final date of follow-up, when all the patients were discharged.', 'cite_spans': [], 'section': 'Study design and participants ::: Methods', 'ref_spans': []}, {'text': 'The patients included in this study underwent various surgical procedures as shown in Table 1\\n, and were categorized into four levels based on the degree of technical difficulty, complexity and risk according to the measures for the hierarchical management of surgical procedures published by the National Heath Commission of China. [15] Briefly, level-1, various operations with low risks, simple procedures and low technical difficulty; level-2, various operations with mild risks, general complexity of procedures and general technical difficulty; level-3, various operations with moderate risks, complex procedures, and moderate technical difficulty; level-4, various operations with high risks, highly complex procedures and high technical difficulty.', 'cite_spans': [{'start': 333, 'end': 337, 'mention': '[15]', 'ref_id': 'BIBREF14'}], 'section': 'Surgical difficulty category ::: Methods', 'ref_spans': [{'start': 86, 'end': 93, 'mention': 'Table 1', 'ref_id': 'TABREF0'}]}, {'text': 'We reviewed clinical records, nursing records, laboratory findings, and chest computed tomographic (CT) scans for all 34 operative patients. The admission date of these patients was from January 1 to February 5, 2020. Epidemiological, clinical, laboratory, and radiological characteristics and treatment and outcomes data were obtained with data collection forms from electronic medical records. Information included demographic, exposure history, underlying comorbidities, chest CT image, surgical type, surgical time, signs and symptoms, time of surgery to first symptoms, time of first symptom to dyspnea, vita signs and laboratory values on hospital admission, COVID-19 onset and ICU admission, treatments, complications, and prognosis. The durations from hospital admission to surgery, first symptom, dyspnea, ARDS, and ICU admission were also recorded. Two research investigators independently reviewed the data collection forms to verify data accuracy. Any missing or uncertain records were collected and clarified by direct communication with patients and their families.', 'cite_spans': [], 'section': 'Data collection ::: Methods', 'ref_spans': []}, {'text': 'The time of COVID-19 onset was defined as the date when the first sign or symptom was noticed. Acute respiratory distress syndrome (ARDS) was defined according to the Berlin definition. [16] Acute cardiac injury was identified if the cardiac biomarkers (eg, hypersensitive troponin I, Creatine kinase–MB) were above the 99% upper reference limit or new abnormalities were shown in electrocardiography and echocardiography. [5] Acute kidney injury was defined according to the KDIGO clinical practice guidelines. [17] Patients were admitted and transferred to intensive care unit (ICU) based on the progression of organ dysfunction or the need of mechanical ventilation. [2,18] For patients admitted to ICU, myocardial enzymes, inflammatory stress, and blood gas analysis were determined on the day of ICU admission.', 'cite_spans': [{'start': 186, 'end': 190, 'mention': '[16]', 'ref_id': 'BIBREF15'}, {'start': 423, 'end': 426, 'mention': '[5]', 'ref_id': 'BIBREF4'}, {'start': 512, 'end': 516, 'mention': '[17]', 'ref_id': 'BIBREF16'}, {'start': 671, 'end': 672, 'mention': '2', 'ref_id': 'BIBREF1'}, {'start': 673, 'end': 675, 'mention': '18', 'ref_id': 'BIBREF17'}], 'section': 'Definitions ::: Methods', 'ref_spans': []}, {'text': \"Continuous variables were presented as median with interquartile range (IQR) and compared by using independent group t tests when the data were normally distributed; otherwise, the Mann–Whitney test was used. Categorical variables were expressed as frequencies and percentages and compared by Pearson's chi-square or Fisher's exact test between ICU and non-ICU groups or survival and death groups. A two-sided α of less than 0.05 was considered statistically significant. All statistical analyses were performed with the SPSS (version 25·0) software.\", 'cite_spans': [], 'section': 'Statistical analysis ::: Methods', 'ref_spans': []}, {'text': 'The funders had no role in study design, data collection, data analysis, data interpretation, and the writing of the report. The corresponding authors had full access to all the data in the study and had final responsibility for the decision to submit for publication.', 'cite_spans': [], 'section': 'Role of the funding source ::: Methods', 'ref_spans': []}, {'text': 'All 34 patients included in this study were residents of Wuhan City. The age range was 21 to 84 years, and median age was 55 years (IQR 43–63) (Table 2\\n). 20 (58·8%) patients were woman, and 15 (44·1%) patients were admitted to ICU because of the progression of organ dysfunction or the need for mechanical ventilation. 20 (58·8%) patients had 1 or more comorbidities. Hypertension (13 [38·2%]), malignancy (9 [26·5%]), diabetes (8 [23·5%]), and cardiovascular disease (7 [20·6%]) were the most common comorbidities.', 'cite_spans': [], 'section': 'Results', 'ref_spans': [{'start': 144, 'end': 151, 'mention': 'Table 2', 'ref_id': 'TABREF1'}]}, {'text': 'In the present study, most patients underwent surgeries with the surgical difficulty category at level-2 (11 [32·4%]) and level-3 (20 [58·8%]), only 2 (5·9%) patients underwent surgeries with the surgical difficulty category at level-4, the highest surgical difficulty category (Table 2). 13 of 15 patients admitted to ICU underwent level-3 surgeries. By contrast, the surgical difficulty category was level-2 for the majority of non-ICU patients. The overall median surgical time was 178 min (IQR, 70–249). The patients in ICU had longer surgical time (median time, 200 min [IQR, 125–240] vs 70 min [IQR, 53–215]; P = 0·04) and shorter time from surgery to first symptom (median time, 2·0 days [IQR, 1·0–4·0] vs 5·0 days [IQR, 2·0–6·5]; P = 0.02) than that of non-ICU patients.', 'cite_spans': [], 'section': 'Results', 'ref_spans': [{'start': 279, 'end': 286, 'mention': 'Table 2', 'ref_id': 'TABREF1'}]}, {'text': 'The most common symptoms at COVID-19 onset were fever (31 [91·2%]), fatigue (25 [73·5%]), dry cough (18 [52·9%]), dyspnea (15 [44·1%]), myalgia or arthralgia (11 [32·4%]), and expectoration (11 [32·4%]). Less common symptoms were dizziness, headache, pharyngalgia, nausea, diarrhea, and abdominal pain. The median time from surgery to first symptom was 2·0 days (IQR, 1·0–4·0) and 3.0 days (IQR, 2.0–4.5) to diagnosis of pneumonia. The median time from first symptoms to dyspnea was 3·5 days (IQR, 2·0–5·3) (Table 2). The median duration from hospital admission to surgery was 2·5 days (IQR, 1·0–4·0), to first sign or symptoms was 5·0 days (IQR, 3·3–8·8), to dyspnea was 9·5 days (IQR 6·0–14·5), to ARDS was 10·0 days (IQR, 8·0–15·0). For the non-survival patients, the median time from hospital admission to death was 16·0 days (IQR, 11·0–25·0) (Fig. 1\\n).', 'cite_spans': [], 'section': 'Results', 'ref_spans': [{'start': 848, 'end': 854, 'mention': 'Fig. 1', 'ref_id': 'FIGREF0'}, {'start': 508, 'end': 515, 'mention': 'Table 2', 'ref_id': 'TABREF1'}]}, {'text': 'Compared with patients who did not receive ICU care, patients who required ICU care were significantly older (median age, 55 years [IQR, 44–74] vs 47 years [IQR, 29–58]; P = 0·03), and were more likely to have underlying comorbidities (12 [80·0%] vs 8 [42·1%; P = 0.04), including hypertension (9 [60·0%] vs 4 [21·1%]; P = 0·03) and cardiovascular disease (6 [40·0%] vs 1 [5·3%]); P = 0·03) (Table 2).', 'cite_spans': [], 'section': 'Results', 'ref_spans': [{'start': 392, 'end': 399, 'mention': 'Table 2', 'ref_id': 'TABREF1'}]}, {'text': 'There were numerous differences in laboratory findings between ICU and non-ICU patients (Table 3\\n), including higher white blood cell (median count, 8·5 [IQR, 4·7–13·1] × 109/L vs 5·7 [IQR, 4·6–8·1] × 109/L; P = 0·049) and neutrophil counts (median count, 7·9 [IQR, 4·1–10·7] × 109/L vs 4·1 [IQR, 3·1–5·8] × 109/L; P = 0·02), as well as higher levels of total bilirubin (median concentration, 13·0 μmol/L [IQR, 9·1–19·2] vs 8·1μmol/L [IQR, 6·5–12·9]; P = 0·04), blood urea nitrogen (median concentration, 5·8 mmol/L [IQR, 4·9–10·3] vs 4·0 mmol/L [IQR, 3·3–4·2]; P = 0·03) and creatinine (median concentration, 63·0 μmol/L [IQR, 60·0–76·0] vs 46·8 μmol/L [IQR, 42·0–57·0]; P = 0·049). The number of patients with increased concentration of procalcitonin (≥ 0·1 ng/mL) in ICU was more than non-ICU patients (12 [80·0%] vs 7 [36·8%]; P = 0·02). These measures were recorded on the day of COVID-19 onset for all patients. All the 34 patients demonstrated bilateral distribution of patchy shadows or ground glass opacity on chest CT scan (Table 3).', 'cite_spans': [], 'section': 'Results', 'ref_spans': [{'start': 89, 'end': 96, 'mention': 'Table 3', 'ref_id': 'TABREF2'}, {'start': 1034, 'end': 1041, 'mention': 'Table 3', 'ref_id': 'TABREF2'}]}, {'text': 'All patients had developed pneumonia after surgery. Common complications among the 34 patients included ARDS (11 [32·4%]), shock (10 [29·4%]), secondary infection (10 [29·4%]), arrhythmia (8 [23·5%]), acute cardiac injury (5 [14·7%]), and acute kidney injury (2 [5·9%]; Table 3). ICU patients were more likely to have ARDS, shock, second infection and acute cardiac injury than non-ICU patients. All patients received antiviral therapy (lopinavir/ritonavir) and antibiotic therapy. Part of the patients received glucocorticoid therapy (16 [47·1%]) and immunoglobulin therapy (14 [41·2%]), and 1 (2·9%) patient received kidney replacement therapy. In ICU, 7 (46·7%) patients received high-flow oxygen or noninvasive ventilation, and 5 (33·3%) required invasive mechanical ventilation, 1 (6·7%) of whom received extracorporeal membrane oxygenation as rescue therapy (Table 4\\n).', 'cite_spans': [], 'section': 'Results', 'ref_spans': [{'start': 270, 'end': 277, 'mention': 'Table 3', 'ref_id': 'TABREF2'}, {'start': 865, 'end': 872, 'mention': 'Table 4', 'ref_id': 'TABREF3'}]}, {'text': 'Seven patients died after admission to the ICU, they all underwent surgeries at the surgical difficulty category level-3. The age range was 34 to 83 years old, and 4 were women (Table 5\\n). The surgery duration ranged from 110 to 379 min. All these patients had 1 or more coexisting medical conditions. The most common comorbidities were cardiovascular disease (4 [57·1%]), malignancy (4 [57·1%]) and hypertension (3 [42·9%]). 4 of these 7 patients presented with fever as first symptom. The median duration from first symptom to death was 9 days (IQR, 6–11). All these patients developed respiratory failure and had three or more complications. The most common complications among the 7 patients included ARDS (7 [100%]), shock (4 [57·1%]), arrhythmia (4 [57·1%]) and acute cardiac injury (4 [57·1%]).', 'cite_spans': [], 'section': 'Results', 'ref_spans': [{'start': 178, 'end': 185, 'mention': 'Table 5', 'ref_id': 'TABREF4'}]}, {'text': 'This report, to the best of our knowledge, is the first retrospective cohort study to describe the clinical characteristics and outcomes of operative patients with SARS-CoV-2 infection. All the 34 patients involved in this study had a history of direct exposure to Wuhan City before hospital admission, and none of them had any sign or symptom of COVID-19 before surgery. Of note, symptoms of COVID-19 manifested quickly after the completion of surgery, and SARS-CoV-2 infection was laboratory-confirmed soon thereafter. The length of time from hospital admission to surgery (median time, 2·5 days [IQR, 1·0–4·0]) is shorter than the median incubation time of 5·2 days obtained from a study of patients with confirmed SARS-CoV-2 infections in Wuhan, [19] and also shorter than the overall incubation time (median time, 4·0 days [IQR, 2·0–7·0]) derived from a study of patients with COVID-19 from 552 hospitals in China. [6] These evidences collectively support our belief that the patients included in the current study are in their incubation period of COVID-19 infection before undergoing surgeries.', 'cite_spans': [{'start': 750, 'end': 754, 'mention': '[19]', 'ref_id': 'BIBREF18'}, {'start': 920, 'end': 923, 'mention': '[6]', 'ref_id': 'BIBREF5'}], 'section': 'Discussion', 'ref_spans': []}, {'text': 'During the disease progression, 15 of 34 postoperative patients received ICU care. This proportion (44·1%) is much higher than the reported 26·1% in hospitalized COVID-19 patients without surgery. [2] Most patients in ICU were older, had more underlying comorbidities and longer surgical time, and undergone more difficult surgery than patients not admitted to ICU. This suggests that old age, comorbidities, surgical time, and difficulty of operation may be risk factors for poor outcome. Compared with the symptoms in non-ICU patients, dyspnea occurred earlier in critically ill patients. The time of first symptom to dyspnea in ICU patients was much shorter than non-ICU patients. This symptom characteristic may help physicians identify patients with potential poor prognosis.', 'cite_spans': [{'start': 197, 'end': 200, 'mention': '[2]', 'ref_id': 'BIBREF1'}], 'section': 'Discussion', 'ref_spans': []}, {'text': 'The data in this study suggest that surgery may accelerate and exacerbate disease progression of COVID-19. This is derived from the following findings. The patients developed COVID-19 symptoms very shortly (average 2·6 days) after surgery completion. The median time of COVID-19 onset (which was defined as the date when the first sign or symptom was noticed) to dyspnea in the current study was 3·5 days (IQR, 2·0–5·3), which is shorter than the reported time of 8·0 days (IQR, 5·0–13·0) from the onset of first symptom to dyspnea in a study of 41 laboratory-confirmed cases infected with SARS-CoV-2, [5] and also shorter than the reported 5·0 days (IQR, 1·0–10·0) in another study of 138 hospitalized patients with confirmed SARS-CoV-2 in Wuhan. [2] Seven of the 34 operative patients (20·6%) died of COVID-19 associated complications. This mortality rate is much higher than the reported overall case-fatality rate of 2·3% in COVID-19 patients without surgery, [9] and also higher than the case-fatality rate of 7·9% in noncardiac surgical patients without COVID-19 infection who were admitted to multidisciplinary ICU. [20] Furthermore, the average duration from the time of first symptom to death of the 7 non-survivors (8·7 days, range 4 to 13 days) is apparently shorter than that reported previously. In the report of Huang and colleagues, [5] the median time from onset of symptoms to first hospital admission was 7.0 days (4.0–8.0) and to ICU admission was 10.5 days (8.0–17.0) and 6 patients (15% of the total of 41 cases studied) did not survive ICU. The study did not report the exact dates of patient death, but it is safe for us to say that the average time from the onset of symptoms to death was longer than 10.5 days and thus longer than that of the non-survivors in our study. In other words, it is highly likely that surgical stress occurred during the incubation period of SARS-CoV-2 infection exacerbated disease progression and severity.', 'cite_spans': [{'start': 602, 'end': 605, 'mention': '[5]', 'ref_id': 'BIBREF4'}, {'start': 748, 'end': 751, 'mention': '[2]', 'ref_id': 'BIBREF1'}, {'start': 964, 'end': 967, 'mention': '[9]', 'ref_id': 'BIBREF8'}, {'start': 1123, 'end': 1127, 'mention': '[20]', 'ref_id': 'BIBREF19'}, {'start': 1348, 'end': 1351, 'mention': '[5]', 'ref_id': 'BIBREF4'}], 'section': 'Discussion', 'ref_spans': []}, {'text': \"Currently, no specialized medication is available for the treatment of SARS-CoV-2 infection, and supportive measures remain the mainstay of COVID-19 treatment. Thus, the patient's immune function is a major determinant of the disease severity, and populations with low immune function, such as older people are more vulnerable and have high mortality after COVID-19 infection. [7] Surgery may not only cause immediate impairment immune function, [12] but also induce early systemic inflammatory response. [21] Similar to the Middle East Respiratory Syndrome Coronavirus (MERS-CoV) infection, [22] the SARS-CoV infected lung could induce and increase the amount of macrophage and neutrophil infiltration, and increase the levels of pro-inflammatory cytokines and chemokines. [23,24] And, the high levels of circulating inflammatory cytokines has been reported to be correlated with the severity of illness in patients infected with the 2019 novel coronavirus (SARS-CoV-2). [5] Consistent with these findings, the most common laboratory abnormalities in this study were lymphopenia (29 [85·3%] of 34 patients) and increased hypersensitive C-reactive protein. In addition, the patients admitted to ICU showed higher counts of white blood cell and neutrophil.\", 'cite_spans': [{'start': 377, 'end': 380, 'mention': '[7]', 'ref_id': 'BIBREF6'}, {'start': 446, 'end': 450, 'mention': '[12]', 'ref_id': 'BIBREF11'}, {'start': 505, 'end': 509, 'mention': '[21]', 'ref_id': 'BIBREF20'}, {'start': 592, 'end': 596, 'mention': '[22]', 'ref_id': 'BIBREF21'}, {'start': 775, 'end': 777, 'mention': '23', 'ref_id': 'BIBREF22'}, {'start': 778, 'end': 780, 'mention': '24', 'ref_id': 'BIBREF23'}, {'start': 972, 'end': 975, 'mention': '[5]', 'ref_id': 'BIBREF4'}], 'section': 'Discussion', 'ref_spans': []}, {'text': \"Our study has some similarity to a case reported during MERS-CoV outbreak. [25] In the report, a 61-year-old man who was electively admitted for cardiac bypass surgery and the patient's laboratory examinations were normal despite the history of hypertension and diabetes, and MERS-CoV test was negative before the surgery. However, this patient developed MERS-CoV infection-related symptom in the 2nd day after surgery, and was tested positive for MERS-CoV on post-operative day 6, and the patient passed away on post-operative day 9. Possibility exists that this patient could be a healthy carrier of MERS-CoV who then had the virus activated by the stress of surgery. For the patients included in our current study, we performed routine laboratory tests but did not specifically test for SARS-CoV-2 before surgery by use of quantitative RT-PCR with the CDC (Chinese Center for Disease Control and Prevention) recommended kit based on the considerations that these patients had no any sign of COVID-19 related symptoms and the CDC recommended kit was very limited and could only be used for highly suspected case of SARS-CoV-2 infection at that time. However, a thorough review of the patients' profile made us confident to believe that the 34 patients included in this study got infected before their hospital admission. We now retrospectively understand that it might be necessary and very important for a patient to be isolated for a certain period (such as the WHO recommended 14-days quarantine period) or the possibility of the new coronavirus infection being excluded before being considered for an elective surgery during the COVID-19 epidemic.\", 'cite_spans': [{'start': 75, 'end': 79, 'mention': '[25]', 'ref_id': 'BIBREF24'}], 'section': 'Discussion', 'ref_spans': []}, {'text': 'Our study has several notable limitations. The sample size of the present study was small, as it was a rare situation and patients were unintentionally scheduled for elective surgeries during the incubation period of COVID-19 infection. Additionally, the patients were not performed the specific SARS-CoV-2 confirmation test before surgery, due to the limited understanding of the epidemic situation and the shortage of SARS-CoV-2 kits at that time. Thus, our assumption of the intubation periods of the patients included the current study was mainly based on clinical profiles and routine laboratory tests. Nevertheless, it is our hope that the findings of the SARS-CoV-2 associated postoperative morbidity and mortality will alert the global community to be better prepared in the battle against COVID-19 infection.', 'cite_spans': [], 'section': 'Discussion', 'ref_spans': []}, {'text': 'In summary, we described the clinical characteristics and outcomes of the patients who were unintentionally scheduled for elective surgeries during the incubation period of COVID-19 infection. This retrospective cohort study showed 44·1% patients needed ICU care, and mortality was 20·5%. Risk factors for the poor prognosis of operative patients with COVID-19 need to be further study in larger sample size.', 'cite_spans': [], 'section': 'Discussion', 'ref_spans': []}, {'text': 'Z-YX and SL had the idea for the study. Z-YX and ZX designed the study and have full access to all data in the study and take responsibility for the integrity of the data and the accuracy of the data analysis. SL, WS, CC, JC, WM, l-YZ and YJ collected the data. SL, FJ, DL and LZ performed data analysis. SL, FJ and WS drafted the manuscript. ZX and Z-YX revised the final manuscript.', 'cite_spans': [], 'section': 'Contributors', 'ref_spans': []}, {'text': 'None.', 'cite_spans': [], 'section': 'Conflict of Interest', 'ref_spans': []}], 'paragraphs': [{'text': 'In December 2019, an outbreak of the 2019 novel coronavirus disease (COVID-19) caused by the SARS coronavirus 2 (SARS-CoV-2) occurred in Wuhan, China [1,2]. It has spread rapidly to other areas in China and worldwide. [3,4] The most common manifestations of COVID-19 included fever, dry cough, dyspnea, myalgia, fatigue, hypolymphaemia, and radiographic evidence of pneumonia. Complications (eg, acute respiratory distress syndrome [ARDS], arrhythmia, shock, acute cardiac injury, secondary infection, and acute kidney injury) and death may occur in severe cases. [2,[5], [6], [7]] The course of the COVID-19 is long, and COVID-19 is highly contagious even during the incubation period. [8] Furthermore, asymptomatic carrier of SARS-CoV-2, accounting for 1% of the laboratory confirmed cases of SARS-CoV-infection, [9] may potentially transmit the virus during incubation time, [10] which makes the identification and prevention of COVID-19 infection highly challenging. During the early phase of the COVID-19 outbreak, we encountered a small number of asymptomatic patients who underwent elective surgeries during the incubation period of COVID-19 infection, but the clinical manifestations and prognosis of these patients were beyond our expectation. It is our belief that these represent a specific surgical patient population that deserves our attention.', 'id': '00001'}], 'title': 'Clinical characteristics and outcomes of patients undergoing surgeries during the incubation period of COVID-19 infection', 'sha': 'ee9ecf938304f7f068074ccc6271845eb6128c76', 'doi': '10.1016/j.eclinm.2020.100331', 'score': 8.746100425720215}, {'rank': 281, 'paragraphs': [{'text': 'h(v)=αλ{π(vλ)α−1+(1−π)/Γ(1/α)}exp(−(vλ)α),v≥0.', 'id': '00018'}], 'title': 'Estimation of incubation period distribution of COVID-19 using disease onset forward time: a novel cross-sectional and forward follow-up study', 'sha': 'c61be9fc4562d250e2dec241ef75bd22d61f61d1; 6d3b3f4ab80a61c45f82c61c6c756cfc6ddf4bf2', 'doi': '10.1101/2020.03.06.20032417', 'score': 8.746099472045898}, {'rank': 282, 'abstract': 'Since the outbreak of the new coronavirus epidemic, novel coronavirus has infected nearly 100,000 people in more than 110 countries. How to face this new coronavirus epidemic outbreak is an important issue. Basic reproduction number (R0) is an important parameter in epidemiology; The basic reproduction number of an infection can be thought of as the expected number of cases directly generated by one case in a population where all individuals are susceptible to infection. Epidemiology dynamics is a mathematical model based on a susceptibility-infection-recovery epidemic model. Researchers analyzed the epidemiological benefits of different transmission rates for the establishment of effective strategy in prevention and control strategies for epidemic infectious diseases. In this review, the early use of TCM for light and ordinary patients, can rapidly improve symptoms, shorten hospitalization days and reduce severe cases transformed from light and normal. Many TCM formulas and products have wide application in treating infectious and non-infectious diseases. The TCM theoretical system of treating epidemic diseases with TCM and the treatment scheme of integrated Chinese and Western medicine have proved their effectiveness in clinical practice. TCM can cure COVID-19 pneumonia, and also shows that the role of TCM in blocking the progress of COVID-19 pneumonia.', 'body_text': [{'text': 'Since the outbreak of the new coronavirus epidemic, novel coronavirus has infected nearly 100,000 people in more than 110 countries in Asia, Europe, America, Africa and other regions. The epidemic situation in China, South Korea, Iran, Italy, Japan and other countries has shown a rapid growth trend. Nature and Science have issued separate articles saying that the novel coronavirus pneumonia epidemic may have become a “global pandemic” (Callaway, 2020; Cohen & Kupfeschmidt, 2020).', 'cite_spans': [{'start': 440, 'end': 454, 'mention': 'Callaway, 2020', 'ref_id': 'BIBREF1'}, {'start': 456, 'end': 482, 'mention': 'Cohen & Kupfeschmidt, 2020', 'ref_id': 'BIBREF3'}], 'section': 'Introduction: an outbreak of novel coronavirus epidemic', 'ref_spans': []}, {'text': \"The WHO's decision to hold off describing the global outbreaks as a ‘pandemic’ was based in part on data showing that infections in China had peaked between 27th January and 18th February in 2020 and that control measures, such as the partial travel control of cities, where the virus originated, had worked to prevent new cases (Table 1\\n). WHO affirmed that China has taken practical actions and made positive and effective contributions to the prevention and control of the epidemic.\", 'cite_spans': [], 'section': 'Introduction: an outbreak of novel coronavirus epidemic', 'ref_spans': [{'start': 330, 'end': 337, 'mention': 'Table 1', 'ref_id': 'TABREF0'}]}, {'text': \"When a viral infection spreads from person to person in a region, country or countries, it causes persistent outbreaks. From the first case of patients increased gradually, its spread at a particular level also won't be affected by the formation of significant; With the development of a pandemic outbreak to be close to a certain number of infections and severity (such as death). It is more than healthcare system capacity (Table 1), the Chinese Center for Disease Control and Prevention (CDC) will improve the warning level, the administrative department to take strict measures to control of epidemic diseases.\", 'cite_spans': [], 'section': 'Introduction: an outbreak of novel coronavirus epidemic', 'ref_spans': [{'start': 426, 'end': 433, 'mention': 'Table 1', 'ref_id': 'TABREF0'}]}, {'text': 'According to the WHO, a pandemic is the worldwide spread of a new disease. \"A pandemic is when an epidemic spreads between countries\" says David Jones. In the case of COVID-19 specifically, the WHO said that it\\'s the first pandemic caused by a coronavirus.', 'cite_spans': [], 'section': 'Introduction: an outbreak of novel coronavirus epidemic', 'ref_spans': []}, {'text': \"A pandemic is the highest possible level of disease, or a measure of how many people have gotten sick from a particular disease and how far it has spread—but before a common illness reaches pandemic proportions, it has to exceed a few other levels (Brabaw, 2020): (1) Sporadic: When a disease occurs infrequently and irregularly; (2) Endemic: A constant presence and/or usual prevalence of a disease or infection within a geographic area (Hyperendemic, is a situation in which there are persistent, high levels of disease occurrence.); (3) Epidemic: A sudden increase in the number of cases of a disease—more than what's typically expected for the population in that area; (4) Pandemic: An epidemic that has spread over several countries or continents, affecting a large number of people.\", 'cite_spans': [{'start': 249, 'end': 261, 'mention': 'Brabaw, 2020', 'ref_id': 'BIBREF0'}], 'section': 'Introduction: an outbreak of novel coronavirus epidemic', 'ref_spans': []}, {'text': 'According to WHO news, a pandemic is the global spread of a new disease. The global medical community recognized the \"pandemic\" includes the following three criteria: (1) The virus in the human population is pathogenic and deadly, that is, the virus caused a high risk of disease; (2) The virus can be transmitted continuously from person to person; (3) There is evidence that the disease is prevalent worldwide.', 'cite_spans': [], 'section': 'Introduction: an outbreak of novel coronavirus epidemic', 'ref_spans': []}, {'text': 'A new pneumonia outbreak is characterized as a \"pandemic,\" dr. Tedros Adhanom Ghebreyesus (WHO director-general) said on March 11th. He said the extent and severity of the outbreak was deeply concerning, which is why we assess that the novel coronavirus pneumonia outbreak is characterized as a pandemic. The number of new crown pneumonia cases outside China has increased 13-fold in the past two weeks, and the number of affected countries and regions has tripled; More than 118,000 cases have been confirmed in 114 countries and regions, with 4291 deaths and thousands more fighting for their lives in hospitals. The number of confirmed cases, the number of deaths, and the number of affected countries and regions are expected to rise further in the coming weeks. “We have never seen a pandemic caused by a coronavirus before,” he added. “And we\\'ve never seen a controlled pandemic before.” Therefore, the WHO calls on countries to take urgent and active action. We have sounded the alarm loud and clear.', 'cite_spans': [], 'section': 'Introduction: an outbreak of novel coronavirus epidemic', 'ref_spans': []}, {'text': 'It was only a matter of time before the word pandemic was used. Marc Lipsitch, an American epidemiologist of infectious diseases, “I think the epidemiological conditions for a pandemic are met,” in Nature (Callaway, 2020).', 'cite_spans': [{'start': 206, 'end': 220, 'mention': 'Callaway, 2020', 'ref_id': 'BIBREF1'}], 'section': 'Introduction: an outbreak of novel coronavirus epidemic', 'ref_spans': []}, {'text': \"The World Health Organization (WHO) still avoided using the word “pandemic” to describe the burgeoning crisis today. Whatever the rest of the world does, it's essential that it needs take action soon. WHO hopes other countries will learn from China. Burnce Aylward says “The single biggest lesson is: Speed is everything,”. “And you know what worries me most? Has the rest of the world learned the lesson of speed?”\", 'cite_spans': [], 'section': 'WHO: novel coronavirus epidemic for outbreak of pandemic ::: How to face this new coronavirus epidemic for a pandemic outbreak', 'ref_spans': []}, {'text': \"WHO calls on all countries to activate and expand emergency response mechanisms; Communicate with citizens about the risks of the virus and how to protect themselves, because that's everyone's job; Screen, isolate, test and treat each case and follow up each close contact; Prepare the hospital; Protect and train medical workers. Finally, let's help each other, because we need each other. WHO and governments should adopt a series of countermeasures, including (1) strengthening cooperation and sharing of information between national health agencies and international organizations; (2) humanitarian assistance to countries with insufficient medical resources; (3) to speed up the research and development of vaccines and the approval of antiviral drugs; (4) measures taken by states to reduce or suspend activities that cause public gatherings; and (5) countries issue travel warnings according to the situation.\", 'cite_spans': [], 'section': 'WHO: novel coronavirus epidemic for outbreak of pandemic ::: How to face this new coronavirus epidemic for a pandemic outbreak', 'ref_spans': []}, {'text': 'Writing in NEJM, Bill Gates said the new pneumonia outbreak could become a once-in-a-century pandemic and called on governments to act quickly to prepare for a response (Gates, 2020). He is mainly based on the following two points. First, the case fatality rate of the 2019 novel coronavirus is around 1%, which is between the 1957 pandemic (case fatality rate 0.6%) and the 1918 pandemic (case fatality rate 2%), indicating that it is several times worse than typical seasonal influenza. Second, novel coronavirus pneumonia is highly contagious. An average infected person can infect two or three people, an exponential increase, and can be spread from mild to asymptomatic patients. Novel coronavirus pneumonia has caused ten times as many confirmed cases as SARS in just a quarter of the time.', 'cite_spans': [{'start': 170, 'end': 181, 'mention': 'Gates, 2020', 'ref_id': 'BIBREF8'}], 'section': 'WHO: novel coronavirus epidemic for outbreak of pandemic ::: How to face this new coronavirus epidemic for a pandemic outbreak', 'ref_spans': []}, {'text': 'In a NEJM (New England Journal of Medicine) paper, Gates cites five references (Gates, 2015; The Novel Coronavirus Pneumonia Emergency Response Epidemiology Team, 2020; Hoehl, Rabenau & Berger; Frieden et al., 2020; Gates, 2018) to illustrate his point, and clearly points to the establishment of a public health safety system and concerns about its implementation. He said “We need to build a system that can develop safe, effective vaccines and antivirals, get them approved, and deliver billions of doses within a few months after the discovery of a fast-moving pathogen”. In addition, “Governments and other donors will need to fund — as a global public good — manufacturing facilities that can generate a vaccine supply in a matter of weeks”.', 'cite_spans': [{'start': 80, 'end': 91, 'mention': 'Gates, 2015', 'ref_id': 'BIBREF6'}, {'start': 93, 'end': 167, 'mention': 'The Novel Coronavirus Pneumonia Emergency Response Epidemiology Team, 2020', 'ref_id': 'BIBREF22'}, {'start': 169, 'end': 192, 'mention': 'Hoehl, Rabenau & Berger', 'ref_id': 'BIBREF13'}, {'start': 194, 'end': 214, 'mention': 'Frieden et al., 2020', 'ref_id': 'BIBREF5'}, {'start': 216, 'end': 227, 'mention': 'Gates, 2018', 'ref_id': 'BIBREF7'}], 'section': 'WHO: novel coronavirus epidemic for outbreak of pandemic ::: How to face this new coronavirus epidemic for a pandemic outbreak', 'ref_spans': []}, {'text': 'Basic Reproduction Number (R0), sometimes called basic reproductive ratio, or incorrectly basic reproductive rate, and denoted R0, pronounced R zero, is an important index in epidemiology. Expected number of secondary cases of disease produced directly by an average infectious individual entering an entirely susceptible population. As shown in Fig. 1\\n, when R0 > 1, epidemic occurs to form A, B, C, D, E and F generation transmission, and R0 > 1, no epidemic occurs.', 'cite_spans': [], 'section': 'Ability to recognize virus infecting: basic infection number (R0) ::: How to face this new coronavirus epidemic for a pandemic outbreak', 'ref_spans': [{'start': 346, 'end': 352, 'mention': 'Fig. 1', 'ref_id': 'FIGREF0'}]}, {'text': 'The basic reproduction number of an infection can be thought of as the expected number of cases directly generated by one case in a population where all individuals are susceptible to infection (Zhao, Tian et al., 2020a; Zhao et al., 2020b).', 'cite_spans': [{'start': 195, 'end': 219, 'mention': 'Zhao, Tian et al., 2020a', 'ref_id': 'BIBREF26'}, {'start': 221, 'end': 239, 'mention': 'Zhao et al., 2020b', 'ref_id': 'BIBREF27'}], 'section': 'Ability to recognize virus infecting: basic infection number (R0) ::: How to face this new coronavirus epidemic for a pandemic outbreak', 'ref_spans': []}, {'text': 'The outcome of each pathogen depends on its R0 value and the population of this closed community (Table 2\\n). Instead of focusing prevention and control efforts on preventing the spread of the epidemic, reducing the spread rate and losses, two things are waiting to happen: first, it is expected that specific drugs or vaccines for COVID-19 will be developed as soon as possible; Second, it is expected that more and more healthy young and middle-aged people infected with the virus will be able to develop immunity, thus establishing a group immunity barrier to protect the most dangerous old, weak, sick and disabled.', 'cite_spans': [], 'section': 'Ability to recognize virus infecting: basic infection number (R0) ::: How to face this new coronavirus epidemic for a pandemic outbreak', 'ref_spans': [{'start': 98, 'end': 105, 'mention': 'Table 2', 'ref_id': 'TABREF1'}]}, {'text': \"Risk of harm or death can be determined for certain groups of people, whose mortality apparently increases with age. In fact, it is the patient's immunity (acquired immunity ability) that is related to it. The prognosis trend of the patient mortality from viral infection is shown in Fig. 2\\n.\", 'cite_spans': [], 'section': 'Epidemiological dynamics is a basis to establish the healthcare system capacity to meet the needs of the epidemic ::: How to face this new coronavirus epidemic for a pandemic outbreak', 'ref_spans': [{'start': 284, 'end': 290, 'mention': 'Fig. 2', 'ref_id': 'FIGREF1'}]}, {'text': 'The high mortality rate of the elderly or patients with underlying diseases is a real problem. From the above data, it can be seen that the elderly infected persons are the main risk groups. Mathematical statistics or models are based on many obvious or implicit assumptions. Therefore, for situations like the new coronavirus that we do not fully understand, we should need to adjust the control measures or reduce the risks according to the outbreak speed and the changed number of infected and dead people. Many infected persons whose symptoms are not obvious, the possibility of shortage of medical resources still exist.', 'cite_spans': [], 'section': 'Epidemiological dynamics is a basis to establish the healthcare system capacity to meet the needs of the epidemic ::: How to face this new coronavirus epidemic for a pandemic outbreak', 'ref_spans': []}, {'text': 'Good disease prevention and control strategy is an important basis for reducing disease transmission rate and mortality rate. Epidemiology dynamics is a mathematical model based on a susceptibility-infection-recovery epidemic model, which changes with the change of the number of patients found as the epidemic time of infectious diseases passes. Data simulation is carried out based on the time and the number of patients to form a transmission dynamics curve (Fig. 3\\n). Researchers analyzed the epidemiological benefits of different transmission rates. This is an effective strategy to reduce the transmission rate and correspondingly reduce the prevailing level of local stability. Therefore, this epidemiological dynamic model is the basis for establishing prevention and strategies control for epidemic infectious diseases.', 'cite_spans': [], 'section': 'Epidemiological dynamics is a basis to establish the healthcare system capacity to meet the needs of the epidemic ::: How to face this new coronavirus epidemic for a pandemic outbreak', 'ref_spans': [{'start': 462, 'end': 468, 'mention': 'Fig. 3', 'ref_id': 'FIGREF2'}]}, {'text': \"For new pathogens (COVID-19), it is necessary to understand the process of their transmission and the update of risk factors, and the corresponding epidemic prevention policies will be changed accordingly, and the prediction system based on healthcare system capacity should be established instead (Fig. 3). The size of the risk is related to the country's epidemic prevention policy, and the ability to implement healthcare.\", 'cite_spans': [], 'section': 'Epidemiological dynamics is a basis to establish the healthcare system capacity to meet the needs of the epidemic ::: How to face this new coronavirus epidemic for a pandemic outbreak', 'ref_spans': [{'start': 299, 'end': 305, 'mention': 'Fig. 3', 'ref_id': 'FIGREF2'}]}, {'text': 'In Fig. 3, the national healthcare system capacity with high level of control measures helps to reduce spread of diseases. But without control measures to reduce risk of diseases is no action taken during the outbreak of infectious diseases. In fact, the “no action taken” is “non-feasance” for this capacity system.', 'cite_spans': [], 'section': 'Epidemiological dynamics is a basis to establish the healthcare system capacity to meet the needs of the epidemic ::: How to face this new coronavirus epidemic for a pandemic outbreak', 'ref_spans': [{'start': 3, 'end': 9, 'mention': 'Fig. 3', 'ref_id': 'FIGREF2'}]}, {'text': 'Based on an article from Daily Mail, the UK government\\'s new plan urges people with mild coronavirus-like symptoms to self-quarantine (Daily Mail Online, March 13, 2020). Around 60% of the UK population will need to become infected with coronavirus in order for society to have \"herd immunity\" from future outbreaks, Sir Patrick Vallance said. Herd immunity (Community Immunity) is the resistance to a contagious disease within a population because enough people have become immune, and so it is harder for it to spread (Sky News, March 13, 2020). At present, the UK government has adjusted the formulation of \"Herd immunity\" to prevent and control diseases, to a certain extent easing the external worries about the British government\\'s response to the epidemic situation.', 'cite_spans': [{'start': 135, 'end': 168, 'mention': 'Daily Mail Online, March 13, 2020', 'ref_id': 'BIBREF4'}, {'start': 521, 'end': 545, 'mention': 'Sky News, March 13, 2020', 'ref_id': 'BIBREF19'}], 'section': '“Herd immunity” is not suitable for prevention and control strategies in outbreak of infectious diseases ::: How to face this new coronavirus epidemic for a pandemic outbreak', 'ref_spans': []}, {'text': 'The mentioned \"Herd Immunity\" in a report in the UK, that is, after most of the people have antibodies to the virus, the epidemic will not erupt on a large scale. In fact, it is the government\\'s responsibility to evade the establishment of a scientific epidemic prevention and control system, and it is a lie that violates infectious disease prevention and control strategies. The concept of “Community Immunity” was originally an academic concept in the field of epidemiological research, which was essentially the \"minimum Immunity rate\". However, when vaccines were widely used to fight against large-scale infectious diseases, the concept gained new life. But human beings have proved to be different from cattle, and humanity has determined that we will not voluntarily give up protecting anyone, even the old and the sick.', 'cite_spans': [], 'section': '“Herd immunity” is not suitable for prevention and control strategies in outbreak of infectious diseases ::: How to face this new coronavirus epidemic for a pandemic outbreak', 'ref_spans': []}, {'text': 'Theoretically, whether the herd immunity is effective or not, it needs to reach the threshold of the herd immunity, which is determined by the basic infection number R0 of the virus. R0 is the average number of people infected by a patient, and if R0 is less than 1, the more the virus spreads, the fewer people it infects, and it is difficult to form a large-scale infection. Assuming R0=3 for the new coronavirus, that would be an infected person, with an average of three people infected. If more than 67% of a population is immunized, the number of infected people will be less than 1, which will result in R0 < 1, indicating the epidemic is under control, and you need to increase the number of people in the population to believe that herd immunity can be formed.', 'cite_spans': [], 'section': '“Herd immunity” is not suitable for prevention and control strategies in outbreak of infectious diseases ::: How to face this new coronavirus epidemic for a pandemic outbreak', 'ref_spans': []}, {'text': 'According to the above view point, how to prevent novel coronavirus? At present, one of the characteristics of COVID-19 virus is that the incubation period is very long, and there are many asymptomatic infected persons who do not suffer from the disease for a long time. Some discharged patients may have positive nucleic acid tests and still carry the virus. If so, do you think it is difficult to eradicate the virus?', 'cite_spans': [], 'section': '“Herd immunity” is not suitable for prevention and control strategies in outbreak of infectious diseases ::: How to face this new coronavirus epidemic for a pandemic outbreak', 'ref_spans': []}, {'text': 'The best way to prevent and treat serious infectious diseases that have been effective in our country is “to find them early, isolate them early, and treat early”. We will try our best to track and isolate every infected person until they are completely controlled and new infected persons are avoided. For this reason, we will do whatever we can to take some strict measures. This is the successful experience and measures currently adopted in our country.', 'cite_spans': [], 'section': '“Herd immunity” is not suitable for prevention and control strategies in outbreak of infectious diseases ::: How to face this new coronavirus epidemic for a pandemic outbreak', 'ref_spans': []}, {'text': 'Of course, the cost is huge that cannot be copied in all countries. Then, once the strict control measures are relaxed, the virus will come back and may spread very violently. Therefore, measures should be taken to delay the spread of the virus, but do not expect it to be completely eliminated. If the spread of the virus can be delayed and there are not too many patients at any one time, then medical resources will not be overstretched and excessive mortality will not result.', 'cite_spans': [], 'section': '“Herd immunity” is not suitable for prevention and control strategies in outbreak of infectious diseases ::: How to face this new coronavirus epidemic for a pandemic outbreak', 'ref_spans': []}, {'text': 'The local food and drug administration to speed up the examination and approval of hospital preparations in order to facilitate medical institutions to deal with the epidemic situation quickly, hospital preparations approved by Beijing, Guangdong, Shandong, Shanxi, Jilin, Yunnan, Shaanxi, Sichuan and other provinces and cities are used for the treatment of COVID-19. But the application can only be adjusted within the local scope. According to statistics, 21 prescriptions have been approved and used in clinical therapy.', 'cite_spans': [], 'section': 'Hospital preparations ::: Effective prescription for treatment of COVID-19 pneumonia ::: TCM advantages in treatment of SARS-CoV-2 infection', 'ref_spans': []}, {'text': 'Qingfei Paidu Decoction is derived from the combination of five classical prescriptions of TCM, namely Maxing Shigan Decoction, Shegan Mahuang Decoction, Xiaochaihu Decoction and Wuling Powder. These classic prescriptions were recorded in \"Treatise on Febrile and Miscellaneous Diseases\" written by Zhang Zhong-jing in the Han Dynasty. Most of them are classic prescriptions for treating exogenous febrile diseases caused by cold pathogens with the characteristics of reasonable prescription, mild nature and taste.', 'cite_spans': [], 'section': 'Qingfei Paidu decoction ::: Effective prescription for treatment of COVID-19 pneumonia ::: TCM advantages in treatment of SARS-CoV-2 infection', 'ref_spans': []}, {'text': \"According to statistics, 214 confirmed cases were treated with Qingfei Paidu Decoction in four pilot provinces. Three days is a course of treatment, with a total effective rate of more than 90%, of which more than 60% patients' symptoms and imaging performance improved significantly, and 30% patients' symptoms were stable and without aggravation. The formula can also be used for patients with common cold and influenza. Chemical substances in Qingfei Paidu Decoction also showed inhibitory effect on 3CL protein of coronavirus (State Administration of Traditional Chinese Medicine, 2020; Zhao, Tian et al., 2020a; Zhao et al., 2020b).\", 'cite_spans': [{'start': 531, 'end': 589, 'mention': 'State Administration of Traditional Chinese Medicine, 2020', 'ref_id': 'BIBREF20'}, {'start': 591, 'end': 615, 'mention': 'Zhao, Tian et al., 2020a', 'ref_id': 'BIBREF26'}, {'start': 617, 'end': 635, 'mention': 'Zhao et al., 2020b', 'ref_id': 'BIBREF27'}], 'section': 'Qingfei Paidu decoction ::: Effective prescription for treatment of COVID-19 pneumonia ::: TCM advantages in treatment of SARS-CoV-2 infection', 'ref_spans': []}, {'text': 'Pneumonia No.1 Formula (officially named as \"Thoroughly Resolving Antifungal Granules\") is a clinical experience formula of the Chinese Medicine Department of Guangzhou Eighth People\\'s Hospital (Guangdong Provincial Drug Administration, 2020). The recipe is mainly composed of five Chinese herbal medicines. Under the guidance of the theory of febrile diseases, this prescription is determined to treat COVID-19 with the principles of clearing away heat and toxic materials, dispelling wind and penetrating the exterior, invigorating qi and nourishing yin.', 'cite_spans': [{'start': 195, 'end': 241, 'mention': 'Guangdong Provincial Drug Administration, 2020', 'ref_id': 'BIBREF9'}], 'section': 'Pneumonia no.1 formula ::: Effective prescription for treatment of COVID-19 pneumonia ::: TCM advantages in treatment of SARS-CoV-2 infection', 'ref_spans': []}, {'text': 'Clinical application was reported on the treatment of 50 patients with confirmed diagnosis of COVID-19 (mild disease). After one week of clinical observation, the body temperature of all patients returned to normal, cough symptoms disappeared in 50% of patients, the pharyngalgia symptoms disappeared in 52.4%, fatigue symptoms disappeared in 69.6%, and no patient turned to severe disease. Because TCM pays attention to syndrome differentiation and treatment, according to different clinical manifestations and pathological changes, taking this prescription as the basic prescription, the use of drugs will be different (Guangzhou Eighth Hospital, 2020).', 'cite_spans': [{'start': 622, 'end': 653, 'mention': 'Guangzhou Eighth Hospital, 2020', 'ref_id': 'BIBREF10'}], 'section': 'Pneumonia no.1 formula ::: Effective prescription for treatment of COVID-19 pneumonia ::: TCM advantages in treatment of SARS-CoV-2 infection', 'ref_spans': []}, {'text': \"Xiao et al. observed the clinical therapeutic effect of Shufeng Jiedu Capsule (SFJDC) combined with arbidol in the treatment of mild disease with COVID pneumonia (Xiao et al., 2020). Two hundred patients were divided into control group and treatment group with 100 cases in each group. The control group was treated with arbidol while the observation group was treated with SFJDC. The course of treatment was 2 weeks. The symptoms of fever and cough, white blood cell count, lymphocyte percentage and chest CT manifestations were compared between the two groups. After treatment, the white blood cell count, lymphocyte percentage and chest CT were significantly improved in the two groups (P < 0.05). The improvement in the observation group was better than that in the control group (P < 0.05). Fever regression time in observation group was shorter than that in control group (P < 0.05). The total effective rate of the observation group was 88%, higher than 75% of the control group (P < 0.05). SFJDC has a good antiviral effect, and has a good effect on respiratory system symptoms such as fever, cough, chest tightness and shortness of breath. Combined with abidor in the treatment of mild NPC, it can significantly increase the percentage level of white blood cells and lymphocytes in patients' blood routine and make chest CT infection foci absorb obviously. TCM can participate in prevention and treatment in the whole process and in all directions, complementing the advantages of chemical drug and improving the curative effect. In this clinical observation, the adverse reactions of SFJDC are less and can be better popularized clinically. Later, large sample and multi-center research are needed to obtain more evidence-based medical evidence to further guide clinical treatment.\", 'cite_spans': [{'start': 163, 'end': 180, 'mention': 'Xiao et al., 2020', 'ref_id': 'BIBREF25'}], 'section': 'Shufeng Jiedu capsule ::: Effective prescription for treatment of COVID-19 pneumonia ::: TCM advantages in treatment of SARS-CoV-2 infection', 'ref_spans': []}, {'text': '\\nWang, Chen, Lu, Chen and Zhang (2020) reported here the clinical characteristics and therapeutic procedure for four patients with mild or severe COVID-19 pneumonia admitted to Shanghai Public Health Clinical Center. All the patients were given antiviral treatment including lopinavir/ritonavir (Kaletra), arbidol, and SFJDC, a TCM product and other necessary support care. After treatment, three patients gained significant improvement in pneumonia associated symptoms, two of whom were confirmed SARS-1 negative and discharged, and one of whom was virus negative at the first test. The remaining patient with severe pneumonia had shown signs of improvement by the cutoff date for data collection. Results obtained in the current study may provide clues for treatment of 2019-nCoV pneumonia. In clinical study, two mild and two severe COVID-19 pneumonia patients were given combined Chinese and Western medicine treatment, three of whom gained significant improvement in pneumonia associated symptoms. The remaining patient with severe pneumonia has shown signs improvement by the cutoff date for data collection. The efficacy of antiviral treatment including lopinavir/ritonavir, arbidol, and SFJDC warrants further verification in future study.', 'cite_spans': [{'start': 1, 'end': 38, 'mention': 'Wang, Chen, Lu, Chen and Zhang (2020)', 'ref_id': 'BIBREF24'}], 'section': 'Shufeng Jiedu capsule ::: Effective prescription for treatment of COVID-19 pneumonia ::: TCM advantages in treatment of SARS-CoV-2 infection', 'ref_spans': []}, {'text': 'The research has noted that the type II transmembrane serine proteases TMPRSS2 and HAT can cleave and activate the spike protein (S) of the SARS-CoV for membrane fusion. Heurich et al’ results in conjunction with those of previous studies indicate that TMPRSS2 and potentially related proter omote SARS-CoV entry by two separate mechanisms: ACE2 cleavage, which might promoteviral uptake, and SARS-S-protein cleavage, which activates the S-protein for membrane fusion. Their observations have interesting implications for the development of novel therapeutic drugs using peptidases as entry receptor (Heurich et al., 2014). Network pharmacology studies reveal that more than 10 kinds of ingredients may through the HRAS, 3-phosphoinositide dependent protein kinase 1 (PDPK1), silk crack the original activated protein kinase phosphatase in the role of 1 (MAP2K1) 31 targets such as inflammation, adhesive (focal adhesion), silk crack the original amp-activated protein kinase (MAPK), Fc epsilon RI, peroxidase growth activated receptor (PPAR), vascular endothelial growth factor (VEGF), B cell receptor and T cell receptors signal 19 pathways (Tao, Meng & Han, 2017).', 'cite_spans': [{'start': 601, 'end': 621, 'mention': 'Heurich et al., 2014', 'ref_id': 'BIBREF12'}, {'start': 1144, 'end': 1165, 'mention': 'Tao, Meng & Han, 2017', 'ref_id': 'BIBREF21'}], 'section': 'Basic research on compatibility of traditional Chinese medicine was expounded by using the synergetic effect of multi-targets ::: TCM advantages in treatment of SARS-CoV-2 infection', 'ref_spans': []}, {'text': 'Using natural product library, 720 compounds were screened to inhibit the activity of 3CL Pro (Li, Chang & Han, 2017). The centers for disease control in the area provided the homogeneous purified total RNA from a throat swab taken from a patient with severe acute respiratory syndrome. These results suggest that TF2B and TF3 compounds (Fig. 4\\n) may be a good starting point for the design of a more effective 3CL Pro-inhibitor encoded by SARS-CoV. It is necessary to further study the inhibitory effect of these natural products on SARS-CoV replication in cell culture (Chen, Lin Coney & Huang, 2005). Researchers suggested that attention should be paid to the clinical treatment of SARS-Cov-2 patients infected with pneumonia (The observer, 2020), and 30 drugs that may be effective against SARS-CoV-2 were also found, such as the chemical components in Chinese medicinal materials such as Chinese knotwort (Polygonum cuspidatum Siebold & Zucc) and Rhizoma Alba (Atractylodes macrocephala Koidz.). The study showed that the chemical constituents of Radix Bupleurae (Bupleurum chinense DC), liquorice, honeysuckle and other traditional Chinese medicines had inhibitory effects on Mpro.', 'cite_spans': [{'start': 95, 'end': 116, 'mention': 'Li, Chang & Han, 2017', 'ref_id': 'BIBREF14'}, {'start': 572, 'end': 601, 'mention': 'Chen, Lin Coney & Huang, 2005', 'ref_id': 'BIBREF2'}, {'start': 730, 'end': 748, 'mention': 'The observer, 2020', 'ref_id': 'BIBREF23'}], 'section': 'Basic research on compatibility of traditional Chinese medicine was expounded by using the synergetic effect of multi-targets ::: TCM advantages in treatment of SARS-CoV-2 infection', 'ref_spans': [{'start': 338, 'end': 344, 'mention': 'Fig. 4', 'ref_id': 'FIGREF3'}]}, {'text': 'To investigate the mechanism of treating COVID-19 with TCM and monomers based on ACE2 receptor. A study on TCM and monomers acting on angiotensin converting enzyme II (ACE2) receptor was retrieved by TCMSP database. UniProt, GeneCards and other databases were used in this work to query the gene names corresponding to the target of Chinese medicine monomer, and then Cytoscape 3.2.1 and used to construct the compound-target (gene) network. DAVID was used to carry out the gene ontology (GO) functional enrichment analysis and KEGG pathway enrichment analysis based on the Kyoto encyclopedia of genes and genomes to predict its mechanism of action. From the results, it showed that there were 54 targets in the network, including AKT1, VEGFA and TNF. GO function enrichment analysis revealed 554 GO items including 486 biological process (BP) items, 26 cell component (CC) items, and 42 molecular function (MF) items. There were 162 signaling pathways involved in small cell lung cancer, non-small cell lung cancer, renin-angiotensin system pathway. The results of molecular docking showed that the affinity of puerarin with ACE2 and SARS-CoV-2 was similar to the recommended drugs. The study suggested that puerarin may regulate multiple signaling pathways by binding ACE2 to AGTR1, NOS3, HIF1A and other targets, which may have therapeutic effects on COVID-19 (Zong, Dig, Ma & Ju, 2020).', 'cite_spans': [{'start': 1364, 'end': 1388, 'mention': 'Zong, Dig, Ma & Ju, 2020', 'ref_id': 'BIBREF28'}], 'section': 'Basic research on compatibility of traditional Chinese medicine was expounded by using the synergetic effect of multi-targets ::: TCM advantages in treatment of SARS-CoV-2 infection', 'ref_spans': []}, {'text': 'The compatibility theory of TCM is based on the analysis of etiology and pathogenesis of diseases in traditional Chinese medicine. According to the understanding of diseases in traditional Chinese medicine, the occurrence of diseases is a complex process of life movement. The external diseases of COVID-19 also had pathological damage and disease transmission, such as \"cytokine storm\", due to wind and temperature epidemic virus. The compatibility of traditional Chinese medicine can meet the treatment needs of multiple pathological mechanisms.', 'cite_spans': [], 'section': 'Basic research on compatibility of traditional Chinese medicine was expounded by using the synergetic effect of multi-targets ::: TCM advantages in treatment of SARS-CoV-2 infection', 'ref_spans': []}, {'text': 'Our project team studied the compatibility rules of Shufeng Jiedu Capsule (SFJDC), and studied the compatibility rules of SFJDC with heat-clearing and detoxification effects in the acute pneumonia model of rats. SFJDC components can significantly reduce the IL-1α, IL-1β, IL-2, IL-4, and I-10 levels. SFJDC group, nourish group and licorice group have common set of targets and pathways, and each have lay particular stress on, targets related to inflammation, immune response, bacterial endotoxin reaction, defense, sweating, antipyretic, glucocorticoid reaction and so on. Each link, each path through mutual targets for connections between groups shows different ingredients target and pathway of synergy (Han, Zhu & Dong, 2019).', 'cite_spans': [{'start': 709, 'end': 730, 'mention': 'Han, Zhu & Dong, 2019', 'ref_id': 'BIBREF11'}], 'section': 'Basic research on compatibility of traditional Chinese medicine was expounded by using the synergetic effect of multi-targets ::: TCM advantages in treatment of SARS-CoV-2 infection', 'ref_spans': []}, {'text': 'Objectives of this review are to gather and highlight the information on different TCM products and plant formulations with antiviral properties in the backdrop of prior art patents, patent applications, non-patent art and commonly available traditional knowledge. In order to prevent and control the epidemic situation, different teams and researchers in our country have also published some valuable discussions on drug innovation and research, application of traditional drug research and evaluation research (Liu, 2020; Liu, Wang, Yan 2020a; Liu, Wang, Yi et al. 2020b; Liu, Wang, Zhang 2020c).', 'cite_spans': [{'start': 513, 'end': 522, 'mention': 'Liu, 2020', 'ref_id': 'BIBREF15'}, {'start': 524, 'end': 544, 'mention': 'Liu, Wang, Yan 2020a', 'ref_id': 'BIBREF16'}, {'start': 546, 'end': 572, 'mention': 'Liu, Wang, Yi et al. 2020b', 'ref_id': 'BIBREF17'}, {'start': 574, 'end': 596, 'mention': 'Liu, Wang, Zhang 2020c', 'ref_id': 'BIBREF18'}], 'section': 'Summary', 'ref_spans': []}, {'text': 'In summary, since the outbreak of COVID-19 pneumonia, Chinese medicine diagnosis and treatment methods and a variety of proprietary Chinese medicines and prescriptions have been listed in various versions of the national version of the diagnosis and treatment plan and recommended for many times. A great deal of research and application has been done to bring the unique effect of TCM on COVID-19 pneumonia.', 'cite_spans': [], 'section': 'Summary', 'ref_spans': []}, {'text': 'In therapy of COVID-19, early use of TCM for light and ordinary patients, can rapidly improve symptoms, shorten the hospitalization days and reduce light, normal to the heavy development, for heavy duty, and critically ill patients with a combination of Chinese and western medicine, to improve the symptoms of fever, cough, shortness of breath aspect, and has significant advantages to promote the heavy type, critical to light, normal.', 'cite_spans': [], 'section': 'Summary', 'ref_spans': []}, {'text': 'It is found that there are three theoretical characteristics worthy of recognition in the use of medicine. First, different from the unified diagnosis and treatment standards of western medicine, the treatment of TCM must be added and subtracted according to the actual situation of patients, which can be described as \"one person, one formula\". Second, TCM has been fully involved in the prevention and treatment of the COVID-19 pneumonia epidemic, effectively reducing the incidence of mild cases becoming severe ones and severe cases becoming critical ones, and improving the cure rate. \"In the frontal battle against the epidemic, TCM should not be absent and has never been absent. Third, the overall regulation of TCM, treatment based on syndrome differentiation, to improve the ability to resist disease, in the regulation of \"vital, blood, essence, fluid, viscera\" function is the basis of TCM clinical therapy.', 'cite_spans': [], 'section': 'Summary', 'ref_spans': []}], 'paragraphs': [{'text': 'According to the above view point, how to prevent novel coronavirus? At present, one of the characteristics of COVID-19 virus is that the incubation period is very long, and there are many asymptomatic infected persons who do not suffer from the disease for a long time. Some discharged patients may have positive nucleic acid tests and still carry the virus. If so, do you think it is difficult to eradicate the virus?', 'id': '00024'}], 'title': 'Pay attention to situation of SARS-CoV-2 and TCM advantages in treatment of novel coronavirus infection', 'sha': 'ed99ea6c7af5c77db2bf336d263db55646686098; ee18fa6db68e8a0521d7a3812bba3a6f94bd6709', 'doi': '10.1016/j.chmed.2020.03.004', 'score': 8.734600067138672}, {'rank': 283, 'paragraphs': [{'text': 'With mean infectious period (from compartments → ) set as 3.9 days and mean incubation period (from compartments → ) set as 5 days, we first use ordinary differential equations based on Markov processes to estimate the infection rate from compartments → . Assuming that the epidemic starts from December 1 st , is equal to 0.68 to match the number of cumulative confirmed cases in Wuhan city.', 'id': '00051'}], 'title': 'Short-term forecasts and long-term mitigation evaluations for the COVID-19 epidemic in Hubei Province, China', 'sha': '26a2d4b805cdb48bccb4d640da82fa4c4d15c7c4', 'doi': '10.1101/2020.03.27.20045625', 'score': 8.731100082397461}, {'rank': 284, 'paragraphs': [{'text': 'Among the seven known human coronavirus, four of them are common pathogens of human influenza. SARS-CoV, MERS-CoV and 2019-nCoV will cause fatal respiratory diseases [1] . Although people have identified and analyzed the coronavirus for a long time, knowledge of the coronavirus is quite limited and there are no vaccines or antiviral drugs to prevent or treat human coronavirus infections. Last major outbreak in China was SARS which is an acute respiratory infectious disease with high fatality rate caused by SARS-CoV in 2003. Chinese people managed the outbreak of SARS through multiple control and prevention measures effectively. Compared with SARS, the incubation period of COVID-19 is significant and rather long. Different mean incubation period of COVID-19 is reported, 5.2 days [9] in the early stage, 3.0 days [10] and 4.75 days [11] in recent research. The patient with up to 24 days of incubation period is reported in [10] , even 38 days is also reported in Enshi Tujia and Miao Autonomous Prefecture in Hubei Province. Notice that there is quite a lot of people infected with asymptomatic [6] and the fatality is much lower than SARS-CoV and MERS-CoV [10] . Genetic studies of viruses show that the homology of SARS-CoV and 2019-nCoV is 85% [12] . But 2019-nCoV binds ACE2 with higher affinity than SARS-CoV S [13] . By the end of January 29, 2020, the confirmed cases caused by COVID-19 are surpassed SARS. The uncertain of incubation period, asymptomatic cases and super transmissibility of the virus bring great difficulties in epidemic control.', 'id': '00002'}], 'title': 'Modeling the Control of COVID-19: Impact of Policy Interventions and Meteorological Factors', 'sha': '35180f822b0debe3a1bb120a1ab64c46b11f02c7', 'doi': '', 'score': 8.72719955444336}, {'rank': 285, 'abstract': 'Background: Coronavirus disease 2019 (COVID-19) was first identified in Wuhan, China, in December 2019 and quickly spread throughout China and the rest of the world. Many mathematical models have been developed to understand and predict the infectiousness of COVID-19. We aim to summarize these models to inform efforts to manage the current outbreak. Methods: We searched PubMed, Web of science, EMBASE, bioRxiv, medRxiv, arXiv, Preprints, and National Knowledge Infrastructure (Chinese database) for relevant studies published between 1 December 2019 and 21 February 2020. References were screened for additional publications. Crucial indicators were extracted and analysed. We also built a mathematical model for the evolution of the epidemic in Wuhan that synthesised extracted indicators. Results: Fifty-two articles involving 75 mathematical or statistical models were included in our systematic review. The overall median basic reproduction number (R(0)) was 3.77 [interquartile range (IQR) 2.78–5.13], which dropped to a controlled reproduction number (R(c)) of 1.88 (IQR 1.41–2.24) after city lockdown. The median incubation and infectious periods were 5.90 (IQR 4.78–6.25) and 9.94 (IQR 3.93–13.50) days, respectively. The median case-fatality rate (CFR) was 2.9% (IQR 2.3–5.4%). Our mathematical model showed that, in Wuhan, the peak time of infection is likely to be March 2020 with a median size of 98,333 infected cases (range 55,225–188,284). The earliest elimination of ongoing transmission is likely to be achieved around 7 May 2020. Conclusions: Our analysis found a sustained R(c) and prolonged incubation/ infectious periods, suggesting COVID-19 is highly infectious. Although interventions in China have been effective in controlling secondary transmission, sustained global efforts are needed to contain an emerging pandemic. Alternative interventions can be explored using modelling studies to better inform policymaking as the outbreak continues.', 'body_text': [{'text': 'An outbreak of atypical pneumonia (Coronavirus Disease 2019, COVID-19) caused by the novel coronavirus (2019-nCoV) emerged in Wuhan, China, at the end of 2019. The virus rapidly spread across China and the rest of the world. As of 9 May, 83,976 confirmed infections and 4,639 deaths had been reported within China1,2. The majority of cases in China have been identified in Hubei Province, especially within Wuhan. The Wuhan Municipal Government began a citywide lockdown on 23 January 2020 to slow the spread of the disease, and other cities in Hubei Province soon followed suit3. The lockdown effectively curbed further exportation of the epidemic from Hubei to the other provinces of China (1–4). Within China, the outbreak has been effectively under control and the main effort was put in identifying the imported cases from overseas1. However, the WHO declared COVID-19 a global pandemic on 11 March due to its spread and severity worldwide, with 2,361,998 confirmed infections and 272,094 deaths outsides China as of 9 May.', 'cite_spans': [{'start': 693, 'end': 694, 'mention': '1', 'ref_id': 'BIBREF0'}, {'start': 695, 'end': 696, 'mention': '4', 'ref_id': 'BIBREF33'}], 'section': 'Introduction', 'ref_spans': []}, {'text': 'Mathematical modelling, including statistical modelling, is a useful tool to understand the dynamics of new diseases. Since COVID-19 was first identified, many mathematical models have been developed to simulate the transmission across populations and assess the potential impact of public health interventions. Crucial parameters of new diseases can be derived from models, including the basic reproduction number (R0), peak time, peak size, incubation period, infectious period, case-fatality rate (CFR), and elimination time. By definition, R0 measures the average number of secondary cases that are expected to be generated from a single case of a disease entering a completely susceptible population (5). R0 decreases if intervention measures are implemented or the susceptible population size decreases. The controlled reproduction number (Rc) denotes R0 after interventional measures are undertaken. If R0 < 1, then one infectious person will infect fewer than one person, and an epidemic will ultimately resolve (6). Thus, R0 is an important parameter to assess potential control strategies during an outbreak. Peak time refers to the time when a disease infects the largest number of people (peak size) and is an inflection point during an outbreak.', 'cite_spans': [{'start': 706, 'end': 707, 'mention': '5', 'ref_id': 'BIBREF44'}, {'start': 1021, 'end': 1022, 'mention': '6', 'ref_id': 'BIBREF54'}], 'section': 'Introduction', 'ref_spans': []}, {'text': 'Published models of COVID-19 have reported a huge range of estimated R0 and peak times. For example, Zhang et al. (7) estimated an R0 of 1.44 while Mizumoto et al. (8) reported an R0 of 7.05. To better inform efforts to control the current outbreak, we systematically reviewed existing mathematical and statistical models and built our own mathematical model to estimate the transmission capacity, epidemiological characteristics, potential peak time and size, and elimination time of COVID-19.', 'cite_spans': [{'start': 115, 'end': 116, 'mention': '7', 'ref_id': 'BIBREF55'}, {'start': 165, 'end': 166, 'mention': '8', 'ref_id': 'BIBREF56'}], 'section': 'Introduction', 'ref_spans': []}, {'text': 'Our systematic review was conducted according to PRISMA guidelines (9). We searched PubMed, Web of Science, EMBASE, bioRxiv, medRxiv, arXiv, Preprints, and National Knowledge Infrastructure (CNKI) for studies published between 1 December 2019 and 21 February 2020. We used the search terms “Coronavirus Disease 2019,” “COVID-19,” “SARS-CoV-2,” “2019-nCoV,” “coronavirus,” OR “pneumonia” AND “model,” “modelling,” “modeling,” “dynamic,” “estimation,” “prediction,” OR “transmission.” Search terms were translated into Chinese when searching Chinese database. The database search was supplemented by screening references of retrieved articles.', 'cite_spans': [{'start': 68, 'end': 69, 'mention': '9', 'ref_id': 'BIBREF57'}], 'section': 'Search Strategy and Selection Criteria ::: Methods', 'ref_spans': []}, {'text': 'Studies were included if they presented a mathematical/statistical model of COVID-19 and reported any of the following—R0, incubation period, infectious period, fatality, peak time, peak size, total infection number, or elimination time. Studies were excluded if they were purely methodological and did not report the aforementioned parameters. If one study was concurrently published in a journal and preprint website, only the journal version was included. Two reviewers (YL and YZ) independently performed the literature search and screened titles and abstracts. Disagreements were resolved by discussion among all authors.', 'cite_spans': [], 'section': 'Search Strategy and Selection Criteria ::: Methods', 'ref_spans': []}, {'text': 'Data extraction was performed by QD, Y-FL, and YZho independently, and results were summarised by Y-FL. Abstracted variables included the first author, model type, type and period of data used for model fitting, setting, region of interest, estimated R0, estimated incubation period, estimated CFR, estimated peak time and peak size, and impact of outbreak response if available. Assumed values of the incubation period based on other studies were excluded. Quality of mathematical models was assessed according to a quality-appraisal tool developed upon the recommendations by the International Society for Pharmacoeconomics and Outcomes Research and Society for Medical Decision Making (ISPOR-SMDM) Modelling Good Research Practices Task Force (10, 11). Such a tool brings up questions regarding 14 criteria, e.g., model setting and population, modelling methodology and structure, and fitting methodology (see Appendix in Supplementary Material). Each criterion of a paper was scored zero, one, or two. If a criterion was not relevant for a paper, then a score of one was assigned. QD, Y-FL, and YZho assessed the quality of mathematical models, and discrepancies were resolved through discussion with a senior investigator (HZo).', 'cite_spans': [{'start': 747, 'end': 749, 'mention': '10', 'ref_id': 'BIBREF1'}, {'start': 751, 'end': 753, 'mention': '11', 'ref_id': 'BIBREF2'}], 'section': 'Data Analysis ::: Methods', 'ref_spans': []}, {'text': 'A comprehensive meta-analysis of extracted data was not performed due to the high level of heterogeneity between the studies in terms of model type, model setting, type and period of data used for model calibration, and region of interest. However, some key parameters were analysed, including estimated parameters (R0/Rc, incubation period, infectious period, and CFR) and model predictions (peak size and peak time, total infections, and elimination time). R0/Rc was analysed by stratifying regions, namely “Wuhan,” “Hubei (including Wuhan),” “mainland China (excluding Taiwan, Hong Kong and Macau),” and “regions other than Hubei in mainland China.”', 'cite_spans': [], 'section': 'Data Analysis ::: Methods', 'ref_spans': []}, {'text': 'We reported distributions of point estimates and reported medians and interquartile ranges (IQR). We did not pool point estimates from various mathematical models. Categorical variables were presented as frequencies or proportions. Data were combined by interval segments and/or grouped by the same characteristics according to the sparsity of data. Some statistical methods, mainly non-parametric methods, including the Wilcoxon rank sum test (R0/Rc), Kruskal-Wallis H test (regions), and Quade test (R0/Rc and regions), were used to analyze differences between segments/groups.', 'cite_spans': [], 'section': 'Data Analysis ::: Methods', 'ref_spans': []}, {'text': 'We modelled COVID-19 transmission using a classic susceptible (S)-exposed (E)-infectious (I)-recovery (R) (SEIR) structure model (see Appendix in Supplementary Material) to predict future trends and expected peak time in Wuhan. Two assumptions were separately considered in this SEIR model: (1) individuals in the incubation period are infectious and (2) individuals in the incubation period are not infectious. We used the parameter values that were obtained from our review of previous models for our model simulations. When calibrating the model, the top 20 of 256 best-fit simulations, selected by least square error, were used to obtain estimates of epidemic trends. We calculated peak times and eliminations (total infections <100) based on normal (median), optimistic, and pessimistic scenarios. All analyses were conducted with R software version 3.6.2 (R Foundation for Statistical Computing). Confirmed reported cases between 12 February 2020 and 21 February 2020 were downloaded from the China National Health Commission website 1.', 'cite_spans': [], 'section': 'Data Analysis ::: Methods', 'ref_spans': []}, {'text': 'We identified 1,451 studies; 269 were duplicates, which left us with 1,182 unique studies (Figure 1). After screening titles and abstracts, 90 studies underwent a full-text review. Of these, 38 were excluded because they did not report necessary parameters or were not models specifically targeting COVID-19. Fifty-two publications were eligible for inclusion. Details of each included study is summarised in the Appendix (Supplementary Material) (4, 7, 8, 12–36).', 'cite_spans': [{'start': 448, 'end': 449, 'mention': '4', 'ref_id': 'BIBREF33'}, {'start': 451, 'end': 452, 'mention': '7', 'ref_id': 'BIBREF55'}, {'start': 454, 'end': 455, 'mention': '8', 'ref_id': 'BIBREF56'}, {'start': 457, 'end': 459, 'mention': '12', 'ref_id': 'BIBREF3'}, {'start': 460, 'end': 462, 'mention': '36', 'ref_id': 'BIBREF29'}], 'section': 'Results', 'ref_spans': [{'start': 91, 'end': 99, 'mention': 'Figure 1', 'ref_id': 'FIGREF0'}]}, {'text': 'The 52 included studies reported a total of 75 unique models, including 88% (66/75) of articles calibrated models using original data (i.e., reported cases), 7% (5/75) used adjusted data modified by reported rates prior to model fitting, and 5% (4/75) articles used simulations. 35% (26/75), 16% (12/75), 37% (28/75), and 12% (9/75) of models refer to Wuhan, Hubei, mainland China, and regions other than Hubei in mainland China. The other regions mentioned in these nine articles were too wide, such as Guangdong Province, Beijing, and Chongqing, so we decided to focus on the first three regions only. The usage frequency of different models is summarised in the Appendix (Supplementary Material).', 'cite_spans': [], 'section': 'Results', 'ref_spans': []}, {'text': 'Thirty studies reported estimated R0 and/or Rc. Models used to generate R0/Rc varied in terms of model type, model structure, model setting, and data used for model fitting (see Appendix in Supplementary Material). R0 differed significantly before and after the citywide lockdown in Wuhan (Figures 2, 3, p < 0.001). The median R0 was 3.77 (IQR, 2.78–5.13), and median Rc was 1.88 (IQR, 1.41–2.24). After aggregating data by regions, median R0 for Wuhan, Hubei, and mainland China over the whole outbreak period were 3.16 (IQR, 2.36–4.40), 4.39 (IQR, 3.18–5.15), and 3.03 (IQR, 2.30–4.19), respectively. Differences between these estimated R0 were not statistically significant (p = 0.180).', 'cite_spans': [], 'section': 'Results', 'ref_spans': [{'start': 290, 'end': 299, 'mention': 'Figures 2', 'ref_id': 'FIGREF1'}, {'start': 301, 'end': 302, 'mention': '3', 'ref_id': 'FIGREF2'}]}, {'text': 'When R0 and Rc were stratified by region, differences in R0 and Rc remained statistically significant across all four regions. In Wuhan, the median of R0 was 3.22 (IQR, 2.50–5.03) and Rc was 2.09 (IQR, 1.95–2.96). In Hubei, the median R0 and Rc were 4.80 (IQR, 4.07–5.58) and 1.48 (IQR, 1.01–2.08), respectively. Across mainland China, R0 was 3.55 (IQR, 3.03–4.66), and the median Rc was 2.19 (IQR, 1.76–2.48).', 'cite_spans': [], 'section': 'Results', 'ref_spans': []}, {'text': 'Nineteen studies predicted peak time and peak size of infections. Estimates varied from late January to late March with peak size ranging from 7,000 to 90,000 (Table 1). Elimination time was predicted in five studies, with estimates ranging from March to August 2020 (7, 37–40).', 'cite_spans': [{'start': 268, 'end': 269, 'mention': '7', 'ref_id': 'BIBREF55'}, {'start': 271, 'end': 273, 'mention': '37', 'ref_id': 'BIBREF30'}, {'start': 274, 'end': 276, 'mention': '40', 'ref_id': 'BIBREF34'}], 'section': 'Results', 'ref_spans': [{'start': 160, 'end': 167, 'mention': 'Table 1', 'ref_id': 'TABREF0'}]}, {'text': 'Incubation period was estimated in 9 studies, with the median estimate being 5.90 days (IQR 4.78–6.25) (3, 7, 20, 22, 28, 41–44). Among the six studies reporting infection period, the median estimate was 9.94 days (IQR, 3.93–13.50) [Figure 4; (7, 14, 15, 17, 25)]. Six studies reported CFR, and median estimated CFR was 2.94% (IQR, 2.25%−5.40%) (3, 8, 15, 20, 22, 31).', 'cite_spans': [{'start': 104, 'end': 105, 'mention': '3', 'ref_id': 'BIBREF22'}, {'start': 107, 'end': 108, 'mention': '7', 'ref_id': 'BIBREF55'}, {'start': 110, 'end': 112, 'mention': '20', 'ref_id': 'BIBREF12'}, {'start': 114, 'end': 116, 'mention': '22', 'ref_id': 'BIBREF14'}, {'start': 118, 'end': 120, 'mention': '28', 'ref_id': 'BIBREF20'}, {'start': 122, 'end': 124, 'mention': '41', 'ref_id': 'BIBREF35'}, {'start': 125, 'end': 127, 'mention': '44', 'ref_id': 'BIBREF38'}, {'start': 244, 'end': 245, 'mention': '7', 'ref_id': 'BIBREF55'}, {'start': 247, 'end': 249, 'mention': '14', 'ref_id': 'BIBREF5'}, {'start': 251, 'end': 253, 'mention': '15', 'ref_id': 'BIBREF6'}, {'start': 255, 'end': 257, 'mention': '17', 'ref_id': 'BIBREF8'}, {'start': 259, 'end': 261, 'mention': '25', 'ref_id': 'BIBREF17'}, {'start': 346, 'end': 347, 'mention': '3', 'ref_id': 'BIBREF22'}, {'start': 349, 'end': 350, 'mention': '8', 'ref_id': 'BIBREF56'}, {'start': 352, 'end': 354, 'mention': '15', 'ref_id': 'BIBREF6'}, {'start': 356, 'end': 358, 'mention': '20', 'ref_id': 'BIBREF12'}, {'start': 360, 'end': 362, 'mention': '22', 'ref_id': 'BIBREF14'}, {'start': 364, 'end': 366, 'mention': '31', 'ref_id': 'BIBREF24'}], 'section': 'Results', 'ref_spans': [{'start': 233, 'end': 241, 'mention': 'Figure 4', 'ref_id': 'FIGREF3'}]}, {'text': 'The estimated number of total infections varied by region (see Appendix in Supplementary Material). Median estimated number of total infections in Wuhan, Hubei, and mainland China were 56,565 (IQR, 49,795–280,255), 61,028 (IQR, 43,750–111,682), and 87,525 (IQR, 59,784–461,652), respectively (3, 8, 15, 17, 20, 38, 39, 45–48). Most studies used data from 23 January 2020 or earlier when building models. Two thirds of studies that predicted abnormally high totals came from Probabilistic/likelihood-based models that used data from after 23 January 2020 (15, 46, 48).', 'cite_spans': [{'start': 293, 'end': 294, 'mention': '3', 'ref_id': 'BIBREF22'}, {'start': 296, 'end': 297, 'mention': '8', 'ref_id': 'BIBREF56'}, {'start': 299, 'end': 301, 'mention': '15', 'ref_id': 'BIBREF6'}, {'start': 303, 'end': 305, 'mention': '17', 'ref_id': 'BIBREF8'}, {'start': 307, 'end': 309, 'mention': '20', 'ref_id': 'BIBREF12'}, {'start': 311, 'end': 313, 'mention': '38', 'ref_id': 'BIBREF31'}, {'start': 315, 'end': 317, 'mention': '39', 'ref_id': 'BIBREF32'}, {'start': 319, 'end': 321, 'mention': '45', 'ref_id': 'BIBREF39'}, {'start': 322, 'end': 324, 'mention': '48', 'ref_id': 'BIBREF42'}, {'start': 555, 'end': 557, 'mention': '15', 'ref_id': 'BIBREF6'}, {'start': 559, 'end': 561, 'mention': '46', 'ref_id': 'BIBREF40'}, {'start': 563, 'end': 565, 'mention': '48', 'ref_id': 'BIBREF42'}], 'section': 'Results', 'ref_spans': []}, {'text': 'The estimated impact of interventions is summarised in the Appendix (Supplementary Material). Four of 10 studies found that after implementing citywide lockdown in Wuhan, R0 would be reduced by 87–95%, peak size would be reduced by 21.06–22.38%, and deaths would be reduced by 56.87–62.95% (1–4). Three studies predicted delay of lockdown measures by 1 or 7 days would increase the number of infections at peak size by 722–6,351 and 8,618–28,274, respectively (3, 16, 49). Increasing diagnosis efficacy to 70% was predicted to reduce infections by 90% as of 10 February 2020 (50).', 'cite_spans': [{'start': 291, 'end': 292, 'mention': '1', 'ref_id': 'BIBREF0'}, {'start': 293, 'end': 294, 'mention': '4', 'ref_id': 'BIBREF33'}, {'start': 461, 'end': 462, 'mention': '3', 'ref_id': 'BIBREF22'}, {'start': 464, 'end': 466, 'mention': '16', 'ref_id': 'BIBREF7'}, {'start': 468, 'end': 470, 'mention': '49', 'ref_id': 'BIBREF43'}, {'start': 576, 'end': 578, 'mention': '50', 'ref_id': 'BIBREF45'}], 'section': 'Results', 'ref_spans': []}, {'text': 'We generated models under two separate assumptions: (A1) individuals are infectious during the incubation period and (A2) individuals are not infectious during the incubation period. In the A1 model, mean Rc was 2.15 (SD, 0.15); mean incubation period was 5.19 days (SD, 0.53); mean infectious period was 11.87 days (SD, 1.35); and CFR was 2.68% (SD, 0.67%). In the A2 model, mean Rc was 2.14 (SD, 0.16); mean incubation period was 5.17 days (SD, 0.50); mean infectious period was 12.00 days (SD, 1.51); and CFR was 2.43% (SD, 0.39%). Mean of least square error in the top 20 best-fit simulations was 17,944 (SD, 1,140) and 27,750 (SD, 1,754) in the A1 and A2 models, respectively (see Appendix in Supplementary Material).', 'cite_spans': [], 'section': 'Results', 'ref_spans': []}, {'text': 'The top 20 best-fit simulations with best- and worst-case scenarios are presented in Figure 5. In the A1 model, Rc, peak size was 55,225 and 188,284 in the best- and worst-case scenarios, respectively. In the A2 model, peak size was 28,237 and 36,248 in the best- and worst-case scenarios, respectively. In Wuhan, the A1 model predicted peak time would be 17 March (Range, 12–22 March 2020), and elimination time would be 7 May (25 April−21 May 2020). In the A2 model, peak time was estimated to be 2 March (Range, 13 February−5 March 2020) and elimination time to be 17 May (8–27 May 2020).', 'cite_spans': [], 'section': 'Results', 'ref_spans': [{'start': 85, 'end': 93, 'mention': 'Figure 5', 'ref_id': 'FIGREF4'}]}, {'text': 'Our systematic review and data synthesis is the first study to synthesize mathematical models on the transmission of COVID-19. The estimated values of R0, incubation period, infectious period, peak time, and peak size for COVID-19 were consistently higher than that of SARS or MERS, suggesting this novel coronavirus is highly infectious. We also found that the citywide lockdown of Wuhan resulted in significantly reduced R0, with the earliest elimination time in China now estimated to be late April, though the complex dynamics of an evolving global pandemic were not incorporated into included models.', 'cite_spans': [], 'section': 'Discussion', 'ref_spans': []}, {'text': 'In this systematic review of transmission-dynamic models predicting the spread of COVID-19, we found the median estimated R0 to be 3.77, suggesting this novel virus is highly infectious. Estimated R0 of COVID-19 was higher than that of middle east respiratory syndrome coronavirus (MERS-CoV, <1) and severe acute respiratory syndrome (SARS, 2–4) (51, 52). This corresponds to the difference between the total number of infections seen in the current COVID-19 and 2003–2004 SARS outbreaks in China (>80,000 vs. ~5,327)1.', 'cite_spans': [{'start': 347, 'end': 349, 'mention': '51', 'ref_id': 'BIBREF46'}, {'start': 351, 'end': 353, 'mention': '52', 'ref_id': 'BIBREF47'}], 'section': 'Discussion', 'ref_spans': []}, {'text': 'After a citywide lockdown began in Wuhan, the median estimated Rc dropped to 1.88, suggesting a large drop in infections coincided with the implementation of this outbreak control intervention. By limiting interactions and preventing travel, the lockdown effort has dramatically reduced contact rates between infected and non-infected persons. However, an Rc>1 suggests that COVID-19 would continue to spread, which is at odds with recently published epidemiological reports, suggesting the outbreak in China is slowing down1. This discrepancy may suggest that other recently implemented public health measures beyond citywide lockdowns, including contact tracing, intensification of screening, quarantine of infected individuals, and mask utilisation, may also be contributing to the containment of COVID-19. Future models should attempt to capture the impact of these additional interventions on COVID-19 transmission.', 'cite_spans': [], 'section': 'Discussion', 'ref_spans': []}, {'text': 'We found median estimated incubation period, infectious period, and fatality were 5.90 days, 9.94 days, and 2.94%, respectively. If these estimations are accurate, a 14-day quarantine period would be long enough to assess for infection in an asymptomatic person exposed to COVID-19. It should be noted that the maximum incubation period reported for COVID-19 was 24 days, and additional research is needed to confirm these estimations. However, this estimated incubation period is similar to that of SARS and MERS (51, 52). Estimated CFR of COVID-19 (2.94%) was substantially lower than that of SARS (14–15%) and MERS (34.4%), suggesting COVID-19 may be a less virulent strain in the coronavirus family (20, 22, 51, 53). As of 9 May, the epidemic in China has almost come to end and local asymptomatic infected cases have been captured by the surveillance system, with the death toll come to 4,643 and fatality rate about 5.5%.', 'cite_spans': [{'start': 515, 'end': 517, 'mention': '51', 'ref_id': 'BIBREF46'}, {'start': 519, 'end': 521, 'mention': '52', 'ref_id': 'BIBREF47'}, {'start': 704, 'end': 706, 'mention': '20', 'ref_id': 'BIBREF12'}, {'start': 708, 'end': 710, 'mention': '22', 'ref_id': 'BIBREF14'}, {'start': 712, 'end': 714, 'mention': '51', 'ref_id': 'BIBREF46'}, {'start': 716, 'end': 718, 'mention': '53', 'ref_id': 'BIBREF48'}], 'section': 'Discussion', 'ref_spans': []}, {'text': 'We found significant variation in estimated R0 and Rc in our review of the published literature. These variations may be due to the wide variety of modelling methods used and different assumptions used to build each model. Additionally, limited healthcare resources and immature diagnostic algorithms resulted in under-diagnosis and delayed treatment at the beginning of the outbreak in Wuhan. Consequently, models calibrated using only the official number of confirmed infections may be impacted by a systematic underreporting of infections, leading to a higher estimated R0 compared to models that adjusted for potential underreporting. Several studies included in this review concluded that underreported infections may have had an significant impact on estimated R0 (15, 21, 54), and five studies attempted to approximate the number of unreported cases (13, 18, 21, 22, 24).', 'cite_spans': [{'start': 771, 'end': 773, 'mention': '15', 'ref_id': 'BIBREF6'}, {'start': 775, 'end': 777, 'mention': '21', 'ref_id': 'BIBREF13'}, {'start': 779, 'end': 781, 'mention': '54', 'ref_id': 'BIBREF49'}, {'start': 858, 'end': 860, 'mention': '13', 'ref_id': 'BIBREF4'}, {'start': 862, 'end': 864, 'mention': '18', 'ref_id': 'BIBREF9'}, {'start': 866, 'end': 868, 'mention': '21', 'ref_id': 'BIBREF13'}, {'start': 870, 'end': 872, 'mention': '22', 'ref_id': 'BIBREF14'}, {'start': 874, 'end': 876, 'mention': '24', 'ref_id': 'BIBREF16'}], 'section': 'Discussion', 'ref_spans': []}, {'text': 'Our mathematical model predicts peak time for COVID-19 will be in March 2020 and elimination is likely to be achieved by late April 2020 at the earliest, assuming the current intervention level is maintained. This estimate of peak time is close to the reality that there are few locally infected cases after 31 March, which indicates that the peak size of local transmission has been reached in March. Elimination of local transmission has been achieved in April as most of newly infected cases are imported from overseas. COVID-19 continues to spread worldwide2 (55), and the influx of overseas cases may introduce new transmission dynamics that are not possible to predict using current models. Studies have reported that the epidemic of some viruses (e.g., SARS-CoV) or bacteria (e.g., Clostridioides difficile) can be affected by geographical climatic factors such as temperature, humidity, and latitude (56–58), and COVID-19 infections may consequently be impacted by seasonality and latitude in unpredictable ways. However, we were not able to add additional results and analysis of temperature for COVID-19 due to lack of available data. In addition, the difference in social mixing pattern between rural and urban areas may lead to different transmission models. These data are essential to a thorough understanding of the spread of SARS-CoV-2 and formulating appropriate intervention strategies. Through comparing the two scenarios of our mathematical model, scenarios with an infectious incubation period resulted in much better goodness-of-fit. This to some extent support that incubation period is infectious. Finally, the potential impact of new treatment or vaccines for COVID-19 are not represented in the predictions of our models.', 'cite_spans': [{'start': 564, 'end': 566, 'mention': '55', 'ref_id': 'BIBREF50'}, {'start': 909, 'end': 911, 'mention': '56', 'ref_id': 'BIBREF51'}, {'start': 912, 'end': 914, 'mention': '58', 'ref_id': 'BIBREF53'}], 'section': 'Discussion', 'ref_spans': []}, {'text': 'Our study has limitations. First, some studies included multiple models, and, as a result, models developed with certain underlying assumptions and validation methods were overrepresented in our results. Second, none of the included models considered age-related contact rates. Immunity to and fatality from COVID-19 likely differ across age cohorts. Without accounting for this key difference, results of all included models should be interpreted with caution. Third, our synthesis model did not take into account rates of underreported infections, additional quarantine efforts, mask usage, or changes in mass transportation, which may change our predictions. Without readily available data effects of these factors are hard to account for.', 'cite_spans': [], 'section': 'Discussion', 'ref_spans': []}, {'text': 'Findings from our systematic review and mathematical model suggest high infectiousness of COVID-19, and the lockdown of Wuhan significantly reduced R0. If current modelling is accurate, a 14-day quarantine is sufficient for asymptomatic persons exposed to the virus. The effect of age on infection and fatality should be incorporated into future models to more accurately predict transmission dynamics. Interventions besides citywide lockdowns, including mask utilisation and travel restrictions, should be further evaluated through modelling in order to better inform ongoing efforts to contact outbreaks inside and outside China.', 'cite_spans': [], 'section': 'Discussion', 'ref_spans': []}, {'text': 'All datasets generated for this study are included in the article/Supplementary Material.', 'cite_spans': [], 'section': 'Data Availability Statement', 'ref_spans': []}, {'text': 'HZo and YS conceived the study. Y-FL, QD, and YZho designed the protocol and conducted study selection and data extraction. Y-FL, QD, and YZho contributed to statistical analysis and interpretation of data. Y-FL, QD, YZho, TY, and PL drafted the manuscript. All authors critically revising the manuscript.', 'cite_spans': [], 'section': 'Author Contributions', 'ref_spans': []}, {'text': 'The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.', 'cite_spans': [], 'section': 'Conflict of Interest', 'ref_spans': []}], 'paragraphs': [{'text': 'We found median estimated incubation period, infectious period, and fatality were 5.90 days, 9.94 days, and 2.94%, respectively. If these estimations are accurate, a 14-day quarantine period would be long enough to assess for infection in an asymptomatic person exposed to COVID-19. It should be noted that the maximum incubation period reported for COVID-19 was 24 days, and additional research is needed to confirm these estimations. However, this estimated incubation period is similar to that of SARS and MERS (51, 52). Estimated CFR of COVID-19 (2.94%) was substantially lower than that of SARS (14–15%) and MERS (34.4%), suggesting COVID-19 may be a less virulent strain in the coronavirus family (20, 22, 51, 53). As of 9 May, the epidemic in China has almost come to end and local asymptomatic infected cases have been captured by the surveillance system, with the death toll come to 4,643 and fatality rate about 5.5%.', 'id': '00023'}], 'title': 'Spread and Impact of COVID-19 in China: A Systematic Review and Synthesis of Predictions From Transmission-Dynamic Models', 'sha': '1fb1d2d4637ef504b4a2805fa23fc288c7317f9d', 'doi': '10.3389/fmed.2020.00321', 'score': 8.724200248718262}, {'rank': 286, 'abstract': 'Since the COVID-19 outbreak in Wuhan City in December of 2019, numerous model predictions on the COVID-19 epidemics in Wuhan and other parts of China have been reported. These model predictions have shown a wide range of variations. In our study, we demonstrate that nonidentifiability in model calibrations using the confirmed-case data is the main reason for such wide variations. Using the Akaike Information Criterion (AIC) for model selection, we show that an SIR model performs much better than an SEIR model in representing the information contained in the confirmed-case data. This indicates that predictions using more complex models may not be more reliable compared to using a simpler model. We present our model predictions for the COVID-19 epidemic in Wuhan after the lockdown and quarantine of the city on January 23, 2020. We also report our results of modeling the impacts of the strict quarantine measures undertaken in the city after February 7 on the time course of the epidemic, and modeling the potential of a second outbreak after the return-to-work in the city.', 'body_text': [{'text': 'In early December 2019, a novel coronavirus, later labelled as COVID-19, caused an outbreak in the city of Wuhan, Hubei Province, China, and it has further spread to other parts of China and many other countries in the world. By January 31, the global confirmed cases have reached 9,776 with a death toll of 213, and the WHO declared the outbreak as a public health emergency of international concern (WHO, 2020). By February 9, the global death toll has climbed to 811, surpassing the total death toll of the 2003 SARS epidemic, and the confirmed cases continued to climb globally. As governments and public agencies in China and other impacted countries respond to the outbreaks, it is crucial for modelers to estimate the severity of the epidemic in terms of the total number of infected, total number of confirmed cases, total deaths, and the basic reproduction number, and to predict the time course of the epidemic, the arrival of its peak time, and total duration. Such information can help the public health agencies make informed decisions.', 'cite_spans': [{'start': 402, 'end': 411, 'mention': 'WHO, 2020', 'ref_id': 'BIBREF21'}], 'section': 'Introduction', 'ref_spans': []}, {'text': 'Since the start of the outbreak in Wuhan, several modeling groups around the world have reported estimations and predictions for the COVID-19 (formerly called 2019-nCov) epidemic in journal publications or on websites, for an incomplete list see (Bai et al., 2020; Imai, Dorigatti, Cori, Riley, & Ferguson, 2020; Read, Bridgen, Cummings, Ho, & Jewell, 2020; Shen, Peng, Xiao, & Zhang, 2020; Tang et al., 2020b, a; Wu, Leung, & Leung, 2020; You et al., 2020; Yu, 2020; Zhao et al., 2020). The modeling results have shown a wide range of variations (Cyranoski, 2020): estimated basic reproduction number varies from 2 to 6, peak time estimated from mid-February to late March, and the total number of infected people ranges from 50,000 to millions. Why is there such a wide variation in model predictions, even among predictions made using transmission models based on either the SIR or SEIR framework? We attempt to address this variability issue in our study.', 'cite_spans': [{'start': 247, 'end': 263, 'mention': 'Bai et al., 2020', 'ref_id': 'BIBREF11'}, {'start': 265, 'end': 311, 'mention': 'Imai, Dorigatti, Cori, Riley, & Ferguson, 2020', 'ref_id': 'BIBREF33'}, {'start': 313, 'end': 356, 'mention': 'Read, Bridgen, Cummings, Ho, & Jewell, 2020', 'ref_id': 'BIBREF10'}, {'start': 358, 'end': 389, 'mention': 'Shen, Peng, Xiao, & Zhang, 2020', 'ref_id': 'BIBREF14'}, {'start': 391, 'end': 409, 'mention': 'Tang et al., 2020b', 'ref_id': 'BIBREF17'}, {'start': 411, 'end': 412, 'mention': 'a', 'ref_id': 'BIBREF16'}, {'start': 414, 'end': 438, 'mention': 'Wu, Leung, & Leung, 2020', 'ref_id': 'BIBREF23'}, {'start': 440, 'end': 456, 'mention': 'You et al., 2020', 'ref_id': 'BIBREF24'}, {'start': 458, 'end': 466, 'mention': 'Yu, 2020', 'ref_id': 'BIBREF25'}, {'start': 468, 'end': 485, 'mention': 'Zhao et al., 2020', 'ref_id': 'BIBREF27'}, {'start': 548, 'end': 563, 'mention': 'Cyranoski, 2020', 'ref_id': 'BIBREF30'}], 'section': 'Introduction', 'ref_spans': []}, {'text': 'A simple answer for the wide range of model predictions might be that there was too little information at the beginning of the outbreak, especially before January 23 when Wuhan was quarantined and locked down, and that there was a lack of reliable data, except for the confirmed case data that could be used for model calibration. Rigorous model calibration methods, including maximum likelihood methods and the Bayesian inference based MCMC methods, already take into consideration uncertainties in data by allowing the data at each time point to follow a probability distribution with the mean given by the assumed model and the variance τ given by the assumed probability distribution, where the variance may depend on the mean. The lack of data, as we will demonstrate, is a more serious concern for modellers. A key issue that can explain the variability in model predictions is understanding how the available data (confirmed cases) compares with model predictions. Confirmed cases are people with symptoms who made contact with a hospital, got tested, and whose infection of COVID-19 was confirmed by DNA or imaging tests. The infected compartment in by transmission models represents all people who are infected. These include people who may or may not have symptoms and contacts with a hospital, as well as people with confirmed laboratory tests and those who are misdiagnosed. In this sense, confirmed cases (data) are only a fraction of the total infected population (model predictions). A metaphor of an iceberg best represents the difference between data and model predictions. The entire iceberg represents the total infected population, and the tip of the iceberg above the sea surface represents the case data. The part of the iceberg hidden under the water represents the infected people that are unknown to public health surveillance and testing; often called the hidden epidemic. The difference between cases and infections can be measured by the case-infection ratio ρ, between the newly confirmed cases and the number of infected people, or as a surrogate, the ratio between the cumulative confirm cases and the cumulative number of infected people.', 'cite_spans': [], 'section': 'Introduction', 'ref_spans': []}, {'text': 'The case-infection ratio ρ can vary widely for different viral infections that spread through air droplets and close contacts. For the SARS epidemic, the ratio ρ was in the range of 1/5−1/2 (Chowell et al., 2004; Gumel et al., 2004; Lipsitch et al., 2003; Zhang et al., 2005). In contrast, for seasonal influenza in 2019–2020, the ratio ρ can be as small as 1/100, based on estimates from the US CDC (US CDC, 2020). Why should this be a problem for the modellers? In model calibration, in order to estimate key model parameters such as the transmission rate β, by fitting the model output to the confirmed cases data, it is necessary to discount the total number of infectious people, I(t), from the model prediction, by the case-infection ratio ρ to appropriately predict confirmed case data. For each value of the ratio ρ, a corresponding value for the transmission rate β can then be estimated by fitting the model to data, which in turn determines the basic reproduction number R0, the scale of the epidemic, as well as the peak time. Given the potential wide range for the case-infection ratio ρ of the COVID-19, the estimated transmission rate β has a wide range, and hence the wide range of reported model predictions.', 'cite_spans': [{'start': 191, 'end': 211, 'mention': 'Chowell et al., 2004', 'ref_id': 'BIBREF29'}, {'start': 213, 'end': 231, 'mention': 'Gumel et al., 2004', 'ref_id': 'BIBREF32'}, {'start': 233, 'end': 254, 'mention': 'Lipsitch et al., 2003', 'ref_id': 'BIBREF4'}, {'start': 256, 'end': 274, 'mention': 'Zhang et al., 2005', 'ref_id': 'BIBREF26'}, {'start': 401, 'end': 413, 'mention': 'US CDC, 2020', 'ref_id': 'BIBREF18'}], 'section': 'Introduction', 'ref_spans': []}, {'text': 'In modeling terms, given the confirmed-case data, there is a linkage between the model parameter ρ and the transmission rate β, and potentially also with other model parameters. While many different combinations of ρ and β can show good fit to the data, they can produce very different model predictions of the epidemic. This is known as nonidentifiability in the modeling literature, see e.g. (Lintusaari, Gutmann, Kaski, & Corander, 2016; Raue et al., 2009; van der Vaart, 1998). It means that a group of model parameters can not be uniquely determined from the given data during model calibration. Different choices of parameter values with the same good fit to the data can lead to very different model predictions. The ways in which nonidentifiability is addressed in the model calibration process greatly influences the reliability of model predictions.', 'cite_spans': [{'start': 395, 'end': 439, 'mention': 'Lintusaari, Gutmann, Kaski, & Corander, 2016', 'ref_id': 'BIBREF3'}, {'start': 441, 'end': 458, 'mention': 'Raue et al., 2009', 'ref_id': 'BIBREF9'}, {'start': 460, 'end': 479, 'mention': 'van der Vaart, 1998', 'ref_id': 'BIBREF19'}], 'section': 'Introduction', 'ref_spans': []}, {'text': 'The standard nonlinear least squares method is known to be ill suited to detect or address the nonidentifiability issue, since it relies on a rudimentary optimization algorithm. These rudimentary optimization algorithms attempt to find a global minimum of the given objective function, but there are infinitely many global minima given nonidentifiability. Standard Markov chain Monte Carlo (MCMC) procedures based on Bayesian inference often fail to converge to the target posterior distribution in the presence of nonidentifiability, and can produce best-fit parameter values with unreliable credible intervals, since these often relies on elementary MCMC algorithms. Elementary MCMC algorithms converge very slowly given a very skewed posterior distribution. In our study, we used an improved model calibration method using Bayesian inference and affine invariant ensemble MCMC algorithm that can ensure fast convergence to the target posterior distribution when facing nonidentifiability, and provide more reliable credible intervals and model predictions.', 'cite_spans': [], 'section': 'Introduction', 'ref_spans': []}, {'text': 'Another important factor that can significantly influence model predictions is the choice of a suitable model to describe the epidemic under study: a more complex or simpler model. A complex model incorporates more biological and epidemiological information about the epidemic and is more biologically realistic. A drawback of a complex model is that it requires more model parameters to be estimated compared to a simpler model. Given the dataset, such as the confirmed case data of COVID-19, increased number of parameters in a complex model that are unknown and need to be estimated by model fitting can lead to a greater degree of uncertainty in model predictions. In choosing an appropriate model, it is important to draw a balance between biological realism and reducing uncertainty in model predictions, and this choice can significantly influence the reliability of model predictions. The modeling procedure to determine the right balance is model selection using various information criteria, for instance the Akaike Information Criteria (AIC) for nested models (Akaike, 1973; Sugiura, 1978).', 'cite_spans': [{'start': 1072, 'end': 1084, 'mention': 'Akaike, 1973', 'ref_id': 'BIBREF0'}, {'start': 1086, 'end': 1099, 'mention': 'Sugiura, 1978', 'ref_id': 'BIBREF15'}], 'section': 'Introduction', 'ref_spans': []}, {'text': 'In our study, we considered both SEIR and SIR models for model predictions and applied model-selection analysis. For the given dataset of confirmed cases, we determined that the SIR model is a better choice than the SEIR model, and more likely than models that are more complex than an SEIR model (Section 3). Our study focused on the development of the outbreak in Wuhan city after the quarantine and lockdown (January 23, 2020), given the reliability of confirmed case data and definition during this period and the simplicity in our predictions and analysis. We briefly outline in Section 2 the methodology for model calibration using an improved procedure based on Bayesian inference and model selection method using Akaike Information Criteria. In Section 4, using the SIR model, we illustrate the linkage between the transmission rate and case-infection ratio, and the presence of nonidentifiability when only the confirmed-case data is used for model calibration. In Section 5, we present detailed results of the SIR model calibration and our model predictions, including the distribution of peak time, prediction interval of future confirmed cases, as well as the total number of infected people. In Section 6, we estimate the impact of further control measures recommended in Wuhan after February 7 and predicted the changes in peak time under different assumptions on the reduction of transmission achieved by these measures. In Section 7, we estimated the impact of timing the return to work on the course of the epidemic, in terms of peak time, peak values, and the duration of the epidemics. Our results are summarized in Section 8.', 'cite_spans': [], 'section': 'Introduction', 'ref_spans': []}, {'text': 'Mathematical Model. Consider a mathematical model given by a system of differential equations:(1)x’=f(x),where x=(x1,⋯,xk) denotes the vector of state variables, f(x)=(f1(x),⋯fk(x)) the vector field. We let u∈Rn1 be the vector of all model parameters, which often include initial conditions x0=(x01,⋯,x0k). We assume that there exists a unique solution x=x(u,t) for each given u.', 'cite_spans': [], 'section': 'Affine invariant ensemble Markov chain Monte Carlo algorithm for model calibration ::: Model calibration and model selection', 'ref_spans': []}, {'text': 'Data. Data is often given on the observable quantities, such as newly confirmed cases, which are linear or nonlinear combinations of the solutions x(u,t) in the form:y=y(w,t)=y(x(u,t),v),where v∈Rn2 are parameters in the observables y and w=(u,v)∈Rn, n=n1+n2, is the vector of all model parameters to be estimated. Furthermore, the dataset is collected at N time points t1,t2,⋯,tN. We will fit the model outputsyi=y(w,ti)=y(x(u,ti),v),i=1,2,⋯,N,to the time series datasetD={D1,D2,⋯,DN}.', 'cite_spans': [], 'section': 'Affine invariant ensemble Markov chain Monte Carlo algorithm for model calibration ::: Model calibration and model selection', 'ref_spans': []}, {'text': 'Likelihood functions. In order to account for noise in the data, we let the probability of observing Di at time ti be given by fi(Di), with mean yi and variance qi=σi2=1/τi, i=1,2,⋯,N. Common probability distributions used for this purpose include the normal distribution, Poisson distribution, and negative binomial distribution. In our Bayesian inference, the variance qi=1/τi in the noise distribution is also estimated from the data, giving us an accurate posterior distribution and accurate credible intervals for the estimated parameters. The entire set of parameters to be estimated includes model parameters u, parameters v in the observable function y, and the variances q=(1/τ1,1/τ2,⋯,1/τN), and is denoted byθ=(u,v,q).', 'cite_spans': [], 'section': 'Affine invariant ensemble Markov chain Monte Carlo algorithm for model calibration ::: Model calibration and model selection', 'ref_spans': []}, {'text': 'We consider the likelihood functionL(θ)=CP(D|θ)=Cf1(D1)f2(D2)⋯fN(DN),where C is a constant independent of θ used to simplify the likelihood function (Kalbfleisch, 1979).', 'cite_spans': [{'start': 150, 'end': 167, 'mention': 'Kalbfleisch, 1979', 'ref_id': 'BIBREF2'}], 'section': 'Affine invariant ensemble Markov chain Monte Carlo algorithm for model calibration ::: Model calibration and model selection', 'ref_spans': []}, {'text': 'Bayesian framework. The Bayesian framework assumes that a probability model for the observed data D given unknown parameters θ is P(D|θ), and that θ is randomly distributed from the prior distribution P(θ). Statistical inference for θ is based on the posterior distribution P(θ|D). Using Bayes Theorem we obtainPθ|D=PD|θPθPD=PD|θPθ∫ΩPD|θPθdθ∝LθPθ=πθ|D,where Ω is the parameter space of θ and L(θ) is the likelihood function. Constant P(D)=∫ΩP(D|θ)P(θ)dθ is used to normalize the posterior distribution P(θ|D) (Chen, Shao, & Ibrahim, 2000). The unnormalized posterior distribution is given by π(θ|D)=L(θ)P(θ). The Bayesian framework is very useful for statistical inference that occurs in mathematical modeling since it allows utilization of the prior information about the unknown parameters in the literature. Epidemiological information about the infectious disease can often inform a general range for the parameters to be estimated, and the uniform distribution is typically chosen as the prior distribution in such a case.', 'cite_spans': [{'start': 510, 'end': 537, 'mention': 'Chen, Shao, & Ibrahim, 2000', 'ref_id': 'BIBREF28'}], 'section': 'Affine invariant ensemble Markov chain Monte Carlo algorithm for model calibration ::: Model calibration and model selection', 'ref_spans': []}, {'text': 'Markov chain Monte Carlo algorithms. Markov chain Monte Carlo (MCMC) algorithms are used to approximate a posterior distribution of parameters by randomly sampling the parameter space (Lynch, 2007). In MCMC algorithms, a new vector of parameter values θ(t) is sampled iteratively from the posterior distribution, based on the previous vector θ(t−1), until a sample path (also called a chain or walker) has arrived at a stationary process and produces the target unnormalized posterior distribution. Commonly used MCMC algorithms include the Metropolis-Hastings algorithm and Random-Walk Metropolis-Hastings algorithms (Chen et al., 2000).', 'cite_spans': [{'start': 185, 'end': 196, 'mention': 'Lynch, 2007', 'ref_id': 'BIBREF5'}, {'start': 619, 'end': 636, 'mention': 'Chen et al., 2000', 'ref_id': 'BIBREF28'}], 'section': 'Affine invariant ensemble Markov chain Monte Carlo algorithm for model calibration ::: Model calibration and model selection', 'ref_spans': []}, {'text': 'In our study, we used an improved MCMC algorithm, the affine invariant ensemble Markov chain Monte Carlo algorithm, which has been shown to perform better than Metropolis-Hastings and other MCMC algorithms, especially in the presence of nonidentifiability. The algorithm uses a number of walkers and the positions of the walkers are updated based on the present positions of all walkers. For details on this algorithm, we refer the reader to (Goodman and Weare, 2010, May, 2015) and recent lecture notes on this topic (Roda, 2020).', 'cite_spans': [{'start': 443, 'end': 466, 'mention': 'Goodman and Weare, 2010', 'ref_id': 'BIBREF31'}, {'start': 468, 'end': 477, 'mention': 'May, 2015', 'ref_id': 'BIBREF6'}, {'start': 519, 'end': 529, 'mention': 'Roda, 2020', 'ref_id': 'BIBREF12'}], 'section': 'Affine invariant ensemble Markov chain Monte Carlo algorithm for model calibration ::: Model calibration and model selection', 'ref_spans': []}, {'text': 'When using mathematical models to explain data that has been formed by an underlying disease process, the principle of parsimony should be used to select a suitable model. A parsimonious model is the simplest model with the least assumptions and variables but with the greatest explanatory power for the disease process represented by the data (Johnson & Omland, 2015). This principle is also reflected in a well known quotation: “Models should be as simple as possible but not simpler.” This quotation is often ascribed to A. Einstein. The model selection method using Akaike Information Criterion takes into account both how well the model fits the data and the principle of parsimony.', 'cite_spans': [{'start': 345, 'end': 367, 'mention': 'Johnson & Omland, 2015', 'ref_id': 'BIBREF1'}], 'section': 'Method of model selection using Akaike information criterion ::: Model calibration and model selection', 'ref_spans': []}, {'text': 'Akaike Information Criterion (AIC). Let L(θˆMLE) be the maximum likelihood value achieved at a best-fit parameter value θˆMLE. Let K be the number of parameters to be estimated in a model, and N be the number of time points where data are observed. The Akaike Information Criterion (AIC) is defined as (Akaike, 1973):AIC=−2ln(L(θˆMLE))+2K.', 'cite_spans': [{'start': 303, 'end': 315, 'mention': 'Akaike, 1973', 'ref_id': 'BIBREF0'}], 'section': 'Method of model selection using Akaike information criterion ::: Model calibration and model selection', 'ref_spans': []}, {'text': 'This definition should be used when K<N/40, namely when the number of time points N is large in comparison to the number of parameters. When K>N/40, namely when the number of parameters is large in comparison to the number of time points, the following corrected AIC should be used (Sugiura, 1978):AICc=AIC+2K(K+1)N−K−1.', 'cite_spans': [{'start': 283, 'end': 296, 'mention': 'Sugiura, 1978', 'ref_id': 'BIBREF15'}], 'section': 'Method of model selection using Akaike information criterion ::: Model calibration and model selection', 'ref_spans': []}, {'text': 'We note that in the Bayesian inference based calibration, the unnormalized posterior distribution π(θ|D) is equal to the product of the likelihood function L(θ) and the prior distribution P(θ). The Akaike information criterion can be applied if uniform prior distributions are used for each parameter, since π(θˆMLE)=γL(θˆMLE), where γ is a constant.', 'cite_spans': [], 'section': 'Method of model selection using Akaike information criterion ::: Model calibration and model selection', 'ref_spans': []}, {'text': 'Model selection using AIC. When several nested models, each having a different level of complexity, are considered as candidates for the most suitable model, AIC values can be computed for each model, and the model associated with the smallest AIC value is considered the best model. The difference of AICi value of model i with the minimum miniAICi:Δi=AICi−miniAICi.', 'cite_spans': [], 'section': 'Method of model selection using Akaike information criterion ::: Model calibration and model selection', 'ref_spans': []}, {'text': 'This measures the information lost when using model i instead of a model with the smallest AIC value. When Δi is larger, model i is less plausible.', 'cite_spans': [], 'section': 'Method of model selection using Akaike information criterion ::: Model calibration and model selection', 'ref_spans': []}, {'text': 'Useful guidelines for interpreting Δi for nested models are as follows (Burnham & Anderson, 2002):•If 1≤Δi≤2, model i has substantial support and should be considered.•If 4≤Δi≤7, model i has less support.•If Δi>10, model i has no support and can be omitted.', 'cite_spans': [{'start': 72, 'end': 96, 'mention': 'Burnham & Anderson, 2002', 'ref_id': 'BIBREF22'}], 'section': 'Method of model selection using Akaike information criterion ::: Model calibration and model selection', 'ref_spans': []}, {'text': 'When a large number of models are under consideration or the models are not nested, the model selection rules are different. We refer the reader to recent lecture notes (Portet, 2020) for an introduction to model selection.', 'cite_spans': [{'start': 170, 'end': 182, 'mention': 'Portet, 2020', 'ref_id': 'BIBREF8'}], 'section': 'Method of model selection using Akaike information criterion ::: Model calibration and model selection', 'ref_spans': []}, {'text': 'In our SIR and SEIR models, the compartment S denotes the susceptible population in Wuhan, compartment I denotes the infectious population, and R denotes the confirmed cases. In the SEIR model, a latent compartment E is added to denote the individuals who are infected but not infectious. The latency of COVID-19 infection is biologically realistic due to an incubation period as long as 14 days; newly infected individuals may not be infectious while the virus is incubating in the body. Here we note the difference between the latent period, which is the period from the time an individual is infected to the time the individual is infectious, and the incubation period, which is the period between the time an individual is infected to the time clinical symptoms appear, which include fever and coughing for COVID-19. For SARS, infected individuals become infectious on average two days after the onset of symptoms WHO (2003); so, the SARS latent period is on average longer than the incubation period. For COVID-19, evidence has shown that infected individuals can be infectious before the onset of symptoms (Bai et al., 2020), but the length of the latent period is largely unknown. In comparison to the SIR model, the SEIR model has the strength of being more biologically realistic, but the SEIR model has the drawback of having two additional unknown parameters: the latent period and the initial latent population.', 'cite_spans': [{'start': 918, 'end': 928, 'mention': 'WHO (2003)', 'ref_id': 'BIBREF20'}, {'start': 1113, 'end': 1129, 'mention': 'Bai et al., 2020', 'ref_id': 'BIBREF11'}], 'section': 'The models ::: Model selection analysis for an SEIR and an SIR model', 'ref_spans': []}, {'text': 'The transfer diagrams for both models are shown in Fig. 1. The biological meaning of all model parameters are given in Table 1 and Table 2. A key assumption in both models is that deaths occurring in the S, E, and I compartments are negligible during the period of model predictions.', 'cite_spans': [], 'section': 'The models ::: Model selection analysis for an SEIR and an SIR model', 'ref_spans': [{'start': 51, 'end': 57, 'mention': 'Fig. 1', 'ref_id': 'FIGREF0'}, {'start': 119, 'end': 126, 'mention': 'Table 1', 'ref_id': 'TABREF0'}, {'start': 131, 'end': 138, 'mention': 'Table 2', 'ref_id': 'TABREF1'}]}, {'text': '(4 months). Since we use the newly confirmed case data for model calibration, which is matched to the ρI term in both models, the death term in the R compartment has no effect on our model fitting. The systems of differential equations for each model is given below:(2)S’=−βISI’=βIS−(ρ+μ)IR’=ρI−dR(3)S’=−βISE’=βIS−εEI’=εE−(ρ+μ)IR’=ρI−dR', 'cite_spans': [], 'section': 'The models ::: Model selection analysis for an SEIR and an SIR model', 'ref_spans': []}, {'text': 'For data reliability, the data used for both models (2) and (3) is the newly confirmed cases in Wuhan city from the official reports from January 21 to February 4, 2020 (National Health Commission of the People’s Republic of China, 2020). It is common to use a Poisson or negative binomial probability model for observed count data. When the mean of a Poisson or negative binomial distribution is large, it approximates a normal distribution. Since the newly confirmed cases are approaching large values quickly, the distribution of the count data will be approximately normal and the probability model for the observed count data in our study was assumed to a normal distribution with mean given by ρI and variance given by 1/τ. There are four parameters to be estimated in the SIR model from data: transmission rate β, diagnosis rate ρ, the initial population size I0 for the compartment I on January 21, 2019 (t=0), and the variance q=1/τ for the noise distribution in the data. There are six parameters to be estimated for the SEIR model: transfer rate ε from E to I, the initial population size E0 for the latent compartment E on January 21, 2019, and β, ρ, I0, and q=1/τ. Since it was announced at a news conference by the mayor of Wuhan on January 23 that 5 million people have left the city by that date, we set the total population N=S+I+R in Wuhan on January 21 to the conservative estimate of 6 million.', 'cite_spans': [{'start': 170, 'end': 236, 'mention': 'National Health Commission of the People’s Republic of China, 2020', 'ref_id': 'BIBREF7'}], 'section': 'Model calibration from the data ::: Model selection analysis for an SEIR and an SIR model', 'ref_spans': []}, {'text': 'We used the same uniform distributions over the initial range of parameters as the priors for both models, as given in Table 1, Table 2. The affine invariant ensemble Markov chain Monte Carlo algorithm was used to produce posterior distributions for all estimated parameters. From these posterior distributions, we obtain the best-fit values and the 95% credible intervals, as given in Table 1 for the SIR model (2) and in Table 2 for the SEIR model (3).', 'cite_spans': [], 'section': 'Model calibration from the data ::: Model selection analysis for an SEIR and an SIR model', 'ref_spans': [{'start': 119, 'end': 126, 'mention': 'Table 1', 'ref_id': 'TABREF0'}, {'start': 128, 'end': 135, 'mention': 'Table 2', 'ref_id': 'TABREF1'}, {'start': 386, 'end': 393, 'mention': 'Table 1', 'ref_id': 'TABREF0'}, {'start': 423, 'end': 430, 'mention': 'Table 2', 'ref_id': 'TABREF1'}]}, {'text': 'Using the calibration results for both the SIR and SEIR models in Section 3.3, their corrected Akaike Information Criterion AICc are calculated as 174 and 186, respectively. The difference Δ=186−174=12 is sufficiently large and this implies that using the SEIR model (3) will produce a significant loss of information in comparison to using the SIR model (2). Accordingly, our further investigation will be carried out using the SIR model (2).', 'cite_spans': [], 'section': 'Comparing SIR and SEIR models ::: Model selection analysis for an SEIR and an SIR model', 'ref_spans': []}, {'text': 'Based on our calibration results of the SIR model in Section 3.3, we detected a linkage between the transmission rate β and the diagnosis rate ρ. In Fig. 2 (a), we show the projection of the unnormalized posterior distribution in the β-ρ parameter space. It shows that the largest probability are concentrated along a flat strip rather than on a single point. Correspondingly, as shown in Fig. 2 (b), a curve in the β-ρ parameter space can be determined such that every point on the curve has approximately the same large probability. The linkage between two or more parameters implies the following: (1) the best-fit parameter values are effectively not unique; and (2) there is a continuum of parameter values that cause the model to fit the data approximately equally as well. This phenomenon is often referred to as nonidentifiability in the modeling literature.', 'cite_spans': [], 'section': 'Nonidentifiability: linkage between transmission rate β and diagnosis rate ρ', 'ref_spans': [{'start': 149, 'end': 159, 'mention': 'Fig. 2 (a)', 'ref_id': 'FIGREF1'}, {'start': 389, 'end': 399, 'mention': 'Fig. 2 (b)', 'ref_id': 'FIGREF1'}]}, {'text': 'To further illustrate the significant impact of nonidentifiability on model predictions, we choose two endpoints on the curve in Fig. 2 (b), with respective values (β,ρ)=(2.09e-7,0.909) and (β,ρ)=(7.34e-8,0.084), and we plotted the corresponding projected new cases in Fig. 3(a) and (b), respectively. Fig. 3 shows that the peak height, as well as the duration and scale of the epidemic are different in the two projections, even though both choices of parameter values are effectively equally likely to produce the best fit between the model outcomes and the data.', 'cite_spans': [], 'section': 'Nonidentifiability: linkage between transmission rate β and diagnosis rate ρ', 'ref_spans': [{'start': 129, 'end': 139, 'mention': 'Fig. 2 (b)', 'ref_id': 'FIGREF1'}, {'start': 269, 'end': 275, 'mention': 'Fig. 3', 'ref_id': 'FIGREF2'}, {'start': 302, 'end': 308, 'mention': 'Fig. 3', 'ref_id': 'FIGREF2'}]}, {'text': 'A striking feature in Fig. 3 is that the peak time of the two different projections are almost identical. This illustrates that, unlike the peak value, the peak time of the epidemic is insensitive to small parameter changes. This important property of the peak time will also be observed in later sections.', 'cite_spans': [], 'section': 'Nonidentifiability: linkage between transmission rate β and diagnosis rate ρ', 'ref_spans': [{'start': 22, 'end': 28, 'mention': 'Fig. 3', 'ref_id': 'FIGREF2'}]}, {'text': 'We further note that the diagnosis rate ρ is the case-infection ratio that is used to discount of the number of infected individuals I(t) to properly predict the newly confirmed cases. The linkage between β and ρ reflects the dependency of the transmission rate and the case-infection ratio, and hence the scale of the epidemic. We believe that this nonidentifiability is the reason for the wide variability in published model predictions of COVID-19 epidemic.', 'cite_spans': [], 'section': 'Nonidentifiability: linkage between transmission rate β and diagnosis rate ρ', 'ref_spans': []}, {'text': 'To reduce the impact of nonidentifiability in model calibration from data, one approach is to search for more independent data, including clinical, surveillance, or administrative data, and from published literature, that can be used for model calibration. This approach is often difficult when facing an outbreak of unknown pathogens that occur in real time such as SARS in 2003 and the current COVID-19. Another approach is to adopt better inference methods and model fitting algorithms to narrow done the otherwise large confidence or credible intervals. Our fitting procedure using Bayesian inference and the affine invariant ensemble Markov chain Monte Carlo algorithm was able to achieve this objective.', 'cite_spans': [], 'section': 'Nonidentifiability: linkage between transmission rate β and diagnosis rate ρ', 'ref_spans': []}, {'text': 'Our baseline predictions for Wuhan are prediction intervals produced by randomly sampling the posterior distribution. The best-fit parameter values and credible intervals are shown in Table 1. The Bayesian inference used the newly confirmed cases for Wuhan contained in the official reports from January 21 to February 4, 2020. This is the period during the lockdown and travel restrictions in Wuhan, but before the further control measures that were undertaken in Wuhan after February 7, 2020, including the drastic increase in the available hospital beds and the door-to-door visits used to identify and quarantine suspected cases. These projections show our estimation for the hypothetical epidemic in Wuhan if further control measures after February 7 were not implemented.', 'cite_spans': [], 'section': 'Baseline predictions for Wuhan and three scenarios', 'ref_spans': [{'start': 184, 'end': 191, 'mention': 'Table 1', 'ref_id': 'TABREF0'}]}, {'text': 'In Fig. 4(a) and (b), we show the distributions of the projected peak time and the estimated values of the control reproduction number Rc. In Fig. 4 (c), we show the projected time course of newly confirmed cases in Wuhan together with its 95% prediction interval. The fit of our model predictions and the newly-confirmed case data is shown for the period between January 21 to February 4 in Fig. 4 (d). Based on these projections, if the more restrictive control measures after February 7 in the city were not implemented, the most likely peak time would have occurred on February 27, 2020, with the 95% credible interval from February 23 - March 6. The median value of Rc is 1.629 with the first quantile 1.414 and the third quantile 1.979. By our projection, without the strict quarantine measures after February 7, the peak case total would reach approximately 120,000, and the epidemic in Wuhan would not be over before mid-May of 2020.', 'cite_spans': [], 'section': 'Baseline predictions for Wuhan and three scenarios', 'ref_spans': [{'start': 3, 'end': 9, 'mention': 'Fig. 4', 'ref_id': 'FIGREF3'}, {'start': 142, 'end': 152, 'mention': 'Fig. 4 (c)', 'ref_id': 'FIGREF3'}, {'start': 392, 'end': 402, 'mention': 'Fig. 4 (d)', 'ref_id': 'FIGREF3'}]}, {'text': 'At the time of this manuscript, the consensus among medical experts is that the basic reproduction number R0 near the beginning of the COVID-19 outbreak in Wuhan is around 2. Our result in Fig. 4 (b) is comparable with earlier estimates and the current consensus. It also shows that, even without the more restrictive control measures in Wuhan undertaken after February 7, the lockdown and travel restrictions in the city had slowed down the transmission and reduced the basic reproduction number to a control reproduction number Rc with a mean value 1.629. We will estimate the impact of the more restrictive control measures in Section 6.', 'cite_spans': [], 'section': 'Baseline predictions for Wuhan and three scenarios', 'ref_spans': [{'start': 189, 'end': 199, 'mention': 'Fig. 4 (b)', 'ref_id': 'FIGREF3'}]}, {'text': 'The baseline prediction intervals are computed over a large credible interval of the diagnosis rate ρ, (0.0637,0.909), which represents a wide range of assumptions on the case-infection ratio and the scale of the epidemic in Wuhan. We further restricted the parameter ρ to three narrower ranges: (0.02,0.03), (0.05,0.1), and (0.2,1), and recalibrated the SIR model (2) with each of the ρ ranges. The resulting predictions for newly confirmed cases are shown in Fig. 5.', 'cite_spans': [], 'section': 'Baseline predictions for Wuhan and three scenarios', 'ref_spans': [{'start': 461, 'end': 467, 'mention': 'Fig. 5', 'ref_id': 'FIGREF4'}]}, {'text': 'In Fig. 5, different ρ ranges have resulted in significant variations in the peak value of cases and the duration of the epidemic. In contrast, the projected peak times are very similar in all three cases, which further demonstrates that the peak time is insensitive to changes in parameters.', 'cite_spans': [], 'section': 'Baseline predictions for Wuhan and three scenarios', 'ref_spans': [{'start': 3, 'end': 9, 'mention': 'Fig. 5', 'ref_id': 'FIGREF4'}]}, {'text': 'After February 7, 2020, Wuhan implemented more strict quarantine measures that included the following: locking down residential buildings and compounds, strict self quarantine for families, door-to-door inspection for suspected cases, quarantining suspected cases and close contacts in newly established hospitals and other quarantine spaces including vacated hotels and university dormitories. The goal of these measures was to reduce transmissions within family clusters and residential compounds. These measures have a direct impact on two parameters in our SIR model (2): reducing the transmission rate β and increasing the diagnosis rate ρ. It is difficult to estimate the exact impacts on these parameters by these measures. We incorporated several likely scenarios of the effects of these measures by adjusting our baseline estimates of β and ρ and we plotted the resulting time courses in Fig. 6.', 'cite_spans': [], 'section': 'Impacts of more strict quarantine measures in Wuhan after February 7', 'ref_spans': [{'start': 897, 'end': 903, 'mention': 'Fig. 6', 'ref_id': 'FIGREF5'}]}, {'text': 'In Fig. 6 (a), we see that a combination of a 10% reduction in the transmission rate β and a 90% increase in the diagnosis rate ρ can effectively stop the epidemic in its tracks, force the newly diagnosed cases to decline, and significantly shorten the duration of the epidemic.', 'cite_spans': [], 'section': 'Impacts of more strict quarantine measures in Wuhan after February 7', 'ref_spans': [{'start': 3, 'end': 13, 'mention': 'Fig. 6 (a)', 'ref_id': 'FIGREF5'}]}, {'text': 'With newly diagnosed cases on the decline in Wuhan and other cities in China since February 14, an urgent task for the authorities is to decide when to allow people to go back to work. Without lifting the ban on traffic in and out of the city, we tested three hypotheses of allowing people to return to work in Wuhan at three different dates: February 24, March 2, and March 31. As shown in Fig. 7, our results predict a significant second outbreak after the return-to-work day.', 'cite_spans': [], 'section': 'Potential of a second outbreak in Wuhan after the return-to-work', 'ref_spans': [{'start': 391, 'end': 397, 'mention': 'Fig. 7', 'ref_id': 'FIGREF6'}]}, {'text': 'The COVID-19 epidemics have presented China and many other countries in the world with an unprecedented public health challenge in the modern era, with a significant impact on health and public health systems, human lives and national and world economies. Mathematical modeling is an important tool for estimating and predicting the scale and time course of epidemics, evaluating the effectiveness of public health interventions, and informing public health policies. The focus of our study is to demonstrate the challenges facing modelers in predicting outbreaks of this nature and to provide a partial explanation for the wide variability in earlier model predictions of the COVID-19 epidemic.', 'cite_spans': [], 'section': 'Conclusions', 'ref_spans': []}, {'text': 'Our study focused on the COVID-19 epidemic in Wuhan city, the epicentre of the epidemic, during a less volatile period of the epidemic, after the lock down and quarantine of the city. By comparing standard SIR and SEIR models in predicting the epidemic using the Akaike Information Criterion, we showed that, given the same dataset of confirmed cases, more complex models may not necessarily be more reliable in making predictions due to the larger number of model parameters to be estimated.', 'cite_spans': [], 'section': 'Conclusions', 'ref_spans': []}, {'text': 'Using a simple SIR model and the dataset of newly diagnosed cases in Wuhan for model calibration, we demonstrated that there is a linkage between the transmission rate β and the case-infection ratio ρ, which resulted in a continuum of best-fit parameter values, which can produce significantly different model predictions of the epidemic. This is a hallmark of nonidentifiability, and the root cause for variabilities in model predictions. The nonidentifiability should not be interpreted as a shortcoming of transmission models; neither is it caused by the limited number of time points in data. Rather, it is caused by the lack of datasets that are independent of the conirmed cases to allow modelers to produce independent estimates of β and ρ. The reliability in model predictions depends on how rigorously the nonidentifiability is addressed in model calibration and on the choice of parameter values.', 'cite_spans': [], 'section': 'Conclusions', 'ref_spans': []}, {'text': 'We demonstrated that Bayesian inference and an improved Markov chain Monte Carlo algorithm, the affine invariant ensemble Markov chain Monte Carlo algorithm, can significantly reduce the wide parameter ranges in the uniform prior and produce workable credible intervals, even in the presence of nonidentifiability. We showed that the estimated credible intervals for the parameters are sufficiently small to allow our credible interval for the peak time to fall within a week. We have further demonstrated that the peak time of the epidemic is much less sensitive to parameter variations than the peak values and the scale of the epidemic. This was also observed in our previous work on predicting seasonal influenza for the Province of Alberta.', 'cite_spans': [], 'section': 'Conclusions', 'ref_spans': []}, {'text': 'We estimated the impact of the Wuhan lockdown and traffic restrictions in the city after January 23 and before February 6, 2020. We show that if the more restrictive control and prevention measures were not implemented in the city, the epidemic would peak between the end of February and first week of March of 2020. Our results can be used to inform public health authorities on what may happen if the more strict quarantine measures after February 7 were not taken.', 'cite_spans': [], 'section': 'Conclusions', 'ref_spans': []}, {'text': 'When the more restrictive measures are incorporated into our model, including the lock down of residential buildings and compounds, the door-to-door search of suspected cases, and the quarantine of suspected cases and their close contacts in newly established hospitals and quarantine spaces, we showed that these measures can effectively stop the otherwise surging epidemic in its tracks and significantly reduce the duration of the epidemic. These findings provide a theoretical verification of the effectiveness of these measures.', 'cite_spans': [], 'section': 'Conclusions', 'ref_spans': []}, {'text': 'We further considered the impact of the return-to-work order on different dates in February and March on the course of the outbreak. Our results show that a second peak in Wuhan is very likely even if the return-to-work happens near the end of March 2020. This may serve as a warning to the public health authorities.', 'cite_spans': [], 'section': 'Conclusions', 'ref_spans': []}, {'text': 'The authors claim no conflict of interests.', 'cite_spans': [], 'section': 'Declaration of competing interest', 'ref_spans': []}], 'paragraphs': [{'text': 'In our SIR and SEIR models, the compartment S denotes the susceptible population in Wuhan, compartment I denotes the infectious population, and R denotes the confirmed cases. In the SEIR model, a latent compartment E is added to denote the individuals who are infected but not infectious. The latency of COVID-19 infection is biologically realistic due to an incubation period as long as 14 days; newly infected individuals may not be infectious while the virus is incubating in the body. Here we note the difference between the latent period, which is the period from the time an individual is infected to the time the individual is infectious, and the incubation period, which is the period between the time an individual is infected to the time clinical symptoms appear, which include fever and coughing for COVID-19. For SARS, infected individuals become infectious on average two days after the onset of symptoms WHO (2003); so, the SARS latent period is on average longer than the incubation period. For COVID-19, evidence has shown that infected individuals can be infectious before the onset of symptoms (Bai et al., 2020), but the length of the latent period is largely unknown. In comparison to the SIR model, the SEIR model has the strength of being more biologically realistic, but the SEIR model has the drawback of having two additional unknown parameters: the latent period and the initial latent population.', 'id': '00024'}], 'title': 'Why is it difficult to accurately predict the COVID-19 epidemic?', 'sha': '4d71eda036617bbbfc9d6b8b916d94aa4de9a705; 5b0cb1777c2f78d0e598794f26eb85c79471210f', 'doi': '10.1016/j.idm.2020.03.001', 'score': 8.714599609375}, {'rank': 287, 'abstract': 'Background: As the COVID-19 epidemic is spreading, incoming data allows us to quantify values of key variables that determine the transmission and the effort required to control the epidemic. We determine the incubation period and serial interval distribution for transmission clusters in Singapore and in Tianjin. We infer the basic reproduction number and identify the extent of pre-symptomatic transmission. Methods: We collected outbreak information from Singapore and Tianjin, China, reported from Jan.19-Feb.26 and Jan.21-Feb.27, respectively. We estimated incubation periods and serial intervals in both populations. Results: The mean incubation period was 7.1 (6.13, 8.25) days for Singapore and 9 (7.92, 10.2)days for Tianjin. Both datasets had shorter incubation periods for earlier-occurring cases. The mean serial interval was 4.56 (2.69, 6.42) days for Singapore and 4.22 (3.43, 5.01) for Tianjin. We inferred that early in the outbreaks, infection was transmitted on average 2.55 and 2.89days before symptom onset (Singapore, Tianjin). The estimated basic reproduction number for Singapore was 1.97 (1.45, 2.48) secondary cases per infective; for Tianjin it was 1.87 (1.65,2.09) secondary cases per infective. Conclusions: Estimated serial intervals are shorter than incubation periods in both Singapore and Tianjin, suggesting that pre-symptomatic transmission is occurring. Shorter serial intervals lead to lower estimates of R0, which suggest that half of all secondary infections should be prevented to control spread.', 'body_text': [{'text': 'quarantined and sections of the Baodi District where the store is located were sealed and put under security patrol.', 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': 'We screened publicly available data to identify datasets for two COVID-19 clusters that can be used to estimate transmission dynamics within a relatively closed system, where immediate public health responses were implemented, contacts were identified and quarantined, and key infection dates were tracked and published. We use these clusters to estimate both incubation period and serial interval for COVID-19, in both datasets, and discuss the implications of our findings for R0 estimates and pre-symptomatic transmission.', 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': 'All datasets and R code are available on GitHub (github.com/carolinecolijn/ClustersCOVID19).', 'cite_spans': [], 'ref_spans': [], 'section': 'Data'}, {'text': 'Singapore data was obtained from the Ministry of Health Singapore [14] online press releases.', 'cite_spans': [{'start': 66, 'end': 70, 'text': '[14]', 'ref_id': 'BIBREF13'}], 'ref_spans': [], 'section': 'Data'}, {'text': 'The Singapore dataset comprised 93 confirmed cases from the date of the initial case on January 19, 2020 until February 26, 2020. Tianjin data was obtained from the Tianjin Health Commission [15] online press releases. The Tianjin dataset comprises 135 cases confirmed from January 21 to February 22, 2020. The symptom onsets were available on the official website for all but a few patients who had not had symptoms before being diagnosed at a quarantine center. Both datasets contained information on exposure times, contacts among cases, time of symptom onset and more (see Supplementary Information for column descriptions and data processing).', 'cite_spans': [{'start': 191, 'end': 195, 'text': '[15]', 'ref_id': None}], 'ref_spans': [], 'section': 'Data'}, {'text': '. CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '4'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.03.20029983 doi: medRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': '4'}, {'text': 'For both Singapore and Tianjin, the daily incidence of hospitalization and mortality was plotted with the cumulative number of cases confirmed and discharged. The daily incidence rate was also visualized by date of symptom onset per probable source of infection. For the symptom onset plots, any cases that did not have information on date of onset of symptoms were removed. Cases were then grouped based on information provided in the \"presumed reason\" column for Singapore dataset and by the \"Infection source\" column in the Tianjin dataset. In the Tianjin dataset, there were a small number of cases (n = 10) that could be classified into two possible infection source groups (e.g. from Wuhan and has a close relationship with another known case). These cases were assigned their infection source groups based on the following hierarchy of possible sources:', 'cite_spans': [], 'ref_spans': [], 'section': 'Statistical analysis'}, {'text': '(highest priority) Known relationship > Wuhan origin > Other China travel > Location unclear travel > Mall (for shoppers, workers, or individuals living near to the Baodi mall outbreak) > Unknown (lowest priority). All cases in the Singapore dataset were categorized into an infection source group without conflict. The group designations were not used in the statistical estimates.', 'cite_spans': [], 'ref_spans': [], 'section': 'Statistical analysis'}, {'text': 'Incubation period estimates were based on data for the earliest and latest possible exposure times, and on the reported times of symptom onset. It is impossible to confirm the exact times of exposure, so we use interval censoring (R package icenReg [16] ) to make parametric estimates of the incubation period distribution. Where the earliest exposure is not specified in the \"start source\" column, we noted that the case must have been exposed some time since the beginning of the outbreak (Dec 01, 2019). Where the last possible exposure is not given in \"end source\", we noted that exposure must have occurred before symptom onset. In the Singapore dataset, some cases had a travel history or contact with a known location or presumed source of the virus and this defined a most recent time of exposure. For these we proceed as in Tianjin. Other individuals had estimated exposure times based on the symptom times for their presumed infector. For these, we define an exposure window using the symptoms of their presumed infector ±4 days. Having defined exposure windows, we proceed with interval censoring as in the Tianjin data. In both datasets 5 . CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 249, 'end': 253, 'text': '[16]', 'ref_id': 'BIBREF15'}], 'ref_spans': [], 'section': 'Statistical analysis'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.03.20029983 doi: medRxiv preprint we stratified the data according to whether symptom onset occurred early or late and estimated incubation periods separately.', 'cite_spans': [], 'ref_spans': [], 'section': 'Statistical analysis'}, {'text': 'Serial intervals were estimated with the expectation-maximization approach described in Vink et al [17] . Briefly, this approach assigns the case with earliest symptom onset in the cluster as a \"putative index\" (PI) status, and uses the time difference between symptom onset of subsequent cases in the cluster and the putative index as \"index case to case\" (ICC) intervals for putative index cases in small clusters. The ICC intervals are the time differences between the symptom onset in the putative index (PI) and the others, so the collection of times t j − t pi for each j in the cluster. These intervals are not samples of the serial interval distribution, because it need not be the case that the PI infected the others. Vink et al [17] used a mixture model in which ICC intervals t j − t pi can arise in four ways: (1) an outside case infects PI and j; (2) PI infects j; (3) PI infects an unknown who infects j and (4) PI infects unknown 1 who infects unknown 2 who infects j.', 'cite_spans': [{'start': 99, 'end': 103, 'text': '[17]', 'ref_id': 'BIBREF16'}, {'start': 739, 'end': 743, 'text': '[17]', 'ref_id': 'BIBREF16'}], 'ref_spans': [], 'section': 'Statistical analysis'}, {'text': 'Accordingly, if the serial interval x ∼ N (µ, σ 2 ), the density for the ICC intervals is', 'cite_spans': [], 'ref_spans': [], 'section': 'Statistical analysis'}, {'text': 'where w i are weights of the i th component density and f i are the component densities for the i th transmission route. See [17] for more details. Expectation-maximization is used to determine µ and σ.', 'cite_spans': [{'start': 125, 'end': 129, 'text': '[17]', 'ref_id': 'BIBREF16'}], 'ref_spans': [], 'section': 'Statistical analysis'}, {'text': 'For each dataset we create a graph whose nodes are individuals and whose edges are reported direct contacts between individuals. We use the connected components of the graph to define the small clusters. We also explored using earliest end exposure time, in case the first symptomatic case was not the index case for the cluster. A few clusters are large enough that the four models in the mixture are likely insufficient to cover the transmission possibilities, so we restrict the analysis to only the first 3, 4, 5 or 6 cases per cluster.', 'cite_spans': [], 'ref_spans': [], 'section': 'Statistical analysis'}, {'text': 'The mean time difference between transmitting infection by an infector and symptom onset is 6 . CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Statistical analysis'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.03.20029983 doi: medRxiv preprint (assuming independence) then the difference between the mean serial interval and the mean incubation time. We calculate the reproduction number as in [18] : R = exp rµ − 1/2r 2 σ 2 with r the exponential growth rate, µ the mean serial interval and σ the standard deviation of the serial interval. To obtain confidence intervals for R we resample µ and σ from our bootstrap samples; this preserved covariation of µ and σ. Statistical analyses were performed using R [19] . Figure 1c indicates that symptom onset occurred 6.6 ± 4.8 (mean ± SD) days after the initial presumed viral exposure and hospitalization occurred 5.9 ± 5.1 days after symptom onset. The mean length of hospital stay was 13.3 ± 6.0 days before individuals recovered and were discharged.', 'cite_spans': [{'start': 294, 'end': 298, 'text': '[18]', 'ref_id': 'BIBREF17'}, {'start': 609, 'end': 613, 'text': '[19]', 'ref_id': 'BIBREF18'}], 'ref_spans': [{'start': 616, 'end': 625, 'text': 'Figure 1c', 'ref_id': 'FIGREF0'}], 'section': 'Statistical analysis'}, {'text': 'In the Tianjin dataset, new confirmed cases were documented daily from January 21 to 4.5 (mean ± SD) days after the latest presumed viral exposure. Cases were confirmed 5.2 ± 4.2 days after symptom onset. The duration of hospital stay of the Tianjin cases is unknown as the discharge date of each case was not available. In both datasets, daily counts decline over time, which is likely a combination of delays to symptom onset and between symptom onset and reporting, combined with the effects of strong social distancing and contact tracing.', 'cite_spans': [], 'ref_spans': [], 'section': 'Descriptive Analyses'}, {'text': '. CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 3  4  5  6  7  8  9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41  42  43  44  45  46  47  48  49  50  51  52  53  54  55  56  57  58  59  60  61  62  63  64  65  66  67  68  69  70  71  72  73  74  75  76  77  78  79  80  81  82  83  84  85  86  87  88  89  90  91  92 Incidence curve with probable source of infection (c) Disease progression timeline, including dates at which each case is unexposed, exposed, symptomatic, hospitalized, and discharged. Not all cases go through each status as a result of missing dates for some cases.', 'cite_spans': [], 'ref_spans': [{'start': 154, 'end': 505, 'text': '3  4  5  6  7  8  9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41  42  43  44  45  46  47  48  49  50  51  52  53  54  55  56  57  58  59  60  61  62  63  64  65  66  67  68  69  70  71  72  73  74  75  76  77  78  79  80  81  82  83  84  85  86  87  88  89  90  91  92', 'ref_id': 'TABREF1'}], 'section': '7'}, {'text': '8 . CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 3  4  5  6  7  8  9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41  42  43  44  45  46  47  48  49  50  51  52  53  54  55  56  57  58  59  60  61  62  63  64  65  66  67  68  69  70  71  72  73  74  75  76  77  78  79  80  81  82  83  84  85  86  87  88  89  90  91  92  93  94  95  96  97  98  99  100  101  102  103  104  105  106  107  108  109  110  111  112  113  114  115  116  117  118  119  120  121  122  123  124  125  126  127  128  129  130  131  132  133  134 . CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [{'start': 156, 'end': 710, 'text': '3  4  5  6  7  8  9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41  42  43  44  45  46  47  48  49  50  51  52  53  54  55  56  57  58  59  60  61  62  63  64  65  66  67  68  69  70  71  72  73  74  75  76  77  78  79  80  81  82  83  84  85  86  87  88  89  90  91  92  93  94  95  96  97  98  99  100  101  102  103  104  105  106  107  108  109  110  111  112  113  114  115  116  117  118  119  120  121  122  123  124  125  126  127  128  129  130  131  132  133  134', 'ref_id': 'TABREF1'}], 'section': '7'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.03.20029983 doi: medRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': '7'}, {'text': 'In the Singapore dataset, we find that the median incubation period is 6.55 days with the Weibull 8.91, 11 .56). The mean is 9 (7.92, 10.2) days. We also fitted gamma and log normal distributions; see Tables 1 and S1. These are consistent with previous estimates.', 'cite_spans': [], 'ref_spans': [{'start': 98, 'end': 106, 'text': '8.91, 11', 'ref_id': 'FIGREF0'}], 'section': 'Incubation period'}, {'text': 'However, in Singapore they based on a combination of cases for whom last possible exposure is given by travel, and later cases (for whom the presumed infector was used). In Tianjin, social distancing measures took place. We find that the estimated incubation period is different, particularly in Tianjin, for cases with symptom onset prior to January 31. The estimated median incubation period for pre-Jan 31 cases in Tianjin is 6.9 days; the q = (0.025, 0.975) quantiles are (2, 12.7) days. In contrast, post-Jan 31 the median is 12.4 days with q = (0.025, 0.975) quantiles (5.4, 19) days. The means are 7 (6.1, 8.0) days for early cases and 12.4 (10.8, 14.2) days for later cases.', 'cite_spans': [{'start': 476, 'end': 485, 'text': '(2, 12.7)', 'ref_id': None}, {'start': 575, 'end': 584, 'text': '(5.4, 19)', 'ref_id': None}], 'ref_spans': [], 'section': 'Incubation period'}, {'text': 'Social distancing seems unlikely to change the natural course of infection, but these results might be explained if exposure occurred during group quarantine or otherwise later than the last time individuals thought they could have been exposed. Pre-symptomatic transmission would enable this.', 'cite_spans': [], 'ref_spans': [], 'section': 'Incubation period'}, {'text': \"In Singapore we find the same effect, though less pronounced. The estimated median incubation time is 5.46, with (0.025, 0.975) quantiles of (1.34, 11.1) days for early cases and 7.27 days (quantiles (1.31, 17. 3)) days for late-arising cases. The means are 5.71 (4.55, 7.06) days for early cases and 7.86 (6.57, 9.38) days for later cases. Fits of gamma and log-normal distributions are similar; see Table S2 . In Singapore, this difference could be explained by the fact that presumed Top panels show unstratified data (all cases with symptom onset given). Bottom panels show 'early' and 'late' cases, defined in Tianjin by those with symptom onset prior to January 31 vs later, and in Singapore, those with a specified last possible exposure (all prior to or on January 30).\", 'cite_spans': [], 'ref_spans': [{'start': 200, 'end': 210, 'text': '(1.31, 17.', 'ref_id': 'FIGREF0'}, {'start': 401, 'end': 409, 'text': 'Table S2', 'ref_id': 'TABREF4'}], 'section': 'Incubation period'}, {'text': '. CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '11'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.03.20029983 doi: medRxiv preprint Our serial intervals are notably shorter than our incubation period estimates, suggesting that there is pre-symptomatic transmission, with infection occurring on average 2.89 and 2.55 days before symptom onset of the infector (Tianjin, Singapore). Because the incubation period is different for early-and late-occurring cases in our data, on average transmission for early-occurring cases is 2.79 and 1.2 days before symptom onset (Tianjin, Singapore) and 8.2, 3.3 days before (Tianjin, Singapore) for late-occurring cases. The fact that serial intervals are shorter than incubation periods is robust in our sensitivity analysis (Table S3 ). These estimates are strengthened by the fact that we have estimated both incubation period and serial interval in the same population and by the fact that we obtain the same result in two distinct datasets. In both sets of estimates, samples of the incubation period minus serial interval are negative with probability 0.8 or higher (Tianjin) and 0.7 or higher (Singapore), suggesting that a substantial portion of transmission may occur before symptom onset (see Supplementary Information and Figure S2 ), consistent with the clinical observations reported by Rothe et al. [21] and Bai et al. [22] .', 'cite_spans': [{'start': 1360, 'end': 1364, 'text': '[21]', 'ref_id': 'BIBREF20'}, {'start': 1380, 'end': 1384, 'text': '[22]', 'ref_id': 'BIBREF21'}], 'ref_spans': [{'start': 773, 'end': 782, 'text': '(Table S3', 'ref_id': 'TABREF5'}, {'start': 1280, 'end': 1289, 'text': 'Figure S2', 'ref_id': 'FIGREF6'}], 'section': '11'}, {'text': 'Shorter serial intervals yield lower reproduction number estimates. For example, when the epidemic grows at a rate of 0.15 (doubling time of 6.6 days; [23] scenario 1), the reproduction number for Tianjin is R = 1.87 (1.65, 2.07) and for Singapore it is 1.97 (1.43, 2.51). In contrast, if the previous serial interval (7.5 days [23, 1] ) is used, the estimate is R = 3.05.', 'cite_spans': [{'start': 151, 'end': 155, 'text': '[23]', 'ref_id': 'BIBREF22'}, {'start': 328, 'end': 332, 'text': '[23,', 'ref_id': 'BIBREF22'}, {'start': 333, 'end': 335, 'text': '1]', 'ref_id': None}], 'ref_spans': [], 'section': 'Serial interval, pre-symptomatic transmission and R0'}, {'text': '. CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. ', 'cite_spans': [], 'ref_spans': [], 'section': '12'}, {'text': 'Here we use transmission clusters in two locations where cases have reported links, exposure and symptom onset times to estimate both the incubation period and serial interval of COVID-19.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'These are integral parameters for disease forecasting and for informing public health interventions.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'Serial intervals, together with R0, control the shape and distribution of the epidemic curve [24] .', 'cite_spans': [{'start': 93, 'end': 97, 'text': '[24]', 'ref_id': 'BIBREF23'}], 'ref_spans': [], 'section': 'Discussion'}, {'text': \"They influence the disease's incidence and prevalence, how quickly an epidemic grows, and how quickly intervention methods need to be implemented by public health officials to control the disease [24, 25] . In particular, the portion of transmission events that occur before symptom onset is a central quantity for infection control [25] .\", 'cite_spans': [{'start': 196, 'end': 200, 'text': '[24,', 'ref_id': 'BIBREF23'}, {'start': 201, 'end': 204, 'text': '25]', 'ref_id': 'BIBREF24'}, {'start': 333, 'end': 337, 'text': '[25]', 'ref_id': 'BIBREF24'}], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'In Singapore and Tianjin we estimated relatively short serial intervals. Of particular note, early estimates of R0 for COVID-19 used the SARS serial interval of 8.4 days [7, 8, 26] . Our serial interval findings from two populations mirror those of and Zhao et al [27] and Nishiura et al [13] , who estimated a serial interval of 4.4 and 4.0 days. Furthermore, we estimate the serial interval to be shorter than the incubation period in both clusters, which suggests pre-symptomatic transmission. This indicates that spread of SARS-CoV-2 is likely to be difficult to stop by isolation of detected cases alone. However, shorter serial intervals also lead to lower estimates of R0, and 13 . CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 170, 'end': 173, 'text': '[7,', 'ref_id': None}, {'start': 174, 'end': 176, 'text': '8,', 'ref_id': None}, {'start': 177, 'end': 180, 'text': '26]', 'ref_id': 'BIBREF25'}, {'start': 264, 'end': 268, 'text': '[27]', 'ref_id': 'BIBREF26'}, {'start': 288, 'end': 292, 'text': '[13]', 'ref_id': 'BIBREF12'}], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.03.20029983 doi: medRxiv preprint our serial intervals support R0 values close to 2; if correct this means that half of the transmissions need to be prevented to contain outbreaks.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'We stratified the incubation period analysis for Tianjin by time of symptom onset (pre-or post-Jan 31, 2020; motivated by quarantine/social distancing measures) and found that the apparent incubation period was longer for those with post-quarantine symptom onset. The reason for this is unclear, but one possible explanation is that there were (unknown, therefore unreported) exposures during the quarantine period. If people are quarantined in groups of (presumed) uninfected cases, pre-symptomatic transmission in quarantine would result in true exposure times that are more recent than reported last possible exposure times.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'There are several limitations to this work. First, the times of exposure and the presumed infectors are uncertain, and the incubation period is variable. We have not incorporated uncertainty in the dates of symptom onset. We have used the mixture model approach for serial intervals to avoid assuming that the presumed infector is always the true infector, but the mixture does not capture all possible transmission configurations. Our R0 estimates are very simple and could be refined with more sophisticated modelling in combination with case count data. We have not adjusted for truncation (eg shorter serial intervals are likely to be observed first). However, the serial interval estimates are consistent between the two datasets, are robust to the parameter choices, and are consistently shorter than the estimated incubation times.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'In conclusion, in both the Singapore and Tianjin COVID-19 clusters we identified both the incubation period and the serial interval. Our results suggest that there is substantial pre-symptomatic transmission, with the serial interval shorter than incubation period by 2-4 days. We find differences in estimated incubation period between early and later cases; this may be due to presymptomatic transmission or differences in reporting and/or in perceived exposure as the outbreak progressed, in the context of social distancing measures. Finally, our shorter serial intervals lead to an estimate of the basic reproduction number of approximately 2 in both datasets, suggesting that stopping half the transmission events may be sufficient to control outbreaks.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': '14 . CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.03.20029983 doi: medRxiv preprint ', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': '. CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. 1 . CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '19'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.03.20029983 doi: medRxiv preprint 6 Table S1   7   7 Table S2  8   8 Table S3  9 9 Table S4  10 1 Details of the Singapore and Tianjin datasets', 'cite_spans': [], 'ref_spans': [{'start': 144, 'end': 190, 'text': '6 Table S1   7   7 Table S2  8   8 Table S3  9', 'ref_id': 'TABREF1'}, {'start': 193, 'end': 205, 'text': 'Table S4  10', 'ref_id': 'TABREF1'}], 'section': '19'}, {'text': 'In the Singapore dataset: \"related cases\" are direct known contacts between cases; \"cluster links\"', 'cite_spans': [], 'ref_spans': [], 'section': '19'}, {'text': 'are cases that are linked together through an identified cluster event; \"presumed infected date\"', 'cite_spans': [], 'ref_spans': [], 'section': '19'}, {'text': 'and \"presumed reason\" are the earliest known date and the reason that each case was known to likely be infected; \"last poss exposure\" and \"symptom presumed infector\" are sub-classifications of \"presumed infected date,\" representing either the last date that each case could have been infected -the date of arrival in Singapore for travellers from Wuhan -or the date that each case was likely infected during a local transmission event in Singapore, respectively; \"cluster\" is the Ministry of Health Singapore\\'s classification of cases into transmission cluster events.', 'cite_spans': [], 'ref_spans': [], 'section': '19'}, {'text': 'In the Tianjin cases summary spreadsheet, the main columns are: gender, age, symptom onset, symptom type, confirmation date, severity and death date [1] ; detailed information from daily reports for the first 80 patients provided travel or exposure history and contact information, from which we obtained exposure windows (start source, end source). For backup and to complete missing information for later cases we also referred to Jinyun News, Tianjin official local media: [2] who used Baodi local government reports [3] . They reported detailed activity for those confirmed cases when their corresponding epidemiological history investigation was finished.', 'cite_spans': [{'start': 149, 'end': 152, 'text': '[1]', 'ref_id': None}, {'start': 476, 'end': 479, 'text': '[2]', 'ref_id': None}, {'start': 520, 'end': 523, 'text': '[3]', 'ref_id': 'BIBREF2'}], 'ref_spans': [], 'section': '19'}, {'text': '2 . CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '19'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.03.20029983 doi: medRxiv preprint 2 Statistical methods', 'cite_spans': [], 'ref_spans': [], 'section': '19'}, {'text': 'We explored several distributions for the incubation period: Weibull, gamma and log normal. As shown in Figure 3 , once fit the resulting distributions all provide very similar results. Table S1 summarizes the parameter estimates for the gamma and log normal distributions. Table S2 gives the parameters for the incubation period for early-and late-occurring cases in both datasets.', 'cite_spans': [], 'ref_spans': [{'start': 104, 'end': 112, 'text': 'Figure 3', 'ref_id': 'FIGREF7'}, {'start': 186, 'end': 194, 'text': 'Table S1', 'ref_id': 'TABREF1'}, {'start': 274, 'end': 282, 'text': 'Table S2', 'ref_id': 'TABREF4'}], 'section': 'Incubation period'}, {'text': 'We used bootstrapping to explore the range for the point estimates of µ and σ from the mixture model. Figure S1 shows the results. The mean of the bootstrapped mean estimates is 4.26 ± 1.02 for Tianjin and 4.57 ± 0.95 days in Singapore. Bootstrap values are consistent with a serial interval that is considerably shorter than the incubation periods in both datasets. Table S3 shows the sensitivity analysis; we varied the the number of cases per cluster to include in the ICC interval data and we explored sorting the cases in the clusters according to the time of last exposure (ie the putative index status assigned to the individual with the earliest end to their exposure window, instead of the first symptomatic individual).', 'cite_spans': [], 'ref_spans': [{'start': 102, 'end': 111, 'text': 'Figure S1', 'ref_id': 'FIGREF0'}, {'start': 367, 'end': 375, 'text': 'Table S3', 'ref_id': 'TABREF5'}], 'section': 'Serial interval'}, {'text': 'We sampled the estimated serial interval and incubation period distributions to estimate the probability that the incubation period minus the serial interval is negative. This could indicate the portion of transmission that occurs before symptom onset, but it makes the strong assumption that incubation period and onset of symptoms occur independently and it does not take the growth curve of 3 . CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Pre-symptomatic transmission'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.03.20029983 doi: medRxiv preprint the outbreak, or censoring or truncation, into account. We find that the portion of negative samples is 0.67 for early cases and 0.73 for unstratified (overall incubation period) cases in Singapore, and 0.82 (early) and 0.87 (all) cases in Tianjin. Early cases have shorter estimated incubation periods so the fractions are lower. Late cases have higher portions (0.78, 0.98) but this is due to the long incubation estimate and fixed serial interval distribution; it may be influenced by censoring and truncation. Figure S2 illustrates the distributions.', 'cite_spans': [], 'ref_spans': [{'start': 658, 'end': 667, 'text': 'Figure S2', 'ref_id': 'FIGREF6'}], 'section': 'Pre-symptomatic transmission'}, {'text': 'Estimates of incubation period and serial interval from other studies are shown in Table S4 . Of note, the majority of studies do not estimate both incubation period and serial interval in the same population.', 'cite_spans': [], 'ref_spans': [{'start': 83, 'end': 91, 'text': 'Table S4', 'ref_id': 'TABREF6'}], 'section': 'Additional published estimations'}, {'text': '4 . CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Additional published estimations'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.03.20029983 doi: medRxiv preprint 4 Figure S1 Figure S2 : Samples of the incubation period distribution minus the serial interval distribution, assuming independence and without considering the (unknown) growth curve of the outbreaks or the extent of censoring.', 'cite_spans': [], 'ref_spans': [{'start': 156, 'end': 165, 'text': 'Figure S2', 'ref_id': 'FIGREF6'}], 'section': 'Additional published estimations'}, {'text': '6 . CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Additional published estimations'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.03.20029983 doi: medRxiv preprint Table S1 : Incubation period estimates using a gamma distribution. 95% confidence intervals for the shape and scale (log mean and sd for log normal) parameters are shown in brackets. 8 . CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [{'start': 144, 'end': 152, 'text': 'Table S1', 'ref_id': 'TABREF1'}], 'section': 'Additional published estimations'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.03.20029983 doi: medRxiv preprint 8 Table S3 . CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Additional published estimations'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.03.20029983 doi: medRxiv preprint Infector-infectee pairs* 28 pairs -4.0 (95CI 3.1-4.9) Nishiura et al 2020 [9] *Note: included 3 infector-infectee pairs from this Singapore cluster. Remainder from Vietnam (4), South Korea (7), Germany (4), Taiwan (1) and China (9) 10 . CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 218, 'end': 221, 'text': '[9]', 'ref_id': None}], 'ref_spans': [], 'section': 'Additional published estimations'}, {'text': 'is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.03.03.20029983 doi: medRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': 'Additional published estimations'}], 'paragraphs': [{'text': \"They influence the disease's incidence and prevalence, how quickly an epidemic grows, and how quickly intervention methods need to be implemented by public health officials to control the disease [24, 25] . In particular, the portion of transmission events that occur before symptom onset is a central quantity for infection control [25] .\", 'id': '00033'}], 'title': 'Transmission interval estimates suggest pre-symptomatic spread of COVID-19', 'sha': '3028628066ec2401f3981f4e70c5b1acd4cef573', 'doi': '10.1101/2020.03.03.20029983', 'score': 8.71399974822998}, {'rank': 288, 'abstract': '', 'body_text': [{'text': 'In the current pandemic crisis, remember to ‘Follow WOMEN’ and ‘Avoid touching MEN’. Follow ‘WOMEN’, i.e. Wash your hands frequently with soap and water; Obey social distancing; Mask is a must if you leave your house; Eat a healthy diet rich in vegetables and fruits; N – No going out if you are not feeling well. Avoid touching ‘MEN’ i.e. Mouth, Eyes, and Nose …. because the COVID 19 virus can enter your body via any of these gateways. This is the advice from the Departments of Health worldwide. Of the three entry points, the least considered, has been the eyes.', 'cite_spans': [], 'section': '', 'ref_spans': []}, {'text': 'Many people wear face masks, but far fewer consider the need for eye protection. Although the eye is probably a less important gateway for the virus, in the last few days, evidence has emerged of the likelihood of transmission via the conjunctival epithelium [1]. So, the question arises “What does this mean for optometrists and ophthalmologists and their patients?”', 'cite_spans': [{'start': 260, 'end': 261, 'mention': '1', 'ref_id': 'BIBREF0'}], 'section': '', 'ref_spans': []}, {'text': 'Wearing glasses or a visor may reduce risk of contamination, but it does not protect against COVID 19 infections [2]. This is a similar situation to wearing a regular surgical or other face covering in that there is an imperfect seal allowing unfiltered air to reach the nose, mouth, and eyes – it helps, but it does not prevent [2,3]. This is why frontline health care workers need to use a higher standard of face mask, the N95, which needs to be individually fitted, together with well-fitted goggles.', 'cite_spans': [{'start': 114, 'end': 115, 'mention': '2', 'ref_id': 'BIBREF1'}, {'start': 330, 'end': 331, 'mention': '2', 'ref_id': 'BIBREF1'}, {'start': 332, 'end': 333, 'mention': '3', 'ref_id': 'BIBREF2'}], 'section': '', 'ref_spans': []}, {'text': 'It is currently unknown how the organism can infect the eye, whether it is direct transmission by droplets or aerosol from infected patients or spread from the nasal tissues via the nasolacrimal duct [4]. The long and variable incubation period of this viral infection makes it difficult to determine the exact time and/or method of spread. Many patients with COVID 19 are asymptomatic, increasing the challenge of pinpointing the actual time or person responsible for passing on the infection. Regardless of the route, the presence of the virus in the eye makes this infection of particular importance to practitioners and their clients.', 'cite_spans': [{'start': 201, 'end': 202, 'mention': '4', 'ref_id': 'BIBREF3'}], 'section': '', 'ref_spans': []}, {'text': 'Optometrists, whilst not considered frontline health care workers for COVID 19, may nonetheless, be exposed to asymptomatic patients or their close contacts. Therefore, they need to take precautions to protect themselves and others in their practices. Depending on local regulations, it is a sensible precaution to check the temperature of anyone entering the practice, including staff, and for all present to wear a surgical mask, and use hand sanitizers. Ideally, this should be supplemented with a face visor, as certain diagnostic procedures do require close contact with patients. In addition, ophthalmic instruments should be shielded as far as possible to prevent contamination and wiped down with disinfectant or alcohol after each patient. Clients may try on several frames when choosing new spectacles, so, although it is not normal practice to clean these after every customer, at this extraordinary time, all items that come into contact with potentially infected persons must be wiped with alcohol or equivalently effective method. This means that clients trying on frames must be supervised to ensure that frames tried by the clients can be identified for cleaning.', 'cite_spans': [], 'section': '', 'ref_spans': []}, {'text': 'Although the optometry practice is not the correct setting, some patients may present with ocular manifestations, such as conjunctivitis [5], rather than attending the hospital or a medical practitioner. An ocular condition may be the first symptom of COVID 19 [6] and it is considered that one of the first practitioners to draw attention to this new infection was Dr Li Wenliang, an ophthalmologist [5]. It is thought that he contracted the virus from a glaucoma patient. In China, Guan et al. [7] reported that less than 1% of COVID 19 patients had conjunctival congestion, but subsequent reports have observed higher percentages [6,8,9], although not all conjunctivitis present in COVID 19 patients was attributable to the virus [6].', 'cite_spans': [{'start': 138, 'end': 139, 'mention': '5', 'ref_id': 'BIBREF4'}, {'start': 262, 'end': 263, 'mention': '6', 'ref_id': 'BIBREF5'}, {'start': 402, 'end': 403, 'mention': '5', 'ref_id': 'BIBREF4'}, {'start': 497, 'end': 498, 'mention': '7', 'ref_id': 'BIBREF6'}, {'start': 634, 'end': 635, 'mention': '6', 'ref_id': 'BIBREF5'}, {'start': 636, 'end': 637, 'mention': '8', 'ref_id': 'BIBREF7'}, {'start': 638, 'end': 639, 'mention': '9', 'ref_id': 'BIBREF8'}, {'start': 734, 'end': 735, 'mention': '6', 'ref_id': 'BIBREF5'}], 'section': '', 'ref_spans': []}, {'text': 'Regarding the use of contact lenses during this period of worldwide infection, mixed messages have emerged from various health sources, making it difficult for practitioners to provide absolute guidance to their patients. The most frequent question regards the safety of wearing contact lenses for vision correction during the crisis. Contact lens wear itself is safe if there is strict compliance with care and wear routines [10]. This has not been changed by the pandemic. There is no definitive evidence so far of direct transmission via contact lenses. However, it is well recognized that contact lens patients are frequently not compliant [[11], [12], [13]]. In addition, contact lens wearers may experience minor discomfort or irritation more frequently than spectacle wearers [14], and this in turn increases the chance of the natural response to touch or rub eyes [15]. This is especially true for patients who experience some dryness when using contact lenses. For these patients in particular, it may be prudent to switch to spectacles for this period. Alternatively, those who are more dependent on their contact lenses may consider wearing plano spectacles or sunglasses, in addition to their lenses, in public. Nevertheless, as spectacle wearers may also rub their eyes after removing their glasses, they also need to be reminded to avoid doing so.', 'cite_spans': [{'start': 427, 'end': 429, 'mention': '10', 'ref_id': 'BIBREF9'}, {'start': 645, 'end': 649, 'mention': '[11]', 'ref_id': 'BIBREF10'}, {'start': 651, 'end': 655, 'mention': '[12]', 'ref_id': 'BIBREF11'}, {'start': 657, 'end': 661, 'mention': '[13]', 'ref_id': 'BIBREF12'}, {'start': 784, 'end': 786, 'mention': '14', 'ref_id': 'BIBREF13'}, {'start': 873, 'end': 875, 'mention': '15', 'ref_id': 'BIBREF14'}], 'section': '', 'ref_spans': []}, {'text': 'During this pandemic, many countries have instituted formal lockdowns, or working from home. Whilst this reduces the risk of contact with others, it may encourage another form of non-compliance – napping while wearing contact lenses. This can lead to hypoxia, which increases the risk of infection [16]. Once again, practitioners should remind their patients of the importance of not sleeping with their lenses on, unless prescribed otherwise.', 'cite_spans': [{'start': 299, 'end': 301, 'mention': '16', 'ref_id': 'BIBREF15'}], 'section': '', 'ref_spans': []}, {'text': 'Another issue regarding lockdown is the lack of accessibility to aftercare consultations. Some practitioners are sending out refill lenses on request, which allows them to remind patients to attend the clinic after lockdown is lifted. For other patients, a general reminder sent by social media helps to maintain rapport with the practitioners and may prevent patients from regularly ordering lenses online even after the lockdown.', 'cite_spans': [], 'section': '', 'ref_spans': []}, {'text': 'Practitioners of course, are responsible for impressing upon their patients the importance of good hygiene, but some additional precautions are needed during this time of pandemic. Spectacle wearers should be reminded to clean their glasses on returning home, either using alcohol wipes or with soap and water. Clients should also be reminded that other frequently contacted items, especially mobile phones, should also be regularly cleaned to prevent cross transmission. Any patient who does contract COVID 19 should be advised to discard all previously worn contact lenses and used accessories (e.g. solutions, lens storage case) after recovery.', 'cite_spans': [], 'section': '', 'ref_spans': []}, {'text': 'Since the beginning of the pandemic, it was believed that children were relatively safe from COVID 19. However, according to a recent study in the US, the risk has been underemphasized [17], as several cases of severe, even fatal, disease have been reported. It appears to be relayed to an immense over response of the immune system to the infection, which is very difficult to treat. The authors consider a surge of severe pediatric cases of COVID 19 would be disastrous, in view of inadequate pediatric hospital care resources. This observation is likely to be the case in many other countries as well. At this time, it is essential to increase awareness of children wearing contact lenses for myopia control. These include those wearing specialized soft contact lenses in the daytime and overnight orthokeratology. It is more difficult to impress on children the importance of not touching ‘MEN’. Therefore, the case for temporary discontinuation of daytime lens wear at this time may be stronger, but the final decision must be made by the practitioner after careful consideration of the likelihood of good compliance and amount of time spent out of the home. In many places, schools are closed and time outdoors is strictly limited, reducing opportunities for contamination and infection. Once again, when outdoors, the risk can be ameliorated by the use of plano glasses or sunglasses with the contact lenses.', 'cite_spans': [{'start': 186, 'end': 188, 'mention': '17', 'ref_id': 'BIBREF16'}], 'section': '', 'ref_spans': []}, {'text': 'With respect to orthokeratology, lenses are only worn at home and insertion/removal is often performed with the assistance of parents. Children sleep with the lenses in place, thereby reducing the chance of touching their eyes; after all, no one wears a face mask to sleep! One problem of school closures is that children may be allowed to sleep in longer. Practitioners need to caution parents that extended hours of sleep with the orthokeratology lenses may lead to development of corneal oedema. This condition has been observed in a few children undergoing orthokeratology treatment just recently in Hong Kong. As patients may be unable to visit clinics or practices due to lockdown, it may be advisable for practitioners to contact the parents of all of their orthokeratology patients to alert them about this problem. After all, as Boris Johnson said, “We must all be alert!” (10 May, 2020).', 'cite_spans': [], 'section': '', 'ref_spans': []}], 'paragraphs': [{'text': 'It is currently unknown how the organism can infect the eye, whether it is direct transmission by droplets or aerosol from infected patients or spread from the nasal tissues via the nasolacrimal duct [4]. The long and variable incubation period of this viral infection makes it difficult to determine the exact time and/or method of spread. Many patients with COVID 19 are asymptomatic, increasing the challenge of pinpointing the actual time or person responsible for passing on the infection. Regardless of the route, the presence of the virus in the eye makes this infection of particular importance to practitioners and their clients.', 'id': '00004'}], 'title': 'COVID 19—An eye on the virus', 'sha': '2ae6ae5d1645dda13544fb2b2671cdfc49f6cc18', 'doi': '10.1016/j.clae.2020.05.011', 'score': 8.70359992980957}, {'rank': 289, 'abstract': 'The world is currently facing the worst pandemic in a century and we were caught unprepared. COVID-19 has proven highly contagious and with severe consequences that are still unfolding. As of 16 April 2020, there were over 2 million confirmed cases and over 136,000 related deaths reported worldwide. Over 1 million of those confirmed cases were in the preceding 14 days, with the USA accounting for nearly half of those. Furthermore, the International Monetary Fund (IMF) is now warning that the world is about to suffer the worst economic recession since the Great Depression in the 1920s.', 'body_text': [{'text': 'Whilst the nation state and Westphalian sovereignty remain the starting points when considering regulatory powers within a territory and engagement on the international sphere, these are not set in stone and considerable variations arise. Thus, multilevel governance and regulation theories acknowledge the \"reallocation of authority upwards, downwards and sideways from central states\". 5 This begs the question of how to determine where the core powers ought to rest.', 'cite_spans': [{'start': 388, 'end': 389, 'text': '5', 'ref_id': None}], 'ref_spans': [], 'section': 'I. SUBSIDIARITY-BASED MULTILEVEL GOVERNANCE 4'}, {'text': 'One potential mechanism is by applying subsidiaritya broad concept with roots in ideas of democracy, Catholicism and economics or effectiveness. 6 It focuses on \"the proper geographic distribution of power\". 7 This broadly argues that powers ought to rest at the lowest level possible (due to democracy), unless it would be more effective to allocate them at a higher level. 8 There are three key steps in order to apply subsidiarity, with a range of considerations within them. 9 The first relates to the interest(s) in question. It is necessary to identify them and consider how significant they are to the various levels or constituents, to what extent homogeneity or heterogeneity exists (eg regarding objectives, balance with other interests and broad approaches) and the capacity of other levels to accommodate the heterogeneity. In the context of public health, this normally includes considering issues such as whether there is broad consensus on acting as a welfare state or not and the balance with other societal issues where resources are insufficient, as well as opinions on related issues such as the approach to the economy and markets. Whilst each state shares values and goals of strong public health and also a resilient economy, with both closely intertwined in the long term, there is clearly no broad global consensus on the balance between values and approaches to them.', 'cite_spans': [{'start': 145, 'end': 146, 'text': '6', 'ref_id': None}, {'start': 375, 'end': 376, 'text': '8', 'ref_id': None}, {'start': 479, 'end': 480, 'text': '9', 'ref_id': None}], 'ref_spans': [], 'section': 'I. SUBSIDIARITY-BASED MULTILEVEL GOVERNANCE 4'}, {'text': 'The second entails considering the question of effectiveness or efficiency. This includes identifying where the relevant expertise and/or knowledge lie, including whether there is access to resources at a different level or not. In many contexts, this may include local and experiential knowledge. Where scientific or other expertise is central to decision-making, centralisation of both the research and the decisionmaking may be efficient, as lower levels may not have the necessary resources and \"gaps\" could arise. 10 However, in cases of uncertainty, the value of full centralisation may be more questionable. It also includes identifying the potential for externalities, whereby one body\\'s decisions can impact on external bodies and vice versa, and the potential to internalise those externalities or not by centralising. Where there are significant negative externalities, this would support centralisation.', 'cite_spans': [], 'ref_spans': [], 'section': 'I. SUBSIDIARITY-BASED MULTILEVEL GOVERNANCE 4'}, {'text': 'Then finally a balancing act must be undertakenmaking for a very complex calculation, where the division and (re-)allocation of different powers across several levels may be appropriate.', 'cite_spans': [], 'ref_spans': [], 'section': 'I. SUBSIDIARITY-BASED MULTILEVEL GOVERNANCE 4'}, {'text': 'But how does this then apply to COVID-19 and the surrounding decision-making?', 'cite_spans': [], 'ref_spans': [], 'section': 'I. SUBSIDIARITY-BASED MULTILEVEL GOVERNANCE 4'}, {'text': 'Whilst still part of public health, COVID-19 goes beyond the norm. Firstly, COVID-19 is highly contagious and spreads swiftly and easily. 11 This is accentuated by potentially long incubation periods and the potential for an individual to be contagious even though they are asymptomatic. The virus also remains viable on surfaces such as paper and plastic for some time, facilitating the spread further. Without control measures, the spread has proven to be exponentialwith us all now becoming too familiar with steeply increasing graphs and the term \"R0\". 12 Nor is this limited to one area or country. Due to a combination of the above factors, in conjunction with the widespread travel of individuals and products, the virus has spread globally and become a pandemic. Furthermore, COVID-19 has devastating effects. We have no vaccines, no tailored effective treatments and no built-in resistance or partial immunity to COVID-19. Whilst most will be asymptomatic or have mild symptoms, it can be fatal, with the elderly, ill and those with \"underlying conditions\" 13 being particularly vulnerable, and it also can cause severe impairments in the short and long termalthough the extent of these impairments is still to be fully understood. 14 This has further knock-on effects, as some public health systems are overburdened to the point of collapseillnesses or injuries that should be manageable cannot be properly treated, appointments and treatments are postponed and discussions arise about triage for both COVID-19 and non-COVID-19 patients. 15 The combination of the highly contagious nature and severity of impacts is worrying. It also provides for clusters of cases affecting specific groups and industries, including both healthcare workers (exacerbating the public health impacts once more) and other 11 Eg RM Anderson, H Heesterbeek and D Klinkenberg, \"How Will Country-Based Mitigation Measures Influence the Course of the COVID-19 Epidemic?\" (2020) 395(10228) The Lancet 931. 12 Reproduction number: the mean number of new cases an infected person will create. 13 A vague term that presumably is meant to reassure us, as the majority who die have \"underlying conditions\", but without clarity to date as to what this involvescould this be a broken leg?and also deprecating the value of those who have died who are elderly, ill or with such conditions. 14 \"essential workers\" in global supply chains, 16 as well as the economy reflected in the IMF\\'s report. All of these feed into each other and risk circular short-and long-term effectswhere the virus is present and elsewhere. Consequently, there is the potential for severe externalities by the virus itself if left unmanagedbut this does not mean that the governance of it would lead necessarily to externalities also. But is effective governance possible? A final major concern for the moment, which impacts upon every nation as they react to this, is the underlying uncertainties. 17 We do not understand the disease fully, gathering data and hopefully understanding as we go. That uncertainty relates to when an individual is contagious, the morbidity levels, how to test for the virus, how to test for antibodies, how long a person might be immune for and how to treat or prevent it in future. Whilst major strides are being made with each of these aspects, uncertainty still remainsin particular regarding antibodies, immunity and vaccines. Furthermore, testing (and subsequent contact tracing) for the disease is sometimes flawed and frequently limitedhampering understanding and making for patchy data to underpin decision-making. Consequently, any decision-making is based on the precautionary principle (whether express or otherwise) and it becomes more challenging to identify clear pathways to address the pandemic effectively that also minimise countervailing riskssomething that may in itself justify national rather than international approaches, even whilst benefiting from the centralisation and sharing of scientific data.', 'cite_spans': [{'start': 1548, 'end': 1550, 'text': '15', 'ref_id': None}, {'start': 2365, 'end': 2367, 'text': '14', 'ref_id': None}, {'start': 2949, 'end': 2951, 'text': '17', 'ref_id': None}], 'ref_spans': [], 'section': 'II. COVID-19'}, {'text': 'In light of the goal of public health, the ideal options would be widespread immunity of the entire population and/or complete eradication of the virus, with effective treatment as a support mechanism. However, besides the surrounding uncertainty, nation states have different capacities, and any response has countervailing risks. Consequently, nation states have chosen different approaches.', 'cite_spans': [], 'ref_spans': [], 'section': 'III. PANDEMICS POLICIES?'}, {'text': 'The core responses include herd immunity, 18 \"flattening the curve\" 19 and complete eradication. 20', 'cite_spans': [], 'ref_spans': [], 'section': 'III. PANDEMICS POLICIES?'}, {'text': '(1) Herd immunity is where a sufficient proportion of the population have immunity to a disease so that no further members of the population are likely to be affected in the future. For diseases such as polio, this is largely achieved through vaccination, 16 L Petetin, \"The COVID- but as there is no current vaccination for COVID-19, to achieve this now would entail initially enabling the spread of a disease through a population (as with chicken pox parties). (2) \"Flattening the curve\" involves trying to control the spread of the disease so that the increase in cases, especially those requiring medical treatment, hospitalisation and specifically the use of ventilators, is slowed (eg until R0 = 1) and remains within the capacity of the public health system. This is about buying time, enabling capacity-building, development of effective treatments and hopefully a vaccine. In order to achieve this, social distancing is essential and contact tracing is a key mechanism.', 'cite_spans': [{'start': 256, 'end': 258, 'text': '16', 'ref_id': None}], 'ref_spans': [], 'section': 'III. PANDEMICS POLICIES?'}, {'text': '(3) Eradication could be achieved through complete immunity of the population, but equally can be achieved through preventing the spread of the disease (R0 < 1 and eventually R0 = 0). Identification and quarantining of every single actual or potential case in the population, as well as cleansing all potential contaminated areas, are essential.', 'cite_spans': [], 'ref_spans': [], 'section': 'III. PANDEMICS POLICIES?'}, {'text': 'It should be noted that these can overlap and also may include \"cocooning\" vulnerable individuals, testing for the disease and testing subsequently for antibodies. Each of these approaches in theory could be viable and acceptablewe use these approaches when addressing issues such as colds, flus or chicken pox. However, there are numerous issues that challenge their effectiveness and appropriateness in the context of COVID-19. Beyond issues of the rate of transmission, the potential for the virus to remain viable on surfaces or to transmit to and from animals, the potential for asymptomatic individuals to transmit the virus and the virus already being in situ within the population, there are a few other key concerns.', 'cite_spans': [], 'ref_spans': [], 'section': 'III. PANDEMICS POLICIES?'}, {'text': 'Firstly, the impacts of the virus are not negligible. Herd immunity is desirable, but the current mechanism to achieve it is dependent on the spread of the disease, with resulting deaths and severe impairments for members of the population. Cocooning may protect the vulnerable, but we do not know who precisely is vulnerable, and from the daily death rates in hotspots we are seeing that either cocooning is not working or the vulnerable are a much larger proportion of the population than we thought. Those pushing for flattening the curve may try to claim a moral high ground, but individuals are still badly affected by COVID-19, and for both there are knock-on effects on other treatments and the economy. On the other hand, social distancing and lockdowns can impact negatively on both the economy and mental and physical health.', 'cite_spans': [], 'ref_spans': [], 'section': 'III. PANDEMICS POLICIES?'}, {'text': \"Secondly, there is no certainty that immunity following recovery will arise or last long enough to prevent continued outbreaks. 21 Due to the ongoing uncertainty, we cannot conclude that an individual cannot contract the virus a second time. If they do, it might not be as severe, but their bodies may already be physically damaged and it could lead to COVID-19's continued circulationnecessitating continued cocooning of the vulnerable. This is relevant to all three approaches, as it undermines the herd immunity approach entirely, but also may necessitate more extended and more restrictive measures for flattening the curve and eradicationfor instance, public healthcare workers treating COVID-19 patients could repeatedly contract and transmit the virus.\", 'cite_spans': [], 'ref_spans': [], 'section': 'III. PANDEMICS POLICIES?'}, {'text': 'Thirdly, we do not know when an effective vaccine will be developed and made available in sufficient quantities. The same applies to more effective treatments. It is hoped that these will be developed rapidly, but this is by no means guaranteed, and testing will be required to ensure their safety. It might be that the only (short-or long-term) immunity possible is through contracting the disease and recovering. States may not be able to maintain social distancing or lockdowns due to practical constraints, such as a lack of resources, including finances.', 'cite_spans': [], 'ref_spans': [], 'section': 'III. PANDEMICS POLICIES?'}, {'text': 'Clearly, we need further information in order to know what approaches might be the most effective and acceptable in the long term. Returning to subsidiarity, though, perhaps this indicates that a decentralised approach to policy-making here is appropriate? After all, it relates to public health (varied balance of values worldwide), with little scientific certainty and where culture may play an important role.', 'cite_spans': [], 'ref_spans': [], 'section': 'III. PANDEMICS POLICIES?'}, {'text': 'However, even if effective, the variations raise a further significant issue that challenges individual national approaches: as we shall see, each nation state has the potential to impact negatively on the others and likewise to be impacted upon.', 'cite_spans': [], 'ref_spans': [], 'section': 'III. PANDEMICS POLICIES?'}, {'text': \"If one returns to the three core approaches and presumes that they are each effective in principleherd immunity will arise, case numbers and severity can be controlled and/or the disease can be eradicated within the populationsimply reflect on the global nature of our society and the continued shifting nature of our populations. If we change the proportion in a population who have immunity, or introduce new sick individuals who need treatment, or simply introduce new carriers to the population, each approach will be set back and have to start again. If vulnerable individuals are no longer cocooning, they risk catching the disease and being severely impacted. Consequently, one state's approach(es) will impact on others. 22 The alternative? Continued controls on entry into each nation state and restrictions on travelwith entry potentially limited in future to those who have certificates of immunity (akin to yellow fever certificates) or who are willing to be quarantined for weeks. One reason why New Zealand 23 has been able to take the eradication approach is that they started testing prior to any cases presenting symptoms and quarantined anyone entering the state. However, crucially, they are also an island distant from most other countries and entry other than by plane or ship is highly challengingand even so, a single case in the future could lead to a new outbreak before it can be controlled, and bear in mind, for instance, the potential for unreliable tests and false negatives. Contrast this with the EU's normally porous borders or the federal states within the USAentry can be limited via airports and ports, but the extent of the land borders makes absolute control infeasible. Furthermore, the photos of Romanians flying to Germany and the UK to pick fruit and vegetables in the midst of a pandemic highlight the existing supply chains' dependence on migrant workers. 24 Similarly, the importance of a supranational or global approach, or at least considerable cooperation and collaboration, is seen in the flurry of activity and competition for resources. Most nation states are under-resourced generally, but especially are not prepared for pandemics and do not have adequate resources in situ, in part due to the economic costs and the lack of need on an ordinary basis. 25 Nor are states typically self-sufficient in the production of protective, testing or treatment equipment or components, including ventilators or chemical reagentswe are dependent on both domestic and imported materials. That dependency is now coming to the fore, with shipments to one country being intercepted by other countries, bidding wars between States in the USA 26 (with President Trump refusing to intervene when asked by States) 27 and threats to block exports to other countries including Canada. 28 Yet the stories are not all negativecountries are fast-tracking the production of necessary supplies and distributing these worldwide, including as donations, whilst others are taking in patients for treatment from overburdened countries or sending their own healthcare workers to assist. 29 The pandemic merely highlights what we already know: the issue is not merely the limited resources, but also the control of the distribution of resources and access to them.\", 'cite_spans': [{'start': 729, 'end': 731, 'text': '22', 'ref_id': None}, {'start': 1900, 'end': 1902, 'text': '24', 'ref_id': None}, {'start': 2306, 'end': 2308, 'text': '25', 'ref_id': None}, {'start': 3109, 'end': 3111, 'text': '29', 'ref_id': None}], 'ref_spans': [], 'section': 'IV. GLOBAL IMPACTS: THE NEED TO ADAPT FOR PANDEMICS?'}, {'text': 'The benefits of centralisation for decision-making regarding policies and resource distribution are clear. They facilitate coordinated decision-making where externalities become internalised and the long-term effectiveness of the policies is facilitated. Whilst there is some centralisation internationally (with the WHO) and within the EU (with the European Centre for Disease Prevention and Control (ECDC)), this primarily involves the gathering and sharing of scientific evidence and provision of guidance. 30 The EU does also, however, have the Health Threats Decision, 31 which has enabled joint procurement of medical equipment and facilitated 24 L O\\'Carroll, \"Romanian Fruit Pickers Flown to UK amid Crisis in Farming Sector\", The Guardian, 15 April 2020 <https://www.theguardian.com/world/2020/apr/15/romanian-fruit-pickers-flown-uk-crisis-farming-sector-coronavirus>. 25 The main obvious exception being Cuba, which has been providing considerable assistance to other states. 26 D Smith, \"New York\\'s Andrew Cuomo Decries \\'EBay\\'-Style Bidding War for Ventilators\", The Guardian, 31 March 2020 <https://www.theguardian.com/us-news/2020/mar/31/new-york-andrew-cuomo-coronavirus-ventilators>. 27 Q Forgey, \"\\'We\\'re Not a Shipping Clerk\\': Trump Tells Governors to Step up Efforts to Get Medical Supplies\", Politico, 19 March 2020 <https://www.politico.com/news/2020/03/19/trump-governors-coronavirus-medical-supplies-137658>. 28 H Jackson, \"Coronavirus: Canada to Receive \\'millions\\' of Masks from China, Trudeau Says\", Global News, 4 April 2020 <https://globalnews.ca/news/6778318/coronavirus-trudeau-aid-vulnerable>. 29 Eg J Bateman, \"\\'Solidarity Knows No Borders\\': Germany Treating Dozens of Coronavirus Patients from Italy and France\", The Independent, 1 April 2020 <https://www.independent.co.uk/news/world/europe/coronavirus-germanyitaly-france-hospital-treatment-covid-19-a9440906.html>; NG Torres and J Charles, \"Despite US Warnings, Cuba\\'s Medical Diplomacy Triumphs in the Caribbean during Pandemic\", Miami Herald, 15 April 2020 <https://www. miamiherald.com/news/nation-world/world/americas/cuba/article241745281.html>. 30 The WHO\\'s Pandemic Influenza Pandemic Framework, which was created by non-binding resolution, is of particular interest. Although not applicable to COVID-19, it demonstrates the potential role of the WHO in encouraging and facilitating sharing of research and resourcesbut equally through its uniqueness demonstrates the challenges in creating even soft instruments at an international level, much less legally enforceable ones. 31 targeted distributions across the EU. Furthermore, the European Commission has taken an innovative and impressive step recently by publishing a roadmap for exiting lockdowns across the EU 32thereby seeking to try and coordinate the exit strategies for the pandemic, even though the powers still rest officially with the Member States. 33 Such centralisation is essential to address externalities and is clearly possible in principle.', 'cite_spans': [{'start': 510, 'end': 512, 'text': '30', 'ref_id': None}, {'start': 877, 'end': 879, 'text': '25', 'ref_id': None}, {'start': 2133, 'end': 2135, 'text': '30', 'ref_id': None}, {'start': 2565, 'end': 2567, 'text': '31', 'ref_id': None}, {'start': 2903, 'end': 2905, 'text': '33', 'ref_id': 'BIBREF1'}], 'ref_spans': [], 'section': 'IV. GLOBAL IMPACTS: THE NEED TO ADAPT FOR PANDEMICS?'}, {'text': 'However, three obvious difficulties in particular arise here. Firstly, we would need to acknowledge and address the different circumstances of each state and region. This would include the state of progression of the disease, but also the capacity limitations of states and their population to implement policiesakin to \"common but differentiated responsibilities\", 34 but leading to more proactive support internationally. Just as we put in place systems to help citizens within individual nations, there would need to be global responsibility and packages for states that did not have the means to address it. We are seeing this to an extent with the World Bank, IMF and G20\\'s temporary debt relief for poor countries, 35 but more would be needed if there is to be a global approachgenuine, full-blown assistance for countries. \"Public\" becomes a global public. This is going to be very challenging where, for instance, there is no capacity to store food for a few weeks or to self-isolate.', 'cite_spans': [{'start': 721, 'end': 723, 'text': '35', 'ref_id': 'BIBREF3'}], 'ref_spans': [], 'section': 'IV. GLOBAL IMPACTS: THE NEED TO ADAPT FOR PANDEMICS?'}, {'text': 'Secondly, a desire for centralisation is based on the presumption that the centralised approach will be the preferable one. Whilst there is the potential to improve the effectiveness of any of the three approaches through a closely implemented and enforced approach across all populations, this does not mean it will be the best onein light of public health objectives or otherwise. For instance, aiming for herd immunity without also effectively protecting the vulnerable and flattening the curve at the same time appears to be prioritising the economy above human health and a weaker precautionary approach. In contrast, if a complete lockdown were undertaken, this might have considerable negative impacts on society and economies if maintained for lengthy periods, but would initially seem to prioritise human health and reflect a strong precautionary approach. But which approach is preferable to society? What would happen if the \"wrong\" approach were taken globally, whether excessively or insufficiently precautionary?', 'cite_spans': [], 'ref_spans': [], 'section': 'IV. GLOBAL IMPACTS: THE NEED TO ADAPT FOR PANDEMICS?'}, {'text': 'Thirdly, linked to this, decentralisation and varied approaches provide for learning opportunities and peer pressure. Each state is looking to see what others are doing and adjusting in light of not merely new scientific data and understanding, but also in light of what seems to be working or failing. 36 Furthermore, early, substantial restrictions and generally more precautionary approaches have influenced both the timing of responses and the underlying approach in some states (eg the UK switched from a laissez-faire, soft-regulation approach facilitating long-term herd immunity to one that is more restrictive and reflects approaches taken broadly across the EU). Consequently, there is \"soft centralisation\" due to comparisons facilitated by national decision-making.', 'cite_spans': [], 'ref_spans': [], 'section': 'IV. GLOBAL IMPACTS: THE NEED TO ADAPT FOR PANDEMICS?'}, {'text': \"Considering the importance of the values at stake and the extensive uncertainty, there is no absolute answer as to which approach will prove right in the long run or as to whether there should be full centralisation of decision-making powers (with recognition of differentiated circumstances and a relatively strong precautionary approach initially whilst information is gathered). It may be that mutual collaboration is the most feasible approach where efforts are taken to minimise externalitiesbut at the end of the day, unless there is a uniform approach taken globally or a vaccine is available for the entire population, the potential for one state to disrupt another's approach will remain.\", 'cite_spans': [], 'ref_spans': [], 'section': 'IV. GLOBAL IMPACTS: THE NEED TO ADAPT FOR PANDEMICS?'}, {'text': \"It is necessary to consider whether public health decision-making should be elevated to the global level in the case of pandemics. Subsidiarity provides a means to consider whether such powers should be reallocated, even on a temporary basis. Both public health/human life and the economy are fundamental values within each state, and the extent of the surrounding uncertainties makes the identification of the appropriate precautionary pathway in the long term more challengingwhich arguably suggests national approaches and trial and error. Nonetheless, the potential to undermine another state's approach (externalities) and the nature and level of the potential consequences of continued circulation of the disease supports centralisation of decision-makingat least within epidemiological unitswith caveats as noted.\", 'cite_spans': [], 'ref_spans': [], 'section': 'V. CONCLUSION'}, {'text': 'Fundamentally, it also raises broader questions about public health, interconnectedness and values. Serious health crises exist continuously across the world, both in developed and developing countries; they affect human life and broader society, yet until the issue comes knocking at our own door, we do not step up adequately. So, yes, under ideas of subsidiarity, some centralisation of decision-making, science and distribution of resources appears logical in the context of a serious pandemic. But this leaves unanswered the question of whether some centralisation and responsibility needs to be taken outside of pandemicsof whether we need to acknowledge the relative security and privilege of some and the corresponding insecurity and vulnerability of others in both our local and global societies. 36 ', 'cite_spans': [{'start': 806, 'end': 808, 'text': '36', 'ref_id': None}], 'ref_spans': [], 'section': 'V. CONCLUSION'}], 'paragraphs': [{'text': 'Whilst still part of public health, COVID-19 goes beyond the norm. Firstly, COVID-19 is highly contagious and spreads swiftly and easily. 11 This is accentuated by potentially long incubation periods and the potential for an individual to be contagious even though they are asymptomatic. The virus also remains viable on surfaces such as paper and plastic for some time, facilitating the spread further. Without control measures, the spread has proven to be exponentialwith us all now becoming too familiar with steeply increasing graphs and the term \"R0\". 12 Nor is this limited to one area or country. Due to a combination of the above factors, in conjunction with the widespread travel of individuals and products, the virus has spread globally and become a pandemic. Furthermore, COVID-19 has devastating effects. We have no vaccines, no tailored effective treatments and no built-in resistance or partial immunity to COVID-19. Whilst most will be asymptomatic or have mild symptoms, it can be fatal, with the elderly, ill and those with \"underlying conditions\" 13 being particularly vulnerable, and it also can cause severe impairments in the short and long termalthough the extent of these impairments is still to be fully understood. 14 This has further knock-on effects, as some public health systems are overburdened to the point of collapseillnesses or injuries that should be manageable cannot be properly treated, appointments and treatments are postponed and discussions arise about triage for both COVID-19 and non-COVID-19 patients. 15 The combination of the highly contagious nature and severity of impacts is worrying. It also provides for clusters of cases affecting specific groups and industries, including both healthcare workers (exacerbating the public health impacts once more) and other 11 Eg RM Anderson, H Heesterbeek and D Klinkenberg, \"How Will Country-Based Mitigation Measures Influence the Course of the COVID-19 Epidemic?\" (2020) 395(10228) The Lancet 931. 12 Reproduction number: the mean number of new cases an infected person will create. 13 A vague term that presumably is meant to reassure us, as the majority who die have \"underlying conditions\", but without clarity to date as to what this involvescould this be a broken leg?and also deprecating the value of those who have died who are elderly, ill or with such conditions. 14 \"essential workers\" in global supply chains, 16 as well as the economy reflected in the IMF\\'s report. All of these feed into each other and risk circular short-and long-term effectswhere the virus is present and elsewhere. Consequently, there is the potential for severe externalities by the virus itself if left unmanagedbut this does not mean that the governance of it would lead necessarily to externalities also. But is effective governance possible? A final major concern for the moment, which impacts upon every nation as they react to this, is the underlying uncertainties. 17 We do not understand the disease fully, gathering data and hopefully understanding as we go. That uncertainty relates to when an individual is contagious, the morbidity levels, how to test for the virus, how to test for antibodies, how long a person might be immune for and how to treat or prevent it in future. Whilst major strides are being made with each of these aspects, uncertainty still remainsin particular regarding antibodies, immunity and vaccines. Furthermore, testing (and subsequent contact tracing) for the disease is sometimes flawed and frequently limitedhampering understanding and making for patchy data to underpin decision-making. Consequently, any decision-making is based on the precautionary principle (whether express or otherwise) and it becomes more challenging to identify clear pathways to address the pandemic effectively that also minimise countervailing riskssomething that may in itself justify national rather than international approaches, even whilst benefiting from the centralisation and sharing of scientific data.', 'id': '00006'}], 'title': 'National Governance of Public Health Responses in a Pandemic?', 'sha': 'adf4d6ce9f1250714251304d5d1a06094ed3cb7d', 'doi': '10.1017/err.2020.39', 'score': 8.701600074768066}, {'rank': 290, 'abstract': 'The SIR model is modified, which may be called an SIQR model, so as to be appropriate for COVID-19 which has the following characteristics: [1] a long incubation period, [2] transmission of the virus by asymptomatic patients and [3] quarantine of patients identified through PCR testing. It is assumed that the society consists of four compartments; susceptibles (S), infecteds at large (simply called infecteds) (I), quarantined patients (Q) and recovered individuals (R), and the time evolution of the pandemic is described by a set of ordinary differential equations. It is shown that the quarantine rate can be determined from the time dependence of the daily confirmed new cases, from which the number of the infecteds at large can be estimated. The number of daily confirmed new cases is shown to be proportional to the number of infecteds a characteristic time earlier, and the infection rate and quarantine rate are determined for the period from mid-February to mid-April in Japan, and transmission characteristics of the initial stages of the outbreak in Japan are analyzed. The effectiveness of different measures is discussed for controlling the outbreak and it is shown that identifying patients through PCR testing and isolating them in a quarantine is more effective than lockdown measures aimed at inhibiting social interactions of the general population. An effective reproduction number for infecteds at large is introduced which is appropriate to epidemics controlled by quarantine measures.', 'body_text': [{'text': 'The pandemic due to the novel coronavirus (SARS-CoV-2) is still expanding throughout the world 6 months after the first outbreak in November 2019 in China (1) . Compared to other viral outbreaks such as Influenza, for which the SIR model is applied, COVID-19 has the following characteristics (2, 3) . [1] A long incubation period. [2] Prevalence of asymptomatic transmission. [3] Detection of infection by PCR tests. The biggest challenge in modeling the COVID-19 outbreak is that a large number of patients are hidden in the general population as they continuously infect others, and that the only observable metric on which we can base estimates is the number of daily new cases confirmed by positive tests.', 'cite_spans': [{'start': 155, 'end': 158, 'text': '(1)', 'ref_id': 'BIBREF0'}, {'start': 293, 'end': 296, 'text': '(2,', 'ref_id': 'BIBREF1'}, {'start': 297, 'end': 299, 'text': '3)', 'ref_id': None}, {'start': 302, 'end': 305, 'text': '[1]', 'ref_id': 'BIBREF0'}, {'start': 332, 'end': 335, 'text': '[2]', 'ref_id': 'BIBREF1'}, {'start': 377, 'end': 380, 'text': '[3]', 'ref_id': None}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'Traditionally, the dynamics of infectious diseases have been understood by the deterministic compartmental models described by differential equations such as the SIR model (4) and the SEIR model (5, 6) or by the stochastic model described by a renewal integral equation (7) . In the SEIR model, the population consists of susceptibles (S), exposed individuals (E), infectious individuals (I) and removed individuals (R). The SIR model does not include the compartment of exposed individuals. In these compartmental models, quarantine is considered to be an efficient way for medical treatment of patients and one of the removal processes. The removal time is usually treated as the scale of time, and thus the quarantine rate is included only in the 2 . CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 172, 'end': 175, 'text': '(4)', 'ref_id': None}, {'start': 195, 'end': 198, 'text': '(5,', 'ref_id': None}, {'start': 199, 'end': 201, 'text': '6)', 'ref_id': None}, {'start': 270, 'end': 273, 'text': '(7)', 'ref_id': 'BIBREF6'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'The copyright holder for this preprint this version posted June 5, 2020. . https://doi.org/10.1101/2020.06.02.20117341 doi: medRxiv preprint reproduction number. The SIR-X model proposed by Maier and Brockmann (8) includes the quarantine process in the analysis of the early state of the outbreak in China.', 'cite_spans': [{'start': 190, 'end': 213, 'text': 'Maier and Brockmann (8)', 'ref_id': None}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'In the renewal model, the number of infecteds at a given time is assumed to be determined by a convolution of a time-dependent transmission coefficient and the number of infecteds at earlier times with the reproduction number as a multiplier, and the quarantine process is hidden in the reproduction number.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'There are several attempts to understand the nature of the outbreak of COVID-19 on the basis of the SEIR model (8, 9) and the renewal model (10) . Apparently no attempts have treated quarantined patients as a compartment and the quarantine rate as a controllable parameter, although it plays an essential role in the measures against the COVID-19. At this moment in the middle of outbreak of COVID-19, it is important to establish a universal, predictable and simple model for the outbreak which can be understood by citizens and policy makers.', 'cite_spans': [{'start': 111, 'end': 114, 'text': '(8,', 'ref_id': None}, {'start': 115, 'end': 117, 'text': '9)', 'ref_id': None}, {'start': 140, 'end': 144, 'text': '(10)', 'ref_id': None}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'In this paper, keeping the characteristics of the COVID-19 explained above in mind, I introduce four compartments; susceptibles (S), infecteds at large or simply infecteds (I), quarantined patients (Q) and recovered individuals (R), and describe the transmission process by a system of ordinary differential equations where the quarantine rate of infecteds is introduced as a key parameter. This model may be called an SIQR model. Although the basic differential equations are similar to those of the SIR model, the present SIQR model treats the quarantine process as a measure controlled by policy. Namely, I consider the quarantined patients not only as individuals to be treated in the hospital but also as key players in combating the spread of the virus. CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review) Figure 1 : Flow diagram of the SIQR model. If the quarantine process is included in the recovery process and ¼ is assumed, the SIQR model is reduced to the SIR model. the data from March 1st to April 29th by four pieacewise exponential functions, and compare the time evolution of parameters derived from fitting with the measures taken by the Japanese government from mid-February to mid-April. This period saw a rapid rise in daily confirmed new cases, and is also notable for a succession of policy milestones, one being the planned Olympiad postponed by IOC on March 24th, and the other being the prime minister announcing on April 5th that a state of emergency would be formally declared two days later on the 7th. The government issued stay-at-home request, asking 80% reduction of social contact. I compare the time dependence of daily confirmed new cases if different measures would have been taken at the beginning of April in Japan. I compare various policies characterized by a combination of differing degrees of testing/quarantine and societal lockdown and show that the quarantine measure is more efficient than the lockdown measure.', 'cite_spans': [], 'ref_spans': [{'start': 960, 'end': 968, 'text': 'Figure 1', 'ref_id': 'FIGREF0'}], 'section': '(which was not certified by peer review)'}, {'text': '. CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'cite_spans': [], 'ref_spans': [], 'section': '4'}, {'text': 'The copyright holder for this preprint this version posted June 5, 2020. . Table  2 . Data are from Ministry of Health, Labor and Welfare, Japan (11).', 'cite_spans': [], 'ref_spans': [{'start': 75, 'end': 83, 'text': 'Table  2', 'ref_id': 'TABREF1'}], 'section': '4'}, {'text': 'The dynamics of the SIQR model are described by the following set of ordinary differential equations:', 'cite_spans': [], 'ref_spans': [], 'section': 'SIQR model'}, {'text': 'where Ø is the time, and Ë, Á, É and Ê are the stock of susceptible population, infecteds at large, quarantined patients and recovered (and died) patients, respectively. Here, I assumed that the net rate at which infections spread is in proportion to the number of encounters between susceptibles and infecteds at large, ¬ Ë Á , where ¬ is a transmission coefficient. The parameter 5 . CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'cite_spans': [], 'ref_spans': [], 'section': 'SIQR model'}, {'text': 'The copyright holder for this preprint this version posted June 5, 2020. . https://doi.org/10.1101/2020.06.02.20117341 doi: medRxiv preprint Õ ¼ represents a per capita rate of quarantine of the new patients immediately after they get infected. For COVID-19, Õ ¼ can be set to zero since the incubation period is long. Infecteds at large, regardless of wheter they are symptomatic or asymptomatic, are quarantined at a per capita rate Õ and become non-infectious to the population. Quarantined patients recover at a per capita rate ¼ (where ½ ¼ is the average time it takes for recovery) and infecteds at large become non-infectious at a per capita rate (where ½ is the average time that an infected patient at large is capable of infecting others). It is apparent that Eqs. (1) (4) guarantee the conservation of population AE Ë · Á · É · Ê.', 'cite_spans': [], 'ref_spans': [], 'section': 'SIQR model'}, {'text': 'If we assume ¼ and regard É · Ê as the recovered compartment, Eqs. (1) (4) are identical to those of the SIR model with removal rate of infecteds Õ · . Since the value of Õ depends strongly on government policies and the only observable is the number of quarantined patients on each day, the SIQR model which treats the quarantine process explicitly is superior to the SIR and SEIR models in the analysis of the outbreak of the COVID-19.', 'cite_spans': [], 'ref_spans': [], 'section': 'SIQR model'}, {'text': 'In the following discussion, I concentrate on the number of infecteds in the early stage of the outbreak where Á · É · Ê AE is satisfied so that I can assume Ë ³ AE. Then the basic equations of the SIQR model become', 'cite_spans': [], 'ref_spans': [], 'section': 'SIQR model'}, {'text': 'where the rate of change of the infecteds ¬ AE Õ', 'cite_spans': [], 'ref_spans': [], 'section': 'SIQR model'}, {'text': 'determines the short-term behavior of the number of infecteds.', 'cite_spans': [], 'ref_spans': [], 'section': 'SIQR model'}, {'text': 'The parameter ¬ determines the initial growth of infecteds and is determined by the cumulative effect of the virulence of the virus and social factors. The longer latent period and 6 . CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'SIQR model'}, {'text': 'The copyright holder for this preprint this version posted June 5, 2020. . https://doi.org/10.1101/2020.06.02.20117341 doi: medRxiv preprint shorter infectious periods reduces the number of infectious patients in the infecteds at a given time which in turn makes ¬ effectively smaller. Similarly, the shorter incubation period and the higher onset probability increases the number of quarantined individuals which makes the quarantine rate Õ larger.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The parameters are dependent on the lifestyle of each community and measures taken by governments. For example, availability of a vaccine, implementation of social distancing or lock down measures will all decrease ¬, while widespread PCR testing and isolation of patients will increase Õ. Although the recovery rate for quarantined patients ¼ will be increased when a new drug for SARS-CoV-2 is developed, it would not increase since a patient is unlikely to seek treatment unless they are first identified.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'It should be noted that the characteristics of the outbreak of the COVID-19 can be understood by effective values of three parameters ¬, Õ and , It is usual to take the inverse of the removal rate (Õ · ) or the recovery rate ( ¼ ) as the scale of time. However, these parameters are policy dependent and cannot be considered as an intrinsic time scale of the problem. In order to present results in the style that citizens and policy makers can easily understand, I take one day as the scale of time in this paper.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'Although an exact solution to Eqs. (1) (4) can be obtained by introducing a scaled time when parameters do not depend on time (12) , the solution is impractical for the purpose of analyzing the outbreak and predicting the outcome of various measures.', 'cite_spans': [{'start': 126, 'end': 130, 'text': '(12)', 'ref_id': None}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The term ÕÁ´Øµ in Eqs. (5) and (6) is the number of daily confirmed new cases ¡É´Øµ and thus the number of infecteds is given by Á´Øµ ¡É´Øµ ¬ AE (8) 7 . CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [{'start': 23, 'end': 26, 'text': '(5)', 'ref_id': None}, {'start': 145, 'end': 148, 'text': '(8)', 'ref_id': None}], 'ref_spans': [], 'section': 'Estimation of the number of infecteds at large'}, {'text': 'The copyright holder for this preprint this version posted June 5, 2020. . This relation indicates that if we can determine from the time dependence of the daily confirmed new cases, ¬ from the initial increase of infecteds and infectious period ½ of asymptomatic patients, we can determine the number of infected at large from the daily confirmed new cases.', 'cite_spans': [{'start': 59, 'end': 74, 'text': 'June 5, 2020. .', 'ref_id': None}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'In the data reported in Japan (11), the daily confirmed new cases are apparently a constant around ¿¼ in the period from 11th to 19 th of April, 2020 as shown in Fig. 2 . If this is the case, must be negligibly small in this period, and thus Á´Øµ ¡É´Øµ ¬ AE (9) As will be seen later, ³ ¼ and Õ ³ ¼ ¼ in this period when ¼ ¼¿ (or 33 days of the latent plus infectious periods) is used, and thus I find Á´Øµ ³ ½¼¡É´Øµ (10) This means that there were about 5300 infecteds in Japan at the beginning of April. This number increases to 8030 if ¼ ¼ (17days for the recovering time) is used.', 'cite_spans': [{'start': 258, 'end': 261, 'text': '(9)', 'ref_id': None}, {'start': 417, 'end': 421, 'text': '(10)', 'ref_id': None}], 'ref_spans': [{'start': 162, 'end': 168, 'text': 'Fig. 2', 'ref_id': 'FIGREF1'}], 'section': '(which was not certified by peer review)'}, {'text': 'In the time period when the parameters do not depend on time, Eqs. (5) and (6) can be readily solved:', 'cite_spans': [], 'ref_spans': [], 'section': 'Time evolution of outbreak'}, {'text': 'Á´Øµ', 'cite_spans': [], 'ref_spans': [], 'section': 'Time evolution of outbreak'}, {'text': 'Equation (11) indicates that the number of infecteds is determined by and its initial condition.', 'cite_spans': [{'start': 9, 'end': 13, 'text': '(11)', 'ref_id': 'BIBREF10'}], 'ref_spans': [], 'section': 'Time evolution of outbreak'}, {'text': 'The parameter depends on measures that the government employs, and we can assess the measures from the time dependence of the number of infecteds.', 'cite_spans': [], 'ref_spans': [], 'section': 'Time evolution of outbreak'}, {'text': '8 . CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Time evolution of outbreak'}, {'text': 'The copyright holder for this preprint this version posted June 5, 2020. . https://doi.org/10.1101/2020.06.02.20117341 doi: medRxiv preprint At the beginning of the outbreak, infecteds cannot be detected since symptoms do not appear for the incubation period and thus cannot be quarantined. Therefore, the number of infecteds increases exponentially as Á´Øµ Á´¼µ ÜÔ´¬AEØ µ (13) About a few weeks later Ø ½ (13), some infecteds will recover and thus the growth rate will be reduced by :', 'cite_spans': [{'start': 373, 'end': 377, 'text': '(13)', 'ref_id': 'BIBREF12'}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'Á´Øµ Á´Ø ½ µ ÜÔ ´¬AE µ´Ø Ø ½ µ (14) After some time, a certain number of patients with severe symptoms will appear and the government will introduce various measures at time Ø ¾ such as social distancing, and societal lockdown measures. If the measures in which ¬ is reduced to´½ Üµ¬ keeping Õ as before is effective, the number of infecteds will start to decline exponentially, following Á´Øµ Á´Ø ¾ µ Ü Ô ´´½ Üµ¬ AE Õ µ´Ø Ø ¾ µ (15) where Ü represents the degree of social distancing and lockdown. The effectiveness of the measure can be judged by the size of the decay constant ´´½ Üµ¬ AE Õ µ .', 'cite_spans': [{'start': 31, 'end': 35, 'text': '(14)', 'ref_id': None}, {'start': 429, 'end': 433, 'text': '(15)', 'ref_id': None}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'If no measures are taken, the number of infecteds increases exponentially at the beginning. I assume when no measure is taken, the doubling time of infecteds is 7 days and the recovering time is 30 days, that is ¬ AE ¼ ¼ and ¼ ¼¿¿.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'For the first 30 days, the number of infecteds simply increases exponentially with ¼ ¼ , and for the next 30 days it increases with reduced rate ¼ ¼ due to the recovering process.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': '. CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '9'}, {'text': 'The copyright holder for this preprint this version posted June 5, 2020. . https://doi.org/10.1101/2020.06.02.20117341 doi: medRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'In the following two periods, days 60 75 and days 75 , each country takes different measures.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'Country U introduces a strict PCR testings and qurantine Õ ¼ ¼¼½ in the first period and in the second period 50% reduction of social contact Ü ¼ and a strong quarantine measure Õ ¼ ½. Country J introduces a moderate quarantine measure Õ ¼ ¼ in the first period.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'In the second period, stay-at-home request with 80% reduction of outings is issued, keeping the same quarantine measure. Country K introduces a complete lockdown preventing all social contact Ü ½ and strong quarantine measure Õ ¼ ½ from the first period. Table 1 summarizes the parameters of each model country. Next, I will take a look at the relation of Á´Øµ to the number of daily confirmed new cases ¡É´Øµ. I assume that patients tested positive are quarantined either in a hospital or at home, and are no more infectious to the population. The daily confirmed new cases ¡É´Øµ is proportional to the number of infecteds who have an onset of symptoms. Since the latter is given by a convolution of the incubation period distribution function ´Øµ and the number of infecteds Á´Øµ. Therefore, ¡É´Øµ can be expressed as', 'cite_spans': [], 'ref_spans': [{'start': 255, 'end': 273, 'text': 'Table 1 summarizes', 'ref_id': 'TABREF0'}], 'section': '(which was not certified by peer review)'}, {'text': '10 . CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted June 5, 2020. . Table 1 for the detailed setting of the parameters for the latter half.', 'cite_spans': [], 'ref_spans': [{'start': 75, 'end': 82, 'text': 'Table 1', 'ref_id': 'TABREF0'}], 'section': '(which was not certified by peer review)'}, {'text': 'The incubation period distribution is a well behaved function with a single peak (14, 15) . I assume, for the sake of simplicity, that ´Øµ is given by ´Øµ', 'cite_spans': [{'start': 81, 'end': 85, 'text': '(14,', 'ref_id': None}, {'start': 86, 'end': 89, 'text': '15)', 'ref_id': None}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'where is a positive constant and denotes the incubation period. When Á´Øµ Á´¼µ ÜÔ´ Øµ, it is straitforward to show that (16) ¡É´Øµ » ¾ × Ò ¾ Á´Ø µ (18) Therefore, the daily confirmed new cases is related to the number of infecteds mean-incubationtime earlier. In fact, the number of daily confirmed new cases of many countries seem to be represented by piecewise exponential functions (17) .', 'cite_spans': [{'start': 147, 'end': 151, 'text': '(18)', 'ref_id': 'BIBREF18'}, {'start': 385, 'end': 389, 'text': '(17)', 'ref_id': 'BIBREF17'}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'I analyze the daily confirmed new cases ¡É´Øµ in Japan by fitting it with piecewise exponential functions, where I neglected the effect of the transition periods. From the fitting, I 11 . CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted June 5, 2020. . https://doi.org/10.1101/2020.06.02.20117341 doi: medRxiv preprint determined the rate of change , from which parameters ¬ AEand Õ are estimated. The solid curve in Fig. 2 ', 'cite_spans': [], 'ref_spans': [{'start': 239, 'end': 245, 'text': 'Fig. 2', 'ref_id': 'FIGREF1'}], 'section': '(which was not certified by peer review)'}, {'text': 'April 29th by four exponential functions with listed in Table 2 . Table 2 .', 'cite_spans': [], 'ref_spans': [{'start': 56, 'end': 63, 'text': 'Table 2', 'ref_id': 'TABREF1'}, {'start': 66, 'end': 73, 'text': 'Table 2', 'ref_id': 'TABREF1'}], 'section': 'shows a piecewise fitting in the period of March 1st'}, {'text': 'Several comments are in order: (1) the basic reproduction number Ê ¼ ¬ ´Õ · µ is estimated to be for the first two periods and becomes 0.4 after April 5th passing the period where Ê ¼ ½. (2) The reduction of ¬ AEafter April 5th is about ¼ ¼ ½ ¼ ½¾ ¢½¼¼ ¼% which may be attributed to the emergency declaration by the government on April 7th. Apparently the effect is seen two days earlier since the government announced it on April 5th in advance.', 'cite_spans': [{'start': 187, 'end': 190, 'text': '(2)', 'ref_id': 'BIBREF1'}], 'ref_spans': [], 'section': 'shows a piecewise fitting in the period of March 1st'}, {'text': '(3) The effectiveness of PCR test and quarantine, that is the probability of a patient being qurantined when symptoms appear, is rather low around 25 50% due to the strict conditions set by the Japanese government for PCR tests (such as a mandatory 4 day waiting period before 12 . CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'shows a piecewise fitting in the period of March 1st'}, {'text': 'The copyright holder for this preprint this version posted June 5, 2020. . https://doi.org/10.1101/2020.06.02.20117341 doi: medRxiv preprint a symptomatic patient is allowed a test ). Since Õ onset rate ¢ the probability of getting PCR test, the onset probability is estimated to be 20 40%. These values are in the wide range 20 95% of estimations by other researchers (18) . (4) Since the fitting is very much rough as the data are, one might fit the data by different sets of exponential functions, and the estimated value of parameters might be a little different from those obtained above.', 'cite_spans': [{'start': 369, 'end': 373, 'text': '(18)', 'ref_id': 'BIBREF18'}], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'If the estimation given in the previous section is correct, there must have been 5300 infecteds at large in Japan at the beginning of April. In this section, I compare how the number of daily confirmed new cases ¡É´Øµ would have reduced if Japanese government had taken different measures.', 'cite_spans': [], 'ref_spans': [], 'section': 'Comparison of possible measures'}, {'text': 'A measure can be characterized by two parameters Ü and Ý, which define an effective decay constant ´Ü Ýµ as ´Ü Ýµ 1 Üµ¬ AE ÝÕ (19) For the purposes of this paper, I will treat social distancing and lockdown measures as part of the same effort to reduce social contact, whose cumulative effectiveness is represented by Ü (¼ Ü ½), with Ü ¼ representing no change to social behavior, and Ü ½ representing the most extreme form of societal lockdown.', 'cite_spans': [{'start': 126, 'end': 130, 'text': '(19)', 'ref_id': 'BIBREF19'}], 'ref_spans': [], 'section': 'Comparison of possible measures'}, {'text': 'Parameter Ý denotes the enhancement factor of quarantine rate. Since´½ Üµ¬ AE ÝÕ is supposed to be negative when the number of infecteds is declining, the smaller´½ Üµ and the larger Ý are, the decay rate ´Ü Ýµ becomes larger.', 'cite_spans': [], 'ref_spans': [], 'section': 'Comparison of possible measures'}, {'text': 'From the values of ¬ AEin the third and fourth periods in Table 2 , the reduction rate of social contact is estimated to be 60% after the emergency declaration. Therefore, I compare, relative to the third period of Table 2 (¬ AE ¼ ½¾ , Õ ¼ ¼ , Ü ¼ and Ý ½ ), hypothetical measures of (1) 80% reduction of outings that the government requested and at the same level of the PCR 13 . CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [{'start': 58, 'end': 65, 'text': 'Table 2', 'ref_id': 'TABREF1'}, {'start': 215, 'end': 222, 'text': 'Table 2', 'ref_id': 'TABREF1'}], 'section': 'Comparison of possible measures'}, {'text': 'The copyright holder for this preprint this version posted June 5, 2020. . https://doi.org/10.1101/2020.06.02.20117341 doi: medRxiv preprint Table 2 , if it was taken on April 5th (Ø ¼ ).', 'cite_spans': [], 'ref_spans': [{'start': 141, 'end': 148, 'text': 'Table 2', 'ref_id': 'TABREF1'}], 'section': '(which was not certified by peer review)'}, {'text': 'Observed decay given in the fourth period is also shown by curve (0). If is set to a larger value, the effect in decay rate due to the enhanced quarantine rate becomes somewhat moderate. The horizontal line represents ¡É´Øµ ¡É´¼µ ¼ ½.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'test (Ü ¼ , Ý ½ ) and (2) 100% reduction of outings, that is the strictest form of lockdown, and at the same level of the PCR test (Ü ½ ¼, Ý ½). I also estimate the reduction rate for (3) 50% reduction of outings and doubled quarantine rate by introducing an expanded policy for PCR testing (Ü ¼ , Ý ¾ ) and (4) quadrupled quarantine rate without any restriction on social interactions (Ü ¼ ¼, Ý ). Figure 4 compares the reduction rate of the daily confirmed new cases for different measures, had they been taken from April 5th. Figure 4 shows clearly that partial or strict lockdowns are less effective than increase of the quarantine rate. Countries like Korea, Vietnam and Taiwan which took strong lockdown and quarantine measures have apparently succeeded in controlling the first wave outbreak of COVID-19 (19) (20) (21) .', 'cite_spans': [{'start': 811, 'end': 815, 'text': '(19)', 'ref_id': 'BIBREF19'}, {'start': 816, 'end': 820, 'text': '(20)', 'ref_id': 'BIBREF20'}, {'start': 821, 'end': 825, 'text': '(21)', 'ref_id': 'BIBREF21'}], 'ref_spans': [{'start': 399, 'end': 407, 'text': 'Figure 4', 'ref_id': 'FIGREF5'}, {'start': 529, 'end': 537, 'text': 'Figure 4', 'ref_id': 'FIGREF5'}], 'section': '(which was not certified by peer review)'}, {'text': 'In order to see the effectiveness of measures characterized by parameters Ü and Ý, the effec-', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': '.', 'cite_spans': [], 'ref_spans': [], 'section': '14'}, {'text': 'CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '14'}, {'text': 'The copyright holder for this preprint this version posted June 5, 2020. . https://doi.org/10.1101/2020.06.02.20117341 doi: medRxiv preprint Figure 5 : Dependence of the effective decay rate ´Ü Ýµ on parameters Ü and Ý which represent the strength of lockdown and the enhancement factor of the quarantine rate, respectively. The values for ¬ AE , Õ and are taken from the third period of Table 2. tive decay rate ´Ü Ýµ is plotted as a function of Ü and Ý in Fig. 5 . The decay constant Eq. (19) shows that Ý has more leverage than Ü in making (´½ Üµ¬ AE ÝÕ ) a large negative value. Therefore, we see that increasing the quarantine rate is a more effective measure than any forms of societal lockdown. Although this figure depends on the value assumed for the parameters, effectiveness of the quarantine does not change (22) .', 'cite_spans': [{'start': 490, 'end': 494, 'text': '(19)', 'ref_id': 'BIBREF19'}, {'start': 820, 'end': 824, 'text': '(22)', 'ref_id': None}], 'ref_spans': [{'start': 141, 'end': 149, 'text': 'Figure 5', 'ref_id': None}, {'start': 388, 'end': 396, 'text': 'Table 2.', 'ref_id': 'TABREF1'}, {'start': 458, 'end': 464, 'text': 'Fig. 5', 'ref_id': None}], 'section': '(which was not certified by peer review)'}, {'text': 'The pandemic COVID-19 has unusual nature such as infectious asymptomatic patients and a long incubation period in comparison to known epidemics (1). In the typical epidemics, all infected individuals become infectious after a latent period and show onset of symptoms after an incubation period. Therefore the number of infecteds can be estimated from the number of symptomatic patients. We cannot do the same for COVID-19, however, because of the prevalence of asymptomatic patients in the general population. Short of testing the entire population in a single day, the only visible metric we have for the purpose of estimating the total number of infecteds is the number of daily confirmed new cases ¡É´Øµ. Therefore, what we need is to 15 . CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'The copyright holder for this preprint this version posted June 5, 2020. . https://doi.org/10.1101/2020.06.02.20117341 doi: medRxiv preprint construct a physical model for the pandemic in which ¡É´Øµ is treated explicitly and can be related to the number of infecteds at large Á´Øµ.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The infection process itself is a complex phenomenon with numerous parameters. What we need now is a simple theory in which those parameters can be effectively absorbed in essential parameters describing COVID-19. I have presented in this paper a universal, predictable and simple model which is constructed on the basis of the mean field theory and distinguishes the quarantine process explicitly. In order to determine parameters of the model, I fitted the data of the daily confirmed new cases reported in Japan. Using the time dependence of the parameters, I analyzed the characteristics of the outbreak in Japan from mid-February to mid-April. Because of the advance notice of the declaration of the national emergency expected on April 7th, the effect is seen apparently from April 5th. After April 5th, the partial lockdown appears to have resulted in 60% reduction of the social contacts, which is 20% less than the value 80% that the government requested, Has the government introduced other measures as well at the same time, the reduction of infecteds would have been much faster. In particular, the four-day waiting period required for symptomatic patients before they were allowed a PCR test has made the situation much worse. If the waiting period was halved, the quarantine rate would have been doubled. The curves (3) and (4) in Fig. 4 clearly show the effect of efficiency of the PCR test.', 'cite_spans': [], 'ref_spans': [{'start': 1345, 'end': 1351, 'text': 'Fig. 4', 'ref_id': 'FIGREF5'}], 'section': '(which was not certified by peer review)'}, {'text': 'The Japanese government could have taken much better and stronger measures.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The basic equation can be modified so as to include the memory effect in the infection process and the quarantine process. To this end, the first two terms in the middle part on the right-hand side of Eq. (5) can be written as', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'where ¬´Ø Ø ¼ µ denotes the time dependence of the transmission coefficient and Õ´Ø Ø ¼ µ represents the rate of quarantine of infecteds at time Ø who got infected at time Ø ¼ ,', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': '. CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'cite_spans': [], 'ref_spans': [], 'section': '16'}, {'text': 'The copyright holder for this preprint this version posted June 5, 2020. In order to check the consistency of the present analysis in Japan, I verified that the daily confirmed new cases in Japan up until the first two weeks of May are on the curve (0) in Fig. 4 . Although the present model is based on the same compartmental approach as the SIR and SEIR models, it proposes a novel concept in that the quarantine is an effective measure in combatting the spread of the COVID-19 (22) . According to the SIR model apparently employed by the Japanese government, the quarantine was considered as a part of treatment and the last two terms in Eq. (2) ÕÁ· Áare considered together for effective treatments in a hospital. Therefore, many doctors in Japan were strongly against increasing the number of PCR tests, fearing the collapse of the medical care system and false positives in PCR tests leading to placing healthy individuals in quarantine. The policy on quarantine set by the Japanese government was not appropriate to COVID-19.', 'cite_spans': [{'start': 480, 'end': 484, 'text': '(22)', 'ref_id': None}], 'ref_spans': [{'start': 256, 'end': 262, 'text': 'Fig. 4', 'ref_id': 'FIGREF5'}], 'section': '(which was not certified by peer review)'}, {'text': 'Since the early stages of civilized history, the quaratine process was considered an effective method for reducing the spread of infection. Therefore, the term´¬ AE ÕµÁ in Eq. (5) is considered the number of infecteds newly added in the population, and the quarantined ÔÁ is considered a means of removing infecteds from the population. It is apparent that increasing the quarantine rate is more effective in making ´Ü Ýµ negative than lockdown measure, since the former can reduce ´Ü Ýµ by several times of Õ while the latter can contribute at most ¬ AE .', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'For COVID-19, we should go back to this tradition with contemporary method in identifying pre-symptomatic and asymptomatic patients by PCR tests. In this sense, an effective reproduction number Ê relevant to COVID-19 should be defined by Ê ¬ AE Õ (21) which represents the rate of change of infecteds at large. The effective reproduction number in Japan in each period shown in Table 2 is 4.2, 4.0, 1 and -1.5, where Ê ½ indicates that the number of infecteds in the population is steady. When the epidemic is expanding, Ê becomes 17 . CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'cite_spans': [{'start': 247, 'end': 251, 'text': '(21)', 'ref_id': 'BIBREF21'}], 'ref_spans': [{'start': 378, 'end': 385, 'text': 'Table 2', 'ref_id': 'TABREF1'}], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted June 5, 2020. . https://doi.org/10.1101/2020.06.02.20117341 doi: medRxiv preprint larger than 1. On the other hand if Ê becomes less than 1, the epidemic is declining. When ¼ Ê ½, the convergence of the epidemic is achieved by the recovery of patients. The measure which makes Ê ¼ must be introduced by each government suitable to the country to accelerate the convergence.', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': '. CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted June 5, 2020. . https://doi.org/10.1101/2020.06.02.20117341 doi: medRxiv preprint . CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}, {'text': 'The copyright holder for this preprint this version posted June 5, 2020. . https://doi.org/10.1101/2020.06.02.20117341 doi: medRxiv preprint', 'cite_spans': [], 'ref_spans': [], 'section': '(which was not certified by peer review)'}], 'paragraphs': [{'text': 'The pandemic COVID-19 has unusual nature such as infectious asymptomatic patients and a long incubation period in comparison to known epidemics (1). In the typical epidemics, all infected individuals become infectious after a latent period and show onset of symptoms after an incubation period. Therefore the number of infecteds can be estimated from the number of symptomatic patients. We cannot do the same for COVID-19, however, because of the prevalence of asymptomatic patients in the general population. Short of testing the entire population in a single day, the only visible metric we have for the purpose of estimating the total number of infecteds is the number of daily confirmed new cases ¡É´Øµ. Therefore, what we need is to 15 . CC-BY-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.', 'id': '00059'}], 'title': 'Analysis of the outbreak of COVID-19 in Japan on the basis of an SIQR model', 'sha': 'f9cad7144c76b0515fd12253f6199b650efe8e8c', 'doi': '10.1101/2020.06.02.20117341', 'score': 8.701199531555176}, {'rank': 291, 'paragraphs': [{'text': 'designed to capture the overall fraction of infected individuals throughout the spreading period.', 'id': '00011'}], 'title': 'Alternating quarantine for sustainable mitigation of COVID-19', 'sha': 'b923a8fc7d406d0f149ecd5293d1a72dec7ee526', 'doi': '', 'score': 8.695899963378906}, {'rank': 292, 'paragraphs': [{'text': 'Our results indicate that, unmitigated, COVID-19 will lead to an order of R ∞ ≈ 0.65, i.e. ∼ 65% total infection (purple). Next we seek', 'id': '00012'}], 'title': 'Alternating quarantine for sustainable mitigation of COVID-19', 'sha': 'b923a8fc7d406d0f149ecd5293d1a72dec7ee526', 'doi': '', 'score': 8.678400039672852}, {'rank': 293, 'paragraphs': [{'text': 'However, in absence of vaccine and other uncertainties regarding the virus, herd immunity is debatable issue as a preventive measure and achieving herd immunity is likely to be a long-drawn process.', 'id': '00026'}], 'title': 'Incubation Period and Reproduction Number for novel coronavirus (COVID-19) infections in India', 'sha': '4cc2e02fbf932137fb5dda889927d090206f248e', 'doi': '10.1101/2020.06.27.20141424', 'score': 8.676199913024902}, {'rank': 294, 'paragraphs': [{'text': 'We introduce a method for determining the first date at which the case counts of two models begin to diverge. The simulated datasets for a model are treated as random samples from a distribution of prevalence in each component of the ODE. In order to determine when the effect of physical distancing first appears in a model (i.e., determining when a model diverges from a baseline model), we compute the empirical probability that there is a difference between the two models, according to the distributions of prevalence implied by the two models. In this work, we mainly consider a difference of 10 prevalent and non-quarantined cases, which correspond to I +I d in the model, to be a substantial difference, as this may be a number for which the risk of a return to community spreading is imminent. We refer to this threshold as the number of active cases (i.e., the I + I d component is the number of active cases). To compute the empirical probability that a substantial difference occurs on a given day after introducing distancing measures, we consider the proportion of simulated samples for which the model with physical distancing shows fewer infected counts than the non-distancing model by at least 10. (Alternatively, if distancing measures are being relaxed rather than introduced, then prevalent cases would increase and we consider the proportion of simulated samples for which the model without physical distancing shows more than 10 active cases.) We form this empirical probability for each day. The first day on which the empirical probability of observing a 10-case difference is at least 0.95 is our estimate for the first day at which the effect of modifying physical distancing is substantial (this is the days until threshold in our figures). In most of our experiments involving this threshold, we use a value of 10 for the threshold. In addition, we conduct an experiment in which the threshold is varied in the set 6 . CC-BY-NC 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. (which was not certified by peer review)', 'id': '00016'}], 'title': 'Long time frames to detect the impact of changing COVID-19 control measures', 'sha': '8fb24f1b94e8bff0fdbc612929d6152135bf242c', 'doi': '10.1101/2020.06.14.20131177', 'score': 8.674500465393066}, {'rank': 295, 'abstract': 'Severe acute respiratory syndrome-coronavirus 2 (SARS-CoV-2), the virus strain that causes coronavirus disease 2019 (COVID-19), was first identified in Wuhan, China in December 2019. It spread to several countries across continents and infected more than one million people within three months. While there is no consensus on the treatment of the disease yet, understanding the virus and its transmission is a cardinal priority. SARS-CoV-2 can be transmitted through bodily fluid. Upon inoculation, the surface enzyme angiotensin-converting enzyme 2 (ACE2) acts as a receptor protein for viral entry. The mean incubation period is 5.1 days, and infected individuals can exhibit a variety of symptoms from fever, cough, dyspnea, and respiratory failure to even multiorgan failure. Given the current situation, it is of paramount importance to understand the virus as thoroughly as possible. In this review, we discuss the background, epidemiology, possible pathophysiology, clinical presentation, and diagnostic studies related to SARS-CoV-2 infection. We also elaborate on the current research and evidence on treatment options and vaccine development based on the literature.', 'paragraphs': [{'text': 'SARS-CoV-2 infection or COVID-19 is a highly transmissible novel zoonotic infection with high case fatality rates. The transmission of infection from both asymptomatic and symptomatic patients and its relatively long incubation period make it even harder to control its spread. Unfortunately, there is no consensus on how to treat this infectious disease yet. More studies and data are needed. A good understanding of the disease transmission, its genetics, pathophysiology, clinical presentation, prognostic factors, diagnostic studies, and preventive measures are of prime importance in making clinical decisions and designing research projects for best treatment while responding to minimize the threat to people.', 'id': '00045'}], 'title': 'A Comprehensive Review of Severe Acute Respiratory Syndrome Coronavirus 2', 'sha': 'c5e9acf0143b9745a91ad7110deae6a999c92d74', 'doi': '10.7759/cureus.7943', 'score': 8.674099922180176}, {'rank': 296, 'abstract': 'Radiology departments have been directly involved from the beginning of the novel coronavirus disease (COVID-19) emergency to provide imaging lung assessment of suspected and positive patients while ensuring the execution of other routine and emergency examinations for non-COVID-19 patients. To limit the risk of the infection spread, radiology departments should be reconfigured. We propose the example of the reorganization of the Radiology Department of our hospital, in the center of Milan, in Northern Italy, which consisted of the creation of 2 completely distinct pathways and distinct radiological machines for COVID-19 positive or suspected positive and for non-COVID-19 patients.', 'body_text': [{'text': 'Radiology technologists (RTs) are among the first-line health care workers who might be exposed to COVID-19; therefore, they should be particularly careful in the use of adequate precautions.', 'cite_spans': [], 'section': 'Personal Protective Equipment ::: STAFF TRAINING', 'ref_spans': []}, {'text': 'COVID-19 spreads through droplet and contact transmission. When dealing with suspected patients, the PPE to protect from contact, droplet, and airborne transmission is needed. The PPE set includes eye protection (goggles or face shield) to prevent virus exposure of the eye mucosa, a filtering face-piece respirator (FFP2), a surgical cap, a double pair of disposable gloves (inner and outer layer), a disposable long-sleeved fluid-resistant gown, and shoe covers.', 'cite_spans': [], 'section': 'Personal Protective Equipment ::: STAFF TRAINING', 'ref_spans': []}, {'text': 'Before putting on PPE, proper hand hygiene should be performed.5 Alcohol-based hand gels for hand hygiene dispensers are available in all dressing rooms, reading rooms, and diagnostic units. The wearing and removal of PPE are performed in 2 different dedicated rooms. The PPE wearing takes place in a room with direct access to the COVID-19 Radiology and includes the following items in sequence: disposable gown, surgical cap, protective FFP2 mask (fitting the metal nose clip), goggles over the mask straps, inner latex gloves, outer latex gloves layer, covering over the gown’s cuff, and shoe covers.6 A second operator helps in closing the gown back and checks that the dressing procedure is carried out correctly.', 'cite_spans': [{'start': 63, 'end': 64, 'mention': '5', 'ref_id': None}, {'start': 603, 'end': 604, 'mention': '6', 'ref_id': None}], 'section': 'Personal Protective Equipment ::: STAFF TRAINING', 'ref_spans': []}, {'text': 'The proper PPE removal sequence should be performed to avoid self-contamination. The cleaning of visible dirt, if present, followed by hand hygiene with alcohol-based disinfectant, the removal of shoe covers, followed by hand hygiene, and removal of the outer pairs of gloves, followed by hand hygiene take place in the CT or X-ray unit rooms7; the used PPE items are put in biohazard containers for special medical waste.', 'cite_spans': [{'start': 342, 'end': 343, 'mention': '7', 'ref_id': None}], 'section': 'Personal Protective Equipment ::: STAFF TRAINING', 'ref_spans': []}, {'text': 'The second part of decontamination and removal of PPE take place in a second dedicated room with direct access from COVID-19 Radiology and involve a similar sequence of steps: taking off the disposable gown with the help of an assistant, hand hygiene, removal of the goggles or face shield, hand hygiene, taking off medical protective masks, hand hygiene, taking off the cap and of inner gloves, and final hand-washing.6 A second operator helps the first one in the gown removal and checks that the undressing steps are carried out correctly.', 'cite_spans': [{'start': 419, 'end': 420, 'mention': '6', 'ref_id': None}], 'section': 'Personal Protective Equipment ::: STAFF TRAINING', 'ref_spans': []}, {'text': 'Because a certain division between truly positive and negative cases at patient arrival is impossible due to the presence of COVID-19 positive but asymptomatic patients, and considering the long incubation period of the disease in both radiology units, the staff should consider all of the patients as potentially COVID-19 positive until proven otherwise and, therefore, use the highest level available of PPE against droplets and contact transmission.', 'cite_spans': [], 'section': 'Personal Protective Equipment ::: STAFF TRAINING', 'ref_spans': []}, {'text': 'Checkpoints for temperature measurements of all outpatients and staff have been set at the entrances of the hospital. Signs to remember the respect of the safety distance and the mandatory use of surgical masks were hung at the entrances and in all waiting rooms.', 'cite_spans': [], 'section': 'Patients’ Entrance ::: STAFF TRAINING', 'ref_spans': []}, {'text': 'As already suggested,7,8 the creation of 2 separate pathways for COVID-19 positive (and suspected positive) and COVID-19 negative patients represents the best solution to avoid the spread of the infection. We divided our Radiology Department into a dedicated COVID-19 radiology unit and a COVID-19-free radiology unit.', 'cite_spans': [{'start': 21, 'end': 22, 'mention': '7', 'ref_id': None}, {'start': 23, 'end': 24, 'mention': '8', 'ref_id': None}], 'section': 'Creation of Distinct Routes and Radiology Unit Reconfiguration ::: STAFF TRAINING', 'ref_spans': []}, {'text': 'The division of patients begins in the triage area of the emergency department (ED), where trained staff aim to select subjects with high or low probability to have COVID-19 infection, according to their symptoms, clinical parameters, and history of exposure. Therefore, 2 distinct waiting areas for patients with and without respiratory symptoms and fever have been created.', 'cite_spans': [], 'section': 'Creation of Distinct Routes and Radiology Unit Reconfiguration ::: STAFF TRAINING', 'ref_spans': []}, {'text': 'Regardless of the level of suspicion, each patient receives a surgical face mask at entrances of the ED and the Radiology Department.', 'cite_spans': [], 'section': 'Creation of Distinct Routes and Radiology Unit Reconfiguration ::: STAFF TRAINING', 'ref_spans': []}, {'text': 'From the ED, health care assistants with appropriate PPE bring patients to perform the required radiological examinations in the 2 radiology units with distinct paths, according to the assigned risk. The 2 paths are indicated by signs on the wall and are separated by barriers.', 'cite_spans': [], 'section': 'Creation of Distinct Routes and Radiology Unit Reconfiguration ::: STAFF TRAINING', 'ref_spans': []}, {'text': 'The COVID-19 Radiology Unit is located in the basement of the building where the ED and intensive and subintensive COVID-19 units are located and accessible from a dedicated elevator.9 This unit includes a CT scanner, 2 X-ray machines, 1 ultrasound machine, and rooms dedicated to personnel, 1 for wearing the PPE and 1 for decontamination, with removing PPE items, which are put into special waste bins. These were originally reading rooms that have been converted.', 'cite_spans': [{'start': 183, 'end': 184, 'mention': '9', 'ref_id': None}], 'section': 'Creation of Distinct Routes and Radiology Unit Reconfiguration ::: STAFF TRAINING', 'ref_spans': []}, {'text': 'According to the American College of Radiology recommendations,10 portable radiography machines have been positioned in the ED and in COVID-19 units to perform chest X-rays when a lung evaluation is medically needed, reducing patient handling risks. The activity should work 24/7; therefore, a reorganization of the radiological staff is needed.', 'cite_spans': [{'start': 63, 'end': 65, 'mention': '10', 'ref_id': None}], 'section': 'Creation of Distinct Routes and Radiology Unit Reconfiguration ::: STAFF TRAINING', 'ref_spans': []}, {'text': 'For the execution of chest CT, the presence of 2 RTs represents the best solution11,12: 1 RT with PPE sets up the patient positioning on the scanner table, whereas a second RT with a surgical mask and gloves operates the machine console. Disposable surgical sheets are employed to cover the CT scanner table to avoid contacts between the patient and the CT equipment. In contrast-enhanced CT, the medical radiologist and nurse wear PPE for direct assistance to the patient.', 'cite_spans': [{'start': 81, 'end': 83, 'mention': '11', 'ref_id': None}, {'start': 84, 'end': 86, 'mention': '12', 'ref_id': None}], 'section': 'Creation of Distinct Routes and Radiology Unit Reconfiguration ::: STAFF TRAINING', 'ref_spans': []}, {'text': 'Two RTs are dedicated to the execution of X-rays: 1 wears PPE for direct assistance, for positioning the patient or the X-ray cassette behind the patient’s back and to center the patient; and the other RT, equipped with a surgical mask and gloves, sets the machine (exposure data), delivers X-rays, and checks the images to be sent to Picture Archiving and Communication System (PACS).', 'cite_spans': [], 'section': 'Creation of Distinct Routes and Radiology Unit Reconfiguration ::: STAFF TRAINING', 'ref_spans': []}, {'text': 'The non-COVID-19 patients admitted in the ED have a separate route with access to the clean Radiology Unit, dedicated also to non-COVID-19 inpatients and outpatients.', 'cite_spans': [], 'section': 'Creation of Distinct Routes and Radiology Unit Reconfiguration ::: STAFF TRAINING', 'ref_spans': []}, {'text': 'All RTs and health care assistants have also been trained adequately on the disinfection procedures that need to be applied to each machine. After each examination, the CT suite is closed for the next hour to exchange the room air and to allow a specifically trained health care assistant, protected with PPE, to perform the removal of the disposable sheet and the cleaning of the CT gantry and all contact surfaces with a cloth soaked with alcohol-based disinfectants (75% ethanol). Disinfection of the radiography equipment is performed between uses by RTs (protected with PPE) wiping the surface using 75% ethanol. After each examination, the X-ray equipment, the contact surfaces of the CT gantry and table, and CT console rooms are sanitized with alcohol-based disinfectants.', 'cite_spans': [], 'section': 'Disinfection Procedures ::: STAFF TRAINING', 'ref_spans': []}, {'text': 'It is likely that radiology departments will always have a key role in the diagnosis and follow-up of infectious diseases and in the infection control, on account of a timely imaging diagnosis.13,14 The experience gained in this emergency must not only help establish a rapid reconfiguration of the units, when needed, to minimize in-hospital transmission, but also spread a culture of infection control practice to the radiological and the whole hospital staff. The collaboration among the staff is essential to get the right safety measures within the Radiology Department and the whole hospital. Development and continuous implementation of strict procedures can provide protection against the transmission of any viral agent.', 'cite_spans': [{'start': 193, 'end': 195, 'mention': '13', 'ref_id': None}, {'start': 196, 'end': 198, 'mention': '14', 'ref_id': None}], 'section': 'CONCLUSION', 'ref_spans': []}, {'text': 'The reorganization of our Radiology Department, with the creation of 2 distinct pathways, could be a possible way to face the COVID-19 outbreak. Dedicated radiology units and dedicated radiology equipment can reduce the risk of infection transmission and contribute in the successful management of this emergency.', 'cite_spans': [], 'section': 'CONCLUSION', 'ref_spans': []}, {'text': '\\nCOVID-19novel coronavirus diseaseCTcomputed tomographyEDemergency departmentFFP2filtering face-piece respiratorPPEpersonal protective equipmentRTradiology technologist\\n', 'cite_spans': [], 'section': 'Abbreviations', 'ref_spans': []}], 'paragraphs': [{'text': 'Because a certain division between truly positive and negative cases at patient arrival is impossible due to the presence of COVID-19 positive but asymptomatic patients, and considering the long incubation period of the disease in both radiology units, the staff should consider all of the patients as potentially COVID-19 positive until proven otherwise and, therefore, use the highest level available of PPE against droplets and contact transmission.', 'id': '00006'}], 'title': 'How to Reorganize the Radiology Departments to Face the 2019 Coronavirus Disease Outbreak', 'sha': 'deec036df7cfcac7c2b29ccb8fbfd3dfd00b4af2', 'doi': '10.1017/dmp.2020.159', 'score': 8.671600341796875}, {'rank': 297, 'paragraphs': [{'text': ' Evidence from more chains of transmission can better delineate the borderline between the latent period and infectious period.', 'id': '00016'}], 'title': 'Pre‐symptomatic transmission of novel coronavirus in community settings', 'sha': '33d4618670f943618b03739410e3a7b023c7d9d9', 'doi': '10.1111/irv.12773', 'score': 8.669899940490723}, {'rank': 298, 'abstract': 'Abstract From 71 laboratory-confirmed cases, we found three presymptomatic patients and 10 entirely asymptomatic infections. In two of the three incubation period patients, the viral titer in the presymptomatic period was very high (Ct value <20). The median days to first negative RT-PCR in the entirely asymptomatic carriers was 4.5 (range 2.5–9) days and all asymptomatic carriers reached a first Ct>35 RT-PCR within 14 days after diagnosis. Patients who have COVID-19 may already be infectious before there are symptoms, and 14 days of isolation after diagnosis may be sufficient in entirely asymptomatic cases.', 'body_text': [{'text': 'J o u r n a l P r e -p r o o f An outbreak of a novel human coronavirus, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), was first detected in Wuhan, China in December 2019 (Lu et al., 2020) and became a public health emergency of international concern. The World Health Organization defines coronavirus disease 2019 (COVID-19) as the infectious disease caused by laboratory confirmed SARS-CoV-2, with real-time reverse transcription polymerase chain reaction (RT-PCR) (WHO, 2020). COVID-19 can present as an asymptomatic carrier state, acute respiratory disease, or pneumonia. Some studies found that people infected with SARS-CoV-2 in the asymptomatic or presymptomatic period can be infectious (Bai et al., 2020 , Mao et al., 2020 . Current evidence on how long to quarantine asymptomatic carriers is limited. In recent paper, Mao was suspicious about the 14-day isolation period for asymptomatic carriers because it was based only on observations of two cases (Mao et al., 2020) . The shortage of isolation capacity in some countries is expected to lead to an explosive increase in COVID-19.', 'cite_spans': [{'start': 187, 'end': 204, 'text': '(Lu et al., 2020)', 'ref_id': 'BIBREF0'}, {'start': 711, 'end': 728, 'text': '(Bai et al., 2020', 'ref_id': None}, {'start': 729, 'end': 747, 'text': ', Mao et al., 2020', 'ref_id': 'BIBREF3'}, {'start': 978, 'end': 996, 'text': '(Mao et al., 2020)', 'ref_id': 'BIBREF3'}], 'ref_spans': [], 'section': ''}, {'text': 'Here, we report the viral kinetics according to absence or presence of symptoms.', 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': 'Our study included laboratory-confirmed COVID-19 patients admitted to Affiliated Hospitals of Chonnam National University between February 04 and April 07, 2020. RT-PCR was positive if the cycle threshold (Ct) value of both the envelope (E) and RNA-dependent RNA polymerase (RdRp) genes of SARS-CoV-2 were less than 35. Specimens were collected from all patients at least 2 days after hospitalization and physicians checked their symptoms and signs daily. Epidemiological and clinical information were obtained.', 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': 'Seventy-one patients were hospitalized for treatment or isolation. We analyzed patients who had any of the following features at quarantine release: (1) asymptomatic carrier (n=10) who had no symptoms in the 14 days preceding diagnosis until quarantine release; (2) incubation period patients (n=3) who were asymptomatic during the 14 days preceding diagnosis, but became symptomatic during quarantine. Table 1 shows the clinical J o u r n a l P r e -p r o o f characteristics of the patients analyzed. The median age of the asymptomatic carriers was 31 (interquartile range 17.8-55.8) years. Three patients who were asymptomatic on admission developed myalgia, fever, and cough 1 or 2 days afterwards without pneumonia. In two of the three patients, the viral load in the incubation period was very high (Ct value <20), but all patients first reached Ct>35 by day 14 (Fig 1A) . In the asymptomatic COVID-19 carriers, the viral load on admission was not relatively high (Fig 1B) . The median time to first Ct >35 RT-PCR result after diagnosis in asymptomatic carriers was 4.5 (2.0-8.5) days and all asymptomatic cases first reached a Ct >35 result within 14 days after diagnosis (Fig 1B) .', 'cite_spans': [{'start': 262, 'end': 265, 'text': '(2)', 'ref_id': 'BIBREF1'}], 'ref_spans': [{'start': 403, 'end': 410, 'text': 'Table 1', 'ref_id': 'TABREF0'}, {'start': 868, 'end': 876, 'text': '(Fig 1A)', 'ref_id': 'FIGREF1'}, {'start': 970, 'end': 978, 'text': '(Fig 1B)', 'ref_id': 'FIGREF1'}, {'start': 1179, 'end': 1187, 'text': '(Fig 1B)', 'ref_id': 'FIGREF1'}], 'section': ''}, {'text': 'We found that RT-PCR was indeterminate or negative 14 days after diagnosis in entirely asymptomatic individuals who were not given any antiviral agents. Although the presence of viral RNA in specimens doesnot distinguish between infective and non-infective PCR was positive on the 19th day of isolation in patients receiving antiviral agents, which might have prolonged virus detection in the two asymptomatic carriers. A modelling study estimated that 18% would be true asymptomatic cases, including the incubation period (Mizumoto et al., 2020) . If the percentage of asymptomatic carriers is large, the cost of isolation for public health management may be enormous. Our data suggest that 14-days quarantine at home after diagnosis may be adequate in entirely asymptomatic carriers.', 'cite_spans': [{'start': 523, 'end': 546, 'text': '(Mizumoto et al., 2020)', 'ref_id': 'BIBREF6'}], 'ref_spans': [], 'section': ''}, {'text': 'We found that incubation-period patients had high viral loads before the onset of symptoms, and that some asymptomatic carriers had an initial viral load that might be a living virus. Interestingly, this differs from the transmission of SARS-CoV, which occurred mainly J o u r n a l P r e -p r o o f ', 'cite_spans': [], 'ref_spans': [], 'section': ''}], 'paragraphs': [{'text': 'J o u r n a l P r e -p r o o f An outbreak of a novel human coronavirus, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), was first detected in Wuhan, China in December 2019 (Lu et al., 2020) and became a public health emergency of international concern. The World Health Organization defines coronavirus disease 2019 (COVID-19) as the infectious disease caused by laboratory confirmed SARS-CoV-2, with real-time reverse transcription polymerase chain reaction (RT-PCR) (WHO, 2020). COVID-19 can present as an asymptomatic carrier state, acute respiratory disease, or pneumonia. Some studies found that people infected with SARS-CoV-2 in the asymptomatic or presymptomatic period can be infectious (Bai et al., 2020 , Mao et al., 2020 . Current evidence on how long to quarantine asymptomatic carriers is limited. In recent paper, Mao was suspicious about the 14-day isolation period for asymptomatic carriers because it was based only on observations of two cases (Mao et al., 2020) . The shortage of isolation capacity in some countries is expected to lead to an explosive increase in COVID-19.', 'id': '00001'}], 'title': 'Viral kinetics of SARS-CoV-2 in asymptomatic carriers and presymptomatic patients', 'sha': '9e7db18df3cbdfef8b00ece6c4c6f74151a83262', 'doi': '10.1016/j.ijid.2020.04.083', 'score': 8.664899826049805}, {'rank': 299, 'paragraphs': [{'text': \"Recent meta-reviews estimate the basic reproduction number R 0 for COVID-19 in the range of 2-3 and the incubation period as 4-8 days [37, 38] . Accordingly, we use R 0 = 3 and a recovery rate of µ = 1/(8d), close to values previously used for the analysis of the disease's spread in Germany [15] .\", 'id': '00052'}], 'title': 'COVID-19 lockdown induces structural changes in mobility networks -- Implication for mitigating disease dynamics', 'sha': 'e45405c41ee94b1e5d4507ea11b73150768380b7', 'doi': '', 'score': 8.659500122070312}, {'rank': 300, 'abstract': 'Abstract Since the first cases were detected in China in December 2019, the COVID-19 pandemic has spread rapidly, collapsing many healthcare systems, and forcing them to adapt. Hand surgery has been indirectly affected by this scenario. This article aims to provide an overview on how Spanish hand surgeons have modified their daily practice. Based on a survey conducted nationwide, we observed a decrease in the number of emergency cases and cancellation of elective cases, shift to a more conservative treatment approach, use of personal protective equipment, and decrease in the number of outpatient visits and tests. Without definitive evidence at this point, knowing how we have dealt with the situation so far will help us adopt the needed measures to ensure both the patient’s and surgeon’s safety and manage available resources in future pandemics.', 'body_text': [{'text': 'On December 31, 2019, the Wuhan Municipal Health Commission (Hubei province, China) reported a cluster of 27 pneumonia cases of unknown etiology but with common exposure, including seven severe cases [1] . The agent causing the outbreak was identified as a new type of Coronaviridae, subsequently named SARS-CoV-2. This virus and its associated disease (the so called COVID- 19) have triggered a dramatic and rapidly evolving health emergency around the world. On March 11, 2020 , the WHO declared a global pandemic [2] . As of May 12, 2020, 4,178,750 cases have been registered worldwide. In Spain, a state of alarm was declared on March 14 [3] , and when this article was submitted, it had the second-highest number of coronavirus cases in the world: 228,030 had been diagnosed, and 26,920 deaths had been reported.', 'cite_spans': [{'start': 200, 'end': 203, 'text': '[1]', 'ref_id': None}, {'start': 375, 'end': 378, 'text': '19)', 'ref_id': 'BIBREF27'}, {'start': 464, 'end': 478, 'text': 'March 11, 2020', 'ref_id': None}, {'start': 516, 'end': 519, 'text': '[2]', 'ref_id': None}, {'start': 642, 'end': 645, 'text': '[3]', 'ref_id': 'BIBREF11'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'SARS-CoV-2 is highly contagious [4] ; it causes an illness that can affect the lungs and airways, potentially escalating to severe acute respiratory syndrome [5] . The ease of spread and severity of the disease translates into a high hospital and intensive care unit (ICU) admission rate. This situation has overloaded many healthcare systems that have been forced to increase the number of ICU beds and create specific units to isolate COVID-19 patients. To this end, different measures have been taken following guidelines derived from the experience in China and Italy [6, 7, 8] such as the cancellation of elective surgeries, the transformation of surgical wards into COVID-19 wards, the use of surgical areas as ICUs, the conversion of surgeons into supporting staff for COVID-19 units, or the closing of outpatient clinics. These measures and the new social situation derived from the population lockdown have collided with the usual activity of surgical departments, who have had to adapt to this situation.', 'cite_spans': [{'start': 32, 'end': 35, 'text': '[4]', 'ref_id': 'BIBREF12'}, {'start': 158, 'end': 161, 'text': '[5]', 'ref_id': 'BIBREF13'}, {'start': 572, 'end': 575, 'text': '[6,', 'ref_id': 'BIBREF14'}, {'start': 576, 'end': 578, 'text': '7,', 'ref_id': 'BIBREF15'}, {'start': 579, 'end': 581, 'text': '8]', 'ref_id': 'BIBREF16'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'Regarding hand surgery, each country has taken different approaches depending on their own pandemic situation and available resources [9, 10] . To our knowledge, there are no published studies on how the peak of COVID-19 pandemic Page 5 of 15 J o u r n a l P r e -p r o o f has affected hand surgery in Spain. We believe that with a global view of the situation, we will be able to improve the current and future care of patients with hand injuries.', 'cite_spans': [{'start': 134, 'end': 137, 'text': '[9,', 'ref_id': 'BIBREF17'}, {'start': 138, 'end': 141, 'text': '10]', 'ref_id': 'BIBREF18'}], 'ref_spans': [], 'section': 'Introduction'}, {'text': 'We prepared an online questionnaire (Google Forms; Alphabet Co., Mountain View, CA) https://forms.gle/sxw1U4zTLH6XmmXY7 that was distributed online with the help of the Spanish Society of Hand Surgery (SECMAmore than 400 members) to orthopedic and plastic surgeons who are currently performing hand surgery in Spain.', 'cite_spans': [], 'ref_spans': [], 'section': 'Materials and methods'}, {'text': 'The questionnaire was disseminated among surgeons of all levels of expertise [11] and was completed between April 17 and April 28, 2020. It focused on the management of acute hand cases, obviating the elective cases as most of them were cancelled nationwide. Various topics were covered: level of activity during COVID-19 pandemic, activity prior to the pandemic, protective methods used during surgery, potential changes in surgical indication, hospital admission, transfers between centers, and changes in follow-up protocols. When asking about the use of COVID-19 tests, we did not make a difference between swab and blood tests.', 'cite_spans': [{'start': 77, 'end': 81, 'text': '[11]', 'ref_id': 'BIBREF19'}], 'ref_spans': [], 'section': 'Materials and methods'}, {'text': \"At the end of the survey's period, a total of 111 replies were analyzed, duplicate answers were manually screened and deleted, and statistical analysis was carried out.\", 'cite_spans': [], 'ref_spans': [], 'section': 'Materials and methods'}, {'text': \"Categorical variables were expressed as percentages, while differences between groups were assessed using the Chi-square test. Before and after comparisons were made using McNemar's test. Statistical analyses were performed using R software (version 3.6.1., R Foundation for Statistical Computing, Vienna, Austria).\", 'cite_spans': [], 'ref_spans': [], 'section': 'Materials and methods'}, {'text': 'After manually removing duplicates, a total of 108 surgeons from 64 centers from all over Spain had answered the questionnaire (Table 1) (45%) and follow-up visits (81%) was also found. Some changes in follow-up methods were also considered and 81% started using telemedicine for follow-up visits, while 14% of the surgeons increased their use of it. Among centers which often transfer patients or receive patients from other centers, 73% of those surveyed observed a reduction in these transfers. When a patient was COVID-19 positive, a more conservative approach (i.e., managing fractures non operatively that in normal times would be operated on) was adopted by 48% of surgeons.', 'cite_spans': [], 'ref_spans': [{'start': 127, 'end': 136, 'text': '(Table 1)', 'ref_id': None}], 'section': 'Results'}, {'text': 'No major differences were detected when comparing this conservative approach between more experienced (>10 years of experience) and less experienced surgeons (<10 years), as older surgeons adopted a more conservative approach in 53% of cases, and younger surgeons in 37% (p=0.124). This trend towards conservative management was also observed in the percentage of closed fractures that were managed non-operatively after the COVID-19 pandemic started (Fig. 3) . In spite of this more conservative approach, replantation criteria did not change for 75% of the surgeons, although a more restrictive approach was seen in 24% of them. Other changes were noted in surgical indications: 24% of surgeons performed fewer flaps for soft tissue coverage, and 64% reduced the use of implanted devices for non-elective', 'cite_spans': [], 'ref_spans': [{'start': 451, 'end': 459, 'text': '(Fig. 3)', 'ref_id': 'FIGREF5'}], 'section': 'Results'}, {'text': 'cases.', 'cite_spans': [], 'ref_spans': [], 'section': 'Results'}, {'text': 'An increase in the use of regional or local anesthesia instead of general anesthesia was noted by 67% of the surgeons. And minor procedures were performed outside the operating room by 46% to free up the OR.', 'cite_spans': [], 'ref_spans': [], 'section': 'Results'}, {'text': 'Whenever surgery could be delayed at least a few hours, 82% of the surveyed would ask for a COVID-19 test for the patient. When the surgery could be delayed a few days, this test was ordered 24 hours before surgery by 60% and 48 hours by 32% (Fig. 4) .', 'cite_spans': [], 'ref_spans': [{'start': 242, 'end': 250, 'text': '(Fig. 4)', 'ref_id': 'FIGREF6'}], 'section': 'Results'}, {'text': 'We are all facing a new disease and a new healthcare scenario worldwide. In this situation, there is global consensus that any treatment other than urgent or emergent care should be avoided in order to reduce the rate of new case development, minimize resource exhaustion, and conserve disposable medical supplies [7, 12] . At first glance, the current pandemic does not directly affect hand surgeons; however, surgical safety has fallen under the spotlight and surgeons play a role in the continuation of emergency surgical care as well as the protection of medical staff and patients [9, 13] .', 'cite_spans': [{'start': 314, 'end': 317, 'text': '[7,', 'ref_id': 'BIBREF15'}, {'start': 318, 'end': 321, 'text': '12]', 'ref_id': 'BIBREF20'}, {'start': 586, 'end': 589, 'text': '[9,', 'ref_id': 'BIBREF17'}, {'start': 590, 'end': 593, 'text': '13]', 'ref_id': 'BIBREF21'}], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'As shown in this survey, Spanish hand surgeons have adapted to the situation by protecting themselves in the operating room, being more conservative in their indications and protecting patients from exposure by reducing the number of visits, as shown in other articles [9, 10, 14] .', 'cite_spans': [{'start': 269, 'end': 272, 'text': '[9,', 'ref_id': 'BIBREF17'}, {'start': 273, 'end': 276, 'text': '10,', 'ref_id': 'BIBREF18'}, {'start': 277, 'end': 280, 'text': '14]', 'ref_id': 'BIBREF22'}], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'Protecting healthcare workers should be a priority when facing any pandemic.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'Up to 20.4% of COVID-19 cases in Spain have been detected in healthcare workers [15] . It is known that COVID-19 has a long incubation period (5-7 days) [4, 16, 17] , resulting in many asymptomatic carriers [18] . For this reason, appropriate measures should also be taken in asymptomatic patients. Most of the surgeons who responded to the questionnaire stated that surgical patients underwent pre-surgery screening to determine their risk of COVID-19 infection whenever the procedure could be postponed [19, 20] . Contrary to other surgeries, hand surgery can often be delayed at least a few hours in order to get the test done before proceeding with surgery.', 'cite_spans': [{'start': 80, 'end': 84, 'text': '[15]', 'ref_id': 'BIBREF23'}, {'start': 153, 'end': 156, 'text': '[4,', 'ref_id': 'BIBREF12'}, {'start': 157, 'end': 160, 'text': '16,', 'ref_id': None}, {'start': 161, 'end': 164, 'text': '17]', 'ref_id': 'BIBREF25'}, {'start': 207, 'end': 211, 'text': '[18]', 'ref_id': 'BIBREF26'}, {'start': 505, 'end': 509, 'text': '[19,', 'ref_id': 'BIBREF27'}, {'start': 510, 'end': 513, 'text': '20]', 'ref_id': 'BIBREF28'}], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'Our systems are generally well-designed to deal with the occasional high-risk case, but general measures should be adopted during a pandemic [19] . Operating rooms are high risk areas for SARS-CoV-2 transmission that require specific protective measures. Emergency surgery and surgery that cannot be delayed should only proceed with adequate staffing and personal protective equipment (PPE) [7] , as used anesthesia. More widespread use of local or regional anesthesia was documented, which helps avoid airway instrumentation and the patient coughing during intubation and extubation, thus reducing the risk of infecting staff via the associated aerosol generation [20] . This would also theoretically reduce postoperative pulmonary complications in COVID-19 patients [21] . We have seen an increase in minor surgical procedures, such as wound debridement and closure, done as outpatient procedures in the emergency department to decrease the use of operation rooms. This means decreasing the number of transfers inside the hospital and decreasing the risk of contagion for both patients and workers. Orthopedic procedures using saws, drills and electrocautery are also aerosol generating; although not proven, their use for implant placement should be reduced and power settings should be as low as possible [22, 23] .', 'cite_spans': [{'start': 141, 'end': 145, 'text': '[19]', 'ref_id': 'BIBREF27'}, {'start': 391, 'end': 394, 'text': '[7]', 'ref_id': 'BIBREF15'}, {'start': 665, 'end': 669, 'text': '[20]', 'ref_id': 'BIBREF28'}, {'start': 768, 'end': 772, 'text': '[21]', 'ref_id': 'BIBREF29'}, {'start': 1309, 'end': 1313, 'text': '[22,', 'ref_id': 'BIBREF30'}, {'start': 1314, 'end': 1317, 'text': '23]', 'ref_id': 'BIBREF31'}], 'ref_spans': [], 'section': 'Discussion'}, {'text': \"Most surgeons use FFP2/3 masks as well as protective goggles/screens and reinforced gowns when operating whether the patient is confirmed positive for COVID-19 or not. Risk stratification of patients is important to avoid over-depletion of PPE supplies [24] by using them mainly when the patient is a confirmed or suspected case of COVID-19. The use of essential PPE can be challenging to hand surgeons because it can physically interfere with the magnification loupes or the microscope's bifocal eye piece [10, 25] . When the patient is COVID-19 positive, the risks and benefits of proceeding versus postponing surgery should be weighed [19] ; this has apparently led to a more conservative management of these patients in Spain. We should keep in mind that this conservative approach might imply an acceptance of potential suboptimal clinical outcomes to decrease the risk of COVID-19 infection [12] .\", 'cite_spans': [{'start': 253, 'end': 257, 'text': '[24]', 'ref_id': 'BIBREF32'}, {'start': 507, 'end': 511, 'text': '[10,', 'ref_id': 'BIBREF18'}, {'start': 512, 'end': 515, 'text': '25]', 'ref_id': 'BIBREF33'}, {'start': 638, 'end': 642, 'text': '[19]', 'ref_id': 'BIBREF27'}, {'start': 897, 'end': 901, 'text': '[12]', 'ref_id': 'BIBREF20'}], 'ref_spans': [], 'section': 'Discussion'}, {'text': \"The safest place for patients during the COVID-19 pandemic is not in the hospital; measures aimed at decreasing the length of hospital stay as well as the number of follow-up visits have been taken by Spanish hand surgeons [26] . There has J o u r n a l P r e -p r o o f been a reduction in the number of visits to the outpatient clinic for follow-up and a reduction in the number of follow-up tests such as X-rays, to decrease the patient's exposure to densely packed waiting rooms.\", 'cite_spans': [{'start': 223, 'end': 227, 'text': '[26]', 'ref_id': None}], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'Although non-urgent outpatient clinic visits have been postponed until the situation improves, we still must ensure that we maintain the quality of care expected by our patients. This can be done using novel technologies [13] , including the adoption of telemedicine for follow-up visits. However, these novel technologies are not standardized, and their long-term efficacy are not proven. A mid-and long-term study to analyze follow-up strategies and functional outcomes of these patients would be useful so we can adapt our healthcare systems to high demand situations like this pandemic.', 'cite_spans': [{'start': 221, 'end': 225, 'text': '[13]', 'ref_id': 'BIBREF21'}], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'Spain has imposed a lockdown on its citizens that resulted in a decrease in the number of patients going to the emergency departments for hand trauma, both due to a decrease in manufacturing activity and a fear of going to the hospital for minor injuries.', 'cite_spans': [], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'This marked decrease in the number of patients in some hospitals that decreased their number of daily hand emergency cases to a point where there were zero hand emergency cases on some days. This situation is not uniform, as emergency cases have not decreased in other countries [10] .', 'cite_spans': [{'start': 279, 'end': 283, 'text': '[10]', 'ref_id': 'BIBREF18'}], 'ref_spans': [], 'section': 'Discussion'}, {'text': 'This study is based on the personal opinion of surgeons who replied to the questionnaire and numbers such as reduction in cases or number of cases treated non-operatively have not been calculated. One of the main drawbacks of this study is that it lacks accurate data and does not necessarily reflect the official protocols adopted in each center. It reflects the situation during the peak of the pandemic in Spain (officially 3 rd of April), capturing a moment in time where most of the attention was focused on treating COVID-19 patients. How the disease will behave in the midand long-term is still unclear, and surgical policies will have to change as the pandemic disease burden evolves. To our knowledge, this is the first study to capture hand surgery practice in Spain during the pandemic and all regions of the country were As of the date of submission of this article, there were no set protocols for how surgical departments should adapt to an unusual situation like this pandemic. The studies needed to obtain such evidence would require lengthy follow-up and a vast number of patients and professionals. Although reliable information on the potential risk of contagion during surgery is not available, this does not mean that it is safe to proceed with usual practices. A multicenter study with accurate data from the different units across the country as well as a mid-and long-term follow-up of these patients would provide us with better information on how we have dealt with the disease and if we should change anything before facing a similar situation again in the future. This will help us define the measures needed to ensure both the patient\\'s and surgeon\\'s safety and manage available resources in future pandemics. Fig. 1 . Decrease in the number of urgent cases seen daily relative to a hospital\\'s previous case load. \"Pre\" shows the number of patients seen before the pandemic, and \"Post\" shows the number of patients seen after the pandemic started. Small (< 5 cases per day), medium (6-10 cases per day) and large (>10 cases per day) units are shown (p<0.001). ', 'cite_spans': [], 'ref_spans': [{'start': 1739, 'end': 1745, 'text': 'Fig. 1', 'ref_id': None}], 'section': 'Discussion'}], 'paragraphs': [{'text': 'Up to 20.4% of COVID-19 cases in Spain have been detected in healthcare workers [15] . It is known that COVID-19 has a long incubation period (5-7 days) [4, 16, 17] , resulting in many asymptomatic carriers [18] . For this reason, appropriate measures should also be taken in asymptomatic patients. Most of the surgeons who responded to the questionnaire stated that surgical patients underwent pre-surgery screening to determine their risk of COVID-19 infection whenever the procedure could be postponed [19, 20] . Contrary to other surgeries, hand surgery can often be delayed at least a few hours in order to get the test done before proceeding with surgery.', 'id': '00016'}], 'title': 'An overview of the situation of hand surgery in Spain during the peak of COVID-19 pandemic', 'sha': '91415b52471af11251ea6c600c39f7bf65442430', 'doi': '10.1016/j.hansur.2020.06.002', 'score': 8.658499717712402}, {'rank': 301, 'abstract': 'Since first reported in December 2019, the novel RNA betacoronavirus disease (COVID‐19) has spread rapidly across the world with, as of the 3rd May 2020, 3.3 million confirmed cases with more than 230,000 deaths worldwide and more than 200 countries affected. The most common symptoms reported include dry cough, fever and fatigue with ground‐glass opacification bilaterally in the lungs on imaging and lymphopenia on haematological analysis. In severe cases the patients can develop acute respiratory distress syndrome (ARDS) or multi organ failure which can lead to death. Transmission of the virus (SARS‐CoV‐2) is via direct contact or via droplets spread by a cough or sneeze from an infected individual. The difficulty in disease detection and containment has been the long course of COVID‐19. Current evidence shows an incubation period of up to 14 days post exposure to the virus, and on average most patients first display symptoms on day five after initial exposure. More importantly, carriers are contagious even during this preceding asymptomatic incubation phase of the disease.', 'body_text': [{'text': 'cases with more than 230,000 deaths worldwide and more than 200 countries affected. 1 The most common symptoms reported include dry cough, fever and fatigue with ground-glass opacification bilaterally in the lungs on imaging and lymphopenia on haematological analysis. In severe cases the patients can develop acute respiratory distress syndrome (ARDS) or multi organ failure which can lead to death. 2 ', 'cite_spans': [{'start': 84, 'end': 85, 'text': '1', 'ref_id': None}, {'start': 401, 'end': 402, 'text': '2', 'ref_id': 'BIBREF1'}], 'ref_spans': [], 'section': ''}, {'text': 'CoV-2) is via direct contact or via droplets spread by a cough or sneeze from an infected individual. The difficulty in disease detection and containment has been the long course of COVID-19. Current evidence shows an incubation period of up to 14 days post exposure to the virus, and on average most patients first display symptoms on day five after initial exposure. More importantly, carriers are contagious even during this preceding asymptomatic incubation phase of the disease. 2, 4 In the United Kingdom (UK), a general lockdown was introduced on the 23rd March 2020 as the major public health measure to slow viral transmission. Even before then, hospitals in the National Health Service (NHS) had been preparing for a surge in demand by redeploying All patients had standard pre-COVID-19 pre-operative assessments with no COVID-19 swabs taken preceding surgery. In addition, a HHFT standardised screening consent form for COVID-19 symptoms were included as part of the pre assessment ( Figure 1 ) and patients were excluded if any symptoms were present.', 'cite_spans': [{'start': 484, 'end': 486, 'text': '2,', 'ref_id': 'BIBREF1'}, {'start': 487, 'end': 488, 'text': '4', 'ref_id': 'BIBREF3'}], 'ref_spans': [{'start': 995, 'end': 1003, 'text': 'Figure 1', 'ref_id': 'FIGREF2'}], 'section': ',3 Transmission of the virus (SARS-'}, {'text': 'A consecutive series of patients listed for operations during the first three weeks since the UK lockdown were identified for prospective follow up.', 'cite_spans': [], 'ref_spans': [], 'section': ',3 Transmission of the virus (SARS-'}, {'text': 'Standard operating procedures were followed as per the Public Health England (PHE) COVID-19 infection control policy. 6 During the operation, standard Personal Protective Equipment (PPE) for aerosol generating procedures were used by the anaesthetics team.', 'cite_spans': [{'start': 118, 'end': 119, 'text': '6', 'ref_id': 'BIBREF5'}], 'ref_spans': [], 'section': ',3 Transmission of the virus (SARS-'}, {'text': 'The theatre team used standard PPE for non-aerosol generating procedures.', 'cite_spans': [], 'ref_spans': [], 'section': ',3 Transmission of the virus (SARS-'}, {'text': 'During the postoperative period, all patients were recovered on designated \"COVID-19', 'cite_spans': [], 'ref_spans': [], 'section': ',3 Transmission of the virus (SARS-'}, {'text': 'negative\" surgical wards in multi-patient shared bays.', 'cite_spans': [], 'ref_spans': [], 'section': ',3 Transmission of the virus (SARS-'}, {'text': 'After 14 days, all patients were contacted via a telephone consultation to assess for any potential postoperative or COVID-19 complications.', 'cite_spans': [], 'ref_spans': [], 'section': ',3 Transmission of the virus (SARS-'}, {'text': 'A total of 52 consecutive patients underwent a urological operation between 23rd March and 9th April 2020. The average age was 66 (range 22 -89). 45 (86·5%) patients were male. SIx', 'cite_spans': [], 'ref_spans': [], 'section': ',3 Transmission of the virus (SARS-'}, {'text': 'This article is protected by copyright. All rights reserved ', 'cite_spans': [], 'ref_spans': [], 'section': 'Accepted Article'}, {'text': 'This article is protected by copyright. All rights reserved However our cohort included five cases of laparoscopic radical nephrectomies which are counted as high risk major surgeries in urology. Furthermore, by the lockdown stage in the UK, the awareness of COVID-19 within the medical and general population had increased, with social distancing in the community already in place by this stage which may have decreased the incidence of infection within the community. In addition, PPE advice from PHE had already been implemented by this stage to prevent transmission both from patient to staff as well as the other way round. However as of the 3rd May 2020, there have been 2827 confirmed cases of COVID-19 in the Hampshire Upper Tier Local Authority area. 7 This is the third highest number of cases in England, with an estimated overall local population of around 1·3 million people. 8 Therefore the disease by the lockdown stage was likely already prevalent in the local community, and in particular the actual number of COVID-19 cases will be higher than the official confirmed cases as people with COVID-19 symptoms who did not require hospital admission will have neither been swabbed nor counted.', 'cite_spans': [], 'ref_spans': [], 'section': 'Accepted Article'}, {'text': 'Cases were performed where possible as either a day-case or short inpatient admission in order to limit patient exposure to the clinical environment, with over 90% of cases discharged home within 48 hours. One patient stayed longer than 73 hours and had a 50 day length of stay. He was admitted with urosepsis following a collapse at home and has known advanced prostate cancer with bilateral stents in situ. An emergency stent change was performed on day 45 of admission due to bacterial colonisation and the patient was discharged on day five post-operatively after appropriate social care support was arranged.', 'cite_spans': [], 'ref_spans': [], 'section': 'Accepted Article'}, {'text': 'There were two complications discovered in the postoperative period, both were of urinary tract infection symptoms post endoscopic procedure, which were treated in the community with a standard seven day course of oral antibiotics which fully resolved the symptoms. This is within the normal parameters expected post general endoscopic urological operations where the risk of infections are quoted as between 10 -50%.', 'cite_spans': [], 'ref_spans': [], 'section': 'Accepted Article'}, {'text': 'Our study supports the continued delivery of appropriately triaged urgent urological surgery during the COVID-19 pandemic. A thorough assessment accounting for the risks and benefits for each case is necessary, and as the duration of the pandemic progresses, the ability to continue delivering urgent surgical care will greatly help mitigate the longer term impact on delaying treatment and meet the urgent needs of non-COVID-19 patients.', 'cite_spans': [], 'ref_spans': [], 'section': 'Accepted Article'}, {'text': 'This article is protected by copyright. All rights reserved', 'cite_spans': [], 'ref_spans': [], 'section': 'Accepted Article'}, {'text': 'LP and BY designed the study, collected and analysed the data. All authors participated in data collection. LP and BY drafted the manuscript. RH and TN revised the final manuscript. ', 'cite_spans': [], 'ref_spans': [], 'section': 'Contributors'}], 'paragraphs': [{'text': 'CoV-2) is via direct contact or via droplets spread by a cough or sneeze from an infected individual. The difficulty in disease detection and containment has been the long course of COVID-19. Current evidence shows an incubation period of up to 14 days post exposure to the virus, and on average most patients first display symptoms on day five after initial exposure. More importantly, carriers are contagious even during this preceding asymptomatic incubation phase of the disease. 2, 4 In the United Kingdom (UK), a general lockdown was introduced on the 23rd March 2020 as the major public health measure to slow viral transmission. Even before then, hospitals in the National Health Service (NHS) had been preparing for a surge in demand by redeploying All patients had standard pre-COVID-19 pre-operative assessments with no COVID-19 swabs taken preceding surgery. In addition, a HHFT standardised screening consent form for COVID-19 symptoms were included as part of the pre assessment ( Figure 1 ) and patients were excluded if any symptoms were present.', 'id': '00002'}], 'title': 'Delivering urgent urological surgery during the COVID‐19 pandemic in the United Kingdom: Outcomes from our initial 52 patients.', 'sha': '3221d5a662ad8d97228b5a340252ca1854fe7629', 'doi': '10.1111/bju.15110', 'score': 8.640899658203125}, {'rank': 302, 'abstract': \"Abstract Hong Kong has been recently attacked by the coronavirus disease-2019 (COVID-19). In late January 2020, it's shown a steadily increasing trend of confirmed cases. There is a 257 in total infected cases confirmed including 4 deaths until 20th of March 2020. To prevent further outbreak of COVID-19, this article discusses the current understanding of COVID-19 and compares with the outbreak of SARS-CoV-2 in 2003 of Hong Kong from the causes, transmission, symptoms, diagnosis, treatments and preventions to study for an applicable measurement to control COVID-19.\", 'body_text': [{'text': 'An outbreak of pneumonia with unknown etiology emerged in Wuhan of Hubei Province, China on December in 2019. Chinese scientists confirmed that a new coronavirus named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is the cause of pneumonia outbreak. Now, this severe acute respiratory syndrome has been termed as coronavirus disease 2019 (COVID-19) by the World Health of Organization (WHO) (Peng et al., 2020).', 'cite_spans': [{'start': 407, 'end': 424, 'mention': 'Peng et al., 2020', 'ref_id': 'BIBREF57'}], 'section': 'Introduction', 'ref_spans': []}, {'text': \"Coronavirus is a group of viruses categorized into alphacoronavirus and betacoronavirus often causing cold and other mild upper respiratory tract infections in the human body. Alphacoronavirus consists of HCoV-229E and HCoV-NL63; Betacoronavirus consists of HCoV-OC43 and HCoV-HKU1. Nevertheless, the rare form would be lethal such as SARS, MERS and COVID-19 (Geller et al., 2012). SARS-CoV, MERS-CoV and SARS-CoV-2 are betacoronavirus (Gorbalenya et al., 2020). However, SARSr-CoVs belong to the subgenus Sarbecovirus (previously lineage B) of genus Betacoronavirus and occupy a unique phylogenetic position. It's the most lethal (Guan et al., 2020).\", 'cite_spans': [{'start': 360, 'end': 379, 'mention': 'Geller et al., 2012', 'ref_id': 'BIBREF31'}, {'start': 437, 'end': 460, 'mention': 'Gorbalenya et al., 2020', 'ref_id': 'BIBREF32'}, {'start': 632, 'end': 649, 'mention': 'Guan et al., 2020', 'ref_id': 'BIBREF34'}], 'section': 'Introduction', 'ref_spans': []}, {'text': 'On the 20th of March 2020, 178 countries or territories reported that there are 234073 in the total confirmed case with 9840 deaths globally; 81300 confirmed cases and 3253 deaths reported in China (Coronavirus disease, 2020); 257 confirmed case with 4 deaths in Hong Kong (CHP, 2020a). In response to the outbreak, we summarize the current knowledge of COVID-19 and compare it with the previous experiences of the SARS outbreak in Hong Kong to study for an effective measurement to control the COVID-19 epidemic.', 'cite_spans': [{'start': 199, 'end': 224, 'mention': 'Coronavirus disease, 2020', 'ref_id': 'BIBREF27'}, {'start': 274, 'end': 284, 'mention': 'CHP, 2020a', 'ref_id': 'BIBREF14'}], 'section': 'Introduction', 'ref_spans': []}, {'text': '\\nSARS was caused by a coronavirus (CoV) and identified in 2003 according to the World Health Organization (WHO). Coronavirus (CoV) is an animal virus from bats which infected human in Guangdong province, China (Peiris et al., 2003, Poon et al., 2003).', 'cite_spans': [{'start': 211, 'end': 230, 'mention': 'Peiris et al., 2003', 'ref_id': 'BIBREF56'}, {'start': 232, 'end': 249, 'mention': 'Poon et al., 2003', 'ref_id': 'BIBREF59'}], 'section': 'Causes', 'ref_spans': []}, {'text': \"Hong Kong's SARS index-patient was Dr. LIU Jianlun in February 2003, who came to attend a family wedding gathering. He was a Provinciale doctor contacting and treating SARS patients from Sun Yat-Sen Memorial Hospital in Guangdong (Hughes, 2006). On 21st of February 2003, Dr. LIU and his wife checked into room 911 on the 9th floor of Metropole Hotel in Kowloon Hong Kong. He was a SARS super-spreader infected 23 other hotel guests from 7th to the 9th floor. The hotel guests traveled to Canada, Singapore, Taiwan and Vietnam transmitting SARS to these locations. Dr. LIU was hospitalized in Kwong Wah Hospital of Hong Kong, he never recovered and died in the Intensive Care Unit on 4th of March, while he caused around 80% of the Hong Kong cases (Hanage, 2010, BWHO, 2011).\", 'cite_spans': [{'start': 231, 'end': 243, 'mention': 'Hughes, 2006', 'ref_id': 'BIBREF43'}, {'start': 749, 'end': 761, 'mention': 'Hanage, 2010', 'ref_id': 'BIBREF36'}, {'start': 763, 'end': 773, 'mention': 'BWHO, 2011', 'ref_id': 'BIBREF7'}], 'section': 'Causes', 'ref_spans': []}, {'text': '\\nCOVID-19 is similar to SARS, also caused by a coronavirus (CoV) that occurred in the seafood and wet animal wholesale market in Wuhan, Hubei Province, China (Bogoch et al., 2020, Lu et al., 2020).', 'cite_spans': [{'start': 159, 'end': 178, 'mention': 'Bogoch et al., 2020', 'ref_id': 'BIBREF5'}, {'start': 180, 'end': 195, 'mention': 'Lu et al., 2020', 'ref_id': 'BIBREF52'}], 'section': 'Causes', 'ref_spans': []}, {'text': \"On 22th of January 2020, the first COVID-19 case was confirmed in Hong Kong and announced by the Centre of Health Protection (CHP). However, no index patient has been identified at present. Initially, most of the confirmed cases had a travel history to Mainland China by High-Speed Rail, especially from Wuhan. Only a confirmed case had no recent travel history on 5th of February 2020 (CHP, 2020b, CHP, 2020c, CHP, 2020d, CHP, 2020e, CHP, 2020f, CHP, 2020g, CHP, 2020h). There was an increase in the largest number of 10 confirmed cases per day from 9th of February 2020. In the Chinese lunar new year family gathering with two relatives from Guangdong Province, China (Chan et al., 2004). 11 of 19 participants were diagnosed with COVID-19 infection. 2 other confirmed cases were the colleagues of the infected participants (CHP, 2020i, CHP, 2020j, CHP, 2020k). Besides, the largest community spreading was reported at Fook Wai Ching She from Maylun Apartments in North Point Hong Kong on 4th of March. 18 patients in total were tested with a positive result of COVID-19 who's visited the temple or association with the confirmed cases in January or February (CHP, 2020h). The accumulated number of cases was raised to 257 with a steadily increasing number of infected cases in Hong Kong on 20th of March 2020 (CHP, 2020a).\", 'cite_spans': [{'start': 387, 'end': 397, 'mention': 'CHP, 2020b', 'ref_id': 'BIBREF15'}, {'start': 399, 'end': 409, 'mention': 'CHP, 2020c', 'ref_id': 'BIBREF16'}, {'start': 411, 'end': 421, 'mention': 'CHP, 2020d', 'ref_id': 'BIBREF17'}, {'start': 423, 'end': 433, 'mention': 'CHP, 2020e', 'ref_id': 'BIBREF18'}, {'start': 435, 'end': 445, 'mention': 'CHP, 2020f', 'ref_id': 'BIBREF19'}, {'start': 447, 'end': 457, 'mention': 'CHP, 2020g', 'ref_id': 'BIBREF20'}, {'start': 459, 'end': 469, 'mention': 'CHP, 2020h', 'ref_id': 'BIBREF21'}, {'start': 671, 'end': 688, 'mention': 'Chan et al., 2004', 'ref_id': 'BIBREF10'}, {'start': 827, 'end': 837, 'mention': 'CHP, 2020i', 'ref_id': 'BIBREF22'}, {'start': 839, 'end': 849, 'mention': 'CHP, 2020j', 'ref_id': 'BIBREF23'}, {'start': 851, 'end': 861, 'mention': 'CHP, 2020k', 'ref_id': 'BIBREF24'}, {'start': 1162, 'end': 1172, 'mention': 'CHP, 2020h', 'ref_id': 'BIBREF21'}, {'start': 1313, 'end': 1323, 'mention': 'CHP, 2020a', 'ref_id': 'BIBREF14'}], 'section': 'Causes', 'ref_spans': []}, {'text': 'A total of 1718 infected cases appeared in Hong Kong from 11th of March to 6th of June 2003 (Lee, 2003a). The SARS epidemic had three phases from March to June. First, the outbreak in Prince Wales Hospital was in March because of the local infected person admission while the spreading of SARS infection affected a large number of medical staffs and students; Second outbreak related to the spread from hospital to the community on 15th of April 2003 which influenced block E of Amoy Garden with a total 329 estate residents infected cases; Third phase was from the early of May to the mid of June 2003, continuous new cases were reported but in a declining trend. According to the previous studied, SARS induced 253 dead in Hong Kong (Lee, 2003b), it was approximately 15% dead rate ranging from 1% in those aged less than 24 years to over 50% in those aged over 65 years within this period (Abdullah et al., 2004).', 'cite_spans': [{'start': 93, 'end': 103, 'mention': 'Lee, 2003a', 'ref_id': 'BIBREF48'}, {'start': 736, 'end': 746, 'mention': 'Lee, 2003b', 'ref_id': 'BIBREF49'}, {'start': 893, 'end': 914, 'mention': 'Abdullah et al., 2004', 'ref_id': 'BIBREF0'}], 'section': 'The trend of SARS outbreak', 'ref_spans': []}, {'text': 'In Hong Kong, there were 257 infected cases from 23th of January to 20th of March 2020. It consisted of 140 males and 117 females within ten age groups from 0 to 100 years old. The age and gender distribution of infected patients were shown in Figure 1\\n(CHP, 2020a).', 'cite_spans': [{'start': 254, 'end': 264, 'mention': 'CHP, 2020a', 'ref_id': 'BIBREF14'}], 'section': 'The trend of COVID-19 outbreak', 'ref_spans': [{'start': 244, 'end': 252, 'mention': 'Figure 1', 'ref_id': 'FIGREF0'}]}, {'text': 'The identity of confirmed cases were divided into three categories, 9 (3%) person of unknown (waiting for confirmation), 10 (4%) of Non-Hong Kong residents and 238 (93%) of Hong Kong residents (Figure 2\\n) (CHP, 2020a).', 'cite_spans': [{'start': 206, 'end': 216, 'mention': 'CHP, 2020a', 'ref_id': 'BIBREF14'}], 'section': 'The trend of COVID-19 outbreak', 'ref_spans': [{'start': 194, 'end': 202, 'mention': 'Figure 2', 'ref_id': 'FIGREF1'}]}, {'text': 'There were only a few infection cases from 23th of January to 8th of February. However, the infection cases rose quickly on the 9th of February as the Hong Kong boundary control points such as Lo Wu, Lok Ma Chau, Lok Ma Chau Spur Line and Macau Ferry Terminal were not closed at the beginning for reducing the flow of people and traffic (MBPC, 2020). The reported cases were increased sharply from 23rd of January to 20th of March especially after 17th of March which was double per day (Figure 3\\n) possibly because some Hong Kong residents returned to Hong Kong from overseas countries with infected COVID-19 (CHP, 2020a).', 'cite_spans': [{'start': 338, 'end': 348, 'mention': 'MBPC, 2020', 'ref_id': 'BIBREF53'}, {'start': 611, 'end': 621, 'mention': 'CHP, 2020a', 'ref_id': 'BIBREF14'}], 'section': 'The trend of COVID-19 outbreak', 'ref_spans': [{'start': 488, 'end': 496, 'mention': 'Figure 3', 'ref_id': 'FIGREF2'}]}, {'text': 'On 12th of February, the first infected person was discharged and the accumulated discharged persons were 97 at present (Figure 4\\n).', 'cite_spans': [], 'section': 'The trend of COVID-19 outbreak', 'ref_spans': [{'start': 121, 'end': 129, 'mention': 'Figure 4', 'ref_id': 'FIGREF3'}]}, {'text': \"\\nSARS-CoV is most readily transmitted by respiratory droplets when an infected person coughs or sneezes within a short distance. The virus could also spread when a person touches a surface or object contaminated with infectious droplets then touched his/her mouth, nose or eyes (SARS, 2017). The average incubation period for SARS within human is 4 to 6 days, although rarely it can be as short as 1 day or as long as 14 days (SARS, 2003). SARS patients secret the virus excretion in respiratory and stool during the second week of illness and start to deteriorate. It resulted in more than 8000 persons suffered from Severe Acute Respiratory Syndrome (SARS) and caused 774 person's death in 2002–2003 (SARS, 2015).\", 'cite_spans': [{'start': 279, 'end': 289, 'mention': 'SARS, 2017', 'ref_id': 'BIBREF62'}, {'start': 427, 'end': 437, 'mention': 'SARS, 2003', 'ref_id': 'BIBREF60'}, {'start': 703, 'end': 713, 'mention': 'SARS, 2015', 'ref_id': 'BIBREF61'}], 'section': 'Transmission', 'ref_spans': []}, {'text': '\\nSARS-CoV-2 was indicated as animal-to-human transmission because of the linkage between fish and wild animal market based on the early studied (Li et al., 2020a). However, there is a rapidly increasing incidence of infections by asymptomatic carriers (Chang et al., 2020, Carlos et al., 2020), therefore, SARS-CoV-2 may have human-to-human transmission through droplets or direct contact (Zhao et al., 2020, Biscayart et al., 2020).', 'cite_spans': [{'start': 145, 'end': 161, 'mention': 'Li et al., 2020a', 'ref_id': 'BIBREF50'}, {'start': 253, 'end': 271, 'mention': 'Chang et al., 2020', 'ref_id': 'BIBREF11'}, {'start': 273, 'end': 292, 'mention': 'Carlos et al., 2020', 'ref_id': 'BIBREF8'}, {'start': 390, 'end': 407, 'mention': 'Zhao et al., 2020', 'ref_id': 'BIBREF74'}, {'start': 409, 'end': 431, 'mention': 'Biscayart et al., 2020', 'ref_id': 'BIBREF4'}], 'section': 'Transmission', 'ref_spans': []}, {'text': \"Compared to SARS-CoV, SARS-CoV-2 might be transmitted by respiratory droplets within a long distance of 2m or by contaminated surfaces leading to infection through contact transmission (Peng et al., 2020). The average incubation period for COVID-19 expands to 5 days and the period of quarantine is 14 days from the last date of exposure. It was the longest incubation period for similar types of coronaviruses. However, it's remained argumentative since the longest incubation period for COVID-19 can be 27 days or above according to the present-day finding (CIP, 2020). Perhaps, respiratory droplets were not the major factor for transmission (nCoV, 2020). Recently, there is a patient dog infected from its nasal and oral cavity with a low level of COVID-19 virus. It's not shown any signs of disease before. This is the first case of human-to-animal transmission that has been confirmed by the experts from School of Public Health in the Hong Kong University, College of Veterinary Medicine & Life Sciences in the City University of Hong Kong and the World Organization for Animal Health (COVID, 2020).\", 'cite_spans': [{'start': 186, 'end': 203, 'mention': 'Peng et al., 2020', 'ref_id': 'BIBREF57'}, {'start': 560, 'end': 569, 'mention': 'CIP, 2020', 'ref_id': 'BIBREF26'}, {'start': 646, 'end': 656, 'mention': 'nCoV, 2020', 'ref_id': 'BIBREF54'}, {'start': 1093, 'end': 1104, 'mention': 'COVID, 2020', 'ref_id': 'BIBREF28'}], 'section': 'Transmission', 'ref_spans': []}, {'text': 'The symptoms of SARS and COVID-19 are divided into systematic and respiratory disorders (Table 1\\n). Systematic disorders of SARS and COVID-19 are fever, cough and fatigue. The major respiratory disorders of COVID-19 and SARS are rhinorrhea, sneezing, sore throat and pneumonia but COVID-19 patients have more respiratory symptoms than the patients with SARS (SARS, 2017, Carlos et al., 2020, Chen et al., 2020, Wang et al., 2020a, Huang et al., 2020). Leukopenia, lymphopenia and lower platelet count are possibly observed in the hematology (Yang et al., 2020, Chen et al., 2006).', 'cite_spans': [{'start': 359, 'end': 369, 'mention': 'SARS, 2017', 'ref_id': 'BIBREF62'}, {'start': 371, 'end': 390, 'mention': 'Carlos et al., 2020', 'ref_id': 'BIBREF8'}, {'start': 392, 'end': 409, 'mention': 'Chen et al., 2020', 'ref_id': 'BIBREF13'}, {'start': 411, 'end': 429, 'mention': 'Wang et al., 2020a', 'ref_id': 'BIBREF69'}, {'start': 431, 'end': 449, 'mention': 'Huang et al., 2020', 'ref_id': 'BIBREF42'}, {'start': 542, 'end': 559, 'mention': 'Yang et al., 2020', 'ref_id': 'BIBREF73'}, {'start': 561, 'end': 578, 'mention': 'Chen et al., 2006', 'ref_id': 'BIBREF12'}], 'section': 'Symptoms', 'ref_spans': [{'start': 89, 'end': 96, 'mention': 'Table 1', 'ref_id': 'TABREF0'}]}, {'text': 'Reverse transcription polymerase chain reaction(RT-PCR) is the golden test for the diagnosis of SARS and COVID-19. Computed Tomography (CT) scan is the auxiliary diagnostic method. The images of CT sometimes appear bilateral pulmonary parenchymal ground-glass and consolidative pulmonary opacities with a rounded morphology and a peripheral lung distribution when patients infected with SARS and COVID-19 (Li et al., 2020b).', 'cite_spans': [{'start': 406, 'end': 422, 'mention': 'Li et al., 2020b', 'ref_id': 'BIBREF51'}], 'section': 'Diagnosis', 'ref_spans': []}, {'text': \"RT-PCR is a technique that combines RNA reverse transcription with polymerase chain amplification (PCR) of cDNA. It's the rapid and specific diagnostic test for SARS following the protocols of the World Health Organization (WHO) and commonly used in the SARS outbreak 2003. Respiratory specimen, stool specimen and urine specimen could be used for RT-PCR testing while respiratory specimens were collected in the SARS epidemic (Yam et al., 2005). The peak detection period in SARS-CoV after illness: respiratory specimen is week 2; stool and rectal swab specimens are weeks 2 to 3 and urine specimen is week 4 (Chan et al., 2004).\", 'cite_spans': [{'start': 428, 'end': 444, 'mention': 'Yam et al., 2005', 'ref_id': 'BIBREF71'}, {'start': 611, 'end': 628, 'mention': 'Chan et al., 2004', 'ref_id': 'BIBREF10'}], 'section': 'SARS', 'ref_spans': []}, {'text': \"Real-time quantitative polymerase chain reaction (RT-qPCR) is the most common and effective nucleic acid detection technologies for SARS-CoV-2. Meanwhile, high-throughput sequencing technology has been applied for diagnosis. It's limited usage because of the high cost. Immune identification technologies such as Point-of-care Testing (POCT) of IgM/IgG and enzyme-linked immunosorbent assay (ELISA) are other possible diagnostic methods for COVID-19 but remain to be studied (Li et al., 2020b).\", 'cite_spans': [{'start': 476, 'end': 492, 'mention': 'Li et al., 2020b', 'ref_id': 'BIBREF51'}], 'section': 'COVID-19', 'ref_spans': []}, {'text': 'The combination regimen of steroids and ribavirin was the treatment protocols for SARS in Hong Kong (So et al., 2003).', 'cite_spans': [{'start': 101, 'end': 116, 'mention': 'So et al., 2003', 'ref_id': 'BIBREF63'}], 'section': 'SARS ::: Treatments', 'ref_spans': []}, {'text': 'Corticosteroid is commonly used as an immunomodulatory agent to modulate inflammatory cytokines to achieve immune homeostasis (Yam et al., 2007). Steroid with high dosage was used to treat patients with SARS (Ho et al., 2003).', 'cite_spans': [{'start': 127, 'end': 143, 'mention': 'Yam et al., 2007', 'ref_id': 'BIBREF72'}, {'start': 209, 'end': 224, 'mention': 'Ho et al., 2003', 'ref_id': 'BIBREF40'}], 'section': 'SARS ::: Treatments', 'ref_spans': []}, {'text': 'Ribavirin can suppress the replication of RNA virus. It has been used in treating hepatitis C virus, lassa fever virus and severe respiratory syncytial virus infection (Lauer and Walker, 2001). However, the usage of ribavirin did not reduce the intra-tracheal intubation or mortality rates (Booth et al., 2003). It has significant toxicity which may cause hemolytic anemia and electrolyte disturbances (Lauer and Walker, 2001).', 'cite_spans': [{'start': 169, 'end': 191, 'mention': 'Lauer and Walker, 2001', 'ref_id': 'BIBREF47'}, {'start': 291, 'end': 309, 'mention': 'Booth et al., 2003', 'ref_id': 'BIBREF6'}, {'start': 403, 'end': 425, 'mention': 'Lauer and Walker, 2001', 'ref_id': 'BIBREF47'}], 'section': 'SARS ::: Treatments', 'ref_spans': []}, {'text': 'Protease inhibitor for treating human immunodeficiency virus (HIV) infection has also been considered as a treatment for SARS. Lopinavir-ritonavir coformulation (Kaletra) blocks the virus replicase polyprotein and prevents the replication of RNA. This is able to reduce the intubation and mortality rates especially administered in the early stage with ribavirin and corticosteroid (Sung, 2003). Glycyrrhizin that can inhibit viral absorption and penetration has been found to be effective against SARS-CoV during and after the viral adsorption period (Cinatl et al., 2003). The Interferons are partly effective against coronaviruses but the efficacy is not accurately tested (Pitkäranta et al., 1999). Gamma immunoglobulins are used in SARS but the effectiveness remains to be evaluated (Fujii et al., 2004).', 'cite_spans': [{'start': 383, 'end': 393, 'mention': 'Sung, 2003', 'ref_id': 'BIBREF64'}, {'start': 553, 'end': 572, 'mention': 'Cinatl et al., 2003', 'ref_id': 'BIBREF25'}, {'start': 677, 'end': 700, 'mention': 'Pitkäranta et al., 1999', 'ref_id': 'BIBREF58'}, {'start': 789, 'end': 807, 'mention': 'Fujii et al., 2004', 'ref_id': 'BIBREF30'}], 'section': 'SARS ::: Treatments', 'ref_spans': []}, {'text': 'Broad spectrum antibiotics such as fluoroquinolone or beta-lactams and macrolide are also utilized to treat nonspecific features and assisted ventilation is commonly used for respiratory failure in SARS patients (Fujii et al., 2004).', 'cite_spans': [{'start': 213, 'end': 231, 'mention': 'Fujii et al., 2004', 'ref_id': 'BIBREF30'}], 'section': 'SARS ::: Treatments', 'ref_spans': []}, {'text': 'Broad spectrum antiviral drugs have been used in treating COVID-19 infection e.g oseltamivir, lopinavir, ritonavir and ganciclovir regimen; remdesivir and chloroquine regimen (Chen et al., 2020). The invention of a specific antiviral drug for COVID-19 is still progressing. Potential compounds are available for further investigation such as EIDD-2801 (isopropyl ester of N4-hydroxycytidine), Sofosbuvir, IDX-184, Ribavirin and Remidisvir may be considered as its high therapeutic potential in treating emerged viral infection (Wang et al., 2020b, Elfiky, 2020). Meanwhile, there are not only the antiviral drugs needed for an infected patient but also the prophylaxis e.g. vaccine is required for the uninfected person especially for the healthcare workers. Antibiotics and ventilation support are also utilized in COVID19 patients (Yang et al., 2020).', 'cite_spans': [{'start': 176, 'end': 193, 'mention': 'Chen et al., 2020', 'ref_id': 'BIBREF13'}, {'start': 528, 'end': 546, 'mention': 'Wang et al., 2020b', 'ref_id': 'BIBREF70'}, {'start': 548, 'end': 560, 'mention': 'Elfiky, 2020', 'ref_id': 'BIBREF29'}, {'start': 834, 'end': 851, 'mention': 'Yang et al., 2020', 'ref_id': 'BIBREF73'}], 'section': 'COVID-19 ::: Treatments', 'ref_spans': []}, {'text': 'The exercise capacity and health status of SARS recoverees were significantly lower than the general population. In the early rehabilitation phase, the short and weak breath caused limitations in physical function because of reduced lung function with patchy changes. A minority of survivors had a mild decrease in carbon monoxide diffusing capacity (Hui et al., 2009a). Steroid resulted in some psycho-behavioral problems such as anxiety and depression, it might be improved gradually in most of the patients (Chan et al., 2003). However, high dose corticosteroid treatment induced avascular necrosis of the joint and hip which further decreases the health status and mobility (Hui et al., 2009b).', 'cite_spans': [{'start': 351, 'end': 368, 'mention': 'Hui et al., 2009a', 'ref_id': 'BIBREF44'}, {'start': 511, 'end': 528, 'mention': 'Chan et al., 2003', 'ref_id': 'BIBREF9'}, {'start': 679, 'end': 696, 'mention': 'Hui et al., 2009b', 'ref_id': 'BIBREF45'}], 'section': 'SARS ::: Sequela', 'ref_spans': []}, {'text': 'Some patients recovered have 20–30% decrease of the lung capacity because of the reveal irreversible fibrosis (Barbara Walsh, 2020). This causes lung tissue scarring and stiffening as well as the immune system damage. Lung permanent damage may be leading to severe respiratory issues such as acute respiratory distress syndrome (ARDS) (Bell, 2020). Younger patients have a better chance of recovery but patients within 40 to 50 years old only have little chance and difficulty back to their pre-illness status.', 'cite_spans': [{'start': 111, 'end': 130, 'mention': 'Barbara Walsh, 2020', 'ref_id': 'BIBREF1'}, {'start': 336, 'end': 346, 'mention': 'Bell, 2020', 'ref_id': 'BIBREF3'}], 'section': 'COVID-19 ::: Sequela', 'ref_spans': []}, {'text': 'On 12th of March 2003, the Department of Health in Hong Kong alerted Word Health Organization (WHO) for the first notable suspected infection condition in Prince of Wales Hospital (PWH) ward 8A for handling the confirmed cases. The department of Food and Health Secretary started daily press briefings on the latest SARS condition and government measures for informing the public and increasing the effectiveness of tracing close contact person since 16th of March 2003. In the mid of April 2003, a temperature check was required for a person who arriving in Hong Kong to detect any suspected case of SARS (Tomlinson and Cockram, 2003).', 'cite_spans': [{'start': 607, 'end': 634, 'mention': 'Tomlinson and Cockram, 2003', 'ref_id': 'BIBREF67'}], 'section': 'Confirmed cases ::: SARS ::: Preventions', 'ref_spans': []}, {'text': 'On 31st of March 2003, all close contactors of SARS were mandated to report daily and designated medical centers by Department of Health for close contacted handling. Home quarantine measure extend to close contactors of a suspected case to minimize the opportunities of transmission for the virus since 25th of April 2003. Due to the increasing report of a confirmed case in block E Amoy Garden, isolation camp was opened for the evacuation of block E Amoy Gardens residents in April 2003 (Pang et al., 2003).', 'cite_spans': [{'start': 491, 'end': 508, 'mention': 'Pang et al., 2003', 'ref_id': 'BIBREF55'}], 'section': 'Close contacted handling ::: SARS ::: Preventions', 'ref_spans': []}, {'text': 'On 27th of March 2003, Hong Kong government announced the suspension of classes in all schools from 29th of March to 6th of April 2003 to prevent clustering. While there was no visiting policy extending to all acute ward in HA hospital for preventing further close contact cases. On 2nd of April 2003, the Word Health Organization (WHO) issued a travel advisory recommended to postpone for all non-essential travel to Hong Kong and Guangdong. Education for the use of face mask, hand hygiene, disinfection of surfaces for fomites, avoiding contact with SARS patients and their body fluids, covering your mouth by tissue when coughing or sneezing and other simple hygiene measures were provided by the government to prevent further community outbreak (Bell, 2004).', 'cite_spans': [{'start': 751, 'end': 761, 'mention': 'Bell, 2004', 'ref_id': 'BIBREF2'}], 'section': 'Community prevention ::: SARS ::: Preventions', 'ref_spans': []}, {'text': 'Healthcare workers were advised to use personal protective equipment (PPE) including N95 masks for an aerosol generated procedure, face shield or goggles, gloves and isolation gowns with strict infection control to prevent the droplet and contact transmission. During the SARS period, health workers worked in a pair for mutually monitoring the infection control implement, especially wearing and removing the PPE. When transferring the confirmed patient, advise the patient putting on the mask to prevent the spread of droplets with viruses. Daily self-monitoring of fever and any other symptoms of SARS were implemented in Hospital Authority for early detection of SARS infection. The high-risk areas such as A&E, ICU and Isolation ward were adequately ventilated with negative pressure to prevent infection of the health workers (Lau et al., 2004).', 'cite_spans': [{'start': 833, 'end': 849, 'mention': 'Lau et al., 2004', 'ref_id': 'BIBREF46'}], 'section': 'Protection on healthcare workers ::: SARS ::: Preventions', 'ref_spans': []}, {'text': \"On 8th of January 2020, Centre for Health Protection added “Severe respiratory disease associated with a new infectious agent” as the notifiable diseases in Hong Kong which gained the authority on quarantine for handling the confirmed case. All confirmed cases should stay in the hospital for isolation and treatment until fully recovered. Suspected cases were also required to isolate in the hospital until a negative result of SARS-CoV-2 observed. To increase the effectiveness of detecting high risk cases on admission to the hospital in early February 2020, the patient's travel history was connected from the immigration department and shown in the hospital authority computer system (Clinical Management System). Temperature check maintained the routine check for person arriving in Hong Kong. The Health declaration form should be submitted compulsorily by paper or electronic for all travellers who arrived at Hong Kong International Airport from 8th of March 2020. Private doctors were allowed to collect a respiratory specimen for the symptomatic patient to send for viral testing at accredited private laboratories on 9th of March 2020 (HKSAR, 2020). Started from 20th March, 2020, the Hospital Authority (HA) established 2 temporary COVID-19 virus test centers at the AsiaWorld-Expo and the North Lantau Hospital for people arriving in Hong Kong with upper respiratory symptoms to undergo viral test and to wait for the laboratory results (Temporary test, 2020).\", 'cite_spans': [{'start': 1148, 'end': 1159, 'mention': 'HKSAR, 2020', 'ref_id': 'BIBREF39'}, {'start': 1452, 'end': 1472, 'mention': 'Temporary test, 2020', 'ref_id': 'BIBREF65'}], 'section': 'Confirmed cases ::: COVID 19 ::: Preventions', 'ref_spans': []}, {'text': 'All visitors arrived at Hong Kong who has been to Hubei Province in the past 14 days must be transferred to 3 quarantine centers (Chai Wan Lei Yue Mun Park and Holiday Village, Jao Tsung-I Academy Heritage Lodge, Chun Yeung Estate) (HKSAR, 2019). On 19th of March 2020, all people arriving in Hong Kong who have been to Iran, Daegu and Gyeongsangbuk-do in Korea, Emilia-Romagna, Lombardy and Veneto regions in Italy within the past 14 days have to stay in a quarantine center for quarantine (Guidelines on Prevention, 2019). Compulsory home or accommodation quarantine for 14 days implemented to people who arrived in Hong Kong from Mainland China (but not include Hubei Province) on 8th February 2020 has been extended to people who arrived to Hong Kong from all any other overseas countries and area except Macau, Hong Kong and Taiwan since 19th of March 2020 (HKSAR, 2019). Home or accommodation quarantine for 14 days, daily body temperature, sign and symptoms monitoring are required and electronic wristbands are also worn to monitor the location of who have been isolated (Thematic Website, 2020).', 'cite_spans': [{'start': 233, 'end': 244, 'mention': 'HKSAR, 2019', 'ref_id': 'BIBREF38'}, {'start': 492, 'end': 522, 'mention': 'Guidelines on Prevention, 2019', 'ref_id': 'BIBREF35'}, {'start': 863, 'end': 874, 'mention': 'HKSAR, 2019', 'ref_id': 'BIBREF38'}, {'start': 1080, 'end': 1102, 'mention': 'Thematic Website, 2020', 'ref_id': 'BIBREF66'}], 'section': 'Close contacted handling ::: COVID 19 ::: Preventions', 'ref_spans': []}, {'text': 'In the late of January government gradually declared to close all kindergarten, primary schools, secondary schools and special schools for community prevention; all amusement parks in Hong Kong including Hong Kong Disneyland Resort, Ocean Park Hong Kong and Madame Tussauds Hong Kong; all facilities of the Leisure and Cultural Services Department (LCSD) including public museums, public libraries, sports centers and venue were also closed to prevent clustering. Home office for working was advised. Special arrangements for the Hong Kong Diploma of Secondary Education examination, such as widen seating plan, compulsory use of mask and temperature measuring will be implemented to prevent cross infection. On 17th of January 2020, Hong Kong Government prohibited the entry of non-Hong Kong residents who have visited the Hubei Province in the past 14 days since 27th of January 2020 (Thematic Website, 2020). The High-Speed Rail train services have been suspended from 29th of January 2020 and the cross border train station has been suspended from 4th of February 2020 to prevent close contacted high-risk cases to Hong Kong. On 17th of March 2020, the Hong Kong Security Bureau extended the red outbound travel alert to all overseas countries/territories based on public health concerns (Government issues Red, 2020). Hospital Authority (HA) raised the emergency response level to emergency with no visiting, exceptions will be made for clinical consideration and suspended volunteer services or clinical attachment in all public hospitals to prevent further close contact cases on 25th of January (Transport Department, 2020, Hospital Authority, 2020).', 'cite_spans': [{'start': 887, 'end': 909, 'mention': 'Thematic Website, 2020', 'ref_id': 'BIBREF66'}, {'start': 1293, 'end': 1320, 'mention': 'Government issues Red, 2020', 'ref_id': 'BIBREF33'}, {'start': 1604, 'end': 1630, 'mention': 'Transport Department, 2020', 'ref_id': 'BIBREF68'}, {'start': 1632, 'end': 1656, 'mention': 'Hospital Authority, 2020', 'ref_id': 'BIBREF41'}], 'section': 'Community prevention ::: COVID 19 ::: Preventions', 'ref_spans': []}, {'text': 'Every person in a public hospital was required to wear a surgical mask for preventing cross contamination with the emergency response level. Same to SARS conditions, the hospital authority advised using personal protective equipment (PPE) as discussed above. A special allowance as housing allowance (HK$500/day) was provided for the healthcare staff working in high-risk areas for the purpose of renting accommodation for self-isolation to prevent close contact with their family so as to prevent from spreading to the community (Healthcare staff, 2020).', 'cite_spans': [{'start': 531, 'end': 553, 'mention': 'Healthcare staff, 2020', 'ref_id': 'BIBREF37'}], 'section': 'Protection on healthcare workers ::: COVID 19 ::: Preventions', 'ref_spans': []}, {'text': 'The causes, transmissions, symptoms and preventive measures of SARS and COVID-19 are reviewed above. However, diagnosis and treatments remain for further study. Polymerase chain reaction for confirming the interaction of SARS or COVID-19 sometimes may show a false negative. There is no specific treatment regimen, only has a broad spectrum of antiviral drugs for COVID-19 patients and no vaccine for prevention at present. Nowadays, personal hygiene and protection are the most important for preventing the spread of COVID-19 such as wearing a mask and washing hands as well as reducing social contact including avoiding crowds, working in home, so on.', 'cite_spans': [], 'section': 'Conclusion', 'ref_spans': []}, {'text': 'The authors received no specific funding for this work.', 'cite_spans': [], 'section': 'Funding', 'ref_spans': []}, {'text': 'All authors declare no competing interests exist.', 'cite_spans': [], 'section': 'Conflict of interest', 'ref_spans': []}, {'text': 'Ethical approval is not required.', 'cite_spans': [], 'section': 'Ethical approval', 'ref_spans': []}], 'paragraphs': [{'text': \"Compared to SARS-CoV, SARS-CoV-2 might be transmitted by respiratory droplets within a long distance of 2m or by contaminated surfaces leading to infection through contact transmission (Peng et al., 2020). The average incubation period for COVID-19 expands to 5 days and the period of quarantine is 14 days from the last date of exposure. It was the longest incubation period for similar types of coronaviruses. However, it's remained argumentative since the longest incubation period for COVID-19 can be 27 days or above according to the present-day finding (CIP, 2020). Perhaps, respiratory droplets were not the major factor for transmission (nCoV, 2020). Recently, there is a patient dog infected from its nasal and oral cavity with a low level of COVID-19 virus. It's not shown any signs of disease before. This is the first case of human-to-animal transmission that has been confirmed by the experts from School of Public Health in the Hong Kong University, College of Veterinary Medicine & Life Sciences in the City University of Hong Kong and the World Organization for Animal Health (COVID, 2020).\", 'id': '00015'}], 'title': 'Severe acute respiratory syndrome (SARS) and coronavirus disease-2019 (COVID-19): From causes to preventions in Hong Kong', 'sha': 'a8d5d102f3a7f3956eed948da8d0df868f4e4a12', 'doi': '10.1016/j.ijid.2020.03.059', 'score': 8.630000114440918}, {'rank': 303, 'abstract': 'In December 2019, the 2019 novel coronavirus (SARS-CoV-2) began spreading in China. At present, there are no special protocols for treating lumbar burst fracture (LBF) patients infected with SARS-CoV-2. Here, we present our lessons and experiences with a patient presenting with a severe LBF complicated by an occult SARS-CoV-2 infection. The clinical data for a 52-year-old male LBF patient were collected during the incubation period of COVID-19. The patient exhibited no obvious COVID-19-related symptoms prior to his surgery, and his vital signs were stable on the first day after the operation. By postoperative day 3, however, the patient was exhibiting chills and high fever. A chest CT showed a patchy high-density shadow surrounded by ground-glass opacity in the lower portion of his right lung. A nucleic acid test for SARS-CoV-2 was positive, and the patient was then transferred to the Department of Infectious Disease for further special treatment. This case taught that when treating patients with severe trauma within an epicenter of this pandemic, it is crucial for healthcare workers to be vigilant so as to avoid potential widespread outbreaks of COVID-19 within hospitals.', 'body_text': [{'text': 'SARS-CoV-2 refers to a novel coronavirus that firstly reported in Wuhan, Hubei province of China in December 2019 and quickly spread to the world [1]. SARS-CoV-2 belongs to the genus coronavirus, which is able to infect mammals, including humans [2]. The genetic characteristics of SARS-CoV-2 are significantly different from the acute respiratory syndrome coronavirus (SARS-CoV) and the middle east respiratory syndrome coronavirus (MERS-CoV) [3]. A recent study reported that SARS-CoV-2 is very similar to another virus in its family, which is carried by bats, leading some investigators to speculate that bats may host this new virus [4]. Currently, around 3.27 million cases have been confirmed, with 234,000 deaths worldwide [5].', 'cite_spans': [{'start': 147, 'end': 148, 'mention': '1', 'ref_id': 'BIBREF0'}, {'start': 247, 'end': 248, 'mention': '2', 'ref_id': 'BIBREF1'}, {'start': 445, 'end': 446, 'mention': '3', 'ref_id': 'BIBREF2'}, {'start': 638, 'end': 639, 'mention': '4', 'ref_id': 'BIBREF3'}, {'start': 731, 'end': 732, 'mention': '5', 'ref_id': 'BIBREF4'}], 'section': 'INTRODUCTION', 'ref_spans': []}, {'text': 'Lumbar burst fracture (LBF) is very common in the elderly, and it is well known that elderly fracture patients are susceptible to pulmonary infection. This is especially true for patients for spinal fractures, which make it difficult for them to walk. This raises the question, what do we do when a fracture patient is infected with SARS-CoV-2? There is currently no protocol for this situation. Here, we summarize our experience and lessons learned when diagnosing and surgically treating a patient with a severe LBF who is also infected with SARS-CoV-2.', 'cite_spans': [], 'section': 'INTRODUCTION', 'ref_spans': []}, {'text': 'A 52-year-old male suffering with a L3 LBF presented with lower back pain and serious weakness in both lower limbs. The patient had history of shopping while wearing an ordinary mask. Physical examination indicated that his bilateral hallux dorsal extensor muscle strength level was 2, bilateral tibialis anterior muscle strength was level 2, bilateral quadriceps muscle strength was level 4, and bilateral iliopsoas muscle strength was level. X-ray, magnetic resonance imaging (MRI), and computed tomography (CT) examinations revealed an L3 LBF as well as severe spinal cord compression (Figure 1). The patient denied experiencing fever, cough, sputum production, dyspnea, nausea or vomiting. There was no significant lung abnormality on preoperative CT examination (Figure 2), and tests for viruses indicated the patient to be influenza a virus RNA (-), influenza b virus RNA (-), and respiratory syncytial virus RNA (-). Routine blood counts showed leucocytes 11.43 g/L, erythrocytes 4.87 g/L, hemoglobin 139 g/L, platelets 208 g/L, neutrophils 88.5%, lymphocytes 5.9%, monocytes 5.5%, eosinophils 0%, and basophils 0.1%.', 'cite_spans': [], 'section': 'Patient and treatment', 'ref_spans': [{'start': 589, 'end': 597, 'mention': 'Figure 1', 'ref_id': 'FIGREF0'}, {'start': 768, 'end': 776, 'mention': 'Figure 2', 'ref_id': 'FIGREF1'}]}, {'text': 'Lumbar posterior decompression and fixation was performed in a laminar flow, negative pressure operating room. All procedures were performed while strictly adhering to biosafety level 3 standards. In addition to routine personal protection, goggles were worn and postoperative disinfection was performed. The surgery took around 1 hour and 30 minutes, and there were 200 ml of blood loss without blood transfusion. After the surgery, the patient was transferred to the intensive care unit for medical observation and treatment. On the second postoperative day, the patient was safely returned back to the ward with a lumbar brace. Postoperative radiography and CT revealed that the internal repair was well positioned (Figure 3).', 'cite_spans': [], 'section': 'Patient and treatment', 'ref_spans': [{'start': 719, 'end': 727, 'mention': 'Figure 3', 'ref_id': 'FIGREF2'}]}, {'text': 'On the first postoperative day, the patient had no obvious cough or sputum production, and his vital signs were stable. By the third postoperative day, however, the patient had developed chills and a high fever, which reached 39.5°C. Routine examination showed that his leukocyte count was 7.7 g/L, hemoglobin 126 g/L, neutrophils 81.3%, lymphocytes 8.9%, monocytes 9.5%, eosinophils 0% and basophils 0.3%. Levels of procalcitonin and c-reactive protein were 0.146 ng/mL and 144.9 mg/L, respectively. A chest CT examination showed a patchy high-density shadow surrounded by a ground-glass opacity in the lower portion of the right lung (Figure 4). After consultation with specialists from the Department of Infectious Disease, the patient was immediately isolated in a single room to prevent potential widespread infection. By postoperative day 5, the patient’s temperature had decreased to 37.2°C, and his cough had significantly diminished. A nucleic acid test for SARS-CoV-2 was positive, and the patient was transferred to the Department of Infectious Disease for further treatment, which consisted of cefoperazone sulbactam 1.5 g every 12 hours, abidole hydrochloride 0.2 g orally three times a day, and oseltamivir 75 mg orally twice a day. The patient’s vital signs remained stable, and his cough and fever eventually resolved.', 'cite_spans': [], 'section': 'Patient and treatment', 'ref_spans': [{'start': 637, 'end': 645, 'mention': 'Figure 4', 'ref_id': 'FIGREF3'}]}, {'text': 'A previous study reported that the incubation period for COVID-19 ranges from 1 to 14 days, with a average period from 3 to 7 days [6]. However, a recent retrospective study by the team of Nanshan Zhong, who is credited with detecting SARS in 2003, indicated that the incubation period for COVID-19 could be as long as 24 days, or 10 days longer than previously understood. That study, which has yet to be peer-reviewed, also suggested that patients can infect other people during the incubation period, echoing the findings of other reports about this virus. The most common symptoms at onset of COVID-19 are fever, fatigue, dry cough, myalgia, and dyspnea. Less common symptoms are headache, dizziness, abdominal pain, diarrhea, nausea, and vomiting [7]. The data from a recent study showed that up to 29% of COVID-19 patients are healthcare workers. Indeed, updated data indicate that in China, a total of 1716 healthcare workers were infected by SARS-CoV-2, and 16 of them have died. The adverse effect of infection of healthcare workers on the prevention and treatment of COVID-19 is enormous. On the one hand, loss of workers due to hospital-related transmission and infection exacerbates the existing shortage of healthcare workers. On the other hand, widespread infection of healthcare workers increases the public’s fear of COVID-19, which is harmful to social stability.', 'cite_spans': [{'start': 132, 'end': 133, 'mention': '6', 'ref_id': 'BIBREF5'}, {'start': 753, 'end': 754, 'mention': '7', 'ref_id': 'BIBREF6'}], 'section': 'DISCUSSION', 'ref_spans': []}, {'text': 'Fever occasionally develops after surgery as a stress response to the procedure [8–9]. In the present case, the elevated neutrophils and the reduction in lymphocytes indicated the possibility of infection, though the chest CT revealed no obvious abnormity. However, by the time the patient developed fever with an obvious dry cough on the day postoperative day 5, the chest CT definitely showed signs of pneumonia, and a nucleic acid test confirmed the SARS-CoV-2 infection on postoperative day 7. The symptoms were initially occult in this case. This patient had no significant symptoms of pulmonary infection at admission, and the chest CT results also did not support a diagnosis of pulmonary infection. Nonetheless, because of the ongoing COVID-19 epidemic, we decided to perform the spinal surgery in a laminar flow, negative pressure operating room. All procedures were performed strictly according biosafety level 3 standards, which are vital for avoiding widespread outbreak of COVID-19. Moreover, to avoid potential cross-infection, all the healthcare workers involved in this case self-isolated for 14 days. This including 3 surgeons, 1 anesthesiologist, 2 operating room nurses, and 2 ward nurses. Fortunately, no one was infected by this patient.', 'cite_spans': [{'start': 81, 'end': 82, 'mention': '8', 'ref_id': 'BIBREF7'}, {'start': 83, 'end': 84, 'mention': '9', 'ref_id': 'BIBREF8'}], 'section': 'DISCUSSION', 'ref_spans': []}, {'text': 'Several lessons can be learned from this case. First, for emergency surgery, comprehensive preoperative examinations should be performed to exclude the possibility of SARS-CoV-2 infection, and use of laminar flow, negative pressure operating rooms is strongly recommended during this COVID-19 pandemic. Second, the incubation period of COVID-19 should be given full attention, even for patients with negative results on the routine tests, perioperative isolation measures should be strong enough to prevent cross-infection among healthcare workers. Finally, for patients with confirmed SARS-CoV-2 infections, consultation with infectious disease specialists should be completed in a timely manner, and the necessary reports should be produced and submitted. If available, patients should be transferred to designated or specialized departments or hospitals for further diagnosis and treatment when necessary.', 'cite_spans': [], 'section': 'DISCUSSION', 'ref_spans': []}, {'text': 'All procedures performed in studies involving human participants were in accordance with the ethical standards of the institutional and/or national research committee (Puren Hospital, Wuhan University of Science and Technology, reference number 3-2020-0131) and with the 1964 Helsinki declaration and its later amendments or comparable ethical standards.', 'cite_spans': [], 'section': 'Ethical standards ::: CONCLUSION', 'ref_spans': []}], 'paragraphs': [{'text': 'A previous study reported that the incubation period for COVID-19 ranges from 1 to 14 days, with a average period from 3 to 7 days [6]. However, a recent retrospective study by the team of Nanshan Zhong, who is credited with detecting SARS in 2003, indicated that the incubation period for COVID-19 could be as long as 24 days, or 10 days longer than previously understood. That study, which has yet to be peer-reviewed, also suggested that patients can infect other people during the incubation period, echoing the findings of other reports about this virus. The most common symptoms at onset of COVID-19 are fever, fatigue, dry cough, myalgia, and dyspnea. Less common symptoms are headache, dizziness, abdominal pain, diarrhea, nausea, and vomiting [7]. The data from a recent study showed that up to 29% of COVID-19 patients are healthcare workers. Indeed, updated data indicate that in China, a total of 1716 healthcare workers were infected by SARS-CoV-2, and 16 of them have died. The adverse effect of infection of healthcare workers on the prevention and treatment of COVID-19 is enormous. On the one hand, loss of workers due to hospital-related transmission and infection exacerbates the existing shortage of healthcare workers. On the other hand, widespread infection of healthcare workers increases the public’s fear of COVID-19, which is harmful to social stability.', 'id': '00006'}], 'title': 'Lessons from a lumbar burst fracture patient infected with SARS-CoV-2', 'sha': '', 'doi': '10.18632/aging.103414', 'score': 8.62660026550293}, {'rank': 304, 'abstract': \"Since the first case of coronavirus disease 2019 (COVID‐19) was identified in Taiwan 2020.01.21. Several family cluster infections were found later. This study aimed to report family cluster infections and observe subsequent development. We collected domestic family cluster infections among COVID‐19 confirmed cases from 21 January 2020 to 16 March 2020. There were three domestic family clusters infections in this period. The first cluster was cases 19 to 23. The infectious source was a Taiwanese passenger from Zhejiang. The second cluster was cases 24 to 26 and the third cluster was cases 27 to 32. The infectious sources of the latter clusters are currently uncertain. All contacts of three clusters have been isolated and no new confirmed cases have been identified to date. Some measures which have reduced the spread of these three clusters included: First, high suspicion of COVID‐19 for unexplained pneumonia is very important for early detection. Second, immediate epidemic investigation is taken especially COVID‐19 is infectious during the incubation period. Third, when the second and third clusters could not find infectious sources, CECC's press conference let the public know immediately the epidemic situation, so that people could raise their awareness and seek medical treatment or quarantine.\", 'body_text': [{'text': 'In December 2019, since coronavirus disease 2019 (COVID‐19) was confirmed that the pathogen was severe acute respiratory syndrome coronavirus 2 (SARS‐CoV2),\\n1\\n, \\n2\\n, \\n3\\n, \\n4\\n many countries around the world have been infected in recent months.\\n5\\n The WHO announced that the COVID‐19 epidemic was listed as an international public health emergency.\\n6\\n Taiwan is adjacent to mainland China and there are many cultural and economic exchanges and tourism between the two sides of Taiwan straits.\\n7\\n, \\n8\\n, \\n9\\n Taiwan is at great risk from being infected by SARS‐CoV2.\\n7\\n\\n', 'cite_spans': [], 'section': 'INTRODUCTION', 'ref_spans': []}, {'text': 'Since the first case of COVID‐19 was identified in Taiwan 2020.01.21, there have been sporadic imported infections, infections between couples, family cluster infections, and nosocomial infections later. In 2004, the second year after the SARS outbreak, the Taiwan government established the National Health Command Center (NHCC). NHCC unifies a central command system, including the Central Epidemic Command Center (CECC), the Biological Pathogen Disaster Command Center, and so on.\\n10\\n The Taiwan center for disease control (CDC) also announced the establishment of a CECC for severe special infectious pneumonia. The level has been increased to the second level and has now been increased to the first level.\\n11\\n\\n', 'cite_spans': [], 'section': 'INTRODUCTION', 'ref_spans': []}, {'text': \"Having new cases in Taiwan, or major changes in the international epidemic situation, the CECC will hold a press conference to tell people about Taiwan case epidemic information and corresponding strategies. Therefore, all people, medical institutions, and health unit personnel can understand the epidemic and work together to cope with COVID‐19. This study aims to report the domestic family cluster infection of COVID‐19 in Taiwan, to explore the public health strategy of Taiwan's government and to observe the development of subsequent epidemics.\", 'cite_spans': [], 'section': 'INTRODUCTION', 'ref_spans': []}, {'text': \"This is a retrospective study. Patient information was obtained from the content of CECC's press conference and the Taiwan CDC website.\\n12\\n The authors who do not know the patient's additional personal information will not involve privacy and personal security. In the face of the sudden outbreak of COVID‐19, we do not know where the patients are, nor can we obtain the patient's informed consent. Given that the rights of patients are not infringed and the medical knowledge thus obtained can help more patients, I believe scholars will have the same consensus.\", 'cite_spans': [], 'section': 'Patient informed consents and ethic issue ::: MATERIALS AND METHODS', 'ref_spans': []}, {'text': 'We collected cases of domestic family cluster infection among COVID‐19 confirmed cases from 21 January 2020 to 16 March 2020.', 'cite_spans': [], 'section': 'Study population ::: MATERIALS AND METHODS', 'ref_spans': []}, {'text': \"The first domestic family cluster is cases 19 to 23 (Figures 1 and 2). Case 19 is a male driver in his 60 seconds with a history of hepatitis B and diabetes, but no travel abroad history. Symptoms appeared on 1/27 and he was hospitalized on 3 February with the diagnosis of pneumonia at that time, died of severe pneumonia complicated with sepsis on 15 February. This case was confirmed later due to the patient with negative influenza screening and an afterward test for SARS‐CoV‐2. Case 19 was the first death case of COVID‐19 in Taiwan. Case 19 once carried Taiwanese businessman. The Taiwanese businessman from Zhejiang returned to Taiwan on 22 January and had a cough. Cases 20 to 23 are cases of brother, mother, nephew son‐in‐law, and sister of case 19, who had no symptoms and were suspected to be infected during the dinner at the same table during the New Year. All contacts have been quarantined according to the CECC's protocol.\", 'cite_spans': [], 'section': 'The first domestic family cluster infections ::: MATERIALS AND METHODS', 'ref_spans': [{'start': 61, 'end': 62, 'mention': '1', 'ref_id': 'FIGREF0'}, {'start': 67, 'end': 68, 'mention': '2', 'ref_id': 'FIGREF1'}]}, {'text': \"The second domestic family cluster is cases 24 to 26 (Figure 3). The 24th confirmed case was a 67 years old woman with gout and hypertension but had no history of going abroad for nearly 2 years. She developed fever, cough and other symptoms on 22 January. He went to the clinic four times from 22 to 29 January. Due to worsening symptoms and shortness of breath, he went to the hospital for emergency treatment on the evening of the 29th and was diagnosed with pneumonia on the examination. He was admitted to the hospital on the 30th. Due to the deterioration of his condition, she was transferred to the intensive care unit on 10 February. To follow the CECC's measure that cases of severe complicated influenza that tested negative for influenza are required to test for COVID‐19, she was transferred to a negative pressure isolation ward on 17 February. The patient tested positive for COVID‐19 on 19 February and is receiving medical treatment in isolation. Case 25 and 26 are the granddaughter and younger daughter of Case 24. Neither had they had a history of foreign travel. Case 26 developed symptoms of cough on 28 January. He sought medical treatment on 31 January due to a fever and continued to have cough symptoms on 31 February. He also sought medical treatment on 4 and 11 February. Case 25 had no fever or respiratory symptoms as of 21 February and had a medical consultation on 3 and 6 February because of gastroesophageal reflux. Tracing the history, on 6 January, Case 24 had a dinner with Case 25's classmate who came back from China. All contacts have been quarantined according to the CECC's protocol.\", 'cite_spans': [], 'section': 'The second domestic family cluster infections ::: MATERIALS AND METHODS', 'ref_spans': [{'start': 61, 'end': 62, 'mention': '3', 'ref_id': 'FIGREF2'}]}, {'text': \"Case 27 to 32 is the third domestic family cluster infection (Figure 4). Case 27 was an 80 seconds male with hypertension, diabetes, and end‐stage renal disease with hemodialysis. He has no recent history of going abroad. On 6 February, he developed symptoms such as cough and runny nose. On the 9th, he was admitted to a single ward for fever and a diagnosis of pneumonia. On the 16th, he had shortness of breath and was transferred to the intensive care unit. On the 20th, he was transferred to a negative pressure isolation ward because of unexplained pneumonia. Later, the diagnosis COVID‐19 was confirmed on the 23rd. Finally, Case 27 died of multiple organ failure and septic shock. Cases 28 to 32 were the two sons, wife, grandson, and foreign caregiver of Case 27. Case 29's younger son frequently traveled to and from Wuhan due to his work relationship. He returned to Taiwan from Guangzhou on 2 December 2019. During the Chinese New Year, he also had dinner with a friend with a travel history of China. Case 31 was the 11‐year‐old grandson of Case 27. He lives with him and visited the hospital on 11th and 19th February. Because he was listed as close contact, he was admitted to the hospital for isolation and observation and was confirmed later. He had currently no obvious symptoms. All contacts have been quarantined according to the CECC's protocol.\", 'cite_spans': [], 'section': 'The third domestic family cluster infections ::: MATERIALS AND METHODS', 'ref_spans': [{'start': 69, 'end': 70, 'mention': '4', 'ref_id': 'FIGREF3'}]}, {'text': 'There are 57 confirmed cases with COVID‐19 in Taiwan as of 16 March 2020. Of confirmed cases, there are three family clusters infections.', 'cite_spans': [], 'section': 'RESULTS', 'ref_spans': []}, {'text': 'The first domestic family cluster was cases 19 to 23. CECC intervened in the epidemic. After interrogating the medical history, it estimated the date of symptom onset and the date of exposure of the confirmed case to find the index patient. It estimated all contacts of the confirmed case during the infection period including family members, pharmacy buying medicine, clinic visits, and driver group companions, and all passengers from communication records. It also involved index patient medical history and related contacts (Figure 1). A Zhejiang Taiwanese businessman is the index patient. The businessman returned to Taiwan on 22 January and had a cough. The result was negative for the first time by PCR. Further blood tests found that SARS‐CoV‐2 antibodies were positive. However, antibody testing was not listed as a diagnostic tool. Therefore, Zhejiang Taiwanese businessmen were not included in the confirmed case. As for Zhejiang Taiwanese businessmen, his three Taiwan family members who tested were negative. About 257 contacts were collected and 256 were tested (one died not due to COVID‐19, including 4 of whom were confirmed (cases 20 to 23), and the remaining 252 were all negative (Figure 2). Above contacts have been isolated and no new confirmed cases have been identified to date.\\n15\\n\\n', 'cite_spans': [], 'section': 'RESULTS', 'ref_spans': [{'start': 536, 'end': 537, 'mention': '1', 'ref_id': 'FIGREF0'}, {'start': 1209, 'end': 1210, 'mention': '2', 'ref_id': 'FIGREF1'}]}, {'text': \"The secondary domestic family cluster was cases 24 to 26. There were 853 contacts, 242 of whom had been in close contact, 2 of whom were confirmed positive (case 25‐26) and 240 had negative reactions (Figure 3). Although Case 24 had a contact history with Case 25's classmate who came back from China, Case 25's classmate's test was negative and CECC still listed as an unknown index patient. In addition, the woman's eldest daughter was negative on three tests, she was still suspected cases and was isolated because she had persistent cough symptoms and had close contact with her mother 14 days before the onset of illness. Above contacts have been isolated and no new confirmed cases have been identified to date.\\n16\\n\\n\", 'cite_spans': [], 'section': 'RESULTS', 'ref_spans': [{'start': 208, 'end': 209, 'mention': '3', 'ref_id': 'FIGREF2'}]}, {'text': \"The third domestic family cluster infection was case 27 to 32. There were 580 contacts, 144 of whom have been in close contact, 4 of whom have confirmed positive (case 28‐31) and 140 were negative (Figure 4). Case 29 was suspected of index patient initially but was ruled out later because his friend's test was negative. CECC still listed the index patient was currently uncertain. All contacts have been isolated and no new confirmed cases have been identified to date.\\n17\\n\\n\", 'cite_spans': [], 'section': 'RESULTS', 'ref_spans': [{'start': 205, 'end': 206, 'mention': '4', 'ref_id': 'FIGREF3'}]}, {'text': 'In the first family cluster infections, only case 19 had a history of exposure history with Taiwanese businessmen from Zhejiang. Case 20 and 21 live together with case 19, should be infected by case 19 during the incubation period or after symptoms. Case 22 and 23 had participated in a family gathering on January 27, maybe infected by case 19 or case 20 or 21 during the incubation period, or infected by case 19, 20, 21 to one of both, and then infected to another.', 'cite_spans': [], 'section': 'DISCUSSION', 'ref_spans': []}, {'text': 'In the second family cluster infections, case 24 has no recent travel history. Case 26 and case 24 live together. Case 26 had a friend traveling to the mainland and suspected case 26 infected case 24 during the incubation period, but the friend of the case 26 was negative, but this possibility was not ruled out, or case 24 with unknown source of infection was transmitted to case 26 during the incubation period or after the onset of the symptoms, but this probability of the latter was low. Because Case 24 frequently does exercise in the park, there have been no new confirmed cases or community infections in this area. Case 25 was infected due to a history of medical visit contact. CECC is still logged as the index patient was not found in this cluster.', 'cite_spans': [], 'section': 'DISCUSSION', 'ref_spans': []}, {'text': \"In the third family cluster infections, Case 27 long term lived in the house without a history of exposure. Case 28, 30, and 31 also had no history of exposure. It is supposed that Case 29 transmitted case 30 and case 27 during the incubation period or after symptoms. Case 29 used to be frequent in Wuhan and other regions of China. The last return to Taiwan from Guangzhou on 12 December 2019 has exceeded the incubation period of 14 days. It is not excluded that case 29 has a longer incubation period. Case 29 had contacts with friends who had a history of Chinese tourism in Taiwan Lunar New Year. Although the friend's test was negative, he was not excluded from being an Index patient. Case 28 and 31 is asymptomatic and lives with case 27.30. They were infected by case 27 or case 30 who was during the incubation period or after the onset of symptoms. However, CECC is still logged as the index patient was not found in this cluster.\", 'cite_spans': [], 'section': 'DISCUSSION', 'ref_spans': []}, {'text': 'SARS cases are infectious only during their symptomatic period and are noninfectious during the incubation period.\\n18\\n Evidence has demonstrated that human‐to‐human transmission of 2019 novel coronaviruses has occurred among close contacts,\\n19\\n, \\n20\\n, \\n21\\n even during the incubation period.\\n22\\n Therefore, the determination of close contacts should also include those who are exposed to case‐patients during their incubation. The incubation period of COVID‐19 is considered to be within 14 days after exposure, and most cases occur about 4 to 5 days after exposure.\\n19\\n, \\n21\\n, \\n23\\n However, the interval at which individuals with COVID‐19 are infected is uncertain. It seems that SARS‐CoV‐2 can spread before symptoms appear and throughout the disease.\\n24\\n, \\n25\\n The Case 25 and Case 29 are still suspected as the index patients of the family cluster infections which may be confirmed by further knowledge of COVID‐19.', 'cite_spans': [], 'section': 'DISCUSSION', 'ref_spans': []}, {'text': 'The severity of COVID‐19 symptoms ranges from very mild to severe.\\n23\\n, \\n26\\n, \\n27\\n, \\n28\\n, \\n29\\n, \\n30\\n, \\n31\\n, \\n32\\n, \\n33\\n The patients, who are older or have underlying diseases, may be at higher risk of serious illness. The most common symptoms are fever and cough.\\n23\\n, \\n26\\n, \\n27\\n, \\n28\\n, \\n29\\n, \\n30\\n In addition, there are gastrointestinal symptoms, sore throat, runny nose, etc. The symptoms may vary depending on the progress of the disease and individual differences. Owing to some patients with COVID‐19 has early symptoms that are not obvious and without fever, some of them are mild or asymptomatic\\n31\\n, \\n32\\n (for example, case 31, asymptomatic at confirmation), they often visit the local clinic due to mild respiratory infection. The local clinic has also become an important role in this epidemic prevention. Case 19 has no history of travel and no obvious contact history at initial history taking. Fortunately, the local clinician was alert and referred to a large hospital. He was admitted to a negative pressure isolation ward and thus prevented virus spreading.', 'cite_spans': [], 'section': 'DISCUSSION', 'ref_spans': []}, {'text': 'The measures taken by the Taiwan CECC have reduced the spread of these three family cluster infections of CARS‐CoV‐2 without causing community infections.', 'cite_spans': [], 'section': 'DISCUSSION', 'ref_spans': []}, {'text': 'We have some suggestions like the following: First, early detection of COVID‐19 in patients with unexplained pneumonia with negative influenza test can prevent infection spread. The starters of these three family clusters were not easy to detect exposure history at first, high suspicion of COVID‐19 for atypical pneumonia is very important. Second, the strict contact history tracking and immediate epidemic investigation of the confirmed cases, especially COVID‐19 is infectious during the incubation period or asymptomatic period. Recognition of infections, time inference, and quarantine scope require new considerations. Third, a series of press conferences will let the public know immediately the changes in the epidemic situation in Taiwan and the world, and immediately educate the public on how to prevent the epidemic\\n34\\n including mask policies,\\n35\\n hand hygiene, and maintain appropriate social distance. Especially when the second and third family group infections could not find the source of the infection, the whereabouts of the diagnosed person will be announced, so that the public can raise their awareness and seek medical treatment or quarantine to prevent community infections.', 'cite_spans': [], 'section': 'DISCUSSION', 'ref_spans': []}, {'text': \"The data source of this study is the information released by the press conference of the Taiwan CECC. The authenticity of the information cannot be doubted, and there is no violation of personal privacy. There are still some limitations, such as deficiency in more detail of individual patient's medical history, laboratory data, or radiographic characterization. Because this study focuses on the development of the epidemic, the individual's clinical details did not affect the purpose of this study.\", 'cite_spans': [], 'section': 'DISCUSSION', 'ref_spans': []}, {'text': 'In conclusion, the measures taken by the Taiwan CECC have reduced the spread of these three family cluster infections of CARS‐CoV‐2 without causing community infections.', 'cite_spans': [], 'section': 'DISCUSSION', 'ref_spans': []}, {'text': 'The authors declare that there are no conflict of interests.', 'cite_spans': [], 'section': 'CONFLICT OF INTERESTS', 'ref_spans': []}, {'text': 'S‐FL: study conceptualization; formal analysis; methodology; software; supervision; roles/writing‐original draft; writing‐ review, & editing. H‐CK: data curation; formal analysis; methodology; software; formal analysis; methodology; writing‐review, & editing.', 'cite_spans': [], 'section': 'AUTHOR CONTRIBUTIONS', 'ref_spans': []}], 'paragraphs': [{'text': \"In the third family cluster infections, Case 27 long term lived in the house without a history of exposure. Case 28, 30, and 31 also had no history of exposure. It is supposed that Case 29 transmitted case 30 and case 27 during the incubation period or after symptoms. Case 29 used to be frequent in Wuhan and other regions of China. The last return to Taiwan from Guangzhou on 12 December 2019 has exceeded the incubation period of 14 days. It is not excluded that case 29 has a longer incubation period. Case 29 had contacts with friends who had a history of Chinese tourism in Taiwan Lunar New Year. Although the friend's test was negative, he was not excluded from being an Index patient. Case 28 and 31 is asymptomatic and lives with case 27.30. They were infected by case 27 or case 30 who was during the incubation period or after the onset of symptoms. However, CECC is still logged as the index patient was not found in this cluster.\", 'id': '00015'}], 'title': \"Three Taiwan's domestic family cluster infections of coronavirus disease 2019\", 'sha': 'bb4b07ea340ec1ef42d3f127e7a3c7461bb29c3b', 'doi': '10.1002/jmv.25949', 'score': 8.610699653625488}, {'rank': 305, 'abstract': \"BACKGROUND: In December, 2019, the newly identified severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) emerged in Wuhan, China, causing COVID-19, a respiratory disease presenting with fever, cough, and often pneumonia. WHO has set the strategic objective to interrupt spread of SARS-CoV-2 worldwide. An outbreak in Bavaria, Germany, starting at the end of January, 2020, provided the opportunity to study transmission events, incubation period, and secondary attack rates. METHODS: A case was defined as a person with SARS-CoV-2 infection confirmed by RT-PCR. Case interviews were done to describe timing of onset and nature of symptoms and to identify and classify contacts as high risk (had cumulative face-to-face contact with a confirmed case for ≥15 min, direct contact with secretions or body fluids of a patient with confirmed COVID-19, or, in the case of health-care workers, had worked within 2 m of a patient with confirmed COVID-19 without personal protective equipment) or low risk (all other contacts). High-risk contacts were ordered to stay at home in quarantine for 14 days and were actively followed up and monitored for symptoms, and low-risk contacts were tested upon self-reporting of symptoms. We defined fever and cough as specific symptoms, and defined a prodromal phase as the presence of non-specific symptoms for at least 1 day before the onset of specific symptoms. Whole genome sequencing was used to confirm epidemiological links and clarify transmission events where contact histories were ambiguous; integration with epidemiological data enabled precise reconstruction of exposure events and incubation periods. Secondary attack rates were calculated as the number of cases divided by the number of contacts, using Fisher's exact test for the 95% CIs. FINDINGS: Patient 0 was a Chinese resident who visited Germany for professional reasons. 16 subsequent cases, often with mild and non-specific symptoms, emerged in four transmission generations. Signature mutations in the viral genome occurred upon foundation of generation 2, as well as in one case pertaining to generation 4. The median incubation period was 4·0 days (IQR 2·3–4·3) and the median serial interval was 4·0 days (3·0–5·0). Transmission events were likely to have occurred presymptomatically for one case (possibly five more), at the day of symptom onset for four cases (possibly five more), and the remainder after the day of symptom onset or unknown. One or two cases resulted from contact with a case during the prodromal phase. Secondary attack rates were 75·0% (95% CI 19·0–99·0; three of four people) among members of a household cluster in common isolation, 10·0% (1·2–32·0; two of 20) among household contacts only together until isolation of the patient, and 5·1% (2·6–8·9; 11 of 217) among non-household, high-risk contacts. INTERPRETATION: Although patients in our study presented with predominately mild, non-specific symptoms, infectiousness before or on the day of symptom onset was substantial. Additionally, the incubation period was often very short and false-negative tests occurred. These results suggest that although the outbreak was controlled, successful long-term and global containment of COVID-19 could be difficult to achieve. FUNDING: All authors are employed and all expenses covered by governmental, federal state, or other publicly funded institutions.\", 'body_text': [{'text': 'On Dec 31, 2019, Chinese officials reported a cluster of cases of pneumonia in Wuhan, China. The newly discovered severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) was identified to be responsible for the ensuing outbreak.1 As of May 13, 2020, more than 4·1 million confirmed cases of COVID-19 have been reported with more than 280 000 attributable deaths worldwide.2\\n', 'cite_spans': [{'start': 233, 'end': 234, 'mention': '1', 'ref_id': 'BIBREF0'}, {'start': 377, 'end': 378, 'mention': '2', 'ref_id': 'BIBREF5'}], 'section': 'Introduction', 'ref_spans': []}, {'text': 'As of March 1, 2020, WHO adhered to the strategic objective to “interrupt human-to-human transmission including reducing secondary infections among close contacts and health-care workers, preventing transmission amplification events, and preventing further international spread”.2 The International Health Regulations (IHR) Emergency Committee stated in its most recent declaration that it “believes that it is still possible to interrupt virus spread, provided that countries put in place strong measures to detect disease early, isolate and treat cases [and] trace contacts”.3 China implemented unprecedented measures to curb the epidemic, including cordoning off entire cities and implementing rigorous contact restrictions.4 In the meantime, countries outside of China attempted to contain the spread of the virus upon detection of travel-associated cases.', 'cite_spans': [{'start': 279, 'end': 280, 'mention': '2', 'ref_id': 'BIBREF5'}, {'start': 577, 'end': 578, 'mention': '3', 'ref_id': 'BIBREF6'}, {'start': 727, 'end': 728, 'mention': '4', 'ref_id': 'BIBREF7'}], 'section': 'Introduction', 'ref_spans': []}, {'text': '\\nResearch in context\\nEvidence before this study\\nWe searched MEDLINE (via PubMed) for articles published until March 2, 2020, using the key words “novel coronavirus”, “2019-nCoV”, “COVID-19”, “SARS-CoV-2”, “transmission”, “incubation period”, “serial interval”, and “Europe”. Moreover, we screened preprint servers such as medrxiv and SSRN for relevant articles and consulted the webpages of organisations such as WHO, European Centre for Disease Prevention and Control, and the Robert Koch Institute. No reports describing events of human-to-human transmission of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) within Europe other than the Bavarian outbreak were identified.\\nAdded value of this study\\nOriginating from a single travel-associated primary case from China, we describe the first reported cluster of COVID-19 cases with human-to-human transmission of SARS-CoV-2 within Europe and outside of Asia. The outbreak, which resulted in 16 subsequent cases, comprised four generations of virus transmission in only 16 days, corresponding to a median incubation period and serial interval of 4 days. By testing all people who had high-risk contact with confirmed patients with SARS-CoV-2 infection (and people with low-risk contact upon onset of symptoms), we were additionally able to detect and follow up patients with only very mild clinical symptoms of COVID-19 that would have probably remained undetected otherwise. By combining methods of epidemiology and whole genome sequencing, we were able to reconstruct and describe transmission events precisely. Virus transmission before or on the day of symptom onset or during a prodromal phase was substantial in this outbreak. Moreover, we were able to estimate attack rates according to different scenarios of contact intensity. As of May 2, 2020, no further cases associated with the outbreak were detected, suggesting that applied containment measures have worked.\\nImplications of all the available evidence\\nIn the beginning of an epidemic caused by a newly discovered virus, the assessment of key epidemiological parameters such as attack rates, incubation period, and serial interval is important, as it provides essential information for estimating the potential size of the epidemic and developing containment measures. Detailed reports on transmission events of SARS-CoV-2, especially in a well described setting, could improve the understanding of transmissibility and further spread of the virus.\\n', 'cite_spans': [], 'section': 'Introduction', 'ref_spans': []}, {'text': 'On Jan 27, 2020, the Bavarian Health and Food Safety Authority, Germany, was informed of the first human case of infection with SARS-CoV-2 in a German national working for a company in the greater Munich area. The primary case in this satellite outbreak of COVID-19 is a person from Shanghai, China, who had been in contact with their parents from Wuhan before visiting Germany for a business meeting in the aforementioned company. Between Jan 27 and Feb 11, 16 cases of COVID-19 were identified in this cluster.', 'cite_spans': [], 'section': 'Introduction', 'ref_spans': []}, {'text': 'Management and investigation of the outbreak was immediately initiated on Jan 27 to identify further cases and contacts and to understand transmission events and parameters that are relevant for successful containment, such as incubation period and secondary attack rates.', 'cite_spans': [], 'section': 'Introduction', 'ref_spans': []}, {'text': 'This outbreak investigation was done in Bavaria, Germany, as a collaboration of state (Bavarian Health and Food Safety Autority) and national level (Robert Koch Institute) public health authorities and four public health laboratories. It included all people with confirmed SARS-CoV-2 infection and contacts (ie, people who had contact with a person with confirmed SARS-CoV-2 infection) linked to patient 0 of the Bavarian cluster.', 'cite_spans': [], 'section': 'Study design and participants ::: Methods', 'ref_spans': []}, {'text': 'The outbreak investigation was conducted as part of the authoritative, official tasks of the county health departments as well as the state health department of the Bavarian Health and Food Safety Authority, supported by the Robert Koch Institute. As conducted in response to a public health emergency, this study was exempt from institutional review board approval.', 'cite_spans': [], 'section': 'Study design and participants ::: Methods', 'ref_spans': []}, {'text': 'On Jan 27, 2020, all employees of the affected company were informed about the potential risk of COVID-19 infection. International public health authorities were informed via the Early Warning and Response System or IHR National Focal points. Employees were actively queried for any contact with the first two known cases. New cases were asked about professional and private contacts. Contacts were classified as high risk if they had cumulative face-to-face contact with a patient with laboratory-confirmed SARS-CoV-2 infection for at least 15 min, had direct contact with secretions or body fluids of a patient with confirmed COVID-19, or, in the case of health-care workers, had worked within 2 m of a patient with confirmed COVID-19 without personal protective equipment. All other contacts were classified as low-risk contacts.', 'cite_spans': [], 'section': 'Contact classification and management ::: Methods', 'ref_spans': []}, {'text': 'High-risk contacts were ordered to stay in home quarantine for 14 days after the last known contact event with a confirmed case. Their health status was monitored daily, usually by self-report. Laboratory testing for SARS-CoV-2 was done at the beginning and end of home quarantine periods, irrespective of the presence of symptoms. If a contact tested positive for SARS-CoV-2, they were immediately hospitalised and isolated. Low-risk contacts were asked to self-monitor their health status and report any symptoms. In both high-risk and low-risk contacts, further laboratory testing was triggered upon onset of any symptoms.', 'cite_spans': [], 'section': 'Contact classification and management ::: Methods', 'ref_spans': []}, {'text': 'Case interviews were done in a two-stage procedure. In the first stage, patients with confirmed SARS-CoV-2 infection or their household members were interviewed to determine date of symptom onset, links between cases, contact events during the incubation period, and contact classification. In a second stage, in-depth interviews with ten patients with COVID-19 were done by teams of two professionals (one physician and one epidemiologist; SB, KP, NM, NZ, and TSB) using a semi-structured questionnaire. Additional topics included characteristics of symptoms and details on type, setting, and environment of contact with other patients and with high-risk contacts. The in-depth interview with the primary case from Shanghai, China, was done twice (Jan 30 and 31, 2020) and supported by a Chinese native speaker (CW). We defined fever or cough as specific symptoms; a prodromal phase (with non-specific symptoms) was defined as presence of symptoms other than fever and cough for at least 1 day before the onset of specific symptoms. The day of symptom onset was defined as the day when any symptom (specific or non-specific) occurred, and thus could coincide with the start of the prodromal phase. The potential infectious period was defined as the period 2 days before onset of any kind of symptoms (specific or non-specific) until 14 days later.', 'cite_spans': [], 'section': 'Case interviews ::: Methods', 'ref_spans': []}, {'text': 'Laboratory testing involved two swabs (nasopharyngeal and oropharyngeal, pooled) that were stored in viral transport medium and cooled. RNA was extracted using the QiAamp Bio Robot Kit (Qiagen; Hilden, Germany) on a Hamilton Microlab Star as recommended by the manufacturer. Real-time RT-PCR was done with the QuantiTect Virus +Rox Vial Kit (Qiagen, Hilden, Germany) on the Bio-Rad CFX96 Touch Real-Time PCR Detection System. Primer and probes were used as described by Corman and colleagues5 and provided by Tib-Molbiol (Berlin, Germany). The reference laboratory worked exactly as described in Corman and colleagues.5 Whole genome sequencing involved Roche KAPA HyperPlus library preparation and sequencing on Illumina NextSeq and MiSeq instruments as well as RT-PCR product sequencing on Oxford Nanopore MinION using the primers described in Corman and colleagues.6 Patient 1 was sequenced on all three platforms; patients 2–7 were sequenced on Illumina NextSeq, both with and without RT-PCR product sequencing with primers as in Corman and colleagues;6 and patients 8–11, 14, and 16 were sequenced on Oxford Nanopore MinION. Sequencing of patient 15 was not successful. Sequence gaps were filled by Sanger sequencing.', 'cite_spans': [{'start': 491, 'end': 492, 'mention': '5', 'ref_id': 'BIBREF8'}, {'start': 618, 'end': 619, 'mention': '5', 'ref_id': 'BIBREF8'}, {'start': 867, 'end': 868, 'mention': '6', 'ref_id': 'BIBREF9'}, {'start': 1055, 'end': 1056, 'mention': '6', 'ref_id': 'BIBREF9'}], 'section': 'Laboratory testing ::: Methods', 'ref_spans': []}, {'text': \"We calculated secondary attack rates among contacts and case–contact pairs. A case–contact pair is the connection between a given contact and a potentially infectious case with contact time of at least 15 min; the number of case–contact pairs is higher than the number of contacts because a single contact person could have had contact with more than one case. To calculate secondary attack rates, we considered four distinct groups: (1) a household cluster where one household member had COVID-19 and all household members were quarantined together in one hospital room; (2) any other household contacts (ie, anyone sharing living space with a patient with COVID-19); (3) non-household high-risk contacts; (4) known low-risk contacts. Secondary attack rates were calculated as number of cases divided by number of contacts or case–contact pairs, using Fisher's exact test for the 95% CIs.\", 'cite_spans': [], 'section': 'Secondary attack rate among contacts ::: Methods', 'ref_spans': []}, {'text': 'The sponsors of the study had no role in study design, data collection, data analysis, data interpretation, or writing of the report. The corresponding author had full access to all the data in the study and had final responsibility for the decision to submit for publication.', 'cite_spans': [], 'section': 'Role of the funding source ::: Methods', 'ref_spans': []}, {'text': \"Details on SARS-CoV-2 importation to Germany have been previously described by Rothe and colleagues.7 In brief, the primary case (patient 0) stated that her parents, who normally live in Wuhan, had arrived for a visit in Shanghai on Jan 16. Both parents recalled cold-like symptoms the week before, and one parent showed fatigue and loss of appetite while visiting. Patient 0, who was an employee of the Chinese branch of a German company based in greater Munich, travelled from Shanghai to Munich by aeroplane on Jan 19, 2020, to facilitate workshops and attend meetings in the company building. The day after arrival (Jan 20, 2020), patient 0 felt chest and back aches—which she reported to be unusual—and took a single dose of medicine containing paracetamol. The patient reported fatigue during her whole stay in Germany and attributed the symptom to jetlag. After an overnight flight back to Shanghai on Jan 22, the patient felt feverish. With a self-measured temperature of 38·6°C and cough on Jan 24, the patient visited a physician's office on Jan 25. The patient tested positive for SARS-CoV-2 on Jan 26 and was hospitalised the next day. The clinical situation in both parents also deteriorated during the primary case's stay in Germany and both were laboratory-confirmed with COVID-19 later.7 The German company was informed of the primary case's infection in the morning of Jan 27, 2020, and immediately informed its employees as well as the local health authority.\", 'cite_spans': [{'start': 100, 'end': 101, 'mention': '7', 'ref_id': 'BIBREF10'}, {'start': 1302, 'end': 1303, 'mention': '7', 'ref_id': 'BIBREF10'}], 'section': 'Results', 'ref_spans': []}, {'text': \"The initial testing by RT-PCR of high-risk contacts between Jan 27–29 identified patients 1–4 as first-generation cases (figure\\n; table 1\\n).7 All four patients were immediately hospitalised and isolated. Their immediate contacts, including people they had contact with 2 days before symptom onset, were traced. A 14-day home quarantine was ordered for all newly identified high-risk contacts, starting at the day of the last known contact event with a case during the potential infectious period. Contacts were actively followed up on a daily basis. All high-risk contacts were instructed to minimise contact with other people, including household members in home quarantine. The affected company site was closed on the company's own initiative until Feb 11, 2020, and on-site disinfection measures were applied.\", 'cite_spans': [{'start': 140, 'end': 141, 'mention': '7', 'ref_id': 'BIBREF10'}], 'section': 'Results', 'ref_spans': [{'start': 121, 'end': 127, 'mention': 'figure', 'ref_id': 'FIGREF0'}, {'start': 130, 'end': 137, 'mention': 'table 1', 'ref_id': 'TABREF0'}]}, {'text': 'By Feb 19, 2020, 16 subsequent cases had been identified: four female and 12 male. All patients had been registered as high-risk contacts of the primary case or subsequent cases before being identified. The median age of the 16 patients was 35 years (IQR 27–42; range 2–58). Ten patients (patients 1–5, 7, 8, 10, 13, and 16) as well as the primary case (patient 0) are employees of the company. A Chinese colleague (patient 13) of patient 0 accompanied them in multiple activities while in Germany. Patient 13 travelled back to China with patient 0, developed symptoms on Jan 27, and tested SARS-CoV-2 positive a few days later.', 'cite_spans': [], 'section': 'Results', 'ref_spans': []}, {'text': \"Patient 1 was an employee who attended a 1-h business meeting with patient 0 and two other colleagues on Jan 20, 2020. The meeting took place in a small room (approximately 12 m2); patient 1 sat next to patient 0, and the two other colleagues sat at the opposite side of the table. The two other colleagues did not test positive during follow-up. Patient 1 had another brief contact event with patient 0 on Jan 21, and developed a sore throat on Jan 23 (figure). Over the following weekend (Jan 25–26), they developed cold-like symptoms with self-measured fever up to 39°C and mild productive cough. On Jan 27, patient 1 felt well enough to go to work. There, patient 1 learned about patient 0's infection and was tested positive on the same day.\", 'cite_spans': [], 'section': 'Results', 'ref_spans': [{'start': 454, 'end': 460, 'mention': 'figure', 'ref_id': 'FIGREF0'}]}, {'text': \"Patient 2 was not aware of any direct person-to-person contact with patient 0; however, virus sequence analysis supports the assumption that patient 0 transmitted the virus to patient 2. Patient 3 also did not have direct contact with patient 0; however, patient 3 had contact with patient 1 on Jan 24 (figure), when both worked simultaneously on the same computer for a short period of time. Thus, transmission from patient 1 to patient 3 most likely took place the day after symptom onset (Jan 23). On Jan 25, the day of symptom onset of patient 3, patient 3 had a private meeting with patient 12, sitting next to him for approximately 90 min. Afterwards, they spent the rest of the evening together at patient 3's home, joined by patient 3's partner, who did not test positive during follow-up. Patient 12 departed for vacation to Spain 3 days later (Jan 28). After Spanish authorities were informed, patient 12 was isolated in hospital on Jan 30 and diagnosed with COVID-19.\", 'cite_spans': [], 'section': 'Results', 'ref_spans': [{'start': 303, 'end': 309, 'mention': 'figure', 'ref_id': 'FIGREF0'}]}, {'text': 'Patient 4 had contact with patient 0 on Jan 20, 21, and 22, and reported chills on Jan 24. They had subsequent mild symptoms with slight malaise and slight nose and sinus congestion and were isolated on Jan 28. Patient 5 did not meet patient 0 but did meet patient 4 on Jan 22. Their only encounter was a canteen visit, sitting back to back, when patient 5 turned to patient 4 to borrow the salt shaker from their table. The encounter was 2 days before symptom onset in patient 4. Presymptomatic transmission from patient 4 to patient 5 is strongly supported by virus sequence analysis (table 2\\n): a nonsynonymous nucleotide polymorphism (a G6446A substitution) was found in the virus from patients 4 and 5 onwards but not in any cases detected before this point (patients 1–3). Later cases with available specimens, all containing this same substitution, were all traced back to patient 5. The possibility that patient 4 could have been infected by patient 5 was excluded by detailed sequence analysis:8 patient 4 had the novel G6446A virus detected in a throat swab and the original 6446G virus detected in her sputum, whereas patient 5 had a homogeneous virus population containing the novel G6446A substitution in the throat swab.', 'cite_spans': [{'start': 1003, 'end': 1004, 'mention': '8', 'ref_id': 'BIBREF11'}], 'section': 'Results', 'ref_spans': [{'start': 587, 'end': 594, 'mention': 'table 2', 'ref_id': 'TABREF1'}]}, {'text': 'The household of patient 5 consisted of five members, who were all hospitalised and kept together in one room in the hospital after patient 5 was confirmed positive. Three members—patients, 11, 6, and 9—developed symptoms and tested SARS-CoV-2 positive, whereas one member had no symptoms and did not test positive based on RT-PCR during follow-up. Patient 11 became symptomatic first; however, her first two tests were negative. Patient 9 became symptomatic last, with the virus showing an additional C22323T substitution. This same mutation was found as a minority virus population of approximately 4% reads also in the sputum of patient 11, but in none of the other household members. Because patient 11 had become symptomatic 4 days before patient 9, patient 11 infected patient 9.', 'cite_spans': [], 'section': 'Results', 'ref_spans': []}, {'text': 'Patient 7 met patient 5 for a 1·5-h meeting with a distance of approximately 1·5 m on Jan 24, the day of symptom onset in patient 5. Symptoms in patient 7 started 4 days later, on Jan 28, when patient 7 had a 1-h meeting with patient 10, in which patient 16 participated. Patients 8 and 5 had regular daily meetings at work, including on Jan 22, 23, and 24. Because the onset of symptoms for patient 5 was on Jan 24, transmission from patient 5 to patient 8 could therefore be presymptomatic or on the day of symptom onset. Patients 8 and 10 were identified as cases when a large number of employees, including both high-risk and low-risk contacts, were invited to be sampled at the company during the 3 days following the initial discovery of the cluster (Jan 29–31). Retrospectively, during detailed interviews, both patients recalled mild non-specific symptoms before testing, but no symptoms on the day of testing.', 'cite_spans': [], 'section': 'Results', 'ref_spans': []}, {'text': \"Patient 14 was a household member of patient 7 and both spent multiple days together after patient 7 was sent to home quarantine. Patient 15 is a household member of patient 2 and was asymptomatic. Patients 15 and 16 initially tested negative between Jan 29 and 31 at the company and tested positive when the test was repeated at the end of the follow-up period, thus 12–14 days after the last exposure to a case. Retrospectively, patient 16 had experienced a short period of very mild rhinorrhoea from Feb 4 and had not perceived this as a relevant symptom. Two possible transmission options could have occurred for patient 16, through patients 7 and 8, both on Jan 28. Patient 16 participated in the same meeting in which patient 10 was probably infected by patient 7. Patient 16 had also met patient 8 on the same day in a meeting, at a distance greater than 1 m; notably, Jan 28 was also the day of patient 8's symptom onset. Transmission history cannot be resolved because viral sequences in patients 7, 8, 10, and 16 are identical (table 2).\", 'cite_spans': [], 'section': 'Results', 'ref_spans': [{'start': 1038, 'end': 1045, 'mention': 'table 2', 'ref_id': 'TABREF1'}]}, {'text': 'Notification via IHR focal points resulted in identification of patient 12 by Spanish authorities. Extensive contact tracing involved the international flights from Munich to Shanghai (patient 0 on Jan 22, 2020) and from Munich to Tenerife (patient 12 on Jan 28, 2020). As of May 2, no further cases have been identified among flight passengers or other (personal) contacts.', 'cite_spans': [], 'section': 'Results', 'ref_spans': []}, {'text': \"All cases except for patient 15 were symptomatic, even if the symptoms were mostly mild (table 1). However, no transmission was documented from asymptomatic patient 15 to the patient's contacts until the end of the observation period (beginning of March, 2020). Presymptomatic transmission was the only possible explanation for the transmission from patient 4 to patient 5. It was also a possibility in the transmission from patient 5 to patient 8 and could not be ruled out for the household clusters (ie, transmission from patient 5 to patients 11 and 6 and transmission from patient 11 to patient 9) or transmission from patient 7 to patient 14. Patients 4 and 5 had symptom onset on the same day, which is only plausible if patient 4 shed virus 2 days before symptom onset and patient 5 had a 2-day incubation period. Four transmission events occurred certainly on the day of symptom onset (from patient 3 to patient 12, from patient 5 to patient 7, from patient 7 to patient 10, and from patient 7 or 8 to patient 16) and for one transmission event (patient 5 to patient 8), the only other possibility is presymptomatic transmission (table 1; figure). Five patients were possibly infected when they had contact with their source case at the day of the source case's symptom onset (patients 5 to 8, 5 to 11, 5 to 6, 11 to 9, and 7 to 14; table). Transmission during the prodromal phase of the illness occurred from patient 0 to patients 1, 2, and 4, as well as from patient 3 to patient 12. The incubation period ranged from 1 to 7 days. When using the most likely duration of the incubation periods or, in case of two equally likely durations, the mean duration, the median incubation period was 4·0 days (IQR 2·3–4·3). The median serial interval was also 4·0 days (IQR 3·0–5·0).\", 'cite_spans': [], 'section': 'Results', 'ref_spans': [{'start': 1148, 'end': 1154, 'mention': 'figure', 'ref_id': 'FIGREF0'}, {'start': 89, 'end': 96, 'mention': 'table 1', 'ref_id': 'TABREF0'}, {'start': 1139, 'end': 1146, 'mention': 'table 1', 'ref_id': 'TABREF0'}]}, {'text': 'After the infection of patient 0 was confirmed, and after identification of every new case, high-risk contacts were identified and put under home quarantine. By Feb 19, 2020, 241 high-risk contacts had been identified: four household contacts of patient 5 who were isolated together with the patient in a single hospital room, 20 household contacts of other cases, and 217 close, non-household contacts (table 3\\n). Of the family isolated in one room, three (75%) members subsequently became cases, resulting in a secondary attack rate of 75·0% (95% CI 19·0–99·0; table 3). Among 20 household contacts of other patients who lived with the patients until they were isolated in hospital, two further cases were identified, resulting in a secondary attack rate of 10·0% (1·2–32·0). Because these 20 household contacts had 80 contact days (mean 4 days), the secondary attack rate can also be expressed as one infection per 40 household contact days. Among 217 further high-risk contacts, 11 transmissions occurred, leading to a secondary attack rate of 5·1% (2·6–8·9). No cases occurred among the 108 identified low-risk contacts (table 3).', 'cite_spans': [], 'section': 'Results', 'ref_spans': [{'start': 404, 'end': 411, 'mention': 'table 3', 'ref_id': 'TABREF2'}, {'start': 563, 'end': 570, 'mention': 'table 3', 'ref_id': 'TABREF2'}, {'start': 1126, 'end': 1133, 'mention': 'table 3', 'ref_id': 'TABREF2'}]}, {'text': 'Over the four generations of transmission in this outbreak, the virus acquired two mutations, both of them non-silent (table 2). The G6446A exchange leads to a valin-to-isoleucine change in the betacoronavirus-specific marker domain in non-structural protein 3. No structure and function is known for this domain in SARS coronavirus.9 The C22323T exchange causes a serine-to-phenylalanine change in the spike protein S1 domain, which is outside of the receptor binding domain.10 The directly observed substitution rate was two substitutions per 29 903 nucleotides per 11 days, equalling 2·2  × 10−3 substitutions per site, per year.', 'cite_spans': [{'start': 333, 'end': 334, 'mention': '9', 'ref_id': 'BIBREF12'}, {'start': 476, 'end': 478, 'mention': '10', 'ref_id': 'BIBREF1'}], 'section': 'Results', 'ref_spans': [{'start': 119, 'end': 126, 'mention': 'table 2', 'ref_id': 'TABREF1'}]}, {'text': 'This case series of the Bavarian cluster constitutes the first documented chain of multiple human-to-human transmissions of SARS-CoV-2 outside of Asia. Due to the particular setting centred around a business company with encounters tractable through electronic calendars, the timing and setting of most contact events were well defined. The sensitising and close monitoring of people involved might have triggered sensitive reporting of prodromal symptoms. This might explain a rather short median incubation period of 4·0 days in our study, which is substantially shorter than the 5·2 days calculated from data of the first 425 confirmed cases in China11 but similar to the findings of Guan and colleagues.12 For comparison, the incubation period for SARS was 6·4 days, with a range of 1–14 days.13\\n', 'cite_spans': [{'start': 653, 'end': 655, 'mention': '11', 'ref_id': 'BIBREF2'}, {'start': 707, 'end': 709, 'mention': '12', 'ref_id': 'BIBREF3'}, {'start': 797, 'end': 799, 'mention': '13', 'ref_id': 'BIBREF4'}], 'section': 'Discussion', 'ref_spans': []}, {'text': 'The short incubation period combined with a median serial interval of the same length correspond to transmission early in the course of disease, or even before disease onset, which is corroborated by studies of viral shedding in the same patients.8 High-level replication with frequent virus isolation from the pharynx stands by contrast with the SARS coronavirus—genetically similar to SARS-CoV-2—that was shed at lower concentrations from the upper respiratory tract and was effectively transmitted roughly a week after symptom onset.11 This delayed shedding in SARS caused substantial problems with the sensitivity of RT-PCR based on upper respiratory tract swabs. In our study, two tests from patient 11 came back negative although the patient had already developed symptoms, suggesting that these results were false negative. Nevertheless, it is evident from our study that RT-PCR on throat swabs can discover asymptomatic or oligosymptomatic people who shed the virus.', 'cite_spans': [{'start': 247, 'end': 248, 'mention': '8', 'ref_id': 'BIBREF11'}, {'start': 536, 'end': 538, 'mention': '11', 'ref_id': 'BIBREF2'}], 'section': 'Discussion', 'ref_spans': []}, {'text': 'The present investigation enables us to determine secondary attack rates based on closely monitored high-risk contacts. The secondary attack rate thereby decreases with the intensity of contact: among members of the cohorted household, the secondary attack rate was 75%, but decreased to 10% among household contacts that were only together until isolation of the case. Although the experience with the cohorted family is based on a single observation, it shows the high risk of close household contacts—ie, sharing a room—of a case in home isolation and calls for strong measures within households to prevent transmission, particularly if susceptible people are present. Among 217 non-household high-risk contacts that were cumulatively followed up because of direct contact with a confirmed case, 11 were infected. The resulting secondary attack rate of 5% seems low, indicating little spread in this cluster. However, more effective spread of the virus might have been prevented by the proactive quarantine of high-risk contacts later identified as cases while still being presymptomatic or mildly symptomatic, as well as the proactive closure of the affected company. Transmission originating from people with more distinct respiratory symptoms might have resulted in a higher number of secondary cases.', 'cite_spans': [], 'section': 'Discussion', 'ref_spans': []}, {'text': 'Although two patients developed signs of pneumonia later in the course of disease, all others had a light course of disease and all patients recovered fully.8 However, it should be taken into account that the outbreak occurred in a population of working-age, generally healthy individuals. The overall clinical picture might have been different in a population including older individuals, or those with underlying chronic diseases. Another limitation of our study is that, naturally, not all infectious encounters could be reconstructed. Because the company held larger business and social events during the exposure period, it is possible that an infectious case might have met a successor case so briefly that neither of the two remembered the encounter. Some transmission events were nevertheless confirmed by viral sequencing and population analysis. The contact investigation and interviews followed a general hypothesis of direct human-to-human transmission, and we believe that the concurring epidemiological and genetic results in reconstructing the transmission network support this assumption.', 'cite_spans': [{'start': 157, 'end': 158, 'mention': '8', 'ref_id': 'BIBREF11'}], 'section': 'Discussion', 'ref_spans': []}, {'text': 'In conclusion, while COVID-19 cases presented partially with mild, non-specific symptoms in the Bavarian outbreak studied here, we found that infectiousness before symptom onset, on the day of symptom onset, and during mild prodromal symptoms was substantial and poses a huge challenge on the implementation of public health measures. Additionally, the incubation period was often very short and false negative tests can occur. Thus, although the outbreak was controlled and therefore might have granted valuable time before more intense transmission occurred in Germany, successful long-term and global containment of COVID-19 could be difficult to achieve.', 'cite_spans': [], 'section': 'Discussion', 'ref_spans': []}], 'paragraphs': [{'text': 'In conclusion, while COVID-19 cases presented partially with mild, non-specific symptoms in the Bavarian outbreak studied here, we found that infectiousness before symptom onset, on the day of symptom onset, and during mild prodromal symptoms was substantial and poses a huge challenge on the implementation of public health measures. Additionally, the incubation period was often very short and false negative tests can occur. Thus, although the outbreak was controlled and therefore might have granted valuable time before more intense transmission occurred in Germany, successful long-term and global containment of COVID-19 could be difficult to achieve.', 'id': '00031'}], 'title': 'Investigation of a COVID-19 outbreak in Germany resulting from a single travel-associated primary case: a case series', 'sha': '0f24818c6adcadeb15290cf2ca3ed4f499560b9d', 'doi': '10.1016/s1473-3099(20)30314-5', 'score': 8.605500221252441}, {'rank': 306, 'paragraphs': [{'text': 'The third author (Y.Z.) conducted a literature screening for the articles published between January 24, 2020 and March 31, 2020 by using online databases, including PubMed, Web of Science, Google Scholar and the official websites of core scientific and biomedical journals including Science, Nature, The Lancet, The New England Journal of Medicine, and The Journal of American Medical Association, as well as some preprint platforms such as BioRxiv and MedRxir, with search terms specified as COVID-19, SARS-CoV-2, 2019-nCov, and novel coronavirus. Forty-three articles were found with the theme on the basic reproduction number, the incubation period, the percentage of asymptomatic cases, and the case fatality rate. Among those articles, 20 articles, described in Table 1 , were identified by the first author (W.H.) to be included in the analysis, together with [24, 25] which were found on April 2. The inclusion criteria are the availability of both point estimates and 95% confidence intervals (or equivalently, standard deviations) for the basic transmission number, the average incubation time, the asymptomatic rate, or the case fatality rate. Table 1 presents the summary information of the selected articles together with the descriptions of the data used in those articles. We extract the results for the basic reproduction number from [3, 4, 5, 6, 7, 9, 10] and the results for the average incubation time from [3, 4, 8, 12, 15] . The results from [20, 21, 22, 23, 24, 25] are extracted for estimation of the asymptomatic infection rate. The estimates for the case fatality rate together with their 95% confidence intervals are taken from [10, 11, 13, 14, 16, 17, 18] . In the articles [4, 6, 8] , the reported 95% confidence intervals were asymmetric which we suspect were caused by employing a transformation (such as the exponential transformation) to the initial confidence intervals for the reparameterized effective size; for example, some authors may apply the logarithm to reparameterize the basic reproduction number or the average incubation time before performing the analysis. Using the inverse transformation, we convert the reported asymmetric confidence intervals and work out the associated standard deviations which are used in determining the weights for the meta-analysis.', 'id': '00007'}], 'title': 'Estimation of the basic reproduction number, average incubation time, asymptomatic infection rate, and case fatality rate forCOVID-19: Meta-analysis and sensitivity analysis', 'sha': '4bc132d5407496a57dabc2d8db6e46536f0a57f3', 'doi': '10.1101/2020.04.28.20083758', 'score': 8.597999572753906}, {'rank': 307, 'abstract': '', 'body_text': [{'text': 'Starting in December 2019, some hospitals in Wuhan identified several cases of unexplained pneumonia, which have since been confirmed as acute respiratory infections caused by SARS-CoV-2 infection. 1 Wuhan is an international metropolis with a population of >12 million. Due to the outbreak of COVID-19, the government took the extraordinary step of locking down Wuhan on 23 January 2020, to quickly contain the spread of the epidemic. As COVID-19 became a pandemic, many countries imposed quarantines and travel bans on an unprecedented scale. 2 Nevertheless, the number of cases and deaths continued to rise.', 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': 'Wuhan was the epicentre in China, with >50 000 confirmed cases. The mortality rate was dynamic because the data were constantly updated. Currently there are no specific antiviral drugs or vaccines available against COVID-19 in humans. Therefore, prevention is the key step to control this pandemic.', 'cite_spans': [], 'ref_spans': [], 'section': ''}, {'text': 'Cardiologists in Wuhan faced substantial challenges. During the epidemic, the hospitals in Wuhan had been divided into COVID-19designated hospitals (for the treatment of confirmed COVID-19 patients) and non-designated hospitals (for the treatment of non-COVID-19 patients in critical condition, such as haemodialysis, blood cancers, acute cardiovascular and cerebrovascular diseases, etc.).', 'cite_spans': [], 'ref_spans': [], 'section': 'Challenges facing cardiologists'}, {'text': \"First, there was a shortage of medical staff. A significant proportion of cardiologists in non-designated hospitals were allocated to assist other medical units, such as fever clinics, emergency rooms, isolation wards, designated hospitals, and 'Fangcang' Shelter hospitals. Unfortunately, some doctors were infected in the early stage, which exacerbated the problem of shortage of medical staff.\", 'cite_spans': [], 'ref_spans': [], 'section': 'Challenges facing cardiologists'}, {'text': 'Secondly, the medical resources were insufficient. Because COVID-19 is highly contagious and insidious, a large number of patients rushed to the fever clinic, resulting in a serious shortage of beds in the infectious disease department. Meanwhile, medical equipment was insufficient in the early stage, such as ventilators, extracorporeal membrane oxygenation (ECMO) systems, and personal protection equipment (PPE).', 'cite_spans': [], 'ref_spans': [], 'section': 'Challenges facing cardiologists'}, {'text': 'Thirdly, there were difficulties in treating cardiovascular emergencies. Cardiovascular"
          ],
          "name": "stdout"
        },
        {
          "output_type": "stream",
          "text": [
            "IOPub data rate exceeded.\n",
            "The notebook server will temporarily stop sending output\n",
            "to the client in order to avoid crashing it.\n",
            "To change this limit, set the config variable\n",
            "`--NotebookApp.iopub_data_rate_limit`.\n",
            "\n",
            "Current values:\n",
            "NotebookApp.iopub_data_rate_limit=1000000.0 (bytes/sec)\n",
            "NotebookApp.rate_limit_window=3.0 (secs)\n",
            "\n"
          ],
          "name": "stderr"
        }
      ]
    }
  ]
}